<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

  <style>
  .panel-body {
    -ms-word-break: break-all;
    word-break: break-all;

    word-break: break-word;

    -webkit-hyphens: auto;
    -moz-hyphens: auto;
    hyphens: auto;
  }
  </style>

</head>
<body>

<div class="container">
  <center>
    <h2>Data</h2>
  </center>
  <div class="panel-group" id="accordion">
    <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" data-parent="#accordion" href="#collapse1">Chronic myeloid leukemia (CML)</a>
        </h4>
      </div>
      <div id="collapse1" class="panel-collapse collapse">
        <div class="panel-body">
          <p>
            15	83215190	.	C	G	63.20	highCVG	AC1=2;AF1=0.2897;DP=17;DP4=3,4,0,8;FQ=62.9;MQ=60;PV4=0.077,1,1,1;VDB=0.0006;GM=NM_001079533.1,NM_001079534.1,NM_001079535.1,NM_030594.3,XM_005272441.1,XM_005272442.1,XM_005272443.1,XM_005272444.1,XM_005272445.1,XM_005272446.1;GL=CPEB1;FG=splice-donor;FD=unknown;CP=1.000;CG=4.850;AA=C;DSP=0;KP=hsa04114:OocyteMeiosis|hsa04320:Dorso-ventralAxisFormation|hsa04914:Progesterone-mediatedOocyteMaturation;TFBS=V$AREB6_02,V$SREBP1_01;GESP=C:12084;PPI=UBC:943|SPI1:845|GRIN3A:830|MRTO4:830;PAC=NP_001073001.1,NP_001073002.1,NP_001073003.1,NP_085097.3,XP_005272498.1,XP_005272499.1,XP_005272500.1,XP_005272501.1,XP_005272502.1,XP_005272503.1	GT:GQ:PL	0/0:8:0,6,37	0/0:6:0,3,27	1/1:6:106,12,0	0/0:11:0,9,45
          </p>
          <p>
            1	17657616	rs11203367	T	C	17.40	lowCVG	AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:19,3,0	0/1:37:33,0,50
          </p>
          <p>
            1	17660468	rs34309058	C	A	35.70	midCVG	AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/0:16:0,15,97	0/0:3:0,0,0	1/1:6:78,12,0	0/0:13:0,12,80
          </p>
          <p>
            3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0
          </p>
          <p>
            6	43011188	.	G	T	66.80	highCVG	AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1	GT:GQ:PL	0/0:7:0,6,35	0/0:5:0,3,19	1/1:23:112,30,0	0/0:16:0,15,97
          </p>
          <p>
            7	77789554	.	G	A	51.40	highCVG	AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1	GT:GQ:PL	0/0:13:0,12,79	0/0:13:0,12,69	1/1:17:97,24,0	0/0:5:0,3,25
          </p>
          <p>
            7	154738409	rs3501	T	C	124	highCVG	AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1	GT:GQ:PL	0/0:8:0,9,63	0/0:6:0,6,48	1/1:20:149,24,0	0/1:20:22,0,18
          </p>
          <p>
            8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75
          </p>
          <p>
            11	69466030	.	G	A	20.20	lowCVG	AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1	GT:GQ:PL	0/0:6:0,3,17	0/0:6:0,3,14	1/1:4:61,9,0	0/0:11:0,9,66
          </p>
          <p>
            19	55143083	rs1061679	T	C	34	lowCVG	AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA	GT:GQ:PL	0/0:12:0,15,73	0/1:3:0,0,0	1/1:5:50,6,0	0/1:3:26,3,0
          </p>
          <p>
            <strong>IDs and variants associated:</strong> H00004;CDKN2A|ABL1|RB1|BCR|TP53|WT1|RUNX1;hsa05220
          </p>
          <p>
            <strong>Nonsense Mutations:</strong>  0
          </p>
          <p>
            <strong>Missense Mutations:</strong>  9
          </p>
          <p>
            <strong>Disease Rank Score:</strong>  0.9
          </p>

        </div>
      </div>
    </div>
    <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" data-parent="#accordion" href="#collapse2">Small cell lung cancer</a>
        </h4>
      </div>
      <div id="collapse2" class="panel-collapse collapse">
        <div class="panel-body">
          <p>
            7	77789554	.	G	A	51.40	highCVG	AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1	GT:GQ:PL	0/0:13:0,12,79	0/0:13:0,12,69	1/1:17:97,24,0	0/0:5:0,3,25
          </p>
          <p>
            8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75
          </p>
          <p>
            11	69466030	.	G	A	20.20	lowCVG	AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1	GT:GQ:PL	0/0:6:0,3,17	0/0:6:0,3,14	1/1:4:61,9,0	0/0:11:0,9,66
          </p>
          <p>
            17	48598785	rs1380657	G	A	174	highCVG	AC1=6;AF1=0.7336;DB;DP=19;DP4=3,2,5,7;FQ=44.9;MQ=60;PV4=0.62,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=A424T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYCBPAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032133.4;VDB=0.0177;DN=88;DA=G/A;GM=NM_032133.4,XM_005257725.1,XM_005257726.1,XM_005257727.1;GL=MYCBPAP;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=424/985,439/1001,439/1000,424/986;CDP=1270,1315,1315,1270;GS=58;PH=0;CP=0.000;CG=-4.290;AA=A;HA=7.6;HE=27.6;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=A:10167/G:2839;PPI=MYCBP:846;PAC=NP_115509.4,XP_005257782.1,XP_005257783.1,XP_005257784.1	GT:GQ:PL	0/1:14:16,0,16	0/1:38:31,0,68	1/1:22:130,21,0	1/1:8:42,6,0
          </p>
          <p>
            19	55143083	rs1061679	T	C	34	lowCVG	AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA	GT:GQ:PL	0/0:12:0,15,73	0/1:3:0,0,0	1/1:5:50,6,0	0/1:3:26,3,0
          </p>
          <p>
            22	50728062	rs28379706	T	C	62.50	midCVG	AC1=4;AF1=0.5564;DB;DP=11;DP4=1,2,6,0;FQ=26.1;MQ=60;PV4=0.083,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K318E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012401.3;VDB=0.0214;DN=125;DA=T/C;GM=NM_012401.3,XM_005261908.1,XM_005261909.1,XM_005261910.1,XM_005261911.1;GL=PLXNB2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=318/1839,400/1921,318/1839,318/1839,318/1839;CDP=952,1198,952,952,952;GS=56;PH=0;CP=0.001;CG=-1.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=117;KP=hsa04360:AxonGuidance;CPG=50727442-50728654;GESP=C:5524/T:6698;PPI=TUBA3E:962|MYC:846|ARHGEF11:845;PAC=NP_036533.2,XP_005261965.1,XP_005261966.1,XP_005261967.1,XP_005261968.1	GT:GQ:PL	0/1:3:0,3,24	0/0:5:0,6,44	1/1:3:21,3,0	1/1:13:87,15,0
          </p>
          <p>
            <strong>IDs and variants associated:</strong> H00013;ENO2|FHIT|GRP|MYC|PTEN|RARB|RB1|BCL2|TP53;hsa05222
          </p>
          <p>
            <strong>Nonsense Mutations:</strong> 0
          </p>
          <p>
            <strong>Missense Mutations:</strong> 6
          </p>
          <p>
            <strong>Disease Rank Score:</strong> 0.06
          </p>
        </div>
      </div>
    </div>
    <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a data-toggle="collapse" data-parent="#accordion" href="#collapse3">Non-small cell lung cancer</a>
        </h4>
      </div>
      <div id="collapse3" class="panel-collapse collapse">
        <div class="panel-body">
          <p>
            4	25678199	rs6448389	A	G	52.40	midCVG	AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1	GT:GQ:PL	0/1:3:0,0,0	0/0:6:0,9,58	1/1:4:18,3,0	1/1:11:77,12,0
          </p>
          <p>
            19	6374890	rs7540	G	A	55.70	midCVG	AC1=4;AF1=0.4563;DB;DP=10;DP4=5,0,3,1;FQ=47.1;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R191Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALKBH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032306.3;VDB=0.0086;DN=52;DA=G/A;GM=NM_032306.3,XM_005259658.1;GL=ALKBH7;FG=missense;FD=missense,unknown;AAC=ARG/GLN;PP=191/222,165/196;CDP=572,494;GS=43;PH=0.275;CP=0.021;CG=3.030;AA=G;HA=0.0;HE=10.3;HC=2.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=69;TFBS=V$OLF1_01;GESP=A:862/G:12144;PAC=NP_115682.1,XP_005259715.1	GT:GQ:PL	0/1:18:43,0,14	0/1:3:0,0,0	1/1:4:54,6,0	0/0:11:0,12,72
          </p>
          <p>
            <strong>IDs and variants associated:</strong> H00014;CDKN2A|TFG|CEACAM5|SLC34A2|RASSF1|FHIT|ALK|EML4|KRAS|KRT19|RARB|ROS1|TP53;hsa05223
          </p>
          <p>
            <strong>Nonsense Mutations:</strong> 0
          </p>
          <p>
            <strong>Missense Mutations:</strong> 2
          </p>
          <p>
            <strong>Disease Rank Score:</strong> 0.02
          </p>
        </div>
      </div>
    </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse4">Pancreatic cancer</a>
      </h4>
    </div>
    <div id="collapse4" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	17657616	rs11203367	T	C	17.40	lowCVG	AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:19,3,0	0/1:37:33,0,50

        </p>
        <p>
          1	17660468	rs34309058	C	A	35.70	midCVG	AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/0:16:0,15,97	0/0:3:0,0,0	1/1:6:78,12,0	0/0:13:0,12,80

        </p>
        <p>
          3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0

        </p>
        <p>
          6	43011188	.	G	T	66.80	highCVG	AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1	GT:GQ:PL	0/0:7:0,6,35	0/0:5:0,3,19	1/1:23:112,30,0	0/0:16:0,15,97

        </p>
        <p>
          7	154738409	rs3501	T	C	124	highCVG	AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1	GT:GQ:PL	0/0:8:0,9,63	0/0:6:0,6,48	1/1:20:149,24,0	0/1:20:22,0,18

        </p>
        <p>
          8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00019;CDKN2A|ERBB2|KRAS|SMAD4|SPP1|BRCA2|STK11|TP53|GDF15;hsa05212
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 6
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.06
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse5">Colorectal cancer</a>
      </h4>
    </div>
    <div id="collapse5" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	242035438	rs4149965	G	A	6.86	lowCVG	AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1	GT:GQ:PL	0/0:3:0,0,0	0/0:3:0,0,0	1/1:4:44,9,0	0/0:16:0,15,101

        </p>
        <p>
          3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0

        </p>
        <p>
          12	49443799	rs112236653	G	A	49.50	midCVG	AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1	GT:GQ:PL	0/1:14:11,0,64	0/1:3:0,0,0	1/1:9:81,12,0	0/0:7:0,9,52

        </p>
        <p>
          15	65676726	rs33918653	C	T	78.50	midCVG	AC1=4;AF1=0.5302;DB;DP=10;DP4=0,4,5,1;FQ=24.8;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C1125Y;SNPEFF_CODON_CHANGE=tGc/tAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0003;DN=126;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=CYS/TYR;PP=1125/1251;CDP=3374;GS=194;PH=0.981;CP=1.000;CG=4.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=T:2295/C:10705;PAC=NP_066013.1	GT:GQ:PL	0/0:5:0,6,38	0/1:3:0,0,0	1/1:7:70,9,0	0/1:28:50,0,25

        </p>
        <p>
          15	65684513	rs148990036	C	T	29.40	lowCVG	AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,3,0;FQ=21.1;MQ=59;PV4=0.4,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R694Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0000;DN=134;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=ARG/GLN;PP=694/1251;CDP=2081;GS=43;PH=0.968;CP=0.939;CG=5.490;AA=C;DV=by-frequency,by-cluster;DSP=42;GESP=T:88/C:12878;PAC=NP_066013.1	GT:GQ:PL	0/0:9:0,9,56	0/1:3:0,0,0	1/1:6:67,9,0	0/1:3:0,0,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00020;APC2|CTNNB1|DCC|MSH6|HBA1|HBA2|HBB|KRAS|SMAD2|SMAD4|MLH1|MSH2|MSH3|BAX|TGFBR2|TP53;hsa05210
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 5
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.05
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse6">Renal cell carcinoma</a>
      </h4>
    </div>
    <div id="collapse6" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	17085502	rs2329006	T	C	10.30	highCVG	AC1=3;AF1=0.3679;DB;DP=20;DP4=4,5,7,3;FQ=11.7;MQ=16;PV4=0.37,7e-06,0.3,0.15;SNPEFF_AMINO_ACID_CHANGE=M397V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MST1L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271733.1;VDB=0.0107;DN=100;DA=T/C;GM=NM_001271733.1;GL=MST1L;FG=missense;FD=missense;AAC=MET/VAL;PP=397/690;CDP=1189;GS=21;PH=0.235;CP=0.000;AA=C;DG;DV=by-cluster,by-hapmap;DSP=29;CPG=17085412-17086974;PAC=NP_001258662.1	GT:GQ:PL	0/1:7:7,0,12	0/0:5:0,3,18	1/1:3:45,7,0	0/0:11:0,10,33

        </p>
        <p>
          1	17657616	rs11203367	T	C	17.40	lowCVG	AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:19,3,0	0/1:37:33,0,50

        </p>
        <p>
          1	17660468	rs34309058	C	A	35.70	midCVG	AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/0:16:0,15,97	0/0:3:0,0,0	1/1:6:78,12,0	0/0:13:0,12,80

        </p>
        <p>
          1	33161212	rs360041	T	C	100	highCVG	AC1=4;AF1=0.5158;DB;DP=34;DP4=5,0,28,0;FQ=52.7;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=M163V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030786.2;VDB=0.0000;DN=79;DA=T/C;GM=NM_001161708.1,NM_030786.2;GL=SYNC;FG=missense;FD=unknown;AAC=MET/VAL;PP=163/477,163/483;CDP=487;GS=21;PH=0;CP=0.005;CG=-2.750;AA=T;HA=18.1;HE=4.0;HC=13.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=434;GESP=C:4162/T:404;PAC=NP_001155180.1,NP_110413.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:80:141,84,0	0/0:13:0,15,90

        </p>
        <p>
          1	242035438	rs4149965	G	A	6.86	lowCVG	AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1	GT:GQ:PL	0/0:3:0,0,0	0/0:3:0,0,0	1/1:4:44,9,0	0/0:16:0,15,101

        </p>
        <p>
          1	248039451	rs3811444	C	T	107	highCVG	AC1=4;AF1=0.5108;DB;DP=24;DP4=8,4,4,7;FQ=110;MQ=60;PV4=0.22,0.14,1,0.15;SNPEFF_AMINO_ACID_CHANGE=T374M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM58;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015431.3;VDB=0.0144;DN=107;DA=C/T;GM=NM_015431.3,XM_005273091.1,XM_005273092.1;GL=TRIM58;FG=missense;FD=missense,unknown,unknown;AAC=THR/MET;PP=374/487,230/343,230/343;CDP=1121,689,689;GS=81;PH=0.999;CP=0.011;CG=2.210;AA=C;HA=3.3;HE=33.2;HC=23.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=250;GESP=T:3365/C:9641;PAC=NP_056246.3,XP_005273148.1,XP_005273149.1	GT:GQ:PL	0/0:12:0,15,107	0/1:13:19,0,11	1/1:21:112,24,0	0/1:27:24,0,91

        </p>
        <p>
          1	248685719	rs9330305	A	T	35.40	highCVG	AC1=3;AF1=0.3917;DB;DP=16;DP4=3,9,3,1;FQ=37.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M258L;SNPEFF_CODON_CHANGE=Atg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2G6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001013355.1;VDB=0.0168;DN=119;DA=A/T;GM=NM_001013355.1;GL=OR2G6;FG=missense;FD=missense;AAC=MET/LEU;PP=258/317;CDP=772;GS=15;PH=0.934;CP=0.966;CG=3.830;AA=A;HA=33.6;HE=12.1;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=159907;KP=hsa04740:OlfactoryTransduction;GESP=T:2221/A:10785;PAC=NP_001013373.1	GT:GQ:PL	0/0:15:0,15,89	0/1:4:17,3,0	1/1:5:66,9,0	0/0:20:0,21,135

        </p>
        <p>
          2	1320143	rs4927635	C	T	220	highCVG	AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1	GT:GQ:PL	0/1:6:0,3,25	0/1:8:20,0,8	1/1:19:119,18,0	1/1:22:127,21,0

        </p>
        <p>
          2	96993757	rs2279105	C	T	69.10	lowCVG	AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3	GT:GQ:PL	0/1:16:12,0,21	0/1:4:0,3,17	1/1:4:21,3,0	1/1:12:80,12,0

        </p>
        <p>
          2	152527572	rs7426114	C	T	226	highCVG	AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1	GT:GQ:PL	0/1:13:55,0,13	0/1:3:0,6,35	1/1:15:90,15,0	1/1:21:129,21,0

        </p>
        <p>
          2	186656956	rs17229201	T	C	27.70	lowCVG	AC1=5;AF1=0.6553;DB;DP=5;DP4=2,0,2,1;FQ=12.6;MQ=60;PV4=1,0.22,1,0.25;SNPEFF_AMINO_ACID_CHANGE=M1787T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FSIP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173651.2;VDB=0.0061;DN=123;DA=T/C;GM=NM_173651.2;GL=FSIP2;FG=missense;FD=missense;AAC=MET/THR;PP=1787/6997;CDP=5360;GS=81;CP=0.371;CG=3.390;AA=C;HA=42.4;HE=48.8;HC=46.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3587;PAC=NP_775922.2	GT:GQ:PL	0/1:3:0,0,0	0/0:4:0,6,48	1/1:4:21,3,0	1/1:5:47,6,0

        </p>
        <p>
          2	207820276	rs13420911	G	A	43.40	highCVG	AC1=2;AF1=0.2979;DB;DP=18;DP4=6,8,2,2;FQ=44.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M85I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173077.2;VDB=0.0134;DN=121;DA=G/A;GM=NM_173077.2;GL=CPO;FG=missense;FD=missense;AAC=MET/ILE;PP=85/375;CDP=255;GS=10;PH=0.021;CP=0.999;CG=5.110;AA=G;HA=33.2;HE=44.8;HC=39.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=13;GESP=A:5799/G:7207;PAC=NP_775100.1	GT:GQ:PL	0/0:20:0,18,108	0/0:3:0,0,0	1/1:6:86,12,0	0/0:25:0,24,153

        </p>
        <p>
          3	13361287	rs354478	C	T	70.90	midCVG	AC1=5;AF1=0.5672;DB;DP=11;DP4=4,1,4,2;FQ=39.6;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=V1787M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=37;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUP210;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024923.3;VDB=0.0032;DN=79;DA=C/T;GM=NM_024923.3;GL=NUP210;FG=missense;FD=missense;AAC=VAL/MET;PP=1787/1888;CDP=5359;GS=21;PH=0;CP=0.698;CG=-1.130;AA=T;HA=43.7;HE=48.6;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=25;GESP=T:6670/C:6336;PAC=NP_079199.2	GT:GQ:PL	0/1:26:49,0,24	0/0:6:0,9,57	1/1:5:44,6,0	1/1:3:23,3,0

        </p>
        <p>
          3	57136585	rs17057718	C	T	51.40	midCVG	AC1=4;AF1=0.493;DB;DP=12;DP4=6,0,6,0;FQ=54.2;MQ=60;PV4=1,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=V301M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=123;DA=C/T;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=VAL/MET;PP=301/740,277/716,273/712,157/596;CDP=901,829,817,469;GS=21;PH=0.777;CP=0.215;CG=3.570;AA=C;HA=4.1;HE=17.7;HC=38.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;TFBS=V$GFI1_01;GESP=T:1623/C:11383;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1	GT:GQ:PL	0/1:8:5,0,26	0/0:7:0,9,66	1/1:9:72,12,0	0/1:20:20,0,19

        </p>
        <p>
          3	66419956	rs13874	C	T	145	highCVG	AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA	GT:GQ:PL	0/0:7:0,9,70	0/1:3:0,0,0	1/1:18:112,21,0	0/1:31:77,0,28

        </p>
        <p>
          3	98600385	rs9838238	T	C	36.30	midCVG	AC1=3;AF1=0.3969;DB;DP=9;DP4=1,3,2,2;FQ=36.7;MQ=60;PV4=1,0.28,1,0.067;SNPEFF_AMINO_ACID_CHANGE=I144M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCBLD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080927.3;VDB=0.0156;DN=119;DA=T/C;GM=NM_080927.3,XM_005247118.1;GL=DCBLD2;FG=missense-near-splice;FD=missense,unknown;AAC=ILE/MET;PP=144/776,144/790;CDP=432;GS=10;PH=1;CP=1.000;CG=4.900;AA=T;HA=1.2;HE=2.3;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=C:437/T:11499;PAC=NP_563615.3,XP_005247175.1	GT:GQ:PL	0/1:3:0,0,0	0/0:5:0,3,22	1/1:7:77,12,0	0/0:9:0,9,58

        </p>
        <p>
          3	111901099	rs74840030	T	C	15.40	midCVG	AC1=3;AF1=0.3387;DB;DP=11;DP4=4,3,2,2;FQ=17;MQ=60;PV4=1,0.00097,1,0.41;SNPEFF_AMINO_ACID_CHANGE=M844V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC9C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_183061.1;VDB=0.0154;DN=132;DA=T/C;GM=NM_183061.1,XM_005247381.1;GL=SLC9C1;FG=missense;FD=missense,unknown;AAC=MET/VAL;PP=844/1178,796/1130;CDP=2530,2386;GS=21;PH=0.39;CP=1.000;CG=4.590;AA=T;DV=by-frequency,by-cluster;DSP=7;TFBS=V$HNF1_01;GESP=C:495/T:12501;PAC=NP_898884.1,XP_005247438.1	GT:GQ:PL	0/0:10:0,9,74	0/1:5:4,0,40	1/1:4:54,9,0	0/0:8:0,6,51

        </p>
        <p>
          3	140406947	rs28594654	G	A	110	highCVG	AC1=4;AF1=0.5408;DB;DP=15;DP4=3,2,9,1;FQ=41;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V475M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM42;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152616.4;VDB=0.0045;DN=125;DA=G/A;GM=NM_152616.4,XM_005247397.1,XM_005247398.1;GL=TRIM42;FG=missense;FD=missense,unknown,unknown;AAC=VAL/MET;PP=475/724,475/600,328/453;CDP=1423,1423,982;GS=21;PH=0;CP=0.926;CG=1.790;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=66;GESP=A:6351/G:6655;PAC=NP_689829.3,XP_005247454.1,XP_005247455.1	GT:GQ:PL	0/1:54:96,0,51	0/1:3:0,0,0	1/1:5:53,6,0	0/0:3:0,3,26

        </p>
        <p>
          3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0

        </p>
        <p>
          3	193209178	rs6788448	T	C	32.90	lowCVG	AC1=3;AF1=0.3939;DB;DP=6;DP4=2,1,1,2;FQ=30.9;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=I181M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP13A4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032279.2;VDB=0.0095;DN=116;DA=T/C;GM=NM_032279.2,XM_005247829.1;GL=ATP13A4;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=181/1197,181/1128;CDP=543;GS=10;PH=0.999;CP=1.000;CG=4.630;AA=T;HA=48.5;HE=41.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=C:5746/T:7260;PAC=NP_115655.2,XP_005247886.1	GT:GQ:PL	0/0:5:0,3,23	0/0:3:0,0,0	1/1:5:72,9,0	0/0:7:0,6,47

        </p>
        <p>
          4	129793281	.	A	G	98.60	highCVG	AC1=4;AF1=0.5296;DP=39;DP4=1,6,3,23;FQ=30.8;MQ=60;PV4=1,0.39,1,0.19;SNPEFF_AMINO_ACID_CHANGE=N798S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=JADE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287439.1;VDB=0.0093;GM=NM_199320.2,XM_005263229.1,XM_005263230.1,XM_005263231.1,XM_005263232.1,XM_005263233.1;GL=PHF17;FG=missense;FD=unknown;AAC=ASN/SER;PP=798/843,825/870,798/843,798/843,798/843,786/831;CDP=2393,2474,2393,2393,2393,2357;GS=46;PH=0;CP=0.014;CG=-6.450;AA=A;DSP=772;GESP=A:13006;PPI=KAT7:965|ING5:913|VHL:845|MEAF6:845|ING4:788;PAC=NP_955352.1,XP_005263286.1,XP_005263287.1,XP_005263288.1,XP_005263289.1,XP_005263290.1	GT:GQ:PL	0/0:3:0,3,26	0/1:3:23,3,0	1/1:7:120,9,0	0/0:5:0,6,46

        </p>
        <p>
          4	169928842	rs2877380	G	T	108	midCVG	AC1=5;AF1=0.6024;DB;DP=14;DP4=2,2,6,2;FQ=49.6;MQ=60;PV4=0.55,0.03,1,1;SNPEFF_AMINO_ACID_CHANGE=L70M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CBR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032783.4;VDB=0.0052;DN=101;DA=G/T;GM=NM_032783.4,XM_005263315.1,XM_005263316.1,XM_005263317.1;GL=CBR4;FG=missense;FD=missense,unknown,unknown,unknown;AAC=LEU/MET;PP=70/238,70/209,70/194,70/146;CDP=208;GS=15;PH=0.005;CP=1.000;CG=5.560;AA=T;HA=38.2;HE=21.8;HC=3.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=56;GESP=T:8677/G:4325;PAC=NP_116172.2,XP_005263372.1,XP_005263373.1,XP_005263374.1	GT:GQ:PL	0/1:18:51,0,16	0/0:6:0,9,76	1/1:11:74,12,0	1/1:4:29,3,0

        </p>
        <p>
          5	78181477	rs1065757	C	T	31.80	midCVG	AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1	GT:GQ:PL	0/0:18:0,21,102	0/1:3:0,0,0	1/1:7:56,9,0	1/1:3:19,3,0

        </p>
        <p>
          5	151771838	rs4958531	T	C	25.70	midCVG	AC1=3;AF1=0.3673;DB;DP=11;DP4=2,6,1,2;FQ=27.7;MQ=60;PV4=1,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=M388V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NMUR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020167.4;VDB=0.0057;DN=111;DA=T/C;GM=NM_020167.4;GL=NMUR2;FG=missense;FD=missense;AAC=MET/VAL;PP=388/416;CDP=1162;GS=21;PH=0;CP=0.000;CG=-7.170;AA=C;HA=0.0;HE=18.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=225;KP=hsa04080:NeuroactiveLigand-receptorInteraction;TFBS=V$TCF11_01;GESP=C:1791/T:11215;PPI=NMU:974;PAC=NP_064552.3	GT:GQ:PL	0/1:14:18,0,10	0/0:5:0,3,23	1/1:3:51,6,0	0/0:18:0,18,106

        </p>
        <p>
          5	180030322	rs79620092	C	T	69.90	lowCVG	AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2	GT:GQ:PL	0/1:15:20,0,13	0/1:3:0,0,0	1/1:13:91,15,0	0/1:3:0,3,26

        </p>
        <p>
          6	43011188	.	G	T	66.80	highCVG	AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1	GT:GQ:PL	0/0:7:0,6,35	0/0:5:0,3,19	1/1:23:112,30,0	0/0:16:0,15,97

        </p>
        <p>
          6	97063522	rs9373985	C	G	49.80	highCVG	AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1	GT:GQ:PL	0/0:9:0,9,63	0/1:27:26,0,61	1/1:13:72,18,0	0/0:35:0,36,176

        </p>
        <p>
          6	152443744	rs2295190	G	T	29	lowCVG	AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:34,6,0	0/1:24:31,0,23

        </p>
        <p>
          7	111580166	rs144867634	T	C	90.70	highCVG	AC1=3;AF1=0.3681;DB;DP=23;DP4=12,3,3,4;FQ=92.7;MQ=60;PV4=0.14,0.45,1,0.35;SNPEFF_AMINO_ACID_CHANGE=M326V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DOCK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014705.3;VDB=0.0036;DN=134;DA=T/C;GM=NM_014705.3,XM_005250724.1;GL=DOCK4;FG=missense-near-splice;FD=missense,unknown;AAC=MET/VAL;PP=326/1967,326/2012;CDP=976;GS=21;PH=0.066;CP=1.000;CG=6.060;AA=T;DV=by-frequency,by-cluster;DSP=2;GESP=C:225/T:12057;PAC=NP_055520.3,XP_005250781.1	GT:GQ:PL	0/0:26:0,27,130	0/1:24:48,0,18	1/1:10:91,15,0	0/0:15:0,15,111

        </p>
        <p>
          7	154738409	rs3501	T	C	124	highCVG	AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1	GT:GQ:PL	0/0:8:0,9,63	0/0:6:0,6,48	1/1:20:149,24,0	0/1:20:22,0,18

        </p>
        <p>
          8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75

        </p>
        <p>
          9	113208250	rs7863519	T	G	81.60	midCVG	AC1=4;AF1=0.5301;DB;DP=11;DP4=3,1,3,4;FQ=42.9;MQ=60;PV4=0.55,0.4,1,1;SNPEFF_AMINO_ACID_CHANGE=M1444L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SVEP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153366.3;VDB=0.0187;DN=116;DA=T/G;GM=NM_153366.3;GL=SVEP1;FG=missense;FD=missense;AAC=MET/LEU;PP=1444/3572;CDP=4330;GS=15;PH=0.002;CP=0.023;CG=3.620;AA=C;HA=44.7;HE=20.1;HC=37.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=69;TFBS=V$YY1_01;GESP=G:3078/T:9136;PAC=NP_699197.3	GT:GQ:PL	0/0:3:0,3,25	0/1:3:16,3,0	1/1:15:112,18,0	0/0:7:0,9,61

        </p>
        <p>
          9	135786904	rs1073123	A	G	103	highCVG	AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1	GT:GQ:PL	0/0:4:0,3,23	0/1:84:83,0,87	1/1:6:64,9,0	0/0:6:0,6,39

        </p>
        <p>
          10	129903016	rs7071768	T	C	19.60	midCVG	AC1=5;AF1=0.5991;DB;DP=9;DP4=2,1,1,2;FQ=16.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N2363S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MKI67;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002417.4;VDB=0.0101;DN=116;DA=T/C;GM=NM_001145966.1,NM_002417.4,XM_005252683.1;GL=MKI67;FG=missense;FD=missense,missense,unknown;AAC=ASN/SER;PP=2003/2897,2363/3257,2166/3060;CDP=6008,7088,6497;GS=46;PH=0;CP=0.000;CG=-5.360;AA=C;HA=41.1;HE=46.0;HC=38.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2174;GESP=C:6499/T:6507;PPI=MKI67IP:983|SUMO2:748;PAC=NP_001139438.1,NP_002408.3,XP_005252740.1	GT:GQ:PL	0/1:18:19,0,17	0/1:18:14,0,35	1/1:3:25,3,0	0/1:3:0,0,0

        </p>
        <p>
          11	6816137	.	T	A	48.90	midCVG	AC1=5;AF1=0.5835;DP=13;DP4=1,1,5,0;FQ=-7.18;MQ=60;PV4=0.29,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=K268M;SNPEFF_CODON_CHANGE=aAg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003696.2;VDB=0.0075;GM=NM_003696.2;GL=OR6A2;FG=missense;FD=unknown;AAC=LYS/MET;PP=268/328;CDP=803;GS=95;PH=0.999;CP=0.995;CG=3.850;AA=T;DSP=111784;KP=hsa04740:OlfactoryTransduction;GESP=T:12994;PAC=NP_003687.2	GT:GQ:PL	0/1:20:34,0,17	0/1:3:0,0,1/1:7:55,9,0	0/1:3:0,3,13

        </p>
        <p>
          11	17632651	rs7111528	C	T	102	midCVG	AC1=5;AF1=0.5888;DB;DP=10;DP4=1,2,4,3;FQ=15.1;MQ=60;PV4=1,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=T1947M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0076;DN=116;DA=C/T;GM=NM_001277269.1,XM_005252894.1;GL=OTOG;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=1947/2926,953/1509;CDP=5840,2858;GS=81;PH=0.011;CP=0.000;CG=-0.815;AA=T;HA=48.7;HE=42.3;HC=17.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=320;PAC=NP_001264198.1,XP_005252951.1	GT:GQ:PL	0/0:4:0,6,33	0/1:3:0,3,25	1/1:5:50,6,0	1/1:13:99,15,0

        </p>
        <p>
          11	18735947	rs7125943	T	C	65	lowCVG	AC1=4;AF1=0.4853;DB;DP=7;DP4=1,1,3,2;FQ=17.9;MQ=60;PV4=1,0.094,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M559V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGSF22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173588.3;VDB=0.0024;DN=116;DA=T/C;GM=NM_173588.3;GL=IGSF22;FG=missense;FD=missense;AAC=MET/VAL;PP=559/1327;CDP=1675;GS=21;PH=0;CP=0.001;CG=-2.120;AA=C;HA=15.3;HE=22.1;HC=20.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=19;GESP=C:9686/T:2872;PAC=NP_775859.3	GT:GQ:PL	0/1:3:0,0,0	0/0:4:0,3,25	1/1:11:105,15,0	0/0:4:0,3,30

        </p>
        <p>
          11	63233721	rs149917445	G	A	18	midCVG	AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1	GT:GQ:PL	0/0:18:0,15,108	0/0:9:0,6,46	1/1:3:60,9,0	0/0:9:0,6,41

        </p>
        <p>
          11	69466030	.	G	A	20.20	lowCVG	AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1	GT:GQ:PL	0/0:6:0,3,17	0/0:6:0,3,14	1/1:4:61,9,0	0/0:11:0,9,66

        </p>
        <p>
          11	77909014	rs2510044	G	A	62.90	highCVG	AC1=4;AF1=0.528;DB;DP=28;DP4=0,20,3,5;FQ=65.9;MQ=60;PV4=0.017,1,1,0.089;SNPEFF_AMINO_ACID_CHANGE=V236M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP35;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020798.2;VDB=0.0191;DN=100;DA=G/A;GM=NM_020798.2;GL=USP35;FG=missense;FD=missense;AAC=VAL/MET;PP=236/1019;CDP=706;GS=21;PH=0.999;CP=0.992;CG=4.820;AA=G;HA=0.4;HE=10.7;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=33;TFBS=V$NRSF_01;GESP=A:1344/G:11488;PAC=NP_065849.1	GT:GQ:PL	0/1:20:20,0,19	0/1:3:0,3,25	1/1:5:53,6,0	0/1:36:33,0,76

        </p>
        <p>
          11	99715891	rs61749255	T	G	78.20	highCVG	AC1=3;AF1=0.3664;DB;DP=19;DP4=3,8,1,5;FQ=80.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I158M;SNPEFF_CODON_CHANGE=atT/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001243270.1;VDB=0.0015;DN=129;DA=T/G;GM=NM_001243270.1,NM_001243271.1,NM_014361.3,NM_175566.2;GL=CNTN5;FG=missense;FD=missense;AAC=ILE/MET;PP=158/1101,158/912,158/1101,84/1027;CDP=474,474,474,252;GS=10;PH=1;CP=0.990;CG=0.247;AA=T;DV=by-frequency,by-cluster;DSP=73;GESP=G:64/T:11852;PAC=NP_001230199.1,NP_001230200.1,NP_055176.1,NP_780775.1	GT:GQ:PL	0/1:34:54,0,28	0/0:21:0,21,127	1/1:5:70,9,0	0/0:7:0,6,42

        </p>
        <p>
          12	40758652	rs3761863	T	C	14.60	lowCVG	AC1=6;AF1=0.6706;DB;DP=5;DP4=1,0,2,1;FQ=-12.1;MQ=60;PV4=1,0.14,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M2397T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=49;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198578.3;VDB=0.0050;DN=107;DA=T/C;GM=NM_198578.3,XM_005268629.1,XM_005268630.1,XM_005268631.1;GL=LRRK2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=MET/THR;PP=2397/2528,2397/2522,2397/2528,2357/2488;CDP=7190,7190,7190,7070;GS=81;PH=0;CP=0.005;CG=3.030;AA=C;HA=46.5;HE=34.3;HC=42.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=9;KP=hsa05012:Parkinson'sDisease;GESP=C:7962/T:5042;PPI=STUB1:846;PAC=NP_940980.3,XP_005268686.1,XP_005268687.1,XP_005268688.1	GT:GQ:PL	0/1:20:30,0,18	0/1:3:0,0,0	1/1:3:19,3,0	0/1:3:0,0,0

        </p>
        <p>
          12	101560328	rs164365	C	A	44.40	midCVG	AC1=4;AF1=0.4752;DB;DP=14;DP4=4,4,2,3;FQ=47.1;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=M490I;SNPEFF_CODON_CHANGE=atG/atT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC5A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145913.3;VDB=0.0176;DN=79;DA=C/A;GM=NM_145913.3;GL=SLC5A8;FG=missense;FD=missense;AAC=MET/ILE;PP=490/611;CDP=1470;GS=10;PH=0.002;CP=0.751;CG=2.950;AA=C;HA=11.3;HE=19.6;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57;GESP=A:2529/C:10477;PAC=NP_666018.3	GT:GQ:PL	0/0:10:0,12,72	0/1:11:8,0,55	1/1:6:67,9,0	0/1:15:15,0,15

        </p>
        <p>
          12	125467158	rs11558556	C	T	62	highCVG	AC1=2;AF1=0.2591;DB;DP=25;DP4=3,16,2,4;FQ=63.3;G3=0.75,7.094e-07,0.25;HWE=0.0457;MQ=60;PV4=0.56,0.034,1,1;SNPEFF_AMINO_ACID_CHANGE=M96I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHX37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032656.3;VDB=0.0142;DN=120;DA=C/G/T;GM=NM_032656.3,XM_005253590.1;GL=DHX37;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=96/1158,96/1176;CDP=288;GS=10;PH=0;CP=1.000;CG=4.050;AA=C;HA=41.2;HE=31.7;HC=38.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12;GESP=T:4715/C:8291;PAC=NP_116045.2,XP_005253647.1	GT:GQ:PL	0/0:9:0,6,41	0/0:32:0,30,147	1/1:11:108,18,0	0/0:23:0,21,121

        </p>
        <p>
          12	133423662	rs2306536	C	T	17.40	lowCVG	AC1=5;AF1=0.666;DB;DP=8;DP4=3,0,3,0;FQ=12.8;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V580M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHFR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001161344.1;VDB=0.0028;DN=100;DA=C/T;GM=NM_001161344.1,NM_001161345.1,NM_001161346.1,NM_001161347.1,NM_018223.2,XM_005266170.1;GL=CHFR;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=VAL/MET;PP=580/665,579/664,568/653,488/573,539/624,621/706;CDP=1738,1735,1702,1462,1615,1861;GS=21;PH=0.099;CP=0.012;CG=-2.610;AA=C;HA=31.0;HE=16.7;HC=4.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=34;GESP=T:2911/C:10095;PPI=UBC:937|AURKA:846;PAC=NP_001154816.1,NP_001154817.1,NP_001154818.1,NP_001154819.1,NP_060693.2,XP_005266227.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:24,3,0	0/1:32:28,0,43

        </p>
        <p>
          13	96540204	rs12876018	T	G	17.80	midCVG	AC1=3;AF1=0.3733;DB;DP=13;DP4=1,7,2,2;FQ=19.7;MQ=60;PV4=0.24,0.25,1,0.15;SNPEFF_AMINO_ACID_CHANGE=M994L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UGGT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020121.3;VDB=0.0164;DN=121;DA=T/G;GM=NM_020121.3,XM_005254066.1,XM_005254067.1,XM_005254068.1,XM_005254069.1;GL=UGGT2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=MET/LEU;PP=994/1517,992/1515,994/1485,950/1473,640/1163;CDP=2980,2974,2980,2848,1918;GS=15;PH=0.131;CP=0.995;CG=2.860;AA=T;HA=26.0;HE=44.2;HC=28.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4835/T:8169;PPI=BRE:892|BABAM1:871;PAC=NP_064506.3,XP_005254123.1,XP_005254124.1,XP_005254125.1,XP_005254126.1	GT:GQ:PL	0/0:12:0,12,66	0/0:5:0,3,21	1/1:3:31,6,0	0/1:30:30,0,50

        </p>
        <p>
          13	113479820	rs368865	A	G	144	highCVG	AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1	GT:GQ:PL	0/0:10:0,13,159	0/1:3:0,0,0	1/1:21:145,24,0	0/1:46:43,0,56

        </p>
        <p>
          14	20528321	rs1958716	A	G	62.10	midCVG	AC1=4;AF1=0.5177;DB;DP=12;DP4=2,4,3,3;FQ=65.2;MQ=60;PV4=1,0.028,1,0.098;SNPEFF_AMINO_ACID_CHANGE=M40V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004717.1;VDB=0.0138;DN=92;DA=A/G;GM=NM_001004717.1;GL=OR4L1;FG=missense;FD=missense;AAC=MET/VAL;PP=40/313;CDP=118;GS=21;PH=0;CP=0.001;CG=-5.830;HA=45.1;HE=44.5;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=229585;KP=hsa04740:OlfactoryTransduction;GESP=G:5619/A:7387;PAC=NP_001004717.1	GT:GQ:PL	0/1:3:0,0,0	0/0:7:0,9,77	1/1:9:79,12,0	0/1:29:26,0,46

        </p>
        <p>
          14	64491695	rs4902264	T	C	97.50	midCVG	AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:18:122,18,0	0/1:19:13,0,55

        </p>
        <p>
          16	55562466	rs837550	G	A	132	highCVG	AC1=5;AF1=0.5658;DB;DP=21;DP4=11,0,4,5;FQ=111;MQ=60;PV4=0.0081,1,1,0.063;SNPEFF_AMINO_ACID_CHANGE=M163I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LPCAT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017839.4;VDB=0.0146;DN=86;DA=G/A;GM=NM_017839.4,XM_005256006.1;GL=LPCAT2;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=163/545,163/331;CDP=489;GS=10;PH=0;CP=0.006;CG=-3.800;AA=G;HA=38.7;HE=35.2;HC=41.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00564:GlycerophospholipidMetabolism|hsa00565:EtherLipidMetabolism|hsa01100:MetabolicPathways;GESP=A:7296/G:5700;PAC=NP_060309.2,XP_005256063.1	GT:GQ:PL	0/1:14:10,0,42	0/0:20:0,24,113	1/1:3:31,3,0	1/1:19:139,21,0

        </p>
        <p>
          16	75563330	rs3826107	G	A	36.20	lowCVG	AC1=5;AF1=0.6482;DB;DP=5;DP4=0,1,4,0;FQ=-9.91;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T318M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHST5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024533.4;VDB=0.0001;DN=107;DA=G/A;GM=NM_024533.4,XM_005255880.1;GL=CHST5;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=318/412,324/418;CDP=953,971;GS=81;PH=0.999;CP=0.994;CG=2.840;AA=G;HA=9.7;HE=15.6;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=906;CPG=75563114-75564112;GESP=A:2238/G:10758;PAC=NP_078809.2,XP_005255937.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,3,23	1/1:10:73,12,0	0/1:3:0,0,0

        </p>
        <p>
          17	21318867	rs73979893	G	A	999	highCVG	AC1=5;AF1=0.6076;DB;DP=37;DP4=5,10,16,5;FQ=193;MQ=60;PV4=0.017,0.45,1,0.069;SNPEFF_AMINO_ACID_CHANGE=M71I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNJ12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021012.4;VDB=0.0094;DN=130;DA=G/A;GM=NM_021012.4,XM_005256625.1;GL=KCNJ12;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=71/434;CDP=213;GS=10;PH=0;CP=1.000;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=269;CPG=21318651-21320041;TFBS=V$MEIS1AHOXA9_01,V$OCT1_06;PPI=DLG1:972|CASK:833|LIN7B:833|LIN7A:833|APBA1:833|LIN7C:833|DLG4:833;PAC=NP_066292.2,XP_005256682.1	GT:GQ:PL	0/1:54:49,0,91	0/1:35:62,0,34	1/1:8:120,9,0	0/1:59:54,0,108

        </p>
        <p>
          17	26694861	rs704	G	A	31.80	highCVG	AC1=4;AF1=0.4318;DB;DP=15;DP4=3,7,1,3;FQ=34;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T400M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000638.3;VDB=0.0089;DN=36;DA=G/A;GM=NM_000638.3;GL=VTN;FG=missense;FD=missense;AAC=THR/MET;PP=400/479;CDP=1199;GS=81;PH=0.943;CP=0.000;CG=-8.940;AA=G;HA=35.0;HE=46.9;HC=25.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=126;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=A:6627/G:6379;PAC=NP_000629.3	GT:GQ:PL	0/0:12:0,12,67	0/1:3:0,0,0	1/1:5:69,9,0	0/1:5:3,0,95

        </p>
        <p>
          17	27284443	rs12941884	A	G	58.10	midCVG	AC1=5;AF1=0.587;DB;DP=9;DP4=2,0,4,1;FQ=5.4;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=M806T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEZ6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178860.4;VDB=0.0169;DN=121;DA=A/G;GM=NM_001098635.1,NM_178860.4,XM_005257919.1;GL=SEZ6;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=806/994,806/995,681/870;CDP=2417,2417,2042;GS=81;PH=0;CP=0.999;CG=3.650;AA=G;HA=20.8;HE=14.6;HC=8.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=78;GESP=G:1806/A:10402;PAC=NP_001092105.1,NP_849191.3,XP_005257976.1	GT:GQ:PL	0/1:3:0,0,0	0/1:18:21,0,17	1/1:10:78,12,0	0/1:3:0,3,24

        </p>
        <p>
          19	20133846	rs7255165	C	T	23.50	midCVG	AC1=3;AF1=0.4339;DB;DP=12;DP4=5,4,2,1;FQ=25.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V65M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF682;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033196.2;VDB=0.0044;DN=116;DA=C/T;GM=NM_001077349.1,NM_033196.2;GL=ZNF682;FG=missense;FD=missense;AAC=VAL/MET;PP=33/467,65/499;CDP=97,193;GS=21;PH=0.005;CP=0.300;CG=0.898;AA=C;HA=42.2;HE=18.1;HC=6.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=34;GESP=T:2867/C:9861;PAC=NP_001070817.1,NP_149973.1	GT:GQ:PL	0/0:20:0,21,136	0/1:3:0,0,0	1/1:4:48,6,0	0/1:18:17,0,43

        </p>
        <p>
          19	37118439	rs61732180	C	T	83.80	midCVG	AC1=5;AF1=0.6451;DB;DP=9;DP4=3,0,3,3;FQ=15.4;MQ=60;PV4=0.46,1,1,0.002;SNPEFF_AMINO_ACID_CHANGE=T547M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0028;DN=129;DA=C/T;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=THR/MET;PP=546/550,547/551,547/551,546/550;CDP=1637,1640,1640,1637;GS=81;PH=0;CP=0.009;CG=1.990;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1408;GESP=T:2043/C:10963;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1	GT:GQ:PL	0/1:17:27,0,16	0/0:4:0,6,41	1/1:9:59,9,0	1/1:6:44,6,0

        </p>
        <p>
          19	48495304	rs60213124	G	A	79	midCVG	AC1=4;AF1=0.5008;DB;DP=12;DP4=3,3,3,3;FQ=79.1;MQ=60;PV4=1,0.42,1,0.17;SNPEFF_AMINO_ACID_CHANGE=T12M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BSPH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128326.2;VDB=0.0230;DN=129;DA=G/A;GM=NM_001128326.2;GL=BSPH1;FG=missense;FD=missense;AAC=THR/MET;PP=12/133;CDP=35;GS=81;PH=0.962;CP=0.000;CG=-0.642;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=G:4566;PAC=NP_001121798.1	GT:GQ:PL	0/0:13:0,15,105	0/1:3:27,3,0	1/1:12:99,15,0	0/0:3:0,3,21

        </p>
        <p>
          19	59023174	rs35350976	A	G	35.70	lowCVG	AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1	GT:GQ:PL	0/1:3:0,0,0	0/0:3:0,3,33	1/1:6:69,9,0	0/1:9:6,0,61

        </p>
        <p>
          20	60899206	rs2427283	C	T	73.40	midCVG	AC1=5;AF1=0.6657;DB;DP=10;DP4=0,1,1,6;FQ=-7.71;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=V1900M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0025;DN=100;DA=C/T;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=missense;AAC=VAL/MET;PP=1900/3696;CDP=5698;GS=21;PH=0.839;CP=0.017;CG=1.990;AA=C;HE=31.7;HC=20.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=T:8377/C:4565;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3	GT:GQ:PL	0/1:4:0,3,26	0/1:3:0,0,0	1/1:19:111,21,0	0/1:3:0,0,0

        </p>
        <p>
          21	31812772	rs2832873	C	A	95.50	highCVG	AC1=6;AF1=0.744;DB;DP=17;DP4=3,3,4,6;FQ=70.4;MQ=60;PV4=1,0.023,1,0.032;SNPEFF_AMINO_ACID_CHANGE=L43M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRTAP15-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181623.1;VDB=0.0148;DN=100;DA=C/A;GM=NM_181623.1;GL=KRTAP15-1;FG=missense;FD=missense;AAC=LEU/MET;PP=43/138;CDP=127;GS=15;PH=0.956;CP=0.961;CG=0.826;AA=C;HA=26.2;HE=45.1;HC=34.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=228721;GESP=A:7882/C:5124;PAC=NP_853654.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:24:124,24,0	0/1:15:9,0,102

        </p>
        <p>
          21	38525356	rs1053808	T	C	81.50	midCVG	AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1	GT:GQ:PL	0/1:30:24,0,88	0/1:14:50,0,13	1/1:5:28,3,0	1/1:5:21,3,0

        </p>
        <p>
          22	21800806	.	T	C	11.20	highCVG	AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1	GT:GQ:PL	0/0:39:0,36,185	0/0:4:0,0,0	1/1:3:52,9,0	0/0:18:0,15,98

        </p>
        <p>
          22	30888527	rs17738540	C	T	33.50	lowCVG	AC1=3;AF1=0.4062;DB;DP=8;DP4=2,2,3,1;FQ=34.5;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=V200M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC14L4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174977.3;VDB=0.0080;DN=123;DA=C/T;GM=NM_001161368.1,NM_174977.3,XM_005261583.1,XM_005261584.1,XM_005261585.1,XM_005261586.1;GL=SEC14L4;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=200/361,200/407,201/408,185/392,146/361,117/324;CDP=598,598,601,553,436,349;GS=21;PH=1;CP=0.855;CG=2.760;AA=C;HA=4.6;HE=24.7;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18;TFBS=V$HNF4_01,V$MIF1_01;GESP=T:2478/C:10528;PAC=NP_001154840.1,NP_777637.1,XP_005261640.1,XP_005261641.1,XP_005261642.1,XP_005261643.1	GT:GQ:PL	0/0:6:0,6,44	0/0:4:0,3,20	1/1:3:43,6,0	0/1:22:33,0,17

        </p>
        <p>
          22	46655779	rs36125344	G	C	91.40	midCVG	AC1=5;AF1=0.6118;DB;DP=12;DP4=3,2,1,6;FQ=52.5;MQ=60;PV4=0.22,1,1,0.45;SNPEFF_AMINO_ACID_CHANGE=I1147M;SNPEFF_CODON_CHANGE=atC/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKDREJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006071.1;VDB=0.0056;DN=126;DA=G/C;GM=NM_006071.1,XM_005261302.1;GL=PKDREJ;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=1147/2254,1147/2163;CDP=3441;GS=10;PH=0.022;CP=0.000;CG=-8.750;AA=G;HA=3.3;HE=15.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=643;GESP=C:1242/G:11764;PAC=NP_006062.1,XP_005261359.1	GT:GQ:PL	0/0:8:0,12,77	0/1:19:19,0,20	1/1:8:65,9,0	1/1:8:56,9,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00021;FLCN|FH|FIGF|GSN|MET|MKI67|PRCC|TFE3|TP53|VEGFA|VEGFB|VEGFC|VHL|VIM|CA9|CA12|CD44;hsa05211

        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 64
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse7">Bladder cancer</a>
      </h4>
    </div>
    <div id="collapse7" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	17657616	rs11203367	T	C	17.40	lowCVG	AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:19,3,0	0/1:37:33,0,50

        </p>
        <p>
          1	17660468	rs34309058	C	A	35.70	midCVG	AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2	GT:GQ:PL	0/0:16:0,15,97	0/0:3:0,0,0	1/1:6:78,12,0	0/0:13:0,12,80

        </p>
        <p>
          3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0

        </p>
        <p>
          6	43011188	.	G	T	66.80	highCVG	AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1	GT:GQ:PL	0/0:7:0,6,35	0/0:5:0,3,19	1/1:23:112,30,0	0/0:16:0,15,97

        </p>
        <p>
          7	77789554	.	G	A	51.40	highCVG	AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1	GT:GQ:PL	0/0:13:0,12,79	0/0:13:0,12,69	1/1:17:97,24,0	0/0:5:0,3,25

        </p>
        <p>
          7	154738409	rs3501	T	C	124	highCVG	AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1	GT:GQ:PL	0/0:8:0,9,63	0/0:6:0,6,48	1/1:20:149,24,0	0/1:20:22,0,18

        </p>
        <p>
          8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75

        </p>
        <p>
          11	63233721	rs149917445	G	A	18	midCVG	AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1	GT:GQ:PL	0/0:18:0,15,108	0/0:9:0,6,46	1/1:3:60,9,0	0/0:9:0,6,41

        </p>
        <p>
          11	69466030	.	G	A	20.20	lowCVG	AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1	GT:GQ:PL	0/0:6:0,3,17	0/0:6:0,3,14	1/1:4:61,9,0	0/0:11:0,9,66

        </p>
        <p>
          19	55143083	rs1061679	T	C	34	lowCVG	AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA	GT:GQ:PL	0/0:12:0,15,73	0/1:3:0,0,0	1/1:5:50,6,0	0/1:3:26,3,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00022;CDKN2A|CEACAM5|MGEA5|CFHR4|CFHR3|RASSF1|DAPK1|EGFR|ERBB2|FGFR3|HBA1|HBA2|HBB|CFH|CFHR1|CFHR2|HRAS|BIRC5|KRT8|KRT18|KRT19|NUMA1|KRT20|RB1|TERT|TP53|CFHR5;hsa05219

        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 10
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse8">Prostate cancer</a>
      </h4>
    </div>
    <div id="collapse8" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	12855878	rs1052908	A	C	37.70	midCVG	AC1=6;AF1=0.6449;DB;DP=11;DP4=3,1,1,4;FQ=-6.46;MQ=20;PV4=0.21,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R386S;SNPEFF_CODON_CHANGE=agA/agC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRAMEF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_023013.4;VDB=0.0080;DN=86;DA=A/C;GM=NM_023013.2,XM_005263490.1;GL=PRAMEF1;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=386/475,141/230;CDP=1158,423;GS=110;PH=0;CP=0.004;CG=0.600;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,suspect;DSP=292;GESP=A:12994;PAC=NP_075389.1,XP_005263547.1	GT:GQ:PL	0/1:2:4,3,0	0/1:3:0,0,0	1/1:3:70,4,0	0/0:3:0,3,27

        </p>
        <p>
          1	15909850	rs6429757	C	G	23.50	midCVG	AC1=3;AF1=0.403;DB;DP=14;DP4=9,2,3,0;FQ=25.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G105R;SNPEFF_CODON_CHANGE=Ggg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AGMAT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024758.4;VDB=0.0052;DN=116;DA=C/G;GM=NM_024758.4;GL=AGMAT;FG=missense;FD=missense;AAC=GLY/ARG;PP=105/353;CDP=313;GS=125;PH=0;CP=0.010;CG=-1.410;AA=G;HA=7.9;HE=49.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;TFBS=V$COMP1_01;GESP=G:8504/C:4502;PAC=NP_079034.3	GT:GQ:PL	0/0:6:0,6,49	0/1:4:22,3,0	1/1:3:47,6,0	0/0:26:0,27,74

        </p>
        <p>
          1	24201920	rs2501432	T	C	86.40	highCVG	AC1=5;AF1=0.5586;DB;DP=20;DP4=8,2,3,5;FQ=69.9;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q63R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001841.2;VDB=0.0031;DN=100;DA=T/C;GM=NM_001841.2,XM_005245736.1,XM_005245737.1,XM_005245738.1;GL=CNR2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLN/ARG;PP=63/361;CDP=188;GS=43;PH=0;CP=0.910;CG=5.120;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=233;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=C:7940/T:5066;PAC=NP_001832.1,XP_005245793.1,XP_005245794.1,XP_005245795.1	GT:GQ:PL	0/1:23:46,0,21	0/0:20:0,24,93	1/1:3:16,3,0	1/1:10:72,12,0

        </p>
        <p>
          1	36550658	rs12043423	C	T	16.90	lowCVG	AC1=4;AF1=0.4561;DB;DP=8;DP4=2,3,2,1;FQ=19.5;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=R46C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TEKT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014466.2;VDB=0.0094;DN=120;DA=C/T;GM=NM_014466.2,XM_005270753.1;GL=TEKT2;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=46/431;CDP=136;GS=180;PH=1;CP=1.000;CG=3.680;AA=C;HA=6.1;HE=0.0;HC=19.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=21;GESP=T:273/C:12733;PAC=NP_055281.2,XP_005270810.1	GT:GQ:PL	0/1:3:0,0,0	0/0:11:0,12,80	1/1:4:37,6,0	0/1:16:21,0,13

        </p>
        <p>
          1	53676448	rs1799821	G	A	24.80	lowCVG	AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1	GT:GQ:PL	0/1:3:0,0,0	0/0:3:0,3,22	1/1:5:45,6,0	0/1:20:19,0,22

        </p>
        <p>
          1	54661150	rs946448	C	T	61.80	highCVG	AC1=4;AF1=0.5145;DB;DP=17;DP4=6,2,5,1;FQ=64.8;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=R47K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYB5RL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001031672.2;VDB=0.0105;DN=86;DA=C/T;GM=NM_001031672.2,XM_005271096.1;GL=CYB5RL;FG=missense;FD=missense,unknown;AAC=ARG/LYS;PP=47/316,47/237;CDP=140;GS=26;PH=0.159;CP=1.000;CG=4.240;AA=C;HA=17.4;HE=37.5;HC=24.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=59;GESP=T:3741/C:8713;PAC=NP_001026842.2,XP_005271153.1	GT:GQ:PL	0/0:3:0,3,18	0/0:18:0,21,113	1/1:9:69,12,0	1/1:5:41,6,0

        </p>
        <p>
          1	152285603	.	G	A	73.80	midCVG	AC1=5;AF1=0.6441;DP=9;DP4=2,1,1,5;FQ=9.97;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R587C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0003;GM=NM_002016.1;GL=FLG;FG=missense;FD=unknown;AAC=ARG/CYS;PP=587/4062;CDP=1759;GS=180;PH=1;CP=0.000;CG=-3.110;AA=G;DSP=912;GESP=G:13006;PPI=FGFR1:845;PAC=NP_002007.1	GT:GQ:PL	0/1:10:19,0,9	0/0:4:0,6,41	1/1:4:26,3,0	1/1:11:74,12,0

        </p>
        <p>
          1	152770521	rs41268492	G	A	13.80	lowCVG	AC1=6;AF1=0.699;DB;DP=5;DP4=1,0,3,0;FQ=-21.4;MQ=47;PV4=1,1,0.48,1;SNPEFF_AMINO_ACID_CHANGE=R84H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LCE1D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178352.2;VDB=0.0045;DN=127;DA=G/A;GM=NM_178352.2;GL=LCE1D;FG=missense;FD=missense;AAC=ARG/HIS;PP=84/115;CDP=251;GS=29;PH=0;CP=0.000;CG=-7.330;AA=A;DV=by-frequency,by-cluster;DSP=273;GESP=A:2436/G:9006;PAC=NP_848129.1	GT:GQ:PL	0/1:3:0,0,0/1:4:0,1,8	1/1:4:47,6,0	0/1:3:0,0,0

        </p>
        <p>
          1	156883546	rs12137505	A	G	17.30	lowCVG	AC1=4;AF1=0.4665;DB;DP=8;DP4=3,1,2,1;FQ=20;MQ=60;PV4=1,0.24,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N903D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PEAR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080471.1;VDB=0.0107;DN=120;DA=A/G;GM=NM_001080471.1,XM_005245141.1,XM_005245142.1;GL=PEAR1;FG=missense;FD=missense,unknown,unknown;AAC=ASN/ASP;PP=903/1038,903/1038,839/974;CDP=2707,2707,2515;GS=23;PH=0.094;CP=0.000;CG=-3.910;AA=A;HA=20.6;HE=35.7;HC=48.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=7;GESP=G:4616/A:8390;PAC=NP_001073940.1,XP_005245198.1,XP_005245199.1	GT:GQ:PL	0/0:4:0,3,27	0/0:8:0,9,64	1/1:4:44,6,0	0/1:4:17,3,0

        </p>
        <p>
          1	169823718	rs4656197	T	C	47.80	highCVG	AC1=4;AF1=0.5453;DB;DP=15;DP4=3,3,0,7;FQ=39.1;MQ=60;PV4=0.07,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=Q621R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCYL3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181093.3;VDB=0.0088;DN=111;DA=T/A/C/G;GM=NM_020423.6,NM_181093.3;GL=SCYL3;FG=missense;FD=missense;AAC=GLN/ARG;PP=567/689,621/743;CDP=1700,1862;GS=43;PH=0.589;CP=0.000;CG=-1.020;AA=T;HA=27.3;HE=43.9;HC=34.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=308;GESP=C:8150/T:4856;PAC=NP_065156.5,NP_851607.2	GT:GQ:PL	0/1:5:52,0,0	0/1:3:0,0,0	1/1:7:37,9,0	0/0:12:0,15,77

        </p>
        <p>
          1	182025558	.	T	C	23	midCVG	AC1=2;AF1=0.2266;DP=12;DP4=0,9,0,3;FQ=24;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R530G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF648;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001009992.1;VDB=0.0015;GM=NM_001009992.1;GL=ZNF648;FG=missense;FD=unknown;AAC=ARG/GLY;PP=530/569;CDP=1588;GS=125;PH=0.797;CP=0.999;CG=1.640;AA=T;DSP=1651;GESP=T:13006;PAC=NP_001009992.1	GT:GQ:PL	0/0:9:0,6,35	0/0:9:0,6,44	1/1:3:65,9,0	0/0:18:0,15,92

        </p>
        <p>
          1	206225011	rs33990840	G	C	30	lowCVG	AC1=4;AF1=0.5539;DB;DP=5;DP4=2,0,1,2;FQ=11.7;MQ=60;PV4=0.4,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G191R;SNPEFF_CODON_CHANGE=Ggc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVPR1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000707.3;VDB=0.0112;DN=126;DA=G/C;GM=NM_000707.3;GL=AVPR1B;FG=missense;FD=missense;AAC=GLY/ARG;PP=191/425;CDP=571;GS=125;PH=0;CP=1.000;CG=4.980;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=370;KP=hsa04020:CalciumSignalingPathway|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04270:VascularSmoothMuscleContraction;GESP=C:802/G:12188;PAC=NP_000698.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:67,9,0	0/0:6:0,6,45

        </p>
        <p>
          1	207790088	rs3811381	C	G	67.70	midCVG	AC1=5;AF1=0.5957;DB;DP=9;DP4=0,4,1,4;FQ=29.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2277R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=41;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000651.4;VDB=0.0076;DN=107;DA=C/G;GM=NM_000573.3,NM_000651.4,XM_005273064.1;GL=CR1;FG=missense;FD=missense,missense,unknown;AAC=PRO/ARG;PP=1827/2040,2277/2490,2149/2362;CDP=5480,6830,6446;GS=103;PH=0.643;CP=0.000;CG=-5.580;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;KP=hsa04610:ComplementAndCoagulationCascades|hsa04640:HematopoieticCellLineage|hsa05140:Leishmaniasis;GESP=G:2006/C:10176;PPI=CR2:846;PAC=NP_000564.2,NP_000642.3,XP_005273121.1	GT:GQ:PL	0/1:35:36,0,34	0/0:4:0,6,36	1/1:6:51,6,0	1/1:4:25,3,0

        </p>
        <p>
          1	214830617	rs438034	A	G	57	lowCVG	AC1=5;AF1=0.6559;DB;DP=8;DP4=2,0,2,4;FQ=7.46;MQ=60;PV4=0.43,0.015,1,1;SNPEFF_AMINO_ACID_CHANGE=R2943G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0106;DN=80;DA=A/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ARG/GLY;PP=2943/3115,2921/3093;CDP=8827,8761;GS=125;PH=0;CP=0.000;CG=2.520;AA=A;HA=36.6;HE=47.7;HC=9.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=136;GESP=G:6399/A:6607;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1	GT:GQ:PL	0/1:18:16,0,20	0/1:18:14,0,23	1/1:11:68,12,0	0/1:3:0,0,0

        </p>
        <p>
          1	228559430	rs41305136	G	A	31.70	lowCVG	AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1	GT:GQ:PL	0/0:4:0,6,43	0/1:3:0,0,0	1/1:4:20,3,0	1/1:5:52,6,0

        </p>
        <p>
          1	247420421	rs41308154	C	T	25.10	midCVG	AC1=3;AF1=0.3809;DB;DP=11;DP4=2,3,2,2;FQ=27.1;MQ=60;PV4=1,0.0023,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R350C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VN1R5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173858.1;VDB=0.0123;DN=127;DA=C/T;GM=NM_173858.1;GL=VN1R5;FG=missense;FD=unknown;AAC=ARG/CYS;PP=350/358;CDP=1048;GS=180;PH=0.987;CP=0.337;AA=C;DG;DV=by-cluster,by-1000G;RM=MSTC;DSP=44158;GESP=T:2053/C:9883;PAC=NP_776257.1	GT:GQ:PL	0/0:3:0,0,0	0/0:12:0,12,89	1/1:7:66,12,0	0/0:5:0,3,23

        </p>
        <p>
          1	248458718	rs79635253	T	C	14.40	midCVG	AC1=4;AF1=0.443;DB;DP=9;DP4=4,1,1,2;FQ=16.8;MQ=45;PV4=0.46,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R55G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004692.1;VDB=0.0077;DN=131;DA=T/C;GM=NM_001004692.1;GL=OR2T12;FG=missense;FD=missense;AAC=ARG/GLY;PP=55/321;CDP=163;GS=125;PH=0.005;CP=0.159;CG=1.550;AA=T;DV=by-frequency,by-cluster;DSP=196059;KP=hsa04740:OlfactoryTransduction;GESP=C:613/T:12389;PAC=NP_001004692.1	GT:GQ:PL	0/0:4:0,3,23	0/1:4:13,3,0	1/1:3:45,6,0	0/0:12:0,12,73

        </p>
        <p>
          1	248722777	rs77589892	T	A	89	highCVG	AC1=5;AF1=0.7783;DB;DP=29;DP4=7,2,14,5;FQ=-25.8;MQ=12;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R6W;SNPEFF_CODON_CHANGE=Agg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004694.2;VDB=0.0185;DN=131;DA=T/A;GM=NM_001004694.2;GL=OR2T29;FG=missense;FD=missense;AAC=ARG/TRP;PP=6/316;CDP=16;GS=101;CP=0.001;CG=-0.604;AA=A;DG;DV=by-frequency,by-cluster;DSP=122849;KP=hsa04740:OlfactoryTransduction;GESP=A:4597/T:8405;PAC=NP_001004694.2	GT:GQ:PL	0/0:4:0,12,14	0/1:3:6,0,2	1/1:22:86,20,0	1/1:23:44,21,0

        </p>
        <p>
          2	25022598	rs1550116	A	G	39.40	lowCVG	AC1=6;AF1=0.7255;DB;DP=6;DP4=1,1,1,3;FQ=3.46;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q34R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024322.2;VDB=0.0085;DN=88;DA=A/G;GM=NM_001199803.1,NM_024322.2;GL=CENPO;FG=missense;FD=missense;AAC=GLN/ARG;PP=28/295,34/301;CDP=83,101;GS=43;PH=0;CP=0.000;CG=-3.630;AA=A;HA=5.8;HE=18.0;HC=19.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=55;GESP=G:1761/A:11245;PAC=NP_001186732.1,NP_077298.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:9:64,9,0	0/1:17:12,0,32

        </p>
        <p>
          2	131414176	rs62165276	C	T	7.93	midCVG	AC1=4;AF1=0.5176;DB;DP=10;DP4=1,3,3,2;FQ=10.6;MQ=18;PV4=0.52,0.35,0.061,0.14;SNPEFF_AMINO_ACID_CHANGE=R615W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POTEJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277083.1;VDB=0.0090;DN=129;DA=C/T;GM=NM_001277083.1,XM_005263739.1,XM_005263740.1;GL=POTEJ;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=615/1039,497/921,492/916;CDP=1843,1489,1474;GS=101;CP=0.246;CG=0.993;AA=C;DG;DV=by-frequency,by-cluster;DSP=55;PAC=NP_001264012.1,XP_005263796.1,XP_005263797.1	GT:GQ:PL	0/0:6:0,6,33	0/1:3:0,0,0	1/1:5:39,9,0	0/1:10:9,0,34

        </p>
        <p>
          2	167099158	rs6746030	A	G	150	highCVG	AC1=6;AF1=0.7278;DB;DP=16;DP4=2,4,7,3;FQ=65.7;MQ=60;PV4=0.3,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W1150R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCN9A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002977.3;VDB=0.0076;DN=116;DA=A/G;GM=NM_002977.3,XM_005246757.1;GL=SCN9A;FG=missense;FD=missense,unknown;AAC=TRP/ARG;PP=1150/1978,1161/1989;CDP=3448,3481;GS=101;PH=0;CP=0.788;CG=3.830;AA=G;HA=13.1;HE=12.3;HC=4.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=9;GESP=G:10303/A:1439;PAC=NP_002968.1,XP_005246814.1	GT:GQ:PL	0/1:43:47,0,44	0/1:22:15,0,61	1/1:10:59,9,0	1/1:10:74,9,0

        </p>
        <p>
          2	178417142	rs11694988	C	T	97.90	midCVG	AC1=5;AF1=0.5858;DB;DP=13;DP4=2,4,3,4;FQ=78.7;MQ=50;PV4=1,0.46,0.044,0.32;SNPEFF_AMINO_ACID_CHANGE=R117H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152517.2;VDB=0.0034;DN=120;DA=C/T;GM=NM_152517.2;GL=TTC30B;FG=missense;FD=missense;AAC=ARG/HIS;PP=117/666;CDP=350;GS=29;PH=0.944;CP=0.996;CG=4.300;AA=C;HA=15.8;HE=14.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14341;GESP=T:11079/C:1927;PAC=NP_689730.2	GT:GQ:PL	0/0:11:0,15,107	0/1:18:35,0,16	1/1:8:64,9,0	1/1:6:47,6,0

        </p>
        <p>
          2	207570529	rs56067462	G	C	69.40	midCVG	AC1=4;AF1=0.5258;DB;DP=14;DP4=2,5,4,2;FQ=67.2;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P122R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DYTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001093730.1;VDB=0.0048;DN=129;DA=G/C;GM=NM_001093730.1;GL=DYTN;FG=missense;FD=missense;AAC=PRO/ARG;PP=122/579;CDP=365;GS=103;PH=1;CP=1.000;CG=4.150;AA=G;DV=by-frequency,by-cluster;DSP=18;GESP=C:854/G:11154;PAC=NP_001087199.1	GT:GQ:PL	0/1:3:0,0,0	0/0:15:0,18,106	1/1:7:57,9,0	0/1:6:54,0,1

        </p>
        <p>
          2	240969312	rs10187574	A	G	13.60	midCVG	AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1	GT:GQ:PL	0/0:11:0,12,14	0/1:3:6,3,0	1/1:7:53,12,0	0/0:8:0,9,11

        </p>
        <p>
          3	101283792	rs3762735	C	G	93	highCVG	AC1=4;AF1=0.4986;DB;DP=20;DP4=6,4,7,2;FQ=96;MQ=60;PV4=0.63,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=P56R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRMT10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017819.3;VDB=0.0174;DN=107;DA=C/G;GM=NM_017819.3;GL=TRMT10C;FG=missense;FD=missense;AAC=PRO/ARG;PP=56/404;CDP=167;GS=103;PH=0.001;CP=0.084;CG=5.160;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=179;GESP=G:1711/C:10225;PAC=NP_060289.2	GT:GQ:PL	0/0:15:0,18,120	0/1:29:26,0,47	1/1:15:98,18,0	0/1:20:17,0,29

        </p>
        <p>
          3	121208833	rs3218649	G	C	208	highCVG	AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3	GT:GQ:PL	0/1:37:59,0,36	0/0:43:0,48,136	1/1:20:109,21,0	1/1:11:91,12,0

        </p>
        <p>
          3	185990096	rs2193587	C	T	115	midCVG	AC1=5;AF1=0.651;DB;DP=13;DP4=1,3,1,7;FQ=49.8;MQ=60;PV4=1,0.2,1,0.13;SNPEFF_AMINO_ACID_CHANGE=R316K;SNPEFF_CODON_CHANGE=aGa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DGKG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001346.2;VDB=0.0103;DN=96;DA=C/T;GM=NM_001080744.1,NM_001080745.1,NM_001346.2,XM_005247156.1,XM_005247157.1,XM_005247158.1;GL=DGKG;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=316/767,316/753,316/792,316/768,258/734,316/733;CDP=947,947,947,947,773,947;GS=26;PH=0;CP=1.000;CG=3.920;AA=T;HA=11.8;HE=14.2;HC=29.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=37;KP=hsa00561:GlycerolipidMetabolism|hsa00564:GlycerophospholipidMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=T:10726/C:2280;PAC=NP_001074213.1,NP_001074214.1,NP_001337.2,XP_005247213.1,XP_005247214.1,XP_005247215.1	GT:GQ:PL	0/0:3:0,6,53	0/1:37:36,0,39	1/1:11:72,12,0	1/1:6:54,6,0

        </p>
        <p>
          4	2451694	rs3108494	T	C	127	midCVG	AC1=6;AF1=0.6856;DB;DP=10;DP4=1,1,6,2;FQ=6.6;MQ=60;PV4=1,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=C253R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0035;DN=103;DA=T/C;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=CYS/ARG;PP=253/647;CDP=757;GS=180;CP=0.976;CG=3.340;AA=C;DG;DV=by-frequency,by-cluster,by-1000G,by-other-pop;DSP=34;PAC=NP_001180211.1	GT:GQ:PL	0/1:20:22,0,20	0/1:5:0,3,26	1/1:7:51,6,0	1/1:15:99,15,0

        </p>
        <p>
          4	8603105	rs115392829	G	A	26.80	lowCVG	AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA	GT:GQ:PL	0/0:7:0,6,47	0/0:4:0,3,18	1/1:7:67,12,0	0/1:3:0,0,0

        </p>
        <p>
          4	71232388	rs10031844	G	A	39.10	lowCVG	AC1=5;AF1=0.5822;DB;DP=6;DP4=0,2,2,2;FQ=11.2;MQ=58;PV4=0.47,0.22,1,0.3;SNPEFF_AMINO_ACID_CHANGE=G28R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMR3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012390.3;VDB=0.0103;DN=119;DA=G/A;GM=NM_012390.3;GL=SMR3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=28/135;CDP=82;GS=125;PH=0.001;CP=0.000;CG=-5.140;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=28;GESP=A:3028/G:9978;PAC=NP_036522.3	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:9:77,12,0	0/0:5:0,6,45

        </p>
        <p>
          4	77631425	rs3821979	T	A	48.70	midCVG	AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA	GT:GQ:PL	0/1:35:32,0,38	0/1:21:20,0,22	1/1:4:23,3,0	0/1:20:16,0,43

        </p>
        <p>
          4	88415636	rs1049544	G	C	88.60	midCVG	AC1=4;AF1=0.5004;DB;DP=12;DP4=2,3,5,2;FQ=51.6;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H106D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPARCL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128310.2;VDB=0.0073;DN=86;DA=G/C;GM=NM_001128310.1,NM_004684.4,XM_005263272.1,XM_005263273.1;GL=SPARCL1;FG=missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown;AAC=HIS/ASP,HIS/ASP,HIS/ASP,none;PP=106/665,106/665,106/665,NA;CDP=316,316,316,NA;GS=81,81,81,NA;PH=0;CP=0.015;CG=4.250;AA=C;HA=8.4;HE=33.5;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=115;GESP=C:6754/G:6252;PAC=NP_001121782.1,NP_004675.3,XP_005263329.1,NA	GT:GQ:PL	0/0:7:0,9,68	0/1:15:50,0,12	1/1:7:74,9,0	0/1:12:10,0,14

        </p>
        <p>
          4	95170839	rs11722476	G	A	30.20	highCVG	AC1=3;AF1=0.351;DB;DP=19;DP4=5,7,0,5;FQ=32;MQ=60;PV4=0.24,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=S247N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0126;DN=120;DA=G/A;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none;PP=247/1029,247/1029,247/1027,NA;CDP=740,740,740,1066;GS=46,46,46,NA;PH=0;CP=0.987;CG=0.199;AA=G;HA=20.5;HE=45.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:4424/G:8574;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA	GT:GQ:PL	0/0:10:0,9,68	0/1:8:7,0,75	1/1:7:69,12,0	0/0:15:0,15,90

        </p>
        <p>
          4	95173779	rs7439869	T	C	130	midCVG	AC1=3;AF1=0.4198;DB;DP=14;DP4=2,3,3,6;FQ=65;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=V301A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0162;DN=116;DA=T/C;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=VAL/ALA,VAL/ALA,VAL/ALA,none;PP=301/1029,301/1029,301/1027,NA;CDP=902,902,902,1228;GS=64,64,64,NA;PH=0;CP=0.012;CG=4.400;AA=C;HA=5.0;HE=41.6;HC=28.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=C:9055/T:3845;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA	GT:GQ:PL	0/1:11:100,0,6	0/0:6:0,6,45	1/1:6:74,9,0	0/0:6:0,6,56

        </p>
        <p>
          4	128996129	.	G	T	28.60	midCVG	AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1	GT:GQ:PL	0/0:8:0,6,51	0/0:6:0,3,17	1/1:6:71,12,0	0/0:8:0,6,45

        </p>
        <p>
          4	155287535	rs10017772	T	C	73.60	midCVG	AC1=5;AF1=0.5889;DB;DP=10;DP4=2,1,5,2;FQ=5.24;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=H174R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCHS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017639.3;VDB=0.0092;DN=119;DA=T/C;GM=NM_001142552.1,NM_017639.3;GL=DCHS2;FG=missense;FD=missense;AAC=HIS/ARG;PP=768/1370,174/2917;CDP=2303,521;GS=29;PH=0.007;CP=0.001;CG=2.100;AA=T;HA=36.4;HE=48.0;HC=45.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;TFBS=V$CART1_01;GESP=C:5817/T:7189;PAC=NP_001136024.1,NP_060109.2	GT:GQ:PL	0/1:7:54,0,3	0/1:3:0,0,0	1/1:7:60,9,0	0/0:4:0,6,37

        </p>
        <p>
          4	169611811	.	C	T	14.60	midCVG	AC1=2;AF1=0.2475;DP=11;DP4=6,2,0,3;FQ=15.7;MQ=60;PV4=0.061,1,1,0.052;SNPEFF_AMINO_ACID_CHANGE=R465C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PALLD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001166108.1;VDB=0.0128;GM=NM_001166108.1,NM_001166109.1,NM_016081.3,XM_005262861.1,XM_005262862.1,XM_005262863.1,XM_005262864.1,XM_005262865.1,XM_005262866.1;GL=PALLD;FG=missense;FD=unknown;AAC=ARG/CYS;PP=465/1124,83/778,465/1107,465/1401,465/1384,465/1348,465/1310,465/1277,83/1019;CDP=1393,247,1393,1393,1393,1393,1393,1393,247;GS=180;PH=1;CP=0.789;CG=3.960;AA=C;DSP=58;TFBS=V$TCF11MAFG_01;GESP=C:13006;PPI=EZR:846|ACTN1:846;PAC=NP_001159580.1,NP_001159581.1,NP_057165.3,XP_005262918.1,XP_005262919.1,XP_005262920.1,XP_005262921.1,XP_005262922.1,XP_005262923.1	GT:GQ:PL	0/0:7:0,3,16	0/0:9:0,6,36	1/1:3:56,9,0	0/0:18:0,15,104

        </p>
        <p>
          4	187627792	rs11939575	T	C	10.40	lowCVG	AC1=4;AF1=0.5539;DB;DP=5;DP4=0,2,3,0;FQ=9.79;MQ=60;PV4=0.1,0.00034,1,1;SNPEFF_AMINO_ACID_CHANGE=R1064G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0075;DN=120;DA=T/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLY;PP=1064/4589,1064/4601,1064/4601;CDP=3190;GS=125;PH=0;CP=0.635;CG=4.600;AA=C;HA=9.0;HE=23.5;HC=49.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=76;GESP=C:10198/T:2130;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1	GT:GQ:PL	0/0:6:0,6,45	0/1:3:0,0,0	1/1:5:47,9,0	0/1:3:0,0,0

        </p>
        <p>
          5	54439466	rs1021580	G	A	260	highCVG	AC1=6;AF1=0.805;DB;DP=19;DP4=1,2,4,12;FQ=-17.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R121C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170402.1;VDB=0.0082;DN=86;DA=G/A;GM=NM_001145734.2,NM_001170402.1,NM_152623.2,XM_005248450.1;GL=CDC20B;FG=missense;FD=missense,missense,missense,unknown;AAC=ARG/CYS;PP=121/478,121/520,121/516,121/373;CDP=361;GS=180;PH=0;CP=0.001;CG=-1.690;AA=A;HA=49.1;HE=18.6;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;GESP=A:9758/G:3248;PAC=NP_001139206.2,NP_001163873.1,NP_689836.2,XP_005248507.1	GT:GQ:PL	0/1:4:87,0,4	0/1:13:40,0,15	1/1:12:63,9,0	1/1:18:113,15,0

        </p>
        <p>
          5	75003678	rs2307111	T	C	211	highCVG	AC1=6;AF1=0.7083;DB;DP=19;DP4=2,3,3,11;FQ=53.7;MQ=60;PV4=0.57,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=H36R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099271.1;VDB=0.0147;DN=100;DA=T/C;GM=NM_001099271.1,NM_152408.2,XM_005248435.1,XM_005248436.1,XM_005248437.1;GL=POC5;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ARG;PP=36/576,11/551,9/549,36/548,36/399;CDP=107,32,26,107,107;GS=29;PH=0;CP=0.937;CG=5.850;AA=C;HA=6.7;HE=38.0;HC=46.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=23;GESP=C:6453/T:5401;PAC=NP_001092741.1,NP_689621.2,XP_005248492.1,XP_005248493.1,XP_005248494.1	GT:GQ:PL	0/1:53:47,0,73	0/1:5:0,3,22	1/1:22:132,21,0	1/1:13:79,12,0

        </p>
        <p>
          5	78181477	rs1065757	C	T	31.80	midCVG	AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1	GT:GQ:PL	0/0:18:0,21,102	0/1:3:0,0,0	1/1:7:56,9,0	1/1:3:19,3,0

        </p>
        <p>
          5	111504479	rs7703522	C	T	58.60	midCVG	AC1=4;AF1=0.4913;DB;DP=11;DP4=0,5,2,3;FQ=36.6;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R630H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L4A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022140.3;VDB=0.0074;DN=116;DA=C/T;GM=NM_022140.3,XM_005272043.1,XM_005272044.1;GL=EPB41L4A;FG=missense;FD=unknown;AAC=ARG/HIS;PP=630/687,602/659,257/339;CDP=1889,1805,770;GS=29;PH=1;CP=0.728;CG=3.770;AA=C;HA=4.9;HE=35.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;GESP=T:8586/C:3412;PAC=NP_071423.3,XP_005272100.1,XP_005272101.1	GT:GQ:PL	0/0:10:0,12,66	0/1:3:0,0,0	1/1:6:68,9,0	0/1:14:33,0,10

        </p>
        <p>
          5	134782450	rs12520799	T	A	123	midCVG	AC1=4;AF1=0.5277;DB;DP=14;DP4=3,2,5,4;FQ=60;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=R117*;SNPEFF_CODON_CHANGE=Aga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5orf20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_130848.2;VDB=0.0030;DN=120;DA=T/A;GM=NM_130848.2;GL=C5orf20;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=117/245;CDP=349;CP=0.000;CG=-5.910;AA=A;HA=5.0;HE=48.3;HC=46.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3190;GESP=A:7784/T:5222;PAC=NP_570900.1	GT:GQ:PL	0/1:39:61,0,36	0/1:3:0,0,0	1/1:12:105,15,0	0/0:7:0,9,64

        </p>
        <p>
          5	140572763	rs2907323	C	G	71.10	midCVG	AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1	GT:GQ:PL	0/0:10:0,12,73	0/0:5:0,6,45	1/1:9:74,12,0	1/1:4:46,6,0

        </p>
        <p>
          5	145499996	rs10988	C	T	67.30	midCVG	AC1=5;AF1=0.6441;DB;DP=12;DP4=4,1,4,3;FQ=17.8;MQ=60;PV4=0.58,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=R1088K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020117.9;VDB=0.0086;DN=52;DA=C/T;GM=NM_020117.9,XM_005268458.1,XM_005268459.1,XM_005268460.1;GL=LARS;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=1088/1177,1061/1150,1042/1131,1034/1123;CDP=3263,3182,3125,3101;GS=26;PH=0.073;CP=1.000;CG=4.460;AA=C;HA=1.4;HE=29.1;HC=22.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa00290:Valine,LeucineAndIsoleucineBiosynthesis|hsa00970:Aminoacyl-tRNABiosynthesis;GESP=T:2696/C:10310;PAC=NP_064502.9,XP_005268515.1,XP_005268516.1,XP_005268517.1	GT:GQ:PL	0/1:13:39,0,11	0/1:3:0,0,0	1/1:8:63,9,0	0/1:10:5,0,42

        </p>
        <p>
          5	149006726	rs7732714	A	C	48.70	midCVG	AC1=4;AF1=0.5;DB;DP=10;DP4=1,4,2,3;FQ=50.5;MQ=60;PV4=1,0.047,1,0.025;SNPEFF_AMINO_ACID_CHANGE=S518R;SNPEFF_CODON_CHANGE=Agt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001669.2;VDB=0.0090;DN=116;DA=A/C;GM=NM_001001669.2,XM_005268447.1,XM_005268448.1;GL=ARHGEF37;FG=missense;FD=missense,unknown,unknown;AAC=SER/ARG;PP=518/676,518/676,304/462;CDP=1552,1552,910;GS=110;PH=0.906;CP=0.938;CG=3.930;AA=A;HA=32.4;HE=7.0;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=C:1584/A:11172;PAC=NP_001001669.2,XP_005268504.1,XP_005268505.1	GT:GQ:PL	0/0:5:0,6,47	0/1:3:0,0,0	1/1:4:44,6,0	0/1:46:45,0,47

        </p>
        <p>
          5	180030322	rs79620092	C	T	69.90	lowCVG	AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2	GT:GQ:PL	0/1:15:20,0,13	0/1:3:0,0,0	1/1:13:91,15,0	0/1:3:0,3,26

        </p>
        <p>
          5	180582604	rs2546423	A	G	65.20	midCVG	AC1=5;AF1=0.6414;DB;DP=10;DP4=3,0,2,4;FQ=12;MQ=60;PV4=0.17,0.39,1,0.2;SNPEFF_AMINO_ACID_CHANGE=H221R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2V2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206880.1;VDB=0.0064;DN=100;DA=A/G;GM=NM_206880.1;GL=OR2V2;FG=missense;FD=missense;AAC=HIS/ARG;PP=221/316;CDP=662;GS=29;PH=0;CP=0.001;CG=0.494;AA=G;HA=19.4;HE=41.3;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26085;KP=hsa04740:OlfactoryTransduction;GESP=G:8019/A:4987;PAC=NP_996763.1	GT:GQ:PL	0/1:11:8,0,13	0/0:4:0,6,40	1/1:6:43,6,0	1/1:8:60,9,0

        </p>
        <p>
          6	5431340	rs11243011	A	G	68.10	highCVG	AC1=4;AF1=0.5135;DB;DP=19;DP4=6,3,5,3;FQ=71.2;MQ=60;PV4=1,0.29,1,0.13;SNPEFF_AMINO_ACID_CHANGE=N280S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FARS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006567.3;VDB=0.0157;DN=120;DA=A/G;GM=NM_006567.3,XM_005248811.1,XM_005248812.1,XM_005248813.1;GL=FARS2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ASN/SER;PP=280/452,280/452,280/452,280/337;CDP=839;GS=46;PH=0.001;CP=1.000;CG=3.130;AA=A;HA=6.4;HE=15.1;HC=37.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;KP=hsa00970:Aminoacyl-tRNABiosynthesis;GESP=G:1979/A:11027;PAC=NP_006558.1,XP_005248868.1,XP_005248869.1,XP_005248870.1	GT:GQ:PL	0/1:3:0,0,0	0/0:10:0,12,63	1/1:15:94,18,0	0/1:21:18,0,80

        </p>
        <p>
          6	7211818	rs1334576	G	A	14	lowCVG	AC1=4;AF1=0.4604;DB;DP=7;DP4=0,4,2,1;FQ=15.2;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G195R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RREB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001003699.3;VDB=0.0098;DN=88;DA=G/A;GM=NM_001003698.3,NM_001003699.3,NM_001003700.1,NM_001168344.1,XM_005249272.1,XM_005249273.1,XM_005249274.1,XM_005249275.1,XM_005249276.1,XM_005249277.1;GL=RREB1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/ARG;PP=195/1688,195/1743,195/1477,195/1688,195/1743,195/1743,195/1743,195/1743,195/1743,195/1688;CDP=583;GS=125;PH=0.998;CP=0.039;CG=2.930;AA=G;HA=35.5;HE=45.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=A:4926/G:8080;PAC=NP_001003698.1,NP_001003699.1,NP_001003700.1,NP_001161816.1,XP_005249329.1,XP_005249330.1,XP_005249331.1,XP_005249332.1,XP_005249333.1,XP_005249334.1	GT:GQ:PL	0/1:4:21,3,0	0/0:6:0,6,32	1/1:4:37,6,0	0/0:6:0,6,30

        </p>
        <p>
          6	54173413	rs2297980	A	G	54.70	midCVG	AC1=5;AF1=0.5683;DB;DP=12;DP4=3,2,4,3;FQ=51.3;MQ=60;PV4=1,0.035,1,1;SNPEFF_AMINO_ACID_CHANGE=Q22R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0132;DN=100;DA=A/G;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=GLN/ARG;PP=22/477;CDP=65;GS=43;PH=0;CP=0.000;CG=-4.040;AA=A;HA=9.2;HE=10.5;HC=0.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=291;GESP=G:1365/A:11641;PAC=NP_055279.3	GT:GQ:PL	0/1:53:50,0,61	0/1:3:0,0,0	1/1:7:45,9,0	0/1:3:0,3,28

        </p>
        <p>
          6	54219326	rs3195579	G	A	24.40	midCVG	AC1=4;AF1=0.4386;DB;DP=11;DP4=6,1,2,1;FQ=26.9;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=R381H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0086;DN=105;DA=G/A/C;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=ARG/HIS;PP=381/477;CDP=1142;GS=29;PH=0;CP=0.594;CG=2.850;AA=G;HA=10.2;HE=17.7;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$PAX2_01;GESP=A:2158/G:10842;PAC=NP_055279.3	GT:GQ:PL	0/0:17:0,18,123	0/0:4:0,3,25	1/1:4:43,6,0	0/1:4:26,3,0

        </p>
        <p>
          6	97063522	rs9373985	C	G	49.80	highCVG	AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1	GT:GQ:PL	0/0:9:0,9,63	0/1:27:26,0,61	1/1:13:72,18,0	0/0:35:0,36,176

        </p>
        <p>
          6	117150008	rs6907580	G	A	108	highCVG	AC1=4;AF1=0.4941;DB;DP=17;DP4=2,6,5,3;FQ=107;MQ=60;PV4=0.31,0.089,1,0.23;SNPEFF_AMINO_ACID_CHANGE=R57*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0127;DN=116;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=stop-gained;FD=stop-gained,unknown,unknown;AAC=ARG/stop;PP=57/927,57/856,57/752;CDP=169;CP=0.862;CG=-2.840;AA=G;HA=11.5;HE=1.3;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=26;GESP=A:908/G:12098;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1	GT:GQ:PL	0/1:15:35,0,12	0/1:21:39,0,18	1/1:9:81,12,0	0/0:15:0,18,123

        </p>
        <p>
          6	155451352	rs931312	G	A	154	highCVG	AC1=5;AF1=0.6241;DB;DP=19;DP4=5,1,3,10;FQ=43.7;MQ=60;PV4=0.041,0.11,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R332H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TIAM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012454.3;VDB=0.0045;DN=86;DA=G/A;GM=NM_012454.3,XM_005266927.1;GL=TIAM2;FG=missense;FD=missense,unknown;AAC=ARG/HIS;PP=332/1702,332/1731;CDP=995;GS=29;PH=1;CP=0.427;CG=4.990;AA=G;HA=9.0;HE=27.2;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=200;KP=hsa04062:ChemokineSignalingPathway|hsa04810:RegulationOfActinCytoskeleton;GESP=A:2063/G:10943;PAC=NP_036586.2,XP_005266984.1	GT:GQ:PL	0/1:18:43,0,16	0/1:52:48,0,63	1/1:5:103,5,0	0/1:3:0,0,0

        </p>
        <p>
          7	2646796	rs61729413	G	A	20	lowCVG	AC1=4;AF1=0.5539;DB;DP=6;DP4=2,0,0,3;FQ=12.4;MQ=60;PV4=0.1,0.33,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R635K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IQCE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287499.1;VDB=0.0117;DN=129;DA=G/A;GM=NM_001100390.1,NM_152558.3,XM_005249675.1,XM_005249676.1,XM_005249677.1,XM_005249678.1;GL=IQCE;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=619/680,635/696,635/703,619/687,570/638,570/631;CDP=1856,1904,1904,1856,1709,1709;GS=26;PH=0;CP=0.000;CG=-3.560;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=A:1500/G:10556;PAC=NP_001093860.1,NP_689771.3,XP_005249732.1,XP_005249733.1,XP_005249734.1,XP_005249735.1	GT:GQ:PL	0/0:6:0,6,46	0/1:3:0,0,0	1/1:6:57,9,0	0/1:3:0,0,0

        </p>
        <p>
          7	4213975	rs671694	A	G	129	highCVG	AC1=5;AF1=0.6104;DB;DP=18;DP4=4,5,5,4;FQ=109;MQ=60;PV4=1,1,1,0.36;SNPEFF_AMINO_ACID_CHANGE=H1641R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152744.3;VDB=0.0088;DN=83;DA=A/G;GM=NM_001079653.1,NM_152744.3;GL=SDK1;FG=missense;FD=missense;AAC=HIS/ARG;PP=128/680,1641/2214;CDP=383,4922;GS=29;PH=0;CP=0.086;CG=4.910;AA=G;HA=0.0;HE=29.8;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=G:10231/A:2775;PAC=NP_001073121.1,NP_689957.3	GT:GQ:PL	0/1:37:32,0,78	0/0:8:0,12,76	1/1:6:47,6,0	1/1:14:99,15,0

        </p>
        <p>
          7	6656830	rs2243563	G	A	41.40	lowCVG	AC1=6;AF1=0.6978;DB;DP=7;DP4=0,2,3,1;FQ=-9.9;MQ=60;PV4=0.4,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=G8R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF853;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017560.1;VDB=0.0058;DN=100;DA=G/A;GM=NM_017560.1,XM_005249790.1;GL=ZNF853;FG=missense;FD=missense,unknown;AAC=GLY/ARG;PP=8/660,8/203;CDP=22;GS=125;PH=0.001;CP=0.000;CG=-7.700;AA=G;HA=14.0;HE=49.1;HC=26.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=A:2834/G:1732;PAC=NP_060030.1,XP_005249847.1	GT:GQ:PL	0/1:3:0,0,0	0/0:4:0,6,27	1/1:6:46,6,0	1/1:6:37,6,0

        </p>
        <p>
          7	13978809	rs9639168	T	C	137	highCVG	AC1=5;AF1=0.6375;DB;DP=18;DP4=3,1,5,8;FQ=21.4;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S100G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ETV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004956.4;VDB=0.0073;DN=119;DA=T/C;GM=NM_001163147.1,NM_001163148.1,NM_001163149.1,NM_001163150.1,NM_001163151.1,NM_001163152.1,NM_004956.4,XM_005249635.1,XM_005249636.1,XM_005249637.1,XM_005249638.1;GL=ETV1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/GLY;PP=100/455,82/460,82/460,60/438,42/420,60/375,100/478,100/478,100/478,60/438,42/420;CDP=298,244,244,178,124,178,298,298,298,178,124;GS=56;PH=0;CP=0.998;CG=4.570;AA=C;HA=11.6;HE=29.4;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=C:3493/T:8433;PPI=RFWD2:845;PAC=NP_001156619.1,NP_001156620.1,NP_001156621.1,NP_001156622.1,NP_001156623.1,NP_001156624.1,NP_004947.2,XP_005249692.1,XP_005249693.1,XP_005249694.1,XP_005249695.1	GT:GQ:PL	0/1:25:51,0,24	0/0:3:0,6,39	1/1:20:106,21,0	1/1:4:26,3,0

        </p>
        <p>
          7	72397435	.	A	G	6.99	lowCVG	AC1=5;AF1=0.625;DP=5;DP4=0,1,0,3;FQ=-8.69;MQ=50;PV4=1,0.036,0.14,1;SNPEFF_AMINO_ACID_CHANGE=K54R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POM121;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001257190.2;VDB=0.0004;GM=NM_001257190.2,NM_172020.4,XM_005250726.1,XM_005250727.1,XM_005250728.1,XM_005250729.1,XM_005250730.1,XM_005250731.1,XM_005250732.1;GL=POM121;FG=missense;FD=unknown;AAC=LYS/ARG;PP=54/1000,54/985,319/1250,319/1191,54/985,54/985,54/985,54/985,54/985;CDP=161,161,956,956,161,161,161,161,161;GS=26;PH=0.018;CP=0.966;AA=-;DSP=67;GESP=A:13006;PAC=NP_001244119.1,NP_742017.1,XP_005250783.1,XP_005250784.1,XP_005250785.1,XP_005250786.1,XP_005250787.1,XP_005250788.1,XP_005250789.1	GT:GQ:PL	0/1:3:0,0,0	0/0:4:0,3,24	1/1:5:42,9,0	0/1:3:0,0,0

        </p>
        <p>
          7	99306685	rs2257401	C	G	76	midCVG	AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:45,6,0	0/1:64:67,0,63

        </p>
        <p>
          7	107593989	rs20556	T	C	80.60	midCVG	AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1	GT:GQ:PL	0/1:17:41,0,14	0/1:3:0,0,0	1/1:13:82,15,0	0/0:4:0,6,52

        </p>
        <p>
          7	122635024	rs860170	C	T	171	highCVG	AC1=5;AF1=0.609;DB;DP=24;DP4=3,8,2,10;FQ=110;MQ=60;PV4=0.64,1,1,0.064;SNPEFF_AMINO_ACID_CHANGE=R222H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TAS2R16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016945.2;VDB=0.0143;DN=86;DA=C/T;GM=NM_016945.2;GL=TAS2R16;FG=missense;FD=missense;AAC=ARG/HIS;PP=222/292;CDP=665;GS=29;PH=0;CP=0.138;CG=2.210;AA=T;HA=3.2;HE=27.6;HC=47.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=108972;KP=hsa04742:TasteTransduction;GESP=T:9806/C:3200;PAC=NP_058641.1	GT:GQ:PL	0/1:13:69,0,12	0/0:25:0,30,144	1/1:14:84,15,0	1/1:8:68,9,0

        </p>
        <p>
          8	12879539	rs608052	A	G	159	highCVG	AC1=5;AF1=0.5689;DB;DP=16;DP4=3,2,4,7;FQ=62.2;MQ=60;PV4=0.6,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R451G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1456;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020844.2;VDB=0.0150;DN=83;DA=A/G;GM=NM_001099677.1,NM_020844.2,XM_005273584.1,XM_005273585.1,XM_005273586.1,XM_005273587.1,XM_005273588.1,XM_005273589.1,XM_005273590.1,XM_005273591.1,XM_005273592.1,XM_005273593.1,XM_005273594.1,XM_005273595.1;GL=KIAA1456;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLY;PP=325/329,451/455,520/524,471/475,451/455,420/424,420/424,415/419,415/419,415/419,415/419,364/368,325/329,364/368;CDP=973,1351,1558,1411,1351,1258,1258,1243,1243,1243,1243,1090,973,1090;GS=125;PH=0;CP=0.034;CG=0.894;AA=G;HA=45.0;HE=18.9;HC=25.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1023;GESP=G:8818/A:3100;PAC=NP_001093147.1,NP_065895.2,XP_005273641.1,XP_005273642.1,XP_005273643.1,XP_005273644.1,XP_005273645.1,XP_005273646.1,XP_005273647.1,XP_005273648.1,XP_005273649.1,XP_005273650.1,XP_005273651.1,XP_005273652.1	GT:GQ:PL	0/0:9:0,12,85	0/1:3:0,3,23	1/1:8:69,9,0	1/1:22:140,24,0

        </p>
        <p>
          8	32453358	rs3924999	G	A	31.20	lowCVG	AC1=6;AF1=0.7171;DB;DP=6;DP4=0,1,2,1;FQ=-15.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R38Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013956.3;VDB=0.0076;DN=108;DA=G/A;GM=NM_001159995.1,NM_001159999.1,NM_001160001.1,NM_001160002.1,NM_001160004.1,NM_001160005.1,NM_001160007.1,NM_001160008.1,NM_004495.3,NM_013956.3,NM_013957.3,NM_013958.3,NM_013960.3,NM_013962.2,NM_013964.3,XM_005273488.1;GL=NRG1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,unknown;AAC=ARG/GLN;PP=17/608,17/625,17/591,38/195,38/460,38/208,38/178,38/421,38/212,38/646,38/638,38/242,38/463,253/423,38/641,45/636;CDP=50,50,50,113,113,113,113,113,113,113,113,113,113,758,113,134;GS=43;PH=0.966;CP=0.998;CG=5.650;AA=G;HA=1.3;HE=43.2;HC=21.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=13;KP=hsa04012:ErbBSignalingPathway;GESP=A:3990/G:9016;PPI=ERBB3:989|ERBB4:983;PAC=NP_001153467.1,NP_001153471.1,NP_001153473.1,NP_001153474.1,NP_001153476.1,NP_001153477.1,NP_001153479.1,NP_001153480.1,NP_004486.2,NP_039250.2,NP_039251.2,NP_039252.2,NP_039254.1,NP_039256.2,NP_039258.1,XP_005273545.1	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,18	1/1:4:20,3,0	1/1:6:50,6,0

        </p>
        <p>
          8	48739305	rs4278157	G	A	137	midCVG	AC1=4;AF1=0.5579;DB;DP=14;DP4=2,2,4,6;FQ=14.3;MQ=60;PV4=1,1,1,1;VDB=0.0263;DN=111;DA=G/A;GM=NM_001081640.1,NM_006904.6;GL=PRKDC;FG=missense;FD=missense;AAC=ARG/CYS;PP=2899/4098,2899/4129;CDP=8694;GS=180;PH=0.966;CP=0.005;CG=4.670;AA=G;HA=0.0;HE=5.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=89;KP=hsa03450:Non-homologousEnd-joining|hsa04110:CellCycle;GESP=A:442/G:12052;PAC=NP_001075109.1,NP_008835.5	GT:GQ:PL	0/0:5:0,6,35	0/1:19:21,0,18	1/1:13:158,15,0	0/1:3:0,0,0

        </p>
        <p>
          8	53586706	rs17337252	A	G	65.40	midCVG	AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:103,9,0	0/0:8:0,9,75

        </p>
        <p>
          8	67380528	rs1060242	T	C	74.90	highCVG	AC1=5;AF1=0.62;DB;DP=19;DP4=3,3,4,4;FQ=64;MQ=60;PV4=1,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=C449R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADHFE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144650.2;VDB=0.0170;DN=86;DA=T/C;GM=NM_144650.2,XM_005251152.1,XM_005251153.1,XM_005251154.1,XM_005251155.1,XM_005251156.1,XM_005251157.1,XM_005251158.1,XM_005251159.1,XM_005251160.1,XM_005251161.1,XM_005251162.1,XM_005251163.1;GL=ADHFE1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=CYS/ARG;PP=449/468,430/449,401/420,401/420,401/420,401/420,401/420,379/398,401/420,219/238,219/238,219/238,219/238;CDP=1345,1288,1201,1201,1201,1201,1201,1135,1201,655,655,655,655;GS=180;PH=0;CP=0.818;CG=3.680;AA=C;HA=23.7;HE=42.6;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;GESP=C:7238/T:5768;PAC=NP_653251.2,XP_005251209.1,XP_005251210.1,XP_005251211.1,XP_005251212.1,XP_005251213.1,XP_005251214.1,XP_005251215.1,XP_005251216.1,XP_005251217.1,XP_005251218.1,XP_005251219.1,XP_005251220.1	GT:GQ:PL	0/0:8:0,12,80	0/1:21:17,0,29	1/1:8:35,9,0	1/1:11:72,12,0

        </p>
        <p>
          8	124449466	rs3824250	C	T	38.30	midCVG	AC1=4;AF1=0.4784;DB;DP=10;DP4=5,1,1,3;FQ=41.1;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R134C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDYHV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018024.2;VDB=0.0061;DN=107;DA=C/T;GM=NM_018024.1,XM_005250997.1,XM_005250998.1;GL=WDYHV1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/CYS;PP=134/206,74/146,66/138;CDP=400,220,196;GS=180;PH=0.99;CP=1.000;CG=5.480;AA=C;HA=34.8;HE=28.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;TFBS=V$AHRARNT_02,V$TCF11MAFG_01;GESP=T:4627/C:8379;PAC=NP_060494.1,XP_005251054.1,XP_005251055.1	GT:GQ:PL	0/1:16:14,0,41	0/0:7:0,9,51	1/1:4:47,6,0	0/1:17:22,0,14

        </p>
        <p>
          8	142367335	rs10875472	T	C	48.60	lowCVG	AC1=6;AF1=0.6934;DB;DP=6;DP4=1,0,3,2;FQ=-18.7;MQ=60;PV4=1,0.17,1,0.27;SNPEFF_AMINO_ACID_CHANGE=H230R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005293.2;VDB=0.0059;DN=120;DA=T/C;GM=NM_005293.2;GL=GPR20;FG=missense;FD=missense;AAC=HIS/ARG;PP=230/359;CDP=689;GS=29;PH=0;CP=0.989;CG=-0.037;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=713;CPG=142367051-142367860;GESP=C:10416/T:2188;PAC=NP_005284.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:4:21,3,0	0/1:13:62,0,11

        </p>
        <p>
          8	142517227	rs2613648	C	T	79.90	midCVG	AC1=5;AF1=0.6558;DB;DP=13;DP4=3,2,8,0;FQ=19.6;MQ=59;PV4=0.13,1,0.23,0.41;SNPEFF_AMINO_ACID_CHANGE=R8K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0030;DN=100;DA=C/T;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ARG/LYS,none,none;PP=8/1319,NA,NA;CDP=23,104,104;GS=26,NA,NA;CP=0.000;CG=-2.700;AA=C;HA=21.0;HE=49.1;HC=36.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=58;GESP=T:4996/C:7690;PAC=NP_997297.2,NA,NA	GT:GQ:PL	0/1:3:0,0,0	0/1:21:47,0,20	1/1:11:61,12,0	0/1:18:13,0,36

        </p>
        <p>
          9	26978259	rs11555693	A	G	90.20	highCVG	AC1=6;AF1=0.7286;DB;DP=22;DP4=0,6,4,5;FQ=48.8;MQ=60;PV4=0.1,0.013,1,0.18;SNPEFF_AMINO_ACID_CHANGE=K85R;SNPEFF_CODON_CHANGE=aAa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IFT74;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099222.1;VDB=0.0093;DN=120;DA=A/G;GM=NM_001099222.1,NM_001099223.1,NM_001099224.1,NM_025103.2;GL=IFT74;FG=missense;FD=missense;AAC=LYS/ARG;PP=85/601,85/601,85/373,85/601;CDP=254;GS=26;PH=0.459;CP=1.000;CG=5.620;AA=A;DV=by-frequency,by-cluster;DSP=3;GESP=G:735/A:11169;PAC=NP_001092692.1,NP_001092693.1,NP_001092694.1,NP_079379.2	GT:GQ:PL	0/1:78:77,0,80	0/1:3:0,0,0	1/1:9:50,9,0	0/1:3:0,0,0

        </p>
        <p>
          9	33385863	rs62542746	G	T	17.60	highCVG	AC1=5;AF1=0.6224;DB;DP=17;DP4=2,5,5,5;FQ=21.6;MQ=33;PV4=0.62,1,1.5e-05,0.36;SNPEFF_AMINO_ACID_CHANGE=A176E;SNPEFF_CODON_CHANGE=gCg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0114;DN=129;DA=G/T;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense-near-splice;FD=missense,unknown,unknown;AAC=ALA/GLU;PP=176/343,176/343,176/274;CDP=527;GS=107;PH=0.006;CP=0.040;CG=1.570;AA=T;DV=by-cluster;DSP=2;KP=hsa03320:PPARSignalingPathway;GESP=G:13002;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1	GT:GQ:PL	0/1:11:7,0,29	0/0:4:0,5,74	1/1:5:18,6,0	1/1:5:37,6,0

        </p>
        <p>
          9	33387047	rs4008658	T	G	66.90	highCVG	AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1	GT:GQ:PL	0/1:27:37,0,25	0/1:17:22,0,15	1/1:10:52,12,0	0/1:4:0,2,50

        </p>
        <p>
          9	88651370	rs2297002	T	C	82.10	midCVG	AC1=3;AF1=0.3754;DB;DP=12;DP4=5,1,4,2;FQ=82;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H217R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GOLM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016548.3;VDB=0.0035;DN=100;DA=T/C;GM=NM_016548.3,NM_177937.2,XM_005252040.1,XM_005252041.1,XM_005252042.1,XM_005252043.1,XM_005252044.1;GL=GOLM1;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=HIS/ARG;PP=217/402,217/402,217/459,217/459,217/403,217/385,217/385;CDP=650;GS=29;PH=0;CP=0.000;CG=-3.200;AA=C;HA=49.2;HE=2.5;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=C:1785/T:11221;PAC=NP_057632.2,NP_808800.1,XP_005252097.1,XP_005252098.1,XP_005252099.1,XP_005252100.1,XP_005252101.1	GT:GQ:PL	0/0:12:0,12,76	0/0:7:0,6,51	1/1:12:125,18,0	0/1:3:0,0,0

        </p>
        <p>
          9	104433235	rs10989589	C	T	82.80	highCVG	AC1=4;AF1=0.5477;DB;DP=16;DP4=2,2,4,3;FQ=34.3;MQ=60;PV4=1,0.22,1,0.16;SNPEFF_AMINO_ACID_CHANGE=G487R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRIN3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133445.2;VDB=0.0068;DN=120;DA=C/T;GM=NM_133445.2;GL=GRIN3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=487/1116;CDP=1459;GS=125;PH=0.956;CP=0.252;CG=5.760;AA=C;HA=10.0;HE=35.8;HC=20.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=155;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=T:4290/C:8716;PPI=GRIN1:846|CPEB1:830;PAC=NP_597702.2	GT:GQ:PL	0/0:5:0,6,49	0/1:3:0,0,0	1/1:10:83,12,0	0/1:27:42,0,24

        </p>
        <p>
          9	117050998	rs10982134	G	A	55.90	lowCVG	AC1=6;AF1=0.7419;DB;DP=6;DP4=1,0,3,1;FQ=-9.04;MQ=60;PV4=1,0.38,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R1354Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL27A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032888.2;VDB=0.0146;DN=120;DA=G/A;GM=NM_032888.2;GL=COL27A1;FG=missense;FD=missense;AAC=ARG/GLN;PP=1354/1861;CDP=4061;GS=43;PH=0.016;CP=0.999;CG=1.840;AA=G;HA=21.2;HE=25.9;HC=38.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=A:3908/G:9098;PAC=NP_116277.2	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,26	1/1:7:48,6,0	1/1:7:48,6,0

        </p>
        <p>
          9	117846580	rs1061494	T	C	25.30	midCVG	AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1	GT:GQ:PL	0/1:22:18,0,120	0/1:3:0,0,0	1/1:6:35,6,0	0/1:15:12,0,20

        </p>
        <p>
          9	125315557	rs1831370	T	C	80.20	highCVG	AC1=4;AF1=0.548;DB;DP=15;DP4=2,5,4,3;FQ=74.2;MQ=60;PV4=0.59,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W37R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1N2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004457.1;VDB=0.0200;DN=92;DA=T/C;GM=NM_001004457.1;GL=OR1N2;FG=missense;FD=missense;AAC=TRP/ARG;PP=37/331;CDP=109;GS=101;PH=0;CP=0.002;CG=-2.620;AA=C;HA=41.0;HE=45.5;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42490;KP=hsa04740:OlfactoryTransduction;GESP=C:6740/T:6266;PAC=NP_001004457.1	GT:GQ:PL	0/1:3:0,3,19	0/0:5:0,7,98	1/1:12:96,15,0	1/1:3:30,3,0

        </p>
        <p>
          9	125330125	rs10739614	C	G	163	highCVG	AC1=6;AF1=0.7385;DB;DP=20;DP4=0,5,11,3;FQ=29.2;MQ=60;PV4=0.0048,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=R211P;SNPEFF_CODON_CHANGE=cGt/cCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0083;DN=120;DA=C/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=ARG/PRO;PP=211/310;CDP=632;GS=103;PH=0;CP=0.008;CG=3.600;AA=G;HA=32.8;HE=45.0;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57058;KP=hsa04740:OlfactoryTransduction;GESP=G:8273/C:4733;PAC=NP_001004454.1	GT:GQ:PL	0/1:12:16,0,13	0/1:46:40,0,55	1/1:8:43,6,0	1/1:25:109,24,0

        </p>
        <p>
          9	125391241	rs1476860	G	A	91.10	midCVG	AC1=6;AF1=0.6853;DB;DP=9;DP4=2,1,1,5;FQ=19.3;MQ=60;PV4=0.23,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R192*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0081;DN=88;DA=G/A;GM=NM_001004450.1;GL=OR1B1;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=192/319;CDP=574;CP=0.142;CG=-6.050;AA=G;HA=3.8;HE=30.8;HC=46.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118174;KP=hsa04740:OlfactoryTransduction;GESP=A:2952/G:10054;PAC=NP_001004450.1	GT:GQ:PL	0/0:5:0,9,58	0/1:3:0,0,0	1/1:6:45,6,0	1/1:11:90,12,0

        </p>
        <p>
          9	140201604	rs28685844	C	T	15.90	lowCVG	AC1=4;AF1=0.4461;DB;DP=5;DP4=3,0,1,1;FQ=15;MQ=60;PV4=0.4,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R810Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017820.4;VDB=0.0130;DN=125;DA=C/T;GM=NM_017820.3,XM_005266091.1,XM_005266092.1;GL=EXD3;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=810/877,490/557,448/515;CDP=2429,1469,1343;GS=43;PH=0.193;CP=0.000;CG=-3.990;AA=C;HA=3.6;HE=0.4;HC=8.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=112;GESP=T:123/C:12191;PAC=NP_060290.3,XP_005266148.1,XP_005266149.1	GT:GQ:PL	0/1:3:0,0,0	0/0:9:0,9,51	1/1:3:52,6,0	0/1:3:0,0,0

        </p>
        <p>
          10	5415954	rs10904481	A	G	49.50	lowCVG	AC1=6;AF1=0.685;DB;DP=8;DP4=0,3,1,4;FQ=13.7;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R91G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053049.2;VDB=0.0100;DN=120;DA=A/G;GM=NM_053049.2;GL=UCN3;FG=missense;FD=missense;AAC=ARG/GLY;PP=91/162;CDP=271;GS=125;PH=0;CP=0.000;CG=2.520;AA=G;HA=29.2;HE=35.7;HC=48.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=277;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4033/A:8969;PAC=NP_444277.2	GT:GQ:PL	0/1:17:11,0,30	0/1:19:16,0,21	1/1:7:46,6,0	1/1:5:19,3,0

        </p>
        <p>
          10	69933969	rs7916821	G	A	187	highCVG	AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA	GT:GQ:PL	0/1:45:68,0,45	0/1:5:0,3,28	1/1:15:86,15,0	1/1:13:80,12,0

        </p>
        <p>
          10	95459817	rs726817	C	T	164	highCVG	AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1	GT:GQ:PL	0/1:34:33,0,36	0/1:13:61,0,12	1/1:14:98,15,0	0/1:20:17,0,24

        </p>
        <p>
          11	5528918	rs2227271	C	T	10.70	lowCVG	AC1=5;AF1=0.5785;DB;DP=5;DP4=0,1,2,0;FQ=-6.4;MQ=60;PV4=0.33,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=R624Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBQLN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017481.2;VDB=0.0061;DN=98;DA=C/T;GM=NM_017481.2;GL=UBQLN3;FG=missense;FD=missense;AAC=ARG/GLN;PP=624/656;CDP=1871;GS=43;PH=0.045;CP=0.213;CG=1.230;AA=C;HA=6.4;HE=37.7;HC=6.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1907;GESP=T:3709/C:9287;PAC=NP_059509.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:4:45,6,0	0/0:4:0,3,26

        </p>
        <p>
          11	5809673	rs10769224	C	T	24	highCVG	AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1	GT:GQ:PL	0/0:24:0,21,130	0/0:7:0,3,25	1/1:3:66,9,0	0/0:15:0,12,88

        </p>
        <p>
          11	5906205	rs4757987	G	A	63.50	midCVG	AC1=4;AF1=0.4597;DB;DP=12;DP4=3,4,2,3;FQ=66.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R228H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52E4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005165.1;VDB=0.0135;DN=111;DA=G/A;GM=NM_001005165.1;GL=OR52E4;FG=missense;FD=missense;AAC=ARG/HIS;PP=228/313;CDP=683;GS=29;PH=0.003;CP=0.023;CG=0.047;AA=G;HA=47.5;HC=42.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=64694;KP=hsa04740:OlfactoryTransduction;GESP=A:5225/G:7769;PAC=NP_001005165.1	GT:GQ:PL	0/0:6:0,6,44	0/1:4:25,3,0	1/1:8:86,12,0	0/0:13:0,15,95

        </p>
        <p>
          11	6007680	rs4436524	A	G	189	highCVG	AC1=6;AF1=0.7132;DB;DP=17;DP4=0,2,8,7;FQ=-4.85;MQ=47;PV4=0.47,0.4,0.081,0.3;SNPEFF_AMINO_ACID_CHANGE=C161R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0140;DN=111;DA=A/G/T;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=CYS/ARG;PP=161/330;CDP=481;GS=180;PH=0;CP=0.012;CG=0.241;AA=G;HA=43.3;HE=15.3;HC=11.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=166169;KP=hsa04740:OlfactoryTransduction;GESP=G:9200/A:3246;PAC=NP_001005173.2	GT:GQ:PL	0/1:16:39,0,17	0/1:5:0,3,21	1/1:13:53,12,0	1/1:28:144,27,0

        </p>
        <p>
          11	6341684	rs2682123	C	G	35.60	lowCVG	AC1=6;AF1=0.7558;DB;DP=5;DP4=0,1,2,2;FQ=-21.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R8P;SNPEFF_CODON_CHANGE=cGg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRKCDBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145040.2;VDB=0.0113;DN=100;DA=C/G;GM=NM_145040.2;GL=PRKCDBP;FG=missense;FD=missense;AAC=ARG/PRO;PP=8/262;CDP=23;GS=103;PH=0;CP=0.004;CG=2.010;AA=G;HA=13.6;HE=1.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=362;CPG=6340446-6341909;GESP=G:11875/C:705;PAC=NP_659477.2	GT:GQ:PL	0/1:10:28,0,9	0/1:3:0,0,0	1/1:6:43,6,0	0/1:3:0,0,0

        </p>
        <p>
          11	47270255	rs2167079	C	T	15.90	midCVG	AC1=3;AF1=0.4192;DB;DP=11;DP4=4,2,1,2;FQ=18.3;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R29Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001610.2;VDB=0.0055;DN=96;DA=C/T;GM=NM_001131064.1,NM_001251934.1,NM_001251935.1,NM_001610.2,XM_005252982.1,XM_005252983.1,XM_005252984.1,XM_005252985.1;GL=ACP2/NR1H3;FG=missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,5-prime-UTR,missense;FD=missense,utr-variant-5-prime,utr-variant-5-prime,missense,unknown,unknown,unknown,unknown;AAC=ARG/GLN,none,none,ARG/GLN,ARG/GLN,ARG/GLN,none,ARG/GLN;PP=29/161,NA,NA,29/424,29/392,29/361,NA,29/218;CDP=86,NA,NA,86,86,86,NA,86;GS=43,NA,NA,43,43,43,NA,43;PH=0.003;CP=0.987;CG=5.690;AA=T;HA=38.3;HE=31.2;HC=27.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;KP=hsa00740:RiboflavinMetabolism|hsa04142:Lysosome|hsa03320:PPARSignalingPathway;CPG=47270235-47270633;TFBS=V$AHR_01;GESP=T:4930/C:8048;PPI=RXRA:937|RXRB:900;PAC=NP_001124536.1,NA,NA,NP_001601.1,XP_005253039.1,XP_005253040.1,NA,XP_005253042.1	GT:GQ:PL	0/1:4:24,3,0	0/0:6:0,6,30	1/1:3:37,6,0	0/0:11:0,12,67

        </p>
        <p>
          11	55406524	rs61896975	C	T	78.80	highCVG	AC1=4;AF1=0.4977;DB;DP=15;DP4=2,3,1,9;FQ=54.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R231C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4P4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004124.1;VDB=0.0025;DN=129;DA=C/T;GM=NM_001004124.1;GL=OR4P4;FG=missense;FD=missense;AAC=ARG/CYS;PP=231/313;CDP=691;GS=180;PH=0.407;CP=0.000;CG=-1.540;AA=C;DG;DV=by-cluster,by-1000G;DSP=244337;KP=hsa04740:OlfactoryTransduction;GESP=T:719/C:11695;PAC=NP_001004124.1	GT:GQ:PL	0/1:3:0,0,0	0/0:5:0,6,55	1/1:4:102,6,0	0/1:19:17,0,37

        </p>
        <p>
          11	55541620	rs7124871	G	T	59.50	midCVG	AC1=4;AF1=0.4901;DB;DP=13;DP4=3,4,4,2;FQ=62.4;MQ=59;PV4=0.59,1,0.15,1;SNPEFF_AMINO_ACID_CHANGE=R236L;SNPEFF_CODON_CHANGE=cGc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5D13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001967.1;VDB=0.0105;DN=116;DA=G/A/T;GM=NM_001001967.1;GL=OR5D13;FG=missense;FD=missense;AAC=ARG/LEU;PP=236/315;CDP=707;GS=102;PH=0.01;CP=0.000;CG=2.850;AA=G;HA=0.0;HE=7.6;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=109241;KP=hsa04740:OlfactoryTransduction;GESP=T:576/G:12416;PAC=NP_001001967.1	GT:GQ:PL	0/1:3:0,0,0	0/0:13:0,15,100	1/1:9:74,12,0	0/1:31:29,0,37

        </p>
        <p>
          11	55703861	rs17597625	T	C	22.50	midCVG	AC1=4;AF1=0.508;DB;DP=12;DP4=3,1,4,0;FQ=25.4;MQ=60;PV4=1,0.019,1,0.18;SNPEFF_AMINO_ACID_CHANGE=R6G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006637.1;VDB=0.0044;DN=123;DA=T/C;GM=NM_006637.1;GL=OR5I1;FG=missense;FD=missense;AAC=ARG/GLY;PP=6/315;CDP=16;GS=125;PH=0;CP=0.238;CG=4.060;AA=C;HA=0.6;HE=10.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=43693;KP=hsa04740:OlfactoryTransduction;GESP=C:787/T:12195;PAC=NP_006628.1	GT:GQ:PL	0/0:8:0,9,62	0/0:3:0,3,22	1/1:4:30,6,0	1/1:4:38,6,0

        </p>
        <p>
          11	61165741	rs10897158	G	C	74.40	midCVG	AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1	GT:GQ:PL	0/1:10:35,0,8	0/0:6:0,9,70	1/1:4:24,3,0	1/1:8:61,9,0

        </p>
        <p>
          11	87013438	rs2276102	G	A	98	midCVG	AC1=6;AF1=0.7212;DB;DP=11;DP4=0,2,5,3;FQ=-16.6;MQ=60;PV4=0.44,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=G218R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM135;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022918.3;VDB=0.0208;DN=100;DA=G/A;GM=NM_001168724.1,NM_022918.3,NR_033149.1;GL=TMEM135;FG=missense,missense,non-coding-exon;FD=missense,missense,nc-transcript-variant;AAC=GLY/ARG,GLY/ARG,none;PP=196/437,218/459,NA;CDP=586,652,820;GS=125,125,NA;PH=0;CP=0.664;CG=4.310;AA=A;HA=43.6;HE=31.1;HC=46.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;GESP=A:8629/G:4371;PAC=NP_001162195.1,NP_075069.3,NA	GT:GQ:PL	0/1:14:15,0,14	0/1:3:0,0,0	1/1:5:26,3,0	0/1:3:93,0,0

        </p>
        <p>
          11	89531540	rs12417980	C	T	84.10	highCVG	AC1=6;AF1=0.7306;DB;DP=18;DP4=3,3,3,9;FQ=8.51;MQ=21;PV4=0.34,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G373R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM49;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020358.2;VDB=0.0085;DN=120;DA=C/T;GM=NM_020358.2;GL=TRIM49;FG=missense;FD=missense;AAC=GLY/ARG;PP=373/453;CDP=1117;GS=125;PH=0.834;CP=0.000;CG=-1.800;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=258;GESP=T:3992/C:8964;PAC=NP_065091.1	GT:GQ:PL	0/1:4:13,0,1	0/1:3:0,0,0	1/1:8:80,7,0	0/1:34:29,0,39

        </p>
        <p>
          11	124908352	rs7107487	T	C	999	highCVG	AC1=4;AF1=0.5057;DB;DP=29;DP4=6,9,4,10;FQ=205;MQ=60;PV4=0.7,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=C813R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025004.2;VDB=0.0034;DN=116;DA=T/C;GM=NM_025004.2;GL=CCDC15;FG=missense;FD=missense;AAC=CYS/ARG;PP=813/952;CDP=2437;GS=180;PH=0;CP=0.973;CG=3.110;AA=C;HA=36.7;HE=12.8;HC=17.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=C:2453/T:9283;PAC=NP_079280.2	GT:GQ:PL	0/1:48:45,0,74	0/1:63:60,0,82	1/1:18:134,21,0	0/0:12:0,15,94

        </p>
        <p>
          12	53605545	rs2229774	G	A	41.20	midCVG	AC1=4;AF1=0.5599;DB;DP=12;DP4=0,3,0,9;FQ=13.2;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=S427L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RARG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000966.5;VDB=0.0000;DN=98;DA=G/A;GM=NM_000966.5,NM_001042728.2,NM_001243730.1,NM_001243731.1,NM_001243732.1,XM_005269054.1,XM_005269055.1,XM_005269056.1,XM_005269057.1;GL=RARG;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/LEU;PP=427/455,416/444,355/383,306/334,405/433,514/542,514/542,427/455,427/455;CDP=1280,1247,1064,917,1214,1541,1541,1280,1280;GS=145;PH=0.525;CP=1.000;CG=4.530;AA=G;HA=11.1;HE=6.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=103;TFBS=V$ARP1_01;GESP=A:949/G:12057;PAC=NP_000957.1,NP_001036193.1,NP_001230659.1,NP_001230660.1,NP_001230661.1,XP_005269111.1,XP_005269112.1,XP_005269113.1,XP_005269114.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:24:81,27,0	0/0:7:0,9,49

        </p>
        <p>
          12	117383320	rs4076700	G	A	78.70	midCVG	AC1=5;AF1=0.6582;DB;DP=13;DP4=3,1,7,1;FQ=32.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R192Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXW8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153348.2;VDB=0.0037;DN=108;DA=G/A;GM=NM_012174.1,NM_153348.2,XM_005253866.1,XM_005253867.1,XM_005253868.1;GL=FBXW8;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=ARG/GLN,ARG/GLN,ARG/GLN,GLU/LYS,GLU/LYS;PP=126/533,192/599,192/423,41/418,41/418;CDP=377,575,575,121,121;GS=43,43,43,56,56;PH=1;CP=0.990;CG=5.190;AA=G;HA=22.8;HE=17.4;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=A:8670/G:4336;PPI=SKP1:994|CUL7:994|CUL1:932|RBX1:929;PAC=NP_036306.1,NP_699179.2,XP_005253923.1,XP_005253924.1,XP_005253925.1	GT:GQ:PL	0/1:23:18,0,39	0/1:3:0,0,0	1/1:17:93,18,0	0/1:14:9,0,30

        </p>
        <p>
          13	103384824	rs9300756	G	A	50.40	highCVG	AC1=3;AF1=0.3894;DB;DP=15;DP4=2,6,5,0;FQ=52.5;MQ=60;PV4=0.021,1,1,0.032;SNPEFF_AMINO_ACID_CHANGE=R6075C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC168;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146197.1;VDB=0.0090;DN=119;DA=G/A;GM=NM_001146197.1;GL=CCDC168;FG=missense;FD=missense;AAC=ARG/CYS;PP=6075/7082;CDP=18223;GS=180;PH=0;CP=0.000;CG=0.010;AA=G;HA=35.0;HE=41.0;HC=16.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17915;GESP=A:2470/G:2094;PAC=NP_001139669.1	GT:GQ:PL	0/0:14:0,15,100	0/0:6:0,6,34	1/1:8:75,12,0	0/1:20:22,0,19

        </p>
        <p>
          13	113479820	rs368865	A	G	144	highCVG	AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1	GT:GQ:PL	0/0:10:0,13,159	0/1:3:0,0,0	1/1:21:145,24,0	0/1:46:43,0,56

        </p>
        <p>
          14	20404736	rs2792146	G	A	143	highCVG	AC1=4;AF1=0.4995;DB;DP=17;DP4=3,1,8,3;FQ=42.3;MQ=60;PV4=1,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=R304H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004063.2;VDB=0.0150;DN=100;DA=G/A;GM=NM_001004063.2;GL=OR4K1;FG=missense;FD=missense;AAC=ARG/HIS;PP=304/312;CDP=911;GS=29;PH=0.003;CP=0.000;CG=-4.590;AA=G;HA=23.3;HC=30.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=222291;KP=hsa04740:OlfactoryTransduction;GESP=A:4807/G:8199;PAC=NP_001004063.2	GT:GQ:PL	0/1:34:39,0,31	0/0:3:0,3,28	1/1:21:149,24,0	0/0:3:0,3,25

        </p>
        <p>
          14	20482919	rs17308108	A	C	27.60	highCVG	AC1=2;AF1=0.2756;DB;DP=32;DP4=12,11,5,1;FQ=28.9;MQ=60;PV4=0.35,4.3e-07,1,0.024;SNPEFF_AMINO_ACID_CHANGE=L145R;SNPEFF_CODON_CHANGE=cTg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004712.1;VDB=0.0161;DN=123;DA=A/C;GM=NM_001004712.1;GL=OR4K14;FG=missense;FD=missense;AAC=LEU/ARG;PP=145/311;CDP=434;GS=102;PH=0.994;CP=0.198;CG=4.040;AA=A;HA=5.8;HE=10.2;HC=11.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=274987;KP=hsa04740:OlfactoryTransduction;GESP=C:894/A:12112;PAC=NP_001004712.1	GT:GQ:PL	0/0:44:0,42,160	0/0:6:0,3,19	1/1:11:73,18,0	0/0:26:0,24,111

        </p>
        <p>
          14	58681951	.	T	C	41	lowCVG	AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1	GT:GQ:PL	0/1:33:31,0,37	0/0:4:0,6,50	1/1:3:26,3,0	1/1:3:27,3,0

        </p>
        <p>
          14	92465749	rs35007347	C	T	31.40	highCVG	AC1=3;AF1=0.362;DB;DP=18;DP4=9,5,2,2;FQ=33.3;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=R1576H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIP11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004239.3;VDB=0.0065;DN=126;DA=C/T;GM=NM_004239.3,XM_005268213.1,XM_005268214.1,XM_005268215.1;GL=TRIP11;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/HIS;PP=1576/1980,1547/1951,1134/1538,566/970;CDP=4727,4640,3401,1697;GS=29;PH=1;CP=0.895;CG=5.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=T:254/C:12752;PAC=NP_004230.2,XP_005268270.1,XP_005268271.1,XP_005268272.1	GT:GQ:PL	0/0:30:0,30,154	0/1:36:36,0,57	1/1:3:39,6,0	0/0:5:0,3,21

        </p>
        <p>
          14	102815042	rs7011	C	T	107	midCVG	AC1=5;AF1=0.6413;DB;DP=13;DP4=0,5,3,5;FQ=6.84;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R164H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CINP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032630.2;VDB=0.0109;DN=52;DA=C/T;GM=NM_032630.2,XM_005267753.1;GL=CINP;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=ARG/HIS,none;PP=164/213,NA;CDP=491,NA;GS=29,NA;PH=0.054;CP=0.701;CG=2.760;AA=C;HA=5.8;HE=31.7;HC=11.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=T:3032/C:9974;PAC=NP_116019.1,NA	GT:GQ:PL	0/1:3:0,0,0	0/1:12:53,0,11	1/1:6:48,6,0	0/1:31:46,0,30

        </p>
        <p>
          14	105413204	rs2582514	G	T	89	midCVG	AC1=6;AF1=0.707;DB;DP=10;DP4=1,2,5,2;FQ=7.98;MQ=55;PV4=0.5,1,0.15,0.061;SNPEFF_AMINO_ACID_CHANGE=R2862S;SNPEFF_CODON_CHANGE=Cgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0059;DN=100;DA=G/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=2862/5796,2762/5696;CDP=8584,8284;GS=110;PH=0;CP=0.000;CG=0.525;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7933;GESP=T:7253/G:4947;PAC=NP_612429.2,XP_005267356.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:46,6,0	0/1:39:79,0,38

        </p>
        <p>
          15	29418520	rs2256273	C	T	23.10	lowCVG	AC1=5;AF1=0.5568;DB;DP=8;DP4=4,1,3,0;FQ=26.6;MQ=60;PV4=1,0.058,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R393H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM189A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015307.1;VDB=0.0105;DN=100;DA=C/T;GM=NM_015307.1,XM_005254261.1,XM_005254262.1;GL=FAM189A1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=393/540,430/577,252/399;CDP=1178,1289,755;GS=29;PH=0.005;CP=0.000;CG=-2.140;AA=C;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=51;TFBS=V$P300_01;GESP=T:3037/C:1529;PAC=NP_056122.1,XP_005254318.1,XP_005254319.1	GT:GQ:PL	0/1:3:22,3,0	0/1:3:0,0,1/1:5:43,6,0	0/0:12:0,15,102

        </p>
        <p>
          15	42154738	rs16972193	G	A	26.10	midCVG	AC1=5;AF1=0.5822;DB;DP=9;DP4=2,0,4,0;FQ=9.38;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R2462W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016642.3;VDB=0.0109;DN=123;DA=G/A;GM=NM_016642.3,XM_005254438.1;GL=SPTBN5;FG=missense-near-splice;FD=missense,unknown;AAC=ARG/TRP;PP=2462/3675,2482/3695;CDP=7384,7444;GS=101;PH=0.213;CP=0.837;CG=-4.960;AA=G;HE=13.8;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=A:1149/G:11369;PAC=NP_057726.4,XP_005254495.1	GT:GQ:PL	0/0:5:0,6,43	0/1:3:0,0,0	1/1:9:64,12,0	0/1:3:0,0,0

        </p>
        <p>
          15	42570718	rs8024732	A	G	106	highCVG	AC1=4;AF1=0.501;DB;DP=26;DP4=3,9,5,8;FQ=108;MQ=60;PV4=0.67,0.04,1,1;SNPEFF_AMINO_ACID_CHANGE=Q44R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GANC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198141.2;VDB=0.0051;DN=116;DA=A/G;GM=NM_198141.2,XM_005254286.1;GL=GANC;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=44/915,44/913;CDP=131;GS=43;PH=0;CP=0.454;CG=-2.850;AA=A;HA=31.5;HE=11.0;HC=11.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;KP=hsa00052:GalactoseMetabolism|hsa00500:StarchAndSucroseMetabolism|hsa01100:MetabolicPathways;GESP=G:9331/A:3673;PAC=NP_937784.2,XP_005254343.1	GT:GQ:PL	0/1:13:11,0,15	0/0:11:0,14,130	1/1:12:59,15,0	0/1:18:83,0,15

        </p>
        <p>
          15	65684513	rs148990036	C	T	29.40	lowCVG	AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,3,0;FQ=21.1;MQ=59;PV4=0.4,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R694Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0000;DN=134;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=ARG/GLN;PP=694/1251;CDP=2081;GS=43;PH=0.968;CP=0.939;CG=5.490;AA=C;DV=by-frequency,by-cluster;DSP=42;GESP=T:88/C:12878;PAC=NP_066013.1	GT:GQ:PL	0/0:9:0,9,56	0/1:3:0,0,0	1/1:6:67,9,0	0/1:3:0,0,0
        </p>
        <p>
          15	83215190	.	C	G	63.20	highCVG	AC1=2;AF1=0.2897;DP=17;DP4=3,4,0,8;FQ=62.9;MQ=60;PV4=0.077,1,1,1;VDB=0.0006;GM=NM_001079533.1,NM_001079534.1,NM_001079535.1,NM_030594.3,XM_005272441.1,XM_005272442.1,XM_005272443.1,XM_005272444.1,XM_005272445.1,XM_005272446.1;GL=CPEB1;FG=splice-donor;FD=unknown;CP=1.000;CG=4.850;AA=C;DSP=0;KP=hsa04114:OocyteMeiosis|hsa04320:Dorso-ventralAxisFormation|hsa04914:Progesterone-mediatedOocyteMaturation;TFBS=V$AREB6_02,V$SREBP1_01;GESP=C:12084;PPI=UBC:943|SPI1:845|GRIN3A:830|MRTO4:830;PAC=NP_001073001.1,NP_001073002.1,NP_001073003.1,NP_085097.3,XP_005272498.1,XP_005272499.1,XP_005272500.1,XP_005272501.1,XP_005272502.1,XP_005272503.1	GT:GQ:PL	0/0:8:0,6,37	0/0:6:0,3,27	1/1:6:106,12,0	0/0:11:0,9,45

        </p>
        <p>
          15	90174824	rs12900805	C	T	23.70	midCVG	AC1=6;AF1=0.8319;DB;DP=9;DP4=1,0,1,2;FQ=-24.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G1005R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198525.2;VDB=0.0106;DN=121;DA=C/T;GM=NM_198525.2,XM_005254902.1,XM_005254903.1,XM_005254904.1;GL=KIF7;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ARG;PP=1005/1344,1005/1344,1004/1343,1004/1343;CDP=3013,3013,3010,3010;GS=125;PH=0.004;CP=0.067;CG=-0.473;AA=C;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=99;TFBS=V$SREBP1_02;GESP=T:6862/C:6054;PAC=NP_940927.2,XP_005254959.1,XP_005254960.1,XP_005254961.1	GT:GQ:PL	0/1:3:0,0,0/1:6:29,0,4	1/1:4:28,3,0	0/1:3:0,0,0

        </p>
        <p>
          16	715990	rs7190775	G	A	37.20	lowCVG	AC1=5;AF1=0.6489;DB;DP=5;DP4=0,1,4,0;FQ=-10.8;MQ=60;PV4=0.2,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R1492H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR90;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145294.4;VDB=0.0001;DN=116;DA=G/A;GM=NM_145294.4,XM_005255160.1,XM_005255161.1;GL=WDR90;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=1492/1749,1495/1752,1494/1751;CDP=4475,4484,4481;GS=29;PH=0.169;CP=0.000;CG=-4.830;AA=G;HA=11.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;GESP=A:5864/G:6472;PAC=NP_660337.3,XP_005255217.1,XP_005255218.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,3,22	1/1:10:74,12,0	0/1:3:0,0,0

        </p>
        <p>
          16	840378	rs3765263	A	G	66.70	midCVG	AC1=3;AF1=0.3829;DB;DP=14;DP4=6,0,8,0;FQ=58.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K244R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHTF18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022092.2;VDB=0.0000;DN=107;DA=A/G;GM=NM_022092.2,XM_005255470.1,XM_005255471.1,XM_005255472.1;GL=CHTF18;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=LYS/ARG,LYS/ARG,LYS/ARG,none;PP=244/976,128/860,272/1004,NA;CDP=731,383,815,NA;GS=26,26,26,NA;PH=0;CP=0.081;CG=-5.390;AA=G;HA=33.2;HE=22.3;HC=42.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4000/A:8074;PPI=PCNA:939|RFC4:846|DSCC1:845|RFC3:845|RFC2:845;PAC=NP_071375.1,XP_005255527.1,XP_005255528.1,NA	GT:GQ:PL	0/0:12:0,12,60	0/1:3:0,0,0	1/1:19:110,24,0	0/0:7:0,6,40

        </p>
        <p>
          16	2905689	rs199712440	G	A	23.60	lowCVG	AC1=3;AF1=0.4317;DB;DP=7;DP4=2,1,2,1;FQ=25.1;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R149C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022119.3;VDB=0.0096;DN=137;DA=G/A;GM=NM_022119.3,XM_005255473.1;GL=PRSS22;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=149/318;CDP=445;GS=180;PH=0.99;CP=0.006;CG=3.430;AA=G;DV=by-cluster;DSP=115;TFBS=V$PAX4_01;GESP=A:1/G:12995;PAC=NP_071402.1,XP_005255530.1	GT:GQ:PL	0/0:4:0,3,24	0/0:4:0,3,29	1/1:4:50,6,0	0/1:16:15,0,19

        </p>
        <p>
          16	15111218	rs4985162	C	T	42.50	highCVG	AC1=5;AF1=0.5528;DB;DP=27;DP4=8,9,4,3;FQ=46;MQ=32;PV4=1,1,0.18,1;SNPEFF_AMINO_ACID_CHANGE=P301L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PDXDC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015027.3;VDB=0.0062;DN=111;DA=C/T;GM=NM_015027.2,XM_005255173.1,XM_005255174.1,XM_005255175.1,XM_005255176.1,XM_005255177.1,XM_005255178.1,XM_005255179.1,XM_005255180.1,XM_005255181.1;GL=PDXDC1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=301/789,319/807,310/798,300/788,286/774,286/774,282/770,274/762,273/761,210/698;CDP=902,956,929,899,857,857,845,821,818,629;GS=98;PH=0.015;CP=0.484;CG=4.490;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;TFBS=V$PPARA_01;GESP=T:2213/C:10775;PAC=NP_055842.2,XP_005255230.1,XP_005255231.1,XP_005255232.1,XP_005255233.1,XP_005255234.1,XP_005255235.1,XP_005255236.1,XP_005255237.1,XP_005255238.1	GT:GQ:PL	0/0:35:0,39,138	0/1:18:14,0,36	1/1:10:62,12,0	1/1:3:14,3,0

        </p>
        <p>
          16	67183799	.	A	G	7.55	lowCVG	AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2	GT:GQ:PL	0/0:5:0,3,20	0/0:5:0,3,18	1/1:4:45,9,0	0/0:3:0,0,0

        </p>
        <p>
          17	1690752	rs11549830	G	A	24.30	lowCVG	AC1=3;AF1=0.3495;DB;DP=8;DP4=2,3,3,0;FQ=26.1;MQ=60;PV4=0.2,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R562W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMYD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052928.2;VDB=0.0000;DN=120;DA=G/A;GM=NM_052928.2,XM_005256435.1,XM_005256436.1;GL=SMYD4;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=562/805,562/680,562/636;CDP=1684;GS=101;PH=0.997;CP=0.266;CG=2.540;AA=G;HA=45.0;HE=30.0;HC=45.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=37;GESP=A:4552/G:8438;PAC=NP_443160.2,XP_005256492.1,XP_005256493.1	GT:GQ:PL	0/0:13:0,12,73	0/0:3:0,0,0	1/1:4:64,9,0	0/0:5:0,3,21

        </p>
        <p>
          17	5058808	rs9899177	G	A	14.40	midCVG	AC1=4;AF1=0.4652;DB;DP=9;DP4=1,5,0,3;FQ=17.1;MQ=56;PV4=1,1,6.1e-05,0.17;SNPEFF_AMINO_ACID_CHANGE=R912Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004505.2;VDB=0.0065;DN=119;DA=G/A;GM=NM_004505.2,XM_005256845.1,XM_005256846.1;GL=USP6;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=912/1407,912/1407,595/1090;CDP=2735,2735,1784;GS=43;PH=0.999;CP=1.000;CG=2.910;AA=G;HA=15.2;HE=33.3;HC=10.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT1_01;GESP=A:6766/G:6240;PAC=NP_004496.2,XP_005256902.1,XP_005256903.1	GT:GQ:PL	0/0:8:0,9,57	0/1:3:0,0,0	1/1:4:41,6,0	0/1:15:14,0,40

        </p>
        <p>
          17	7951819	rs4792147	A	G	25	lowCVG	AC1=5;AF1=0.6327;DB;DP=8;DP4=2,1,3,1;FQ=27.7;MQ=60;PV4=1,0.054,1,1;SNPEFF_AMINO_ACID_CHANGE=Q656R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALOX15B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001141.2;VDB=0.0083;DN=111;DA=A/C/G;GM=NM_001039130.1,NM_001039131.1,NM_001141.2,XM_005256591.1;GL=ALOX15B;FG=missense;FD=missense,missense,missense,unknown;AAC=GLN/ARG;PP=627/648,582/603,656/677,644/665;CDP=1880,1745,1967,1931;GS=43;PH=0.001;CP=0.017;CG=-1.690;AA=G;HA=7.1;HE=47.0;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=116;KP=hsa00590:ArachidonicAcidMetabolism|hsa01100:MetabolicPathways;GESP=G:7866/A:5140;PAC=NP_001034219.1,NP_001034220.1,NP_001132.2,XP_005256648.1	GT:GQ:PL	0/0:6:0,9,70	0/1:3:0,0,0	1/1:8:52,9,0	1/1:3:15,3,0

        </p>
        <p>
          17	8216468	rs871841	T	C	58.40	lowCVG	AC1=5;AF1=0.6499;DB;DP=8;DP4=0,3,2,3;FQ=10.5;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L277P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025014.1;VDB=0.0003;DN=86;DA=T/C;GM=NM_025014.1,NM_173728.3,XM_005256536.1;GL=ARHGEF15;FG=missense;FD=missense,missense,unknown;AAC=LEU/PRO;PP=277/842,277/842,277/778;CDP=830;GS=98;PH=0;CP=0.995;CG=5.130;AA=C;HA=29.6;HE=47.3;HC=26.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=105;GESP=C:7328/T:5678;PAC=NP_079290.1,NP_776089.2,XP_005256593.1	GT:GQ:PL	0/0:6:0,9,48	0/1:3:0,0,0	1/1:11:83,12,0	1/1:4:18,3,0

        </p>
        <p>
          17	13980350	rs2072279	G	A	65.90	highCVG	AC1=4;AF1=0.4922;DB;DP=16;DP4=0,8,6,0;FQ=68.8;MQ=60;PV4=0.00033,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R159Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COX10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001303.3;VDB=0.0150;DN=96;DA=G/A;GM=NM_001303.3,XM_005256458.1,XM_005256459.1;GL=COX10;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=159/444,159/240,159/238;CDP=476;GS=43;PH=0.178;CP=0.605;CG=5.490;AA=G;HA=37.0;HE=44.2;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=24;KP=hsa00190:OxidativePhosphorylation|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa01100:MetabolicPathways;GESP=A:6378/G:6628;PAC=NP_001294.2,XP_005256515.1,XP_005256516.1	GT:GQ:PL	0/0:3:0,3,26	0/1:3:25,3,0	1/1:12:88,15,0	0/0:19:0,21,111

        </p>
        <p>
          17	19861458	rs2108978	C	T	98.40	highCVG	AC1=3;AF1=0.3904;DB;DP=18;DP4=7,4,1,6;FQ=101;MQ=60;PV4=0.066,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R249H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AKAP10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007202.3;VDB=0.0017;DN=96;DA=C/T;GM=NM_007202.3;GL=AKAP10;FG=missense;FD=missense;AAC=ARG/HIS;PP=249/663;CDP=746;GS=29;PH=0;CP=0.995;CG=2.360;AA=T;HA=44.6;HE=39.4;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=132;TFBS=V$GATA1_03;GESP=T:5787/C:7219;PPI=PDZK1:959|PRKAR1A:784;PAC=NP_009133.2	GT:GQ:PL	0/0:9:0,9,75	0/0:11:0,12,64	1/1:10:115,15,0	0/1:32:31,0,84

        </p>
        <p>
          17	38121993	rs3894194	G	A	35.30	lowCVG	AC1=6;AF1=0.7326;DB;DP=5;DP4=1,0,2,2;FQ=-12.4;MQ=60;PV4=1,0.21,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R18Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSDMA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178171.4;VDB=0.0024;DN=108;DA=G/A;GM=NM_178171.4;GL=GSDMA;FG=missense;FD=missense;AAC=ARG/GLN;PP=18/446;CDP=53;GS=43;PH=0.016;CP=1.000;CG=3.340;AA=G;HA=28.6;HE=47.7;HC=46.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=58;GESP=A:4966/G:7324;PAC=NP_835465.2	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,22	1/1:9:54,9,0	1/1:4:21,3,0

        </p>
        <p>
          17	41361960	rs8482	A	G	76.10	midCVG	AC1=4;AF1=0.5419;DB;DP=12;DP4=0,4,2,5;FQ=27.5;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H923R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NBR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005899.4;VDB=0.0215;DN=52;DA=A/G;GM=NM_005899.3,NM_031858.2,NM_031862.2;GL=NBR1;FG=missense;FD=missense;AAC=HIS/ARG;PP=923/967;CDP=2768;GS=29;PH=0.968;CP=1.000;CG=5.860;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;GESP=G:3148/A:7152;PPI=SQSTM1:994|MAP1LC3A:937|UBC:757|MAP1LC3C:756;PAC=NP_005890.2,NP_114064.1,NP_114068.1	GT:GQ:PL	0/1:9:6,0,31	0/1:3:0,0,0	1/1:15:112,18,0	0/0:5:0,6,35

        </p>
        <p>
          17	48629458	rs8065903	A	G	42.80	lowCVG	AC1=5;AF1=0.6395;DB;DP=8;DP4=3,0,3,2;FQ=9.67;MQ=60;PV4=0.46,1,1,0.071;SNPEFF_AMINO_ACID_CHANGE=K625R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022827.3;VDB=0.0000;DN=116;DA=A/G;GM=NM_001258372.1,NM_001258373.1,NM_022827.3,XM_005257591.1;GL=SPATA20;FG=missense;FD=missense,missense,missense,unknown;AAC=LYS/ARG;PP=609/787,565/743,625/803,609/787;CDP=1826,1694,1874,1826;GS=26;PH=0;CP=0.996;CG=1.610;AA=G;HA=0.4;HE=23.2;HC=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;TFBS=V$GR_Q6,V$OLF1_01;GESP=G:10556/A:2450;PAC=NP_001245301.1,NP_001245302.1,NP_073738.2,XP_005257648.1	GT:GQ:PL	0/1:4:0,3,20	0/1:12:7,0,28	1/1:8:57,9,0	1/1:4:22,3,0

        </p>
        <p>
          17	72346868	rs2382644	C	T	25.90	lowCVG	AC1=5;AF1=0.6043;DB;DP=8;DP4=1,2,1,2;FQ=24.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R471W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153209.3;VDB=0.0128;DN=100;DA=C/T;GM=NM_153209.3,XM_005257029.1,XM_005257030.1;GL=KIF19;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=471/999,471/1000,429/958;CDP=1411,1411,1285;GS=101;PH=0.999;CP=0.000;CG=-11.500;DG;DV=by-frequency,by-cluster,by-1000G;DSP=23;TFBS=V$HOX13_01;GESP=T:1007/C:11997;PAC=NP_694941.2,XP_005257086.1,XP_005257087.1	GT:GQ:PL	0/1:3:0,0,0	0/0:7:0,9,63	1/1:3:17,3,0	1/1:5:50,6,0

        </p>
        <p>
          18	72103782	rs2278154	T	C	999	highCVG	AC1=5;AF1=0.603;DB;DP=20;DP4=5,1,8,6;FQ=35.4;MQ=60;PV4=0.35,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q405R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM69C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001044369.2;VDB=0.0039;DN=100;DA=T/C;GM=NM_001044369.2,XM_005266647.1;GL=FAM69C;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=405/420,264/279;CDP=1214,791;GS=43;PH=0;CP=0.021;CG=4.080;AA=C;HA=27.7;HE=36.4;HC=29.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=173;GESP=C:4387/T:7701;PAC=NP_001037834.2,XP_005266704.1	GT:GQ:PL	0/1:3:0,0,0	0/1:53:53,0,54	1/1:28:173,30,0	0/1:4:0,3,19

        </p>
        <p>
          19	1068738	rs1801284	G	A	31.20	lowCVG	AC1=6;AF1=0.749;DB;DP=5;DP4=0,1,2,2;FQ=-19.7;MQ=60;PV4=1,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R155H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HMHA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001258328.2;VDB=0.0157;DN=89;DA=G/A;GM=NM_001258328.1,NM_012292.3;GL=HMHA1;FG=missense;FD=missense;AAC=ARG/HIS;PP=155/1153,139/1137;CDP=464,416;GS=29;PH=0.008;CP=0.000;CG=-8.440;AA=A;HA=45.1;HE=34.4;HC=41.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;CPG=1068439-1068764;GESP=A:5253/G:7735;PAC=NP_001245257.1,NP_036424.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:9:60,9,0	0/1:10:7,0,11

        </p>
        <p>
          19	6374890	rs7540	G	A	55.70	midCVG	AC1=4;AF1=0.4563;DB;DP=10;DP4=5,0,3,1;FQ=47.1;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R191Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALKBH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032306.3;VDB=0.0086;DN=52;DA=G/A;GM=NM_032306.3,XM_005259658.1;GL=ALKBH7;FG=missense;FD=missense,unknown;AAC=ARG/GLN;PP=191/222,165/196;CDP=572,494;GS=43;PH=0.275;CP=0.021;CG=3.030;AA=G;HA=0.0;HE=10.3;HC=2.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=69;TFBS=V$OLF1_01;GESP=A:862/G:12144;PAC=NP_115682.1,XP_005259715.1	GT:GQ:PL	0/1:18:43,0,14	0/1:3:0,0,0	1/1:4:54,6,0	0/0:11:0,12,72

        </p>
        <p>
          19	8997482	.	C	T	52.20	highCVG	AC1=5;AF1=0.5614;DP=30;DP4=3,9,6,12;FQ=55.8;MQ=39;PV4=0.7,1,3.5e-17,1;SNPEFF_AMINO_ACID_CHANGE=R13647Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0008;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=ARG/GLN;PP=13647/14508,5561/6405,5561/6025,442/1303;CDP=40940,16682,16682,1325;GS=43;PH=0.984;CP=0.000;CG=-5.720;AA=C;DSP=55;GESP=C:12276;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1	GT:GQ:PL	0/1:3:0,3,59	0/1:17:15,0,19	1/1:13:69,15,0	0/1:17:13,0,77

        </p>
        <p>
          19	10112346	rs2287803	T	C	55.50	lowCVG	AC1=5;AF1=0.647;DB;DP=8;DP4=3,0,0,5;FQ=23;MQ=60;PV4=0.018,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R322G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL5A3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015719.3;VDB=0.0071;DN=100;DA=T/C;GM=NM_015719.3;GL=COL5A3;FG=missense-near-splice;FD=missense;AAC=ARG/GLY;PP=322/1746;CDP=964;GS=125;PH=0;CP=0.002;CG=2.170;AA=C;HA=3.2;HE=2.3;HC=13.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:12695/T:311;PPI=COL5A1:999;PAC=NP_056534.2	GT:GQ:PL	0/1:17:20,0,16	0/1:17:12,0,43	1/1:9:64,9,0	0/1:3:0,0,0

        </p>
        <p>
          19	15587185	rs733731	C	T	22.30	lowCVG	AC1=5;AF1=0.6333;DB;DP=6;DP4=1,1,4,0;FQ=-3.17;MQ=60;PV4=0.33,0.097,1,1;SNPEFF_AMINO_ACID_CHANGE=R99Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PGLYRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052890.3;VDB=0.0069;DN=86;DA=C/T;GM=NM_052890.3;GL=PGLYRP2;FG=missense;FD=missense;AAC=ARG/GLN;PP=99/577;CDP=296;GS=43;PH=0.03;CP=0.001;CG=-5.850;AA=C;HA=34.3;HE=39.5;HC=38.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=235;GESP=T:4832/C:8174;PAC=NP_443122.3	GT:GQ:PL	0/1:14:15,0,14	0/1:4:0,3,24	1/1:4:19,3,0	1/1:6:31,6,0

        </p>
        <p>
          19	15852363	rs11670007	G	A	21	lowCVG	AC1=3;AF1=0.3936;DB;DP=6;DP4=1,2,0,3;FQ=22.5;MQ=58;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R54H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10H3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013938.1;VDB=0.0067;DN=120;DA=G/A;GM=NM_013938.1;GL=OR10H3;FG=missense;FD=missense;AAC=ARG/HIS;PP=54/317;CDP=161;GS=29;PH=0.001;CP=0.225;CG=-4.700;AA=G;HA=43.3;HE=31.4;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18868;KP=hsa04740:OlfactoryTransduction;GESP=A:4268/G:8738;PAC=NP_039226.1	GT:GQ:PL	0/0:5:0,3,26	0/0:3:0,0,0	1/1:5:60,9,0	0/0:7:0,6,43

        </p>
        <p>
          19	17170885	rs1130222	C	T	99.90	highCVG	AC1=5;AF1=0.5805;DB;DP=17;DP4=4,4,3,5;FQ=74.7;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G83R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033417.1;VDB=0.0107;DN=86;DA=C/A/G/T;GM=NM_001011699.1,NM_033417.1,XM_005260154.1;GL=HAUS8;FG=missense,missense,5-prime-UTR;FD=missense/stop-gained,missense/stop-gained,unknown;AAC=GLY/ARG,GLY/ARG,none;PP=82/410,83/411,NA;CDP=244,247,NA;GS=125,125,NA;PH=0.062;CP=0.001;CG=1.070;AA=C;HA=45.0;HE=33.3;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;GESP=T:5621/C:7385;PPI=HAUS1:999|HAUS6:994|HAUS2:987|HAUS4:987;PAC=NP_001011699.1,NP_219485.1,NA	GT:GQ:PL	0/1:46:42,0,63	0/0:9:0,12,55	1/1:4:28,3,0	1/1:10:77,12,0

        </p>
        <p>
          19	36303664	rs3848666	G	C	17.50	lowCVG	AC1=4;AF1=0.5;DB;DP=8;DP4=2,1,2,1;FQ=20;MQ=60;PV4=1,0.19,1,0.28;SNPEFF_AMINO_ACID_CHANGE=P91R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRODH2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021232.1;VDB=0.0075;DN=108;DA=G/C;GM=NM_021232.1,XM_005259123.1;GL=PRODH2;FG=missense;FD=missense,unknown;AAC=PRO/ARG;PP=91/537,15/379;CDP=272,44;GS=103;PH=0.004;CP=0.000;CG=0.647;AA=A;HA=0.8;HE=14.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=131;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;GESP=C:1572/G:11434;PAC=NP_067055.1,XP_005259180.1	GT:GQ:PL	0/0:9:0,9,65	0/1:3:0,0,0	1/1:5:55,9,0	0/1:3:0,0,0

        </p>
        <p>
          19	37488481	rs1363753	G	C	93.70	midCVG	AC1=4;AF1=0.5436;DB;DP=10;DP4=1,2,1,6;FQ=6.29;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G630R;SNPEFF_CODON_CHANGE=Ggt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0155;DN=88;DA=G/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=GLY/ARG;PP=630/636,566/572;CDP=1888,1696;GS=125;PH=0;CP=0.022;CG=1.640;AA=C;HA=29.2;HE=49.2;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1263;PAC=NP_001191767.1,NP_001191768.1	GT:GQ:PL	0/0:3:0,3,20	0/1:3:0,0,0	1/1:5:48,6,0	0/1:24:85,0,21

        </p>
        <p>
          19	37488499	rs1667366	T	C	105	midCVG	AC1=6;AF1=0.6771;DB;DP=10;DP4=1,1,1,7;FQ=-10.4;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*636R;SNPEFF_CODON_CHANGE=Tga/Cga;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0136;DN=89;DA=T/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=stop-lost;FD=stop-lost;AAC=stop/ARG;PP=636/636,572/572;CDP=1906,1714;CP=0.084;CG=2.660;AA=C;HA=43.3;HE=48.3;HC=26.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1281;PAC=NP_001191767.1,NP_001191768.1	GT:GQ:PL	0/1:4:0,3,23	0/1:3:0,0,0	1/1:6:47,6,0	0/1:3:96,1,0

        </p>
        <p>
          19	41035024	.	C	T	33.50	midCVG	AC1=4;AF1=0.5209;DP=10;DP4=0,3,5,0;FQ=22.5;MQ=60;PV4=0.018,0.066,1,0.086;SNPEFF_AMINO_ACID_CHANGE=R1312*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_020971.2;VDB=0.0182;GM=NM_020971.2,XM_005259112.1,XM_005259113.1;GL=SPTBN4;FG=stop-gained;FD=unknown;AAC=ARG/stop;PP=1312/2565,1307/2560,55/1308;CDP=3934,3919,163;CP=1.000;CG=2.100;AA=C;DSP=15;GESP=C:13006;PPI=DISC1:789;PAC=NP_066022.2,XP_005259169.1,XP_005259170.1	GT:GQ:PL	0/1:11:8,0,21	0/0:3:0,3,28	1/1:5:45,6,0	0/1:16:23,0,13

        </p>
        <p>
          19	45716364	rs189063316	C	T	11.60	lowCVG	AC1=5;AF1=0.5837;DB;DP=5;DP4=1,0,2,0;FQ=-12.4;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R398Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXOC3L2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138568.3;VDB=0.0116;DN=135;DA=C/T;GM=NM_138568.3;GL=EXOC3L2;FG=missense;FD=missense;AAC=ARG/GLN;PP=398/410;CDP=1193;GS=43;PH=0.46;CP=0.112;CG=1.260;AA=C;DV=by-cluster;DSP=252;GESP=T:121/C:12673;PAC=NP_612635.3	GT:GQ:PL	0/1:3:0,0,0/0:4:0,3,19	1/1:4:46,6,0	0/1:3:0,0,0

        </p>
        <p>
          19	49377319	rs610308	A	G	17.80	midCVG	AC1=3;AF1=0.4136;DB;DP=11;DP4=3,1,0,3;FQ=20;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K277E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R15A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014330.3;VDB=0.0071;DN=83;DA=A/G;GM=NM_014330.3;GL=PPP1R15A;FG=missense;FD=missense;AAC=LYS/GLU;PP=277/675;CDP=829;GS=56;PH=0;CP=0.000;CG=-3.110;AA=G;HA=22.5;HE=32.3;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=837;GESP=G:6297/A:6709;PPI=PPP1CA:942|PPP1CC:937|SMARCB1:846|PPP1CB:845;PAC=NP_055145.3	GT:GQ:PL	0/0:6:0,6,46	0/0:4:0,3,24	1/1:3:39,6,0	0/1:13:21,0,8

        </p>
        <p>
          19	52496149	rs16983353	C	T	125	highCVG	AC1=5;AF1=0.6541;DB;DP=17;DP4=1,5,4,6;FQ=50.5;MQ=59;PV4=0.59,1,0.23,0.15;SNPEFF_AMINO_ACID_CHANGE=R738K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF615;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199324.1;VDB=0.0050;DN=123;DA=C/T;GM=NM_001199324.1,NM_198480.3,XM_005258801.1,XM_005258802.1,XM_005258803.1,XM_005258804.1,XM_005258805.1,XM_005258806.1;GL=ZNF615;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=738/743,727/732,743/748,738/743,732/737,727/732,619/624,619/624;CDP=2213,2180,2228,2213,2195,2180,1856,1856;GS=26;PH=0;CP=0.003;CG=2.500;AA=T;HA=35.3;HE=28.2;HC=33.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1942;GESP=T:4942/C:8064;PAC=NP_001186253.1,NP_940882.3,XP_005258858.1,XP_005258859.1,XP_005258860.1,XP_005258861.1,XP_005258862.1,XP_005258863.1	GT:GQ:PL	0/1:18:13,0,48	0/1:3:0,0,0	1/1:11:88,12,0	0/1:40:65,0,39

        </p>
        <p>
          19	52888245	rs55748277	A	G	23.70	midCVG	AC1=4;AF1=0.4732;DB;DP=11;DP4=3,3,2,3;FQ=26.4;MQ=59;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=K471R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF880;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145434.1;VDB=0.0110;DN=129;DA=A/G;GM=NM_001145434.1;GL=ZNF880;FG=missense;FD=missense;AAC=LYS/ARG;PP=471/578;CDP=1412;GS=26;PH=0.991;CP=0.007;CG=-0.460;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1144;GESP=G:1745/A:2821;PAC=NP_001138906.1	GT:GQ:PL	0/1:6:3,0,37	0/1:13:24,0,9	1/1:4:39,6,0	0/0:6:0,6,45

        </p>
        <p>
          19	55106778	rs10418391	G	A	52.20	lowCVG	AC1=6;AF1=0.7307;DB;DP=6;DP4=0,1,0,4;FQ=-9.7;MQ=55;PV4=1,1,0.34,1;SNPEFF_AMINO_ACID_CHANGE=R191H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006863.3;VDB=0.0197;DN=119;DA=G/A;GM=NM_001278318.1,NM_001278319.1,NM_006863.3,NR_103501.1,NR_103502.1,NR_103503.1;GL=LILRA1;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=unknown,unknown,missense,unknown,unknown,unknown;AAC=ARG/HIS,ARG/HIS,ARG/HIS,none,none,none;PP=191/290,191/440,191/490,NA,NA,NA;CDP=572,572,572,744,988,744;GS=29,29,29,NA,NA,NA;PH=0.042;CP=0.000;CG=-0.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=90;GESP=A:2052/G:10954;PAC=NP_001265247.1,NP_001265248.1,NP_006854.1,NA,NA,NA	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,25	1/1:4:27,3,0	1/1:9:65,9,0

        </p>
        <p>
          19	55494157	rs4306647	G	A	14.20	highCVG	AC1=3;AF1=0.3892;DB;DP=15;DP4=6,3,3,1;FQ=16.2;MQ=41;PV4=1,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R364K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001174081.1;VDB=0.0098;DN=111;DA=G/A;GM=NM_001174081.1,NM_001174082.1,NM_001174083.1,NM_017852.3,XM_005259050.1;GL=NLRP2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/LYS;PP=364/1063,342/1041,341/1040,364/1063,361/1060;CDP=1091,1025,1022,1091,1082;GS=26;PH=0.363;CP=0.019;CG=1.640;AA=G;HA=19.6;HE=2.2;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=628;GESP=A:915/G:12091;PAC=NP_001167552.1,NP_001167553.1,NP_001167554.1,NP_060322.1,XP_005259107.1	GT:GQ:PL	0/0:5:0,3,19	0/1:11:10,0,32	1/1:5:48,9,0	0/0:12:0,12,50

        </p>
        <p>
          19	59023166	rs34415062	G	A	24	lowCVG	AC1=4;AF1=0.5357;DB;DP=7;DP4=2,1,2,1;FQ=25.4;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=R53W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=G/A;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=53/691,53/607,53/396;CDP=157;GS=101;PH=0;CP=0.000;CG=-8.620;AA=A;HA=22.8;HE=22.2;HC=5.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=280;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=A:2195/G:10597;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1	GT:GQ:PL	0/1:3:0,0,0	0/0:3:0,3,26	1/1:4:50,6,0	0/1:16:13,0,40

        </p>
        <p>
          19	59023174	rs35350976	A	G	35.70	lowCVG	AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1	GT:GQ:PL	0/1:3:0,0,0	0/0:3:0,3,33	1/1:6:69,9,0	0/1:9:6,0,61

        </p>
        <p>
          20	1210647	rs450739	T	C	57.60	lowCVG	AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1	GT:GQ:PL	0/1:18:17,0,20	0/1:3:0,0,0	1/1:12:62,12,0	0/1:8:19,0,7

        </p>
        <p>
          20	33874720	rs2425049	C	T	49.50	midCVG	AC1=3;AF1=0.4207;DB;DP=14;DP4=1,6,0,5;FQ=51.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R621Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM83C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178468.5;VDB=0.0155;DN=100;DA=C/T;GM=NM_178468.5;GL=FAM83C;FG=missense;FD=missense;AAC=ARG/GLN;PP=621/748;CDP=1862;GS=43;PH=0;CP=0.334;CG=2.850;AA=T;HA=26.5;HE=21.9;HC=15.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1056;GESP=T:4644/C:8362;PAC=NP_848563.1	GT:GQ:PL	0/0:4:0,3,23	0/0:14:0,15,87	1/1:8:76,12,0	0/1:12:19,0,8

        </p>
        <p>
          21	34907047	.	T	C	80.50	highCVG	AC1=3;AF1=0.4181;DP=25;DP4=16,1,5,2;FQ=82.8;MQ=59;PV4=0.19,0.071,1,0.48;SNPEFF_AMINO_ACID_CHANGE=N85S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GART;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000819.4;VDB=0.0037;GM=NM_000819.4,NM_001136005.1,NM_001136006.1,NM_175085.2,XM_005260941.1;GL=GART;FG=missense;FD=unknown;AAC=ASN/SER;PP=85/1011,85/1011,85/1011,85/434,85/1011;CDP=254;GS=46;PH=0;CP=0.763;CG=2.160;AA=T;DSP=13;KP=hsa00230:PurineMetabolism|hsa00670:OneCarbonPoolByFolate|hsa01100:MetabolicPathways;GESP=T:13006;PPI=UBC:730;PAC=NP_000810.1,NP_001129477.1,NP_001129478.1,NP_780294.1,XP_005260998.1	GT:GQ:PL	0/1:32:31,0,53	0/0:38:0,39,130	1/1:11:96,15,0	0/0:4:0,3,18

        </p>
        <p>
          21	47851753	rs2073376	A	G	39.40	lowCVG	AC1=6;AF1=0.6957;DB;DP=6;DP4=1,1,1,3;FQ=10.9;MQ=60;PV4=1,0.27,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q2792R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0093;DN=96;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=2792/3337,2803/3348,2802/3347,2803/3269,2803/3259,2685/3230,2674/3140;CDP=8375,8408,8405,8408,8408,8054,8021;GS=43;PH=0;CP=0.314;CG=2.320;AA=G;HA=15.0;HE=33.5;HC=31.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=140;GESP=G:8998/A:4008;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1	GT:GQ:PL	0/0:4:0,6,47	0/1:3:0,0,0	1/1:6:34,6,0	1/1:6:47,6,0

        </p>
        <p>
          22	29913272	rs737976	C	T	154	midCVG	AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,25	1/1:14:103,15,0	0/1:5:91,0,2

        </p>
        <p>
          22	32586875	rs136468	G	A	7	midCVG	AC1=3;AF1=0.3215;DB;DP=12;DP4=0,7,4,1;FQ=8.3;MQ=41;PV4=0.01,1,0.0013,0.38;SNPEFF_AMINO_ACID_CHANGE=R341C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=G/A;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/CYS;PP=341/379,251/289,251/289,280/318,251/289;CDP=1021,751,751,838,751;GS=180;PH=0.999;CP=0.036;CG=0.582;AA=G;HA=1.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=465;GESP=G:12936;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1	GT:GQ:PL	0/0:6:0,3,27	0/0:5:0,1,22	1/1:5:48,12,0	0/0:17:0,15,80

        </p>
        <p>
          22	50728062	rs28379706	T	C	62.50	midCVG	AC1=4;AF1=0.5564;DB;DP=11;DP4=1,2,6,0;FQ=26.1;MQ=60;PV4=0.083,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K318E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012401.3;VDB=0.0214;DN=125;DA=T/C;GM=NM_012401.3,XM_005261908.1,XM_005261909.1,XM_005261910.1,XM_005261911.1;GL=PLXNB2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=318/1839,400/1921,318/1839,318/1839,318/1839;CDP=952,1198,952,952,952;GS=56;PH=0;CP=0.001;CG=-1.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=117;KP=hsa04360:AxonGuidance;CPG=50727442-50728654;GESP=C:5524/T:6698;PPI=TUBA3E:962|MYC:846|ARHGEF11:845;PAC=NP_036533.2,XP_005261965.1,XP_005261966.1,XP_005261967.1,XP_005261968.1	GT:GQ:PL	0/1:3:0,3,24	0/0:5:0,6,44	1/1:3:21,3,0	1/1:13:87,15,0

        </p>
        <p>
          22	51009953	rs470117	C	T	122	highCVG	AC1=4;AF1=0.5;DB;DP=21;DP4=5,3,7,6;FQ=74.8;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=E531K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145135.1;VDB=0.0103;DN=80;DA=C/T;GM=NM_001145134.1,NM_001145135.1,NM_001145136.1,NM_001145137.1,NM_004377.3,NM_152245.2,NM_152246.2,NR_027928.2;GL=CPT1B/CHKB-CPT1B;FG=missense,missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,none;PP=497/739,531/773,450/692,531/773,531/773,531/773,531/773,NA;CDP=1489,1591,1348,1591,1591,1591,1591,3161;GS=56,56,56,56,56,56,56,NA;PH=0.966;CP=0.993;CG=5.310;AA=C;HA=10.1;HE=48.3;HC=47.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway|hsa04920:AdipocytokineSignalingPathway;GESP=T:4624/C:8382;PAC=NP_001138606.1,NP_001138607.1,NP_001138608.1,NP_001138609.1,NP_004368.1,NP_689451.1,NP_689452.1,NA	GT:GQ:PL	0/0:8:0,9,56	0/1:62:106,0,58	1/1:6:58,9,0	0/1:3:0,0,0

        </p>
        <p>
          X	27839617	rs12557898	G	A	18.40	lowCVG	AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,0,3;FQ=16.7;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=R65Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGEB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182506.3;VDB=0.0106;DN=120;DA=G/A;GM=NM_182506.3;GL=MAGEB10;FG=missense;FD=missense;AAC=ARG/GLN;PP=65/348;CDP=194;GS=43;PH=0.012;CP=0.000;CG=-5.040;AA=G;HA=14.0;HE=44.2;HC=8.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=243;GESP=A:4054/G:6506;PAC=NP_872312.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:5:56,9,0	0/0:9:0,9,53

        </p>
        <p>
          X	49179719	rs6520418	C	G	20.80	midCVG	AC1=5;AF1=0.592;DB;DP=14;DP4=0,8,2,4;FQ=24.7;MQ=26;PV4=0.16,0.013,7.5e-06,0.011;SNPEFF_AMINO_ACID_CHANGE=P16R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0167;DN=116;DA=C/G;GM=NM_001098406.1;GL=GAGE12J;FG=missense;FD=missense;AAC=PRO/ARG;PP=16/118;CDP=47;GS=103;PH=0;CP=0.000;CG=0.012;AA=G;DV=by-cluster,by-2hit-2allele;DSP=38;GESP=G:1901/C:4835;PAC=NP_001091876.1	GT:GQ:PL	0/1:3:0,0,0	0/0:20:0,24,118	1/1:3:17,3,0	1/1:13:48,15,0

        </p>
        <p>
          X	49179755	rs7064530	G	A	5.88	midCVG	AC1=5;AF1=0.6096;DB;DP=13;DP4=0,6,0,5;FQ=9.09;MQ=25;PV4=1,1,6.2e-05,1;SNPEFF_AMINO_ACID_CHANGE=R28Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0001;DN=116;DA=G/A;GM=NM_001098406.1;GL=GAGE12J;FG=missense-near-splice;FD=missense;AAC=ARG/GLN;PP=28/118;CDP=83;GS=43;PH=0.881;CP=0.000;CG=-1.910;AA=G;DV=by-cluster;DSP=2;GESP=A:2418/G:4398;PAC=NP_001091876.1	GT:GQ:PL	0/1:3:0,0,0	0/0:17:0,18,67	1/1:4:19,6,0	1/1:6:30,9,0

        </p>
        <p>
          X	88008807	rs5984611	G	A	58	lowCVG	AC1=6;AF1=0.7071;DB;DP=7;DP4=0,2,1,4;FQ=4.26;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=R131H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPXCR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184771.1;VDB=0.0098;DN=114;DA=G/A;GM=NM_001184771.1,NM_033048.5;GL=CPXCR1;FG=missense;FD=missense;AAC=ARG/HIS;PP=131/302;CDP=392;GS=29;PH=0.995;CP=0.000;CG=0.096;AA=G;HA=16.1;HE=40.7;HC=5.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=400;GESP=A:3478/G:7085;PAC=NP_001171700.1,NP_149037.4	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:40,6,0	0/1:34:54,0,33

        </p>
        <p>
          X	153039502	rs6643791	G	C	15.60	lowCVG	AC1=6;AF1=0.683;DB;DP=5;DP4=0,2,3,0;FQ=-16.8;MQ=60;PV4=0.1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=E1179D;SNPEFF_CODON_CHANGE=gaG/gaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163257.1;VDB=0.0123;DN=116;DA=G/C;GM=NM_001163257.1,NM_005393.2,XM_005274694.1,XM_005274695.1,XM_005274696.1;GL=PLXNB3;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=GLU/ASP;PP=1179/1933,1156/1910,1203/1957,1203/1933,809/1563;CDP=3537,3468,3609,3609,2427;GS=45;PH=0.005;CP=0.008;CG=-0.654;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=52;KP=hsa04360:AxonGuidance;GESP=C:7729/G:2554;PPI=ARHGEF11:895;PAC=NP_001156729.1,NP_005384.2,XP_005274751.1,XP_005274752.1,XP_005274753.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:4:27,3,0	0/1:15:23,0,13

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00024;CDKN1B|DDX5|ELK4|ERG|ETV1|ETV4|ETV5|AMACR|GSTP1|KLK3|AR|NKX3|PTEN|TERT|TMPRSS2|SLC45A3;hsa05215

        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 4
        </p>
        <p>
          <strong>Missense Mutations:</strong> 145
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.95
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse9">Endometrial Cancer</a>
      </h4>
    </div>
    <div id="collapse9" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>1 242035438 rs4149965 G A 6.86 lowCVG AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1 GT:GQ:PL 0/0:3:0,0,0 0/0:3:0,0,0 1/1:4:44,9,0 0/0:16:0,15,101</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

<p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>12 49443799 rs112236653 G A 49.50 midCVG AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1 GT:GQ:PL 0/1:14:11,0,64 0/1:3:0,0,0 1/1:9:81,12,0 0/0:7:0,9,52</p>

<p>14 21770730 rs6571751 A G 150 midCVG AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3 GT:GQ:PL 0/1:18:39,0,18 0/1:21:83,0,21 1/1:7:49,6,0 1/1:5:22,3,0</p>

<p>19 8997482 . C T 52.20 highCVG AC1=5;AF1=0.5614;DP=30;DP4=3,9,6,12;FQ=55.8;MQ=39;PV4=0.7,1,3.5e-17,1;SNPEFF_AMINO_ACID_CHANGE=R13647Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0008;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=ARG/GLN;PP=13647/14508,5561/6405,5561/6025,442/1303;CDP=40940,16682,16682,1325;GS=43;PH=0.984;CP=0.000;CG=-5.720;AA=C;DSP=55;GESP=C:12276;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/1:3:0,3,59 0/1:17:15,0,19 1/1:13:69,15,0 0/1:17:13,0,77</p>

<p>19 8997507 . G A 30.10 highCVG AC1=3;AF1=0.3713;DP=29;DP4=8,10,2,8;FQ=32.1;MQ=43;PV4=0.25,0.045,0.00017,0.46;SNPEFF_AMINO_ACID_CHANGE=L13639F;SNPEFF_CODON_CHANGE=Ctc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0023;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=LEU/PHE;PP=13639/14508,5553/6405,5553/6025,434/1303;CDP=40915,16657,16657,1300;GS=22;PH=0.984;CP=0.046;CG=-2.050;AA=G;DSP=30;GESP=G:12308;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/0:21:0,21,114 0/1:16:16,0,29 1/1:7:61,12,0 0/0:8:0,8,111</p>

<p>19 9024994 rs67631215 C T 90.10 highCVG AC1=5;AF1=0.6778;DB;DP=16;DP4=2,2,11,0;FQ=-17;MQ=54;PV4=0.057,0.07,1,0.16;SNPEFF_AMINO_ACID_CHANGE=E12290K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0020;DN=130;DA=C/T;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=GLU/LYS;PP=12290/14508,4204/6405,4204/6025;CDP=36868,12610,12610;GS=56;PH=0.009;CP=0.004;CG=1.510;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=5;GESP=T:4021/C:7885;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:3:0,0,0 0/1:9:41,0,8 1/1:6:33,6,0 0/1:10:55,0,9</p>

<p>19 9056982 rs11670318 G A 109 midCVG AC1=5;AF1=0.6629;DB;DP=11;DP4=1,2,6,2;FQ=5.06;MQ=60;PV4=0.49,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T10155I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0045;DN=120;DA=G/A;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=THR/ILE;PP=10155/14508,2069/6405,2069/6025;CDP=30464,6206,6206;GS=89;PH=0.997;CP=0.000;CG=0.467;AA=G;HA=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=810;GESP=A:3367/G:8779;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:30:47,0,29 0/1:3:0,0,0 1/1:9:61,9,0 0/1:11:41,0,10</p>

<p>19 9069792 rs1862458 G A 52.70 highCVG AC1=4;AF1=0.4499;DB;DP=16;DP4=1,9,4,2;FQ=55.2;MQ=60;PV4=0.036,0.29,1,1;SNPEFF_AMINO_ACID_CHANGE=S5885F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0050;DN=92;DA=G/A;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=SER/PHE;PP=5885/14508;CDP=17654;GS=155;PH=0.995;CP=0.002;CG=1.870;AA=G;HA=12.4;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8074;GESP=A:1913/G:10339;PAC=NP_078966.2 GT:GQ:PL 0/1:13:11,0,32 0/0:19:0,21,95 1/1:4:44,6,0 0/1:19:43,0,15</p>

<p>19 9069892 rs1862460 A C 33.40 highCVG AC1=3;AF1=0.3838;DB;DP=20;DP4=2,9,3,1;FQ=35.4;MQ=60;PV4=0.077,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=F5852V;SNPEFF_CODON_CHANGE=Ttt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0084;DN=92;DA=A/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=PHE/VAL;PP=5852/14508;CDP=17554;GS=50;PH=0;CP=0.000;CG=-0.715;AA=C;HA=30.1;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7974;GESP=C:2522/A:9570;PAC=NP_078966.2 GT:GQ:PL 0/1:13:13,0,15 0/0:27:0,27,130 1/1:5:65,9,0 0/0:4:0,3,28</p>

<p>19 9070225 rs1559172 G C 130 midCVG AC1=6;AF1=0.682;DB;DP=12;DP4=0,3,4,5;FQ=5.24;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H5741D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0092;DN=88;DA=G/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=HIS/ASP;PP=5741/14508;CDP=17221;GS=81;PH=0;CP=0.000;CG=-0.660;AA=C;HA=28.3;HE=15.0;HC=11.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7641;GESP=C:10349/G:2229;PAC=NP_078966.2 GT:GQ:PL 0/1:22:52,0,22 0/1:5:0,3,22 1/1:12:76,12,0 1/1:7:47,6,0</p>


        <p>
          <strong>IDs and variants associated:</strong> H00026;CEACAM5|CTNNB1|ERBB2|ESR1|ESR2|KRAS|MLH1|PGR|PTEN|TP53|MUC16;hsa05213
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 17
        </p>
        <p>
          <strong>Disease Rank Score:</strong>0.17
        </p>
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse10">Thyroid cancer</a>
      </h4>
    </div>
    <div id="collapse10" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>1 115829313 rs6330 G A 9.39 midCVG AC1=4;AF1=0.4545;DB;DP=10;DP4=5,0,2,1;FQ=11;MQ=60;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=A35V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NGF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002506.2;VDB=0.0110;DN=52;DA=G/A;GM=NM_002506.2;GL=NGF;FG=missense;FD=missense;AAC=ALA/VAL;PP=35/242;CDP=104;GS=64;PH=0.002;CP=0.977;CG=4.450;AA=G;HA=17.7;HE=44.1;HC=18.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=116;KP=hsa04010:MAPKSignalingPathway|hsa04210:Apoptosis|hsa04722:NeurotrophinSignalingPathway;TFBS=V$AP2REP_01;GESP=A:4801/G:8205;PPI=NTRK1:998|NGFR:937|NTF4:937|SORT1:845;PAC=NP_002497.2 GT:GQ:PL 0/0:3:0,0,0 0/0:15:0,15,54 1/1:5:47,9,0 0/0:3:0,0,0</p>

<p>1 117311147 rs699738 C A 39 midCVG AC1=3;AF1=0.4223;DB;DP=13;DP4=1,6,5,1;FQ=41.3;MQ=60;PV4=0.029,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=H266Q;SNPEFF_CODON_CHANGE=caC/caA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001767.3;VDB=0.0019;DN=86;DA=C/A;GM=NM_001767.3,XM_005271319.1,XM_005271320.1;GL=CD2;FG=missense;FD=missense,unknown,unknown;AAC=HIS/GLN;PP=266/352,292/378,225/311;CDP=798,876,675;GS=24;PH=0.085;CP=0.000;CG=-2.990;AA=C;HA=30.3;HE=9.4;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=62;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04640:HematopoieticCellLineage;GESP=A:2986/C:10020;PPI=CD58:996|CD2BP2:995|CD2AP:974|SH3KBP1:937|YES1:937|LCK:846|PTPRC:846;PAC=NP_001758.2,XP_005271376.1,XP_005271377.1 GT:GQ:PL 0/0:4:0,3,22 0/1:14:14,0,14 1/1:11:71,15,0 0/0:14:0,15,77</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>2 96993757 rs2279105 C T 69.10 lowCVG AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3 GT:GQ:PL 0/1:16:12,0,21 0/1:4:0,3,17 1/1:4:21,3,0 1/1:12:80,12,0</p>

<p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

<p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

<p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

<p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 103573006 rs2916569 G A 17.80 midCVG AC1=4;AF1=0.504;DB;DP=10;DP4=2,1,4,0;FQ=4.62;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S216N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024410.3;VDB=0.0051;DN=101;DA=G/A;GM=NM_024410.3;GL=ODF1;FG=missense;FD=missense;AAC=SER/ASN;PP=216/251;CDP=647;GS=46;PH=0;CP=0.053;CG=-7.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;RT=TGCAGCCCC;DSP=327;GESP=G:13006;PAC=NP_077721.2 GT:GQ:PL 0/0:5:0,6,25 0/1:3:0,0,0 1/1:4:34,6,0 0/1:18:24,0,15</p>

<p>9 114359624 rs1053959 C A 122 highCVG AC1=3;AF1=0.3851;DB;DP=23;DP4=6,8,4,5;FQ=124;MQ=59;PV4=1,1,1,0.0013;SNPEFF_AMINO_ACID_CHANGE=A27S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTGR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146108.1;VDB=0.0165;DN=86;DA=C/A;GM=NM_001146108.1,NM_001146109.1,NM_012212.3;GL=PTGR1;FG=missense;FD=missense;AAC=ALA/SER;PP=27/330,27/302,27/330;CDP=79;GS=99;PH=0.019;CP=0.038;CG=-0.431;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=28;GESP=A:7644/C:5362;PAC=NP_001139580.1,NP_001139581.1,NP_036344.2 GT:GQ:PL 0/0:29:0,30,158 0/1:10:91,0,4 1/1:8:78,12,0 0/0:9:0,9,72</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

<p>12 49443799 rs112236653 G A 49.50 midCVG AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1 GT:GQ:PL 0/1:14:11,0,64 0/1:3:0,0,0 1/1:9:81,12,0 0/0:7:0,9,52</p>

<p>12 53682468 . G T 35.30 lowCVG AC1=5;AF1=0.6499;DP=5;DP4=1,0,0,4;FQ=-11.7;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1565S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ESPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012291.4;VDB=0.0084;GM=NM_012291.4;GL=ESPL1;FG=missense;FD=unknown;AAC=ALA/SER;PP=1565/2121;CDP=4693;GS=99;PH=0.01;CP=0.003;CG=2.840;AA=G;DSP=16;KP=hsa04110:CellCycle|hsa04114:OocyteMeiosis;GESP=G:12990;PPI=PTTG1:987|IRF2BPL:987;PAC=NP_036423.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:72,12,0 0/1:3:0,3,21</p>

<p>12 70963637 . C T 6.02 midCVG AC1=6;AF1=0.7277;DP=10;DP4=1,0,4,0;FQ=-14.1;MQ=60;PV4=1,0.00024,1,0.38;SNPEFF_AMINO_ACID_CHANGE=G1151E;SNPEFF_CODON_CHANGE=gGa/gAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTPRB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001109754.2;VDB=0.0143;GM=NM_001109754.2,NM_001206971.1,NM_001206972.1,NM_002837.4;GL=PTPRB;FG=missense;FD=unknown;AAC=GLY/GLU;PP=1151/2216,843/1908,933/1908,933/1998;CDP=3452,2528,2798,2798;GS=98;PH=1;CP=0.169;CG=5.300;AA=C;DSP=82;KP=hsa04520:AdherensJunction;GESP=C:12348;PPI=CNTN1:937;PAC=NP_001103224.1,NP_001193900.1,NP_001193901.1,NP_002828.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:33,9,0 0/1:10:8,0,17</p>

<p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

<p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

<p>21 46271460 rs2236442 T C 20.70 midCVG AC1=2;AF1=0.3147;DB;DP=9;DP4=4,2,0,3;FQ=22.3;MQ=58;PV4=0.17,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=E84G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTTG1IP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001286822.1;VDB=0.0020;DN=98;DA=T/C;GM=NM_004339.3,XM_005261175.1,XM_005261176.1,XM_005261177.1;GL=PTTG1IP;FG=3-prime-UTR,missense,missense,3-prime-UTR;FD=utr-variant-3-prime,unknown,unknown,unknown;AAC=none,GLU/GLY,GLU/GLY,none;PP=NA,104/163,84/143,NA;CDP=NA,311,251,NA;GS=NA,98,98,NA;PH=0.914;CP=0.000;CG=0.454;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=83;GESP=C:902/T:12104;PAC=NA,XP_005261232.1,XP_005261233.1,NA GT:GQ:PL 0/0:14:0,12,80 0/0:3:0,0,0 1/1:4:61,9,0 0/0:8:0,6,40</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

        <p>
          <strong>IDs and variants associated:</strong> H00032;TFG|CTNNB1|DPP4|HRAS|KRAS|KRT19|LGALS3|NRAS|NTRK1|PPARG|RET|BRAF|TG|TP53|TPM3|TPO|TPR|PAX8|CCDC6|NCOA4|CDH1;hsa05216
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 27
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.27
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse11">Malignant melanoma</a>
      </h4>
    </div>
    <div id="collapse11" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>2 113537223 rs17561 C A 31.40 midCVG AC1=2;AF1=0.2957;DB;DP=14;DP4=9,1,4,0;FQ=32.8;MQ=60;PV4=1,0.45,1,1;SNPEFF_AMINO_ACID_CHANGE=A114S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000575.3;VDB=0.0004;DN=63;DA=C/A;GM=NM_000575.3;GL=IL1A;FG=missense;FD=missense;AAC=ALA/SER;PP=114/272;CDP=340;GS=99;PH=0.982;CP=0.009;CG=4.440;AA=C;HA=14.5;HE=27.6;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;KP=hsa04010:MAPKSignalingPathway|hsa04060:Cytokine-cytokineReceptorInteraction|hsa04210:Apoptosis|hsa04640:HematopoieticCellLineage|hsa04940:TypeIDiabetesMellitus|hsa05020:PrionDiseases|hsa05140:Leishmaniasis|hsa05332:Graft-versus-hostDisease;GESP=A:3403/C:9603;PPI=S100A13:959;PAC=NP_000566.3 GT:GQ:PL 0/0:26:0,24,122 0/0:6:0,3,27 1/1:6:74,12,0 0/0:6:0,3,27</p>

<p>3 67411166 rs902321 A G 144 highCVG AC1=5;AF1=0.6662;DB;DP=15;DP4=2,3,6,4;FQ=34.2;MQ=60;PV4=0.61,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Y404H;SNPEFF_CODON_CHANGE=Tat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0085;DN=86;DA=A/G;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=TYR/HIS;PP=404/441;CDP=1210;GS=83;CP=0.001;CG=3.220;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=27;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:33:42,0,33 0/0:3:0,6,44 1/1:9:70,9,0 1/1:12:80,12,0</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>5 7897191 rs10380 C T 46.90 midCVG AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1 GT:GQ:PL 0/0:11:0,12,87 0/0:8:0,9,73 1/1:6:65,9,0 0/1:4:29,3,0</p>

<p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

<p>7 64452830 rs34639489 C T 86 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,4,2;FQ=5.73;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C192Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERV3-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007253.3;VDB=0.0141;DN=126;DA=C/T;GM=NM_001007253.3;GL=ERV3-1;FG=missense;FD=missense;AAC=CYS/TYR;PP=192/605;CDP=575;GS=194;PH=0.991;CP=0.325;CG=0.109;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;RM=HERV3-int;DSP=963;GESP=T:5840/C:6396;PAC=NP_001007254.2 GT:GQ:PL 0/0:3:0,6,41 0/1:3:0,0,0 1/1:14:102,15,0 1/1:4:26,3,0</p>

<p>7 113518434 rs1799999 C A 68 lowCVG AC1=6;AF1=0.7069;DB;DP=8;DP4=1,2,1,4;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D905Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002711.3;VDB=0.0058;DN=89;DA=C/A;GM=NM_002711.3,XM_005250473.1;GL=PPP1R3A;FG=missense;FD=missense,unknown;AAC=ASP/TYR;PP=905/1123,704/922;CDP=2713,2110;GS=160;PH=0.976;CP=0.997;CG=5.640;AA=C;HA=23.4;HE=6.3;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=1747;KP=hsa04910:InsulinSignalingPathway;GESP=A:1760/C:11244;PAC=NP_002702.2,XP_005250530.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:59,6,0 0/1:49:45,0,55</p>

<p>7 133848257 rs17167553 G T 35.10 highCVG AC1=3;AF1=0.3866;DB;DP=16;DP4=5,6,0,4;FQ=37.2;MQ=60;PV4=0.23,0.31,1,0.17;SNPEFF_AMINO_ACID_CHANGE=D302Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRGUK;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144648.1;VDB=0.0117;DN=123;DA=G/T;GM=NM_144648.1;GL=LRGUK;FG=missense;FD=unknown;AAC=ASP/TYR;PP=302/826;CDP=904;GS=160;PH=0.902;CP=0.983;CG=3.770;AA=G;HA=13.5;HE=7.3;HC=32.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=36;GESP=T:1632/G:11374;PAC=NP_653249.1 GT:GQ:PL 0/0:23:0,24,157 0/0:4:0,3,23 1/1:5:63,9,0 0/1:18:17,0,43</p>

<p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 104412688 rs141856398 T C 58.40 highCVG AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA GT:GQ:PL 0/0:4:0,3,24 0/0:6:0,6,40 1/1:3:38,6,0 0/1:64:63,0,86</p>

<p>8 110457011 rs72687015 A G 85.70 midCVG AC1=5;AF1=0.6401;DB;DP=10;DP4=3,1,4,2;FQ=43.1;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=Y1638C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKHD1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_177531.4;VDB=0.0097;DN=130;DA=A/G;GM=NM_177531.4;GL=PKHD1L1;FG=missense;FD=missense;AAC=TYR/CYS;PP=1638/4244;CDP=4913;GS=194;PH=0.951;CP=1.000;CG=3.250;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=122;GESP=G:2556/A:9592;PAC=NP_803875.2 GT:GQ:PL 0/1:21:16,0,37 0/0:4:0,6,48 1/1:9:70,9,0 1/1:6:46,6,0</p>

<p>8 142517222 rs2748421 A G 92.90 midCVG AC1=5;AF1=0.6549;DB;DP=13;DP4=3,2,8,0;FQ=41.5;MQ=59;PV4=0.13,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=Y10H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0027;DN=100;DA=A/G;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/HIS,none,none;PP=10/1319,NA,NA;CDP=28,109,109;GS=83,NA,NA;CP=0.000;CG=1.370;AA=G;HA=20.8;HE=49.1;HC=36.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=G:4995/A:7673;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:28:52,0,27 1/1:11:68,12,0 0/1:19:14,0,51</p>

<p>9 114464487 rs1407390 T C 45.60 lowCVG AC1=6;AF1=0.7452;DB;DP=8;DP4=0,1,0,6;FQ=-7.45;MQ=60;PV4=1,0.0038,1,0.36;SNPEFF_AMINO_ACID_CHANGE=Y932C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C9orf84;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173521.4;VDB=0.0037;DN=88;DA=T/C;GM=NM_001080551.1,NM_173521.3,XM_005251738.1;GL=C9orf84;FG=missense;FD=missense,missense,unknown;AAC=TYR/CYS;PP=893/1406,932/1445,996/1509;CDP=2678,2795,2987;GS=194;PH=0;CP=0.930;CG=-2.510;AA=C;HA=15.6;HE=21.5;HC=7.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=20;GESP=C:10539/T:2451;PAC=NP_001074020.1,NP_775792.3,XP_005251795.1 GT:GQ:PL 0/1:5:0,3,28 0/1:3:0,0,0 1/1:5:18,3,0 1/1:15:68,15,0</p>

<p>10 70641860 rs1341667 T C 76 midCVG AC1=5;AF1=0.6593;DB;DP=10;DP4=1,2,3,4;FQ=26.9;MQ=60;PV4=1,0.21,1,0.26;SNPEFF_AMINO_ACID_CHANGE=Y153H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130161.2;VDB=0.0014;DN=88;DA=T/C;GM=NM_001130159.2,NM_001130160.2,NM_001130161.2,NM_001130162.2,NM_152709.4,XM_005269605.1;GL=STOX1;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=TYR/HIS;PP=153/228,153/170,153/990,43/880,153/990,43/880;CDP=457,457,457,127,457,127;GS=83;PH=0.687;CP=0.990;CG=4.440;AA=T;HA=33.8;HE=38.1;HC=11.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=7;TFBS=V$HNF1_C;GESP=C:6772/T:5074;PAC=NP_001123631.1,NP_001123632.1,NP_001123633.1,NP_001123634.1,NP_689922.3,XP_005269662.1 GT:GQ:PL 0/0:6:0,9,65 0/1:3:0,0,0 1/1:17:106,18,0 1/1:4:13,3,0</p>

<p>10 84733562 . C T 20.10 midCVG AC1=3;AF1=0.3373;DP=11;DP4=6,1,3,0;FQ=21.8;MQ=60;PV4=1,0.18,1,0.066;SNPEFF_AMINO_ACID_CHANGE=H435Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010848.3;VDB=0.0000;GM=NM_001010848.3,NM_001165972.1,NM_001165973.1,XM_005269444.1,XM_005269445.1,XM_005269446.1,XM_005269447.1,XM_005269448.1,XM_005269449.1,XM_005269450.1,XM_005269451.1;GL=NRG3;FG=missense;FD=unknown;AAC=HIS/TYR;PP=435/697,434/696,214/500,435/721,265/527,239/525,239/501,214/476,171/457,171/433,85/371;CDP=1303,1300,640,1303,793,715,715,640,511,511,253;GS=83;PH=0.952;CP=1.000;CG=5.050;AA=C;DSP=19;KP=hsa04012:ErbBSignalingPathway;TFBS=V$CETS1P54_01;GESP=C:13006;PAC=NP_001010848.2,NP_001159444.1,NP_001159445.1,XP_005269501.1,XP_005269502.1,XP_005269503.1,XP_005269504.1,XP_005269505.1,XP_005269506.1,XP_005269507.1,XP_005269508.1 GT:GQ:PL 0/0:5:0,3,24 0/0:19:0,18,100 1/1:4:60,9,0 0/0:3:0,0,0</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>11 51411914 rs61890334 T C 57.90 midCVG AC1=4;AF1=0.4616;DB;DP=12;DP4=4,3,1,4;FQ=60.5;MQ=55;PV4=0.29,1,1,0.074;SNPEFF_AMINO_ACID_CHANGE=Y161C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005272.3;VDB=0.0086;DN=129;DA=T/C;GM=NM_001005272.3;GL=OR4A5;FG=missense;FD=missense;AAC=TYR/CYS;PP=161/316;CDP=482;GS=194;PH=0.027;CP=0.042;CG=1.930;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1032140;KP=hsa04740:OlfactoryTransduction;GESP=C:765/T:12229;PAC=NP_001005272.3 GT:GQ:PL 0/0:13:0,15,95 0/0:4:0,3,25 1/1:6:63,9,0 0/1:4:40,2,0</p>

<p>11 124135009 rs2512168 G A 85.50 highCVG AC1=3;AF1=0.3452;DB;DP=15;DP4=1,4,4,6;FQ=75.1;MQ=42;PV4=0.6,1,0.082,1;SNPEFF_AMINO_ACID_CHANGE=C96Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR8G5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005198.1;VDB=0.0030;DN=100;DA=G/A;GM=NM_001005198.1;GL=OR8G5;FG=missense;FD=missense;AAC=CYS/TYR;PP=96/347;CDP=287;GS=194;CP=0.533;CG=-5.000;AA=A;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118248;GESP=A:6010/G:6910;PAC=NP_001005198.1 GT:GQ:PL 0/0:5:0,3,25 0/0:7:0,6,48 1/1:24:130,30,0 0/0:7:0,6,45</p>

<p>12 8374812 rs17855656 G A 37.30 lowCVG AC1=5;AF1=0.6547;DB;DP=5;DP4=1,0,4,0;FQ=-14.5;MQ=48;PV4=1,0.073,1,1;SNPEFF_AMINO_ACID_CHANGE=P334L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM90A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018088.3;VDB=0.0130;DN=123;DA=G/A;GM=NM_018088.3,XM_005253403.1,XM_005253404.1;GL=FAM90A1;FG=missense;FD=missense,unknown,unknown;AAC=PRO/LEU;PP=334/465;CDP=1001;GS=98;PH=0.023;CP=0.006;CG=-1.440;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=569;GESP=A:1659/G:11279;PAC=NP_060558.3,XP_005253460.1,XP_005253461.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,18 1/1:10:74,12,0 0/1:3:0,0,0</p>

<p>15 65676726 rs33918653 C T 78.50 midCVG AC1=4;AF1=0.5302;DB;DP=10;DP4=0,4,5,1;FQ=24.8;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C1125Y;SNPEFF_CODON_CHANGE=tGc/tAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0003;DN=126;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=CYS/TYR;PP=1125/1251;CDP=3374;GS=194;PH=0.981;CP=1.000;CG=4.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=T:2295/C:10705;PAC=NP_066013.1 GT:GQ:PL 0/0:5:0,6,38 0/1:3:0,0,0 1/1:7:70,9,0 0/1:28:50,0,25</p>

<p>17 76433899 rs61742072 G A 107 midCVG AC1=5;AF1=0.5795;DB;DP=14;DP4=6,0,2,6;FQ=60.9;MQ=60;PV4=0.0097,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=H3953Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=74;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH17;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173628.3;VDB=0.0018;DN=129;DA=G/A;GM=NM_173628.3;GL=DNAH17;FG=missense;FD=missense;AAC=HIS/TYR;PP=3953/4463;CDP=11857;GS=83;PH=0;CP=0.997;CG=2.960;AA=G;DV=by-frequency,by-cluster;DSP=52;GESP=A:1641/G:11359;PAC=NP_775899.3 GT:GQ:PL 0/1:21:20,0,22 0/0:11:0,15,83 1/1:4:26,3,0 1/1:16:109,18,0</p>

<p>18 11610073 rs75301503 G A 72.40 midCVG AC1=4;AF1=0.5075;DB;DP=14;DP4=2,5,3,3;FQ=75.1;MQ=53;PV4=0.59,1,0.44,0.48;SNPEFF_AMINO_ACID_CHANGE=C160Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=G/A;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=CYS/TYR;PP=160/339;CDP=479;GS=194;CP=0.000;AA=G;DG;DV=by-cluster,by-1000G;DSP=79866;PAC=XP_934053.1 GT:GQ:PL 0/0:10:0,12,79 0/1:3:0,0,1/1:12:100,15,0 0/1:19:16,0,48</p>

<p>19 9764230 . G A 25.50 lowCVG AC1=2;AF1=0.3027;DP=7;DP4=2,2,0,3;FQ=26.8;MQ=60;PV4=0.43,1,1,0;SNPEFF_AMINO_ACID_CHANGE=H226Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF562;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130031.1;VDB=0.0003;GM=NM_001130031.1,NM_001130032.1,NM_017656.3,XM_005259941.1,XM_005259942.1,XM_005259943.1,XM_005259944.1,XM_005259945.1,XM_005259946.1,XM_005259947.1,XM_005259948.1,XM_005259949.1,XM_005259950.1,XM_005259951.1,XM_005259952.1,XM_005259953.1;GL=ZNF562;FG=missense;FD=unknown;AAC=HIS/TYR;PP=226/427,226/427,154/355,226/427,225/426,218/419,189/390,157/358,157/358,157/358,157/358,148/349,111/312,110/311,83/284,83/284;CDP=676,676,460,676,673,652,565,469,469,469,469,442,331,328,247,247;GS=83;PH=0.965;CP=0.000;CG=-0.749;AA=G;DSP=325;GESP=G:13006;PAC=NP_001123503.1,NP_001123504.1,NP_060126.2,XP_005259998.1,XP_005259999.1,XP_005260000.1,XP_005260001.1,XP_005260002.1,XP_005260003.1,XP_005260004.1,XP_005260005.1,XP_005260006.1,XP_005260007.1,XP_005260008.1,XP_005260009.1,XP_005260010.1 GT:GQ:PL 0/0:8:0,6,34 0/0:6:0,3,28 1/1:4:66,9,0 0/0:6:0,3,19</p>

<p>19 15198631 rs8105737 A C 43.30 lowCVG AC1=5;AF1=0.6343;DB;DP=7;DP4=3,0,3,1;FQ=15.8;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=Y252S;SNPEFF_CODON_CHANGE=tAt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004713.1;VDB=0.0061;DN=116;DA=A/C;GM=NM_001004713.1;GL=OR1I1;FG=missense;FD=missense;AAC=TYR/SER;PP=252/356;CDP=755;GS=144;PH=0.999;CP=0.743;CG=3.590;AA=A;HA=36.4;HE=33.0;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19702;KP=hsa04740:OlfactoryTransduction;TFBS=V$GR_Q6,V$YY1_02;GESP=C:7526/A:5480;PAC=NP_001004713.1 GT:GQ:PL 0/1:14:17,0,13 0/1:3:0,0,0 1/1:6:49,6,0 0/1:22:17,0,38</p>

<p>19 23159486 rs289318 T C 119 highCVG AC1=4;AF1=0.5102;DB;DP=20;DP4=6,3,3,7;FQ=106;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Y218C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0021;DN=79;DA=T/C;GM=NM_001267716.1,XM_005259908.1;GL=ZNF728;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=TYR/CYS,none;PP=218/623,NA;CDP=653,NA;GS=194,NA;CP=0.272;CG=0.945;AA=T;HE=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=427;PAC=NP_001254645.1,NA GT:GQ:PL 0/0:9:0,12,78 0/1:20:34,0,17 1/1:18:121,21,0 0/1:15:12,0,56</p>

<p>19 35435069 rs765746 A G 47.60 midCVG AC1=4;AF1=0.4586;DB;DP=12;DP4=6,1,4,1;FQ=50.2;MQ=60;PV4=1,0.4,1,0.13;SNPEFF_AMINO_ACID_CHANGE=Y401C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF30;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099437.1;VDB=0.0179;DN=86;DA=A/G;GM=NM_001099437.1,NM_001099438.1,NM_194325.2,NR_024018.1;GL=ZNF30;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=TYR/CYS,TYR/CYS,TYR/CYS,none;PP=401/625,401/625,400/624,NA;CDP=1202,1202,1199,1657;GS=194,194,194,NA;PH=1;CP=0.040;CG=1.040;AA=A;HA=22.5;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=943;GESP=G:2295/A:10709;PAC=NP_001092907.1,NP_001092908.1,NP_919306.2,NA GT:GQ:PL 0/0:6:0,6,39 0/0:13:0,15,92 1/1:8:75,12,0 0/1:4:20,3,0</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 56499279 rs306457 G C 68.10 midCVG AC1=4;AF1=0.4813;DB;DP=12;DP4=0,6,0,6;FQ=46;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=*1049Y;SNPEFF_CODON_CHANGE=taG/taC;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_176811.2;VDB=0.0070;DN=79;DA=G/C;GM=NM_176811.2,XM_005258511.1;GL=NLRP8;FG=stop-lost;FD=stop-lost,unknown;AAC=stop/TYR;PP=1049/1049,1030/1030;CDP=3147,3090;CP=0.000;CG=0.372;AA=C;HA=26.7;HE=30.0;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=100;GESP=C:9537/G:3469;PAC=NP_789781.2,XP_005258568.1 GT:GQ:PL 0/0:7:0,9,50 0/1:17:21,0,15 1/1:4:45,6,0 0/1:27:47,0,23</p>

<p>X 23693193 . C A 54.30 highCVG AC1=2;AF1=0.2857;DP=20;DP4=6,6,0,7;FQ=55.8;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S149Y;SNPEFF_CODON_CHANGE=tCt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRDX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006406.1;VDB=0.0080;GM=NM_006406.1,XM_005274438.1;GL=PRDX4;FG=missense;FD=unknown;AAC=SER/TYR;PP=149/272,135/258;CDP=446,404;GS=144;PH=1;CP=1.000;CG=5.840;DSP=31;GESP=C:13006;PAC=NP_006397.1,XP_005274495.1 GT:GQ:PL 0/0:16:0,15,96 0/0:6:0,3,25 1/1:14:100,21,0 0/0:19:0,18,118</p>


        <p>
          <strong>IDs and variants associated:</strong> H00038;CDK4|CDKN2A|DCT|MLANA|MITF|NRAS|PTEN|0A1|BRAF|TP53|TYR|TYRP1|VIM;hsa05218
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 32
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 3.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse12">Basal cell canciroma</a>
      </h4>
    </div>
    <div id="collapse12" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

<p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>


        <p>
          <strong>IDs and variants associated:</strong> H00039;PTCH1|SHH|SMO|TP53|TP63;hsa05217
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 6
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.6
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse13">Glioma</a>
      </h4>
    </div>
    <div id="collapse13" class="panel-collapse collapse">
      <div class="panel-body">

        <p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>10 126678087 rs111888382 C T 16.70 highCVG AC1=6;AF1=0.6827;DB;DP=36;DP4=3,5,13,14;FQ=21.4;MQ=11;PV4=0.7,0.29,1.9e-05,0.13;SNPEFF_AMINO_ACID_CHANGE=*986;SNPEFF_CODON_CHANGE=taG/taA;SNPEFF_EFFECT=SYNONYMOUS_STOP;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0149;DN=132;DA=C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=synonymous;FD=synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=stop;PP=446/446,446/446,986/986,514/514,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,421/421,421/421;CDP=1338,1338,2958,1542,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1263,1263;CP=1.000;CG=5.120;AA=C;DV=by-cluster;DSP=181;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:4:0,3,38 1/1:11:16,12,0 1/1:30:45,31,0</p>

<p>10 126678092 rs73373169 G A 25.10 highCVG AC1=5;AF1=0.6731;DB;DP=36;DP4=2,5,12,13;FQ=29.5;MQ=12;PV4=0.43,1,1.3e-05,0.07;SNPEFF_AMINO_ACID_CHANGE=Q985*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0155;DN=131;DA=G/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=stop-gained;FD=stop-gained,stop-gained,stop-gained,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/stop;PP=445/446,445/446,985/986,513/514,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,420/421,420/421;CDP=1333,1333,2953,1537,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1258,1258;CP=1.000;CG=5.120;AA=G;DG;DV=by-cluster;DSP=176;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$MYB_Q6;GESP=G:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:3:0,5,41 1/1:17:22,18,0 1/1:28:49,28,0</p>

<p>CTBP2 10 126678177 . G T 39.20 highCVG AC1=6;AF1=0.7163;DP=34;DP4=3,4,8,17;FQ=44.8;MQ=28;PV4=0.67,0.00015,8e-15,1;SNPEFF_AMINO_ACID_CHANGE=N956K;SNPEFF_CODON_CHANGE=aaC/aaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0013;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=unknown;AAC=ASN/LYS;PP=416/446,416/446,956/986,484/514,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,391/421,391/421;CDP=1248,1248,2868,1452,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1173,1173;GS=94;PH=0.613;CP=1.000;CG=3.120;AA=G;DSP=91;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$BACH1_01,V$TAXCREB_02;GESP=G:12998;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:5:1,0,7 0/1:4:0,4,98 1/1:24:24,24,0 1/1:33:60,33,0</p>

<p>10 126678221 rs77752853 C A 43.30 highCVG AC1=6;AF1=0.7316;DB;DP=30;DP4=3,4,3,17;FQ=49.1;MQ=30;PV4=0.29,0.031,2.4e-12,0.15;SNPEFF_AMINO_ACID_CHANGE=A942S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=131;DA=C/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/SER;PP=402/446,402/446,942/986,470/514,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,377/421,377/421;CDP=1204,1204,2824,1408,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1129,1129;GS=99;PH=0.599;CP=0.998;CG=5.120;AA=C;DV=by-cluster;DSP=47;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:6:0,1,11 0/1:11:4,0,114 1/1:21:24,21,0 1/1:25:60,24,0</p>

<p>10 126678241 rs113985797 G C 8.10 highCVG AC1=4;AF1=0.4981;DB;DP=26;DP4=4,7,2,13;FQ=10.6;MQ=39;PV4=0.35,1.5e-07,2.9e-13,1;SNPEFF_AMINO_ACID_CHANGE=A935G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=132;DA=G/C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=395/446,395/446,935/986,463/514,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,370/421,370/421;CDP=1184,1184,2804,1388,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1109,1109;GS=60;PH=0.016;CP=1.000;CG=5.120;AA=G;DV=by-frequency,by-cluster;DSP=27;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/0:4:0,2,39 0/0:6:0,6,121 1/1:10:17,18,0 0/1:10:34,0,4</p>

<p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 55143083 rs1061679 T C 34 lowCVG AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA GT:GQ:PL 0/0:12:0,15,73 0/1:3:0,0,0 1/1:5:50,6,0 0/1:3:26,3,0</p>


        <p>
          <strong>IDs and variants associated:</strong> H00042;CDK4|CDKN2A|EGFR|MDM2|PDGFA|PDGFB|PDGFRA|PDGFRB|PTEN|RB1|TP53;hsa05214
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 1
        </p>
        <p>
          <strong>Missense Mutations:</strong> 8
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.8
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse14">Alzheimer's disease (AD)</a>
      </h4>
    </div>
    <div id="collapse14" class="panel-collapse collapse">
      <div class="panel-body">

        <p>4 76878716 rs2242471 G C 19.60 lowCVG AC1=6;AF1=0.6681;DB;DP=8;DP4=1,1,4,0;FQ=-12.1;MQ=54;PV4=0.33,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=S575C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018115.3;VDB=0.0091;DN=98;DA=G/C;GM=NM_018115.2,XM_005263102.1,XM_005263103.1,XM_005263104.1,XM_005263105.1;GL=SDAD1;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=SER/CYS;PP=575/688,550/663,538/651,478/591,478/591;CDP=1724,1649,1613,1433,1433;GS=112;PH=0.941;CP=1.000;CG=5.150;AA=G;HA=27.9;HE=41.7;HC=6.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=46;TFBS=V$CREL_01,V$IK1_01,V$NFKAPPAB_01,V$NFKB_C,V$STAT1_03,V$STAT5A_02;GESP=C:4469/G:8537;PAC=NP_060585.2,XP_005263159.1,XP_005263160.1,XP_005263161.1,XP_005263162.1 GT:GQ:PL 0/1:20:40,0,18 0/1:3:0,0,0 1/1:3:14,3,0 0/1:3:0,0,0</p>

<p>4 184618936 rs62617790 G C 120 highCVG AC1=4;AF1=0.5361;DB;DP=15;DP4=1,3,4,7;FQ=46.7;MQ=60;PV4=1,0.0017,1,0.28;SNPEFF_AMINO_ACID_CHANGE=Q933H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021942.5;VDB=0.0163;DN=129;DA=G/C;GM=NM_021942.5,NM_199053.2;GL=TRAPPC11;FG=missense;FD=missense;AAC=GLN/HIS;PP=933/1134,933/1087;CDP=2799;GS=24;PH=0.999;CP=0.998;CG=4.670;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=53;GESP=C:1255/G:11751;PAC=NP_068761.4,NP_951008.1 GT:GQ:PL 0/0:7:0,9,65 0/1:3:0,3,25 1/1:27:149,30,0 1/1:3:18,3,0</p>

<p>8 141468645 . G A 29.60 midCVG AC1=3;AF1=0.3656;DP=12;DP4=6,0,5,0;FQ=31.5;MQ=60;PV4=1,0.034,1,1.3e-09;SNPEFF_AMINO_ACID_CHANGE=Q7*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC9;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_031466.6;VDB=0.0000;GM=NM_031466.5;GL=TRAPPC9;FG=stop-gained;FD=unknown;AAC=GLN/stop;PP=7/1247;CDP=19;CP=0.000;CG=-2.520;AA=G;DSP=266;GESP=G:12786;PAC=NP_113654.4 GT:GQ:PL 0/0:3:0,0,0 0/0:7:0,6,51 1/1:9:72,15,0 0/0:12:0,12,70</p>

<p>TRAPPC11 14 104139526 . T C 20.20 midCVG AC1=2;AF1=0.2343;DP=13;DP4=3,6,3,0;FQ=21.3;MQ=60;PV4=0.18,0.47,1,1;VDB=0.0000;GM=NM_001130107.1,NM_005552.4,NM_182923.3,XM_005267599.1,XM_005267600.1,XM_005267601.1,XM_005267602.1,XM_005267603.1,XM_005267604.1,XM_005267605.1,XM_005267606.1,XM_005267607.1,XM_005267608.1,XM_005267609.1,XM_005267610.1,XM_005267611.1,XM_005267612.1,XM_005267613.1,XM_005267614.1,XM_005267615.1,XM_005267616.1,XM_005267617.1,XM_005267618.1,XM_005267619.1,XM_005267620.1,XM_005267621.1,XM_005267622.1,XM_005267623.1,XM_005267624.1,XM_005267625.1,XM_005267626.1,XM_005267627.1,XM_005267628.1,XM_005267629.1,XM_005267630.1,XM_005267631.1;GL=KLC1;FG=splice-donor;FD=unknown;CP=0.923;CG=5.560;AA=T;DSP=0;GESP=T:13006;PPI=KIF5B:937|APP:846|KIF5A:845|KIF5C:845;PAC=NP_001123579.1,NP_005543.2,NP_891553.2,XP_005267656.1,XP_005267657.1,XP_005267658.1,XP_005267659.1,XP_005267660.1,XP_005267661.1,XP_005267662.1,XP_005267663.1,XP_005267664.1,XP_005267665.1,XP_005267666.1,XP_005267667.1,XP_005267668.1,XP_005267669.1,XP_005267670.1,XP_005267671.1,XP_005267672.1,XP_005267673.1,XP_005267674.1,XP_005267675.1,XP_005267676.1,XP_005267677.1,XP_005267678.1,XP_005267679.1,XP_005267680.1,XP_005267681.1,XP_005267682.1,XP_005267683.1,XP_005267684.1,XP_005267685.1,XP_005267686.1,XP_005267687.1,XP_005267688.1 GT:GQ:PL 0/0:12:0,9,64 0/0:7:0,3,26 1/1:3:62,9,0 0/0:18:0,15,104</p>

        <p>
          <strong>IDs and variants associated:</strong> H00056;APOE|APP|PSEN1|PSEN2;hsa05010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 1
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse15">Parkinson's disease (PD)</a>
      </h4>
    </div>
    <div id="collapse15" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 11015165 rs75130475 A G 149 midCVG AC1=5;AF1=0.6109;DB;DP=14;DP4=0,2,7,4;FQ=-4.32;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I286T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf127;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170754.1;VDB=0.0208;DN=131;DA=A/G;GM=NM_001170754.1,XM_005263428.1,XM_005263429.1,XM_005263430.1;GL=C1orf127;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=286/824,228/766,129/667,147/659;CDP=857,683,386,440;GS=89;PH=0.397;CP=0.000;CG=-6.010;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79;GESP=G:1746/A:11260;PAC=NP_001164225.1,XP_005263485.1,XP_005263486.1,XP_005263487.1 GT:GQ:PL 0/1:4:0,3,24 0/1:4:0,3,14 1/1:4:26,3,0 1/1:28:168,30,0</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>1 18808526 rs2992752 A C 43.10 lowCVG AC1=6;AF1=0.7115;DB;DP=8;DP4=2,0,0,6;FQ=-12;MQ=60;PV4=0.036,0.43,1,0.35;SNPEFF_AMINO_ACID_CHANGE=T351P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHDC7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152375.2;VDB=0.0098;DN=101;DA=A/C;GM=NM_152375.2,XM_005245739.1,XM_005245740.1;GL=KLHDC7A;FG=missense;FD=missense,unknown,unknown;AAC=THR/PRO;PP=351/778,351/759,208/635;CDP=1051,1051,622;GS=38;PH=0;CP=0.000;CG=0.690;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=255;GESP=C:8092/A:4532;PAC=NP_689588.2,XP_005245796.1,XP_005245797.1 GT:GQ:PL 0/1:20:44,0,19 0/1:3:0,0,0 1/1:6:35,6,0 0/1:3:0,0,0</p>

<p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>1 55638075 rs1165222 G A 13.90 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,0,2;FQ=-26.8;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T226I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP24;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015306.2;VDB=0.0081;DN=87;DA=G/A;GM=NM_015306.2,XM_005270689.1;GL=USP24;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=226/2621,51/2444;CDP=677,152;GS=89;PH=0;CP=0.988;CG=2.190;AA=A;HA=12.5;HE=16.1;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=26;TFBS=V$BRACH_01,V$CDPCR3_01;GESP=A:10080/G:1770;PPI=UBC:709;PAC=NP_056121.2,XP_005270746.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:3:0,0,0</p>

<p>1 93672723 rs2815413 A C 107 highCVG AC1=6;AF1=0.7028;DB;DP=15;DP4=0,6,5,3;FQ=45.9;MQ=60;PV4=0.031,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=N326T;SNPEFF_CODON_CHANGE=aAt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206886.4;VDB=0.0073;DN=100;DA=A/C;GM=NM_206886.3,XM_005270815.1,XM_005270816.1;GL=CCDC18;FG=missense;FD=missense,unknown,unknown;AAC=ASN/THR;PP=326/1300,444/1574,444/1418;CDP=977,1331,1331;GS=65;PH=0.649;CP=1.000;CG=4.580;AA=A;HA=25.6;HE=15.0;HC=3.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=60;GESP=C:8247/A:3559;PAC=NP_996769.3,XP_005270872.1,XP_005270873.1 GT:GQ:PL 0/1:18:44,0,18 0/1:38:32,0,66 1/1:10:53,9,0 1/1:5:21,3,0</p>

<p>1 112308953 rs197412 T C 58.10 midCVG AC1=5;AF1=0.6066;DB;DP=10;DP4=1,1,7,0;FQ=8.93;MQ=60;PV4=0.22,1,1,0.045;SNPEFF_AMINO_ACID_CHANGE=I636T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DDX20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007204.4;VDB=0.0159;DN=79;DA=T/C;GM=NM_007204.4;GL=DDX20;FG=missense;FD=missense;AAC=ILE/THR;PP=636/825;CDP=1907;GS=89;PH=0;CP=0.331;CG=3.860;AA=C;HA=19.7;HE=43.3;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=595;GESP=C:6295/T:6711;PPI=SMN1:984|SMN2:983|GEMIN4:973|GEMIN2:933|AGO2:885|GEMIN5:836|PRRC2A:786|HDAC5:716;PAC=NP_009135.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:97,21,0 0/0:5:0,6,43</p>

<p>1 205801872 rs1361754 A G 26.70 lowCVG AC1=5;AF1=0.6576;DB;DP=6;DP4=2,0,0,3;FQ=-9.17;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I380T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PM20D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152491.4;VDB=0.0040;DN=88;DA=A/G;GM=NM_152491.4;GL=PM20D1;FG=missense;FD=missense;AAC=ILE/THR;PP=380/503;CDP=1139;GS=89;PH=0.016;CP=0.270;CG=-0.108;AA=A;HA=33.9;HE=42.0;HC=0.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=23;GESP=G:6267/A:6739;PAC=NP_689704.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:41,6,0 1/1:4:26,3,0</p>

<p>1 248039451 rs3811444 C T 107 highCVG AC1=4;AF1=0.5108;DB;DP=24;DP4=8,4,4,7;FQ=110;MQ=60;PV4=0.22,0.14,1,0.15;SNPEFF_AMINO_ACID_CHANGE=T374M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM58;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015431.3;VDB=0.0144;DN=107;DA=C/T;GM=NM_015431.3,XM_005273091.1,XM_005273092.1;GL=TRIM58;FG=missense;FD=missense,unknown,unknown;AAC=THR/MET;PP=374/487,230/343,230/343;CDP=1121,689,689;GS=81;PH=0.999;CP=0.011;CG=2.210;AA=C;HA=3.3;HE=33.2;HC=23.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=250;GESP=T:3365/C:9641;PAC=NP_056246.3,XP_005273148.1,XP_005273149.1 GT:GQ:PL 0/0:12:0,15,107 0/1:13:19,0,11 1/1:21:112,24,0 0/1:27:24,0,91</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>2 26536694 rs2052937 C T 46.40 midCVG AC1=4;AF1=0.5336;DB;DP=10;DP4=5,0,1,4;FQ=47.7;MQ=60;PV4=0.048,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=A404T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR113;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145168.1;VDB=0.0034;DN=94;DA=C/T;GM=NM_001145168.1,NM_001145169.1,NM_153835.3,XM_005264163.1;GL=GPR113;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=404/1080,335/998,205/874,335/1004;CDP=1210,1003,613,1003;GS=58;PH=0.066;CP=0.007;CG=3.470;AA=T;HA=34.1;HE=32.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=67;GESP=T:8972/C:4034;PAC=NP_001138640.1,NP_001138641.1,NP_722577.2,XP_005264220.1 GT:GQ:PL 0/0:7:0,9,60 0/1:35:49,0,32 1/1:3:26,3,0 1/1:3:15,3,0</p>

<p>2 29295186 rs75276619 C T 29.50 midCVG AC1=3;AF1=0.3733;DB;DP=12;DP4=4,4,3,1;FQ=31.5;MQ=60;PV4=0.58,0.41,1,0.46;SNPEFF_AMINO_ACID_CHANGE=A648T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf71;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001029883.2;VDB=0.0002;DN=131;DA=C/T;GM=NM_001029883.2;GL=C2orf71;FG=missense;FD=missense;AAC=ALA/THR;PP=648/1289;CDP=1942;GS=58;PH=0.2;CP=0.000;CG=-3.320;AA=C;DV=by-frequency,by-cluster;DSP=1727;GESP=T:113/C:12423;PAC=NP_001025054.1 GT:GQ:PL 0/1:11:10,0,51 0/0:9:0,9,67 1/1:5:64,9,0 0/0:7:0,6,37</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 96993757 rs2279105 C T 69.10 lowCVG AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3 GT:GQ:PL 0/1:16:12,0,21 0/1:4:0,3,17 1/1:4:21,3,0 1/1:12:80,12,0</p>

<p>2 120885310 rs1034489 G A 53.80 highCVG AC1=4;AF1=0.5281;DB;DP=21;DP4=0,9,6,2;FQ=57.1;MQ=60;PV4=0.0023,0.00019,1,1;SNPEFF_AMINO_ACID_CHANGE=A462T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020909.3;VDB=0.0023;DN=86;DA=G/A;GM=NM_001184937.1,NM_020909.3,XM_005263719.1,XM_005263720.1,XM_005263721.1,XM_005263722.1;GL=EPB41L5;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=462/688,462/734,465/737,465/736,462/734,390/662;CDP=1384,1384,1393,1393,1384,1168;GS=58;PH=0;CP=0.973;CG=1.730;AA=A;HA=41.4;HE=26.4;HC=32.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$ROAZ_01;GESP=A:8169/G:4837;PAC=NP_001171866.1,NP_065960.2,XP_005263776.1,XP_005263777.1,XP_005263778.1,XP_005263779.1 GT:GQ:PL 0/1:35:32,0,51 0/1:3:0,3,21 1/1:5:32,6,0 0/1:36:33,0,69</p>

<p>2 185802243 rs1366842 C A 30.40 highCVG AC1=2;AF1=0.2953;DB;DP=17;DP4=3,8,4,0;FQ=31.8;MQ=60;PV4=0.026,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T707K;SNPEFF_CODON_CHANGE=aCa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF804A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_194250.1;VDB=0.0182;DN=88;DA=C/A;GM=NM_194250.1;GL=ZNF804A;FG=missense;FD=missense;AAC=THR/LYS;PP=707/1210;CDP=2120;GS=78;PH=0.002;CP=0.000;CG=1.860;AA=C;HA=4.5;HE=40.1;HC=13.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1734;GESP=A:5568/C:7416;PAC=NP_919226.1 GT:GQ:PL 0/0:6:0,3,26 0/0:29:0,27,124 1/1:6:73,12,0 0/0:6:0,3,28</p>

<p>2 186656956 rs17229201 T C 27.70 lowCVG AC1=5;AF1=0.6553;DB;DP=5;DP4=2,0,2,1;FQ=12.6;MQ=60;PV4=1,0.22,1,0.25;SNPEFF_AMINO_ACID_CHANGE=M1787T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FSIP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173651.2;VDB=0.0061;DN=123;DA=T/C;GM=NM_173651.2;GL=FSIP2;FG=missense;FD=missense;AAC=MET/THR;PP=1787/6997;CDP=5360;GS=81;CP=0.371;CG=3.390;AA=C;HA=42.4;HE=48.8;HC=46.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3587;PAC=NP_775922.2 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,48 1/1:4:21,3,0 1/1:5:47,6,0</p>

<p>2 220353532 rs13026308 C A 58.90 midCVG AC1=6;AF1=0.7369;DB;DP=11;DP4=1,1,5,1;FQ=-4.99;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2687T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=34;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005876.4;VDB=0.0094;DN=121;DA=C/A;GM=NM_005876.4,XM_005246235.1,XM_005246236.1,XM_005246237.1,XM_005246238.1,XM_005246239.1,XM_005246240.1,XM_005246241.1,XM_005246242.1;GL=SPEG;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/THR;PP=2687/3268,2628/3209,2626/3207,2593/3174,2534/3115,1895/2476,1838/2419,1838/2419,1900/2481;CDP=8059,7882,7876,7777,7600,5683,5512,5512,5698;GS=38;PH=0.997;CP=0.999;CG=4.380;AA=C;HA=0.0;HE=16.1;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;GESP=A:1183/C:11149;PAC=NP_005867.3,XP_005246292.1,XP_005246293.1,XP_005246294.1,XP_005246295.1,XP_005246296.1,XP_005246297.1,XP_005246298.1,XP_005246299.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:9:54,9,0 1/1:9:48,9,0</p>

<p>2 231865090 rs2271376 T C 84.80 midCVG AC1=4;AF1=0.5465;DB;DP=9;DP4=1,2,4,2;FQ=5.61;MQ=60;PV4=0.52,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=I104T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139073.3;VDB=0.0134;DN=100;DA=T/C;GM=NM_139073.3,XM_005246303.1;GL=SPATA3;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=104/193;CDP=311;GS=89;PH=0;CP=0.001;CG=-0.717;AA=C;HA=48.8;HE=35.5;HC=40.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;GESP=C:1865/T:2701;PAC=NP_620712.2,XP_005246360.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,16 1/1:5:47,6,0 0/1:27:77,0,24</p>

<p>2 237149941 rs6756597 C T 24.40 highCVG AC1=2;AF1=0.254;DB;DP=18;DP4=7,5,4,0;FQ=25.6;MQ=60;PV4=0.24,0.36,1,0.041;SNPEFF_AMINO_ACID_CHANGE=A104T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ASB18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_212556.2;VDB=0.0223;DN=116;DA=C/T;GM=NM_212556.2;GL=ASB18;FG=missense;FD=missense;AAC=ALA/THR;PP=104/467;CDP=310;GS=58;PH=0.76;CP=0.928;CG=4.820;AA=C;HA=29.5;HE=46.2;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19;TFBS=V$P53_01,V$SRF_Q6;GESP=T:5089/C:6967;PAC=NP_997721.2 GT:GQ:PL 0/0:6:0,3,20 0/0:23:0,21,119 1/1:5:68,12,0 0/0:14:0,12,79</p>

<p>3 45637026 . C A 59.40 midCVG AC1=4;AF1=0.5;DP=14;DP4=1,3,0,9;FQ=9.86;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P219T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIMD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014240.2;VDB=0.0059;GM=NM_014240.2;GL=LIMD1;FG=missense;FD=unknown;AAC=PRO/THR;PP=219/677;CDP=655;GS=38;PH=0.004;CP=0.002;CG=-0.748;AA=C;DSP=754;GESP=C:13006;PAC=NP_055055.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:97,9,0 0/0:8:0,9,44</p>

<p>3 57136593 rs368070640 A G 14.20 midCVG AC1=3;AF1=0.3818;DB;DP=12;DP4=6,0,4,0;FQ=16.2;MQ=60;PV4=1,0.044,1,1;SNPEFF_AMINO_ACID_CHANGE=I298T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=138;DA=A/G;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=298/740,274/716,270/712,154/596;CDP=893,821,809,461;GS=89;PH=0.986;CP=1.000;CG=5.480;AA=A;DSP=25;TFBS=V$GFI1_01;GESP=A:13006;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/0:7:0,6,34 0/0:9:0,9,61 1/1:5:45,9,0 0/1:14:14,0,19</p>

<p>3 57414071 rs4681982 G A 107 highCVG AC1=4;AF1=0.4999;DB;DP=19;DP4=3,7,5,4;FQ=110;MQ=60;PV4=0.37,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=T1786I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0121;DN=111;DA=G/A;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=THR/ILE;PP=1763/3093;CDP=5288;GS=89;PH=0;CP=0.003;CG=-0.547;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6;GESP=G:4506;PAC=NP_848599.3 GT:GQ:PL 0/1:32:30,0,41 0/0:5:0,6,44 1/1:5:50,6,0 0/1:74:71,0,91</p>

<p>3 66419956 rs13874 C T 145 highCVG AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA GT:GQ:PL 0/0:7:0,9,70 0/1:3:0,0,0 1/1:18:112,21,0 0/1:31:77,0,28</p>

<p>3 69245454 rs9310141 T C 6.32 lowCVG AC1=8;AF1=1;DB;DP=5;DP4=0,0,2,0;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T396A;SNPEFF_CODON_CHANGE=Acg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMD4B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015123.2;VDB=0.0175;DN=119;DA=T/C;GM=NM_015123.1,XM_005264720.1,XM_005264721.1,XM_005264722.1,XM_005264723.1;GL=FRMD4B;FG=missense;FD=unknown;AAC=THR/ALA;PP=396/1035,365/1004,363/1002,351/990,48/687;CDP=1186,1093,1087,1051,142;GS=58;PH=0;CP=0.803;CG=6.070;AA=C;HA=13.4;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;GESP=C:11705/T:393;PAC=NP_055938.1,XP_005264777.1,XP_005264778.1,XP_005264779.1,XP_005264780.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 1/1:4:25,3,0</p>

<p>3 98512540 rs28489284 G A 55.60 midCVG AC1=3;AF1=0.4126;DB;DP=12;DP4=5,1,1,4;FQ=57.2;MQ=60;PV4=0.08,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A364T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ST3GAL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271145.1;VDB=0.0070;DN=125;DA=G/A;GM=NM_001271142.1,NM_001271145.1,NM_001271146.1,NM_001271147.1,NM_001271148.1,NM_006100.3,XM_005247067.1,XM_005247068.1,XM_005247069.1,XM_005247070.1,XM_005247071.1,XM_005247072.1;GL=ST3GAL6;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=225/246,364/385,311/332,193/214,167/188,311/332,334/355,333/354,311/332,311/332,167/188,167/188;CDP=673,1090,931,577,499,931,1000,997,931,931,499,499;GS=58;PH=0.934;CP=1.000;CG=5.930;AA=G;HA=13.0;HE=2.9;HC=2.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa00601:GlycosphingolipidBiosynthesis-LactoAndNeolactoSeries|hsa01100:MetabolicPathways;GESP=A:682/G:12324;PAC=NP_001258071.1,NP_001258074.1,NP_001258075.1,NP_001258076.1,NP_001258077.1,NP_006091.1,XP_005247124.1,XP_005247125.1,XP_005247126.1,XP_005247127.1,XP_005247128.1,XP_005247129.1 GT:GQ:PL 0/0:4:0,3,25 0/0:9:0,9,53 1/1:5:69,9,0 0/1:30:31,0,30</p>

<p>3 121208833 rs3218649 G C 208 highCVG AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3 GT:GQ:PL 0/1:37:59,0,36 0/0:43:0,48,136 1/1:20:109,21,0 1/1:11:91,12,0</p>

<p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>3 194062519 rs3732477 C T 62.10 highCVG AC1=3;AF1=0.3624;DB;DP=16;DP4=0,8,1,5;FQ=64;MQ=60;PV4=0.43,0.0087,1,1;SNPEFF_AMINO_ACID_CHANGE=A305T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080513.3;VDB=0.0065;DN=107;DA=C/T;GM=NM_001080513.2,XM_005269280.1,XM_005269281.1;GL=CPN2;FG=missense;FD=missense,unknown,unknown;AAC=ALA/THR;PP=305/546;CDP=913;GS=58;PH=0.017;CP=0.000;CG=-10.600;AA=C;HA=44.2;HE=26.5;HC=32.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=913;GESP=T:4238/C:8768;PAC=NP_001073982.2,XP_005269337.1,XP_005269338.1 GT:GQ:PL 0/0:9:0,9,59 0/0:12:0,12,79 1/1:5:63,9,0 0/1:19:45,0,13</p>

<p>3 195452799 rs2550232 C T 11.30 midCVG AC1=8;AF1=1;DB;DP=9;DP4=4,1,1,3;FQ=-26.1;MQ=21;PV4=0.21,0.34,1,0.42;SNPEFF_AMINO_ACID_CHANGE=T442I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282506.1;VDB=0.0102;DN=100;DA=C/T;GM=NM_001098516.1,NM_152673.2,XM_005269303.1,XM_005269304.1,XM_005269305.1,XM_005269306.1,XM_005269307.1,XM_005269308.1,XM_005269309.1;GL=MUC20;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ILE;PP=236/504,271/539,442/727,442/710,407/692,407/692,407/692,271/553,272/540;CDP=707,812,1325,1325,1220,1220,1220,812,815;GS=89;PH=0.002;CP=0.000;CG=-2.630;AA=C;DG;DV=by-cluster,by-2hit-2allele;DSP=131;GESP=C:12316;PAC=NP_001091986.1,NP_689886.2,XP_005269360.1,XP_005269361.1,XP_005269362.1,XP_005269363.1,XP_005269364.1,XP_005269365.1,XP_005269366.1 GT:GQ:PL 0/0:2:0,6,0/1:3:16,0,0 1/1:4:20,3,0 1/1:4:14,3,0</p>

<p>4 71068489 rs3196714 T C 34.90 lowCVG AC1=5;AF1=0.5673;DB;DP=7;DP4=0,2,0,4;FQ=7.37;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=I222T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODAM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017855.3;VDB=0.0087;DN=105;DA=T/C;GM=NM_017855.3,XM_005265698.1;GL=ODAM;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=222/280,239/297;CDP=665,716;GS=89;PH=0.007;CP=0.000;CG=0.676;AA=T;HA=15.6;HE=26.6;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=C:3310/T:9690;PAC=NP_060325.3,XP_005265755.1 GT:GQ:PL 0/1:15:13,0,17 0/1:3:0,0,0 1/1:7:62,9,0 0/1:3:0,3,26</p>

<p>4 100504664 rs3816873 T C 35.40 lowCVG AC1=6;AF1=0.695;DB;DP=7;DP4=1,1,3,2;FQ=10.9;MQ=60;PV4=1,0.018,1,0.15;SNPEFF_AMINO_ACID_CHANGE=I155T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTTP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300785.1;VDB=0.0031;DN=107;DA=T/C;GM=NM_000253.2,XM_005263025.1;GL=MTTP;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=128/895,155/922;CDP=383,464;GS=89;PH=0.007;CP=0.810;CG=4.100;AA=T;HA=22.5;HE=25.3;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=11;GESP=C:3415/T:9587;PPI=APOB:846;PAC=NP_000244.2,XP_005263082.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:11:61,12,0 1/1:4:16,3,0</p>

<p>5 140023238 rs17208187 C G 65.40 midCVG AC1=5;AF1=0.6275;DB;DP=12;DP4=3,1,1,6;FQ=18.7;MQ=60;PV4=0.088,1,1,0.029;SNPEFF_AMINO_ACID_CHANGE=T305S;SNPEFF_CODON_CHANGE=aCc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMCO6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300980.1;VDB=0.0001;DN=123;DA=C/G;GM=NM_018502.3,XM_005268476.1,XM_005268477.1;GL=TMCO6;FG=missense;FD=missense,unknown,unknown;AAC=THR/SER;PP=299/494,305/500,300/495;CDP=896,914,899;GS=58;PH=0.001;CP=0.341;CG=2.370;AA=C;HA=7.1;HE=26.0;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=23;GESP=G:2322/C:9960;PAC=NP_060972.3,XP_005268533.1,XP_005268534.1 GT:GQ:PL 0/1:16:11,0,28 0/0:4:0,6,32 1/1:4:16,3,0 1/1:14:84,15,0</p>

<p>5 140572763 rs2907323 C G 71.10 midCVG AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1 GT:GQ:PL 0/0:10:0,12,73 0/0:5:0,6,45 1/1:9:74,12,0 1/1:4:46,6,0</p>

<p>6 52617731 rs2180314 C G 37.70 midCVG AC1=4;AF1=0.5406;DB;DP=9;DP4=4,0,3,1;FQ=23;MQ=60;PV4=1,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S112T;SNPEFF_CODON_CHANGE=aGt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTA2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000846.4;VDB=0.0058;DN=96;DA=C/G;GM=NM_000846.4;GL=GSTA2;FG=missense;FD=missense;AAC=SER/THR;PP=112/223;CDP=335;GS=58;PH=0;CP=0.000;CG=-4.510;AA=C;HA=22.9;HE=43.0;HC=33.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=G:6507/C:6499;PPI=GSTM2:937;PAC=NP_000837.3 GT:GQ:PL 0/0:3:0,3,27 0/1:3:0,0,1/1:5:48,6,0 0/1:30:29,0,32</p>

<p>6 118886961 rs3734382 G T 108 midCVG AC1=5;AF1=0.5972;DB;DP=14;DP4=1,3,6,2;FQ=43.4;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P254T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CEP85L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001178035.1;VDB=0.0162;DN=107;DA=G/T;GM=NM_001042475.2,NM_001178035.1,NM_206921.2,XM_005266970.1,XM_005266971.1;GL=CEP85L;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/THR;PP=251/806,254/809,251/497,149/704,149/704;CDP=751,760,751,445,445;GS=38;PH=0;CP=0.066;CG=-4.210;AA=T;HA=17.8;HE=26.0;HC=31.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=270;GESP=T:9623/G:3383;PAC=NP_001035940.1,NP_001171506.1,NP_996804.2,XP_005267027.1,XP_005267028.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:16:112,18,0 0/1:42:38,0,58</p>

<p>7 99306685 rs2257401 C G 76 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:64:67,0,63</p>

<p>7 101200727 rs17393069 G A 47.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=2,0,4,1;FQ=9.59;MQ=60;PV4=1,0.015,1,0.29;VDB=0.0078;DN=123;DA=G/A;GM=NM_001278563.1,NM_133457.3;GL=COL26A1;FG=missense;FD=unknown,missense;AAC=ALA/THR;PP=415/442,413/440;CDP=1242,1236;GS=58;PH=0.226;CP=0.199;CG=-0.864;AA=G;HA=45.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=77;GESP=A:4643/G:7635;PAC=NP_001265492.1,NP_597714.2 GT:GQ:PL 0/0:3:0,6,46 0/1:3:0,0,0 1/1:9:54,9,0 1/1:6:36,6,0</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 97614625 rs3816208 G A 21 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1 GT:GQ:PL 0/0:9:0,6,45 0/0:18:0,15,111 1/1:3:63,9,0 0/0:9:0,6,47</p>

<p>8 144801593 rs60732298 C A 16.80 lowCVG AC1=5;AF1=0.6552;DB;DP=5;DP4=2,0,3,0;FQ=7.18;MQ=60;PV4=1,0.027,1,1;SNPEFF_AMINO_ACID_CHANGE=T221K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAPK15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139021.2;VDB=0.0066;DN=129;DA=C/A;GM=NM_139021.2;GL=MAPK15;FG=missense;FD=missense;AAC=THR/LYS;PP=221/545;CDP=662;GS=78;PH=0.992;CP=0.104;CG=4.140;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;GESP=A:1162/C:11844;PAC=NP_620590.2 GT:GQ:PL 0/0:4:0,6,42 0/1:3:0,0,0 1/1:3:20,3,0 1/1:5:37,6,0</p>

<p>9 33387047 rs4008658 T G 66.90 highCVG AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:27:37,0,25 0/1:17:22,0,15 1/1:10:52,12,0 0/1:4:0,2,50</p>

<p>9 107267352 rs7847413 T C 13.20 midCVG AC1=2;AF1=0.2756;DB;DP=14;DP4=2,9,3,0;FQ=14.5;MQ=60;PV4=0.027,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=I270T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0024;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=ILE/THR;PP=270/320;CDP=809;GS=89;PH=0.016;CP=0.942;CG=1.920;AA=T;HA=40.2;HE=8.4;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=242782;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFAT_Q6;GESP=C:2246/T:10760;PAC=NP_001004485.1 GT:GQ:PL 0/0:14:0,12,74 0/0:23:0,21,130 1/1:3:54,9,0 0/0:4:0,0,0</p>

<p>9 125330678 rs10985704 T G 52.30 midCVG AC1=5;AF1=0.6368;DB;DP=9;DP4=2,0,6,0;FQ=-3.17;MQ=60;PV4=1,0.0061,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T27P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0026;DN=120;DA=T/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=THR/PRO;PP=27/310;CDP=79;GS=38;PH=0;CP=0.733;CG=3.480;AA=G;HA=43.0;HE=49.1;HC=1.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57611;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:7411/T:5595;PAC=NP_001004454.1 GT:GQ:PL 0/1:4:0,3,25 0/1:14:37,0,13 1/1:4:20,3,0 1/1:6:38,6,0</p>

<p>9 130165973 rs34064803 T C 79.40 highCVG AC1=3;AF1=0.4231;DB;DP=25;DP4=0,4,2,15;FQ=51.8;MQ=60;PV4=1,0.0002,1,0.08;SNPEFF_AMINO_ACID_CHANGE=I253T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC2A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014580.4;VDB=0.0144;DN=126;DA=T/C;GM=NM_001271711.1,NM_001271712.1,NM_014580.4,NR_073416.1,XM_005251949.1;GL=SLC2A8;FG=missense,missense,missense,non-coding-exon,missense;FD=missense,missense,missense,nc-transcript-variant,unknown;AAC=ILE/THR,ILE/THR,ILE/THR,none,ILE/THR;PP=253/412,90/315,253/478,NA,90/315;CDP=758,269,758,284,269;GS=89,89,89,NA,89;PH=0.878;CP=0.927;CG=5.420;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=C:1021/T:11985;PAC=NP_001258640.1,NP_001258641.1,NP_055395.2,NA,XP_005252006.1 GT:GQ:PL 0/0:6:0,6,47 0/1:3:0,0,0 1/1:46:122,51,0 0/0:6:0,6,45</p>

<p>9 134401335 rs2296957 C T 63.50 midCVG AC1=4;AF1=0.5532;DB;DP=9;DP4=1,2,1,5;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A180T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135954.1;VDB=0.0243;DN=100;DA=C/A/T;GM=NM_001135954.1,NM_001261450.1,NM_001261451.1,NM_031432.2,XM_005272224.1,XM_005272225.1;GL=UCK1;FG=missense,synonymous,synonymous,synonymous,synonymous,synonymous;FD=missense,synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown;AAC=ALA/THR,PRO,PRO,PRO,PRO,PRO;PP=180/202,202/269,216/283,211/278,225/292,188/255;CDP=538,606,648,633,675,564;GS=58,NA,NA,NA,NA,NA;PH=0.104;CP=0.908;CG=-5.080;AA=C;HA=41.2;HE=8.1;HC=7.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=20;KP=hsa00240:PyrimidineMetabolism|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;TFBS=V$SREBP1_02;GESP=T:10799/C:2207;PAC=NP_001129426.1,NP_001248379.1,NP_001248380.1,NP_113620.1,XP_005272281.1,XP_005272282.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,60 1/1:15:103,18,0 0/1:3:0,0,0</p>

<p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

<p>10 91007360 rs1051338 T G 24.80 lowCVG AC1=4;AF1=0.5432;DB;DP=5;DP4=1,1,0,3;FQ=4.88;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T16P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000235.3;VDB=0.0127;DN=86;DA=T/G;GM=NM_000235.2,NM_001127605.1;GL=LIPA;FG=missense;FD=missense;AAC=THR/PRO;PP=16/400;CDP=46;GS=38;PH=0.002;CP=0.997;CG=3.720;AA=T;HA=8.8;HE=33.0;HC=28.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=47;KP=hsa00100:SteroidBiosynthesis|hsa04142:Lysosome;GESP=G:3134/T:9872;PAC=NP_000226.2,NP_001121077.1 GT:GQ:PL 0/1:19:20,0,18 0/1:3:0,0,0 1/1:5:44,6,0 0/0:3:0,3,21</p>

<p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

<p>11 6007613 rs4354673 T G 168 highCVG AC1=6;AF1=0.6957;DB;DP=18;DP4=0,3,5,9;FQ=4.33;MQ=48;PV4=0.51,1,0.054,1;SNPEFF_AMINO_ACID_CHANGE=K183T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0083;DN=111;DA=T/G;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=LYS/THR;PP=183/330;CDP=548;GS=78;PH=0.549;CP=0.001;CG=-0.167;AA=T;HA=44.5;HE=13.2;HC=10.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=166102;KP=hsa04740:OlfactoryTransduction;GESP=G:8663/T:3675;PAC=NP_001005173.2 GT:GQ:PL 0/1:11:38,0,11 0/1:3:0,6,34 1/1:15:87,15,0 1/1:15:91,15,0</p>

<p>11 17632651 rs7111528 C T 102 midCVG AC1=5;AF1=0.5888;DB;DP=10;DP4=1,2,4,3;FQ=15.1;MQ=60;PV4=1,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=T1947M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0076;DN=116;DA=C/T;GM=NM_001277269.1,XM_005252894.1;GL=OTOG;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=1947/2926,953/1509;CDP=5840,2858;GS=81;PH=0.011;CP=0.000;CG=-0.815;AA=T;HA=48.7;HE=42.3;HC=17.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=320;PAC=NP_001264198.1,XP_005252951.1 GT:GQ:PL 0/0:4:0,6,33 0/1:3:0,3,25 1/1:5:50,6,0 1/1:13:99,15,0</p>

<p>11 56468047 rs2865519 A G 112 highCVG AC1=5;AF1=0.5869;DB;DP=23;DP4=3,4,8,7;FQ=50.7;MQ=40;PV4=1,1,0.0065,0.36;SNPEFF_AMINO_ACID_CHANGE=T62A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=101;DA=A/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=THR/ALA;PP=62/306;CDP=184;GS=58;PH=0.004;CP=0.961;CG=4.540;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482111;TFBS=V$POU3F2_01;GESP=A:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:28:24,0,64 0/1:11:62,0,9 1/1:13:70,15,0 0/1:4:0,2,18</p>

<p>11 59940599 rs7232 T A 17.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=0,3,1,1;FQ=18;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T213S;SNPEFF_CODON_CHANGE=Act/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MS4A6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001247999.1;VDB=0.0040;DN=52;DA=T/A;GM=NM_001247999.1,NM_022349.3,NM_152851.2,NM_152852.2,XM_005274177.1,XM_005274178.1,XM_005274179.1,XM_005274180.1,XM_005274181.1;GL=MS4A6A;FG=missense;FD=unknown,unknown,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=THR/SER,THR/SER,ASN/ILE,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER;PP=213/254,185/226,150/179,185/249,213/277,213/277,185/249,168/232,168/209;CDP=637,553,449,553,637,637,553,502,502;GS=58,58,149,58,58,58,58,58,58;PH=0.827;CP=0.001;CG=-5.110;AA=T;HA=1.8;HE=44.6;HC=8.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=4;GESP=A:3608/T:9384;PAC=NP_001234928.1,NP_071744.2,NP_690590.1,NP_690591.1,XP_005274234.1,XP_005274235.1,XP_005274236.1,XP_005274237.1,XP_005274238.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:54,6,0 0/0:9:0,9,55</p>

<p>11 61165741 rs10897158 G C 74.40 midCVG AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1 GT:GQ:PL 0/1:10:35,0,8 0/0:6:0,9,70 1/1:4:24,3,0 1/1:8:61,9,0</p>

<p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

<p>11 75152243 rs571353 C T 59.30 midCVG AC1=5;AF1=0.5871;DB;DP=10;DP4=4,1,4,1;FQ=40.4;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=A480T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDPD5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030792.6;VDB=0.0080;DN=83;DA=C/T;GM=NM_030792.6,XM_005274313.1,XM_005274314.1,XM_005274315.1,XM_005274316.1,XM_005274317.1,XM_005274318.1,XM_005274319.1;GL=GDPD5;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=480/606,480/607,480/607,480/607,480/607,480/606,235/362,235/361;CDP=1438,1438,1438,1438,1438,1438,703,703;GS=58;PH=0;CP=0.014;CG=-0.094;HA=44.1;HE=26.6;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8514/C:4472;PAC=NP_110419.5,XP_005274370.1,XP_005274371.1,XP_005274372.1,XP_005274373.1,XP_005274374.1,XP_005274375.1,XP_005274376.1 GT:GQ:PL 0/1:7:30,0,4 0/0:9:0,12,77 1/1:4:26,3,0 1/1:5:48,6,0</p>

<p>11 123624658 rs12364099 G T 84.70 midCVG AC1=4;AF1=0.5037;DB;DP=14;DP4=1,3,8,1;FQ=24.8;MQ=60;PV4=0.052,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T190N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6X1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005188.1;VDB=0.0009;DN=120;DA=G/T;GM=NM_001005188.1;GL=OR6X1;FG=missense;FD=missense;AAC=THR/ASN;PP=190/313;CDP=569;GS=65;PH=0.029;CP=0.386;CG=3.460;AA=G;HA=1.7;HE=14.0;HC=2.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12402;KP=hsa04740:OlfactoryTransduction;TFBS=V$YY1_01;GESP=T:1529/G:11473;PAC=NP_001005188.1 GT:GQ:PL 0/1:19:84,0,15 0/1:3:0,0,0 1/1:4:41,6,0 0/0:5:0,6,47</p>

<p>12 9307373 rs2377741 T G 29.10 midCVG AC1=6;AF1=0.7339;DB;DP=14;DP4=0,2,5,1;FQ=-4.32;MQ=50;PV4=0.11,0.007,0.18,1;SNPEFF_AMINO_ACID_CHANGE=T1205P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PZP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002864.2;VDB=0.0199;DN=100;DA=T/G;GM=NM_002864.2,XM_005253459.1;GL=PZP;FG=missense;FD=missense,unknown;AAC=THR/PRO;PP=1205/1483,991/1269;CDP=3613,2971;GS=38;PH=0;CP=0.039;CG=0.627;AA=G;HA=4.9;HE=0.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:12716/T:290;PAC=NP_002855.2,XP_005253516.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:14:60,15,0 0/1:12:6,0,29</p>

<p>12 32135186 rs3759302 T A 62.90 midCVG AC1=4;AF1=0.5521;DB;DP=9;DP4=2,1,1,4;FQ=12.7;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S433T;SNPEFF_CODON_CHANGE=Tct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0034;DN=107;DA=T/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=SER/THR;PP=433/1748;CDP=1297;GS=58;PH=0.999;CP=0.001;CG=-1.130;AA=T;HA=3.4;HE=18.1;HC=11.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1375;GESP=A:2107/T:10899;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:3:0,3,16 0/0:5:0,6,38 1/1:3:26,3,0 1/1:10:82,12,0</p>

<p>12 40758652 rs3761863 T C 14.60 lowCVG AC1=6;AF1=0.6706;DB;DP=5;DP4=1,0,2,1;FQ=-12.1;MQ=60;PV4=1,0.14,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M2397T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=49;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198578.3;VDB=0.0050;DN=107;DA=T/C;GM=NM_198578.3,XM_005268629.1,XM_005268630.1,XM_005268631.1;GL=LRRK2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=MET/THR;PP=2397/2528,2397/2522,2397/2528,2357/2488;CDP=7190,7190,7190,7070;GS=81;PH=0;CP=0.005;CG=3.030;AA=C;HA=46.5;HE=34.3;HC=42.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=9;KP=hsa05012:Parkinson'sDisease;GESP=C:7962/T:5042;PPI=STUB1:846;PAC=NP_940980.3,XP_005268686.1,XP_005268687.1,XP_005268688.1 GT:GQ:PL 0/1:20:30,0,18 0/1:3:0,0,0 1/1:3:19,3,0 0/1:3:0,0,0</p>

<p>12 42863266 rs79087668 C T 63.10 midCVG AC1=5;AF1=0.5836;DB;DP=10;DP4=2,2,2,4;FQ=39.8;MQ=60;PV4=1,0.25,1,0.22;SNPEFF_AMINO_ACID_CHANGE=A124T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRICKLE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144881.1;VDB=0.0077;DN=131;DA=C/T;GM=NM_001144881.1,NM_001144882.1,NM_001144883.1,NM_153026.2;GL=PRICKLE1;FG=missense;FD=missense;AAC=ALA/THR;PP=124/832;CDP=370;GS=58;PH=0.002;CP=0.927;CG=3.550;AA=C;DV=by-frequency,by-cluster;DSP=15;KP=hsa04310:WntSignalingPathway;GESP=T:22/C:12984;PAC=NP_001138353.1,NP_001138354.1,NP_001138355.1,NP_694571.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,26 1/1:10:83,12,0 0/1:25:21,0,51</p>

<p>12 49952449 . C T 14.40 lowCVG AC1=4;AF1=0.5125;DP=6;DP4=3,0,0,3;FQ=12.6;MQ=60;PV4=0.1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A469T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCRS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001012300.1;VDB=0.0104;GM=NM_001012300.1,NM_001278341.1,NM_006337.4,XM_005268572.1,XM_005268573.1;GL=MCRS1;FG=missense;FD=unknown;AAC=ALA/THR;PP=469/476,265/272,456/463,456/463,367/374;CDP=1405,793,1366,1366,1099;GS=58;PH=0.997;CP=1.000;CG=5.280;AA=C;DSP=64;TFBS=V$PAX4_01;GESP=C:13006;PPI=INO80E:892|DAXX:846|INO80:811|PBX2:786;PAC=NP_001012300.1,NP_001265270.1,NP_006328.2,XP_005268629.1,XP_005268630.1 GT:GQ:PL 0/1:7:5,0,32 0/1:3:0,0,0 1/1:4:47,6,0 0/0:4:0,3,17</p>

<p>12 53165940 rs6580904 C T 76.30 midCVG AC1=5;AF1=0.5683;DB;DP=11;DP4=2,1,5,3;FQ=13.2;MQ=60;PV4=1,0.37,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A359T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015848.4;VDB=0.0153;DN=116;DA=C/T;GM=NM_015848.4;GL=KRT76;FG=missense;FD=missense;AAC=ALA/THR;PP=359/639;CDP=1075;GS=58;PH=0.986;CP=0.890;CG=4.420;AA=C;HA=39.2;HE=5.8;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=T:9195/C:3811;PAC=NP_056932.2 GT:GQ:PL 0/1:3:0,3,15 0/1:15:12,0,21 1/1:13:85,15,0 0/1:19:24,0,18</p>

<p>13 51854694 . C A 42.70 highCVG AC1=3;AF1=0.3358;DP=25;DP4=6,7,12,0;FQ=44.4;MQ=60;PV4=0.0052,0.15,1,1;SNPEFF_AMINO_ACID_CHANGE=P351T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM124A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145019.3;VDB=0.0000;GM=NM_001242312.1,NM_145019.3;GL=FAM124A;FG=missense;FD=unknown;AAC=PRO/THR;PP=315/547,351/583;CDP=943,1051;GS=38;PH=0.034;CP=0.000;CG=0.795;AA=C;DSP=109;GESP=C:13006;PAC=NP_001229241.1,NP_659456.3 GT:GQ:PL 0/0:15:0,15,93 0/0:3:0,0,0 1/1:16:87,22,0 0/0:21:0,21,125</p>

<p>14 63174960 rs4902176 C T 17 highCVG AC1=4;AF1=0.4355;DB;DP=15;DP4=2,8,1,3;FQ=19.4;MQ=60;PV4=1,1,1,0.21;SNPEFF_AMINO_ACID_CHANGE=A745T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139318.4;VDB=0.0080;DN=111;DA=C/T;GM=NM_139318.4,NM_172375.2;GL=KCNH5;FG=missense,3-prime-UTR;FD=unknown;AAC=ALA/THR,none;PP=745/989,NA;CDP=2233,NA;GS=58,NA;PH=0.017;CP=0.997;CG=5.520;AA=C;HA=19.0;HE=34.4;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=214;GESP=T:3891/C:9115;PAC=NP_647479.2,NA GT:GQ:PL 0/0:14:0,15,81 0/1:13:12,0,62 1/1:3:46,6,0 0/1:3:0,0,0</p>

<p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

<p>14 74759477 rs4148077 C T 120 midCVG AC1=6;AF1=0.7273;DB;DP=13;DP4=3,0,6,3;FQ=8.59;MQ=60;PV4=0.51,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A304T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005050.3;VDB=0.0175;DN=110;DA=C/T;GM=NM_005050.3,NR_003256.2,XM_005267935.1,XM_005267936.1,XM_005267937.1,XM_005267938.1,XM_005267939.1,XM_005267940.1,XM_005267941.1,XM_005267942.1,XM_005267943.1,XM_005267944.1,XM_005267945.1,XM_005267946.1,XM_005267947.1,XM_005267948.1,XM_005267949.1,XM_005267950.1,XM_005267951.1,XM_005267952.1,XM_005267953.1,XM_005267954.1;GL=ABCD4;FG=missense,non-coding-exon,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR,none,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR;PP=304/607,NA,287/576,304/529,200/489,304/475,166/469,166/469,166/469,166/469,304/455,145/448,145/448,145/448,145/448,145/448,145/448,145/448,41/344,41/344,41/344,41/344;CDP=910,934,859,910,598,910,496,496,496,496,910,433,433,433,433,433,433,433,121,121,121,121;GS=58,NA,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58;PH=0.003;CP=0.998;CG=2.140;AA=C;HA=16.7;HE=29.2;HC=23.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=27;KP=hsa02010:ABCTransporters|hsa04146:Peroxisome;GESP=T:3999/C:9007;PAC=NP_005041.1,NA,XP_005267992.1,XP_005267993.1,XP_005267994.1,XP_005267995.1,XP_005267996.1,XP_005267997.1,XP_005267998.1,XP_005267999.1,XP_005268000.1,XP_005268001.1,XP_005268002.1,XP_005268003.1,XP_005268004.1,XP_005268005.1,XP_005268006.1,XP_005268007.1,XP_005268008.1,XP_005268009.1,XP_005268010.1,XP_005268011.1 GT:GQ:PL 0/1:18:12,0,27 0/1:18:17,0,20 1/1:16:88,15,0 1/1:8:48,6,0</p>

<p>14 88459795 rs73312836 C T 22.20 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=3,5,2,1;FQ=24.4;MQ=60;PV4=0.55,1,1,0.005;SNPEFF_AMINO_ACID_CHANGE=A17T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GALC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001201402.1;VDB=0.0092;DN=130;DA=C/T;GM=NM_001201402.1;GL=GALC;FG=missense;FD=missense;AAC=ALA/THR;PP=17/660;CDP=49;GS=58;CP=0.000;CG=-0.125;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=69;KP=hsa00600:SphingolipidMetabolism|hsa01100:MetabolicPathways|hsa04142:Lysosome;PAC=NP_001188331.1 GT:GQ:PL 0/0:15:0,15,85 0/1:4:20,3,0 1/1:3:48,6,0 0/0:9:0,9,64</p>

<p>16 56673227 rs11640851 C A 13.60 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=2,0,1,1;FQ=13.6;MQ=60;PV4=1,0.23,1,1;SNPEFF_AMINO_ACID_CHANGE=T27N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MT1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005946.2;VDB=0.0011;DN=120;DA=C/A;GM=NM_005946.2;GL=MT1A;FG=missense;FD=missense;AAC=THR/ASN;PP=27/62;CDP=80;GS=65;PH=0.928;CP=0.839;CG=1.960;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=A:6838/C:6158;PAC=NP_005937.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:49,6,0 0/0:7:0,6,49</p>

<p>16 75563330 rs3826107 G A 36.20 lowCVG AC1=5;AF1=0.6482;DB;DP=5;DP4=0,1,4,0;FQ=-9.91;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T318M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHST5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024533.4;VDB=0.0001;DN=107;DA=G/A;GM=NM_024533.4,XM_005255880.1;GL=CHST5;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=318/412,324/418;CDP=953,971;GS=81;PH=0.999;CP=0.994;CG=2.840;AA=G;HA=9.7;HE=15.6;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=906;CPG=75563114-75564112;GESP=A:2238/G:10758;PAC=NP_078809.2,XP_005255937.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:10:73,12,0 0/1:3:0,0,0</p>

<p>17 5087040 rs9908414 A G 60.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=4,4,0,5;FQ=63.2;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I171T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF594;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032530.1;VDB=0.0105;DN=119;DA=A/G;GM=NM_032530.1,XM_005256827.1;GL=ZNF594;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=171/808;CDP=512;GS=89;PH=0;CP=0.203;CG=-2.720;AA=A;HA=9.7;HE=47.3;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=532;GESP=A:12386;PAC=NP_115919.1,XP_005256884.1 GT:GQ:PL 0/0:16:0,18,107 0/0:6:0,6,37 1/1:6:70,9,0 1/1:4:39,6,0</p>

<p>17 26694861 rs704 G A 31.80 highCVG AC1=4;AF1=0.4318;DB;DP=15;DP4=3,7,1,3;FQ=34;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T400M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000638.3;VDB=0.0089;DN=36;DA=G/A;GM=NM_000638.3;GL=VTN;FG=missense;FD=missense;AAC=THR/MET;PP=400/479;CDP=1199;GS=81;PH=0.943;CP=0.000;CG=-8.940;AA=G;HA=35.0;HE=46.9;HC=25.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=126;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=A:6627/G:6379;PAC=NP_000629.3 GT:GQ:PL 0/0:12:0,12,67 0/1:3:0,0,0 1/1:5:69,9,0 0/1:5:3,0,95</p>

<p>17 27284443 rs12941884 A G 58.10 midCVG AC1=5;AF1=0.587;DB;DP=9;DP4=2,0,4,1;FQ=5.4;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=M806T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEZ6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178860.4;VDB=0.0169;DN=121;DA=A/G;GM=NM_001098635.1,NM_178860.4,XM_005257919.1;GL=SEZ6;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=806/994,806/995,681/870;CDP=2417,2417,2042;GS=81;PH=0;CP=0.999;CG=3.650;AA=G;HA=20.8;HE=14.6;HC=8.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=78;GESP=G:1806/A:10402;PAC=NP_001092105.1,NP_849191.3,XP_005257976.1 GT:GQ:PL 0/1:3:0,0,0 0/1:18:21,0,17 1/1:10:78,12,0 0/1:3:0,3,24</p>

<p>17 48598785 rs1380657 G A 174 highCVG AC1=6;AF1=0.7336;DB;DP=19;DP4=3,2,5,7;FQ=44.9;MQ=60;PV4=0.62,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=A424T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYCBPAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032133.4;VDB=0.0177;DN=88;DA=G/A;GM=NM_032133.4,XM_005257725.1,XM_005257726.1,XM_005257727.1;GL=MYCBPAP;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=424/985,439/1001,439/1000,424/986;CDP=1270,1315,1315,1270;GS=58;PH=0;CP=0.000;CG=-4.290;AA=A;HA=7.6;HE=27.6;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=A:10167/G:2839;PPI=MYCBP:846;PAC=NP_115509.4,XP_005257782.1,XP_005257783.1,XP_005257784.1 GT:GQ:PL 0/1:14:16,0,16 0/1:38:31,0,68 1/1:22:130,21,0 1/1:8:42,6,0</p>

<p>17 59544932 . C A 13.30 midCVG AC1=3;AF1=0.3359;DP=11;DP4=2,6,0,3;FQ=14.9;MQ=60;PV4=1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=P155T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018488.2;VDB=0.0039;GM=NM_018488.2,XM_005257835.1,XM_005257836.1,XM_005257837.1,XM_005257838.1;GL=TBX4;FG=missense;FD=unknown;AAC=PRO/THR;PP=155/546,174/566,174/565,155/547,155/547;CDP=463,520,520,463,463;GS=38;PH=1;CP=1.000;CG=5.770;AA=C;DSP=62;GESP=C:13006;PAC=NP_060958.2,XP_005257892.1,XP_005257893.1,XP_005257894.1,XP_005257895.1 GT:GQ:PL 0/0:6:0,3,19 0/0:3:0,0,0 1/1:4:53,9,0 0/0:22:0,21,120</p>

<p>17 74151720 rs201646108 C T 56.70 highCVG AC1=3;AF1=0.3694;DB;DP=26;DP4=6,12,3,3;FQ=58.7;MQ=59;PV4=0.63,1,0.041,0.25;SNPEFF_AMINO_ACID_CHANGE=A541T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RNF157;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052916.2;VDB=0.0188;DN=137;DA=C/T;GM=NM_052916.2,XM_005257006.1,XM_005257007.1,XM_005257008.1;GL=RNF157;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=541/680,541/658,541/654,474/613;CDP=1621,1621,1621,1420;GS=58;PH=1;CP=0.755;CG=5.430;AA=C;DV=by-cluster;DSP=78;GESP=T:5/C:13001;PAC=NP_443148.1,XP_005257063.1,XP_005257064.1,XP_005257065.1 GT:GQ:PL 0/0:9:0,9,60 0/1:29:29,0,78 1/1:7:75,12,0 0/0:24:0,24,130</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>19 4512669 rs55901236 T C 66.40 midCVG AC1=6;AF1=0.6769;DB;DP=10;DP4=0,2,6,1;FQ=-15;MQ=60;PV4=0.083,1,1,0.31;SNPEFF_AMINO_ACID_CHANGE=T421A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080400.1;VDB=0.0122;DN=129;DA=T/C;GM=NM_001080400.1,XM_005259640.1,XM_005259641.1;GL=PLIN4;FG=missense;FD=missense,unknown,unknown;AAC=THR/ALA;PP=421/1358,436/1373,435/1372;CDP=1261,1306,1303;GS=58;PH=0.009;CP=0.001;CG=0.087;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1045;TFBS=V$GR_Q6;GESP=C:5321/T:7027;PAC=NP_001073869.1,XP_005259697.1,XP_005259698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:17:76,0,15</p>

<p>19 9056982 rs11670318 G A 109 midCVG AC1=5;AF1=0.6629;DB;DP=11;DP4=1,2,6,2;FQ=5.06;MQ=60;PV4=0.49,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T10155I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0045;DN=120;DA=G/A;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=THR/ILE;PP=10155/14508,2069/6405,2069/6025;CDP=30464,6206,6206;GS=89;PH=0.997;CP=0.000;CG=0.467;AA=G;HA=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=810;GESP=A:3367/G:8779;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:30:47,0,29 0/1:3:0,0,0 1/1:9:61,9,0 0/1:11:41,0,10</p>

<p>19 12772165 rs1054487 G A 27.30 lowCVG AC1=3;AF1=0.3966;DB;DP=8;DP4=1,3,3,1;FQ=29.4;MQ=60;PV4=0.49,0.0045,1,1;SNPEFF_AMINO_ACID_CHANGE=T312I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAN2B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000528.3;VDB=0.0063;DN=86;DA=G/A/T;GM=NM_000528.3,NM_001173498.1,XM_005259913.1;GL=MAN2B1;FG=missense;FD=missense,missense,unknown;AAC=THR/ILE;PP=312/1012,312/1011,312/1013;CDP=935;GS=89;PH=0.003;CP=0.015;CG=-2.190;AA=A;HA=40.2;HE=38.7;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=A:5899/G:7107;PAC=NP_000519.2,NP_001166969.1,XP_005259970.1 GT:GQ:PL 0/0:5:0,3,25 0/1:3:0,0,0 1/1:7:68,12,0 0/0:9:0,9,72</p>

<p>19 21909948 rs7246400 G A 38.50 lowCVG AC1=5;AF1=0.6627;DB;DP=8;DP4=0,1,0,6;FQ=-6.85;MQ=34;PV4=1,0.01,1,0.36;SNPEFF_AMINO_ACID_CHANGE=T389I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF100;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173531.3;VDB=0.0289;DN=116;DA=G/A;GM=NM_173531.3,XM_005259783.1,XM_005259784.1,XM_005259785.1;GL=ZNF100;FG=missense;FD=missense,unknown,unknown,unknown;AAC=THR/ILE;PP=389/543,388/542,356/510,174/328;CDP=1166,1163,1067,521;GS=89;PH=0.804;CP=0.029;CG=0.841;AA=G;HA=30.4;HC=40.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=844;GESP=A:6476/G:6444;PAC=NP_775802.2,XP_005259840.1,XP_005259841.1,XP_005259842.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:16:76,18,0 0/1:3:0,3,27</p>

<p>19 37118439 rs61732180 C T 83.80 midCVG AC1=5;AF1=0.6451;DB;DP=9;DP4=3,0,3,3;FQ=15.4;MQ=60;PV4=0.46,1,1,0.002;SNPEFF_AMINO_ACID_CHANGE=T547M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0028;DN=129;DA=C/T;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=THR/MET;PP=546/550,547/551,547/551,546/550;CDP=1637,1640,1640,1637;GS=81;PH=0;CP=0.009;CG=1.990;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1408;GESP=T:2043/C:10963;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:17:27,0,16 0/0:4:0,6,41 1/1:9:59,9,0 1/1:6:44,6,0</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 48495304 rs60213124 G A 79 midCVG AC1=4;AF1=0.5008;DB;DP=12;DP4=3,3,3,3;FQ=79.1;MQ=60;PV4=1,0.42,1,0.17;SNPEFF_AMINO_ACID_CHANGE=T12M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BSPH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128326.2;VDB=0.0230;DN=129;DA=G/A;GM=NM_001128326.2;GL=BSPH1;FG=missense;FD=missense;AAC=THR/MET;PP=12/133;CDP=35;GS=81;PH=0.962;CP=0.000;CG=-0.642;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=G:4566;PAC=NP_001121798.1 GT:GQ:PL 0/0:13:0,15,105 0/1:3:27,3,0 1/1:12:99,15,0 0/0:3:0,3,21</p>

<p>19 49438363 rs2270941 G A 47.70 midCVG AC1=5;AF1=0.5687;DB;DP=10;DP4=0,4,2,3;FQ=37.1;MQ=60;PV4=0.44,0.036,1,1;SNPEFF_AMINO_ACID_CHANGE=S66N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHDH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014475.3;VDB=0.0081;DN=100;DA=G/A;GM=NM_014475.3,XM_005258748.1;GL=DHDH;FG=missense;FD=missense,unknown;AAC=SER/ASN,ALA/THR;PP=66/335,9/223;CDP=197,25;GS=46,58;PH=0;CP=0.282;CG=4.740;AA=A;HA=22.1;HE=21.4;HC=45.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00980:MetabolismOfXenobioticsByCytochromeP450;GESP=A:4923/G:8051;PAC=NP_055290.1,XP_005258805.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:7:65,9,0 0/1:26:23,0,51</p>

<p>19 53086124 rs373554 C A 100 highCVG AC1=5;AF1=0.6181;DB;DP=15;DP4=5,1,5,4;FQ=54.4;MQ=59;PV4=0.58,0.21,0.22,0.18;SNPEFF_AMINO_ACID_CHANGE=T337K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF701;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172655.1;VDB=0.0155;DN=80;DA=C/A;GM=NM_001172655.1,NM_018260.2;GL=ZNF701;FG=missense;FD=missense;AAC=THR/LYS;PP=337/532,271/466;CDP=1010,812;GS=78;PH=0;CP=0.001;CG=-0.451;AA=A;HA=0.0;HE=17.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=670;GESP=A:9195/C:3811;PAC=NP_001166126.1,NP_060730.2 GT:GQ:PL 0/1:22:17,0,33 0/1:16:21,0,15 1/1:11:58,12,0 0/1:47:49,0,47</p>

<p>19 54755922 rs12977062 C T 24.20 highCVG AC1=3;AF1=0.4158;DB;DP=22;DP4=1,11,4,0;FQ=26.4;MQ=48;PV4=0.0027,1,0.05,1;SNPEFF_AMINO_ACID_CHANGE=A542T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081442.1;VDB=0.0083;DN=121;DA=C/T;GM=NM_001081442.1,NM_001081443.1,NM_006840.3,XM_005258435.1;GL=LILRB5;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=542/592,442/492,541/591,525/575;CDP=1624,1324,1621,1573;GS=58;PH=0.013;CP=0.000;CG=-2.320;AA=C;HA=0.0;HE=4.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=T:2224/C:10782;PAC=NP_001074911.1,NP_001074912.1,NP_006831.1,XP_005258492.1 GT:GQ:PL 0/0:26:0,27,82 0/0:9:0,9,70 1/1:4:37,6,0 1/1:4:35,6,0</p>

<p>19 58928786 rs7257872 A G 26.90 highCVG AC1=2;AF1=0.2698;DB;DP=17;DP4=12,2,3,0;FQ=28.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T301A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF584;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173548.1;VDB=0.0062;DN=116;DA=A/G;GM=NM_173548.1,XM_005258628.1,XM_005258629.1,XM_005258630.1,XM_005258631.1;GL=ZNF584;FG=missense,missense,missense,3-prime-UTR,3-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=THR/ALA,THR/ALA,THR/ALA,none,none;PP=301/422,293/414,256/377,NA,NA;CDP=901,877,766,NA,NA;GS=58,58,58,NA,NA;PH=0.001;CP=0.822;CG=0.242;AA=A;HA=49.6;HE=26.3;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=609;GESP=G:4324/A:8682;PAC=NP_775819.1,XP_005258685.1,XP_005258686.1,NA,NA GT:GQ:PL 0/0:29:0,27,139 0/0:4:0,0,0 1/1:3:68,9,0 0/0:17:0,15,76</p>

<p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

<p>20 17716416 rs1053993 C G 89 midCVG AC1=5;AF1=0.6077;DB;DP=14;DP4=3,4,3,4;FQ=77.9;MQ=60;PV4=1,0.18,1,0.1;SNPEFF_AMINO_ACID_CHANGE=T85S;SNPEFF_CODON_CHANGE=aCt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BANF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159495.1;VDB=0.0033;DN=86;DA=C/G;GM=NM_001014977.3,NM_001159495.1,NM_178477.4,XM_005260668.1;GL=BANF2;FG=missense;FD=missense,missense,missense,unknown;AAC=THR/SER;PP=78/91,85/98,78/91,78/91;CDP=233,254,233,233;GS=58;PH=0;CP=0.107;CG=3.360;AA=G;HA=45.0;HE=22.5;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=107;GESP=G:4533/C:8473;PAC=NP_001014977.2,NP_001152967.1,NP_848572.3,XP_005260725.1 GT:GQ:PL 0/1:8:3,0,60 0/0:6:0,9,61 1/1:8:66,9,0 1/1:8:67,9,0</p>

<p>21 38525356 rs1053808 T C 81.50 midCVG AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1 GT:GQ:PL 0/1:30:24,0,88 0/1:14:50,0,13 1/1:5:28,3,0 1/1:5:21,3,0</p>

<p>21 47786524 rs2839227 A G 27.90 midCVG AC1=4;AF1=0.5236;DB;DP=10;DP4=6,0,0,4;FQ=31;MQ=60;PV4=0.0048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T879A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0116;DN=100;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1,XM_005261130.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ALA;PP=879/3337,879/3348,879/3347,879/3269,879/3259,761/3230,761/3140,879/2707;CDP=2635,2635,2635,2635,2635,2281,2281,2635;GS=58;PH=0.264;CP=0.002;CG=1.810;AA=A;HA=38.8;HE=17.7;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26;GESP=G:3367/A:9597;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1,XP_005261187.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,40 1/1:6:57,9,0 0/1:15:12,0,61</p>

<p>22 21357119 rs71314757 G A 67 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,1,5;FQ=8.97;MQ=60;PV4=1,0.3,1,1;VDB=0.0209;DN=130;DA=G/A;GM=NR_027051.1,NR_027052.1;GL=THAP7-AS1;FG=splice-donor;FD=splice-donor-variant;CP=0.010;CG=-2.790;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=0 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,45 1/1:14:81,15,0 1/1:4:28,3,0</p>

<p>22 21800806 . T C 11.20 highCVG AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1 GT:GQ:PL 0/0:39:0,36,185 0/0:4:0,0,0 1/1:3:52,9,0 0/0:18:0,15,98</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

<p>X 48418659 rs235836 G A 55.30 midCVG AC1=6;AF1=0.6676;DB;DP=9;DP4=2,0,7,0;FQ=-7.74;MQ=60;PV4=1,0.14,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A455T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D25;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002536.2;VDB=0.0091;DN=79;DA=G/A;GM=NM_002536.2;GL=TBC1D25;FG=missense;FD=missense;AAC=ALA/THR;PP=455/689;CDP=1363;GS=58;PH=0;CP=0.015;CG=-7.340;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=658;GESP=A:5802/G:4761;PAC=NP_002527.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:22,3,0 0/1:25:68,0,23</p>


        <p>
          <strong>IDs and variants associated:</strong> H00057;PARK7|LRRK2|DDC|ATP13A2|FBXO7|GIGYF2|HTRA2|NR4A2|PARK2|PINK1|SNCA|TH|UCHL1|PLA2G6;hsa05012
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 94
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 9.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse16">Amyotrophic Lateral Sclerosis (ALS); Lou Gehrig's disease</a>
      </h4>
    </div>
    <div id="collapse16" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 11015165 rs75130475 A G 149 midCVG AC1=5;AF1=0.6109;DB;DP=14;DP4=0,2,7,4;FQ=-4.32;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I286T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf127;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170754.1;VDB=0.0208;DN=131;DA=A/G;GM=NM_001170754.1,XM_005263428.1,XM_005263429.1,XM_005263430.1;GL=C1orf127;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=286/824,228/766,129/667,147/659;CDP=857,683,386,440;GS=89;PH=0.397;CP=0.000;CG=-6.010;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79;GESP=G:1746/A:11260;PAC=NP_001164225.1,XP_005263485.1,XP_005263486.1,XP_005263487.1 GT:GQ:PL 0/1:4:0,3,24 0/1:4:0,3,14 1/1:4:26,3,0 1/1:28:168,30,0</p>

<p>1 11838841 rs56001051 A G 59.70 midCVG AC1=4;AF1=0.4991;DB;DP=11;DP4=0,6,3,2;FQ=62.4;MQ=60;PV4=0.061,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=S822G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf167;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010881.1;VDB=0.0016;DN=129;DA=A/G;GM=XM_003118845.3;GL=C1orf167;FG=missense;FD=unknown;AAC=SER/GLY;PP=822/1450;CDP=2464;GS=56;PH=0;CP=0.000;CG=2.600;AA=G;DV=by-frequency,by-cluster;DSP=33;PAC=XP_003118893.3 GT:GQ:PL 0/0:10:0,12,75 0/1:3:0,0,0 1/1:9:83,12,0 0/1:22:20,0,39</p>

<p>1 12855878 rs1052908 A C 37.70 midCVG AC1=6;AF1=0.6449;DB;DP=11;DP4=3,1,1,4;FQ=-6.46;MQ=20;PV4=0.21,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R386S;SNPEFF_CODON_CHANGE=agA/agC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRAMEF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_023013.4;VDB=0.0080;DN=86;DA=A/C;GM=NM_023013.2,XM_005263490.1;GL=PRAMEF1;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=386/475,141/230;CDP=1158,423;GS=110;PH=0;CP=0.004;CG=0.600;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,suspect;DSP=292;GESP=A:12994;PAC=NP_075389.1,XP_005263547.1 GT:GQ:PL 0/1:2:4,3,0 0/1:3:0,0,0 1/1:3:70,4,0 0/0:3:0,3,27</p>

<p>1 13183249 rs139566100 C A 11.60 highCVG AC1=4;AF1=0.5476;DB;DP=15;DP4=6,2,2,4;FQ=-3.2;MQ=13;PV4=0.28,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q208H;SNPEFF_CODON_CHANGE=caG/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HNRNPCL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136561.3;VDB=0.0090;DN=134;DA=C/A;GM=NM_001136561.2;GL=HNRNPCP5;FG=missense;FD=missense;AAC=GLN/HIS;PP=208/294;CDP=624;GS=24;PH=0.002;CP=0.004;CG=-0.298;AA=C;DV=by-cluster;DSP=805;GESP=C:4540;PAC=NP_001130033.2 GT:GQ:PL 0/0:4:0,4,20 0/1:18:20,0,16 1/1:4:29,6,0 0/1:4:3,0,1</p>

<p>1 13802325 rs3013105 T C 173 midCVG AC1=6;AF1=0.7094;DB;DP=13;DP4=2,1,5,5;FQ=5.86;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K292E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRC38;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010847.1;VDB=0.0076;DN=101;DA=T/C;GM=NM_001010847.1;GL=LRRC38;FG=missense;FD=missense;AAC=LYS/GLU;PP=292/295;CDP=874;GS=56;PH=0;CP=0.701;CG=5.250;AA=C;HA=48.9;HE=39.6;HC=39.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=243;PAC=NP_001010847.1 GT:GQ:PL 0/1:30:36,0,31 0/1:5:0,3,14 1/1:10:76,9,0 1/1:15:107,15,0</p>

<p>1 15909850 rs6429757 C G 23.50 midCVG AC1=3;AF1=0.403;DB;DP=14;DP4=9,2,3,0;FQ=25.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G105R;SNPEFF_CODON_CHANGE=Ggg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AGMAT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024758.4;VDB=0.0052;DN=116;DA=C/G;GM=NM_024758.4;GL=AGMAT;FG=missense;FD=missense;AAC=GLY/ARG;PP=105/353;CDP=313;GS=125;PH=0;CP=0.010;CG=-1.410;AA=G;HA=7.9;HE=49.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;TFBS=V$COMP1_01;GESP=G:8504/C:4502;PAC=NP_079034.3 GT:GQ:PL 0/0:6:0,6,49 0/1:4:22,3,0 1/1:3:47,6,0 0/0:26:0,27,74</p>

<p>1 17085502 rs2329006 T C 10.30 highCVG AC1=3;AF1=0.3679;DB;DP=20;DP4=4,5,7,3;FQ=11.7;MQ=16;PV4=0.37,7e-06,0.3,0.15;SNPEFF_AMINO_ACID_CHANGE=M397V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MST1L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271733.1;VDB=0.0107;DN=100;DA=T/C;GM=NM_001271733.1;GL=MST1L;FG=missense;FD=missense;AAC=MET/VAL;PP=397/690;CDP=1189;GS=21;PH=0.235;CP=0.000;AA=C;DG;DV=by-cluster,by-hapmap;DSP=29;CPG=17085412-17086974;PAC=NP_001258662.1 GT:GQ:PL 0/1:7:7,0,12 0/0:5:0,3,18 1/1:3:45,7,0 0/0:11:0,10,33</p>

<p>1 17586134 rs3750300 A G 26.60 midCVG AC1=3;AF1=0.3603;DB;DP=9;DP4=5,0,4,0;FQ=28.5;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I52V;SNPEFF_CODON_CHANGE=Atc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016233.2;VDB=0.0149;DN=107;DA=A/G;GM=NM_016233.2;GL=PADI3;FG=missense;FD=missense;AAC=ILE/VAL;PP=52/665;CDP=154;GS=29;PH=0;CP=0.038;CG=0.158;AA=G;HA=8.2;HE=8.2;HC=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=62;GESP=G:1157/A:11849;PAC=NP_057317.2 GT:GQ:PL 0/0:7:0,6,42 0/0:3:0,0,0 1/1:7:68,12,0 0/0:10:0,9,48</p>

<p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>1 18808526 rs2992752 A C 43.10 lowCVG AC1=6;AF1=0.7115;DB;DP=8;DP4=2,0,0,6;FQ=-12;MQ=60;PV4=0.036,0.43,1,0.35;SNPEFF_AMINO_ACID_CHANGE=T351P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHDC7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152375.2;VDB=0.0098;DN=101;DA=A/C;GM=NM_152375.2,XM_005245739.1,XM_005245740.1;GL=KLHDC7A;FG=missense;FD=missense,unknown,unknown;AAC=THR/PRO;PP=351/778,351/759,208/635;CDP=1051,1051,622;GS=38;PH=0;CP=0.000;CG=0.690;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=255;GESP=C:8092/A:4532;PAC=NP_689588.2,XP_005245796.1,XP_005245797.1 GT:GQ:PL 0/1:20:44,0,19 0/1:3:0,0,0 1/1:6:35,6,0 0/1:3:0,0,0</p>

<p>1 19431096 . G T 9.57 lowCVG AC1=8;AF1=1;DP=7;DP4=0,0,0,2;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=A4237D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=86;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020765.2;GM=NM_020765.2,XM_005245798.1,XM_005245799.1,XM_005245800.1,XM_005245801.1,XM_005245802.1,XM_005245803.1,XM_005245804.1,XM_005245805.1,XM_005245806.1,XM_005245807.1,XM_005245808.1;GL=UBR4;FG=missense;FD=unknown;AAC=ALA/ASP;PP=4237/5184,4277/5245,4276/5244,4275/5243,4266/5234,4277/5224,4277/5222,4249/5217,4242/5210,4231/5199,4277/5197,4277/5184;CDP=12710,12830,12827,12824,12797,12830,12830,12746,12725,12692,12830,12830;GS=126;PH=0.998;CP=1.000;CG=6.070;AA=G;DSP=18;TFBS=V$OCT1_02;GESP=G:13006;PPI=UBC:705;PAC=NP_065816.2,XP_005245855.1,XP_005245856.1,XP_005245857.1,XP_005245858.1,XP_005245859.1,XP_005245860.1,XP_005245861.1,XP_005245862.1,XP_005245863.1,XP_005245864.1,XP_005245865.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:19,3,0 1/1:5:23,3,0</p>

<p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>1 24201920 rs2501432 T C 86.40 highCVG AC1=5;AF1=0.5586;DB;DP=20;DP4=8,2,3,5;FQ=69.9;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q63R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001841.2;VDB=0.0031;DN=100;DA=T/C;GM=NM_001841.2,XM_005245736.1,XM_005245737.1,XM_005245738.1;GL=CNR2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLN/ARG;PP=63/361;CDP=188;GS=43;PH=0;CP=0.910;CG=5.120;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=233;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=C:7940/T:5066;PAC=NP_001832.1,XP_005245793.1,XP_005245794.1,XP_005245795.1 GT:GQ:PL 0/1:23:46,0,21 0/0:20:0,24,93 1/1:3:16,3,0 1/1:10:72,12,0</p>

<p>1 31902279 rs34728687 G A 95.60 highCVG AC1=3;AF1=0.4377;DB;DP=19;DP4=8,2,3,6;FQ=98.1;MQ=59;PV4=0.07,0.45,0.15,0.065;SNPEFF_AMINO_ACID_CHANGE=G312S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERINC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199038.1;VDB=0.0042;DN=126;DA=G/A;GM=NM_001199037.1,NM_001199038.1,NM_001199039.1,NM_018565.3,NM_178865.4,XM_005270822.1;GL=SERINC2;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=GLY/SER;PP=307/460,312/465,248/401,307/460,303/456,248/401;CDP=919,934,742,919,907,742;GS=56;PH=0.003;CP=0.000;CG=-4.810;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=36;GESP=A:1075/G:11931;PAC=NP_001185966.1,NP_001185967.1,NP_001185968.1,NP_061035.2,NP_849196.2,XP_005270879.1 GT:GQ:PL 0/0:4:0,3,16 0/0:11:0,12,92 1/1:14:100,18,0 0/1:44:42,0,63</p>

<p>1 33161212 rs360041 T C 100 highCVG AC1=4;AF1=0.5158;DB;DP=34;DP4=5,0,28,0;FQ=52.7;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=M163V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030786.2;VDB=0.0000;DN=79;DA=T/C;GM=NM_001161708.1,NM_030786.2;GL=SYNC;FG=missense;FD=unknown;AAC=MET/VAL;PP=163/477,163/483;CDP=487;GS=21;PH=0;CP=0.005;CG=-2.750;AA=T;HA=18.1;HE=4.0;HC=13.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=434;GESP=C:4162/T:404;PAC=NP_001155180.1,NP_110413.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:80:141,84,0 0/0:13:0,15,90</p>

<p>1 34677925 rs373171181 C G 55.50 midCVG AC1=5;AF1=0.6509;DB;DP=9;DP4=1,2,2,4;FQ=20.1;MQ=60;PV4=1,0.39,1,0.085;SNPEFF_AMINO_ACID_CHANGE=P547A;SNPEFF_CODON_CHANGE=Cct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf94;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001134734.1;VDB=0.0018;DN=138;DA=C/G;GM=NM_001134734.1,NM_032884.3;GL=C1orf94;FG=missense;FD=missense;AAC=PRO/ALA;PP=547/599,357/409;CDP=1639,1069;GS=27;PH=0.987;CP=0.991;CG=4.250;AA=C;DSP=83;GESP=C:13006;PAC=NP_001128206.1,NP_116273.2 GT:GQ:PL 0/1:15:10,0,42 0/1:3:0,0,0 1/1:11:78,12,0 0/1:11:8,0,14</p>

<p>1 36550658 rs12043423 C T 16.90 lowCVG AC1=4;AF1=0.4561;DB;DP=8;DP4=2,3,2,1;FQ=19.5;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=R46C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TEKT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014466.2;VDB=0.0094;DN=120;DA=C/T;GM=NM_014466.2,XM_005270753.1;GL=TEKT2;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=46/431;CDP=136;GS=180;PH=1;CP=1.000;CG=3.680;AA=C;HA=6.1;HE=0.0;HC=19.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=21;GESP=T:273/C:12733;PAC=NP_055281.2,XP_005270810.1 GT:GQ:PL 0/1:3:0,0,0 0/0:11:0,12,80 1/1:4:37,6,0 0/1:16:21,0,13</p>

<p>1 45797505 rs3219489 C G 19 midCVG AC1=4;AF1=0.4355;DB;DP=11;DP4=3,5,1,2;FQ=21.3;MQ=60;PV4=1,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=Q338H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUTYH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128425.1;VDB=0.0034;DN=106;DA=C/G;GM=NM_001048171.1,NM_001048172.1,NM_001048173.1,NM_001048174.1,NM_001128425.1,NM_012222.2,XM_005270880.1,XM_005270881.1,XM_005270882.1,XM_005270883.1,XM_005270884.1,XM_005270885.1,XM_005270886.1;GL=MUTYH;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/HIS;PP=324/536,311/523,310/522,310/522,338/550,335/547,325/537,321/533,321/533,282/494,218/430,218/430,184/396;CDP=972,933,930,930,1014,1005,975,963,963,846,654,654,552;GS=24;PH=0.645;CP=1.000;CG=1.260;AA=C;HA=30.5;HE=28.4;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17;KP=hsa03410:BaseExcisionRepair;GESP=G:3304/C:9698;PPI=APEX1:846|PCNA:846;PAC=NP_001041636.1,NP_001041637.1,NP_001041638.1,NP_001041639.1,NP_001121897.1,NP_036354.1,XP_005270937.1,XP_005270938.1,XP_005270939.1,XP_005270940.1,XP_005270941.1,XP_005270942.1,XP_005270943.1 GT:GQ:PL 0/1:13:12,0,51 0/0:14:0,15,79 1/1:4:48,6,0 0/1:3:0,0,0</p>

<p>1 46827456 rs9865 A G 20.90 highCVG AC1=2;AF1=0.2753;DB;DP=15;DP4=7,2,2,2;FQ=22.2;MQ=60;PV4=0.53,0.075,1,1;SNPEFF_AMINO_ACID_CHANGE=I365V;SNPEFF_CODON_CHANGE=Ata/Gta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NSUN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199044.3;VDB=0.0005;DN=52;DA=A/G;GM=NM_001256127.1,NM_001256128.1,NM_199044.3,NR_045789.1,NR_045790.1,NR_045791.1;GL=NSUN4;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant;AAC=ILE/VAL,ILE/VAL,ILE/VAL,none,none,none;PP=316/336,316/336,365/385,NA,NA,NA;CDP=946,946,1093,1918,1588,1406;GS=29,29,29,NA,NA,NA;PH=0.001;CP=1.000;CG=3.010;AA=A;HA=2.3;HE=12.5;HC=29.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=215;TFBS=V$IK3_01;GESP=G:1441/A:11565;PPI=MTERFD2:900;PAC=NP_001243056.1,NP_001243057.1,NP_950245.2,NA,NA,NA GT:GQ:PL 0/0:9:0,6,44 0/0:6:0,3,17 1/1:6:64,12,0 0/0:20:0,18,80</p>

<p>1 53676448 rs1799821 G A 24.80 lowCVG AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,22 1/1:5:45,6,0 0/1:20:19,0,22</p>

<p>1 54661150 rs946448 C T 61.80 highCVG AC1=4;AF1=0.5145;DB;DP=17;DP4=6,2,5,1;FQ=64.8;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=R47K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYB5RL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001031672.2;VDB=0.0105;DN=86;DA=C/T;GM=NM_001031672.2,XM_005271096.1;GL=CYB5RL;FG=missense;FD=missense,unknown;AAC=ARG/LYS;PP=47/316,47/237;CDP=140;GS=26;PH=0.159;CP=1.000;CG=4.240;AA=C;HA=17.4;HE=37.5;HC=24.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=59;GESP=T:3741/C:8713;PAC=NP_001026842.2,XP_005271153.1 GT:GQ:PL 0/0:3:0,3,18 0/0:18:0,21,113 1/1:9:69,12,0 1/1:5:41,6,0</p>

<p>1 55638075 rs1165222 G A 13.90 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,0,2;FQ=-26.8;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T226I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP24;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015306.2;VDB=0.0081;DN=87;DA=G/A;GM=NM_015306.2,XM_005270689.1;GL=USP24;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=226/2621,51/2444;CDP=677,152;GS=89;PH=0;CP=0.988;CG=2.190;AA=A;HA=12.5;HE=16.1;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=26;TFBS=V$BRACH_01,V$CDPCR3_01;GESP=A:10080/G:1770;PPI=UBC:709;PAC=NP_056121.2,XP_005270746.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:3:0,0,0</p>

<p>1 89847372 rs4658359 G T 65 highCVG AC1=5;AF1=0.5773;DB;DP=17;DP4=8,2,3,3;FQ=68.7;MQ=60;PV4=0.3,0.47,0.1,1;SNPEFF_AMINO_ACID_CHANGE=A331S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GBP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198460.2;VDB=0.0087;DN=111;DA=G/T;GM=NM_198460.2,XM_005270548.1,XM_005270549.1;GL=GBP6;FG=missense;FD=missense,unknown,unknown;AAC=ALA/SER;PP=331/634,331/634,201/504;CDP=991,991,601;GS=99;PH=0.835;CP=0.198;CG=-0.023;AA=G;HA=9.3;HE=27.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=120;GESP=T:10289/G:2717;PAC=NP_940862.2,XP_005270605.1,XP_005270606.1 GT:GQ:PL 0/0:9:0,12,87 0/1:9:5,0,75 1/1:5:37,6,0 1/1:8:70,9,0</p>

<p>1 89849742 rs959460 A T 34.50 lowCVG AC1=6;AF1=0.6772;DB;DP=6;DP4=0,1,4,0;FQ=-19.2;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D520V;SNPEFF_CODON_CHANGE=gAt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GBP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198460.2;VDB=0.0024;DN=86;DA=A/T;GM=NM_198460.2,XM_005270548.1,XM_005270549.1;GL=GBP6;FG=missense;FD=missense,unknown,unknown;AAC=ASP/VAL;PP=520/634,520/634,390/504;CDP=1559,1559,1169;GS=152;PH=0.011;CP=0.001;CG=3.740;AA=A;HA=9.0;HE=13.0;HC=25.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=91;GESP=T:11671/A:1335;PAC=NP_940862.2,XP_005270605.1,XP_005270606.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,13 1/1:10:71,12,0 0/1:3:0,0,0</p>

<p>1 92445257 rs3088232 C G 193 highCVG AC1=6;AF1=0.6924;DB;DP=15;DP4=2,0,6,7;FQ=3.84;MQ=60;PV4=0.47,0.12,1,0.087;SNPEFF_AMINO_ACID_CHANGE=N414K;SNPEFF_CODON_CHANGE=aaC/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BRDT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242806.1;VDB=0.0047;DN=102;DA=C/G;GM=NM_001242805.1,NM_001242806.1,NM_001242807.1,NM_001242808.1,NM_001242810.1,NM_001726.3,NM_207189.2,XM_005271151.1,XM_005271152.1;GL=BRDT;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,unknown,unknown;AAC=ASN/LYS;PP=410/948,414/952,364/902,364/902,337/875,410/948,410/948,410/948,410/948;CDP=1230,1242,1092,1092,1011,1230,1230,1230,1230;GS=94;PH=0.013;CP=0.889;CG=3.760;AA=C;HA=1.7;HE=24.6;HC=3.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=58;GESP=G:2397/C:10609;PAC=NP_001229734.1,NP_001229735.1,NP_001229736.1,NP_001229737.1,NP_001229739.1,NP_001717.2,NP_997072.1,XP_005271208.1,XP_005271209.1 GT:GQ:PL 0/1:5:0,3,26 0/1:16:55,0,16 1/1:7:47,6,0 1/1:24:137,24,0</p>

<p>1 93672723 rs2815413 A C 107 highCVG AC1=6;AF1=0.7028;DB;DP=15;DP4=0,6,5,3;FQ=45.9;MQ=60;PV4=0.031,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=N326T;SNPEFF_CODON_CHANGE=aAt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206886.4;VDB=0.0073;DN=100;DA=A/C;GM=NM_206886.3,XM_005270815.1,XM_005270816.1;GL=CCDC18;FG=missense;FD=missense,unknown,unknown;AAC=ASN/THR;PP=326/1300,444/1574,444/1418;CDP=977,1331,1331;GS=65;PH=0.649;CP=1.000;CG=4.580;AA=A;HA=25.6;HE=15.0;HC=3.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=60;GESP=C:8247/A:3559;PAC=NP_996769.3,XP_005270872.1,XP_005270873.1 GT:GQ:PL 0/1:18:44,0,18 0/1:38:32,0,66 1/1:10:53,9,0 1/1:5:21,3,0</p>

<p>1 109268441 rs11582005 G A 9.32 midCVG AC1=4;AF1=0.5274;DB;DP=14;DP4=8,1,4,1;FQ=12.2;MQ=60;PV4=1,1.1e-05,1,1;SNPEFF_AMINO_ACID_CHANGE=S309N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FNDC7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144937.1;VDB=0.0066;DN=120;DA=G/A;GM=NM_001144937.1,XM_005270550.1;GL=FNDC7;FG=missense;FD=missense,unknown;AAC=SER/ASN;PP=309/734,76/491;CDP=926,227;GS=46;PH=0;CP=0.001;CG=-8.020;AA=A;HA=28.9;HE=10.5;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=70;GESP=A:2609/G:10397;PAC=NP_001138409.1,XP_005270607.1 GT:GQ:PL 0/1:3:0,1,75 0/1:12:10,0,37 1/1:4:32,6,0 0/1:10:8,0,51</p>

<p>1 110279701 rs7483 C T 240 highCVG AC1=6;AF1=0.6918;DB;DP=21;DP4=3,0,11,7;FQ=-9.17;MQ=60;PV4=0.52,0.11,0.35,0.17;SNPEFF_AMINO_ACID_CHANGE=V224I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000849.4;VDB=0.0111;DN=52;DA=C/T;GM=NM_000849.4,NR_024537.1;GL=GSTM3;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=VAL/ILE,none;PP=224/226,NA;CDP=670,904;GS=29,NA;PH=0;CP=0.001;CG=-5.250;AA=C;HA=3.5;HE=30.8;HC=26.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=T:3043/C:9963;PPI=GSTM2:959;PAC=NP_000840.2,NA GT:GQ:PL 0/1:19:93,0,19 0/1:3:0,0,0 1/1:18:98,18,0 0/1:9:90,0,8</p>

<p>1 112018657 rs6703267 G A 20.50 highCVG AC1=2;AF1=0.3037;DB;DP=32;DP4=2,21,0,5;FQ=22;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G3S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf162;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174896.3;VDB=0.0100;DN=116;DA=G/A;GM=NM_174896.2,XM_005270475.1,XM_005270476.1;GL=C1orf162;FG=missense,missense,synonymous;FD=missense,unknown,unknown;AAC=GLY/SER,GLY/SER,GLU;PP=3/156,3/131,5/127;CDP=7,7,15;GS=56,56,NA;PH=0.408;CP=0.008;CG=-1.890;AA=G;HA=22.1;HE=27.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=18;GESP=A:3183/G:9823;PAC=NP_777556.1,XP_005270532.1,XP_005270533.1 GT:GQ:PL 0/0:3:0,0,0 0/0:22:0,20,88 1/1:4:61,9,0 0/0:8:0,6,45</p>

<p>1 112308953 rs197412 T C 58.10 midCVG AC1=5;AF1=0.6066;DB;DP=10;DP4=1,1,7,0;FQ=8.93;MQ=60;PV4=0.22,1,1,0.045;SNPEFF_AMINO_ACID_CHANGE=I636T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DDX20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007204.4;VDB=0.0159;DN=79;DA=T/C;GM=NM_007204.4;GL=DDX20;FG=missense;FD=missense;AAC=ILE/THR;PP=636/825;CDP=1907;GS=89;PH=0;CP=0.331;CG=3.860;AA=C;HA=19.7;HE=43.3;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=595;GESP=C:6295/T:6711;PPI=SMN1:984|SMN2:983|GEMIN4:973|GEMIN2:933|AGO2:885|GEMIN5:836|PRRC2A:786|HDAC5:716;PAC=NP_009135.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:97,21,0 0/0:5:0,6,43</p>

<p>1 115829313 rs6330 G A 9.39 midCVG AC1=4;AF1=0.4545;DB;DP=10;DP4=5,0,2,1;FQ=11;MQ=60;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=A35V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NGF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002506.2;VDB=0.0110;DN=52;DA=G/A;GM=NM_002506.2;GL=NGF;FG=missense;FD=missense;AAC=ALA/VAL;PP=35/242;CDP=104;GS=64;PH=0.002;CP=0.977;CG=4.450;AA=G;HA=17.7;HE=44.1;HC=18.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=116;KP=hsa04010:MAPKSignalingPathway|hsa04210:Apoptosis|hsa04722:NeurotrophinSignalingPathway;TFBS=V$AP2REP_01;GESP=A:4801/G:8205;PPI=NTRK1:998|NGFR:937|NTF4:937|SORT1:845;PAC=NP_002497.2 GT:GQ:PL 0/0:3:0,0,0 0/0:15:0,15,54 1/1:5:47,9,0 0/0:3:0,0,0</p>

<p>1 117311147 rs699738 C A 39 midCVG AC1=3;AF1=0.4223;DB;DP=13;DP4=1,6,5,1;FQ=41.3;MQ=60;PV4=0.029,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=H266Q;SNPEFF_CODON_CHANGE=caC/caA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001767.3;VDB=0.0019;DN=86;DA=C/A;GM=NM_001767.3,XM_005271319.1,XM_005271320.1;GL=CD2;FG=missense;FD=missense,unknown,unknown;AAC=HIS/GLN;PP=266/352,292/378,225/311;CDP=798,876,675;GS=24;PH=0.085;CP=0.000;CG=-2.990;AA=C;HA=30.3;HE=9.4;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=62;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04640:HematopoieticCellLineage;GESP=A:2986/C:10020;PPI=CD58:996|CD2BP2:995|CD2AP:974|SH3KBP1:937|YES1:937|LCK:846|PTPRC:846;PAC=NP_001758.2,XP_005271376.1,XP_005271377.1 GT:GQ:PL 0/0:4:0,3,22 0/1:14:14,0,14 1/1:11:71,15,0 0/0:14:0,15,77</p>

<p>1 152277055 rs2065955 C G 55.20 midCVG AC1=4;AF1=0.5684;DB;DP=10;DP4=0,3,5,2;FQ=5.52;MQ=34;PV4=0.17,1,0.004,1;SNPEFF_AMINO_ACID_CHANGE=G3436A;SNPEFF_CODON_CHANGE=gGa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0107;DN=94;DA=C/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLY/ALA;PP=3436/4062;CDP=10307;GS=60;PH=0.876;CP=0.000;CG=-1.140;AA=C;HE=14.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=9460;GESP=G:4311/C:8695;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:8:60,0,5 0/0:5:0,6,36 1/1:7:36,9,0 0/1:3:0,0,0</p>

<p>1 152280900 rs74129452 T G 22.80 midCVG AC1=5;AF1=0.5654;DB;DP=9;DP4=0,4,0,5;FQ=17.7;MQ=41;PV4=1,1,0.043,0.2;SNPEFF_AMINO_ACID_CHANGE=Q2154H;SNPEFF_CODON_CHANGE=caA/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0064;DN=130;DA=T/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLN/HIS;PP=2154/4062;CDP=6462;GS=24;PH=0;CP=0.000;CG=-5.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G,suspect;DSP=5615;GESP=T:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:4:0,1,28 0/1:3:0,0,0 1/1:5:43,6,0 0/1:20:18,0,25</p>

<p>1 152285603 . G A 73.80 midCVG AC1=5;AF1=0.6441;DP=9;DP4=2,1,1,5;FQ=9.97;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R587C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0003;GM=NM_002016.1;GL=FLG;FG=missense;FD=unknown;AAC=ARG/CYS;PP=587/4062;CDP=1759;GS=180;PH=1;CP=0.000;CG=-3.110;AA=G;DSP=912;GESP=G:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:10:19,0,9 0/0:4:0,6,41 1/1:4:26,3,0 1/1:11:74,12,0</p>

<p>1 152770521 rs41268492 G A 13.80 lowCVG AC1=6;AF1=0.699;DB;DP=5;DP4=1,0,3,0;FQ=-21.4;MQ=47;PV4=1,1,0.48,1;SNPEFF_AMINO_ACID_CHANGE=R84H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LCE1D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178352.2;VDB=0.0045;DN=127;DA=G/A;GM=NM_178352.2;GL=LCE1D;FG=missense;FD=missense;AAC=ARG/HIS;PP=84/115;CDP=251;GS=29;PH=0;CP=0.000;CG=-7.330;AA=A;DV=by-frequency,by-cluster;DSP=273;GESP=A:2436/G:9006;PAC=NP_848129.1 GT:GQ:PL 0/1:3:0,0,0/1:4:0,1,8 1/1:4:47,6,0 0/1:3:0,0,0</p>

<p>1 156642841 . G T 17.30 lowCVG AC1=3;AF1=0.3522;DP=7;DP4=4,0,3,0;FQ=17.7;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=A380D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NES;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006617.1;VDB=0.0007;GM=NM_006617.1;GL=NES;FG=missense;FD=unknown;AAC=ALA/ASP;PP=380/1622;CDP=1139;GS=126;PH=0.94;CP=0.515;CG=3.090;AA=G;DSP=157;TFBS=V$HOX13_01;GESP=G:13006;PPI=CDK5:937|CDK5R1:845;PAC=NP_006608.1 GT:GQ:PL 0/0:8:0,6,32 0/0:8:0,6,29 1/1:4:57,9,0 0/0:3:0,0,0</p>

<p>1 156883546 rs12137505 A G 17.30 lowCVG AC1=4;AF1=0.4665;DB;DP=8;DP4=3,1,2,1;FQ=20;MQ=60;PV4=1,0.24,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N903D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PEAR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080471.1;VDB=0.0107;DN=120;DA=A/G;GM=NM_001080471.1,XM_005245141.1,XM_005245142.1;GL=PEAR1;FG=missense;FD=missense,unknown,unknown;AAC=ASN/ASP;PP=903/1038,903/1038,839/974;CDP=2707,2707,2515;GS=23;PH=0.094;CP=0.000;CG=-3.910;AA=A;HA=20.6;HE=35.7;HC=48.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=7;GESP=G:4616/A:8390;PAC=NP_001073940.1,XP_005245198.1,XP_005245199.1 GT:GQ:PL 0/0:4:0,3,27 0/0:8:0,9,64 1/1:4:44,6,0 0/1:4:17,3,0</p>

<p>1 169578854 . C G 75.50 highCVG AC1=3;AF1=0.3761;DP=18;DP4=9,1,3,3;FQ=77.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q407H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SELP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003005.3;VDB=0.0130;GM=NM_003005.3,XM_005245435.1,XM_005245436.1,XM_005245437.1,XM_005245438.1,XM_005245439.1,XM_005245440.1;GL=SELP;FG=missense;FD=unknown;AAC=GLN/HIS;PP=407/831,407/831,407/831,407/830,407/791,407/791,345/769;CDP=1221,1221,1221,1221,1221,1221,1035;GS=24;PH=0.973;CP=0.018;CG=1.770;AA=C;DSP=74;KP=hsa04514:CellAdhesionMolecules(CAMs);GESP=C:13006;PPI=SELPLG:974;PAC=NP_002996.2,XP_005245492.1,XP_005245493.1,XP_005245494.1,XP_005245495.1,XP_005245496.1,XP_005245497.1 GT:GQ:PL 0/0:12:0,12,64 0/0:7:0,6,46 1/1:5:71,9,0 0/1:50:50,0,72</p>

<p>1 169673838 rs2229569 G A 156 highCVG AC1=3;AF1=0.4287;DB;DP=22;DP4=5,2,3,9;FQ=101;MQ=60;PV4=0.074,1,1,0.33;SNPEFF_AMINO_ACID_CHANGE=P226S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SELL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000655.4;VDB=0.0143;DN=98;DA=G/A;GM=NM_000655.4,NR_029467.1;GL=SELL;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=PRO/SER,none;PP=226/386,NA;CDP=676,605;GS=74,NA;PH=1;CP=0.872;CG=5.310;AA=G;HA=26.2;HE=10.5;HC=27.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=57;KP=hsa04514:CellAdhesionMolecules(CAMs);GESP=A:1885/G:9953;PAC=NP_000646.2,NA GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,46 1/1:26:172,30,0 0/1:32:31,0,37</p>

<p>1 169823718 rs4656197 T C 47.80 highCVG AC1=4;AF1=0.5453;DB;DP=15;DP4=3,3,0,7;FQ=39.1;MQ=60;PV4=0.07,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=Q621R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCYL3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181093.3;VDB=0.0088;DN=111;DA=T/A/C/G;GM=NM_020423.6,NM_181093.3;GL=SCYL3;FG=missense;FD=missense;AAC=GLN/ARG;PP=567/689,621/743;CDP=1700,1862;GS=43;PH=0.589;CP=0.000;CG=-1.020;AA=T;HA=27.3;HE=43.9;HC=34.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=308;GESP=C:8150/T:4856;PAC=NP_065156.5,NP_851607.2 GT:GQ:PL 0/1:5:52,0,0 0/1:3:0,0,0 1/1:7:37,9,0 0/0:12:0,15,77</p>

<p>1 179504043 rs6658180 C G 13.70 lowCVG AC1=3;AF1=0.3645;DB;DP=8;DP4=1,3,1,2;FQ=15.5;MQ=59;PV4=1,0.026,0.14,0.14;SNPEFF_AMINO_ACID_CHANGE=Q993E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AXDND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144696.5;VDB=0.0075;DN=116;DA=C/G;GM=NM_144696.5,NR_073544.1;GL=AXDND1;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=GLN/GLU,none;PP=993/1013,NA;CDP=2977,3097;GS=29,NA;PH=0;CP=0.000;CG=-4.940;AA=C;HA=6.0;HE=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=GA-rich;DSP=55;GESP=G:4250/C:8608;PAC=NP_653297.3,NA GT:GQ:PL 0/0:5:0,3,27 0/0:3:0,0,0 1/1:4:53,9,0 0/0:10:0,9,69</p>

<p>1 180905448 rs3795504 G T 53.90 highCVG AC1=3;AF1=0.3552;DB;DP=17;DP4=8,3,5,1;FQ=55.7;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=L801F;SNPEFF_CODON_CHANGE=ttG/ttT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1614;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020950.1;VDB=0.0071;DN=107;DA=G/T;GM=NM_020950.1,XM_005245376.1;GL=KIAA1614;FG=missense;FD=missense,unknown;AAC=LEU/PHE;PP=801/1191,801/1222;CDP=2403;GS=22;PH=0;CP=0.002;CG=2.380;AA=T;HA=32.9;HE=45.0;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=359;GESP=T:5139/G:7173;PAC=NP_066001.1,XP_005245433.1 GT:GQ:PL 0/0:9:0,9,70 0/0:15:0,15,73 1/1:5:49,9,0 0/1:46:51,0,42</p>

<p>1 182025558 . T C 23 midCVG AC1=2;AF1=0.2266;DP=12;DP4=0,9,0,3;FQ=24;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R530G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF648;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001009992.1;VDB=0.0015;GM=NM_001009992.1;GL=ZNF648;FG=missense;FD=unknown;AAC=ARG/GLY;PP=530/569;CDP=1588;GS=125;PH=0.797;CP=0.999;CG=1.640;AA=T;DSP=1651;GESP=T:13006;PAC=NP_001009992.1 GT:GQ:PL 0/0:9:0,6,35 0/0:9:0,6,44 1/1:3:65,9,0 0/0:18:0,15,92</p>

<p>1 200880702 rs61740234 C T 13.70 lowCVG AC1=2;AF1=0.3055;DB;DP=8;DP4=0,4,3,0;FQ=13.5;MQ=59;PV4=0.029,1,0.14,1.7e-08;SNPEFF_AMINO_ACID_CHANGE=P460S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf106;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018265.3;VDB=0.0002;DN=129;DA=C/T;GM=NM_001142569.2,NM_018265.3;GL=C1orf106;FG=missense;FD=missense;AAC=PRO/SER;PP=361/579,460/678;CDP=1081,1378;GS=74;PH=0.689;CP=0.908;CG=3.690;AA=C;DV=by-frequency,by-cluster;DSP=122;CPG=200880594-200881270;GESP=T:477/C:12529;PAC=NP_001136041.1,NP_060735.3 GT:GQ:PL 0/0:6:0,3,15 0/0:6:0,3,15 1/1:4:54,9,0 0/0:8:0,6,27</p>

<p>1 201176108 rs12094334 G A 111 midCVG AC1=6;AF1=0.7369;DB;DP=14;DP4=0,2,6,2;FQ=12.9;MQ=60;PV4=0.13,0.11,1,0.24;SNPEFF_AMINO_ACID_CHANGE=G696D;SNPEFF_CODON_CHANGE=gGt/gAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGFN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001164586.1;VDB=0.0191;DN=120;DA=G/A;GM=NM_001164586.1,XM_005245580.1,XM_005245581.1;GL=IGFN1;FG=missense;FD=missense,unknown,unknown;AAC=GLY/ASP;PP=696/3709,696/3684,696/3424;CDP=2087;GS=94;CP=0.000;CG=0.077;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=898;GESP=A:1359/G:3207;PAC=NP_001158058.1,XP_005245637.1,XP_005245638.1 GT:GQ:PL 0/0:3:0,6,50 0/1:3:0,0,0 1/1:7:50,6,0 1/1:18:104,18,0</p>

<p>1 205801872 rs1361754 A G 26.70 lowCVG AC1=5;AF1=0.6576;DB;DP=6;DP4=2,0,0,3;FQ=-9.17;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I380T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PM20D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152491.4;VDB=0.0040;DN=88;DA=A/G;GM=NM_152491.4;GL=PM20D1;FG=missense;FD=missense;AAC=ILE/THR;PP=380/503;CDP=1139;GS=89;PH=0.016;CP=0.270;CG=-0.108;AA=A;HA=33.9;HE=42.0;HC=0.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=23;GESP=G:6267/A:6739;PAC=NP_689704.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:41,6,0 1/1:4:26,3,0</p>

<p>1 206225011 rs33990840 G C 30 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=2,0,1,2;FQ=11.7;MQ=60;PV4=0.4,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G191R;SNPEFF_CODON_CHANGE=Ggc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVPR1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000707.3;VDB=0.0112;DN=126;DA=G/C;GM=NM_000707.3;GL=AVPR1B;FG=missense;FD=missense;AAC=GLY/ARG;PP=191/425;CDP=571;GS=125;PH=0;CP=1.000;CG=4.980;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=370;KP=hsa04020:CalciumSignalingPathway|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04270:VascularSmoothMuscleContraction;GESP=C:802/G:12188;PAC=NP_000698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:67,9,0 0/0:6:0,6,45</p>

<p>1 207790088 rs3811381 C G 67.70 midCVG AC1=5;AF1=0.5957;DB;DP=9;DP4=0,4,1,4;FQ=29.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2277R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=41;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000651.4;VDB=0.0076;DN=107;DA=C/G;GM=NM_000573.3,NM_000651.4,XM_005273064.1;GL=CR1;FG=missense;FD=missense,missense,unknown;AAC=PRO/ARG;PP=1827/2040,2277/2490,2149/2362;CDP=5480,6830,6446;GS=103;PH=0.643;CP=0.000;CG=-5.580;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;KP=hsa04610:ComplementAndCoagulationCascades|hsa04640:HematopoieticCellLineage|hsa05140:Leishmaniasis;GESP=G:2006/C:10176;PPI=CR2:846;PAC=NP_000564.2,NP_000642.3,XP_005273121.1 GT:GQ:PL 0/1:35:36,0,34 0/0:4:0,6,36 1/1:6:51,6,0 1/1:4:25,3,0</p>

<p>1 214830617 rs438034 A G 57 lowCVG AC1=5;AF1=0.6559;DB;DP=8;DP4=2,0,2,4;FQ=7.46;MQ=60;PV4=0.43,0.015,1,1;SNPEFF_AMINO_ACID_CHANGE=R2943G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0106;DN=80;DA=A/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ARG/GLY;PP=2943/3115,2921/3093;CDP=8827,8761;GS=125;PH=0;CP=0.000;CG=2.520;AA=A;HA=36.6;HE=47.7;HC=9.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=136;GESP=G:6399/A:6607;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1 GT:GQ:PL 0/1:18:16,0,20 0/1:18:14,0,23 1/1:11:68,12,0 0/1:3:0,0,0</p>

<p>1 214837110 rs7289 C G 17.40 lowCVG AC1=6;AF1=0.7293;DB;DP=5;DP4=0,1,0,4;FQ=-8.8;MQ=60;PV4=1,0.0073,1,1;SNPEFF_AMINO_ACID_CHANGE=N3106K;SNPEFF_CODON_CHANGE=aaC/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0020;DN=52;DA=C/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ASN/LYS;PP=3106/3115,3084/3093;CDP=9318,9252;GS=94;PH=0;CP=0.002;CG=3.740;AA=G;HA=38.2;HE=49.2;HC=9.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=177;GESP=G:6554/C:6452;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:4:17,3,0 1/1:9:40,9,0</p>

<p>1 223284528 rs5744174 A G 264 highCVG AC1=5;AF1=0.6321;DB;DP=26;DP4=4,2,5,13;FQ=58.7;MQ=60;PV4=0.15,1,1,0.16;SNPEFF_AMINO_ACID_CHANGE=F616L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003268.5;VDB=0.0132;DN=114;DA=A/G;GM=NM_003268.5,XM_005273241.1,XM_005273242.1,XM_005273243.1;GL=TLR5;FG=missense;FD=missense,unknown,unknown,unknown;AAC=PHE/LEU;PP=616/859;CDP=1846;GS=22;PH=0;CP=0.000;CG=-11.200;AA=G;HA=10.7;HE=42.7;HC=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1850;KP=hsa04620:Toll-likeReceptorSignalingPathway|hsa05130:PathogenicEscherichiaColiInfection;GESP=G:4352/A:8654;PPI=MYD88:937;PAC=NP_003259.2,XP_005273298.1,XP_005273299.1,XP_005273300.1 GT:GQ:PL 0/1:22:75,0,21 0/0:8:0,12,83 1/1:9:60,9,0 1/1:32:179,33,0</p>

<p>1 225555602 rs12737248 T G 75.40 highCVG AC1=4;AF1=0.4691;DB;DP=16;DP4=4,2,8,1;FQ=56.2;MQ=60;PV4=0.53,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N3969K;SNPEFF_CODON_CHANGE=aaT/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=75;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001373.1;VDB=0.0002;DN=121;DA=T/G;GM=NM_001373.1;GL=DNAH14;FG=missense;FD=missense;AAC=ASN/LYS;PP=3969/4516;CDP=11907;GS=94;PH=0.001;CP=0.001;CG=-10.100;AA=T;HA=23.0;HE=37.9;HC=25.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=96;GESP=G:2139/T:2427;PAC=NP_001364.1 GT:GQ:PL 0/1:17:15,0,39 0/0:3:0,3,25 1/1:20:106,24,0 0/0:5:0,6,35</p>

<p>1 227216775 rs1929860 C T 138 highCVG AC1=5;AF1=0.6658;DB;DP=24;DP4=2,7,6,6;FQ=110;MQ=60;PV4=0.37,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=V1304I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC42BPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003607.3;VDB=0.0143;DN=92;DA=C/A/G/T;GM=NM_003607.3,NM_014826.4,XM_005273317.1,XM_005273318.1,XM_005273319.1,XM_005273320.1,XM_005273321.1,XM_005273322.1,XM_005273323.1,XM_005273324.1,XM_005273325.1,XM_005273326.1,XM_005273327.1;GL=CDC42BPA;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=1304/1720,1223/1639,1355/1833,1352/1830,1333/1811,1317/1795,1304/1782,1339/1755,1274/1752,1317/1733,1251/1729,1284/1700,1276/1692;CDP=3910,3667,4063,4054,3997,3949,3910,4015,3820,3949,3751,3850,3826;GS=29;PH=0.001;CP=0.994;CG=4.180;AA=T;HA=25.3;HE=36.6;HC=43.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=177;GESP=T:3673/C:9333;PPI=CDC42:882;PAC=NP_003598.2,NP_055641.3,XP_005273374.1,XP_005273375.1,XP_005273376.1,XP_005273377.1,XP_005273378.1,XP_005273379.1,XP_005273380.1,XP_005273381.1,XP_005273382.1,XP_005273383.1,XP_005273384.1 GT:GQ:PL 0/1:31:25,0,72 0/1:3:0,0,0 1/1:24:127,24,0 0/1:34:28,0,75</p>

<p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

<p>1 242035438 rs4149965 G A 6.86 lowCVG AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1 GT:GQ:PL 0/0:3:0,0,0 0/0:3:0,0,0 1/1:4:44,9,0 0/0:16:0,15,101</p>

<p>1 247048834 rs2642990 T C 119 highCVG AC1=4;AF1=0.5131;DB;DP=15;DP4=2,4,4,4;FQ=67.9;MQ=58;PV4=0.63,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N883S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHCTF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015446.4;VDB=0.0042;DN=100;DA=T/C;GM=NM_015446.4;GL=AHCTF1;FG=missense;FD=missense;AAC=ASN/SER;PP=883/2276;CDP=2648;GS=46;PH=0;CP=0.002;CG=-0.677;AA=C;HA=0.0;HE=49.2;HC=32.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=40;GESP=C:8196/T:4808;PAC=NP_056261.4 GT:GQ:PL 0/0:9:0,12,85 0/1:16:38,0,13 1/1:12:105,15,0 0/1:16:23,0,14</p>

<p>1 247420421 rs41308154 C T 25.10 midCVG AC1=3;AF1=0.3809;DB;DP=11;DP4=2,3,2,2;FQ=27.1;MQ=60;PV4=1,0.0023,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R350C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VN1R5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173858.1;VDB=0.0123;DN=127;DA=C/T;GM=NM_173858.1;GL=VN1R5;FG=missense;FD=unknown;AAC=ARG/CYS;PP=350/358;CDP=1048;GS=180;PH=0.987;CP=0.337;AA=C;DG;DV=by-cluster,by-1000G;RM=MSTC;DSP=44158;GESP=T:2053/C:9883;PAC=NP_776257.1 GT:GQ:PL 0/0:3:0,0,0 0/0:12:0,12,89 1/1:7:66,12,0 0/0:5:0,3,23</p>

<p>1 248039451 rs3811444 C T 107 highCVG AC1=4;AF1=0.5108;DB;DP=24;DP4=8,4,4,7;FQ=110;MQ=60;PV4=0.22,0.14,1,0.15;SNPEFF_AMINO_ACID_CHANGE=T374M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM58;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015431.3;VDB=0.0144;DN=107;DA=C/T;GM=NM_015431.3,XM_005273091.1,XM_005273092.1;GL=TRIM58;FG=missense;FD=missense,unknown,unknown;AAC=THR/MET;PP=374/487,230/343,230/343;CDP=1121,689,689;GS=81;PH=0.999;CP=0.011;CG=2.210;AA=C;HA=3.3;HE=33.2;HC=23.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=250;GESP=T:3365/C:9641;PAC=NP_056246.3,XP_005273148.1,XP_005273149.1 GT:GQ:PL 0/0:12:0,15,107 0/1:13:19,0,11 1/1:21:112,24,0 0/1:27:24,0,91</p>

<p>1 248458718 rs79635253 T C 14.40 midCVG AC1=4;AF1=0.443;DB;DP=9;DP4=4,1,1,2;FQ=16.8;MQ=45;PV4=0.46,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R55G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004692.1;VDB=0.0077;DN=131;DA=T/C;GM=NM_001004692.1;GL=OR2T12;FG=missense;FD=missense;AAC=ARG/GLY;PP=55/321;CDP=163;GS=125;PH=0.005;CP=0.159;CG=1.550;AA=T;DV=by-frequency,by-cluster;DSP=196059;KP=hsa04740:OlfactoryTransduction;GESP=C:613/T:12389;PAC=NP_001004692.1 GT:GQ:PL 0/0:4:0,3,23 0/1:4:13,3,0 1/1:3:45,6,0 0/0:12:0,12,73</p>

<p>1 248685719 rs9330305 A T 35.40 highCVG AC1=3;AF1=0.3917;DB;DP=16;DP4=3,9,3,1;FQ=37.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M258L;SNPEFF_CODON_CHANGE=Atg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2G6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001013355.1;VDB=0.0168;DN=119;DA=A/T;GM=NM_001013355.1;GL=OR2G6;FG=missense;FD=missense;AAC=MET/LEU;PP=258/317;CDP=772;GS=15;PH=0.934;CP=0.966;CG=3.830;AA=A;HA=33.6;HE=12.1;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=159907;KP=hsa04740:OlfactoryTransduction;GESP=T:2221/A:10785;PAC=NP_001013373.1 GT:GQ:PL 0/0:15:0,15,89 0/1:4:17,3,0 1/1:5:66,9,0 0/0:20:0,21,135</p>

<p>1 248722777 rs77589892 T A 89 highCVG AC1=5;AF1=0.7783;DB;DP=29;DP4=7,2,14,5;FQ=-25.8;MQ=12;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R6W;SNPEFF_CODON_CHANGE=Agg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004694.2;VDB=0.0185;DN=131;DA=T/A;GM=NM_001004694.2;GL=OR2T29;FG=missense;FD=missense;AAC=ARG/TRP;PP=6/316;CDP=16;GS=101;CP=0.001;CG=-0.604;AA=A;DG;DV=by-frequency,by-cluster;DSP=122849;KP=hsa04740:OlfactoryTransduction;GESP=A:4597/T:8405;PAC=NP_001004694.2 GT:GQ:PL 0/0:4:0,12,14 0/1:3:6,0,2 1/1:22:86,20,0 1/1:23:44,21,0</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>2 25022598 rs1550116 A G 39.40 lowCVG AC1=6;AF1=0.7255;DB;DP=6;DP4=1,1,1,3;FQ=3.46;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q34R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024322.2;VDB=0.0085;DN=88;DA=A/G;GM=NM_001199803.1,NM_024322.2;GL=CENPO;FG=missense;FD=missense;AAC=GLN/ARG;PP=28/295,34/301;CDP=83,101;GS=43;PH=0;CP=0.000;CG=-3.630;AA=A;HA=5.8;HE=18.0;HC=19.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=55;GESP=G:1761/A:11245;PAC=NP_001186732.1,NP_077298.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:64,9,0 0/1:17:12,0,32</p>

<p>2 26536694 rs2052937 C T 46.40 midCVG AC1=4;AF1=0.5336;DB;DP=10;DP4=5,0,1,4;FQ=47.7;MQ=60;PV4=0.048,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=A404T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR113;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145168.1;VDB=0.0034;DN=94;DA=C/T;GM=NM_001145168.1,NM_001145169.1,NM_153835.3,XM_005264163.1;GL=GPR113;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=404/1080,335/998,205/874,335/1004;CDP=1210,1003,613,1003;GS=58;PH=0.066;CP=0.007;CG=3.470;AA=T;HA=34.1;HE=32.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=67;GESP=T:8972/C:4034;PAC=NP_001138640.1,NP_001138641.1,NP_722577.2,XP_005264220.1 GT:GQ:PL 0/0:7:0,9,60 0/1:35:49,0,32 1/1:3:26,3,0 1/1:3:15,3,0</p>

<p>2 27464018 . C A 18.10 highCVG AC1=2;AF1=0.2604;DP=16;DP4=3,7,0,5;FQ=19.3;MQ=60;PV4=0.51,0.0092,1,0.00055;SNPEFF_AMINO_ACID_CHANGE=H1911N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=37;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CAD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004341.3;VDB=0.0000;GM=NM_004341.3,XM_005264555.1,XM_005264556.1,XM_005264557.1,XM_005264558.1;GL=CAD;FG=missense;FD=unknown;AAC=HIS/ASN;PP=1911/2226,1928/2243,1894/2209,1877/2192,1848/2163;CDP=5731,5782,5680,5629,5542;GS=68;PH=0.283;CP=1.000;CG=4.920;AA=C;DSP=78;KP=hsa00240:PyrimidineMetabolism|hsa00250:Alanine,AspartateAndGlutamateMetabolism|hsa01100:MetabolicPathways;TFBS=V$TAXCREB_01;GESP=C:13006;PPI=GRB2:814|SHC1:764;PAC=NP_004332.2,XP_005264612.1,XP_005264613.1,XP_005264614.1,XP_005264615.1 GT:GQ:PL 0/0:14:0,12,65 0/0:9:0,6,50 1/1:8:63,15,0 0/0:14:0,12,88</p>

<p>2 27717440 . C A 19.60 midCVG AC1=2;AF1=0.3049;DP=14;DP4=5,3,0,3;FQ=21;MQ=60;PV4=0.18,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=S36I;SNPEFF_CODON_CHANGE=aGc/aTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FNDC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022823.2;VDB=0.0046;GM=NM_022823.2,XM_005264499.1;GL=FNDC4;FG=missense;FD=unknown;AAC=SER/ILE;PP=36/235,36/209;CDP=107;GS=142;PH=0.053;CP=0.995;CG=4.380;AA=C;DSP=27;GESP=C:13004;PAC=NP_073734.1,XP_005264556.1 GT:GQ:PL 0/0:8:0,6,46 0/0:3:0,0,1/1:4:60,9,0 0/0:20:0,18,97</p>

<p>2 29295186 rs75276619 C T 29.50 midCVG AC1=3;AF1=0.3733;DB;DP=12;DP4=4,4,3,1;FQ=31.5;MQ=60;PV4=0.58,0.41,1,0.46;SNPEFF_AMINO_ACID_CHANGE=A648T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf71;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001029883.2;VDB=0.0002;DN=131;DA=C/T;GM=NM_001029883.2;GL=C2orf71;FG=missense;FD=missense;AAC=ALA/THR;PP=648/1289;CDP=1942;GS=58;PH=0.2;CP=0.000;CG=-3.320;AA=C;DV=by-frequency,by-cluster;DSP=1727;GESP=T:113/C:12423;PAC=NP_001025054.1 GT:GQ:PL 0/1:11:10,0,51 0/0:9:0,9,67 1/1:5:64,9,0 0/0:7:0,6,37</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 70488470 rs2706762 C T 88.20 highCVG AC1=5;AF1=0.6256;DB;DP=20;DP4=9,1,1,7;FQ=81.1;MQ=60;PV4=0.0029,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=S149F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCYOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016297.3;VDB=0.0031;DN=100;DA=C/T;GM=NM_016297.3,XM_005264362.1;GL=PCYOX1;FG=missense;FD=missense,unknown;AAC=SER/PHE;PP=149/506,131/488;CDP=446,392;GS=155;PH=0;CP=1.000;CG=4.580;AA=C;HA=0.0;HE=14.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=49;GESP=T:1447/C:11559;PAC=NP_057381.3,XP_005264419.1 GT:GQ:PL 0/0:26:0,30,123 0/1:3:0,0,0 1/1:17:89,18,0 1/1:6:45,6,0</p>

<p>2 71058835 rs741326 A G 155 highCVG AC1=6;AF1=0.7158;DB;DP=19;DP4=1,3,12,3;FQ=25.5;MQ=60;PV4=0.071,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V278A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD207;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015717.4;VDB=0.0056;DN=86;DA=A/G;GM=NM_015717.3;GL=CD207;FG=missense;FD=unknown;AAC=VAL/ALA;PP=278/329;CDP=833;GS=64;PH=0;CP=0.000;CG=-9.570;AA=G;HA=49.4;HE=47.6;HC=30.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=4;GESP=G:5453/A:7019;PAC=NP_056532.3 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:25:130,25,0 1/1:9:70,9,0</p>

<p>2 74755959 . G A 6.27 midCVG AC1=3;AF1=0.3379;DP=10;DP4=6,1,0,3;FQ=7.61;MQ=60;PV4=0.033,0.067,1,0.092;SNPEFF_AMINO_ACID_CHANGE=S148F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AUP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181575.3;VDB=0.0037;GM=NM_181575.3,XM_005264392.1;GL=AUP1;FG=missense;FD=unknown;AAC=SER/PHE;PP=148/411,205/468;CDP=443,614;GS=155;PH=0.99;CP=1.000;CG=4.820;AA=G;DSP=82;GESP=G:12058;PPI=UBE2G2:873|UBC:852|SEL1L:846;PAC=NP_853553.1,XP_005264449.1 GT:GQ:PL 0/0:4:0,0,0 0/0:7:0,3,19 1/1:3:45,9,0 0/0:21:0,18,92</p>

<p>2 96993757 rs2279105 C T 69.10 lowCVG AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3 GT:GQ:PL 0/1:16:12,0,21 0/1:4:0,3,17 1/1:4:21,3,0 1/1:12:80,12,0</p>

<p>2 97638038 rs1730120 A G 94.30 midCVG AC1=5;AF1=0.5991;DB;DP=9;DP4=1,1,4,3;FQ=-3.35;MQ=60;PV4=1,0.4,1,0.26;SNPEFF_AMINO_ACID_CHANGE=V55A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM178B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001122646.2;VDB=0.0156;DN=89;DA=A/G;GM=NM_001122646.2,XM_005263964.1,XM_005263965.1;GL=FAM178B;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=55/680,29/654,55/494;CDP=164,86,164;GS=64;CP=0.000;CG=-4.560;AA=A;HA=30.4;HE=13.6;HC=45.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=22;GESP=G:3773/A:771;PAC=NP_001116118.2,XP_005264021.1,XP_005264022.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:12:60,0,10</p>

<p>2 98165911 rs1839230 T C 170 highCVG AC1=5;AF1=0.6034;DB;DP=35;DP4=0,3,12,4;FQ=5.73;MQ=50;PV4=0.036,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E483G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD36B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025190.3;VDB=0.0137;DN=92;DA=T/C;GM=NM_025190.3,XM_005263995.1,XM_005263996.1,XM_005263997.1,XM_005263998.1,XM_005263999.1,XM_005264000.1,XM_005264001.1;GL=ANKRD36B;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLU/GLY;PP=483/1354,483/1354,483/1354,483/1353,483/1320,251/1122,232/1103,483/939;CDP=1448,1448,1448,1448,1448,752,695,1448;GS=98;PH=0;CP=0.002;CG=-0.893;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=30;GESP=T:11082;PAC=NP_079466.3,XP_005264052.1,XP_005264053.1,XP_005264054.1,XP_005264055.1,XP_005264056.1,XP_005264057.1,XP_005264058.1 GT:GQ:PL 0/1:4:0,3,24 0/1:3:0,0,0 1/1:34:154,36,0 0/1:20:58,0,18</p>

<p>2 100055158 rs3087399 T C 126 highCVG AC1=5;AF1=0.5732;DB;DP=19;DP4=8,3,4,4;FQ=128;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N373S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=REV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016316.2;VDB=0.0092;DN=102;DA=T/C;GM=NM_001037872.1,NM_016316.2,XM_005263967.1,XM_005263968.1,XM_005263969.1;GL=REV1;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown,unknown;AAC=ASN/SER,ASN/SER,ASN/SER,ASN/SER,none;PP=373/1251,373/1252,373/1252,303/1182,NA;CDP=1118,1118,1118,908,NA;GS=46,46,46,46,NA;PH=0.069;CP=0.794;CG=-0.648;AA=T;HA=22.9;HE=10.8;HC=3.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=96;GESP=C:2153/T:10853;PPI=MAD2L2:999|REV3L:998|POLH:995|POLI:846|POLK:846|UBC:845;PAC=NP_001032961.1,NP_057400.1,XP_005264024.1,XP_005264025.1,NA GT:GQ:PL 0/1:19:54,0,17 0/0:26:0,30,163 1/1:8:72,9,0 1/1:5:48,6,0</p>

<p>2 101086966 rs13411940 C T 23.20 midCVG AC1=3;AF1=0.4057;DB;DP=11;DP4=4,1,2,2;FQ=25.4;MQ=60;PV4=0.52,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=P6S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NMS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001011717.1;VDB=0.0096;DN=121;DA=C/T;GM=NM_001011717.1;GL=NMS;FG=missense;FD=missense;AAC=PRO/SER;PP=6/154;CDP=16;GS=74;PH=0.363;CP=0.003;CG=2.530;AA=C;HA=28.1;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=61;GESP=T:3092/C:9914;PAC=NP_001011717.1 GT:GQ:PL 0/0:9:0,9,52 0/1:11:11,0,13 1/1:5:56,9,0 0/0:4:0,3,29</p>

<p>2 113537223 rs17561 C A 31.40 midCVG AC1=2;AF1=0.2957;DB;DP=14;DP4=9,1,4,0;FQ=32.8;MQ=60;PV4=1,0.45,1,1;SNPEFF_AMINO_ACID_CHANGE=A114S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000575.3;VDB=0.0004;DN=63;DA=C/A;GM=NM_000575.3;GL=IL1A;FG=missense;FD=missense;AAC=ALA/SER;PP=114/272;CDP=340;GS=99;PH=0.982;CP=0.009;CG=4.440;AA=C;HA=14.5;HE=27.6;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;KP=hsa04010:MAPKSignalingPathway|hsa04060:Cytokine-cytokineReceptorInteraction|hsa04210:Apoptosis|hsa04640:HematopoieticCellLineage|hsa04940:TypeIDiabetesMellitus|hsa05020:PrionDiseases|hsa05140:Leishmaniasis|hsa05332:Graft-versus-hostDisease;GESP=A:3403/C:9603;PPI=S100A13:959;PAC=NP_000566.3 GT:GQ:PL 0/0:26:0,24,122 0/0:6:0,3,27 1/1:6:74,12,0 0/0:6:0,3,27</p>

<p>2 120885310 rs1034489 G A 53.80 highCVG AC1=4;AF1=0.5281;DB;DP=21;DP4=0,9,6,2;FQ=57.1;MQ=60;PV4=0.0023,0.00019,1,1;SNPEFF_AMINO_ACID_CHANGE=A462T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020909.3;VDB=0.0023;DN=86;DA=G/A;GM=NM_001184937.1,NM_020909.3,XM_005263719.1,XM_005263720.1,XM_005263721.1,XM_005263722.1;GL=EPB41L5;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=462/688,462/734,465/737,465/736,462/734,390/662;CDP=1384,1384,1393,1393,1384,1168;GS=58;PH=0;CP=0.973;CG=1.730;AA=A;HA=41.4;HE=26.4;HC=32.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$ROAZ_01;GESP=A:8169/G:4837;PAC=NP_001171866.1,NP_065960.2,XP_005263776.1,XP_005263777.1,XP_005263778.1,XP_005263779.1 GT:GQ:PL 0/1:35:32,0,51 0/1:3:0,3,21 1/1:5:32,6,0 0/1:36:33,0,69</p>

<p>2 131414176 rs62165276 C T 7.93 midCVG AC1=4;AF1=0.5176;DB;DP=10;DP4=1,3,3,2;FQ=10.6;MQ=18;PV4=0.52,0.35,0.061,0.14;SNPEFF_AMINO_ACID_CHANGE=R615W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POTEJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277083.1;VDB=0.0090;DN=129;DA=C/T;GM=NM_001277083.1,XM_005263739.1,XM_005263740.1;GL=POTEJ;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=615/1039,497/921,492/916;CDP=1843,1489,1474;GS=101;CP=0.246;CG=0.993;AA=C;DG;DV=by-frequency,by-cluster;DSP=55;PAC=NP_001264012.1,XP_005263796.1,XP_005263797.1 GT:GQ:PL 0/0:6:0,6,33 0/1:3:0,0,0 1/1:5:39,9,0 0/1:10:9,0,34</p>

<p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

<p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>2 159954175 rs34588551 C T 73.30 midCVG AC1=4;AF1=0.5077;DB;DP=12;DP4=4,2,2,4;FQ=64.1;MQ=60;PV4=0.57,1,1,0.077;SNPEFF_AMINO_ACID_CHANGE=P30S;SNPEFF_CODON_CHANGE=Cca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TANC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033394.2;VDB=0.0215;DN=126;DA=C/T;GM=NM_001145909.1,NM_033394.2,XM_005246918.1,XM_005246919.1,XM_005246920.1;GL=TANC1;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=PRO/SER;PP=30/1391,30/1862,40/1872,40/1865,40/1401;CDP=88,88,118,118,118;GS=74;PH=0.004;CP=0.003;CG=0.726;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=27;GESP=T:3302/C:8928;PAC=NP_001139381.1,NP_203752.2,XP_005246975.1,XP_005246976.1,XP_005246977.1 GT:GQ:PL 0/0:13:0,15,95 0/1:3:0,0,0 1/1:12:98,15,0 0/1:13:19,0,10</p>

<p>2 160711045 rs3951216 G C 49.60 lowCVG AC1=4;AF1=0.531;DB;DP=8;DP4=1,2,2,3;FQ=23.9;MQ=60;PV4=1,0.4,1,0.026;SNPEFF_AMINO_ACID_CHANGE=D807E;SNPEFF_CODON_CHANGE=gaC/gaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LY75-CD302;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198759.1;VDB=0.0092;DN=108;DA=G/C;GM=NM_001198759.1,NM_001198760.1,NM_002349.3,XM_005246553.1;GL=LY75/LY75-CD302;FG=missense-near-splice;FD=missense,missense,missense,unknown;AAC=ASP/GLU;PP=807/1874,807/1818,807/1723,807/1625;CDP=2421;GS=45;PH=0.002;CP=1.000;CG=3.670;AA=C;HA=39.2;HE=41.7;HC=32.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2;GESP=C:5483/G:7523;PAC=NP_001185688.1,NP_001185689.1,NP_002340.2,XP_005246610.1 GT:GQ:PL 0/0:5:0,6,42 0/1:3:0,0,0 1/1:7:59,9,0 0/1:23:32,0,20</p>

<p>2 167099158 rs6746030 A G 150 highCVG AC1=6;AF1=0.7278;DB;DP=16;DP4=2,4,7,3;FQ=65.7;MQ=60;PV4=0.3,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W1150R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCN9A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002977.3;VDB=0.0076;DN=116;DA=A/G;GM=NM_002977.3,XM_005246757.1;GL=SCN9A;FG=missense;FD=missense,unknown;AAC=TRP/ARG;PP=1150/1978,1161/1989;CDP=3448,3481;GS=101;PH=0;CP=0.788;CG=3.830;AA=G;HA=13.1;HE=12.3;HC=4.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=9;GESP=G:10303/A:1439;PAC=NP_002968.1,XP_005246814.1 GT:GQ:PL 0/1:43:47,0,44 0/1:22:15,0,61 1/1:10:59,9,0 1/1:10:74,9,0</p>

<p>2 178417142 rs11694988 C T 97.90 midCVG AC1=5;AF1=0.5858;DB;DP=13;DP4=2,4,3,4;FQ=78.7;MQ=50;PV4=1,0.46,0.044,0.32;SNPEFF_AMINO_ACID_CHANGE=R117H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152517.2;VDB=0.0034;DN=120;DA=C/T;GM=NM_152517.2;GL=TTC30B;FG=missense;FD=missense;AAC=ARG/HIS;PP=117/666;CDP=350;GS=29;PH=0.944;CP=0.996;CG=4.300;AA=C;HA=15.8;HE=14.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14341;GESP=T:11079/C:1927;PAC=NP_689730.2 GT:GQ:PL 0/0:11:0,15,107 0/1:18:35,0,16 1/1:8:64,9,0 1/1:6:47,6,0</p>

<p>2 185802243 rs1366842 C A 30.40 highCVG AC1=2;AF1=0.2953;DB;DP=17;DP4=3,8,4,0;FQ=31.8;MQ=60;PV4=0.026,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T707K;SNPEFF_CODON_CHANGE=aCa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF804A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_194250.1;VDB=0.0182;DN=88;DA=C/A;GM=NM_194250.1;GL=ZNF804A;FG=missense;FD=missense;AAC=THR/LYS;PP=707/1210;CDP=2120;GS=78;PH=0.002;CP=0.000;CG=1.860;AA=C;HA=4.5;HE=40.1;HC=13.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1734;GESP=A:5568/C:7416;PAC=NP_919226.1 GT:GQ:PL 0/0:6:0,3,26 0/0:29:0,27,124 1/1:6:73,12,0 0/0:6:0,3,28</p>

<p>2 186656956 rs17229201 T C 27.70 lowCVG AC1=5;AF1=0.6553;DB;DP=5;DP4=2,0,2,1;FQ=12.6;MQ=60;PV4=1,0.22,1,0.25;SNPEFF_AMINO_ACID_CHANGE=M1787T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FSIP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173651.2;VDB=0.0061;DN=123;DA=T/C;GM=NM_173651.2;GL=FSIP2;FG=missense;FD=missense;AAC=MET/THR;PP=1787/6997;CDP=5360;GS=81;CP=0.371;CG=3.390;AA=C;HA=42.4;HE=48.8;HC=46.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3587;PAC=NP_775922.2 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,48 1/1:4:21,3,0 1/1:5:47,6,0</p>

<p>2 196749503 rs371128668 C T 102 highCVG AC1=5;AF1=0.5615;DB;DP=15;DP4=2,2,5,4;FQ=23.9;MQ=60;PV4=1,0.15,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V1857I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018897.2;VDB=0.0045;DN=138;DA=C/T;GM=NM_018897.2;GL=DNAH7;FG=missense;FD=missense;AAC=VAL/ILE;PP=1857/4025;CDP=5569;GS=29;PH=0.001;CP=0.000;CG=-11.500;AA=C;DSP=31;GESP=T:4/C:11878;PAC=NP_061720.2 GT:GQ:PL 0/1:27:24,0,31 0/1:3:0,3,25 1/1:8:64,9,0 0/1:11:58,0,8</p>

<p>2 196837129 rs73042525 G C 20.80 midCVG AC1=4;AF1=0.4605;DB;DP=9;DP4=2,2,0,3;FQ=23.2;MQ=60;PV4=0.43,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=S632C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018897.2;VDB=0.0121;DN=130;DA=G/C;GM=NM_018897.2;GL=DNAH7;FG=missense;FD=missense;AAC=SER/CYS;PP=632/4025;CDP=1895;GS=112;PH=0.009;CP=0.997;CG=4.750;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=62;GESP=C:856/G:11112;PAC=NP_061720.2 GT:GQ:PL 0/0:6:0,6,37 0/1:4:22,3,0 1/1:4:43,6,0 0/0:6:0,6,39</p>

<p>2 207570529 rs56067462 G C 69.40 midCVG AC1=4;AF1=0.5258;DB;DP=14;DP4=2,5,4,2;FQ=67.2;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P122R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DYTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001093730.1;VDB=0.0048;DN=129;DA=G/C;GM=NM_001093730.1;GL=DYTN;FG=missense;FD=missense;AAC=PRO/ARG;PP=122/579;CDP=365;GS=103;PH=1;CP=1.000;CG=4.150;AA=G;DV=by-frequency,by-cluster;DSP=18;GESP=C:854/G:11154;PAC=NP_001087199.1 GT:GQ:PL 0/1:3:0,0,0 0/0:15:0,18,106 1/1:7:57,9,0 0/1:6:54,0,1</p>

<p>2 207820276 rs13420911 G A 43.40 highCVG AC1=2;AF1=0.2979;DB;DP=18;DP4=6,8,2,2;FQ=44.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M85I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173077.2;VDB=0.0134;DN=121;DA=G/A;GM=NM_173077.2;GL=CPO;FG=missense;FD=missense;AAC=MET/ILE;PP=85/375;CDP=255;GS=10;PH=0.021;CP=0.999;CG=5.110;AA=G;HA=33.2;HE=44.8;HC=39.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=13;GESP=A:5799/G:7207;PAC=NP_775100.1 GT:GQ:PL 0/0:20:0,18,108 0/0:3:0,0,0 1/1:6:86,12,0 0/0:25:0,24,153</p>

<p>2 218674697 rs918949 C T 84.60 midCVG AC1=5;AF1=0.6725;DB;DP=9;DP4=0,3,2,3;FQ=25.6;MQ=60;PV4=0.46,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V1604I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022648.4;VDB=0.0060;DN=86;DA=C/T;GM=NM_022648.4,XM_005246821.1,XM_005246822.1,XM_005246823.1,XM_005246824.1,XM_005246825.1,XM_005246826.1,XM_005246827.1;GL=TNS1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=1604/1736,1708/1840,1677/1809,1677/1809,1583/1715,1387/1519,1221/1353,1221/1353;CDP=4810,5122,5029,5029,4747,4159,3661,3661;GS=29;PH=0;CP=0.368;CG=-4.710;HA=17.3;HE=39.5;HC=47.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=18;GESP=T:8637/C:4369;PAC=NP_072174.3,XP_005246878.1,XP_005246879.1,XP_005246880.1,XP_005246881.1,XP_005246882.1,XP_005246883.1,XP_005246884.1 GT:GQ:PL 0/0:6:0,9,64 0/1:3:0,0,0 1/1:6:50,6,0 1/1:8:78,9,0</p>

<p>2 220353532 rs13026308 C A 58.90 midCVG AC1=6;AF1=0.7369;DB;DP=11;DP4=1,1,5,1;FQ=-4.99;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2687T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=34;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005876.4;VDB=0.0094;DN=121;DA=C/A;GM=NM_005876.4,XM_005246235.1,XM_005246236.1,XM_005246237.1,XM_005246238.1,XM_005246239.1,XM_005246240.1,XM_005246241.1,XM_005246242.1;GL=SPEG;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/THR;PP=2687/3268,2628/3209,2626/3207,2593/3174,2534/3115,1895/2476,1838/2419,1838/2419,1900/2481;CDP=8059,7882,7876,7777,7600,5683,5512,5512,5698;GS=38;PH=0.997;CP=0.999;CG=4.380;AA=C;HA=0.0;HE=16.1;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;GESP=A:1183/C:11149;PAC=NP_005867.3,XP_005246292.1,XP_005246293.1,XP_005246294.1,XP_005246295.1,XP_005246296.1,XP_005246297.1,XP_005246298.1,XP_005246299.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:9:54,9,0 1/1:9:48,9,0</p>

<p>2 228476253 rs2138402 C G 23.70 lowCVG AC1=4;AF1=0.4527;DB;DP=8;DP4=5,0,1,2;FQ=26.3;MQ=59;PV4=0.11,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E104Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf83;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020161.3;VDB=0.0071;DN=96;DA=C/G;GM=NM_001162483.1,NM_020161.3;GL=C2orf83;FG=3-prime-UTR,missense;FD=utr-variant-3-prime,missense;AAC=none,GLU/GLN;PP=NA,104/151;CDP=NA,310;GS=NA,29;PH=0;CP=0.000;CG=0.457;AA=G;HA=33.2;HE=6.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=120;GESP=G:10654/C:2352;PAC=NA,NP_064546.3 GT:GQ:PL 0/0:11:0,12,62 0/0:4:0,3,25 1/1:4:42,6,0 0/1:4:26,3,0</p>

<p>2 228883029 rs3811515 A C 134 midCVG AC1=5;AF1=0.5723;DB;DP=12;DP4=1,2,4,5;FQ=24.2;MQ=60;PV4=1,0.15,1,0.21;SNPEFF_AMINO_ACID_CHANGE=H847Q;SNPEFF_CODON_CHANGE=caT/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPHKAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001142644.1;VDB=0.0094;DN=107;DA=A/C;GM=NM_001142644.1,NM_030623.3,XM_005246867.1,XM_005246868.1,XM_005246869.1,XM_005246870.1;GL=SPHKAP;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=HIS/GLN;PP=847/1701,847/1672,860/1714,843/1697,860/1685,830/1684;CDP=2541,2541,2580,2529,2580,2490;GS=24;PH=0.019;CP=0.000;CG=0.984;AA=A;HA=0.9;HE=27.9;HC=30.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1844;GESP=C:2449/A:10557;PAC=NP_001136116.1,NP_085126.2,XP_005246924.1,XP_005246925.1,XP_005246926.1,XP_005246927.1 GT:GQ:PL 0/0:4:0,6,55 0/1:3:0,0,0 1/1:10:84,12,0 0/1:10:92,0,7</p>

<p>2 228883721 rs3811514 T C 19.80 midCVG AC1=3;AF1=0.4098;DB;DP=10;DP4=5,2,1,2;FQ=22;MQ=60;PV4=0.5,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K617E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPHKAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001142644.1;VDB=0.0098;DN=107;DA=T/C;GM=NM_001142644.1,NM_030623.3,XM_005246867.1,XM_005246868.1,XM_005246869.1,XM_005246870.1;GL=SPHKAP;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=617/1701,617/1672,630/1714,613/1697,630/1685,600/1684;CDP=1849,1849,1888,1837,1888,1798;GS=56;PH=0;CP=0.074;CG=4.050;AA=C;HA=17.4;HE=44.8;HC=31.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1152;GESP=C:7539/T:5467;PAC=NP_001136116.1,NP_085126.2,XP_005246924.1,XP_005246925.1,XP_005246926.1,XP_005246927.1 GT:GQ:PL 0/0:14:0,15,90 0/0:6:0,6,42 1/1:3:45,6,0 0/1:4:20,3,0</p>

<p>2 231865090 rs2271376 T C 84.80 midCVG AC1=4;AF1=0.5465;DB;DP=9;DP4=1,2,4,2;FQ=5.61;MQ=60;PV4=0.52,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=I104T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139073.3;VDB=0.0134;DN=100;DA=T/C;GM=NM_139073.3,XM_005246303.1;GL=SPATA3;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=104/193;CDP=311;GS=89;PH=0;CP=0.001;CG=-0.717;AA=C;HA=48.8;HE=35.5;HC=40.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;GESP=C:1865/T:2701;PAC=NP_620712.2,XP_005246360.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,16 1/1:5:47,6,0 0/1:27:77,0,24</p>

<p>2 234526871 rs1042597 C G 32.10 midCVG AC1=4;AF1=0.5;DB;DP=9;DP4=0,4,0,4;FQ=34.9;MQ=50;PV4=1,0.18,0.04,1;SNPEFF_AMINO_ACID_CHANGE=A173G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UGT1A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019076.4;VDB=0.0066;DN=86;DA=C/G/T;GM=NM_019076.4;GL=UGT1A8;FG=missense;FD=missense;AAC=ALA/GLY;PP=173/531;CDP=518;GS=60;PH=0.044;CP=0.002;CG=-0.180;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=338;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00053:AscorbateAndAldarateMetabolism|hsa00140:SteroidHormoneBiosynthesis|hsa00500:StarchAndSucroseMetabolism|hsa00830:RetinolMetabolism|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:2472/C:10534;PAC=NP_061949.3 GT:GQ:PL 0/0:5:0,6,48 0/0:5:0,6,44 1/1:4:39,6,0 1/1:4:39,6,0</p>

<p>2 237149941 rs6756597 C T 24.40 highCVG AC1=2;AF1=0.254;DB;DP=18;DP4=7,5,4,0;FQ=25.6;MQ=60;PV4=0.24,0.36,1,0.041;SNPEFF_AMINO_ACID_CHANGE=A104T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ASB18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_212556.2;VDB=0.0223;DN=116;DA=C/T;GM=NM_212556.2;GL=ASB18;FG=missense;FD=missense;AAC=ALA/THR;PP=104/467;CDP=310;GS=58;PH=0.76;CP=0.928;CG=4.820;AA=C;HA=29.5;HE=46.2;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19;TFBS=V$P53_01,V$SRF_Q6;GESP=T:5089/C:6967;PAC=NP_997721.2 GT:GQ:PL 0/0:6:0,3,20 0/0:23:0,21,119 1/1:5:68,12,0 0/0:14:0,12,79</p>

<p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

<p>2 242129524 rs34069570 G A 33.10 lowCVG AC1=4;AF1=0.4716;DB;DP=8;DP4=0,4,1,3;FQ=32.3;MQ=60;PV4=1,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=D70N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANO7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001891.3;VDB=0.0068;DN=126;DA=G/A;GM=NM_001001666.3,NM_001001891.3;GL=ANO7;FG=missense;FD=missense;AAC=ASP/ASN;PP=70/180,70/934;CDP=208;GS=23;PH=0.025;CP=0.001;CG=-1.970;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=53;TFBS=V$CDP_02;GESP=A:264/G:12706;PAC=NP_001001666.1,NP_001001891.2 GT:GQ:PL 0/1:3:0,0,0 0/1:22:42,0,18 1/1:4:32,6,0 0/0:8:0,9,55</p>

<p>3 361508 rs2272522 C T 27.20 midCVG AC1=5;AF1=0.5739;DB;DP=10;DP4=2,2,1,3;FQ=30.5;MQ=60;PV4=1,0.13,1,1;SNPEFF_AMINO_ACID_CHANGE=L17F;SNPEFF_CODON_CHANGE=Ctc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006614.3;VDB=0.0091;DN=100;DA=C/T;GM=NM_001253387.1,NM_001253388.1,NM_006614.3;GL=CHL1;FG=missense;FD=missense;AAC=LEU/PHE;PP=17/1209,17/1172,17/1225;CDP=49;GS=22;PH=0.001;CP=0.003;CG=-2.030;AA=T;HA=37.9;HE=21.9;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=43;GESP=T:4018/C:8986;PAC=NP_001240316.1,NP_001240317.1,NP_006605.2 GT:GQ:PL 0/1:13:16,0,11 0/0:6:0,9,70 1/1:3:20,3,0 1/1:5:36,6,0</p>

<p>3 12613601 . C T 92.20 highCVG AC1=3;AF1=0.4135;DP=17;DP4=2,6,4,3;FQ=80.3;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P124L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MKRN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014160.4;VDB=0.0150;GM=NM_001271707.1,NM_014160.4;GL=MKRN2;FG=missense;FD=unknown;AAC=PRO/LEU;PP=81/374,124/417;CDP=242,371;GS=98;PH=0.856;CP=0.000;CG=0.672;AA=C;DSP=34;GESP=C:13006;PAC=NP_001258636.1,NP_054879.3 GT:GQ:PL 0/0:14:0,15,75 0/0:4:0,3,26 1/1:8:84,12,0 0/1:28:54,0,23</p>

<p>3 12859004 rs3732678 A C 27.40 lowCVG AC1=5;AF1=0.5732;DB;DP=7;DP4=3,1,2,1;FQ=30.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H858P;SNPEFF_CODON_CHANGE=cAc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CAND2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001162499.1;VDB=0.0098;DN=107;DA=A/C/G/T;GM=NM_001162499.1,NM_012298.2;GL=CAND2;FG=missense;FD=missense;AAC=HIS/PRO;PP=858/1237,765/1120;CDP=2573,2294;GS=77;PH=0;CP=1.000;CG=3.880;AA=C;HA=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=372;TFBS=V$PAX5_02;GESP=C:8707/A:3773;PAC=NP_001155971.1,NP_036430.1 GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:19,3,0</p>

<p>3 13361287 rs354478 C T 70.90 midCVG AC1=5;AF1=0.5672;DB;DP=11;DP4=4,1,4,2;FQ=39.6;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=V1787M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=37;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUP210;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024923.3;VDB=0.0032;DN=79;DA=C/T;GM=NM_024923.3;GL=NUP210;FG=missense;FD=missense;AAC=VAL/MET;PP=1787/1888;CDP=5359;GS=21;PH=0;CP=0.698;CG=-1.130;AA=T;HA=43.7;HE=48.6;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=25;GESP=T:6670/C:6336;PAC=NP_079199.2 GT:GQ:PL 0/1:26:49,0,24 0/0:6:0,9,57 1/1:5:44,6,0 1/1:3:23,3,0</p>

<p>3 14755617 rs6790129 C G 50.50 midCVG AC1=3;AF1=0.3771;DB;DP=13;DP4=5,2,5,0;FQ=51.9;MQ=60;PV4=0.47,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L422V;SNPEFF_CODON_CHANGE=Cta/Gta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C3orf20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032137.4;VDB=0.0072;DN=116;DA=C/G;GM=NM_001184957.1,NM_001184958.1,NM_032137.4,XM_005265511.1;GL=C3orf20;FG=missense;FD=missense,missense,missense,unknown;AAC=LEU/VAL;PP=300/783,300/783,422/905,422/905;CDP=898,898,1264,1264;GS=32;PH=0.873;CP=0.948;CG=1.570;AA=C;HA=43.4;HE=39.8;HC=34.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=50;TFBS=V$HOX13_01;GESP=G:5559/C:7447;PAC=NP_001171886.1,NP_001171887.1,NP_115513.4,XP_005265568.GT:GQ:PL 0/0:9:0,9,40 0/0:9:0,9,56 1/1:5:56,9,0 0/1:14:40,0,8</p>

<p>3 45637026 . C A 59.40 midCVG AC1=4;AF1=0.5;DP=14;DP4=1,3,0,9;FQ=9.86;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P219T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIMD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014240.2;VDB=0.0059;GM=NM_014240.2;GL=LIMD1;FG=missense;FD=unknown;AAC=PRO/THR;PP=219/677;CDP=655;GS=38;PH=0.004;CP=0.002;CG=-0.748;AA=C;DSP=754;GESP=C:13006;PAC=NP_055055.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:97,9,0 0/0:8:0,9,44</p>

<p>3 57136585 rs17057718 C T 51.40 midCVG AC1=4;AF1=0.493;DB;DP=12;DP4=6,0,6,0;FQ=54.2;MQ=60;PV4=1,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=V301M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=123;DA=C/T;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=VAL/MET;PP=301/740,277/716,273/712,157/596;CDP=901,829,817,469;GS=21;PH=0.777;CP=0.215;CG=3.570;AA=C;HA=4.1;HE=17.7;HC=38.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;TFBS=V$GFI1_01;GESP=T:1623/C:11383;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/1:8:5,0,26 0/0:7:0,9,66 1/1:9:72,12,0 0/1:20:20,0,19</p>

<p>3 57136593 rs368070640 A G 14.20 midCVG AC1=3;AF1=0.3818;DB;DP=12;DP4=6,0,4,0;FQ=16.2;MQ=60;PV4=1,0.044,1,1;SNPEFF_AMINO_ACID_CHANGE=I298T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=138;DA=A/G;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=298/740,274/716,270/712,154/596;CDP=893,821,809,461;GS=89;PH=0.986;CP=1.000;CG=5.480;AA=A;DSP=25;TFBS=V$GFI1_01;GESP=A:13006;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/0:7:0,6,34 0/0:9:0,9,61 1/1:5:45,9,0 0/1:14:14,0,19</p>

<p>3 57335876 rs4060726 C T 103 midCVG AC1=5;AF1=0.6609;DB;DP=10;DP4=0,3,4,3;FQ=22.3;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G2904S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=54;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0001;DN=108;DA=C/T;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=GLY/SER;PP=2893/3093;CDP=8677;GS=56;PH=0.12;CP=0.988;CG=2.340;AA=C;HA=44.4;HE=41.5;HC=8.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT5A_02;GESP=T:2634/C:1932;PAC=NP_848599.3 GT:GQ:PL 0/1:22:17,0,32 0/1:4:0,3,30 1/1:4:29,3,0 1/1:14:101,15,0</p>

<p>3 57414071 rs4681982 G A 107 highCVG AC1=4;AF1=0.4999;DB;DP=19;DP4=3,7,5,4;FQ=110;MQ=60;PV4=0.37,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=T1786I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0121;DN=111;DA=G/A;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=THR/ILE;PP=1763/3093;CDP=5288;GS=89;PH=0;CP=0.003;CG=-0.547;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6;GESP=G:4506;PAC=NP_848599.3 GT:GQ:PL 0/1:32:30,0,41 0/0:5:0,6,44 1/1:5:50,6,0 0/1:74:71,0,91</p>

<p>3 66419956 rs13874 C T 145 highCVG AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA GT:GQ:PL 0/0:7:0,9,70 0/1:3:0,0,0 1/1:18:112,21,0 0/1:31:77,0,28</p>

<p>3 67411103 rs902320 C T 117 highCVG AC1=5;AF1=0.6304;DB;DP=20;DP4=1,9,6,3;FQ=117;MQ=60;PV4=0.02,0.13,1,1;SNPEFF_AMINO_ACID_CHANGE=V425I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0045;DN=86;DA=C/T;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=VAL/ILE;PP=425/441;CDP=1273;GS=29;CP=0.002;CG=1.640;AA=C;HA=20.5;HE=27.2;HC=25.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=90;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:39:34,0,100 0/0:6:0,9,63 1/1:9:62,9,0 1/1:9:69,9,0</p>

<p>3 67411166 rs902321 A G 144 highCVG AC1=5;AF1=0.6662;DB;DP=15;DP4=2,3,6,4;FQ=34.2;MQ=60;PV4=0.61,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Y404H;SNPEFF_CODON_CHANGE=Tat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0085;DN=86;DA=A/G;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=TYR/HIS;PP=404/441;CDP=1210;GS=83;CP=0.001;CG=3.220;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=27;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:33:42,0,33 0/0:3:0,6,44 1/1:9:70,9,0 1/1:12:80,12,0</p>

<p>3 69245454 rs9310141 T C 6.32 lowCVG AC1=8;AF1=1;DB;DP=5;DP4=0,0,2,0;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T396A;SNPEFF_CODON_CHANGE=Acg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMD4B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015123.2;VDB=0.0175;DN=119;DA=T/C;GM=NM_015123.1,XM_005264720.1,XM_005264721.1,XM_005264722.1,XM_005264723.1;GL=FRMD4B;FG=missense;FD=unknown;AAC=THR/ALA;PP=396/1035,365/1004,363/1002,351/990,48/687;CDP=1186,1093,1087,1051,142;GS=58;PH=0;CP=0.803;CG=6.070;AA=C;HA=13.4;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;GESP=C:11705/T:393;PAC=NP_055938.1,XP_005264777.1,XP_005264778.1,XP_005264779.1,XP_005264780.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 1/1:4:25,3,0</p>

<p>3 75786748 rs112332186 C T 54.70 midCVG AC1=6;AF1=0.7558;DB;DP=13;DP4=3,0,3,3;FQ=-23.2;MQ=32;PV4=0.46,0.25,1,1;SNPEFF_AMINO_ACID_CHANGE=D676N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF717;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128223.1;VDB=0.0067;DN=132;DA=C/T;GM=NM_001128223.1,XM_005264710.1,XM_005264711.1,XM_005264712.1,XM_005264713.1;GL=ZNF717;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=ASP/ASN;PP=676/915;CDP=2026;GS=23;PH=0;CP=0.000;CG=-3.140;AA=C;DV=by-frequency,by-cluster;DSP=1749;GESP=C:4354;PAC=NP_001121695.1,XP_005264767.1,XP_005264768.1,XP_005264769.1,XP_005264770.1 GT:GQ:PL 0/1:6:49,0,5 0/1:4:27,0,1 1/1:3:13,1,0 0/1:3:0,0,0</p>

<p>3 98512540 rs28489284 G A 55.60 midCVG AC1=3;AF1=0.4126;DB;DP=12;DP4=5,1,1,4;FQ=57.2;MQ=60;PV4=0.08,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A364T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ST3GAL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271145.1;VDB=0.0070;DN=125;DA=G/A;GM=NM_001271142.1,NM_001271145.1,NM_001271146.1,NM_001271147.1,NM_001271148.1,NM_006100.3,XM_005247067.1,XM_005247068.1,XM_005247069.1,XM_005247070.1,XM_005247071.1,XM_005247072.1;GL=ST3GAL6;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=225/246,364/385,311/332,193/214,167/188,311/332,334/355,333/354,311/332,311/332,167/188,167/188;CDP=673,1090,931,577,499,931,1000,997,931,931,499,499;GS=58;PH=0.934;CP=1.000;CG=5.930;AA=G;HA=13.0;HE=2.9;HC=2.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa00601:GlycosphingolipidBiosynthesis-LactoAndNeolactoSeries|hsa01100:MetabolicPathways;GESP=A:682/G:12324;PAC=NP_001258071.1,NP_001258074.1,NP_001258075.1,NP_001258076.1,NP_001258077.1,NP_006091.1,XP_005247124.1,XP_005247125.1,XP_005247126.1,XP_005247127.1,XP_005247128.1,XP_005247129.1 GT:GQ:PL 0/0:4:0,3,25 0/0:9:0,9,53 1/1:5:69,9,0 0/1:30:31,0,30</p>

<p>3 98600385 rs9838238 T C 36.30 midCVG AC1=3;AF1=0.3969;DB;DP=9;DP4=1,3,2,2;FQ=36.7;MQ=60;PV4=1,0.28,1,0.067;SNPEFF_AMINO_ACID_CHANGE=I144M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCBLD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080927.3;VDB=0.0156;DN=119;DA=T/C;GM=NM_080927.3,XM_005247118.1;GL=DCBLD2;FG=missense-near-splice;FD=missense,unknown;AAC=ILE/MET;PP=144/776,144/790;CDP=432;GS=10;PH=1;CP=1.000;CG=4.900;AA=T;HA=1.2;HE=2.3;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=C:437/T:11499;PAC=NP_563615.3,XP_005247175.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,3,22 1/1:7:77,12,0 0/0:9:0,9,58</p>

<p>3 101283792 rs3762735 C G 93 highCVG AC1=4;AF1=0.4986;DB;DP=20;DP4=6,4,7,2;FQ=96;MQ=60;PV4=0.63,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=P56R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRMT10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017819.3;VDB=0.0174;DN=107;DA=C/G;GM=NM_017819.3;GL=TRMT10C;FG=missense;FD=missense;AAC=PRO/ARG;PP=56/404;CDP=167;GS=103;PH=0.001;CP=0.084;CG=5.160;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=179;GESP=G:1711/C:10225;PAC=NP_060289.2 GT:GQ:PL 0/0:15:0,18,120 0/1:29:26,0,47 1/1:15:98,18,0 0/1:20:17,0,29</p>

<p>3 111901099 rs74840030 T C 15.40 midCVG AC1=3;AF1=0.3387;DB;DP=11;DP4=4,3,2,2;FQ=17;MQ=60;PV4=1,0.00097,1,0.41;SNPEFF_AMINO_ACID_CHANGE=M844V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC9C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_183061.1;VDB=0.0154;DN=132;DA=T/C;GM=NM_183061.1,XM_005247381.1;GL=SLC9C1;FG=missense;FD=missense,unknown;AAC=MET/VAL;PP=844/1178,796/1130;CDP=2530,2386;GS=21;PH=0.39;CP=1.000;CG=4.590;AA=T;DV=by-frequency,by-cluster;DSP=7;TFBS=V$HNF1_01;GESP=C:495/T:12501;PAC=NP_898884.1,XP_005247438.1 GT:GQ:PL 0/0:10:0,9,74 0/1:5:4,0,40 1/1:4:54,9,0 0/0:8:0,6,51</p>

<p>3 121208833 rs3218649 G C 208 highCVG AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3 GT:GQ:PL 0/1:37:59,0,36 0/0:43:0,48,136 1/1:20:109,21,0 1/1:11:91,12,0</p>

<p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>3 140406947 rs28594654 G A 110 highCVG AC1=4;AF1=0.5408;DB;DP=15;DP4=3,2,9,1;FQ=41;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V475M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM42;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152616.4;VDB=0.0045;DN=125;DA=G/A;GM=NM_152616.4,XM_005247397.1,XM_005247398.1;GL=TRIM42;FG=missense;FD=missense,unknown,unknown;AAC=VAL/MET;PP=475/724,475/600,328/453;CDP=1423,1423,982;GS=21;PH=0;CP=0.926;CG=1.790;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=66;GESP=A:6351/G:6655;PAC=NP_689829.3,XP_005247454.1,XP_005247455.1 GT:GQ:PL 0/1:54:96,0,51 0/1:3:0,0,0 1/1:5:53,6,0 0/0:3:0,3,26</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>3 145938619 rs3762685 T C 14 lowCVG AC1=4;AF1=0.4982;DB;DP=7;DP4=3,0,2,1;FQ=16.2;MQ=60;PV4=1,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=N34S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLSCR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128304.1;VDB=0.0043;DN=107;DA=T/C;GM=NM_001128304.1,NM_001128305.1,NM_001128306.1,NM_001177304.1,NM_020353.2,XM_005247654.1,XM_005247655.1;GL=PLSCR4;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ASN/SER;PP=34/330,34/330,34/240,19/225,34/330,34/330,34/330;CDP=101,101,101,56,101,101,101;GS=46;PH=0.001;CP=0.075;CG=-8.490;AA=T;HA=34.4;HE=37.5;HC=22.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18;GESP=C:4565/T:8441;PAC=NP_001121776.1,NP_001121777.1,NP_001121778.1,NP_001170775.1,NP_065086.2,XP_005247711.1,XP_005247712.1 GT:GQ:PL 0/0:6:0,6,39 0/0:4:0,3,26 1/1:4:38,6,0 0/1:4:19,3,0</p>

<p>3 185990096 rs2193587 C T 115 midCVG AC1=5;AF1=0.651;DB;DP=13;DP4=1,3,1,7;FQ=49.8;MQ=60;PV4=1,0.2,1,0.13;SNPEFF_AMINO_ACID_CHANGE=R316K;SNPEFF_CODON_CHANGE=aGa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DGKG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001346.2;VDB=0.0103;DN=96;DA=C/T;GM=NM_001080744.1,NM_001080745.1,NM_001346.2,XM_005247156.1,XM_005247157.1,XM_005247158.1;GL=DGKG;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=316/767,316/753,316/792,316/768,258/734,316/733;CDP=947,947,947,947,773,947;GS=26;PH=0;CP=1.000;CG=3.920;AA=T;HA=11.8;HE=14.2;HC=29.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=37;KP=hsa00561:GlycerolipidMetabolism|hsa00564:GlycerophospholipidMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=T:10726/C:2280;PAC=NP_001074213.1,NP_001074214.1,NP_001337.2,XP_005247213.1,XP_005247214.1,XP_005247215.1 GT:GQ:PL 0/0:3:0,6,53 0/1:37:36,0,39 1/1:11:72,12,0 1/1:6:54,6,0</p>

<p>3 190999917 rs6788319 C A 22.20 midCVG AC1=6;AF1=0.703;DB;DP=10;DP4=0,2,0,3;FQ=8.03;MQ=60;PV4=1,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=S21I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UTS2B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198152.3;VDB=0.0019;DN=116;DA=C/A;GM=NM_198152.3;GL=UTS2B;FG=missense;FD=missense;AAC=SER/ILE;PP=21/120;CDP=62;GS=142;PH=0;CP=0.000;CG=1.170;AA=T;HA=21.0;HE=39.2;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42;GESP=A:6408/C:6598;PAC=NP_937795.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:17:12,0,38</p>

<p>3 193209178 rs6788448 T C 32.90 lowCVG AC1=3;AF1=0.3939;DB;DP=6;DP4=2,1,1,2;FQ=30.9;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=I181M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP13A4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032279.2;VDB=0.0095;DN=116;DA=T/C;GM=NM_032279.2,XM_005247829.1;GL=ATP13A4;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=181/1197,181/1128;CDP=543;GS=10;PH=0.999;CP=1.000;CG=4.630;AA=T;HA=48.5;HE=41.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=C:5746/T:7260;PAC=NP_115655.2,XP_005247886.1 GT:GQ:PL 0/0:5:0,3,23 0/0:3:0,0,0 1/1:5:72,9,0 0/0:7:0,6,47</p>

<p>3 194062519 rs3732477 C T 62.10 highCVG AC1=3;AF1=0.3624;DB;DP=16;DP4=0,8,1,5;FQ=64;MQ=60;PV4=0.43,0.0087,1,1;SNPEFF_AMINO_ACID_CHANGE=A305T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080513.3;VDB=0.0065;DN=107;DA=C/T;GM=NM_001080513.2,XM_005269280.1,XM_005269281.1;GL=CPN2;FG=missense;FD=missense,unknown,unknown;AAC=ALA/THR;PP=305/546;CDP=913;GS=58;PH=0.017;CP=0.000;CG=-10.600;AA=C;HA=44.2;HE=26.5;HC=32.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=913;GESP=T:4238/C:8768;PAC=NP_001073982.2,XP_005269337.1,XP_005269338.1 GT:GQ:PL 0/0:9:0,9,59 0/0:12:0,12,79 1/1:5:63,9,0 0/1:19:45,0,13</p>

<p>3 195452799 rs2550232 C T 11.30 midCVG AC1=8;AF1=1;DB;DP=9;DP4=4,1,1,3;FQ=-26.1;MQ=21;PV4=0.21,0.34,1,0.42;SNPEFF_AMINO_ACID_CHANGE=T442I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282506.1;VDB=0.0102;DN=100;DA=C/T;GM=NM_001098516.1,NM_152673.2,XM_005269303.1,XM_005269304.1,XM_005269305.1,XM_005269306.1,XM_005269307.1,XM_005269308.1,XM_005269309.1;GL=MUC20;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ILE;PP=236/504,271/539,442/727,442/710,407/692,407/692,407/692,271/553,272/540;CDP=707,812,1325,1325,1220,1220,1220,812,815;GS=89;PH=0.002;CP=0.000;CG=-2.630;AA=C;DG;DV=by-cluster,by-2hit-2allele;DSP=131;GESP=C:12316;PAC=NP_001091986.1,NP_689886.2,XP_005269360.1,XP_005269361.1,XP_005269362.1,XP_005269363.1,XP_005269364.1,XP_005269365.1,XP_005269366.1 GT:GQ:PL 0/0:2:0,6,0/1:3:16,0,0 1/1:4:20,3,0 1/1:4:14,3,0</p>

<p>4 1018705 rs4647930 C A 10.40 lowCVG AC1=3;AF1=0.3941;DB;DP=6;DP4=0,3,0,3;FQ=12.2;MQ=60;PV4=1,0.0081,1,1;SNPEFF_AMINO_ACID_CHANGE=P362Q;SNPEFF_CODON_CHANGE=cCa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFRL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004356.2;VDB=0.0000;DN=111;DA=C/A;GM=NM_001004356.2,NM_001004358.1,NM_021923.3,XM_005272281.1,XM_005272282.1;GL=FGFRL1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/GLN;PP=362/505;CDP=1085;GS=76;PH=0.105;CP=0.879;CG=3.860;HA=0.9;HE=27.0;HC=6.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;CPG=1017525-1018938;TFBS=V$CMYB_01,V$RREB1_01;GESP=A:2641/C:10337;PAC=NP_001004356.1,NP_001004358.1,NP_068742.2,XP_005272338.1,XP_005272339.1 GT:GQ:PL 0/0:8:0,6,45 0/0:3:0,0,0 1/1:4:49,9,0 0/0:5:0,3,22</p>

<p>4 2451694 rs3108494 T C 127 midCVG AC1=6;AF1=0.6856;DB;DP=10;DP4=1,1,6,2;FQ=6.6;MQ=60;PV4=1,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=C253R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0035;DN=103;DA=T/C;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=CYS/ARG;PP=253/647;CDP=757;GS=180;CP=0.976;CG=3.340;AA=C;DG;DV=by-frequency,by-cluster,by-1000G,by-other-pop;DSP=34;PAC=NP_001180211.1 GT:GQ:PL 0/1:20:22,0,20 0/1:5:0,3,26 1/1:7:51,6,0 1/1:15:99,15,0</p>

<p>4 2460571 rs73205493 C T 76.30 highCVG AC1=4;AF1=0.4854;DB;DP=17;DP4=11,0,4,2;FQ=78.6;MQ=60;PV4=0.11,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=A428V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0077;DN=130;DA=C/T;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=ALA/VAL;PP=428/647;CDP=1283;GS=64;CP=0.021;CG=0.034;AA=C;DV=by-cluster;DSP=21;PAC=NP_001180211.1 GT:GQ:PL 0/1:3:0,0,0 0/0:28:0,30,116 1/1:9:84,12,0 0/1:16:36,0,12</p>

<p>4 3015553 rs1140085 G A 18.90 midCVG AC1=3;AF1=0.3786;DB;DP=12;DP4=2,7,3,0;FQ=21;MQ=60;PV4=0.045,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=V247I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182982.2;VDB=0.0058;DN=86;DA=G/A;GM=NM_001004056.1,NM_001004057.1,NM_182982.2,XM_005247957.1,XM_005247958.1,XM_005247959.1,XM_005247960.1,XM_005247961.1,XM_005247962.1,XM_005247963.1;GL=GRK4;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=215/547,247/533,247/579,247/533,215/501,247/487,247/441,53/385,36/368,247/332;CDP=643,739,739,739,643,739,739,157,106,739;GS=29;PH=0.918;CP=0.998;CG=4.760;AA=G;HA=0.0;HE=15.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=3;KP=hsa04062:ChemokineSignalingPathway|hsa04144:Endocytosis;GESP=A:1261/G:11745;PAC=NP_001004056.1,NP_001004057.1,NP_892027.2,XP_005248014.1,XP_005248015.1,XP_005248016.1,XP_005248017.1,XP_005248018.1,XP_005248019.1,XP_005248020.1 GT:GQ:PL 0/0:12:0,12,76 0/1:4:24,3,0 1/1:3:41,6,0 0/0:15:0,15,101</p>

<p>4 8603105 rs115392829 G A 26.80 lowCVG AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA GT:GQ:PL 0/0:7:0,6,47 0/0:4:0,3,18 1/1:7:67,12,0 0/1:3:0,0,0</p>

<p>4 15964485 rs758329 G A 64.30 midCVG AC1=5;AF1=0.6667;DB;DP=10;DP4=0,1,0,8;FQ=-6.93;MQ=60;PV4=1,0.17,1,0.3;SNPEFF_AMINO_ACID_CHANGE=P90S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031950.3;VDB=0.0006;DN=86;DA=G/A;GM=NM_031950.3;GL=FGFBP2;FG=missense;FD=missense;AAC=PRO/SER;PP=90/224;CDP=268;GS=74;PH=0.126;CP=0.160;CG=1.080;AA=G;HA=21.1;HE=46.8;HC=41.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=425;GESP=A:5041/G:7965;PAC=NP_114156.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:22:102,24,0 0/1:4:0,3,27</p>

<p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

<p>4 38774889 rs4129009 T C 35.10 midCVG AC1=3;AF1=0.3337;DB;DP=9;DP4=1,3,3,1;FQ=36.8;MQ=60;PV4=0.49,0.0012,1,0.18;SNPEFF_AMINO_ACID_CHANGE=I775V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001017388.2;VDB=0.0028;DN=108;DA=T/A/C/G;GM=NM_001017388.2,NM_001195106.1,NM_001195107.1,NM_001195108.1,NM_030956.3,XM_005262671.1;GL=TLR10;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=ILE/VAL;PP=775/812,775/812,775/812,761/798,775/812,761/798;CDP=2323,2323,2323,2281,2323,2281;GS=29;PH=0;CP=0.010;CG=-5.340;AA=T;HE=20.1;HC=43.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2288;GESP=C:1621/T:11385;PAC=NP_001017388.1,NP_001182035.1,NP_001182036.1,NP_001182037.1,NP_112218.2,XP_005262728.1 GT:GQ:PL 0/0:8:0,6,48 0/0:5:0,3,26 1/1:6:77,12,0 0/0:5:0,3,26</p>

<p>4 70807771 rs10030475 C T 36 midCVG AC1=4;AF1=0.4967;DB;DP=11;DP4=2,3,4,0;FQ=38.9;MQ=60;PV4=0.17,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A117V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CSN1S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001890.1;VDB=0.0071;DN=119;DA=C/T;GM=NM_001025104.1,NM_001890.1;GL=CSN1S1;FG=missense;FD=missense;AAC=ALA/VAL;PP=108/177,117/186;CDP=323,350;GS=64;PH=0;CP=0.000;CG=-4.500;AA=T;HA=32.5;HE=45.1;HC=25.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=8;GESP=T:4267/C:7325;PAC=NP_001020275.1,NP_001881.1 GT:GQ:PL 0/0:8:0,9,59 0/1:3:0,0,0 1/1:6:63,9,0 0/1:17:15,0,36</p>

<p>4 71068489 rs3196714 T C 34.90 lowCVG AC1=5;AF1=0.5673;DB;DP=7;DP4=0,2,0,4;FQ=7.37;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=I222T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODAM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017855.3;VDB=0.0087;DN=105;DA=T/C;GM=NM_017855.3,XM_005265698.1;GL=ODAM;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=222/280,239/297;CDP=665,716;GS=89;PH=0.007;CP=0.000;CG=0.676;AA=T;HA=15.6;HE=26.6;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=C:3310/T:9690;PAC=NP_060325.3,XP_005265755.1 GT:GQ:PL 0/1:15:13,0,17 0/1:3:0,0,0 1/1:7:62,9,0 0/1:3:0,3,26</p>

<p>4 71232388 rs10031844 G A 39.10 lowCVG AC1=5;AF1=0.5822;DB;DP=6;DP4=0,2,2,2;FQ=11.2;MQ=58;PV4=0.47,0.22,1,0.3;SNPEFF_AMINO_ACID_CHANGE=G28R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMR3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012390.3;VDB=0.0103;DN=119;DA=G/A;GM=NM_012390.3;GL=SMR3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=28/135;CDP=82;GS=125;PH=0.001;CP=0.000;CG=-5.140;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=28;GESP=A:3028/G:9978;PAC=NP_036522.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:77,12,0 0/0:5:0,6,45</p>

<p>4 76878716 rs2242471 G C 19.60 lowCVG AC1=6;AF1=0.6681;DB;DP=8;DP4=1,1,4,0;FQ=-12.1;MQ=54;PV4=0.33,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=S575C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018115.3;VDB=0.0091;DN=98;DA=G/C;GM=NM_018115.2,XM_005263102.1,XM_005263103.1,XM_005263104.1,XM_005263105.1;GL=SDAD1;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=SER/CYS;PP=575/688,550/663,538/651,478/591,478/591;CDP=1724,1649,1613,1433,1433;GS=112;PH=0.941;CP=1.000;CG=5.150;AA=G;HA=27.9;HE=41.7;HC=6.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=46;TFBS=V$CREL_01,V$IK1_01,V$NFKAPPAB_01,V$NFKB_C,V$STAT1_03,V$STAT5A_02;GESP=C:4469/G:8537;PAC=NP_060585.2,XP_005263159.1,XP_005263160.1,XP_005263161.1,XP_005263162.1 GT:GQ:PL 0/1:20:40,0,18 0/1:3:0,0,0 1/1:3:14,3,0 0/1:3:0,0,0</p>

<p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

<p>4 77675505 rs3733242 C T 76.80 highCVG AC1=4;AF1=0.5211;DB;DP=17;DP4=3,5,1,6;FQ=76.9;MQ=60;PV4=0.57,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1290L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=0.0062;DN=107;DA=C/T;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=1290/1997,1290/1997,1114/1821,1114/1821,1114/1821;CDP=3869,3869,3341,3341,3341;GS=98;PH=0;CP=0.000;CG=-1.610;AA=T;HA=34.1;HE=40.1;HC=47.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=42;GESP=T:6227/C:6779;PAC=NP_065910.3,XP_005263219.1,XP_005263220.1,XP_005263221.1,XP_005263222.1 GT:GQ:PL 0/1:42:39,0,69 0/1:19:19,0,18 1/1:7:64,9,0 0/0:5:0,6,35</p>

<p>4 88415636 rs1049544 G C 88.60 midCVG AC1=4;AF1=0.5004;DB;DP=12;DP4=2,3,5,2;FQ=51.6;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H106D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPARCL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128310.2;VDB=0.0073;DN=86;DA=G/C;GM=NM_001128310.1,NM_004684.4,XM_005263272.1,XM_005263273.1;GL=SPARCL1;FG=missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown;AAC=HIS/ASP,HIS/ASP,HIS/ASP,none;PP=106/665,106/665,106/665,NA;CDP=316,316,316,NA;GS=81,81,81,NA;PH=0;CP=0.015;CG=4.250;AA=C;HA=8.4;HE=33.5;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=115;GESP=C:6754/G:6252;PAC=NP_001121782.1,NP_004675.3,XP_005263329.1,NA GT:GQ:PL 0/0:7:0,9,68 0/1:15:50,0,12 1/1:7:74,9,0 0/1:12:10,0,14</p>

<p>4 95170839 rs11722476 G A 30.20 highCVG AC1=3;AF1=0.351;DB;DP=19;DP4=5,7,0,5;FQ=32;MQ=60;PV4=0.24,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=S247N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0126;DN=120;DA=G/A;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none;PP=247/1029,247/1029,247/1027,NA;CDP=740,740,740,1066;GS=46,46,46,NA;PH=0;CP=0.987;CG=0.199;AA=G;HA=20.5;HE=45.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:4424/G:8574;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/0:10:0,9,68 0/1:8:7,0,75 1/1:7:69,12,0 0/0:15:0,15,90</p>

<p>4 95173779 rs7439869 T C 130 midCVG AC1=3;AF1=0.4198;DB;DP=14;DP4=2,3,3,6;FQ=65;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=V301A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0162;DN=116;DA=T/C;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=VAL/ALA,VAL/ALA,VAL/ALA,none;PP=301/1029,301/1029,301/1027,NA;CDP=902,902,902,1228;GS=64,64,64,NA;PH=0;CP=0.012;CG=4.400;AA=C;HA=5.0;HE=41.6;HC=28.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=C:9055/T:3845;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/1:11:100,0,6 0/0:6:0,6,45 1/1:6:74,9,0 0/0:6:0,6,56</p>

<p>4 100504664 rs3816873 T C 35.40 lowCVG AC1=6;AF1=0.695;DB;DP=7;DP4=1,1,3,2;FQ=10.9;MQ=60;PV4=1,0.018,1,0.15;SNPEFF_AMINO_ACID_CHANGE=I155T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTTP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300785.1;VDB=0.0031;DN=107;DA=T/C;GM=NM_000253.2,XM_005263025.1;GL=MTTP;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=128/895,155/922;CDP=383,464;GS=89;PH=0.007;CP=0.810;CG=4.100;AA=T;HA=22.5;HE=25.3;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=11;GESP=C:3415/T:9587;PPI=APOB:846;PAC=NP_000244.2,XP_005263082.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:11:61,12,0 1/1:4:16,3,0</p>

<p>4 103611845 rs227368 C T 89.90 lowCVG AC1=6;AF1=0.7365;DB;DP=8;DP4=1,1,4,2;FQ=12.9;MQ=60;PV4=1,1,1,0.29;SNPEFF_AMINO_ACID_CHANGE=V253I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MANBA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005908.3;VDB=0.0092;DN=79;DA=C/A/G/T;GM=NM_005908.3,XM_005262994.1;GL=MANBA;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=253/880,196/823;CDP=757,586;GS=29;PH=0.037;CP=0.007;CG=1.290;AA=C;HA=10.1;HE=46.2;HC=46.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=84;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=T:5010/C:7996;PAC=NP_005899.3,XP_005263051.1 GT:GQ:PL 0/0:3:0,6,50 0/1:3:0,0,0 1/1:9:70,9,0 1/1:9:63,9,0</p>

<p>4 106859549 rs35132891 G C 52.10 midCVG AC1=5;AF1=0.5498;DB;DP=10;DP4=2,2,3,3;FQ=36.5;MQ=60;PV4=1,0.16,1,1;SNPEFF_AMINO_ACID_CHANGE=Q189H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NPNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184691.1;VDB=0.0030;DN=126;DA=G/C;GM=NM_001033047.2,NM_001184690.1,NM_001184691.1,NM_001184692.1,NM_001184693.1,XM_005262888.1,XM_005262889.1,XM_005262890.1;GL=NPNT;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown;AAC=GLN/HIS;PP=159/566,176/583,189/596,159/537,189/567,206/613,206/584,55/462;CDP=477,528,567,477,567,618,618,165;GS=24;PH=0.997;CP=0.992;CG=3.490;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=C:4607/G:8399;PAC=NP_001028219.1,NP_001171619.1,NP_001171620.1,NP_001171621.1,NP_001171622.1,XP_005262945.1,XP_005262946.1,XP_005262947.1 GT:GQ:PL 0/1:14:15,0,13 0/1:25:22,0,41 1/1:7:59,9,0 0/1:3:0,3,23</p>

<p>4 110773067 rs4698797 G A 30.80 lowCVG AC1=5;AF1=0.6065;DB;DP=7;DP4=2,1,0,3;FQ=33.6;MQ=60;PV4=0.4,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=S175N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRIT3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198506.4;VDB=0.0062;DN=111;DA=G/A;GM=NM_198506.4,XM_005262979.1;GL=LRIT3;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=SER/ASN,none;PP=175/680,NA;CDP=524,NA;GS=46,NA;PH=0.964;CP=1.000;CG=3.780;AA=G;HA=43.9;HE=35.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;TFBS=V$STAT5B_01;GESP=A:2688/G:1878;PAC=NP_940908.3,NA GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,76 1/1:3:27,3,0 1/1:5:45,6,0</p>

<p>4 128814964 rs17012739 G T 226 highCVG AC1=5;AF1=0.6047;DB;DP=33;DP4=7,4,9,12;FQ=139;MQ=60;PV4=0.46,0.29,1,0.095;SNPEFF_AMINO_ACID_CHANGE=E830D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014264.4;VDB=0.0063;DN=123;DA=G/T;GM=NM_001190799.1,NM_001190801.1,NM_014264.4,XM_005262701.1;GL=PLK4;FG=missense;FD=missense,missense,missense,unknown;AAC=GLU/ASP;PP=798/939,789/930,830/971,796/937;CDP=2394,2367,2490,2388;GS=45;PH=0;CP=0.000;CG=-8.440;AA=T;HA=22.5;HE=34.3;HC=43.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=73;GESP=T:9270/G:3736;PAC=NP_001177728.1,NP_001177730.1,NP_055079.3,XP_005262758.1 GT:GQ:PL 0/1:34:32,0,37 0/0:19:0,24,148 1/1:8:127,8,0 1/1:26:117,27,0</p>

<p>4 128996129 . G T 28.60 midCVG AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1 GT:GQ:PL 0/0:8:0,6,51 0/0:6:0,3,17 1/1:6:71,12,0 0/0:8:0,6,45</p>

<p>4 129793281 . A G 98.60 highCVG AC1=4;AF1=0.5296;DP=39;DP4=1,6,3,23;FQ=30.8;MQ=60;PV4=1,0.39,1,0.19;SNPEFF_AMINO_ACID_CHANGE=N798S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=JADE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287439.1;VDB=0.0093;GM=NM_199320.2,XM_005263229.1,XM_005263230.1,XM_005263231.1,XM_005263232.1,XM_005263233.1;GL=PHF17;FG=missense;FD=unknown;AAC=ASN/SER;PP=798/843,825/870,798/843,798/843,798/843,786/831;CDP=2393,2474,2393,2393,2393,2357;GS=46;PH=0;CP=0.014;CG=-6.450;AA=A;DSP=772;GESP=A:13006;PPI=KAT7:965|ING5:913|VHL:845|MEAF6:845|ING4:788;PAC=NP_955352.1,XP_005263286.1,XP_005263287.1,XP_005263288.1,XP_005263289.1,XP_005263290.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:23,3,0 1/1:7:120,9,0 0/0:5:0,6,46</p>

<p>4 146653620 rs10008599 G A 23.70 midCVG AC1=3;AF1=0.3727;DB;DP=11;DP4=3,4,1,3;FQ=25.7;MQ=60;PV4=1,0.083,1,1;SNPEFF_AMINO_ACID_CHANGE=D173N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C4orf51;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080531.1;VDB=0.0141;DN=119;DA=G/A;GM=NM_001080531.1;GL=C4orf51;FG=missense;FD=missense;AAC=ASP/ASN;PP=173/203;CDP=517;GS=23;PH=0.059;CP=0.000;CG=-0.356;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=16;GESP=A:1483/G:10759;PAC=NP_001074000.1 GT:GQ:PL 0/0:9:0,9,71 0/1:12:15,0,8 1/1:5:54,9,0 0/0:9:0,9,74</p>

<p>4 147741397 rs149464457 T A 23.60 midCVG AC1=2;AF1=0.3051;DB;DP=14;DP4=8,2,0,4;FQ=25.1;MQ=60;PV4=0.015,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=Q353H;SNPEFF_CODON_CHANGE=caA/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300761.1;VDB=0.0056;DN=134;DA=T/A;GM=NM_031956.2,XM_005263269.1,XM_005263270.1;GL=TTC29;FG=missense;FD=missense,unknown,unknown;AAC=GLN/HIS;PP=327/476,353/502,353/493;CDP=981,1059,1059;GS=24;PH=0.999;CP=1.000;CG=5.910;AA=T;DV=by-cluster;DSP=4;GESP=A:4/T:11824;PAC=NP_114162.2,XP_005263326.1,XP_005263327.1 GT:GQ:PL 0/0:16:0,15,85 0/0:3:0,0,0 1/1:6:66,12,0 0/0:16:0,15,87</p>

<p>4 155287535 rs10017772 T C 73.60 midCVG AC1=5;AF1=0.5889;DB;DP=10;DP4=2,1,5,2;FQ=5.24;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=H174R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCHS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017639.3;VDB=0.0092;DN=119;DA=T/C;GM=NM_001142552.1,NM_017639.3;GL=DCHS2;FG=missense;FD=missense;AAC=HIS/ARG;PP=768/1370,174/2917;CDP=2303,521;GS=29;PH=0.007;CP=0.001;CG=2.100;AA=T;HA=36.4;HE=48.0;HC=45.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;TFBS=V$CART1_01;GESP=C:5817/T:7189;PAC=NP_001136024.1,NP_060109.2 GT:GQ:PL 0/1:7:54,0,3 0/1:3:0,0,0 1/1:7:60,9,0 0/0:4:0,6,37</p>

<p>4 169611811 . C T 14.60 midCVG AC1=2;AF1=0.2475;DP=11;DP4=6,2,0,3;FQ=15.7;MQ=60;PV4=0.061,1,1,0.052;SNPEFF_AMINO_ACID_CHANGE=R465C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PALLD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001166108.1;VDB=0.0128;GM=NM_001166108.1,NM_001166109.1,NM_016081.3,XM_005262861.1,XM_005262862.1,XM_005262863.1,XM_005262864.1,XM_005262865.1,XM_005262866.1;GL=PALLD;FG=missense;FD=unknown;AAC=ARG/CYS;PP=465/1124,83/778,465/1107,465/1401,465/1384,465/1348,465/1310,465/1277,83/1019;CDP=1393,247,1393,1393,1393,1393,1393,1393,247;GS=180;PH=1;CP=0.789;CG=3.960;AA=C;DSP=58;TFBS=V$TCF11MAFG_01;GESP=C:13006;PPI=EZR:846|ACTN1:846;PAC=NP_001159580.1,NP_001159581.1,NP_057165.3,XP_005262918.1,XP_005262919.1,XP_005262920.1,XP_005262921.1,XP_005262922.1,XP_005262923.1 GT:GQ:PL 0/0:7:0,3,16 0/0:9:0,6,36 1/1:3:56,9,0 0/0:18:0,15,104</p>

<p>4 169928842 rs2877380 G T 108 midCVG AC1=5;AF1=0.6024;DB;DP=14;DP4=2,2,6,2;FQ=49.6;MQ=60;PV4=0.55,0.03,1,1;SNPEFF_AMINO_ACID_CHANGE=L70M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CBR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032783.4;VDB=0.0052;DN=101;DA=G/T;GM=NM_032783.4,XM_005263315.1,XM_005263316.1,XM_005263317.1;GL=CBR4;FG=missense;FD=missense,unknown,unknown,unknown;AAC=LEU/MET;PP=70/238,70/209,70/194,70/146;CDP=208;GS=15;PH=0.005;CP=1.000;CG=5.560;AA=T;HA=38.2;HE=21.8;HC=3.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=56;GESP=T:8677/G:4325;PAC=NP_116172.2,XP_005263372.1,XP_005263373.1,XP_005263374.1 GT:GQ:PL 0/1:18:51,0,16 0/0:6:0,9,76 1/1:11:74,12,0 1/1:4:29,3,0</p>

<p>4 184190233 rs11941467 G T 54.10 highCVG AC1=3;AF1=0.395;DB;DP=17;DP4=5,8,1,3;FQ=56.3;MQ=60;PV4=1,1,1,0.073;SNPEFF_AMINO_ACID_CHANGE=A773S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WWC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024949.5;VDB=0.0099;DN=120;DA=G/T;GM=NM_024949.5;GL=WWC2;FG=missense;FD=missense;AAC=ALA/SER;PP=773/1193;CDP=2317;GS=99;PH=0.099;CP=1.000;CG=3.500;AA=G;HA=1.4;HE=10.4;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;GESP=T:1111/G:11895;PAC=NP_079225.5 GT:GQ:PL 0/0:14:0,15,112 0/1:4:31,3,0 1/1:5:71,9,0 0/0:23:0,24,151</p>

<p>4 184618936 rs62617790 G C 120 highCVG AC1=4;AF1=0.5361;DB;DP=15;DP4=1,3,4,7;FQ=46.7;MQ=60;PV4=1,0.0017,1,0.28;SNPEFF_AMINO_ACID_CHANGE=Q933H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021942.5;VDB=0.0163;DN=129;DA=G/C;GM=NM_021942.5,NM_199053.2;GL=TRAPPC11;FG=missense;FD=missense;AAC=GLN/HIS;PP=933/1134,933/1087;CDP=2799;GS=24;PH=0.999;CP=0.998;CG=4.670;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=53;GESP=C:1255/G:11751;PAC=NP_068761.4,NP_951008.1 GT:GQ:PL 0/0:7:0,9,65 0/1:3:0,3,25 1/1:27:149,30,0 1/1:3:18,3,0</p>

<p>4 187627792 rs11939575 T C 10.40 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=0,2,3,0;FQ=9.79;MQ=60;PV4=0.1,0.00034,1,1;SNPEFF_AMINO_ACID_CHANGE=R1064G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0075;DN=120;DA=T/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLY;PP=1064/4589,1064/4601,1064/4601;CDP=3190;GS=125;PH=0;CP=0.635;CG=4.600;AA=C;HA=9.0;HE=23.5;HC=49.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=76;GESP=C:10198/T:2130;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1 GT:GQ:PL 0/0:6:0,6,45 0/1:3:0,0,0 1/1:5:47,9,0 0/1:3:0,0,0</p>

<p>4 187629140 rs367863 G C 173 highCVG AC1=5;AF1=0.6658;DB;DP=17;DP4=2,2,6,6;FQ=22.5;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F614L;SNPEFF_CODON_CHANGE=ttC/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0127;DN=80;DA=G/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=PHE/LEU;PP=614/4589,614/4601,614/4601;CDP=1842;GS=22;PH=0;CP=0.966;CG=3.530;AA=C;HA=24.9;HE=20.1;HC=48.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1424;GESP=C:9311/G:2625;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1 GT:GQ:PL 0/1:10:64,0,9 0/0:5:0,9,57 1/1:12:76,12,0 1/1:12:81,12,0</p>

<p>5 5462406 . C T 26.90 lowCVG AC1=3;AF1=0.3937;DP=6;DP4=3,0,0,3;FQ=24.3;MQ=60;PV4=0.1,1,1,4e-06;SNPEFF_AMINO_ACID_CHANGE=P987S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ICE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015325.2;VDB=0.0000;GM=NM_015325.2;GL=KIAA0947;FG=missense;FD=unknown;AAC=PRO/SER;PP=987/2267;CDP=2959;GS=74;PH=0.001;CP=0.000;CG=-5.280;DSP=1858;GESP=C:12622;PAC=NP_056140.1 GT:GQ:PL 0/0:7:0,6,38 0/0:3:0,0,0 1/1:5:66,9,0 0/0:5:0,3,25</p>

<p>5 7897191 rs10380 C T 46.90 midCVG AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1 GT:GQ:PL 0/0:11:0,12,87 0/0:8:0,9,73 1/1:6:65,9,0 0/1:4:29,3,0</p>

<p>5 32786389 rs2270915 A G 30.80 midCVG AC1=4;AF1=0.5475;DB;DP=9;DP4=1,4,2,2;FQ=33;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N522D;SNPEFF_CODON_CHANGE=Aac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NPR3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204375.1;VDB=0.0096;DN=100;DA=A/C/G/T;GM=NM_000908.3,NM_001204375.1,NM_001204376.1,XM_005248309.1;GL=NPR3;FG=missense;FD=missense,missense,missense,unknown;AAC=ASN/ASP;PP=521/541,522/542,305/325,306/326;CDP=1561,1564,913,916;GS=23;PH=0.001;CP=0.741;CG=5.000;AA=A;HA=8.1;HE=15.1;HC=23.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=50;GESP=G:2077/A:9753;PPI=NPPC:846|NPPB:846|NPPA:846;PAC=NP_000899.1,NP_001191304.1,NP_001191305.1,XP_005248366.1 GT:GQ:PL 0/0:5:0,6,41 0/1:3:21,3,0 1/1:5:51,6,0 0/1:5:1,0,37</p>

<p>5 33951693 rs16891982 C G 118 midCVG AC1=6;AF1=0.667;DB;DP=9;DP4=0,1,4,4;FQ=-7.74;MQ=60;PV4=1,0.4,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L374F;SNPEFF_CODON_CHANGE=ttG/ttC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC45A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016180.3;VDB=0.0064;DN=123;DA=C/G;GM=NM_001012509.2,NM_016180.3,XM_005248311.1;GL=SLC45A2;FG=missense,missense,3-prime-UTR;FD=missense,missense,unknown;AAC=LEU/PHE,LEU/PHE,none;PP=374/461,374/531,NA;CDP=1122,1122,NA;GS=22,22,NA;PH=0.995;CP=1.000;CG=4.320;AA=C;HA=0.0;HE=1.7;HC=0.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=35;TFBS=V$MRF2_01;GESP=G:9063/C:3943;PAC=NP_001012527.1,NP_057264.3,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,1/1:22:156,24,0 0/1:4:0,3,26</p>

<p>5 35709095 rs13170082 C T 25.90 lowCVG AC1=6;AF1=0.7423;DB;DP=7;DP4=0,1,1,3;FQ=-9.92;MQ=60;PV4=1,0.026,1,0.32;SNPEFF_AMINO_ACID_CHANGE=A904V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024867.3;VDB=0.0080;DN=121;DA=C/T;GM=NM_024867.3,XM_005248376.1,XM_005248377.1,XM_005248378.1;GL=SPEF2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/VAL;PP=904/1823,904/1824,899/1819,904/973;CDP=2711,2711,2696,2711;GS=64;PH=0.001;CP=0.001;CG=-2.830;AA=C;HA=33.6;HE=14.7;HC=17.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=46;GESP=T:9050/C:2912;PAC=NP_079143.3,XP_005248433.1,XP_005248434.1,XP_005248435.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:6:36,6,0 1/1:6:30,6,0</p>

<p>5 36995903 . G A 18 lowCVG AC1=3;AF1=0.3939;DP=7;DP4=2,1,3,0;FQ=18.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E1101K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPBL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133433.3;VDB=0.0000;GM=NM_015384.4,NM_133433.3,XM_005248280.1,XM_005248281.1,XM_005248282.1,XM_005248283.1;GL=NIPBL;FG=missense;FD=unknown;AAC=GLU/LYS;PP=1101/2698,1101/2805,1101/2699,981/2685,881/2585,562/2266;CDP=3301,3301,3301,2941,2641,1684;GS=56;PH=0.61;CP=1.000;CG=5.430;AA=G;DSP=4;GESP=G:13006;PPI=CBX5:765;PAC=NP_056199.2,NP_597677.2,XP_005248337.1,XP_005248338.1,XP_005248339.1,XP_005248340.1 GT:GQ:PL 0/0:7:0,6,39 0/0:5:0,3,23 1/1:5:57,9,0 0/0:3:0,0,0</p>

<p>5 42719239 rs6180 A C 98.30 midCVG AC1=6;AF1=0.705;DB;DP=10;DP4=0,3,0,7;FQ=25.8;MQ=60;PV4=1,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=I551L;SNPEFF_CODON_CHANGE=Atc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GHR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242399.2;VDB=0.0032;DN=52;DA=A/C;GM=NM_000163.4,NM_001242399.2,NM_001242400.2,NM_001242401.3,NM_001242402.2,NM_001242403.2,NM_001242404.2,NM_001242405.2,NM_001242406.2,NM_001242460.1,NM_001242462.1,XM_005248288.1;GL=GHR;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,3-prime-UTR,missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,utr-variant-3-prime,unknown;AAC=ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,none,ILE/LEU;PP=544/639,551/646,544/639,544/639,544/639,544/639,544/639,544/639,544/639,522/617,NA,522/617;CDP=1630,1651,1630,1630,1630,1630,1630,1630,1630,1564,NA,1564;GS=5,5,5,5,5,5,5,5,5,5,NA,5;PH=0.933;CP=0.951;CG=3.430;AA=A;HA=41.6;HE=46.2;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=685;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04630:Jak-STATSignalingPathway;TFBS=V$PAX4_04;GESP=C:5675/A:7331;PPI=CSHL1:998|SOCS2:937|JAK2:937|PTPN11:846|UBC:846|BTRC:846|NCOA6:845|SHC1:845|CISH:845;PAC=NP_000154.1,NP_001229328.1,NP_001229329.1,NP_001229330.1,NP_001229331.1,NP_001229332.1,NP_001229333.1,NP_001229334.1,NP_001229335.1,NP_001229389.1,NA,XP_005248345.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:12:83,12,0 1/1:9:60,9,0</p>

<p>5 54439466 rs1021580 G A 260 highCVG AC1=6;AF1=0.805;DB;DP=19;DP4=1,2,4,12;FQ=-17.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R121C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170402.1;VDB=0.0082;DN=86;DA=G/A;GM=NM_001145734.2,NM_001170402.1,NM_152623.2,XM_005248450.1;GL=CDC20B;FG=missense;FD=missense,missense,missense,unknown;AAC=ARG/CYS;PP=121/478,121/520,121/516,121/373;CDP=361;GS=180;PH=0;CP=0.001;CG=-1.690;AA=A;HA=49.1;HE=18.6;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;GESP=A:9758/G:3248;PAC=NP_001139206.2,NP_001163873.1,NP_689836.2,XP_005248507.1 GT:GQ:PL 0/1:4:87,0,4 0/1:13:40,0,15 1/1:12:63,9,0 1/1:18:113,15,0</p>

<p>5 56233515 . G A 20 lowCVG AC1=4;AF1=0.5539;DP=5;DP4=1,1,3,0;FQ=14.9;MQ=60;PV4=0.4,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=A114V;SNPEFF_CODON_CHANGE=gCa/gTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MIER3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001297598.1;VDB=0.0000;GM=NM_152622.3,XM_005248448.1,XM_005248449.1;GL=MIER3;FG=missense;FD=unknown;AAC=ALA/VAL;PP=109/550,109/551,46/488;CDP=326,326,137;GS=64;PH=1;CP=1.000;CG=6.070;AA=G;DSP=11;GESP=G:13006;PAC=NP_689835.3,XP_005248505.1,XP_005248506.1GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:57,9,0 0/0:6:0,6,49</p>

<p>5 75003678 rs2307111 T C 211 highCVG AC1=6;AF1=0.7083;DB;DP=19;DP4=2,3,3,11;FQ=53.7;MQ=60;PV4=0.57,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=H36R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099271.1;VDB=0.0147;DN=100;DA=T/C;GM=NM_001099271.1,NM_152408.2,XM_005248435.1,XM_005248436.1,XM_005248437.1;GL=POC5;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ARG;PP=36/576,11/551,9/549,36/548,36/399;CDP=107,32,26,107,107;GS=29;PH=0;CP=0.937;CG=5.850;AA=C;HA=6.7;HE=38.0;HC=46.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=23;GESP=C:6453/T:5401;PAC=NP_001092741.1,NP_689621.2,XP_005248492.1,XP_005248493.1,XP_005248494.1 GT:GQ:PL 0/1:53:47,0,73 0/1:5:0,3,22 1/1:22:132,21,0 1/1:13:79,12,0</p>

<p>5 78181477 rs1065757 C T 31.80 midCVG AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1 GT:GQ:PL 0/0:18:0,21,102 0/1:3:0,0,0 1/1:7:56,9,0 1/1:3:19,3,0</p>

<p>5 79028472 rs4704585 C T 98.30 midCVG AC1=5;AF1=0.6256;DB;DP=13;DP4=2,2,8,0;FQ=34.6;MQ=60;PV4=0.091,1,0.25,1;SNPEFF_AMINO_ACID_CHANGE=A1295V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CMYA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153610.4;VDB=0.0069;DN=111;DA=C/T;GM=NM_153610.3;GL=CMYA5;FG=missense;FD=missense;AAC=ALA/VAL;PP=1295/4070;CDP=3884;GS=64;PH=0.004;CP=0.079;CG=-0.023;AA=T;HA=20.1;HE=42.9;HC=29.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3735;GESP=T:5852/C:6092;PAC=NP_705838.3 GT:GQ:PL 0/1:20:35,0,19 0/0:6:0,9,59 1/1:11:67,12,0 1/1:6:44,6,0</p>

<p>5 86695274 rs2230641 A G 50.70 midCVG AC1=4;AF1=0.4542;DB;DP=9;DP4=1,4,2,2;FQ=51.2;MQ=60;PV4=0.52,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V270A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCNH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001239.3;VDB=0.0061;DN=98;DA=A/G/T;GM=NM_001199189.1,NM_001239.3,XM_005248627.1,XM_005248628.1,XM_005248629.1;GL=CCNH;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=VAL/ALA;PP=217/271,270/324,270/330,270/326,217/277;CDP=650,809,809,809,650;GS=64;PH=0.965;CP=1.000;CG=5.680;AA=A;HA=5.0;HE=20.3;HC=8.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;KP=hsa03420:NucleotideExcisionRepair|hsa04110:CellCycle;GESP=G:2278/A:10728;PPI=CDK7:999|ERCC3:997|GTF2H1:989|CDK2:989|MNAT1:905|CDK8:846|ERCC2:845;PAC=NP_001186118.1,NP_001230.1,XP_005248684.1,XP_005248685.1,XP_005248686.1 GT:GQ:PL 0/0:11:0,12,76 0/1:3:0,0,0 1/1:4:48,6,0 0/1:22:44,0,18</p>

<p>5 89988504 rs2366926 A G 62.10 midCVG AC1=5;AF1=0.6067;DB;DP=9;DP4=2,0,7,0;FQ=4.33;MQ=60;PV4=1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=N2345S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=32;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0009;DN=100;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2345/6307,NA,51/4013;CDP=7034,7047,152;GS=46,NA,46;PH=1;CP=1.000;CG=5.950;AA=A;HA=17.9;HE=29.9;HC=47.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=83;GESP=G:3478/A:8458;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:5:0,6,37 0/1:3:0,0,0 1/1:18:101,21,0 0/1:3:0,0,0</p>

<p>5 89990324 rs1878878 A G 184 highCVG AC1=6;AF1=0.7366;DB;DP=16;DP4=1,3,8,4;FQ=36.4;MQ=60;PV4=0.26,1,1,0.041;SNPEFF_AMINO_ACID_CHANGE=N2584S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0007;DN=92;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2584/6307,NA,290/4013;CDP=7751,7764,869;GS=46,NA,46;PH=0;CP=0.995;CG=5.880;AA=G;HA=4.0;HE=40.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=195;GESP=G:8914/A:3276;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/1:25:18,0,41 0/1:34:51,0,35 1/1:16:97,15,0 1/1:11:63,9,0</p>

<p>5 90012379 rs13157270 G A 132 highCVG AC1=4;AF1=0.4566;DB;DP=24;DP4=6,6,2,8;FQ=135;MQ=60;PV4=0.2,1,1,0.054;SNPEFF_AMINO_ACID_CHANGE=V3094I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0154;DN=121;DA=G/A;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=VAL/ILE,none,VAL/ILE;PP=3094/6307,NA,800/4013;CDP=9280,9293,2398;GS=29,NA,29;PH=0.01;CP=0.001;CG=-2.950;AA=G;HA=0.5;HE=9.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=96;GESP=A:774/G:11128;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:16:0,18,131 0/1:28:101,0,24 1/1:4:47,6,0 0/1:32:30,0,52</p>

<p>5 95091201 rs34899 A G 48.40 highCVG AC1=3;AF1=0.4188;DB;DP=17;DP4=3,6,3,2;FQ=50.4;MQ=60;PV4=0.58,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N262D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RHOBTB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014899.3;VDB=0.0132;DN=76;DA=A/G;GM=NM_014899.3,XM_005271925.1;GL=RHOBTB3;FG=missense;FD=missense,unknown;AAC=ASN/ASP;PP=262/612,294/644;CDP=784,880;GS=23;PH=0;CP=0.013;CG=5.210;AA=G;HA=30.5;HE=41.1;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=102;GESP=G:5290/A:7716;PAC=NP_055714.3,XP_005271982.1 GT:GQ:PL 0/0:4:0,3,27 0/0:4:0,3,15 1/1:7:88,12,0 0/1:4:1,0,63</p>

<p>5 111504479 rs7703522 C T 58.60 midCVG AC1=4;AF1=0.4913;DB;DP=11;DP4=0,5,2,3;FQ=36.6;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R630H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L4A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022140.3;VDB=0.0074;DN=116;DA=C/T;GM=NM_022140.3,XM_005272043.1,XM_005272044.1;GL=EPB41L4A;FG=missense;FD=unknown;AAC=ARG/HIS;PP=630/687,602/659,257/339;CDP=1889,1805,770;GS=29;PH=1;CP=0.728;CG=3.770;AA=C;HA=4.9;HE=35.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;GESP=T:8586/C:3412;PAC=NP_071423.3,XP_005272100.1,XP_005272101.1 GT:GQ:PL 0/0:10:0,12,66 0/1:3:0,0,0 1/1:6:68,9,0 0/1:14:33,0,10</p>

<p>5 127685135 rs154001 C T 84.40 midCVG AC1=2;AF1=0.3246;DB;DP=13;DP4=3,3,3,4;FQ=86;MQ=60;PV4=1,0.09,1,1;SNPEFF_AMINO_ACID_CHANGE=V965I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001999.3;VDB=0.0006;DN=79;DA=C/T;GM=NM_001999.3;GL=FBN2;FG=missense;FD=missense;AAC=VAL/ILE;PP=965/2913;CDP=2893;GS=29;PH=0.118;CP=0.989;CG=3.320;AA=C;HA=40.8;HE=31.3;HC=11.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=30;TFBS=V$ATF_01,V$CEBP_C,V$MYOGNF1_01;GESP=T:8612/C:4394;PAC=NP_001990.2 GT:GQ:PL 0/0:10:0,9,72 0/0:5:0,3,23 1/1:15:129,21,0 0/0:7:0,6,52</p>

<p>5 134782450 rs12520799 T A 123 midCVG AC1=4;AF1=0.5277;DB;DP=14;DP4=3,2,5,4;FQ=60;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=R117*;SNPEFF_CODON_CHANGE=Aga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5orf20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_130848.2;VDB=0.0030;DN=120;DA=T/A;GM=NM_130848.2;GL=C5orf20;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=117/245;CDP=349;CP=0.000;CG=-5.910;AA=A;HA=5.0;HE=48.3;HC=46.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3190;GESP=A:7784/T:5222;PAC=NP_570900.1 GT:GQ:PL 0/1:39:61,0,36 0/1:3:0,0,0 1/1:12:105,15,0 0/0:7:0,9,64</p>

<p>FBN2 5 140023238 rs17208187 C G 65.40 midCVG AC1=5;AF1=0.6275;DB;DP=12;DP4=3,1,1,6;FQ=18.7;MQ=60;PV4=0.088,1,1,0.029;SNPEFF_AMINO_ACID_CHANGE=T305S;SNPEFF_CODON_CHANGE=aCc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMCO6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300980.1;VDB=0.0001;DN=123;DA=C/G;GM=NM_018502.3,XM_005268476.1,XM_005268477.1;GL=TMCO6;FG=missense;FD=missense,unknown,unknown;AAC=THR/SER;PP=299/494,305/500,300/495;CDP=896,914,899;GS=58;PH=0.001;CP=0.341;CG=2.370;AA=C;HA=7.1;HE=26.0;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=23;GESP=G:2322/C:9960;PAC=NP_060972.3,XP_005268533.1,XP_005268534.1 GT:GQ:PL 0/1:16:11,0,28 0/0:4:0,6,32 1/1:4:16,3,0 1/1:14:84,15,0</p>

<p>5 140558212 rs2950845 A C 62.90 highCVG AC1=4;AF1=0.4824;DB;DP=28;DP4=10,2,10,2;FQ=52.5;MQ=26;PV4=1,1,0.16,1;SNPEFF_AMINO_ACID_CHANGE=K199N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019120.3;VDB=0.0196;DN=101;DA=A/C/T;GM=NM_019120.3;GL=PCDHB8;FG=missense;FD=missense;AAC=LYS/ASN;PP=199/802;CDP=597;GS=94;PH=0.988;CP=0.833;CG=4.250;AA=A;HA=0.8;HE=14.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6207;GESP=C:1504/A:11452;PAC=NP_061993.2 GT:GQ:PL 0/0:18:0,20,62 0/0:5:0,6,37 1/1:3:75,4,0 1/1:17:37,21,0</p>

<p>5 140559596 rs2697541 T G 19.30 lowCVG AC1=3;AF1=0.352;DB;DP=7;DP4=3,1,0,3;FQ=21.1;MQ=56;PV4=0.14,0.29,0.42,0.14;SNPEFF_AMINO_ACID_CHANGE=L661V;SNPEFF_CODON_CHANGE=Ttg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019120.3;VDB=0.0059;DN=100;DA=T/G;GM=NM_019120.3;GL=PCDHB8;FG=missense;FD=missense;AAC=LEU/VAL;PP=661/802;CDP=1981;GS=32;PH=0;CP=1.000;CG=4.220;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4823;CPG=140558932-140559789;TFBS=V$TCF11MAFG_01;GESP=G:10886/T:1780;PAC=NP_061993.2 GT:GQ:PL 0/0:8:0,6,49 0/0:3:0,0,0 1/1:4:59,9,0 0/0:8:0,6,47</p>

<p>5 140572763 rs2907323 C G 71.10 midCVG AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1 GT:GQ:PL 0/0:10:0,12,73 0/0:5:0,6,45 1/1:9:74,12,0 1/1:4:46,6,0</p>

<p>5 145499996 rs10988 C T 67.30 midCVG AC1=5;AF1=0.6441;DB;DP=12;DP4=4,1,4,3;FQ=17.8;MQ=60;PV4=0.58,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=R1088K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020117.9;VDB=0.0086;DN=52;DA=C/T;GM=NM_020117.9,XM_005268458.1,XM_005268459.1,XM_005268460.1;GL=LARS;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=1088/1177,1061/1150,1042/1131,1034/1123;CDP=3263,3182,3125,3101;GS=26;PH=0.073;CP=1.000;CG=4.460;AA=C;HA=1.4;HE=29.1;HC=22.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa00290:Valine,LeucineAndIsoleucineBiosynthesis|hsa00970:Aminoacyl-tRNABiosynthesis;GESP=T:2696/C:10310;PAC=NP_064502.9,XP_005268515.1,XP_005268516.1,XP_005268517.1 GT:GQ:PL 0/1:13:39,0,11 0/1:3:0,0,0 1/1:8:63,9,0 0/1:10:5,0,42</p>

<p>5 149006726 rs7732714 A C 48.70 midCVG AC1=4;AF1=0.5;DB;DP=10;DP4=1,4,2,3;FQ=50.5;MQ=60;PV4=1,0.047,1,0.025;SNPEFF_AMINO_ACID_CHANGE=S518R;SNPEFF_CODON_CHANGE=Agt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001669.2;VDB=0.0090;DN=116;DA=A/C;GM=NM_001001669.2,XM_005268447.1,XM_005268448.1;GL=ARHGEF37;FG=missense;FD=missense,unknown,unknown;AAC=SER/ARG;PP=518/676,518/676,304/462;CDP=1552,1552,910;GS=110;PH=0.906;CP=0.938;CG=3.930;AA=A;HA=32.4;HE=7.0;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=C:1584/A:11172;PAC=NP_001001669.2,XP_005268504.1,XP_005268505.1 GT:GQ:PL 0/0:5:0,6,47 0/1:3:0,0,0 1/1:4:44,6,0 0/1:46:45,0,47</p>

<p>5 150886882 rs1105168 G A 61.20 lowCVG AC1=5;AF1=0.6338;DB;DP=7;DP4=1,1,3,2;FQ=3.28;MQ=60;PV4=1,0.44,1,1;SNPEFF_AMINO_ACID_CHANGE=P4117L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001447.2;VDB=0.0096;DN=86;DA=G/A;GM=NM_001447.2;GL=FAT2;FG=missense;FD=missense;AAC=PRO/LEU;PP=4117/4350;CDP=12350;GS=98;PH=0;CP=0.001;CG=0.264;AA=G;HA=7.2;HE=46.2;HC=16.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=168;GESP=A:8501/G:4505;PAC=NP_001438.1 GT:GQ:PL 0/1:4:0,3,18 0/1:22:43,0,21 1/1:4:22,3,0 1/1:6:39,6,0</p>

<p>5 151771838 rs4958531 T C 25.70 midCVG AC1=3;AF1=0.3673;DB;DP=11;DP4=2,6,1,2;FQ=27.7;MQ=60;PV4=1,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=M388V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NMUR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020167.4;VDB=0.0057;DN=111;DA=T/C;GM=NM_020167.4;GL=NMUR2;FG=missense;FD=missense;AAC=MET/VAL;PP=388/416;CDP=1162;GS=21;PH=0;CP=0.000;CG=-7.170;AA=C;HA=0.0;HE=18.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=225;KP=hsa04080:NeuroactiveLigand-receptorInteraction;TFBS=V$TCF11_01;GESP=C:1791/T:11215;PPI=NMU:974;PAC=NP_064552.3 GT:GQ:PL 0/1:14:18,0,10 0/0:5:0,3,23 1/1:3:51,6,0 0/0:18:0,18,106</p>

<p>5 176518082 . G T 10.60 midCVG AC1=3;AF1=0.3412;DP=13;DP4=3,3,0,3;FQ=12.1;MQ=60;PV4=0.46,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=E194*;SNPEFF_CODON_CHANGE=Gag/Tag;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFR4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_002011.4;VDB=0.0002;GM=NM_002011.3,NM_022963.2,NM_213647.1,XM_005265837.1,XM_005265838.1,XM_005265839.1;GL=FGFR4;FG=stop-gained;FD=unknown;AAC=GLU/stop;PP=194/803,194/763,194/803,226/835,194/803,194/735;CDP=580,580,580,676,580,580;CP=1.000;CG=4.740;AA=G;DSP=24;KP=hsa04010:MAPKSignalingPathway|hsa04144:Endocytosis|hsa04810:RegulationOfActinCytoskeleton;GESP=G:13006;PPI=FGF5:999|FGF3:999|FGF9:999|FGF4:999|FGF1:937|FGF2:846;PAC=NP_002002.3,NP_075252.2,NP_998812.1,XP_005265894.1,XP_005265895.1,XP_005265896.1 GT:GQ:PL 0/0:17:0,15,78 0/0:6:0,3,25 1/1:4:50,9,0 0/0:3:0,0,0</p>

<p>NMUR2 5 178540975 rs1054480 G A 49.80 lowCVG AC1=6;AF1=0.6833;DB;DP=6;DP4=2,0,1,3;FQ=-5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1177S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADAMTS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014244.4;VDB=0.0112;DN=86;DA=G/A;GM=NM_014244.4;GL=ADAMTS2;FG=missense;FD=missense;AAC=PRO/SER;PP=1177/1212;CDP=3529;GS=74;PH=0.651;CP=0.973;CG=5.050;AA=G;HA=2.7;HE=25.0;HC=32.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=351;GESP=A:2884/G:10122;PAC=NP_055059.2 GT:GQ:PL 0/0:4:0,6,32 0/1:3:0,0,0 1/1:4:26,3,0 1/1:8:65,9,0</p>

<p>5 180030322 rs79620092 C T 69.90 lowCVG AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2 GT:GQ:PL 0/1:15:20,0,13 0/1:3:0,0,0 1/1:13:91,15,0 0/1:3:0,3,26</p>

<p>5 180582604 rs2546423 A G 65.20 midCVG AC1=5;AF1=0.6414;DB;DP=10;DP4=3,0,2,4;FQ=12;MQ=60;PV4=0.17,0.39,1,0.2;SNPEFF_AMINO_ACID_CHANGE=H221R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2V2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206880.1;VDB=0.0064;DN=100;DA=A/G;GM=NM_206880.1;GL=OR2V2;FG=missense;FD=missense;AAC=HIS/ARG;PP=221/316;CDP=662;GS=29;PH=0;CP=0.001;CG=0.494;AA=G;HA=19.4;HE=41.3;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26085;KP=hsa04740:OlfactoryTransduction;GESP=G:8019/A:4987;PAC=NP_996763.1 GT:GQ:PL 0/1:11:8,0,13 0/0:4:0,6,40 1/1:6:43,6,0 1/1:8:60,9,0</p>

<p>6 5431340 rs11243011 A G 68.10 highCVG AC1=4;AF1=0.5135;DB;DP=19;DP4=6,3,5,3;FQ=71.2;MQ=60;PV4=1,0.29,1,0.13;SNPEFF_AMINO_ACID_CHANGE=N280S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FARS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006567.3;VDB=0.0157;DN=120;DA=A/G;GM=NM_006567.3,XM_005248811.1,XM_005248812.1,XM_005248813.1;GL=FARS2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ASN/SER;PP=280/452,280/452,280/452,280/337;CDP=839;GS=46;PH=0.001;CP=1.000;CG=3.130;AA=A;HA=6.4;HE=15.1;HC=37.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;KP=hsa00970:Aminoacyl-tRNABiosynthesis;GESP=G:1979/A:11027;PAC=NP_006558.1,XP_005248868.1,XP_005248869.1,XP_005248870.1 GT:GQ:PL 0/1:3:0,0,0 0/0:10:0,12,63 1/1:15:94,18,0 0/1:21:18,0,80</p>

<p>6 7211818 rs1334576 G A 14 lowCVG AC1=4;AF1=0.4604;DB;DP=7;DP4=0,4,2,1;FQ=15.2;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G195R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RREB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001003699.3;VDB=0.0098;DN=88;DA=G/A;GM=NM_001003698.3,NM_001003699.3,NM_001003700.1,NM_001168344.1,XM_005249272.1,XM_005249273.1,XM_005249274.1,XM_005249275.1,XM_005249276.1,XM_005249277.1;GL=RREB1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/ARG;PP=195/1688,195/1743,195/1477,195/1688,195/1743,195/1743,195/1743,195/1743,195/1743,195/1688;CDP=583;GS=125;PH=0.998;CP=0.039;CG=2.930;AA=G;HA=35.5;HE=45.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=A:4926/G:8080;PAC=NP_001003698.1,NP_001003699.1,NP_001003700.1,NP_001161816.1,XP_005249329.1,XP_005249330.1,XP_005249331.1,XP_005249332.1,XP_005249333.1,XP_005249334.1 GT:GQ:PL 0/1:4:21,3,0 0/0:6:0,6,32 1/1:4:37,6,0 0/0:6:0,6,30</p>

<p>6 18171661 rs149734475 A C 67.40 midCVG AC1=4;AF1=0.4868;DB;DP=13;DP4=4,3,2,4;FQ=70.3;MQ=60;PV4=0.59,0.00091,1,0.47;SNPEFF_AMINO_ACID_CHANGE=Q162P;SNPEFF_CODON_CHANGE=cAg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KDM1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153042.3;VDB=0.0143;DN=134;DA=A/C;GM=NM_153042.3,XM_005248925.1,XM_005248926.1,XM_005248927.1,XM_005248928.1,XM_005248929.1;GL=KDM1B;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown;AAC=GLN/PRO;PP=162/591,162/824,162/823,132/794,162/679,162/456;CDP=485,485,485,395,485,485;GS=76;PH=0;CP=0.981;CG=4.240;AA=A;DV=by-frequency,by-cluster;DSP=50;TFBS=V$CEBPB_01;GESP=C:44/A:12962;PPI=BAP1:941;PAC=NP_694587.3,XP_005248982.1,XP_005248983.1,XP_005248984.1,XP_005248985.1,XP_005248986.1 GT:GQ:PL 0/1:21:32,0,17 0/1:3:0,0,0 1/1:9:79,12,0 0/0:16:0,18,127</p>

<p>6 26184102 rs7766641 G A 188 highCVG AC1=5;AF1=0.626;DB;DP=19;DP4=1,2,12,4;FQ=6.53;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G27S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIST1H2BE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003523.2;VDB=0.0118;DN=116;DA=G/A;GM=NM_003523.2;GL=HIST1H2BE;FG=missense;FD=missense;AAC=GLY/SER;PP=27/127;CDP=79;GS=56;CP=0.986;CG=4.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=12900;KP=hsa05322:SystemicLupusErythematosus;CPG=26184037-26184336;GESP=A:2705/G:10301;PAC=NP_003514.2 GT:GQ:PL 0/0:4:0,6,35 0/1:4:0,3,14 1/1:38:168,39,0 1/1:8:68,9,0</p>

<p>6 28228342 rs1679709 A G 23.10 lowCVG AC1=4;AF1=0.4699;DB;DP=7;DP4=3,1,2,1;FQ=25.8;MQ=60;PV4=1,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=E398G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NKAPL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007531.2;VDB=0.0074;DN=89;DA=A/G;GM=NM_001007531.2;GL=NKAPL;FG=missense;FD=missense;AAC=GLU/GLY;PP=398/403;CDP=1193;GS=98;PH=0.002;CP=1.000;CG=4.630;AA=G;HA=27.2;HE=13.4;HC=14.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1382;TFBS=V$IRF1_01,V$IRF2_01,V$TCF11_01;GESP=G:10661/A:2345;PAC=NP_001007532.1 GT:GQ:PL 0/1:19:19,0,19 0/0:8:0,9,64 1/1:4:45,6,0 0/1:3:0,0,0</p>

<p>6 37614058 . G C 8.05 lowCVG AC1=8;AF1=1;DP=5;DP4=0,0,0,3;FQ=-28.7;MQ=60;SNPEFF_AMINO_ACID_CHANGE=P714A;SNPEFF_CODON_CHANGE=Ccc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MDGA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153487.3;VDB=0.0088;GM=NM_153487.3,XM_005249003.1;GL=MDGA1;FG=missense;FD=unknown;AAC=PRO/ALA;PP=714/956,714/948;CDP=2140;GS=27;PH=0.999;CP=0.998;CG=5.680;AA=G;DSP=85;GESP=G:12708;PAC=NP_705691.1,XP_005249060.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:14,3,0 1/1:7:27,6,0</p>

<p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

<p>6 52617731 rs2180314 C G 37.70 midCVG AC1=4;AF1=0.5406;DB;DP=9;DP4=4,0,3,1;FQ=23;MQ=60;PV4=1,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S112T;SNPEFF_CODON_CHANGE=aGt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTA2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000846.4;VDB=0.0058;DN=96;DA=C/G;GM=NM_000846.4;GL=GSTA2;FG=missense;FD=missense;AAC=SER/THR;PP=112/223;CDP=335;GS=58;PH=0;CP=0.000;CG=-4.510;AA=C;HA=22.9;HE=43.0;HC=33.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=G:6507/C:6499;PPI=GSTM2:937;PAC=NP_000837.3 GT:GQ:PL 0/0:3:0,3,27 0/1:3:0,0,1/1:5:48,6,0 0/1:30:29,0,32</p>

<p>6 54173413 rs2297980 A G 54.70 midCVG AC1=5;AF1=0.5683;DB;DP=12;DP4=3,2,4,3;FQ=51.3;MQ=60;PV4=1,0.035,1,1;SNPEFF_AMINO_ACID_CHANGE=Q22R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0132;DN=100;DA=A/G;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=GLN/ARG;PP=22/477;CDP=65;GS=43;PH=0;CP=0.000;CG=-4.040;AA=A;HA=9.2;HE=10.5;HC=0.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=291;GESP=G:1365/A:11641;PAC=NP_055279.3 GT:GQ:PL 0/1:53:50,0,61 0/1:3:0,0,0 1/1:7:45,9,0 0/1:3:0,3,28</p>

<p>6 54219326 rs3195579 G A 24.40 midCVG AC1=4;AF1=0.4386;DB;DP=11;DP4=6,1,2,1;FQ=26.9;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=R381H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0086;DN=105;DA=G/A/C;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=ARG/HIS;PP=381/477;CDP=1142;GS=29;PH=0;CP=0.594;CG=2.850;AA=G;HA=10.2;HE=17.7;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$PAX2_01;GESP=A:2158/G:10842;PAC=NP_055279.3 GT:GQ:PL 0/0:17:0,18,123 0/0:4:0,3,25 1/1:4:43,6,0 0/1:4:26,3,0</p>

<p>6 57393125 rs927192 A G 133 highCVG AC1=5;AF1=0.603;DB;DP=38;DP4=8,8,5,14;FQ=137;MQ=60;PV4=0.18,0.005,1,1;SNPEFF_AMINO_ACID_CHANGE=S259G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRIM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000947.4;VDB=0.0076;DN=86;DA=A/G;GM=none;FG=intergenic;FD=nc-transcript-variant;CP=1.000;CG=5.860;AA=G;HA=0.8;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap;DSP=306133 GT:GQ:PL 0/1:42:38,0,54 0/1:57:52,0,70 1/1:7:67,8,0 0/1:25:20,0,89</p>

<p>6 66005888 rs9342464 C T 35.50 highCVG AC1=4;AF1=0.47;DB;DP=19;DP4=2,11,2,3;FQ=38.2;MQ=60;PV4=0.53,0.3,1,0.074;SNPEFF_AMINO_ACID_CHANGE=G631S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0077;DN=119;DA=C/T;GM=NM_001142800.1;GL=EYS;FG=missense;FD=missense;AAC=GLY/SER;PP=631/3145;CDP=1891;GS=56;PH=0;CP=0.025;CG=1.710;AA=C;HA=23.3;HE=45.0;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=125;GESP=T:2272/C:2294;PAC=NP_001136272.1 GT:GQ:PL 0/0:10:0,12,80 0/1:11:9,0,78 1/1:6:62,9,0 0/1:12:10,0,69</p>

<p>6 66053934 rs61753611 T G 82 midCVG AC1=4;AF1=0.5267;DB;DP=13;DP4=4,2,2,4;FQ=59.3;MQ=60;PV4=0.57,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K532N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0126;DN=129;DA=T/G;GM=NM_001142800.1,NM_001142801.1,NM_198283.1;GL=EYS;FG=missense;FD=missense;AAC=LYS/ASN;PP=532/3145,532/620,532/595;CDP=1596;GS=94;PH=0.246;CP=0.001;CG=-3.870;AA=T;DV=by-frequency,by-cluster;DSP=4;GESP=G:108/T:12896;PAC=NP_001136272.1,NP_001136273.1,NP_938024.1 GT:GQ:PL 0/0:12:0,15,90 0/0:3:0,3,14 1/1:9:87,12,0 1/1:5:43,6,0</p>

<p>6 97063522 rs9373985 C G 49.80 highCVG AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1 GT:GQ:PL 0/0:9:0,9,63 0/1:27:26,0,61 1/1:13:72,18,0 0/0:35:0,36,176</p>

<p>6 111628626 rs3204953 C T 26.80 midCVG AC1=4;AF1=0.4468;DB;DP=9;DP4=0,6,3,0;FQ=29.2;MQ=60;PV4=0.012,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V3064I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=32;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=REV3L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002912.4;VDB=0.0080;DN=105;DA=C/T;GM=NM_002912.3,XM_005267088.1,XM_005267089.1;GL=REV3L;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ILE;PP=3064/3131,2986/3053,2942/3009;CDP=9190,8956,8824;GS=29;PH=0.389;CP=0.976;CG=5.710;AA=C;HA=0.0;HE=13.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa01100:MetabolicPathways;GESP=T:1410/C:11596;PPI=MAD2L2:999|REV1:998;PAC=NP_002903.3,XP_005267145.1,XP_005267146.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:65,9,0 0/0:18:0,18,99</p>

<p>6 117130704 rs2274911 G A 78.50 midCVG AC1=4;AF1=0.5298;DB;DP=12;DP4=2,3,1,6;FQ=48.8;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P91S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0097;DN=100;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=missense;FD=missense,unknown,unknown;AAC=PRO/SER;PP=91/927,91/856,91/752;CDP=271;GS=74;PH=0.002;CP=0.635;CG=-0.330;AA=A;HA=19.9;HE=29.9;HC=42.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=77;GESP=A:9803/G:3203;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1 GT:GQ:PL 0/0:5:0,6,45 0/1:3:0,0,0 1/1:7:60,9,0 0/1:45:60,0,42</p>

<p>6 117150008 rs6907580 G A 108 highCVG AC1=4;AF1=0.4941;DB;DP=17;DP4=2,6,5,3;FQ=107;MQ=60;PV4=0.31,0.089,1,0.23;SNPEFF_AMINO_ACID_CHANGE=R57*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0127;DN=116;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=stop-gained;FD=stop-gained,unknown,unknown;AAC=ARG/stop;PP=57/927,57/856,57/752;CDP=169;CP=0.862;CG=-2.840;AA=G;HA=11.5;HE=1.3;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=26;GESP=A:908/G:12098;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1 GT:GQ:PL 0/1:15:35,0,12 0/1:21:39,0,18 1/1:9:81,12,0 0/0:15:0,18,123</p>

<p>GPRC6A 6 118886961 rs3734382 G T 108 midCVG AC1=5;AF1=0.5972;DB;DP=14;DP4=1,3,6,2;FQ=43.4;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P254T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CEP85L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001178035.1;VDB=0.0162;DN=107;DA=G/T;GM=NM_001042475.2,NM_001178035.1,NM_206921.2,XM_005266970.1,XM_005266971.1;GL=CEP85L;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/THR;PP=251/806,254/809,251/497,149/704,149/704;CDP=751,760,751,445,445;GS=38;PH=0;CP=0.066;CG=-4.210;AA=T;HA=17.8;HE=26.0;HC=31.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=270;GESP=T:9623/G:3383;PAC=NP_001035940.1,NP_001171506.1,NP_996804.2,XP_005267027.1,XP_005267028.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:16:112,18,0 0/1:42:38,0,58</p>

<p>6 152443744 rs2295190 G T 29 lowCVG AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:34,6,0 0/1:24:31,0,23</p>

<p>6 155451352 rs931312 G A 154 highCVG AC1=5;AF1=0.6241;DB;DP=19;DP4=5,1,3,10;FQ=43.7;MQ=60;PV4=0.041,0.11,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R332H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TIAM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012454.3;VDB=0.0045;DN=86;DA=G/A;GM=NM_012454.3,XM_005266927.1;GL=TIAM2;FG=missense;FD=missense,unknown;AAC=ARG/HIS;PP=332/1702,332/1731;CDP=995;GS=29;PH=1;CP=0.427;CG=4.990;AA=G;HA=9.0;HE=27.2;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=200;KP=hsa04062:ChemokineSignalingPathway|hsa04810:RegulationOfActinCytoskeleton;GESP=A:2063/G:10943;PAC=NP_036586.2,XP_005266984.1 GT:GQ:PL 0/1:18:43,0,16 0/1:52:48,0,63 1/1:5:103,5,0 0/1:3:0,0,0</p>

<p>7 2646796 rs61729413 G A 20 lowCVG AC1=4;AF1=0.5539;DB;DP=6;DP4=2,0,0,3;FQ=12.4;MQ=60;PV4=0.1,0.33,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R635K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IQCE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287499.1;VDB=0.0117;DN=129;DA=G/A;GM=NM_001100390.1,NM_152558.3,XM_005249675.1,XM_005249676.1,XM_005249677.1,XM_005249678.1;GL=IQCE;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=619/680,635/696,635/703,619/687,570/638,570/631;CDP=1856,1904,1904,1856,1709,1709;GS=26;PH=0;CP=0.000;CG=-3.560;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=A:1500/G:10556;PAC=NP_001093860.1,NP_689771.3,XP_005249732.1,XP_005249733.1,XP_005249734.1,XP_005249735.1 GT:GQ:PL 0/0:6:0,6,46 0/1:3:0,0,0 1/1:6:57,9,0 0/1:3:0,0,0</p>

<p>7 4213975 rs671694 A G 129 highCVG AC1=5;AF1=0.6104;DB;DP=18;DP4=4,5,5,4;FQ=109;MQ=60;PV4=1,1,1,0.36;SNPEFF_AMINO_ACID_CHANGE=H1641R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152744.3;VDB=0.0088;DN=83;DA=A/G;GM=NM_001079653.1,NM_152744.3;GL=SDK1;FG=missense;FD=missense;AAC=HIS/ARG;PP=128/680,1641/2214;CDP=383,4922;GS=29;PH=0;CP=0.086;CG=4.910;AA=G;HA=0.0;HE=29.8;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=G:10231/A:2775;PAC=NP_001073121.1,NP_689957.3 GT:GQ:PL 0/1:37:32,0,78 0/0:8:0,12,76 1/1:6:47,6,0 1/1:14:99,15,0</p>

<p>7 6656830 rs2243563 G A 41.40 lowCVG AC1=6;AF1=0.6978;DB;DP=7;DP4=0,2,3,1;FQ=-9.9;MQ=60;PV4=0.4,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=G8R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF853;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017560.1;VDB=0.0058;DN=100;DA=G/A;GM=NM_017560.1,XM_005249790.1;GL=ZNF853;FG=missense;FD=missense,unknown;AAC=GLY/ARG;PP=8/660,8/203;CDP=22;GS=125;PH=0.001;CP=0.000;CG=-7.700;AA=G;HA=14.0;HE=49.1;HC=26.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=A:2834/G:1732;PAC=NP_060030.1,XP_005249847.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:46,6,0 1/1:6:37,6,0</p>

<p>7 6683454 rs2253869 A G 154 highCVG AC1=6;AF1=0.735;DB;DP=17;DP4=3,3,2,9;FQ=64.8;MQ=60;PV4=0.28,0.18,1,1;SNPEFF_AMINO_ACID_CHANGE=E156G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF316;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001278559.1;VDB=0.0121;DN=100;DA=A/G;GM=NM_001278559.1;GL=ZNF316;FG=missense-near-splice;FD=unknown;AAC=GLU/GLY;PP=156/1005;CDP=467;GS=98;CP=1.000;CG=3.130;AA=G;HA=22.1;HE=47.8;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2;PAC=NP_001265488.1 GT:GQ:PL 0/1:16:9,0,92 0/1:10:63,0,11 1/1:13:77,12,0 1/1:8:49,6,0</p>

<p>7 13978809 rs9639168 T C 137 highCVG AC1=5;AF1=0.6375;DB;DP=18;DP4=3,1,5,8;FQ=21.4;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S100G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ETV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004956.4;VDB=0.0073;DN=119;DA=T/C;GM=NM_001163147.1,NM_001163148.1,NM_001163149.1,NM_001163150.1,NM_001163151.1,NM_001163152.1,NM_004956.4,XM_005249635.1,XM_005249636.1,XM_005249637.1,XM_005249638.1;GL=ETV1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/GLY;PP=100/455,82/460,82/460,60/438,42/420,60/375,100/478,100/478,100/478,60/438,42/420;CDP=298,244,244,178,124,178,298,298,298,178,124;GS=56;PH=0;CP=0.998;CG=4.570;AA=C;HA=11.6;HE=29.4;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=C:3493/T:8433;PPI=RFWD2:845;PAC=NP_001156619.1,NP_001156620.1,NP_001156621.1,NP_001156622.1,NP_001156623.1,NP_001156624.1,NP_004947.2,XP_005249692.1,XP_005249693.1,XP_005249694.1,XP_005249695.1 GT:GQ:PL 0/1:25:51,0,24 0/0:3:0,6,39 1/1:20:106,21,0 1/1:4:26,3,0</p>

<p>7 20687604 rs61732039 A G 12.90 midCVG AC1=4;AF1=0.4542;DB;DP=10;DP4=3,4,1,2;FQ=15.2;MQ=60;PV4=1,1,1,0.00055;SNPEFF_AMINO_ACID_CHANGE=D370G;SNPEFF_CODON_CHANGE=gAt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163941.1;VDB=0.0080;DN=129;DA=A/G;GM=NM_001163941.1,NM_001163942.1,NM_001163993.1,NM_178559.5,XM_005249737.1;GL=ABCB5;FG=missense,5-prime-UTR,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,utr-variant-5-prime,utr-variant-5-prime,utr-variant-5-prime,unknown;AAC=ASP/GLY,none,none,none,none;PP=370/1258,NA,NA,NA,NA;CDP=1109,NA,NA,NA,NA;GS=94,NA,NA,NA,NA;PH=1;CP=1.000;CG=4.150;AA=A;DV=by-frequency,by-cluster;DSP=14;KP=hsa02010:ABCTransporters;TFBS=V$CDP_02;GESP=G:1051/A:9243;PAC=NP_001157413.1,NA,NA,NA,NA GT:GQ:PL 0/1:6:4,0,77 0/1:3:0,0,0 1/1:3:47,6,0 0/0:7:0,6,49</p>

<p>7 21468916 . C T 57.50 lowCVG AC1=8;AF1=1;DP=8;DP4=0,1,0,6;FQ=-26.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P45S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SP4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003112.3;VDB=0.0235;GM=NM_003112.3,XM_005249828.1,XM_005249829.1;GL=SP4;FG=missense;FD=unknown;AAC=PRO/SER;PP=45/785,28/768,45/707;CDP=133,82,133;GS=74;PH=0.952;CP=1.000;CG=4.610;AA=C;DSP=10;GESP=C:13002;PAC=NP_003103.2,XP_005249885.1,XP_005249886.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:88,4,0 0/1:3:0,0,0</p>

<p>7 23757162 rs6945306 G C 114 highCVG AC1=4;AF1=0.4594;DB;DP=23;DP4=2,7,8,5;FQ=71.5;MQ=60;PV4=0.099,1,0.21,1;SNPEFF_AMINO_ACID_CHANGE=Q71H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STK31;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031414.4;VDB=0.0004;DN=116;DA=G/C;GM=NM_001260504.1,NM_001260505.1,NM_031414.4,NM_032944.3,NR_048542.1,XM_005249810.1,XM_005249811.1;GL=STK31;FG=missense,missense,missense,missense,non-coding-exon,missense,missense;FD=missense,missense,missense,missense,nc-transcript-variant,unknown,unknown;AAC=GLN/HIS,GLN/HIS,GLN/HIS,GLN/HIS,none,GLN/HIS,GLN/HIS;PP=48/997,71/997,71/1020,48/997,NA,48/997,71/924;CDP=144,213,213,144,324,144,213;GS=24,24,24,24,NA,24,24;PH=0.001;CP=0.888;CG=2.950;AA=G;HA=13.6;HE=27.7;HC=32.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=37;GESP=C:3223/G:9783;PAC=NP_001247433.1,NP_001247434.1,NP_113602.2,NP_116562.2,NA,XP_005249867.1,XP_005249868.1 GT:GQ:PL 0/1:14:13,0,15 0/1:17:20,0,14 1/1:4:126,6,0 0/0:13:0,15,85</p>

<p>7 34979905 rs1637696 C A 8.89 lowCVG AC1=4;AF1=0.4657;DB;DP=8;DP4=3,1,2,1;FQ=11.3;MQ=58;PV4=1,0.008,0.14,0.28;SNPEFF_AMINO_ACID_CHANGE=G502V;SNPEFF_CODON_CHANGE=gGt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DPY19L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015283.1;VDB=0.0085;DN=89;DA=C/A;GM=NM_015283.1,XM_005249679.1;GL=DPY19L1;FG=missense;FD=missense,unknown;AAC=GLY/VAL;PP=502/676,445/619;CDP=1505,1334;GS=109;PH=0.549;CP=1.000;CG=5.240;AA=C;HA=18.3;HE=30.0;HC=10.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:3398/C:8506;PAC=NP_056098.1,XP_005249736.1 GT:GQ:PL 0/1:4:16,3,0 0/0:4:0,3,24 1/1:3:36,6,0 0/0:9:0,9,58</p>

<p>7 39649275 rs4723884 G A 50.20 lowCVG AC1=5;AF1=0.6456;DB;DP=6;DP4=1,0,1,3;FQ=-3.25;MQ=60;PV4=0.4,0.072,1,1;SNPEFF_AMINO_ACID_CHANGE=G104E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=YAE1D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282446.1;VDB=0.0224;DN=111;DA=G/A;GM=NR_024390.1,XM_005249815.1;GL=YAE1D1/LOC646999;FG=non-coding-exon,missense;FD=nc-transcript-variant,unknown;AAC=none,GLY/GLU;PP=NA,104/207;CDP=190,311;GS=NA,98;CP=0.000;CG=-1.570;AA=G;HA=46.9;HE=17.0;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=60;CPG=39649254-39649510;PAC=NA,XP_005249872.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:87,12,0 0/1:3:0,3,32</p>

<p>7 44112873 rs3218660 G T 12.40 lowCVG AC1=6;AF1=0.7387;DB;DP=5;DP4=1,0,3,0;FQ=-22.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H439N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284330.1;VDB=0.0047;DN=106;DA=G/T;GM=NM_013284.2,XM_005249708.1,XM_005249709.1,XM_005249710.1,XM_005249711.1,XM_005249712.1,XM_005249713.1;GL=POLM;FG=3-prime-UTR,missense,missense,3-prime-UTR,missense,missense,missense;FD=utr-variant-3-prime,unknown,unknown,unknown,unknown,unknown,unknown;AAC=none,HIS/ASN,HIS/ASN,none,HIS/ASN,HIS/ASN,HIS/ASN;PP=NA,469/539,439/509,NA,379/449,255/325,255/325;CDP=NA,1405,1315,NA,1135,763,763;GS=NA,68,68,NA,68,68,68;PH=0;CP=0.000;CG=-3.990;AA=G;HA=34.5;HE=19.2;HC=13.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=104;KP=hsa03450:Non-homologousEnd-joining;GESP=T:3329/G:9677;PPI=PCNA:846;PAC=NA,XP_005249765.1,XP_005249766.1,NA,XP_005249768.1,XP_005249769.1,XP_005249770.1 GT:GQ:PL 0/1:9:29,0,7 0/1:3:0,0,0 1/1:4:17,3,0 0/1:3:0,0,0</p>

<p>7 64452830 rs34639489 C T 86 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,4,2;FQ=5.73;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C192Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERV3-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007253.3;VDB=0.0141;DN=126;DA=C/T;GM=NM_001007253.3;GL=ERV3-1;FG=missense;FD=missense;AAC=CYS/TYR;PP=192/605;CDP=575;GS=194;PH=0.991;CP=0.325;CG=0.109;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;RM=HERV3-int;DSP=963;GESP=T:5840/C:6396;PAC=NP_001007254.2 GT:GQ:PL 0/0:3:0,6,41 0/1:3:0,0,0 1/1:14:102,15,0 1/1:4:26,3,0</p>

<p>7 72397435 . A G 6.99 lowCVG AC1=5;AF1=0.625;DP=5;DP4=0,1,0,3;FQ=-8.69;MQ=50;PV4=1,0.036,0.14,1;SNPEFF_AMINO_ACID_CHANGE=K54R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POM121;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001257190.2;VDB=0.0004;GM=NM_001257190.2,NM_172020.4,XM_005250726.1,XM_005250727.1,XM_005250728.1,XM_005250729.1,XM_005250730.1,XM_005250731.1,XM_005250732.1;GL=POM121;FG=missense;FD=unknown;AAC=LYS/ARG;PP=54/1000,54/985,319/1250,319/1191,54/985,54/985,54/985,54/985,54/985;CDP=161,161,956,956,161,161,161,161,161;GS=26;PH=0.018;CP=0.966;AA=-;DSP=67;GESP=A:13006;PAC=NP_001244119.1,NP_742017.1,XP_005250783.1,XP_005250784.1,XP_005250785.1,XP_005250786.1,XP_005250787.1,XP_005250788.1,XP_005250789.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,24 1/1:5:42,9,0 0/1:3:0,0,0</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>7 82453708 rs2522833 A C 74.20 lowCVG AC1=6;AF1=0.7286;DB;DP=8;DP4=1,1,1,5;FQ=15.9;MQ=60;PV4=0.46,0.24,1,0.2;SNPEFF_AMINO_ACID_CHANGE=S4814A;SNPEFF_CODON_CHANGE=Tct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCLO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033026.5;VDB=0.0027;DN=100;DA=A/C;GM=NM_014510.2,NM_033026.5;GL=PCLO;FG=missense;FD=missense;AAC=SER/ALA;PP=4814/4936,4814/5143;CDP=14440;GS=99;PH=0.013;CP=1.000;CG=5.720;AA=A;HA=3.2;HE=41.2;HC=27.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=25;GESP=C:3860/A:8250;PAC=NP_055325.2,NP_149015.2 GT:GQ:PL 0/0:3:0,6,53 0/1:3:0,0,0 1/1:7:49,6,0 1/1:12:68,12,0</p>

<p>7 95041016 rs12026 G C 34.40 midCVG AC1=3;AF1=0.4042;DB;DP=11;DP4=3,3,2,3;FQ=36.7;MQ=60;PV4=1,0.035,1,0.43;SNPEFF_AMINO_ACID_CHANGE=A148G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PON2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000305.2;VDB=0.0045;DN=52;DA=G/C;GM=NM_000305.2,NM_001018161.1,XM_005250453.1,XM_005250454.1;GL=PON2;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=148/355,136/343,80/287,78/285;CDP=443,407,239,233;GS=60;PH=0.009;CP=1.000;CG=2.550;AA=G;HA=34.9;HE=24.2;HC=20.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=40;KP=hsa01100:MetabolicPathways;GESP=C:3203/G:9803;PAC=NP_000296.2,NP_001018171.1,XP_005250510.1,XP_005250511.1 GT:GQ:PL 0/0:4:0,3,26 0/0:6:0,6,41 1/1:3:33,6,0 0/1:45:44,0,55</p>

<p>7 95822389 . G A 54.60 highCVG AC1=3;AF1=0.3723;DP=16;DP4=4,2,0,7;FQ=56.7;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P192L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001160210.1;VDB=0.0003;GM=NM_001160210.1,NM_014251.2,NR_027662.1;GL=SLC25A13;FG=missense,missense,non-coding-exon;FD=unknown;AAC=PRO/LEU,PRO/LEU,none;PP=192/677,192/676,NA;CDP=575,575,650;GS=98,98,NA;PH=0.243;CP=1.000;CG=4.100;AA=G;DSP=41;GESP=G:13006;PPI=BRE:923|BABAM1:871;PAC=NP_001153682.1,NP_055066.1,NA GT:GQ:PL 0/0:9:0,9,60 0/0:9:0,9,66 1/1:15:98,21,0 0/1:3:0,0,0</p>

<p>7 99306685 rs2257401 C G 76 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:64:67,0,63</p>

<p>7 99757612 rs3823646 G A 61.30 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=2,3,5,1;FQ=59.4;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A467V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAL3ST4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024637.4;VDB=0.0171;DN=107;DA=G/A;GM=NM_024637.4,XM_005250590.1;GL=GAL3ST4;FG=missense;FD=missense,unknown;AAC=ALA/VAL;PP=467/487,405/425;CDP=1400,1214;GS=64;PH=0.056;CP=1.000;CG=5.040;AA=G;HA=38.2;HE=41.3;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=971;GESP=A:6586/G:6420;PAC=NP_078913.3,XP_005250647.1 GT:GQ:PL 0/0:7:0,6,43 0/1:3:0,0,0 1/1:13:104,18,0 0/0:9:0,9,59</p>

<p>7 107593989 rs20556 T C 80.60 midCVG AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1 GT:GQ:PL 0/1:17:41,0,14 0/1:3:0,0,0 1/1:13:82,15,0 0/0:4:0,6,52</p>

<p>7 111580166 rs144867634 T C 90.70 highCVG AC1=3;AF1=0.3681;DB;DP=23;DP4=12,3,3,4;FQ=92.7;MQ=60;PV4=0.14,0.45,1,0.35;SNPEFF_AMINO_ACID_CHANGE=M326V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DOCK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014705.3;VDB=0.0036;DN=134;DA=T/C;GM=NM_014705.3,XM_005250724.1;GL=DOCK4;FG=missense-near-splice;FD=missense,unknown;AAC=MET/VAL;PP=326/1967,326/2012;CDP=976;GS=21;PH=0.066;CP=1.000;CG=6.060;AA=T;DV=by-frequency,by-cluster;DSP=2;GESP=C:225/T:12057;PAC=NP_055520.3,XP_005250781.1 GT:GQ:PL 0/0:26:0,27,130 0/1:24:48,0,18 1/1:10:91,15,0 0/0:15:0,15,111</p>

<p>7 113518434 rs1799999 C A 68 lowCVG AC1=6;AF1=0.7069;DB;DP=8;DP4=1,2,1,4;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D905Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002711.3;VDB=0.0058;DN=89;DA=C/A;GM=NM_002711.3,XM_005250473.1;GL=PPP1R3A;FG=missense;FD=missense,unknown;AAC=ASP/TYR;PP=905/1123,704/922;CDP=2713,2110;GS=160;PH=0.976;CP=0.997;CG=5.640;AA=C;HA=23.4;HE=6.3;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=1747;KP=hsa04910:InsulinSignalingPathway;GESP=A:1760/C:11244;PAC=NP_002702.2,XP_005250530.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:59,6,0 0/1:49:45,0,55</p>

<p>7 120776097 rs41281692 C G 153 midCVG AC1=6;AF1=0.7444;DB;DP=14;DP4=2,1,4,7;FQ=14.1;MQ=60;PV4=0.54,0.2,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A551G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024913.4;VDB=0.0084;DN=127;DA=C/G;GM=NM_001105533.1,NM_024913.4,XM_005250604.1,XM_005250605.1;GL=CPED1;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=551/784,551/1027,331/807,331/564;CDP=1652,1652,992,992;GS=60;PH=0.403;CP=0.000;CG=-0.512;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4;GESP=G:4925/C:8057;PAC=NP_001099003.1,NP_079189.4,XP_005250661.1,XP_005250662.1 GT:GQ:PL 0/1:12:17,0,13 0/1:20:13,0,40 1/1:16:101,15,0 1/1:11:67,9,0</p>

<p>7 122635024 rs860170 C T 171 highCVG AC1=5;AF1=0.609;DB;DP=24;DP4=3,8,2,10;FQ=110;MQ=60;PV4=0.64,1,1,0.064;SNPEFF_AMINO_ACID_CHANGE=R222H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TAS2R16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016945.2;VDB=0.0143;DN=86;DA=C/T;GM=NM_016945.2;GL=TAS2R16;FG=missense;FD=missense;AAC=ARG/HIS;PP=222/292;CDP=665;GS=29;PH=0;CP=0.138;CG=2.210;AA=T;HA=3.2;HE=27.6;HC=47.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=108972;KP=hsa04742:TasteTransduction;GESP=T:9806/C:3200;PAC=NP_058641.1 GT:GQ:PL 0/1:13:69,0,12 0/0:25:0,30,144 1/1:14:84,15,0 1/1:8:68,9,0</p>

<p>7 129950740 rs2171492 G T 30.70 lowCVG AC1=4;AF1=0.5413;DB;DP=5;DP4=0,2,3,0;FQ=11.4;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G303C;SNPEFF_CODON_CHANGE=Ggc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPA4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016352.3;VDB=0.0110;DN=96;DA=G/T;GM=NM_001163446.1,NM_016352.3;GL=CPA4;FG=missense;FD=missense;AAC=GLY/CYS;PP=270/389,303/422;CDP=808,907;GS=159;PH=0.012;CP=0.992;CG=0.479;AA=T;HA=17.1;HE=34.6;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=87;GESP=T:4313/G:8693;PPI=LXN:959;PAC=NP_001156918.1,NP_057436.2 GT:GQ:PL 0/1:3:26,3,0 0/0:3:0,3,25 1/1:5:47,6,0 0/0:3:0,3,25</p>

<p>7 133848257 rs17167553 G T 35.10 highCVG AC1=3;AF1=0.3866;DB;DP=16;DP4=5,6,0,4;FQ=37.2;MQ=60;PV4=0.23,0.31,1,0.17;SNPEFF_AMINO_ACID_CHANGE=D302Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRGUK;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144648.1;VDB=0.0117;DN=123;DA=G/T;GM=NM_144648.1;GL=LRGUK;FG=missense;FD=unknown;AAC=ASP/TYR;PP=302/826;CDP=904;GS=160;PH=0.902;CP=0.983;CG=3.770;AA=G;HA=13.5;HE=7.3;HC=32.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=36;GESP=T:1632/G:11374;PAC=NP_653249.1 GT:GQ:PL 0/0:23:0,24,157 0/0:4:0,3,23 1/1:5:63,9,0 0/1:18:17,0,43</p>

<p>7 149419635 rs75701347 T G 36.70 lowCVG AC1=5;AF1=0.5703;DB;DP=6;DP4=1,1,0,4;FQ=7.23;MQ=60;PV4=0.33,0.44,1,1;SNPEFF_AMINO_ACID_CHANGE=S231A;SNPEFF_CODON_CHANGE=Tca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRBA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001290187.1;VDB=0.0121;DN=132;DA=T/G;GM=NM_032534.2,XM_005250063.1,XM_005250064.1,XM_005250065.1,XM_005250066.1,XM_005250067.1,XM_005250068.1;GL=KRBA1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=SER/ALA;PP=197/1031,250/1023,249/1022,248/1021,235/1008,234/1007,250/735;CDP=589,748,745,742,703,700,748;GS=99;PH=0.001;CP=0.000;CG=-7.850;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=10;GESP=G:1451/T:10543;PAC=NP_115923.2,XP_005250120.1,XP_005250121.1,XP_005250122.1,XP_005250123.1,XP_005250124.1,XP_005250125.1 GT:GQ:PL 0/1:3:0,3,25 0/1:3:0,3,21 1/1:3:25,3,0 1/1:7:55,9,0</p>

<p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

<p>8 10390452 rs4406360 C T 170 highCVG AC1=5;AF1=0.6627;DB;DP=16;DP4=4,0,5,7;FQ=10.7;MQ=60;PV4=0.088,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A212V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS55;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198464.3;VDB=0.0094;DN=111;DA=C/T;GM=NM_001197020.1,NM_198464.3,XM_005272372.1;GL=PRSS55;FG=missense;FD=missense,missense,unknown;AAC=ALA/VAL;PP=212/277,212/353,212/257;CDP=635;GS=64;PH=0.108;CP=0.642;CG=-4.080;AA=C;HA=18.5;HE=34.2;HC=40.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=37;GESP=T:4047/C:8959;PAC=NP_001183949.1,NP_940866.2,XP_005272429.1 GT:GQ:PL 0/1:25:88,0,24 0/1:4:0,3,26 1/1:4:26,3,0 1/1:15:100,15,0</p>

<p>8 10468976 . G C 25.70 lowCVG AC1=5;AF1=0.6557;DP=5;DP4=2,0,2,1;FQ=8.94;MQ=60;PV4=1,0.4,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Q878E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RP1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178857.5;VDB=0.0119;GM=NM_178857.5;GL=RP1L1;FG=missense;FD=unknown;AAC=GLN/GLU;PP=878/2401;CDP=2632;GS=29;PH=0.005;CP=0.002;CG=2.580;AA=G;DSP=1881;GESP=G:10756;PAC=NP_849188.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,44 1/1:4:24,3,0 1/1:5:42,6,0</p>

<p>8 12879539 rs608052 A G 159 highCVG AC1=5;AF1=0.5689;DB;DP=16;DP4=3,2,4,7;FQ=62.2;MQ=60;PV4=0.6,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R451G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1456;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020844.2;VDB=0.0150;DN=83;DA=A/G;GM=NM_001099677.1,NM_020844.2,XM_005273584.1,XM_005273585.1,XM_005273586.1,XM_005273587.1,XM_005273588.1,XM_005273589.1,XM_005273590.1,XM_005273591.1,XM_005273592.1,XM_005273593.1,XM_005273594.1,XM_005273595.1;GL=KIAA1456;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLY;PP=325/329,451/455,520/524,471/475,451/455,420/424,420/424,415/419,415/419,415/419,415/419,364/368,325/329,364/368;CDP=973,1351,1558,1411,1351,1258,1258,1243,1243,1243,1243,1090,973,1090;GS=125;PH=0;CP=0.034;CG=0.894;AA=G;HA=45.0;HE=18.9;HC=25.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1023;GESP=G:8818/A:3100;PAC=NP_001093147.1,NP_065895.2,XP_005273641.1,XP_005273642.1,XP_005273643.1,XP_005273644.1,XP_005273645.1,XP_005273646.1,XP_005273647.1,XP_005273648.1,XP_005273649.1,XP_005273650.1,XP_005273651.1,XP_005273652.1 GT:GQ:PL 0/0:9:0,12,85 0/1:3:0,3,23 1/1:8:69,9,0 1/1:22:140,24,0</p>

<p>8 17796382 rs412750 A G 121 highCVG AC1=5;AF1=0.6136;DB;DP=19;DP4=4,1,8,3;FQ=7.28;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N159S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006197.3;VDB=0.0111;DN=80;DA=A/G;GM=NM_006197.3,XM_005273509.1,XM_005273510.1,XM_005273511.1,XM_005273512.1,XM_005273513.1,XM_005273514.1,XM_005273515.1,XM_005273516.1,XM_005273517.1,XM_005273518.1,XM_005273519.1,XM_005273520.1,XM_005273521.1,XM_005273522.1,XM_005273523.1,XM_005273524.1,XM_005273525.1;GL=PCM1;FG=missense;FD=unknown;AAC=ASN/SER;PP=159/2025,159/2088,159/2088,159/2088,159/2087,159/2084,159/2080,159/2076,159/2069,159/2064,126/2055,159/2049,159/2049,159/2017,159/2014,159/2010,159/2002,159/1536;CDP=476,476,476,476,476,476,476,476,476,476,377,476,476,476,476,476,476,476;GS=46;PH=0;CP=0.511;CG=4.850;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=G:9737/A:2227;PPI=BBS4:983|BBS1:775;PAC=NP_006188.3,XP_005273566.1,XP_005273567.1,XP_005273568.1,XP_005273569.1,XP_005273570.1,XP_005273571.1,XP_005273572.1,XP_005273573.1,XP_005273574.1,XP_005273575.1,XP_005273576.1,XP_005273577.1,XP_005273578.1,XP_005273579.1,XP_005273580.1,XP_005273581.1,XP_005273582.1 GT:GQ:PL 0/1:17:13,0,23 0/1:9:76,0,7 1/1:6:45,6,0 0/1:16:30,0,15</p>

<p>8 32453358 rs3924999 G A 31.20 lowCVG AC1=6;AF1=0.7171;DB;DP=6;DP4=0,1,2,1;FQ=-15.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R38Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013956.3;VDB=0.0076;DN=108;DA=G/A;GM=NM_001159995.1,NM_001159999.1,NM_001160001.1,NM_001160002.1,NM_001160004.1,NM_001160005.1,NM_001160007.1,NM_001160008.1,NM_004495.3,NM_013956.3,NM_013957.3,NM_013958.3,NM_013960.3,NM_013962.2,NM_013964.3,XM_005273488.1;GL=NRG1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,unknown;AAC=ARG/GLN;PP=17/608,17/625,17/591,38/195,38/460,38/208,38/178,38/421,38/212,38/646,38/638,38/242,38/463,253/423,38/641,45/636;CDP=50,50,50,113,113,113,113,113,113,113,113,113,113,758,113,134;GS=43;PH=0.966;CP=0.998;CG=5.650;AA=G;HA=1.3;HE=43.2;HC=21.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=13;KP=hsa04012:ErbBSignalingPathway;GESP=A:3990/G:9016;PPI=ERBB3:989|ERBB4:983;PAC=NP_001153467.1,NP_001153471.1,NP_001153473.1,NP_001153474.1,NP_001153476.1,NP_001153477.1,NP_001153479.1,NP_001153480.1,NP_004486.2,NP_039250.2,NP_039251.2,NP_039252.2,NP_039254.1,NP_039256.2,NP_039258.1,XP_005273545.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,18 1/1:4:20,3,0 1/1:6:50,6,0</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 67380528 rs1060242 T C 74.90 highCVG AC1=5;AF1=0.62;DB;DP=19;DP4=3,3,4,4;FQ=64;MQ=60;PV4=1,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=C449R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADHFE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144650.2;VDB=0.0170;DN=86;DA=T/C;GM=NM_144650.2,XM_005251152.1,XM_005251153.1,XM_005251154.1,XM_005251155.1,XM_005251156.1,XM_005251157.1,XM_005251158.1,XM_005251159.1,XM_005251160.1,XM_005251161.1,XM_005251162.1,XM_005251163.1;GL=ADHFE1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=CYS/ARG;PP=449/468,430/449,401/420,401/420,401/420,401/420,401/420,379/398,401/420,219/238,219/238,219/238,219/238;CDP=1345,1288,1201,1201,1201,1201,1201,1135,1201,655,655,655,655;GS=180;PH=0;CP=0.818;CG=3.680;AA=C;HA=23.7;HE=42.6;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;GESP=C:7238/T:5768;PAC=NP_653251.2,XP_005251209.1,XP_005251210.1,XP_005251211.1,XP_005251212.1,XP_005251213.1,XP_005251214.1,XP_005251215.1,XP_005251216.1,XP_005251217.1,XP_005251218.1,XP_005251219.1,XP_005251220.1 GT:GQ:PL 0/0:8:0,12,80 0/1:21:17,0,29 1/1:8:35,9,0 1/1:11:72,12,0</p>

<p>8 67592152 rs1057463 T C 68.70 highCVG AC1=5;AF1=0.6134;DB;DP=22;DP4=3,8,6,2;FQ=72.8;MQ=60;PV4=0.07,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=F148S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C8orf44;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019607.2;VDB=0.0098;DN=86;DA=T/C;GM=NM_001204173.1,NM_019607.2,XM_005251270.1,XM_005251271.1,XM_005251272.1;GL=C8orf44/C8orf44-SGK3;FG=5-prime-UTR,missense,missense,missense,missense;FD=utr-variant-5-prime,missense,unknown,unknown,unknown;AAC=none,PHE/SER,PHE/SER,PHE/SER,PHE/SER;PP=NA,148/160,148/160,148/160,148/160;CDP=NA,443,443,443,443;GS=NA,155,155,155,155;PH=0.713;CP=0.020;CG=2.840;AA=T;HA=4.9;HE=33.5;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=108;GESP=C:2939/T:10067;PPI=GSK3B:869;PAC=NA,NP_062553.1,XP_005251327.1,XP_005251328.1,XP_005251329.1 GT:GQ:PL 0/1:32:27,0,82 0/0:6:0,9,55 1/1:6:35,6,0 1/1:8:54,9,0</p>

<p>8 67796120 rs17332410 C G 17.80 midCVG AC1=3;AF1=0.3679;DB;DP=14;DP4=6,5,2,1;FQ=19.7;MQ=60;PV4=1,0.028,1,1;SNPEFF_AMINO_ACID_CHANGE=L322V;SNPEFF_CODON_CHANGE=Ctc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCMDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173518.4;VDB=0.0061;DN=123;DA=C/G;GM=NM_001136160.1,NM_001136161.1,NM_173518.4,XM_005251173.1,XM_005251174.1;GL=MCMDC2;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=LEU/VAL;PP=322/633,322/591,322/682,322/682,259/619;CDP=964,964,964,964,775;GS=32;PH=0.946;CP=1.000;CG=4.680;AA=C;HA=0.0;HE=6.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=110;TFBS=V$CEBPA_01;GESP=G:735/C:12271;PAC=NP_001129632.1,NP_001129633.1,NP_775789.3,XP_005251230.1,XP_005251231.1 GT:GQ:PL 0/0:15:0,15,96 0/1:5:20,3,0 1/1:3:44,6,0 0/0:18:0,18,122</p>

<p>8 97614625 rs3816208 G A 21 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1 GT:GQ:PL 0/0:9:0,6,45 0/0:18:0,15,111 1/1:3:63,9,0 0/0:9:0,6,47</p>

<p>8 103573006 rs2916569 G A 17.80 midCVG AC1=4;AF1=0.504;DB;DP=10;DP4=2,1,4,0;FQ=4.62;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S216N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024410.3;VDB=0.0051;DN=101;DA=G/A;GM=NM_024410.3;GL=ODF1;FG=missense;FD=missense;AAC=SER/ASN;PP=216/251;CDP=647;GS=46;PH=0;CP=0.053;CG=-7.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;RT=TGCAGCCCC;DSP=327;GESP=G:13006;PAC=NP_077721.2 GT:GQ:PL 0/0:5:0,6,25 0/1:3:0,0,0 1/1:4:34,6,0 0/1:18:24,0,15</p>

<p>8 104412688 rs141856398 T C 58.40 highCVG AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA GT:GQ:PL 0/0:4:0,3,24 0/0:6:0,6,40 1/1:3:38,6,0 0/1:64:63,0,86</p>

<p>8 110457011 rs72687015 A G 85.70 midCVG AC1=5;AF1=0.6401;DB;DP=10;DP4=3,1,4,2;FQ=43.1;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=Y1638C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKHD1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_177531.4;VDB=0.0097;DN=130;DA=A/G;GM=NM_177531.4;GL=PKHD1L1;FG=missense;FD=missense;AAC=TYR/CYS;PP=1638/4244;CDP=4913;GS=194;PH=0.951;CP=1.000;CG=3.250;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=122;GESP=G:2556/A:9592;PAC=NP_803875.2 GT:GQ:PL 0/1:21:16,0,37 0/0:4:0,6,48 1/1:9:70,9,0 1/1:6:46,6,0</p>

<p>8 124449466 rs3824250 C T 38.30 midCVG AC1=4;AF1=0.4784;DB;DP=10;DP4=5,1,1,3;FQ=41.1;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R134C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDYHV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018024.2;VDB=0.0061;DN=107;DA=C/T;GM=NM_018024.1,XM_005250997.1,XM_005250998.1;GL=WDYHV1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/CYS;PP=134/206,74/146,66/138;CDP=400,220,196;GS=180;PH=0.99;CP=1.000;CG=5.480;AA=C;HA=34.8;HE=28.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;TFBS=V$AHRARNT_02,V$TCF11MAFG_01;GESP=T:4627/C:8379;PAC=NP_060494.1,XP_005251054.1,XP_005251055.1 GT:GQ:PL 0/1:16:14,0,41 0/0:7:0,9,51 1/1:4:47,6,0 0/1:17:22,0,14</p>

<p>8 141468645 . G A 29.60 midCVG AC1=3;AF1=0.3656;DP=12;DP4=6,0,5,0;FQ=31.5;MQ=60;PV4=1,0.034,1,1.3e-09;SNPEFF_AMINO_ACID_CHANGE=Q7*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC9;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_031466.6;VDB=0.0000;GM=NM_031466.5;GL=TRAPPC9;FG=stop-gained;FD=unknown;AAC=GLN/stop;PP=7/1247;CDP=19;CP=0.000;CG=-2.520;AA=G;DSP=266;GESP=G:12786;PAC=NP_113654.4 GT:GQ:PL 0/0:3:0,0,0 0/0:7:0,6,51 1/1:9:72,15,0 0/0:12:0,12,70</p>

<p>WDYHV1 8 142367335 rs10875472 T C 48.60 lowCVG AC1=6;AF1=0.6934;DB;DP=6;DP4=1,0,3,2;FQ=-18.7;MQ=60;PV4=1,0.17,1,0.27;SNPEFF_AMINO_ACID_CHANGE=H230R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005293.2;VDB=0.0059;DN=120;DA=T/C;GM=NM_005293.2;GL=GPR20;FG=missense;FD=missense;AAC=HIS/ARG;PP=230/359;CDP=689;GS=29;PH=0;CP=0.989;CG=-0.037;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=713;CPG=142367051-142367860;GESP=C:10416/T:2188;PAC=NP_005284.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:21,3,0 0/1:13:62,0,11</p>

<p>8 142517222 rs2748421 A G 92.90 midCVG AC1=5;AF1=0.6549;DB;DP=13;DP4=3,2,8,0;FQ=41.5;MQ=59;PV4=0.13,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=Y10H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0027;DN=100;DA=A/G;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/HIS,none,none;PP=10/1319,NA,NA;CDP=28,109,109;GS=83,NA,NA;CP=0.000;CG=1.370;AA=G;HA=20.8;HE=49.1;HC=36.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=G:4995/A:7673;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:28:52,0,27 1/1:11:68,12,0 0/1:19:14,0,51</p>

<p>8 142517227 rs2613648 C T 79.90 midCVG AC1=5;AF1=0.6558;DB;DP=13;DP4=3,2,8,0;FQ=19.6;MQ=59;PV4=0.13,1,0.23,0.41;SNPEFF_AMINO_ACID_CHANGE=R8K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0030;DN=100;DA=C/T;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ARG/LYS,none,none;PP=8/1319,NA,NA;CDP=23,104,104;GS=26,NA,NA;CP=0.000;CG=-2.700;AA=C;HA=21.0;HE=49.1;HC=36.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=58;GESP=T:4996/C:7690;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:21:47,0,20 1/1:11:61,12,0 0/1:18:13,0,36</p>

<p>8 144801593 rs60732298 C A 16.80 lowCVG AC1=5;AF1=0.6552;DB;DP=5;DP4=2,0,3,0;FQ=7.18;MQ=60;PV4=1,0.027,1,1;SNPEFF_AMINO_ACID_CHANGE=T221K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAPK15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139021.2;VDB=0.0066;DN=129;DA=C/A;GM=NM_139021.2;GL=MAPK15;FG=missense;FD=missense;AAC=THR/LYS;PP=221/545;CDP=662;GS=78;PH=0.992;CP=0.104;CG=4.140;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;GESP=A:1162/C:11844;PAC=NP_620590.2 GT:GQ:PL 0/0:4:0,6,42 0/1:3:0,0,0 1/1:3:20,3,0 1/1:5:37,6,0</p>

<p>9 742388 rs140893133 G A 40.80 midCVG AC1=2;AF1=0.3159;DB;DP=13;DP4=7,1,3,1;FQ=42.4;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=G1294S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KANK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256876.1;VDB=0.0053;DN=134;DA=G/A;GM=NM_001256876.1,NM_001256877.1,NM_015158.3,NM_153186.4,XM_005251410.1,XM_005251411.1,XM_005251412.1,XM_005251413.1,XM_005251414.1,XM_005251415.1,XM_005251416.1,XM_005251417.1,XM_005251418.1,XM_005251419.1;GL=KANK1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/SER;PP=1294/1353,1294/1353,1294/1353,1136/1195,1294/1365,1294/1365,1294/1365,1214/1285,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207;CDP=3880,3880,3880,3406,3880,3880,3880,3640,3406,3406,3406,3406,3406,3406;GS=56;PH=0.073;CP=0.955;CG=5.330;AA=G;DV=by-frequency,by-cluster;DSP=18;TFBS=V$CHOP_01;GESP=A:16/G:12990;PPI=YWHAQ:846;PAC=NP_001243805.1,NP_001243806.1,NP_055973.2,NP_694856.1,XP_005251467.1,XP_005251468.1,XP_005251469.1,XP_005251470.1,XP_005251471.1,XP_005251472.1,XP_005251473.1,XP_005251474.1,XP_005251475.1,XP_005251476.1 GT:GQ:PL 0/0:13:0,12,85 0/0:3:0,0,0 1/1:6:83,12,0 0/0:13:0,12,70</p>

<p>9 14722477 rs3747532 G C 33 midCVG AC1=5;AF1=0.5574;DB;DP=10;DP4=2,3,3,0;FQ=36.1;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A65G;SNPEFF_CODON_CHANGE=gCc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CER1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005454.2;VDB=-0.0014;DN=107;DA=G/C;GM=NM_005454.2;GL=CER1;FG=missense;FD=missense;AAC=ALA/GLY;PP=65/268;CDP=194;GS=60;PH=0;CP=0.001;CG=2.060;AA=C;HA=12.7;HE=35.8;HC=4.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=314;KP=hsa04310:WntSignalingPathway;GESP=C:6095/G:6911;PAC=NP_005445.1 GT:GQ:PL 0/1:3:27,3,0 0/1:3:0,0,0 1/1:5:48,6,0 0/0:12:0,15,84</p>

<p>9 19058743 rs10511670 G C 22.10 highCVG AC1=4;AF1=0.463;DB;DP=20;DP4=7,8,4,1;FQ=24.7;MQ=28;PV4=0.32,1,0.41,0.28;SNPEFF_AMINO_ACID_CHANGE=H674Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017645.4;VDB=0.0076;DN=119;DA=G/C;GM=NM_001270890.1,NM_017645.4,XM_005251491.1;GL=HAUS6;FG=missense;FD=missense,missense,unknown;AAC=HIS/GLN;PP=639/921,674/956,674/939;CDP=1917,2022,2022;GS=24;PH=0.008;CP=0.000;CG=-4.360;AA=G;HA=6.2;HE=6.7;HC=13.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=257;GESP=C:1067/G:11939;PPI=HAUS1:999|HAUS4:994|HAUS3:994|HAUS8:994|HAUS2:994|HAUS7:994;PAC=NP_001257819.1,NP_060115.3,XP_005251548.1 GT:GQ:PL 0/1:25:23,0,67 0/1:3:0,0,0 1/1:6:42,9,0 0/0:22:0,24,114</p>

<p>9 19290794 rs41269019 A C 75.40 midCVG AC1=3;AF1=0.3437;DB;DP=11;DP4=0,5,3,3;FQ=69.4;MQ=60;PV4=0.18,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=I241L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DENND4C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017925.5;VDB=0.0195;DN=127;DA=A/C;GM=NM_017925.5,NR_073201.2;GL=DENND4C;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=ILE/LEU,none;PP=241/1910,NA;CDP=721,807;GS=5,NA;PH=0.996;CP=1.000;CG=5.440;AA=A;DV=by-frequency,by-cluster;DSP=81;GESP=C:267/A:11599;PAC=NP_060395.5,NA GT:GQ:PL 0/0:5:0,3,26 0/0:5:0,3,25 1/1:12:119,18,0 0/0:10:0,9,61</p>

<p>9 26116150 rs7019716 G T 117 highCVG AC1=5;AF1=0.6707;DB;DP=18;DP4=3,3,1,10;FQ=65.1;MQ=60;PV4=0.099,0.0034,1,1;SNPEFF_AMINO_ACID_CHANGE=S109I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LOC100506422;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004352.2;VDB=0.0134;DN=116;DA=G/T;GM=NM_001004352.2;GL=LOC100506422;FG=missense;FD=missense;AAC=SER/ILE;PP=109/142;CDP=326;GS=142;CP=0.006;CG=-5.100;AA=G;HA=26.5;HE=31.2;HC=6.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=24;PAC=NP_001004352.1 GT:GQ:PL 0/1:37:31,0,62 0/0:3:0,6,46 1/1:9:50,9,0 1/1:15:84,15,0</p>

<p>9 26978259 rs11555693 A G 90.20 highCVG AC1=6;AF1=0.7286;DB;DP=22;DP4=0,6,4,5;FQ=48.8;MQ=60;PV4=0.1,0.013,1,0.18;SNPEFF_AMINO_ACID_CHANGE=K85R;SNPEFF_CODON_CHANGE=aAa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IFT74;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099222.1;VDB=0.0093;DN=120;DA=A/G;GM=NM_001099222.1,NM_001099223.1,NM_001099224.1,NM_025103.2;GL=IFT74;FG=missense;FD=missense;AAC=LYS/ARG;PP=85/601,85/601,85/373,85/601;CDP=254;GS=26;PH=0.459;CP=1.000;CG=5.620;AA=A;DV=by-frequency,by-cluster;DSP=3;GESP=G:735/A:11169;PAC=NP_001092692.1,NP_001092693.1,NP_001092694.1,NP_079379.2 GT:GQ:PL 0/1:78:77,0,80 0/1:3:0,0,0 1/1:9:50,9,0 0/1:3:0,0,0</p>

<p>9 33385863 rs62542746 G T 17.60 highCVG AC1=5;AF1=0.6224;DB;DP=17;DP4=2,5,5,5;FQ=21.6;MQ=33;PV4=0.62,1,1.5e-05,0.36;SNPEFF_AMINO_ACID_CHANGE=A176E;SNPEFF_CODON_CHANGE=gCg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0114;DN=129;DA=G/T;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense-near-splice;FD=missense,unknown,unknown;AAC=ALA/GLU;PP=176/343,176/343,176/274;CDP=527;GS=107;PH=0.006;CP=0.040;CG=1.570;AA=T;DV=by-cluster;DSP=2;KP=hsa03320:PPARSignalingPathway;GESP=G:13002;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:11:7,0,29 0/0:4:0,5,74 1/1:5:18,6,0 1/1:5:37,6,0</p>

<p>9 33387047 rs4008658 T G 66.90 highCVG AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:27:37,0,25 0/1:17:22,0,15 1/1:10:52,12,0 0/1:4:0,2,50</p>

<p>9 75315438 rs1796993 G A 69.40 midCVG AC1=4;AF1=0.4398;DB;DP=14;DP4=3,4,4,3;FQ=70.4;MQ=60;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=E81K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138691.2;VDB=0.0098;DN=89;DA=G/A;GM=NM_138691.2;GL=TMC1;FG=missense;FD=missense;AAC=GLU/LYS;PP=81/761;CDP=241;GS=56;PH=0.155;CP=1.000;CG=5.480;AA=G;HA=32.0;HE=16.5;HC=43.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=5;GESP=A:2606/G:10392;PAC=NP_619636.2 GT:GQ:PL 0/1:16:72,0,11 0/0:8:0,9,66 1/1:4:38,6,0 0/1:5:2,0,39</p>

<p>9 88651370 rs2297002 T C 82.10 midCVG AC1=3;AF1=0.3754;DB;DP=12;DP4=5,1,4,2;FQ=82;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H217R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GOLM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016548.3;VDB=0.0035;DN=100;DA=T/C;GM=NM_016548.3,NM_177937.2,XM_005252040.1,XM_005252041.1,XM_005252042.1,XM_005252043.1,XM_005252044.1;GL=GOLM1;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=HIS/ARG;PP=217/402,217/402,217/459,217/459,217/403,217/385,217/385;CDP=650;GS=29;PH=0;CP=0.000;CG=-3.200;AA=C;HA=49.2;HE=2.5;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=C:1785/T:11221;PAC=NP_057632.2,NP_808800.1,XP_005252097.1,XP_005252098.1,XP_005252099.1,XP_005252100.1,XP_005252101.1 GT:GQ:PL 0/0:12:0,12,76 0/0:7:0,6,51 1/1:12:125,18,0 0/1:3:0,0,0</p>

<p>9 95782118 . A G 18 midCVG AC1=2;AF1=0.2756;DP=9;DP4=4,2,3,0;FQ=19.4;MQ=60;PV4=0.5,0.43,1,1;SNPEFF_AMINO_ACID_CHANGE=E454G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001083536.1;VDB=0.0003;GM=NM_001083536.1,NM_033086.2,XM_005252296.1;GL=FGD3;FG=missense;FD=unknown;AAC=GLU/GLY;PP=454/726,454/726,454/725;CDP=1361;GS=98;PH=0.033;CP=0.997;CG=4.620;AA=A;DSP=6;KP=hsa04810:RegulationOfActinCytoskeleton;GESP=A:12070;PAC=NP_001077005.1,NP_149077.2,XP_005252353.1 GT:GQ:PL 0/0:14:0,12,91 0/0:6:0,3,16 1/1:3:59,9,0 0/0:6:0,3,26</p>

<p>9 104433235 rs10989589 C T 82.80 highCVG AC1=4;AF1=0.5477;DB;DP=16;DP4=2,2,4,3;FQ=34.3;MQ=60;PV4=1,0.22,1,0.16;SNPEFF_AMINO_ACID_CHANGE=G487R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRIN3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133445.2;VDB=0.0068;DN=120;DA=C/T;GM=NM_133445.2;GL=GRIN3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=487/1116;CDP=1459;GS=125;PH=0.956;CP=0.252;CG=5.760;AA=C;HA=10.0;HE=35.8;HC=20.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=155;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=T:4290/C:8716;PPI=GRIN1:846|CPEB1:830;PAC=NP_597702.2 GT:GQ:PL 0/0:5:0,6,49 0/1:3:0,0,0 1/1:10:83,12,0 0/1:27:42,0,24</p>

<p>9 107266596 rs7049042 T C 101 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=0,2,2,5;FQ=3.71;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F18S;SNPEFF_CODON_CHANGE=tTt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0171;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=PHE/SER;PP=18/320;CDP=53;GS=155;PH=0;CP=0.855;CG=3.000;AA=C;HA=8.7;HE=27.3;HC=22.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=243538;KP=hsa04740:OlfactoryTransduction;GESP=C:10035/T:2971;PAC=NP_001004485.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,38 1/1:17:117,18,0 1/1:4:26,3,0</p>

<p>9 107267352 rs7847413 T C 13.20 midCVG AC1=2;AF1=0.2756;DB;DP=14;DP4=2,9,3,0;FQ=14.5;MQ=60;PV4=0.027,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=I270T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0024;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=ILE/THR;PP=270/320;CDP=809;GS=89;PH=0.016;CP=0.942;CG=1.920;AA=T;HA=40.2;HE=8.4;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=242782;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFAT_Q6;GESP=C:2246/T:10760;PAC=NP_001004485.1 GT:GQ:PL 0/0:14:0,12,74 0/0:23:0,21,130 1/1:3:54,9,0 0/0:4:0,0,0</p>

<p>9 107367008 rs10156474 T C 41.50 lowCVG AC1=6;AF1=0.7416;DB;DP=5;DP4=1,0,2,2;FQ=-17.1;MQ=40;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K301E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13C2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004481.1;VDB=0.0107;DN=119;DA=T/C;GM=NM_001004481.1;GL=OR13C2;FG=missense;FD=missense;AAC=LYS/GLU;PP=301/319;CDP=901;GS=56;PH=0.693;CP=0.996;CG=3.530;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=143126;KP=hsa04740:OlfactoryTransduction;GESP=C:4176/T:8826;PAC=NP_001004481.1 GT:GQ:PL 0/1:3:0,0,0 0/1:14:14,0,14 1/1:9:64,9,0 0/1:3:0,0,0</p>

<p>9 107380414 rs2900373 C A 67.60 midCVG AC1=5;AF1=0.6029;DB;DP=13;DP4=4,1,2,5;FQ=29.7;MQ=44;PV4=0.24,0.14,1,0.38;SNPEFF_AMINO_ACID_CHANGE=E24D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13C9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001956.1;VDB=0.0115;DN=101;DA=C/A/T;GM=NM_001001956.1;GL=OR13C9;FG=missense;FD=missense/synonymous-codon;AAC=GLU/ASP;PP=24/319;CDP=72;GS=45;PH=0.229;CP=0.982;CG=3.890;AA=C;HA=34.1;HE=20.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=129720;KP=hsa04740:OlfactoryTransduction;GESP=A:4083/C:8923;PAC=NP_001001956.1 GT:GQ:PL 0/1:12:9,0,14 0/0:6:0,9,58 1/1:4:19,3,0 1/1:14:86,15,0</p>

<p>9 107533175 rs3739741 C G 117 highCVG AC1=5;AF1=0.5707;DB;DP=19;DP4=1,7,4,7;FQ=84.8;MQ=60;PV4=0.34,0.018,1,1;SNPEFF_AMINO_ACID_CHANGE=A159G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPSNAP3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018376.2;VDB=0.0089;DN=107;DA=C/G;GM=NM_018376.2;GL=NIPSNAP3B;FG=missense;FD=unknown;AAC=ALA/GLY;PP=159/248;CDP=476;GS=60;PH=0.999;CP=1.000;CG=2.440;AA=C;HA=0.0;HE=20.7;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=46;GESP=G:1830/C:11176;PAC=NP_060846.2 GT:GQ:PL 0/1:28:67,0,26 0/0:14:0,18,103 1/1:4:27,3,0 1/1:13:71,15,0</p>

<p>9 113208250 rs7863519 T G 81.60 midCVG AC1=4;AF1=0.5301;DB;DP=11;DP4=3,1,3,4;FQ=42.9;MQ=60;PV4=0.55,0.4,1,1;SNPEFF_AMINO_ACID_CHANGE=M1444L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SVEP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153366.3;VDB=0.0187;DN=116;DA=T/G;GM=NM_153366.3;GL=SVEP1;FG=missense;FD=missense;AAC=MET/LEU;PP=1444/3572;CDP=4330;GS=15;PH=0.002;CP=0.023;CG=3.620;AA=C;HA=44.7;HE=20.1;HC=37.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=69;TFBS=V$YY1_01;GESP=G:3078/T:9136;PAC=NP_699197.3 GT:GQ:PL 0/0:3:0,3,25 0/1:3:16,3,0 1/1:15:112,18,0 0/0:7:0,9,61</p>

<p>9 114359624 rs1053959 C A 122 highCVG AC1=3;AF1=0.3851;DB;DP=23;DP4=6,8,4,5;FQ=124;MQ=59;PV4=1,1,1,0.0013;SNPEFF_AMINO_ACID_CHANGE=A27S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTGR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146108.1;VDB=0.0165;DN=86;DA=C/A;GM=NM_001146108.1,NM_001146109.1,NM_012212.3;GL=PTGR1;FG=missense;FD=missense;AAC=ALA/SER;PP=27/330,27/302,27/330;CDP=79;GS=99;PH=0.019;CP=0.038;CG=-0.431;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=28;GESP=A:7644/C:5362;PAC=NP_001139580.1,NP_001139581.1,NP_036344.2 GT:GQ:PL 0/0:29:0,30,158 0/1:10:91,0,4 1/1:8:78,12,0 0/0:9:0,9,72</p>

<p>9 114464487 rs1407390 T C 45.60 lowCVG AC1=6;AF1=0.7452;DB;DP=8;DP4=0,1,0,6;FQ=-7.45;MQ=60;PV4=1,0.0038,1,0.36;SNPEFF_AMINO_ACID_CHANGE=Y932C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C9orf84;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173521.4;VDB=0.0037;DN=88;DA=T/C;GM=NM_001080551.1,NM_173521.3,XM_005251738.1;GL=C9orf84;FG=missense;FD=missense,missense,unknown;AAC=TYR/CYS;PP=893/1406,932/1445,996/1509;CDP=2678,2795,2987;GS=194;PH=0;CP=0.930;CG=-2.510;AA=C;HA=15.6;HE=21.5;HC=7.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=20;GESP=C:10539/T:2451;PAC=NP_001074020.1,NP_775792.3,XP_005251795.1 GT:GQ:PL 0/1:5:0,3,28 0/1:3:0,0,0 1/1:5:18,3,0 1/1:15:68,15,0</p>

<p>9 117050998 rs10982134 G A 55.90 lowCVG AC1=6;AF1=0.7419;DB;DP=6;DP4=1,0,3,1;FQ=-9.04;MQ=60;PV4=1,0.38,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R1354Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL27A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032888.2;VDB=0.0146;DN=120;DA=G/A;GM=NM_032888.2;GL=COL27A1;FG=missense;FD=missense;AAC=ARG/GLN;PP=1354/1861;CDP=4061;GS=43;PH=0.016;CP=0.999;CG=1.840;AA=G;HA=21.2;HE=25.9;HC=38.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=A:3908/G:9098;PAC=NP_116277.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,26 1/1:7:48,6,0 1/1:7:48,6,0</p>

<p>9 117846580 rs1061494 T C 25.30 midCVG AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1 GT:GQ:PL 0/1:22:18,0,120 0/1:3:0,0,0 1/1:6:35,6,0 0/1:15:12,0,20</p>

<p>9 123170733 rs4837768 C G 142 highCVG AC1=6;AF1=0.7872;DB;DP=17;DP4=3,0,5,5;FQ=-19.3;MQ=60;PV4=0.23,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=V1540L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDK5RAP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018249.5;VDB=0.0119;DN=111;DA=C/G;GM=NM_001011649.2,NM_001272039.1,NM_018249.5,NR_073554.1,NR_073555.1,NR_073556.1,NR_073557.1,NR_073558.1;GL=CDK5RAP2;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant;AAC=VAL/LEU,VAL/LEU,VAL/LEU,none,none,none,none,none;PP=1540/1815,1310/1664,1540/1894,NA,NA,NA,NA,NA;CDP=4618,3928,4618,4887,4810,5017,4890,4887;GS=32,32,32,NA,NA,NA,NA,NA;PH=0.105;CP=0.112;CG=-0.555;AA=C;HA=41.5;HE=27.5;HC=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:9047/C:3959;PPI=CDK5R1:845;PAC=NP_001011649.1,NP_001258968.1,NP_060719.4,NA,NA,NA,NA,NA GT:GQ:PL 0/1:4:27,0,4 0/1:10:31,0,12 1/1:6:21,3,0 1/1:17:105,15,0</p>

<p>9 123769200 rs17611 C T 77.90 midCVG AC1=6;AF1=0.677;DB;DP=11;DP4=4,0,5,2;FQ=41.7;MQ=60;PV4=0.49,0.026,1,0.15;SNPEFF_AMINO_ACID_CHANGE=V802I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001735.2;VDB=0.0065;DN=63;DA=C/T;GM=NM_001735.2;GL=C5;FG=missense;FD=missense;AAC=VAL/ILE;PP=802/1677;CDP=2404;GS=29;PH=0.127;CP=0.986;CG=1.060;AA=C;HA=0.0;HE=39.0;HC=41.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=19;KP=hsa04610:ComplementAndCoagulationCascades|hsa05020:PrionDiseases|hsa05322:SystemicLupusErythematosus;GESP=T:4225/C:8781;PPI=C6:983;PAC=NP_001726.2 GT:GQ:PL 0/1:4:0,3,24 0/1:17:11,0,58 1/1:12:71,12,0 1/1:7:41,6,0</p>

<p>9 123860689 rs10818504 C T 202 highCVG AC1=5;AF1=0.6874;DB;DP=18;DP4=1,2,13,2;FQ=9.15;MQ=60;PV4=0.11,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=P216L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007018.4;VDB=0.0128;DN=120;DA=C/T;GM=NM_007018.4,XM_005251679.1,XM_005251680.1;GL=CNTRL;FG=missense;FD=missense,unknown,unknown;AAC=PRO/LEU;PP=216/2326,216/2326,49/2159;CDP=647,647,146;GS=98;PH=0.998;CP=0.998;CG=4.730;AA=T;HA=8.7;HE=39.0;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=T:4541/C:8465;PAC=NP_008949.4,XP_005251736.1,XP_005251737.1 GT:GQ:PL 0/1:3:0,6,40 0/1:11:70,0,11 1/1:24:132,24,0 1/1:10:48,9,0</p>

<p>9 125273127 . G A 20 midCVG AC1=2;AF1=0.2748;DP=14;DP4=1,5,0,3;FQ=21.3;MQ=57;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G16D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1J2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054107.1;VDB=0.0008;GM=NM_054107.1,XM_005251920.1;GL=OR1J2;FG=missense;FD=unknown;AAC=GLY/ASP;PP=16/314;CDP=47;GS=94;PH=0.862;CP=0.998;CG=4.120;AA=G;DSP=60;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFKB_C;GESP=G:13006;PAC=NP_473448.1,XP_005251977.1 GT:GQ:PL 0/0:6:0,3,24 0/0:6:0,3,24 1/1:4:61,9,0 0/0:14:0,12,73</p>

<p>9 125315557 rs1831370 T C 80.20 highCVG AC1=4;AF1=0.548;DB;DP=15;DP4=2,5,4,3;FQ=74.2;MQ=60;PV4=0.59,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W37R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1N2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004457.1;VDB=0.0200;DN=92;DA=T/C;GM=NM_001004457.1;GL=OR1N2;FG=missense;FD=missense;AAC=TRP/ARG;PP=37/331;CDP=109;GS=101;PH=0;CP=0.002;CG=-2.620;AA=C;HA=41.0;HE=45.5;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42490;KP=hsa04740:OlfactoryTransduction;GESP=C:6740/T:6266;PAC=NP_001004457.1 GT:GQ:PL 0/1:3:0,3,19 0/0:5:0,7,98 1/1:12:96,15,0 1/1:3:30,3,0</p>

<p>9 125330125 rs10739614 C G 163 highCVG AC1=6;AF1=0.7385;DB;DP=20;DP4=0,5,11,3;FQ=29.2;MQ=60;PV4=0.0048,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=R211P;SNPEFF_CODON_CHANGE=cGt/cCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0083;DN=120;DA=C/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=ARG/PRO;PP=211/310;CDP=632;GS=103;PH=0;CP=0.008;CG=3.600;AA=G;HA=32.8;HE=45.0;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57058;KP=hsa04740:OlfactoryTransduction;GESP=G:8273/C:4733;PAC=NP_001004454.1 GT:GQ:PL 0/1:12:16,0,13 0/1:46:40,0,55 1/1:8:43,6,0 1/1:25:109,24,0</p>

<p>9 125330678 rs10985704 T G 52.30 midCVG AC1=5;AF1=0.6368;DB;DP=9;DP4=2,0,6,0;FQ=-3.17;MQ=60;PV4=1,0.0061,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T27P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0026;DN=120;DA=T/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=THR/PRO;PP=27/310;CDP=79;GS=38;PH=0;CP=0.733;CG=3.480;AA=G;HA=43.0;HE=49.1;HC=1.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57611;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:7411/T:5595;PAC=NP_001004454.1 GT:GQ:PL 0/1:4:0,3,25 0/1:14:37,0,13 1/1:4:20,3,0 1/1:6:38,6,0</p>

<p>9 125390874 rs1556189 A C 63.70 midCVG AC1=4;AF1=0.5408;DB;DP=11;DP4=0,5,0,6;FQ=48;MQ=60;PV4=1,0.4,1,1;SNPEFF_AMINO_ACID_CHANGE=V314G;SNPEFF_CODON_CHANGE=gTg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0169;DN=88;DA=A/C;GM=NM_001004450.1;GL=OR1B1;FG=missense;FD=missense;AAC=VAL/GLY;PP=314/319;CDP=941;GS=109;PH=0;CP=0.000;CG=-0.410;HA=39.3;HE=14.2;HC=4.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=117807;KP=hsa04740:OlfactoryTransduction;GESP=C:9912/A:3094;PAC=NP_001004450.1 GT:GQ:PL 0/1:57:56,0,58 0/0:3:0,3,26 1/1:5:47,6,0 0/1:3:0,0,0</p>

<p>9 125391026 rs1476858 A C 98.50 highCVG AC1=5;AF1=0.6085;DB;DP=20;DP4=2,9,7,2;FQ=103;MQ=60;PV4=0.022,0.0062,1,0.29;SNPEFF_AMINO_ACID_CHANGE=C263W;SNPEFF_CODON_CHANGE=tgT/tgG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0050;DN=88;DA=A/C;GM=NM_001004450.1;GL=OR1B1;FG=missense;FD=missense;AAC=CYS/TRP;PP=263/319;CDP=789;GS=215;PH=0;CP=0.921;CG=2.820;HA=42.5;HE=15.5;HC=6.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=117959;KP=hsa04740:OlfactoryTransduction;GESP=C:9234/A:3772;PAC=NP_001004450.1 GT:GQ:PL 0/0:8:0,12,90 0/1:19:14,0,103 1/1:6:46,6,0 1/1:14:87,15,0</p>

<p>9 125391241 rs1476860 G A 91.10 midCVG AC1=6;AF1=0.6853;DB;DP=9;DP4=2,1,1,5;FQ=19.3;MQ=60;PV4=0.23,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R192*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0081;DN=88;DA=G/A;GM=NM_001004450.1;GL=OR1B1;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=192/319;CDP=574;CP=0.142;CG=-6.050;AA=G;HA=3.8;HE=30.8;HC=46.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118174;KP=hsa04740:OlfactoryTransduction;GESP=A:2952/G:10054;PAC=NP_001004450.1 GT:GQ:PL 0/0:5:0,9,58 0/1:3:0,0,0 1/1:6:45,6,0 1/1:11:90,12,0</p>

<p>OR1B1 9 130165973 rs34064803 T C 79.40 highCVG AC1=3;AF1=0.4231;DB;DP=25;DP4=0,4,2,15;FQ=51.8;MQ=60;PV4=1,0.0002,1,0.08;SNPEFF_AMINO_ACID_CHANGE=I253T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC2A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014580.4;VDB=0.0144;DN=126;DA=T/C;GM=NM_001271711.1,NM_001271712.1,NM_014580.4,NR_073416.1,XM_005251949.1;GL=SLC2A8;FG=missense,missense,missense,non-coding-exon,missense;FD=missense,missense,missense,nc-transcript-variant,unknown;AAC=ILE/THR,ILE/THR,ILE/THR,none,ILE/THR;PP=253/412,90/315,253/478,NA,90/315;CDP=758,269,758,284,269;GS=89,89,89,NA,89;PH=0.878;CP=0.927;CG=5.420;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=C:1021/T:11985;PAC=NP_001258640.1,NP_001258641.1,NP_055395.2,NA,XP_005252006.1 GT:GQ:PL 0/0:6:0,6,47 0/1:3:0,0,0 1/1:46:122,51,0 0/0:6:0,6,45</p>

<p>9 134401335 rs2296957 C T 63.50 midCVG AC1=4;AF1=0.5532;DB;DP=9;DP4=1,2,1,5;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A180T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135954.1;VDB=0.0243;DN=100;DA=C/A/T;GM=NM_001135954.1,NM_001261450.1,NM_001261451.1,NM_031432.2,XM_005272224.1,XM_005272225.1;GL=UCK1;FG=missense,synonymous,synonymous,synonymous,synonymous,synonymous;FD=missense,synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown;AAC=ALA/THR,PRO,PRO,PRO,PRO,PRO;PP=180/202,202/269,216/283,211/278,225/292,188/255;CDP=538,606,648,633,675,564;GS=58,NA,NA,NA,NA,NA;PH=0.104;CP=0.908;CG=-5.080;AA=C;HA=41.2;HE=8.1;HC=7.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=20;KP=hsa00240:PyrimidineMetabolism|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;TFBS=V$SREBP1_02;GESP=T:10799/C:2207;PAC=NP_001129426.1,NP_001248379.1,NP_001248380.1,NP_113620.1,XP_005272281.1,XP_005272282.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,60 1/1:15:103,18,0 0/1:3:0,0,0</p>

<p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

<p>9 137779091 rs7851696 G T 152 highCVG AC1=4;AF1=0.5192;DB;DP=19;DP4=3,4,3,9;FQ=78.5;MQ=60;PV4=0.62,0.15,1,0.038;SNPEFF_AMINO_ACID_CHANGE=A258S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FCN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004108.2;VDB=0.0132;DN=116;DA=G/T;GM=NM_004108.2,NM_015837.2;GL=FCN2;FG=missense;FD=missense;AAC=ALA/SER;PP=258/314,220/276;CDP=772,658;GS=99;PH=0.946;CP=0.077;CG=4.050;AA=G;HA=17.6;HE=11.0;HC=16.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=78;GESP=T:1769/G:11237;PAC=NP_004099.2,NP_056652.1 GT:GQ:PL 0/1:10:77,0,6 0/0:5:0,6,49 1/1:5:45,6,0 0/1:67:74,0,64</p>

<p>9 138376972 rs3928777 A G 77.10 midCVG AC1=6;AF1=0.6853;DB;DP=9;DP4=0,3,0,6;FQ=11.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K206E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014811.3;VDB=0.0151;DN=108;DA=A/G;GM=NM_014811.3,XM_005263411.1,XM_005263412.1,XM_005263413.1,XM_005263414.1,XM_005263415.1;GL=PPP1R26;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=206/1210;CDP=616;GS=56;PH=0;CP=0.000;CG=-4.970;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=678;GESP=G:2902/A:10090;PAC=NP_055626.3,XP_005263468.1,XP_005263469.1,XP_005263470.1,XP_005263471.1,XP_005263472.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,50 1/1:11:80,12,0 1/1:6:41,6,0</p>

<p>9 139118673 rs12380852 T C 17.20 midCVG AC1=3;AF1=0.3906;DB;DP=12;DP4=3,5,1,2;FQ=19.3;MQ=60;PV4=1,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K126E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=QSOX2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181701.3;VDB=0.0103;DN=120;DA=T/C;GM=NM_181701.3;GL=QSOX2;FG=missense;FD=missense;AAC=LYS/GLU;PP=126/699;CDP=376;GS=56;PH=0.002;CP=0.615;CG=2.180;AA=T;HA=4.2;HE=7.5;HC=25.8;DG;DV=by-frequency,by-cluster,by-1000G;DSP=48;GESP=C:1203/T:11803;PAC=NP_859052.3 GT:GQ:PL 0/0:9:0,9,60 0/1:4:26,3,0 1/1:3:37,6,0 0/0:15:0,15,95</p>

<p>9 140201604 rs28685844 C T 15.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=3,0,1,1;FQ=15;MQ=60;PV4=0.4,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R810Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017820.4;VDB=0.0130;DN=125;DA=C/T;GM=NM_017820.3,XM_005266091.1,XM_005266092.1;GL=EXD3;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=810/877,490/557,448/515;CDP=2429,1469,1343;GS=43;PH=0.193;CP=0.000;CG=-3.990;AA=C;HA=3.6;HE=0.4;HC=8.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=112;GESP=T:123/C:12191;PAC=NP_060290.3,XP_005266148.1,XP_005266149.1 GT:GQ:PL 0/1:3:0,0,0 0/0:9:0,9,51 1/1:3:52,6,0 0/1:3:0,0,0</p>

<p>10 1065457 rs41314629 T A 60.60 lowCVG AC1=5;AF1=0.5962;DB;DP=7;DP4=2,0,4,1;FQ=10.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*228C;SNPEFF_CODON_CHANGE=tgA/tgT;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IDI2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_033261.2;VDB=0.0111;DN=127;DA=T/A;GM=NM_033261.2;GL=IDI2;FG=stop-lost;FD=stop-lost;AAC=stop/CYS;PP=228/228;CDP=684;CP=0.004;CG=3.300;AA=T;DV=by-frequency,by-cluster;DSP=318;KP=hsa00900:TerpenoidBackboneBiosynthesis|hsa01100:MetabolicPathways;GESP=A:847/T:12159;PAC=NP_150286.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:12:99,15,0 0/0:5:0,6,45</p>

<p>10 5415954 rs10904481 A G 49.50 lowCVG AC1=6;AF1=0.685;DB;DP=8;DP4=0,3,1,4;FQ=13.7;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R91G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053049.2;VDB=0.0100;DN=120;DA=A/G;GM=NM_053049.2;GL=UCN3;FG=missense;FD=missense;AAC=ARG/GLY;PP=91/162;CDP=271;GS=125;PH=0;CP=0.000;CG=2.520;AA=G;HA=29.2;HE=35.7;HC=48.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=277;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4033/A:8969;PAC=NP_444277.2 GT:GQ:PL 0/1:17:11,0,30 0/1:19:16,0,21 1/1:7:46,6,0 1/1:5:19,3,0</p>

<p>10 16979714 rs41289305 T C 71.40 highCVG AC1=5;AF1=0.6228;DB;DP=16;DP4=5,2,7,1;FQ=70;MQ=60;PV4=0.57,0.03,1,0.14;SNPEFF_AMINO_ACID_CHANGE=S1935G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=39;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUBN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081.3;VDB=0.0066;DN=127;DA=T/C;GM=NM_001081.3;GL=CUBN;FG=missense;FD=missense;AAC=SER/GLY;PP=1935/3624;CDP=5803;GS=56;PH=0.507;CP=0.136;CG=0.044;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=C:1887/T:11119;PPI=GIF:983;PAC=NP_001072.2 GT:GQ:PL 0/1:8:3,0,76 0/1:3:0,0,0 1/1:8:48,9,0 0/1:33:60,0,31</p>

<p>10 24669877 rs17506606 C G 29.60 lowCVG AC1=4;AF1=0.5278;DB;DP=8;DP4=2,2,0,4;FQ=32.8;MQ=60;PV4=0.43,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=A145G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1217;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019590.4;VDB=0.0046;DN=123;DA=C/G;GM=NM_001098500.1,NM_001098501.1,NM_019590.3,XM_005252500.1,XM_005252501.1,XM_005252502.1,XM_005252503.1,XM_005252504.1,XM_005252505.1,XM_005252506.1,XM_005252507.1,XM_005252508.1,XM_005252509.1,XM_005252510.1,XM_005252511.1,XM_005252512.1,XM_005252513.1,XM_005252517.1,XM_005252518.1,XM_005252519.1,XM_005252520.1,XM_005252521.1,XM_005252522.1,XM_005252523.1,XM_005252524.1;GL=KIAA1217;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,none,none,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY;PP=65/1265,145/1310,145/1944,172/1983,172/1982,172/1971,145/1956,172/1948,172/1936,172/1931,145/1921,145/1909,28/1839,172/1822,92/1903,NA,NA,172/1450,172/1412,145/1375,172/1372,145/1350,145/1345,172/1242,172/1176;CDP=194,434,434,515,515,515,434,515,515,515,434,434,83,515,275,NA,NA,515,515,434,515,434,434,515,515;GS=60,60,60,60,60,60,60,60,60,60,60,60,60,60,60,NA,NA,60,60,60,60,60,60,60,60;PH=0.003;CP=0.008;CG=-0.211;AA=C;HA=0.0;HE=26.6;HC=0.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=80;GESP=G:2468/C:10538;PAC=NP_001091970.1,NP_001091971.1,NP_062536.2,XP_005252557.1,XP_005252558.1,XP_005252559.1,XP_005252560.1,XP_005252561.1,XP_005252562.1,XP_005252563.1,XP_005252564.1,XP_005252565.1,XP_005252566.1,XP_005252567.1,XP_005252568.1,NA,NA,XP_005252574.1,XP_005252575.1,XP_005252576.1,XP_005252577.1,XP_005252578.1,XP_005252579.1,XP_005252580.1,XP_005252581.1 GT:GQ:PL 0/1:3:0,0,0 0/1:20:17,0,42 1/1:7:54,9,0 0/0:5:0,6,46</p>

<p>10 48416456 . T C 53.30 highCVG AC1=3;AF1=0.3888;DP=15;DP4=0,5,0,6;FQ=54.1;MQ=60;PV4=1,1,1,0.0011;SNPEFF_AMINO_ACID_CHANGE=K80E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016204.2;VDB=0.0215;GM=NM_016204.1;GL=GDF2;FG=missense;FD=unknown;AAC=LYS/GLU;PP=80/430;CDP=238;GS=56;PH=0.684;CP=0.393;CG=4.220;AA=T;DSP=109;GESP=T:13006;PPI=ACVRL1:900|ACVR2B:900;PAC=NP_057288.1 GT:GQ:PL 0/0:7:0,6,36 0/0:9:0,9,64 1/1:13:96,18,0 0/1:3:0,0,0</p>

<p>10 69933969 rs7916821 G A 187 highCVG AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA GT:GQ:PL 0/1:45:68,0,45 0/1:5:0,3,28 1/1:15:86,15,0 1/1:13:80,12,0</p>

<p>10 70641860 rs1341667 T C 76 midCVG AC1=5;AF1=0.6593;DB;DP=10;DP4=1,2,3,4;FQ=26.9;MQ=60;PV4=1,0.21,1,0.26;SNPEFF_AMINO_ACID_CHANGE=Y153H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130161.2;VDB=0.0014;DN=88;DA=T/C;GM=NM_001130159.2,NM_001130160.2,NM_001130161.2,NM_001130162.2,NM_152709.4,XM_005269605.1;GL=STOX1;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=TYR/HIS;PP=153/228,153/170,153/990,43/880,153/990,43/880;CDP=457,457,457,127,457,127;GS=83;PH=0.687;CP=0.990;CG=4.440;AA=T;HA=33.8;HE=38.1;HC=11.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=7;TFBS=V$HNF1_C;GESP=C:6772/T:5074;PAC=NP_001123631.1,NP_001123632.1,NP_001123633.1,NP_001123634.1,NP_689922.3,XP_005269662.1 GT:GQ:PL 0/0:6:0,9,65 0/1:3:0,0,0 1/1:17:106,18,0 1/1:4:13,3,0</p>

<p>10 84733562 . C T 20.10 midCVG AC1=3;AF1=0.3373;DP=11;DP4=6,1,3,0;FQ=21.8;MQ=60;PV4=1,0.18,1,0.066;SNPEFF_AMINO_ACID_CHANGE=H435Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010848.3;VDB=0.0000;GM=NM_001010848.3,NM_001165972.1,NM_001165973.1,XM_005269444.1,XM_005269445.1,XM_005269446.1,XM_005269447.1,XM_005269448.1,XM_005269449.1,XM_005269450.1,XM_005269451.1;GL=NRG3;FG=missense;FD=unknown;AAC=HIS/TYR;PP=435/697,434/696,214/500,435/721,265/527,239/525,239/501,214/476,171/457,171/433,85/371;CDP=1303,1300,640,1303,793,715,715,640,511,511,253;GS=83;PH=0.952;CP=1.000;CG=5.050;AA=C;DSP=19;KP=hsa04012:ErbBSignalingPathway;TFBS=V$CETS1P54_01;GESP=C:13006;PAC=NP_001010848.2,NP_001159444.1,NP_001159445.1,XP_005269501.1,XP_005269502.1,XP_005269503.1,XP_005269504.1,XP_005269505.1,XP_005269506.1,XP_005269507.1,XP_005269508.1 GT:GQ:PL 0/0:5:0,3,24 0/0:19:0,18,100 1/1:4:60,9,0 0/0:3:0,0,0</p>

<p>10 85991784 rs3814211 T C 28.60 midCVG AC1=2;AF1=0.3082;DB;DP=11;DP4=2,5,0,3;FQ=30.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S591G;SNPEFF_CODON_CHANGE=Agt/Ggt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRIT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015613.2;VDB=0.0116;DN=107;DA=T/C;GM=NM_015613.2;GL=LRIT1;FG=missense;FD=missense;AAC=SER/GLY;PP=591/624;CDP=1771;GS=56;PH=0.242;CP=0.475;CG=2.790;AA=T;HA=0.0;HE=3.8;HC=2.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=876;GESP=C:495/T:12511;PAC=NP_056428.1 GT:GQ:PL 0/0:8:0,6,42 0/0:3:0,0,0 1/1:4:69,9,0 0/0:17:0,15,113</p>

<p>10 90356598 rs1409136 A G 55 midCVG AC1=3;AF1=0.3912;DB;DP=14;DP4=7,2,2,3;FQ=57.1;MQ=60;PV4=0.27,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=I210V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010939.2;VDB=0.0121;DN=88;DA=A/G;GM=NM_001010939.2,XM_005269542.1;GL=LIPJ;FG=missense;FD=missense,unknown;AAC=ILE/VAL;PP=210/367,97/254;CDP=628,289;GS=29;PH=0;CP=0.000;CG=-8.230;AA=A;HA=1.1;HE=28.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=52;GESP=G:10260/A:2734;PAC=NP_001010939.2,XP_005269599.1 GT:GQ:PL 0/0:4:0,3,26 0/0:20:0,21,97 1/1:5:58,9,0 0/1:21:42,0,16</p>

<p>10 91007360 rs1051338 T G 24.80 lowCVG AC1=4;AF1=0.5432;DB;DP=5;DP4=1,1,0,3;FQ=4.88;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T16P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000235.3;VDB=0.0127;DN=86;DA=T/G;GM=NM_000235.2,NM_001127605.1;GL=LIPA;FG=missense;FD=missense;AAC=THR/PRO;PP=16/400;CDP=46;GS=38;PH=0.002;CP=0.997;CG=3.720;AA=T;HA=8.8;HE=33.0;HC=28.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=47;KP=hsa00100:SteroidBiosynthesis|hsa04142:Lysosome;GESP=G:3134/T:9872;PAC=NP_000226.2,NP_001121077.1 GT:GQ:PL 0/1:19:20,0,18 0/1:3:0,0,0 1/1:5:44,6,0 0/0:3:0,3,21</p>

<p>10 91488979 rs3758388 A T 121 highCVG AC1=4;AF1=0.546;DB;DP=19;DP4=2,7,3,7;FQ=95.9;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=H789L;SNPEFF_CODON_CHANGE=cAt/cTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284259.1;VDB=0.0119;DN=107;DA=A/T;GM=NM_016195.2,XM_005270290.1;GL=KIF20B;FG=missense;FD=missense,unknown;AAC=HIS/LEU;PP=749/1781,789/1821;CDP=2246,2366;GS=99;PH=0.069;CP=0.001;CG=0.696;AA=A;HA=2.9;HE=26.2;HC=39.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=T:1992/A:10770;PAC=NP_057279.2,XP_005270347.1 GT:GQ:PL 0/1:65:62,0,80 0/0:4:0,6,50 1/1:10:77,12,0 0/1:11:28,0,8</p>

<p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

<p>10 102265183 rs2295772 G A 205 highCVG AC1=5;AF1=0.6183;DB;DP=23;DP4=2,4,6,8;FQ=50.4;MQ=60;PV4=1,1,0.26,0.38;SNPEFF_AMINO_ACID_CHANGE=P372S;SNPEFF_CODON_CHANGE=Cca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC31B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015490.3;VDB=0.0054;DN=100;DA=G/A;GM=NM_015490.3,XM_005269676.1,XM_005269677.1,XM_005269678.1,XM_005269679.1;GL=SEC31B;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=PRO/SER,PRO/SER,PRO/SER,PRO/SER,none;PP=372/1180,372/1263,372/1263,215/1106,NA;CDP=1114,1114,1114,643,NA;GS=74,74,74,74,NA;PH=0;CP=0.368;CG=3.470;AA=A;HA=14.0;HE=18.7;HC=16.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=66;TFBS=V$ZID_01;GESP=A:2751/G:10255;PPI=SF3A2:944;PAC=NP_056305.1,XP_005269733.1,XP_005269734.1,XP_005269735.1,NA GT:GQ:PL 0/1:18:44,0,17 0/0:8:0,12,78 1/1:6:52,6,0 1/1:26:158,27,0</p>

<p>10 105657316 rs10786775 G C 120 midCVG AC1=6;AF1=0.7314;DB;DP=12;DP4=0,3,4,4;FQ=10.4;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S248C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBFC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024928.4;VDB=0.0047;DN=120;DA=G/C;GM=NM_024928.4;GL=OBFC1;FG=missense;FD=missense;AAC=SER/CYS;PP=248/369;CDP=743;GS=112;PH=0;CP=0.000;CG=1.390;AA=C;HA=41.6;HE=7.3;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=11;GESP=C:10702/G:2304;PAC=NP_079204.2 GT:GQ:PL 0/1:22:15,0,32 0/1:16:23,0,17 1/1:10:66,9,0 1/1:10:61,9,0</p>

<p>10 115529576 rs41292632 T A 52.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=1,1,4,1;FQ=13.3;MQ=60;PV4=1,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=F151I;SNPEFF_CODON_CHANGE=Ttc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLEKHS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182601.1;VDB=0.0055;DN=127;DA=T/A;GM=NM_001193434.1,NM_001193435.1,NM_024889.4,NM_182601.1,XM_005270162.1,XM_005270163.1;GL=PLEKHS1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown;AAC=PHE/ILE;PP=69/367,69/367,157/364,151/466,157/469,157/455;CDP=205,205,469,451,469,469;GS=21;PH=0.893;CP=0.009;CG=4.760;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=34;TFBS=V$PAX4_04;GESP=A:1962/T:11044;PAC=NP_001180363.1,NP_001180364.1,NP_079165.3,NP_872407.1,XP_005270219.1,XP_005270220.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,50 1/1:6:35,6,0 1/1:9:60,9,0</p>

<p>10 126678087 rs111888382 C T 16.70 highCVG AC1=6;AF1=0.6827;DB;DP=36;DP4=3,5,13,14;FQ=21.4;MQ=11;PV4=0.7,0.29,1.9e-05,0.13;SNPEFF_AMINO_ACID_CHANGE=*986;SNPEFF_CODON_CHANGE=taG/taA;SNPEFF_EFFECT=SYNONYMOUS_STOP;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0149;DN=132;DA=C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=synonymous;FD=synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=stop;PP=446/446,446/446,986/986,514/514,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,421/421,421/421;CDP=1338,1338,2958,1542,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1263,1263;CP=1.000;CG=5.120;AA=C;DV=by-cluster;DSP=181;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:4:0,3,38 1/1:11:16,12,0 1/1:30:45,31,0</p>

<p>10 126678092 rs73373169 G A 25.10 highCVG AC1=5;AF1=0.6731;DB;DP=36;DP4=2,5,12,13;FQ=29.5;MQ=12;PV4=0.43,1,1.3e-05,0.07;SNPEFF_AMINO_ACID_CHANGE=Q985*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0155;DN=131;DA=G/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=stop-gained;FD=stop-gained,stop-gained,stop-gained,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/stop;PP=445/446,445/446,985/986,513/514,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,420/421,420/421;CDP=1333,1333,2953,1537,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1258,1258;CP=1.000;CG=5.120;AA=G;DG;DV=by-cluster;DSP=176;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$MYB_Q6;GESP=G:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:3:0,5,41 1/1:17:22,18,0 1/1:28:49,28,0</p>

<p>CTBP2 10 126678177 . G T 39.20 highCVG AC1=6;AF1=0.7163;DP=34;DP4=3,4,8,17;FQ=44.8;MQ=28;PV4=0.67,0.00015,8e-15,1;SNPEFF_AMINO_ACID_CHANGE=N956K;SNPEFF_CODON_CHANGE=aaC/aaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0013;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=unknown;AAC=ASN/LYS;PP=416/446,416/446,956/986,484/514,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,391/421,391/421;CDP=1248,1248,2868,1452,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1173,1173;GS=94;PH=0.613;CP=1.000;CG=3.120;AA=G;DSP=91;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$BACH1_01,V$TAXCREB_02;GESP=G:12998;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:5:1,0,7 0/1:4:0,4,98 1/1:24:24,24,0 1/1:33:60,33,0</p>

<p>10 126678221 rs77752853 C A 43.30 highCVG AC1=6;AF1=0.7316;DB;DP=30;DP4=3,4,3,17;FQ=49.1;MQ=30;PV4=0.29,0.031,2.4e-12,0.15;SNPEFF_AMINO_ACID_CHANGE=A942S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=131;DA=C/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/SER;PP=402/446,402/446,942/986,470/514,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,377/421,377/421;CDP=1204,1204,2824,1408,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1129,1129;GS=99;PH=0.599;CP=0.998;CG=5.120;AA=C;DV=by-cluster;DSP=47;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:6:0,1,11 0/1:11:4,0,114 1/1:21:24,21,0 1/1:25:60,24,0</p>

<p>10 126678241 rs113985797 G C 8.10 highCVG AC1=4;AF1=0.4981;DB;DP=26;DP4=4,7,2,13;FQ=10.6;MQ=39;PV4=0.35,1.5e-07,2.9e-13,1;SNPEFF_AMINO_ACID_CHANGE=A935G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=132;DA=G/C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=395/446,395/446,935/986,463/514,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,370/421,370/421;CDP=1184,1184,2804,1388,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1109,1109;GS=60;PH=0.016;CP=1.000;CG=5.120;AA=G;DV=by-frequency,by-cluster;DSP=27;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/0:4:0,2,39 0/0:6:0,6,121 1/1:10:17,18,0 0/1:10:34,0,4</p>

<p>10 129903016 rs7071768 T C 19.60 midCVG AC1=5;AF1=0.5991;DB;DP=9;DP4=2,1,1,2;FQ=16.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N2363S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MKI67;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002417.4;VDB=0.0101;DN=116;DA=T/C;GM=NM_001145966.1,NM_002417.4,XM_005252683.1;GL=MKI67;FG=missense;FD=missense,missense,unknown;AAC=ASN/SER;PP=2003/2897,2363/3257,2166/3060;CDP=6008,7088,6497;GS=46;PH=0;CP=0.000;CG=-5.360;AA=C;HA=41.1;HE=46.0;HC=38.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2174;GESP=C:6499/T:6507;PPI=MKI67IP:983|SUMO2:748;PAC=NP_001139438.1,NP_002408.3,XP_005252740.1 GT:GQ:PL 0/1:18:19,0,17 0/1:18:14,0,35 1/1:3:25,3,0 0/1:3:0,0,0</p>

<p>11 1507731 rs1881503 A G 14.10 lowCVG AC1=4;AF1=0.5387;DB;DP=5;DP4=1,1,2,1;FQ=9.13;MQ=60;PV4=1,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=L19P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MOB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172223.2;VDB=0.0097;DN=92;DA=A/G;GM=NM_001172223.1;GL=MOB2;FG=missense;FD=missense;AAC=LEU/PRO;PP=19/269;CDP=56;GS=98;PH=0;CP=0.823;CG=3.930;AA=G;HA=0.0;HE=12.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;PAC=NP_001165694.1 GT:GQ:PL 0/0:3:0,3,21 0/1:3:16,3,0 1/1:4:40,6,0 0/0:3:0,3,27</p>

<p>11 4870261 rs11602499 G C 80.20 midCVG AC1=5;AF1=0.6315;DB;DP=12;DP4=3,1,6,1;FQ=55.7;MQ=59;PV4=1,0.086,0.24,1;SNPEFF_AMINO_ACID_CHANGE=Q60E;SNPEFF_CODON_CHANGE=Cag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR51S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004758.1;VDB=0.0002;DN=120;DA=G/C;GM=NM_001004758.1;GL=OR51S1;FG=missense;FD=missense;AAC=GLN/GLU;PP=60/324;CDP=178;GS=29;PH=0;CP=0.205;CG=4.850;AA=C;HA=12.2;HE=36.9;HC=2.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=139280;KP=hsa04740:OlfactoryTransduction;GESP=C:4937/G:8061;PAC=NP_001004758.1 GT:GQ:PL 0/0:8:0,12,95 0/1:3:0,0,0 1/1:4:24,3,0 1/1:17:100,18,0</p>

<p>11 4945233 rs35264256 C T 27.60 lowCVG AC1=5;AF1=0.6618;DB;DP=7;DP4=1,2,2,2;FQ=7.91;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=E113K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR51G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005237.1;VDB=0.0141;DN=126;DA=C/T;GM=NM_001005237.1;GL=OR51G1;FG=missense;FD=missense;AAC=GLU/LYS;PP=113/322;CDP=337;GS=56;PH=1;CP=0.508;CG=4.200;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=64308;KP=hsa04740:OlfactoryTransduction;TFBS=V$AP1_Q6;GESP=T:1207/C:11791;PAC=NP_001005237.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 0/1:39:49,0,37</p>

<p>11 5528918 rs2227271 C T 10.70 lowCVG AC1=5;AF1=0.5785;DB;DP=5;DP4=0,1,2,0;FQ=-6.4;MQ=60;PV4=0.33,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=R624Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBQLN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017481.2;VDB=0.0061;DN=98;DA=C/T;GM=NM_017481.2;GL=UBQLN3;FG=missense;FD=missense;AAC=ARG/GLN;PP=624/656;CDP=1871;GS=43;PH=0.045;CP=0.213;CG=1.230;AA=C;HA=6.4;HE=37.7;HC=6.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1907;GESP=T:3709/C:9287;PAC=NP_059509.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:45,6,0 0/0:4:0,3,26</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>11 5906205 rs4757987 G A 63.50 midCVG AC1=4;AF1=0.4597;DB;DP=12;DP4=3,4,2,3;FQ=66.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R228H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52E4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005165.1;VDB=0.0135;DN=111;DA=G/A;GM=NM_001005165.1;GL=OR52E4;FG=missense;FD=missense;AAC=ARG/HIS;PP=228/313;CDP=683;GS=29;PH=0.003;CP=0.023;CG=0.047;AA=G;HA=47.5;HC=42.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=64694;KP=hsa04740:OlfactoryTransduction;GESP=A:5225/G:7769;PAC=NP_001005165.1 GT:GQ:PL 0/0:6:0,6,44 0/1:4:25,3,0 1/1:8:86,12,0 0/0:13:0,15,95</p>

<p>11 6007613 rs4354673 T G 168 highCVG AC1=6;AF1=0.6957;DB;DP=18;DP4=0,3,5,9;FQ=4.33;MQ=48;PV4=0.51,1,0.054,1;SNPEFF_AMINO_ACID_CHANGE=K183T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0083;DN=111;DA=T/G;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=LYS/THR;PP=183/330;CDP=548;GS=78;PH=0.549;CP=0.001;CG=-0.167;AA=T;HA=44.5;HE=13.2;HC=10.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=166102;KP=hsa04740:OlfactoryTransduction;GESP=G:8663/T:3675;PAC=NP_001005173.2 GT:GQ:PL 0/1:11:38,0,11 0/1:3:0,6,34 1/1:15:87,15,0 1/1:15:91,15,0</p>

<p>11 6007680 rs4436524 A G 189 highCVG AC1=6;AF1=0.7132;DB;DP=17;DP4=0,2,8,7;FQ=-4.85;MQ=47;PV4=0.47,0.4,0.081,0.3;SNPEFF_AMINO_ACID_CHANGE=C161R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0140;DN=111;DA=A/G/T;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=CYS/ARG;PP=161/330;CDP=481;GS=180;PH=0;CP=0.012;CG=0.241;AA=G;HA=43.3;HE=15.3;HC=11.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=166169;KP=hsa04740:OlfactoryTransduction;GESP=G:9200/A:3246;PAC=NP_001005173.2 GT:GQ:PL 0/1:16:39,0,17 0/1:5:0,3,21 1/1:13:53,12,0 1/1:28:144,27,0</p>

<p>11 6341684 rs2682123 C G 35.60 lowCVG AC1=6;AF1=0.7558;DB;DP=5;DP4=0,1,2,2;FQ=-21.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R8P;SNPEFF_CODON_CHANGE=cGg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRKCDBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145040.2;VDB=0.0113;DN=100;DA=C/G;GM=NM_145040.2;GL=PRKCDBP;FG=missense;FD=missense;AAC=ARG/PRO;PP=8/262;CDP=23;GS=103;PH=0;CP=0.004;CG=2.010;AA=G;HA=13.6;HE=1.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=362;CPG=6340446-6341909;GESP=G:11875/C:705;PAC=NP_659477.2 GT:GQ:PL 0/1:10:28,0,9 0/1:3:0,0,0 1/1:6:43,6,0 0/1:3:0,0,0</p>

<p>11 6816137 . T A 48.90 midCVG AC1=5;AF1=0.5835;DP=13;DP4=1,1,5,0;FQ=-7.18;MQ=60;PV4=0.29,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=K268M;SNPEFF_CODON_CHANGE=aAg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003696.2;VDB=0.0075;GM=NM_003696.2;GL=OR6A2;FG=missense;FD=unknown;AAC=LYS/MET;PP=268/328;CDP=803;GS=95;PH=0.999;CP=0.995;CG=3.850;AA=T;DSP=111784;KP=hsa04740:OlfactoryTransduction;GESP=T:12994;PAC=NP_003687.2 GT:GQ:PL 0/1:20:34,0,17 0/1:3:0,0,1/1:7:55,9,0 0/1:3:0,3,13</p>

<p>11 17593709 rs7106548 T C 84.60 midCVG AC1=4;AF1=0.5314;DB;DP=11;DP4=3,1,2,5;FQ=47;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S692P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0167;DN=116;DA=T/C;GM=NM_001277269.1;GL=OTOG;FG=missense;FD=missense;AAC=SER/PRO;PP=692/2926;CDP=2074;GS=74;CP=0.003;CG=-4.210;AA=C;HA=30.5;HE=39.7;HC=22.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=43;PAC=NP_001264198.1 GT:GQ:PL 0/1:17:15,0,19 0/0:7:0,9,68 1/1:15:113,18,0 0/1:3:0,0,0</p>

<p>11 17632651 rs7111528 C T 102 midCVG AC1=5;AF1=0.5888;DB;DP=10;DP4=1,2,4,3;FQ=15.1;MQ=60;PV4=1,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=T1947M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0076;DN=116;DA=C/T;GM=NM_001277269.1,XM_005252894.1;GL=OTOG;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=1947/2926,953/1509;CDP=5840,2858;GS=81;PH=0.011;CP=0.000;CG=-0.815;AA=T;HA=48.7;HE=42.3;HC=17.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=320;PAC=NP_001264198.1,XP_005252951.1 GT:GQ:PL 0/0:4:0,6,33 0/1:3:0,3,25 1/1:5:50,6,0 1/1:13:99,15,0</p>

<p>11 18735947 rs7125943 T C 65 lowCVG AC1=4;AF1=0.4853;DB;DP=7;DP4=1,1,3,2;FQ=17.9;MQ=60;PV4=1,0.094,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M559V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGSF22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173588.3;VDB=0.0024;DN=116;DA=T/C;GM=NM_173588.3;GL=IGSF22;FG=missense;FD=missense;AAC=MET/VAL;PP=559/1327;CDP=1675;GS=21;PH=0;CP=0.001;CG=-2.120;AA=C;HA=15.3;HE=22.1;HC=20.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=19;GESP=C:9686/T:2872;PAC=NP_775859.3 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,25 1/1:11:105,15,0 0/0:4:0,3,30</p>

<p>11 20065673 rs3802799 G T 174 highCVG AC1=5;AF1=0.6294;DB;DP=19;DP4=1,3,8,7;FQ=19.4;MQ=60;PV4=0.58,0.057,1,1;SNPEFF_AMINO_ACID_CHANGE=E1041D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NAV2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001244963.1;VDB=0.0108;DN=107;DA=G/T;GM=NM_001111018.1,NM_001111019.2,NM_001244963.1,NM_145117.4,NM_182964.5,XM_005253212.1,XM_005253213.1,XM_005253214.1,XM_005253215.1;GL=NAV2;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=GLU/ASP;PP=954/2366,104/1494,1041/2489,1018/2430,1018/2433,104/1519,104/1494,104/1482,17/1432;CDP=2862,312,3123,3054,3054,312,312,312,51;GS=45;PH=0.999;CP=0.994;CG=4.500;AA=G;HA=0.3;HE=14.8;HC=49.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=146;GESP=T:1921/G:11085;PAC=NP_001104488.1,NP_001104489.1,NP_001231892.1,NP_660093.2,NP_892009.3,XP_005253269.1,XP_005253270.1,XP_005253271.1,XP_005253272.1 GT:GQ:PL 0/1:36:64,0,35 0/1:4:0,3,23 1/1:4:21,3,0 1/1:6:132,6,0</p>

<p>11 28057837 . C G 72.40 midCVG AC1=5;AF1=0.6429;DP=14;DP4=2,4,5,1;FQ=57.8;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D775H;SNPEFF_CODON_CHANGE=Gac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF18A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031217.3;VDB=0.0011;GM=NM_031217.3;GL=KIF18A;FG=missense;FD=unknown;AAC=ASP/HIS;PP=775/899;CDP=2323;GS=81;PH=0.664;CP=0.965;CG=4.630;AA=C;DSP=74;GESP=C:13000;PAC=NP_112494.3 GT:GQ:PL 0/1:16:11,0,67 0/1:3:0,0,0 1/1:8:65,9,0 0/1:28:37,0,27</p>

<p>11 47270255 rs2167079 C T 15.90 midCVG AC1=3;AF1=0.4192;DB;DP=11;DP4=4,2,1,2;FQ=18.3;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R29Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001610.2;VDB=0.0055;DN=96;DA=C/T;GM=NM_001131064.1,NM_001251934.1,NM_001251935.1,NM_001610.2,XM_005252982.1,XM_005252983.1,XM_005252984.1,XM_005252985.1;GL=ACP2/NR1H3;FG=missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,5-prime-UTR,missense;FD=missense,utr-variant-5-prime,utr-variant-5-prime,missense,unknown,unknown,unknown,unknown;AAC=ARG/GLN,none,none,ARG/GLN,ARG/GLN,ARG/GLN,none,ARG/GLN;PP=29/161,NA,NA,29/424,29/392,29/361,NA,29/218;CDP=86,NA,NA,86,86,86,NA,86;GS=43,NA,NA,43,43,43,NA,43;PH=0.003;CP=0.987;CG=5.690;AA=T;HA=38.3;HE=31.2;HC=27.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;KP=hsa00740:RiboflavinMetabolism|hsa04142:Lysosome|hsa03320:PPARSignalingPathway;CPG=47270235-47270633;TFBS=V$AHR_01;GESP=T:4930/C:8048;PPI=RXRA:937|RXRB:900;PAC=NP_001124536.1,NA,NA,NP_001601.1,XP_005253039.1,XP_005253040.1,NA,XP_005253042.1 GT:GQ:PL 0/1:4:24,3,0 0/0:6:0,6,30 1/1:3:37,6,0 0/0:11:0,12,67</p>

<p>11 48346547 rs79042268 C T 999 highCVG AC1=5;AF1=0.5818;DB;DP=30;DP4=1,10,12,7;FQ=67.7;MQ=60;PV4=0.0067,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=P19S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0115;DN=131;DA=C/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=PRO/SER;PP=19/330;CDP=55;GS=74;CP=0.000;CG=-5.180;AA=C;DG;DV=by-cluster;DSP=157792;KP=hsa04740:OlfactoryTransduction;GESP=C:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:29:93,0,27 0/1:14:10,0,42 1/1:5:108,5,0 0/1:40:84,0,38</p>

<p>11 48347306 rs73465911 G T 151 highCVG AC1=5;AF1=0.6041;DB;DP=29;DP4=11,2,7,7;FQ=149;MQ=60;PV4=0.1,0.051,1,1;SNPEFF_AMINO_ACID_CHANGE=V272F;SNPEFF_CODON_CHANGE=Gtt/Ttt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0060;DN=131;DA=G/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=VAL/PHE;PP=272/330;CDP=814;GS=50;PH=0.175;CP=0.997;CG=5.970;AA=G;DG;DV=by-frequency,by-cluster;DSP=158551;KP=hsa04740:OlfactoryTransduction;TFBS=V$RFX1_01;GESP=G:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:47:101,0,46 0/1:16:11,0,66 1/1:8:46,9,0 0/1:42:37,0,79</p>

<p>11 50003191 rs4598671 C A 45.80 midCVG AC1=4;AF1=0.4411;DB;DP=12;DP4=0,6,3,2;FQ=48.3;MQ=57;PV4=0.061,0.0084,0.036,1;SNPEFF_AMINO_ACID_CHANGE=V283L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005270.4;VDB=0.0080;DN=111;DA=C/A;GM=NM_001005270.2;GL=OR4C12;FG=missense;FD=missense;AAC=VAL/LEU;PP=283/310;CDP=847;GS=32;PH=0;CP=0.806;CG=2.980;AA=A;HA=23.3;HE=42.5;HC=44.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=171898;KP=hsa04740:OlfactoryTransduction;GESP=A:6703/C:6291;PAC=NP_001005270.1 GT:GQ:PL 0/0:4:0,3,29 0/0:6:0,6,57 1/1:8:80,12,0 0/1:12:10,0,54</p>

<p>11 51411914 rs61890334 T C 57.90 midCVG AC1=4;AF1=0.4616;DB;DP=12;DP4=4,3,1,4;FQ=60.5;MQ=55;PV4=0.29,1,1,0.074;SNPEFF_AMINO_ACID_CHANGE=Y161C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005272.3;VDB=0.0086;DN=129;DA=T/C;GM=NM_001005272.3;GL=OR4A5;FG=missense;FD=missense;AAC=TYR/CYS;PP=161/316;CDP=482;GS=194;PH=0.027;CP=0.042;CG=1.930;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1032140;KP=hsa04740:OlfactoryTransduction;GESP=C:765/T:12229;PAC=NP_001005272.3 GT:GQ:PL 0/0:13:0,15,95 0/0:4:0,3,25 1/1:6:63,9,0 0/1:4:40,2,0</p>

<p>11 55110861 rs200705953 C A 60.70 highCVG AC1=3;AF1=0.3502;DB;DP=28;DP4=6,14,1,5;FQ=62.6;MQ=60;PV4=1,1,1,2.5e-05;SNPEFF_AMINO_ACID_CHANGE=A62D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0110;DN=137;DA=C/A;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=ALA/ASP;PP=62/329;CDP=185;GS=126;PH=0.109;CP=0.839;CG=2.570;AA=C;DV=by-frequency;DSP=45527;KP=hsa04740:OlfactoryTransduction;GESP=C:12994;PAC=NP_001005274.1 GT:GQ:PL 0/0:9:0,9,61 0/0:36:0,36,110 1/1:5:57,9,0 0/1:50:50,0,67</p>

<p>11 55111580 rs76200064 G A 19.70 highCVG AC1=2;AF1=0.2824;DB;DP=16;DP4=5,4,5,0;FQ=21;MQ=56;PV4=0.22,0.00029,0.0037,1;SNPEFF_AMINO_ACID_CHANGE=E302K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0049;DN=131;DA=G/A;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=GLU/LYS;PP=302/329;CDP=904;GS=56;PH=0;CP=0.002;CG=0.439;AA=G;DG;DSP=46246;KP=hsa04740:OlfactoryTransduction;GESP=G:12990;PAC=NP_001005274.1 GT:GQ:PL 0/0:17:0,15,113 0/0:6:0,3,26 1/1:8:64,15,0 0/0:11:0,9,66</p>

<p>11 55111602 rs78987961 A T 121 highCVG AC1=3;AF1=0.3854;DB;DP=23;DP4=8,4,9,2;FQ=123;MQ=56;PV4=0.64,0.049,0.0029,1;SNPEFF_AMINO_ACID_CHANGE=K309I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0051;DN=131;DA=A/T;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=LYS/ILE;PP=309/329;CDP=926;GS=102;PH=0;CP=0.000;CG=-6.030;AA=A;DG;DSP=46268;KP=hsa04740:OlfactoryTransduction;GESP=A:12988;PAC=NP_001005274.1GT:GQ:PL 0/0:14:0,15,112 0/0:9:0,9,61 1/1:10:85,15,0 0/1:65:84,0,60</p>

<p>11 55111607 rs77602694 G T 129 highCVG AC1=3;AF1=0.3814;DB;DP=25;DP4=10,4,9,2;FQ=131;MQ=56;PV4=0.66,0.32,0.0014,1;SNPEFF_AMINO_ACID_CHANGE=V311L;SNPEFF_CODON_CHANGE=Gta/Tta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0051;DN=131;DA=G/T;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=VAL/LEU;PP=311/329;CDP=931;GS=32;PH=0;CP=0.001;CG=-3.470;AA=G;DG;DSP=46273;KP=hsa04740:OlfactoryTransduction;GESP=G:12986;PAC=NP_001005274.1 GT:GQ:PL 0/0:20:0,21,137 0/0:9:0,9,65 1/1:10:89,15,0 0/1:63:88,0,58</p>

<p>11 55322606 rs17581700 T C 30.70 highCVG AC1=3;AF1=0.3416;DB;DP=16;DP4=2,10,2,2;FQ=32.4;MQ=60;PV4=0.24,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=L275S;SNPEFF_CODON_CHANGE=tTg/tCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001920.1;VDB=0.0127;DN=123;DA=T/C;GM=NM_001001920.1;GL=OR4C15;FG=missense;FD=missense;AAC=LEU/SER;PP=275/371;CDP=824;GS=145;PH=1;CP=0.921;CG=5.020;AA=T;HA=0.0;HE=7.1;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=257272;KP=hsa04740:OlfactoryTransduction;GESP=C:567/T:12427;PAC=NP_001001920.1 GT:GQ:PL 0/0:10:0,9,72 0/0:16:0,15,91 1/1:4:66,9,0 0/1:10:10,0,64</p>

<p>11 55406524 rs61896975 C T 78.80 highCVG AC1=4;AF1=0.4977;DB;DP=15;DP4=2,3,1,9;FQ=54.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R231C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4P4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004124.1;VDB=0.0025;DN=129;DA=C/T;GM=NM_001004124.1;GL=OR4P4;FG=missense;FD=missense;AAC=ARG/CYS;PP=231/313;CDP=691;GS=180;PH=0.407;CP=0.000;CG=-1.540;AA=C;DG;DV=by-cluster,by-1000G;DSP=244337;KP=hsa04740:OlfactoryTransduction;GESP=T:719/C:11695;PAC=NP_001004124.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,55 1/1:4:102,6,0 0/1:19:17,0,37</p>

<p>11 55541620 rs7124871 G T 59.50 midCVG AC1=4;AF1=0.4901;DB;DP=13;DP4=3,4,4,2;FQ=62.4;MQ=59;PV4=0.59,1,0.15,1;SNPEFF_AMINO_ACID_CHANGE=R236L;SNPEFF_CODON_CHANGE=cGc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5D13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001967.1;VDB=0.0105;DN=116;DA=G/A/T;GM=NM_001001967.1;GL=OR5D13;FG=missense;FD=missense;AAC=ARG/LEU;PP=236/315;CDP=707;GS=102;PH=0.01;CP=0.000;CG=2.850;AA=G;HA=0.0;HE=7.6;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=109241;KP=hsa04740:OlfactoryTransduction;GESP=T:576/G:12416;PAC=NP_001001967.1 GT:GQ:PL 0/1:3:0,0,0 0/0:13:0,15,100 1/1:9:74,12,0 0/1:31:29,0,37</p>

<p>11 55703861 rs17597625 T C 22.50 midCVG AC1=4;AF1=0.508;DB;DP=12;DP4=3,1,4,0;FQ=25.4;MQ=60;PV4=1,0.019,1,0.18;SNPEFF_AMINO_ACID_CHANGE=R6G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006637.1;VDB=0.0044;DN=123;DA=T/C;GM=NM_006637.1;GL=OR5I1;FG=missense;FD=missense;AAC=ARG/GLY;PP=6/315;CDP=16;GS=125;PH=0;CP=0.238;CG=4.060;AA=C;HA=0.6;HE=10.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=43693;KP=hsa04740:OlfactoryTransduction;GESP=C:787/T:12195;PAC=NP_006628.1 GT:GQ:PL 0/0:8:0,9,62 0/0:3:0,3,22 1/1:4:30,6,0 1/1:4:38,6,0</p>

<p>11 56468044 rs3975155 T G 113 highCVG AC1=5;AF1=0.5811;DB;DP=23;DP4=3,3,8,7;FQ=41.5;MQ=39;PV4=1,0.053,0.014,0.27;SNPEFF_AMINO_ACID_CHANGE=F61V;SNPEFF_CODON_CHANGE=Ttc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=108;DA=T/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=PHE/VAL;PP=61/306;CDP=181;GS=50;PH=0.108;CP=0.998;CG=3.370;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482114;TFBS=V$POU3F2_01;GESP=T:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:29:25,0,52 0/1:14:60,0,12 1/1:13:72,15,0 0/1:4:0,2,18</p>

<p>11 56468047 rs2865519 A G 112 highCVG AC1=5;AF1=0.5869;DB;DP=23;DP4=3,4,8,7;FQ=50.7;MQ=40;PV4=1,1,0.0065,0.36;SNPEFF_AMINO_ACID_CHANGE=T62A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=101;DA=A/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=THR/ALA;PP=62/306;CDP=184;GS=58;PH=0.004;CP=0.961;CG=4.540;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482111;TFBS=V$POU3F2_01;GESP=A:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:28:24,0,64 0/1:11:62,0,9 1/1:13:70,15,0 0/1:4:0,2,18</p>

<p>11 56756399 rs10896563 G T 28.10 midCVG AC1=3;AF1=0.3852;DB;DP=9;DP4=2,2,0,4;FQ=29.8;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G4V;SNPEFF_CODON_CHANGE=gGa/gTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5AK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005323.1;VDB=0.0173;DN=120;DA=G/T;GM=NM_001005323.1;GL=OR5AK2;FG=missense;FD=missense;AAC=GLY/VAL;PP=4/310;CDP=11;GS=109;PH=0.262;CP=0.000;CG=-4.160;AA=G;HA=5.0;HC=41.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=193759;KP=hsa04740:OlfactoryTransduction;GESP=T:720/G:12274;PAC=NP_001005323.1 GT:GQ:PL 0/0:7:0,6,44 0/0:7:0,6,36 1/1:7:69,12,0 0/0:3:0,0,0</p>

<p>11 57982763 rs1993088 A G 15.60 lowCVG AC1=5;AF1=0.5967;DB;DP=7;DP4=3,1,1,2;FQ=18.5;MQ=44;PV4=0.49,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N183D;SNPEFF_CODON_CHANGE=Aac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004458.1;DN=92;DA=A/G;GM=NM_001004458.1;GL=OR1S1;FG=missense;FD=missense;AAC=ASN/ASP;PP=183/326;CDP=547;GS=23;PH=0;CP=0.017;CG=-1.100;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=35861;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:3379/A:9603;PAC=NP_001004458.1 GT:GQ:PL 0/1:21:18,0,28 0/1:3:0,0,0 1/1:3:19,3,0 0/1:21:17,0,32</p>

<p>11 59940599 rs7232 T A 17.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=0,3,1,1;FQ=18;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T213S;SNPEFF_CODON_CHANGE=Act/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MS4A6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001247999.1;VDB=0.0040;DN=52;DA=T/A;GM=NM_001247999.1,NM_022349.3,NM_152851.2,NM_152852.2,XM_005274177.1,XM_005274178.1,XM_005274179.1,XM_005274180.1,XM_005274181.1;GL=MS4A6A;FG=missense;FD=unknown,unknown,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=THR/SER,THR/SER,ASN/ILE,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER;PP=213/254,185/226,150/179,185/249,213/277,213/277,185/249,168/232,168/209;CDP=637,553,449,553,637,637,553,502,502;GS=58,58,149,58,58,58,58,58,58;PH=0.827;CP=0.001;CG=-5.110;AA=T;HA=1.8;HE=44.6;HC=8.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=4;GESP=A:3608/T:9384;PAC=NP_001234928.1,NP_071744.2,NP_690590.1,NP_690591.1,XP_005274234.1,XP_005274235.1,XP_005274236.1,XP_005274237.1,XP_005274238.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:54,6,0 0/0:9:0,9,55</p>

<p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

<p>11 67173164 . G T 39.60 lowCVG AC1=5;AF1=0.5973;DP=8;DP4=2,0,0,5;FQ=-7.3;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q153H;SNPEFF_CODON_CHANGE=caG/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198517.3;VDB=0.0003;GM=NM_001256508.1,NM_198517.3,NR_046266.1;GL=TBC1D10C;FG=missense,missense,non-coding-exon;FD=unknown;AAC=GLN/HIS,GLN/HIS,none;PP=153/311,153/447,NA;CDP=459,459,686;GS=24,24,NA;PH=0.071;CP=0.996;CG=0.600;AA=G;DSP=9;GESP=G:12990;PAC=NP_001243437.1,NP_940919.1,NA GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,28 1/1:12:78,15,0 0/1:3:0,0,0</p>

<p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>11 71169547 rs2276360 G C 15.50 lowCVG AC1=5;AF1=0.5737;DB;DP=8;DP4=2,2,0,3;FQ=19.2;MQ=60;PV4=0.43,0.24,1,0.37;SNPEFF_AMINO_ACID_CHANGE=V74L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NADSYN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018161.4;VDB=0.0099;DN=100;DA=G/C;GM=NM_018161.4;GL=NADSYN1;FG=missense;FD=missense;AAC=VAL/LEU;PP=74/707;CDP=220;GS=32;PH=0;CP=0.037;CG=-2.340;AA=C;HA=33.2;HE=27.2;HC=40.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=44;KP=hsa00760:NicotinateAndNicotinamideMetabolism|hsa01100:MetabolicPathways;GESP=C:8267/G:4721;PAC=NP_060631.2 GT:GQ:PL 0/1:3:21,3,0 0/1:3:0,0,0 1/1:5:36,6,0 0/0:10:0,12,81</p>

<p>11 74413955 rs35903991 G A 36.30 midCVG AC1=3;AF1=0.3987;DB;DP=12;DP4=3,2,0,5;FQ=38.3;MQ=60;PV4=0.17,1,1,0.15;SNPEFF_AMINO_ACID_CHANGE=P335L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHRDL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015424.4;VDB=0.0091;DN=126;DA=G/A;GM=NM_001278473.1,NM_015424.4,XM_005273884.1;GL=CHRDL2;FG=missense;FD=unknown,missense,unknown;AAC=PRO/LEU;PP=335/430,335/452,335/415;CDP=1004;GS=98;PH=1;CP=0.010;CG=4.150;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=58;TFBS=V$CEBP_C,V$LUN1_01,V$MIF1_01;GESP=A:141/G:12845;PAC=NP_001265402.1,NP_056239.3,XP_005273941.1 GT:GQ:PL 0/0:12:0,12,70 0/1:3:0,0,0 1/1:10:78,15,0 0/0:5:0,3,22</p>

<p>11 75152243 rs571353 C T 59.30 midCVG AC1=5;AF1=0.5871;DB;DP=10;DP4=4,1,4,1;FQ=40.4;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=A480T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDPD5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030792.6;VDB=0.0080;DN=83;DA=C/T;GM=NM_030792.6,XM_005274313.1,XM_005274314.1,XM_005274315.1,XM_005274316.1,XM_005274317.1,XM_005274318.1,XM_005274319.1;GL=GDPD5;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=480/606,480/607,480/607,480/607,480/607,480/606,235/362,235/361;CDP=1438,1438,1438,1438,1438,1438,703,703;GS=58;PH=0;CP=0.014;CG=-0.094;HA=44.1;HE=26.6;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8514/C:4472;PAC=NP_110419.5,XP_005274370.1,XP_005274371.1,XP_005274372.1,XP_005274373.1,XP_005274374.1,XP_005274375.1,XP_005274376.1 GT:GQ:PL 0/1:7:30,0,4 0/0:9:0,12,77 1/1:4:26,3,0 1/1:5:48,6,0</p>

<p>11 76912636 rs2276288 A T 117 midCVG AC1=5;AF1=0.6164;DB;DP=13;DP4=2,3,6,2;FQ=39.5;MQ=60;PV4=0.29,1,1,0.33;SNPEFF_AMINO_ACID_CHANGE=S1666C;SNPEFF_CODON_CHANGE=Agt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000260.3;VDB=0.0049;DN=100;DA=A/T;GM=NM_000260.3,NM_001127180.1,XM_005274011.1,XM_005274012.1;GL=MYO7A;FG=missense;FD=missense,missense,unknown,unknown;AAC=SER/CYS;PP=1666/2216,1628/2176,1666/2218,1627/2177;CDP=4996,4882,4996,4879;GS=112;PH=0;CP=0.581;CG=3.460;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=48;GESP=T:7510/A:5238;PPI=USH1C:846;PAC=NP_000251.3,NP_001120652.1,XP_005274068.1,XP_005274069.1 GT:GQ:PL 0/1:4:59,0,1 0/1:17:12,0,58 1/1:11:89,12,0 0/1:4:0,3,19</p>

<p>11 77909014 rs2510044 G A 62.90 highCVG AC1=4;AF1=0.528;DB;DP=28;DP4=0,20,3,5;FQ=65.9;MQ=60;PV4=0.017,1,1,0.089;SNPEFF_AMINO_ACID_CHANGE=V236M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP35;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020798.2;VDB=0.0191;DN=100;DA=G/A;GM=NM_020798.2;GL=USP35;FG=missense;FD=missense;AAC=VAL/MET;PP=236/1019;CDP=706;GS=21;PH=0.999;CP=0.992;CG=4.820;AA=G;HA=0.4;HE=10.7;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=33;TFBS=V$NRSF_01;GESP=A:1344/G:11488;PAC=NP_065849.1 GT:GQ:PL 0/1:20:20,0,19 0/1:3:0,3,25 1/1:5:53,6,0 0/1:36:33,0,76</p>

<p>11 87013438 rs2276102 G A 98 midCVG AC1=6;AF1=0.7212;DB;DP=11;DP4=0,2,5,3;FQ=-16.6;MQ=60;PV4=0.44,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=G218R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM135;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022918.3;VDB=0.0208;DN=100;DA=G/A;GM=NM_001168724.1,NM_022918.3,NR_033149.1;GL=TMEM135;FG=missense,missense,non-coding-exon;FD=missense,missense,nc-transcript-variant;AAC=GLY/ARG,GLY/ARG,none;PP=196/437,218/459,NA;CDP=586,652,820;GS=125,125,NA;PH=0;CP=0.664;CG=4.310;AA=A;HA=43.6;HE=31.1;HC=46.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;GESP=A:8629/G:4371;PAC=NP_001162195.1,NP_075069.3,NA GT:GQ:PL 0/1:14:15,0,14 0/1:3:0,0,0 1/1:5:26,3,0 0/1:3:93,0,0</p>

<p>11 89531540 rs12417980 C T 84.10 highCVG AC1=6;AF1=0.7306;DB;DP=18;DP4=3,3,3,9;FQ=8.51;MQ=21;PV4=0.34,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G373R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM49;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020358.2;VDB=0.0085;DN=120;DA=C/T;GM=NM_020358.2;GL=TRIM49;FG=missense;FD=missense;AAC=GLY/ARG;PP=373/453;CDP=1117;GS=125;PH=0.834;CP=0.000;CG=-1.800;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=258;GESP=T:3992/C:8964;PAC=NP_065091.1 GT:GQ:PL 0/1:4:13,0,1 0/1:3:0,0,0 1/1:8:80,7,0 0/1:34:29,0,39</p>

<p>11 99715891 rs61749255 T G 78.20 highCVG AC1=3;AF1=0.3664;DB;DP=19;DP4=3,8,1,5;FQ=80.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I158M;SNPEFF_CODON_CHANGE=atT/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001243270.1;VDB=0.0015;DN=129;DA=T/G;GM=NM_001243270.1,NM_001243271.1,NM_014361.3,NM_175566.2;GL=CNTN5;FG=missense;FD=missense;AAC=ILE/MET;PP=158/1101,158/912,158/1101,84/1027;CDP=474,474,474,252;GS=10;PH=1;CP=0.990;CG=0.247;AA=T;DV=by-frequency,by-cluster;DSP=73;GESP=G:64/T:11852;PAC=NP_001230199.1,NP_001230200.1,NP_055176.1,NP_780775.1 GT:GQ:PL 0/1:34:54,0,28 0/0:21:0,21,127 1/1:5:70,9,0 0/0:7:0,6,42</p>

<p>11 102593248 rs1940475 T C 26.10 highCVG AC1=4;AF1=0.4625;DB;DP=31;DP4=0,5,1,3;FQ=28.7;MQ=60;PV4=0.44,0.31,1,0.045;SNPEFF_AMINO_ACID_CHANGE=K87E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MMP8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002424.2;VDB=0.0166;DN=92;DA=T/C;GM=NM_002424.2,XM_005271556.1,XM_005271557.1,XM_005271558.1;GL=MMP8;FG=missense;FD=missense,unknown,unknown,unknown;AAC=LYS/GLU;PP=87/468,64/445,64/445,87/369;CDP=259,190,190,259;GS=56;PH=0.01;CP=0.001;CG=-6.060;AA=T;HA=25.4;HE=48.8;HC=41.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=89;GESP=C:7197/T:5807;PAC=NP_002415.1,XP_005271613.1,XP_005271614.1,XP_005271615.1 GT:GQ:PL 0/0:8:0,9,64 0/1:4:17,3,0 1/1:6:55,9,0 0/0:6:0,6,25</p>

<p>11 123624658 rs12364099 G T 84.70 midCVG AC1=4;AF1=0.5037;DB;DP=14;DP4=1,3,8,1;FQ=24.8;MQ=60;PV4=0.052,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T190N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6X1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005188.1;VDB=0.0009;DN=120;DA=G/T;GM=NM_001005188.1;GL=OR6X1;FG=missense;FD=missense;AAC=THR/ASN;PP=190/313;CDP=569;GS=65;PH=0.029;CP=0.386;CG=3.460;AA=G;HA=1.7;HE=14.0;HC=2.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12402;KP=hsa04740:OlfactoryTransduction;TFBS=V$YY1_01;GESP=T:1529/G:11473;PAC=NP_001005188.1 GT:GQ:PL 0/1:19:84,0,15 0/1:3:0,0,0 1/1:4:41,6,0 0/0:5:0,6,47</p>

<p>11 124135009 rs2512168 G A 85.50 highCVG AC1=3;AF1=0.3452;DB;DP=15;DP4=1,4,4,6;FQ=75.1;MQ=42;PV4=0.6,1,0.082,1;SNPEFF_AMINO_ACID_CHANGE=C96Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR8G5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005198.1;VDB=0.0030;DN=100;DA=G/A;GM=NM_001005198.1;GL=OR8G5;FG=missense;FD=missense;AAC=CYS/TYR;PP=96/347;CDP=287;GS=194;CP=0.533;CG=-5.000;AA=A;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118248;GESP=A:6010/G:6910;PAC=NP_001005198.1 GT:GQ:PL 0/0:5:0,3,25 0/0:7:0,6,48 1/1:24:130,30,0 0/0:7:0,6,45</p>

<p>11 124908352 rs7107487 T C 999 highCVG AC1=4;AF1=0.5057;DB;DP=29;DP4=6,9,4,10;FQ=205;MQ=60;PV4=0.7,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=C813R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025004.2;VDB=0.0034;DN=116;DA=T/C;GM=NM_025004.2;GL=CCDC15;FG=missense;FD=missense;AAC=CYS/ARG;PP=813/952;CDP=2437;GS=180;PH=0;CP=0.973;CG=3.110;AA=C;HA=36.7;HE=12.8;HC=17.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=C:2453/T:9283;PAC=NP_079280.2 GT:GQ:PL 0/1:48:45,0,74 0/1:63:60,0,82 1/1:18:134,21,0 0/0:12:0,15,94</p>

<p>11 129724614 . T G 7.84 lowCVG AC1=6;AF1=0.7749;DP=5;DP4=1,0,0,3;FQ=-20.3;MQ=60;PV4=0.25,0.47,1,1;SNPEFF_AMINO_ACID_CHANGE=F96L;SNPEFF_CODON_CHANGE=ttT/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM45B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138788.3;GM=NM_138788.3,XM_005271400.1,XM_005271401.1;GL=TMEM45B;FG=missense;FD=unknown;AAC=PHE/LEU;PP=96/276;CDP=288;GS=22;PH=0.989;CP=0.013;CG=-9.770;AA=T;DSP=98;GESP=T:12996;PAC=NP_620143.1,XP_005271457.1,XP_005271458.1 GT:GQ:PL 0/1:3:0,0,0/1:11:10,0,11 1/1:4:20,3,0 1/1:4:14,3,0</p>

<p>12 6948468 rs1129649 T C 24.30 lowCVG AC1=3;AF1=0.3968;DB;DP=8;DP4=2,2,0,4;FQ=25.9;MQ=59;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I684T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LEPREL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014262.4;VDB=0.0226;DN=86;DA=T/C;GM=NM_014262.3;GL=LEPREL2;FG=coding-unknown;FD=missense;CDP=2053;CP=0.892;CG=2.360;AA=T;HA=43.1;HE=37.2;HC=35.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=C:4320/T:8170;PAC=NP_055077.2 GT:GQ:PL 0/0:5:0,3,23 0/1:3:0,0,0 1/1:7:65,12,0 0/0:9:0,9,51</p>

<p>12 8374812 rs17855656 G A 37.30 lowCVG AC1=5;AF1=0.6547;DB;DP=5;DP4=1,0,4,0;FQ=-14.5;MQ=48;PV4=1,0.073,1,1;SNPEFF_AMINO_ACID_CHANGE=P334L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM90A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018088.3;VDB=0.0130;DN=123;DA=G/A;GM=NM_018088.3,XM_005253403.1,XM_005253404.1;GL=FAM90A1;FG=missense;FD=missense,unknown,unknown;AAC=PRO/LEU;PP=334/465;CDP=1001;GS=98;PH=0.023;CP=0.006;CG=-1.440;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=569;GESP=A:1659/G:11279;PAC=NP_060558.3,XP_005253460.1,XP_005253461.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,18 1/1:10:74,12,0 0/1:3:0,0,0</p>

<p>12 9307373 rs2377741 T G 29.10 midCVG AC1=6;AF1=0.7339;DB;DP=14;DP4=0,2,5,1;FQ=-4.32;MQ=50;PV4=0.11,0.007,0.18,1;SNPEFF_AMINO_ACID_CHANGE=T1205P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PZP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002864.2;VDB=0.0199;DN=100;DA=T/G;GM=NM_002864.2,XM_005253459.1;GL=PZP;FG=missense;FD=missense,unknown;AAC=THR/PRO;PP=1205/1483,991/1269;CDP=3613,2971;GS=38;PH=0;CP=0.039;CG=0.627;AA=G;HA=4.9;HE=0.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:12716/T:290;PAC=NP_002855.2,XP_005253516.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:14:60,15,0 0/1:12:6,0,29</p>

<p>12 10041364 rs1797517 G A 210 highCVG AC1=6;AF1=0.7824;DB;DP=15;DP4=2,1,2,10;FQ=-5.09;MQ=60;PV4=0.15,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V68I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLRF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001190765.1;VDB=0.0087;DN=89;DA=G/A;GM=NM_001190765.1;GL=KLRF2;FG=missense;FD=missense;AAC=VAL/ILE;PP=68/208;CDP=202;GS=29;CP=0.328;CG=1.980;AA=A;HA=42.5;HE=22.7;HC=31.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=16;PAC=NP_001177694.1 GT:GQ:PL 0/1:22:16,0,28 0/1:5:44,0,6 1/1:17:103,15,0 1/1:14:91,12,0</p>

<p>12 31135579 rs2075333 T C 18.20 midCVG AC1=3;AF1=0.3809;DB;DP=10;DP4=6,1,2,1;FQ=20.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V190A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSPAN11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080509.2;VDB=0.0080;DN=96;DA=T/C;GM=NM_001080509.2,XM_005253378.1,XM_005253379.1;GL=TSPAN11;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=190/254,190/254,180/244;CDP=569,569,539;GS=64;PH=0;CP=0.008;CG=-1.760;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$PAX5_02;GESP=C:8630/T:4352;PAC=NP_001073978.1,XP_005253435.1,XP_005253436.1 GT:GQ:PL 0/1:16:16,0,47 0/0:9:0,9,58 1/1:3:45,6,0 0/0:5:0,3,26</p>

<p>12 32135186 rs3759302 T A 62.90 midCVG AC1=4;AF1=0.5521;DB;DP=9;DP4=2,1,1,4;FQ=12.7;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S433T;SNPEFF_CODON_CHANGE=Tct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0034;DN=107;DA=T/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=SER/THR;PP=433/1748;CDP=1297;GS=58;PH=0.999;CP=0.001;CG=-1.130;AA=T;HA=3.4;HE=18.1;HC=11.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1375;GESP=A:2107/T:10899;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:3:0,3,16 0/0:5:0,6,38 1/1:3:26,3,0 1/1:10:82,12,0</p>

<p>12 32137565 rs1057994 G A 109 midCVG AC1=6;AF1=0.6731;DB;DP=10;DP4=1,2,1,6;FQ=8.62;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V1226I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0055;DN=86;DA=G/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=1226/1748;CDP=3676;GS=29;PH=0.02;CP=0.000;CG=-2.920;AA=G;HA=4.3;HE=19.2;HC=9.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1327;GESP=A:2074/G:10932;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:27:37,0,27 0/1:4:0,3,21 1/1:7:42,6,0 1/1:9:75,9,0</p>

<p>12 40758652 rs3761863 T C 14.60 lowCVG AC1=6;AF1=0.6706;DB;DP=5;DP4=1,0,2,1;FQ=-12.1;MQ=60;PV4=1,0.14,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M2397T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=49;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198578.3;VDB=0.0050;DN=107;DA=T/C;GM=NM_198578.3,XM_005268629.1,XM_005268630.1,XM_005268631.1;GL=LRRK2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=MET/THR;PP=2397/2528,2397/2522,2397/2528,2357/2488;CDP=7190,7190,7190,7070;GS=81;PH=0;CP=0.005;CG=3.030;AA=C;HA=46.5;HE=34.3;HC=42.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=9;KP=hsa05012:Parkinson'sDisease;GESP=C:7962/T:5042;PPI=STUB1:846;PAC=NP_940980.3,XP_005268686.1,XP_005268687.1,XP_005268688.1 GT:GQ:PL 0/1:20:30,0,18 0/1:3:0,0,0 1/1:3:19,3,0 0/1:3:0,0,0</p>

<p>12 42863266 rs79087668 C T 63.10 midCVG AC1=5;AF1=0.5836;DB;DP=10;DP4=2,2,2,4;FQ=39.8;MQ=60;PV4=1,0.25,1,0.22;SNPEFF_AMINO_ACID_CHANGE=A124T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRICKLE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144881.1;VDB=0.0077;DN=131;DA=C/T;GM=NM_001144881.1,NM_001144882.1,NM_001144883.1,NM_153026.2;GL=PRICKLE1;FG=missense;FD=missense;AAC=ALA/THR;PP=124/832;CDP=370;GS=58;PH=0.002;CP=0.927;CG=3.550;AA=C;DV=by-frequency,by-cluster;DSP=15;KP=hsa04310:WntSignalingPathway;GESP=T:22/C:12984;PAC=NP_001138353.1,NP_001138354.1,NP_001138355.1,NP_694571.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,26 1/1:10:83,12,0 0/1:25:21,0,51</p>

<p>12 49443799 rs112236653 G A 49.50 midCVG AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1 GT:GQ:PL 0/1:14:11,0,64 0/1:3:0,0,0 1/1:9:81,12,0 0/0:7:0,9,52</p>

<p>12 49952449 . C T 14.40 lowCVG AC1=4;AF1=0.5125;DP=6;DP4=3,0,0,3;FQ=12.6;MQ=60;PV4=0.1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A469T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCRS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001012300.1;VDB=0.0104;GM=NM_001012300.1,NM_001278341.1,NM_006337.4,XM_005268572.1,XM_005268573.1;GL=MCRS1;FG=missense;FD=unknown;AAC=ALA/THR;PP=469/476,265/272,456/463,456/463,367/374;CDP=1405,793,1366,1366,1099;GS=58;PH=0.997;CP=1.000;CG=5.280;AA=C;DSP=64;TFBS=V$PAX4_01;GESP=C:13006;PPI=INO80E:892|DAXX:846|INO80:811|PBX2:786;PAC=NP_001012300.1,NP_001265270.1,NP_006328.2,XP_005268629.1,XP_005268630.1 GT:GQ:PL 0/1:7:5,0,32 0/1:3:0,0,0 1/1:4:47,6,0 0/0:4:0,3,17</p>

<p>12 52908917 rs11549950 T C 25.50 lowCVG AC1=3;AF1=0.3646;DB;DP=8;DP4=3,1,0,3;FQ=27.4;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S528G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000424.3;VDB=0.0108;DN=120;DA=T/C;GM=NM_000424.3,XM_005268862.1;GL=KRT5;FG=missense;FD=missense,unknown;AAC=SER/GLY;PP=528/591;CDP=1582;GS=56;PH=0;CP=0.000;CG=0.084;AA=C;HA=25.2;HE=9.0;HC=4.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CCG)n;CA;DSP=108;GESP=C:2251/T:10755;PPI=KRT14:959;PAC=NP_000415.2,XP_005268919.1 GT:GQ:PL 0/0:5:0,3,24 0/0:3:0,0,0 1/1:4:65,9,0 0/0:10:0,9,70</p>

<p>12 53091566 rs1567759 C A 107 midCVG AC1=5;AF1=0.6042;DB;DP=10;DP4=1,2,3,4;FQ=27.6;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=G220C;SNPEFF_CODON_CHANGE=Ggt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT77;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175078.2;VDB=0.0106;DN=88;DA=C/A;GM=NM_175078.2,XM_005268853.1;GL=KRT77;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=GLY/CYS,none;PP=220/579,NA;CDP=658,NA;GS=159,NA;PH=0.999;CP=0.010;CG=-2.050;AA=C;HA=44.7;HE=42.9;HC=20.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=101;GESP=A:5613/C:7393;PPI=USP32:916;PAC=NP_778253.2,NA GT:GQ:PL 0/1:4:0,3,20 0/0:4:0,6,51 1/1:10:90,12,0 1/1:8:64,9,0</p>

<p>12 53165940 rs6580904 C T 76.30 midCVG AC1=5;AF1=0.5683;DB;DP=11;DP4=2,1,5,3;FQ=13.2;MQ=60;PV4=1,0.37,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A359T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015848.4;VDB=0.0153;DN=116;DA=C/T;GM=NM_015848.4;GL=KRT76;FG=missense;FD=missense;AAC=ALA/THR;PP=359/639;CDP=1075;GS=58;PH=0.986;CP=0.890;CG=4.420;AA=C;HA=39.2;HE=5.8;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=T:9195/C:3811;PAC=NP_056932.2 GT:GQ:PL 0/1:3:0,3,15 0/1:15:12,0,21 1/1:13:85,15,0 0/1:19:24,0,18</p>

<p>12 53242440 rs2013335 A G 31 lowCVG AC1=6;AF1=0.7073;DB;DP=6;DP4=2,0,2,2;FQ=-3.98;MQ=60;PV4=0.47,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L92P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT78;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173352.3;VDB=0.0085;DN=92;DA=A/G;GM=NM_173352.2,XM_005268696.1;GL=KRT78;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=LEU/PRO,none;PP=92/521,NA;CDP=275,NA;GS=98,NA;PH=1;CP=0.944;CG=5.180;AA=A;HA=28.3;HE=37.7;HC=48.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=70;GESP=G:6269/A:6737;PAC=NP_775487.2,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:38,6,0 0/1:29:29,0,29</p>

<p>12 53605545 rs2229774 G A 41.20 midCVG AC1=4;AF1=0.5599;DB;DP=12;DP4=0,3,0,9;FQ=13.2;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=S427L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RARG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000966.5;VDB=0.0000;DN=98;DA=G/A;GM=NM_000966.5,NM_001042728.2,NM_001243730.1,NM_001243731.1,NM_001243732.1,XM_005269054.1,XM_005269055.1,XM_005269056.1,XM_005269057.1;GL=RARG;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/LEU;PP=427/455,416/444,355/383,306/334,405/433,514/542,514/542,427/455,427/455;CDP=1280,1247,1064,917,1214,1541,1541,1280,1280;GS=145;PH=0.525;CP=1.000;CG=4.530;AA=G;HA=11.1;HE=6.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=103;TFBS=V$ARP1_01;GESP=A:949/G:12057;PAC=NP_000957.1,NP_001036193.1,NP_001230659.1,NP_001230660.1,NP_001230661.1,XP_005269111.1,XP_005269112.1,XP_005269113.1,XP_005269114.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:81,27,0 0/0:7:0,9,49</p>

<p>12 53682468 . G T 35.30 lowCVG AC1=5;AF1=0.6499;DP=5;DP4=1,0,0,4;FQ=-11.7;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1565S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ESPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012291.4;VDB=0.0084;GM=NM_012291.4;GL=ESPL1;FG=missense;FD=unknown;AAC=ALA/SER;PP=1565/2121;CDP=4693;GS=99;PH=0.01;CP=0.003;CG=2.840;AA=G;DSP=16;KP=hsa04110:CellCycle|hsa04114:OocyteMeiosis;GESP=G:12990;PPI=PTTG1:987|IRF2BPL:987;PAC=NP_036423.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:72,12,0 0/1:3:0,3,21</p>

<p>12 55355033 rs997173 C T 9.97 midCVG AC1=2;AF1=0.3049;DB;DP=11;DP4=3,5,3,0;FQ=11.3;MQ=60;PV4=0.18,0.06,1,1;SNPEFF_AMINO_ACID_CHANGE=E496K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TESPA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098815.2;VDB=0.0017;DN=86;DA=C/T;GM=NM_001098815.2,NM_001136030.2,NM_001261844.1,NM_014796.2,XM_005269243.1,XM_005269244.1,XM_005269245.1,XM_005269246.1,XM_005269247.1,XM_005269248.1,XM_005269249.1;GL=TESPA1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLU/LYS;PP=496/522,496/522,358/384,358/384,496/522,496/522,496/522,496/522,358/384,243/269,243/269;CDP=1486,1486,1072,1072,1486,1486,1486,1486,1072,727,727;GS=56;PH=0.004;CP=0.842;CG=2.340;AA=C;HA=7.5;HE=7.6;HC=35.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=19;GESP=T:871/C:12133;PAC=NP_001092285.1,NP_001129502.1,NP_001248773.1,NP_055611.1,XP_005269300.1,XP_005269301.1,XP_005269302.1,XP_005269303.1,XP_005269304.1,XP_005269305.1,XP_005269306.1 GT:GQ:PL 0/0:9:0,6,47 0/0:20:0,18,111 1/1:3:50,9,0 0/0:4:0,0,0</p>

<p>12 56030938 rs10876838 C T 17.80 lowCVG AC1=4;AF1=0.4457;DB;DP=6;DP4=0,2,1,3;FQ=4.19;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=P88L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10P1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206899.1;VDB=0.0174;DN=120;DA=C/T;GM=NM_206899.1;GL=OR10P1;FG=missense;FD=missense;AAC=PRO/LEU;PP=88/314;CDP=263;GS=98;PH=0.002;CP=0.000;CG=-2.440;AA=C;HA=8.2;HE=39.8;HC=24.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=45099;KP=hsa04740:OlfactoryTransduction;GESP=T:4598/C:8408;PAC=NP_996782.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,3,23 1/1:5:63,16,7 0/0:4:0,3,15</p>

<p>12 58201241 . G A 49.20 midCVG AC1=3;AF1=0.4181;DP=13;DP4=2,4,6,0;FQ=49.8;MQ=60;PV4=0.061,1,1,0.01;SNPEFF_AMINO_ACID_CHANGE=A455V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVIL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006576.3;VDB=0.0142;GM=NM_006576.3,XM_005268575.1;GL=AVIL;FG=missense;FD=unknown;AAC=ALA/VAL;PP=455/820,448/813;CDP=1364,1343;GS=64;PH=1;CP=1.000;CG=4.770;AA=G;DSP=32;GESP=G:13006;PAC=NP_006567.3,XP_005268632.1 GT:GQ:PL 0/1:11:20,0,6 0/0:8:0,9,54 1/1:8:75,12,0 0/0:6:0,6,38</p>

<p>12 70963637 . C T 6.02 midCVG AC1=6;AF1=0.7277;DP=10;DP4=1,0,4,0;FQ=-14.1;MQ=60;PV4=1,0.00024,1,0.38;SNPEFF_AMINO_ACID_CHANGE=G1151E;SNPEFF_CODON_CHANGE=gGa/gAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTPRB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001109754.2;VDB=0.0143;GM=NM_001109754.2,NM_001206971.1,NM_001206972.1,NM_002837.4;GL=PTPRB;FG=missense;FD=unknown;AAC=GLY/GLU;PP=1151/2216,843/1908,933/1908,933/1998;CDP=3452,2528,2798,2798;GS=98;PH=1;CP=0.169;CG=5.300;AA=C;DSP=82;KP=hsa04520:AdherensJunction;GESP=C:12348;PPI=CNTN1:937;PAC=NP_001103224.1,NP_001193900.1,NP_001193901.1,NP_002828.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:33,9,0 0/1:10:8,0,17</p>

<p>12 71523134 rs1051334 A C 65.60 midCVG AC1=3;AF1=0.4149;DB;DP=10;DP4=1,4,1,4;FQ=59.3;MQ=60;PV4=1,1,1,0.024;SNPEFF_AMINO_ACID_CHANGE=S213A;SNPEFF_CODON_CHANGE=Tca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSPAN8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004616.2;VDB=0.0070;DN=86;DA=A/C;GM=NM_004616.2;GL=TSPAN8;FG=missense;FD=missense;AAC=SER/ALA;PP=213/238;CDP=637;GS=99;PH=0;CP=0.992;CG=4.880;AA=C;HA=11.3;HE=40.1;HC=32.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=24;GESP=C:4358/A:8632;PAC=NP_004607.1 GT:GQ:PL 0/1:20:35,0,15 0/0:4:0,3,21 1/1:6:75,9,0 0/0:9:0,9,66</p>

<p>12 100774572 . C A 19.30 lowCVG AC1=3;AF1=0.352;DP=8;DP4=1,3,3,0;FQ=21.1;MQ=60;PV4=0.14,1,1,6.6e-07;SNPEFF_AMINO_ACID_CHANGE=C65*;SNPEFF_CODON_CHANGE=tgC/tgA;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC17A8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_139319.2;VDB=0.0078;GM=NM_001145288.1,NM_139319.2;GL=SLC17A8;FG=stop-gained;FD=unknown;AAC=CYS/stop;PP=65/540,65/590;CDP=195;CP=0.061;CG=-8.160;AA=C;DSP=94;GESP=C:13006;PAC=NP_001138760.1,NP_647480.1 GT:GQ:PL 0/0:8:0,6,45 0/0:3:0,0,0 1/1:4:59,9,0 0/0:8:0,6,42</p>

<p>TSPAN8 12 101560328 rs164365 C A 44.40 midCVG AC1=4;AF1=0.4752;DB;DP=14;DP4=4,4,2,3;FQ=47.1;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=M490I;SNPEFF_CODON_CHANGE=atG/atT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC5A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145913.3;VDB=0.0176;DN=79;DA=C/A;GM=NM_145913.3;GL=SLC5A8;FG=missense;FD=missense;AAC=MET/ILE;PP=490/611;CDP=1470;GS=10;PH=0.002;CP=0.751;CG=2.950;AA=C;HA=11.3;HE=19.6;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57;GESP=A:2529/C:10477;PAC=NP_666018.3 GT:GQ:PL 0/0:10:0,12,72 0/1:11:8,0,55 1/1:6:67,9,0 0/1:15:15,0,15</p>

<p>12 101750341 . G T 60.60 highCVG AC1=3;AF1=0.4122;DP=15;DP4=0,5,0,7;FQ=40.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E1802*;SNPEFF_CODON_CHANGE=Gaa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=42;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UTP20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_014503.2;VDB=0.0074;GM=NM_014503.2;GL=UTP20;FG=stop-gained;FD=unknown;AAC=GLU/stop;PP=1802/2786;CDP=5404;CP=0.981;CG=5.900;AA=G;DSP=10;GESP=G:13006;PAC=NP_055318.2 GT:GQ:PL 0/0:4:0,3,26 0/1:3:0,0,0 1/1:16:103,21,0 0/0:11:0,12,55</p>

<p>SLC5A8 12 103249087 rs77958223 T C 77.80 highCVG AC1=4;AF1=0.4954;DB;DP=15;DP4=5,3,3,4;FQ=80.7;MQ=60;PV4=0.62,0.25,1,0.071;SNPEFF_AMINO_ACID_CHANGE=E178G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000277.1;VDB=0.0177;DN=131;DA=T/A/C;GM=NM_000277.1;GL=PAH;FG=missense;FD=missense;AAC=GLU/GLY;PP=178/453;CDP=533;GS=98;PH=0.961;CP=1.000;CG=5.730;AA=T;DV=by-cluster;CA;DSP=24;KP=hsa00360:PhenylalanineMetabolism|hsa00400:Phenylalanine,TyrosineAndTryptophanBiosynthesis|hsa01100:MetabolicPathways;GESP=T:13006;PAC=NP_000268.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:16,3,0 1/1:15:109,18,0 0/0:19:0,21,126</p>

<p>12 117383320 rs4076700 G A 78.70 midCVG AC1=5;AF1=0.6582;DB;DP=13;DP4=3,1,7,1;FQ=32.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R192Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXW8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153348.2;VDB=0.0037;DN=108;DA=G/A;GM=NM_012174.1,NM_153348.2,XM_005253866.1,XM_005253867.1,XM_005253868.1;GL=FBXW8;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=ARG/GLN,ARG/GLN,ARG/GLN,GLU/LYS,GLU/LYS;PP=126/533,192/599,192/423,41/418,41/418;CDP=377,575,575,121,121;GS=43,43,43,56,56;PH=1;CP=0.990;CG=5.190;AA=G;HA=22.8;HE=17.4;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=A:8670/G:4336;PPI=SKP1:994|CUL7:994|CUL1:932|RBX1:929;PAC=NP_036306.1,NP_699179.2,XP_005253923.1,XP_005253924.1,XP_005253925.1 GT:GQ:PL 0/1:23:18,0,39 0/1:3:0,0,0 1/1:17:93,18,0 0/1:14:9,0,30</p>

<p>12 120805836 . G A 14.20 lowCVG AC1=4;AF1=0.5539;DP=5;DP4=0,2,0,3;FQ=12.9;MQ=60;PV4=1,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=S81L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MSI1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002442.3;VDB=0.0000;GM=NM_002442.3;GL=MSI1;FG=missense;FD=unknown;AAC=SER/LEU;PP=81/363;CDP=242;GS=145;PH=0.539;CP=1.000;CG=3.240;AA=G;DSP=26;TFBS=V$CEBPA_01,V$CEBPB_02,V$CREB_Q2,V$GATA1_01,V$GFI1_01;GESP=G:13006;PAC=NP_002433.1 GT:GQ:PL 0/1:3:0,0,0/1:3:0,0,0 1/1:6:51,9,0 0/0:6:0,6,48</p>

<p>12 125467158 rs11558556 C T 62 highCVG AC1=2;AF1=0.2591;DB;DP=25;DP4=3,16,2,4;FQ=63.3;G3=0.75,7.094e-07,0.25;HWE=0.0457;MQ=60;PV4=0.56,0.034,1,1;SNPEFF_AMINO_ACID_CHANGE=M96I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHX37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032656.3;VDB=0.0142;DN=120;DA=C/G/T;GM=NM_032656.3,XM_005253590.1;GL=DHX37;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=96/1158,96/1176;CDP=288;GS=10;PH=0;CP=1.000;CG=4.050;AA=C;HA=41.2;HE=31.7;HC=38.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12;GESP=T:4715/C:8291;PAC=NP_116045.2,XP_005253647.1 GT:GQ:PL 0/0:9:0,6,41 0/0:32:0,30,147 1/1:11:108,18,0 0/0:23:0,21,121</p>

<p>12 133423662 rs2306536 C T 17.40 lowCVG AC1=5;AF1=0.666;DB;DP=8;DP4=3,0,3,0;FQ=12.8;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V580M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHFR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001161344.1;VDB=0.0028;DN=100;DA=C/T;GM=NM_001161344.1,NM_001161345.1,NM_001161346.1,NM_001161347.1,NM_018223.2,XM_005266170.1;GL=CHFR;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=VAL/MET;PP=580/665,579/664,568/653,488/573,539/624,621/706;CDP=1738,1735,1702,1462,1615,1861;GS=21;PH=0.099;CP=0.012;CG=-2.610;AA=C;HA=31.0;HE=16.7;HC=4.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=34;GESP=T:2911/C:10095;PPI=UBC:937|AURKA:846;PAC=NP_001154816.1,NP_001154817.1,NP_001154818.1,NP_001154819.1,NP_060693.2,XP_005266227.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:24,3,0 0/1:32:28,0,43</p>

<p>12 133803551 rs36146434 G C 26.70 midCVG AC1=4;AF1=0.4988;DB;DP=12;DP4=7,1,2,2;FQ=29.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S230C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANHX;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001191054.1;VDB=0.0086;DN=126;DA=G/C;GM=NM_001191054.1;GL=ANHX;FG=missense;FD=missense;AAC=SER/CYS;PP=230/380;CDP=689;GS=112;PH=0.983;CP=0.001;CG=3.110;AA=G;DV=by-cluster;DSP=30;PAC=NP_001177983.1 GT:GQ:PL 0/1:22:20,0,80 0/1:3:0,0,1/1:4:47,6,0 0/0:5:0,6,44</p>

<p>13 46170609 rs3014939 C T 28.50 lowCVG AC1=3;AF1=0.4056;DB;DP=8;DP4=0,4,4,0;FQ=30.6;MQ=60;PV4=0.029,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=E178K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERICH6B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182542.2;VDB=0.0092;DN=101;DA=C/T;GM=NM_182542.2;GL=FAM194B;FG=missense;FD=missense;AAC=GLU/LYS;PP=178/697;CDP=532;GS=56;PH=0.988;CP=0.002;CG=1.170;AA=C;HA=13.3;HE=49.7;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=106;GESP=T:1855/C:2711;PAC=NP_872348.2 GT:GQ:PL 0/0:4:0,3,26 0/1:22:32,0,17 1/1:3:39,6,0 0/0:6:0,6,44</p>

<p>13 50141345 rs4942848 G A 41.70 midCVG AC1=4;AF1=0.5416;DB;DP=11;DP4=4,1,0,5;FQ=35.5;MQ=60;PV4=0.048,1,1,0.0057;SNPEFF_AMINO_ACID_CHANGE=A24V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RCBTB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018191.3;VDB=0.0072;DN=111;DA=G/A;GM=NM_018191.3,XM_005266440.1,XM_005266441.1,XM_005266442.1;GL=RCBTB1;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/VAL;PP=24/532,24/532,24/532,24/356;CDP=71;GS=64;PH=1;CP=0.989;CG=5.840;AA=G;HA=38.1;HE=32.9;HC=36.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=56;GESP=A:7547/G:5459;PAC=NP_060661.3,XP_005266497.1,XP_005266498.1,XP_005266499.1 GT:GQ:PL 0/1:42:40,0,50 0/1:3:0,0,0 1/1:5:41,6,0 0/0:3:0,3,22</p>

<p>13 51854694 . C A 42.70 highCVG AC1=3;AF1=0.3358;DP=25;DP4=6,7,12,0;FQ=44.4;MQ=60;PV4=0.0052,0.15,1,1;SNPEFF_AMINO_ACID_CHANGE=P351T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM124A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145019.3;VDB=0.0000;GM=NM_001242312.1,NM_145019.3;GL=FAM124A;FG=missense;FD=unknown;AAC=PRO/THR;PP=315/547,351/583;CDP=943,1051;GS=38;PH=0.034;CP=0.000;CG=0.795;AA=C;DSP=109;GESP=C:13006;PAC=NP_001229241.1,NP_659456.3 GT:GQ:PL 0/0:15:0,15,93 0/0:3:0,0,0 1/1:16:87,22,0 0/0:21:0,21,125</p>

<p>13 92345579 rs553717 C T 77.30 highCVG AC1=4;AF1=0.4896;DB;DP=16;DP4=4,6,0,6;FQ=80.1;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A155V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004466.5;VDB=0.0059;DN=83;DA=C/T;GM=NM_004466.4;GL=GPC5;FG=missense;FD=missense;AAC=ALA/VAL;PP=155/573;CDP=464;GS=64;PH=0.013;CP=0.790;CG=4.190;AA=C;HA=19.4;HE=14.8;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=139;GESP=T:1792/C:11214;PAC=NP_004457.1 GT:GQ:PL 0/0:22:0,24,144 0/0:5:0,6,49 1/1:6:66,9,0 1/1:6:61,9,0</p>

<p>13 96540204 rs12876018 T G 17.80 midCVG AC1=3;AF1=0.3733;DB;DP=13;DP4=1,7,2,2;FQ=19.7;MQ=60;PV4=0.24,0.25,1,0.15;SNPEFF_AMINO_ACID_CHANGE=M994L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UGGT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020121.3;VDB=0.0164;DN=121;DA=T/G;GM=NM_020121.3,XM_005254066.1,XM_005254067.1,XM_005254068.1,XM_005254069.1;GL=UGGT2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=MET/LEU;PP=994/1517,992/1515,994/1485,950/1473,640/1163;CDP=2980,2974,2980,2848,1918;GS=15;PH=0.131;CP=0.995;CG=2.860;AA=T;HA=26.0;HE=44.2;HC=28.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4835/T:8169;PPI=BRE:892|BABAM1:871;PAC=NP_064506.3,XP_005254123.1,XP_005254124.1,XP_005254125.1,XP_005254126.1 GT:GQ:PL 0/0:12:0,12,66 0/0:5:0,3,21 1/1:3:31,6,0 0/1:30:30,0,50</p>

<p>13 103384824 rs9300756 G A 50.40 highCVG AC1=3;AF1=0.3894;DB;DP=15;DP4=2,6,5,0;FQ=52.5;MQ=60;PV4=0.021,1,1,0.032;SNPEFF_AMINO_ACID_CHANGE=R6075C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC168;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146197.1;VDB=0.0090;DN=119;DA=G/A;GM=NM_001146197.1;GL=CCDC168;FG=missense;FD=missense;AAC=ARG/CYS;PP=6075/7082;CDP=18223;GS=180;PH=0;CP=0.000;CG=0.010;AA=G;HA=35.0;HE=41.0;HC=16.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17915;GESP=A:2470/G:2094;PAC=NP_001139669.1 GT:GQ:PL 0/0:14:0,15,100 0/0:6:0,6,34 1/1:8:75,12,0 0/1:20:22,0,19</p>

<p>13 109438084 rs911973 T A 59.60 midCVG AC1=5;AF1=0.5535;DB;DP=11;DP4=0,5,1,5;FQ=45;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D203E;SNPEFF_CODON_CHANGE=gaT/gaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198950.1;VDB=0.0067;DN=86;DA=T/A;GM=NM_001198950.1,NM_015011.1;GL=MYO16;FG=missense;FD=unknown;AAC=ASP/GLU;PP=203/1881,181/1859;CDP=609,543;GS=45;PH=0.025;CP=1.000;CG=2.710;AA=T;HA=0.0;HE=2.5;HC=22.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=8;GESP=A:306/T:12700;PAC=NP_001185879.1,NP_055826.1 GT:GQ:PL 0/1:21:18,0,29 0/1:3:0,3,26 1/1:10:75,12,0 0/1:15:11,0,31</p>

<p>13 109661359 rs3825491 C G 999 maxCVG AC1=6;AF1=0.7326;DB;DP=50;DP4=3,3,13,30;FQ=35.7;MQ=60;PV4=0.38,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=P853A;SNPEFF_CODON_CHANGE=Cct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198950.1;VDB=0.0041;DN=107;DA=C/G;GM=NM_001198950.1,NM_015011.1;GL=MYO16;FG=missense;FD=unknown;AAC=PRO/ALA;PP=853/1881,831/1859;CDP=2557,2491;GS=27;PH=0.97;CP=1.000;CG=5.660;AA=C;HA=28.8;HE=39.6;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=39;GESP=G:4727/C:8279;PAC=NP_001185879.1,NP_055826.1 GT:GQ:PL 0/1:18:21,0,19 0/1:55:84,0,56 1/1:97:160,96,0 1/1:8:46,6,0</p>

<p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

<p>14 20404736 rs2792146 G A 143 highCVG AC1=4;AF1=0.4995;DB;DP=17;DP4=3,1,8,3;FQ=42.3;MQ=60;PV4=1,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=R304H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004063.2;VDB=0.0150;DN=100;DA=G/A;GM=NM_001004063.2;GL=OR4K1;FG=missense;FD=missense;AAC=ARG/HIS;PP=304/312;CDP=911;GS=29;PH=0.003;CP=0.000;CG=-4.590;AA=G;HA=23.3;HC=30.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=222291;KP=hsa04740:OlfactoryTransduction;GESP=A:4807/G:8199;PAC=NP_001004063.2 GT:GQ:PL 0/1:34:39,0,31 0/0:3:0,3,28 1/1:21:149,24,0 0/0:3:0,3,25</p>

<p>14 20482919 rs17308108 A C 27.60 highCVG AC1=2;AF1=0.2756;DB;DP=32;DP4=12,11,5,1;FQ=28.9;MQ=60;PV4=0.35,4.3e-07,1,0.024;SNPEFF_AMINO_ACID_CHANGE=L145R;SNPEFF_CODON_CHANGE=cTg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004712.1;VDB=0.0161;DN=123;DA=A/C;GM=NM_001004712.1;GL=OR4K14;FG=missense;FD=missense;AAC=LEU/ARG;PP=145/311;CDP=434;GS=102;PH=0.994;CP=0.198;CG=4.040;AA=A;HA=5.8;HE=10.2;HC=11.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=274987;KP=hsa04740:OlfactoryTransduction;GESP=C:894/A:12112;PAC=NP_001004712.1 GT:GQ:PL 0/0:44:0,42,160 0/0:6:0,3,19 1/1:11:73,18,0 0/0:26:0,24,111</p>

<p>14 20528207 rs1958715 G A 143 highCVG AC1=5;AF1=0.58;DB;DP=15;DP4=3,1,7,4;FQ=17.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D2N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004717.1;VDB=0.0157;DN=92;DA=G/A;GM=NM_001004717.1;GL=OR4L1;FG=missense;FD=missense;AAC=ASP/ASN;PP=2/313;CDP=4;GS=23;PH=0.002;CP=0.786;CG=1.400;HA=45.1;HE=44.5;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=229699;KP=hsa04740:OlfactoryTransduction;GESP=A:5603/G:7401;PAC=NP_001004717.1 GT:GQ:PL 0/1:26:34,0,24 0/1:4:0,3,25 1/1:19:113,21,0 0/1:12:41,0,10</p>

<p>14 20528321 rs1958716 A G 62.10 midCVG AC1=4;AF1=0.5177;DB;DP=12;DP4=2,4,3,3;FQ=65.2;MQ=60;PV4=1,0.028,1,0.098;SNPEFF_AMINO_ACID_CHANGE=M40V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004717.1;VDB=0.0138;DN=92;DA=A/G;GM=NM_001004717.1;GL=OR4L1;FG=missense;FD=missense;AAC=MET/VAL;PP=40/313;CDP=118;GS=21;PH=0;CP=0.001;CG=-5.830;HA=45.1;HE=44.5;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=229585;KP=hsa04740:OlfactoryTransduction;GESP=G:5619/A:7387;PAC=NP_001004717.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,77 1/1:9:79,12,0 0/1:29:26,0,46</p>

<p>14 20897676 rs1953225 T C 48 highCVG AC1=4;AF1=0.4622;DB;DP=33;DP4=22,4,0,5;FQ=50.6;MQ=60;PV4=0.00074,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E345G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHL33;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001109997.2;VDB=0.0074;DN=92;DA=T/C;GM=NM_001109997.2,XM_005267347.1;GL=KLHL33;FG=missense;FD=missense,unknown;AAC=GLU/GLY;PP=345/534,609/798;CDP=1034,1826;GS=98;PH=0.018;CP=0.986;CG=5.340;AA=T;HA=38.4;HE=21.5;HC=25.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;GESP=C:1053/T:3513;PAC=NP_001103467.2,XP_005267404.1 GT:GQ:PL 0/0:67:0,69,127 0/1:3:0,0,0 1/1:6:56,9,0 0/1:37:35,0,47</p>

<p>14 21770730 rs6571751 A G 150 midCVG AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3 GT:GQ:PL 0/1:18:39,0,18 0/1:21:83,0,21 1/1:7:49,6,0 1/1:5:22,3,0</p>

<p>14 39556185 rs8018720 G C 105 midCVG AC1=5;AF1=0.6629;DB;DP=14;DP4=1,3,6,3;FQ=23.7;MQ=59;PV4=0.27,0.14,0.26,0.26;SNPEFF_AMINO_ACID_CHANGE=L211V;SNPEFF_CODON_CHANGE=Ctt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006364.2;VDB=0.0026;DN=116;DA=G/C;GM=NM_006364.2,XM_005267262.1,XM_005267263.1;GL=SEC23A;FG=missense;FD=missense,unknown,unknown;AAC=LEU/VAL;PP=211/766,211/790,182/737;CDP=631,631,544;GS=32;PH=0;CP=0.826;CG=4.770;AA=C;HA=10.8;HE=12.5;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;TFBS=V$MEF2_03;GESP=C:10877/G:2129;PPI=SEC24D:959|SEC24A:900|SEC24C:845;PAC=NP_006355.2,XP_005267319.1,XP_005267320.1 GT:GQ:PL 0/1:3:0,0,0 0/1:43:41,0,46 1/1:12:74,12,0 0/1:15:31,0,14</p>

<p>14 58681951 . T C 41 lowCVG AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1 GT:GQ:PL 0/1:33:31,0,37 0/0:4:0,6,50 1/1:3:26,3,0 1/1:3:27,3,0</p>

<p>14 60903757 rs1254319 G A 70.80 midCVG AC1=6;AF1=0.7392;DB;DP=12;DP4=1,2,1,5;FQ=25.5;MQ=60;PV4=1,0.16,1,0.093;SNPEFF_AMINO_ACID_CHANGE=L524F;SNPEFF_CODON_CHANGE=Ctt/Ttt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C14orf39;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174978.2;VDB=0.0032;DN=87;DA=G/A;GM=NM_174978.2;GL=C14orf39;FG=missense;FD=unknown;AAC=LEU/PHE;PP=524/588;CDP=1570;GS=22;PH=0;CP=0.998;CG=4.030;AA=A;HA=45.3;HE=28.7;HC=36.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=9;GESP=A:4716/G:8288;PAC=NP_777638.2 GT:GQ:PL 0/1:32:26,0,57 0/1:3:0,0,0 1/1:12:82,12,0 0/1:3:0,0,0</p>

<p>14 63174960 rs4902176 C T 17 highCVG AC1=4;AF1=0.4355;DB;DP=15;DP4=2,8,1,3;FQ=19.4;MQ=60;PV4=1,1,1,0.21;SNPEFF_AMINO_ACID_CHANGE=A745T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139318.4;VDB=0.0080;DN=111;DA=C/T;GM=NM_139318.4,NM_172375.2;GL=KCNH5;FG=missense,3-prime-UTR;FD=unknown;AAC=ALA/THR,none;PP=745/989,NA;CDP=2233,NA;GS=58,NA;PH=0.017;CP=0.997;CG=5.520;AA=C;HA=19.0;HE=34.4;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=214;GESP=T:3891/C:9115;PAC=NP_647479.2,NA GT:GQ:PL 0/0:14:0,15,81 0/1:13:12,0,62 1/1:3:46,6,0 0/1:3:0,0,0</p>

<p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

<p>14 74759477 rs4148077 C T 120 midCVG AC1=6;AF1=0.7273;DB;DP=13;DP4=3,0,6,3;FQ=8.59;MQ=60;PV4=0.51,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A304T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005050.3;VDB=0.0175;DN=110;DA=C/T;GM=NM_005050.3,NR_003256.2,XM_005267935.1,XM_005267936.1,XM_005267937.1,XM_005267938.1,XM_005267939.1,XM_005267940.1,XM_005267941.1,XM_005267942.1,XM_005267943.1,XM_005267944.1,XM_005267945.1,XM_005267946.1,XM_005267947.1,XM_005267948.1,XM_005267949.1,XM_005267950.1,XM_005267951.1,XM_005267952.1,XM_005267953.1,XM_005267954.1;GL=ABCD4;FG=missense,non-coding-exon,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR,none,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR;PP=304/607,NA,287/576,304/529,200/489,304/475,166/469,166/469,166/469,166/469,304/455,145/448,145/448,145/448,145/448,145/448,145/448,145/448,41/344,41/344,41/344,41/344;CDP=910,934,859,910,598,910,496,496,496,496,910,433,433,433,433,433,433,433,121,121,121,121;GS=58,NA,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58;PH=0.003;CP=0.998;CG=2.140;AA=C;HA=16.7;HE=29.2;HC=23.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=27;KP=hsa02010:ABCTransporters|hsa04146:Peroxisome;GESP=T:3999/C:9007;PAC=NP_005041.1,NA,XP_005267992.1,XP_005267993.1,XP_005267994.1,XP_005267995.1,XP_005267996.1,XP_005267997.1,XP_005267998.1,XP_005267999.1,XP_005268000.1,XP_005268001.1,XP_005268002.1,XP_005268003.1,XP_005268004.1,XP_005268005.1,XP_005268006.1,XP_005268007.1,XP_005268008.1,XP_005268009.1,XP_005268010.1,XP_005268011.1 GT:GQ:PL 0/1:18:12,0,27 0/1:18:17,0,20 1/1:16:88,15,0 1/1:8:48,6,0</p>

<p>14 75475817 rs145117455 A G 117 highCVG AC1=4;AF1=0.4529;DB;DP=18;DP4=3,4,8,2;FQ=64.3;MQ=60;PV4=0.16,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I328V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EIF2B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014239.3;VDB=0.0051;DN=134;DA=A/G;GM=NM_014239.3;GL=EIF2B2;FG=missense;FD=missense;AAC=ILE/VAL;PP=328/352;CDP=982;GS=29;PH=0.146;CP=1.000;CG=5.770;AA=A;DV=by-frequency,by-cluster;DSP=84;GESP=G:4/A:13002;PAC=NP_055054.1 GT:GQ:PL 0/0:11:0,12,73 0/1:3:0,0,0 1/1:4:44,6,0 0/1:34:114,0,30</p>

<p>14 88459795 rs73312836 C T 22.20 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=3,5,2,1;FQ=24.4;MQ=60;PV4=0.55,1,1,0.005;SNPEFF_AMINO_ACID_CHANGE=A17T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GALC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001201402.1;VDB=0.0092;DN=130;DA=C/T;GM=NM_001201402.1;GL=GALC;FG=missense;FD=missense;AAC=ALA/THR;PP=17/660;CDP=49;GS=58;CP=0.000;CG=-0.125;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=69;KP=hsa00600:SphingolipidMetabolism|hsa01100:MetabolicPathways|hsa04142:Lysosome;PAC=NP_001188331.1 GT:GQ:PL 0/0:15:0,15,85 0/1:4:20,3,0 1/1:3:48,6,0 0/0:9:0,9,64</p>

<p>14 92465749 rs35007347 C T 31.40 highCVG AC1=3;AF1=0.362;DB;DP=18;DP4=9,5,2,2;FQ=33.3;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=R1576H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIP11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004239.3;VDB=0.0065;DN=126;DA=C/T;GM=NM_004239.3,XM_005268213.1,XM_005268214.1,XM_005268215.1;GL=TRIP11;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/HIS;PP=1576/1980,1547/1951,1134/1538,566/970;CDP=4727,4640,3401,1697;GS=29;PH=1;CP=0.895;CG=5.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=T:254/C:12752;PAC=NP_004230.2,XP_005268270.1,XP_005268271.1,XP_005268272.1 GT:GQ:PL 0/0:30:0,30,154 0/1:36:36,0,57 1/1:3:39,6,0 0/0:5:0,3,21</p>

<p>14 94088435 . A G 9.11 lowCVG AC1=7;AF1=0.9732;DP=8;DP4=1,0,4,0;FQ=-25.6;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=E1442G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UNC79;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020818.3;VDB=0.0000;GM=NM_020818.3,XM_005267917.1;GL=UNC79;FG=missense;FD=unknown;AAC=GLU/GLY;PP=1442/2459,1642/2659;CDP=4325,4925;GS=98;PH=1;CP=0.501;CG=5.650;AA=A;DSP=469;GESP=A:13006;PAC=NP_065869.3,XP_005267974.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:14,3,0 0/1:5:27,0,2</p>

<p>14 102815042 rs7011 C T 107 midCVG AC1=5;AF1=0.6413;DB;DP=13;DP4=0,5,3,5;FQ=6.84;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R164H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CINP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032630.2;VDB=0.0109;DN=52;DA=C/T;GM=NM_032630.2,XM_005267753.1;GL=CINP;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=ARG/HIS,none;PP=164/213,NA;CDP=491,NA;GS=29,NA;PH=0.054;CP=0.701;CG=2.760;AA=C;HA=5.8;HE=31.7;HC=11.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=T:3032/C:9974;PAC=NP_116019.1,NA GT:GQ:PL 0/1:3:0,0,0 0/1:12:53,0,11 1/1:6:48,6,0 0/1:31:46,0,30</p>

<p>14 105413204 rs2582514 G T 89 midCVG AC1=6;AF1=0.707;DB;DP=10;DP4=1,2,5,2;FQ=7.98;MQ=55;PV4=0.5,1,0.15,0.061;SNPEFF_AMINO_ACID_CHANGE=R2862S;SNPEFF_CODON_CHANGE=Cgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0059;DN=100;DA=G/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=2862/5796,2762/5696;CDP=8584,8284;GS=110;PH=0;CP=0.000;CG=0.525;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7933;GESP=T:7253/G:4947;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:39:79,0,38</p>

<p>14 105415608 rs11850848 G C 19.70 midCVG AC1=4;AF1=0.4707;DB;DP=12;DP4=8,0,4,0;FQ=22.3;MQ=57;PV4=1,3.7e-05,0.00042,1;SNPEFF_AMINO_ACID_CHANGE=H2060Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0001;DN=120;DA=G/C;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=HIS/GLN;PP=2060/5796,1960/5696;CDP=6180,5880;GS=24;PH=0.022;CP=0.000;CG=-2.470;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=5529;GESP=C:2565/G:6715;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/0:23:0,24,114 1/1:8:59,12,0 0/1:3:0,0,0</p>

<p>14 105418391 rs11625007 C T 60.30 midCVG AC1=5;AF1=0.6662;DB;DP=10;DP4=2,1,0,6;FQ=18.1;MQ=60;PV4=0.083,1,0.26,1;SNPEFF_AMINO_ACID_CHANGE=V1133I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0023;DN=120;DA=C/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=1133/5796,1033/5696;CDP=3397,3097;GS=29;PH=0.017;CP=0.001;CG=-7.740;AA=T;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2746;GESP=T:5916/C:6260;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:50:70,0,48</p>

<p>15 22368862 rs1835183 G A 35.10 lowCVG AC1=4;AF1=0.4751;DB;DP=8;DP4=2,2,3,1;FQ=27.6;MQ=57;PV4=1,1,0.45,0.28;SNPEFF_AMINO_ACID_CHANGE=G96E;SNPEFF_CODON_CHANGE=gGa/gAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4M2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004719.2;VDB=0.0087;DN=92;DA=G/A;GM=NM_001004719.2,NR_015416.1;GL=OR4M2/LOC727924;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=GLY/GLU,none;PP=96/314,NA;CDP=287,1319;GS=98,NA;PH=0.97;CP=0.990;CG=2.500;HE=1.9;HC=28.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=55;KP=hsa04740:OlfactoryTransduction;GESP=A:7271/G:5735;PAC=NP_001004719.2,NA GT:GQ:PL 0/0:8:0,9,53 0/1:3:0,0,0 1/1:4:42,6,0 0/1:17:34,0,13</p>

<p>15 29418520 rs2256273 C T 23.10 lowCVG AC1=5;AF1=0.5568;DB;DP=8;DP4=4,1,3,0;FQ=26.6;MQ=60;PV4=1,0.058,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R393H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM189A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015307.1;VDB=0.0105;DN=100;DA=C/T;GM=NM_015307.1,XM_005254261.1,XM_005254262.1;GL=FAM189A1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=393/540,430/577,252/399;CDP=1178,1289,755;GS=29;PH=0.005;CP=0.000;CG=-2.140;AA=C;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=51;TFBS=V$P300_01;GESP=T:3037/C:1529;PAC=NP_056122.1,XP_005254318.1,XP_005254319.1 GT:GQ:PL 0/1:3:22,3,0 0/1:3:0,0,1/1:5:43,6,0 0/0:12:0,15,102</p>

<p>15 41105926 rs690347 T G 87.70 midCVG AC1=6;AF1=0.706;DB;DP=11;DP4=2,0,2,6;FQ=-18.1;MQ=59;PV4=0.13,1,0.32,1;SNPEFF_AMINO_ACID_CHANGE=S376A;SNPEFF_CODON_CHANGE=Tcc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZFYVE19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001077268.1;VDB=0.0034;DN=83;DA=T/G;GM=NM_001077268.1,NM_001258420.1,NM_001258421.1,NM_032850.4;GL=ZFYVE19;FG=missense;FD=missense;AAC=SER/ALA;PP=376/472,308/404,201/297,366/462;CDP=1126,922,601,1096;GS=99;PH=0;CP=1.000;CG=5.270;AA=G;HA=15.5;HE=28.1;HC=7.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;GESP=G:9475/T:3081;PAC=NP_001070736.1,NP_001245349.1,NP_001245350.1,NP_116239.3 GT:GQ:PL 0/1:6:63,0,4 0/1:3:0,0,0 1/1:7:41,6,0 0/1:11:22,0,11</p>

<p>15 42154738 rs16972193 G A 26.10 midCVG AC1=5;AF1=0.5822;DB;DP=9;DP4=2,0,4,0;FQ=9.38;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R2462W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016642.3;VDB=0.0109;DN=123;DA=G/A;GM=NM_016642.3,XM_005254438.1;GL=SPTBN5;FG=missense-near-splice;FD=missense,unknown;AAC=ARG/TRP;PP=2462/3675,2482/3695;CDP=7384,7444;GS=101;PH=0.213;CP=0.837;CG=-4.960;AA=G;HE=13.8;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=A:1149/G:11369;PAC=NP_057726.4,XP_005254495.1 GT:GQ:PL 0/0:5:0,6,43 0/1:3:0,0,0 1/1:9:64,12,0 0/1:3:0,0,0</p>

<p>15 42570718 rs8024732 A G 106 highCVG AC1=4;AF1=0.501;DB;DP=26;DP4=3,9,5,8;FQ=108;MQ=60;PV4=0.67,0.04,1,1;SNPEFF_AMINO_ACID_CHANGE=Q44R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GANC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198141.2;VDB=0.0051;DN=116;DA=A/G;GM=NM_198141.2,XM_005254286.1;GL=GANC;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=44/915,44/913;CDP=131;GS=43;PH=0;CP=0.454;CG=-2.850;AA=A;HA=31.5;HE=11.0;HC=11.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;KP=hsa00052:GalactoseMetabolism|hsa00500:StarchAndSucroseMetabolism|hsa01100:MetabolicPathways;GESP=G:9331/A:3673;PAC=NP_937784.2,XP_005254343.1 GT:GQ:PL 0/1:13:11,0,15 0/0:11:0,14,130 1/1:12:59,15,0 0/1:18:83,0,15</p>

<p>15 44120559 rs678084 T G 24.40 lowCVG AC1=4;AF1=0.5372;DB;DP=7;DP4=3,1,3,0;FQ=27.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S153A;SNPEFF_CODON_CHANGE=Tcc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024908.3;VDB=0.0062;DN=83;DA=T/G;GM=NM_001167941.1,NM_024908.3,XM_005254680.1,XM_005254681.1;GL=WDR76;FG=missense;FD=missense,missense,unknown,unknown;AAC=SER/ALA;PP=89/563,153/627,89/563,153/473;CDP=265,457,265,457;GS=99;PH=0;CP=0.524;CG=2.540;AA=G;HA=30.1;HE=20.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=G:10282/T:2710;PAC=NP_001161413.1,NP_079184.2,XP_005254737.1,XP_005254738.1 GT:GQ:PL 0/1:19:20,0,19 0/0:7:0,9,58 1/1:3:23,3,0 1/1:3:25,3,0</p>

<p>15 48057210 . G A 92.30 maxCVG AC1=3;AF1=0.3739;DP=51;DP4=2,8,39,2;FQ=94.3;MQ=60;PV4=2.9e-06,2.1e-06,1,1;SNPEFF_AMINO_ACID_CHANGE=D462N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEMA6D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153618.1;VDB=0.0007;GM=NM_001198999.1,NM_020858.1,NM_024966.2,NM_153616.1,NM_153617.1,NM_153618.1,NM_153619.1,XM_005254685.1,XM_005254686.1,XM_005254687.1,XM_005254688.1,XM_005254689.1,XM_005254690.1;GL=SEMA6D;FG=missense;FD=unknown;AAC=ASP/ASN;PP=462/1012,462/1012,462/477,462/999,462/1018,462/1074,462/598,462/1087,462/1087,462/1074,462/1068,462/1055,462/1031;CDP=1384;GS=23;PH=0.993;CP=1.000;CG=5.790;AA=G;DSP=44;KP=hsa04360:AxonGuidance;GESP=G:12990;PAC=NP_001185928.1,NP_065909.1,NP_079242.2,NP_705869.1,NP_705870.1,NP_705871.1,NP_705872.1,XP_005254742.1,XP_005254743.1,XP_005254744.1,XP_005254745.1,XP_005254746.1,XP_005254747.1 GT:GQ:PL 0/0:9:0,9,67 0/0:12:0,12,80 1/1:99:134,120,0 0/1:7:5,0,60</p>

<p>15 65676726 rs33918653 C T 78.50 midCVG AC1=4;AF1=0.5302;DB;DP=10;DP4=0,4,5,1;FQ=24.8;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C1125Y;SNPEFF_CODON_CHANGE=tGc/tAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0003;DN=126;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=CYS/TYR;PP=1125/1251;CDP=3374;GS=194;PH=0.981;CP=1.000;CG=4.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=T:2295/C:10705;PAC=NP_066013.1 GT:GQ:PL 0/0:5:0,6,38 0/1:3:0,0,0 1/1:7:70,9,0 0/1:28:50,0,25</p>

<p>15 65684513 rs148990036 C T 29.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,3,0;FQ=21.1;MQ=59;PV4=0.4,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R694Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0000;DN=134;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=ARG/GLN;PP=694/1251;CDP=2081;GS=43;PH=0.968;CP=0.939;CG=5.490;AA=C;DV=by-frequency,by-cluster;DSP=42;GESP=T:88/C:12878;PAC=NP_066013.1 GT:GQ:PL 0/0:9:0,9,56 0/1:3:0,0,0 1/1:6:67,9,0 0/1:3:0,0,0</p>

<p>15 81598269 rs11556218 T G 14.60 lowCVG AC1=2;AF1=0.3027;DB;DP=7;DP4=3,1,1,2;FQ=16;MQ=60;PV4=0.49,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=N1147K;SNPEFF_CODON_CHANGE=aaT/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_172217.3;VDB=0.0093;DN=120;DA=T/G;GM=NM_001172128.1,NM_004513.5,NM_172217.3,XM_005254342.1,XM_005254343.1,XM_005254344.1,XM_005254345.1,XM_005254346.1;GL=IL16;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=ASN/LYS;PP=1147/1332,446/632,1147/1333,1194/1380,1194/1379,537/723,537/722,446/632;CDP=3441,1338,3441,3582,3582,1611,1611,1338;GS=94;PH=1;CP=0.007;CG=-6.300;AA=T;HA=25.0;HE=10.0;HC=20.6;DG;DV=by-frequency,by-cluster,by-1000G;DSP=21;TFBS=V$YY1_01;GESP=G:1601/T:11405;PPI=CD4:846;PAC=NP_001165599.1,NP_004504.3,NP_757366.2,XP_005254399.1,XP_005254400.1,XP_005254401.1,XP_005254402.1,XP_005254403.1 GT:GQ:PL 0/0:6:0,3,19 0/0:8:0,6,35 1/1:4:55,9,0 0/0:6:0,3,26</p>

<p>15 90174824 rs12900805 C T 23.70 midCVG AC1=6;AF1=0.8319;DB;DP=9;DP4=1,0,1,2;FQ=-24.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G1005R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198525.2;VDB=0.0106;DN=121;DA=C/T;GM=NM_198525.2,XM_005254902.1,XM_005254903.1,XM_005254904.1;GL=KIF7;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ARG;PP=1005/1344,1005/1344,1004/1343,1004/1343;CDP=3013,3013,3010,3010;GS=125;PH=0.004;CP=0.067;CG=-0.473;AA=C;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=99;TFBS=V$SREBP1_02;GESP=T:6862/C:6054;PAC=NP_940927.2,XP_005254959.1,XP_005254960.1,XP_005254961.1 GT:GQ:PL 0/1:3:0,0,0/1:6:29,0,4 1/1:4:28,3,0 0/1:3:0,0,0</p>

<p>16 715990 rs7190775 G A 37.20 lowCVG AC1=5;AF1=0.6489;DB;DP=5;DP4=0,1,4,0;FQ=-10.8;MQ=60;PV4=0.2,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R1492H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR90;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145294.4;VDB=0.0001;DN=116;DA=G/A;GM=NM_145294.4,XM_005255160.1,XM_005255161.1;GL=WDR90;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=1492/1749,1495/1752,1494/1751;CDP=4475,4484,4481;GS=29;PH=0.169;CP=0.000;CG=-4.830;AA=G;HA=11.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;GESP=A:5864/G:6472;PAC=NP_660337.3,XP_005255217.1,XP_005255218.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,22 1/1:10:74,12,0 0/1:3:0,0,0</p>

<p>16 840378 rs3765263 A G 66.70 midCVG AC1=3;AF1=0.3829;DB;DP=14;DP4=6,0,8,0;FQ=58.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K244R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHTF18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022092.2;VDB=0.0000;DN=107;DA=A/G;GM=NM_022092.2,XM_005255470.1,XM_005255471.1,XM_005255472.1;GL=CHTF18;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=LYS/ARG,LYS/ARG,LYS/ARG,none;PP=244/976,128/860,272/1004,NA;CDP=731,383,815,NA;GS=26,26,26,NA;PH=0;CP=0.081;CG=-5.390;AA=G;HA=33.2;HE=22.3;HC=42.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4000/A:8074;PPI=PCNA:939|RFC4:846|DSCC1:845|RFC3:845|RFC2:845;PAC=NP_071375.1,XP_005255527.1,XP_005255528.1,NA GT:GQ:PL 0/0:12:0,12,60 0/1:3:0,0,0 1/1:19:110,24,0 0/0:7:0,6,40</p>

<p>16 1838640 rs344359 T C 5.70 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,3,0;FQ=-28.6;MQ=60;SNPEFF_AMINO_ACID_CHANGE=W99R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284502.1;VDB=0.0025;DN=79;DA=T/C;GM=NM_012225.2,XM_005255025.1,XM_005255026.1,XM_005255027.1,XM_005255028.1,XM_005255029.1,XM_005255030.1;GL=NUBP2;FG=synonymous;FD=synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO;PP=247/272,201/226,187/212,187/212,187/212,106/131,106/131;CDP=741,603,561,561,561,318,318;CP=0.022;CG=0.180;AA=C;HA=3.6;HE=20.7;HC=32.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;GESP=C:11162/T:1824;PAC=NP_036357.1,XP_005255082.1,XP_005255083.1,XP_005255084.1,XP_005255085.1,XP_005255086.1,XP_005255087.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:25,6,0 1/1:4:13,3,0</p>

<p>16 2854507 rs61747737 T G 133 highCVG AC1=4;AF1=0.4823;DB;DP=35;DP4=6,1,23,2;FQ=81.4;MQ=60;PV4=0.54,1,0.3,1;SNPEFF_AMINO_ACID_CHANGE=F233C;SNPEFF_CODON_CHANGE=tTt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS41;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135086.1;VDB=0.0087;DN=129;DA=T/G;GM=NM_001135086.1;GL=PRSS41;FG=missense;FD=missense;AAC=PHE/CYS;PP=233/319;CDP=698;GS=205;PH=0.99;CP=0.167;CG=4.230;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=41;GESP=G:264/T:4302;PAC=NP_001128558.1 GT:GQ:PL 0/0:7:0,9,62 0/1:3:24,3,0 1/1:48:158,52,0 0/0:7:0,9,65</p>

<p>16 2905689 rs199712440 G A 23.60 lowCVG AC1=3;AF1=0.4317;DB;DP=7;DP4=2,1,2,1;FQ=25.1;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R149C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022119.3;VDB=0.0096;DN=137;DA=G/A;GM=NM_022119.3,XM_005255473.1;GL=PRSS22;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=149/318;CDP=445;GS=180;PH=0.99;CP=0.006;CG=3.430;AA=G;DV=by-cluster;DSP=115;TFBS=V$PAX4_01;GESP=A:1/G:12995;PAC=NP_071402.1,XP_005255530.1 GT:GQ:PL 0/0:4:0,3,24 0/0:4:0,3,29 1/1:4:50,6,0 0/1:16:15,0,19</p>

<p>16 5139186 rs12928528 G C 82.20 highCVG AC1=4;AF1=0.5055;DB;DP=16;DP4=2,5,2,6;FQ=74.9;MQ=54;PV4=1,0.19,1,0.18;SNPEFF_AMINO_ACID_CHANGE=H272D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM86A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_201400.3;VDB=0.0194;DN=121;DA=G/C;GM=NM_201400.2,NM_201598.2,XM_005255157.1,XM_005255158.1,XM_005255159.1;GL=FAM86A;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ASP;PP=272/331,238/297,245/304,172/231,172/231;CDP=814,712,733,514,514;GS=81;PH=0;CP=0.000;CG=-2.010;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=72;GESP=C:3557/G:4861;PAC=NP_958802.1,NP_963892.1,XP_005255214.1,XP_005255215.1,XP_005255216.1 GT:GQ:PL 0/0:5:0,6,25 0/0:13:0,15,97 1/1:15:88,18,0 1/1:5:44,6,0</p>

<p>16 10837913 rs2233531 C G 30.10 highCVG AC1=3;AF1=0.3384;DB;DP=23;DP4=2,15,4,1;FQ=31.8;MQ=60;PV4=0.0093,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P39A;SNPEFF_CODON_CHANGE=Ccg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUBP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002484.3;VDB=0.0101;DN=98;DA=C/G;GM=NM_001278506.1,NM_002484.3;GL=NUBP1;FG=missense;FD=unknown,missense;AAC=PRO/ALA;PP=39/310,39/321;CDP=115;GS=27;PH=0.002;CP=0.509;CG=3.830;AA=C;HA=15.8;HE=10.0;HC=2.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;CPG=10837585-10838196;GESP=G:1334/C:11630;PAC=NP_001265435.1,NP_002475.2 GT:GQ:PL 0/0:3:0,0,0 0/0:5:0,3,23 1/1:4:70,9,0 0/0:18:0,17,85</p>

<p>16 12897578 rs3748976 G T 56.70 midCVG AC1=5;AF1=0.5742;DB;DP=11;DP4=1,2,2,3;FQ=7.33;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A19D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018340.2;VDB=0.0072;DN=107;DA=G/T;GM=NM_001099455.1,NM_018340.2;GL=CPPED1;FG=missense;FD=missense;AAC=ALA/ASP;PP=19/173,19/315;CDP=56;GS=126;PH=0;CP=1.000;CG=4.810;AA=T;HA=26.9;HE=11.4;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;CPG=12897366-12897889;GESP=T:3654/G:8256;PAC=NP_001092925.1,NP_060810.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,16 1/1:7:69,9,0 0/1:27:28,0,27</p>

<p>16 15111218 rs4985162 C T 42.50 highCVG AC1=5;AF1=0.5528;DB;DP=27;DP4=8,9,4,3;FQ=46;MQ=32;PV4=1,1,0.18,1;SNPEFF_AMINO_ACID_CHANGE=P301L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PDXDC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015027.3;VDB=0.0062;DN=111;DA=C/T;GM=NM_015027.2,XM_005255173.1,XM_005255174.1,XM_005255175.1,XM_005255176.1,XM_005255177.1,XM_005255178.1,XM_005255179.1,XM_005255180.1,XM_005255181.1;GL=PDXDC1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=301/789,319/807,310/798,300/788,286/774,286/774,282/770,274/762,273/761,210/698;CDP=902,956,929,899,857,857,845,821,818,629;GS=98;PH=0.015;CP=0.484;CG=4.490;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;TFBS=V$PPARA_01;GESP=T:2213/C:10775;PAC=NP_055842.2,XP_005255230.1,XP_005255231.1,XP_005255232.1,XP_005255233.1,XP_005255234.1,XP_005255235.1,XP_005255236.1,XP_005255237.1,XP_005255238.1 GT:GQ:PL 0/0:35:0,39,138 0/1:18:14,0,36 1/1:10:62,12,0 1/1:3:14,3,0</p>

<p>16 15131962 rs1135999 A G 59.20 highCVG AC1=5;AF1=0.6381;DB;DP=21;DP4=3,1,1,13;FQ=9.29;MQ=57;PV4=0.019,0.29,0.23,1;SNPEFF_AMINO_ACID_CHANGE=S287P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0035;DN=86;DA=A/G;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=SER/PRO;PP=182/206,182/206,287/311;CDP=544,544,859;GS=74;PH=0;CP=1.000;CG=5.870;AA=G;HA=13.3;HE=27.7;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;TFBS=V$FREAC3_01,V$STAT1_01,V$STAT3_01,V$STAT5A_01,V$STAT5B_01;GESP=G:3550/A:9444;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/0:4:0,6,44 0/1:3:0,0,0 1/1:5:23,6,0 0/1:3:76,2,0</p>

<p>16 15131974 rs1136001 G T 95.50 highCVG AC1=6;AF1=0.7476;DB;DP=17;DP4=2,0,2,12;FQ=-10.1;MQ=57;PV4=0.05,0.05,0.28,1;SNPEFF_AMINO_ACID_CHANGE=H283N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0015;DN=86;DA=G/T;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=HIS/ASN;PP=178/206,178/206,283/311;CDP=532,532,847;GS=68;PH=0;CP=0.000;CG=1.840;AA=T;HA=13.4;HE=27.5;HC=41.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=94;GESP=T:3575/G:9419;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/1:5:0,3,25 0/1:3:0,0,0 1/1:5:20,3,0 1/1:16:116,16,0</p>

<p>16 55562466 rs837550 G A 132 highCVG AC1=5;AF1=0.5658;DB;DP=21;DP4=11,0,4,5;FQ=111;MQ=60;PV4=0.0081,1,1,0.063;SNPEFF_AMINO_ACID_CHANGE=M163I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LPCAT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017839.4;VDB=0.0146;DN=86;DA=G/A;GM=NM_017839.4,XM_005256006.1;GL=LPCAT2;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=163/545,163/331;CDP=489;GS=10;PH=0;CP=0.006;CG=-3.800;AA=G;HA=38.7;HE=35.2;HC=41.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00564:GlycerophospholipidMetabolism|hsa00565:EtherLipidMetabolism|hsa01100:MetabolicPathways;GESP=A:7296/G:5700;PAC=NP_060309.2,XP_005256063.1 GT:GQ:PL 0/1:14:10,0,42 0/0:20:0,24,113 1/1:3:31,3,0 1/1:19:139,21,0</p>

<p>16 56673227 rs11640851 C A 13.60 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=2,0,1,1;FQ=13.6;MQ=60;PV4=1,0.23,1,1;SNPEFF_AMINO_ACID_CHANGE=T27N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MT1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005946.2;VDB=0.0011;DN=120;DA=C/A;GM=NM_005946.2;GL=MT1A;FG=missense;FD=missense;AAC=THR/ASN;PP=27/62;CDP=80;GS=65;PH=0.928;CP=0.839;CG=1.960;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=A:6838/C:6158;PAC=NP_005937.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:49,6,0 0/0:7:0,6,49</p>

<p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

<p>16 70690989 rs8046424 G C 61.30 lowCVG AC1=5;AF1=0.603;DB;DP=7;DP4=0,2,5,0;FQ=10.6;MQ=60;PV4=0.048,0.29,1,1;SNPEFF_AMINO_ACID_CHANGE=E123Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL34;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172772.1;VDB=0.0024;DN=116;DA=G/C;GM=NM_001172771.1,NM_001172772.1,NM_152456.2,XM_005255810.1;GL=IL34;FG=missense;FD=missense,missense,missense,unknown;AAC=GLU/GLN;PP=122/242,123/243,123/243,123/243;CDP=364,367,367,367;GS=29;PH=0;CP=0.115;CG=3.040;AA=C;HA=0.0;HE=45.8;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=C:8157/G:4839;PPI=CSF1R:959;PAC=NP_001166242.1,NP_001166243.1,NP_689669.2,XP_005255867.1 GT:GQ:PL 0/1:3:0,3,24 0/1:3:0,3,27 1/1:5:44,6,0 1/1:8:62,9,0</p>

<p>16 75563330 rs3826107 G A 36.20 lowCVG AC1=5;AF1=0.6482;DB;DP=5;DP4=0,1,4,0;FQ=-9.91;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T318M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHST5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024533.4;VDB=0.0001;DN=107;DA=G/A;GM=NM_024533.4,XM_005255880.1;GL=CHST5;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=318/412,324/418;CDP=953,971;GS=81;PH=0.999;CP=0.994;CG=2.840;AA=G;HA=9.7;HE=15.6;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=906;CPG=75563114-75564112;GESP=A:2238/G:10758;PAC=NP_078809.2,XP_005255937.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:10:73,12,0 0/1:3:0,0,0</p>

<p>16 89350038 rs2279348 G A 53.40 lowCVG AC1=6;AF1=0.6958;DB;DP=6;DP4=0,2,1,3;FQ=8.26;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A971V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256182.1;VDB=0.0084;DN=100;DA=G/A;GM=NM_001256182.1,NM_001256183.1,NM_013275.5;GL=ANKRD11;FG=missense;FD=missense;AAC=ALA/VAL;PP=971/2664;CDP=2912;GS=64;PH=0.014;CP=0.002;CG=-3.500;AA=G;HA=40.8;HE=27.5;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2020;GESP=A:9057/G:3937;PAC=NP_001243111.1,NP_001243112.1,NP_037407.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,44 1/1:6:46,6,0 1/1:6:49,6,0</p>

<p>17 649547 rs910925 G C 39.60 midCVG AC1=4;AF1=0.5306;DB;DP=10;DP4=2,1,4,0;FQ=11.2;MQ=60;PV4=0.43,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=A579G;SNPEFF_CODON_CHANGE=gCc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GEMIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015721.2;VDB=0.0040;DN=86;DA=G/C;GM=NM_015721.2,XM_005256667.1,XM_005256668.1,XM_005256669.1,XM_005256670.1;GL=GEMIN4;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=579/1059,583/1063,583/1063,568/1048,568/1048;CDP=1736,1748,1748,1703,1703;GS=60;PH=1;CP=0.997;CG=5.840;AA=G;HA=11.7;HE=47.5;HC=33.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1726;GESP=C:8046/G:4332;PPI=DDX20:973|SMN1:904|SMN2:898|AGO2:885;PAC=NP_056536.2,XP_005256724.1,XP_005256725.1,XP_005256726.1,XP_005256727.1 GT:GQ:PL 0/0:3:0,3,24 0/0:5:0,6,28 1/1:7:59,9,0 0/1:3:25,3,0</p>

<p>17 663472 rs77166377 C A 77.80 highCVG AC1=3;AF1=0.3934;DB;DP=18;DP4=4,8,3,3;FQ=79.9;MQ=59;PV4=0.63,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A280S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GLOD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016080.3;VDB=0.0035;DN=131;DA=C/A;GM=NM_016080.3;GL=GLOD4;FG=missense;FD=missense;AAC=ALA/SER;PP=280/299;CDP=838;GS=99;PH=0.103;CP=0.999;CG=5.050;AA=C;DV=by-frequency,by-cluster;DSP=7;GESP=A:318/C:12688;PAC=NP_057164.3 GT:GQ:PL 0/0:6:0,6,40 0/1:18:17,0,31 1/1:10:108,15,0 0/0:23:0,24,101</p>

<p>17 1690752 rs11549830 G A 24.30 lowCVG AC1=3;AF1=0.3495;DB;DP=8;DP4=2,3,3,0;FQ=26.1;MQ=60;PV4=0.2,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R562W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMYD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052928.2;VDB=0.0000;DN=120;DA=G/A;GM=NM_052928.2,XM_005256435.1,XM_005256436.1;GL=SMYD4;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=562/805,562/680,562/636;CDP=1684;GS=101;PH=0.997;CP=0.266;CG=2.540;AA=G;HA=45.0;HE=30.0;HC=45.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=37;GESP=A:4552/G:8438;PAC=NP_443160.2,XP_005256492.1,XP_005256493.1 GT:GQ:PL 0/0:13:0,12,73 0/0:3:0,0,0 1/1:4:64,9,0 0/0:5:0,3,21</p>

<p>17 5058808 rs9899177 G A 14.40 midCVG AC1=4;AF1=0.4652;DB;DP=9;DP4=1,5,0,3;FQ=17.1;MQ=56;PV4=1,1,6.1e-05,0.17;SNPEFF_AMINO_ACID_CHANGE=R912Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004505.2;VDB=0.0065;DN=119;DA=G/A;GM=NM_004505.2,XM_005256845.1,XM_005256846.1;GL=USP6;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=912/1407,912/1407,595/1090;CDP=2735,2735,1784;GS=43;PH=0.999;CP=1.000;CG=2.910;AA=G;HA=15.2;HE=33.3;HC=10.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT1_01;GESP=A:6766/G:6240;PAC=NP_004496.2,XP_005256902.1,XP_005256903.1 GT:GQ:PL 0/0:8:0,9,57 0/1:3:0,0,0 1/1:4:41,6,0 0/1:15:14,0,40</p>

<p>17 5087040 rs9908414 A G 60.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=4,4,0,5;FQ=63.2;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I171T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF594;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032530.1;VDB=0.0105;DN=119;DA=A/G;GM=NM_032530.1,XM_005256827.1;GL=ZNF594;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=171/808;CDP=512;GS=89;PH=0;CP=0.203;CG=-2.720;AA=A;HA=9.7;HE=47.3;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=532;GESP=A:12386;PAC=NP_115919.1,XP_005256884.1 GT:GQ:PL 0/0:16:0,18,107 0/0:6:0,6,37 1/1:6:70,9,0 1/1:4:39,6,0</p>

<p>17 7250481 . C A 31.70 midCVG AC1=5;AF1=0.6147;DP=10;DP4=3,0,5,0;FQ=-5.48;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=C421*;SNPEFF_CODON_CHANGE=tgC/tgA;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACAP1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_014716.3;VDB=0.0000;GM=NM_014716.3,XM_005256878.1;GL=ACAP1;FG=stop-gained;FD=unknown;AAC=CYS/stop;PP=421/741,347/667;CDP=1263,1041;CP=0.931;CG=-5.370;AA=C;DSP=81;KP=hsa04144:Endocytosis;GESP=C:13006;PAC=NP_055531.1,XP_005256935.1 GT:GQ:PL 0/1:19:17,0,20 0/1:3:0,0,1/1:4:25,3,0 0/1:12:28,0,10</p>

<p>ZNF594 17 7951819 rs4792147 A G 25 lowCVG AC1=5;AF1=0.6327;DB;DP=8;DP4=2,1,3,1;FQ=27.7;MQ=60;PV4=1,0.054,1,1;SNPEFF_AMINO_ACID_CHANGE=Q656R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALOX15B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001141.2;VDB=0.0083;DN=111;DA=A/C/G;GM=NM_001039130.1,NM_001039131.1,NM_001141.2,XM_005256591.1;GL=ALOX15B;FG=missense;FD=missense,missense,missense,unknown;AAC=GLN/ARG;PP=627/648,582/603,656/677,644/665;CDP=1880,1745,1967,1931;GS=43;PH=0.001;CP=0.017;CG=-1.690;AA=G;HA=7.1;HE=47.0;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=116;KP=hsa00590:ArachidonicAcidMetabolism|hsa01100:MetabolicPathways;GESP=G:7866/A:5140;PAC=NP_001034219.1,NP_001034220.1,NP_001132.2,XP_005256648.1 GT:GQ:PL 0/0:6:0,9,70 0/1:3:0,0,0 1/1:8:52,9,0 1/1:3:15,3,0</p>

<p>17 8216468 rs871841 T C 58.40 lowCVG AC1=5;AF1=0.6499;DB;DP=8;DP4=0,3,2,3;FQ=10.5;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L277P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025014.1;VDB=0.0003;DN=86;DA=T/C;GM=NM_025014.1,NM_173728.3,XM_005256536.1;GL=ARHGEF15;FG=missense;FD=missense,missense,unknown;AAC=LEU/PRO;PP=277/842,277/842,277/778;CDP=830;GS=98;PH=0;CP=0.995;CG=5.130;AA=C;HA=29.6;HE=47.3;HC=26.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=105;GESP=C:7328/T:5678;PAC=NP_079290.1,NP_776089.2,XP_005256593.1 GT:GQ:PL 0/0:6:0,9,48 0/1:3:0,0,0 1/1:11:83,12,0 1/1:4:18,3,0</p>

<p>17 10223697 rs2074876 G T 61.70 lowCVG AC1=5;AF1=0.5935;DB;DP=8;DP4=1,1,5,1;FQ=9.15;MQ=60;PV4=0.46,0.34,1,0.18;SNPEFF_AMINO_ACID_CHANGE=D1076E;SNPEFF_CODON_CHANGE=gaC/gaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYH13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003802.2;VDB=0.0194;DN=96;DA=G/T;GM=NM_003802.2,XM_005256843.1;GL=MYH13;FG=missense;FD=missense,unknown;AAC=ASP/GLU;PP=1076/1939;CDP=3228;GS=45;PH=0.001;CP=1.000;CG=3.680;AA=G;HA=12.7;HE=8.7;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=30;KP=hsa04530:TightJunction|hsa05416:ViralMyocarditis;GESP=T:1345/G:10557;PAC=NP_003793.2,XP_005256900.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,3,23 1/1:3:27,3,0 1/1:13:79,15,0</p>

<p>17 13980350 rs2072279 G A 65.90 highCVG AC1=4;AF1=0.4922;DB;DP=16;DP4=0,8,6,0;FQ=68.8;MQ=60;PV4=0.00033,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R159Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COX10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001303.3;VDB=0.0150;DN=96;DA=G/A;GM=NM_001303.3,XM_005256458.1,XM_005256459.1;GL=COX10;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=159/444,159/240,159/238;CDP=476;GS=43;PH=0.178;CP=0.605;CG=5.490;AA=G;HA=37.0;HE=44.2;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=24;KP=hsa00190:OxidativePhosphorylation|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa01100:MetabolicPathways;GESP=A:6378/G:6628;PAC=NP_001294.2,XP_005256515.1,XP_005256516.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:25,3,0 1/1:12:88,15,0 0/0:19:0,21,111</p>

<p>17 19861458 rs2108978 C T 98.40 highCVG AC1=3;AF1=0.3904;DB;DP=18;DP4=7,4,1,6;FQ=101;MQ=60;PV4=0.066,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R249H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AKAP10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007202.3;VDB=0.0017;DN=96;DA=C/T;GM=NM_007202.3;GL=AKAP10;FG=missense;FD=missense;AAC=ARG/HIS;PP=249/663;CDP=746;GS=29;PH=0;CP=0.995;CG=2.360;AA=T;HA=44.6;HE=39.4;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=132;TFBS=V$GATA1_03;GESP=T:5787/C:7219;PPI=PDZK1:959|PRKAR1A:784;PAC=NP_009133.2 GT:GQ:PL 0/0:9:0,9,75 0/0:11:0,12,64 1/1:10:115,15,0 0/1:32:31,0,84</p>

<p>17 21318867 rs73979893 G A 999 highCVG AC1=5;AF1=0.6076;DB;DP=37;DP4=5,10,16,5;FQ=193;MQ=60;PV4=0.017,0.45,1,0.069;SNPEFF_AMINO_ACID_CHANGE=M71I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNJ12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021012.4;VDB=0.0094;DN=130;DA=G/A;GM=NM_021012.4,XM_005256625.1;GL=KCNJ12;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=71/434;CDP=213;GS=10;PH=0;CP=1.000;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=269;CPG=21318651-21320041;TFBS=V$MEIS1AHOXA9_01,V$OCT1_06;PPI=DLG1:972|CASK:833|LIN7B:833|LIN7A:833|APBA1:833|LIN7C:833|DLG4:833;PAC=NP_066292.2,XP_005256682.1 GT:GQ:PL 0/1:54:49,0,91 0/1:35:62,0,34 1/1:8:120,9,0 0/1:59:54,0,108</p>

<p>17 26694861 rs704 G A 31.80 highCVG AC1=4;AF1=0.4318;DB;DP=15;DP4=3,7,1,3;FQ=34;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T400M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000638.3;VDB=0.0089;DN=36;DA=G/A;GM=NM_000638.3;GL=VTN;FG=missense;FD=missense;AAC=THR/MET;PP=400/479;CDP=1199;GS=81;PH=0.943;CP=0.000;CG=-8.940;AA=G;HA=35.0;HE=46.9;HC=25.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=126;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=A:6627/G:6379;PAC=NP_000629.3 GT:GQ:PL 0/0:12:0,12,67 0/1:3:0,0,0 1/1:5:69,9,0 0/1:5:3,0,95</p>

<p>17 27284443 rs12941884 A G 58.10 midCVG AC1=5;AF1=0.587;DB;DP=9;DP4=2,0,4,1;FQ=5.4;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=M806T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEZ6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178860.4;VDB=0.0169;DN=121;DA=A/G;GM=NM_001098635.1,NM_178860.4,XM_005257919.1;GL=SEZ6;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=806/994,806/995,681/870;CDP=2417,2417,2042;GS=81;PH=0;CP=0.999;CG=3.650;AA=G;HA=20.8;HE=14.6;HC=8.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=78;GESP=G:1806/A:10402;PAC=NP_001092105.1,NP_849191.3,XP_005257976.1 GT:GQ:PL 0/1:3:0,0,0 0/1:18:21,0,17 1/1:10:78,12,0 0/1:3:0,3,24</p>

<p>17 33585770 . G C 33.60 midCVG AC1=2;AF1=0.3076;DP=14;DP4=6,4,4,0;FQ=35.1;MQ=60;PV4=0.25,1,1,0;SNPEFF_AMINO_ACID_CHANGE=V21L;SNPEFF_CODON_CHANGE=Gtc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLFN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144975.3;VDB=0.0001;GM=NM_144975.3,XM_005257934.1;GL=SLFN5;FG=missense;FD=unknown;AAC=VAL/LEU;PP=21/892,21/390;CDP=61;GS=32;PH=0.936;CP=0.006;CG=-3.630;AA=G;DSP=101;GESP=G:13006;PAC=NP_659412.3,XP_005257991.1 GT:GQ:PL 0/0:19:0,18,118 0/0:3:0,0,1/1:6:76,12,0 0/0:13:0,12,76</p>

<p>17 37093510 rs113358400 T C 115 highCVG AC1=4;AF1=0.5505;DB;DP=17;DP4=3,1,5,7;FQ=13.6;MQ=60;PV4=0.57,0.38,1,0.38;SNPEFF_AMINO_ACID_CHANGE=N426S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXO47;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008777.2;VDB=0.0129;DN=132;DA=T/C;GM=NM_001008777.2;GL=FBXO47;FG=missense;FD=missense;AAC=ASN/SER;PP=426/453;CDP=1277;GS=46;PH=0.208;CP=0.984;CG=4.810;AA=T;DV=by-frequency,by-cluster;DSP=34;GESP=C:292/T:12714;PAC=NP_001008777.2 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,36 1/1:10:124,12,0 0/1:19:33,0,16</p>

<p>17 38121993 rs3894194 G A 35.30 lowCVG AC1=6;AF1=0.7326;DB;DP=5;DP4=1,0,2,2;FQ=-12.4;MQ=60;PV4=1,0.21,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R18Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSDMA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178171.4;VDB=0.0024;DN=108;DA=G/A;GM=NM_178171.4;GL=GSDMA;FG=missense;FD=missense;AAC=ARG/GLN;PP=18/446;CDP=53;GS=43;PH=0.016;CP=1.000;CG=3.340;AA=G;HA=28.6;HE=47.7;HC=46.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=58;GESP=A:4966/G:7324;PAC=NP_835465.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,22 1/1:9:54,9,0 1/1:4:21,3,0</p>

<p>17 38936659 rs17558560 C T 89.80 highCVG AC1=4;AF1=0.4888;DB;DP=16;DP4=3,3,0,9;FQ=51.7;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G193S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT27;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181537.3;VDB=0.0046;DN=123;DA=C/T;GM=NM_181537.3;GL=KRT27;FG=missense;FD=missense;AAC=GLY/SER;PP=193/460;CDP=577;GS=56;PH=0.152;CP=0.032;CG=3.410;AA=C;HA=11.4;HE=45.7;HC=39.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=50;TFBS=V$CDPCR3HD_01;GESP=T:4441/C:8565;PAC=NP_853515.2 GT:GQ:PL 0/0:10:0,12,63 0/0:5:0,6,34 1/1:21:112,24,0 0/1:3:26,3,0</p>

<p>17 39983808 rs1046404 G C 71.20 lowCVG AC1=6;AF1=0.6722;DB;DP=8;DP4=1,1,1,5;FQ=6.96;MQ=60;PV4=0.46,0.14,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S213C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0127;DN=86;DA=G/C;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/CYS,none,none;PP=213/301,NA,NA;CDP=638,784,940;GS=112,NA,NA;PH=0.991;CP=0.812;CG=5.070;AA=G;HA=20.0;HE=24.2;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=71;GESP=C:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:18,0,20 0/1:4:0,3,26 1/1:9:55,9,0 1/1:7:43,6,0</p>

<p>17 39983820 rs1046403 G A 58.40 lowCVG AC1=5;AF1=0.6474;DB;DP=7;DP4=1,1,1,4;FQ=7.67;MQ=60;PV4=1,0.17,1,0.19;SNPEFF_AMINO_ACID_CHANGE=A209V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0201;DN=86;DA=G/A;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ALA/VAL,none,none;PP=209/301,NA,NA;CDP=626,772,928;GS=64,NA,NA;PH=0.001;CP=0.060;CG=2.870;AA=G;HA=20.8;HE=23.8;HC=17.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;GESP=A:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:17,0,20 0/1:4:0,3,26 1/1:9:64,9,0 1/1:4:21,3,0</p>

<p>17 41361960 rs8482 A G 76.10 midCVG AC1=4;AF1=0.5419;DB;DP=12;DP4=0,4,2,5;FQ=27.5;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H923R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NBR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005899.4;VDB=0.0215;DN=52;DA=A/G;GM=NM_005899.3,NM_031858.2,NM_031862.2;GL=NBR1;FG=missense;FD=missense;AAC=HIS/ARG;PP=923/967;CDP=2768;GS=29;PH=0.968;CP=1.000;CG=5.860;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;GESP=G:3148/A:7152;PPI=SQSTM1:994|MAP1LC3A:937|UBC:757|MAP1LC3C:756;PAC=NP_005890.2,NP_114064.1,NP_114068.1 GT:GQ:PL 0/1:9:6,0,31 0/1:3:0,0,0 1/1:15:112,18,0 0/0:5:0,6,35</p>

<p>17 46847364 rs2032844 C A 24.80 lowCVG AC1=4;AF1=0.5405;DB;DP=5;DP4=1,1,1,2;FQ=9.57;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E712D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTLL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130918.1;VDB=0.0135;DN=94;DA=C/A;GM=NM_001130918.1,NM_173623.3;GL=TTLL6;FG=missense;FD=missense;AAC=GLU/ASP;PP=712/892,405/585;CDP=2136,1215;GS=45;PH=0.999;CP=0.002;CG=-3.840;AA=C;HA=24.4;HE=29.9;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=96;GESP=A:3281/C:9725;PAC=NP_001124390.1,NP_775894.2 GT:GQ:PL 0/0:3:0,3,24 0/1:3:20,3,0 1/1:4:47,6,0 0/0:3:0,3,24</p>

<p>17 48598785 rs1380657 G A 174 highCVG AC1=6;AF1=0.7336;DB;DP=19;DP4=3,2,5,7;FQ=44.9;MQ=60;PV4=0.62,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=A424T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYCBPAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032133.4;VDB=0.0177;DN=88;DA=G/A;GM=NM_032133.4,XM_005257725.1,XM_005257726.1,XM_005257727.1;GL=MYCBPAP;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=424/985,439/1001,439/1000,424/986;CDP=1270,1315,1315,1270;GS=58;PH=0;CP=0.000;CG=-4.290;AA=A;HA=7.6;HE=27.6;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=A:10167/G:2839;PPI=MYCBP:846;PAC=NP_115509.4,XP_005257782.1,XP_005257783.1,XP_005257784.1 GT:GQ:PL 0/1:14:16,0,16 0/1:38:31,0,68 1/1:22:130,21,0 1/1:8:42,6,0</p>

<p>17 48629458 rs8065903 A G 42.80 lowCVG AC1=5;AF1=0.6395;DB;DP=8;DP4=3,0,3,2;FQ=9.67;MQ=60;PV4=0.46,1,1,0.071;SNPEFF_AMINO_ACID_CHANGE=K625R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022827.3;VDB=0.0000;DN=116;DA=A/G;GM=NM_001258372.1,NM_001258373.1,NM_022827.3,XM_005257591.1;GL=SPATA20;FG=missense;FD=missense,missense,missense,unknown;AAC=LYS/ARG;PP=609/787,565/743,625/803,609/787;CDP=1826,1694,1874,1826;GS=26;PH=0;CP=0.996;CG=1.610;AA=G;HA=0.4;HE=23.2;HC=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;TFBS=V$GR_Q6,V$OLF1_01;GESP=G:10556/A:2450;PAC=NP_001245301.1,NP_001245302.1,NP_073738.2,XP_005257648.1 GT:GQ:PL 0/1:4:0,3,20 0/1:12:7,0,28 1/1:8:57,9,0 1/1:4:22,3,0</p>

<p>17 56233034 rs7218964 C A 103 midCVG AC1=5;AF1=0.6659;DB;DP=10;DP4=0,2,6,2;FQ=-6.33;MQ=60;PV4=0.13,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=L174I;SNPEFF_CODON_CHANGE=Cta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012374.1;VDB=0.0134;DN=116;DA=C/A;GM=NM_012374.1;GL=OR4D1;FG=missense;FD=missense;AAC=LEU/ILE;PP=174/311;CDP=520;GS=5;PH=0.329;CP=0.984;CG=5.630;AA=C;HA=48.5;HE=22.7;HC=12.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=37250;KP=hsa04740:OlfactoryTransduction;GESP=A:3410/C:9596;PAC=NP_036506.1 GT:GQ:PL 0/1:3:0,0,0 0/1:17:36,0,16 1/1:12:76,12,0 0/1:16:32,0,15</p>

<p>17 56388238 rs2680704 C G 16.90 highCVG AC1=3;AF1=0.3661;DB;DP=16;DP4=6,4,4,0;FQ=18.7;MQ=60;PV4=0.25,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=A1140P;SNPEFF_CODON_CHANGE=Gca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BZRAP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004758.3;VDB=0.0162;DN=100;DA=C/G;GM=NM_001261835.1,NM_004758.3,NM_024418.2,XM_005257807.1,XM_005257808.1,XM_005257809.1,XM_005257810.1;GL=BZRAP1;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/PRO;PP=1140/1849,1140/1858,1080/1798,1140/1899,1133/1892,1080/1839,990/1749;CDP=3418,3418,3238,3418,3397,3238,2968;GS=27;PH=0;CP=0.000;CG=-1.050;AA=G;HA=22.5;HE=38.3;HC=21.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42;GESP=G:3461/C:9395;PAC=NP_001248764.1,NP_004749.2,NP_077729.1,XP_005257864.1,XP_005257865.1,XP_005257866.1,XP_005257867.1 GT:GQ:PL 0/0:5:0,3,17 0/0:28:0,27,161 1/1:7:58,12,0 0/0:3:0,0,0</p>

<p>17 56388303 rs3744099 T A 41.60 midCVG AC1=2;AF1=0.3053;DB;DP=10;DP4=2,4,3,1;FQ=43.1;MQ=60;PV4=0.52,0.25,1,0.057;SNPEFF_AMINO_ACID_CHANGE=H1118L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BZRAP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004758.3;VDB=0.0175;DN=107;DA=T/A;GM=NM_001261835.1,NM_004758.3,NM_024418.2,XM_005257807.1,XM_005257808.1,XM_005257809.1,XM_005257810.1;GL=BZRAP1;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown;AAC=HIS/LEU;PP=1118/1849,1118/1858,1058/1798,1118/1899,1111/1892,1058/1839,968/1749;CDP=3353,3353,3173,3353,3332,3173,2903;GS=99;PH=0.05;CP=0.000;CG=-11.500;AA=T;HA=0.0;HE=6.0;HC=10.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=107;TFBS=V$MYOGNF1_01,V$PAX5_02;GESP=A:660/T:12322;PAC=NP_001248764.1,NP_004749.2,NP_077729.1,XP_005257864.1,XP_005257865.1,XP_005257866.1,XP_005257867.1 GT:GQ:PL 0/0:6:0,3,26 0/0:14:0,12,84 1/1:6:84,12,0 0/0:6:0,3,25</p>

<p>17 59544932 . C A 13.30 midCVG AC1=3;AF1=0.3359;DP=11;DP4=2,6,0,3;FQ=14.9;MQ=60;PV4=1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=P155T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018488.2;VDB=0.0039;GM=NM_018488.2,XM_005257835.1,XM_005257836.1,XM_005257837.1,XM_005257838.1;GL=TBX4;FG=missense;FD=unknown;AAC=PRO/THR;PP=155/546,174/566,174/565,155/547,155/547;CDP=463,520,520,463,463;GS=38;PH=1;CP=1.000;CG=5.770;AA=C;DSP=62;GESP=C:13006;PAC=NP_060958.2,XP_005257892.1,XP_005257893.1,XP_005257894.1,XP_005257895.1 GT:GQ:PL 0/0:6:0,3,19 0/0:3:0,0,0 1/1:4:53,9,0 0/0:22:0,21,120</p>

<p>17 66246416 rs3213690 A G 90.80 midCVG AC1=5;AF1=0.6141;DB;DP=10;DP4=1,1,3,5;FQ=-8.56;MQ=59;PV4=1,1,0.32,1;SNPEFF_AMINO_ACID_CHANGE=N30D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AMZ2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001033569.1;VDB=0.0143;DN=106;DA=A/G;GM=NM_001033569.1,NM_001033570.1,NM_001033571.1,NM_001033572.1,NM_001033574.1,NM_016627.4,XM_005257432.1,XM_005257433.1,XM_005257434.1,XM_005257435.1,XM_005257436.1,XM_005257437.1,XM_005257438.1;GL=AMZ2;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ASN/ASP;PP=30/361,30/361,30/361,30/361,30/303,30/361,94/425,30/361,30/361,30/361,30/361,30/361,94/266;CDP=88,88,88,88,88,88,280,88,88,88,88,88,280;GS=23;PH=0;CP=0.009;CG=1.320;AA=G;HA=2.0;HE=37.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=G:9868/A:3138;PAC=NP_001028741.1,NP_001028742.1,NP_001028743.1,NP_001028744.1,NP_001028746.1,NP_057711.3,XP_005257489.1,XP_005257490.1,XP_005257491.1,XP_005257492.1,XP_005257493.1,XP_005257494.1,XP_005257495.1 GT:GQ:PL 0/1:4:0,3,19 0/1:3:0,0,0 1/1:16:107,18,0 0/1:12:25,0,10</p>

<p>17 72346868 rs2382644 C T 25.90 lowCVG AC1=5;AF1=0.6043;DB;DP=8;DP4=1,2,1,2;FQ=24.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R471W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153209.3;VDB=0.0128;DN=100;DA=C/T;GM=NM_153209.3,XM_005257029.1,XM_005257030.1;GL=KIF19;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=471/999,471/1000,429/958;CDP=1411,1411,1285;GS=101;PH=0.999;CP=0.000;CG=-11.500;DG;DV=by-frequency,by-cluster,by-1000G;DSP=23;TFBS=V$HOX13_01;GESP=T:1007/C:11997;PAC=NP_694941.2,XP_005257086.1,XP_005257087.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,63 1/1:3:17,3,0 1/1:5:50,6,0</p>

<p>17 73886888 rs3760128 A G 19.20 highCVG AC1=3;AF1=0.3332;DB;DP=24;DP4=12,3,1,3;FQ=20.8;MQ=60;PV4=0.071,0.1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=L509P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM65;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173547.3;VDB=0.0187;DN=107;DA=A/G;GM=NM_001256124.1,NM_173547.3;GL=TRIM65;FG=missense;FD=missense;AAC=LEU/PRO;PP=487/496,509/518;CDP=1460,1526;GS=98;PH=0;CP=0.001;CG=3.690;AA=G;HA=7.2;HE=33.3;HC=28.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=541;GESP=G:6549/A:6433;PAC=NP_001243053.1,NP_775818.2 GT:GQ:PL 0/0:3:0,0,0 0/0:8:0,6,42 1/1:6:61,12,0 0/0:40:0,39,144</p>

<p>17 74006201 . C T 36.70 highCVG AC1=2;AF1=0.2266;DP=18;DP4=9,4,0,4;FQ=37.8;MQ=60;PV4=0.029,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D1029N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EVPL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001988.2;VDB=0.0002;GM=NM_001988.2,XM_005257143.1;GL=EVPL;FG=missense;FD=unknown;AAC=ASP/ASN;PP=1029/2034,1051/2056;CDP=3085,3151;GS=23;PH=0.964;CP=0.992;CG=5.340;AA=C;DSP=424;GESP=C:13004;PPI=PPL:937;PAC=NP_001979.2,XP_005257200.1 GT:GQ:PL 0/0:12:0,9,55 0/0:15:0,12,74 1/1:5:81,12,0 0/0:21:0,18,102</p>

<p>17 74151720 rs201646108 C T 56.70 highCVG AC1=3;AF1=0.3694;DB;DP=26;DP4=6,12,3,3;FQ=58.7;MQ=59;PV4=0.63,1,0.041,0.25;SNPEFF_AMINO_ACID_CHANGE=A541T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RNF157;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052916.2;VDB=0.0188;DN=137;DA=C/T;GM=NM_052916.2,XM_005257006.1,XM_005257007.1,XM_005257008.1;GL=RNF157;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=541/680,541/658,541/654,474/613;CDP=1621,1621,1621,1420;GS=58;PH=1;CP=0.755;CG=5.430;AA=C;DV=by-cluster;DSP=78;GESP=T:5/C:13001;PAC=NP_443148.1,XP_005257063.1,XP_005257064.1,XP_005257065.1 GT:GQ:PL 0/0:9:0,9,60 0/1:29:29,0,78 1/1:7:75,12,0 0/0:24:0,24,130</p>

<p>17 76433899 rs61742072 G A 107 midCVG AC1=5;AF1=0.5795;DB;DP=14;DP4=6,0,2,6;FQ=60.9;MQ=60;PV4=0.0097,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=H3953Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=74;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH17;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173628.3;VDB=0.0018;DN=129;DA=G/A;GM=NM_173628.3;GL=DNAH17;FG=missense;FD=missense;AAC=HIS/TYR;PP=3953/4463;CDP=11857;GS=83;PH=0;CP=0.997;CG=2.960;AA=G;DV=by-frequency,by-cluster;DSP=52;GESP=A:1641/G:11359;PAC=NP_775899.3 GT:GQ:PL 0/1:21:20,0,22 0/0:11:0,15,83 1/1:4:26,3,0 1/1:16:109,18,0</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>17 79634774 rs61742303 A T 27.20 midCVG AC1=2;AF1=0.2865;DB;DP=13;DP4=4,5,1,2;FQ=28.6;MQ=60;PV4=1,1,0.041,0.3;SNPEFF_AMINO_ACID_CHANGE=K50N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC137;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199287.2;VDB=0.0222;DN=129;DA=A/T;GM=NM_199287.2;GL=CCDC137;FG=missense;FD=unknown;AAC=LYS/ASN;PP=50/290;CDP=150;GS=94;PH=0.982;CP=0.997;CG=0.759;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=16;GESP=T:2092/A:9860;PAC=NP_954981.1 GT:GQ:PL 0/0:20:0,18,97 0/0:4:0,0,0 1/1:4:68,9,0 0/0:11:0,9,59</p>

<p>17 80086389 rs7406163 C T 19.80 lowCVG AC1=4;AF1=0.4474;DB;DP=6;DP4=0,2,0,3;FQ=3.11;MQ=60;PV4=1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=D776N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC57;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198082.2;VDB=0.0077;DN=116;DA=C/T;GM=NM_198082.2,XM_005256343.1,XM_005256344.1;GL=CCDC57;FG=missense;FD=missense,unknown,unknown;AAC=ASP/ASN;PP=776/916,888/1033,888/1028;CDP=2326,2662,2662;GS=23;PH=0;CP=0.000;CG=-3.210;AA=C;HA=49.1;HE=44.6;HC=9.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=21;GESP=T:7026/C:5548;PAC=NP_932348.2,XP_005256400.1,XP_005256401.1 GT:GQ:PL 0/0:4:0,3,22 0/1:3:0,0,0 1/1:5:58,9,0 0/0:4:0,3,14</p>

<p>18 3215158 rs1791085 C G 46.10 lowCVG AC1=3;AF1=0.4432;DB;DP=8;DP4=1,2,5,0;FQ=30.6;MQ=60;PV4=0.11,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V22L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYOM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003803.3;VDB=0.0017;DN=89;DA=C/G;GM=NM_003803.3,NM_019856.1;GL=MYOM1;FG=missense;FD=missense;AAC=VAL/LEU;PP=22/1686,22/1590;CDP=64;GS=32;PH=0;CP=0.984;CG=5.670;AA=G;HA=13.4;HE=8.7;HC=15.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;GESP=G:1230/C:11464;PAC=NP_003794.3,NP_062830.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,22 1/1:10:87,15,0 0/0:6:0,6,47</p>

<p>18 5956238 rs3737353 C T 87.70 highCVG AC1=3;AF1=0.4121;DB;DP=16;DP4=7,0,2,5;FQ=85.9;MQ=60;PV4=0.021,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G618D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=L3MBTL4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173464.3;VDB=0.0033;DN=107;DA=C/T;GM=NM_173464.3,XM_005258165.1,XM_005258166.1,XM_005258167.1;GL=L3MBTL4;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ASP;PP=618/624,618/624,609/615,588/594;CDP=1853,1853,1826,1763;GS=94;PH=0.053;CP=0.010;CG=2.780;AA=C;HA=33.1;HE=38.3;HC=46.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=149;GESP=T:4306/C:7788;PAC=NP_775735.2,XP_005258222.1,XP_005258223.1,XP_005258224.1 GT:GQ:PL 0/0:11:0,12,74 0/1:18:19,0,17 1/1:13:116,18,0 0/0:6:0,6,42</p>

<p>18 11610052 rs74410464 T C 70.60 highCVG AC1=5;AF1=0.6397;DB;DP=15;DP4=3,5,2,4;FQ=70.8;MQ=55;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V153A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=T/C;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=VAL/ALA;PP=153/339;CDP=458;GS=64;CP=0.000;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79887;PAC=XP_934053.1 GT:GQ:PL 0/1:10:5,0,68 0/1:3:0,0,0 1/1:11:93,12,0 0/1:18:13,0,41</p>

<p>18 11610073 rs75301503 G A 72.40 midCVG AC1=4;AF1=0.5075;DB;DP=14;DP4=2,5,3,3;FQ=75.1;MQ=53;PV4=0.59,1,0.44,0.48;SNPEFF_AMINO_ACID_CHANGE=C160Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=G/A;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=CYS/TYR;PP=160/339;CDP=479;GS=194;CP=0.000;AA=G;DG;DV=by-cluster,by-1000G;DSP=79866;PAC=XP_934053.1 GT:GQ:PL 0/0:10:0,12,79 0/1:3:0,0,1/1:12:100,15,0 0/1:19:16,0,48</p>

<p>18 11610163 rs62084100 G C 19.60 highCVG AC1=4;AF1=0.4585;DB;DP=15;DP4=4,4,3,3;FQ=22.1;MQ=47;PV4=1,0.45,0.072,0.13;SNPEFF_AMINO_ACID_CHANGE=G190A;SNPEFF_CODON_CHANGE=gGc/gCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0171;DN=129;DA=G/C/T;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=GLY/ALA;PP=190/339;CDP=569;GS=60;CP=0.105;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79776;PAC=XP_934053.1 GT:GQ:PL 0/0:6:0,6,100 0/1:3:0,0,0 1/1:8:59,12,0 0/0:3:0,1,17</p>

<p>18 14779969 rs9675365 C G 147 highCVG AC1=4;AF1=0.5542;DB;DP=24;DP4=0,8,5,11;FQ=15.9;MQ=55;PV4=0.13,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F477L;SNPEFF_CODON_CHANGE=ttC/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145029.1;VDB=0.0092;DN=119;DA=C/G;GM=NM_001145029.1;GL=ANKRD30B;FG=missense;FD=missense;AAC=PHE/LEU;PP=477/1393;CDP=1431;GS=22;PH=0.286;CP=0.000;CG=-2.400;AA=C;HA=41.2;HE=48.8;HC=36.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=11;GESP=C:4566;PAC=NP_001138501.1 GT:GQ:PL 0/0:4:0,6,34 0/1:5:92,0,1 1/1:5:69,6,0 0/1:22:29,0,20</p>

<p>18 28986333 rs4799570 A C 83.20 midCVG AC1=5;AF1=0.5962;DB;DP=10;DP4=3,1,3,3;FQ=24.2;MQ=60;PV4=0.57,1,1,0.43;SNPEFF_AMINO_ACID_CHANGE=I644L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DSG4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001134453.1;VDB=0.0057;DN=111;DA=A/C;GM=NM_001134453.1,NM_177986.3;GL=DSG4;FG=missense;FD=missense;AAC=ILE/LEU;PP=644/1060,644/1041;CDP=1930;GS=5;PH=0;CP=0.762;CG=4.770;AA=C;HA=0.0;HE=11.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=4;GESP=C:12022/A:984;PAC=NP_001127925.1,NP_817123.1 GT:GQ:PL 0/1:3:0,3,13 0/1:3:0,0,0 1/1:10:93,12,0 0/1:35:31,0,48</p>

<p>18 29339975 . C T 29.30 lowCVG AC1=3;AF1=0.4714;DP=7;DP4=2,1,0,4;FQ=-8.34;MQ=24;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G227D;SNPEFF_CODON_CHANGE=gGt/gAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A52;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001034172.2;VDB=0.0001;GM=NM_001034172.2;GL=SLC25A52;FG=missense;FD=unknown;AAC=GLY/ASP;PP=227/308;CDP=680;GS=94;PH=0.006;CP=0.077;CG=-1.260;AA=C;DSP=35519;GESP=C:13006;PAC=NP_001029344.3 GT:GQ:PL 0/0:4:0,3,22 0/1:3:0,0,0 1/1:8:69,12,0 0/0:5:0,6,7</p>

<p>18 29797921 rs616114 C T 34.30 midCVG AC1=3;AF1=0.4037;DB;DP=9;DP4=1,3,0,5;FQ=36.2;MQ=60;PV4=0.44,0.28,1,1;SNPEFF_AMINO_ACID_CHANGE=P695L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MEP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005925.2;VDB=0.0113;DN=83;DA=C/T;GM=NM_005925.2,XM_005258275.1;GL=MEP1B;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=695/702,695/701;CDP=2084;GS=98;PH=0;CP=0.004;CG=0.414;AA=C;HA=45.9;HE=37.9;HC=26.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=8;GESP=T:4754/C:7364;PAC=NP_005916.2,XP_005258332.1 GT:GQ:PL 0/0:7:0,6,40 0/1:3:0,0,0 1/1:10:76,15,0 0/0:7:0,6,47</p>

<p>18 43497710 rs3744998 A G 87.20 highCVG AC1=5;AF1=0.6466;DB;DP=18;DP4=6,1,2,6;FQ=33.7;MQ=60;PV4=0.041,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=V1058A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPG5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020964.2;VDB=0.0029;DN=107;DA=A/G;GM=NM_020964.2,XM_005258323.1,XM_005258324.1;GL=EPG5;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=1058/2580,1058/2579,703/2225;CDP=3173,3173,2108;GS=64;PH=0.884;CP=1.000;CG=5.400;AA=A;HA=26.5;HE=41.1;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=67;GESP=G:5759/A:6247;PAC=NP_066015.2,XP_005258380.1,XP_005258381.1 GT:GQ:PL 0/1:36:35,0,38 0/1:3:0,0,0 1/1:9:44,9,0 0/1:33:49,0,32</p>

<p>18 65179829 rs2279269 G A 25.80 lowCVG AC1=4;AF1=0.4987;DB;DP=8;DP4=2,2,1,2;FQ=27;MQ=60;PV4=1,1,1,0.04;SNPEFF_AMINO_ACID_CHANGE=P683S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DSEL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032160.2;VDB=0.0085;DN=100;DA=G/A;GM=NM_032160.2;GL=DSEL;FG=missense;FD=missense;AAC=PRO/SER;PP=683/1223;CDP=2047;GS=74;PH=0;CP=0.987;CG=2.920;AA=A;HA=29.6;HE=47.7;HC=50.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=2901;GESP=A:5871/G:7135;PAC=NP_115536.1 GT:GQ:PL 0/1:19:18,0,23 0/1:3:0,0,0 1/1:4:48,6,0 0/0:5:0,6,46</p>

<p>18 72103782 rs2278154 T C 999 highCVG AC1=5;AF1=0.603;DB;DP=20;DP4=5,1,8,6;FQ=35.4;MQ=60;PV4=0.35,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q405R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM69C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001044369.2;VDB=0.0039;DN=100;DA=T/C;GM=NM_001044369.2,XM_005266647.1;GL=FAM69C;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=405/420,264/279;CDP=1214,791;GS=43;PH=0;CP=0.021;CG=4.080;AA=C;HA=27.7;HE=36.4;HC=29.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=173;GESP=C:4387/T:7701;PAC=NP_001037834.2,XP_005266704.1 GT:GQ:PL 0/1:3:0,0,0 0/1:53:53,0,54 1/1:28:173,30,0 0/1:4:0,3,19</p>

<p>19 1010629 . G A 33.70 lowCVG AC1=6;AF1=0.8319;DP=6;DP4=1,0,4,0;FQ=-24.4;MQ=60;PV4=1,1,1,0.043;SNPEFF_AMINO_ACID_CHANGE=A528V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM259;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001033026.1;VDB=0.0000;GM=NM_001033026.1,NM_033420.3,XM_005259675.1,XM_005259676.1,XM_005259677.1,XM_005259678.1,XM_005259679.1;GL=TMEM259;FG=missense,3-prime-UTR,missense,missense,missense,missense,missense;FD=unknown;AAC=ALA/VAL,none,ALA/VAL,ALA/VAL,ALA/VAL,ALA/VAL,ALA/VAL;PP=528/621,NA,569/662,563/656,552/645,534/627,348/441;CDP=1583,NA,1706,1688,1655,1601,1043;GS=64,NA,64,64,64,64,64;PH=0.99;CP=0.057;CG=3.370;DSP=266;CPG=1007552-1012222;GESP=G:8630;PAC=NP_001028198.1,NA,XP_005259732.1,XP_005259733.1,XP_005259734.1,XP_005259735.1,XP_005259736.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:26,3,0 0/1:6:41,0,4</p>

<p>19 1068738 rs1801284 G A 31.20 lowCVG AC1=6;AF1=0.749;DB;DP=5;DP4=0,1,2,2;FQ=-19.7;MQ=60;PV4=1,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R155H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HMHA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001258328.2;VDB=0.0157;DN=89;DA=G/A;GM=NM_001258328.1,NM_012292.3;GL=HMHA1;FG=missense;FD=missense;AAC=ARG/HIS;PP=155/1153,139/1137;CDP=464,416;GS=29;PH=0.008;CP=0.000;CG=-8.440;AA=A;HA=45.1;HE=34.4;HC=41.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;CPG=1068439-1068764;GESP=A:5253/G:7735;PAC=NP_001245257.1,NP_036424.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:60,9,0 0/1:10:7,0,11</p>

<p>19 4512669 rs55901236 T C 66.40 midCVG AC1=6;AF1=0.6769;DB;DP=10;DP4=0,2,6,1;FQ=-15;MQ=60;PV4=0.083,1,1,0.31;SNPEFF_AMINO_ACID_CHANGE=T421A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080400.1;VDB=0.0122;DN=129;DA=T/C;GM=NM_001080400.1,XM_005259640.1,XM_005259641.1;GL=PLIN4;FG=missense;FD=missense,unknown,unknown;AAC=THR/ALA;PP=421/1358,436/1373,435/1372;CDP=1261,1306,1303;GS=58;PH=0.009;CP=0.001;CG=0.087;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1045;TFBS=V$GR_Q6;GESP=C:5321/T:7027;PAC=NP_001073869.1,XP_005259697.1,XP_005259698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:17:76,0,15</p>

<p>19 4538904 . G A 14.10 lowCVG AC1=5;AF1=0.6551;DP=7;DP4=0,2,3,0;FQ=-10.9;MQ=60;PV4=0.1,1,1,2.3e-05;SNPEFF_AMINO_ACID_CHANGE=A31V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052972.2;VDB=0.0085;GM=NM_052972.2;GL=LRG1;FG=missense;FD=unknown;AAC=ALA/VAL;PP=31/348;CDP=92;GS=64;PH=0.006;CP=0.003;CG=0.951;AA=G;DSP=60;GESP=G:13006;PAC=NP_443204.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,25 1/1:5:40,6,0 1/1:3:14,3,0</p>

<p>19 6374890 rs7540 G A 55.70 midCVG AC1=4;AF1=0.4563;DB;DP=10;DP4=5,0,3,1;FQ=47.1;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R191Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALKBH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032306.3;VDB=0.0086;DN=52;DA=G/A;GM=NM_032306.3,XM_005259658.1;GL=ALKBH7;FG=missense;FD=missense,unknown;AAC=ARG/GLN;PP=191/222,165/196;CDP=572,494;GS=43;PH=0.275;CP=0.021;CG=3.030;AA=G;HA=0.0;HE=10.3;HC=2.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=69;TFBS=V$OLF1_01;GESP=A:862/G:12144;PAC=NP_115682.1,XP_005259715.1 GT:GQ:PL 0/1:18:43,0,14 0/1:3:0,0,0 1/1:4:54,6,0 0/0:11:0,12,72</p>

<p>19 8997482 . C T 52.20 highCVG AC1=5;AF1=0.5614;DP=30;DP4=3,9,6,12;FQ=55.8;MQ=39;PV4=0.7,1,3.5e-17,1;SNPEFF_AMINO_ACID_CHANGE=R13647Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0008;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=ARG/GLN;PP=13647/14508,5561/6405,5561/6025,442/1303;CDP=40940,16682,16682,1325;GS=43;PH=0.984;CP=0.000;CG=-5.720;AA=C;DSP=55;GESP=C:12276;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/1:3:0,3,59 0/1:17:15,0,19 1/1:13:69,15,0 0/1:17:13,0,77</p>

<p>19 8997507 . G A 30.10 highCVG AC1=3;AF1=0.3713;DP=29;DP4=8,10,2,8;FQ=32.1;MQ=43;PV4=0.25,0.045,0.00017,0.46;SNPEFF_AMINO_ACID_CHANGE=L13639F;SNPEFF_CODON_CHANGE=Ctc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0023;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=LEU/PHE;PP=13639/14508,5553/6405,5553/6025,434/1303;CDP=40915,16657,16657,1300;GS=22;PH=0.984;CP=0.046;CG=-2.050;AA=G;DSP=30;GESP=G:12308;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/0:21:0,21,114 0/1:16:16,0,29 1/1:7:61,12,0 0/0:8:0,8,111</p>

<p>19 9024994 rs67631215 C T 90.10 highCVG AC1=5;AF1=0.6778;DB;DP=16;DP4=2,2,11,0;FQ=-17;MQ=54;PV4=0.057,0.07,1,0.16;SNPEFF_AMINO_ACID_CHANGE=E12290K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0020;DN=130;DA=C/T;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=GLU/LYS;PP=12290/14508,4204/6405,4204/6025;CDP=36868,12610,12610;GS=56;PH=0.009;CP=0.004;CG=1.510;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=5;GESP=T:4021/C:7885;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:3:0,0,0 0/1:9:41,0,8 1/1:6:33,6,0 0/1:10:55,0,9</p>

<p>19 9056982 rs11670318 G A 109 midCVG AC1=5;AF1=0.6629;DB;DP=11;DP4=1,2,6,2;FQ=5.06;MQ=60;PV4=0.49,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T10155I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0045;DN=120;DA=G/A;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=THR/ILE;PP=10155/14508,2069/6405,2069/6025;CDP=30464,6206,6206;GS=89;PH=0.997;CP=0.000;CG=0.467;AA=G;HA=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=810;GESP=A:3367/G:8779;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:30:47,0,29 0/1:3:0,0,0 1/1:9:61,9,0 0/1:11:41,0,10</p>

<p>19 9069792 rs1862458 G A 52.70 highCVG AC1=4;AF1=0.4499;DB;DP=16;DP4=1,9,4,2;FQ=55.2;MQ=60;PV4=0.036,0.29,1,1;SNPEFF_AMINO_ACID_CHANGE=S5885F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0050;DN=92;DA=G/A;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=SER/PHE;PP=5885/14508;CDP=17654;GS=155;PH=0.995;CP=0.002;CG=1.870;AA=G;HA=12.4;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8074;GESP=A:1913/G:10339;PAC=NP_078966.2 GT:GQ:PL 0/1:13:11,0,32 0/0:19:0,21,95 1/1:4:44,6,0 0/1:19:43,0,15</p>

<p>19 9069892 rs1862460 A C 33.40 highCVG AC1=3;AF1=0.3838;DB;DP=20;DP4=2,9,3,1;FQ=35.4;MQ=60;PV4=0.077,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=F5852V;SNPEFF_CODON_CHANGE=Ttt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0084;DN=92;DA=A/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=PHE/VAL;PP=5852/14508;CDP=17554;GS=50;PH=0;CP=0.000;CG=-0.715;AA=C;HA=30.1;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7974;GESP=C:2522/A:9570;PAC=NP_078966.2 GT:GQ:PL 0/1:13:13,0,15 0/0:27:0,27,130 1/1:5:65,9,0 0/0:4:0,3,28</p>

<p>19 9070225 rs1559172 G C 130 midCVG AC1=6;AF1=0.682;DB;DP=12;DP4=0,3,4,5;FQ=5.24;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H5741D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0092;DN=88;DA=G/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=HIS/ASP;PP=5741/14508;CDP=17221;GS=81;PH=0;CP=0.000;CG=-0.660;AA=C;HA=28.3;HE=15.0;HC=11.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7641;GESP=C:10349/G:2229;PAC=NP_078966.2 GT:GQ:PL 0/1:22:52,0,22 0/1:5:0,3,22 1/1:12:76,12,0 1/1:7:47,6,0</p>

<p>19 9764230 . G A 25.50 lowCVG AC1=2;AF1=0.3027;DP=7;DP4=2,2,0,3;FQ=26.8;MQ=60;PV4=0.43,1,1,0;SNPEFF_AMINO_ACID_CHANGE=H226Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF562;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130031.1;VDB=0.0003;GM=NM_001130031.1,NM_001130032.1,NM_017656.3,XM_005259941.1,XM_005259942.1,XM_005259943.1,XM_005259944.1,XM_005259945.1,XM_005259946.1,XM_005259947.1,XM_005259948.1,XM_005259949.1,XM_005259950.1,XM_005259951.1,XM_005259952.1,XM_005259953.1;GL=ZNF562;FG=missense;FD=unknown;AAC=HIS/TYR;PP=226/427,226/427,154/355,226/427,225/426,218/419,189/390,157/358,157/358,157/358,157/358,148/349,111/312,110/311,83/284,83/284;CDP=676,676,460,676,673,652,565,469,469,469,469,442,331,328,247,247;GS=83;PH=0.965;CP=0.000;CG=-0.749;AA=G;DSP=325;GESP=G:13006;PAC=NP_001123503.1,NP_001123504.1,NP_060126.2,XP_005259998.1,XP_005259999.1,XP_005260000.1,XP_005260001.1,XP_005260002.1,XP_005260003.1,XP_005260004.1,XP_005260005.1,XP_005260006.1,XP_005260007.1,XP_005260008.1,XP_005260009.1,XP_005260010.1 GT:GQ:PL 0/0:8:0,6,34 0/0:6:0,3,28 1/1:4:66,9,0 0/0:6:0,3,19</p>

<p>19 10112346 rs2287803 T C 55.50 lowCVG AC1=5;AF1=0.647;DB;DP=8;DP4=3,0,0,5;FQ=23;MQ=60;PV4=0.018,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R322G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL5A3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015719.3;VDB=0.0071;DN=100;DA=T/C;GM=NM_015719.3;GL=COL5A3;FG=missense-near-splice;FD=missense;AAC=ARG/GLY;PP=322/1746;CDP=964;GS=125;PH=0;CP=0.002;CG=2.170;AA=C;HA=3.2;HE=2.3;HC=13.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:12695/T:311;PPI=COL5A1:999;PAC=NP_056534.2 GT:GQ:PL 0/1:17:20,0,16 0/1:17:12,0,43 1/1:9:64,9,0 0/1:3:0,0,0</p>

<p>19 10224526 rs3745600 C T 89.90 lowCVG AC1=6;AF1=0.7365;DB;DP=8;DP4=0,2,2,4;FQ=11;MQ=60;PV4=1,0.13,1,0.14;SNPEFF_AMINO_ACID_CHANGE=P520L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPAN-P2RY11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198690.1;VDB=0.0011;DN=107;DA=C/T;GM=NM_001040664.2,NM_001198690.1,NM_002566.4;GL=P2RY11/PPAN-P2RY11;FG=synonymous,missense,synonymous;FD=synonymous-codon,missense,synonymous-codon;AAC=ALA,PRO/LEU,ALA;PP=499/795,520/521,79/375;CDP=1497,1559,237;GS=NA,98,NA;PH=0;CP=0.553;CG=-4.370;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=218;KP=hsa04080:NeuroactiveLigand-receptorInteraction;CPG=10224443-10225138;GESP=T:4862/C:8140;PAC=NP_001035754.1,NP_001185619.1,NP_002557.2 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,48 1/1:9:71,9,0 1/1:9:62,9,0</p>

<p>19 12154799 rs67102109 G C 83.90 lowCVG AC1=6;AF1=0.7469;DB;DP=7;DP4=0,1,2,4;FQ=-14.5;MQ=60;PV4=1,0.36,1,0.21;SNPEFF_AMINO_ACID_CHANGE=H473D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF878;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080404.2;VDB=0.0169;DN=130;DA=G/C;GM=NM_001080404.2;GL=ZNF878;FG=missense;FD=missense;AAC=HIS/ASP;PP=473/532;CDP=1417;GS=81;PH=0.997;CP=0.016;CG=1.300;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1226;GESP=C:908/G:12058;PAC=NP_001073873.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:12:87,12,0 0/1:17:34,0,17</p>

<p>19 12502457 rs4804669 A G 15.20 midCVG AC1=3;AF1=0.3874;DB;DP=12;DP4=4,3,2,3;FQ=17.3;MQ=50;PV4=1,0.014,0.37,1;SNPEFF_AMINO_ACID_CHANGE=L252P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF799;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080821.2;VDB=0.0207;DN=111;DA=A/G;GM=NM_001080821.2,XM_005260140.1;GL=ZNF799;FG=missense;FD=missense,unknown;AAC=LEU/PRO;PP=252/644,254/646;CDP=755,761;GS=98;PH=0;CP=0.001;CG=1.310;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=564;GESP=G:10882/A:2116;PAC=NP_001074290.1,XP_005260197.1 GT:GQ:PL 0/0:7:0,6,51 0/1:11:11,0,15 1/1:7:50,12,0 0/0:12:0,12,85</p>

<p>19 12542653 rs4239550 T G 25.90 midCVG AC1=3;AF1=0.3275;DB;DP=11;DP4=4,1,1,2;FQ=27.6;MQ=50;PV4=0.46,0.23,0.01,1;SNPEFF_AMINO_ACID_CHANGE=K111N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF443;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005815.4;VDB=0.0205;DN=111;DA=T/A/G;GM=NM_005815.4,XM_005259705.1;GL=ZNF443;FG=missense;FD=missense,unknown;AAC=LYS/ASN;PP=111/672,113/674;CDP=333,339;GS=94;PH=0.009;CP=0.000;CG=-0.007;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=142;GESP=G:4380/T:8626;PAC=NP_005806.2,XP_005259762.1 GT:GQ:PL 0/0:11:0,9,62 0/0:8:0,6,48 1/1:4:66,9,0 0/0:3:0,0,0</p>

<p>19 12542654 rs28599549 T A 22.40 midCVG AC1=2;AF1=0.2468;DB;DP=12;DP4=4,4,1,2;FQ=23.6;MQ=52;PV4=1,0.082,0.0011,1;SNPEFF_AMINO_ACID_CHANGE=K111I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF443;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005815.4;VDB=0.0205;DN=125;DA=T/A;GM=NM_005815.4,XM_005259705.1;GL=ZNF443;FG=missense;FD=missense,unknown;AAC=LYS/ILE;PP=111/672,113/674;CDP=332,338;GS=102;PH=0;CP=0.000;CG=-1.340;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=141;GESP=A:4376/T:8630;PAC=NP_005806.2,XP_005259762.1 GT:GQ:PL 0/0:18:0,15,110 0/0:9:0,6,48 1/1:3:64,9,0 0/0:7:0,3,28</p>

<p>19 12772165 rs1054487 G A 27.30 lowCVG AC1=3;AF1=0.3966;DB;DP=8;DP4=1,3,3,1;FQ=29.4;MQ=60;PV4=0.49,0.0045,1,1;SNPEFF_AMINO_ACID_CHANGE=T312I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAN2B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000528.3;VDB=0.0063;DN=86;DA=G/A/T;GM=NM_000528.3,NM_001173498.1,XM_005259913.1;GL=MAN2B1;FG=missense;FD=missense,missense,unknown;AAC=THR/ILE;PP=312/1012,312/1011,312/1013;CDP=935;GS=89;PH=0.003;CP=0.015;CG=-2.190;AA=A;HA=40.2;HE=38.7;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=A:5899/G:7107;PAC=NP_000519.2,NP_001166969.1,XP_005259970.1 GT:GQ:PL 0/0:5:0,3,25 0/1:3:0,0,0 1/1:7:68,12,0 0/0:9:0,9,72</p>

<p>19 14185226 rs7258963 T C 15.40 lowCVG AC1=4;AF1=0.4664;DB;DP=7;DP4=1,3,2,1;FQ=18.1;MQ=60;PV4=0.49,0.27,1,0.35;SNPEFF_AMINO_ACID_CHANGE=V198A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LOC113230;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291291.1;VDB=0.0059;DN=116;DA=T/C;GM=NR_024282.1;GL=LOC113230;FG=non-coding-exon;FD=nc-transcript-variant;CDP=894;CP=0.984;CG=3.330;AA=C;HA=21.4;HE=37.5;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;CPG=14185108-14185601 GT:GQ:PL 0/1:3:0,0,0 0/0:8:0,9,62 1/1:4:42,6,0 0/1:15:14,0,19</p>

<p>19 14875388 rs35612307 G A 41.60 highCVG AC1=3;AF1=0.4114;DB;DP=35;DP4=3,23,3,1;FQ=43.9;MQ=60;PV4=0.018,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=A314V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EMR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013447.3;VDB=0.0063;DN=126;DA=G/A;GM=NM_001271052.1,NM_013447.3,XM_005259884.1,XM_005259885.1,XM_005259886.1,XM_005259887.1;GL=EMR2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/VAL;PP=314/766,314/824,383/893,383/835,53/552,383/462;CDP=941,941,1148,1148,158,1148;GS=64;PH=0.294;CP=0.000;CG=-3.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=A:3066/G:9936;PAC=NP_001257981.1,NP_038475.2,XP_005259941.1,XP_005259942.1,XP_005259943.1,XP_005259944.1 GT:GQ:PL 0/0:68:0,69,135 0/1:4:24,3,0 1/1:5:65,9,0 0/0:9:0,9,34</p>

<p>19 14910573 rs10415562 C T 54.20 lowCVG AC1=6;AF1=0.7302;DB;DP=6;DP4=0,1,2,2;FQ=-8.83;MQ=60;PV4=1,0.37,1,0.34;SNPEFF_AMINO_ACID_CHANGE=V126I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR7C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198944.1;VDB=0.0040;DN=119;DA=C/T;GM=NM_198944.1,XM_005259853.1;GL=OR7C1;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=126/321;CDP=376;GS=29;PH=0.007;CP=0.999;CG=2.590;AA=T;HA=14.3;HE=25.8;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=389;KP=hsa04740:OlfactoryTransduction;GESP=T:10304/C:2702;PAC=NP_945182.1,XP_005259910.1 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:4:29,3,0 1/1:9:65,9,0</p>

<p>19 15198631 rs8105737 A C 43.30 lowCVG AC1=5;AF1=0.6343;DB;DP=7;DP4=3,0,3,1;FQ=15.8;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=Y252S;SNPEFF_CODON_CHANGE=tAt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004713.1;VDB=0.0061;DN=116;DA=A/C;GM=NM_001004713.1;GL=OR1I1;FG=missense;FD=missense;AAC=TYR/SER;PP=252/356;CDP=755;GS=144;PH=0.999;CP=0.743;CG=3.590;AA=A;HA=36.4;HE=33.0;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19702;KP=hsa04740:OlfactoryTransduction;TFBS=V$GR_Q6,V$YY1_02;GESP=C:7526/A:5480;PAC=NP_001004713.1 GT:GQ:PL 0/1:14:17,0,13 0/1:3:0,0,0 1/1:6:49,6,0 0/1:22:17,0,38</p>

<p>19 15587185 rs733731 C T 22.30 lowCVG AC1=5;AF1=0.6333;DB;DP=6;DP4=1,1,4,0;FQ=-3.17;MQ=60;PV4=0.33,0.097,1,1;SNPEFF_AMINO_ACID_CHANGE=R99Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PGLYRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052890.3;VDB=0.0069;DN=86;DA=C/T;GM=NM_052890.3;GL=PGLYRP2;FG=missense;FD=missense;AAC=ARG/GLN;PP=99/577;CDP=296;GS=43;PH=0.03;CP=0.001;CG=-5.850;AA=C;HA=34.3;HE=39.5;HC=38.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=235;GESP=T:4832/C:8174;PAC=NP_443122.3 GT:GQ:PL 0/1:14:15,0,14 0/1:4:0,3,24 1/1:4:19,3,0 1/1:6:31,6,0</p>

<p>19 15852363 rs11670007 G A 21 lowCVG AC1=3;AF1=0.3936;DB;DP=6;DP4=1,2,0,3;FQ=22.5;MQ=58;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R54H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10H3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013938.1;VDB=0.0067;DN=120;DA=G/A;GM=NM_013938.1;GL=OR10H3;FG=missense;FD=missense;AAC=ARG/HIS;PP=54/317;CDP=161;GS=29;PH=0.001;CP=0.225;CG=-4.700;AA=G;HA=43.3;HE=31.4;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18868;KP=hsa04740:OlfactoryTransduction;GESP=A:4268/G:8738;PAC=NP_039226.1 GT:GQ:PL 0/0:5:0,3,26 0/0:3:0,0,0 1/1:5:60,9,0 0/0:7:0,6,43</p>

<p>19 17017878 rs56408444 G A 20.80 lowCVG AC1=4;AF1=0.4992;DB;DP=6;DP4=2,1,1,2;FQ=23.3;MQ=60;PV4=1,0.004,1,0.41;SNPEFF_AMINO_ACID_CHANGE=A1351V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPAMD8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015692.2;VDB=0.0285;DN=129;DA=G/A;GM=NM_015692.2;GL=CPAMD8;FG=missense;FD=missense;AAC=ALA/VAL;PP=1351/1933;CDP=4052;GS=64;PH=0.068;CP=0.000;CG=-4.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=50;GESP=A:1872/G:10942;PAC=NP_056507.2GT:GQ:PL 0/0:3:0,3,27 0/1:3:22,3,0 1/1:4:42,6,0 0/0:5:0,6,47</p>

<p>19 17170885 rs1130222 C T 99.90 highCVG AC1=5;AF1=0.5805;DB;DP=17;DP4=4,4,3,5;FQ=74.7;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G83R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033417.1;VDB=0.0107;DN=86;DA=C/A/G/T;GM=NM_001011699.1,NM_033417.1,XM_005260154.1;GL=HAUS8;FG=missense,missense,5-prime-UTR;FD=missense/stop-gained,missense/stop-gained,unknown;AAC=GLY/ARG,GLY/ARG,none;PP=82/410,83/411,NA;CDP=244,247,NA;GS=125,125,NA;PH=0.062;CP=0.001;CG=1.070;AA=C;HA=45.0;HE=33.3;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;GESP=T:5621/C:7385;PPI=HAUS1:999|HAUS6:994|HAUS2:987|HAUS4:987;PAC=NP_001011699.1,NP_219485.1,NA GT:GQ:PL 0/1:46:42,0,63 0/0:9:0,12,55 1/1:4:28,3,0 1/1:10:77,12,0</p>

<p>19 17743622 rs75595663 C A 43.40 highCVG AC1=2;AF1=0.2548;DB;DP=15;DP4=4,7,3,1;FQ=44.6;MQ=60;PV4=0.28,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1133S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UNC13A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080421.2;VDB=0.0071;DN=132;DA=C/A;GM=NM_001080421.2,XM_005259821.1;GL=UNC13A;FG=missense;FD=missense,unknown;AAC=ALA/SER;PP=1133/1704,1133/1681;CDP=3397;GS=99;PH=0.001;CP=0.987;CG=-0.725;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=A:782/C:11986;PAC=NP_001073890.2,XP_005259878.1 GT:GQ:PL 0/0:14:0,12,89 0/0:6:0,3,25 1/1:5:87,12,0 0/0:20:0,18,104</p>

<p>19 20133846 rs7255165 C T 23.50 midCVG AC1=3;AF1=0.4339;DB;DP=12;DP4=5,4,2,1;FQ=25.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V65M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF682;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033196.2;VDB=0.0044;DN=116;DA=C/T;GM=NM_001077349.1,NM_033196.2;GL=ZNF682;FG=missense;FD=missense;AAC=VAL/MET;PP=33/467,65/499;CDP=97,193;GS=21;PH=0.005;CP=0.300;CG=0.898;AA=C;HA=42.2;HE=18.1;HC=6.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=34;GESP=T:2867/C:9861;PAC=NP_001070817.1,NP_149973.1 GT:GQ:PL 0/0:20:0,21,136 0/1:3:0,0,0 1/1:4:48,6,0 0/1:18:17,0,43</p>

<p>19 21909948 rs7246400 G A 38.50 lowCVG AC1=5;AF1=0.6627;DB;DP=8;DP4=0,1,0,6;FQ=-6.85;MQ=34;PV4=1,0.01,1,0.36;SNPEFF_AMINO_ACID_CHANGE=T389I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF100;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173531.3;VDB=0.0289;DN=116;DA=G/A;GM=NM_173531.3,XM_005259783.1,XM_005259784.1,XM_005259785.1;GL=ZNF100;FG=missense;FD=missense,unknown,unknown,unknown;AAC=THR/ILE;PP=389/543,388/542,356/510,174/328;CDP=1166,1163,1067,521;GS=89;PH=0.804;CP=0.029;CG=0.841;AA=G;HA=30.4;HC=40.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=844;GESP=A:6476/G:6444;PAC=NP_775802.2,XP_005259840.1,XP_005259841.1,XP_005259842.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:16:76,18,0 0/1:3:0,3,27</p>

<p>19 22941591 rs62119159 A T 42.80 midCVG AC1=6;AF1=0.739;DB;DP=13;DP4=2,1,0,10;FQ=19.6;MQ=46;PV4=0.038,4.5e-06,1,1;SNPEFF_AMINO_ACID_CHANGE=C374S;SNPEFF_CODON_CHANGE=Tgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080409.2;VDB=0.0101;DN=129;DA=A/T;GM=NM_001080409.2;GL=ZNF99;FG=missense;FD=missense;AAC=CYS/SER;PP=374/865;CDP=1120;GS=112;PH=1;CP=0.032;CG=1.120;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=894;GESP=T:1032/A:11442;PAC=NP_001073878.2 GT:GQ:PL 0/1:12:6,0,51 0/1:3:0,0,0 1/1:27:74,27,0 0/1:3:0,0,0</p>

<p>19 23158847 rs1433111 T A 17 lowCVG AC1=3;AF1=0.4009;DB;DP=8;DP4=1,3,3,0;FQ=19;MQ=58;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K431I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0063;DN=88;DA=T/A;GM=NM_001267716.1;GL=ZNF728;FG=missense;FD=missense;AAC=LYS/ILE;PP=431/623;CDP=1292;GS=102;CP=0.729;CG=-1.320;AA=T;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1066;PAC=NP_001254645.1 GT:GQ:PL 0/0:4:0,3,16 0/0:7:0,6,42 1/1:3:46,6,0 0/1:13:13,0,16</p>

<p>19 23159486 rs289318 T C 119 highCVG AC1=4;AF1=0.5102;DB;DP=20;DP4=6,3,3,7;FQ=106;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Y218C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0021;DN=79;DA=T/C;GM=NM_001267716.1,XM_005259908.1;GL=ZNF728;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=TYR/CYS,none;PP=218/623,NA;CDP=653,NA;GS=194,NA;CP=0.272;CG=0.945;AA=T;HE=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=427;PAC=NP_001254645.1,NA GT:GQ:PL 0/0:9:0,12,78 0/1:20:34,0,17 1/1:18:121,21,0 0/1:15:12,0,56</p>

<p>19 35435069 rs765746 A G 47.60 midCVG AC1=4;AF1=0.4586;DB;DP=12;DP4=6,1,4,1;FQ=50.2;MQ=60;PV4=1,0.4,1,0.13;SNPEFF_AMINO_ACID_CHANGE=Y401C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF30;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099437.1;VDB=0.0179;DN=86;DA=A/G;GM=NM_001099437.1,NM_001099438.1,NM_194325.2,NR_024018.1;GL=ZNF30;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=TYR/CYS,TYR/CYS,TYR/CYS,none;PP=401/625,401/625,400/624,NA;CDP=1202,1202,1199,1657;GS=194,194,194,NA;PH=1;CP=0.040;CG=1.040;AA=A;HA=22.5;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=943;GESP=G:2295/A:10709;PAC=NP_001092907.1,NP_001092908.1,NP_919306.2,NA GT:GQ:PL 0/0:6:0,6,39 0/0:13:0,15,92 1/1:8:75,12,0 0/1:4:20,3,0</p>

<p>19 36303664 rs3848666 G C 17.50 lowCVG AC1=4;AF1=0.5;DB;DP=8;DP4=2,1,2,1;FQ=20;MQ=60;PV4=1,0.19,1,0.28;SNPEFF_AMINO_ACID_CHANGE=P91R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRODH2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021232.1;VDB=0.0075;DN=108;DA=G/C;GM=NM_021232.1,XM_005259123.1;GL=PRODH2;FG=missense;FD=missense,unknown;AAC=PRO/ARG;PP=91/537,15/379;CDP=272,44;GS=103;PH=0.004;CP=0.000;CG=0.647;AA=A;HA=0.8;HE=14.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=131;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;GESP=C:1572/G:11434;PAC=NP_067055.1,XP_005259180.1 GT:GQ:PL 0/0:9:0,9,65 0/1:3:0,0,0 1/1:5:55,9,0 0/1:3:0,0,0</p>

<p>19 37117302 rs3108171 A G 50.10 lowCVG AC1=5;AF1=0.5824;DB;DP=8;DP4=1,2,5,0;FQ=19.9;MQ=60;PV4=0.11,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=E168G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0027;DN=103;DA=A/G;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=GLU/GLY;PP=167/550,168/551,168/551,167/550;CDP=500,503,503,500;GS=98;PH=0.103;CP=1.000;CG=3.820;AA=A;HA=28.0;HE=40.7;HC=16.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=271;GESP=G:8458/A:4542;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:19:16,0,31</p>

<p>19 37118439 rs61732180 C T 83.80 midCVG AC1=5;AF1=0.6451;DB;DP=9;DP4=3,0,3,3;FQ=15.4;MQ=60;PV4=0.46,1,1,0.002;SNPEFF_AMINO_ACID_CHANGE=T547M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0028;DN=129;DA=C/T;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=THR/MET;PP=546/550,547/551,547/551,546/550;CDP=1637,1640,1640,1637;GS=81;PH=0;CP=0.009;CG=1.990;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1408;GESP=T:2043/C:10963;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:17:27,0,16 0/0:4:0,6,41 1/1:9:59,9,0 1/1:6:44,6,0</p>

<p>19 37488472 rs1644698 C G 25.20 lowCVG AC1=6;AF1=0.7081;DB;DP=8;DP4=1,0,0,3;FQ=-11;MQ=60;PV4=0.25,0.34,1,0.21;SNPEFF_AMINO_ACID_CHANGE=P627A;SNPEFF_CODON_CHANGE=Ccc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0049;DN=89;DA=C/G;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=PRO/ALA;PP=627/636,563/572;CDP=1879,1687;GS=27;PH=0;CP=0.004;CG=-3.900;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=1254;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:47,6,0 0/1:17:14,0,20</p>

<p>19 37488481 rs1363753 G C 93.70 midCVG AC1=4;AF1=0.5436;DB;DP=10;DP4=1,2,1,6;FQ=6.29;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G630R;SNPEFF_CODON_CHANGE=Ggt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0155;DN=88;DA=G/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=GLY/ARG;PP=630/636,566/572;CDP=1888,1696;GS=125;PH=0;CP=0.022;CG=1.640;AA=C;HA=29.2;HE=49.2;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1263;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:0,0,0 1/1:5:48,6,0 0/1:24:85,0,21</p>

<p>19 37488499 rs1667366 T C 105 midCVG AC1=6;AF1=0.6771;DB;DP=10;DP4=1,1,1,7;FQ=-10.4;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*636R;SNPEFF_CODON_CHANGE=Tga/Cga;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0136;DN=89;DA=T/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=stop-lost;FD=stop-lost;AAC=stop/ARG;PP=636/636,572/572;CDP=1906,1714;CP=0.084;CG=2.660;AA=C;HA=43.3;HE=48.3;HC=26.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1281;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:6:47,6,0 0/1:3:96,1,0</p>

<p>19 39401520 . G C 15.90 midCVG AC1=4;AF1=0.497;DP=11;DP4=1,6,4,0;FQ=18.8;MQ=60;PV4=0.015,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q132E;SNPEFF_CODON_CHANGE=Cag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCER2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001243212.1;VDB=0.0133;GM=NM_001243212.1;GL=LOC643669;FG=missense;FD=unknown;AAC=GLN/GLU;PP=132/267;CDP=394;GS=29;CP=0.080;CG=2.700;DSP=204;PAC=NP_001230141.1 GT:GQ:PL 0/0:6:0,6,38 0/1:3:0,0,0 1/1:4:40,6,0 0/1:18:16,0,58</p>

<p>19 40363926 rs3746013 C G 69.60 midCVG AC1=3;AF1=0.4126;DB;DP=13;DP4=3,3,3,3;FQ=71.8;MQ=60;PV4=1,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=D4906H;SNPEFF_CODON_CHANGE=Gat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FCGBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003890.2;VDB=0.0085;DN=107;DA=C/G;GM=NM_003890.2,XM_005259365.1;GL=FCGBP;FG=missense;FD=missense,unknown;AAC=ASP/HIS;PP=4906/5406;CDP=14716;GS=81;PH=0.997;CP=0.005;CG=-1.040;AA=C;HA=34.2;HE=19.2;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:2841/C:10165;PAC=NP_003881.2,XP_005259422.1 GT:GQ:PL 0/0:9:0,9,75 0/0:4:0,3,26 1/1:5:63,9,0 0/1:42:51,0,38</p>

<p>19 40729096 rs147729392 C G 71.20 highCVG AC1=4;AF1=0.5608;DB;DP=20;DP4=3,0,1,15;FQ=25;MQ=60;PV4=0.0041,0.00036,1,0.34;SNPEFF_AMINO_ACID_CHANGE=D263H;SNPEFF_CODON_CHANGE=Gac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024877.3;VDB=0.0024;DN=134;DA=C/G;GM=NM_024877.3;GL=CNTD2;FG=missense;FD=missense;AAC=ASP/HIS;PP=263/308;CDP=787;GS=81;PH=0.742;CP=0.000;CG=-0.127;AA=C;DV=by-cluster;DSP=115;CPG=40729088-40729430;GESP=C:4564;PAC=NP_079153.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:45:111,48,0 0/0:7:0,9,61</p>

<p>19 40901614 rs268671 A G 70.30 highCVG AC1=4;AF1=0.4851;DB;DP=16;DP4=3,6,2,4;FQ=73.2;MQ=60;PV4=1,0.33,1,0.16;SNPEFF_AMINO_ACID_CHANGE=V882A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRX;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181882.2;VDB=0.0157;DN=79;DA=A/G;GM=NM_020956.2,NM_181882.2,XM_005259111.1;GL=PRX;FG=3-prime-UTR,missense,missense;FD=utr-variant-3-prime,missense,unknown;AAC=none,VAL/ALA,VAL/ALA;PP=NA,882/1462,856/1436;CDP=NA,2645,2567;GS=NA,64,64;PH=0;CP=0.033;CG=2.700;AA=G;HA=9.5;HE=49.1;HC=20.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1088;GESP=G:7930/A:5076;PAC=NA,NP_870998.2,XP_005259168.1 GT:GQ:PL 0/0:16:0,18,111 0/1:3:0,0,1/1:9:84,12,0 0/1:32:30,0,49</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 41035024 . C T 33.50 midCVG AC1=4;AF1=0.5209;DP=10;DP4=0,3,5,0;FQ=22.5;MQ=60;PV4=0.018,0.066,1,0.086;SNPEFF_AMINO_ACID_CHANGE=R1312*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_020971.2;VDB=0.0182;GM=NM_020971.2,XM_005259112.1,XM_005259113.1;GL=SPTBN4;FG=stop-gained;FD=unknown;AAC=ARG/stop;PP=1312/2565,1307/2560,55/1308;CDP=3934,3919,163;CP=1.000;CG=2.100;AA=C;DSP=15;GESP=C:13006;PPI=DISC1:789;PAC=NP_066022.2,XP_005259169.1,XP_005259170.1 GT:GQ:PL 0/1:11:8,0,21 0/0:3:0,3,28 1/1:5:45,6,0 0/1:16:23,0,13</p>

<p>SERTAD1 19 43243238 rs11559136 A G 43.70 lowCVG AC1=5;AF1=0.6561;DB;DP=7;DP4=2,1,1,3;FQ=11.7;MQ=51;PV4=0.49,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L23S;SNPEFF_CODON_CHANGE=tTa/tCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PSG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021016.3;VDB=0.0075;DN=120;DA=A/G;GM=NM_021016.3,XM_005259068.1;GL=PSG3;FG=missense;FD=missense,unknown;AAC=LEU/SER;PP=23/429,45/459;CDP=68,134;GS=145;PH=0;CP=0.977;CG=1.390;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4;GESP=G:8155/A:4843;PAC=NP_066296.2,XP_005259125.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,47 1/1:6:43,6,0 1/1:4:41,3,0</p>

<p>19 44591360 rs8113249 G A 146 highCVG AC1=4;AF1=0.5099;DB;DP=21;DP4=3,5,6,7;FQ=114;MQ=60;PV4=1,0.021,1,0.056;SNPEFF_AMINO_ACID_CHANGE=E577K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF284;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001037813.2;VDB=0.0032;DN=116;DA=G/A;GM=NM_001037813.2,XM_005258859.1;GL=ZNF284;FG=missense;FD=missense,unknown;AAC=GLU/LYS;PP=577/594;CDP=1729;GS=56;PH=0;CP=0.707;CG=-0.036;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=1494;GESP=A:10995/G:1991;PAC=NP_001032902.1,XP_005258916.1 GT:GQ:PL 0/0:12:0,15,106 0/1:3:0,0,0 1/1:27:160,30,0 0/1:33:30,0,49</p>

<p>19 44610798 rs4239529 A T 111 highCVG AC1=4;AF1=0.5334;DB;DP=15;DP4=3,3,2,7;FQ=45.2;MQ=60;PV4=0.33,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H162L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF224;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013398.2;VDB=0.0096;DN=111;DA=A/T;GM=NM_013398.2,NR_033341.1,XM_005259221.1;GL=ZNF224/LOC100379224;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=HIS/LEU,none,HIS/LEU;PP=162/708,NA,162/708;CDP=485,2055,485;GS=99,NA,99;PH=0;CP=0.000;CG=-1.700;AA=T;HA=9.2;HE=12.5;HC=37.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=250;GESP=T:11402/A:1604;PAC=NP_037530.2,NA,XP_005259278.1 GT:GQ:PL 0/0:12:0,15,81 0/1:3:0,0,0 1/1:12:78,15,0 0/1:8:76,0,4</p>

<p>19 45716364 rs189063316 C T 11.60 lowCVG AC1=5;AF1=0.5837;DB;DP=5;DP4=1,0,2,0;FQ=-12.4;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R398Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXOC3L2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138568.3;VDB=0.0116;DN=135;DA=C/T;GM=NM_138568.3;GL=EXOC3L2;FG=missense;FD=missense;AAC=ARG/GLN;PP=398/410;CDP=1193;GS=43;PH=0.46;CP=0.112;CG=1.260;AA=C;DV=by-cluster;DSP=252;GESP=T:121/C:12673;PAC=NP_612635.3 GT:GQ:PL 0/1:3:0,0,0/0:4:0,3,19 1/1:4:46,6,0 0/1:3:0,0,0</p>

<p>19 46269076 rs2014576 G A 95.10 midCVG AC1=6;AF1=0.6715;DB;DP=11;DP4=2,1,3,4;FQ=14.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P635S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SIX5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175875.4;VDB=0.0122;DN=92;DA=G/A;GM=NM_175875.4;GL=SIX5;FG=missense;FD=missense;AAC=PRO/SER;PP=635/740;CDP=1903;GS=74;PH=0.004;CP=0.983;CG=-0.760;AA=G;HA=33.3;HE=47.5;HC=29.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=294;GESP=A:5213/G:7717;PAC=NP_787071.2 GT:GQ:PL 0/1:29:32,0,29 0/1:4:0,3,25 1/1:7:54,6,0 1/1:9:54,9,0</p>

<p>19 48495304 rs60213124 G A 79 midCVG AC1=4;AF1=0.5008;DB;DP=12;DP4=3,3,3,3;FQ=79.1;MQ=60;PV4=1,0.42,1,0.17;SNPEFF_AMINO_ACID_CHANGE=T12M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BSPH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128326.2;VDB=0.0230;DN=129;DA=G/A;GM=NM_001128326.2;GL=BSPH1;FG=missense;FD=missense;AAC=THR/MET;PP=12/133;CDP=35;GS=81;PH=0.962;CP=0.000;CG=-0.642;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=G:4566;PAC=NP_001121798.1 GT:GQ:PL 0/0:13:0,15,105 0/1:3:27,3,0 1/1:12:99,15,0 0/0:3:0,3,21</p>

<p>19 48525507 rs2303690 G A 67.70 midCVG AC1=5;AF1=0.6528;DB;DP=13;DP4=3,0,5,3;FQ=4.34;MQ=60;PV4=0.49,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E199K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ELSPBP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022142.4;VDB=0.0107;DN=100;DA=G/A;GM=NM_022142.4;GL=ELSPBP1;FG=missense;FD=missense;AAC=GLU/LYS;PP=199/224;CDP=595;GS=56;PH=0.011;CP=0.044;CG=1.540;AA=A;HA=31.1;HE=47.0;HC=13.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=81;GESP=A:7578/G:5428;PPI=MYF5:832|MYOG:832|MYOD1:832;PAC=NP_071425.3 GT:GQ:PL 0/1:10:52,0,8 0/1:3:0,0,0 1/1:8:53,11,2 0/1:9:4,0,29</p>

<p>19 48608598 rs2307279 C G 24.60 lowCVG AC1=5;AF1=0.6092;DB;DP=5;DP4=0,2,0,3;FQ=-7.18;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A48P;SNPEFF_CODON_CHANGE=Gct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLA2G4C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159322.1;DN=100;DA=C/G;GM=NM_001159322.1,NM_001159323.1,NM_003706.2,XM_005259361.1;GL=PLA2G4C;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/PRO;PP=48/552,38/528,38/542,48/534;CDP=142,112,112,142;GS=27;PH=0.74;CP=0.000;CG=0.300;AA=C;HA=38.7;HE=4.2;HC=30.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=9;GESP=G:2512/C:10494;PAC=NP_001152794.1,NP_001152795.1,NP_003697.2,XP_005259418.1 GT:GQ:PL 0/1:15:17,0,14 0/1:4:0,3,17 1/1:4:24,3,0 1/1:4:25,3,0</p>

<p>19 49377319 rs610308 A G 17.80 midCVG AC1=3;AF1=0.4136;DB;DP=11;DP4=3,1,0,3;FQ=20;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K277E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R15A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014330.3;VDB=0.0071;DN=83;DA=A/G;GM=NM_014330.3;GL=PPP1R15A;FG=missense;FD=missense;AAC=LYS/GLU;PP=277/675;CDP=829;GS=56;PH=0;CP=0.000;CG=-3.110;AA=G;HA=22.5;HE=32.3;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=837;GESP=G:6297/A:6709;PPI=PPP1CA:942|PPP1CC:937|SMARCB1:846|PPP1CB:845;PAC=NP_055145.3 GT:GQ:PL 0/0:6:0,6,46 0/0:4:0,3,24 1/1:3:39,6,0 0/1:13:21,0,8</p>

<p>19 49438363 rs2270941 G A 47.70 midCVG AC1=5;AF1=0.5687;DB;DP=10;DP4=0,4,2,3;FQ=37.1;MQ=60;PV4=0.44,0.036,1,1;SNPEFF_AMINO_ACID_CHANGE=S66N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHDH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014475.3;VDB=0.0081;DN=100;DA=G/A;GM=NM_014475.3,XM_005258748.1;GL=DHDH;FG=missense;FD=missense,unknown;AAC=SER/ASN,ALA/THR;PP=66/335,9/223;CDP=197,25;GS=46,58;PH=0;CP=0.282;CG=4.740;AA=A;HA=22.1;HE=21.4;HC=45.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00980:MetabolismOfXenobioticsByCytochromeP450;GESP=A:4923/G:8051;PAC=NP_055290.1,XP_005258805.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:7:65,9,0 0/1:26:23,0,51</p>

<p>19 52496149 rs16983353 C T 125 highCVG AC1=5;AF1=0.6541;DB;DP=17;DP4=1,5,4,6;FQ=50.5;MQ=59;PV4=0.59,1,0.23,0.15;SNPEFF_AMINO_ACID_CHANGE=R738K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF615;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199324.1;VDB=0.0050;DN=123;DA=C/T;GM=NM_001199324.1,NM_198480.3,XM_005258801.1,XM_005258802.1,XM_005258803.1,XM_005258804.1,XM_005258805.1,XM_005258806.1;GL=ZNF615;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=738/743,727/732,743/748,738/743,732/737,727/732,619/624,619/624;CDP=2213,2180,2228,2213,2195,2180,1856,1856;GS=26;PH=0;CP=0.003;CG=2.500;AA=T;HA=35.3;HE=28.2;HC=33.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1942;GESP=T:4942/C:8064;PAC=NP_001186253.1,NP_940882.3,XP_005258858.1,XP_005258859.1,XP_005258860.1,XP_005258861.1,XP_005258862.1,XP_005258863.1 GT:GQ:PL 0/1:18:13,0,48 0/1:3:0,0,0 1/1:11:88,12,0 0/1:40:65,0,39</p>

<p>19 52888245 rs55748277 A G 23.70 midCVG AC1=4;AF1=0.4732;DB;DP=11;DP4=3,3,2,3;FQ=26.4;MQ=59;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=K471R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF880;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145434.1;VDB=0.0110;DN=129;DA=A/G;GM=NM_001145434.1;GL=ZNF880;FG=missense;FD=missense;AAC=LYS/ARG;PP=471/578;CDP=1412;GS=26;PH=0.991;CP=0.007;CG=-0.460;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1144;GESP=G:1745/A:2821;PAC=NP_001138906.1 GT:GQ:PL 0/1:6:3,0,37 0/1:13:24,0,9 1/1:4:39,6,0 0/0:6:0,6,45</p>

<p>19 53086124 rs373554 C A 100 highCVG AC1=5;AF1=0.6181;DB;DP=15;DP4=5,1,5,4;FQ=54.4;MQ=59;PV4=0.58,0.21,0.22,0.18;SNPEFF_AMINO_ACID_CHANGE=T337K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF701;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172655.1;VDB=0.0155;DN=80;DA=C/A;GM=NM_001172655.1,NM_018260.2;GL=ZNF701;FG=missense;FD=missense;AAC=THR/LYS;PP=337/532,271/466;CDP=1010,812;GS=78;PH=0;CP=0.001;CG=-0.451;AA=A;HA=0.0;HE=17.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=670;GESP=A:9195/C:3811;PAC=NP_001166126.1,NP_060730.2 GT:GQ:PL 0/1:22:17,0,33 0/1:16:21,0,15 1/1:11:58,12,0 0/1:47:49,0,47</p>

<p>19 53117531 rs1056185 C T 9.83 lowCVG AC1=6;AF1=0.6684;DB;DP=7;DP4=1,0,1,1;FQ=-10.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S96N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF83;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001105549.1;VDB=0.0014;DN=86;DA=C/T;GM=NM_001105549.1,NM_001105550.1,NM_001105551.1,NM_001105552.1,NM_001277945.1,NM_001277946.1,NM_001277947.1,NM_001277948.1,NM_001277949.1,NM_001277951.1,NM_001277952.1,NM_018300.3;GL=ZNF83;FG=missense;FD=missense;AAC=SER/ASN;PP=96/517;CDP=287;GS=46;PH=0;CP=0.000;CG=-2.410;AA=T;HA=8.7;HE=37.5;HC=27.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=520;GESP=T:8689/C:4317;PAC=NP_001099019.1,NP_001099020.1,NP_001099021.1,NP_001099022.1,NP_001264874.1,NP_001264875.1,NP_001264876.1,NP_001264877.1,NP_001264878.1,NP_001264880.1,NP_001264881.1,NP_060770.3 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,0,0 1/1:3:27,3,0 1/1:3:20,3,0</p>

<p>19 53959343 rs2708742 G A 58.30 lowCVG AC1=5;AF1=0.6547;DB;DP=7;DP4=0,1,3,1;FQ=-14.5;MQ=60;PV4=0.4,1,1,0.35;SNPEFF_AMINO_ACID_CHANGE=G528S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF761;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008401.3;VDB=0.0002;DN=100;DA=G/A;GM=NM_001008401.3;GL=ZNF761;FG=missense;FD=missense;AAC=GLY/SER;PP=528/747;CDP=1583;GS=56;CP=0.001;CG=1.140;AA=G;HA=29.9;HE=41.9;HC=24.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1440;GESP=A:4990/G:8016;PAC=NP_001008401.3 GT:GQ:PL 0/1:3:0,3,18 0/1:3:0,0,0 1/1:10:95,12,0 0/1:3:0,0,0</p>

<p>19 53959568 rs2617726 G C 74.70 highCVG AC1=4;AF1=0.5121;DB;DP=15;DP4=3,2,5,2;FQ=27;MQ=52;PV4=1,0.28,1,1;SNPEFF_AMINO_ACID_CHANGE=E603Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF761;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008401.3;VDB=0.0125;DN=100;DA=G/C;GM=NM_001008401.3;GL=ZNF761;FG=missense;FD=missense;AAC=GLU/GLN;PP=603/747;CDP=1808;GS=29;CP=0.228;CG=1.130;AA=G;HA=39.2;HE=41.4;HC=24.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1665;GESP=C:5269/G:7733;PAC=NP_001008401.3 GT:GQ:PL 0/0:3:0,3,19 0/0:7:0,9,50 1/1:15:98,18,0 0/1:4:22,1,0</p>

<p>19 54755922 rs12977062 C T 24.20 highCVG AC1=3;AF1=0.4158;DB;DP=22;DP4=1,11,4,0;FQ=26.4;MQ=48;PV4=0.0027,1,0.05,1;SNPEFF_AMINO_ACID_CHANGE=A542T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081442.1;VDB=0.0083;DN=121;DA=C/T;GM=NM_001081442.1,NM_001081443.1,NM_006840.3,XM_005258435.1;GL=LILRB5;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=542/592,442/492,541/591,525/575;CDP=1624,1324,1621,1573;GS=58;PH=0.013;CP=0.000;CG=-2.320;AA=C;HA=0.0;HE=4.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=T:2224/C:10782;PAC=NP_001074911.1,NP_001074912.1,NP_006831.1,XP_005258492.1 GT:GQ:PL 0/0:26:0,27,82 0/0:9:0,9,70 1/1:4:37,6,0 1/1:4:35,6,0</p>

<p>19 55106778 rs10418391 G A 52.20 lowCVG AC1=6;AF1=0.7307;DB;DP=6;DP4=0,1,0,4;FQ=-9.7;MQ=55;PV4=1,1,0.34,1;SNPEFF_AMINO_ACID_CHANGE=R191H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006863.3;VDB=0.0197;DN=119;DA=G/A;GM=NM_001278318.1,NM_001278319.1,NM_006863.3,NR_103501.1,NR_103502.1,NR_103503.1;GL=LILRA1;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=unknown,unknown,missense,unknown,unknown,unknown;AAC=ARG/HIS,ARG/HIS,ARG/HIS,none,none,none;PP=191/290,191/440,191/490,NA,NA,NA;CDP=572,572,572,744,988,744;GS=29,29,29,NA,NA,NA;PH=0.042;CP=0.000;CG=-0.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=90;GESP=A:2052/G:10954;PAC=NP_001265247.1,NP_001265248.1,NP_006854.1,NA,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:4:27,3,0 1/1:9:65,9,0</p>

<p>19 55143083 rs1061679 T C 34 lowCVG AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA GT:GQ:PL 0/0:12:0,15,73 0/1:3:0,0,0 1/1:5:50,6,0 0/1:3:26,3,0</p>

<p>19 55174498 rs28366008 T C 53.10 midCVG AC1=3;AF1=0.4334;DB;DP=10;DP4=2,3,0,5;FQ=52;MQ=58;PV4=0.44,1,0.17,1;SNPEFF_AMINO_ACID_CHANGE=F46L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001278429.3;VDB=0.0117;DN=125;DA=T/C;GM=NM_001278426.2,NM_001278427.2,NM_001278428.2,NM_001278429.2,NM_001278430.2;GL=LILRB4;FG=missense;FD=unknown;AAC=PHE/LEU;PP=5/449,5/448,5/449,46/479,5/255;CDP=13,13,13,136,13;GS=22;PH=0.001;CP=0.000;CG=0.476;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=22;GESP=C:4017/T:8989;PPI=PTPN11:846|PTPN6:846;PAC=NP_001265355.1,NP_001265356.1,NP_001265357.1,NP_001265358.1,NP_001265359.1 GT:GQ:PL 0/0:6:0,6,44 0/0:4:0,3,22 1/1:6:66,9,0 0/1:30:31,0,30</p>

<p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>

<p>19 55494157 rs4306647 G A 14.20 highCVG AC1=3;AF1=0.3892;DB;DP=15;DP4=6,3,3,1;FQ=16.2;MQ=41;PV4=1,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R364K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001174081.1;VDB=0.0098;DN=111;DA=G/A;GM=NM_001174081.1,NM_001174082.1,NM_001174083.1,NM_017852.3,XM_005259050.1;GL=NLRP2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/LYS;PP=364/1063,342/1041,341/1040,364/1063,361/1060;CDP=1091,1025,1022,1091,1082;GS=26;PH=0.363;CP=0.019;CG=1.640;AA=G;HA=19.6;HE=2.2;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=628;GESP=A:915/G:12091;PAC=NP_001167552.1,NP_001167553.1,NP_001167554.1,NP_060322.1,XP_005259107.1 GT:GQ:PL 0/0:5:0,3,19 0/1:11:10,0,32 1/1:5:48,9,0 0/0:12:0,12,50</p>

<p>19 56499279 rs306457 G C 68.10 midCVG AC1=4;AF1=0.4813;DB;DP=12;DP4=0,6,0,6;FQ=46;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=*1049Y;SNPEFF_CODON_CHANGE=taG/taC;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_176811.2;VDB=0.0070;DN=79;DA=G/C;GM=NM_176811.2,XM_005258511.1;GL=NLRP8;FG=stop-lost;FD=stop-lost,unknown;AAC=stop/TYR;PP=1049/1049,1030/1030;CDP=3147,3090;CP=0.000;CG=0.372;AA=C;HA=26.7;HE=30.0;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=100;GESP=C:9537/G:3469;PAC=NP_789781.2,XP_005258568.1 GT:GQ:PL 0/0:7:0,9,50 0/1:17:21,0,15 1/1:4:45,6,0 0/1:27:47,0,23</p>

<p>19 57641146 rs1027392 A G 124 highCVG AC1=5;AF1=0.6123;DB;DP=15;DP4=1,4,2,8;FQ=44.9;MQ=60;PV4=1,0.41,1,0.22;SNPEFF_AMINO_ACID_CHANGE=N368S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020903.2;VDB=0.0091;DN=86;DA=A/G;GM=NM_020903.2;GL=USP29;FG=missense;FD=missense;AAC=ASN/SER;PP=368/923;CDP=1103;GS=46;PH=0.916;CP=0.000;CG=-0.648;AA=G;HA=25.9;HE=41.4;HC=6.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1119;GESP=G:8646/A:4358;PAC=NP_065954.1 GT:GQ:PL 0/0:8:0,12,80 0/1:10:75,0,8 1/1:8:57,9,0 1/1:6:39,6,0</p>

<p>19 57646570 rs4801433 T C 80.30 midCVG AC1=5;AF1=0.6545;DB;DP=14;DP4=3,2,0,7;FQ=66.9;MQ=60;PV4=0.045,0.067,1,0.21;SNPEFF_AMINO_ACID_CHANGE=I379V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZIM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052882.1;VDB=0.0156;DN=111;DA=T/C;GM=NM_052882.1;GL=ZIM3;FG=missense;FD=missense;AAC=ILE/VAL;PP=379/473;CDP=1135;GS=29;PH=0.978;CP=0.326;CG=1.680;AA=T;HA=24.0;HE=49.7;HC=38.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=894;GESP=C:5945/T:7061;PAC=NP_443114.1 GT:GQ:PL 0/0:11:0,15,108 0/1:3:0,0,0 1/1:11:66,12,0 1/1:8:60,9,0</p>

<p>19 57647236 rs7252632 T C 26.30 midCVG AC1=4;AF1=0.4772;DB;DP=11;DP4=1,5,1,3;FQ=29;MQ=60;PV4=1,0.41,1,0.12;SNPEFF_AMINO_ACID_CHANGE=N157D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZIM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052882.1;VDB=0.0086;DN=116;DA=T/C;GM=NM_052882.1;GL=ZIM3;FG=missense;FD=missense;AAC=ASN/ASP;PP=157/473;CDP=469;GS=23;PH=0;CP=0.000;CG=-4.360;AA=C;HA=24.3;HE=40.9;HC=6.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=228;GESP=C:8716/T:4290;PAC=NP_443114.1 GT:GQ:PL 0/0:4:0,3,20 0/1:5:1,0,45 1/1:4:48,6,0 0/1:20:18,0,35</p>

<p>19 57760018 rs2014572 G A 89.90 midCVG AC1=4;AF1=0.5523;DB;DP=10;DP4=2,1,5,2;FQ=19.5;MQ=46;PV4=1,0.47,0.1,1;SNPEFF_AMINO_ACID_CHANGE=G68E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0081;DN=92;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense,5-prime-UTR;FD=missense,utr-variant-5-prime;AAC=GLY/GLU,none;PP=68/628,NA;CDP=203,NA;GS=98,NA;PH=0.682;CP=0.000;CG=-0.004;AA=A;HA=26.7;HE=42.5;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=46;GESP=A:2030/G:2536;PAC=NP_001018857.2,NA GT:GQ:PL 0/0:5:0,6,44 0/1:3:0,0,0 1/1:10:82,12,0 0/1:17:50,0,14</p>

<p>19 57760057 rs3746231 G A 82.30 midCVG AC1=4;AF1=0.5287;DB;DP=11;DP4=2,2,5,2;FQ=27.7;MQ=47;PV4=0.58,1,0.055,0.28;SNPEFF_AMINO_ACID_CHANGE=G81D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0081;DN=107;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense,5-prime-UTR;FD=missense,utr-variant-5-prime;AAC=GLY/ASP,none;PP=81/628,NA;CDP=242,NA;GS=94,NA;PH=0;CP=0.000;CG=2.540;AA=G;HA=26.7;HE=42.2;HC=21.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=12;GESP=A:2029/G:2537;PAC=NP_001018857.2,NA GT:GQ:PL 0/0:7:0,9,57 0/1:3:0,0,0 1/1:9:87,12,0 0/1:13:38,0,10</p>

<p>19 57764770 rs10421632 G A 40 midCVG AC1=5;AF1=0.612;DB;DP=10;DP4=2,2,3,2;FQ=32.8;MQ=60;PV4=1,0.13,1,0.11;SNPEFF_AMINO_ACID_CHANGE=V195I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0083;DN=119;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense;FD=missense;AAC=VAL/ILE;PP=195/628,62/495;CDP=583,184;GS=29;PH=0.026;CP=0.000;CG=-8.700;AA=A;HA=24.6;HE=41.9;HC=20.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=330;GESP=A:2022/G:2544;PAC=NP_001018857.2,NP_001138550.1 GT:GQ:PL 0/0:4:0,6,39 0/1:18:14,0,35 1/1:8:54,9,0 1/1:4:17,3,0</p>

<p>19 58038964 rs12461014 T A 13.70 lowCVG AC1=6;AF1=0.7458;DB;DP=5;DP4=2,0,3,0;FQ=3.71;MQ=60;PV4=1,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=I8N;SNPEFF_CODON_CHANGE=aTt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF549;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199295.1;DN=120;DA=T/A;GM=NM_001199295.1,NM_153263.2;GL=ZNF549;FG=missense;FD=unknown;AAC=ILE/ASN;PP=8/641,8/628;CDP=23;GS=149;PH=0;CP=0.000;CG=0.703;AA=A;HA=0.0;HE=5.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=11;CPG=58038573-58039208;GESP=A:12747/T:259;PAC=NP_001186224.1,NP_694995.2 GT:GQ:PL 0/1:16:11,0,33 0/1:3:0,0,0 1/1:4:21,3,0 1/1:4:19,3,0</p>

<p>19 58928786 rs7257872 A G 26.90 highCVG AC1=2;AF1=0.2698;DB;DP=17;DP4=12,2,3,0;FQ=28.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T301A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF584;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173548.1;VDB=0.0062;DN=116;DA=A/G;GM=NM_173548.1,XM_005258628.1,XM_005258629.1,XM_005258630.1,XM_005258631.1;GL=ZNF584;FG=missense,missense,missense,3-prime-UTR,3-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=THR/ALA,THR/ALA,THR/ALA,none,none;PP=301/422,293/414,256/377,NA,NA;CDP=901,877,766,NA,NA;GS=58,58,58,NA,NA;PH=0.001;CP=0.822;CG=0.242;AA=A;HA=49.6;HE=26.3;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=609;GESP=G:4324/A:8682;PAC=NP_775819.1,XP_005258685.1,XP_005258686.1,NA,NA GT:GQ:PL 0/0:29:0,27,139 0/0:4:0,0,0 1/1:3:68,9,0 0/0:17:0,15,76</p>

<p>19 58946056 rs1465789 G A 19.10 midCVG AC1=3;AF1=0.3689;DB;DP=9;DP4=5,1,1,2;FQ=21.1;MQ=60;PV4=0.23,0.3,1,0.085;SNPEFF_AMINO_ACID_CHANGE=P252L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF132;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003433.3;VDB=0.0079;DN=88;DA=G/A;GM=NM_003433.3,XM_005259210.1;GL=ZNF132;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=252/707,239/694;CDP=755,716;GS=98;PH=0;CP=0.002;CG=0.980;AA=A;HA=14.9;HE=46.1;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=523;GESP=A:7487/G:5519;PAC=NP_003424.3,XP_005259267.1 GT:GQ:PL 0/0:7:0,6,53 0/0:7:0,6,49 1/1:3:49,6,0 0/1:13:13,0,30</p>

<p>19 59023166 rs34415062 G A 24 lowCVG AC1=4;AF1=0.5357;DB;DP=7;DP4=2,1,2,1;FQ=25.4;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=R53W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=G/A;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=53/691,53/607,53/396;CDP=157;GS=101;PH=0;CP=0.000;CG=-8.620;AA=A;HA=22.8;HE=22.2;HC=5.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=280;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=A:2195/G:10597;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,26 1/1:4:50,6,0 0/1:16:13,0,40</p>

<p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

<p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

<p>20 17716416 rs1053993 C G 89 midCVG AC1=5;AF1=0.6077;DB;DP=14;DP4=3,4,3,4;FQ=77.9;MQ=60;PV4=1,0.18,1,0.1;SNPEFF_AMINO_ACID_CHANGE=T85S;SNPEFF_CODON_CHANGE=aCt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BANF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159495.1;VDB=0.0033;DN=86;DA=C/G;GM=NM_001014977.3,NM_001159495.1,NM_178477.4,XM_005260668.1;GL=BANF2;FG=missense;FD=missense,missense,missense,unknown;AAC=THR/SER;PP=78/91,85/98,78/91,78/91;CDP=233,254,233,233;GS=58;PH=0;CP=0.107;CG=3.360;AA=G;HA=45.0;HE=22.5;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=107;GESP=G:4533/C:8473;PAC=NP_001014977.2,NP_001152967.1,NP_848572.3,XP_005260725.1 GT:GQ:PL 0/1:8:3,0,60 0/0:6:0,9,61 1/1:8:66,9,0 1/1:8:67,9,0</p>

<p>20 18505200 rs36023150 G T 94.50 highCVG AC1=5;AF1=0.6403;DB;DP=15;DP4=1,6,4,4;FQ=80.8;MQ=60;PV4=0.28,0.1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V164L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172745.1;VDB=0.0134;DN=126;DA=G/T;GM=NM_001172745.1,NM_001172746.1,NM_006363.4,NM_032985.4,NM_032986.3;GL=SEC23B;FG=missense;FD=missense;AAC=VAL/LEU;PP=164/768,146/750,164/768,164/768,164/768;CDP=490,436,490,490,490;GS=32;PH=0.994;CP=0.996;CG=4.910;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=T:97/G:12909;PAC=NP_001166216.1,NP_001166217.1,NP_006354.2,NP_116780.1,NP_116781.1 GT:GQ:PL 0/1:3:0,0,0 0/1:34:29,0,53 1/1:14:103,15,0 0/1:8:3,0,64</p>

<p>20 33874720 rs2425049 C T 49.50 midCVG AC1=3;AF1=0.4207;DB;DP=14;DP4=1,6,0,5;FQ=51.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R621Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM83C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178468.5;VDB=0.0155;DN=100;DA=C/T;GM=NM_178468.5;GL=FAM83C;FG=missense;FD=missense;AAC=ARG/GLN;PP=621/748;CDP=1862;GS=43;PH=0;CP=0.334;CG=2.850;AA=T;HA=26.5;HE=21.9;HC=15.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1056;GESP=T:4644/C:8362;PAC=NP_848563.1 GT:GQ:PL 0/0:4:0,3,23 0/0:14:0,15,87 1/1:8:76,12,0 0/1:12:19,0,8</p>

<p>20 34214173 rs11543239 G A 32.30 midCVG AC1=5;AF1=0.5743;DB;DP=10;DP4=1,3,1,2;FQ=34.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P540L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003915.5;VDB=0.0111;DN=120;DA=G/A;GM=NM_001198863.1,NM_003915.5,NM_152925.2,NM_152926.2,NM_152927.2,NM_152928.2,NR_037188.1;GL=CPNE1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,none;PP=534/537,540/543,535/538,535/538,535/538,535/538,NA;CDP=1601,1619,1604,1604,1604,1604,1978;GS=98,98,98,98,98,98,NA;PH=0.008;CP=0.016;CG=5.550;AA=G;HA=11.2;HE=5.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=131;GESP=A:911/G:12095;PAC=NP_001185792.1,NP_003906.2,NP_690902.1,NP_690903.1,NP_690904.1,NP_690905.1,NA GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:24,3,0</p>

<p>20 52645167 rs61754122 C A 66.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=6,2,0,5;FQ=69.2;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V163F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BCAS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003657.2;VDB=0.0055;DN=129;DA=C/A;GM=NM_003657.2,XM_005260587.1,XM_005260588.1,XM_005260589.1,XM_005260590.1,XM_005260591.1,XM_005260592.1,XM_005260593.1,XM_005260594.1,XM_005260595.1,XM_005260596.1;GL=BCAS1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/PHE;PP=163/585,163/630,163/630,163/616,163/608,163/594,124/591,163/571,163/552,163/538,163/507;CDP=487,487,487,487,487,487,370,487,487,487,487;GS=50;PH=0.868;CP=0.003;CG=0.450;AA=C;DV=by-frequency,by-cluster;DSP=237;GESP=A:93/C:12913;PAC=NP_003648.2,XP_005260644.1,XP_005260645.1,XP_005260646.1,XP_005260647.1,XP_005260648.1,XP_005260649.1,XP_005260650.1,XP_005260651.1,XP_005260652.1,XP_005260653.1 GT:GQ:PL 0/0:16:0,18,101 0/0:6:0,6,48 1/1:4:50,6,0 1/1:6:65,9,0</p>

<p>20 60887027 . C T 11.30 lowCVG AC1=4;AF1=0.554;DP=5;DP4=2,0,0,3;FQ=4.23;MQ=56;PV4=0.1,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G3195D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=70;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0028;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=unknown;AAC=GLY/ASP;PP=3195/3696;CDP=9584;GS=94;PH=0.299;CP=0.001;CG=-2.870;AA=C;DSP=52;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:13002;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:48,9,0 0/0:6:0,6,36</p>

<p>20 60899206 rs2427283 C T 73.40 midCVG AC1=5;AF1=0.6657;DB;DP=10;DP4=0,1,1,6;FQ=-7.71;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=V1900M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0025;DN=100;DA=C/T;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=missense;AAC=VAL/MET;PP=1900/3696;CDP=5698;GS=21;PH=0.839;CP=0.017;CG=1.990;AA=C;HE=31.7;HC=20.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=T:8377/C:4565;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:19:111,21,0 0/1:3:0,0,0</p>

<p>21 31812772 rs2832873 C A 95.50 highCVG AC1=6;AF1=0.744;DB;DP=17;DP4=3,3,4,6;FQ=70.4;MQ=60;PV4=1,0.023,1,0.032;SNPEFF_AMINO_ACID_CHANGE=L43M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRTAP15-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181623.1;VDB=0.0148;DN=100;DA=C/A;GM=NM_181623.1;GL=KRTAP15-1;FG=missense;FD=missense;AAC=LEU/MET;PP=43/138;CDP=127;GS=15;PH=0.956;CP=0.961;CG=0.826;AA=C;HA=26.2;HE=45.1;HC=34.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=228721;GESP=A:7882/C:5124;PAC=NP_853654.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:124,24,0 0/1:15:9,0,102</p>

<p>21 34907047 . T C 80.50 highCVG AC1=3;AF1=0.4181;DP=25;DP4=16,1,5,2;FQ=82.8;MQ=59;PV4=0.19,0.071,1,0.48;SNPEFF_AMINO_ACID_CHANGE=N85S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GART;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000819.4;VDB=0.0037;GM=NM_000819.4,NM_001136005.1,NM_001136006.1,NM_175085.2,XM_005260941.1;GL=GART;FG=missense;FD=unknown;AAC=ASN/SER;PP=85/1011,85/1011,85/1011,85/434,85/1011;CDP=254;GS=46;PH=0;CP=0.763;CG=2.160;AA=T;DSP=13;KP=hsa00230:PurineMetabolism|hsa00670:OneCarbonPoolByFolate|hsa01100:MetabolicPathways;GESP=T:13006;PPI=UBC:730;PAC=NP_000810.1,NP_001129477.1,NP_001129478.1,NP_780294.1,XP_005260998.1 GT:GQ:PL 0/1:32:31,0,53 0/0:38:0,39,130 1/1:11:96,15,0 0/0:4:0,3,18</p>

<p>21 37661405 rs3827183 G A 106 highCVG AC1=3;AF1=0.4579;DB;DP=17;DP4=3,2,6,3;FQ=57.2;MQ=60;PV4=1,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=G2139E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DOPEY2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005128.2;VDB=0.0104;DN=107;DA=G/A;GM=NM_005128.2;GL=DOPEY2;FG=missense;FD=missense;AAC=GLY/GLU;PP=2139/2299;CDP=6416;GS=98;PH=0.215;CP=0.199;CG=2.870;AA=G;HA=11.5;HE=11.2;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=A:1551/G:11455;PAC=NP_005119.2 GT:GQ:PL 0/0:6:0,6,53 0/1:6:85,0,0 1/1:6:64,9,0 0/0:6:0,6,45</p>

<p>21 38525356 rs1053808 T C 81.50 midCVG AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1 GT:GQ:PL 0/1:30:24,0,88 0/1:14:50,0,13 1/1:5:28,3,0 1/1:5:21,3,0</p>

<p>21 43547873 rs220159 G A 23.40 midCVG AC1=5;AF1=0.6013;DB;DP=12;DP4=5,3,3,1;FQ=27.2;MQ=60;PV4=1,0.006,1,1;SNPEFF_AMINO_ACID_CHANGE=D1336N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UMODL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173568.3;VDB=0.0075;DN=79;DA=G/A;GM=NM_001004416.2,NM_001199527.1,NM_001199528.2,NM_173568.3;GL=UMODL1;FG=missense;FD=missense;AAC=ASP/ASN;PP=1208/1319,1264/1375,1136/1247,1336/1447;CDP=3622,3790,3406,4006;GS=23;PH=0.001;CP=0.924;CG=2.420;AA=A;HA=34.4;HE=36.0;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=69;CPG=43547872-43548089;TFBS=V$ZID_01;GESP=A:4181/G:8217;PAC=NP_001004416.2,NP_001186456.1,NP_001186457.2,NP_775839.3 GT:GQ:PL 0/1:8:4,0,87 0/1:3:0,0,0 1/1:5:47,6,0 0/1:15:11,0,63</p>

<p>21 46271460 rs2236442 T C 20.70 midCVG AC1=2;AF1=0.3147;DB;DP=9;DP4=4,2,0,3;FQ=22.3;MQ=58;PV4=0.17,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=E84G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTTG1IP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001286822.1;VDB=0.0020;DN=98;DA=T/C;GM=NM_004339.3,XM_005261175.1,XM_005261176.1,XM_005261177.1;GL=PTTG1IP;FG=3-prime-UTR,missense,missense,3-prime-UTR;FD=utr-variant-3-prime,unknown,unknown,unknown;AAC=none,GLU/GLY,GLU/GLY,none;PP=NA,104/163,84/143,NA;CDP=NA,311,251,NA;GS=NA,98,98,NA;PH=0.914;CP=0.000;CG=0.454;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=83;GESP=C:902/T:12104;PAC=NA,XP_005261232.1,XP_005261233.1,NA GT:GQ:PL 0/0:14:0,12,80 0/0:3:0,0,0 1/1:4:61,9,0 0/0:8:0,6,40</p>

<p>21 47786524 rs2839227 A G 27.90 midCVG AC1=4;AF1=0.5236;DB;DP=10;DP4=6,0,0,4;FQ=31;MQ=60;PV4=0.0048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T879A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0116;DN=100;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1,XM_005261130.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ALA;PP=879/3337,879/3348,879/3347,879/3269,879/3259,761/3230,761/3140,879/2707;CDP=2635,2635,2635,2635,2635,2281,2281,2635;GS=58;PH=0.264;CP=0.002;CG=1.810;AA=A;HA=38.8;HE=17.7;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26;GESP=G:3367/A:9597;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1,XP_005261187.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,40 1/1:6:57,9,0 0/1:15:12,0,61</p>

<p>21 47851753 rs2073376 A G 39.40 lowCVG AC1=6;AF1=0.6957;DB;DP=6;DP4=1,1,1,3;FQ=10.9;MQ=60;PV4=1,0.27,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q2792R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0093;DN=96;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=2792/3337,2803/3348,2802/3347,2803/3269,2803/3259,2685/3230,2674/3140;CDP=8375,8408,8405,8408,8408,8054,8021;GS=43;PH=0;CP=0.314;CG=2.320;AA=G;HA=15.0;HE=33.5;HC=31.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=140;GESP=G:8998/A:4008;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:6:34,6,0 1/1:6:47,6,0</p>

<p>22 21800806 . T C 11.20 highCVG AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1 GT:GQ:PL 0/0:39:0,36,185 0/0:4:0,0,0 1/1:3:52,9,0 0/0:18:0,15,98</p>

<p>22 22868841 . A G 36.10 highCVG AC1=3;AF1=0.3333;DP=17;DP4=2,11,0,4;FQ=37.8;MQ=59;PV4=1,0.42,0.11,1;SNPEFF_AMINO_ACID_CHANGE=S372P;SNPEFF_CODON_CHANGE=Tca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF280A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080740.4;VDB=0.0001;GM=NM_080740.3;GL=ZNF280A;FG=missense;FD=unknown;AAC=SER/PRO;PP=372/543;CDP=1114;GS=74;PH=1;CP=0.993;CG=2.860;AA=A;DSP=1185;GESP=A:13006;PAC=NP_542778.1 GT:GQ:PL 0/0:8:0,6,48 0/0:3:0,0,0 1/1:6:78,12,0 0/0:34:0,33,165</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

<p>22 30888527 rs17738540 C T 33.50 lowCVG AC1=3;AF1=0.4062;DB;DP=8;DP4=2,2,3,1;FQ=34.5;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=V200M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC14L4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174977.3;VDB=0.0080;DN=123;DA=C/T;GM=NM_001161368.1,NM_174977.3,XM_005261583.1,XM_005261584.1,XM_005261585.1,XM_005261586.1;GL=SEC14L4;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=200/361,200/407,201/408,185/392,146/361,117/324;CDP=598,598,601,553,436,349;GS=21;PH=1;CP=0.855;CG=2.760;AA=C;HA=4.6;HE=24.7;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18;TFBS=V$HNF4_01,V$MIF1_01;GESP=T:2478/C:10528;PAC=NP_001154840.1,NP_777637.1,XP_005261640.1,XP_005261641.1,XP_005261642.1,XP_005261643.1 GT:GQ:PL 0/0:6:0,6,44 0/0:4:0,3,20 1/1:3:43,6,0 0/1:22:33,0,17</p>

<p>22 32112249 rs117719230 G A 43.30 midCVG AC1=3;AF1=0.4398;DB;DP=12;DP4=5,1,2,2;FQ=45.9;MQ=60;PV4=0.5,0.41,1,0.33;SNPEFF_AMINO_ACID_CHANGE=P526S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRR14L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173566.2;VDB=0.0050;DN=132;DA=G/A;GM=NM_173566.2,XM_005261480.1;GL=PRR14L;FG=missense;FD=missense,unknown;AAC=PRO/SER;PP=526/2152;CDP=1576;GS=74;PH=0.048;CP=0.020;CG=0.731;AA=G;DV=by-cluster;DSP=1029;GESP=G:4566;PPI=PPP2CA:808|PPP2CB:808|PPP2R1A:808;PAC=NP_775837.2,XP_005261537.1 GT:GQ:PL 0/0:8:0,9,71 0/0:8:0,9,59 1/1:6:69,9,0 0/1:4:21,3,0</p>

<p>22 32586854 rs136467 C T 6.51 midCVG AC1=3;AF1=0.3919;DB;DP=12;DP4=0,7,3,2;FQ=8.17;MQ=45;PV4=0.045,0.095,0.028,1;SNPEFF_AMINO_ACID_CHANGE=V348I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=C/T;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=VAL/ILE;PP=348/379,258/289,258/289,287/318,258/289;CDP=1042,772,772,859,772;GS=29;PH=0;CP=0.000;CG=-2.480;AA=T;HA=16.7;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=486;GESP=C:12726;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1 GT:GQ:PL 0/0:6:0,3,26 0/1:7:7,0,26 1/1:6:43,12,0 0/0:16:0,15,85</p>

<p>22 32586875 rs136468 G A 7 midCVG AC1=3;AF1=0.3215;DB;DP=12;DP4=0,7,4,1;FQ=8.3;MQ=41;PV4=0.01,1,0.0013,0.38;SNPEFF_AMINO_ACID_CHANGE=R341C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=G/A;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/CYS;PP=341/379,251/289,251/289,280/318,251/289;CDP=1021,751,751,838,751;GS=180;PH=0.999;CP=0.036;CG=0.582;AA=G;HA=1.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=465;GESP=G:12936;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1 GT:GQ:PL 0/0:6:0,3,27 0/0:5:0,1,22 1/1:5:48,12,0 0/0:17:0,15,80</p>

<p>22 42486723 rs1801311 G A 45.20 lowCVG AC1=5;AF1=0.6464;DB;DP=6;DP4=0,1,2,2;FQ=-6.55;MQ=60;PV4=1,0.24,1,0.25;SNPEFF_AMINO_ACID_CHANGE=A35V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NDUFA6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002490.3;VDB=0.0099;DN=89;DA=G/A;GM=NM_002490.3;GL=NDUFA6;FG=missense;FD=missense;AAC=ALA/VAL;PP=35/155;CDP=104;GS=64;PH=0.024;CP=0.066;CG=2.390;AA=G;HA=49.1;HE=31.4;HC=18.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=114;KP=hsa00190:OxidativePhosphorylation|hsa01100:MetabolicPathways|hsa05010:Alzheimer'sDisease|hsa05012:Parkinson'sDisease|hsa05016:Huntington'sDisease;CPG=42486350-42487246;GESP=A:5063/G:7943;PPI=NDUFA2:999;PAC=NP_002481.2 GT:GQ:PL 0/1:3:0,3,27 0/1:3:0,0,0 1/1:10:82,12,0 0/1:3:0,0,0</p>

<p>22 46644168 rs9627281 A G 73.20 highCVG AC1=3;AF1=0.3788;DB;DP=17;DP4=1,9,2,4;FQ=75.2;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V5A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDPF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207327.4;VDB=0.0179;DN=119;DA=A/G;GM=NM_207327.4;GL=CDPF1;FG=missense;FD=missense;AAC=VAL/ALA;PP=5/124;CDP=14;GS=64;PH=0;CP=0.000;CG=0.371;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=14;GESP=G:4606/A:8400;PAC=NP_997210.3 GT:GQ:PL 0/0:12:0,12,87 0/0:14:0,15,85 1/1:10:101,15,0 0/1:15:20,0,11</p>

<p>22 46655779 rs36125344 G C 91.40 midCVG AC1=5;AF1=0.6118;DB;DP=12;DP4=3,2,1,6;FQ=52.5;MQ=60;PV4=0.22,1,1,0.45;SNPEFF_AMINO_ACID_CHANGE=I1147M;SNPEFF_CODON_CHANGE=atC/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKDREJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006071.1;VDB=0.0056;DN=126;DA=G/C;GM=NM_006071.1,XM_005261302.1;GL=PKDREJ;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=1147/2254,1147/2163;CDP=3441;GS=10;PH=0.022;CP=0.000;CG=-8.750;AA=G;HA=3.3;HE=15.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=643;GESP=C:1242/G:11764;PAC=NP_006062.1,XP_005261359.1 GT:GQ:PL 0/0:8:0,12,77 0/1:19:19,0,20 1/1:8:65,9,0 1/1:8:56,9,0</p>

<p>22 50555686 rs760749 A C 999 highCVG AC1=4;AF1=0.5314;DB;DP=31;DP4=5,4,3,11;FQ=93.2;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I454L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MOV10L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018995.2;VDB=0.0115;DN=86;DA=A/C;GM=NM_001164104.1,NM_001164105.1,NM_018995.2,XM_005261923.1,XM_005261924.1;GL=MOV10L1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=ILE/LEU;PP=454/1166,434/1166,454/1212,434/1192,213/971;CDP=1360,1300,1360,1300,637;GS=5;PH=0.472;CP=0.048;CG=5.760;AA=A;HA=0.9;HE=14.8;HC=23.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=95;GESP=C:1564/A:11442;PAC=NP_001157576.1,NP_001157577.1,NP_061868.1,XP_005261980.1,XP_005261981.1 GT:GQ:PL 0/1:14:81,0,11 0/1:50:47,0,57 1/1:19:130,21,0 0/0:7:0,9,69</p>

<p>22 50728062 rs28379706 T C 62.50 midCVG AC1=4;AF1=0.5564;DB;DP=11;DP4=1,2,6,0;FQ=26.1;MQ=60;PV4=0.083,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K318E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012401.3;VDB=0.0214;DN=125;DA=T/C;GM=NM_012401.3,XM_005261908.1,XM_005261909.1,XM_005261910.1,XM_005261911.1;GL=PLXNB2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=318/1839,400/1921,318/1839,318/1839,318/1839;CDP=952,1198,952,952,952;GS=56;PH=0;CP=0.001;CG=-1.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=117;KP=hsa04360:AxonGuidance;CPG=50727442-50728654;GESP=C:5524/T:6698;PPI=TUBA3E:962|MYC:846|ARHGEF11:845;PAC=NP_036533.2,XP_005261965.1,XP_005261966.1,XP_005261967.1,XP_005261968.1 GT:GQ:PL 0/1:3:0,3,24 0/0:5:0,6,44 1/1:3:21,3,0 1/1:13:87,15,0</p>

<p>22 51009953 rs470117 C T 122 highCVG AC1=4;AF1=0.5;DB;DP=21;DP4=5,3,7,6;FQ=74.8;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=E531K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145135.1;VDB=0.0103;DN=80;DA=C/T;GM=NM_001145134.1,NM_001145135.1,NM_001145136.1,NM_001145137.1,NM_004377.3,NM_152245.2,NM_152246.2,NR_027928.2;GL=CPT1B/CHKB-CPT1B;FG=missense,missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,none;PP=497/739,531/773,450/692,531/773,531/773,531/773,531/773,NA;CDP=1489,1591,1348,1591,1591,1591,1591,3161;GS=56,56,56,56,56,56,56,NA;PH=0.966;CP=0.993;CG=5.310;AA=C;HA=10.1;HE=48.3;HC=47.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway|hsa04920:AdipocytokineSignalingPathway;GESP=T:4624/C:8382;PAC=NP_001138606.1,NP_001138607.1,NP_001138608.1,NP_001138609.1,NP_004368.1,NP_689451.1,NP_689452.1,NA GT:GQ:PL 0/0:8:0,9,56 0/1:62:106,0,58 1/1:6:58,9,0 0/1:3:0,0,0</p>

<p>X 12708329 . C A 66.40 highCVG AC1=2;AF1=0.276;DP=28;DP4=11,3,12,0;FQ=67.8;G3=0.75,7.644e-07,0.25;HWE=0.0436;MQ=60;PV4=0.22,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=L233I;SNPEFF_CODON_CHANGE=Ctt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMPD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014728.3;VDB=0.0000;GM=NM_014728.3,XM_005274632.1;GL=FRMPD4;FG=missense;FD=unknown;AAC=LEU/ILE;PP=233/1323,233/1763;CDP=697;GS=5;PH=0.999;CP=0.982;CG=3.770;AA=C;DSP=16;TFBS=V$CDC5_01;GESP=C:13006;PAC=NP_055543.2,XP_005274689.1 GT:GQ:PL 0/0:13:0,12,66 0/0:25:0,24,77 1/1:29:113,36,0 0/0:8:0,6,47</p>

<p>X 23693193 . C A 54.30 highCVG AC1=2;AF1=0.2857;DP=20;DP4=6,6,0,7;FQ=55.8;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S149Y;SNPEFF_CODON_CHANGE=tCt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRDX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006406.1;VDB=0.0080;GM=NM_006406.1,XM_005274438.1;GL=PRDX4;FG=missense;FD=unknown;AAC=SER/TYR;PP=149/272,135/258;CDP=446,404;GS=144;PH=1;CP=1.000;CG=5.840;DSP=31;GESP=C:13006;PAC=NP_006397.1,XP_005274495.1 GT:GQ:PL 0/0:16:0,15,96 0/0:6:0,3,25 1/1:14:100,21,0 0/0:19:0,18,118</p>

<p>X 27839617 rs12557898 G A 18.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,0,3;FQ=16.7;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=R65Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGEB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182506.3;VDB=0.0106;DN=120;DA=G/A;GM=NM_182506.3;GL=MAGEB10;FG=missense;FD=missense;AAC=ARG/GLN;PP=65/348;CDP=194;GS=43;PH=0.012;CP=0.000;CG=-5.040;AA=G;HA=14.0;HE=44.2;HC=8.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=243;GESP=A:4054/G:6506;PAC=NP_872312.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:56,9,0 0/0:9:0,9,53</p>

<p>X 48418659 rs235836 G A 55.30 midCVG AC1=6;AF1=0.6676;DB;DP=9;DP4=2,0,7,0;FQ=-7.74;MQ=60;PV4=1,0.14,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A455T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D25;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002536.2;VDB=0.0091;DN=79;DA=G/A;GM=NM_002536.2;GL=TBC1D25;FG=missense;FD=missense;AAC=ALA/THR;PP=455/689;CDP=1363;GS=58;PH=0;CP=0.015;CG=-7.340;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=658;GESP=A:5802/G:4761;PAC=NP_002527.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:22,3,0 0/1:25:68,0,23</p>

<p>X 49179719 rs6520418 C G 20.80 midCVG AC1=5;AF1=0.592;DB;DP=14;DP4=0,8,2,4;FQ=24.7;MQ=26;PV4=0.16,0.013,7.5e-06,0.011;SNPEFF_AMINO_ACID_CHANGE=P16R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0167;DN=116;DA=C/G;GM=NM_001098406.1;GL=GAGE12J;FG=missense;FD=missense;AAC=PRO/ARG;PP=16/118;CDP=47;GS=103;PH=0;CP=0.000;CG=0.012;AA=G;DV=by-cluster,by-2hit-2allele;DSP=38;GESP=G:1901/C:4835;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:20:0,24,118 1/1:3:17,3,0 1/1:13:48,15,0</p>

<p>X 49179755 rs7064530 G A 5.88 midCVG AC1=5;AF1=0.6096;DB;DP=13;DP4=0,6,0,5;FQ=9.09;MQ=25;PV4=1,1,6.2e-05,1;SNPEFF_AMINO_ACID_CHANGE=R28Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0001;DN=116;DA=G/A;GM=NM_001098406.1;GL=GAGE12J;FG=missense-near-splice;FD=missense;AAC=ARG/GLN;PP=28/118;CDP=83;GS=43;PH=0.881;CP=0.000;CG=-1.910;AA=G;DV=by-cluster;DSP=2;GESP=A:2418/G:4398;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:17:0,18,67 1/1:4:19,6,0 1/1:6:30,9,0</p>

<p>X 88008807 rs5984611 G A 58 lowCVG AC1=6;AF1=0.7071;DB;DP=7;DP4=0,2,1,4;FQ=4.26;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=R131H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPXCR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184771.1;VDB=0.0098;DN=114;DA=G/A;GM=NM_001184771.1,NM_033048.5;GL=CPXCR1;FG=missense;FD=missense;AAC=ARG/HIS;PP=131/302;CDP=392;GS=29;PH=0.995;CP=0.000;CG=0.096;AA=G;HA=16.1;HE=40.7;HC=5.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=400;GESP=A:3478/G:7085;PAC=NP_001171700.1,NP_149037.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:40,6,0 0/1:34:54,0,33</p>

<p>X 118281545 . A G 25.70 midCVG AC1=4;AF1=0.4454;DP=10;DP4=1,1,3,0;FQ=5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F101L;SNPEFF_CODON_CHANGE=Ttt/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1210;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020721.1;VDB=0.0000;GM=NM_020721.1;GL=KIAA1210;FG=missense;FD=unknown;AAC=PHE/LEU;PP=101/1710;CDP=301;GS=22;PH=0.6;CP=0.000;CG=-2.860;AA=A;DSP=12;GESP=A:12338;PAC=NP_065772.1 GT:GQ:PL 0/0:4:0,3,25 0/1:3:0,0,0 1/1:5:64,9,0 0/0:4:0,3,15</p>

<p>X 120183030 rs9697983 T G 66.90 midCVG AC1=3;AF1=0.4074;DB;DP=11;DP4=5,0,5,1;FQ=62.1;MQ=60;PV4=1,0.25,1,0.26;SNPEFF_AMINO_ACID_CHANGE=S498A;SNPEFF_CODON_CHANGE=Tcg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GLUD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012084.3;VDB=0.0053;DN=119;DA=T/G;GM=NM_012084.3;GL=GLUD2;FG=missense;FD=missense;AAC=SER/ALA;PP=498/559;CDP=1492;GS=99;PH=0;CP=0.092;CG=0.710;AA=T;HA=4.9;HE=3.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=64389;KP=hsa00250:Alanine,AspartateAndGlutamateMetabolism|hsa00330:ArginineAndProlineMetabolism|hsa00471:D-GlutamineAndD-glutamateMetabolism|hsa00910:NitrogenMetabolism|hsa01100:MetabolicPathways|hsa04964:ProximalTubuleBicarbonateReclamation;GESP=G:390/T:10173;PAC=NP_036216.2 GT:GQ:PL 0/1:3:0,0,0 0/0:12:0,12,76 1/1:13:109,18,0 0/0:4:0,3,27</p>

<p>X 128691863 . G T 18.30 lowCVG AC1=5;AF1=0.6661;DP=7;DP4=0,3,0,4;FQ=6.16;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=E125D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OCRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000276.3;VDB=0.0039;GM=NM_000276.3,NM_001587.3,XM_005262421.1,XM_005262422.1;GL=OCRL;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=GLU/ASP,GLU/ASP,GLU/ASP,none;PP=125/902,125/894,59/836,NA;CDP=375,375,177,NA;GS=45,45,45,NA;PH=0.001;CP=0.999;CG=0.803;AA=G;DSP=26;KP=hsa00562:InositolPhosphateMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=G:13006;PPI=RAB8A:959|CLTC:937;PAC=NP_000267.2,NP_001578.2,XP_005262478.1,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:24,3,0 0/1:32:29,0,35</p>

<p>X 148038075 . A C 11.70 midCVG AC1=5;AF1=0.5727;DP=14;DP4=3,1,0,3;FQ=15.3;MQ=60;PV4=0.14,0.022,1,1;SNPEFF_AMINO_ACID_CHANGE=K834Q;SNPEFF_CODON_CHANGE=Aaa/Caa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AFF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002025.3;VDB=0.0005;GM=NM_001169122.1,NM_001169123.1,NM_001169124.1,NM_001169125.1,NM_001170628.1,NM_002025.3;GL=AFF2;FG=missense;FD=unknown;AAC=LYS/GLN;PP=801/1277,824/1302,799/1277,795/1273,475/953,834/1312;CDP=2401,2470,2395,2383,1423,2500;GS=53;PH=1;CP=0.878;CG=5.870;AA=A;DSP=69;TFBS=V$PAX6_01;GESP=A:13006;PAC=NP_001162593.1,NP_001162594.1,NP_001162595.1,NP_001162596.1,NP_001164099.1,NP_002016.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:18,3,0 1/1:4:35,6,0 0/0:10:0,12,77</p>

<p>X 150349871 rs13440581 A G 56.20 highCVG AC1=3;AF1=0.3587;DB;DP=15;DP4=5,4,4,2;FQ=58.1;MQ=60;PV4=1,0.31,1,0.021;SNPEFF_AMINO_ACID_CHANGE=I606V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR50;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004224.3;VDB=0.0088;DN=121;DA=A/G;GM=NM_004224.3;GL=GPR50;FG=missense;FD=missense;AAC=ILE/VAL;PP=606/618;CDP=1816;GS=29;PH=0.01;CP=0.002;CG=2.130;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;CA;DSP=969;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4866/A:5438;PAC=NP_004215.2 GT:GQ:PL 0/0:9:0,9,73 0/0:7:0,6,42 1/1:3:43,6,0 0/1:55:57,0,53</p>

<p>X 153039502 rs6643791 G C 15.60 lowCVG AC1=6;AF1=0.683;DB;DP=5;DP4=0,2,3,0;FQ=-16.8;MQ=60;PV4=0.1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=E1179D;SNPEFF_CODON_CHANGE=gaG/gaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163257.1;VDB=0.0123;DN=116;DA=G/C;GM=NM_001163257.1,NM_005393.2,XM_005274694.1,XM_005274695.1,XM_005274696.1;GL=PLXNB3;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=GLU/ASP;PP=1179/1933,1156/1910,1203/1957,1203/1933,809/1563;CDP=3537,3468,3609,3609,2427;GS=45;PH=0.005;CP=0.008;CG=-0.654;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=52;KP=hsa04360:AxonGuidance;GESP=C:7729/G:2554;PPI=ARHGEF11:895;PAC=NP_001156729.1,NP_005384.2,XP_005274751.1,XP_005274752.1,XP_005274753.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:27,3,0 0/1:15:23,0,13</p>

<p>X 153284192 rs1059702 A G 86.90 midCVG AC1=5;AF1=0.64;DB;DP=10;DP4=2,0,5,3;FQ=-11.8;MQ=60;PV4=1,0.45,1,0.18;SNPEFF_AMINO_ACID_CHANGE=F196S;SNPEFF_CODON_CHANGE=tTt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IRAK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001569.3;VDB=0.0123;DN=86;DA=A/G;GM=NM_001025242.1,NM_001025243.1,NM_001569.3,XM_005274668.1;GL=IRAK1;FG=missense;FD=missense,missense,missense,unknown;AAC=PHE/SER;PP=196/683,196/634,196/713,222/709;CDP=587,587,587,665;GS=155;PH=0.002;CP=0.002;CG=3.680;AA=G;HA=1.8;HE=20.5;HC=19.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=47;KP=hsa04210:Apoptosis|hsa04620:Toll-likeReceptorSignalingPathway|hsa04722:NeurotrophinSignalingPathway|hsa05140:Leishmaniasis|hs</p>


        <p>
          <strong>IDs and variants associated:</strong> H00058;OPTN|DCTN1|SETX|TARDBP|FUS|ANG|MAPT|NEFH|PRPH|ALS2|SOD1|VAPB|FIG4;hsa05014
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 10
        </p>
        <p>
          <strong>Missense Mutations:</strong> 623
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 72.3
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse17">Spinocerebellar ataxia (SCA); Machado-Joseph disease (SCA3)</a>
      </h4>
    </div>
    <div id="collapse17" class="panel-collapse collapse">
      <div class="panel-body">
        <p>6 152443744 rs2295190 G T 29 lowCVG AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:34,6,0 0/1:24:31,0,23</p>

<p>10 126678087 rs111888382 C T 16.70 highCVG AC1=6;AF1=0.6827;DB;DP=36;DP4=3,5,13,14;FQ=21.4;MQ=11;PV4=0.7,0.29,1.9e-05,0.13;SNPEFF_AMINO_ACID_CHANGE=*986;SNPEFF_CODON_CHANGE=taG/taA;SNPEFF_EFFECT=SYNONYMOUS_STOP;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0149;DN=132;DA=C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=synonymous;FD=synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=stop;PP=446/446,446/446,986/986,514/514,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,421/421,421/421;CDP=1338,1338,2958,1542,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1263,1263;CP=1.000;CG=5.120;AA=C;DV=by-cluster;DSP=181;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:4:0,3,38 1/1:11:16,12,0 1/1:30:45,31,0</p>

<p>10 126678092 rs73373169 G A 25.10 highCVG AC1=5;AF1=0.6731;DB;DP=36;DP4=2,5,12,13;FQ=29.5;MQ=12;PV4=0.43,1,1.3e-05,0.07;SNPEFF_AMINO_ACID_CHANGE=Q985*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0155;DN=131;DA=G/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=stop-gained;FD=stop-gained,stop-gained,stop-gained,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/stop;PP=445/446,445/446,985/986,513/514,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,420/421,420/421;CDP=1333,1333,2953,1537,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1258,1258;CP=1.000;CG=5.120;AA=G;DG;DV=by-cluster;DSP=176;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$MYB_Q6;GESP=G:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:3:0,5,41 1/1:17:22,18,0 1/1:28:49,28,0</p>

<p>CTBP2 10 126678177 . G T 39.20 highCVG AC1=6;AF1=0.7163;DP=34;DP4=3,4,8,17;FQ=44.8;MQ=28;PV4=0.67,0.00015,8e-15,1;SNPEFF_AMINO_ACID_CHANGE=N956K;SNPEFF_CODON_CHANGE=aaC/aaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0013;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=unknown;AAC=ASN/LYS;PP=416/446,416/446,956/986,484/514,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,391/421,391/421;CDP=1248,1248,2868,1452,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1173,1173;GS=94;PH=0.613;CP=1.000;CG=3.120;AA=G;DSP=91;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$BACH1_01,V$TAXCREB_02;GESP=G:12998;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:5:1,0,7 0/1:4:0,4,98 1/1:24:24,24,0 1/1:33:60,33,0</p>

<p>10 126678221 rs77752853 C A 43.30 highCVG AC1=6;AF1=0.7316;DB;DP=30;DP4=3,4,3,17;FQ=49.1;MQ=30;PV4=0.29,0.031,2.4e-12,0.15;SNPEFF_AMINO_ACID_CHANGE=A942S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=131;DA=C/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/SER;PP=402/446,402/446,942/986,470/514,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,377/421,377/421;CDP=1204,1204,2824,1408,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1129,1129;GS=99;PH=0.599;CP=0.998;CG=5.120;AA=C;DV=by-cluster;DSP=47;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:6:0,1,11 0/1:11:4,0,114 1/1:21:24,21,0 1/1:25:60,24,0</p>

<p>10 126678241 rs113985797 G C 8.10 highCVG AC1=4;AF1=0.4981;DB;DP=26;DP4=4,7,2,13;FQ=10.6;MQ=39;PV4=0.35,1.5e-07,2.9e-13,1;SNPEFF_AMINO_ACID_CHANGE=A935G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=132;DA=G/C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=395/446,395/446,935/986,463/514,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,370/421,370/421;CDP=1184,1184,2804,1388,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1109,1109;GS=60;PH=0.016;CP=1.000;CG=5.120;AA=G;DV=by-frequency,by-cluster;DSP=27;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/0:4:0,2,39 0/0:6:0,6,121 1/1:10:17,18,0 0/1:10:34,0,4</p>



        <p>
          <strong>IDs and variants associated:</strong> H00063;NOP56|AFG3L2|TTBK2|BEAN1|FGF14|SYNE1|ATXN10|ITPR1|KCNC3|ATXN3|PDYN|ANO10|PPP2R2B|TDP1|PRKCG|ADCK3|ATXN1|ATXN2|ATXN7|SPTBN2|TBP|CACNA1A|ZNF592;hsa04730
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 1
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse18">Leber optic atrophy; Leber hereditary optic atrophy (LHON)</a>
      </h4>
    </div>
    <div id="collapse18" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 179504043 rs6658180 C G 13.70 lowCVG AC1=3;AF1=0.3645;DB;DP=8;DP4=1,3,1,2;FQ=15.5;MQ=59;PV4=1,0.026,0.14,0.14;SNPEFF_AMINO_ACID_CHANGE=Q993E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AXDND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144696.5;VDB=0.0075;DN=116;DA=C/G;GM=NM_144696.5,NR_073544.1;GL=AXDND1;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=GLN/GLU,none;PP=993/1013,NA;CDP=2977,3097;GS=29,NA;PH=0;CP=0.000;CG=-4.940;AA=C;HA=6.0;HE=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=GA-rich;DSP=55;GESP=G:4250/C:8608;PAC=NP_653297.3,NA GT:GQ:PL 0/0:5:0,3,27 0/0:3:0,0,0 1/1:4:53,9,0 0/0:10:0,9,69</p>

<p>3 12859004 rs3732678 A C 27.40 lowCVG AC1=5;AF1=0.5732;DB;DP=7;DP4=3,1,2,1;FQ=30.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H858P;SNPEFF_CODON_CHANGE=cAc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CAND2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001162499.1;VDB=0.0098;DN=107;DA=A/C/G/T;GM=NM_001162499.1,NM_012298.2;GL=CAND2;FG=missense;FD=missense;AAC=HIS/PRO;PP=858/1237,765/1120;CDP=2573,2294;GS=77;PH=0;CP=1.000;CG=3.880;AA=C;HA=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=372;TFBS=V$PAX5_02;GESP=C:8707/A:3773;PAC=NP_001155971.1,NP_036430.1 GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:19,3,0</p>

<p>9 19290794 rs41269019 A C 75.40 midCVG AC1=3;AF1=0.3437;DB;DP=11;DP4=0,5,3,3;FQ=69.4;MQ=60;PV4=0.18,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=I241L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DENND4C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017925.5;VDB=0.0195;DN=127;DA=A/C;GM=NM_017925.5,NR_073201.2;GL=DENND4C;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=ILE/LEU,none;PP=241/1910,NA;CDP=721,807;GS=5,NA;PH=0.996;CP=1.000;CG=5.440;AA=A;DV=by-frequency,by-cluster;DSP=81;GESP=C:267/A:11599;PAC=NP_060395.5,NA GT:GQ:PL 0/0:5:0,3,26 0/0:5:0,3,25 1/1:12:119,18,0 0/0:10:0,9,61</p>

<p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>17 13980350 rs2072279 G A 65.90 highCVG AC1=4;AF1=0.4922;DB;DP=16;DP4=0,8,6,0;FQ=68.8;MQ=60;PV4=0.00033,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R159Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COX10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001303.3;VDB=0.0150;DN=96;DA=G/A;GM=NM_001303.3,XM_005256458.1,XM_005256459.1;GL=COX10;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=159/444,159/240,159/238;CDP=476;GS=43;PH=0.178;CP=0.605;CG=5.490;AA=G;HA=37.0;HE=44.2;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=24;KP=hsa00190:OxidativePhosphorylation|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa01100:MetabolicPathways;GESP=A:6378/G:6628;PAC=NP_001294.2,XP_005256515.1,XP_005256516.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:25,3,0 1/1:12:88,15,0 0/0:19:0,21,111</p>

        <p>
          <strong>IDs and variants associated:</strong>  H00068;ATP6|COX1|COX3|CYTB|ND1|ND2|ND4|ND4L|ND5|ND6;hsa00190
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 5
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.5
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse19">Asthma</a>
      </h4>
    </div>
    <div id="collapse19" class="panel-collapse collapse">
      <div class="panel-body">
        <p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00079;ADRB2|MS4A2|HLA|DQB1|HLA|DRB1|IL4|IL4R|IL13|TNF|ADAM33|CD14|CD14|CD14;hsa05310
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse20">Systemic lupus erythematosus</a>
      </h4>
    </div>
    <div id="collapse20" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 207790088 rs3811381 C G 67.70 midCVG AC1=5;AF1=0.5957;DB;DP=9;DP4=0,4,1,4;FQ=29.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2277R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=41;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000651.4;VDB=0.0076;DN=107;DA=C/G;GM=NM_000573.3,NM_000651.4,XM_005273064.1;GL=CR1;FG=missense;FD=missense,missense,unknown;AAC=PRO/ARG;PP=1827/2040,2277/2490,2149/2362;CDP=5480,6830,6446;GS=103;PH=0.643;CP=0.000;CG=-5.580;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;KP=hsa04610:ComplementAndCoagulationCascades|hsa04640:HematopoieticCellLineage|hsa05140:Leishmaniasis;GESP=G:2006/C:10176;PPI=CR2:846;PAC=NP_000564.2,NP_000642.3,XP_005273121.1 GT:GQ:PL 0/1:35:36,0,34 0/0:4:0,6,36 1/1:6:51,6,0 1/1:4:25,3,0</p>

<p>1 223284528 rs5744174 A G 264 highCVG AC1=5;AF1=0.6321;DB;DP=26;DP4=4,2,5,13;FQ=58.7;MQ=60;PV4=0.15,1,1,0.16;SNPEFF_AMINO_ACID_CHANGE=F616L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003268.5;VDB=0.0132;DN=114;DA=A/G;GM=NM_003268.5,XM_005273241.1,XM_005273242.1,XM_005273243.1;GL=TLR5;FG=missense;FD=missense,unknown,unknown,unknown;AAC=PHE/LEU;PP=616/859;CDP=1846;GS=22;PH=0;CP=0.000;CG=-11.200;AA=G;HA=10.7;HE=42.7;HC=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1850;KP=hsa04620:Toll-likeReceptorSignalingPathway|hsa05130:PathogenicEscherichiaColiInfection;GESP=G:4352/A:8654;PPI=MYD88:937;PAC=NP_003259.2,XP_005273298.1,XP_005273299.1,XP_005273300.1 GT:GQ:PL 0/1:22:75,0,21 0/0:8:0,12,83 1/1:9:60,9,0 1/1:32:179,33,0</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>15 81598269 rs11556218 T G 14.60 lowCVG AC1=2;AF1=0.3027;DB;DP=7;DP4=3,1,1,2;FQ=16;MQ=60;PV4=0.49,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=N1147K;SNPEFF_CODON_CHANGE=aaT/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_172217.3;VDB=0.0093;DN=120;DA=T/G;GM=NM_001172128.1,NM_004513.5,NM_172217.3,XM_005254342.1,XM_005254343.1,XM_005254344.1,XM_005254345.1,XM_005254346.1;GL=IL16;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=ASN/LYS;PP=1147/1332,446/632,1147/1333,1194/1380,1194/1379,537/723,537/722,446/632;CDP=3441,1338,3441,3582,3582,1611,1611,1338;GS=94;PH=1;CP=0.007;CG=-6.300;AA=T;HA=25.0;HE=10.0;HC=20.6;DG;DV=by-frequency,by-cluster,by-1000G;DSP=21;TFBS=V$YY1_01;GESP=G:1601/T:11405;PPI=CD4:846;PAC=NP_001165599.1,NP_004504.3,NP_757366.2,XP_005254399.1,XP_005254400.1,XP_005254401.1,XP_005254402.1,XP_005254403.1 GT:GQ:PL 0/0:6:0,3,19 0/0:8:0,6,35 1/1:4:55,9,0 0/0:6:0,3,26</p>


        <p>
          <strong>IDs and variants associated:</strong> H00080;CR1|CRP|FCGR2A|FCGR2B|FCGR3A|FCGR3B|ZNF423|GRIN2A|GRIN2B|3F3A|DQA1|DQB1|DRB1|ICAM1|IL1RN|IL2RA|IL2RB|IL2RG|IL4|IL6|IL10|IL12RB1|IL13|IL16|IL18|2AFB1|PDCD1|SNRPB|SNRPD1|TRIM21|SSB|THBD|TLR5|1QA|TNF|VCAM1|HIST1H2BI|HIST1H4A|CD27|CD40LG;hsa05322
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 5
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.5
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse21">Hashimoto's thyroiditis</a>
      </h4>
    </div>
    <div id="collapse21" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>8 103573006 rs2916569 G A 17.80 midCVG AC1=4;AF1=0.504;DB;DP=10;DP4=2,1,4,0;FQ=4.62;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S216N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024410.3;VDB=0.0051;DN=101;DA=G/A;GM=NM_024410.3;GL=ODF1;FG=missense;FD=missense;AAC=SER/ASN;PP=216/251;CDP=647;GS=46;PH=0;CP=0.053;CG=-7.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;RT=TGCAGCCCC;DSP=327;GESP=G:13006;PAC=NP_077721.2 GT:GQ:PL 0/0:5:0,6,25 0/1:3:0,0,0 1/1:4:34,6,0 0/1:18:24,0,15</p>

<p>9 114359624 rs1053959 C A 122 highCVG AC1=3;AF1=0.3851;DB;DP=23;DP4=6,8,4,5;FQ=124;MQ=59;PV4=1,1,1,0.0013;SNPEFF_AMINO_ACID_CHANGE=A27S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTGR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146108.1;VDB=0.0165;DN=86;DA=C/A;GM=NM_001146108.1,NM_001146109.1,NM_012212.3;GL=PTGR1;FG=missense;FD=missense;AAC=ALA/SER;PP=27/330,27/302,27/330;CDP=79;GS=99;PH=0.019;CP=0.038;CG=-0.431;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=28;GESP=A:7644/C:5362;PAC=NP_001139580.1,NP_001139581.1,NP_036344.2 GT:GQ:PL 0/0:29:0,30,158 0/1:10:91,0,4 1/1:8:78,12,0 0/0:9:0,9,72</p>

<p>12 53682468 . G T 35.30 lowCVG AC1=5;AF1=0.6499;DP=5;DP4=1,0,0,4;FQ=-11.7;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1565S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ESPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012291.4;VDB=0.0084;GM=NM_012291.4;GL=ESPL1;FG=missense;FD=unknown;AAC=ALA/SER;PP=1565/2121;CDP=4693;GS=99;PH=0.01;CP=0.003;CG=2.840;AA=G;DSP=16;KP=hsa04110:CellCycle|hsa04114:OocyteMeiosis;GESP=G:12990;PPI=PTTG1:987|IRF2BPL:987;PAC=NP_036423.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:72,12,0 0/1:3:0,3,21</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>

<p>21 46271460 rs2236442 T C 20.70 midCVG AC1=2;AF1=0.3147;DB;DP=9;DP4=4,2,0,3;FQ=22.3;MQ=58;PV4=0.17,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=E84G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTTG1IP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001286822.1;VDB=0.0020;DN=98;DA=T/C;GM=NM_004339.3,XM_005261175.1,XM_005261176.1,XM_005261177.1;GL=PTTG1IP;FG=3-prime-UTR,missense,missense,3-prime-UTR;FD=utr-variant-3-prime,unknown,unknown,unknown;AAC=none,GLU/GLY,GLU/GLY,none;PP=NA,104/163,84/143,NA;CDP=NA,311,251,NA;GS=NA,98,98,NA;PH=0.914;CP=0.000;CG=0.454;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=83;GESP=C:902/T:12104;PAC=NA,XP_005261232.1,XP_005261233.1,NA GT:GQ:PL 0/0:14:0,12,80 0/0:3:0,0,0 1/1:4:61,9,0 0/0:8:0,6,40</p>




        <p>
          <strong>IDs and variants associated:</strong> H00081;CGA|CTLA4|HLA|DQA1|HLA|DQB1|HLA|DRB1|IL2|SLC5A5|TG|TPO|TSHB;hsa05320
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 11
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse22">Graves' disease</a>
      </h4>
    </div>
    <div id="collapse22" class="panel-collapse collapse">
      <div class="panel-body">

        <p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00082;CGA|CTLA4|HLA|DQA1|HLA|DQB1|HLA|DRB1|TSHB|TSHR;hsa05320
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 3
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.3
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse23">Allograft rejection</a>
      </h4>
    </div>
    <div id="collapse23" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          17	38121993	rs3894194	G	A	35.30	lowCVG	AC1=6;AF1=0.7326;DB;DP=5;DP4=1,0,2,2;FQ=-12.4;MQ=60;PV4=1,0.21,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R18Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSDMA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178171.4;VDB=0.0024;DN=108;DA=G/A;GM=NM_178171.4;GL=GSDMA;FG=missense;FD=missense;AAC=ARG/GLN;PP=18/446;CDP=53;GS=43;PH=0.016;CP=1.000;CG=3.340;AA=G;HA=28.6;HE=47.7;HC=46.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=58;GESP=A:4966/G:7324;PAC=NP_835465.2	GT:GQ:PL	0/1:3:0,0,0	0/1:4:0,3,22	1/1:9:54,9,0	1/1:4:21,3,0
        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00083;CTLA4|ACE|DMA|ICAM1|IFNG|IL10|ITGB3|TGFB1|TGFB2|TGFB3|TNF;hsa05330
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse24">Graft-versus-host disease</a>
      </h4>
    </div>
    <div id="collapse24" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          2	113537223	rs17561	C	A	31.40	midCVG	AC1=2;AF1=0.2957;DB;DP=14;DP4=9,1,4,0;FQ=32.8;MQ=60;PV4=1,0.45,1,1;SNPEFF_AMINO_ACID_CHANGE=A114S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000575.3;VDB=0.0004;DN=63;DA=C/A;GM=NM_000575.3;GL=IL1A;FG=missense;FD=missense;AAC=ALA/SER;PP=114/272;CDP=340;GS=99;PH=0.982;CP=0.009;CG=4.440;AA=C;HA=14.5;HE=27.6;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;KP=hsa04010:MAPKSignalingPathway|hsa04060:Cytokine-cytokineReceptorInteraction|hsa04210:Apoptosis|hsa04640:HematopoieticCellLineage|hsa04940:TypeIDiabetesMellitus|hsa05020:PrionDiseases|hsa05140:Leishmaniasis|hsa05332:Graft-versus-hostDisease;GESP=A:3403/C:9603;PPI=S100A13:959;PAC=NP_000566.3	GT:GQ:PL	0/0:26:0,24,122	0/0:6:0,3,27	1/1:6:74,12,0	0/0:6:0,3,27
        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00084;IFNG|IL1A|IL1RN|IL2|IL6|IL10|IL13|TGFB1|TGFB2|TGFB3|TNF|TNFRSF1B;hsa05332
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse25">T-B+Severe combined immunodeficiencies (SCIDs)</a>
      </h4>
    </div>
    <div id="collapse25" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	117311147	rs699738	C	A	39	midCVG	AC1=3;AF1=0.4223;DB;DP=13;DP4=1,6,5,1;FQ=41.3;MQ=60;PV4=0.029,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=H266Q;SNPEFF_CODON_CHANGE=caC/caA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001767.3;VDB=0.0019;DN=86;DA=C/A;GM=NM_001767.3,XM_005271319.1,XM_005271320.1;GL=CD2;FG=missense;FD=missense,unknown,unknown;AAC=HIS/GLN;PP=266/352,292/378,225/311;CDP=798,876,675;GS=24;PH=0.085;CP=0.000;CG=-2.990;AA=C;HA=30.3;HE=9.4;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=62;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04640:HematopoieticCellLineage;GESP=A:2986/C:10020;PPI=CD58:996|CD2BP2:995|CD2AP:974|SH3KBP1:937|YES1:937|LCK:846|PTPRC:846;PAC=NP_001758.2,XP_005271376.1,XP_005271377.1	GT:GQ:PL	0/0:4:0,3,22	0/1:14:14,0,14	1/1:11:71,15,0	0/0:14:0,15,77

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00091;IL2RA|IL2RG|IL7R|JAK3|PTPRC|FOXN1|CD3D|CD3E|CD247;hsa05340
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse26">T-B-Severe combined immunodeficiencies (SCIDs)</a>
      </h4>
    </div>
    <div id="collapse26" class="panel-collapse collapse">
      <div class="panel-body">

        <p>5 42719239 rs6180 A C 98.30 midCVG AC1=6;AF1=0.705;DB;DP=10;DP4=0,3,0,7;FQ=25.8;MQ=60;PV4=1,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=I551L;SNPEFF_CODON_CHANGE=Atc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GHR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242399.2;VDB=0.0032;DN=52;DA=A/C;GM=NM_000163.4,NM_001242399.2,NM_001242400.2,NM_001242401.3,NM_001242402.2,NM_001242403.2,NM_001242404.2,NM_001242405.2,NM_001242406.2,NM_001242460.1,NM_001242462.1,XM_005248288.1;GL=GHR;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,3-prime-UTR,missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,utr-variant-3-prime,unknown;AAC=ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,none,ILE/LEU;PP=544/639,551/646,544/639,544/639,544/639,544/639,544/639,544/639,544/639,522/617,NA,522/617;CDP=1630,1651,1630,1630,1630,1630,1630,1630,1630,1564,NA,1564;GS=5,5,5,5,5,5,5,5,5,5,NA,5;PH=0.933;CP=0.951;CG=3.430;AA=A;HA=41.6;HE=46.2;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=685;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04630:Jak-STATSignalingPathway;TFBS=V$PAX4_04;GESP=C:5675/A:7331;PPI=CSHL1:998|SOCS2:937|JAK2:937|PTPN11:846|UBC:846|BTRC:846|NCOA6:845|SHC1:845|CISH:845;PAC=NP_000154.1,NP_001229328.1,NP_001229329.1,NP_001229330.1,NP_001229331.1,NP_001229332.1,NP_001229333.1,NP_001229334.1,NP_001229335.1,NP_001229389.1,NA,XP_005248345.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:12:83,12,0 1/1:9:60,9,0</p>

<p>5 178540975 rs1054480 G A 49.80 lowCVG AC1=6;AF1=0.6833;DB;DP=6;DP4=2,0,1,3;FQ=-5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1177S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADAMTS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014244.4;VDB=0.0112;DN=86;DA=G/A;GM=NM_014244.4;GL=ADAMTS2;FG=missense;FD=missense;AAC=PRO/SER;PP=1177/1212;CDP=3529;GS=74;PH=0.651;CP=0.973;CG=5.050;AA=G;HA=2.7;HE=25.0;HC=32.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=351;GESP=A:2884/G:10122;PAC=NP_055059.2 GT:GQ:PL 0/0:4:0,6,32 0/1:3:0,0,0 1/1:4:26,3,0 1/1:8:65,9,0</p>

<p>11 56756399 rs10896563 G T 28.10 midCVG AC1=3;AF1=0.3852;DB;DP=9;DP4=2,2,0,4;FQ=29.8;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G4V;SNPEFF_CODON_CHANGE=gGa/gTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5AK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005323.1;VDB=0.0173;DN=120;DA=G/T;GM=NM_001005323.1;GL=OR5AK2;FG=missense;FD=missense;AAC=GLY/VAL;PP=4/310;CDP=11;GS=109;PH=0.262;CP=0.000;CG=-4.160;AA=G;HA=5.0;HC=41.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=193759;KP=hsa04740:OlfactoryTransduction;GESP=T:720/G:12274;PAC=NP_001005323.1 GT:GQ:PL 0/0:7:0,6,44 0/0:7:0,6,36 1/1:7:69,12,0 0/0:3:0,0,0</p>

<p>14 105413204 rs2582514 G T 89 midCVG AC1=6;AF1=0.707;DB;DP=10;DP4=1,2,5,2;FQ=7.98;MQ=55;PV4=0.5,1,0.15,0.061;SNPEFF_AMINO_ACID_CHANGE=R2862S;SNPEFF_CODON_CHANGE=Cgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0059;DN=100;DA=G/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=2862/5796,2762/5696;CDP=8584,8284;GS=110;PH=0;CP=0.000;CG=0.525;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7933;GESP=T:7253/G:4947;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:39:79,0,38</p>

<p>14 105415608 rs11850848 G C 19.70 midCVG AC1=4;AF1=0.4707;DB;DP=12;DP4=8,0,4,0;FQ=22.3;MQ=57;PV4=1,3.7e-05,0.00042,1;SNPEFF_AMINO_ACID_CHANGE=H2060Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0001;DN=120;DA=G/C;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=HIS/GLN;PP=2060/5796,1960/5696;CDP=6180,5880;GS=24;PH=0.022;CP=0.000;CG=-2.470;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=5529;GESP=C:2565/G:6715;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/0:23:0,24,114 1/1:8:59,12,0 0/1:3:0,0,0</p>

<p>14 105418391 rs11625007 C T 60.30 midCVG AC1=5;AF1=0.6662;DB;DP=10;DP4=2,1,0,6;FQ=18.1;MQ=60;PV4=0.083,1,0.26,1;SNPEFF_AMINO_ACID_CHANGE=V1133I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0023;DN=120;DA=C/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=1133/5796,1033/5696;CDP=3397,3097;GS=29;PH=0.017;CP=0.001;CG=-7.740;AA=T;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2746;GESP=T:5916/C:6260;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:50:70,0,48</p>

        <p>
          <strong>IDs and variants associated:</strong> H00092;ADA|AK2|RAG1|RAG2|DCLRE1C;hsa05340
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 6
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.6
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse27">Combined immunodeficiencies (CIDs)</a>
      </h4>
    </div>
    <div id="collapse27" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 117311147 rs699738 C A 39 midCVG AC1=3;AF1=0.4223;DB;DP=13;DP4=1,6,5,1;FQ=41.3;MQ=60;PV4=0.029,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=H266Q;SNPEFF_CODON_CHANGE=caC/caA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001767.3;VDB=0.0019;DN=86;DA=C/A;GM=NM_001767.3,XM_005271319.1,XM_005271320.1;GL=CD2;FG=missense;FD=missense,unknown,unknown;AAC=HIS/GLN;PP=266/352,292/378,225/311;CDP=798,876,675;GS=24;PH=0.085;CP=0.000;CG=-2.990;AA=C;HA=30.3;HE=9.4;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=62;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04640:HematopoieticCellLineage;GESP=A:2986/C:10020;PPI=CD58:996|CD2BP2:995|CD2AP:974|SH3KBP1:937|YES1:937|LCK:846|PTPRC:846;PAC=NP_001758.2,XP_005271376.1,XP_005271377.1 GT:GQ:PL 0/0:4:0,3,22 0/1:14:14,0,14 1/1:11:71,15,0 0/0:14:0,15,77</p>

<p>21 31812772 rs2832873 C A 95.50 highCVG AC1=6;AF1=0.744;DB;DP=17;DP4=3,3,4,6;FQ=70.4;MQ=60;PV4=1,0.023,1,0.032;SNPEFF_AMINO_ACID_CHANGE=L43M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRTAP15-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181623.1;VDB=0.0148;DN=100;DA=C/A;GM=NM_181623.1;GL=KRTAP15-1;FG=missense;FD=missense;AAC=LEU/MET;PP=43/138;CDP=127;GS=15;PH=0.956;CP=0.961;CG=0.826;AA=C;HA=26.2;HE=45.1;HC=34.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=228721;GESP=A:7882/C:5124;PAC=NP_853654.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:124,24,0 0/1:15:9,0,102</p>


        <p>
          <strong>IDs and variants associated:</strong> H00093;IL2RG|LCK|CIITA|PNP|RAG1|RAG2|RFX5|RFXAP|STIM1|TAP1|TAP2|ZAP70|ORAI1|RFXANK|CD8A|CD40|CD40LG;hsa05340
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse28">Classic complement pathway component defects</a>
      </h4>
    </div>
    <div id="collapse28" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 13802325 rs3013105 T C 173 midCVG AC1=6;AF1=0.7094;DB;DP=13;DP4=2,1,5,5;FQ=5.86;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K292E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRC38;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010847.1;VDB=0.0076;DN=101;DA=T/C;GM=NM_001010847.1;GL=LRRC38;FG=missense;FD=missense;AAC=LYS/GLU;PP=292/295;CDP=874;GS=56;PH=0;CP=0.701;CG=5.250;AA=C;HA=48.9;HE=39.6;HC=39.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=243;PAC=NP_001010847.1 GT:GQ:PL 0/1:30:36,0,31 0/1:5:0,3,14 1/1:10:76,9,0 1/1:15:107,15,0</p>

<p>1 31902279 rs34728687 G A 95.60 highCVG AC1=3;AF1=0.4377;DB;DP=19;DP4=8,2,3,6;FQ=98.1;MQ=59;PV4=0.07,0.45,0.15,0.065;SNPEFF_AMINO_ACID_CHANGE=G312S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERINC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199038.1;VDB=0.0042;DN=126;DA=G/A;GM=NM_001199037.1,NM_001199038.1,NM_001199039.1,NM_018565.3,NM_178865.4,XM_005270822.1;GL=SERINC2;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=GLY/SER;PP=307/460,312/465,248/401,307/460,303/456,248/401;CDP=919,934,742,919,907,742;GS=56;PH=0.003;CP=0.000;CG=-4.810;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=36;GESP=A:1075/G:11931;PAC=NP_001185966.1,NP_001185967.1,NP_001185968.1,NP_061035.2,NP_849196.2,XP_005270879.1 GT:GQ:PL 0/0:4:0,3,16 0/0:11:0,12,92 1/1:14:100,18,0 0/1:44:42,0,63</p>

<p>1 112308953 rs197412 T C 58.10 midCVG AC1=5;AF1=0.6066;DB;DP=10;DP4=1,1,7,0;FQ=8.93;MQ=60;PV4=0.22,1,1,0.045;SNPEFF_AMINO_ACID_CHANGE=I636T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DDX20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007204.4;VDB=0.0159;DN=79;DA=T/C;GM=NM_007204.4;GL=DDX20;FG=missense;FD=missense;AAC=ILE/THR;PP=636/825;CDP=1907;GS=89;PH=0;CP=0.331;CG=3.860;AA=C;HA=19.7;HE=43.3;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=595;GESP=C:6295/T:6711;PPI=SMN1:984|SMN2:983|GEMIN4:973|GEMIN2:933|AGO2:885|GEMIN5:836|PRRC2A:786|HDAC5:716;PAC=NP_009135.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:97,21,0 0/0:5:0,6,43</p>

<p>2 29295186 rs75276619 C T 29.50 midCVG AC1=3;AF1=0.3733;DB;DP=12;DP4=4,4,3,1;FQ=31.5;MQ=60;PV4=0.58,0.41,1,0.46;SNPEFF_AMINO_ACID_CHANGE=A648T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf71;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001029883.2;VDB=0.0002;DN=131;DA=C/T;GM=NM_001029883.2;GL=C2orf71;FG=missense;FD=missense;AAC=ALA/THR;PP=648/1289;CDP=1942;GS=58;PH=0.2;CP=0.000;CG=-3.320;AA=C;DV=by-frequency,by-cluster;DSP=1727;GESP=T:113/C:12423;PAC=NP_001025054.1 GT:GQ:PL 0/1:11:10,0,51 0/0:9:0,9,67 1/1:5:64,9,0 0/0:7:0,6,37</p>

<p>2 178417142 rs11694988 C T 97.90 midCVG AC1=5;AF1=0.5858;DB;DP=13;DP4=2,4,3,4;FQ=78.7;MQ=50;PV4=1,0.46,0.044,0.32;SNPEFF_AMINO_ACID_CHANGE=R117H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152517.2;VDB=0.0034;DN=120;DA=C/T;GM=NM_152517.2;GL=TTC30B;FG=missense;FD=missense;AAC=ARG/HIS;PP=117/666;CDP=350;GS=29;PH=0.944;CP=0.996;CG=4.300;AA=C;HA=15.8;HE=14.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14341;GESP=T:11079/C:1927;PAC=NP_689730.2 GT:GQ:PL 0/0:11:0,15,107 0/1:18:35,0,16 1/1:8:64,9,0 1/1:6:47,6,0</p>

<p>2 228476253 rs2138402 C G 23.70 lowCVG AC1=4;AF1=0.4527;DB;DP=8;DP4=5,0,1,2;FQ=26.3;MQ=59;PV4=0.11,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E104Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf83;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020161.3;VDB=0.0071;DN=96;DA=C/G;GM=NM_001162483.1,NM_020161.3;GL=C2orf83;FG=3-prime-UTR,missense;FD=utr-variant-3-prime,missense;AAC=none,GLU/GLN;PP=NA,104/151;CDP=NA,310;GS=NA,29;PH=0;CP=0.000;CG=0.457;AA=G;HA=33.2;HE=6.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=120;GESP=G:10654/C:2352;PAC=NA,NP_064546.3 GT:GQ:PL 0/0:11:0,12,62 0/0:4:0,3,25 1/1:4:42,6,0 0/1:4:26,3,0</p>

<p>3 14755617 rs6790129 C G 50.50 midCVG AC1=3;AF1=0.3771;DB;DP=13;DP4=5,2,5,0;FQ=51.9;MQ=60;PV4=0.47,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L422V;SNPEFF_CODON_CHANGE=Cta/Gta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C3orf20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032137.4;VDB=0.0072;DN=116;DA=C/G;GM=NM_001184957.1,NM_001184958.1,NM_032137.4,XM_005265511.1;GL=C3orf20;FG=missense;FD=missense,missense,missense,unknown;AAC=LEU/VAL;PP=300/783,300/783,422/905,422/905;CDP=898,898,1264,1264;GS=32;PH=0.873;CP=0.948;CG=1.570;AA=C;HA=43.4;HE=39.8;HC=34.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=50;TFBS=V$HOX13_01;GESP=G:5559/C:7447;PAC=NP_001171886.1,NP_001171887.1,NP_115513.4,XP_005265568.GT:GQ:PL 0/0:9:0,9,40 0/0:9:0,9,56 1/1:5:56,9,0 0/1:14:40,0,8</p>

<p>3 66419956 rs13874 C T 145 highCVG AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA GT:GQ:PL 0/0:7:0,9,70 0/1:3:0,0,0 1/1:18:112,21,0 0/1:31:77,0,28</p>

<p>3 195452799 rs2550232 C T 11.30 midCVG AC1=8;AF1=1;DB;DP=9;DP4=4,1,1,3;FQ=-26.1;MQ=21;PV4=0.21,0.34,1,0.42;SNPEFF_AMINO_ACID_CHANGE=T442I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282506.1;VDB=0.0102;DN=100;DA=C/T;GM=NM_001098516.1,NM_152673.2,XM_005269303.1,XM_005269304.1,XM_005269305.1,XM_005269306.1,XM_005269307.1,XM_005269308.1,XM_005269309.1;GL=MUC20;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ILE;PP=236/504,271/539,442/727,442/710,407/692,407/692,407/692,271/553,272/540;CDP=707,812,1325,1325,1220,1220,1220,812,815;GS=89;PH=0.002;CP=0.000;CG=-2.630;AA=C;DG;DV=by-cluster,by-2hit-2allele;DSP=131;GESP=C:12316;PAC=NP_001091986.1,NP_689886.2,XP_005269360.1,XP_005269361.1,XP_005269362.1,XP_005269363.1,XP_005269364.1,XP_005269365.1,XP_005269366.1 GT:GQ:PL 0/0:2:0,6,0/1:3:16,0,0 1/1:4:20,3,0 1/1:4:14,3,0</p>

<p>4 2451694 rs3108494 T C 127 midCVG AC1=6;AF1=0.6856;DB;DP=10;DP4=1,1,6,2;FQ=6.6;MQ=60;PV4=1,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=C253R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0035;DN=103;DA=T/C;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=CYS/ARG;PP=253/647;CDP=757;GS=180;CP=0.976;CG=3.340;AA=C;DG;DV=by-frequency,by-cluster,by-1000G,by-other-pop;DSP=34;PAC=NP_001180211.1 GT:GQ:PL 0/1:20:22,0,20 0/1:5:0,3,26 1/1:7:51,6,0 1/1:15:99,15,0</p>

<p>4 8603105 rs115392829 G A 26.80 lowCVG AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA GT:GQ:PL 0/0:7:0,6,47 0/0:4:0,3,18 1/1:7:67,12,0 0/1:3:0,0,0</p>

<p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

<p>4 147741397 rs149464457 T A 23.60 midCVG AC1=2;AF1=0.3051;DB;DP=14;DP4=8,2,0,4;FQ=25.1;MQ=60;PV4=0.015,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=Q353H;SNPEFF_CODON_CHANGE=caA/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300761.1;VDB=0.0056;DN=134;DA=T/A;GM=NM_031956.2,XM_005263269.1,XM_005263270.1;GL=TTC29;FG=missense;FD=missense,unknown,unknown;AAC=GLN/HIS;PP=327/476,353/502,353/493;CDP=981,1059,1059;GS=24;PH=0.999;CP=1.000;CG=5.910;AA=T;DV=by-cluster;DSP=4;GESP=A:4/T:11824;PAC=NP_114162.2,XP_005263326.1,XP_005263327.1 GT:GQ:PL 0/0:16:0,15,85 0/0:3:0,0,0 1/1:6:66,12,0 0/0:16:0,15,87</p>

<p>4 184190233 rs11941467 G T 54.10 highCVG AC1=3;AF1=0.395;DB;DP=17;DP4=5,8,1,3;FQ=56.3;MQ=60;PV4=1,1,1,0.073;SNPEFF_AMINO_ACID_CHANGE=A773S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WWC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024949.5;VDB=0.0099;DN=120;DA=G/T;GM=NM_024949.5;GL=WWC2;FG=missense;FD=missense;AAC=ALA/SER;PP=773/1193;CDP=2317;GS=99;PH=0.099;CP=1.000;CG=3.500;AA=G;HA=1.4;HE=10.4;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;GESP=T:1111/G:11895;PAC=NP_079225.5 GT:GQ:PL 0/0:14:0,15,112 0/1:4:31,3,0 1/1:5:71,9,0 0/0:23:0,24,151</p>

<p>5 54439466 rs1021580 G A 260 highCVG AC1=6;AF1=0.805;DB;DP=19;DP4=1,2,4,12;FQ=-17.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R121C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170402.1;VDB=0.0082;DN=86;DA=G/A;GM=NM_001145734.2,NM_001170402.1,NM_152623.2,XM_005248450.1;GL=CDC20B;FG=missense;FD=missense,missense,missense,unknown;AAC=ARG/CYS;PP=121/478,121/520,121/516,121/373;CDP=361;GS=180;PH=0;CP=0.001;CG=-1.690;AA=A;HA=49.1;HE=18.6;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;GESP=A:9758/G:3248;PAC=NP_001139206.2,NP_001163873.1,NP_689836.2,XP_005248507.1 GT:GQ:PL 0/1:4:87,0,4 0/1:13:40,0,15 1/1:12:63,9,0 1/1:18:113,15,0</p>

<p>5 86695274 rs2230641 A G 50.70 midCVG AC1=4;AF1=0.4542;DB;DP=9;DP4=1,4,2,2;FQ=51.2;MQ=60;PV4=0.52,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V270A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCNH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001239.3;VDB=0.0061;DN=98;DA=A/G/T;GM=NM_001199189.1,NM_001239.3,XM_005248627.1,XM_005248628.1,XM_005248629.1;GL=CCNH;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=VAL/ALA;PP=217/271,270/324,270/330,270/326,217/277;CDP=650,809,809,809,650;GS=64;PH=0.965;CP=1.000;CG=5.680;AA=A;HA=5.0;HE=20.3;HC=8.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;KP=hsa03420:NucleotideExcisionRepair|hsa04110:CellCycle;GESP=G:2278/A:10728;PPI=CDK7:999|ERCC3:997|GTF2H1:989|CDK2:989|MNAT1:905|CDK8:846|ERCC2:845;PAC=NP_001186118.1,NP_001230.1,XP_005248684.1,XP_005248685.1,XP_005248686.1 GT:GQ:PL 0/0:11:0,12,76 0/1:3:0,0,0 1/1:4:48,6,0 0/1:22:44,0,18</p>

<p>7 95822389 . G A 54.60 highCVG AC1=3;AF1=0.3723;DP=16;DP4=4,2,0,7;FQ=56.7;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P192L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001160210.1;VDB=0.0003;GM=NM_001160210.1,NM_014251.2,NR_027662.1;GL=SLC25A13;FG=missense,missense,non-coding-exon;FD=unknown;AAC=PRO/LEU,PRO/LEU,none;PP=192/677,192/676,NA;CDP=575,575,650;GS=98,98,NA;PH=0.243;CP=1.000;CG=4.100;AA=G;DSP=41;GESP=G:13006;PPI=BRE:923|BABAM1:871;PAC=NP_001153682.1,NP_055066.1,NA GT:GQ:PL 0/0:9:0,9,60 0/0:9:0,9,66 1/1:15:98,21,0 0/1:3:0,0,0</p>

<p>8 67796120 rs17332410 C G 17.80 midCVG AC1=3;AF1=0.3679;DB;DP=14;DP4=6,5,2,1;FQ=19.7;MQ=60;PV4=1,0.028,1,1;SNPEFF_AMINO_ACID_CHANGE=L322V;SNPEFF_CODON_CHANGE=Ctc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCMDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173518.4;VDB=0.0061;DN=123;DA=C/G;GM=NM_001136160.1,NM_001136161.1,NM_173518.4,XM_005251173.1,XM_005251174.1;GL=MCMDC2;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=LEU/VAL;PP=322/633,322/591,322/682,322/682,259/619;CDP=964,964,964,964,775;GS=32;PH=0.946;CP=1.000;CG=4.680;AA=C;HA=0.0;HE=6.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=110;TFBS=V$CEBPA_01;GESP=G:735/C:12271;PAC=NP_001129632.1,NP_001129633.1,NP_775789.3,XP_005251230.1,XP_005251231.1 GT:GQ:PL 0/0:15:0,15,96 0/1:5:20,3,0 1/1:3:44,6,0 0/0:18:0,18,122</p>

<p>8 97614625 rs3816208 G A 21 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1 GT:GQ:PL 0/0:9:0,6,45 0/0:18:0,15,111 1/1:3:63,9,0 0/0:9:0,6,47</p>

<p>8 104412688 rs141856398 T C 58.40 highCVG AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA GT:GQ:PL 0/0:4:0,3,24 0/0:6:0,6,40 1/1:3:38,6,0 0/1:64:63,0,86</p>

<p>9 107367008 rs10156474 T C 41.50 lowCVG AC1=6;AF1=0.7416;DB;DP=5;DP4=1,0,2,2;FQ=-17.1;MQ=40;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K301E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13C2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004481.1;VDB=0.0107;DN=119;DA=T/C;GM=NM_001004481.1;GL=OR13C2;FG=missense;FD=missense;AAC=LYS/GLU;PP=301/319;CDP=901;GS=56;PH=0.693;CP=0.996;CG=3.530;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=143126;KP=hsa04740:OlfactoryTransduction;GESP=C:4176/T:8826;PAC=NP_001004481.1 GT:GQ:PL 0/1:3:0,0,0 0/1:14:14,0,14 1/1:9:64,9,0 0/1:3:0,0,0</p>

<p>9 125391026 rs1476858 A C 98.50 highCVG AC1=5;AF1=0.6085;DB;DP=20;DP4=2,9,7,2;FQ=103;MQ=60;PV4=0.022,0.0062,1,0.29;SNPEFF_AMINO_ACID_CHANGE=C263W;SNPEFF_CODON_CHANGE=tgT/tgG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0050;DN=88;DA=A/C;GM=NM_001004450.1;GL=OR1B1;FG=missense;FD=missense;AAC=CYS/TRP;PP=263/319;CDP=789;GS=215;PH=0;CP=0.921;CG=2.820;HA=42.5;HE=15.5;HC=6.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=117959;KP=hsa04740:OlfactoryTransduction;GESP=C:9234/A:3772;PAC=NP_001004450.1 GT:GQ:PL 0/0:8:0,12,90 0/1:19:14,0,103 1/1:6:46,6,0 1/1:14:87,15,0</p>

<p>9 130165973 rs34064803 T C 79.40 highCVG AC1=3;AF1=0.4231;DB;DP=25;DP4=0,4,2,15;FQ=51.8;MQ=60;PV4=1,0.0002,1,0.08;SNPEFF_AMINO_ACID_CHANGE=I253T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC2A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014580.4;VDB=0.0144;DN=126;DA=T/C;GM=NM_001271711.1,NM_001271712.1,NM_014580.4,NR_073416.1,XM_005251949.1;GL=SLC2A8;FG=missense,missense,missense,non-coding-exon,missense;FD=missense,missense,missense,nc-transcript-variant,unknown;AAC=ILE/THR,ILE/THR,ILE/THR,none,ILE/THR;PP=253/412,90/315,253/478,NA,90/315;CDP=758,269,758,284,269;GS=89,89,89,NA,89;PH=0.878;CP=0.927;CG=5.420;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=C:1021/T:11985;PAC=NP_001258640.1,NP_001258641.1,NP_055395.2,NA,XP_005252006.1 GT:GQ:PL 0/0:6:0,6,47 0/1:3:0,0,0 1/1:46:122,51,0 0/0:6:0,6,45</p>

<p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

<p>10 102265183 rs2295772 G A 205 highCVG AC1=5;AF1=0.6183;DB;DP=23;DP4=2,4,6,8;FQ=50.4;MQ=60;PV4=1,1,0.26,0.38;SNPEFF_AMINO_ACID_CHANGE=P372S;SNPEFF_CODON_CHANGE=Cca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC31B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015490.3;VDB=0.0054;DN=100;DA=G/A;GM=NM_015490.3,XM_005269676.1,XM_005269677.1,XM_005269678.1,XM_005269679.1;GL=SEC31B;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=PRO/SER,PRO/SER,PRO/SER,PRO/SER,none;PP=372/1180,372/1263,372/1263,215/1106,NA;CDP=1114,1114,1114,643,NA;GS=74,74,74,74,NA;PH=0;CP=0.368;CG=3.470;AA=A;HA=14.0;HE=18.7;HC=16.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=66;TFBS=V$ZID_01;GESP=A:2751/G:10255;PPI=SF3A2:944;PAC=NP_056305.1,XP_005269733.1,XP_005269734.1,XP_005269735.1,NA GT:GQ:PL 0/1:18:44,0,17 0/0:8:0,12,78 1/1:6:52,6,0 1/1:26:158,27,0</p>

<p>11 48346547 rs79042268 C T 999 highCVG AC1=5;AF1=0.5818;DB;DP=30;DP4=1,10,12,7;FQ=67.7;MQ=60;PV4=0.0067,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=P19S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0115;DN=131;DA=C/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=PRO/SER;PP=19/330;CDP=55;GS=74;CP=0.000;CG=-5.180;AA=C;DG;DV=by-cluster;DSP=157792;KP=hsa04740:OlfactoryTransduction;GESP=C:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:29:93,0,27 0/1:14:10,0,42 1/1:5:108,5,0 0/1:40:84,0,38</p>

<p>11 48347306 rs73465911 G T 151 highCVG AC1=5;AF1=0.6041;DB;DP=29;DP4=11,2,7,7;FQ=149;MQ=60;PV4=0.1,0.051,1,1;SNPEFF_AMINO_ACID_CHANGE=V272F;SNPEFF_CODON_CHANGE=Gtt/Ttt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0060;DN=131;DA=G/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=VAL/PHE;PP=272/330;CDP=814;GS=50;PH=0.175;CP=0.997;CG=5.970;AA=G;DG;DV=by-frequency,by-cluster;DSP=158551;KP=hsa04740:OlfactoryTransduction;TFBS=V$RFX1_01;GESP=G:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:47:101,0,46 0/1:16:11,0,66 1/1:8:46,9,0 0/1:42:37,0,79</p>

<p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>14 39556185 rs8018720 G C 105 midCVG AC1=5;AF1=0.6629;DB;DP=14;DP4=1,3,6,3;FQ=23.7;MQ=59;PV4=0.27,0.14,0.26,0.26;SNPEFF_AMINO_ACID_CHANGE=L211V;SNPEFF_CODON_CHANGE=Ctt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006364.2;VDB=0.0026;DN=116;DA=G/C;GM=NM_006364.2,XM_005267262.1,XM_005267263.1;GL=SEC23A;FG=missense;FD=missense,unknown,unknown;AAC=LEU/VAL;PP=211/766,211/790,182/737;CDP=631,631,544;GS=32;PH=0;CP=0.826;CG=4.770;AA=C;HA=10.8;HE=12.5;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;TFBS=V$MEF2_03;GESP=C:10877/G:2129;PPI=SEC24D:959|SEC24A:900|SEC24C:845;PAC=NP_006355.2,XP_005267319.1,XP_005267320.1 GT:GQ:PL 0/1:3:0,0,0 0/1:43:41,0,46 1/1:12:74,12,0 0/1:15:31,0,14</p>

<p>16 840378 rs3765263 A G 66.70 midCVG AC1=3;AF1=0.3829;DB;DP=14;DP4=6,0,8,0;FQ=58.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K244R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHTF18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022092.2;VDB=0.0000;DN=107;DA=A/G;GM=NM_022092.2,XM_005255470.1,XM_005255471.1,XM_005255472.1;GL=CHTF18;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=LYS/ARG,LYS/ARG,LYS/ARG,none;PP=244/976,128/860,272/1004,NA;CDP=731,383,815,NA;GS=26,26,26,NA;PH=0;CP=0.081;CG=-5.390;AA=G;HA=33.2;HE=22.3;HC=42.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4000/A:8074;PPI=PCNA:939|RFC4:846|DSCC1:845|RFC3:845|RFC2:845;PAC=NP_071375.1,XP_005255527.1,XP_005255528.1,NA GT:GQ:PL 0/0:12:0,12,60 0/1:3:0,0,0 1/1:19:110,24,0 0/0:7:0,6,40</p>

<p>16 15131962 rs1135999 A G 59.20 highCVG AC1=5;AF1=0.6381;DB;DP=21;DP4=3,1,1,13;FQ=9.29;MQ=57;PV4=0.019,0.29,0.23,1;SNPEFF_AMINO_ACID_CHANGE=S287P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0035;DN=86;DA=A/G;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=SER/PRO;PP=182/206,182/206,287/311;CDP=544,544,859;GS=74;PH=0;CP=1.000;CG=5.870;AA=G;HA=13.3;HE=27.7;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;TFBS=V$FREAC3_01,V$STAT1_01,V$STAT3_01,V$STAT5A_01,V$STAT5B_01;GESP=G:3550/A:9444;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/0:4:0,6,44 0/1:3:0,0,0 1/1:5:23,6,0 0/1:3:76,2,0</p>

<p>17 39983808 rs1046404 G C 71.20 lowCVG AC1=6;AF1=0.6722;DB;DP=8;DP4=1,1,1,5;FQ=6.96;MQ=60;PV4=0.46,0.14,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S213C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0127;DN=86;DA=G/C;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/CYS,none,none;PP=213/301,NA,NA;CDP=638,784,940;GS=112,NA,NA;PH=0.991;CP=0.812;CG=5.070;AA=G;HA=20.0;HE=24.2;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=71;GESP=C:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:18,0,20 0/1:4:0,3,26 1/1:9:55,9,0 1/1:7:43,6,0</p>

<p>17 39983820 rs1046403 G A 58.40 lowCVG AC1=5;AF1=0.6474;DB;DP=7;DP4=1,1,1,4;FQ=7.67;MQ=60;PV4=1,0.17,1,0.19;SNPEFF_AMINO_ACID_CHANGE=A209V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0201;DN=86;DA=G/A;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ALA/VAL,none,none;PP=209/301,NA,NA;CDP=626,772,928;GS=64,NA,NA;PH=0.001;CP=0.060;CG=2.870;AA=G;HA=20.8;HE=23.8;HC=17.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;GESP=A:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:17,0,20 0/1:4:0,3,26 1/1:9:64,9,0 1/1:4:21,3,0</p>

<p>17 41361960 rs8482 A G 76.10 midCVG AC1=4;AF1=0.5419;DB;DP=12;DP4=0,4,2,5;FQ=27.5;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H923R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NBR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005899.4;VDB=0.0215;DN=52;DA=A/G;GM=NM_005899.3,NM_031858.2,NM_031862.2;GL=NBR1;FG=missense;FD=missense;AAC=HIS/ARG;PP=923/967;CDP=2768;GS=29;PH=0.968;CP=1.000;CG=5.860;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;GESP=G:3148/A:7152;PPI=SQSTM1:994|MAP1LC3A:937|UBC:757|MAP1LC3C:756;PAC=NP_005890.2,NP_114064.1,NP_114068.1 GT:GQ:PL 0/1:9:6,0,31 0/1:3:0,0,0 1/1:15:112,18,0 0/0:5:0,6,35</p>

<p>18 11610052 rs74410464 T C 70.60 highCVG AC1=5;AF1=0.6397;DB;DP=15;DP4=3,5,2,4;FQ=70.8;MQ=55;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V153A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=T/C;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=VAL/ALA;PP=153/339;CDP=458;GS=64;CP=0.000;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79887;PAC=XP_934053.1 GT:GQ:PL 0/1:10:5,0,68 0/1:3:0,0,0 1/1:11:93,12,0 0/1:18:13,0,41</p>

<p>18 11610073 rs75301503 G A 72.40 midCVG AC1=4;AF1=0.5075;DB;DP=14;DP4=2,5,3,3;FQ=75.1;MQ=53;PV4=0.59,1,0.44,0.48;SNPEFF_AMINO_ACID_CHANGE=C160Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=G/A;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=CYS/TYR;PP=160/339;CDP=479;GS=194;CP=0.000;AA=G;DG;DV=by-cluster,by-1000G;DSP=79866;PAC=XP_934053.1 GT:GQ:PL 0/0:10:0,12,79 0/1:3:0,0,1/1:12:100,15,0 0/1:19:16,0,48</p>

<p>18 11610163 rs62084100 G C 19.60 highCVG AC1=4;AF1=0.4585;DB;DP=15;DP4=4,4,3,3;FQ=22.1;MQ=47;PV4=1,0.45,0.072,0.13;SNPEFF_AMINO_ACID_CHANGE=G190A;SNPEFF_CODON_CHANGE=gGc/gCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0171;DN=129;DA=G/C/T;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=GLY/ALA;PP=190/339;CDP=569;GS=60;CP=0.105;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79776;PAC=XP_934053.1 GT:GQ:PL 0/0:6:0,6,100 0/1:3:0,0,0 1/1:8:59,12,0 0/0:3:0,1,17</p>

<p>18 29339975 . C T 29.30 lowCVG AC1=3;AF1=0.4714;DP=7;DP4=2,1,0,4;FQ=-8.34;MQ=24;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G227D;SNPEFF_CODON_CHANGE=gGt/gAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A52;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001034172.2;VDB=0.0001;GM=NM_001034172.2;GL=SLC25A52;FG=missense;FD=unknown;AAC=GLY/ASP;PP=227/308;CDP=680;GS=94;PH=0.006;CP=0.077;CG=-1.260;AA=C;DSP=35519;GESP=C:13006;PAC=NP_001029344.3 GT:GQ:PL 0/0:4:0,3,22 0/1:3:0,0,0 1/1:8:69,12,0 0/0:5:0,6,7</p>

<p>19 22941591 rs62119159 A T 42.80 midCVG AC1=6;AF1=0.739;DB;DP=13;DP4=2,1,0,10;FQ=19.6;MQ=46;PV4=0.038,4.5e-06,1,1;SNPEFF_AMINO_ACID_CHANGE=C374S;SNPEFF_CODON_CHANGE=Tgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080409.2;VDB=0.0101;DN=129;DA=A/T;GM=NM_001080409.2;GL=ZNF99;FG=missense;FD=missense;AAC=CYS/SER;PP=374/865;CDP=1120;GS=112;PH=1;CP=0.032;CG=1.120;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=894;GESP=T:1032/A:11442;PAC=NP_001073878.2 GT:GQ:PL 0/1:12:6,0,51 0/1:3:0,0,0 1/1:27:74,27,0 0/1:3:0,0,0</p>

<p>19 45716364 rs189063316 C T 11.60 lowCVG AC1=5;AF1=0.5837;DB;DP=5;DP4=1,0,2,0;FQ=-12.4;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R398Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXOC3L2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138568.3;VDB=0.0116;DN=135;DA=C/T;GM=NM_138568.3;GL=EXOC3L2;FG=missense;FD=missense;AAC=ARG/GLN;PP=398/410;CDP=1193;GS=43;PH=0.46;CP=0.112;CG=1.260;AA=C;DV=by-cluster;DSP=252;GESP=T:121/C:12673;PAC=NP_612635.3 GT:GQ:PL 0/1:3:0,0,0/0:4:0,3,19 1/1:4:46,6,0 0/1:3:0,0,0</p>

<p>19 59023166 rs34415062 G A 24 lowCVG AC1=4;AF1=0.5357;DB;DP=7;DP4=2,1,2,1;FQ=25.4;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=R53W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=G/A;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=53/691,53/607,53/396;CDP=157;GS=101;PH=0;CP=0.000;CG=-8.620;AA=A;HA=22.8;HE=22.2;HC=5.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=280;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=A:2195/G:10597;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,26 1/1:4:50,6,0 0/1:16:13,0,40</p>

<p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

<p>20 18505200 rs36023150 G T 94.50 highCVG AC1=5;AF1=0.6403;DB;DP=15;DP4=1,6,4,4;FQ=80.8;MQ=60;PV4=0.28,0.1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V164L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172745.1;VDB=0.0134;DN=126;DA=G/T;GM=NM_001172745.1,NM_001172746.1,NM_006363.4,NM_032985.4,NM_032986.3;GL=SEC23B;FG=missense;FD=missense;AAC=VAL/LEU;PP=164/768,146/750,164/768,164/768,164/768;CDP=490,436,490,490,490;GS=32;PH=0.994;CP=0.996;CG=4.910;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=T:97/G:12909;PAC=NP_001166216.1,NP_001166217.1,NP_006354.2,NP_116780.1,NP_116781.1 GT:GQ:PL 0/1:3:0,0,0 0/1:34:29,0,53 1/1:14:103,15,0 0/1:8:3,0,64</p>

<p>20 60887027 . C T 11.30 lowCVG AC1=4;AF1=0.554;DP=5;DP4=2,0,0,3;FQ=4.23;MQ=56;PV4=0.1,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G3195D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=70;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0028;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=unknown;AAC=GLY/ASP;PP=3195/3696;CDP=9584;GS=94;PH=0.299;CP=0.001;CG=-2.870;AA=C;DSP=52;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:13002;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:48,9,0 0/0:6:0,6,36</p>

<p>20 60899206 rs2427283 C T 73.40 midCVG AC1=5;AF1=0.6657;DB;DP=10;DP4=0,1,1,6;FQ=-7.71;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=V1900M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0025;DN=100;DA=C/T;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=missense;AAC=VAL/MET;PP=1900/3696;CDP=5698;GS=21;PH=0.839;CP=0.017;CG=1.990;AA=C;HE=31.7;HC=20.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=T:8377/C:4565;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:19:111,21,0 0/1:3:0,0,0</p>

<p>21 38525356 rs1053808 T C 81.50 midCVG AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1 GT:GQ:PL 0/1:30:24,0,88 0/1:14:50,0,13 1/1:5:28,3,0 1/1:5:21,3,0</p>

<p>22 21800806 . T C 11.20 highCVG AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1 GT:GQ:PL 0/0:39:0,36,185 0/0:4:0,0,0 1/1:3:52,9,0 0/0:18:0,15,98</p>
        <p>
          <strong>IDs and variants associated:</strong> H00102;ARVCF|C1QA|C1QB|C1QC|C1R|C1S|C2|C3|C4A;hsa04610
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 46
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.46
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse29">Late complement pathway defects</a>
      </h4>
    </div>
    <div id="collapse29" class="panel-collapse collapse">
      <div class="panel-body">

        <p>2 97638038 rs1730120 A G 94.30 midCVG AC1=5;AF1=0.5991;DB;DP=9;DP4=1,1,4,3;FQ=-3.35;MQ=60;PV4=1,0.4,1,0.26;SNPEFF_AMINO_ACID_CHANGE=V55A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM178B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001122646.2;VDB=0.0156;DN=89;DA=A/G;GM=NM_001122646.2,XM_005263964.1,XM_005263965.1;GL=FAM178B;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=55/680,29/654,55/494;CDP=164,86,164;GS=64;CP=0.000;CG=-4.560;AA=A;HA=30.4;HE=13.6;HC=45.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=22;GESP=G:3773/A:771;PAC=NP_001116118.2,XP_005264021.1,XP_005264022.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:12:60,0,10</p>

<p>6 28228342 rs1679709 A G 23.10 lowCVG AC1=4;AF1=0.4699;DB;DP=7;DP4=3,1,2,1;FQ=25.8;MQ=60;PV4=1,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=E398G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NKAPL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007531.2;VDB=0.0074;DN=89;DA=A/G;GM=NM_001007531.2;GL=NKAPL;FG=missense;FD=missense;AAC=GLU/GLY;PP=398/403;CDP=1193;GS=98;PH=0.002;CP=1.000;CG=4.630;AA=G;HA=27.2;HE=13.4;HC=14.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1382;TFBS=V$IRF1_01,V$IRF2_01,V$TCF11_01;GESP=G:10661/A:2345;PAC=NP_001007532.1 GT:GQ:PL 0/1:19:19,0,19 0/0:8:0,9,64 1/1:4:45,6,0 0/1:3:0,0,0</p>

<p>7 95041016 rs12026 G C 34.40 midCVG AC1=3;AF1=0.4042;DB;DP=11;DP4=3,3,2,3;FQ=36.7;MQ=60;PV4=1,0.035,1,0.43;SNPEFF_AMINO_ACID_CHANGE=A148G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PON2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000305.2;VDB=0.0045;DN=52;DA=G/C;GM=NM_000305.2,NM_001018161.1,XM_005250453.1,XM_005250454.1;GL=PON2;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=148/355,136/343,80/287,78/285;CDP=443,407,239,233;GS=60;PH=0.009;CP=1.000;CG=2.550;AA=G;HA=34.9;HE=24.2;HC=20.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=40;KP=hsa01100:MetabolicPathways;GESP=C:3203/G:9803;PAC=NP_000296.2,NP_001018171.1,XP_005250510.1,XP_005250511.1 GT:GQ:PL 0/0:4:0,3,26 0/0:6:0,6,41 1/1:3:33,6,0 0/1:45:44,0,55</p>

<p>12 52908917 rs11549950 T C 25.50 lowCVG AC1=3;AF1=0.3646;DB;DP=8;DP4=3,1,0,3;FQ=27.4;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S528G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000424.3;VDB=0.0108;DN=120;DA=T/C;GM=NM_000424.3,XM_005268862.1;GL=KRT5;FG=missense;FD=missense,unknown;AAC=SER/GLY;PP=528/591;CDP=1582;GS=56;PH=0;CP=0.000;CG=0.084;AA=C;HA=25.2;HE=9.0;HC=4.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CCG)n;CA;DSP=108;GESP=C:2251/T:10755;PPI=KRT14:959;PAC=NP_000415.2,XP_005268919.1 GT:GQ:PL 0/0:5:0,3,24 0/0:3:0,0,0 1/1:4:65,9,0 0/0:10:0,9,70</p>

<p>12 103249087 rs77958223 T C 77.80 highCVG AC1=4;AF1=0.4954;DB;DP=15;DP4=5,3,3,4;FQ=80.7;MQ=60;PV4=0.62,0.25,1,0.071;SNPEFF_AMINO_ACID_CHANGE=E178G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000277.1;VDB=0.0177;DN=131;DA=T/A/C;GM=NM_000277.1;GL=PAH;FG=missense;FD=missense;AAC=GLU/GLY;PP=178/453;CDP=533;GS=98;PH=0.961;CP=1.000;CG=5.730;AA=T;DV=by-cluster;CA;DSP=24;KP=hsa00360:PhenylalanineMetabolism|hsa00400:Phenylalanine,TyrosineAndTryptophanBiosynthesis|hsa01100:MetabolicPathways;GESP=T:13006;PAC=NP_000268.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:16,3,0 1/1:15:109,18,0 0/0:19:0,21,126</p>

<p>19 37117302 rs3108171 A G 50.10 lowCVG AC1=5;AF1=0.5824;DB;DP=8;DP4=1,2,5,0;FQ=19.9;MQ=60;PV4=0.11,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=E168G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0027;DN=103;DA=A/G;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=GLU/GLY;PP=167/550,168/551,168/551,167/550;CDP=500,503,503,500;GS=98;PH=0.103;CP=1.000;CG=3.820;AA=A;HA=28.0;HE=40.7;HC=16.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=271;GESP=G:8458/A:4542;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:19:16,0,31</p>

      <p>
          <strong>IDs and variants associated:</strong> H00103;8A|8B|8G;hsa04610
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 6
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.6
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse30">Other well-defined immunodeficiency syndromes</a>
      </h4>
    </div>
    <div id="collapse30" class="panel-collapse collapse">
      <div class="panel-body">
        <p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

<p>16 15131962 rs1135999 A G 59.20 highCVG AC1=5;AF1=0.6381;DB;DP=21;DP4=3,1,1,13;FQ=9.29;MQ=57;PV4=0.019,0.29,0.23,1;SNPEFF_AMINO_ACID_CHANGE=S287P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0035;DN=86;DA=A/G;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=SER/PRO;PP=182/206,182/206,287/311;CDP=544,544,859;GS=74;PH=0;CP=1.000;CG=5.870;AA=G;HA=13.3;HE=27.7;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;TFBS=V$FREAC3_01,V$STAT1_01,V$STAT3_01,V$STAT5A_01,V$STAT5B_01;GESP=G:3550/A:9444;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/0:4:0,6,44 0/1:3:0,0,0 1/1:5:23,6,0 0/1:3:76,2,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00107;AIRE|XIAP|ITK|SH2D1A|FOXP3|RMRP|STAT3|TBX1|TYK2|WAS;hsa05340
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse31">Mucopolysaccharidosis type VI (MPS6); Maroteaux-Lamy syndrome</a>
      </h4>
    </div>
    <div id="collapse31" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	78181477	rs1065757	C	T	31.80	midCVG	AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1	GT:GQ:PL	0/0:18:0,21,102	0/1:3:0,0,0	1/1:7:56,9,0	1/1:3:19,3,0
        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00131;ARSB;hsa04142,hsa00531
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse32">Krabbe disease; Globoid cell leukodystrophy</a>
      </h4>
    </div>
    <div id="collapse32" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          14	88459795	rs73312836	C	T	22.20	midCVG	AC1=3;AF1=0.3888;DB;DP=11;DP4=3,5,2,1;FQ=24.4;MQ=60;PV4=0.55,1,1,0.005;SNPEFF_AMINO_ACID_CHANGE=A17T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GALC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001201402.1;VDB=0.0092;DN=130;DA=C/T;GM=NM_001201402.1;GL=GALC;FG=missense;FD=missense;AAC=ALA/THR;PP=17/660;CDP=49;GS=58;CP=0.000;CG=-0.125;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=69;KP=hsa00600:SphingolipidMetabolism|hsa01100:MetabolicPathways|hsa04142:Lysosome;PAC=NP_001188331.1	GT:GQ:PL	0/0:15:0,15,85	0/1:4:20,3,0	1/1:3:48,6,0	0/0:9:0,9,64

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00135;GALC|PSAP;hsa04142,hsa00600
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>


  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse33">alpha-Mannosidosis</a>
      </h4>
    </div>
    <div id="collapse33" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          19	12772165	rs1054487	G	A	27.30	lowCVG	AC1=3;AF1=0.3966;DB;DP=8;DP4=1,3,3,1;FQ=29.4;MQ=60;PV4=0.49,0.0045,1,1;SNPEFF_AMINO_ACID_CHANGE=T312I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAN2B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000528.3;VDB=0.0063;DN=86;DA=G/A/T;GM=NM_000528.3,NM_001173498.1,XM_005259913.1;GL=MAN2B1;FG=missense;FD=missense,missense,unknown;AAC=THR/ILE;PP=312/1012,312/1011,312/1013;CDP=935;GS=89;PH=0.003;CP=0.015;CG=-2.190;AA=A;HA=40.2;HE=38.7;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=A:5899/G:7107;PAC=NP_000519.2,NP_001166969.1,XP_005259970.1	GT:GQ:PL	0/0:5:0,3,25	0/1:3:0,0,0	1/1:7:68,12,0	0/0:9:0,9,72

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00139;MAN2B1;hsa04142,hsa00511
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse34">beta-Mannosidosis</a>
      </h4>
    </div>
    <div id="collapse34" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          4	103611845	rs227368	C	T	89.90	lowCVG	AC1=6;AF1=0.7365;DB;DP=8;DP4=1,1,4,2;FQ=12.9;MQ=60;PV4=1,1,1,0.29;SNPEFF_AMINO_ACID_CHANGE=V253I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MANBA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005908.3;VDB=0.0092;DN=79;DA=C/A/G/T;GM=NM_005908.3,XM_005262994.1;GL=MANBA;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=253/880,196/823;CDP=757,586;GS=29;PH=0.037;CP=0.007;CG=1.290;AA=C;HA=10.1;HE=46.2;HC=46.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=84;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=T:5010/C:7996;PAC=NP_005899.3,XP_005263051.1	GT:GQ:PL	0/0:3:0,6,50	0/1:3:0,0,0	1/1:9:70,9,0	1/1:9:63,9,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00140;MANBA;hsa04142,hsa00511
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse36">Lysosomal acid lipase deficiency</a>
      </h4>
    </div>
    <div id="collapse36" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          10	91007360	rs1051338	T	G	24.80	lowCVG	AC1=4;AF1=0.5432;DB;DP=5;DP4=1,1,0,3;FQ=4.88;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T16P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000235.3;VDB=0.0127;DN=86;DA=T/G;GM=NM_000235.2,NM_001127605.1;GL=LIPA;FG=missense;FD=missense;AAC=THR/PRO;PP=16/400;CDP=46;GS=38;PH=0.002;CP=0.997;CG=3.720;AA=T;HA=8.8;HE=33.0;HC=28.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=47;KP=hsa00100:SteroidBiosynthesis|hsa04142:Lysosome;GESP=G:3134/T:9872;PAC=NP_000226.2,NP_001121077.1	GT:GQ:PL	0/1:19:20,0,18	0/1:3:0,0,0	1/1:5:44,6,0	0/0:3:0,3,21

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00148;LIPA;hsa04142,hsa00100
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse37">Hypercholesterolemia</a>
      </h4>
    </div>
    <div id="collapse37" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          4	100504664	rs3816873	T	C	35.40	lowCVG	AC1=6;AF1=0.695;DB;DP=7;DP4=1,1,3,2;FQ=10.9;MQ=60;PV4=1,0.018,1,0.15;SNPEFF_AMINO_ACID_CHANGE=I155T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTTP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300785.1;VDB=0.0031;DN=107;DA=T/C;GM=NM_000253.2,XM_005263025.1;GL=MTTP;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=128/895,155/922;CDP=383,464;GS=89;PH=0.007;CP=0.810;CG=4.100;AA=T;HA=22.5;HE=25.3;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=11;GESP=C:3415/T:9587;PPI=APOB:846;PAC=NP_000244.2,XP_005263082.1	GT:GQ:PL	0/0:4:0,6,47	0/1:3:0,0,0	1/1:11:61,12,0	1/1:4:16,3,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00155;PCSK9|LDLRAP1|APOB|LDLR;hsa04144
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>


  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse38">Phenylketonuria (PKU); Hyperphenylalaninemia, BH4-deficient</a>
      </h4>
    </div>
    <div id="collapse38" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          12	103249087	rs77958223	T	C	77.80	highCVG	AC1=4;AF1=0.4954;DB;DP=15;DP4=5,3,3,4;FQ=80.7;MQ=60;PV4=0.62,0.25,1,0.071;SNPEFF_AMINO_ACID_CHANGE=E178G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000277.1;VDB=0.0177;DN=131;DA=T/A/C;GM=NM_000277.1;GL=PAH;FG=missense;FD=missense;AAC=GLU/GLY;PP=178/453;CDP=533;GS=98;PH=0.961;CP=1.000;CG=5.730;AA=T;DV=by-cluster;CA;DSP=24;KP=hsa00360:PhenylalanineMetabolism|hsa00400:Phenylalanine,TyrosineAndTryptophanBiosynthesis|hsa01100:MetabolicPathways;GESP=T:13006;PAC=NP_000268.1	GT:GQ:PL	0/0:3:0,3,20	0/1:3:16,3,0	1/1:15:109,18,0	0/0:19:0,21,126

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00167;GCH1|PAH|PCBD1|PTS|QDPR;hsa00790,hsa00400
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse39">Oculocutaneous albinism (OCA)</a>
      </h4>
    </div>
    <div id="collapse39" class="panel-collapse collapse">
      <div class="panel-body">

        <p>3 67411166 rs902321 A G 144 highCVG AC1=5;AF1=0.6662;DB;DP=15;DP4=2,3,6,4;FQ=34.2;MQ=60;PV4=0.61,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Y404H;SNPEFF_CODON_CHANGE=Tat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0085;DN=86;DA=A/G;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=TYR/HIS;PP=404/441;CDP=1210;GS=83;CP=0.001;CG=3.220;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=27;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:33:42,0,33 0/0:3:0,6,44 1/1:9:70,9,0 1/1:12:80,12,0</p>

        <p>5 7897191 rs10380 C T 46.90 midCVG AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1 GT:GQ:PL 0/0:11:0,12,87 0/0:8:0,9,73 1/1:6:65,9,0 0/1:4:29,3,0</p>

        <p>5 33951693 rs16891982 C G 118 midCVG AC1=6;AF1=0.667;DB;DP=9;DP4=0,1,4,4;FQ=-7.74;MQ=60;PV4=1,0.4,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L374F;SNPEFF_CODON_CHANGE=ttG/ttC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC45A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016180.3;VDB=0.0064;DN=123;DA=C/G;GM=NM_001012509.2,NM_016180.3,XM_005248311.1;GL=SLC45A2;FG=missense,missense,3-prime-UTR;FD=missense,missense,unknown;AAC=LEU/PHE,LEU/PHE,none;PP=374/461,374/531,NA;CDP=1122,1122,NA;GS=22,22,NA;PH=0.995;CP=1.000;CG=4.320;AA=C;HA=0.0;HE=1.7;HC=0.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=35;TFBS=V$MRF2_01;GESP=G:9063/C:3943;PAC=NP_001012527.1,NP_057264.3,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,1/1:22:156,24,0 0/1:4:0,3,26</p>

        <p>7 64452830 rs34639489 C T 86 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,4,2;FQ=5.73;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C192Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERV3-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007253.3;VDB=0.0141;DN=126;DA=C/T;GM=NM_001007253.3;GL=ERV3-1;FG=missense;FD=missense;AAC=CYS/TYR;PP=192/605;CDP=575;GS=194;PH=0.991;CP=0.325;CG=0.109;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;RM=HERV3-int;DSP=963;GESP=T:5840/C:6396;PAC=NP_001007254.2 GT:GQ:PL 0/0:3:0,6,41 0/1:3:0,0,0 1/1:14:102,15,0 1/1:4:26,3,0</p>

        <p>7 113518434 rs1799999 C A 68 lowCVG AC1=6;AF1=0.7069;DB;DP=8;DP4=1,2,1,4;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D905Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002711.3;VDB=0.0058;DN=89;DA=C/A;GM=NM_002711.3,XM_005250473.1;GL=PPP1R3A;FG=missense;FD=missense,unknown;AAC=ASP/TYR;PP=905/1123,704/922;CDP=2713,2110;GS=160;PH=0.976;CP=0.997;CG=5.640;AA=C;HA=23.4;HE=6.3;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=1747;KP=hsa04910:InsulinSignalingPathway;GESP=A:1760/C:11244;PAC=NP_002702.2,XP_005250530.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:59,6,0 0/1:49:45,0,55</p>

        <p>7 133848257 rs17167553 G T 35.10 highCVG AC1=3;AF1=0.3866;DB;DP=16;DP4=5,6,0,4;FQ=37.2;MQ=60;PV4=0.23,0.31,1,0.17;SNPEFF_AMINO_ACID_CHANGE=D302Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRGUK;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144648.1;VDB=0.0117;DN=123;DA=G/T;GM=NM_144648.1;GL=LRGUK;FG=missense;FD=unknown;AAC=ASP/TYR;PP=302/826;CDP=904;GS=160;PH=0.902;CP=0.983;CG=3.770;AA=G;HA=13.5;HE=7.3;HC=32.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=36;GESP=T:1632/G:11374;PAC=NP_653249.1 GT:GQ:PL 0/0:23:0,24,157 0/0:4:0,3,23 1/1:5:63,9,0 0/1:18:17,0,43</p>

        <p>8 104412688 rs141856398 T C 58.40 highCVG AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA GT:GQ:PL 0/0:4:0,3,24 0/0:6:0,6,40 1/1:3:38,6,0 0/1:64:63,0,86</p>

        <p>8 110457011 rs72687015 A G 85.70 midCVG AC1=5;AF1=0.6401;DB;DP=10;DP4=3,1,4,2;FQ=43.1;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=Y1638C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKHD1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_177531.4;VDB=0.0097;DN=130;DA=A/G;GM=NM_177531.4;GL=PKHD1L1;FG=missense;FD=missense;AAC=TYR/CYS;PP=1638/4244;CDP=4913;GS=194;PH=0.951;CP=1.000;CG=3.250;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=122;GESP=G:2556/A:9592;PAC=NP_803875.2 GT:GQ:PL 0/1:21:16,0,37 0/0:4:0,6,48 1/1:9:70,9,0 1/1:6:46,6,0</p>

        <p>8 142517222 rs2748421 A G 92.90 midCVG AC1=5;AF1=0.6549;DB;DP=13;DP4=3,2,8,0;FQ=41.5;MQ=59;PV4=0.13,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=Y10H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0027;DN=100;DA=A/G;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/HIS,none,none;PP=10/1319,NA,NA;CDP=28,109,109;GS=83,NA,NA;CP=0.000;CG=1.370;AA=G;HA=20.8;HE=49.1;HC=36.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=G:4995/A:7673;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:28:52,0,27 1/1:11:68,12,0 0/1:19:14,0,51</p>

        <p>9 114464487 rs1407390 T C 45.60 lowCVG AC1=6;AF1=0.7452;DB;DP=8;DP4=0,1,0,6;FQ=-7.45;MQ=60;PV4=1,0.0038,1,0.36;SNPEFF_AMINO_ACID_CHANGE=Y932C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C9orf84;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173521.4;VDB=0.0037;DN=88;DA=T/C;GM=NM_001080551.1,NM_173521.3,XM_005251738.1;GL=C9orf84;FG=missense;FD=missense,missense,unknown;AAC=TYR/CYS;PP=893/1406,932/1445,996/1509;CDP=2678,2795,2987;GS=194;PH=0;CP=0.930;CG=-2.510;AA=C;HA=15.6;HE=21.5;HC=7.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=20;GESP=C:10539/T:2451;PAC=NP_001074020.1,NP_775792.3,XP_005251795.1 GT:GQ:PL 0/1:5:0,3,28 0/1:3:0,0,0 1/1:5:18,3,0 1/1:15:68,15,0</p>

        <p>10 70641860 rs1341667 T C 76 midCVG AC1=5;AF1=0.6593;DB;DP=10;DP4=1,2,3,4;FQ=26.9;MQ=60;PV4=1,0.21,1,0.26;SNPEFF_AMINO_ACID_CHANGE=Y153H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130161.2;VDB=0.0014;DN=88;DA=T/C;GM=NM_001130159.2,NM_001130160.2,NM_001130161.2,NM_001130162.2,NM_152709.4,XM_005269605.1;GL=STOX1;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=TYR/HIS;PP=153/228,153/170,153/990,43/880,153/990,43/880;CDP=457,457,457,127,457,127;GS=83;PH=0.687;CP=0.990;CG=4.440;AA=T;HA=33.8;HE=38.1;HC=11.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=7;TFBS=V$HNF1_C;GESP=C:6772/T:5074;PAC=NP_001123631.1,NP_001123632.1,NP_001123633.1,NP_001123634.1,NP_689922.3,XP_005269662.1 GT:GQ:PL 0/0:6:0,9,65 0/1:3:0,0,0 1/1:17:106,18,0 1/1:4:13,3,0</p>

        <p>10 84733562 . C T 20.10 midCVG AC1=3;AF1=0.3373;DP=11;DP4=6,1,3,0;FQ=21.8;MQ=60;PV4=1,0.18,1,0.066;SNPEFF_AMINO_ACID_CHANGE=H435Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010848.3;VDB=0.0000;GM=NM_001010848.3,NM_001165972.1,NM_001165973.1,XM_005269444.1,XM_005269445.1,XM_005269446.1,XM_005269447.1,XM_005269448.1,XM_005269449.1,XM_005269450.1,XM_005269451.1;GL=NRG3;FG=missense;FD=unknown;AAC=HIS/TYR;PP=435/697,434/696,214/500,435/721,265/527,239/525,239/501,214/476,171/457,171/433,85/371;CDP=1303,1300,640,1303,793,715,715,640,511,511,253;GS=83;PH=0.952;CP=1.000;CG=5.050;AA=C;DSP=19;KP=hsa04012:ErbBSignalingPathway;TFBS=V$CETS1P54_01;GESP=C:13006;PAC=NP_001010848.2,NP_001159444.1,NP_001159445.1,XP_005269501.1,XP_005269502.1,XP_005269503.1,XP_005269504.1,XP_005269505.1,XP_005269506.1,XP_005269507.1,XP_005269508.1 GT:GQ:PL 0/0:5:0,3,24 0/0:19:0,18,100 1/1:4:60,9,0 0/0:3:0,0,0</p>

        <p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

        <p>11 51411914 rs61890334 T C 57.90 midCVG AC1=4;AF1=0.4616;DB;DP=12;DP4=4,3,1,4;FQ=60.5;MQ=55;PV4=0.29,1,1,0.074;SNPEFF_AMINO_ACID_CHANGE=Y161C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005272.3;VDB=0.0086;DN=129;DA=T/C;GM=NM_001005272.3;GL=OR4A5;FG=missense;FD=missense;AAC=TYR/CYS;PP=161/316;CDP=482;GS=194;PH=0.027;CP=0.042;CG=1.930;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1032140;KP=hsa04740:OlfactoryTransduction;GESP=C:765/T:12229;PAC=NP_001005272.3 GT:GQ:PL 0/0:13:0,15,95 0/0:4:0,3,25 1/1:6:63,9,0 0/1:4:40,2,0</p>

        <p>11 124135009 rs2512168 G A 85.50 highCVG AC1=3;AF1=0.3452;DB;DP=15;DP4=1,4,4,6;FQ=75.1;MQ=42;PV4=0.6,1,0.082,1;SNPEFF_AMINO_ACID_CHANGE=C96Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR8G5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005198.1;VDB=0.0030;DN=100;DA=G/A;GM=NM_001005198.1;GL=OR8G5;FG=missense;FD=missense;AAC=CYS/TYR;PP=96/347;CDP=287;GS=194;CP=0.533;CG=-5.000;AA=A;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118248;GESP=A:6010/G:6910;PAC=NP_001005198.1 GT:GQ:PL 0/0:5:0,3,25 0/0:7:0,6,48 1/1:24:130,30,0 0/0:7:0,6,45</p>

        <p>15 65676726 rs33918653 C T 78.50 midCVG AC1=4;AF1=0.5302;DB;DP=10;DP4=0,4,5,1;FQ=24.8;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C1125Y;SNPEFF_CODON_CHANGE=tGc/tAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0003;DN=126;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=CYS/TYR;PP=1125/1251;CDP=3374;GS=194;PH=0.981;CP=1.000;CG=4.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=T:2295/C:10705;PAC=NP_066013.1 GT:GQ:PL 0/0:5:0,6,38 0/1:3:0,0,0 1/1:7:70,9,0 0/1:28:50,0,25</p>

        <p>17 76433899 rs61742072 G A 107 midCVG AC1=5;AF1=0.5795;DB;DP=14;DP4=6,0,2,6;FQ=60.9;MQ=60;PV4=0.0097,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=H3953Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=74;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH17;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173628.3;VDB=0.0018;DN=129;DA=G/A;GM=NM_173628.3;GL=DNAH17;FG=missense;FD=missense;AAC=HIS/TYR;PP=3953/4463;CDP=11857;GS=83;PH=0;CP=0.997;CG=2.960;AA=G;DV=by-frequency,by-cluster;DSP=52;GESP=A:1641/G:11359;PAC=NP_775899.3 GT:GQ:PL 0/1:21:20,0,22 0/0:11:0,15,83 1/1:4:26,3,0 1/1:16:109,18,0</p>

        <p>18 11610073 rs75301503 G A 72.40 midCVG AC1=4;AF1=0.5075;DB;DP=14;DP4=2,5,3,3;FQ=75.1;MQ=53;PV4=0.59,1,0.44,0.48;SNPEFF_AMINO_ACID_CHANGE=C160Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=G/A;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=CYS/TYR;PP=160/339;CDP=479;GS=194;CP=0.000;AA=G;DG;DV=by-cluster,by-1000G;DSP=79866;PAC=XP_934053.1 GT:GQ:PL 0/0:10:0,12,79 0/1:3:0,0,1/1:12:100,15,0 0/1:19:16,0,48</p>

        <p>19 9764230 . G A 25.50 lowCVG AC1=2;AF1=0.3027;DP=7;DP4=2,2,0,3;FQ=26.8;MQ=60;PV4=0.43,1,1,0;SNPEFF_AMINO_ACID_CHANGE=H226Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF562;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130031.1;VDB=0.0003;GM=NM_001130031.1,NM_001130032.1,NM_017656.3,XM_005259941.1,XM_005259942.1,XM_005259943.1,XM_005259944.1,XM_005259945.1,XM_005259946.1,XM_005259947.1,XM_005259948.1,XM_005259949.1,XM_005259950.1,XM_005259951.1,XM_005259952.1,XM_005259953.1;GL=ZNF562;FG=missense;FD=unknown;AAC=HIS/TYR;PP=226/427,226/427,154/355,226/427,225/426,218/419,189/390,157/358,157/358,157/358,157/358,148/349,111/312,110/311,83/284,83/284;CDP=676,676,460,676,673,652,565,469,469,469,469,442,331,328,247,247;GS=83;PH=0.965;CP=0.000;CG=-0.749;AA=G;DSP=325;GESP=G:13006;PAC=NP_001123503.1,NP_001123504.1,NP_060126.2,XP_005259998.1,XP_005259999.1,XP_005260000.1,XP_005260001.1,XP_005260002.1,XP_005260003.1,XP_005260004.1,XP_005260005.1,XP_005260006.1,XP_005260007.1,XP_005260008.1,XP_005260009.1,XP_005260010.1 GT:GQ:PL 0/0:8:0,6,34 0/0:6:0,3,28 1/1:4:66,9,0 0/0:6:0,3,19</p>

        <p>19 15198631 rs8105737 A C 43.30 lowCVG AC1=5;AF1=0.6343;DB;DP=7;DP4=3,0,3,1;FQ=15.8;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=Y252S;SNPEFF_CODON_CHANGE=tAt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004713.1;VDB=0.0061;DN=116;DA=A/C;GM=NM_001004713.1;GL=OR1I1;FG=missense;FD=missense;AAC=TYR/SER;PP=252/356;CDP=755;GS=144;PH=0.999;CP=0.743;CG=3.590;AA=A;HA=36.4;HE=33.0;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19702;KP=hsa04740:OlfactoryTransduction;TFBS=V$GR_Q6,V$YY1_02;GESP=C:7526/A:5480;PAC=NP_001004713.1 GT:GQ:PL 0/1:14:17,0,13 0/1:3:0,0,0 1/1:6:49,6,0 0/1:22:17,0,38</p>

        <p>19 23159486 rs289318 T C 119 highCVG AC1=4;AF1=0.5102;DB;DP=20;DP4=6,3,3,7;FQ=106;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Y218C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0021;DN=79;DA=T/C;GM=NM_001267716.1,XM_005259908.1;GL=ZNF728;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=TYR/CYS,none;PP=218/623,NA;CDP=653,NA;GS=194,NA;CP=0.272;CG=0.945;AA=T;HE=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=427;PAC=NP_001254645.1,NA GT:GQ:PL 0/0:9:0,12,78 0/1:20:34,0,17 1/1:18:121,21,0 0/1:15:12,0,56</p>

        <p>19 35435069 rs765746 A G 47.60 midCVG AC1=4;AF1=0.4586;DB;DP=12;DP4=6,1,4,1;FQ=50.2;MQ=60;PV4=1,0.4,1,0.13;SNPEFF_AMINO_ACID_CHANGE=Y401C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF30;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099437.1;VDB=0.0179;DN=86;DA=A/G;GM=NM_001099437.1,NM_001099438.1,NM_194325.2,NR_024018.1;GL=ZNF30;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=TYR/CYS,TYR/CYS,TYR/CYS,none;PP=401/625,401/625,400/624,NA;CDP=1202,1202,1199,1657;GS=194,194,194,NA;PH=1;CP=0.040;CG=1.040;AA=A;HA=22.5;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=943;GESP=G:2295/A:10709;PAC=NP_001092907.1,NP_001092908.1,NP_919306.2,NA GT:GQ:PL 0/0:6:0,6,39 0/0:13:0,15,92 1/1:8:75,12,0 0/1:4:20,3,0</p>

        <p>19 56499279 rs306457 G C 68.10 midCVG AC1=4;AF1=0.4813;DB;DP=12;DP4=0,6,0,6;FQ=46;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=*1049Y;SNPEFF_CODON_CHANGE=taG/taC;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_176811.2;VDB=0.0070;DN=79;DA=G/C;GM=NM_176811.2,XM_005258511.1;GL=NLRP8;FG=stop-lost;FD=stop-lost,unknown;AAC=stop/TYR;PP=1049/1049,1030/1030;CDP=3147,3090;CP=0.000;CG=0.372;AA=C;HA=26.7;HE=30.0;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=100;GESP=C:9537/G:3469;PAC=NP_789781.2,XP_005258568.1 GT:GQ:PL 0/0:7:0,9,50 0/1:17:21,0,15 1/1:4:45,6,0 0/1:27:47,0,23</p>

        <p>X 23693193 . C A 54.30 highCVG AC1=2;AF1=0.2857;DP=20;DP4=6,6,0,7;FQ=55.8;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S149Y;SNPEFF_CODON_CHANGE=tCt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRDX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006406.1;VDB=0.0080;GM=NM_006406.1,XM_005274438.1;GL=PRDX4;FG=missense;FD=unknown;AAC=SER/TYR;PP=149/272,135/258;CDP=446,404;GS=144;PH=1;CP=1.000;CG=5.840;DSP=31;GESP=C:13006;PAC=NP_006397.1,XP_005274495.1 GT:GQ:PL 0/0:16:0,15,96 0/0:6:0,3,25 1/1:14:100,21,0 0/0:19:0,18,118</p>

        <p>
          <strong>IDs and variants associated:</strong> H00168;OCA2|SLC45A2|TYR|TYRP1;hsa04916,hsa00350
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 24
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.24
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse40">Methylmalonic aciduria (MMA)</a>
      </h4>
    </div>
    <div id="collapse40" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	45797505	rs3219489	C	G	19	midCVG	AC1=4;AF1=0.4355;DB;DP=11;DP4=3,5,1,2;FQ=21.3;MQ=60;PV4=1,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=Q338H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUTYH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128425.1;VDB=0.0034;DN=106;DA=C/G;GM=NM_001048171.1,NM_001048172.1,NM_001048173.1,NM_001048174.1,NM_001128425.1,NM_012222.2,XM_005270880.1,XM_005270881.1,XM_005270882.1,XM_005270883.1,XM_005270884.1,XM_005270885.1,XM_005270886.1;GL=MUTYH;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/HIS;PP=324/536,311/523,310/522,310/522,338/550,335/547,325/537,321/533,321/533,282/494,218/430,218/430,184/396;CDP=972,933,930,930,1014,1005,975,963,963,846,654,654,552;GS=24;PH=0.645;CP=1.000;CG=1.260;AA=C;HA=30.5;HE=28.4;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17;KP=hsa03410:BaseExcisionRepair;GESP=G:3304/C:9698;PPI=APEX1:846|PCNA:846;PAC=NP_001041636.1,NP_001041637.1,NP_001041638.1,NP_001041639.1,NP_001121897.1,NP_036354.1,XP_005270937.1,XP_005270938.1,XP_005270939.1,XP_005270940.1,XP_005270941.1,XP_005270942.1,XP_005270943.1	GT:GQ:PL	0/1:13:12,0,51	0/0:14:0,15,79	1/1:4:48,6,0	0/1:3:0,0,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00174;MMAA|MMACHC|MMADHC|MMAB|MUT|LMBRD1|MCEE;hsa00640,hsa00280
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse41">Homocystinuria</a>
      </h4>
    </div>
    <div id="collapse41" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          5	7897191	rs10380	C	T	46.90	midCVG	AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1	GT:GQ:PL	0/0:11:0,12,87	0/0:8:0,9,73	1/1:6:65,9,0	0/1:4:29,3,0
        </p>
        <p>
          11	71169547	rs2276360	G	C	15.50	lowCVG	AC1=5;AF1=0.5737;DB;DP=8;DP4=2,2,0,3;FQ=19.2;MQ=60;PV4=0.43,0.24,1,0.37;SNPEFF_AMINO_ACID_CHANGE=V74L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NADSYN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018161.4;VDB=0.0099;DN=100;DA=G/C;GM=NM_018161.4;GL=NADSYN1;FG=missense;FD=missense;AAC=VAL/LEU;PP=74/707;CDP=220;GS=32;PH=0;CP=0.037;CG=-2.340;AA=C;HA=33.2;HE=27.2;HC=40.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=44;KP=hsa00760:NicotinateAndNicotinamideMetabolism|hsa01100:MetabolicPathways;GESP=C:8267/G:4721;PAC=NP_060631.2	GT:GQ:PL	0/1:3:21,3,0	0/1:3:0,0,0	1/1:5:36,6,0	0/0:10:0,12,81

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00183;MTHFR|NAD|MTRR|CBS;hsa00670,hsa00450,hsa00270,hsa00260
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse42">Citrullinemia (CTLN)</a>
      </h4>
    </div>
    <div id="collapse42" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          7	95822389	.	G	A	54.60	highCVG	AC1=3;AF1=0.3723;DP=16;DP4=4,2,0,7;FQ=56.7;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P192L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001160210.1;VDB=0.0003;GM=NM_001160210.1,NM_014251.2,NR_027662.1;GL=SLC25A13;FG=missense,missense,non-coding-exon;FD=unknown;AAC=PRO/LEU,PRO/LEU,none;PP=192/677,192/676,NA;CDP=575,575,650;GS=98,98,NA;PH=0.243;CP=1.000;CG=4.100;AA=G;DSP=41;GESP=G:13006;PPI=BRE:923|BABAM1:871;PAC=NP_001153682.1,NP_055066.1,NA	GT:GQ:PL	0/0:9:0,9,60	0/0:9:0,9,66	1/1:15:98,21,0	0/1:3:0,0,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00185;SLC25A13|ASS1;hsa00330,hsa00250
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse43">Hyperprolinemia (HP)</a>
      </h4>
    </div>
    <div id="collapse43" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          19	36303664	rs3848666	G	C	17.50	lowCVG	AC1=4;AF1=0.5;DB;DP=8;DP4=2,1,2,1;FQ=20;MQ=60;PV4=1,0.19,1,0.28;SNPEFF_AMINO_ACID_CHANGE=P91R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRODH2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021232.1;VDB=0.0075;DN=108;DA=G/C;GM=NM_021232.1,XM_005259123.1;GL=PRODH2;FG=missense;FD=missense,unknown;AAC=PRO/ARG;PP=91/537,15/379;CDP=272,44;GS=103;PH=0.004;CP=0.000;CG=0.647;AA=A;HA=0.8;HE=14.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=131;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;GESP=C:1572/G:11434;PAC=NP_067055.1,XP_005259180.1	GT:GQ:PL	0/0:9:0,9,65	0/1:3:0,0,0	1/1:5:55,9,0	0/1:3:0,0,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00190;PRODH|ALDH4A1;hsa00330
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse44">Inherited thrombophilia</a>
      </h4>
    </div>
    <div id="collapse44" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	169578854	.	C	G	75.50	highCVG	AC1=3;AF1=0.3761;DP=18;DP4=9,1,3,3;FQ=77.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q407H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SELP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003005.3;VDB=0.0130;GM=NM_003005.3,XM_005245435.1,XM_005245436.1,XM_005245437.1,XM_005245438.1,XM_005245439.1,XM_005245440.1;GL=SELP;FG=missense;FD=unknown;AAC=GLN/HIS;PP=407/831,407/831,407/831,407/830,407/791,407/791,345/769;CDP=1221,1221,1221,1221,1221,1221,1035;GS=24;PH=0.973;CP=0.018;CG=1.770;AA=C;DSP=74;KP=hsa04514:CellAdhesionMolecules(CAMs);GESP=C:13006;PPI=SELPLG:974;PAC=NP_002996.2,XP_005245492.1,XP_005245493.1,XP_005245494.1,XP_005245495.1,XP_005245496.1,XP_005245497.1	GT:GQ:PL	0/0:12:0,12,64	0/0:7:0,6,46	1/1:5:71,9,0	0/1:50:50,0,72

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00223;FGA|FGB|FGG|SERPIND1|HRG|SERPINC1|SERPINE1|PLG|PROC|PROS1|THBD;hsa04610
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>


  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse45">Bartter syndrome</a>
      </h4>
    </div>
    <div id="collapse45" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          17	21318867	rs73979893	G	A	999	highCVG	AC1=5;AF1=0.6076;DB;DP=37;DP4=5,10,16,5;FQ=193;MQ=60;PV4=0.017,0.45,1,0.069;SNPEFF_AMINO_ACID_CHANGE=M71I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNJ12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021012.4;VDB=0.0094;DN=130;DA=G/A;GM=NM_021012.4,XM_005256625.1;GL=KCNJ12;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=71/434;CDP=213;GS=10;PH=0;CP=1.000;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=269;CPG=21318651-21320041;TFBS=V$MEIS1AHOXA9_01,V$OCT1_06;PPI=DLG1:972|CASK:833|LIN7B:833|LIN7A:833|APBA1:833|LIN7C:833|DLG4:833;PAC=NP_066292.2,XP_005256682.1	GT:GQ:PL	0/1:54:49,0,91	0/1:35:62,0,34	1/1:8:120,9,0	0/1:59:54,0,108

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00239;CLCNKA|CLCNKB|KCNJ1|SLC12A1|BSND|CASR;hsa04960
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse46">Thyroid dyshormonogenesis; Dyshormogenetic goiter</a>
      </h4>
    </div>
    <div id="collapse46" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

<p>8 103573006 rs2916569 G A 17.80 midCVG AC1=4;AF1=0.504;DB;DP=10;DP4=2,1,4,0;FQ=4.62;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S216N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024410.3;VDB=0.0051;DN=101;DA=G/A;GM=NM_024410.3;GL=ODF1;FG=missense;FD=missense;AAC=SER/ASN;PP=216/251;CDP=647;GS=46;PH=0;CP=0.053;CG=-7.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;RT=TGCAGCCCC;DSP=327;GESP=G:13006;PAC=NP_077721.2 GT:GQ:PL 0/0:5:0,6,25 0/1:3:0,0,0 1/1:4:34,6,0 0/1:18:24,0,15</p>

<p>9 114359624 rs1053959 C A 122 highCVG AC1=3;AF1=0.3851;DB;DP=23;DP4=6,8,4,5;FQ=124;MQ=59;PV4=1,1,1,0.0013;SNPEFF_AMINO_ACID_CHANGE=A27S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTGR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146108.1;VDB=0.0165;DN=86;DA=C/A;GM=NM_001146108.1,NM_001146109.1,NM_012212.3;GL=PTGR1;FG=missense;FD=missense;AAC=ALA/SER;PP=27/330,27/302,27/330;CDP=79;GS=99;PH=0.019;CP=0.038;CG=-0.431;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=28;GESP=A:7644/C:5362;PAC=NP_001139580.1,NP_001139581.1,NP_036344.2 GT:GQ:PL 0/0:29:0,30,158 0/1:10:91,0,4 1/1:8:78,12,0 0/0:9:0,9,72</p>

<p>12 53682468 . G T 35.30 lowCVG AC1=5;AF1=0.6499;DP=5;DP4=1,0,0,4;FQ=-11.7;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1565S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ESPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012291.4;VDB=0.0084;GM=NM_012291.4;GL=ESPL1;FG=missense;FD=unknown;AAC=ALA/SER;PP=1565/2121;CDP=4693;GS=99;PH=0.01;CP=0.003;CG=2.840;AA=G;DSP=16;KP=hsa04110:CellCycle|hsa04114:OocyteMeiosis;GESP=G:12990;PPI=PTTG1:987|IRF2BPL:987;PAC=NP_036423.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:72,12,0 0/1:3:0,3,21</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>21 46271460 rs2236442 T C 20.70 midCVG AC1=2;AF1=0.3147;DB;DP=9;DP4=4,2,0,3;FQ=22.3;MQ=58;PV4=0.17,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=E84G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTTG1IP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001286822.1;VDB=0.0020;DN=98;DA=T/C;GM=NM_004339.3,XM_005261175.1,XM_005261176.1,XM_005261177.1;GL=PTTG1IP;FG=3-prime-UTR,missense,missense,3-prime-UTR;FD=utr-variant-3-prime,unknown,unknown,unknown;AAC=none,GLU/GLY,GLU/GLY,none;PP=NA,104/163,84/143,NA;CDP=NA,311,251,NA;GS=NA,98,98,NA;PH=0.914;CP=0.000;CG=0.454;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=83;GESP=C:902/T:12104;PAC=NA,XP_005261232.1,XP_005261233.1,NA GT:GQ:PL 0/0:14:0,12,80 0/0:3:0,0,0 1/1:4:61,9,0 0/0:8:0,6,40</p>

        <p>
          <strong>IDs and variants associated:</strong> H00251;FOXI1|IYD|DUOXA2|DUOX2|SLC26A4|SLC5A5|TG|TPO;hsa00350
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 8
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.8
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse47">Hypogonadotropic hypogonadism, including: Kallmann syndrome (KAL); Fertile eunuch syndrome (FEUNS)</a>
      </h4>
    </div>
    <div id="collapse47" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 152277055 rs2065955 C G 55.20 midCVG AC1=4;AF1=0.5684;DB;DP=10;DP4=0,3,5,2;FQ=5.52;MQ=34;PV4=0.17,1,0.004,1;SNPEFF_AMINO_ACID_CHANGE=G3436A;SNPEFF_CODON_CHANGE=gGa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0107;DN=94;DA=C/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLY/ALA;PP=3436/4062;CDP=10307;GS=60;PH=0.876;CP=0.000;CG=-1.140;AA=C;HE=14.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=9460;GESP=G:4311/C:8695;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:8:60,0,5 0/0:5:0,6,36 1/1:7:36,9,0 0/1:3:0,0,0</p>

<p>1 152280900 rs74129452 T G 22.80 midCVG AC1=5;AF1=0.5654;DB;DP=9;DP4=0,4,0,5;FQ=17.7;MQ=41;PV4=1,1,0.043,0.2;SNPEFF_AMINO_ACID_CHANGE=Q2154H;SNPEFF_CODON_CHANGE=caA/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0064;DN=130;DA=T/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLN/HIS;PP=2154/4062;CDP=6462;GS=24;PH=0;CP=0.000;CG=-5.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G,suspect;DSP=5615;GESP=T:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:4:0,1,28 0/1:3:0,0,0 1/1:5:43,6,0 0/1:20:18,0,25</p>

<p>1 152285603 . G A 73.80 midCVG AC1=5;AF1=0.6441;DP=9;DP4=2,1,1,5;FQ=9.97;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R587C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0003;GM=NM_002016.1;GL=FLG;FG=missense;FD=unknown;AAC=ARG/CYS;PP=587/4062;CDP=1759;GS=180;PH=1;CP=0.000;CG=-3.110;AA=G;DSP=912;GESP=G:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:10:19,0,9 0/0:4:0,6,41 1/1:4:26,3,0 1/1:11:74,12,0</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

        <p>
          <strong>IDs and variants associated:</strong> H00255;PROKR2|FGF8|FGFR1|NSMF|GNRH1|GNRHR|KAL1|CHD7|PROK2|TAC3|TACR3|KISS1R;hsa04912,hsa04810,hsa04080
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse48">Spina bifida</a>
      </h4>
    </div>
    <div id="collapse48" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          5	7897191	rs10380	C	T	46.90	midCVG	AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1	GT:GQ:PL	0/0:11:0,12,87	0/0:8:0,9,73	1/1:6:65,9,0	0/1:4:29,3,0
        </p>
        <p>
          11	71169547	rs2276360	G	C	15.50	lowCVG	AC1=5;AF1=0.5737;DB;DP=8;DP4=2,2,0,3;FQ=19.2;MQ=60;PV4=0.43,0.24,1,0.37;SNPEFF_AMINO_ACID_CHANGE=V74L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NADSYN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018161.4;VDB=0.0099;DN=100;DA=G/C;GM=NM_018161.4;GL=NADSYN1;FG=missense;FD=missense;AAC=VAL/LEU;PP=74/707;CDP=220;GS=32;PH=0;CP=0.037;CG=-2.340;AA=C;HA=33.2;HE=27.2;HC=40.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=44;KP=hsa00760:NicotinateAndNicotinamideMetabolism|hsa01100:MetabolicPathways;GESP=C:8267/G:4721;PAC=NP_060631.2	GT:GQ:PL	0/1:3:21,3,0	0/1:3:0,0,0	1/1:5:36,6,0	0/0:10:0,12,81

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00262;MTHFD1|MTHFR|NAD|MTR|MTRR|VANGL1;hsa00670
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse49">Charcot-Marie-Tooth disease (CMT); Hereditary motor and sensory neuropathy; Peroneal muscular atrophy</a>
      </h4>
    </div>
    <div id="collapse49" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          6	152443744	rs2295190	G	T	29	lowCVG	AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:34,6,0	0/1:24:31,0,23

        </p>
        <p>
          19	40901614	rs268671	A	G	70.30	highCVG	AC1=4;AF1=0.4851;DB;DP=16;DP4=3,6,2,4;FQ=73.2;MQ=60;PV4=1,0.33,1,0.16;SNPEFF_AMINO_ACID_CHANGE=V882A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRX;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181882.2;VDB=0.0157;DN=79;DA=A/G;GM=NM_020956.2,NM_181882.2,XM_005259111.1;GL=PRX;FG=3-prime-UTR,missense,missense;FD=utr-variant-3-prime,missense,unknown;AAC=none,VAL/ALA,VAL/ALA;PP=NA,882/1462,856/1436;CDP=NA,2645,2567;GS=NA,64,64;PH=0;CP=0.033;CG=2.700;AA=G;HA=9.5;HE=49.1;HC=20.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1088;GESP=G:7930/A:5076;PAC=NA,NP_870998.2,XP_005259168.1	GT:GQ:PL	0/0:16:0,18,111	0/1:3:0,0,1/1:9:84,12,0	0/1:32:30,0,49

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00264;NDRG1|FGD4|AARS|DNM2|EGR2|KIF1B|GARS|HSPB8|GJB1|KARS|LMNA|MPZ|NEFL|HSPB11|PMP22|PRPS1|PRX|RAB7A|GDAP1L1|SH3TC2|SBF2|MED25|YARS|MTMR2|LITAF|FIG4|MFN2;hsa00970
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse50">Hypertrophic cardiomyopathy (HCM)</a>
      </h4>
    </div>
    <div id="collapse50" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

<p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>9 117846580 rs1061494 T C 25.30 midCVG AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1 GT:GQ:PL 0/1:22:18,0,120 0/1:3:0,0,0 1/1:6:35,6,0 0/1:15:12,0,20</p>

<p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

        <p>
          <strong>IDs and variants associated:</strong> H00292;MYBPC3|MYH6|MYH7|MYL2|MYL3|PRKAG2|ACTC1|TNNC1|TNNI3|TNNT2|TPM1|TTN|VCL|CSRP3;hsa05410
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 6
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.6
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse51">Arrhythmogenic right ventricular cardiomyopathy (ARVC)</a>
      </h4>
    </div>
    <div id="collapse51" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 11015165 rs75130475 A G 149 midCVG AC1=5;AF1=0.6109;DB;DP=14;DP4=0,2,7,4;FQ=-4.32;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I286T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf127;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170754.1;VDB=0.0208;DN=131;DA=A/G;GM=NM_001170754.1,XM_005263428.1,XM_005263429.1,XM_005263430.1;GL=C1orf127;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=286/824,228/766,129/667,147/659;CDP=857,683,386,440;GS=89;PH=0.397;CP=0.000;CG=-6.010;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79;GESP=G:1746/A:11260;PAC=NP_001164225.1,XP_005263485.1,XP_005263486.1,XP_005263487.1 GT:GQ:PL 0/1:4:0,3,24 0/1:4:0,3,14 1/1:4:26,3,0 1/1:28:168,30,0</p>

      <p>1 11838841 rs56001051 A G 59.70 midCVG AC1=4;AF1=0.4991;DB;DP=11;DP4=0,6,3,2;FQ=62.4;MQ=60;PV4=0.061,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=S822G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf167;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010881.1;VDB=0.0016;DN=129;DA=A/G;GM=XM_003118845.3;GL=C1orf167;FG=missense;FD=unknown;AAC=SER/GLY;PP=822/1450;CDP=2464;GS=56;PH=0;CP=0.000;CG=2.600;AA=G;DV=by-frequency,by-cluster;DSP=33;PAC=XP_003118893.3 GT:GQ:PL 0/0:10:0,12,75 0/1:3:0,0,0 1/1:9:83,12,0 0/1:22:20,0,39</p>

      <p>1 12855878 rs1052908 A C 37.70 midCVG AC1=6;AF1=0.6449;DB;DP=11;DP4=3,1,1,4;FQ=-6.46;MQ=20;PV4=0.21,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R386S;SNPEFF_CODON_CHANGE=agA/agC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRAMEF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_023013.4;VDB=0.0080;DN=86;DA=A/C;GM=NM_023013.2,XM_005263490.1;GL=PRAMEF1;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=386/475,141/230;CDP=1158,423;GS=110;PH=0;CP=0.004;CG=0.600;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,suspect;DSP=292;GESP=A:12994;PAC=NP_075389.1,XP_005263547.1 GT:GQ:PL 0/1:2:4,3,0 0/1:3:0,0,0 1/1:3:70,4,0 0/0:3:0,3,27</p>

      <p>1 13183249 rs139566100 C A 11.60 highCVG AC1=4;AF1=0.5476;DB;DP=15;DP4=6,2,2,4;FQ=-3.2;MQ=13;PV4=0.28,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q208H;SNPEFF_CODON_CHANGE=caG/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HNRNPCL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136561.3;VDB=0.0090;DN=134;DA=C/A;GM=NM_001136561.2;GL=HNRNPCP5;FG=missense;FD=missense;AAC=GLN/HIS;PP=208/294;CDP=624;GS=24;PH=0.002;CP=0.004;CG=-0.298;AA=C;DV=by-cluster;DSP=805;GESP=C:4540;PAC=NP_001130033.2 GT:GQ:PL 0/0:4:0,4,20 0/1:18:20,0,16 1/1:4:29,6,0 0/1:4:3,0,1</p>

      <p>1 13802325 rs3013105 T C 173 midCVG AC1=6;AF1=0.7094;DB;DP=13;DP4=2,1,5,5;FQ=5.86;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K292E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRC38;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010847.1;VDB=0.0076;DN=101;DA=T/C;GM=NM_001010847.1;GL=LRRC38;FG=missense;FD=missense;AAC=LYS/GLU;PP=292/295;CDP=874;GS=56;PH=0;CP=0.701;CG=5.250;AA=C;HA=48.9;HE=39.6;HC=39.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=243;PAC=NP_001010847.1 GT:GQ:PL 0/1:30:36,0,31 0/1:5:0,3,14 1/1:10:76,9,0 1/1:15:107,15,0</p>

      <p>1 15909850 rs6429757 C G 23.50 midCVG AC1=3;AF1=0.403;DB;DP=14;DP4=9,2,3,0;FQ=25.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G105R;SNPEFF_CODON_CHANGE=Ggg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AGMAT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024758.4;VDB=0.0052;DN=116;DA=C/G;GM=NM_024758.4;GL=AGMAT;FG=missense;FD=missense;AAC=GLY/ARG;PP=105/353;CDP=313;GS=125;PH=0;CP=0.010;CG=-1.410;AA=G;HA=7.9;HE=49.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;TFBS=V$COMP1_01;GESP=G:8504/C:4502;PAC=NP_079034.3 GT:GQ:PL 0/0:6:0,6,49 0/1:4:22,3,0 1/1:3:47,6,0 0/0:26:0,27,74</p>

      <p>1 17085502 rs2329006 T C 10.30 highCVG AC1=3;AF1=0.3679;DB;DP=20;DP4=4,5,7,3;FQ=11.7;MQ=16;PV4=0.37,7e-06,0.3,0.15;SNPEFF_AMINO_ACID_CHANGE=M397V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MST1L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271733.1;VDB=0.0107;DN=100;DA=T/C;GM=NM_001271733.1;GL=MST1L;FG=missense;FD=missense;AAC=MET/VAL;PP=397/690;CDP=1189;GS=21;PH=0.235;CP=0.000;AA=C;DG;DV=by-cluster,by-hapmap;DSP=29;CPG=17085412-17086974;PAC=NP_001258662.1 GT:GQ:PL 0/1:7:7,0,12 0/0:5:0,3,18 1/1:3:45,7,0 0/0:11:0,10,33</p>

      <p>1 17586134 rs3750300 A G 26.60 midCVG AC1=3;AF1=0.3603;DB;DP=9;DP4=5,0,4,0;FQ=28.5;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I52V;SNPEFF_CODON_CHANGE=Atc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016233.2;VDB=0.0149;DN=107;DA=A/G;GM=NM_016233.2;GL=PADI3;FG=missense;FD=missense;AAC=ILE/VAL;PP=52/665;CDP=154;GS=29;PH=0;CP=0.038;CG=0.158;AA=G;HA=8.2;HE=8.2;HC=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=62;GESP=G:1157/A:11849;PAC=NP_057317.2 GT:GQ:PL 0/0:7:0,6,42 0/0:3:0,0,0 1/1:7:68,12,0 0/0:10:0,9,48</p>

      <p>1 17657616 rs11203367 T C 17.40 lowCVG AC1=5;AF1=0.6653;DB;DP=6;DP4=2,1,2,1;FQ=18;MQ=60;PV4=1,0.46,1,1;SNPEFF_AMINO_ACID_CHANGE=V82A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0077;DN=120;DA=T/C;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=VAL/ALA;PP=82/664;CDP=245;GS=64;PH=0;CP=0.001;CG=4.660;AA=C;HA=46.2;HE=43.0;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=29;GESP=C:7199/T:5807;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:19,3,0 0/1:37:33,0,50</p>

      <p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

      <p>1 18808526 rs2992752 A C 43.10 lowCVG AC1=6;AF1=0.7115;DB;DP=8;DP4=2,0,0,6;FQ=-12;MQ=60;PV4=0.036,0.43,1,0.35;SNPEFF_AMINO_ACID_CHANGE=T351P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHDC7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152375.2;VDB=0.0098;DN=101;DA=A/C;GM=NM_152375.2,XM_005245739.1,XM_005245740.1;GL=KLHDC7A;FG=missense;FD=missense,unknown,unknown;AAC=THR/PRO;PP=351/778,351/759,208/635;CDP=1051,1051,622;GS=38;PH=0;CP=0.000;CG=0.690;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=255;GESP=C:8092/A:4532;PAC=NP_689588.2,XP_005245796.1,XP_005245797.1 GT:GQ:PL 0/1:20:44,0,19 0/1:3:0,0,0 1/1:6:35,6,0 0/1:3:0,0,0</p>

      <p>1 19431096 . G T 9.57 lowCVG AC1=8;AF1=1;DP=7;DP4=0,0,0,2;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=A4237D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=86;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020765.2;GM=NM_020765.2,XM_005245798.1,XM_005245799.1,XM_005245800.1,XM_005245801.1,XM_005245802.1,XM_005245803.1,XM_005245804.1,XM_005245805.1,XM_005245806.1,XM_005245807.1,XM_005245808.1;GL=UBR4;FG=missense;FD=unknown;AAC=ALA/ASP;PP=4237/5184,4277/5245,4276/5244,4275/5243,4266/5234,4277/5224,4277/5222,4249/5217,4242/5210,4231/5199,4277/5197,4277/5184;CDP=12710,12830,12827,12824,12797,12830,12830,12746,12725,12692,12830,12830;GS=126;PH=0.998;CP=1.000;CG=6.070;AA=G;DSP=18;TFBS=V$OCT1_02;GESP=G:13006;PPI=UBC:705;PAC=NP_065816.2,XP_005245855.1,XP_005245856.1,XP_005245857.1,XP_005245858.1,XP_005245859.1,XP_005245860.1,XP_005245861.1,XP_005245862.1,XP_005245863.1,XP_005245864.1,XP_005245865.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:19,3,0 1/1:5:23,3,0</p>

      <p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

      <p>1 24201920 rs2501432 T C 86.40 highCVG AC1=5;AF1=0.5586;DB;DP=20;DP4=8,2,3,5;FQ=69.9;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q63R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001841.2;VDB=0.0031;DN=100;DA=T/C;GM=NM_001841.2,XM_005245736.1,XM_005245737.1,XM_005245738.1;GL=CNR2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLN/ARG;PP=63/361;CDP=188;GS=43;PH=0;CP=0.910;CG=5.120;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=233;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=C:7940/T:5066;PAC=NP_001832.1,XP_005245793.1,XP_005245794.1,XP_005245795.1 GT:GQ:PL 0/1:23:46,0,21 0/0:20:0,24,93 1/1:3:16,3,0 1/1:10:72,12,0</p>

      <p>1 31902279 rs34728687 G A 95.60 highCVG AC1=3;AF1=0.4377;DB;DP=19;DP4=8,2,3,6;FQ=98.1;MQ=59;PV4=0.07,0.45,0.15,0.065;SNPEFF_AMINO_ACID_CHANGE=G312S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERINC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199038.1;VDB=0.0042;DN=126;DA=G/A;GM=NM_001199037.1,NM_001199038.1,NM_001199039.1,NM_018565.3,NM_178865.4,XM_005270822.1;GL=SERINC2;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=GLY/SER;PP=307/460,312/465,248/401,307/460,303/456,248/401;CDP=919,934,742,919,907,742;GS=56;PH=0.003;CP=0.000;CG=-4.810;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=36;GESP=A:1075/G:11931;PAC=NP_001185966.1,NP_001185967.1,NP_001185968.1,NP_061035.2,NP_849196.2,XP_005270879.1 GT:GQ:PL 0/0:4:0,3,16 0/0:11:0,12,92 1/1:14:100,18,0 0/1:44:42,0,63</p>

      <p>1 33161212 rs360041 T C 100 highCVG AC1=4;AF1=0.5158;DB;DP=34;DP4=5,0,28,0;FQ=52.7;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=M163V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030786.2;VDB=0.0000;DN=79;DA=T/C;GM=NM_001161708.1,NM_030786.2;GL=SYNC;FG=missense;FD=unknown;AAC=MET/VAL;PP=163/477,163/483;CDP=487;GS=21;PH=0;CP=0.005;CG=-2.750;AA=T;HA=18.1;HE=4.0;HC=13.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=434;GESP=C:4162/T:404;PAC=NP_001155180.1,NP_110413.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:80:141,84,0 0/0:13:0,15,90</p>

      <p>1 34677925 rs373171181 C G 55.50 midCVG AC1=5;AF1=0.6509;DB;DP=9;DP4=1,2,2,4;FQ=20.1;MQ=60;PV4=1,0.39,1,0.085;SNPEFF_AMINO_ACID_CHANGE=P547A;SNPEFF_CODON_CHANGE=Cct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf94;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001134734.1;VDB=0.0018;DN=138;DA=C/G;GM=NM_001134734.1,NM_032884.3;GL=C1orf94;FG=missense;FD=missense;AAC=PRO/ALA;PP=547/599,357/409;CDP=1639,1069;GS=27;PH=0.987;CP=0.991;CG=4.250;AA=C;DSP=83;GESP=C:13006;PAC=NP_001128206.1,NP_116273.2 GT:GQ:PL 0/1:15:10,0,42 0/1:3:0,0,0 1/1:11:78,12,0 0/1:11:8,0,14</p>

      <p>1 36550658 rs12043423 C T 16.90 lowCVG AC1=4;AF1=0.4561;DB;DP=8;DP4=2,3,2,1;FQ=19.5;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=R46C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TEKT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014466.2;VDB=0.0094;DN=120;DA=C/T;GM=NM_014466.2,XM_005270753.1;GL=TEKT2;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=46/431;CDP=136;GS=180;PH=1;CP=1.000;CG=3.680;AA=C;HA=6.1;HE=0.0;HC=19.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=21;GESP=T:273/C:12733;PAC=NP_055281.2,XP_005270810.1 GT:GQ:PL 0/1:3:0,0,0 0/0:11:0,12,80 1/1:4:37,6,0 0/1:16:21,0,13</p>

      <p>1 45797505 rs3219489 C G 19 midCVG AC1=4;AF1=0.4355;DB;DP=11;DP4=3,5,1,2;FQ=21.3;MQ=60;PV4=1,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=Q338H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUTYH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128425.1;VDB=0.0034;DN=106;DA=C/G;GM=NM_001048171.1,NM_001048172.1,NM_001048173.1,NM_001048174.1,NM_001128425.1,NM_012222.2,XM_005270880.1,XM_005270881.1,XM_005270882.1,XM_005270883.1,XM_005270884.1,XM_005270885.1,XM_005270886.1;GL=MUTYH;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/HIS;PP=324/536,311/523,310/522,310/522,338/550,335/547,325/537,321/533,321/533,282/494,218/430,218/430,184/396;CDP=972,933,930,930,1014,1005,975,963,963,846,654,654,552;GS=24;PH=0.645;CP=1.000;CG=1.260;AA=C;HA=30.5;HE=28.4;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17;KP=hsa03410:BaseExcisionRepair;GESP=G:3304/C:9698;PPI=APEX1:846|PCNA:846;PAC=NP_001041636.1,NP_001041637.1,NP_001041638.1,NP_001041639.1,NP_001121897.1,NP_036354.1,XP_005270937.1,XP_005270938.1,XP_005270939.1,XP_005270940.1,XP_005270941.1,XP_005270942.1,XP_005270943.1 GT:GQ:PL 0/1:13:12,0,51 0/0:14:0,15,79 1/1:4:48,6,0 0/1:3:0,0,0</p>

      <p>1 46827456 rs9865 A G 20.90 highCVG AC1=2;AF1=0.2753;DB;DP=15;DP4=7,2,2,2;FQ=22.2;MQ=60;PV4=0.53,0.075,1,1;SNPEFF_AMINO_ACID_CHANGE=I365V;SNPEFF_CODON_CHANGE=Ata/Gta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NSUN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199044.3;VDB=0.0005;DN=52;DA=A/G;GM=NM_001256127.1,NM_001256128.1,NM_199044.3,NR_045789.1,NR_045790.1,NR_045791.1;GL=NSUN4;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant;AAC=ILE/VAL,ILE/VAL,ILE/VAL,none,none,none;PP=316/336,316/336,365/385,NA,NA,NA;CDP=946,946,1093,1918,1588,1406;GS=29,29,29,NA,NA,NA;PH=0.001;CP=1.000;CG=3.010;AA=A;HA=2.3;HE=12.5;HC=29.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=215;TFBS=V$IK3_01;GESP=G:1441/A:11565;PPI=MTERFD2:900;PAC=NP_001243056.1,NP_001243057.1,NP_950245.2,NA,NA,NA GT:GQ:PL 0/0:9:0,6,44 0/0:6:0,3,17 1/1:6:64,12,0 0/0:20:0,18,80</p>

      <p>1 47533340 . G T 92.90 highCVG AC1=4;AF1=0.5112;DP=16;DP4=5,1,6,2;FQ=47.1;MQ=39;PV4=1,1,1,0.081;VDB=0.0040;GM=NM_178134.2;GL=CYP4Z1;FG=splice-donor;FD=unknown;CP=0.077;CG=3.100;AA=G;DSP=0;GESP=G:13006;PAC=NP_835235.1 GT:GQ:PL 0/1:13:37,0,10 0/0:10:0,12,76 1/1:5:47,6,0 0/1:6:53,0,2</p>

      <p>1 53676448 rs1799821 G A 24.80 lowCVG AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,22 1/1:5:45,6,0 0/1:20:19,0,22</p>

      <p>1 54661150 rs946448 C T 61.80 highCVG AC1=4;AF1=0.5145;DB;DP=17;DP4=6,2,5,1;FQ=64.8;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=R47K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYB5RL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001031672.2;VDB=0.0105;DN=86;DA=C/T;GM=NM_001031672.2,XM_005271096.1;GL=CYB5RL;FG=missense;FD=missense,unknown;AAC=ARG/LYS;PP=47/316,47/237;CDP=140;GS=26;PH=0.159;CP=1.000;CG=4.240;AA=C;HA=17.4;HE=37.5;HC=24.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=59;GESP=T:3741/C:8713;PAC=NP_001026842.2,XP_005271153.1 GT:GQ:PL 0/0:3:0,3,18 0/0:18:0,21,113 1/1:9:69,12,0 1/1:5:41,6,0</p>

      <p>1 55638075 rs1165222 G A 13.90 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,0,2;FQ=-26.8;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T226I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP24;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015306.2;VDB=0.0081;DN=87;DA=G/A;GM=NM_015306.2,XM_005270689.1;GL=USP24;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=226/2621,51/2444;CDP=677,152;GS=89;PH=0;CP=0.988;CG=2.190;AA=A;HA=12.5;HE=16.1;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=26;TFBS=V$BRACH_01,V$CDPCR3_01;GESP=A:10080/G:1770;PPI=UBC:709;PAC=NP_056121.2,XP_005270746.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:3:0,0,0</p>

      <p>1 63635821 rs7519254 T A 56 midCVG AC1=5;AF1=0.571;DB;DP=11;DP4=5,1,4,1;FQ=55;MQ=60;PV4=1,1,1,1;VDB=0.0082;DN=116;DA=T/A;GM=NR_038252.1;GL=LINC00466;FG=splice-donor;FD=splice-donor-variant;CP=0.003;CG=0.206;AA=T;HA=8.5;HE=23.6;HC=13.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=L1ME4a;DSP=0 GT:GQ:PL 0/0:12:0,15,89 0/1:12:15,0,11 1/1:5:41,6,0 1/1:5:47,6,0</p>

      <p>1 89847372 rs4658359 G T 65 highCVG AC1=5;AF1=0.5773;DB;DP=17;DP4=8,2,3,3;FQ=68.7;MQ=60;PV4=0.3,0.47,0.1,1;SNPEFF_AMINO_ACID_CHANGE=A331S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GBP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198460.2;VDB=0.0087;DN=111;DA=G/T;GM=NM_198460.2,XM_005270548.1,XM_005270549.1;GL=GBP6;FG=missense;FD=missense,unknown,unknown;AAC=ALA/SER;PP=331/634,331/634,201/504;CDP=991,991,601;GS=99;PH=0.835;CP=0.198;CG=-0.023;AA=G;HA=9.3;HE=27.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=120;GESP=T:10289/G:2717;PAC=NP_940862.2,XP_005270605.1,XP_005270606.1 GT:GQ:PL 0/0:9:0,12,87 0/1:9:5,0,75 1/1:5:37,6,0 1/1:8:70,9,0</p>

      <p>1 89849742 rs959460 A T 34.50 lowCVG AC1=6;AF1=0.6772;DB;DP=6;DP4=0,1,4,0;FQ=-19.2;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D520V;SNPEFF_CODON_CHANGE=gAt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GBP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198460.2;VDB=0.0024;DN=86;DA=A/T;GM=NM_198460.2,XM_005270548.1,XM_005270549.1;GL=GBP6;FG=missense;FD=missense,unknown,unknown;AAC=ASP/VAL;PP=520/634,520/634,390/504;CDP=1559,1559,1169;GS=152;PH=0.011;CP=0.001;CG=3.740;AA=A;HA=9.0;HE=13.0;HC=25.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=91;GESP=T:11671/A:1335;PAC=NP_940862.2,XP_005270605.1,XP_005270606.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,13 1/1:10:71,12,0 0/1:3:0,0,0</p>

      <p>1 92445257 rs3088232 C G 193 highCVG AC1=6;AF1=0.6924;DB;DP=15;DP4=2,0,6,7;FQ=3.84;MQ=60;PV4=0.47,0.12,1,0.087;SNPEFF_AMINO_ACID_CHANGE=N414K;SNPEFF_CODON_CHANGE=aaC/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BRDT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242806.1;VDB=0.0047;DN=102;DA=C/G;GM=NM_001242805.1,NM_001242806.1,NM_001242807.1,NM_001242808.1,NM_001242810.1,NM_001726.3,NM_207189.2,XM_005271151.1,XM_005271152.1;GL=BRDT;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,unknown,unknown;AAC=ASN/LYS;PP=410/948,414/952,364/902,364/902,337/875,410/948,410/948,410/948,410/948;CDP=1230,1242,1092,1092,1011,1230,1230,1230,1230;GS=94;PH=0.013;CP=0.889;CG=3.760;AA=C;HA=1.7;HE=24.6;HC=3.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=58;GESP=G:2397/C:10609;PAC=NP_001229734.1,NP_001229735.1,NP_001229736.1,NP_001229737.1,NP_001229739.1,NP_001717.2,NP_997072.1,XP_005271208.1,XP_005271209.1 GT:GQ:PL 0/1:5:0,3,26 0/1:16:55,0,16 1/1:7:47,6,0 1/1:24:137,24,0</p>

      <p>1 93672723 rs2815413 A C 107 highCVG AC1=6;AF1=0.7028;DB;DP=15;DP4=0,6,5,3;FQ=45.9;MQ=60;PV4=0.031,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=N326T;SNPEFF_CODON_CHANGE=aAt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206886.4;VDB=0.0073;DN=100;DA=A/C;GM=NM_206886.3,XM_005270815.1,XM_005270816.1;GL=CCDC18;FG=missense;FD=missense,unknown,unknown;AAC=ASN/THR;PP=326/1300,444/1574,444/1418;CDP=977,1331,1331;GS=65;PH=0.649;CP=1.000;CG=4.580;AA=A;HA=25.6;HE=15.0;HC=3.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=60;GESP=C:8247/A:3559;PAC=NP_996769.3,XP_005270872.1,XP_005270873.1 GT:GQ:PL 0/1:18:44,0,18 0/1:38:32,0,66 1/1:10:53,9,0 1/1:5:21,3,0</p>

      <p>1 109268441 rs11582005 G A 9.32 midCVG AC1=4;AF1=0.5274;DB;DP=14;DP4=8,1,4,1;FQ=12.2;MQ=60;PV4=1,1.1e-05,1,1;SNPEFF_AMINO_ACID_CHANGE=S309N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FNDC7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144937.1;VDB=0.0066;DN=120;DA=G/A;GM=NM_001144937.1,XM_005270550.1;GL=FNDC7;FG=missense;FD=missense,unknown;AAC=SER/ASN;PP=309/734,76/491;CDP=926,227;GS=46;PH=0;CP=0.001;CG=-8.020;AA=A;HA=28.9;HE=10.5;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=70;GESP=A:2609/G:10397;PAC=NP_001138409.1,XP_005270607.1 GT:GQ:PL 0/1:3:0,1,75 0/1:12:10,0,37 1/1:4:32,6,0 0/1:10:8,0,51</p>

      <p>1 110279701 rs7483 C T 240 highCVG AC1=6;AF1=0.6918;DB;DP=21;DP4=3,0,11,7;FQ=-9.17;MQ=60;PV4=0.52,0.11,0.35,0.17;SNPEFF_AMINO_ACID_CHANGE=V224I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000849.4;VDB=0.0111;DN=52;DA=C/T;GM=NM_000849.4,NR_024537.1;GL=GSTM3;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=VAL/ILE,none;PP=224/226,NA;CDP=670,904;GS=29,NA;PH=0;CP=0.001;CG=-5.250;AA=C;HA=3.5;HE=30.8;HC=26.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=T:3043/C:9963;PPI=GSTM2:959;PAC=NP_000840.2,NA GT:GQ:PL 0/1:19:93,0,19 0/1:3:0,0,0 1/1:18:98,18,0 0/1:9:90,0,8</p>

      <p>1 112018657 rs6703267 G A 20.50 highCVG AC1=2;AF1=0.3037;DB;DP=32;DP4=2,21,0,5;FQ=22;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G3S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf162;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174896.3;VDB=0.0100;DN=116;DA=G/A;GM=NM_174896.2,XM_005270475.1,XM_005270476.1;GL=C1orf162;FG=missense,missense,synonymous;FD=missense,unknown,unknown;AAC=GLY/SER,GLY/SER,GLU;PP=3/156,3/131,5/127;CDP=7,7,15;GS=56,56,NA;PH=0.408;CP=0.008;CG=-1.890;AA=G;HA=22.1;HE=27.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=18;GESP=A:3183/G:9823;PAC=NP_777556.1,XP_005270532.1,XP_005270533.1 GT:GQ:PL 0/0:3:0,0,0 0/0:22:0,20,88 1/1:4:61,9,0 0/0:8:0,6,45</p>

      <p>1 112308953 rs197412 T C 58.10 midCVG AC1=5;AF1=0.6066;DB;DP=10;DP4=1,1,7,0;FQ=8.93;MQ=60;PV4=0.22,1,1,0.045;SNPEFF_AMINO_ACID_CHANGE=I636T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DDX20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007204.4;VDB=0.0159;DN=79;DA=T/C;GM=NM_007204.4;GL=DDX20;FG=missense;FD=missense;AAC=ILE/THR;PP=636/825;CDP=1907;GS=89;PH=0;CP=0.331;CG=3.860;AA=C;HA=19.7;HE=43.3;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=595;GESP=C:6295/T:6711;PPI=SMN1:984|SMN2:983|GEMIN4:973|GEMIN2:933|AGO2:885|GEMIN5:836|PRRC2A:786|HDAC5:716;PAC=NP_009135.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:97,21,0 0/0:5:0,6,43</p>

      <p>1 115829313 rs6330 G A 9.39 midCVG AC1=4;AF1=0.4545;DB;DP=10;DP4=5,0,2,1;FQ=11;MQ=60;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=A35V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NGF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002506.2;VDB=0.0110;DN=52;DA=G/A;GM=NM_002506.2;GL=NGF;FG=missense;FD=missense;AAC=ALA/VAL;PP=35/242;CDP=104;GS=64;PH=0.002;CP=0.977;CG=4.450;AA=G;HA=17.7;HE=44.1;HC=18.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=116;KP=hsa04010:MAPKSignalingPathway|hsa04210:Apoptosis|hsa04722:NeurotrophinSignalingPathway;TFBS=V$AP2REP_01;GESP=A:4801/G:8205;PPI=NTRK1:998|NGFR:937|NTF4:937|SORT1:845;PAC=NP_002497.2 GT:GQ:PL 0/0:3:0,0,0 0/0:15:0,15,54 1/1:5:47,9,0 0/0:3:0,0,0</p>

      <p>1 117311147 rs699738 C A 39 midCVG AC1=3;AF1=0.4223;DB;DP=13;DP4=1,6,5,1;FQ=41.3;MQ=60;PV4=0.029,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=H266Q;SNPEFF_CODON_CHANGE=caC/caA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001767.3;VDB=0.0019;DN=86;DA=C/A;GM=NM_001767.3,XM_005271319.1,XM_005271320.1;GL=CD2;FG=missense;FD=missense,unknown,unknown;AAC=HIS/GLN;PP=266/352,292/378,225/311;CDP=798,876,675;GS=24;PH=0.085;CP=0.000;CG=-2.990;AA=C;HA=30.3;HE=9.4;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=62;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04640:HematopoieticCellLineage;GESP=A:2986/C:10020;PPI=CD58:996|CD2BP2:995|CD2AP:974|SH3KBP1:937|YES1:937|LCK:846|PTPRC:846;PAC=NP_001758.2,XP_005271376.1,XP_005271377.1 GT:GQ:PL 0/0:4:0,3,22 0/1:14:14,0,14 1/1:11:71,15,0 0/0:14:0,15,77</p>

      <p>1 152277055 rs2065955 C G 55.20 midCVG AC1=4;AF1=0.5684;DB;DP=10;DP4=0,3,5,2;FQ=5.52;MQ=34;PV4=0.17,1,0.004,1;SNPEFF_AMINO_ACID_CHANGE=G3436A;SNPEFF_CODON_CHANGE=gGa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0107;DN=94;DA=C/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLY/ALA;PP=3436/4062;CDP=10307;GS=60;PH=0.876;CP=0.000;CG=-1.140;AA=C;HE=14.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=9460;GESP=G:4311/C:8695;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:8:60,0,5 0/0:5:0,6,36 1/1:7:36,9,0 0/1:3:0,0,0</p>

      <p>1 152280900 rs74129452 T G 22.80 midCVG AC1=5;AF1=0.5654;DB;DP=9;DP4=0,4,0,5;FQ=17.7;MQ=41;PV4=1,1,0.043,0.2;SNPEFF_AMINO_ACID_CHANGE=Q2154H;SNPEFF_CODON_CHANGE=caA/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0064;DN=130;DA=T/G;GM=NM_002016.1;GL=FLG;FG=missense;FD=missense;AAC=GLN/HIS;PP=2154/4062;CDP=6462;GS=24;PH=0;CP=0.000;CG=-5.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G,suspect;DSP=5615;GESP=T:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:4:0,1,28 0/1:3:0,0,0 1/1:5:43,6,0 0/1:20:18,0,25</p>

      <p>1 152285603 . G A 73.80 midCVG AC1=5;AF1=0.6441;DP=9;DP4=2,1,1,5;FQ=9.97;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R587C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0003;GM=NM_002016.1;GL=FLG;FG=missense;FD=unknown;AAC=ARG/CYS;PP=587/4062;CDP=1759;GS=180;PH=1;CP=0.000;CG=-3.110;AA=G;DSP=912;GESP=G:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:10:19,0,9 0/0:4:0,6,41 1/1:4:26,3,0 1/1:11:74,12,0</p>

      <p>1 152770521 rs41268492 G A 13.80 lowCVG AC1=6;AF1=0.699;DB;DP=5;DP4=1,0,3,0;FQ=-21.4;MQ=47;PV4=1,1,0.48,1;SNPEFF_AMINO_ACID_CHANGE=R84H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LCE1D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178352.2;VDB=0.0045;DN=127;DA=G/A;GM=NM_178352.2;GL=LCE1D;FG=missense;FD=missense;AAC=ARG/HIS;PP=84/115;CDP=251;GS=29;PH=0;CP=0.000;CG=-7.330;AA=A;DV=by-frequency,by-cluster;DSP=273;GESP=A:2436/G:9006;PAC=NP_848129.1 GT:GQ:PL 0/1:3:0,0,0/1:4:0,1,8 1/1:4:47,6,0 0/1:3:0,0,0</p>

      <p>1 156642841 . G T 17.30 lowCVG AC1=3;AF1=0.3522;DP=7;DP4=4,0,3,0;FQ=17.7;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=A380D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NES;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006617.1;VDB=0.0007;GM=NM_006617.1;GL=NES;FG=missense;FD=unknown;AAC=ALA/ASP;PP=380/1622;CDP=1139;GS=126;PH=0.94;CP=0.515;CG=3.090;AA=G;DSP=157;TFBS=V$HOX13_01;GESP=G:13006;PPI=CDK5:937|CDK5R1:845;PAC=NP_006608.1 GT:GQ:PL 0/0:8:0,6,32 0/0:8:0,6,29 1/1:4:57,9,0 0/0:3:0,0,0</p>

      <p>1 156883546 rs12137505 A G 17.30 lowCVG AC1=4;AF1=0.4665;DB;DP=8;DP4=3,1,2,1;FQ=20;MQ=60;PV4=1,0.24,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N903D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PEAR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080471.1;VDB=0.0107;DN=120;DA=A/G;GM=NM_001080471.1,XM_005245141.1,XM_005245142.1;GL=PEAR1;FG=missense;FD=missense,unknown,unknown;AAC=ASN/ASP;PP=903/1038,903/1038,839/974;CDP=2707,2707,2515;GS=23;PH=0.094;CP=0.000;CG=-3.910;AA=A;HA=20.6;HE=35.7;HC=48.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=7;GESP=G:4616/A:8390;PAC=NP_001073940.1,XP_005245198.1,XP_005245199.1 GT:GQ:PL 0/0:4:0,3,27 0/0:8:0,9,64 1/1:4:44,6,0 0/1:4:17,3,0</p>

      <p>1 169578854 . C G 75.50 highCVG AC1=3;AF1=0.3761;DP=18;DP4=9,1,3,3;FQ=77.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q407H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SELP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003005.3;VDB=0.0130;GM=NM_003005.3,XM_005245435.1,XM_005245436.1,XM_005245437.1,XM_005245438.1,XM_005245439.1,XM_005245440.1;GL=SELP;FG=missense;FD=unknown;AAC=GLN/HIS;PP=407/831,407/831,407/831,407/830,407/791,407/791,345/769;CDP=1221,1221,1221,1221,1221,1221,1035;GS=24;PH=0.973;CP=0.018;CG=1.770;AA=C;DSP=74;KP=hsa04514:CellAdhesionMolecules(CAMs);GESP=C:13006;PPI=SELPLG:974;PAC=NP_002996.2,XP_005245492.1,XP_005245493.1,XP_005245494.1,XP_005245495.1,XP_005245496.1,XP_005245497.1 GT:GQ:PL 0/0:12:0,12,64 0/0:7:0,6,46 1/1:5:71,9,0 0/1:50:50,0,72</p>

      <p>1 169673838 rs2229569 G A 156 highCVG AC1=3;AF1=0.4287;DB;DP=22;DP4=5,2,3,9;FQ=101;MQ=60;PV4=0.074,1,1,0.33;SNPEFF_AMINO_ACID_CHANGE=P226S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SELL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000655.4;VDB=0.0143;DN=98;DA=G/A;GM=NM_000655.4,NR_029467.1;GL=SELL;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=PRO/SER,none;PP=226/386,NA;CDP=676,605;GS=74,NA;PH=1;CP=0.872;CG=5.310;AA=G;HA=26.2;HE=10.5;HC=27.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=57;KP=hsa04514:CellAdhesionMolecules(CAMs);GESP=A:1885/G:9953;PAC=NP_000646.2,NA GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,46 1/1:26:172,30,0 0/1:32:31,0,37</p>

      <p>1 169823718 rs4656197 T C 47.80 highCVG AC1=4;AF1=0.5453;DB;DP=15;DP4=3,3,0,7;FQ=39.1;MQ=60;PV4=0.07,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=Q621R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCYL3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181093.3;VDB=0.0088;DN=111;DA=T/A/C/G;GM=NM_020423.6,NM_181093.3;GL=SCYL3;FG=missense;FD=missense;AAC=GLN/ARG;PP=567/689,621/743;CDP=1700,1862;GS=43;PH=0.589;CP=0.000;CG=-1.020;AA=T;HA=27.3;HE=43.9;HC=34.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=308;GESP=C:8150/T:4856;PAC=NP_065156.5,NP_851607.2 GT:GQ:PL 0/1:5:52,0,0 0/1:3:0,0,0 1/1:7:37,9,0 0/0:12:0,15,77</p>

      <p>1 179504043 rs6658180 C G 13.70 lowCVG AC1=3;AF1=0.3645;DB;DP=8;DP4=1,3,1,2;FQ=15.5;MQ=59;PV4=1,0.026,0.14,0.14;SNPEFF_AMINO_ACID_CHANGE=Q993E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AXDND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144696.5;VDB=0.0075;DN=116;DA=C/G;GM=NM_144696.5,NR_073544.1;GL=AXDND1;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=GLN/GLU,none;PP=993/1013,NA;CDP=2977,3097;GS=29,NA;PH=0;CP=0.000;CG=-4.940;AA=C;HA=6.0;HE=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=GA-rich;DSP=55;GESP=G:4250/C:8608;PAC=NP_653297.3,NA GT:GQ:PL 0/0:5:0,3,27 0/0:3:0,0,0 1/1:4:53,9,0 0/0:10:0,9,69</p>

      <p>1 180905448 rs3795504 G T 53.90 highCVG AC1=3;AF1=0.3552;DB;DP=17;DP4=8,3,5,1;FQ=55.7;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=L801F;SNPEFF_CODON_CHANGE=ttG/ttT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1614;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020950.1;VDB=0.0071;DN=107;DA=G/T;GM=NM_020950.1,XM_005245376.1;GL=KIAA1614;FG=missense;FD=missense,unknown;AAC=LEU/PHE;PP=801/1191,801/1222;CDP=2403;GS=22;PH=0;CP=0.002;CG=2.380;AA=T;HA=32.9;HE=45.0;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=359;GESP=T:5139/G:7173;PAC=NP_066001.1,XP_005245433.1 GT:GQ:PL 0/0:9:0,9,70 0/0:15:0,15,73 1/1:5:49,9,0 0/1:46:51,0,42</p>

      <p>1 182025558 . T C 23 midCVG AC1=2;AF1=0.2266;DP=12;DP4=0,9,0,3;FQ=24;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R530G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF648;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001009992.1;VDB=0.0015;GM=NM_001009992.1;GL=ZNF648;FG=missense;FD=unknown;AAC=ARG/GLY;PP=530/569;CDP=1588;GS=125;PH=0.797;CP=0.999;CG=1.640;AA=T;DSP=1651;GESP=T:13006;PAC=NP_001009992.1 GT:GQ:PL 0/0:9:0,6,35 0/0:9:0,6,44 1/1:3:65,9,0 0/0:18:0,15,92</p>

      <p>1 200880702 rs61740234 C T 13.70 lowCVG AC1=2;AF1=0.3055;DB;DP=8;DP4=0,4,3,0;FQ=13.5;MQ=59;PV4=0.029,1,0.14,1.7e-08;SNPEFF_AMINO_ACID_CHANGE=P460S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf106;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018265.3;VDB=0.0002;DN=129;DA=C/T;GM=NM_001142569.2,NM_018265.3;GL=C1orf106;FG=missense;FD=missense;AAC=PRO/SER;PP=361/579,460/678;CDP=1081,1378;GS=74;PH=0.689;CP=0.908;CG=3.690;AA=C;DV=by-frequency,by-cluster;DSP=122;CPG=200880594-200881270;GESP=T:477/C:12529;PAC=NP_001136041.1,NP_060735.3 GT:GQ:PL 0/0:6:0,3,15 0/0:6:0,3,15 1/1:4:54,9,0 0/0:8:0,6,27</p>

      <p>1 201176108 rs12094334 G A 111 midCVG AC1=6;AF1=0.7369;DB;DP=14;DP4=0,2,6,2;FQ=12.9;MQ=60;PV4=0.13,0.11,1,0.24;SNPEFF_AMINO_ACID_CHANGE=G696D;SNPEFF_CODON_CHANGE=gGt/gAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGFN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001164586.1;VDB=0.0191;DN=120;DA=G/A;GM=NM_001164586.1,XM_005245580.1,XM_005245581.1;GL=IGFN1;FG=missense;FD=missense,unknown,unknown;AAC=GLY/ASP;PP=696/3709,696/3684,696/3424;CDP=2087;GS=94;CP=0.000;CG=0.077;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=898;GESP=A:1359/G:3207;PAC=NP_001158058.1,XP_005245637.1,XP_005245638.1 GT:GQ:PL 0/0:3:0,6,50 0/1:3:0,0,0 1/1:7:50,6,0 1/1:18:104,18,0</p>

      <p>1 205801872 rs1361754 A G 26.70 lowCVG AC1=5;AF1=0.6576;DB;DP=6;DP4=2,0,0,3;FQ=-9.17;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I380T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PM20D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152491.4;VDB=0.0040;DN=88;DA=A/G;GM=NM_152491.4;GL=PM20D1;FG=missense;FD=missense;AAC=ILE/THR;PP=380/503;CDP=1139;GS=89;PH=0.016;CP=0.270;CG=-0.108;AA=A;HA=33.9;HE=42.0;HC=0.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=23;GESP=G:6267/A:6739;PAC=NP_689704.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:41,6,0 1/1:4:26,3,0</p>

      <p>1 206225011 rs33990840 G C 30 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=2,0,1,2;FQ=11.7;MQ=60;PV4=0.4,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G191R;SNPEFF_CODON_CHANGE=Ggc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVPR1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000707.3;VDB=0.0112;DN=126;DA=G/C;GM=NM_000707.3;GL=AVPR1B;FG=missense;FD=missense;AAC=GLY/ARG;PP=191/425;CDP=571;GS=125;PH=0;CP=1.000;CG=4.980;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=370;KP=hsa04020:CalciumSignalingPathway|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04270:VascularSmoothMuscleContraction;GESP=C:802/G:12188;PAC=NP_000698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:67,9,0 0/0:6:0,6,45</p>

      <p>1 207790088 rs3811381 C G 67.70 midCVG AC1=5;AF1=0.5957;DB;DP=9;DP4=0,4,1,4;FQ=29.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2277R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=41;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000651.4;VDB=0.0076;DN=107;DA=C/G;GM=NM_000573.3,NM_000651.4,XM_005273064.1;GL=CR1;FG=missense;FD=missense,missense,unknown;AAC=PRO/ARG;PP=1827/2040,2277/2490,2149/2362;CDP=5480,6830,6446;GS=103;PH=0.643;CP=0.000;CG=-5.580;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;KP=hsa04610:ComplementAndCoagulationCascades|hsa04640:HematopoieticCellLineage|hsa05140:Leishmaniasis;GESP=G:2006/C:10176;PPI=CR2:846;PAC=NP_000564.2,NP_000642.3,XP_005273121.1 GT:GQ:PL 0/1:35:36,0,34 0/0:4:0,6,36 1/1:6:51,6,0 1/1:4:25,3,0</p>

      <p>1 214830617 rs438034 A G 57 lowCVG AC1=5;AF1=0.6559;DB;DP=8;DP4=2,0,2,4;FQ=7.46;MQ=60;PV4=0.43,0.015,1,1;SNPEFF_AMINO_ACID_CHANGE=R2943G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0106;DN=80;DA=A/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ARG/GLY;PP=2943/3115,2921/3093;CDP=8827,8761;GS=125;PH=0;CP=0.000;CG=2.520;AA=A;HA=36.6;HE=47.7;HC=9.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=136;GESP=G:6399/A:6607;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1 GT:GQ:PL 0/1:18:16,0,20 0/1:18:14,0,23 1/1:11:68,12,0 0/1:3:0,0,0</p>

      <p>1 214837110 rs7289 C G 17.40 lowCVG AC1=6;AF1=0.7293;DB;DP=5;DP4=0,1,0,4;FQ=-8.8;MQ=60;PV4=1,0.0073,1,1;SNPEFF_AMINO_ACID_CHANGE=N3106K;SNPEFF_CODON_CHANGE=aaC/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0020;DN=52;DA=C/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ASN/LYS;PP=3106/3115,3084/3093;CDP=9318,9252;GS=94;PH=0;CP=0.002;CG=3.740;AA=G;HA=38.2;HE=49.2;HC=9.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=177;GESP=G:6554/C:6452;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:4:17,3,0 1/1:9:40,9,0</p>

      <p>1 223284528 rs5744174 A G 264 highCVG AC1=5;AF1=0.6321;DB;DP=26;DP4=4,2,5,13;FQ=58.7;MQ=60;PV4=0.15,1,1,0.16;SNPEFF_AMINO_ACID_CHANGE=F616L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003268.5;VDB=0.0132;DN=114;DA=A/G;GM=NM_003268.5,XM_005273241.1,XM_005273242.1,XM_005273243.1;GL=TLR5;FG=missense;FD=missense,unknown,unknown,unknown;AAC=PHE/LEU;PP=616/859;CDP=1846;GS=22;PH=0;CP=0.000;CG=-11.200;AA=G;HA=10.7;HE=42.7;HC=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1850;KP=hsa04620:Toll-likeReceptorSignalingPathway|hsa05130:PathogenicEscherichiaColiInfection;GESP=G:4352/A:8654;PPI=MYD88:937;PAC=NP_003259.2,XP_005273298.1,XP_005273299.1,XP_005273300.1 GT:GQ:PL 0/1:22:75,0,21 0/0:8:0,12,83 1/1:9:60,9,0 1/1:32:179,33,0</p>

      <p>1 225555602 rs12737248 T G 75.40 highCVG AC1=4;AF1=0.4691;DB;DP=16;DP4=4,2,8,1;FQ=56.2;MQ=60;PV4=0.53,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N3969K;SNPEFF_CODON_CHANGE=aaT/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=75;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001373.1;VDB=0.0002;DN=121;DA=T/G;GM=NM_001373.1;GL=DNAH14;FG=missense;FD=missense;AAC=ASN/LYS;PP=3969/4516;CDP=11907;GS=94;PH=0.001;CP=0.001;CG=-10.100;AA=T;HA=23.0;HE=37.9;HC=25.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=96;GESP=G:2139/T:2427;PAC=NP_001364.1 GT:GQ:PL 0/1:17:15,0,39 0/0:3:0,3,25 1/1:20:106,24,0 0/0:5:0,6,35</p>

      <p>1 227216775 rs1929860 C T 138 highCVG AC1=5;AF1=0.6658;DB;DP=24;DP4=2,7,6,6;FQ=110;MQ=60;PV4=0.37,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=V1304I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC42BPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003607.3;VDB=0.0143;DN=92;DA=C/A/G/T;GM=NM_003607.3,NM_014826.4,XM_005273317.1,XM_005273318.1,XM_005273319.1,XM_005273320.1,XM_005273321.1,XM_005273322.1,XM_005273323.1,XM_005273324.1,XM_005273325.1,XM_005273326.1,XM_005273327.1;GL=CDC42BPA;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=1304/1720,1223/1639,1355/1833,1352/1830,1333/1811,1317/1795,1304/1782,1339/1755,1274/1752,1317/1733,1251/1729,1284/1700,1276/1692;CDP=3910,3667,4063,4054,3997,3949,3910,4015,3820,3949,3751,3850,3826;GS=29;PH=0.001;CP=0.994;CG=4.180;AA=T;HA=25.3;HE=36.6;HC=43.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=177;GESP=T:3673/C:9333;PPI=CDC42:882;PAC=NP_003598.2,NP_055641.3,XP_005273374.1,XP_005273375.1,XP_005273376.1,XP_005273377.1,XP_005273378.1,XP_005273379.1,XP_005273380.1,XP_005273381.1,XP_005273382.1,XP_005273383.1,XP_005273384.1 GT:GQ:PL 0/1:31:25,0,72 0/1:3:0,0,0 1/1:24:127,24,0 0/1:34:28,0,75</p>

      <p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

      <p>1 242035438 rs4149965 G A 6.86 lowCVG AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1 GT:GQ:PL 0/0:3:0,0,0 0/0:3:0,0,0 1/1:4:44,9,0 0/0:16:0,15,101</p>

      <p>1 247048834 rs2642990 T C 119 highCVG AC1=4;AF1=0.5131;DB;DP=15;DP4=2,4,4,4;FQ=67.9;MQ=58;PV4=0.63,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N883S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHCTF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015446.4;VDB=0.0042;DN=100;DA=T/C;GM=NM_015446.4;GL=AHCTF1;FG=missense;FD=missense;AAC=ASN/SER;PP=883/2276;CDP=2648;GS=46;PH=0;CP=0.002;CG=-0.677;AA=C;HA=0.0;HE=49.2;HC=32.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=40;GESP=C:8196/T:4808;PAC=NP_056261.4 GT:GQ:PL 0/0:9:0,12,85 0/1:16:38,0,13 1/1:12:105,15,0 0/1:16:23,0,14</p>

      <p>1 247420421 rs41308154 C T 25.10 midCVG AC1=3;AF1=0.3809;DB;DP=11;DP4=2,3,2,2;FQ=27.1;MQ=60;PV4=1,0.0023,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R350C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VN1R5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173858.1;VDB=0.0123;DN=127;DA=C/T;GM=NM_173858.1;GL=VN1R5;FG=missense;FD=unknown;AAC=ARG/CYS;PP=350/358;CDP=1048;GS=180;PH=0.987;CP=0.337;AA=C;DG;DV=by-cluster,by-1000G;RM=MSTC;DSP=44158;GESP=T:2053/C:9883;PAC=NP_776257.1 GT:GQ:PL 0/0:3:0,0,0 0/0:12:0,12,89 1/1:7:66,12,0 0/0:5:0,3,23</p>

      <p>1 248039451 rs3811444 C T 107 highCVG AC1=4;AF1=0.5108;DB;DP=24;DP4=8,4,4,7;FQ=110;MQ=60;PV4=0.22,0.14,1,0.15;SNPEFF_AMINO_ACID_CHANGE=T374M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM58;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015431.3;VDB=0.0144;DN=107;DA=C/T;GM=NM_015431.3,XM_005273091.1,XM_005273092.1;GL=TRIM58;FG=missense;FD=missense,unknown,unknown;AAC=THR/MET;PP=374/487,230/343,230/343;CDP=1121,689,689;GS=81;PH=0.999;CP=0.011;CG=2.210;AA=C;HA=3.3;HE=33.2;HC=23.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=250;GESP=T:3365/C:9641;PAC=NP_056246.3,XP_005273148.1,XP_005273149.1 GT:GQ:PL 0/0:12:0,15,107 0/1:13:19,0,11 1/1:21:112,24,0 0/1:27:24,0,91</p>

      <p>1 248458718 rs79635253 T C 14.40 midCVG AC1=4;AF1=0.443;DB;DP=9;DP4=4,1,1,2;FQ=16.8;MQ=45;PV4=0.46,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R55G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004692.1;VDB=0.0077;DN=131;DA=T/C;GM=NM_001004692.1;GL=OR2T12;FG=missense;FD=missense;AAC=ARG/GLY;PP=55/321;CDP=163;GS=125;PH=0.005;CP=0.159;CG=1.550;AA=T;DV=by-frequency,by-cluster;DSP=196059;KP=hsa04740:OlfactoryTransduction;GESP=C:613/T:12389;PAC=NP_001004692.1 GT:GQ:PL 0/0:4:0,3,23 0/1:4:13,3,0 1/1:3:45,6,0 0/0:12:0,12,73</p>

      <p>1 248685719 rs9330305 A T 35.40 highCVG AC1=3;AF1=0.3917;DB;DP=16;DP4=3,9,3,1;FQ=37.5;MQ=60;PV4=0.12,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M258L;SNPEFF_CODON_CHANGE=Atg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2G6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001013355.1;VDB=0.0168;DN=119;DA=A/T;GM=NM_001013355.1;GL=OR2G6;FG=missense;FD=missense;AAC=MET/LEU;PP=258/317;CDP=772;GS=15;PH=0.934;CP=0.966;CG=3.830;AA=A;HA=33.6;HE=12.1;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=159907;KP=hsa04740:OlfactoryTransduction;GESP=T:2221/A:10785;PAC=NP_001013373.1 GT:GQ:PL 0/0:15:0,15,89 0/1:4:17,3,0 1/1:5:66,9,0 0/0:20:0,21,135</p>

      <p>1 248722777 rs77589892 T A 89 highCVG AC1=5;AF1=0.7783;DB;DP=29;DP4=7,2,14,5;FQ=-25.8;MQ=12;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R6W;SNPEFF_CODON_CHANGE=Agg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004694.2;VDB=0.0185;DN=131;DA=T/A;GM=NM_001004694.2;GL=OR2T29;FG=missense;FD=missense;AAC=ARG/TRP;PP=6/316;CDP=16;GS=101;CP=0.001;CG=-0.604;AA=A;DG;DV=by-frequency,by-cluster;DSP=122849;KP=hsa04740:OlfactoryTransduction;GESP=A:4597/T:8405;PAC=NP_001004694.2 GT:GQ:PL 0/0:4:0,12,14 0/1:3:6,0,2 1/1:22:86,20,0 1/1:23:44,21,0</p>

      <p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

      <p>2 25022598 rs1550116 A G 39.40 lowCVG AC1=6;AF1=0.7255;DB;DP=6;DP4=1,1,1,3;FQ=3.46;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q34R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024322.2;VDB=0.0085;DN=88;DA=A/G;GM=NM_001199803.1,NM_024322.2;GL=CENPO;FG=missense;FD=missense;AAC=GLN/ARG;PP=28/295,34/301;CDP=83,101;GS=43;PH=0;CP=0.000;CG=-3.630;AA=A;HA=5.8;HE=18.0;HC=19.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=55;GESP=G:1761/A:11245;PAC=NP_001186732.1,NP_077298.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:64,9,0 0/1:17:12,0,32</p>

      <p>2 26536694 rs2052937 C T 46.40 midCVG AC1=4;AF1=0.5336;DB;DP=10;DP4=5,0,1,4;FQ=47.7;MQ=60;PV4=0.048,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=A404T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR113;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145168.1;VDB=0.0034;DN=94;DA=C/T;GM=NM_001145168.1,NM_001145169.1,NM_153835.3,XM_005264163.1;GL=GPR113;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=404/1080,335/998,205/874,335/1004;CDP=1210,1003,613,1003;GS=58;PH=0.066;CP=0.007;CG=3.470;AA=T;HA=34.1;HE=32.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=67;GESP=T:8972/C:4034;PAC=NP_001138640.1,NP_001138641.1,NP_722577.2,XP_005264220.1 GT:GQ:PL 0/0:7:0,9,60 0/1:35:49,0,32 1/1:3:26,3,0 1/1:3:15,3,0</p>

      <p>2 27464018 . C A 18.10 highCVG AC1=2;AF1=0.2604;DP=16;DP4=3,7,0,5;FQ=19.3;MQ=60;PV4=0.51,0.0092,1,0.00055;SNPEFF_AMINO_ACID_CHANGE=H1911N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=37;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CAD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004341.3;VDB=0.0000;GM=NM_004341.3,XM_005264555.1,XM_005264556.1,XM_005264557.1,XM_005264558.1;GL=CAD;FG=missense;FD=unknown;AAC=HIS/ASN;PP=1911/2226,1928/2243,1894/2209,1877/2192,1848/2163;CDP=5731,5782,5680,5629,5542;GS=68;PH=0.283;CP=1.000;CG=4.920;AA=C;DSP=78;KP=hsa00240:PyrimidineMetabolism|hsa00250:Alanine,AspartateAndGlutamateMetabolism|hsa01100:MetabolicPathways;TFBS=V$TAXCREB_01;GESP=C:13006;PPI=GRB2:814|SHC1:764;PAC=NP_004332.2,XP_005264612.1,XP_005264613.1,XP_005264614.1,XP_005264615.1 GT:GQ:PL 0/0:14:0,12,65 0/0:9:0,6,50 1/1:8:63,15,0 0/0:14:0,12,88</p>

      <p>2 27717440 . C A 19.60 midCVG AC1=2;AF1=0.3049;DP=14;DP4=5,3,0,3;FQ=21;MQ=60;PV4=0.18,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=S36I;SNPEFF_CODON_CHANGE=aGc/aTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FNDC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022823.2;VDB=0.0046;GM=NM_022823.2,XM_005264499.1;GL=FNDC4;FG=missense;FD=unknown;AAC=SER/ILE;PP=36/235,36/209;CDP=107;GS=142;PH=0.053;CP=0.995;CG=4.380;AA=C;DSP=27;GESP=C:13004;PAC=NP_073734.1,XP_005264556.1 GT:GQ:PL 0/0:8:0,6,46 0/0:3:0,0,1/1:4:60,9,0 0/0:20:0,18,97</p>

      <p>2 29295186 rs75276619 C T 29.50 midCVG AC1=3;AF1=0.3733;DB;DP=12;DP4=4,4,3,1;FQ=31.5;MQ=60;PV4=0.58,0.41,1,0.46;SNPEFF_AMINO_ACID_CHANGE=A648T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf71;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001029883.2;VDB=0.0002;DN=131;DA=C/T;GM=NM_001029883.2;GL=C2orf71;FG=missense;FD=missense;AAC=ALA/THR;PP=648/1289;CDP=1942;GS=58;PH=0.2;CP=0.000;CG=-3.320;AA=C;DV=by-frequency,by-cluster;DSP=1727;GESP=T:113/C:12423;PAC=NP_001025054.1 GT:GQ:PL 0/1:11:10,0,51 0/0:9:0,9,67 1/1:5:64,9,0 0/0:7:0,6,37</p>

      <p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

      <p>2 70488470 rs2706762 C T 88.20 highCVG AC1=5;AF1=0.6256;DB;DP=20;DP4=9,1,1,7;FQ=81.1;MQ=60;PV4=0.0029,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=S149F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCYOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016297.3;VDB=0.0031;DN=100;DA=C/T;GM=NM_016297.3,XM_005264362.1;GL=PCYOX1;FG=missense;FD=missense,unknown;AAC=SER/PHE;PP=149/506,131/488;CDP=446,392;GS=155;PH=0;CP=1.000;CG=4.580;AA=C;HA=0.0;HE=14.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=49;GESP=T:1447/C:11559;PAC=NP_057381.3,XP_005264419.1 GT:GQ:PL 0/0:26:0,30,123 0/1:3:0,0,0 1/1:17:89,18,0 1/1:6:45,6,0</p>

      <p>2 71058835 rs741326 A G 155 highCVG AC1=6;AF1=0.7158;DB;DP=19;DP4=1,3,12,3;FQ=25.5;MQ=60;PV4=0.071,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V278A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD207;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015717.4;VDB=0.0056;DN=86;DA=A/G;GM=NM_015717.3;GL=CD207;FG=missense;FD=unknown;AAC=VAL/ALA;PP=278/329;CDP=833;GS=64;PH=0;CP=0.000;CG=-9.570;AA=G;HA=49.4;HE=47.6;HC=30.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=4;GESP=G:5453/A:7019;PAC=NP_056532.3 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:25:130,25,0 1/1:9:70,9,0</p>

      <p>2 74755959 . G A 6.27 midCVG AC1=3;AF1=0.3379;DP=10;DP4=6,1,0,3;FQ=7.61;MQ=60;PV4=0.033,0.067,1,0.092;SNPEFF_AMINO_ACID_CHANGE=S148F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AUP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181575.3;VDB=0.0037;GM=NM_181575.3,XM_005264392.1;GL=AUP1;FG=missense;FD=unknown;AAC=SER/PHE;PP=148/411,205/468;CDP=443,614;GS=155;PH=0.99;CP=1.000;CG=4.820;AA=G;DSP=82;GESP=G:12058;PPI=UBE2G2:873|UBC:852|SEL1L:846;PAC=NP_853553.1,XP_005264449.1 GT:GQ:PL 0/0:4:0,0,0 0/0:7:0,3,19 1/1:3:45,9,0 0/0:21:0,18,92</p>

      <p>2 96993757 rs2279105 C T 69.10 lowCVG AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3 GT:GQ:PL 0/1:16:12,0,21 0/1:4:0,3,17 1/1:4:21,3,0 1/1:12:80,12,0</p>

      <p>2 97638038 rs1730120 A G 94.30 midCVG AC1=5;AF1=0.5991;DB;DP=9;DP4=1,1,4,3;FQ=-3.35;MQ=60;PV4=1,0.4,1,0.26;SNPEFF_AMINO_ACID_CHANGE=V55A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM178B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001122646.2;VDB=0.0156;DN=89;DA=A/G;GM=NM_001122646.2,XM_005263964.1,XM_005263965.1;GL=FAM178B;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=55/680,29/654,55/494;CDP=164,86,164;GS=64;CP=0.000;CG=-4.560;AA=A;HA=30.4;HE=13.6;HC=45.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=22;GESP=G:3773/A:771;PAC=NP_001116118.2,XP_005264021.1,XP_005264022.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:12:60,0,10</p>

      <p>2 98165911 rs1839230 T C 170 highCVG AC1=5;AF1=0.6034;DB;DP=35;DP4=0,3,12,4;FQ=5.73;MQ=50;PV4=0.036,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E483G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD36B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025190.3;VDB=0.0137;DN=92;DA=T/C;GM=NM_025190.3,XM_005263995.1,XM_005263996.1,XM_005263997.1,XM_005263998.1,XM_005263999.1,XM_005264000.1,XM_005264001.1;GL=ANKRD36B;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLU/GLY;PP=483/1354,483/1354,483/1354,483/1353,483/1320,251/1122,232/1103,483/939;CDP=1448,1448,1448,1448,1448,752,695,1448;GS=98;PH=0;CP=0.002;CG=-0.893;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=30;GESP=T:11082;PAC=NP_079466.3,XP_005264052.1,XP_005264053.1,XP_005264054.1,XP_005264055.1,XP_005264056.1,XP_005264057.1,XP_005264058.1 GT:GQ:PL 0/1:4:0,3,24 0/1:3:0,0,0 1/1:34:154,36,0 0/1:20:58,0,18</p>

      <p>2 100055158 rs3087399 T C 126 highCVG AC1=5;AF1=0.5732;DB;DP=19;DP4=8,3,4,4;FQ=128;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N373S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=REV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016316.2;VDB=0.0092;DN=102;DA=T/C;GM=NM_001037872.1,NM_016316.2,XM_005263967.1,XM_005263968.1,XM_005263969.1;GL=REV1;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown,unknown;AAC=ASN/SER,ASN/SER,ASN/SER,ASN/SER,none;PP=373/1251,373/1252,373/1252,303/1182,NA;CDP=1118,1118,1118,908,NA;GS=46,46,46,46,NA;PH=0.069;CP=0.794;CG=-0.648;AA=T;HA=22.9;HE=10.8;HC=3.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=96;GESP=C:2153/T:10853;PPI=MAD2L2:999|REV3L:998|POLH:995|POLI:846|POLK:846|UBC:845;PAC=NP_001032961.1,NP_057400.1,XP_005264024.1,XP_005264025.1,NA GT:GQ:PL 0/1:19:54,0,17 0/0:26:0,30,163 1/1:8:72,9,0 1/1:5:48,6,0</p>

      <p>2 101086966 rs13411940 C T 23.20 midCVG AC1=3;AF1=0.4057;DB;DP=11;DP4=4,1,2,2;FQ=25.4;MQ=60;PV4=0.52,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=P6S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NMS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001011717.1;VDB=0.0096;DN=121;DA=C/T;GM=NM_001011717.1;GL=NMS;FG=missense;FD=missense;AAC=PRO/SER;PP=6/154;CDP=16;GS=74;PH=0.363;CP=0.003;CG=2.530;AA=C;HA=28.1;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=61;GESP=T:3092/C:9914;PAC=NP_001011717.1 GT:GQ:PL 0/0:9:0,9,52 0/1:11:11,0,13 1/1:5:56,9,0 0/0:4:0,3,29</p>

      <p>2 113537223 rs17561 C A 31.40 midCVG AC1=2;AF1=0.2957;DB;DP=14;DP4=9,1,4,0;FQ=32.8;MQ=60;PV4=1,0.45,1,1;SNPEFF_AMINO_ACID_CHANGE=A114S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000575.3;VDB=0.0004;DN=63;DA=C/A;GM=NM_000575.3;GL=IL1A;FG=missense;FD=missense;AAC=ALA/SER;PP=114/272;CDP=340;GS=99;PH=0.982;CP=0.009;CG=4.440;AA=C;HA=14.5;HE=27.6;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;KP=hsa04010:MAPKSignalingPathway|hsa04060:Cytokine-cytokineReceptorInteraction|hsa04210:Apoptosis|hsa04640:HematopoieticCellLineage|hsa04940:TypeIDiabetesMellitus|hsa05020:PrionDiseases|hsa05140:Leishmaniasis|hsa05332:Graft-versus-hostDisease;GESP=A:3403/C:9603;PPI=S100A13:959;PAC=NP_000566.3 GT:GQ:PL 0/0:26:0,24,122 0/0:6:0,3,27 1/1:6:74,12,0 0/0:6:0,3,27</p>

      <p>2 120885310 rs1034489 G A 53.80 highCVG AC1=4;AF1=0.5281;DB;DP=21;DP4=0,9,6,2;FQ=57.1;MQ=60;PV4=0.0023,0.00019,1,1;SNPEFF_AMINO_ACID_CHANGE=A462T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020909.3;VDB=0.0023;DN=86;DA=G/A;GM=NM_001184937.1,NM_020909.3,XM_005263719.1,XM_005263720.1,XM_005263721.1,XM_005263722.1;GL=EPB41L5;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=462/688,462/734,465/737,465/736,462/734,390/662;CDP=1384,1384,1393,1393,1384,1168;GS=58;PH=0;CP=0.973;CG=1.730;AA=A;HA=41.4;HE=26.4;HC=32.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$ROAZ_01;GESP=A:8169/G:4837;PAC=NP_001171866.1,NP_065960.2,XP_005263776.1,XP_005263777.1,XP_005263778.1,XP_005263779.1 GT:GQ:PL 0/1:35:32,0,51 0/1:3:0,3,21 1/1:5:32,6,0 0/1:36:33,0,69</p>

      <p>2 131414176 rs62165276 C T 7.93 midCVG AC1=4;AF1=0.5176;DB;DP=10;DP4=1,3,3,2;FQ=10.6;MQ=18;PV4=0.52,0.35,0.061,0.14;SNPEFF_AMINO_ACID_CHANGE=R615W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POTEJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277083.1;VDB=0.0090;DN=129;DA=C/T;GM=NM_001277083.1,XM_005263739.1,XM_005263740.1;GL=POTEJ;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=615/1039,497/921,492/916;CDP=1843,1489,1474;GS=101;CP=0.246;CG=0.993;AA=C;DG;DV=by-frequency,by-cluster;DSP=55;PAC=NP_001264012.1,XP_005263796.1,XP_005263797.1 GT:GQ:PL 0/0:6:0,6,33 0/1:3:0,0,0 1/1:5:39,9,0 0/1:10:9,0,34</p>

      <p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

      <p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

      <p>2 159954175 rs34588551 C T 73.30 midCVG AC1=4;AF1=0.5077;DB;DP=12;DP4=4,2,2,4;FQ=64.1;MQ=60;PV4=0.57,1,1,0.077;SNPEFF_AMINO_ACID_CHANGE=P30S;SNPEFF_CODON_CHANGE=Cca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TANC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033394.2;VDB=0.0215;DN=126;DA=C/T;GM=NM_001145909.1,NM_033394.2,XM_005246918.1,XM_005246919.1,XM_005246920.1;GL=TANC1;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=PRO/SER;PP=30/1391,30/1862,40/1872,40/1865,40/1401;CDP=88,88,118,118,118;GS=74;PH=0.004;CP=0.003;CG=0.726;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=27;GESP=T:3302/C:8928;PAC=NP_001139381.1,NP_203752.2,XP_005246975.1,XP_005246976.1,XP_005246977.1 GT:GQ:PL 0/0:13:0,15,95 0/1:3:0,0,0 1/1:12:98,15,0 0/1:13:19,0,10</p>

      <p>2 160711045 rs3951216 G C 49.60 lowCVG AC1=4;AF1=0.531;DB;DP=8;DP4=1,2,2,3;FQ=23.9;MQ=60;PV4=1,0.4,1,0.026;SNPEFF_AMINO_ACID_CHANGE=D807E;SNPEFF_CODON_CHANGE=gaC/gaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LY75-CD302;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198759.1;VDB=0.0092;DN=108;DA=G/C;GM=NM_001198759.1,NM_001198760.1,NM_002349.3,XM_005246553.1;GL=LY75/LY75-CD302;FG=missense-near-splice;FD=missense,missense,missense,unknown;AAC=ASP/GLU;PP=807/1874,807/1818,807/1723,807/1625;CDP=2421;GS=45;PH=0.002;CP=1.000;CG=3.670;AA=C;HA=39.2;HE=41.7;HC=32.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2;GESP=C:5483/G:7523;PAC=NP_001185688.1,NP_001185689.1,NP_002340.2,XP_005246610.1 GT:GQ:PL 0/0:5:0,6,42 0/1:3:0,0,0 1/1:7:59,9,0 0/1:23:32,0,20</p>

      <p>2 167099158 rs6746030 A G 150 highCVG AC1=6;AF1=0.7278;DB;DP=16;DP4=2,4,7,3;FQ=65.7;MQ=60;PV4=0.3,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W1150R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCN9A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002977.3;VDB=0.0076;DN=116;DA=A/G;GM=NM_002977.3,XM_005246757.1;GL=SCN9A;FG=missense;FD=missense,unknown;AAC=TRP/ARG;PP=1150/1978,1161/1989;CDP=3448,3481;GS=101;PH=0;CP=0.788;CG=3.830;AA=G;HA=13.1;HE=12.3;HC=4.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=9;GESP=G:10303/A:1439;PAC=NP_002968.1,XP_005246814.1 GT:GQ:PL 0/1:43:47,0,44 0/1:22:15,0,61 1/1:10:59,9,0 1/1:10:74,9,0</p>

      <p>2 178417142 rs11694988 C T 97.90 midCVG AC1=5;AF1=0.5858;DB;DP=13;DP4=2,4,3,4;FQ=78.7;MQ=50;PV4=1,0.46,0.044,0.32;SNPEFF_AMINO_ACID_CHANGE=R117H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152517.2;VDB=0.0034;DN=120;DA=C/T;GM=NM_152517.2;GL=TTC30B;FG=missense;FD=missense;AAC=ARG/HIS;PP=117/666;CDP=350;GS=29;PH=0.944;CP=0.996;CG=4.300;AA=C;HA=15.8;HE=14.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14341;GESP=T:11079/C:1927;PAC=NP_689730.2 GT:GQ:PL 0/0:11:0,15,107 0/1:18:35,0,16 1/1:8:64,9,0 1/1:6:47,6,0</p>

      <p>2 185802243 rs1366842 C A 30.40 highCVG AC1=2;AF1=0.2953;DB;DP=17;DP4=3,8,4,0;FQ=31.8;MQ=60;PV4=0.026,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T707K;SNPEFF_CODON_CHANGE=aCa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF804A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_194250.1;VDB=0.0182;DN=88;DA=C/A;GM=NM_194250.1;GL=ZNF804A;FG=missense;FD=missense;AAC=THR/LYS;PP=707/1210;CDP=2120;GS=78;PH=0.002;CP=0.000;CG=1.860;AA=C;HA=4.5;HE=40.1;HC=13.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1734;GESP=A:5568/C:7416;PAC=NP_919226.1 GT:GQ:PL 0/0:6:0,3,26 0/0:29:0,27,124 1/1:6:73,12,0 0/0:6:0,3,28</p>

      <p>2 186656956 rs17229201 T C 27.70 lowCVG AC1=5;AF1=0.6553;DB;DP=5;DP4=2,0,2,1;FQ=12.6;MQ=60;PV4=1,0.22,1,0.25;SNPEFF_AMINO_ACID_CHANGE=M1787T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FSIP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173651.2;VDB=0.0061;DN=123;DA=T/C;GM=NM_173651.2;GL=FSIP2;FG=missense;FD=missense;AAC=MET/THR;PP=1787/6997;CDP=5360;GS=81;CP=0.371;CG=3.390;AA=C;HA=42.4;HE=48.8;HC=46.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3587;PAC=NP_775922.2 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,48 1/1:4:21,3,0 1/1:5:47,6,0</p>

      <p>2 196749503 rs371128668 C T 102 highCVG AC1=5;AF1=0.5615;DB;DP=15;DP4=2,2,5,4;FQ=23.9;MQ=60;PV4=1,0.15,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V1857I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018897.2;VDB=0.0045;DN=138;DA=C/T;GM=NM_018897.2;GL=DNAH7;FG=missense;FD=missense;AAC=VAL/ILE;PP=1857/4025;CDP=5569;GS=29;PH=0.001;CP=0.000;CG=-11.500;AA=C;DSP=31;GESP=T:4/C:11878;PAC=NP_061720.2 GT:GQ:PL 0/1:27:24,0,31 0/1:3:0,3,25 1/1:8:64,9,0 0/1:11:58,0,8</p>

      <p>2 196837129 rs73042525 G C 20.80 midCVG AC1=4;AF1=0.4605;DB;DP=9;DP4=2,2,0,3;FQ=23.2;MQ=60;PV4=0.43,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=S632C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018897.2;VDB=0.0121;DN=130;DA=G/C;GM=NM_018897.2;GL=DNAH7;FG=missense;FD=missense;AAC=SER/CYS;PP=632/4025;CDP=1895;GS=112;PH=0.009;CP=0.997;CG=4.750;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=62;GESP=C:856/G:11112;PAC=NP_061720.2 GT:GQ:PL 0/0:6:0,6,37 0/1:4:22,3,0 1/1:4:43,6,0 0/0:6:0,6,39</p>

      <p>2 207570529 rs56067462 G C 69.40 midCVG AC1=4;AF1=0.5258;DB;DP=14;DP4=2,5,4,2;FQ=67.2;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P122R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DYTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001093730.1;VDB=0.0048;DN=129;DA=G/C;GM=NM_001093730.1;GL=DYTN;FG=missense;FD=missense;AAC=PRO/ARG;PP=122/579;CDP=365;GS=103;PH=1;CP=1.000;CG=4.150;AA=G;DV=by-frequency,by-cluster;DSP=18;GESP=C:854/G:11154;PAC=NP_001087199.1 GT:GQ:PL 0/1:3:0,0,0 0/0:15:0,18,106 1/1:7:57,9,0 0/1:6:54,0,1</p>

      <p>2 207820276 rs13420911 G A 43.40 highCVG AC1=2;AF1=0.2979;DB;DP=18;DP4=6,8,2,2;FQ=44.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M85I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173077.2;VDB=0.0134;DN=121;DA=G/A;GM=NM_173077.2;GL=CPO;FG=missense;FD=missense;AAC=MET/ILE;PP=85/375;CDP=255;GS=10;PH=0.021;CP=0.999;CG=5.110;AA=G;HA=33.2;HE=44.8;HC=39.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=13;GESP=A:5799/G:7207;PAC=NP_775100.1 GT:GQ:PL 0/0:20:0,18,108 0/0:3:0,0,0 1/1:6:86,12,0 0/0:25:0,24,153</p>

      <p>2 218674697 rs918949 C T 84.60 midCVG AC1=5;AF1=0.6725;DB;DP=9;DP4=0,3,2,3;FQ=25.6;MQ=60;PV4=0.46,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V1604I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022648.4;VDB=0.0060;DN=86;DA=C/T;GM=NM_022648.4,XM_005246821.1,XM_005246822.1,XM_005246823.1,XM_005246824.1,XM_005246825.1,XM_005246826.1,XM_005246827.1;GL=TNS1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=1604/1736,1708/1840,1677/1809,1677/1809,1583/1715,1387/1519,1221/1353,1221/1353;CDP=4810,5122,5029,5029,4747,4159,3661,3661;GS=29;PH=0;CP=0.368;CG=-4.710;HA=17.3;HE=39.5;HC=47.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=18;GESP=T:8637/C:4369;PAC=NP_072174.3,XP_005246878.1,XP_005246879.1,XP_005246880.1,XP_005246881.1,XP_005246882.1,XP_005246883.1,XP_005246884.1 GT:GQ:PL 0/0:6:0,9,64 0/1:3:0,0,0 1/1:6:50,6,0 1/1:8:78,9,0</p>

      <p>2 220353532 rs13026308 C A 58.90 midCVG AC1=6;AF1=0.7369;DB;DP=11;DP4=1,1,5,1;FQ=-4.99;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2687T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=34;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005876.4;VDB=0.0094;DN=121;DA=C/A;GM=NM_005876.4,XM_005246235.1,XM_005246236.1,XM_005246237.1,XM_005246238.1,XM_005246239.1,XM_005246240.1,XM_005246241.1,XM_005246242.1;GL=SPEG;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/THR;PP=2687/3268,2628/3209,2626/3207,2593/3174,2534/3115,1895/2476,1838/2419,1838/2419,1900/2481;CDP=8059,7882,7876,7777,7600,5683,5512,5512,5698;GS=38;PH=0.997;CP=0.999;CG=4.380;AA=C;HA=0.0;HE=16.1;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;GESP=A:1183/C:11149;PAC=NP_005867.3,XP_005246292.1,XP_005246293.1,XP_005246294.1,XP_005246295.1,XP_005246296.1,XP_005246297.1,XP_005246298.1,XP_005246299.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:9:54,9,0 1/1:9:48,9,0</p>

      <p>2 228476253 rs2138402 C G 23.70 lowCVG AC1=4;AF1=0.4527;DB;DP=8;DP4=5,0,1,2;FQ=26.3;MQ=59;PV4=0.11,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E104Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf83;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020161.3;VDB=0.0071;DN=96;DA=C/G;GM=NM_001162483.1,NM_020161.3;GL=C2orf83;FG=3-prime-UTR,missense;FD=utr-variant-3-prime,missense;AAC=none,GLU/GLN;PP=NA,104/151;CDP=NA,310;GS=NA,29;PH=0;CP=0.000;CG=0.457;AA=G;HA=33.2;HE=6.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=120;GESP=G:10654/C:2352;PAC=NA,NP_064546.3 GT:GQ:PL 0/0:11:0,12,62 0/0:4:0,3,25 1/1:4:42,6,0 0/1:4:26,3,0</p>

      <p>2 228883029 rs3811515 A C 134 midCVG AC1=5;AF1=0.5723;DB;DP=12;DP4=1,2,4,5;FQ=24.2;MQ=60;PV4=1,0.15,1,0.21;SNPEFF_AMINO_ACID_CHANGE=H847Q;SNPEFF_CODON_CHANGE=caT/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPHKAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001142644.1;VDB=0.0094;DN=107;DA=A/C;GM=NM_001142644.1,NM_030623.3,XM_005246867.1,XM_005246868.1,XM_005246869.1,XM_005246870.1;GL=SPHKAP;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=HIS/GLN;PP=847/1701,847/1672,860/1714,843/1697,860/1685,830/1684;CDP=2541,2541,2580,2529,2580,2490;GS=24;PH=0.019;CP=0.000;CG=0.984;AA=A;HA=0.9;HE=27.9;HC=30.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1844;GESP=C:2449/A:10557;PAC=NP_001136116.1,NP_085126.2,XP_005246924.1,XP_005246925.1,XP_005246926.1,XP_005246927.1 GT:GQ:PL 0/0:4:0,6,55 0/1:3:0,0,0 1/1:10:84,12,0 0/1:10:92,0,7</p>

      <p>2 228883721 rs3811514 T C 19.80 midCVG AC1=3;AF1=0.4098;DB;DP=10;DP4=5,2,1,2;FQ=22;MQ=60;PV4=0.5,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K617E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPHKAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001142644.1;VDB=0.0098;DN=107;DA=T/C;GM=NM_001142644.1,NM_030623.3,XM_005246867.1,XM_005246868.1,XM_005246869.1,XM_005246870.1;GL=SPHKAP;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=617/1701,617/1672,630/1714,613/1697,630/1685,600/1684;CDP=1849,1849,1888,1837,1888,1798;GS=56;PH=0;CP=0.074;CG=4.050;AA=C;HA=17.4;HE=44.8;HC=31.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1152;GESP=C:7539/T:5467;PAC=NP_001136116.1,NP_085126.2,XP_005246924.1,XP_005246925.1,XP_005246926.1,XP_005246927.1 GT:GQ:PL 0/0:14:0,15,90 0/0:6:0,6,42 1/1:3:45,6,0 0/1:4:20,3,0</p>

      <p>2 231865090 rs2271376 T C 84.80 midCVG AC1=4;AF1=0.5465;DB;DP=9;DP4=1,2,4,2;FQ=5.61;MQ=60;PV4=0.52,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=I104T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139073.3;VDB=0.0134;DN=100;DA=T/C;GM=NM_139073.3,XM_005246303.1;GL=SPATA3;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=104/193;CDP=311;GS=89;PH=0;CP=0.001;CG=-0.717;AA=C;HA=48.8;HE=35.5;HC=40.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;GESP=C:1865/T:2701;PAC=NP_620712.2,XP_005246360.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,16 1/1:5:47,6,0 0/1:27:77,0,24</p>

      <p>2 234526871 rs1042597 C G 32.10 midCVG AC1=4;AF1=0.5;DB;DP=9;DP4=0,4,0,4;FQ=34.9;MQ=50;PV4=1,0.18,0.04,1;SNPEFF_AMINO_ACID_CHANGE=A173G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UGT1A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019076.4;VDB=0.0066;DN=86;DA=C/G/T;GM=NM_019076.4;GL=UGT1A8;FG=missense;FD=missense;AAC=ALA/GLY;PP=173/531;CDP=518;GS=60;PH=0.044;CP=0.002;CG=-0.180;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=338;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00053:AscorbateAndAldarateMetabolism|hsa00140:SteroidHormoneBiosynthesis|hsa00500:StarchAndSucroseMetabolism|hsa00830:RetinolMetabolism|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:2472/C:10534;PAC=NP_061949.3 GT:GQ:PL 0/0:5:0,6,48 0/0:5:0,6,44 1/1:4:39,6,0 1/1:4:39,6,0</p>

      <p>2 237149941 rs6756597 C T 24.40 highCVG AC1=2;AF1=0.254;DB;DP=18;DP4=7,5,4,0;FQ=25.6;MQ=60;PV4=0.24,0.36,1,0.041;SNPEFF_AMINO_ACID_CHANGE=A104T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ASB18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_212556.2;VDB=0.0223;DN=116;DA=C/T;GM=NM_212556.2;GL=ASB18;FG=missense;FD=missense;AAC=ALA/THR;PP=104/467;CDP=310;GS=58;PH=0.76;CP=0.928;CG=4.820;AA=C;HA=29.5;HE=46.2;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19;TFBS=V$P53_01,V$SRF_Q6;GESP=T:5089/C:6967;PAC=NP_997721.2 GT:GQ:PL 0/0:6:0,3,20 0/0:23:0,21,119 1/1:5:68,12,0 0/0:14:0,12,79</p>

      <p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

      <p>2 242129524 rs34069570 G A 33.10 lowCVG AC1=4;AF1=0.4716;DB;DP=8;DP4=0,4,1,3;FQ=32.3;MQ=60;PV4=1,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=D70N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANO7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001891.3;VDB=0.0068;DN=126;DA=G/A;GM=NM_001001666.3,NM_001001891.3;GL=ANO7;FG=missense;FD=missense;AAC=ASP/ASN;PP=70/180,70/934;CDP=208;GS=23;PH=0.025;CP=0.001;CG=-1.970;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=53;TFBS=V$CDP_02;GESP=A:264/G:12706;PAC=NP_001001666.1,NP_001001891.2 GT:GQ:PL 0/1:3:0,0,0 0/1:22:42,0,18 1/1:4:32,6,0 0/0:8:0,9,55</p>

      <p>3 361508 rs2272522 C T 27.20 midCVG AC1=5;AF1=0.5739;DB;DP=10;DP4=2,2,1,3;FQ=30.5;MQ=60;PV4=1,0.13,1,1;SNPEFF_AMINO_ACID_CHANGE=L17F;SNPEFF_CODON_CHANGE=Ctc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006614.3;VDB=0.0091;DN=100;DA=C/T;GM=NM_001253387.1,NM_001253388.1,NM_006614.3;GL=CHL1;FG=missense;FD=missense;AAC=LEU/PHE;PP=17/1209,17/1172,17/1225;CDP=49;GS=22;PH=0.001;CP=0.003;CG=-2.030;AA=T;HA=37.9;HE=21.9;HC=30.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=43;GESP=T:4018/C:8986;PAC=NP_001240316.1,NP_001240317.1,NP_006605.2 GT:GQ:PL 0/1:13:16,0,11 0/0:6:0,9,70 1/1:3:20,3,0 1/1:5:36,6,0</p>

      <p>3 12098716 . C G 4.55 lowCVG AC1=4;AF1=0.4404;DP=6;DP4=0,2,3,0;FQ=5.91;MQ=57;PV4=0.1,0.0095,1,1;VDB=0.0091;GM=none;FG=intergenic;FD=unknown;CP=0.003;CG=-0.658;AA=C;RM=AluSz;DSP=83493 GT:GQ:PL 0/0:6:0,3,25 0/0:6:0,3,26 1/1:3:41,9,0 0/0:4:0,0,0</p>

      <p>3 12613601 . C T 92.20 highCVG AC1=3;AF1=0.4135;DP=17;DP4=2,6,4,3;FQ=80.3;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P124L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MKRN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014160.4;VDB=0.0150;GM=NM_001271707.1,NM_014160.4;GL=MKRN2;FG=missense;FD=unknown;AAC=PRO/LEU;PP=81/374,124/417;CDP=242,371;GS=98;PH=0.856;CP=0.000;CG=0.672;AA=C;DSP=34;GESP=C:13006;PAC=NP_001258636.1,NP_054879.3 GT:GQ:PL 0/0:14:0,15,75 0/0:4:0,3,26 1/1:8:84,12,0 0/1:28:54,0,23</p>

      <p>3 12859004 rs3732678 A C 27.40 lowCVG AC1=5;AF1=0.5732;DB;DP=7;DP4=3,1,2,1;FQ=30.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H858P;SNPEFF_CODON_CHANGE=cAc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CAND2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001162499.1;VDB=0.0098;DN=107;DA=A/C/G/T;GM=NM_001162499.1,NM_012298.2;GL=CAND2;FG=missense;FD=missense;AAC=HIS/PRO;PP=858/1237,765/1120;CDP=2573,2294;GS=77;PH=0;CP=1.000;CG=3.880;AA=C;HA=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=372;TFBS=V$PAX5_02;GESP=C:8707/A:3773;PAC=NP_001155971.1,NP_036430.1 GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:19,3,0</p>

      <p>3 13361287 rs354478 C T 70.90 midCVG AC1=5;AF1=0.5672;DB;DP=11;DP4=4,1,4,2;FQ=39.6;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=V1787M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=37;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUP210;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024923.3;VDB=0.0032;DN=79;DA=C/T;GM=NM_024923.3;GL=NUP210;FG=missense;FD=missense;AAC=VAL/MET;PP=1787/1888;CDP=5359;GS=21;PH=0;CP=0.698;CG=-1.130;AA=T;HA=43.7;HE=48.6;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=25;GESP=T:6670/C:6336;PAC=NP_079199.2 GT:GQ:PL 0/1:26:49,0,24 0/0:6:0,9,57 1/1:5:44,6,0 1/1:3:23,3,0</p>

      <p>3 14755617 rs6790129 C G 50.50 midCVG AC1=3;AF1=0.3771;DB;DP=13;DP4=5,2,5,0;FQ=51.9;MQ=60;PV4=0.47,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L422V;SNPEFF_CODON_CHANGE=Cta/Gta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C3orf20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032137.4;VDB=0.0072;DN=116;DA=C/G;GM=NM_001184957.1,NM_001184958.1,NM_032137.4,XM_005265511.1;GL=C3orf20;FG=missense;FD=missense,missense,missense,unknown;AAC=LEU/VAL;PP=300/783,300/783,422/905,422/905;CDP=898,898,1264,1264;GS=32;PH=0.873;CP=0.948;CG=1.570;AA=C;HA=43.4;HE=39.8;HC=34.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=50;TFBS=V$HOX13_01;GESP=G:5559/C:7447;PAC=NP_001171886.1,NP_001171887.1,NP_115513.4,XP_005265568.GT:GQ:PL 0/0:9:0,9,40 0/0:9:0,9,56 1/1:5:56,9,0 0/1:14:40,0,8</p>

      <p>3 45637026 . C A 59.40 midCVG AC1=4;AF1=0.5;DP=14;DP4=1,3,0,9;FQ=9.86;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P219T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIMD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014240.2;VDB=0.0059;GM=NM_014240.2;GL=LIMD1;FG=missense;FD=unknown;AAC=PRO/THR;PP=219/677;CDP=655;GS=38;PH=0.004;CP=0.002;CG=-0.748;AA=C;DSP=754;GESP=C:13006;PAC=NP_055055.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:97,9,0 0/0:8:0,9,44</p>

      <p>3 57136585 rs17057718 C T 51.40 midCVG AC1=4;AF1=0.493;DB;DP=12;DP4=6,0,6,0;FQ=54.2;MQ=60;PV4=1,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=V301M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=123;DA=C/T;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=VAL/MET;PP=301/740,277/716,273/712,157/596;CDP=901,829,817,469;GS=21;PH=0.777;CP=0.215;CG=3.570;AA=C;HA=4.1;HE=17.7;HC=38.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;TFBS=V$GFI1_01;GESP=T:1623/C:11383;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/1:8:5,0,26 0/0:7:0,9,66 1/1:9:72,12,0 0/1:20:20,0,19</p>

      <p>3 57136593 rs368070640 A G 14.20 midCVG AC1=3;AF1=0.3818;DB;DP=12;DP4=6,0,4,0;FQ=16.2;MQ=60;PV4=1,0.044,1,1;SNPEFF_AMINO_ACID_CHANGE=I298T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=138;DA=A/G;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=298/740,274/716,270/712,154/596;CDP=893,821,809,461;GS=89;PH=0.986;CP=1.000;CG=5.480;AA=A;DSP=25;TFBS=V$GFI1_01;GESP=A:13006;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/0:7:0,6,34 0/0:9:0,9,61 1/1:5:45,9,0 0/1:14:14,0,19</p>

      <p>3 57335876 rs4060726 C T 103 midCVG AC1=5;AF1=0.6609;DB;DP=10;DP4=0,3,4,3;FQ=22.3;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G2904S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=54;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0001;DN=108;DA=C/T;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=GLY/SER;PP=2893/3093;CDP=8677;GS=56;PH=0.12;CP=0.988;CG=2.340;AA=C;HA=44.4;HE=41.5;HC=8.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT5A_02;GESP=T:2634/C:1932;PAC=NP_848599.3 GT:GQ:PL 0/1:22:17,0,32 0/1:4:0,3,30 1/1:4:29,3,0 1/1:14:101,15,0</p>

      <p>3 57414071 rs4681982 G A 107 highCVG AC1=4;AF1=0.4999;DB;DP=19;DP4=3,7,5,4;FQ=110;MQ=60;PV4=0.37,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=T1786I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0121;DN=111;DA=G/A;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=THR/ILE;PP=1763/3093;CDP=5288;GS=89;PH=0;CP=0.003;CG=-0.547;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6;GESP=G:4506;PAC=NP_848599.3 GT:GQ:PL 0/1:32:30,0,41 0/0:5:0,6,44 1/1:5:50,6,0 0/1:74:71,0,91</p>

      <p>3 66419956 rs13874 C T 145 highCVG AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA GT:GQ:PL 0/0:7:0,9,70 0/1:3:0,0,0 1/1:18:112,21,0 0/1:31:77,0,28</p>

      <p>3 67411103 rs902320 C T 117 highCVG AC1=5;AF1=0.6304;DB;DP=20;DP4=1,9,6,3;FQ=117;MQ=60;PV4=0.02,0.13,1,1;SNPEFF_AMINO_ACID_CHANGE=V425I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0045;DN=86;DA=C/T;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=VAL/ILE;PP=425/441;CDP=1273;GS=29;CP=0.002;CG=1.640;AA=C;HA=20.5;HE=27.2;HC=25.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=90;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:39:34,0,100 0/0:6:0,9,63 1/1:9:62,9,0 1/1:9:69,9,0</p>

      <p>3 67411166 rs902321 A G 144 highCVG AC1=5;AF1=0.6662;DB;DP=15;DP4=2,3,6,4;FQ=34.2;MQ=60;PV4=0.61,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Y404H;SNPEFF_CODON_CHANGE=Tat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SUCLG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001177599.1;VDB=0.0085;DN=86;DA=A/G;GM=NM_001177599.1;GL=SUCLG2;FG=missense;FD=missense;AAC=TYR/HIS;PP=404/441;CDP=1210;GS=83;CP=0.001;CG=3.220;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=27;KP=hsa00020:CitrateCycle(TCACycle)|hsa00640:PropanoateMetabolism|hsa01100:MetabolicPathways;PPI=USP19:926|DMWD:871;PAC=NP_001171070.1 GT:GQ:PL 0/1:33:42,0,33 0/0:3:0,6,44 1/1:9:70,9,0 1/1:12:80,12,0</p>

      <p>3 69245454 rs9310141 T C 6.32 lowCVG AC1=8;AF1=1;DB;DP=5;DP4=0,0,2,0;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T396A;SNPEFF_CODON_CHANGE=Acg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMD4B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015123.2;VDB=0.0175;DN=119;DA=T/C;GM=NM_015123.1,XM_005264720.1,XM_005264721.1,XM_005264722.1,XM_005264723.1;GL=FRMD4B;FG=missense;FD=unknown;AAC=THR/ALA;PP=396/1035,365/1004,363/1002,351/990,48/687;CDP=1186,1093,1087,1051,142;GS=58;PH=0;CP=0.803;CG=6.070;AA=C;HA=13.4;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;GESP=C:11705/T:393;PAC=NP_055938.1,XP_005264777.1,XP_005264778.1,XP_005264779.1,XP_005264780.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 1/1:4:25,3,0</p>

      <p>3 75786748 rs112332186 C T 54.70 midCVG AC1=6;AF1=0.7558;DB;DP=13;DP4=3,0,3,3;FQ=-23.2;MQ=32;PV4=0.46,0.25,1,1;SNPEFF_AMINO_ACID_CHANGE=D676N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF717;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128223.1;VDB=0.0067;DN=132;DA=C/T;GM=NM_001128223.1,XM_005264710.1,XM_005264711.1,XM_005264712.1,XM_005264713.1;GL=ZNF717;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=ASP/ASN;PP=676/915;CDP=2026;GS=23;PH=0;CP=0.000;CG=-3.140;AA=C;DV=by-frequency,by-cluster;DSP=1749;GESP=C:4354;PAC=NP_001121695.1,XP_005264767.1,XP_005264768.1,XP_005264769.1,XP_005264770.1 GT:GQ:PL 0/1:6:49,0,5 0/1:4:27,0,1 1/1:3:13,1,0 0/1:3:0,0,0</p>

      <p>3 98512540 rs28489284 G A 55.60 midCVG AC1=3;AF1=0.4126;DB;DP=12;DP4=5,1,1,4;FQ=57.2;MQ=60;PV4=0.08,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A364T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ST3GAL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271145.1;VDB=0.0070;DN=125;DA=G/A;GM=NM_001271142.1,NM_001271145.1,NM_001271146.1,NM_001271147.1,NM_001271148.1,NM_006100.3,XM_005247067.1,XM_005247068.1,XM_005247069.1,XM_005247070.1,XM_005247071.1,XM_005247072.1;GL=ST3GAL6;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=225/246,364/385,311/332,193/214,167/188,311/332,334/355,333/354,311/332,311/332,167/188,167/188;CDP=673,1090,931,577,499,931,1000,997,931,931,499,499;GS=58;PH=0.934;CP=1.000;CG=5.930;AA=G;HA=13.0;HE=2.9;HC=2.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa00601:GlycosphingolipidBiosynthesis-LactoAndNeolactoSeries|hsa01100:MetabolicPathways;GESP=A:682/G:12324;PAC=NP_001258071.1,NP_001258074.1,NP_001258075.1,NP_001258076.1,NP_001258077.1,NP_006091.1,XP_005247124.1,XP_005247125.1,XP_005247126.1,XP_005247127.1,XP_005247128.1,XP_005247129.1 GT:GQ:PL 0/0:4:0,3,25 0/0:9:0,9,53 1/1:5:69,9,0 0/1:30:31,0,30</p>

      <p>3 98600385 rs9838238 T C 36.30 midCVG AC1=3;AF1=0.3969;DB;DP=9;DP4=1,3,2,2;FQ=36.7;MQ=60;PV4=1,0.28,1,0.067;SNPEFF_AMINO_ACID_CHANGE=I144M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCBLD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080927.3;VDB=0.0156;DN=119;DA=T/C;GM=NM_080927.3,XM_005247118.1;GL=DCBLD2;FG=missense-near-splice;FD=missense,unknown;AAC=ILE/MET;PP=144/776,144/790;CDP=432;GS=10;PH=1;CP=1.000;CG=4.900;AA=T;HA=1.2;HE=2.3;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=C:437/T:11499;PAC=NP_563615.3,XP_005247175.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,3,22 1/1:7:77,12,0 0/0:9:0,9,58</p>

      <p>3 101283792 rs3762735 C G 93 highCVG AC1=4;AF1=0.4986;DB;DP=20;DP4=6,4,7,2;FQ=96;MQ=60;PV4=0.63,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=P56R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRMT10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017819.3;VDB=0.0174;DN=107;DA=C/G;GM=NM_017819.3;GL=TRMT10C;FG=missense;FD=missense;AAC=PRO/ARG;PP=56/404;CDP=167;GS=103;PH=0.001;CP=0.084;CG=5.160;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=179;GESP=G:1711/C:10225;PAC=NP_060289.2 GT:GQ:PL 0/0:15:0,18,120 0/1:29:26,0,47 1/1:15:98,18,0 0/1:20:17,0,29</p>

      <p>3 111901099 rs74840030 T C 15.40 midCVG AC1=3;AF1=0.3387;DB;DP=11;DP4=4,3,2,2;FQ=17;MQ=60;PV4=1,0.00097,1,0.41;SNPEFF_AMINO_ACID_CHANGE=M844V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC9C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_183061.1;VDB=0.0154;DN=132;DA=T/C;GM=NM_183061.1,XM_005247381.1;GL=SLC9C1;FG=missense;FD=missense,unknown;AAC=MET/VAL;PP=844/1178,796/1130;CDP=2530,2386;GS=21;PH=0.39;CP=1.000;CG=4.590;AA=T;DV=by-frequency,by-cluster;DSP=7;TFBS=V$HNF1_01;GESP=C:495/T:12501;PAC=NP_898884.1,XP_005247438.1 GT:GQ:PL 0/0:10:0,9,74 0/1:5:4,0,40 1/1:4:54,9,0 0/0:8:0,6,51</p>

      <p>3 121208833 rs3218649 G C 208 highCVG AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3 GT:GQ:PL 0/1:37:59,0,36 0/0:43:0,48,136 1/1:20:109,21,0 1/1:11:91,12,0</p>

      <p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

      <p>3 140406947 rs28594654 G A 110 highCVG AC1=4;AF1=0.5408;DB;DP=15;DP4=3,2,9,1;FQ=41;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V475M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM42;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152616.4;VDB=0.0045;DN=125;DA=G/A;GM=NM_152616.4,XM_005247397.1,XM_005247398.1;GL=TRIM42;FG=missense;FD=missense,unknown,unknown;AAC=VAL/MET;PP=475/724,475/600,328/453;CDP=1423,1423,982;GS=21;PH=0;CP=0.926;CG=1.790;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=66;GESP=A:6351/G:6655;PAC=NP_689829.3,XP_005247454.1,XP_005247455.1 GT:GQ:PL 0/1:54:96,0,51 0/1:3:0,0,0 1/1:5:53,6,0 0/0:3:0,3,26</p>

      <p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

      <p>3 145938619 rs3762685 T C 14 lowCVG AC1=4;AF1=0.4982;DB;DP=7;DP4=3,0,2,1;FQ=16.2;MQ=60;PV4=1,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=N34S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLSCR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128304.1;VDB=0.0043;DN=107;DA=T/C;GM=NM_001128304.1,NM_001128305.1,NM_001128306.1,NM_001177304.1,NM_020353.2,XM_005247654.1,XM_005247655.1;GL=PLSCR4;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ASN/SER;PP=34/330,34/330,34/240,19/225,34/330,34/330,34/330;CDP=101,101,101,56,101,101,101;GS=46;PH=0.001;CP=0.075;CG=-8.490;AA=T;HA=34.4;HE=37.5;HC=22.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18;GESP=C:4565/T:8441;PAC=NP_001121776.1,NP_001121777.1,NP_001121778.1,NP_001170775.1,NP_065086.2,XP_005247711.1,XP_005247712.1 GT:GQ:PL 0/0:6:0,6,39 0/0:4:0,3,26 1/1:4:38,6,0 0/1:4:19,3,0</p>

      <p>3 185990096 rs2193587 C T 115 midCVG AC1=5;AF1=0.651;DB;DP=13;DP4=1,3,1,7;FQ=49.8;MQ=60;PV4=1,0.2,1,0.13;SNPEFF_AMINO_ACID_CHANGE=R316K;SNPEFF_CODON_CHANGE=aGa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DGKG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001346.2;VDB=0.0103;DN=96;DA=C/T;GM=NM_001080744.1,NM_001080745.1,NM_001346.2,XM_005247156.1,XM_005247157.1,XM_005247158.1;GL=DGKG;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=316/767,316/753,316/792,316/768,258/734,316/733;CDP=947,947,947,947,773,947;GS=26;PH=0;CP=1.000;CG=3.920;AA=T;HA=11.8;HE=14.2;HC=29.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=37;KP=hsa00561:GlycerolipidMetabolism|hsa00564:GlycerophospholipidMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=T:10726/C:2280;PAC=NP_001074213.1,NP_001074214.1,NP_001337.2,XP_005247213.1,XP_005247214.1,XP_005247215.1 GT:GQ:PL 0/0:3:0,6,53 0/1:37:36,0,39 1/1:11:72,12,0 1/1:6:54,6,0</p>

      <p>3 190999917 rs6788319 C A 22.20 midCVG AC1=6;AF1=0.703;DB;DP=10;DP4=0,2,0,3;FQ=8.03;MQ=60;PV4=1,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=S21I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UTS2B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198152.3;VDB=0.0019;DN=116;DA=C/A;GM=NM_198152.3;GL=UTS2B;FG=missense;FD=missense;AAC=SER/ILE;PP=21/120;CDP=62;GS=142;PH=0;CP=0.000;CG=1.170;AA=T;HA=21.0;HE=39.2;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42;GESP=A:6408/C:6598;PAC=NP_937795.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:17:12,0,38</p>

      <p>3 193209178 rs6788448 T C 32.90 lowCVG AC1=3;AF1=0.3939;DB;DP=6;DP4=2,1,1,2;FQ=30.9;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=I181M;SNPEFF_CODON_CHANGE=atA/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP13A4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032279.2;VDB=0.0095;DN=116;DA=T/C;GM=NM_032279.2,XM_005247829.1;GL=ATP13A4;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=181/1197,181/1128;CDP=543;GS=10;PH=0.999;CP=1.000;CG=4.630;AA=T;HA=48.5;HE=41.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=C:5746/T:7260;PAC=NP_115655.2,XP_005247886.1 GT:GQ:PL 0/0:5:0,3,23 0/0:3:0,0,0 1/1:5:72,9,0 0/0:7:0,6,47</p>

      <p>3 194062519 rs3732477 C T 62.10 highCVG AC1=3;AF1=0.3624;DB;DP=16;DP4=0,8,1,5;FQ=64;MQ=60;PV4=0.43,0.0087,1,1;SNPEFF_AMINO_ACID_CHANGE=A305T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080513.3;VDB=0.0065;DN=107;DA=C/T;GM=NM_001080513.2,XM_005269280.1,XM_005269281.1;GL=CPN2;FG=missense;FD=missense,unknown,unknown;AAC=ALA/THR;PP=305/546;CDP=913;GS=58;PH=0.017;CP=0.000;CG=-10.600;AA=C;HA=44.2;HE=26.5;HC=32.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=913;GESP=T:4238/C:8768;PAC=NP_001073982.2,XP_005269337.1,XP_005269338.1 GT:GQ:PL 0/0:9:0,9,59 0/0:12:0,12,79 1/1:5:63,9,0 0/1:19:45,0,13</p>

      <p>3 195452799 rs2550232 C T 11.30 midCVG AC1=8;AF1=1;DB;DP=9;DP4=4,1,1,3;FQ=-26.1;MQ=21;PV4=0.21,0.34,1,0.42;SNPEFF_AMINO_ACID_CHANGE=T442I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282506.1;VDB=0.0102;DN=100;DA=C/T;GM=NM_001098516.1,NM_152673.2,XM_005269303.1,XM_005269304.1,XM_005269305.1,XM_005269306.1,XM_005269307.1,XM_005269308.1,XM_005269309.1;GL=MUC20;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ILE;PP=236/504,271/539,442/727,442/710,407/692,407/692,407/692,271/553,272/540;CDP=707,812,1325,1325,1220,1220,1220,812,815;GS=89;PH=0.002;CP=0.000;CG=-2.630;AA=C;DG;DV=by-cluster,by-2hit-2allele;DSP=131;GESP=C:12316;PAC=NP_001091986.1,NP_689886.2,XP_005269360.1,XP_005269361.1,XP_005269362.1,XP_005269363.1,XP_005269364.1,XP_005269365.1,XP_005269366.1 GT:GQ:PL 0/0:2:0,6,0/1:3:16,0,0 1/1:4:20,3,0 1/1:4:14,3,0</p>

      <p>4 1018705 rs4647930 C A 10.40 lowCVG AC1=3;AF1=0.3941;DB;DP=6;DP4=0,3,0,3;FQ=12.2;MQ=60;PV4=1,0.0081,1,1;SNPEFF_AMINO_ACID_CHANGE=P362Q;SNPEFF_CODON_CHANGE=cCa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFRL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004356.2;VDB=0.0000;DN=111;DA=C/A;GM=NM_001004356.2,NM_001004358.1,NM_021923.3,XM_005272281.1,XM_005272282.1;GL=FGFRL1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/GLN;PP=362/505;CDP=1085;GS=76;PH=0.105;CP=0.879;CG=3.860;HA=0.9;HE=27.0;HC=6.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;CPG=1017525-1018938;TFBS=V$CMYB_01,V$RREB1_01;GESP=A:2641/C:10337;PAC=NP_001004356.1,NP_001004358.1,NP_068742.2,XP_005272338.1,XP_005272339.1 GT:GQ:PL 0/0:8:0,6,45 0/0:3:0,0,0 1/1:4:49,9,0 0/0:5:0,3,22</p>

      <p>4 2451694 rs3108494 T C 127 midCVG AC1=6;AF1=0.6856;DB;DP=10;DP4=1,1,6,2;FQ=6.6;MQ=60;PV4=1,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=C253R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0035;DN=103;DA=T/C;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=CYS/ARG;PP=253/647;CDP=757;GS=180;CP=0.976;CG=3.340;AA=C;DG;DV=by-frequency,by-cluster,by-1000G,by-other-pop;DSP=34;PAC=NP_001180211.1 GT:GQ:PL 0/1:20:22,0,20 0/1:5:0,3,26 1/1:7:51,6,0 1/1:15:99,15,0</p>

      <p>4 2460571 rs73205493 C T 76.30 highCVG AC1=4;AF1=0.4854;DB;DP=17;DP4=11,0,4,2;FQ=78.6;MQ=60;PV4=0.11,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=A428V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0077;DN=130;DA=C/T;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=ALA/VAL;PP=428/647;CDP=1283;GS=64;CP=0.021;CG=0.034;AA=C;DV=by-cluster;DSP=21;PAC=NP_001180211.1 GT:GQ:PL 0/1:3:0,0,0 0/0:28:0,30,116 1/1:9:84,12,0 0/1:16:36,0,12</p>

      <p>4 3015553 rs1140085 G A 18.90 midCVG AC1=3;AF1=0.3786;DB;DP=12;DP4=2,7,3,0;FQ=21;MQ=60;PV4=0.045,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=V247I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182982.2;VDB=0.0058;DN=86;DA=G/A;GM=NM_001004056.1,NM_001004057.1,NM_182982.2,XM_005247957.1,XM_005247958.1,XM_005247959.1,XM_005247960.1,XM_005247961.1,XM_005247962.1,XM_005247963.1;GL=GRK4;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=215/547,247/533,247/579,247/533,215/501,247/487,247/441,53/385,36/368,247/332;CDP=643,739,739,739,643,739,739,157,106,739;GS=29;PH=0.918;CP=0.998;CG=4.760;AA=G;HA=0.0;HE=15.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=3;KP=hsa04062:ChemokineSignalingPathway|hsa04144:Endocytosis;GESP=A:1261/G:11745;PAC=NP_001004056.1,NP_001004057.1,NP_892027.2,XP_005248014.1,XP_005248015.1,XP_005248016.1,XP_005248017.1,XP_005248018.1,XP_005248019.1,XP_005248020.1 GT:GQ:PL 0/0:12:0,12,76 0/1:4:24,3,0 1/1:3:41,6,0 0/0:15:0,15,101</p>

      <p>4 8603105 rs115392829 G A 26.80 lowCVG AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA GT:GQ:PL 0/0:7:0,6,47 0/0:4:0,3,18 1/1:7:67,12,0 0/1:3:0,0,0</p>

      <p>4 15964485 rs758329 G A 64.30 midCVG AC1=5;AF1=0.6667;DB;DP=10;DP4=0,1,0,8;FQ=-6.93;MQ=60;PV4=1,0.17,1,0.3;SNPEFF_AMINO_ACID_CHANGE=P90S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031950.3;VDB=0.0006;DN=86;DA=G/A;GM=NM_031950.3;GL=FGFBP2;FG=missense;FD=missense;AAC=PRO/SER;PP=90/224;CDP=268;GS=74;PH=0.126;CP=0.160;CG=1.080;AA=G;HA=21.1;HE=46.8;HC=41.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=425;GESP=A:5041/G:7965;PAC=NP_114156.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:22:102,24,0 0/1:4:0,3,27</p>

      <p>4 25678199 rs6448389 A G 52.40 midCVG AC1=5;AF1=0.6499;DB;DP=10;DP4=3,0,4,1;FQ=20.1;MQ=60;PV4=1,0.13,1,0.13;SNPEFF_AMINO_ACID_CHANGE=D634G;SNPEFF_CODON_CHANGE=gAc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC34A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006424.2;VDB=0.0049;DN=116;DA=A/G;GM=NM_001177998.1,NM_001177999.1,NM_006424.2,XM_005248128.1,XM_005248129.1;GL=SLC34A2;FG=missense;FD=unknown;AAC=ASP/GLY;PP=633/690,633/690,634/691,634/691,634/691;CDP=1898,1898,1901,1901,1901;GS=94;PH=0;CP=0.978;CG=4.990;AA=G;HA=0.0;HE=13.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CTG)n;DSP=443;GESP=G:11393/A:1613;PAC=NP_001171469.1,NP_001171470.1,NP_006415.2,XP_005248185.1,XP_005248186.1 GT:GQ:PL 0/1:3:0,0,0 0/0:6:0,9,58 1/1:4:18,3,0 1/1:11:77,12,0</p>

      <p>4 38774889 rs4129009 T C 35.10 midCVG AC1=3;AF1=0.3337;DB;DP=9;DP4=1,3,3,1;FQ=36.8;MQ=60;PV4=0.49,0.0012,1,0.18;SNPEFF_AMINO_ACID_CHANGE=I775V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001017388.2;VDB=0.0028;DN=108;DA=T/A/C/G;GM=NM_001017388.2,NM_001195106.1,NM_001195107.1,NM_001195108.1,NM_030956.3,XM_005262671.1;GL=TLR10;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=ILE/VAL;PP=775/812,775/812,775/812,761/798,775/812,761/798;CDP=2323,2323,2323,2281,2323,2281;GS=29;PH=0;CP=0.010;CG=-5.340;AA=T;HE=20.1;HC=43.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2288;GESP=C:1621/T:11385;PAC=NP_001017388.1,NP_001182035.1,NP_001182036.1,NP_001182037.1,NP_112218.2,XP_005262728.1 GT:GQ:PL 0/0:8:0,6,48 0/0:5:0,3,26 1/1:6:77,12,0 0/0:5:0,3,26</p>

      <p>4 70807771 rs10030475 C T 36 midCVG AC1=4;AF1=0.4967;DB;DP=11;DP4=2,3,4,0;FQ=38.9;MQ=60;PV4=0.17,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A117V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CSN1S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001890.1;VDB=0.0071;DN=119;DA=C/T;GM=NM_001025104.1,NM_001890.1;GL=CSN1S1;FG=missense;FD=missense;AAC=ALA/VAL;PP=108/177,117/186;CDP=323,350;GS=64;PH=0;CP=0.000;CG=-4.500;AA=T;HA=32.5;HE=45.1;HC=25.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=8;GESP=T:4267/C:7325;PAC=NP_001020275.1,NP_001881.1 GT:GQ:PL 0/0:8:0,9,59 0/1:3:0,0,0 1/1:6:63,9,0 0/1:17:15,0,36</p>

      <p>4 71068489 rs3196714 T C 34.90 lowCVG AC1=5;AF1=0.5673;DB;DP=7;DP4=0,2,0,4;FQ=7.37;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=I222T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODAM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017855.3;VDB=0.0087;DN=105;DA=T/C;GM=NM_017855.3,XM_005265698.1;GL=ODAM;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=222/280,239/297;CDP=665,716;GS=89;PH=0.007;CP=0.000;CG=0.676;AA=T;HA=15.6;HE=26.6;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=C:3310/T:9690;PAC=NP_060325.3,XP_005265755.1 GT:GQ:PL 0/1:15:13,0,17 0/1:3:0,0,0 1/1:7:62,9,0 0/1:3:0,3,26</p>

      <p>4 71232388 rs10031844 G A 39.10 lowCVG AC1=5;AF1=0.5822;DB;DP=6;DP4=0,2,2,2;FQ=11.2;MQ=58;PV4=0.47,0.22,1,0.3;SNPEFF_AMINO_ACID_CHANGE=G28R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMR3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012390.3;VDB=0.0103;DN=119;DA=G/A;GM=NM_012390.3;GL=SMR3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=28/135;CDP=82;GS=125;PH=0.001;CP=0.000;CG=-5.140;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=28;GESP=A:3028/G:9978;PAC=NP_036522.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:77,12,0 0/0:5:0,6,45</p>

      <p>4 76878716 rs2242471 G C 19.60 lowCVG AC1=6;AF1=0.6681;DB;DP=8;DP4=1,1,4,0;FQ=-12.1;MQ=54;PV4=0.33,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=S575C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018115.3;VDB=0.0091;DN=98;DA=G/C;GM=NM_018115.2,XM_005263102.1,XM_005263103.1,XM_005263104.1,XM_005263105.1;GL=SDAD1;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=SER/CYS;PP=575/688,550/663,538/651,478/591,478/591;CDP=1724,1649,1613,1433,1433;GS=112;PH=0.941;CP=1.000;CG=5.150;AA=G;HA=27.9;HE=41.7;HC=6.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=46;TFBS=V$CREL_01,V$IK1_01,V$NFKAPPAB_01,V$NFKB_C,V$STAT1_03,V$STAT5A_02;GESP=C:4469/G:8537;PAC=NP_060585.2,XP_005263159.1,XP_005263160.1,XP_005263161.1,XP_005263162.1 GT:GQ:PL 0/1:20:40,0,18 0/1:3:0,0,0 1/1:3:14,3,0 0/1:3:0,0,0</p>

      <p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

      <p>4 77675505 rs3733242 C T 76.80 highCVG AC1=4;AF1=0.5211;DB;DP=17;DP4=3,5,1,6;FQ=76.9;MQ=60;PV4=0.57,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1290L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=0.0062;DN=107;DA=C/T;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=1290/1997,1290/1997,1114/1821,1114/1821,1114/1821;CDP=3869,3869,3341,3341,3341;GS=98;PH=0;CP=0.000;CG=-1.610;AA=T;HA=34.1;HE=40.1;HC=47.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=42;GESP=T:6227/C:6779;PAC=NP_065910.3,XP_005263219.1,XP_005263220.1,XP_005263221.1,XP_005263222.1 GT:GQ:PL 0/1:42:39,0,69 0/1:19:19,0,18 1/1:7:64,9,0 0/0:5:0,6,35</p>

      <p>4 88415636 rs1049544 G C 88.60 midCVG AC1=4;AF1=0.5004;DB;DP=12;DP4=2,3,5,2;FQ=51.6;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H106D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPARCL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128310.2;VDB=0.0073;DN=86;DA=G/C;GM=NM_001128310.1,NM_004684.4,XM_005263272.1,XM_005263273.1;GL=SPARCL1;FG=missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown;AAC=HIS/ASP,HIS/ASP,HIS/ASP,none;PP=106/665,106/665,106/665,NA;CDP=316,316,316,NA;GS=81,81,81,NA;PH=0;CP=0.015;CG=4.250;AA=C;HA=8.4;HE=33.5;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=115;GESP=C:6754/G:6252;PAC=NP_001121782.1,NP_004675.3,XP_005263329.1,NA GT:GQ:PL 0/0:7:0,9,68 0/1:15:50,0,12 1/1:7:74,9,0 0/1:12:10,0,14</p>

      <p>4 95170839 rs11722476 G A 30.20 highCVG AC1=3;AF1=0.351;DB;DP=19;DP4=5,7,0,5;FQ=32;MQ=60;PV4=0.24,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=S247N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0126;DN=120;DA=G/A;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none;PP=247/1029,247/1029,247/1027,NA;CDP=740,740,740,1066;GS=46,46,46,NA;PH=0;CP=0.987;CG=0.199;AA=G;HA=20.5;HE=45.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:4424/G:8574;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/0:10:0,9,68 0/1:8:7,0,75 1/1:7:69,12,0 0/0:15:0,15,90</p>

      <p>4 95173779 rs7439869 T C 130 midCVG AC1=3;AF1=0.4198;DB;DP=14;DP4=2,3,3,6;FQ=65;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=V301A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0162;DN=116;DA=T/C;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=VAL/ALA,VAL/ALA,VAL/ALA,none;PP=301/1029,301/1029,301/1027,NA;CDP=902,902,902,1228;GS=64,64,64,NA;PH=0;CP=0.012;CG=4.400;AA=C;HA=5.0;HE=41.6;HC=28.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=C:9055/T:3845;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/1:11:100,0,6 0/0:6:0,6,45 1/1:6:74,9,0 0/0:6:0,6,56</p>

      <p>4 100504664 rs3816873 T C 35.40 lowCVG AC1=6;AF1=0.695;DB;DP=7;DP4=1,1,3,2;FQ=10.9;MQ=60;PV4=1,0.018,1,0.15;SNPEFF_AMINO_ACID_CHANGE=I155T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTTP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300785.1;VDB=0.0031;DN=107;DA=T/C;GM=NM_000253.2,XM_005263025.1;GL=MTTP;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=128/895,155/922;CDP=383,464;GS=89;PH=0.007;CP=0.810;CG=4.100;AA=T;HA=22.5;HE=25.3;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=11;GESP=C:3415/T:9587;PPI=APOB:846;PAC=NP_000244.2,XP_005263082.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:11:61,12,0 1/1:4:16,3,0</p>

      <p>4 103611845 rs227368 C T 89.90 lowCVG AC1=6;AF1=0.7365;DB;DP=8;DP4=1,1,4,2;FQ=12.9;MQ=60;PV4=1,1,1,0.29;SNPEFF_AMINO_ACID_CHANGE=V253I;SNPEFF_CODON_CHANGE=Gta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MANBA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005908.3;VDB=0.0092;DN=79;DA=C/A/G/T;GM=NM_005908.3,XM_005262994.1;GL=MANBA;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=253/880,196/823;CDP=757,586;GS=29;PH=0.037;CP=0.007;CG=1.290;AA=C;HA=10.1;HE=46.2;HC=46.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=84;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=T:5010/C:7996;PAC=NP_005899.3,XP_005263051.1 GT:GQ:PL 0/0:3:0,6,50 0/1:3:0,0,0 1/1:9:70,9,0 1/1:9:63,9,0</p>

      <p>4 106859549 rs35132891 G C 52.10 midCVG AC1=5;AF1=0.5498;DB;DP=10;DP4=2,2,3,3;FQ=36.5;MQ=60;PV4=1,0.16,1,1;SNPEFF_AMINO_ACID_CHANGE=Q189H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NPNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184691.1;VDB=0.0030;DN=126;DA=G/C;GM=NM_001033047.2,NM_001184690.1,NM_001184691.1,NM_001184692.1,NM_001184693.1,XM_005262888.1,XM_005262889.1,XM_005262890.1;GL=NPNT;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown;AAC=GLN/HIS;PP=159/566,176/583,189/596,159/537,189/567,206/613,206/584,55/462;CDP=477,528,567,477,567,618,618,165;GS=24;PH=0.997;CP=0.992;CG=3.490;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=C:4607/G:8399;PAC=NP_001028219.1,NP_001171619.1,NP_001171620.1,NP_001171621.1,NP_001171622.1,XP_005262945.1,XP_005262946.1,XP_005262947.1 GT:GQ:PL 0/1:14:15,0,13 0/1:25:22,0,41 1/1:7:59,9,0 0/1:3:0,3,23</p>

      <p>4 110773067 rs4698797 G A 30.80 lowCVG AC1=5;AF1=0.6065;DB;DP=7;DP4=2,1,0,3;FQ=33.6;MQ=60;PV4=0.4,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=S175N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRIT3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198506.4;VDB=0.0062;DN=111;DA=G/A;GM=NM_198506.4,XM_005262979.1;GL=LRIT3;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=SER/ASN,none;PP=175/680,NA;CDP=524,NA;GS=46,NA;PH=0.964;CP=1.000;CG=3.780;AA=G;HA=43.9;HE=35.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;TFBS=V$STAT5B_01;GESP=A:2688/G:1878;PAC=NP_940908.3,NA GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,76 1/1:3:27,3,0 1/1:5:45,6,0</p>

      <p>4 128814964 rs17012739 G T 226 highCVG AC1=5;AF1=0.6047;DB;DP=33;DP4=7,4,9,12;FQ=139;MQ=60;PV4=0.46,0.29,1,0.095;SNPEFF_AMINO_ACID_CHANGE=E830D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014264.4;VDB=0.0063;DN=123;DA=G/T;GM=NM_001190799.1,NM_001190801.1,NM_014264.4,XM_005262701.1;GL=PLK4;FG=missense;FD=missense,missense,missense,unknown;AAC=GLU/ASP;PP=798/939,789/930,830/971,796/937;CDP=2394,2367,2490,2388;GS=45;PH=0;CP=0.000;CG=-8.440;AA=T;HA=22.5;HE=34.3;HC=43.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=73;GESP=T:9270/G:3736;PAC=NP_001177728.1,NP_001177730.1,NP_055079.3,XP_005262758.1 GT:GQ:PL 0/1:34:32,0,37 0/0:19:0,24,148 1/1:8:127,8,0 1/1:26:117,27,0</p>

      <p>4 128996129 . G T 28.60 midCVG AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1 GT:GQ:PL 0/0:8:0,6,51 0/0:6:0,3,17 1/1:6:71,12,0 0/0:8:0,6,45</p>

      <p>4 129793281 . A G 98.60 highCVG AC1=4;AF1=0.5296;DP=39;DP4=1,6,3,23;FQ=30.8;MQ=60;PV4=1,0.39,1,0.19;SNPEFF_AMINO_ACID_CHANGE=N798S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=JADE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287439.1;VDB=0.0093;GM=NM_199320.2,XM_005263229.1,XM_005263230.1,XM_005263231.1,XM_005263232.1,XM_005263233.1;GL=PHF17;FG=missense;FD=unknown;AAC=ASN/SER;PP=798/843,825/870,798/843,798/843,798/843,786/831;CDP=2393,2474,2393,2393,2393,2357;GS=46;PH=0;CP=0.014;CG=-6.450;AA=A;DSP=772;GESP=A:13006;PPI=KAT7:965|ING5:913|VHL:845|MEAF6:845|ING4:788;PAC=NP_955352.1,XP_005263286.1,XP_005263287.1,XP_005263288.1,XP_005263289.1,XP_005263290.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:23,3,0 1/1:7:120,9,0 0/0:5:0,6,46</p>

      <p>4 146653620 rs10008599 G A 23.70 midCVG AC1=3;AF1=0.3727;DB;DP=11;DP4=3,4,1,3;FQ=25.7;MQ=60;PV4=1,0.083,1,1;SNPEFF_AMINO_ACID_CHANGE=D173N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C4orf51;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080531.1;VDB=0.0141;DN=119;DA=G/A;GM=NM_001080531.1;GL=C4orf51;FG=missense;FD=missense;AAC=ASP/ASN;PP=173/203;CDP=517;GS=23;PH=0.059;CP=0.000;CG=-0.356;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=16;GESP=A:1483/G:10759;PAC=NP_001074000.1 GT:GQ:PL 0/0:9:0,9,71 0/1:12:15,0,8 1/1:5:54,9,0 0/0:9:0,9,74</p>

      <p>4 147741397 rs149464457 T A 23.60 midCVG AC1=2;AF1=0.3051;DB;DP=14;DP4=8,2,0,4;FQ=25.1;MQ=60;PV4=0.015,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=Q353H;SNPEFF_CODON_CHANGE=caA/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300761.1;VDB=0.0056;DN=134;DA=T/A;GM=NM_031956.2,XM_005263269.1,XM_005263270.1;GL=TTC29;FG=missense;FD=missense,unknown,unknown;AAC=GLN/HIS;PP=327/476,353/502,353/493;CDP=981,1059,1059;GS=24;PH=0.999;CP=1.000;CG=5.910;AA=T;DV=by-cluster;DSP=4;GESP=A:4/T:11824;PAC=NP_114162.2,XP_005263326.1,XP_005263327.1 GT:GQ:PL 0/0:16:0,15,85 0/0:3:0,0,0 1/1:6:66,12,0 0/0:16:0,15,87</p>

      <p>4 155287535 rs10017772 T C 73.60 midCVG AC1=5;AF1=0.5889;DB;DP=10;DP4=2,1,5,2;FQ=5.24;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=H174R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCHS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017639.3;VDB=0.0092;DN=119;DA=T/C;GM=NM_001142552.1,NM_017639.3;GL=DCHS2;FG=missense;FD=missense;AAC=HIS/ARG;PP=768/1370,174/2917;CDP=2303,521;GS=29;PH=0.007;CP=0.001;CG=2.100;AA=T;HA=36.4;HE=48.0;HC=45.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;TFBS=V$CART1_01;GESP=C:5817/T:7189;PAC=NP_001136024.1,NP_060109.2 GT:GQ:PL 0/1:7:54,0,3 0/1:3:0,0,0 1/1:7:60,9,0 0/0:4:0,6,37</p>

      <p>4 169611811 . C T 14.60 midCVG AC1=2;AF1=0.2475;DP=11;DP4=6,2,0,3;FQ=15.7;MQ=60;PV4=0.061,1,1,0.052;SNPEFF_AMINO_ACID_CHANGE=R465C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PALLD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001166108.1;VDB=0.0128;GM=NM_001166108.1,NM_001166109.1,NM_016081.3,XM_005262861.1,XM_005262862.1,XM_005262863.1,XM_005262864.1,XM_005262865.1,XM_005262866.1;GL=PALLD;FG=missense;FD=unknown;AAC=ARG/CYS;PP=465/1124,83/778,465/1107,465/1401,465/1384,465/1348,465/1310,465/1277,83/1019;CDP=1393,247,1393,1393,1393,1393,1393,1393,247;GS=180;PH=1;CP=0.789;CG=3.960;AA=C;DSP=58;TFBS=V$TCF11MAFG_01;GESP=C:13006;PPI=EZR:846|ACTN1:846;PAC=NP_001159580.1,NP_001159581.1,NP_057165.3,XP_005262918.1,XP_005262919.1,XP_005262920.1,XP_005262921.1,XP_005262922.1,XP_005262923.1 GT:GQ:PL 0/0:7:0,3,16 0/0:9:0,6,36 1/1:3:56,9,0 0/0:18:0,15,104</p>

      <p>4 169928842 rs2877380 G T 108 midCVG AC1=5;AF1=0.6024;DB;DP=14;DP4=2,2,6,2;FQ=49.6;MQ=60;PV4=0.55,0.03,1,1;SNPEFF_AMINO_ACID_CHANGE=L70M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CBR4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032783.4;VDB=0.0052;DN=101;DA=G/T;GM=NM_032783.4,XM_005263315.1,XM_005263316.1,XM_005263317.1;GL=CBR4;FG=missense;FD=missense,unknown,unknown,unknown;AAC=LEU/MET;PP=70/238,70/209,70/194,70/146;CDP=208;GS=15;PH=0.005;CP=1.000;CG=5.560;AA=T;HA=38.2;HE=21.8;HC=3.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=56;GESP=T:8677/G:4325;PAC=NP_116172.2,XP_005263372.1,XP_005263373.1,XP_005263374.1 GT:GQ:PL 0/1:18:51,0,16 0/0:6:0,9,76 1/1:11:74,12,0 1/1:4:29,3,0</p>

      <p>4 184190233 rs11941467 G T 54.10 highCVG AC1=3;AF1=0.395;DB;DP=17;DP4=5,8,1,3;FQ=56.3;MQ=60;PV4=1,1,1,0.073;SNPEFF_AMINO_ACID_CHANGE=A773S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WWC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024949.5;VDB=0.0099;DN=120;DA=G/T;GM=NM_024949.5;GL=WWC2;FG=missense;FD=missense;AAC=ALA/SER;PP=773/1193;CDP=2317;GS=99;PH=0.099;CP=1.000;CG=3.500;AA=G;HA=1.4;HE=10.4;HC=10.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;GESP=T:1111/G:11895;PAC=NP_079225.5 GT:GQ:PL 0/0:14:0,15,112 0/1:4:31,3,0 1/1:5:71,9,0 0/0:23:0,24,151</p>

      <p>4 184618936 rs62617790 G C 120 highCVG AC1=4;AF1=0.5361;DB;DP=15;DP4=1,3,4,7;FQ=46.7;MQ=60;PV4=1,0.0017,1,0.28;SNPEFF_AMINO_ACID_CHANGE=Q933H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021942.5;VDB=0.0163;DN=129;DA=G/C;GM=NM_021942.5,NM_199053.2;GL=TRAPPC11;FG=missense;FD=missense;AAC=GLN/HIS;PP=933/1134,933/1087;CDP=2799;GS=24;PH=0.999;CP=0.998;CG=4.670;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=53;GESP=C:1255/G:11751;PAC=NP_068761.4,NP_951008.1 GT:GQ:PL 0/0:7:0,9,65 0/1:3:0,3,25 1/1:27:149,30,0 1/1:3:18,3,0</p>

      <p>4 187627792 rs11939575 T C 10.40 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=0,2,3,0;FQ=9.79;MQ=60;PV4=0.1,0.00034,1,1;SNPEFF_AMINO_ACID_CHANGE=R1064G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0075;DN=120;DA=T/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLY;PP=1064/4589,1064/4601,1064/4601;CDP=3190;GS=125;PH=0;CP=0.635;CG=4.600;AA=C;HA=9.0;HE=23.5;HC=49.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=76;GESP=C:10198/T:2130;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1 GT:GQ:PL 0/0:6:0,6,45 0/1:3:0,0,0 1/1:5:47,9,0 0/1:3:0,0,0</p>

      <p>4 187629140 rs367863 G C 173 highCVG AC1=5;AF1=0.6658;DB;DP=17;DP4=2,2,6,6;FQ=22.5;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F614L;SNPEFF_CODON_CHANGE=ttC/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0127;DN=80;DA=G/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=PHE/LEU;PP=614/4589,614/4601,614/4601;CDP=1842;GS=22;PH=0;CP=0.966;CG=3.530;AA=C;HA=24.9;HE=20.1;HC=48.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1424;GESP=C:9311/G:2625;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1 GT:GQ:PL 0/1:10:64,0,9 0/0:5:0,9,57 1/1:12:76,12,0 1/1:12:81,12,0</p>

      <p>5 5462406 . C T 26.90 lowCVG AC1=3;AF1=0.3937;DP=6;DP4=3,0,0,3;FQ=24.3;MQ=60;PV4=0.1,1,1,4e-06;SNPEFF_AMINO_ACID_CHANGE=P987S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ICE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015325.2;VDB=0.0000;GM=NM_015325.2;GL=KIAA0947;FG=missense;FD=unknown;AAC=PRO/SER;PP=987/2267;CDP=2959;GS=74;PH=0.001;CP=0.000;CG=-5.280;DSP=1858;GESP=C:12622;PAC=NP_056140.1 GT:GQ:PL 0/0:7:0,6,38 0/0:3:0,0,0 1/1:5:66,9,0 0/0:5:0,3,25</p>

      <p>5 7897191 rs10380 C T 46.90 midCVG AC1=3;AF1=0.4271;DB;DP=11;DP4=3,4,3,1;FQ=49.4;MQ=60;PV4=0.55,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=H622Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTRR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024010.2;VDB=0.0059;DN=52;DA=C/T;GM=NM_002454.2,NM_024010.2,XM_005248304.1,XM_005248305.1;GL=MTRR;FG=missense;FD=missense,missense,unknown,unknown;AAC=HIS/TYR;PP=595/699,622/726,610/714,595/699;CDP=1783,1864,1828,1783;GS=83;PH=0.001;CP=0.109;CG=4.530;AA=C;HA=38.2;HE=17.3;HC=14.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=14;TFBS=V$P53_01;GESP=T:2251/C:10755;PAC=NP_002445.2,NP_076915.2,XP_005248361.1,XP_005248362.1 GT:GQ:PL 0/0:11:0,12,87 0/0:8:0,9,73 1/1:6:65,9,0 0/1:4:29,3,0</p>

      <p>5 32786389 rs2270915 A G 30.80 midCVG AC1=4;AF1=0.5475;DB;DP=9;DP4=1,4,2,2;FQ=33;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N522D;SNPEFF_CODON_CHANGE=Aac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NPR3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204375.1;VDB=0.0096;DN=100;DA=A/C/G/T;GM=NM_000908.3,NM_001204375.1,NM_001204376.1,XM_005248309.1;GL=NPR3;FG=missense;FD=missense,missense,missense,unknown;AAC=ASN/ASP;PP=521/541,522/542,305/325,306/326;CDP=1561,1564,913,916;GS=23;PH=0.001;CP=0.741;CG=5.000;AA=A;HA=8.1;HE=15.1;HC=23.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=50;GESP=G:2077/A:9753;PPI=NPPC:846|NPPB:846|NPPA:846;PAC=NP_000899.1,NP_001191304.1,NP_001191305.1,XP_005248366.1 GT:GQ:PL 0/0:5:0,6,41 0/1:3:21,3,0 1/1:5:51,6,0 0/1:5:1,0,37</p>

      <p>5 33951693 rs16891982 C G 118 midCVG AC1=6;AF1=0.667;DB;DP=9;DP4=0,1,4,4;FQ=-7.74;MQ=60;PV4=1,0.4,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L374F;SNPEFF_CODON_CHANGE=ttG/ttC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC45A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016180.3;VDB=0.0064;DN=123;DA=C/G;GM=NM_001012509.2,NM_016180.3,XM_005248311.1;GL=SLC45A2;FG=missense,missense,3-prime-UTR;FD=missense,missense,unknown;AAC=LEU/PHE,LEU/PHE,none;PP=374/461,374/531,NA;CDP=1122,1122,NA;GS=22,22,NA;PH=0.995;CP=1.000;CG=4.320;AA=C;HA=0.0;HE=1.7;HC=0.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=35;TFBS=V$MRF2_01;GESP=G:9063/C:3943;PAC=NP_001012527.1,NP_057264.3,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,1/1:22:156,24,0 0/1:4:0,3,26</p>

      <p>5 35709095 rs13170082 C T 25.90 lowCVG AC1=6;AF1=0.7423;DB;DP=7;DP4=0,1,1,3;FQ=-9.92;MQ=60;PV4=1,0.026,1,0.32;SNPEFF_AMINO_ACID_CHANGE=A904V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024867.3;VDB=0.0080;DN=121;DA=C/T;GM=NM_024867.3,XM_005248376.1,XM_005248377.1,XM_005248378.1;GL=SPEF2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/VAL;PP=904/1823,904/1824,899/1819,904/973;CDP=2711,2711,2696,2711;GS=64;PH=0.001;CP=0.001;CG=-2.830;AA=C;HA=33.6;HE=14.7;HC=17.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=46;GESP=T:9050/C:2912;PAC=NP_079143.3,XP_005248433.1,XP_005248434.1,XP_005248435.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:6:36,6,0 1/1:6:30,6,0</p>

      <p>5 36995903 . G A 18 lowCVG AC1=3;AF1=0.3939;DP=7;DP4=2,1,3,0;FQ=18.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E1101K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPBL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133433.3;VDB=0.0000;GM=NM_015384.4,NM_133433.3,XM_005248280.1,XM_005248281.1,XM_005248282.1,XM_005248283.1;GL=NIPBL;FG=missense;FD=unknown;AAC=GLU/LYS;PP=1101/2698,1101/2805,1101/2699,981/2685,881/2585,562/2266;CDP=3301,3301,3301,2941,2641,1684;GS=56;PH=0.61;CP=1.000;CG=5.430;AA=G;DSP=4;GESP=G:13006;PPI=CBX5:765;PAC=NP_056199.2,NP_597677.2,XP_005248337.1,XP_005248338.1,XP_005248339.1,XP_005248340.1 GT:GQ:PL 0/0:7:0,6,39 0/0:5:0,3,23 1/1:5:57,9,0 0/0:3:0,0,0</p>

      <p>5 42719239 rs6180 A C 98.30 midCVG AC1=6;AF1=0.705;DB;DP=10;DP4=0,3,0,7;FQ=25.8;MQ=60;PV4=1,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=I551L;SNPEFF_CODON_CHANGE=Atc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GHR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242399.2;VDB=0.0032;DN=52;DA=A/C;GM=NM_000163.4,NM_001242399.2,NM_001242400.2,NM_001242401.3,NM_001242402.2,NM_001242403.2,NM_001242404.2,NM_001242405.2,NM_001242406.2,NM_001242460.1,NM_001242462.1,XM_005248288.1;GL=GHR;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,3-prime-UTR,missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,utr-variant-3-prime,unknown;AAC=ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,none,ILE/LEU;PP=544/639,551/646,544/639,544/639,544/639,544/639,544/639,544/639,544/639,522/617,NA,522/617;CDP=1630,1651,1630,1630,1630,1630,1630,1630,1630,1564,NA,1564;GS=5,5,5,5,5,5,5,5,5,5,NA,5;PH=0.933;CP=0.951;CG=3.430;AA=A;HA=41.6;HE=46.2;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=685;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04630:Jak-STATSignalingPathway;TFBS=V$PAX4_04;GESP=C:5675/A:7331;PPI=CSHL1:998|SOCS2:937|JAK2:937|PTPN11:846|UBC:846|BTRC:846|NCOA6:845|SHC1:845|CISH:845;PAC=NP_000154.1,NP_001229328.1,NP_001229329.1,NP_001229330.1,NP_001229331.1,NP_001229332.1,NP_001229333.1,NP_001229334.1,NP_001229335.1,NP_001229389.1,NA,XP_005248345.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:12:83,12,0 1/1:9:60,9,0</p>

      <p>5 54439466 rs1021580 G A 260 highCVG AC1=6;AF1=0.805;DB;DP=19;DP4=1,2,4,12;FQ=-17.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R121C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170402.1;VDB=0.0082;DN=86;DA=G/A;GM=NM_001145734.2,NM_001170402.1,NM_152623.2,XM_005248450.1;GL=CDC20B;FG=missense;FD=missense,missense,missense,unknown;AAC=ARG/CYS;PP=121/478,121/520,121/516,121/373;CDP=361;GS=180;PH=0;CP=0.001;CG=-1.690;AA=A;HA=49.1;HE=18.6;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;GESP=A:9758/G:3248;PAC=NP_001139206.2,NP_001163873.1,NP_689836.2,XP_005248507.1 GT:GQ:PL 0/1:4:87,0,4 0/1:13:40,0,15 1/1:12:63,9,0 1/1:18:113,15,0</p>

      <p>5 56233515 . G A 20 lowCVG AC1=4;AF1=0.5539;DP=5;DP4=1,1,3,0;FQ=14.9;MQ=60;PV4=0.4,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=A114V;SNPEFF_CODON_CHANGE=gCa/gTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MIER3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001297598.1;VDB=0.0000;GM=NM_152622.3,XM_005248448.1,XM_005248449.1;GL=MIER3;FG=missense;FD=unknown;AAC=ALA/VAL;PP=109/550,109/551,46/488;CDP=326,326,137;GS=64;PH=1;CP=1.000;CG=6.070;AA=G;DSP=11;GESP=G:13006;PAC=NP_689835.3,XP_005248505.1,XP_005248506.1GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:57,9,0 0/0:6:0,6,49</p>

      <p>5 75003678 rs2307111 T C 211 highCVG AC1=6;AF1=0.7083;DB;DP=19;DP4=2,3,3,11;FQ=53.7;MQ=60;PV4=0.57,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=H36R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099271.1;VDB=0.0147;DN=100;DA=T/C;GM=NM_001099271.1,NM_152408.2,XM_005248435.1,XM_005248436.1,XM_005248437.1;GL=POC5;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ARG;PP=36/576,11/551,9/549,36/548,36/399;CDP=107,32,26,107,107;GS=29;PH=0;CP=0.937;CG=5.850;AA=C;HA=6.7;HE=38.0;HC=46.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=23;GESP=C:6453/T:5401;PAC=NP_001092741.1,NP_689621.2,XP_005248492.1,XP_005248493.1,XP_005248494.1 GT:GQ:PL 0/1:53:47,0,73 0/1:5:0,3,22 1/1:22:132,21,0 1/1:13:79,12,0</p>

      <p>5 78181477 rs1065757 C T 31.80 midCVG AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1 GT:GQ:PL 0/0:18:0,21,102 0/1:3:0,0,0 1/1:7:56,9,0 1/1:3:19,3,0</p>

      <p>5 79028472 rs4704585 C T 98.30 midCVG AC1=5;AF1=0.6256;DB;DP=13;DP4=2,2,8,0;FQ=34.6;MQ=60;PV4=0.091,1,0.25,1;SNPEFF_AMINO_ACID_CHANGE=A1295V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CMYA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153610.4;VDB=0.0069;DN=111;DA=C/T;GM=NM_153610.3;GL=CMYA5;FG=missense;FD=missense;AAC=ALA/VAL;PP=1295/4070;CDP=3884;GS=64;PH=0.004;CP=0.079;CG=-0.023;AA=T;HA=20.1;HE=42.9;HC=29.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3735;GESP=T:5852/C:6092;PAC=NP_705838.3 GT:GQ:PL 0/1:20:35,0,19 0/0:6:0,9,59 1/1:11:67,12,0 1/1:6:44,6,0</p>

      <p>5 86044932 rs1363339 G A 99.40 highCVG AC1=5;AF1=0.5944;DB;DP=17;DP4=3,6,3,5;FQ=103;MQ=60;PV4=1,0.33,1,1;VDB=0.0080;DN=88;DA=G/A;GM=none;FG=intergenic;FD=nc-transcript-variant;CP=0.004;CG=1.880;AA=A;HA=19.2;HE=29.2;HC=44.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=MLT1H;DSP=128448 GT:GQ:PL 0/1:37:33,0,52 0/1:20:41,0,18 1/1:6:42,6,0 0/1:31:27,0,88</p>

      <p>5 86695274 rs2230641 A G 50.70 midCVG AC1=4;AF1=0.4542;DB;DP=9;DP4=1,4,2,2;FQ=51.2;MQ=60;PV4=0.52,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V270A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCNH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001239.3;VDB=0.0061;DN=98;DA=A/G/T;GM=NM_001199189.1,NM_001239.3,XM_005248627.1,XM_005248628.1,XM_005248629.1;GL=CCNH;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=VAL/ALA;PP=217/271,270/324,270/330,270/326,217/277;CDP=650,809,809,809,650;GS=64;PH=0.965;CP=1.000;CG=5.680;AA=A;HA=5.0;HE=20.3;HC=8.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;KP=hsa03420:NucleotideExcisionRepair|hsa04110:CellCycle;GESP=G:2278/A:10728;PPI=CDK7:999|ERCC3:997|GTF2H1:989|CDK2:989|MNAT1:905|CDK8:846|ERCC2:845;PAC=NP_001186118.1,NP_001230.1,XP_005248684.1,XP_005248685.1,XP_005248686.1 GT:GQ:PL 0/0:11:0,12,76 0/1:3:0,0,0 1/1:4:48,6,0 0/1:22:44,0,18</p>

      <p>5 89988504 rs2366926 A G 62.10 midCVG AC1=5;AF1=0.6067;DB;DP=9;DP4=2,0,7,0;FQ=4.33;MQ=60;PV4=1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=N2345S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=32;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0009;DN=100;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2345/6307,NA,51/4013;CDP=7034,7047,152;GS=46,NA,46;PH=1;CP=1.000;CG=5.950;AA=A;HA=17.9;HE=29.9;HC=47.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=83;GESP=G:3478/A:8458;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:5:0,6,37 0/1:3:0,0,0 1/1:18:101,21,0 0/1:3:0,0,0</p>

      <p>5 89990324 rs1878878 A G 184 highCVG AC1=6;AF1=0.7366;DB;DP=16;DP4=1,3,8,4;FQ=36.4;MQ=60;PV4=0.26,1,1,0.041;SNPEFF_AMINO_ACID_CHANGE=N2584S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0007;DN=92;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2584/6307,NA,290/4013;CDP=7751,7764,869;GS=46,NA,46;PH=0;CP=0.995;CG=5.880;AA=G;HA=4.0;HE=40.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=195;GESP=G:8914/A:3276;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/1:25:18,0,41 0/1:34:51,0,35 1/1:16:97,15,0 1/1:11:63,9,0</p>

      <p>5 90012379 rs13157270 G A 132 highCVG AC1=4;AF1=0.4566;DB;DP=24;DP4=6,6,2,8;FQ=135;MQ=60;PV4=0.2,1,1,0.054;SNPEFF_AMINO_ACID_CHANGE=V3094I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0154;DN=121;DA=G/A;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=VAL/ILE,none,VAL/ILE;PP=3094/6307,NA,800/4013;CDP=9280,9293,2398;GS=29,NA,29;PH=0.01;CP=0.001;CG=-2.950;AA=G;HA=0.5;HE=9.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=96;GESP=A:774/G:11128;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:16:0,18,131 0/1:28:101,0,24 1/1:4:47,6,0 0/1:32:30,0,52</p>

      <p>5 95091201 rs34899 A G 48.40 highCVG AC1=3;AF1=0.4188;DB;DP=17;DP4=3,6,3,2;FQ=50.4;MQ=60;PV4=0.58,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N262D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RHOBTB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014899.3;VDB=0.0132;DN=76;DA=A/G;GM=NM_014899.3,XM_005271925.1;GL=RHOBTB3;FG=missense;FD=missense,unknown;AAC=ASN/ASP;PP=262/612,294/644;CDP=784,880;GS=23;PH=0;CP=0.013;CG=5.210;AA=G;HA=30.5;HE=41.1;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=102;GESP=G:5290/A:7716;PAC=NP_055714.3,XP_005271982.1 GT:GQ:PL 0/0:4:0,3,27 0/0:4:0,3,15 1/1:7:88,12,0 0/1:4:1,0,63</p>

      <p>5 111504479 rs7703522 C T 58.60 midCVG AC1=4;AF1=0.4913;DB;DP=11;DP4=0,5,2,3;FQ=36.6;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R630H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L4A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022140.3;VDB=0.0074;DN=116;DA=C/T;GM=NM_022140.3,XM_005272043.1,XM_005272044.1;GL=EPB41L4A;FG=missense;FD=unknown;AAC=ARG/HIS;PP=630/687,602/659,257/339;CDP=1889,1805,770;GS=29;PH=1;CP=0.728;CG=3.770;AA=C;HA=4.9;HE=35.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;GESP=T:8586/C:3412;PAC=NP_071423.3,XP_005272100.1,XP_005272101.1 GT:GQ:PL 0/0:10:0,12,66 0/1:3:0,0,0 1/1:6:68,9,0 0/1:14:33,0,10</p>

      <p>5 127685135 rs154001 C T 84.40 midCVG AC1=2;AF1=0.3246;DB;DP=13;DP4=3,3,3,4;FQ=86;MQ=60;PV4=1,0.09,1,1;SNPEFF_AMINO_ACID_CHANGE=V965I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001999.3;VDB=0.0006;DN=79;DA=C/T;GM=NM_001999.3;GL=FBN2;FG=missense;FD=missense;AAC=VAL/ILE;PP=965/2913;CDP=2893;GS=29;PH=0.118;CP=0.989;CG=3.320;AA=C;HA=40.8;HE=31.3;HC=11.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=30;TFBS=V$ATF_01,V$CEBP_C,V$MYOGNF1_01;GESP=T:8612/C:4394;PAC=NP_001990.2 GT:GQ:PL 0/0:10:0,9,72 0/0:5:0,3,23 1/1:15:129,21,0 0/0:7:0,6,52</p>

      <p>5 134782450 rs12520799 T A 123 midCVG AC1=4;AF1=0.5277;DB;DP=14;DP4=3,2,5,4;FQ=60;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=R117*;SNPEFF_CODON_CHANGE=Aga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5orf20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_130848.2;VDB=0.0030;DN=120;DA=T/A;GM=NM_130848.2;GL=C5orf20;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=117/245;CDP=349;CP=0.000;CG=-5.910;AA=A;HA=5.0;HE=48.3;HC=46.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3190;GESP=A:7784/T:5222;PAC=NP_570900.1 GT:GQ:PL 0/1:39:61,0,36 0/1:3:0,0,0 1/1:12:105,15,0 0/0:7:0,9,64</p>

      <p>FBN2 5 140023238 rs17208187 C G 65.40 midCVG AC1=5;AF1=0.6275;DB;DP=12;DP4=3,1,1,6;FQ=18.7;MQ=60;PV4=0.088,1,1,0.029;SNPEFF_AMINO_ACID_CHANGE=T305S;SNPEFF_CODON_CHANGE=aCc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMCO6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300980.1;VDB=0.0001;DN=123;DA=C/G;GM=NM_018502.3,XM_005268476.1,XM_005268477.1;GL=TMCO6;FG=missense;FD=missense,unknown,unknown;AAC=THR/SER;PP=299/494,305/500,300/495;CDP=896,914,899;GS=58;PH=0.001;CP=0.341;CG=2.370;AA=C;HA=7.1;HE=26.0;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=23;GESP=G:2322/C:9960;PAC=NP_060972.3,XP_005268533.1,XP_005268534.1 GT:GQ:PL 0/1:16:11,0,28 0/0:4:0,6,32 1/1:4:16,3,0 1/1:14:84,15,0</p>

      <p>5 140558212 rs2950845 A C 62.90 highCVG AC1=4;AF1=0.4824;DB;DP=28;DP4=10,2,10,2;FQ=52.5;MQ=26;PV4=1,1,0.16,1;SNPEFF_AMINO_ACID_CHANGE=K199N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019120.3;VDB=0.0196;DN=101;DA=A/C/T;GM=NM_019120.3;GL=PCDHB8;FG=missense;FD=missense;AAC=LYS/ASN;PP=199/802;CDP=597;GS=94;PH=0.988;CP=0.833;CG=4.250;AA=A;HA=0.8;HE=14.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6207;GESP=C:1504/A:11452;PAC=NP_061993.2 GT:GQ:PL 0/0:18:0,20,62 0/0:5:0,6,37 1/1:3:75,4,0 1/1:17:37,21,0</p>

      <p>5 140559596 rs2697541 T G 19.30 lowCVG AC1=3;AF1=0.352;DB;DP=7;DP4=3,1,0,3;FQ=21.1;MQ=56;PV4=0.14,0.29,0.42,0.14;SNPEFF_AMINO_ACID_CHANGE=L661V;SNPEFF_CODON_CHANGE=Ttg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019120.3;VDB=0.0059;DN=100;DA=T/G;GM=NM_019120.3;GL=PCDHB8;FG=missense;FD=missense;AAC=LEU/VAL;PP=661/802;CDP=1981;GS=32;PH=0;CP=1.000;CG=4.220;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4823;CPG=140558932-140559789;TFBS=V$TCF11MAFG_01;GESP=G:10886/T:1780;PAC=NP_061993.2 GT:GQ:PL 0/0:8:0,6,49 0/0:3:0,0,0 1/1:4:59,9,0 0/0:8:0,6,47</p>

      <p>5 140572763 rs2907323 C G 71.10 midCVG AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1 GT:GQ:PL 0/0:10:0,12,73 0/0:5:0,6,45 1/1:9:74,12,0 1/1:4:46,6,0</p>

      <p>5 145499996 rs10988 C T 67.30 midCVG AC1=5;AF1=0.6441;DB;DP=12;DP4=4,1,4,3;FQ=17.8;MQ=60;PV4=0.58,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=R1088K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020117.9;VDB=0.0086;DN=52;DA=C/T;GM=NM_020117.9,XM_005268458.1,XM_005268459.1,XM_005268460.1;GL=LARS;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=1088/1177,1061/1150,1042/1131,1034/1123;CDP=3263,3182,3125,3101;GS=26;PH=0.073;CP=1.000;CG=4.460;AA=C;HA=1.4;HE=29.1;HC=22.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa00290:Valine,LeucineAndIsoleucineBiosynthesis|hsa00970:Aminoacyl-tRNABiosynthesis;GESP=T:2696/C:10310;PAC=NP_064502.9,XP_005268515.1,XP_005268516.1,XP_005268517.1 GT:GQ:PL 0/1:13:39,0,11 0/1:3:0,0,0 1/1:8:63,9,0 0/1:10:5,0,42</p>

      <p>5 149006726 rs7732714 A C 48.70 midCVG AC1=4;AF1=0.5;DB;DP=10;DP4=1,4,2,3;FQ=50.5;MQ=60;PV4=1,0.047,1,0.025;SNPEFF_AMINO_ACID_CHANGE=S518R;SNPEFF_CODON_CHANGE=Agt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001669.2;VDB=0.0090;DN=116;DA=A/C;GM=NM_001001669.2,XM_005268447.1,XM_005268448.1;GL=ARHGEF37;FG=missense;FD=missense,unknown,unknown;AAC=SER/ARG;PP=518/676,518/676,304/462;CDP=1552,1552,910;GS=110;PH=0.906;CP=0.938;CG=3.930;AA=A;HA=32.4;HE=7.0;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=C:1584/A:11172;PAC=NP_001001669.2,XP_005268504.1,XP_005268505.1 GT:GQ:PL 0/0:5:0,6,47 0/1:3:0,0,0 1/1:4:44,6,0 0/1:46:45,0,47</p>

      <p>5 150886882 rs1105168 G A 61.20 lowCVG AC1=5;AF1=0.6338;DB;DP=7;DP4=1,1,3,2;FQ=3.28;MQ=60;PV4=1,0.44,1,1;SNPEFF_AMINO_ACID_CHANGE=P4117L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001447.2;VDB=0.0096;DN=86;DA=G/A;GM=NM_001447.2;GL=FAT2;FG=missense;FD=missense;AAC=PRO/LEU;PP=4117/4350;CDP=12350;GS=98;PH=0;CP=0.001;CG=0.264;AA=G;HA=7.2;HE=46.2;HC=16.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=168;GESP=A:8501/G:4505;PAC=NP_001438.1 GT:GQ:PL 0/1:4:0,3,18 0/1:22:43,0,21 1/1:4:22,3,0 1/1:6:39,6,0</p>

      <p>5 151771838 rs4958531 T C 25.70 midCVG AC1=3;AF1=0.3673;DB;DP=11;DP4=2,6,1,2;FQ=27.7;MQ=60;PV4=1,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=M388V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NMUR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020167.4;VDB=0.0057;DN=111;DA=T/C;GM=NM_020167.4;GL=NMUR2;FG=missense;FD=missense;AAC=MET/VAL;PP=388/416;CDP=1162;GS=21;PH=0;CP=0.000;CG=-7.170;AA=C;HA=0.0;HE=18.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=225;KP=hsa04080:NeuroactiveLigand-receptorInteraction;TFBS=V$TCF11_01;GESP=C:1791/T:11215;PPI=NMU:974;PAC=NP_064552.3 GT:GQ:PL 0/1:14:18,0,10 0/0:5:0,3,23 1/1:3:51,6,0 0/0:18:0,18,106</p>

      <p>5 176518082 . G T 10.60 midCVG AC1=3;AF1=0.3412;DP=13;DP4=3,3,0,3;FQ=12.1;MQ=60;PV4=0.46,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=E194*;SNPEFF_CODON_CHANGE=Gag/Tag;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGFR4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_002011.4;VDB=0.0002;GM=NM_002011.3,NM_022963.2,NM_213647.1,XM_005265837.1,XM_005265838.1,XM_005265839.1;GL=FGFR4;FG=stop-gained;FD=unknown;AAC=GLU/stop;PP=194/803,194/763,194/803,226/835,194/803,194/735;CDP=580,580,580,676,580,580;CP=1.000;CG=4.740;AA=G;DSP=24;KP=hsa04010:MAPKSignalingPathway|hsa04144:Endocytosis|hsa04810:RegulationOfActinCytoskeleton;GESP=G:13006;PPI=FGF5:999|FGF3:999|FGF9:999|FGF4:999|FGF1:937|FGF2:846;PAC=NP_002002.3,NP_075252.2,NP_998812.1,XP_005265894.1,XP_005265895.1,XP_005265896.1 GT:GQ:PL 0/0:17:0,15,78 0/0:6:0,3,25 1/1:4:50,9,0 0/0:3:0,0,0</p>

      <p>NMUR2 5 178540975 rs1054480 G A 49.80 lowCVG AC1=6;AF1=0.6833;DB;DP=6;DP4=2,0,1,3;FQ=-5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1177S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADAMTS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014244.4;VDB=0.0112;DN=86;DA=G/A;GM=NM_014244.4;GL=ADAMTS2;FG=missense;FD=missense;AAC=PRO/SER;PP=1177/1212;CDP=3529;GS=74;PH=0.651;CP=0.973;CG=5.050;AA=G;HA=2.7;HE=25.0;HC=32.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=351;GESP=A:2884/G:10122;PAC=NP_055059.2 GT:GQ:PL 0/0:4:0,6,32 0/1:3:0,0,0 1/1:4:26,3,0 1/1:8:65,9,0</p>

      <p>5 180030322 rs79620092 C T 69.90 lowCVG AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2 GT:GQ:PL 0/1:15:20,0,13 0/1:3:0,0,0 1/1:13:91,15,0 0/1:3:0,3,26</p>

      <p>5 180582604 rs2546423 A G 65.20 midCVG AC1=5;AF1=0.6414;DB;DP=10;DP4=3,0,2,4;FQ=12;MQ=60;PV4=0.17,0.39,1,0.2;SNPEFF_AMINO_ACID_CHANGE=H221R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2V2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206880.1;VDB=0.0064;DN=100;DA=A/G;GM=NM_206880.1;GL=OR2V2;FG=missense;FD=missense;AAC=HIS/ARG;PP=221/316;CDP=662;GS=29;PH=0;CP=0.001;CG=0.494;AA=G;HA=19.4;HE=41.3;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26085;KP=hsa04740:OlfactoryTransduction;GESP=G:8019/A:4987;PAC=NP_996763.1 GT:GQ:PL 0/1:11:8,0,13 0/0:4:0,6,40 1/1:6:43,6,0 1/1:8:60,9,0</p>

      <p>6 5431340 rs11243011 A G 68.10 highCVG AC1=4;AF1=0.5135;DB;DP=19;DP4=6,3,5,3;FQ=71.2;MQ=60;PV4=1,0.29,1,0.13;SNPEFF_AMINO_ACID_CHANGE=N280S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FARS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006567.3;VDB=0.0157;DN=120;DA=A/G;GM=NM_006567.3,XM_005248811.1,XM_005248812.1,XM_005248813.1;GL=FARS2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ASN/SER;PP=280/452,280/452,280/452,280/337;CDP=839;GS=46;PH=0.001;CP=1.000;CG=3.130;AA=A;HA=6.4;HE=15.1;HC=37.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;KP=hsa00970:Aminoacyl-tRNABiosynthesis;GESP=G:1979/A:11027;PAC=NP_006558.1,XP_005248868.1,XP_005248869.1,XP_005248870.1 GT:GQ:PL 0/1:3:0,0,0 0/0:10:0,12,63 1/1:15:94,18,0 0/1:21:18,0,80</p>

      <p>6 7211818 rs1334576 G A 14 lowCVG AC1=4;AF1=0.4604;DB;DP=7;DP4=0,4,2,1;FQ=15.2;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G195R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RREB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001003699.3;VDB=0.0098;DN=88;DA=G/A;GM=NM_001003698.3,NM_001003699.3,NM_001003700.1,NM_001168344.1,XM_005249272.1,XM_005249273.1,XM_005249274.1,XM_005249275.1,XM_005249276.1,XM_005249277.1;GL=RREB1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/ARG;PP=195/1688,195/1743,195/1477,195/1688,195/1743,195/1743,195/1743,195/1743,195/1743,195/1688;CDP=583;GS=125;PH=0.998;CP=0.039;CG=2.930;AA=G;HA=35.5;HE=45.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=A:4926/G:8080;PAC=NP_001003698.1,NP_001003699.1,NP_001003700.1,NP_001161816.1,XP_005249329.1,XP_005249330.1,XP_005249331.1,XP_005249332.1,XP_005249333.1,XP_005249334.1 GT:GQ:PL 0/1:4:21,3,0 0/0:6:0,6,32 1/1:4:37,6,0 0/0:6:0,6,30</p>

      <p>6 18171661 rs149734475 A C 67.40 midCVG AC1=4;AF1=0.4868;DB;DP=13;DP4=4,3,2,4;FQ=70.3;MQ=60;PV4=0.59,0.00091,1,0.47;SNPEFF_AMINO_ACID_CHANGE=Q162P;SNPEFF_CODON_CHANGE=cAg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KDM1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153042.3;VDB=0.0143;DN=134;DA=A/C;GM=NM_153042.3,XM_005248925.1,XM_005248926.1,XM_005248927.1,XM_005248928.1,XM_005248929.1;GL=KDM1B;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown;AAC=GLN/PRO;PP=162/591,162/824,162/823,132/794,162/679,162/456;CDP=485,485,485,395,485,485;GS=76;PH=0;CP=0.981;CG=4.240;AA=A;DV=by-frequency,by-cluster;DSP=50;TFBS=V$CEBPB_01;GESP=C:44/A:12962;PPI=BAP1:941;PAC=NP_694587.3,XP_005248982.1,XP_005248983.1,XP_005248984.1,XP_005248985.1,XP_005248986.1 GT:GQ:PL 0/1:21:32,0,17 0/1:3:0,0,0 1/1:9:79,12,0 0/0:16:0,18,127</p>

      <p>6 26184102 rs7766641 G A 188 highCVG AC1=5;AF1=0.626;DB;DP=19;DP4=1,2,12,4;FQ=6.53;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G27S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIST1H2BE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003523.2;VDB=0.0118;DN=116;DA=G/A;GM=NM_003523.2;GL=HIST1H2BE;FG=missense;FD=missense;AAC=GLY/SER;PP=27/127;CDP=79;GS=56;CP=0.986;CG=4.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=12900;KP=hsa05322:SystemicLupusErythematosus;CPG=26184037-26184336;GESP=A:2705/G:10301;PAC=NP_003514.2 GT:GQ:PL 0/0:4:0,6,35 0/1:4:0,3,14 1/1:38:168,39,0 1/1:8:68,9,0</p>

      <p>6 28228342 rs1679709 A G 23.10 lowCVG AC1=4;AF1=0.4699;DB;DP=7;DP4=3,1,2,1;FQ=25.8;MQ=60;PV4=1,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=E398G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NKAPL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007531.2;VDB=0.0074;DN=89;DA=A/G;GM=NM_001007531.2;GL=NKAPL;FG=missense;FD=missense;AAC=GLU/GLY;PP=398/403;CDP=1193;GS=98;PH=0.002;CP=1.000;CG=4.630;AA=G;HA=27.2;HE=13.4;HC=14.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1382;TFBS=V$IRF1_01,V$IRF2_01,V$TCF11_01;GESP=G:10661/A:2345;PAC=NP_001007532.1 GT:GQ:PL 0/1:19:19,0,19 0/0:8:0,9,64 1/1:4:45,6,0 0/1:3:0,0,0</p>

      <p>6 37614058 . G C 8.05 lowCVG AC1=8;AF1=1;DP=5;DP4=0,0,0,3;FQ=-28.7;MQ=60;SNPEFF_AMINO_ACID_CHANGE=P714A;SNPEFF_CODON_CHANGE=Ccc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MDGA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153487.3;VDB=0.0088;GM=NM_153487.3,XM_005249003.1;GL=MDGA1;FG=missense;FD=unknown;AAC=PRO/ALA;PP=714/956,714/948;CDP=2140;GS=27;PH=0.999;CP=0.998;CG=5.680;AA=G;DSP=85;GESP=G:12708;PAC=NP_705691.1,XP_005249060.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:14,3,0 1/1:7:27,6,0</p>

      <p>6 43011188 . G T 66.80 highCVG AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1 GT:GQ:PL 0/0:7:0,6,35 0/0:5:0,3,19 1/1:23:112,30,0 0/0:16:0,15,97</p>

      <p>6 52617731 rs2180314 C G 37.70 midCVG AC1=4;AF1=0.5406;DB;DP=9;DP4=4,0,3,1;FQ=23;MQ=60;PV4=1,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S112T;SNPEFF_CODON_CHANGE=aGt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTA2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000846.4;VDB=0.0058;DN=96;DA=C/G;GM=NM_000846.4;GL=GSTA2;FG=missense;FD=missense;AAC=SER/THR;PP=112/223;CDP=335;GS=58;PH=0;CP=0.000;CG=-4.510;AA=C;HA=22.9;HE=43.0;HC=33.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=G:6507/C:6499;PPI=GSTM2:937;PAC=NP_000837.3 GT:GQ:PL 0/0:3:0,3,27 0/1:3:0,0,1/1:5:48,6,0 0/1:30:29,0,32</p>

      <p>6 54173413 rs2297980 A G 54.70 midCVG AC1=5;AF1=0.5683;DB;DP=12;DP4=3,2,4,3;FQ=51.3;MQ=60;PV4=1,0.035,1,1;SNPEFF_AMINO_ACID_CHANGE=Q22R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0132;DN=100;DA=A/G;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=GLN/ARG;PP=22/477;CDP=65;GS=43;PH=0;CP=0.000;CG=-4.040;AA=A;HA=9.2;HE=10.5;HC=0.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=291;GESP=G:1365/A:11641;PAC=NP_055279.3 GT:GQ:PL 0/1:53:50,0,61 0/1:3:0,0,0 1/1:7:45,9,0 0/1:3:0,3,28</p>

      <p>6 54219326 rs3195579 G A 24.40 midCVG AC1=4;AF1=0.4386;DB;DP=11;DP4=6,1,2,1;FQ=26.9;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=R381H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0086;DN=105;DA=G/A/C;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=ARG/HIS;PP=381/477;CDP=1142;GS=29;PH=0;CP=0.594;CG=2.850;AA=G;HA=10.2;HE=17.7;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$PAX2_01;GESP=A:2158/G:10842;PAC=NP_055279.3 GT:GQ:PL 0/0:17:0,18,123 0/0:4:0,3,25 1/1:4:43,6,0 0/1:4:26,3,0</p>

      <p>6 57393125 rs927192 A G 133 highCVG AC1=5;AF1=0.603;DB;DP=38;DP4=8,8,5,14;FQ=137;MQ=60;PV4=0.18,0.005,1,1;SNPEFF_AMINO_ACID_CHANGE=S259G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRIM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000947.4;VDB=0.0076;DN=86;DA=A/G;GM=none;FG=intergenic;FD=nc-transcript-variant;CP=1.000;CG=5.860;AA=G;HA=0.8;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap;DSP=306133 GT:GQ:PL 0/1:42:38,0,54 0/1:57:52,0,70 1/1:7:67,8,0 0/1:25:20,0,89</p>

      <p>6 66005888 rs9342464 C T 35.50 highCVG AC1=4;AF1=0.47;DB;DP=19;DP4=2,11,2,3;FQ=38.2;MQ=60;PV4=0.53,0.3,1,0.074;SNPEFF_AMINO_ACID_CHANGE=G631S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0077;DN=119;DA=C/T;GM=NM_001142800.1;GL=EYS;FG=missense;FD=missense;AAC=GLY/SER;PP=631/3145;CDP=1891;GS=56;PH=0;CP=0.025;CG=1.710;AA=C;HA=23.3;HE=45.0;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=125;GESP=T:2272/C:2294;PAC=NP_001136272.1 GT:GQ:PL 0/0:10:0,12,80 0/1:11:9,0,78 1/1:6:62,9,0 0/1:12:10,0,69</p>

      <p>6 66053934 rs61753611 T G 82 midCVG AC1=4;AF1=0.5267;DB;DP=13;DP4=4,2,2,4;FQ=59.3;MQ=60;PV4=0.57,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K532N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0126;DN=129;DA=T/G;GM=NM_001142800.1,NM_001142801.1,NM_198283.1;GL=EYS;FG=missense;FD=missense;AAC=LYS/ASN;PP=532/3145,532/620,532/595;CDP=1596;GS=94;PH=0.246;CP=0.001;CG=-3.870;AA=T;DV=by-frequency,by-cluster;DSP=4;GESP=G:108/T:12896;PAC=NP_001136272.1,NP_001136273.1,NP_938024.1 GT:GQ:PL 0/0:12:0,15,90 0/0:3:0,3,14 1/1:9:87,12,0 1/1:5:43,6,0</p>

      <p>6 97063522 rs9373985 C G 49.80 highCVG AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1 GT:GQ:PL 0/0:9:0,9,63 0/1:27:26,0,61 1/1:13:72,18,0 0/0:35:0,36,176</p>

      <p>6 111628626 rs3204953 C T 26.80 midCVG AC1=4;AF1=0.4468;DB;DP=9;DP4=0,6,3,0;FQ=29.2;MQ=60;PV4=0.012,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V3064I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=32;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=REV3L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002912.4;VDB=0.0080;DN=105;DA=C/T;GM=NM_002912.3,XM_005267088.1,XM_005267089.1;GL=REV3L;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ILE;PP=3064/3131,2986/3053,2942/3009;CDP=9190,8956,8824;GS=29;PH=0.389;CP=0.976;CG=5.710;AA=C;HA=0.0;HE=13.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa01100:MetabolicPathways;GESP=T:1410/C:11596;PPI=MAD2L2:999|REV1:998;PAC=NP_002903.3,XP_005267145.1,XP_005267146.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:65,9,0 0/0:18:0,18,99</p>

      <p>6 117130704 rs2274911 G A 78.50 midCVG AC1=4;AF1=0.5298;DB;DP=12;DP4=2,3,1,6;FQ=48.8;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P91S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0097;DN=100;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=missense;FD=missense,unknown,unknown;AAC=PRO/SER;PP=91/927,91/856,91/752;CDP=271;GS=74;PH=0.002;CP=0.635;CG=-0.330;AA=A;HA=19.9;HE=29.9;HC=42.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=77;GESP=A:9803/G:3203;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1 GT:GQ:PL 0/0:5:0,6,45 0/1:3:0,0,0 1/1:7:60,9,0 0/1:45:60,0,42</p>

      <p>6 117150008 rs6907580 G A 108 highCVG AC1=4;AF1=0.4941;DB;DP=17;DP4=2,6,5,3;FQ=107;MQ=60;PV4=0.31,0.089,1,0.23;SNPEFF_AMINO_ACID_CHANGE=R57*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0127;DN=116;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=stop-gained;FD=stop-gained,unknown,unknown;AAC=ARG/stop;PP=57/927,57/856,57/752;CDP=169;CP=0.862;CG=-2.840;AA=G;HA=11.5;HE=1.3;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=26;GESP=A:908/G:12098;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1 GT:GQ:PL 0/1:15:35,0,12 0/1:21:39,0,18 1/1:9:81,12,0 0/0:15:0,18,123</p>

      <p>GPRC6A 6 118886961 rs3734382 G T 108 midCVG AC1=5;AF1=0.5972;DB;DP=14;DP4=1,3,6,2;FQ=43.4;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P254T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CEP85L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001178035.1;VDB=0.0162;DN=107;DA=G/T;GM=NM_001042475.2,NM_001178035.1,NM_206921.2,XM_005266970.1,XM_005266971.1;GL=CEP85L;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/THR;PP=251/806,254/809,251/497,149/704,149/704;CDP=751,760,751,445,445;GS=38;PH=0;CP=0.066;CG=-4.210;AA=T;HA=17.8;HE=26.0;HC=31.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=270;GESP=T:9623/G:3383;PAC=NP_001035940.1,NP_001171506.1,NP_996804.2,XP_005267027.1,XP_005267028.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:16:112,18,0 0/1:42:38,0,58</p>

      <p>6 152443744 rs2295190 G T 29 lowCVG AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:34,6,0 0/1:24:31,0,23</p>

      <p>6 155451352 rs931312 G A 154 highCVG AC1=5;AF1=0.6241;DB;DP=19;DP4=5,1,3,10;FQ=43.7;MQ=60;PV4=0.041,0.11,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R332H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TIAM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012454.3;VDB=0.0045;DN=86;DA=G/A;GM=NM_012454.3,XM_005266927.1;GL=TIAM2;FG=missense;FD=missense,unknown;AAC=ARG/HIS;PP=332/1702,332/1731;CDP=995;GS=29;PH=1;CP=0.427;CG=4.990;AA=G;HA=9.0;HE=27.2;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=200;KP=hsa04062:ChemokineSignalingPathway|hsa04810:RegulationOfActinCytoskeleton;GESP=A:2063/G:10943;PAC=NP_036586.2,XP_005266984.1 GT:GQ:PL 0/1:18:43,0,16 0/1:52:48,0,63 1/1:5:103,5,0 0/1:3:0,0,0</p>

      <p>7 2646796 rs61729413 G A 20 lowCVG AC1=4;AF1=0.5539;DB;DP=6;DP4=2,0,0,3;FQ=12.4;MQ=60;PV4=0.1,0.33,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R635K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IQCE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287499.1;VDB=0.0117;DN=129;DA=G/A;GM=NM_001100390.1,NM_152558.3,XM_005249675.1,XM_005249676.1,XM_005249677.1,XM_005249678.1;GL=IQCE;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=619/680,635/696,635/703,619/687,570/638,570/631;CDP=1856,1904,1904,1856,1709,1709;GS=26;PH=0;CP=0.000;CG=-3.560;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=A:1500/G:10556;PAC=NP_001093860.1,NP_689771.3,XP_005249732.1,XP_005249733.1,XP_005249734.1,XP_005249735.1 GT:GQ:PL 0/0:6:0,6,46 0/1:3:0,0,0 1/1:6:57,9,0 0/1:3:0,0,0</p>

      <p>7 4213975 rs671694 A G 129 highCVG AC1=5;AF1=0.6104;DB;DP=18;DP4=4,5,5,4;FQ=109;MQ=60;PV4=1,1,1,0.36;SNPEFF_AMINO_ACID_CHANGE=H1641R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152744.3;VDB=0.0088;DN=83;DA=A/G;GM=NM_001079653.1,NM_152744.3;GL=SDK1;FG=missense;FD=missense;AAC=HIS/ARG;PP=128/680,1641/2214;CDP=383,4922;GS=29;PH=0;CP=0.086;CG=4.910;AA=G;HA=0.0;HE=29.8;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=G:10231/A:2775;PAC=NP_001073121.1,NP_689957.3 GT:GQ:PL 0/1:37:32,0,78 0/0:8:0,12,76 1/1:6:47,6,0 1/1:14:99,15,0</p>

      <p>7 6656830 rs2243563 G A 41.40 lowCVG AC1=6;AF1=0.6978;DB;DP=7;DP4=0,2,3,1;FQ=-9.9;MQ=60;PV4=0.4,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=G8R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF853;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017560.1;VDB=0.0058;DN=100;DA=G/A;GM=NM_017560.1,XM_005249790.1;GL=ZNF853;FG=missense;FD=missense,unknown;AAC=GLY/ARG;PP=8/660,8/203;CDP=22;GS=125;PH=0.001;CP=0.000;CG=-7.700;AA=G;HA=14.0;HE=49.1;HC=26.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=A:2834/G:1732;PAC=NP_060030.1,XP_005249847.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:46,6,0 1/1:6:37,6,0</p>

      <p>7 6683454 rs2253869 A G 154 highCVG AC1=6;AF1=0.735;DB;DP=17;DP4=3,3,2,9;FQ=64.8;MQ=60;PV4=0.28,0.18,1,1;SNPEFF_AMINO_ACID_CHANGE=E156G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF316;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001278559.1;VDB=0.0121;DN=100;DA=A/G;GM=NM_001278559.1;GL=ZNF316;FG=missense-near-splice;FD=unknown;AAC=GLU/GLY;PP=156/1005;CDP=467;GS=98;CP=1.000;CG=3.130;AA=G;HA=22.1;HE=47.8;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2;PAC=NP_001265488.1 GT:GQ:PL 0/1:16:9,0,92 0/1:10:63,0,11 1/1:13:77,12,0 1/1:8:49,6,0</p>

      <p>7 13978809 rs9639168 T C 137 highCVG AC1=5;AF1=0.6375;DB;DP=18;DP4=3,1,5,8;FQ=21.4;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S100G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ETV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004956.4;VDB=0.0073;DN=119;DA=T/C;GM=NM_001163147.1,NM_001163148.1,NM_001163149.1,NM_001163150.1,NM_001163151.1,NM_001163152.1,NM_004956.4,XM_005249635.1,XM_005249636.1,XM_005249637.1,XM_005249638.1;GL=ETV1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/GLY;PP=100/455,82/460,82/460,60/438,42/420,60/375,100/478,100/478,100/478,60/438,42/420;CDP=298,244,244,178,124,178,298,298,298,178,124;GS=56;PH=0;CP=0.998;CG=4.570;AA=C;HA=11.6;HE=29.4;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=C:3493/T:8433;PPI=RFWD2:845;PAC=NP_001156619.1,NP_001156620.1,NP_001156621.1,NP_001156622.1,NP_001156623.1,NP_001156624.1,NP_004947.2,XP_005249692.1,XP_005249693.1,XP_005249694.1,XP_005249695.1 GT:GQ:PL 0/1:25:51,0,24 0/0:3:0,6,39 1/1:20:106,21,0 1/1:4:26,3,0</p>

      <p>7 20687604 rs61732039 A G 12.90 midCVG AC1=4;AF1=0.4542;DB;DP=10;DP4=3,4,1,2;FQ=15.2;MQ=60;PV4=1,1,1,0.00055;SNPEFF_AMINO_ACID_CHANGE=D370G;SNPEFF_CODON_CHANGE=gAt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163941.1;VDB=0.0080;DN=129;DA=A/G;GM=NM_001163941.1,NM_001163942.1,NM_001163993.1,NM_178559.5,XM_005249737.1;GL=ABCB5;FG=missense,5-prime-UTR,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,utr-variant-5-prime,utr-variant-5-prime,utr-variant-5-prime,unknown;AAC=ASP/GLY,none,none,none,none;PP=370/1258,NA,NA,NA,NA;CDP=1109,NA,NA,NA,NA;GS=94,NA,NA,NA,NA;PH=1;CP=1.000;CG=4.150;AA=A;DV=by-frequency,by-cluster;DSP=14;KP=hsa02010:ABCTransporters;TFBS=V$CDP_02;GESP=G:1051/A:9243;PAC=NP_001157413.1,NA,NA,NA,NA GT:GQ:PL 0/1:6:4,0,77 0/1:3:0,0,0 1/1:3:47,6,0 0/0:7:0,6,49</p>

      <p>7 21468916 . C T 57.50 lowCVG AC1=8;AF1=1;DP=8;DP4=0,1,0,6;FQ=-26.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P45S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SP4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003112.3;VDB=0.0235;GM=NM_003112.3,XM_005249828.1,XM_005249829.1;GL=SP4;FG=missense;FD=unknown;AAC=PRO/SER;PP=45/785,28/768,45/707;CDP=133,82,133;GS=74;PH=0.952;CP=1.000;CG=4.610;AA=C;DSP=10;GESP=C:13002;PAC=NP_003103.2,XP_005249885.1,XP_005249886.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:88,4,0 0/1:3:0,0,0</p>

      <p>7 23757162 rs6945306 G C 114 highCVG AC1=4;AF1=0.4594;DB;DP=23;DP4=2,7,8,5;FQ=71.5;MQ=60;PV4=0.099,1,0.21,1;SNPEFF_AMINO_ACID_CHANGE=Q71H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STK31;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031414.4;VDB=0.0004;DN=116;DA=G/C;GM=NM_001260504.1,NM_001260505.1,NM_031414.4,NM_032944.3,NR_048542.1,XM_005249810.1,XM_005249811.1;GL=STK31;FG=missense,missense,missense,missense,non-coding-exon,missense,missense;FD=missense,missense,missense,missense,nc-transcript-variant,unknown,unknown;AAC=GLN/HIS,GLN/HIS,GLN/HIS,GLN/HIS,none,GLN/HIS,GLN/HIS;PP=48/997,71/997,71/1020,48/997,NA,48/997,71/924;CDP=144,213,213,144,324,144,213;GS=24,24,24,24,NA,24,24;PH=0.001;CP=0.888;CG=2.950;AA=G;HA=13.6;HE=27.7;HC=32.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=37;GESP=C:3223/G:9783;PAC=NP_001247433.1,NP_001247434.1,NP_113602.2,NP_116562.2,NA,XP_005249867.1,XP_005249868.1 GT:GQ:PL 0/1:14:13,0,15 0/1:17:20,0,14 1/1:4:126,6,0 0/0:13:0,15,85</p>

      <p>7 34979905 rs1637696 C A 8.89 lowCVG AC1=4;AF1=0.4657;DB;DP=8;DP4=3,1,2,1;FQ=11.3;MQ=58;PV4=1,0.008,0.14,0.28;SNPEFF_AMINO_ACID_CHANGE=G502V;SNPEFF_CODON_CHANGE=gGt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DPY19L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015283.1;VDB=0.0085;DN=89;DA=C/A;GM=NM_015283.1,XM_005249679.1;GL=DPY19L1;FG=missense;FD=missense,unknown;AAC=GLY/VAL;PP=502/676,445/619;CDP=1505,1334;GS=109;PH=0.549;CP=1.000;CG=5.240;AA=C;HA=18.3;HE=30.0;HC=10.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:3398/C:8506;PAC=NP_056098.1,XP_005249736.1 GT:GQ:PL 0/1:4:16,3,0 0/0:4:0,3,24 1/1:3:36,6,0 0/0:9:0,9,58</p>

      <p>7 39649275 rs4723884 G A 50.20 lowCVG AC1=5;AF1=0.6456;DB;DP=6;DP4=1,0,1,3;FQ=-3.25;MQ=60;PV4=0.4,0.072,1,1;SNPEFF_AMINO_ACID_CHANGE=G104E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=YAE1D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282446.1;VDB=0.0224;DN=111;DA=G/A;GM=NR_024390.1,XM_005249815.1;GL=YAE1D1/LOC646999;FG=non-coding-exon,missense;FD=nc-transcript-variant,unknown;AAC=none,GLY/GLU;PP=NA,104/207;CDP=190,311;GS=NA,98;CP=0.000;CG=-1.570;AA=G;HA=46.9;HE=17.0;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=60;CPG=39649254-39649510;PAC=NA,XP_005249872.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:87,12,0 0/1:3:0,3,32</p>

      <p>7 44112873 rs3218660 G T 12.40 lowCVG AC1=6;AF1=0.7387;DB;DP=5;DP4=1,0,3,0;FQ=-22.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H439N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284330.1;VDB=0.0047;DN=106;DA=G/T;GM=NM_013284.2,XM_005249708.1,XM_005249709.1,XM_005249710.1,XM_005249711.1,XM_005249712.1,XM_005249713.1;GL=POLM;FG=3-prime-UTR,missense,missense,3-prime-UTR,missense,missense,missense;FD=utr-variant-3-prime,unknown,unknown,unknown,unknown,unknown,unknown;AAC=none,HIS/ASN,HIS/ASN,none,HIS/ASN,HIS/ASN,HIS/ASN;PP=NA,469/539,439/509,NA,379/449,255/325,255/325;CDP=NA,1405,1315,NA,1135,763,763;GS=NA,68,68,NA,68,68,68;PH=0;CP=0.000;CG=-3.990;AA=G;HA=34.5;HE=19.2;HC=13.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=104;KP=hsa03450:Non-homologousEnd-joining;GESP=T:3329/G:9677;PPI=PCNA:846;PAC=NA,XP_005249765.1,XP_005249766.1,NA,XP_005249768.1,XP_005249769.1,XP_005249770.1 GT:GQ:PL 0/1:9:29,0,7 0/1:3:0,0,0 1/1:4:17,3,0 0/1:3:0,0,0</p>

      <p>7 64452830 rs34639489 C T 86 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,4,2;FQ=5.73;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C192Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERV3-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001007253.3;VDB=0.0141;DN=126;DA=C/T;GM=NM_001007253.3;GL=ERV3-1;FG=missense;FD=missense;AAC=CYS/TYR;PP=192/605;CDP=575;GS=194;PH=0.991;CP=0.325;CG=0.109;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;RM=HERV3-int;DSP=963;GESP=T:5840/C:6396;PAC=NP_001007254.2 GT:GQ:PL 0/0:3:0,6,41 0/1:3:0,0,0 1/1:14:102,15,0 1/1:4:26,3,0</p>

      <p>7 72397435 . A G 6.99 lowCVG AC1=5;AF1=0.625;DP=5;DP4=0,1,0,3;FQ=-8.69;MQ=50;PV4=1,0.036,0.14,1;SNPEFF_AMINO_ACID_CHANGE=K54R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POM121;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001257190.2;VDB=0.0004;GM=NM_001257190.2,NM_172020.4,XM_005250726.1,XM_005250727.1,XM_005250728.1,XM_005250729.1,XM_005250730.1,XM_005250731.1,XM_005250732.1;GL=POM121;FG=missense;FD=unknown;AAC=LYS/ARG;PP=54/1000,54/985,319/1250,319/1191,54/985,54/985,54/985,54/985,54/985;CDP=161,161,956,956,161,161,161,161,161;GS=26;PH=0.018;CP=0.966;AA=-;DSP=67;GESP=A:13006;PAC=NP_001244119.1,NP_742017.1,XP_005250783.1,XP_005250784.1,XP_005250785.1,XP_005250786.1,XP_005250787.1,XP_005250788.1,XP_005250789.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,24 1/1:5:42,9,0 0/1:3:0,0,0</p>

      <p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

      <p>7 82453708 rs2522833 A C 74.20 lowCVG AC1=6;AF1=0.7286;DB;DP=8;DP4=1,1,1,5;FQ=15.9;MQ=60;PV4=0.46,0.24,1,0.2;SNPEFF_AMINO_ACID_CHANGE=S4814A;SNPEFF_CODON_CHANGE=Tct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCLO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033026.5;VDB=0.0027;DN=100;DA=A/C;GM=NM_014510.2,NM_033026.5;GL=PCLO;FG=missense;FD=missense;AAC=SER/ALA;PP=4814/4936,4814/5143;CDP=14440;GS=99;PH=0.013;CP=1.000;CG=5.720;AA=A;HA=3.2;HE=41.2;HC=27.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=25;GESP=C:3860/A:8250;PAC=NP_055325.2,NP_149015.2 GT:GQ:PL 0/0:3:0,6,53 0/1:3:0,0,0 1/1:7:49,6,0 1/1:12:68,12,0</p>

      <p>7 95041016 rs12026 G C 34.40 midCVG AC1=3;AF1=0.4042;DB;DP=11;DP4=3,3,2,3;FQ=36.7;MQ=60;PV4=1,0.035,1,0.43;SNPEFF_AMINO_ACID_CHANGE=A148G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PON2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000305.2;VDB=0.0045;DN=52;DA=G/C;GM=NM_000305.2,NM_001018161.1,XM_005250453.1,XM_005250454.1;GL=PON2;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=148/355,136/343,80/287,78/285;CDP=443,407,239,233;GS=60;PH=0.009;CP=1.000;CG=2.550;AA=G;HA=34.9;HE=24.2;HC=20.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=40;KP=hsa01100:MetabolicPathways;GESP=C:3203/G:9803;PAC=NP_000296.2,NP_001018171.1,XP_005250510.1,XP_005250511.1 GT:GQ:PL 0/0:4:0,3,26 0/0:6:0,6,41 1/1:3:33,6,0 0/1:45:44,0,55</p>

      <p>7 95822389 . G A 54.60 highCVG AC1=3;AF1=0.3723;DP=16;DP4=4,2,0,7;FQ=56.7;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P192L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001160210.1;VDB=0.0003;GM=NM_001160210.1,NM_014251.2,NR_027662.1;GL=SLC25A13;FG=missense,missense,non-coding-exon;FD=unknown;AAC=PRO/LEU,PRO/LEU,none;PP=192/677,192/676,NA;CDP=575,575,650;GS=98,98,NA;PH=0.243;CP=1.000;CG=4.100;AA=G;DSP=41;GESP=G:13006;PPI=BRE:923|BABAM1:871;PAC=NP_001153682.1,NP_055066.1,NA GT:GQ:PL 0/0:9:0,9,60 0/0:9:0,9,66 1/1:15:98,21,0 0/1:3:0,0,0</p>

      <p>7 99306685 rs2257401 C G 76 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:64:67,0,63</p>

      <p>7 99757612 rs3823646 G A 61.30 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=2,3,5,1;FQ=59.4;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A467V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAL3ST4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024637.4;VDB=0.0171;DN=107;DA=G/A;GM=NM_024637.4,XM_005250590.1;GL=GAL3ST4;FG=missense;FD=missense,unknown;AAC=ALA/VAL;PP=467/487,405/425;CDP=1400,1214;GS=64;PH=0.056;CP=1.000;CG=5.040;AA=G;HA=38.2;HE=41.3;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=971;GESP=A:6586/G:6420;PAC=NP_078913.3,XP_005250647.1 GT:GQ:PL 0/0:7:0,6,43 0/1:3:0,0,0 1/1:13:104,18,0 0/0:9:0,9,59</p>

      <p>7 101200727 rs17393069 G A 47.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=2,0,4,1;FQ=9.59;MQ=60;PV4=1,0.015,1,0.29;VDB=0.0078;DN=123;DA=G/A;GM=NM_001278563.1,NM_133457.3;GL=COL26A1;FG=missense;FD=unknown,missense;AAC=ALA/THR;PP=415/442,413/440;CDP=1242,1236;GS=58;PH=0.226;CP=0.199;CG=-0.864;AA=G;HA=45.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=77;GESP=A:4643/G:7635;PAC=NP_001265492.1,NP_597714.2 GT:GQ:PL 0/0:3:0,6,46 0/1:3:0,0,0 1/1:9:54,9,0 1/1:6:36,6,0</p>

      <p>7 107593989 rs20556 T C 80.60 midCVG AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1 GT:GQ:PL 0/1:17:41,0,14 0/1:3:0,0,0 1/1:13:82,15,0 0/0:4:0,6,52</p>

      <p>7 111580166 rs144867634 T C 90.70 highCVG AC1=3;AF1=0.3681;DB;DP=23;DP4=12,3,3,4;FQ=92.7;MQ=60;PV4=0.14,0.45,1,0.35;SNPEFF_AMINO_ACID_CHANGE=M326V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DOCK4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014705.3;VDB=0.0036;DN=134;DA=T/C;GM=NM_014705.3,XM_005250724.1;GL=DOCK4;FG=missense-near-splice;FD=missense,unknown;AAC=MET/VAL;PP=326/1967,326/2012;CDP=976;GS=21;PH=0.066;CP=1.000;CG=6.060;AA=T;DV=by-frequency,by-cluster;DSP=2;GESP=C:225/T:12057;PAC=NP_055520.3,XP_005250781.1 GT:GQ:PL 0/0:26:0,27,130 0/1:24:48,0,18 1/1:10:91,15,0 0/0:15:0,15,111</p>

      <p>7 113518434 rs1799999 C A 68 lowCVG AC1=6;AF1=0.7069;DB;DP=8;DP4=1,2,1,4;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D905Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002711.3;VDB=0.0058;DN=89;DA=C/A;GM=NM_002711.3,XM_005250473.1;GL=PPP1R3A;FG=missense;FD=missense,unknown;AAC=ASP/TYR;PP=905/1123,704/922;CDP=2713,2110;GS=160;PH=0.976;CP=0.997;CG=5.640;AA=C;HA=23.4;HE=6.3;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=1747;KP=hsa04910:InsulinSignalingPathway;GESP=A:1760/C:11244;PAC=NP_002702.2,XP_005250530.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:59,6,0 0/1:49:45,0,55</p>

      <p>7 120776097 rs41281692 C G 153 midCVG AC1=6;AF1=0.7444;DB;DP=14;DP4=2,1,4,7;FQ=14.1;MQ=60;PV4=0.54,0.2,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A551G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024913.4;VDB=0.0084;DN=127;DA=C/G;GM=NM_001105533.1,NM_024913.4,XM_005250604.1,XM_005250605.1;GL=CPED1;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=551/784,551/1027,331/807,331/564;CDP=1652,1652,992,992;GS=60;PH=0.403;CP=0.000;CG=-0.512;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4;GESP=G:4925/C:8057;PAC=NP_001099003.1,NP_079189.4,XP_005250661.1,XP_005250662.1 GT:GQ:PL 0/1:12:17,0,13 0/1:20:13,0,40 1/1:16:101,15,0 1/1:11:67,9,0</p>

      <p>7 122635024 rs860170 C T 171 highCVG AC1=5;AF1=0.609;DB;DP=24;DP4=3,8,2,10;FQ=110;MQ=60;PV4=0.64,1,1,0.064;SNPEFF_AMINO_ACID_CHANGE=R222H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TAS2R16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016945.2;VDB=0.0143;DN=86;DA=C/T;GM=NM_016945.2;GL=TAS2R16;FG=missense;FD=missense;AAC=ARG/HIS;PP=222/292;CDP=665;GS=29;PH=0;CP=0.138;CG=2.210;AA=T;HA=3.2;HE=27.6;HC=47.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=108972;KP=hsa04742:TasteTransduction;GESP=T:9806/C:3200;PAC=NP_058641.1 GT:GQ:PL 0/1:13:69,0,12 0/0:25:0,30,144 1/1:14:84,15,0 1/1:8:68,9,0</p>

      <p>7 129950740 rs2171492 G T 30.70 lowCVG AC1=4;AF1=0.5413;DB;DP=5;DP4=0,2,3,0;FQ=11.4;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G303C;SNPEFF_CODON_CHANGE=Ggc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPA4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016352.3;VDB=0.0110;DN=96;DA=G/T;GM=NM_001163446.1,NM_016352.3;GL=CPA4;FG=missense;FD=missense;AAC=GLY/CYS;PP=270/389,303/422;CDP=808,907;GS=159;PH=0.012;CP=0.992;CG=0.479;AA=T;HA=17.1;HE=34.6;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=87;GESP=T:4313/G:8693;PPI=LXN:959;PAC=NP_001156918.1,NP_057436.2 GT:GQ:PL 0/1:3:26,3,0 0/0:3:0,3,25 1/1:5:47,6,0 0/0:3:0,3,25</p>

      <p>7 133848257 rs17167553 G T 35.10 highCVG AC1=3;AF1=0.3866;DB;DP=16;DP4=5,6,0,4;FQ=37.2;MQ=60;PV4=0.23,0.31,1,0.17;SNPEFF_AMINO_ACID_CHANGE=D302Y;SNPEFF_CODON_CHANGE=Gac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRGUK;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144648.1;VDB=0.0117;DN=123;DA=G/T;GM=NM_144648.1;GL=LRGUK;FG=missense;FD=unknown;AAC=ASP/TYR;PP=302/826;CDP=904;GS=160;PH=0.902;CP=0.983;CG=3.770;AA=G;HA=13.5;HE=7.3;HC=32.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=36;GESP=T:1632/G:11374;PAC=NP_653249.1 GT:GQ:PL 0/0:23:0,24,157 0/0:4:0,3,23 1/1:5:63,9,0 0/1:18:17,0,43</p>

      <p>7 149419635 rs75701347 T G 36.70 lowCVG AC1=5;AF1=0.5703;DB;DP=6;DP4=1,1,0,4;FQ=7.23;MQ=60;PV4=0.33,0.44,1,1;SNPEFF_AMINO_ACID_CHANGE=S231A;SNPEFF_CODON_CHANGE=Tca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRBA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001290187.1;VDB=0.0121;DN=132;DA=T/G;GM=NM_032534.2,XM_005250063.1,XM_005250064.1,XM_005250065.1,XM_005250066.1,XM_005250067.1,XM_005250068.1;GL=KRBA1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=SER/ALA;PP=197/1031,250/1023,249/1022,248/1021,235/1008,234/1007,250/735;CDP=589,748,745,742,703,700,748;GS=99;PH=0.001;CP=0.000;CG=-7.850;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=10;GESP=G:1451/T:10543;PAC=NP_115923.2,XP_005250120.1,XP_005250121.1,XP_005250122.1,XP_005250123.1,XP_005250124.1,XP_005250125.1 GT:GQ:PL 0/1:3:0,3,25 0/1:3:0,3,21 1/1:3:25,3,0 1/1:7:55,9,0</p>

      <p>7 154738409 rs3501 T C 124 highCVG AC1=3;AF1=0.4278;DB;DP=16;DP4=4,2,4,5;FQ=84.2;MQ=60;PV4=0.61,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M1013V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAXIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007349.3;VDB=0.0178;DN=36;DA=T/C;GM=NM_007349.3,NR_024476.1,NR_024477.1,XM_005249536.1,XM_005249537.1,XM_005249538.1,XM_005249539.1,XM_005249540.1,XM_005249541.1;GL=PAXIP1/PAXIP1-AS2;FG=missense,non-coding-exon,non-coding-exon,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL,none,none,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL,MET/VAL;PP=1013/1070,NA,NA,1013/1075,979/1041,977/1039,979/1036,977/1034,966/1028;CDP=3037,1155,1108,3037,2935,2929,2935,2929,2896;GS=21,NA,NA,21,21,21,21,21,21;PH=0;CP=0.964;CG=0.184;AA=T;HA=6.2;HE=25.8;HC=23.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=21;GESP=C:2642/T:9720;PPI=TP53BP1:971|NCOA6:968|KDM6A:968|PAGR1:925|RBBP5:922|ASH2L:922|WDR5:922|KMT2C:825|RAD50:825|KMT2D:824;PAC=NP_031375.3,NA,NA,XP_005249593.1,XP_005249594.1,XP_005249595.1,XP_005249596.1,XP_005249597.1,XP_005249598.1 GT:GQ:PL 0/0:8:0,9,63 0/0:6:0,6,48 1/1:20:149,24,0 0/1:20:22,0,18</p>

      <p>8 10390452 rs4406360 C T 170 highCVG AC1=5;AF1=0.6627;DB;DP=16;DP4=4,0,5,7;FQ=10.7;MQ=60;PV4=0.088,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A212V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS55;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198464.3;VDB=0.0094;DN=111;DA=C/T;GM=NM_001197020.1,NM_198464.3,XM_005272372.1;GL=PRSS55;FG=missense;FD=missense,missense,unknown;AAC=ALA/VAL;PP=212/277,212/353,212/257;CDP=635;GS=64;PH=0.108;CP=0.642;CG=-4.080;AA=C;HA=18.5;HE=34.2;HC=40.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=37;GESP=T:4047/C:8959;PAC=NP_001183949.1,NP_940866.2,XP_005272429.1 GT:GQ:PL 0/1:25:88,0,24 0/1:4:0,3,26 1/1:4:26,3,0 1/1:15:100,15,0</p>

      <p>8 10468976 . G C 25.70 lowCVG AC1=5;AF1=0.6557;DP=5;DP4=2,0,2,1;FQ=8.94;MQ=60;PV4=1,0.4,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Q878E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RP1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178857.5;VDB=0.0119;GM=NM_178857.5;GL=RP1L1;FG=missense;FD=unknown;AAC=GLN/GLU;PP=878/2401;CDP=2632;GS=29;PH=0.005;CP=0.002;CG=2.580;AA=G;DSP=1881;GESP=G:10756;PAC=NP_849188.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,44 1/1:4:24,3,0 1/1:5:42,6,0</p>

      <p>8 12879539 rs608052 A G 159 highCVG AC1=5;AF1=0.5689;DB;DP=16;DP4=3,2,4,7;FQ=62.2;MQ=60;PV4=0.6,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R451G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1456;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020844.2;VDB=0.0150;DN=83;DA=A/G;GM=NM_001099677.1,NM_020844.2,XM_005273584.1,XM_005273585.1,XM_005273586.1,XM_005273587.1,XM_005273588.1,XM_005273589.1,XM_005273590.1,XM_005273591.1,XM_005273592.1,XM_005273593.1,XM_005273594.1,XM_005273595.1;GL=KIAA1456;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLY;PP=325/329,451/455,520/524,471/475,451/455,420/424,420/424,415/419,415/419,415/419,415/419,364/368,325/329,364/368;CDP=973,1351,1558,1411,1351,1258,1258,1243,1243,1243,1243,1090,973,1090;GS=125;PH=0;CP=0.034;CG=0.894;AA=G;HA=45.0;HE=18.9;HC=25.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1023;GESP=G:8818/A:3100;PAC=NP_001093147.1,NP_065895.2,XP_005273641.1,XP_005273642.1,XP_005273643.1,XP_005273644.1,XP_005273645.1,XP_005273646.1,XP_005273647.1,XP_005273648.1,XP_005273649.1,XP_005273650.1,XP_005273651.1,XP_005273652.1 GT:GQ:PL 0/0:9:0,12,85 0/1:3:0,3,23 1/1:8:69,9,0 1/1:22:140,24,0</p>

      <p>8 17796382 rs412750 A G 121 highCVG AC1=5;AF1=0.6136;DB;DP=19;DP4=4,1,8,3;FQ=7.28;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N159S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006197.3;VDB=0.0111;DN=80;DA=A/G;GM=NM_006197.3,XM_005273509.1,XM_005273510.1,XM_005273511.1,XM_005273512.1,XM_005273513.1,XM_005273514.1,XM_005273515.1,XM_005273516.1,XM_005273517.1,XM_005273518.1,XM_005273519.1,XM_005273520.1,XM_005273521.1,XM_005273522.1,XM_005273523.1,XM_005273524.1,XM_005273525.1;GL=PCM1;FG=missense;FD=unknown;AAC=ASN/SER;PP=159/2025,159/2088,159/2088,159/2088,159/2087,159/2084,159/2080,159/2076,159/2069,159/2064,126/2055,159/2049,159/2049,159/2017,159/2014,159/2010,159/2002,159/1536;CDP=476,476,476,476,476,476,476,476,476,476,377,476,476,476,476,476,476,476;GS=46;PH=0;CP=0.511;CG=4.850;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=G:9737/A:2227;PPI=BBS4:983|BBS1:775;PAC=NP_006188.3,XP_005273566.1,XP_005273567.1,XP_005273568.1,XP_005273569.1,XP_005273570.1,XP_005273571.1,XP_005273572.1,XP_005273573.1,XP_005273574.1,XP_005273575.1,XP_005273576.1,XP_005273577.1,XP_005273578.1,XP_005273579.1,XP_005273580.1,XP_005273581.1,XP_005273582.1 GT:GQ:PL 0/1:17:13,0,23 0/1:9:76,0,7 1/1:6:45,6,0 0/1:16:30,0,15</p>

      <p>8 32453358 rs3924999 G A 31.20 lowCVG AC1=6;AF1=0.7171;DB;DP=6;DP4=0,1,2,1;FQ=-15.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R38Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013956.3;VDB=0.0076;DN=108;DA=G/A;GM=NM_001159995.1,NM_001159999.1,NM_001160001.1,NM_001160002.1,NM_001160004.1,NM_001160005.1,NM_001160007.1,NM_001160008.1,NM_004495.3,NM_013956.3,NM_013957.3,NM_013958.3,NM_013960.3,NM_013962.2,NM_013964.3,XM_005273488.1;GL=NRG1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,unknown;AAC=ARG/GLN;PP=17/608,17/625,17/591,38/195,38/460,38/208,38/178,38/421,38/212,38/646,38/638,38/242,38/463,253/423,38/641,45/636;CDP=50,50,50,113,113,113,113,113,113,113,113,113,113,758,113,134;GS=43;PH=0.966;CP=0.998;CG=5.650;AA=G;HA=1.3;HE=43.2;HC=21.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=13;KP=hsa04012:ErbBSignalingPathway;GESP=A:3990/G:9016;PPI=ERBB3:989|ERBB4:983;PAC=NP_001153467.1,NP_001153471.1,NP_001153473.1,NP_001153474.1,NP_001153476.1,NP_001153477.1,NP_001153479.1,NP_001153480.1,NP_004486.2,NP_039250.2,NP_039251.2,NP_039252.2,NP_039254.1,NP_039256.2,NP_039258.1,XP_005273545.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,18 1/1:4:20,3,0 1/1:6:50,6,0</p>

      <p>8 39362859 rs62510500 C T 82.70 highCVG AC1=4;AF1=0.5274;DB;DP=16;DP4=5,4,5,2;FQ=86;MQ=60;PV4=0.63,0.077,1,1;VDB=0.0134;DN=129;DA=C/T;GM=NR_001569.3,NR_024106.2,NR_024107.2,NR_073423.1;GL=ADAM3A;FG=splice-donor;FD=splice-donor-variant;CP=0.704;CG=3.850;DG;DV=by-frequency,by-cluster,by-1000G;DSP=0 GT:GQ:PL 0/1:29:26,0,105 0/1:3:0,0,0 1/1:12:100,15,0 0/0:7:0,9,72</p>

      <p>8 48739305 rs4278157 G A 137 midCVG AC1=4;AF1=0.5579;DB;DP=14;DP4=2,2,4,6;FQ=14.3;MQ=60;PV4=1,1,1,1;VDB=0.0263;DN=111;DA=G/A;GM=NM_001081640.1,NM_006904.6;GL=PRKDC;FG=missense;FD=missense;AAC=ARG/CYS;PP=2899/4098,2899/4129;CDP=8694;GS=180;PH=0.966;CP=0.005;CG=4.670;AA=G;HA=0.0;HE=5.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=89;KP=hsa03450:Non-homologousEnd-joining|hsa04110:CellCycle;GESP=A:442/G:12052;PAC=NP_001075109.1,NP_008835.5 GT:GQ:PL 0/0:5:0,6,35 0/1:19:21,0,18 1/1:13:158,15,0 0/1:3:0,0,0</p>

      <p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

      <p>8 67380528 rs1060242 T C 74.90 highCVG AC1=5;AF1=0.62;DB;DP=19;DP4=3,3,4,4;FQ=64;MQ=60;PV4=1,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=C449R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADHFE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144650.2;VDB=0.0170;DN=86;DA=T/C;GM=NM_144650.2,XM_005251152.1,XM_005251153.1,XM_005251154.1,XM_005251155.1,XM_005251156.1,XM_005251157.1,XM_005251158.1,XM_005251159.1,XM_005251160.1,XM_005251161.1,XM_005251162.1,XM_005251163.1;GL=ADHFE1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=CYS/ARG;PP=449/468,430/449,401/420,401/420,401/420,401/420,401/420,379/398,401/420,219/238,219/238,219/238,219/238;CDP=1345,1288,1201,1201,1201,1201,1201,1135,1201,655,655,655,655;GS=180;PH=0;CP=0.818;CG=3.680;AA=C;HA=23.7;HE=42.6;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;GESP=C:7238/T:5768;PAC=NP_653251.2,XP_005251209.1,XP_005251210.1,XP_005251211.1,XP_005251212.1,XP_005251213.1,XP_005251214.1,XP_005251215.1,XP_005251216.1,XP_005251217.1,XP_005251218.1,XP_005251219.1,XP_005251220.1 GT:GQ:PL 0/0:8:0,12,80 0/1:21:17,0,29 1/1:8:35,9,0 1/1:11:72,12,0</p>

      <p>8 67592152 rs1057463 T C 68.70 highCVG AC1=5;AF1=0.6134;DB;DP=22;DP4=3,8,6,2;FQ=72.8;MQ=60;PV4=0.07,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=F148S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C8orf44;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019607.2;VDB=0.0098;DN=86;DA=T/C;GM=NM_001204173.1,NM_019607.2,XM_005251270.1,XM_005251271.1,XM_005251272.1;GL=C8orf44/C8orf44-SGK3;FG=5-prime-UTR,missense,missense,missense,missense;FD=utr-variant-5-prime,missense,unknown,unknown,unknown;AAC=none,PHE/SER,PHE/SER,PHE/SER,PHE/SER;PP=NA,148/160,148/160,148/160,148/160;CDP=NA,443,443,443,443;GS=NA,155,155,155,155;PH=0.713;CP=0.020;CG=2.840;AA=T;HA=4.9;HE=33.5;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=108;GESP=C:2939/T:10067;PPI=GSK3B:869;PAC=NA,NP_062553.1,XP_005251327.1,XP_005251328.1,XP_005251329.1 GT:GQ:PL 0/1:32:27,0,82 0/0:6:0,9,55 1/1:6:35,6,0 1/1:8:54,9,0</p>

      <p>8 67796120 rs17332410 C G 17.80 midCVG AC1=3;AF1=0.3679;DB;DP=14;DP4=6,5,2,1;FQ=19.7;MQ=60;PV4=1,0.028,1,1;SNPEFF_AMINO_ACID_CHANGE=L322V;SNPEFF_CODON_CHANGE=Ctc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCMDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173518.4;VDB=0.0061;DN=123;DA=C/G;GM=NM_001136160.1,NM_001136161.1,NM_173518.4,XM_005251173.1,XM_005251174.1;GL=MCMDC2;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=LEU/VAL;PP=322/633,322/591,322/682,322/682,259/619;CDP=964,964,964,964,775;GS=32;PH=0.946;CP=1.000;CG=4.680;AA=C;HA=0.0;HE=6.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=110;TFBS=V$CEBPA_01;GESP=G:735/C:12271;PAC=NP_001129632.1,NP_001129633.1,NP_775789.3,XP_005251230.1,XP_005251231.1 GT:GQ:PL 0/0:15:0,15,96 0/1:5:20,3,0 1/1:3:44,6,0 0/0:18:0,18,122</p>

      <p>8 97614625 rs3816208 G A 21 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1 GT:GQ:PL 0/0:9:0,6,45 0/0:18:0,15,111 1/1:3:63,9,0 0/0:9:0,6,47</p>

      <p>8 103573006 rs2916569 G A 17.80 midCVG AC1=4;AF1=0.504;DB;DP=10;DP4=2,1,4,0;FQ=4.62;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S216N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024410.3;VDB=0.0051;DN=101;DA=G/A;GM=NM_024410.3;GL=ODF1;FG=missense;FD=missense;AAC=SER/ASN;PP=216/251;CDP=647;GS=46;PH=0;CP=0.053;CG=-7.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;RT=TGCAGCCCC;DSP=327;GESP=G:13006;PAC=NP_077721.2 GT:GQ:PL 0/0:5:0,6,25 0/1:3:0,0,0 1/1:4:34,6,0 0/1:18:24,0,15</p>

      <p>8 104412688 rs141856398 T C 58.40 highCVG AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA GT:GQ:PL 0/0:4:0,3,24 0/0:6:0,6,40 1/1:3:38,6,0 0/1:64:63,0,86</p>

      <p>8 110457011 rs72687015 A G 85.70 midCVG AC1=5;AF1=0.6401;DB;DP=10;DP4=3,1,4,2;FQ=43.1;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=Y1638C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKHD1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_177531.4;VDB=0.0097;DN=130;DA=A/G;GM=NM_177531.4;GL=PKHD1L1;FG=missense;FD=missense;AAC=TYR/CYS;PP=1638/4244;CDP=4913;GS=194;PH=0.951;CP=1.000;CG=3.250;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=122;GESP=G:2556/A:9592;PAC=NP_803875.2 GT:GQ:PL 0/1:21:16,0,37 0/0:4:0,6,48 1/1:9:70,9,0 1/1:6:46,6,0</p>

      <p>8 124449466 rs3824250 C T 38.30 midCVG AC1=4;AF1=0.4784;DB;DP=10;DP4=5,1,1,3;FQ=41.1;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R134C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDYHV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018024.2;VDB=0.0061;DN=107;DA=C/T;GM=NM_018024.1,XM_005250997.1,XM_005250998.1;GL=WDYHV1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/CYS;PP=134/206,74/146,66/138;CDP=400,220,196;GS=180;PH=0.99;CP=1.000;CG=5.480;AA=C;HA=34.8;HE=28.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;TFBS=V$AHRARNT_02,V$TCF11MAFG_01;GESP=T:4627/C:8379;PAC=NP_060494.1,XP_005251054.1,XP_005251055.1 GT:GQ:PL 0/1:16:14,0,41 0/0:7:0,9,51 1/1:4:47,6,0 0/1:17:22,0,14</p>

      <p>8 141468645 . G A 29.60 midCVG AC1=3;AF1=0.3656;DP=12;DP4=6,0,5,0;FQ=31.5;MQ=60;PV4=1,0.034,1,1.3e-09;SNPEFF_AMINO_ACID_CHANGE=Q7*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC9;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_031466.6;VDB=0.0000;GM=NM_031466.5;GL=TRAPPC9;FG=stop-gained;FD=unknown;AAC=GLN/stop;PP=7/1247;CDP=19;CP=0.000;CG=-2.520;AA=G;DSP=266;GESP=G:12786;PAC=NP_113654.4 GT:GQ:PL 0/0:3:0,0,0 0/0:7:0,6,51 1/1:9:72,15,0 0/0:12:0,12,70</p>

      <p>WDYHV1 8 142367335 rs10875472 T C 48.60 lowCVG AC1=6;AF1=0.6934;DB;DP=6;DP4=1,0,3,2;FQ=-18.7;MQ=60;PV4=1,0.17,1,0.27;SNPEFF_AMINO_ACID_CHANGE=H230R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005293.2;VDB=0.0059;DN=120;DA=T/C;GM=NM_005293.2;GL=GPR20;FG=missense;FD=missense;AAC=HIS/ARG;PP=230/359;CDP=689;GS=29;PH=0;CP=0.989;CG=-0.037;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=713;CPG=142367051-142367860;GESP=C:10416/T:2188;PAC=NP_005284.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:21,3,0 0/1:13:62,0,11</p>

      <p>8 142517222 rs2748421 A G 92.90 midCVG AC1=5;AF1=0.6549;DB;DP=13;DP4=3,2,8,0;FQ=41.5;MQ=59;PV4=0.13,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=Y10H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0027;DN=100;DA=A/G;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/HIS,none,none;PP=10/1319,NA,NA;CDP=28,109,109;GS=83,NA,NA;CP=0.000;CG=1.370;AA=G;HA=20.8;HE=49.1;HC=36.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=G:4995/A:7673;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:28:52,0,27 1/1:11:68,12,0 0/1:19:14,0,51</p>

      <p>8 142517227 rs2613648 C T 79.90 midCVG AC1=5;AF1=0.6558;DB;DP=13;DP4=3,2,8,0;FQ=19.6;MQ=59;PV4=0.13,1,0.23,0.41;SNPEFF_AMINO_ACID_CHANGE=R8K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0030;DN=100;DA=C/T;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ARG/LYS,none,none;PP=8/1319,NA,NA;CDP=23,104,104;GS=26,NA,NA;CP=0.000;CG=-2.700;AA=C;HA=21.0;HE=49.1;HC=36.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=58;GESP=T:4996/C:7690;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:21:47,0,20 1/1:11:61,12,0 0/1:18:13,0,36</p>

      <p>8 144801593 rs60732298 C A 16.80 lowCVG AC1=5;AF1=0.6552;DB;DP=5;DP4=2,0,3,0;FQ=7.18;MQ=60;PV4=1,0.027,1,1;SNPEFF_AMINO_ACID_CHANGE=T221K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAPK15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139021.2;VDB=0.0066;DN=129;DA=C/A;GM=NM_139021.2;GL=MAPK15;FG=missense;FD=missense;AAC=THR/LYS;PP=221/545;CDP=662;GS=78;PH=0.992;CP=0.104;CG=4.140;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;GESP=A:1162/C:11844;PAC=NP_620590.2 GT:GQ:PL 0/0:4:0,6,42 0/1:3:0,0,0 1/1:3:20,3,0 1/1:5:37,6,0</p>

      <p>9 742388 rs140893133 G A 40.80 midCVG AC1=2;AF1=0.3159;DB;DP=13;DP4=7,1,3,1;FQ=42.4;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=G1294S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KANK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256876.1;VDB=0.0053;DN=134;DA=G/A;GM=NM_001256876.1,NM_001256877.1,NM_015158.3,NM_153186.4,XM_005251410.1,XM_005251411.1,XM_005251412.1,XM_005251413.1,XM_005251414.1,XM_005251415.1,XM_005251416.1,XM_005251417.1,XM_005251418.1,XM_005251419.1;GL=KANK1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/SER;PP=1294/1353,1294/1353,1294/1353,1136/1195,1294/1365,1294/1365,1294/1365,1214/1285,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207;CDP=3880,3880,3880,3406,3880,3880,3880,3640,3406,3406,3406,3406,3406,3406;GS=56;PH=0.073;CP=0.955;CG=5.330;AA=G;DV=by-frequency,by-cluster;DSP=18;TFBS=V$CHOP_01;GESP=A:16/G:12990;PPI=YWHAQ:846;PAC=NP_001243805.1,NP_001243806.1,NP_055973.2,NP_694856.1,XP_005251467.1,XP_005251468.1,XP_005251469.1,XP_005251470.1,XP_005251471.1,XP_005251472.1,XP_005251473.1,XP_005251474.1,XP_005251475.1,XP_005251476.1 GT:GQ:PL 0/0:13:0,12,85 0/0:3:0,0,0 1/1:6:83,12,0 0/0:13:0,12,70</p>

      <p>9 14722477 rs3747532 G C 33 midCVG AC1=5;AF1=0.5574;DB;DP=10;DP4=2,3,3,0;FQ=36.1;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A65G;SNPEFF_CODON_CHANGE=gCc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CER1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005454.2;VDB=-0.0014;DN=107;DA=G/C;GM=NM_005454.2;GL=CER1;FG=missense;FD=missense;AAC=ALA/GLY;PP=65/268;CDP=194;GS=60;PH=0;CP=0.001;CG=2.060;AA=C;HA=12.7;HE=35.8;HC=4.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=314;KP=hsa04310:WntSignalingPathway;GESP=C:6095/G:6911;PAC=NP_005445.1 GT:GQ:PL 0/1:3:27,3,0 0/1:3:0,0,0 1/1:5:48,6,0 0/0:12:0,15,84</p>

      <p>9 19058743 rs10511670 G C 22.10 highCVG AC1=4;AF1=0.463;DB;DP=20;DP4=7,8,4,1;FQ=24.7;MQ=28;PV4=0.32,1,0.41,0.28;SNPEFF_AMINO_ACID_CHANGE=H674Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017645.4;VDB=0.0076;DN=119;DA=G/C;GM=NM_001270890.1,NM_017645.4,XM_005251491.1;GL=HAUS6;FG=missense;FD=missense,missense,unknown;AAC=HIS/GLN;PP=639/921,674/956,674/939;CDP=1917,2022,2022;GS=24;PH=0.008;CP=0.000;CG=-4.360;AA=G;HA=6.2;HE=6.7;HC=13.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=257;GESP=C:1067/G:11939;PPI=HAUS1:999|HAUS4:994|HAUS3:994|HAUS8:994|HAUS2:994|HAUS7:994;PAC=NP_001257819.1,NP_060115.3,XP_005251548.1 GT:GQ:PL 0/1:25:23,0,67 0/1:3:0,0,0 1/1:6:42,9,0 0/0:22:0,24,114</p>

      <p>9 19290794 rs41269019 A C 75.40 midCVG AC1=3;AF1=0.3437;DB;DP=11;DP4=0,5,3,3;FQ=69.4;MQ=60;PV4=0.18,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=I241L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DENND4C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017925.5;VDB=0.0195;DN=127;DA=A/C;GM=NM_017925.5,NR_073201.2;GL=DENND4C;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=ILE/LEU,none;PP=241/1910,NA;CDP=721,807;GS=5,NA;PH=0.996;CP=1.000;CG=5.440;AA=A;DV=by-frequency,by-cluster;DSP=81;GESP=C:267/A:11599;PAC=NP_060395.5,NA GT:GQ:PL 0/0:5:0,3,26 0/0:5:0,3,25 1/1:12:119,18,0 0/0:10:0,9,61</p>

      <p>9 26116150 rs7019716 G T 117 highCVG AC1=5;AF1=0.6707;DB;DP=18;DP4=3,3,1,10;FQ=65.1;MQ=60;PV4=0.099,0.0034,1,1;SNPEFF_AMINO_ACID_CHANGE=S109I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LOC100506422;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004352.2;VDB=0.0134;DN=116;DA=G/T;GM=NM_001004352.2;GL=LOC100506422;FG=missense;FD=missense;AAC=SER/ILE;PP=109/142;CDP=326;GS=142;CP=0.006;CG=-5.100;AA=G;HA=26.5;HE=31.2;HC=6.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=24;PAC=NP_001004352.1 GT:GQ:PL 0/1:37:31,0,62 0/0:3:0,6,46 1/1:9:50,9,0 1/1:15:84,15,0</p>

      <p>9 26978259 rs11555693 A G 90.20 highCVG AC1=6;AF1=0.7286;DB;DP=22;DP4=0,6,4,5;FQ=48.8;MQ=60;PV4=0.1,0.013,1,0.18;SNPEFF_AMINO_ACID_CHANGE=K85R;SNPEFF_CODON_CHANGE=aAa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IFT74;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099222.1;VDB=0.0093;DN=120;DA=A/G;GM=NM_001099222.1,NM_001099223.1,NM_001099224.1,NM_025103.2;GL=IFT74;FG=missense;FD=missense;AAC=LYS/ARG;PP=85/601,85/601,85/373,85/601;CDP=254;GS=26;PH=0.459;CP=1.000;CG=5.620;AA=A;DV=by-frequency,by-cluster;DSP=3;GESP=G:735/A:11169;PAC=NP_001092692.1,NP_001092693.1,NP_001092694.1,NP_079379.2 GT:GQ:PL 0/1:78:77,0,80 0/1:3:0,0,0 1/1:9:50,9,0 0/1:3:0,0,0</p>

      <p>9 33385863 rs62542746 G T 17.60 highCVG AC1=5;AF1=0.6224;DB;DP=17;DP4=2,5,5,5;FQ=21.6;MQ=33;PV4=0.62,1,1.5e-05,0.36;SNPEFF_AMINO_ACID_CHANGE=A176E;SNPEFF_CODON_CHANGE=gCg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0114;DN=129;DA=G/T;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense-near-splice;FD=missense,unknown,unknown;AAC=ALA/GLU;PP=176/343,176/343,176/274;CDP=527;GS=107;PH=0.006;CP=0.040;CG=1.570;AA=T;DV=by-cluster;DSP=2;KP=hsa03320:PPARSignalingPathway;GESP=G:13002;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:11:7,0,29 0/0:4:0,5,74 1/1:5:18,6,0 1/1:5:37,6,0</p>

      <p>9 33387047 rs4008658 T G 66.90 highCVG AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:27:37,0,25 0/1:17:22,0,15 1/1:10:52,12,0 0/1:4:0,2,50</p>

      <p>9 75315438 rs1796993 G A 69.40 midCVG AC1=4;AF1=0.4398;DB;DP=14;DP4=3,4,4,3;FQ=70.4;MQ=60;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=E81K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138691.2;VDB=0.0098;DN=89;DA=G/A;GM=NM_138691.2;GL=TMC1;FG=missense;FD=missense;AAC=GLU/LYS;PP=81/761;CDP=241;GS=56;PH=0.155;CP=1.000;CG=5.480;AA=G;HA=32.0;HE=16.5;HC=43.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=5;GESP=A:2606/G:10392;PAC=NP_619636.2 GT:GQ:PL 0/1:16:72,0,11 0/0:8:0,9,66 1/1:4:38,6,0 0/1:5:2,0,39</p>

      <p>9 88651370 rs2297002 T C 82.10 midCVG AC1=3;AF1=0.3754;DB;DP=12;DP4=5,1,4,2;FQ=82;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H217R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GOLM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016548.3;VDB=0.0035;DN=100;DA=T/C;GM=NM_016548.3,NM_177937.2,XM_005252040.1,XM_005252041.1,XM_005252042.1,XM_005252043.1,XM_005252044.1;GL=GOLM1;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=HIS/ARG;PP=217/402,217/402,217/459,217/459,217/403,217/385,217/385;CDP=650;GS=29;PH=0;CP=0.000;CG=-3.200;AA=C;HA=49.2;HE=2.5;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=C:1785/T:11221;PAC=NP_057632.2,NP_808800.1,XP_005252097.1,XP_005252098.1,XP_005252099.1,XP_005252100.1,XP_005252101.1 GT:GQ:PL 0/0:12:0,12,76 0/0:7:0,6,51 1/1:12:125,18,0 0/1:3:0,0,0</p>

      <p>9 95782118 . A G 18 midCVG AC1=2;AF1=0.2756;DP=9;DP4=4,2,3,0;FQ=19.4;MQ=60;PV4=0.5,0.43,1,1;SNPEFF_AMINO_ACID_CHANGE=E454G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FGD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001083536.1;VDB=0.0003;GM=NM_001083536.1,NM_033086.2,XM_005252296.1;GL=FGD3;FG=missense;FD=unknown;AAC=GLU/GLY;PP=454/726,454/726,454/725;CDP=1361;GS=98;PH=0.033;CP=0.997;CG=4.620;AA=A;DSP=6;KP=hsa04810:RegulationOfActinCytoskeleton;GESP=A:12070;PAC=NP_001077005.1,NP_149077.2,XP_005252353.1 GT:GQ:PL 0/0:14:0,12,91 0/0:6:0,3,16 1/1:3:59,9,0 0/0:6:0,3,26</p>

      <p>9 104433235 rs10989589 C T 82.80 highCVG AC1=4;AF1=0.5477;DB;DP=16;DP4=2,2,4,3;FQ=34.3;MQ=60;PV4=1,0.22,1,0.16;SNPEFF_AMINO_ACID_CHANGE=G487R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRIN3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133445.2;VDB=0.0068;DN=120;DA=C/T;GM=NM_133445.2;GL=GRIN3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=487/1116;CDP=1459;GS=125;PH=0.956;CP=0.252;CG=5.760;AA=C;HA=10.0;HE=35.8;HC=20.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=155;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=T:4290/C:8716;PPI=GRIN1:846|CPEB1:830;PAC=NP_597702.2 GT:GQ:PL 0/0:5:0,6,49 0/1:3:0,0,0 1/1:10:83,12,0 0/1:27:42,0,24</p>

      <p>9 107266596 rs7049042 T C 101 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=0,2,2,5;FQ=3.71;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F18S;SNPEFF_CODON_CHANGE=tTt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0171;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=PHE/SER;PP=18/320;CDP=53;GS=155;PH=0;CP=0.855;CG=3.000;AA=C;HA=8.7;HE=27.3;HC=22.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=243538;KP=hsa04740:OlfactoryTransduction;GESP=C:10035/T:2971;PAC=NP_001004485.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,38 1/1:17:117,18,0 1/1:4:26,3,0</p>

      <p>9 107267352 rs7847413 T C 13.20 midCVG AC1=2;AF1=0.2756;DB;DP=14;DP4=2,9,3,0;FQ=14.5;MQ=60;PV4=0.027,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=I270T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0024;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=ILE/THR;PP=270/320;CDP=809;GS=89;PH=0.016;CP=0.942;CG=1.920;AA=T;HA=40.2;HE=8.4;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=242782;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFAT_Q6;GESP=C:2246/T:10760;PAC=NP_001004485.1 GT:GQ:PL 0/0:14:0,12,74 0/0:23:0,21,130 1/1:3:54,9,0 0/0:4:0,0,0</p>

      <p>9 107367008 rs10156474 T C 41.50 lowCVG AC1=6;AF1=0.7416;DB;DP=5;DP4=1,0,2,2;FQ=-17.1;MQ=40;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K301E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13C2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004481.1;VDB=0.0107;DN=119;DA=T/C;GM=NM_001004481.1;GL=OR13C2;FG=missense;FD=missense;AAC=LYS/GLU;PP=301/319;CDP=901;GS=56;PH=0.693;CP=0.996;CG=3.530;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=143126;KP=hsa04740:OlfactoryTransduction;GESP=C:4176/T:8826;PAC=NP_001004481.1 GT:GQ:PL 0/1:3:0,0,0 0/1:14:14,0,14 1/1:9:64,9,0 0/1:3:0,0,0</p>

      <p>9 107380414 rs2900373 C A 67.60 midCVG AC1=5;AF1=0.6029;DB;DP=13;DP4=4,1,2,5;FQ=29.7;MQ=44;PV4=0.24,0.14,1,0.38;SNPEFF_AMINO_ACID_CHANGE=E24D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13C9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001956.1;VDB=0.0115;DN=101;DA=C/A/T;GM=NM_001001956.1;GL=OR13C9;FG=missense;FD=missense/synonymous-codon;AAC=GLU/ASP;PP=24/319;CDP=72;GS=45;PH=0.229;CP=0.982;CG=3.890;AA=C;HA=34.1;HE=20.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=129720;KP=hsa04740:OlfactoryTransduction;GESP=A:4083/C:8923;PAC=NP_001001956.1 GT:GQ:PL 0/1:12:9,0,14 0/0:6:0,9,58 1/1:4:19,3,0 1/1:14:86,15,0</p>

      <p>9 107533175 rs3739741 C G 117 highCVG AC1=5;AF1=0.5707;DB;DP=19;DP4=1,7,4,7;FQ=84.8;MQ=60;PV4=0.34,0.018,1,1;SNPEFF_AMINO_ACID_CHANGE=A159G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPSNAP3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018376.2;VDB=0.0089;DN=107;DA=C/G;GM=NM_018376.2;GL=NIPSNAP3B;FG=missense;FD=unknown;AAC=ALA/GLY;PP=159/248;CDP=476;GS=60;PH=0.999;CP=1.000;CG=2.440;AA=C;HA=0.0;HE=20.7;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=46;GESP=G:1830/C:11176;PAC=NP_060846.2 GT:GQ:PL 0/1:28:67,0,26 0/0:14:0,18,103 1/1:4:27,3,0 1/1:13:71,15,0</p>

      <p>9 113208250 rs7863519 T G 81.60 midCVG AC1=4;AF1=0.5301;DB;DP=11;DP4=3,1,3,4;FQ=42.9;MQ=60;PV4=0.55,0.4,1,1;SNPEFF_AMINO_ACID_CHANGE=M1444L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SVEP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153366.3;VDB=0.0187;DN=116;DA=T/G;GM=NM_153366.3;GL=SVEP1;FG=missense;FD=missense;AAC=MET/LEU;PP=1444/3572;CDP=4330;GS=15;PH=0.002;CP=0.023;CG=3.620;AA=C;HA=44.7;HE=20.1;HC=37.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=69;TFBS=V$YY1_01;GESP=G:3078/T:9136;PAC=NP_699197.3 GT:GQ:PL 0/0:3:0,3,25 0/1:3:16,3,0 1/1:15:112,18,0 0/0:7:0,9,61</p>

      <p>9 114359624 rs1053959 C A 122 highCVG AC1=3;AF1=0.3851;DB;DP=23;DP4=6,8,4,5;FQ=124;MQ=59;PV4=1,1,1,0.0013;SNPEFF_AMINO_ACID_CHANGE=A27S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTGR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146108.1;VDB=0.0165;DN=86;DA=C/A;GM=NM_001146108.1,NM_001146109.1,NM_012212.3;GL=PTGR1;FG=missense;FD=missense;AAC=ALA/SER;PP=27/330,27/302,27/330;CDP=79;GS=99;PH=0.019;CP=0.038;CG=-0.431;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=28;GESP=A:7644/C:5362;PAC=NP_001139580.1,NP_001139581.1,NP_036344.2 GT:GQ:PL 0/0:29:0,30,158 0/1:10:91,0,4 1/1:8:78,12,0 0/0:9:0,9,72</p>

      <p>9 114464487 rs1407390 T C 45.60 lowCVG AC1=6;AF1=0.7452;DB;DP=8;DP4=0,1,0,6;FQ=-7.45;MQ=60;PV4=1,0.0038,1,0.36;SNPEFF_AMINO_ACID_CHANGE=Y932C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C9orf84;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173521.4;VDB=0.0037;DN=88;DA=T/C;GM=NM_001080551.1,NM_173521.3,XM_005251738.1;GL=C9orf84;FG=missense;FD=missense,missense,unknown;AAC=TYR/CYS;PP=893/1406,932/1445,996/1509;CDP=2678,2795,2987;GS=194;PH=0;CP=0.930;CG=-2.510;AA=C;HA=15.6;HE=21.5;HC=7.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=20;GESP=C:10539/T:2451;PAC=NP_001074020.1,NP_775792.3,XP_005251795.1 GT:GQ:PL 0/1:5:0,3,28 0/1:3:0,0,0 1/1:5:18,3,0 1/1:15:68,15,0</p>

      <p>9 117050998 rs10982134 G A 55.90 lowCVG AC1=6;AF1=0.7419;DB;DP=6;DP4=1,0,3,1;FQ=-9.04;MQ=60;PV4=1,0.38,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R1354Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL27A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032888.2;VDB=0.0146;DN=120;DA=G/A;GM=NM_032888.2;GL=COL27A1;FG=missense;FD=missense;AAC=ARG/GLN;PP=1354/1861;CDP=4061;GS=43;PH=0.016;CP=0.999;CG=1.840;AA=G;HA=21.2;HE=25.9;HC=38.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=A:3908/G:9098;PAC=NP_116277.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,26 1/1:7:48,6,0 1/1:7:48,6,0</p>

      <p>9 117846580 rs1061494 T C 25.30 midCVG AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1 GT:GQ:PL 0/1:22:18,0,120 0/1:3:0,0,0 1/1:6:35,6,0 0/1:15:12,0,20</p>

      <p>9 123170733 rs4837768 C G 142 highCVG AC1=6;AF1=0.7872;DB;DP=17;DP4=3,0,5,5;FQ=-19.3;MQ=60;PV4=0.23,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=V1540L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDK5RAP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018249.5;VDB=0.0119;DN=111;DA=C/G;GM=NM_001011649.2,NM_001272039.1,NM_018249.5,NR_073554.1,NR_073555.1,NR_073556.1,NR_073557.1,NR_073558.1;GL=CDK5RAP2;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant;AAC=VAL/LEU,VAL/LEU,VAL/LEU,none,none,none,none,none;PP=1540/1815,1310/1664,1540/1894,NA,NA,NA,NA,NA;CDP=4618,3928,4618,4887,4810,5017,4890,4887;GS=32,32,32,NA,NA,NA,NA,NA;PH=0.105;CP=0.112;CG=-0.555;AA=C;HA=41.5;HE=27.5;HC=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:9047/C:3959;PPI=CDK5R1:845;PAC=NP_001011649.1,NP_001258968.1,NP_060719.4,NA,NA,NA,NA,NA GT:GQ:PL 0/1:4:27,0,4 0/1:10:31,0,12 1/1:6:21,3,0 1/1:17:105,15,0</p>

      <p>9 123769200 rs17611 C T 77.90 midCVG AC1=6;AF1=0.677;DB;DP=11;DP4=4,0,5,2;FQ=41.7;MQ=60;PV4=0.49,0.026,1,0.15;SNPEFF_AMINO_ACID_CHANGE=V802I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001735.2;VDB=0.0065;DN=63;DA=C/T;GM=NM_001735.2;GL=C5;FG=missense;FD=missense;AAC=VAL/ILE;PP=802/1677;CDP=2404;GS=29;PH=0.127;CP=0.986;CG=1.060;AA=C;HA=0.0;HE=39.0;HC=41.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=19;KP=hsa04610:ComplementAndCoagulationCascades|hsa05020:PrionDiseases|hsa05322:SystemicLupusErythematosus;GESP=T:4225/C:8781;PPI=C6:983;PAC=NP_001726.2 GT:GQ:PL 0/1:4:0,3,24 0/1:17:11,0,58 1/1:12:71,12,0 1/1:7:41,6,0</p>

      <p>9 123860689 rs10818504 C T 202 highCVG AC1=5;AF1=0.6874;DB;DP=18;DP4=1,2,13,2;FQ=9.15;MQ=60;PV4=0.11,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=P216L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007018.4;VDB=0.0128;DN=120;DA=C/T;GM=NM_007018.4,XM_005251679.1,XM_005251680.1;GL=CNTRL;FG=missense;FD=missense,unknown,unknown;AAC=PRO/LEU;PP=216/2326,216/2326,49/2159;CDP=647,647,146;GS=98;PH=0.998;CP=0.998;CG=4.730;AA=T;HA=8.7;HE=39.0;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=T:4541/C:8465;PAC=NP_008949.4,XP_005251736.1,XP_005251737.1 GT:GQ:PL 0/1:3:0,6,40 0/1:11:70,0,11 1/1:24:132,24,0 1/1:10:48,9,0</p>

      <p>9 125273127 . G A 20 midCVG AC1=2;AF1=0.2748;DP=14;DP4=1,5,0,3;FQ=21.3;MQ=57;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G16D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1J2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054107.1;VDB=0.0008;GM=NM_054107.1,XM_005251920.1;GL=OR1J2;FG=missense;FD=unknown;AAC=GLY/ASP;PP=16/314;CDP=47;GS=94;PH=0.862;CP=0.998;CG=4.120;AA=G;DSP=60;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFKB_C;GESP=G:13006;PAC=NP_473448.1,XP_005251977.1 GT:GQ:PL 0/0:6:0,3,24 0/0:6:0,3,24 1/1:4:61,9,0 0/0:14:0,12,73</p>

      <p>9 125315557 rs1831370 T C 80.20 highCVG AC1=4;AF1=0.548;DB;DP=15;DP4=2,5,4,3;FQ=74.2;MQ=60;PV4=0.59,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W37R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1N2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004457.1;VDB=0.0200;DN=92;DA=T/C;GM=NM_001004457.1;GL=OR1N2;FG=missense;FD=missense;AAC=TRP/ARG;PP=37/331;CDP=109;GS=101;PH=0;CP=0.002;CG=-2.620;AA=C;HA=41.0;HE=45.5;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42490;KP=hsa04740:OlfactoryTransduction;GESP=C:6740/T:6266;PAC=NP_001004457.1 GT:GQ:PL 0/1:3:0,3,19 0/0:5:0,7,98 1/1:12:96,15,0 1/1:3:30,3,0</p>

      <p>9 125330125 rs10739614 C G 163 highCVG AC1=6;AF1=0.7385;DB;DP=20;DP4=0,5,11,3;FQ=29.2;MQ=60;PV4=0.0048,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=R211P;SNPEFF_CODON_CHANGE=cGt/cCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0083;DN=120;DA=C/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=ARG/PRO;PP=211/310;CDP=632;GS=103;PH=0;CP=0.008;CG=3.600;AA=G;HA=32.8;HE=45.0;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57058;KP=hsa04740:OlfactoryTransduction;GESP=G:8273/C:4733;PAC=NP_001004454.1 GT:GQ:PL 0/1:12:16,0,13 0/1:46:40,0,55 1/1:8:43,6,0 1/1:25:109,24,0</p>

      <p>9 125330678 rs10985704 T G 52.30 midCVG AC1=5;AF1=0.6368;DB;DP=9;DP4=2,0,6,0;FQ=-3.17;MQ=60;PV4=1,0.0061,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T27P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0026;DN=120;DA=T/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=THR/PRO;PP=27/310;CDP=79;GS=38;PH=0;CP=0.733;CG=3.480;AA=G;HA=43.0;HE=49.1;HC=1.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57611;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:7411/T:5595;PAC=NP_001004454.1 GT:GQ:PL 0/1:4:0,3,25 0/1:14:37,0,13 1/1:4:20,3,0 1/1:6:38,6,0</p>

      <p>9 125390874 rs1556189 A C 63.70 midCVG AC1=4;AF1=0.5408;DB;DP=11;DP4=0,5,0,6;FQ=48;MQ=60;PV4=1,0.4,1,1;SNPEFF_AMINO_ACID_CHANGE=V314G;SNPEFF_CODON_CHANGE=gTg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0169;DN=88;DA=A/C;GM=NM_001004450.1;GL=OR1B1;FG=missense;FD=missense;AAC=VAL/GLY;PP=314/319;CDP=941;GS=109;PH=0;CP=0.000;CG=-0.410;HA=39.3;HE=14.2;HC=4.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=117807;KP=hsa04740:OlfactoryTransduction;GESP=C:9912/A:3094;PAC=NP_001004450.1 GT:GQ:PL 0/1:57:56,0,58 0/0:3:0,3,26 1/1:5:47,6,0 0/1:3:0,0,0</p>

      <p>9 125391026 rs1476858 A C 98.50 highCVG AC1=5;AF1=0.6085;DB;DP=20;DP4=2,9,7,2;FQ=103;MQ=60;PV4=0.022,0.0062,1,0.29;SNPEFF_AMINO_ACID_CHANGE=C263W;SNPEFF_CODON_CHANGE=tgT/tgG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0050;DN=88;DA=A/C;GM=NM_001004450.1;GL=OR1B1;FG=missense;FD=missense;AAC=CYS/TRP;PP=263/319;CDP=789;GS=215;PH=0;CP=0.921;CG=2.820;HA=42.5;HE=15.5;HC=6.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=117959;KP=hsa04740:OlfactoryTransduction;GESP=C:9234/A:3772;PAC=NP_001004450.1 GT:GQ:PL 0/0:8:0,12,90 0/1:19:14,0,103 1/1:6:46,6,0 1/1:14:87,15,0</p>

      <p>9 125391241 rs1476860 G A 91.10 midCVG AC1=6;AF1=0.6853;DB;DP=9;DP4=2,1,1,5;FQ=19.3;MQ=60;PV4=0.23,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R192*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0081;DN=88;DA=G/A;GM=NM_001004450.1;GL=OR1B1;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=192/319;CDP=574;CP=0.142;CG=-6.050;AA=G;HA=3.8;HE=30.8;HC=46.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118174;KP=hsa04740:OlfactoryTransduction;GESP=A:2952/G:10054;PAC=NP_001004450.1 GT:GQ:PL 0/0:5:0,9,58 0/1:3:0,0,0 1/1:6:45,6,0 1/1:11:90,12,0</p>

      <p>OR1B1 9 130165973 rs34064803 T C 79.40 highCVG AC1=3;AF1=0.4231;DB;DP=25;DP4=0,4,2,15;FQ=51.8;MQ=60;PV4=1,0.0002,1,0.08;SNPEFF_AMINO_ACID_CHANGE=I253T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC2A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014580.4;VDB=0.0144;DN=126;DA=T/C;GM=NM_001271711.1,NM_001271712.1,NM_014580.4,NR_073416.1,XM_005251949.1;GL=SLC2A8;FG=missense,missense,missense,non-coding-exon,missense;FD=missense,missense,missense,nc-transcript-variant,unknown;AAC=ILE/THR,ILE/THR,ILE/THR,none,ILE/THR;PP=253/412,90/315,253/478,NA,90/315;CDP=758,269,758,284,269;GS=89,89,89,NA,89;PH=0.878;CP=0.927;CG=5.420;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=C:1021/T:11985;PAC=NP_001258640.1,NP_001258641.1,NP_055395.2,NA,XP_005252006.1 GT:GQ:PL 0/0:6:0,6,47 0/1:3:0,0,0 1/1:46:122,51,0 0/0:6:0,6,45</p>

      <p>9 134401335 rs2296957 C T 63.50 midCVG AC1=4;AF1=0.5532;DB;DP=9;DP4=1,2,1,5;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A180T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135954.1;VDB=0.0243;DN=100;DA=C/A/T;GM=NM_001135954.1,NM_001261450.1,NM_001261451.1,NM_031432.2,XM_005272224.1,XM_005272225.1;GL=UCK1;FG=missense,synonymous,synonymous,synonymous,synonymous,synonymous;FD=missense,synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown;AAC=ALA/THR,PRO,PRO,PRO,PRO,PRO;PP=180/202,202/269,216/283,211/278,225/292,188/255;CDP=538,606,648,633,675,564;GS=58,NA,NA,NA,NA,NA;PH=0.104;CP=0.908;CG=-5.080;AA=C;HA=41.2;HE=8.1;HC=7.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=20;KP=hsa00240:PyrimidineMetabolism|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;TFBS=V$SREBP1_02;GESP=T:10799/C:2207;PAC=NP_001129426.1,NP_001248379.1,NP_001248380.1,NP_113620.1,XP_005272281.1,XP_005272282.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,60 1/1:15:103,18,0 0/1:3:0,0,0</p>

      <p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

      <p>9 137779091 rs7851696 G T 152 highCVG AC1=4;AF1=0.5192;DB;DP=19;DP4=3,4,3,9;FQ=78.5;MQ=60;PV4=0.62,0.15,1,0.038;SNPEFF_AMINO_ACID_CHANGE=A258S;SNPEFF_CODON_CHANGE=Gct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FCN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004108.2;VDB=0.0132;DN=116;DA=G/T;GM=NM_004108.2,NM_015837.2;GL=FCN2;FG=missense;FD=missense;AAC=ALA/SER;PP=258/314,220/276;CDP=772,658;GS=99;PH=0.946;CP=0.077;CG=4.050;AA=G;HA=17.6;HE=11.0;HC=16.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=78;GESP=T:1769/G:11237;PAC=NP_004099.2,NP_056652.1 GT:GQ:PL 0/1:10:77,0,6 0/0:5:0,6,49 1/1:5:45,6,0 0/1:67:74,0,64</p>

      <p>9 138376972 rs3928777 A G 77.10 midCVG AC1=6;AF1=0.6853;DB;DP=9;DP4=0,3,0,6;FQ=11.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K206E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014811.3;VDB=0.0151;DN=108;DA=A/G;GM=NM_014811.3,XM_005263411.1,XM_005263412.1,XM_005263413.1,XM_005263414.1,XM_005263415.1;GL=PPP1R26;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=206/1210;CDP=616;GS=56;PH=0;CP=0.000;CG=-4.970;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=678;GESP=G:2902/A:10090;PAC=NP_055626.3,XP_005263468.1,XP_005263469.1,XP_005263470.1,XP_005263471.1,XP_005263472.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,50 1/1:11:80,12,0 1/1:6:41,6,0</p>

      <p>9 139118673 rs12380852 T C 17.20 midCVG AC1=3;AF1=0.3906;DB;DP=12;DP4=3,5,1,2;FQ=19.3;MQ=60;PV4=1,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K126E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=QSOX2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181701.3;VDB=0.0103;DN=120;DA=T/C;GM=NM_181701.3;GL=QSOX2;FG=missense;FD=missense;AAC=LYS/GLU;PP=126/699;CDP=376;GS=56;PH=0.002;CP=0.615;CG=2.180;AA=T;HA=4.2;HE=7.5;HC=25.8;DG;DV=by-frequency,by-cluster,by-1000G;DSP=48;GESP=C:1203/T:11803;PAC=NP_859052.3 GT:GQ:PL 0/0:9:0,9,60 0/1:4:26,3,0 1/1:3:37,6,0 0/0:15:0,15,95</p>

      <p>9 140201604 rs28685844 C T 15.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=3,0,1,1;FQ=15;MQ=60;PV4=0.4,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R810Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017820.4;VDB=0.0130;DN=125;DA=C/T;GM=NM_017820.3,XM_005266091.1,XM_005266092.1;GL=EXD3;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=810/877,490/557,448/515;CDP=2429,1469,1343;GS=43;PH=0.193;CP=0.000;CG=-3.990;AA=C;HA=3.6;HE=0.4;HC=8.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=112;GESP=T:123/C:12191;PAC=NP_060290.3,XP_005266148.1,XP_005266149.1 GT:GQ:PL 0/1:3:0,0,0 0/0:9:0,9,51 1/1:3:52,6,0 0/1:3:0,0,0</p>

      <p>10 1065457 rs41314629 T A 60.60 lowCVG AC1=5;AF1=0.5962;DB;DP=7;DP4=2,0,4,1;FQ=10.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*228C;SNPEFF_CODON_CHANGE=tgA/tgT;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IDI2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_033261.2;VDB=0.0111;DN=127;DA=T/A;GM=NM_033261.2;GL=IDI2;FG=stop-lost;FD=stop-lost;AAC=stop/CYS;PP=228/228;CDP=684;CP=0.004;CG=3.300;AA=T;DV=by-frequency,by-cluster;DSP=318;KP=hsa00900:TerpenoidBackboneBiosynthesis|hsa01100:MetabolicPathways;GESP=A:847/T:12159;PAC=NP_150286.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:12:99,15,0 0/0:5:0,6,45</p>

      <p>10 5415954 rs10904481 A G 49.50 lowCVG AC1=6;AF1=0.685;DB;DP=8;DP4=0,3,1,4;FQ=13.7;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R91G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053049.2;VDB=0.0100;DN=120;DA=A/G;GM=NM_053049.2;GL=UCN3;FG=missense;FD=missense;AAC=ARG/GLY;PP=91/162;CDP=271;GS=125;PH=0;CP=0.000;CG=2.520;AA=G;HA=29.2;HE=35.7;HC=48.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=277;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4033/A:8969;PAC=NP_444277.2 GT:GQ:PL 0/1:17:11,0,30 0/1:19:16,0,21 1/1:7:46,6,0 1/1:5:19,3,0</p>

      <p>10 16979714 rs41289305 T C 71.40 highCVG AC1=5;AF1=0.6228;DB;DP=16;DP4=5,2,7,1;FQ=70;MQ=60;PV4=0.57,0.03,1,0.14;SNPEFF_AMINO_ACID_CHANGE=S1935G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=39;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUBN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081.3;VDB=0.0066;DN=127;DA=T/C;GM=NM_001081.3;GL=CUBN;FG=missense;FD=missense;AAC=SER/GLY;PP=1935/3624;CDP=5803;GS=56;PH=0.507;CP=0.136;CG=0.044;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=C:1887/T:11119;PPI=GIF:983;PAC=NP_001072.2 GT:GQ:PL 0/1:8:3,0,76 0/1:3:0,0,0 1/1:8:48,9,0 0/1:33:60,0,31</p>

      <p>10 24669877 rs17506606 C G 29.60 lowCVG AC1=4;AF1=0.5278;DB;DP=8;DP4=2,2,0,4;FQ=32.8;MQ=60;PV4=0.43,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=A145G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1217;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019590.4;VDB=0.0046;DN=123;DA=C/G;GM=NM_001098500.1,NM_001098501.1,NM_019590.3,XM_005252500.1,XM_005252501.1,XM_005252502.1,XM_005252503.1,XM_005252504.1,XM_005252505.1,XM_005252506.1,XM_005252507.1,XM_005252508.1,XM_005252509.1,XM_005252510.1,XM_005252511.1,XM_005252512.1,XM_005252513.1,XM_005252517.1,XM_005252518.1,XM_005252519.1,XM_005252520.1,XM_005252521.1,XM_005252522.1,XM_005252523.1,XM_005252524.1;GL=KIAA1217;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,none,none,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY,ALA/GLY;PP=65/1265,145/1310,145/1944,172/1983,172/1982,172/1971,145/1956,172/1948,172/1936,172/1931,145/1921,145/1909,28/1839,172/1822,92/1903,NA,NA,172/1450,172/1412,145/1375,172/1372,145/1350,145/1345,172/1242,172/1176;CDP=194,434,434,515,515,515,434,515,515,515,434,434,83,515,275,NA,NA,515,515,434,515,434,434,515,515;GS=60,60,60,60,60,60,60,60,60,60,60,60,60,60,60,NA,NA,60,60,60,60,60,60,60,60;PH=0.003;CP=0.008;CG=-0.211;AA=C;HA=0.0;HE=26.6;HC=0.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=80;GESP=G:2468/C:10538;PAC=NP_001091970.1,NP_001091971.1,NP_062536.2,XP_005252557.1,XP_005252558.1,XP_005252559.1,XP_005252560.1,XP_005252561.1,XP_005252562.1,XP_005252563.1,XP_005252564.1,XP_005252565.1,XP_005252566.1,XP_005252567.1,XP_005252568.1,NA,NA,XP_005252574.1,XP_005252575.1,XP_005252576.1,XP_005252577.1,XP_005252578.1,XP_005252579.1,XP_005252580.1,XP_005252581.1 GT:GQ:PL 0/1:3:0,0,0 0/1:20:17,0,42 1/1:7:54,9,0 0/0:5:0,6,46</p>

      <p>10 48416456 . T C 53.30 highCVG AC1=3;AF1=0.3888;DP=15;DP4=0,5,0,6;FQ=54.1;MQ=60;PV4=1,1,1,0.0011;SNPEFF_AMINO_ACID_CHANGE=K80E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016204.2;VDB=0.0215;GM=NM_016204.1;GL=GDF2;FG=missense;FD=unknown;AAC=LYS/GLU;PP=80/430;CDP=238;GS=56;PH=0.684;CP=0.393;CG=4.220;AA=T;DSP=109;GESP=T:13006;PPI=ACVRL1:900|ACVR2B:900;PAC=NP_057288.1 GT:GQ:PL 0/0:7:0,6,36 0/0:9:0,9,64 1/1:13:96,18,0 0/1:3:0,0,0</p>

      <p>10 69933969 rs7916821 G A 187 highCVG AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA GT:GQ:PL 0/1:45:68,0,45 0/1:5:0,3,28 1/1:15:86,15,0 1/1:13:80,12,0</p>

      <p>10 70641860 rs1341667 T C 76 midCVG AC1=5;AF1=0.6593;DB;DP=10;DP4=1,2,3,4;FQ=26.9;MQ=60;PV4=1,0.21,1,0.26;SNPEFF_AMINO_ACID_CHANGE=Y153H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130161.2;VDB=0.0014;DN=88;DA=T/C;GM=NM_001130159.2,NM_001130160.2,NM_001130161.2,NM_001130162.2,NM_152709.4,XM_005269605.1;GL=STOX1;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=TYR/HIS;PP=153/228,153/170,153/990,43/880,153/990,43/880;CDP=457,457,457,127,457,127;GS=83;PH=0.687;CP=0.990;CG=4.440;AA=T;HA=33.8;HE=38.1;HC=11.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=7;TFBS=V$HNF1_C;GESP=C:6772/T:5074;PAC=NP_001123631.1,NP_001123632.1,NP_001123633.1,NP_001123634.1,NP_689922.3,XP_005269662.1 GT:GQ:PL 0/0:6:0,9,65 0/1:3:0,0,0 1/1:17:106,18,0 1/1:4:13,3,0</p>

      <p>10 84733562 . C T 20.10 midCVG AC1=3;AF1=0.3373;DP=11;DP4=6,1,3,0;FQ=21.8;MQ=60;PV4=1,0.18,1,0.066;SNPEFF_AMINO_ACID_CHANGE=H435Y;SNPEFF_CODON_CHANGE=Cat/Tat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010848.3;VDB=0.0000;GM=NM_001010848.3,NM_001165972.1,NM_001165973.1,XM_005269444.1,XM_005269445.1,XM_005269446.1,XM_005269447.1,XM_005269448.1,XM_005269449.1,XM_005269450.1,XM_005269451.1;GL=NRG3;FG=missense;FD=unknown;AAC=HIS/TYR;PP=435/697,434/696,214/500,435/721,265/527,239/525,239/501,214/476,171/457,171/433,85/371;CDP=1303,1300,640,1303,793,715,715,640,511,511,253;GS=83;PH=0.952;CP=1.000;CG=5.050;AA=C;DSP=19;KP=hsa04012:ErbBSignalingPathway;TFBS=V$CETS1P54_01;GESP=C:13006;PAC=NP_001010848.2,NP_001159444.1,NP_001159445.1,XP_005269501.1,XP_005269502.1,XP_005269503.1,XP_005269504.1,XP_005269505.1,XP_005269506.1,XP_005269507.1,XP_005269508.1 GT:GQ:PL 0/0:5:0,3,24 0/0:19:0,18,100 1/1:4:60,9,0 0/0:3:0,0,0</p>

      <p>10 85991784 rs3814211 T C 28.60 midCVG AC1=2;AF1=0.3082;DB;DP=11;DP4=2,5,0,3;FQ=30.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S591G;SNPEFF_CODON_CHANGE=Agt/Ggt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRIT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015613.2;VDB=0.0116;DN=107;DA=T/C;GM=NM_015613.2;GL=LRIT1;FG=missense;FD=missense;AAC=SER/GLY;PP=591/624;CDP=1771;GS=56;PH=0.242;CP=0.475;CG=2.790;AA=T;HA=0.0;HE=3.8;HC=2.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=876;GESP=C:495/T:12511;PAC=NP_056428.1 GT:GQ:PL 0/0:8:0,6,42 0/0:3:0,0,0 1/1:4:69,9,0 0/0:17:0,15,113</p>

      <p>10 90356598 rs1409136 A G 55 midCVG AC1=3;AF1=0.3912;DB;DP=14;DP4=7,2,2,3;FQ=57.1;MQ=60;PV4=0.27,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=I210V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001010939.2;VDB=0.0121;DN=88;DA=A/G;GM=NM_001010939.2,XM_005269542.1;GL=LIPJ;FG=missense;FD=missense,unknown;AAC=ILE/VAL;PP=210/367,97/254;CDP=628,289;GS=29;PH=0;CP=0.000;CG=-8.230;AA=A;HA=1.1;HE=28.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=52;GESP=G:10260/A:2734;PAC=NP_001010939.2,XP_005269599.1 GT:GQ:PL 0/0:4:0,3,26 0/0:20:0,21,97 1/1:5:58,9,0 0/1:21:42,0,16</p>

      <p>10 91007360 rs1051338 T G 24.80 lowCVG AC1=4;AF1=0.5432;DB;DP=5;DP4=1,1,0,3;FQ=4.88;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T16P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000235.3;VDB=0.0127;DN=86;DA=T/G;GM=NM_000235.2,NM_001127605.1;GL=LIPA;FG=missense;FD=missense;AAC=THR/PRO;PP=16/400;CDP=46;GS=38;PH=0.002;CP=0.997;CG=3.720;AA=T;HA=8.8;HE=33.0;HC=28.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=47;KP=hsa00100:SteroidBiosynthesis|hsa04142:Lysosome;GESP=G:3134/T:9872;PAC=NP_000226.2,NP_001121077.1 GT:GQ:PL 0/1:19:20,0,18 0/1:3:0,0,0 1/1:5:44,6,0 0/0:3:0,3,21</p>

      <p>10 91488979 rs3758388 A T 121 highCVG AC1=4;AF1=0.546;DB;DP=19;DP4=2,7,3,7;FQ=95.9;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=H789L;SNPEFF_CODON_CHANGE=cAt/cTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284259.1;VDB=0.0119;DN=107;DA=A/T;GM=NM_016195.2,XM_005270290.1;GL=KIF20B;FG=missense;FD=missense,unknown;AAC=HIS/LEU;PP=749/1781,789/1821;CDP=2246,2366;GS=99;PH=0.069;CP=0.001;CG=0.696;AA=A;HA=2.9;HE=26.2;HC=39.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=T:1992/A:10770;PAC=NP_057279.2,XP_005270347.1 GT:GQ:PL 0/1:65:62,0,80 0/0:4:0,6,50 1/1:10:77,12,0 0/1:11:28,0,8</p>

      <p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

      <p>10 102265183 rs2295772 G A 205 highCVG AC1=5;AF1=0.6183;DB;DP=23;DP4=2,4,6,8;FQ=50.4;MQ=60;PV4=1,1,0.26,0.38;SNPEFF_AMINO_ACID_CHANGE=P372S;SNPEFF_CODON_CHANGE=Cca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC31B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015490.3;VDB=0.0054;DN=100;DA=G/A;GM=NM_015490.3,XM_005269676.1,XM_005269677.1,XM_005269678.1,XM_005269679.1;GL=SEC31B;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=PRO/SER,PRO/SER,PRO/SER,PRO/SER,none;PP=372/1180,372/1263,372/1263,215/1106,NA;CDP=1114,1114,1114,643,NA;GS=74,74,74,74,NA;PH=0;CP=0.368;CG=3.470;AA=A;HA=14.0;HE=18.7;HC=16.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=66;TFBS=V$ZID_01;GESP=A:2751/G:10255;PPI=SF3A2:944;PAC=NP_056305.1,XP_005269733.1,XP_005269734.1,XP_005269735.1,NA GT:GQ:PL 0/1:18:44,0,17 0/0:8:0,12,78 1/1:6:52,6,0 1/1:26:158,27,0</p>

      <p>10 105657316 rs10786775 G C 120 midCVG AC1=6;AF1=0.7314;DB;DP=12;DP4=0,3,4,4;FQ=10.4;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S248C;SNPEFF_CODON_CHANGE=tCc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBFC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024928.4;VDB=0.0047;DN=120;DA=G/C;GM=NM_024928.4;GL=OBFC1;FG=missense;FD=missense;AAC=SER/CYS;PP=248/369;CDP=743;GS=112;PH=0;CP=0.000;CG=1.390;AA=C;HA=41.6;HE=7.3;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=11;GESP=C:10702/G:2304;PAC=NP_079204.2 GT:GQ:PL 0/1:22:15,0,32 0/1:16:23,0,17 1/1:10:66,9,0 1/1:10:61,9,0</p>

      <p>10 115529576 rs41292632 T A 52.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=1,1,4,1;FQ=13.3;MQ=60;PV4=1,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=F151I;SNPEFF_CODON_CHANGE=Ttc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLEKHS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182601.1;VDB=0.0055;DN=127;DA=T/A;GM=NM_001193434.1,NM_001193435.1,NM_024889.4,NM_182601.1,XM_005270162.1,XM_005270163.1;GL=PLEKHS1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown;AAC=PHE/ILE;PP=69/367,69/367,157/364,151/466,157/469,157/455;CDP=205,205,469,451,469,469;GS=21;PH=0.893;CP=0.009;CG=4.760;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=34;TFBS=V$PAX4_04;GESP=A:1962/T:11044;PAC=NP_001180363.1,NP_001180364.1,NP_079165.3,NP_872407.1,XP_005270219.1,XP_005270220.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,50 1/1:6:35,6,0 1/1:9:60,9,0</p>

      <p>10 126678087 rs111888382 C T 16.70 highCVG AC1=6;AF1=0.6827;DB;DP=36;DP4=3,5,13,14;FQ=21.4;MQ=11;PV4=0.7,0.29,1.9e-05,0.13;SNPEFF_AMINO_ACID_CHANGE=*986;SNPEFF_CODON_CHANGE=taG/taA;SNPEFF_EFFECT=SYNONYMOUS_STOP;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0149;DN=132;DA=C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=synonymous;FD=synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=stop;PP=446/446,446/446,986/986,514/514,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,421/421,421/421;CDP=1338,1338,2958,1542,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1263,1263;CP=1.000;CG=5.120;AA=C;DV=by-cluster;DSP=181;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:4:0,3,38 1/1:11:16,12,0 1/1:30:45,31,0</p>

      <p>10 126678092 rs73373169 G A 25.10 highCVG AC1=5;AF1=0.6731;DB;DP=36;DP4=2,5,12,13;FQ=29.5;MQ=12;PV4=0.43,1,1.3e-05,0.07;SNPEFF_AMINO_ACID_CHANGE=Q985*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0155;DN=131;DA=G/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=stop-gained;FD=stop-gained,stop-gained,stop-gained,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/stop;PP=445/446,445/446,985/986,513/514,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,420/421,420/421;CDP=1333,1333,2953,1537,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1258,1258;CP=1.000;CG=5.120;AA=G;DG;DV=by-cluster;DSP=176;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$MYB_Q6;GESP=G:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:3:0,5,41 1/1:17:22,18,0 1/1:28:49,28,0</p>

      <p>CTBP2 10 126678177 . G T 39.20 highCVG AC1=6;AF1=0.7163;DP=34;DP4=3,4,8,17;FQ=44.8;MQ=28;PV4=0.67,0.00015,8e-15,1;SNPEFF_AMINO_ACID_CHANGE=N956K;SNPEFF_CODON_CHANGE=aaC/aaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0013;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=unknown;AAC=ASN/LYS;PP=416/446,416/446,956/986,484/514,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,391/421,391/421;CDP=1248,1248,2868,1452,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1173,1173;GS=94;PH=0.613;CP=1.000;CG=3.120;AA=G;DSP=91;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$BACH1_01,V$TAXCREB_02;GESP=G:12998;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:5:1,0,7 0/1:4:0,4,98 1/1:24:24,24,0 1/1:33:60,33,0</p>

      <p>10 126678221 rs77752853 C A 43.30 highCVG AC1=6;AF1=0.7316;DB;DP=30;DP4=3,4,3,17;FQ=49.1;MQ=30;PV4=0.29,0.031,2.4e-12,0.15;SNPEFF_AMINO_ACID_CHANGE=A942S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=131;DA=C/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/SER;PP=402/446,402/446,942/986,470/514,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,377/421,377/421;CDP=1204,1204,2824,1408,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1129,1129;GS=99;PH=0.599;CP=0.998;CG=5.120;AA=C;DV=by-cluster;DSP=47;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:6:0,1,11 0/1:11:4,0,114 1/1:21:24,21,0 1/1:25:60,24,0</p>

      <p>10 126678241 rs113985797 G C 8.10 highCVG AC1=4;AF1=0.4981;DB;DP=26;DP4=4,7,2,13;FQ=10.6;MQ=39;PV4=0.35,1.5e-07,2.9e-13,1;SNPEFF_AMINO_ACID_CHANGE=A935G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=132;DA=G/C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=395/446,395/446,935/986,463/514,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,370/421,370/421;CDP=1184,1184,2804,1388,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1109,1109;GS=60;PH=0.016;CP=1.000;CG=5.120;AA=G;DV=by-frequency,by-cluster;DSP=27;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/0:4:0,2,39 0/0:6:0,6,121 1/1:10:17,18,0 0/1:10:34,0,4</p>

      <p>10 129903016 rs7071768 T C 19.60 midCVG AC1=5;AF1=0.5991;DB;DP=9;DP4=2,1,1,2;FQ=16.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N2363S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MKI67;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002417.4;VDB=0.0101;DN=116;DA=T/C;GM=NM_001145966.1,NM_002417.4,XM_005252683.1;GL=MKI67;FG=missense;FD=missense,missense,unknown;AAC=ASN/SER;PP=2003/2897,2363/3257,2166/3060;CDP=6008,7088,6497;GS=46;PH=0;CP=0.000;CG=-5.360;AA=C;HA=41.1;HE=46.0;HC=38.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2174;GESP=C:6499/T:6507;PPI=MKI67IP:983|SUMO2:748;PAC=NP_001139438.1,NP_002408.3,XP_005252740.1 GT:GQ:PL 0/1:18:19,0,17 0/1:18:14,0,35 1/1:3:25,3,0 0/1:3:0,0,0</p>

      <p>11 1507731 rs1881503 A G 14.10 lowCVG AC1=4;AF1=0.5387;DB;DP=5;DP4=1,1,2,1;FQ=9.13;MQ=60;PV4=1,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=L19P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MOB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172223.2;VDB=0.0097;DN=92;DA=A/G;GM=NM_001172223.1;GL=MOB2;FG=missense;FD=missense;AAC=LEU/PRO;PP=19/269;CDP=56;GS=98;PH=0;CP=0.823;CG=3.930;AA=G;HA=0.0;HE=12.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;PAC=NP_001165694.1 GT:GQ:PL 0/0:3:0,3,21 0/1:3:16,3,0 1/1:4:40,6,0 0/0:3:0,3,27</p>

      <p>11 4870261 rs11602499 G C 80.20 midCVG AC1=5;AF1=0.6315;DB;DP=12;DP4=3,1,6,1;FQ=55.7;MQ=59;PV4=1,0.086,0.24,1;SNPEFF_AMINO_ACID_CHANGE=Q60E;SNPEFF_CODON_CHANGE=Cag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR51S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004758.1;VDB=0.0002;DN=120;DA=G/C;GM=NM_001004758.1;GL=OR51S1;FG=missense;FD=missense;AAC=GLN/GLU;PP=60/324;CDP=178;GS=29;PH=0;CP=0.205;CG=4.850;AA=C;HA=12.2;HE=36.9;HC=2.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=139280;KP=hsa04740:OlfactoryTransduction;GESP=C:4937/G:8061;PAC=NP_001004758.1 GT:GQ:PL 0/0:8:0,12,95 0/1:3:0,0,0 1/1:4:24,3,0 1/1:17:100,18,0</p>

      <p>11 4945233 rs35264256 C T 27.60 lowCVG AC1=5;AF1=0.6618;DB;DP=7;DP4=1,2,2,2;FQ=7.91;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=E113K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR51G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005237.1;VDB=0.0141;DN=126;DA=C/T;GM=NM_001005237.1;GL=OR51G1;FG=missense;FD=missense;AAC=GLU/LYS;PP=113/322;CDP=337;GS=56;PH=1;CP=0.508;CG=4.200;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=64308;KP=hsa04740:OlfactoryTransduction;TFBS=V$AP1_Q6;GESP=T:1207/C:11791;PAC=NP_001005237.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 0/1:39:49,0,37</p>

      <p>11 5528918 rs2227271 C T 10.70 lowCVG AC1=5;AF1=0.5785;DB;DP=5;DP4=0,1,2,0;FQ=-6.4;MQ=60;PV4=0.33,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=R624Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBQLN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017481.2;VDB=0.0061;DN=98;DA=C/T;GM=NM_017481.2;GL=UBQLN3;FG=missense;FD=missense;AAC=ARG/GLN;PP=624/656;CDP=1871;GS=43;PH=0.045;CP=0.213;CG=1.230;AA=C;HA=6.4;HE=37.7;HC=6.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1907;GESP=T:3709/C:9287;PAC=NP_059509.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:45,6,0 0/0:4:0,3,26</p>

      <p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

      <p>11 5906205 rs4757987 G A 63.50 midCVG AC1=4;AF1=0.4597;DB;DP=12;DP4=3,4,2,3;FQ=66.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R228H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52E4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005165.1;VDB=0.0135;DN=111;DA=G/A;GM=NM_001005165.1;GL=OR52E4;FG=missense;FD=missense;AAC=ARG/HIS;PP=228/313;CDP=683;GS=29;PH=0.003;CP=0.023;CG=0.047;AA=G;HA=47.5;HC=42.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=64694;KP=hsa04740:OlfactoryTransduction;GESP=A:5225/G:7769;PAC=NP_001005165.1 GT:GQ:PL 0/0:6:0,6,44 0/1:4:25,3,0 1/1:8:86,12,0 0/0:13:0,15,95</p>

      <p>11 6007613 rs4354673 T G 168 highCVG AC1=6;AF1=0.6957;DB;DP=18;DP4=0,3,5,9;FQ=4.33;MQ=48;PV4=0.51,1,0.054,1;SNPEFF_AMINO_ACID_CHANGE=K183T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0083;DN=111;DA=T/G;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=LYS/THR;PP=183/330;CDP=548;GS=78;PH=0.549;CP=0.001;CG=-0.167;AA=T;HA=44.5;HE=13.2;HC=10.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=166102;KP=hsa04740:OlfactoryTransduction;GESP=G:8663/T:3675;PAC=NP_001005173.2 GT:GQ:PL 0/1:11:38,0,11 0/1:3:0,6,34 1/1:15:87,15,0 1/1:15:91,15,0</p>

      <p>11 6007680 rs4436524 A G 189 highCVG AC1=6;AF1=0.7132;DB;DP=17;DP4=0,2,8,7;FQ=-4.85;MQ=47;PV4=0.47,0.4,0.081,0.3;SNPEFF_AMINO_ACID_CHANGE=C161R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0140;DN=111;DA=A/G/T;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=CYS/ARG;PP=161/330;CDP=481;GS=180;PH=0;CP=0.012;CG=0.241;AA=G;HA=43.3;HE=15.3;HC=11.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=166169;KP=hsa04740:OlfactoryTransduction;GESP=G:9200/A:3246;PAC=NP_001005173.2 GT:GQ:PL 0/1:16:39,0,17 0/1:5:0,3,21 1/1:13:53,12,0 1/1:28:144,27,0</p>

      <p>11 6341684 rs2682123 C G 35.60 lowCVG AC1=6;AF1=0.7558;DB;DP=5;DP4=0,1,2,2;FQ=-21.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R8P;SNPEFF_CODON_CHANGE=cGg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRKCDBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145040.2;VDB=0.0113;DN=100;DA=C/G;GM=NM_145040.2;GL=PRKCDBP;FG=missense;FD=missense;AAC=ARG/PRO;PP=8/262;CDP=23;GS=103;PH=0;CP=0.004;CG=2.010;AA=G;HA=13.6;HE=1.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=362;CPG=6340446-6341909;GESP=G:11875/C:705;PAC=NP_659477.2 GT:GQ:PL 0/1:10:28,0,9 0/1:3:0,0,0 1/1:6:43,6,0 0/1:3:0,0,0</p>

      <p>11 6816137 . T A 48.90 midCVG AC1=5;AF1=0.5835;DP=13;DP4=1,1,5,0;FQ=-7.18;MQ=60;PV4=0.29,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=K268M;SNPEFF_CODON_CHANGE=aAg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6A2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003696.2;VDB=0.0075;GM=NM_003696.2;GL=OR6A2;FG=missense;FD=unknown;AAC=LYS/MET;PP=268/328;CDP=803;GS=95;PH=0.999;CP=0.995;CG=3.850;AA=T;DSP=111784;KP=hsa04740:OlfactoryTransduction;GESP=T:12994;PAC=NP_003687.2 GT:GQ:PL 0/1:20:34,0,17 0/1:3:0,0,1/1:7:55,9,0 0/1:3:0,3,13</p>

      <p>11 12963012 . G A 23.10 midCVG AC1=2;AF1=0.2817;DP=11;DP4=6,1,0,4;FQ=24.4;MQ=60;PV4=0.015,0.11,1,0.24;VDB=0.0039;GM=NM_021961.5,XM_005253090.1,XM_005253091.1,XM_005253092.1,XM_005253093.1,XM_005253094.1,XM_005253095.1,XM_005253096.1,XM_005253097.1,XM_005253098.1;GL=TEAD1;FG=3-prime-UTR;FD=unknown;CP=0.998;CG=0.679;AA=G;DSP=4357;PPI=YAP1:959|SRF:846|MEF2C:845 GT:GQ:PL 0/0:6:0,3,18 0/0:8:0,6,44 1/1:6:66,12,0 0/0:14:0,12,78</p>

      <p>11 17593709 rs7106548 T C 84.60 midCVG AC1=4;AF1=0.5314;DB;DP=11;DP4=3,1,2,5;FQ=47;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S692P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0167;DN=116;DA=T/C;GM=NM_001277269.1;GL=OTOG;FG=missense;FD=missense;AAC=SER/PRO;PP=692/2926;CDP=2074;GS=74;CP=0.003;CG=-4.210;AA=C;HA=30.5;HE=39.7;HC=22.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=43;PAC=NP_001264198.1 GT:GQ:PL 0/1:17:15,0,19 0/0:7:0,9,68 1/1:15:113,18,0 0/1:3:0,0,0</p>

      <p>11 17632651 rs7111528 C T 102 midCVG AC1=5;AF1=0.5888;DB;DP=10;DP4=1,2,4,3;FQ=15.1;MQ=60;PV4=1,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=T1947M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0076;DN=116;DA=C/T;GM=NM_001277269.1,XM_005252894.1;GL=OTOG;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=1947/2926,953/1509;CDP=5840,2858;GS=81;PH=0.011;CP=0.000;CG=-0.815;AA=T;HA=48.7;HE=42.3;HC=17.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=320;PAC=NP_001264198.1,XP_005252951.1 GT:GQ:PL 0/0:4:0,6,33 0/1:3:0,3,25 1/1:5:50,6,0 1/1:13:99,15,0</p>

      <p>11 18735947 rs7125943 T C 65 lowCVG AC1=4;AF1=0.4853;DB;DP=7;DP4=1,1,3,2;FQ=17.9;MQ=60;PV4=1,0.094,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M559V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGSF22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173588.3;VDB=0.0024;DN=116;DA=T/C;GM=NM_173588.3;GL=IGSF22;FG=missense;FD=missense;AAC=MET/VAL;PP=559/1327;CDP=1675;GS=21;PH=0;CP=0.001;CG=-2.120;AA=C;HA=15.3;HE=22.1;HC=20.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=19;GESP=C:9686/T:2872;PAC=NP_775859.3 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,25 1/1:11:105,15,0 0/0:4:0,3,30</p>

      <p>11 20065673 rs3802799 G T 174 highCVG AC1=5;AF1=0.6294;DB;DP=19;DP4=1,3,8,7;FQ=19.4;MQ=60;PV4=0.58,0.057,1,1;SNPEFF_AMINO_ACID_CHANGE=E1041D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NAV2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001244963.1;VDB=0.0108;DN=107;DA=G/T;GM=NM_001111018.1,NM_001111019.2,NM_001244963.1,NM_145117.4,NM_182964.5,XM_005253212.1,XM_005253213.1,XM_005253214.1,XM_005253215.1;GL=NAV2;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=GLU/ASP;PP=954/2366,104/1494,1041/2489,1018/2430,1018/2433,104/1519,104/1494,104/1482,17/1432;CDP=2862,312,3123,3054,3054,312,312,312,51;GS=45;PH=0.999;CP=0.994;CG=4.500;AA=G;HA=0.3;HE=14.8;HC=49.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=146;GESP=T:1921/G:11085;PAC=NP_001104488.1,NP_001104489.1,NP_001231892.1,NP_660093.2,NP_892009.3,XP_005253269.1,XP_005253270.1,XP_005253271.1,XP_005253272.1 GT:GQ:PL 0/1:36:64,0,35 0/1:4:0,3,23 1/1:4:21,3,0 1/1:6:132,6,0</p>

      <p>11 28057837 . C G 72.40 midCVG AC1=5;AF1=0.6429;DP=14;DP4=2,4,5,1;FQ=57.8;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D775H;SNPEFF_CODON_CHANGE=Gac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF18A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_031217.3;VDB=0.0011;GM=NM_031217.3;GL=KIF18A;FG=missense;FD=unknown;AAC=ASP/HIS;PP=775/899;CDP=2323;GS=81;PH=0.664;CP=0.965;CG=4.630;AA=C;DSP=74;GESP=C:13000;PAC=NP_112494.3 GT:GQ:PL 0/1:16:11,0,67 0/1:3:0,0,0 1/1:8:65,9,0 0/1:28:37,0,27</p>

      <p>11 47270255 rs2167079 C T 15.90 midCVG AC1=3;AF1=0.4192;DB;DP=11;DP4=4,2,1,2;FQ=18.3;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R29Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001610.2;VDB=0.0055;DN=96;DA=C/T;GM=NM_001131064.1,NM_001251934.1,NM_001251935.1,NM_001610.2,XM_005252982.1,XM_005252983.1,XM_005252984.1,XM_005252985.1;GL=ACP2/NR1H3;FG=missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,5-prime-UTR,missense;FD=missense,utr-variant-5-prime,utr-variant-5-prime,missense,unknown,unknown,unknown,unknown;AAC=ARG/GLN,none,none,ARG/GLN,ARG/GLN,ARG/GLN,none,ARG/GLN;PP=29/161,NA,NA,29/424,29/392,29/361,NA,29/218;CDP=86,NA,NA,86,86,86,NA,86;GS=43,NA,NA,43,43,43,NA,43;PH=0.003;CP=0.987;CG=5.690;AA=T;HA=38.3;HE=31.2;HC=27.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;KP=hsa00740:RiboflavinMetabolism|hsa04142:Lysosome|hsa03320:PPARSignalingPathway;CPG=47270235-47270633;TFBS=V$AHR_01;GESP=T:4930/C:8048;PPI=RXRA:937|RXRB:900;PAC=NP_001124536.1,NA,NA,NP_001601.1,XP_005253039.1,XP_005253040.1,NA,XP_005253042.1 GT:GQ:PL 0/1:4:24,3,0 0/0:6:0,6,30 1/1:3:37,6,0 0/0:11:0,12,67</p>

      <p>11 48346547 rs79042268 C T 999 highCVG AC1=5;AF1=0.5818;DB;DP=30;DP4=1,10,12,7;FQ=67.7;MQ=60;PV4=0.0067,1,0.23,1;SNPEFF_AMINO_ACID_CHANGE=P19S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0115;DN=131;DA=C/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=PRO/SER;PP=19/330;CDP=55;GS=74;CP=0.000;CG=-5.180;AA=C;DG;DV=by-cluster;DSP=157792;KP=hsa04740:OlfactoryTransduction;GESP=C:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:29:93,0,27 0/1:14:10,0,42 1/1:5:108,5,0 0/1:40:84,0,38</p>

      <p>11 48347306 rs73465911 G T 151 highCVG AC1=5;AF1=0.6041;DB;DP=29;DP4=11,2,7,7;FQ=149;MQ=60;PV4=0.1,0.051,1,1;SNPEFF_AMINO_ACID_CHANGE=V272F;SNPEFF_CODON_CHANGE=Gtt/Ttt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004702.1;VDB=0.0060;DN=131;DA=G/T;GM=NM_001004702.1;GL=OR4C3;FG=missense;FD=missense;AAC=VAL/PHE;PP=272/330;CDP=814;GS=50;PH=0.175;CP=0.997;CG=5.970;AA=G;DG;DV=by-frequency,by-cluster;DSP=158551;KP=hsa04740:OlfactoryTransduction;TFBS=V$RFX1_01;GESP=G:12998;PAC=NP_001004702.1 GT:GQ:PL 0/1:47:101,0,46 0/1:16:11,0,66 1/1:8:46,9,0 0/1:42:37,0,79</p>

      <p>11 48367136 rs78927905 C T 138 highCVG AC1=4;AF1=0.5195;DB;DP=18;DP4=4,1,4,8;FQ=35.5;MQ=60;PV4=0.13,0.48,0.27,1;VDB=0.0095;DN=131;DA=C/T;GM=none;FG=intergenic;FD=unknown;CP=0.566;CG=2.210;AA=C;DG;DV=by-frequency,by-cluster;DSP=178381 GT:GQ:PL 0/1:33:35,0,31 0/1:19:19,0,18 1/1:4:127,5,0 0/0:3:0,3,26</p>

      <p>11 48367850 rs147514008 C T 7.59 highCVG AC1=5;AF1=0.5542;DB;DP=16;DP4=7,2,6,1;FQ=10.5;MQ=42;PV4=1,0.43,3.2e-08,0.02;VDB=0.0100;DN=134;DA=C/T;GM=none;FG=intergenic;FD=unknown;CP=0.024;AA=C;RM=L1PA4;DSP=179095 GT:GQ:PL 0/1:7:5,0,23 0/0:19:0,21,86 1/1:6:17,9,0 1/1:4:32,6,0</p>

      <p>11 48373885 rs76292854 A G 143 highCVG AC1=4;AF1=0.4681;DB;DP=21;DP4=5,3,7,5;FQ=73.3;MQ=60;PV4=1,1,1,1;VDB=0.0169;DN=131;DA=A/G;GM=none;FG=intergenic;FD=unknown;CP=0.984;CG=1.060;AA=G;DG;DV=by-frequency,by-cluster;DSP=185130 GT:GQ:PL 0/1:59:73,0,55 0/0:5:0,6,39 1/1:14:115,18,0 0/0:3:0,3,22</p>

      <p>11 48373900 rs77877396 A G 999 highCVG AC1=4;AF1=0.5364;DB;DP=29;DP4=6,5,11,5;FQ=90.8;MQ=60;PV4=0.69,1,1,0.48;VDB=0.0156;DN=131;DA=A/G;GM=none;FG=intergenic;FD=unknown;CP=0.051;CG=-10.700;AA=G;DG;DV=by-frequency,by-cluster;DSP=185145 GT:GQ:PL 0/1:88:91,0,86 0/1:28:51,0,25 1/1:6:100,7,0 0/1:3:0,3,20</p>

      <p>11 48373936 rs76304554 A G 999 highCVG AC1=5;AF1=0.6201;DB;DP=33;DP4=6,6,12,8;FQ=118;MQ=60;PV4=0.72,1,1,0.47;VDB=0.0119;DN=131;DA=A/G;GM=none;FG=intergenic;FD=unknown;CP=0.368;CG=0.952;AA=G;DG;DV=by-frequency,by-cluster;DSP=185181 GT:GQ:PL 0/1:63:89,0,62 0/1:38:48,0,37 1/1:26:139,27,0 0/1:14:9,0,60</p>

      <p>11 48373976 rs80225178 T G 999 highCVG AC1=5;AF1=0.5853;DB;DP=32;DP4=7,6,9,9;FQ=169;MQ=60;PV4=1,0.046,1,0.12;VDB=0.0076;DN=131;DA=T/G;GM=none;FG=intergenic;FD=unknown;CP=0.593;CG=0.647;AA=G;DG;DV=by-frequency,by-cluster;DSP=185221 GT:GQ:PL 0/1:72:68,0,83 0/1:27:50,0,25 1/1:19:140,21,0 0/1:5:0,1,101</p>

      <p>11 48373986 rs12420054 A G 999 highCVG AC1=5;AF1=0.6237;DB;DP=34;DP4=7,7,11,9;FQ=173;MQ=60;PV4=1,0.36,1,0.0065;VDB=0.0131;DN=120;DA=A/G;GM=none;FG=intergenic;FD=unknown;CP=0.780;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=185231 GT:GQ:PL 0/1:79:98,0,78 0/1:38:58,0,37 1/1:20:127,21,0 0/1:30:25,0,99</p>

      <p>11 50003191 rs4598671 C A 45.80 midCVG AC1=4;AF1=0.4411;DB;DP=12;DP4=0,6,3,2;FQ=48.3;MQ=57;PV4=0.061,0.0084,0.036,1;SNPEFF_AMINO_ACID_CHANGE=V283L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005270.4;VDB=0.0080;DN=111;DA=C/A;GM=NM_001005270.2;GL=OR4C12;FG=missense;FD=missense;AAC=VAL/LEU;PP=283/310;CDP=847;GS=32;PH=0;CP=0.806;CG=2.980;AA=A;HA=23.3;HE=42.5;HC=44.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=171898;KP=hsa04740:OlfactoryTransduction;GESP=A:6703/C:6291;PAC=NP_001005270.1 GT:GQ:PL 0/0:4:0,3,29 0/0:6:0,6,57 1/1:8:80,12,0 0/1:12:10,0,54</p>

      <p>11 51411914 rs61890334 T C 57.90 midCVG AC1=4;AF1=0.4616;DB;DP=12;DP4=4,3,1,4;FQ=60.5;MQ=55;PV4=0.29,1,1,0.074;SNPEFF_AMINO_ACID_CHANGE=Y161C;SNPEFF_CODON_CHANGE=tAc/tGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005272.3;VDB=0.0086;DN=129;DA=T/C;GM=NM_001005272.3;GL=OR4A5;FG=missense;FD=missense;AAC=TYR/CYS;PP=161/316;CDP=482;GS=194;PH=0.027;CP=0.042;CG=1.930;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1032140;KP=hsa04740:OlfactoryTransduction;GESP=C:765/T:12229;PAC=NP_001005272.3 GT:GQ:PL 0/0:13:0,15,95 0/0:4:0,3,25 1/1:6:63,9,0 0/1:4:40,2,0</p>

      <p>11 55110861 rs200705953 C A 60.70 highCVG AC1=3;AF1=0.3502;DB;DP=28;DP4=6,14,1,5;FQ=62.6;MQ=60;PV4=1,1,1,2.5e-05;SNPEFF_AMINO_ACID_CHANGE=A62D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0110;DN=137;DA=C/A;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=ALA/ASP;PP=62/329;CDP=185;GS=126;PH=0.109;CP=0.839;CG=2.570;AA=C;DV=by-frequency;DSP=45527;KP=hsa04740:OlfactoryTransduction;GESP=C:12994;PAC=NP_001005274.1 GT:GQ:PL 0/0:9:0,9,61 0/0:36:0,36,110 1/1:5:57,9,0 0/1:50:50,0,67</p>

      <p>11 55111580 rs76200064 G A 19.70 highCVG AC1=2;AF1=0.2824;DB;DP=16;DP4=5,4,5,0;FQ=21;MQ=56;PV4=0.22,0.00029,0.0037,1;SNPEFF_AMINO_ACID_CHANGE=E302K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0049;DN=131;DA=G/A;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=GLU/LYS;PP=302/329;CDP=904;GS=56;PH=0;CP=0.002;CG=0.439;AA=G;DG;DSP=46246;KP=hsa04740:OlfactoryTransduction;GESP=G:12990;PAC=NP_001005274.1 GT:GQ:PL 0/0:17:0,15,113 0/0:6:0,3,26 1/1:8:64,15,0 0/0:11:0,9,66</p>

      <p>11 55111602 rs78987961 A T 121 highCVG AC1=3;AF1=0.3854;DB;DP=23;DP4=8,4,9,2;FQ=123;MQ=56;PV4=0.64,0.049,0.0029,1;SNPEFF_AMINO_ACID_CHANGE=K309I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0051;DN=131;DA=A/T;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=LYS/ILE;PP=309/329;CDP=926;GS=102;PH=0;CP=0.000;CG=-6.030;AA=A;DG;DSP=46268;KP=hsa04740:OlfactoryTransduction;GESP=A:12988;PAC=NP_001005274.1GT:GQ:PL 0/0:14:0,15,112 0/0:9:0,9,61 1/1:10:85,15,0 0/1:65:84,0,60</p>

      <p>11 55111607 rs77602694 G T 129 highCVG AC1=3;AF1=0.3814;DB;DP=25;DP4=10,4,9,2;FQ=131;MQ=56;PV4=0.66,0.32,0.0014,1;SNPEFF_AMINO_ACID_CHANGE=V311L;SNPEFF_CODON_CHANGE=Gta/Tta;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4A16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005274.1;VDB=0.0051;DN=131;DA=G/T;GM=NM_001005274.1;GL=OR4A16;FG=missense;FD=missense;AAC=VAL/LEU;PP=311/329;CDP=931;GS=32;PH=0;CP=0.001;CG=-3.470;AA=G;DG;DSP=46273;KP=hsa04740:OlfactoryTransduction;GESP=G:12986;PAC=NP_001005274.1 GT:GQ:PL 0/0:20:0,21,137 0/0:9:0,9,65 1/1:10:89,15,0 0/1:63:88,0,58</p>

      <p>11 55322606 rs17581700 T C 30.70 highCVG AC1=3;AF1=0.3416;DB;DP=16;DP4=2,10,2,2;FQ=32.4;MQ=60;PV4=0.24,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=L275S;SNPEFF_CODON_CHANGE=tTg/tCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4C15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001920.1;VDB=0.0127;DN=123;DA=T/C;GM=NM_001001920.1;GL=OR4C15;FG=missense;FD=missense;AAC=LEU/SER;PP=275/371;CDP=824;GS=145;PH=1;CP=0.921;CG=5.020;AA=T;HA=0.0;HE=7.1;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=257272;KP=hsa04740:OlfactoryTransduction;GESP=C:567/T:12427;PAC=NP_001001920.1 GT:GQ:PL 0/0:10:0,9,72 0/0:16:0,15,91 1/1:4:66,9,0 0/1:10:10,0,64</p>

      <p>11 55406524 rs61896975 C T 78.80 highCVG AC1=4;AF1=0.4977;DB;DP=15;DP4=2,3,1,9;FQ=54.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R231C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4P4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004124.1;VDB=0.0025;DN=129;DA=C/T;GM=NM_001004124.1;GL=OR4P4;FG=missense;FD=missense;AAC=ARG/CYS;PP=231/313;CDP=691;GS=180;PH=0.407;CP=0.000;CG=-1.540;AA=C;DG;DV=by-cluster,by-1000G;DSP=244337;KP=hsa04740:OlfactoryTransduction;GESP=T:719/C:11695;PAC=NP_001004124.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,55 1/1:4:102,6,0 0/1:19:17,0,37</p>

      <p>11 55541620 rs7124871 G T 59.50 midCVG AC1=4;AF1=0.4901;DB;DP=13;DP4=3,4,4,2;FQ=62.4;MQ=59;PV4=0.59,1,0.15,1;SNPEFF_AMINO_ACID_CHANGE=R236L;SNPEFF_CODON_CHANGE=cGc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5D13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001967.1;VDB=0.0105;DN=116;DA=G/A/T;GM=NM_001001967.1;GL=OR5D13;FG=missense;FD=missense;AAC=ARG/LEU;PP=236/315;CDP=707;GS=102;PH=0.01;CP=0.000;CG=2.850;AA=G;HA=0.0;HE=7.6;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=109241;KP=hsa04740:OlfactoryTransduction;GESP=T:576/G:12416;PAC=NP_001001967.1 GT:GQ:PL 0/1:3:0,0,0 0/0:13:0,15,100 1/1:9:74,12,0 0/1:31:29,0,37</p>

      <p>11 55703861 rs17597625 T C 22.50 midCVG AC1=4;AF1=0.508;DB;DP=12;DP4=3,1,4,0;FQ=25.4;MQ=60;PV4=1,0.019,1,0.18;SNPEFF_AMINO_ACID_CHANGE=R6G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006637.1;VDB=0.0044;DN=123;DA=T/C;GM=NM_006637.1;GL=OR5I1;FG=missense;FD=missense;AAC=ARG/GLY;PP=6/315;CDP=16;GS=125;PH=0;CP=0.238;CG=4.060;AA=C;HA=0.6;HE=10.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=43693;KP=hsa04740:OlfactoryTransduction;GESP=C:787/T:12195;PAC=NP_006628.1 GT:GQ:PL 0/0:8:0,9,62 0/0:3:0,3,22 1/1:4:30,6,0 1/1:4:38,6,0</p>

      <p>11 56468044 rs3975155 T G 113 highCVG AC1=5;AF1=0.5811;DB;DP=23;DP4=3,3,8,7;FQ=41.5;MQ=39;PV4=1,0.053,0.014,0.27;SNPEFF_AMINO_ACID_CHANGE=F61V;SNPEFF_CODON_CHANGE=Ttc/Gtc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=108;DA=T/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=PHE/VAL;PP=61/306;CDP=181;GS=50;PH=0.108;CP=0.998;CG=3.370;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482114;TFBS=V$POU3F2_01;GESP=T:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:29:25,0,52 0/1:14:60,0,12 1/1:13:72,15,0 0/1:4:0,2,18</p>

      <p>11 56468047 rs2865519 A G 112 highCVG AC1=5;AF1=0.5869;DB;DP=23;DP4=3,4,8,7;FQ=50.7;MQ=40;PV4=1,1,0.0065,0.36;SNPEFF_AMINO_ACID_CHANGE=T62A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=101;DA=A/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=THR/ALA;PP=62/306;CDP=184;GS=58;PH=0.004;CP=0.961;CG=4.540;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482111;TFBS=V$POU3F2_01;GESP=A:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:28:24,0,64 0/1:11:62,0,9 1/1:13:70,15,0 0/1:4:0,2,18</p>

      <p>11 56756399 rs10896563 G T 28.10 midCVG AC1=3;AF1=0.3852;DB;DP=9;DP4=2,2,0,4;FQ=29.8;MQ=60;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G4V;SNPEFF_CODON_CHANGE=gGa/gTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5AK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005323.1;VDB=0.0173;DN=120;DA=G/T;GM=NM_001005323.1;GL=OR5AK2;FG=missense;FD=missense;AAC=GLY/VAL;PP=4/310;CDP=11;GS=109;PH=0.262;CP=0.000;CG=-4.160;AA=G;HA=5.0;HC=41.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=193759;KP=hsa04740:OlfactoryTransduction;GESP=T:720/G:12274;PAC=NP_001005323.1 GT:GQ:PL 0/0:7:0,6,44 0/0:7:0,6,36 1/1:7:69,12,0 0/0:3:0,0,0</p>

      <p>11 57982763 rs1993088 A G 15.60 lowCVG AC1=5;AF1=0.5967;DB;DP=7;DP4=3,1,1,2;FQ=18.5;MQ=44;PV4=0.49,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N183D;SNPEFF_CODON_CHANGE=Aac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1S1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004458.1;DN=92;DA=A/G;GM=NM_001004458.1;GL=OR1S1;FG=missense;FD=missense;AAC=ASN/ASP;PP=183/326;CDP=547;GS=23;PH=0;CP=0.017;CG=-1.100;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=35861;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:3379/A:9603;PAC=NP_001004458.1 GT:GQ:PL 0/1:21:18,0,28 0/1:3:0,0,0 1/1:3:19,3,0 0/1:21:17,0,32</p>

      <p>11 59940599 rs7232 T A 17.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=0,3,1,1;FQ=18;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T213S;SNPEFF_CODON_CHANGE=Act/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MS4A6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001247999.1;VDB=0.0040;DN=52;DA=T/A;GM=NM_001247999.1,NM_022349.3,NM_152851.2,NM_152852.2,XM_005274177.1,XM_005274178.1,XM_005274179.1,XM_005274180.1,XM_005274181.1;GL=MS4A6A;FG=missense;FD=unknown,unknown,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=THR/SER,THR/SER,ASN/ILE,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER;PP=213/254,185/226,150/179,185/249,213/277,213/277,185/249,168/232,168/209;CDP=637,553,449,553,637,637,553,502,502;GS=58,58,149,58,58,58,58,58,58;PH=0.827;CP=0.001;CG=-5.110;AA=T;HA=1.8;HE=44.6;HC=8.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=4;GESP=A:3608/T:9384;PAC=NP_001234928.1,NP_071744.2,NP_690590.1,NP_690591.1,XP_005274234.1,XP_005274235.1,XP_005274236.1,XP_005274237.1,XP_005274238.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:54,6,0 0/0:9:0,9,55</p>

      <p>11 61165741 rs10897158 G C 74.40 midCVG AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1 GT:GQ:PL 0/1:10:35,0,8 0/0:6:0,9,70 1/1:4:24,3,0 1/1:8:61,9,0</p>

      <p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

      <p>11 67173164 . G T 39.60 lowCVG AC1=5;AF1=0.5973;DP=8;DP4=2,0,0,5;FQ=-7.3;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q153H;SNPEFF_CODON_CHANGE=caG/caT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198517.3;VDB=0.0003;GM=NM_001256508.1,NM_198517.3,NR_046266.1;GL=TBC1D10C;FG=missense,missense,non-coding-exon;FD=unknown;AAC=GLN/HIS,GLN/HIS,none;PP=153/311,153/447,NA;CDP=459,459,686;GS=24,24,NA;PH=0.071;CP=0.996;CG=0.600;AA=G;DSP=9;GESP=G:12990;PAC=NP_001243437.1,NP_940919.1,NA GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,28 1/1:12:78,15,0 0/1:3:0,0,0</p>

      <p>11 69466030 . G A 20.20 lowCVG AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1 GT:GQ:PL 0/0:6:0,3,17 0/0:6:0,3,14 1/1:4:61,9,0 0/0:11:0,9,66</p>

      <p>11 71169547 rs2276360 G C 15.50 lowCVG AC1=5;AF1=0.5737;DB;DP=8;DP4=2,2,0,3;FQ=19.2;MQ=60;PV4=0.43,0.24,1,0.37;SNPEFF_AMINO_ACID_CHANGE=V74L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NADSYN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018161.4;VDB=0.0099;DN=100;DA=G/C;GM=NM_018161.4;GL=NADSYN1;FG=missense;FD=missense;AAC=VAL/LEU;PP=74/707;CDP=220;GS=32;PH=0;CP=0.037;CG=-2.340;AA=C;HA=33.2;HE=27.2;HC=40.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=44;KP=hsa00760:NicotinateAndNicotinamideMetabolism|hsa01100:MetabolicPathways;GESP=C:8267/G:4721;PAC=NP_060631.2 GT:GQ:PL 0/1:3:21,3,0 0/1:3:0,0,0 1/1:5:36,6,0 0/0:10:0,12,81</p>

      <p>11 74413955 rs35903991 G A 36.30 midCVG AC1=3;AF1=0.3987;DB;DP=12;DP4=3,2,0,5;FQ=38.3;MQ=60;PV4=0.17,1,1,0.15;SNPEFF_AMINO_ACID_CHANGE=P335L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHRDL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015424.4;VDB=0.0091;DN=126;DA=G/A;GM=NM_001278473.1,NM_015424.4,XM_005273884.1;GL=CHRDL2;FG=missense;FD=unknown,missense,unknown;AAC=PRO/LEU;PP=335/430,335/452,335/415;CDP=1004;GS=98;PH=1;CP=0.010;CG=4.150;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=58;TFBS=V$CEBP_C,V$LUN1_01,V$MIF1_01;GESP=A:141/G:12845;PAC=NP_001265402.1,NP_056239.3,XP_005273941.1 GT:GQ:PL 0/0:12:0,12,70 0/1:3:0,0,0 1/1:10:78,15,0 0/0:5:0,3,22</p>

      <p>11 75152243 rs571353 C T 59.30 midCVG AC1=5;AF1=0.5871;DB;DP=10;DP4=4,1,4,1;FQ=40.4;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=A480T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDPD5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030792.6;VDB=0.0080;DN=83;DA=C/T;GM=NM_030792.6,XM_005274313.1,XM_005274314.1,XM_005274315.1,XM_005274316.1,XM_005274317.1,XM_005274318.1,XM_005274319.1;GL=GDPD5;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=480/606,480/607,480/607,480/607,480/607,480/606,235/362,235/361;CDP=1438,1438,1438,1438,1438,1438,703,703;GS=58;PH=0;CP=0.014;CG=-0.094;HA=44.1;HE=26.6;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8514/C:4472;PAC=NP_110419.5,XP_005274370.1,XP_005274371.1,XP_005274372.1,XP_005274373.1,XP_005274374.1,XP_005274375.1,XP_005274376.1 GT:GQ:PL 0/1:7:30,0,4 0/0:9:0,12,77 1/1:4:26,3,0 1/1:5:48,6,0</p>

      <p>11 76912636 rs2276288 A T 117 midCVG AC1=5;AF1=0.6164;DB;DP=13;DP4=2,3,6,2;FQ=39.5;MQ=60;PV4=0.29,1,1,0.33;SNPEFF_AMINO_ACID_CHANGE=S1666C;SNPEFF_CODON_CHANGE=Agt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000260.3;VDB=0.0049;DN=100;DA=A/T;GM=NM_000260.3,NM_001127180.1,XM_005274011.1,XM_005274012.1;GL=MYO7A;FG=missense;FD=missense,missense,unknown,unknown;AAC=SER/CYS;PP=1666/2216,1628/2176,1666/2218,1627/2177;CDP=4996,4882,4996,4879;GS=112;PH=0;CP=0.581;CG=3.460;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=48;GESP=T:7510/A:5238;PPI=USH1C:846;PAC=NP_000251.3,NP_001120652.1,XP_005274068.1,XP_005274069.1 GT:GQ:PL 0/1:4:59,0,1 0/1:17:12,0,58 1/1:11:89,12,0 0/1:4:0,3,19</p>

      <p>11 77909014 rs2510044 G A 62.90 highCVG AC1=4;AF1=0.528;DB;DP=28;DP4=0,20,3,5;FQ=65.9;MQ=60;PV4=0.017,1,1,0.089;SNPEFF_AMINO_ACID_CHANGE=V236M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP35;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020798.2;VDB=0.0191;DN=100;DA=G/A;GM=NM_020798.2;GL=USP35;FG=missense;FD=missense;AAC=VAL/MET;PP=236/1019;CDP=706;GS=21;PH=0.999;CP=0.992;CG=4.820;AA=G;HA=0.4;HE=10.7;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=33;TFBS=V$NRSF_01;GESP=A:1344/G:11488;PAC=NP_065849.1 GT:GQ:PL 0/1:20:20,0,19 0/1:3:0,3,25 1/1:5:53,6,0 0/1:36:33,0,76</p>

      <p>11 87013438 rs2276102 G A 98 midCVG AC1=6;AF1=0.7212;DB;DP=11;DP4=0,2,5,3;FQ=-16.6;MQ=60;PV4=0.44,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=G218R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM135;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022918.3;VDB=0.0208;DN=100;DA=G/A;GM=NM_001168724.1,NM_022918.3,NR_033149.1;GL=TMEM135;FG=missense,missense,non-coding-exon;FD=missense,missense,nc-transcript-variant;AAC=GLY/ARG,GLY/ARG,none;PP=196/437,218/459,NA;CDP=586,652,820;GS=125,125,NA;PH=0;CP=0.664;CG=4.310;AA=A;HA=43.6;HE=31.1;HC=46.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;GESP=A:8629/G:4371;PAC=NP_001162195.1,NP_075069.3,NA GT:GQ:PL 0/1:14:15,0,14 0/1:3:0,0,0 1/1:5:26,3,0 0/1:3:93,0,0</p>

      <p>11 89531540 rs12417980 C T 84.10 highCVG AC1=6;AF1=0.7306;DB;DP=18;DP4=3,3,3,9;FQ=8.51;MQ=21;PV4=0.34,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G373R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM49;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020358.2;VDB=0.0085;DN=120;DA=C/T;GM=NM_020358.2;GL=TRIM49;FG=missense;FD=missense;AAC=GLY/ARG;PP=373/453;CDP=1117;GS=125;PH=0.834;CP=0.000;CG=-1.800;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=258;GESP=T:3992/C:8964;PAC=NP_065091.1 GT:GQ:PL 0/1:4:13,0,1 0/1:3:0,0,0 1/1:8:80,7,0 0/1:34:29,0,39</p>

      <p>11 99715891 rs61749255 T G 78.20 highCVG AC1=3;AF1=0.3664;DB;DP=19;DP4=3,8,1,5;FQ=80.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I158M;SNPEFF_CODON_CHANGE=atT/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001243270.1;VDB=0.0015;DN=129;DA=T/G;GM=NM_001243270.1,NM_001243271.1,NM_014361.3,NM_175566.2;GL=CNTN5;FG=missense;FD=missense;AAC=ILE/MET;PP=158/1101,158/912,158/1101,84/1027;CDP=474,474,474,252;GS=10;PH=1;CP=0.990;CG=0.247;AA=T;DV=by-frequency,by-cluster;DSP=73;GESP=G:64/T:11852;PAC=NP_001230199.1,NP_001230200.1,NP_055176.1,NP_780775.1 GT:GQ:PL 0/1:34:54,0,28 0/0:21:0,21,127 1/1:5:70,9,0 0/0:7:0,6,42</p>

      <p>11 102593248 rs1940475 T C 26.10 highCVG AC1=4;AF1=0.4625;DB;DP=31;DP4=0,5,1,3;FQ=28.7;MQ=60;PV4=0.44,0.31,1,0.045;SNPEFF_AMINO_ACID_CHANGE=K87E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MMP8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002424.2;VDB=0.0166;DN=92;DA=T/C;GM=NM_002424.2,XM_005271556.1,XM_005271557.1,XM_005271558.1;GL=MMP8;FG=missense;FD=missense,unknown,unknown,unknown;AAC=LYS/GLU;PP=87/468,64/445,64/445,87/369;CDP=259,190,190,259;GS=56;PH=0.01;CP=0.001;CG=-6.060;AA=T;HA=25.4;HE=48.8;HC=41.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=89;GESP=C:7197/T:5807;PAC=NP_002415.1,XP_005271613.1,XP_005271614.1,XP_005271615.1 GT:GQ:PL 0/0:8:0,9,64 0/1:4:17,3,0 1/1:6:55,9,0 0/0:6:0,6,25</p>

      <p>11 123624658 rs12364099 G T 84.70 midCVG AC1=4;AF1=0.5037;DB;DP=14;DP4=1,3,8,1;FQ=24.8;MQ=60;PV4=0.052,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T190N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6X1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005188.1;VDB=0.0009;DN=120;DA=G/T;GM=NM_001005188.1;GL=OR6X1;FG=missense;FD=missense;AAC=THR/ASN;PP=190/313;CDP=569;GS=65;PH=0.029;CP=0.386;CG=3.460;AA=G;HA=1.7;HE=14.0;HC=2.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12402;KP=hsa04740:OlfactoryTransduction;TFBS=V$YY1_01;GESP=T:1529/G:11473;PAC=NP_001005188.1 GT:GQ:PL 0/1:19:84,0,15 0/1:3:0,0,0 1/1:4:41,6,0 0/0:5:0,6,47</p>

      <p>11 124127373 rs2466681 C T 38.60 midCVG AC1=2;AF1=0.3056;DB;DP=10;DP4=1,5,3,1;FQ=40.1;MQ=60;PV4=0.19,0.26,1,1;VDB=0.0158;DN=100;DA=C/T;GM=none;FG=intergenic;FD=unknown;CP=0.043;CG=0.131;AA=C;HA=26.4;HC=42.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=LTR48;DSP=110801 GT:GQ:PL 0/0:14:0,12,88 0/0:6:0,3,20 1/1:6:81,12,0 0/0:6:0,3,28</p>

      <p>11 124127954 rs2512190 T G 50.90 lowCVG AC1=6;AF1=0.746;DB;DP=7;DP4=1,0,4,1;FQ=-14.6;MQ=60;PV4=1,1,1,1;VDB=0.0012;DN=100;DA=T/G;GM=none;FG=intergenic;FD=unknown;CP=0.069;CG=0.438;AA=T;HA=24.2;HE=50.0;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=111382 GT:GQ:PL 0/1:5:0,3,20 0/1:3:0,0,0 1/1:12:71,12,0 1/1:5:20,3,0</p>

      <p>11 124129506 rs2512185 C T 111 highCVG AC1=5;AF1=0.559;DB;DP=19;DP4=7,2,1,9;FQ=111;MQ=60;PV4=0.0055,0.029,1,0.24;VDB=0.0065;DN=100;DA=C/T;GM=none;FG=intergenic;FD=unknown;CP=0.000;CG=-1.710;AA=C;HA=24.2;HE=50.0;HC=39.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=112934 GT:GQ:PL 0/1:51:47,0,77 0/1:3:0,3,25 1/1:10:60,12,0 0/1:51:49,0,53</p>

      <p>11 124129518 rs12807551 A G 111 highCVG AC1=5;AF1=0.5471;DB;DP=19;DP4=7,2,1,8;FQ=106;MQ=60;PV4=0.015,0.35,1,1;VDB=0.0084;DN=121;DA=A/G;GM=none;FG=intergenic;FD=unknown;CP=0.182;CG=-0.228;AA=A;HA=19.2;HE=34.7;HC=46.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=112946 GT:GQ:PL 0/1:66:64,0,69 0/1:3:0,3,26 1/1:7:64,9,0 0/1:30:27,0,53</p>

      <p>11 124135009 rs2512168 G A 85.50 highCVG AC1=3;AF1=0.3452;DB;DP=15;DP4=1,4,4,6;FQ=75.1;MQ=42;PV4=0.6,1,0.082,1;SNPEFF_AMINO_ACID_CHANGE=C96Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR8G5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005198.1;VDB=0.0030;DN=100;DA=G/A;GM=NM_001005198.1;GL=OR8G5;FG=missense;FD=missense;AAC=CYS/TYR;PP=96/347;CDP=287;GS=194;CP=0.533;CG=-5.000;AA=A;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118248;GESP=A:6010/G:6910;PAC=NP_001005198.1 GT:GQ:PL 0/0:5:0,3,25 0/0:7:0,6,48 1/1:24:130,30,0 0/0:7:0,6,45</p>

      <p>11 124675214 rs558814 A G 70.90 highCVG AC1=5;AF1=0.5479;DB;DP=16;DP4=0,8,3,5;FQ=73.5;MQ=60;PV4=0.2,0.082,1,1;VDB=0.0189;DN=83;DA=A/G;GM=NR_103862.1;GL=LOC100507283;FG=splice-acceptor;FD=unknown;CP=0.230;CG=0.225;AA=A;HA=41.0;HE=34.8;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=L2a;DSP=0 GT:GQ:PL 0/1:13:9,0,67 0/1:3:0,3,26 1/1:15:100,18,0 0/1:10:6,0,29</p>

      <p>11 124908352 rs7107487 T C 999 highCVG AC1=4;AF1=0.5057;DB;DP=29;DP4=6,9,4,10;FQ=205;MQ=60;PV4=0.7,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=C813R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025004.2;VDB=0.0034;DN=116;DA=T/C;GM=NM_025004.2;GL=CCDC15;FG=missense;FD=missense;AAC=CYS/ARG;PP=813/952;CDP=2437;GS=180;PH=0;CP=0.973;CG=3.110;AA=C;HA=36.7;HE=12.8;HC=17.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=C:2453/T:9283;PAC=NP_079280.2 GT:GQ:PL 0/1:48:45,0,74 0/1:63:60,0,82 1/1:18:134,21,0 0/0:12:0,15,94</p>

      <p>11 129724614 . T G 7.84 lowCVG AC1=6;AF1=0.7749;DP=5;DP4=1,0,0,3;FQ=-20.3;MQ=60;PV4=0.25,0.47,1,1;SNPEFF_AMINO_ACID_CHANGE=F96L;SNPEFF_CODON_CHANGE=ttT/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM45B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138788.3;GM=NM_138788.3,XM_005271400.1,XM_005271401.1;GL=TMEM45B;FG=missense;FD=unknown;AAC=PHE/LEU;PP=96/276;CDP=288;GS=22;PH=0.989;CP=0.013;CG=-9.770;AA=T;DSP=98;GESP=T:12996;PAC=NP_620143.1,XP_005271457.1,XP_005271458.1 GT:GQ:PL 0/1:3:0,0,0/1:11:10,0,11 1/1:4:20,3,0 1/1:4:14,3,0</p>

      <p>12 6948468 rs1129649 T C 24.30 lowCVG AC1=3;AF1=0.3968;DB;DP=8;DP4=2,2,0,4;FQ=25.9;MQ=59;PV4=0.43,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I684T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LEPREL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014262.4;VDB=0.0226;DN=86;DA=T/C;GM=NM_014262.3;GL=LEPREL2;FG=coding-unknown;FD=missense;CDP=2053;CP=0.892;CG=2.360;AA=T;HA=43.1;HE=37.2;HC=35.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=C:4320/T:8170;PAC=NP_055077.2 GT:GQ:PL 0/0:5:0,3,23 0/1:3:0,0,0 1/1:7:65,12,0 0/0:9:0,9,51</p>

      <p>12 8374812 rs17855656 G A 37.30 lowCVG AC1=5;AF1=0.6547;DB;DP=5;DP4=1,0,4,0;FQ=-14.5;MQ=48;PV4=1,0.073,1,1;SNPEFF_AMINO_ACID_CHANGE=P334L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM90A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018088.3;VDB=0.0130;DN=123;DA=G/A;GM=NM_018088.3,XM_005253403.1,XM_005253404.1;GL=FAM90A1;FG=missense;FD=missense,unknown,unknown;AAC=PRO/LEU;PP=334/465;CDP=1001;GS=98;PH=0.023;CP=0.006;CG=-1.440;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=569;GESP=A:1659/G:11279;PAC=NP_060558.3,XP_005253460.1,XP_005253461.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,18 1/1:10:74,12,0 0/1:3:0,0,0</p>

      <p>12 9307373 rs2377741 T G 29.10 midCVG AC1=6;AF1=0.7339;DB;DP=14;DP4=0,2,5,1;FQ=-4.32;MQ=50;PV4=0.11,0.007,0.18,1;SNPEFF_AMINO_ACID_CHANGE=T1205P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PZP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002864.2;VDB=0.0199;DN=100;DA=T/G;GM=NM_002864.2,XM_005253459.1;GL=PZP;FG=missense;FD=missense,unknown;AAC=THR/PRO;PP=1205/1483,991/1269;CDP=3613,2971;GS=38;PH=0;CP=0.039;CG=0.627;AA=G;HA=4.9;HE=0.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:12716/T:290;PAC=NP_002855.2,XP_005253516.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:14:60,15,0 0/1:12:6,0,29</p>

      <p>12 10041364 rs1797517 G A 210 highCVG AC1=6;AF1=0.7824;DB;DP=15;DP4=2,1,2,10;FQ=-5.09;MQ=60;PV4=0.15,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V68I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLRF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001190765.1;VDB=0.0087;DN=89;DA=G/A;GM=NM_001190765.1;GL=KLRF2;FG=missense;FD=missense;AAC=VAL/ILE;PP=68/208;CDP=202;GS=29;CP=0.328;CG=1.980;AA=A;HA=42.5;HE=22.7;HC=31.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=16;PAC=NP_001177694.1 GT:GQ:PL 0/1:22:16,0,28 0/1:5:44,0,6 1/1:17:103,15,0 1/1:14:91,12,0</p>

      <p>12 31135579 rs2075333 T C 18.20 midCVG AC1=3;AF1=0.3809;DB;DP=10;DP4=6,1,2,1;FQ=20.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V190A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSPAN11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080509.2;VDB=0.0080;DN=96;DA=T/C;GM=NM_001080509.2,XM_005253378.1,XM_005253379.1;GL=TSPAN11;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=190/254,190/254,180/244;CDP=569,569,539;GS=64;PH=0;CP=0.008;CG=-1.760;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$PAX5_02;GESP=C:8630/T:4352;PAC=NP_001073978.1,XP_005253435.1,XP_005253436.1 GT:GQ:PL 0/1:16:16,0,47 0/0:9:0,9,58 1/1:3:45,6,0 0/0:5:0,3,26</p>

      <p>12 32135186 rs3759302 T A 62.90 midCVG AC1=4;AF1=0.5521;DB;DP=9;DP4=2,1,1,4;FQ=12.7;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S433T;SNPEFF_CODON_CHANGE=Tct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0034;DN=107;DA=T/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=SER/THR;PP=433/1748;CDP=1297;GS=58;PH=0.999;CP=0.001;CG=-1.130;AA=T;HA=3.4;HE=18.1;HC=11.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1375;GESP=A:2107/T:10899;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:3:0,3,16 0/0:5:0,6,38 1/1:3:26,3,0 1/1:10:82,12,0</p>

      <p>12 32137565 rs1057994 G A 109 midCVG AC1=6;AF1=0.6731;DB;DP=10;DP4=1,2,1,6;FQ=8.62;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V1226I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0055;DN=86;DA=G/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=1226/1748;CDP=3676;GS=29;PH=0.02;CP=0.000;CG=-2.920;AA=G;HA=4.3;HE=19.2;HC=9.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1327;GESP=A:2074/G:10932;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:27:37,0,27 0/1:4:0,3,21 1/1:7:42,6,0 1/1:9:75,9,0</p>

      <p>12 40758652 rs3761863 T C 14.60 lowCVG AC1=6;AF1=0.6706;DB;DP=5;DP4=1,0,2,1;FQ=-12.1;MQ=60;PV4=1,0.14,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M2397T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=49;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198578.3;VDB=0.0050;DN=107;DA=T/C;GM=NM_198578.3,XM_005268629.1,XM_005268630.1,XM_005268631.1;GL=LRRK2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=MET/THR;PP=2397/2528,2397/2522,2397/2528,2357/2488;CDP=7190,7190,7190,7070;GS=81;PH=0;CP=0.005;CG=3.030;AA=C;HA=46.5;HE=34.3;HC=42.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=9;KP=hsa05012:Parkinson'sDisease;GESP=C:7962/T:5042;PPI=STUB1:846;PAC=NP_940980.3,XP_005268686.1,XP_005268687.1,XP_005268688.1 GT:GQ:PL 0/1:20:30,0,18 0/1:3:0,0,0 1/1:3:19,3,0 0/1:3:0,0,0</p>

      <p>12 40787679 rs11176421 G C 149 highCVG AC1=6;AF1=0.6679;DB;DP=18;DP4=2,2,1,13;FQ=29.1;MQ=60;PV4=0.11,1,1,0.054;VDB=0.0038;DN=120;DA=G/C;GM=none;FG=upstream-gene;FD=unknown;CP=0.000;CG=-0.589;AA=C;HA=39.0;HC=48.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=Tigger1;DSP=4035 GT:GQ:PL 0/1:42:39,0,46 0/1:4:0,3,23 1/1:4:26,3,0 1/1:30:129,30,0</p>

      <p>12 40787912 rs17444605 G A 92.20 midCVG AC1=5;AF1=0.6298;DB;DP=12;DP4=0,4,5,3;FQ=35.6;MQ=60;PV4=0.081,0.18,1,0.3;VDB=0.0144;DN=123;DA=G/A;GM=none;FG=upstream-gene;FD=unknown;CP=0.000;CG=-0.771;AA=A;HA=22.9;HE=35.8;HC=37.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=Tigger1;DSP=3802 GT:GQ:PL 0/1:16:11,0,33 0/0:4:0,6,44 1/1:4:15,3,0 1/1:17:112,18,0</p>

      <p>12 40805948 rs10878538 G C 49.20 lowCVG AC1=6;AF1=0.7301;DB;DP=6;DP4=0,1,3,1;FQ=-8.82;MQ=60;PV4=0.4,0.35,1,0.26;VDB=0.0025;DN=120;DA=G/C;GM=XM_003846356.2;GL=MUC19;FG=missense;FD=missense;AAC=ASP/HIS;PP=160/8189;CDP=478;GS=81;CP=0.001;CG=2.010;AA=G;HA=0.0;HE=15.0;HC=19.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=37;PAC=XP_003846404.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,26 1/1:4:24,3,0 1/1:9:65,9,0</p>

      <p>12 40938458 rs17128403 T C 47.40 highCVG AC1=3;AF1=0.3611;DB;DP=17;DP4=2,10,2,3;FQ=49.2;MQ=60;PV4=0.54,1,1,0.27;VDB=0.0069;DN=123;DA=T/C;GM=XM_003846356.2;GL=MUC19;FG=missense;FD=missense;AAC=SER/PRO;PP=7559/8189;CDP=22673;GS=74;CP=0.093;CG=0.515;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=18;PAC=XP_003846404.1 GT:GQ:PL 0/0:5:0,3,24 0/1:4:2,0,101 1/1:7:88,12,0 0/0:13:0,12,87</p>

      <p>12 42863266 rs79087668 C T 63.10 midCVG AC1=5;AF1=0.5836;DB;DP=10;DP4=2,2,2,4;FQ=39.8;MQ=60;PV4=1,0.25,1,0.22;SNPEFF_AMINO_ACID_CHANGE=A124T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRICKLE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144881.1;VDB=0.0077;DN=131;DA=C/T;GM=NM_001144881.1,NM_001144882.1,NM_001144883.1,NM_153026.2;GL=PRICKLE1;FG=missense;FD=missense;AAC=ALA/THR;PP=124/832;CDP=370;GS=58;PH=0.002;CP=0.927;CG=3.550;AA=C;DV=by-frequency,by-cluster;DSP=15;KP=hsa04310:WntSignalingPathway;GESP=T:22/C:12984;PAC=NP_001138353.1,NP_001138354.1,NP_001138355.1,NP_694571.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,26 1/1:10:83,12,0 0/1:25:21,0,51</p>

      <p>12 49443799 rs112236653 G A 49.50 midCVG AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1 GT:GQ:PL 0/1:14:11,0,64 0/1:3:0,0,0 1/1:9:81,12,0 0/0:7:0,9,52</p>

      <p>12 49952449 . C T 14.40 lowCVG AC1=4;AF1=0.5125;DP=6;DP4=3,0,0,3;FQ=12.6;MQ=60;PV4=0.1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A469T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCRS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001012300.1;VDB=0.0104;GM=NM_001012300.1,NM_001278341.1,NM_006337.4,XM_005268572.1,XM_005268573.1;GL=MCRS1;FG=missense;FD=unknown;AAC=ALA/THR;PP=469/476,265/272,456/463,456/463,367/374;CDP=1405,793,1366,1366,1099;GS=58;PH=0.997;CP=1.000;CG=5.280;AA=C;DSP=64;TFBS=V$PAX4_01;GESP=C:13006;PPI=INO80E:892|DAXX:846|INO80:811|PBX2:786;PAC=NP_001012300.1,NP_001265270.1,NP_006328.2,XP_005268629.1,XP_005268630.1 GT:GQ:PL 0/1:7:5,0,32 0/1:3:0,0,0 1/1:4:47,6,0 0/0:4:0,3,17</p>

      <p>12 52908917 rs11549950 T C 25.50 lowCVG AC1=3;AF1=0.3646;DB;DP=8;DP4=3,1,0,3;FQ=27.4;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S528G;SNPEFF_CODON_CHANGE=Agc/Ggc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000424.3;VDB=0.0108;DN=120;DA=T/C;GM=NM_000424.3,XM_005268862.1;GL=KRT5;FG=missense;FD=missense,unknown;AAC=SER/GLY;PP=528/591;CDP=1582;GS=56;PH=0;CP=0.000;CG=0.084;AA=C;HA=25.2;HE=9.0;HC=4.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;RM=(CCG)n;CA;DSP=108;GESP=C:2251/T:10755;PPI=KRT14:959;PAC=NP_000415.2,XP_005268919.1 GT:GQ:PL 0/0:5:0,3,24 0/0:3:0,0,0 1/1:4:65,9,0 0/0:10:0,9,70</p>

      <p>12 53091566 rs1567759 C A 107 midCVG AC1=5;AF1=0.6042;DB;DP=10;DP4=1,2,3,4;FQ=27.6;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=G220C;SNPEFF_CODON_CHANGE=Ggt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT77;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175078.2;VDB=0.0106;DN=88;DA=C/A;GM=NM_175078.2,XM_005268853.1;GL=KRT77;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=GLY/CYS,none;PP=220/579,NA;CDP=658,NA;GS=159,NA;PH=0.999;CP=0.010;CG=-2.050;AA=C;HA=44.7;HE=42.9;HC=20.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=101;GESP=A:5613/C:7393;PPI=USP32:916;PAC=NP_778253.2,NA GT:GQ:PL 0/1:4:0,3,20 0/0:4:0,6,51 1/1:10:90,12,0 1/1:8:64,9,0</p>

      <p>12 53165940 rs6580904 C T 76.30 midCVG AC1=5;AF1=0.5683;DB;DP=11;DP4=2,1,5,3;FQ=13.2;MQ=60;PV4=1,0.37,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A359T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015848.4;VDB=0.0153;DN=116;DA=C/T;GM=NM_015848.4;GL=KRT76;FG=missense;FD=missense;AAC=ALA/THR;PP=359/639;CDP=1075;GS=58;PH=0.986;CP=0.890;CG=4.420;AA=C;HA=39.2;HE=5.8;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=T:9195/C:3811;PAC=NP_056932.2 GT:GQ:PL 0/1:3:0,3,15 0/1:15:12,0,21 1/1:13:85,15,0 0/1:19:24,0,18</p>

      <p>12 53242440 rs2013335 A G 31 lowCVG AC1=6;AF1=0.7073;DB;DP=6;DP4=2,0,2,2;FQ=-3.98;MQ=60;PV4=0.47,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L92P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT78;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173352.3;VDB=0.0085;DN=92;DA=A/G;GM=NM_173352.2,XM_005268696.1;GL=KRT78;FG=missense,5-prime-UTR;FD=missense,unknown;AAC=LEU/PRO,none;PP=92/521,NA;CDP=275,NA;GS=98,NA;PH=1;CP=0.944;CG=5.180;AA=A;HA=28.3;HE=37.7;HC=48.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=70;GESP=G:6269/A:6737;PAC=NP_775487.2,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:38,6,0 0/1:29:29,0,29</p>

      <p>12 53605545 rs2229774 G A 41.20 midCVG AC1=4;AF1=0.5599;DB;DP=12;DP4=0,3,0,9;FQ=13.2;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=S427L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RARG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000966.5;VDB=0.0000;DN=98;DA=G/A;GM=NM_000966.5,NM_001042728.2,NM_001243730.1,NM_001243731.1,NM_001243732.1,XM_005269054.1,XM_005269055.1,XM_005269056.1,XM_005269057.1;GL=RARG;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/LEU;PP=427/455,416/444,355/383,306/334,405/433,514/542,514/542,427/455,427/455;CDP=1280,1247,1064,917,1214,1541,1541,1280,1280;GS=145;PH=0.525;CP=1.000;CG=4.530;AA=G;HA=11.1;HE=6.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=103;TFBS=V$ARP1_01;GESP=A:949/G:12057;PAC=NP_000957.1,NP_001036193.1,NP_001230659.1,NP_001230660.1,NP_001230661.1,XP_005269111.1,XP_005269112.1,XP_005269113.1,XP_005269114.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:81,27,0 0/0:7:0,9,49</p>

      <p>12 53682468 . G T 35.30 lowCVG AC1=5;AF1=0.6499;DP=5;DP4=1,0,0,4;FQ=-11.7;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1565S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ESPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012291.4;VDB=0.0084;GM=NM_012291.4;GL=ESPL1;FG=missense;FD=unknown;AAC=ALA/SER;PP=1565/2121;CDP=4693;GS=99;PH=0.01;CP=0.003;CG=2.840;AA=G;DSP=16;KP=hsa04110:CellCycle|hsa04114:OocyteMeiosis;GESP=G:12990;PPI=PTTG1:987|IRF2BPL:987;PAC=NP_036423.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:10:72,12,0 0/1:3:0,3,21</p>

      <p>12 55355033 rs997173 C T 9.97 midCVG AC1=2;AF1=0.3049;DB;DP=11;DP4=3,5,3,0;FQ=11.3;MQ=60;PV4=0.18,0.06,1,1;SNPEFF_AMINO_ACID_CHANGE=E496K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TESPA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098815.2;VDB=0.0017;DN=86;DA=C/T;GM=NM_001098815.2,NM_001136030.2,NM_001261844.1,NM_014796.2,XM_005269243.1,XM_005269244.1,XM_005269245.1,XM_005269246.1,XM_005269247.1,XM_005269248.1,XM_005269249.1;GL=TESPA1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLU/LYS;PP=496/522,496/522,358/384,358/384,496/522,496/522,496/522,496/522,358/384,243/269,243/269;CDP=1486,1486,1072,1072,1486,1486,1486,1486,1072,727,727;GS=56;PH=0.004;CP=0.842;CG=2.340;AA=C;HA=7.5;HE=7.6;HC=35.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=19;GESP=T:871/C:12133;PAC=NP_001092285.1,NP_001129502.1,NP_001248773.1,NP_055611.1,XP_005269300.1,XP_005269301.1,XP_005269302.1,XP_005269303.1,XP_005269304.1,XP_005269305.1,XP_005269306.1 GT:GQ:PL 0/0:9:0,6,47 0/0:20:0,18,111 1/1:3:50,9,0 0/0:4:0,0,0</p>

      <p>12 56030938 rs10876838 C T 17.80 lowCVG AC1=4;AF1=0.4457;DB;DP=6;DP4=0,2,1,3;FQ=4.19;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=P88L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10P1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206899.1;VDB=0.0174;DN=120;DA=C/T;GM=NM_206899.1;GL=OR10P1;FG=missense;FD=missense;AAC=PRO/LEU;PP=88/314;CDP=263;GS=98;PH=0.002;CP=0.000;CG=-2.440;AA=C;HA=8.2;HE=39.8;HC=24.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=45099;KP=hsa04740:OlfactoryTransduction;GESP=T:4598/C:8408;PAC=NP_996782.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,3,23 1/1:5:63,16,7 0/0:4:0,3,15</p>

      <p>12 58201241 . G A 49.20 midCVG AC1=3;AF1=0.4181;DP=13;DP4=2,4,6,0;FQ=49.8;MQ=60;PV4=0.061,1,1,0.01;SNPEFF_AMINO_ACID_CHANGE=A455V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVIL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006576.3;VDB=0.0142;GM=NM_006576.3,XM_005268575.1;GL=AVIL;FG=missense;FD=unknown;AAC=ALA/VAL;PP=455/820,448/813;CDP=1364,1343;GS=64;PH=1;CP=1.000;CG=4.770;AA=G;DSP=32;GESP=G:13006;PAC=NP_006567.3,XP_005268632.1 GT:GQ:PL 0/1:11:20,0,6 0/0:8:0,9,54 1/1:8:75,12,0 0/0:6:0,6,38</p>

      <p>12 70963637 . C T 6.02 midCVG AC1=6;AF1=0.7277;DP=10;DP4=1,0,4,0;FQ=-14.1;MQ=60;PV4=1,0.00024,1,0.38;SNPEFF_AMINO_ACID_CHANGE=G1151E;SNPEFF_CODON_CHANGE=gGa/gAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTPRB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001109754.2;VDB=0.0143;GM=NM_001109754.2,NM_001206971.1,NM_001206972.1,NM_002837.4;GL=PTPRB;FG=missense;FD=unknown;AAC=GLY/GLU;PP=1151/2216,843/1908,933/1908,933/1998;CDP=3452,2528,2798,2798;GS=98;PH=1;CP=0.169;CG=5.300;AA=C;DSP=82;KP=hsa04520:AdherensJunction;GESP=C:12348;PPI=CNTN1:937;PAC=NP_001103224.1,NP_001193900.1,NP_001193901.1,NP_002828.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:33,9,0 0/1:10:8,0,17</p>

      <p>12 71523134 rs1051334 A C 65.60 midCVG AC1=3;AF1=0.4149;DB;DP=10;DP4=1,4,1,4;FQ=59.3;MQ=60;PV4=1,1,1,0.024;SNPEFF_AMINO_ACID_CHANGE=S213A;SNPEFF_CODON_CHANGE=Tca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSPAN8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004616.2;VDB=0.0070;DN=86;DA=A/C;GM=NM_004616.2;GL=TSPAN8;FG=missense;FD=missense;AAC=SER/ALA;PP=213/238;CDP=637;GS=99;PH=0;CP=0.992;CG=4.880;AA=C;HA=11.3;HE=40.1;HC=32.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=24;GESP=C:4358/A:8632;PAC=NP_004607.1 GT:GQ:PL 0/1:20:35,0,15 0/0:4:0,3,21 1/1:6:75,9,0 0/0:9:0,9,66</p>

      <p>12 100774572 . C A 19.30 lowCVG AC1=3;AF1=0.352;DP=8;DP4=1,3,3,0;FQ=21.1;MQ=60;PV4=0.14,1,1,6.6e-07;SNPEFF_AMINO_ACID_CHANGE=C65*;SNPEFF_CODON_CHANGE=tgC/tgA;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC17A8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_139319.2;VDB=0.0078;GM=NM_001145288.1,NM_139319.2;GL=SLC17A8;FG=stop-gained;FD=unknown;AAC=CYS/stop;PP=65/540,65/590;CDP=195;CP=0.061;CG=-8.160;AA=C;DSP=94;GESP=C:13006;PAC=NP_001138760.1,NP_647480.1 GT:GQ:PL 0/0:8:0,6,45 0/0:3:0,0,0 1/1:4:59,9,0 0/0:8:0,6,42</p>

      <p>TSPAN8 12 101560328 rs164365 C A 44.40 midCVG AC1=4;AF1=0.4752;DB;DP=14;DP4=4,4,2,3;FQ=47.1;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=M490I;SNPEFF_CODON_CHANGE=atG/atT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC5A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145913.3;VDB=0.0176;DN=79;DA=C/A;GM=NM_145913.3;GL=SLC5A8;FG=missense;FD=missense;AAC=MET/ILE;PP=490/611;CDP=1470;GS=10;PH=0.002;CP=0.751;CG=2.950;AA=C;HA=11.3;HE=19.6;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57;GESP=A:2529/C:10477;PAC=NP_666018.3 GT:GQ:PL 0/0:10:0,12,72 0/1:11:8,0,55 1/1:6:67,9,0 0/1:15:15,0,15</p>

      <p>12 101750341 . G T 60.60 highCVG AC1=3;AF1=0.4122;DP=15;DP4=0,5,0,7;FQ=40.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E1802*;SNPEFF_CODON_CHANGE=Gaa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=42;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UTP20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_014503.2;VDB=0.0074;GM=NM_014503.2;GL=UTP20;FG=stop-gained;FD=unknown;AAC=GLU/stop;PP=1802/2786;CDP=5404;CP=0.981;CG=5.900;AA=G;DSP=10;GESP=G:13006;PAC=NP_055318.2 GT:GQ:PL 0/0:4:0,3,26 0/1:3:0,0,0 1/1:16:103,21,0 0/0:11:0,12,55</p>

      <p>SLC5A8 12 103249087 rs77958223 T C 77.80 highCVG AC1=4;AF1=0.4954;DB;DP=15;DP4=5,3,3,4;FQ=80.7;MQ=60;PV4=0.62,0.25,1,0.071;SNPEFF_AMINO_ACID_CHANGE=E178G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PAH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000277.1;VDB=0.0177;DN=131;DA=T/A/C;GM=NM_000277.1;GL=PAH;FG=missense;FD=missense;AAC=GLU/GLY;PP=178/453;CDP=533;GS=98;PH=0.961;CP=1.000;CG=5.730;AA=T;DV=by-cluster;CA;DSP=24;KP=hsa00360:PhenylalanineMetabolism|hsa00400:Phenylalanine,TyrosineAndTryptophanBiosynthesis|hsa01100:MetabolicPathways;GESP=T:13006;PAC=NP_000268.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:16,3,0 1/1:15:109,18,0 0/0:19:0,21,126</p>

      <p>12 117383320 rs4076700 G A 78.70 midCVG AC1=5;AF1=0.6582;DB;DP=13;DP4=3,1,7,1;FQ=32.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R192Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXW8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153348.2;VDB=0.0037;DN=108;DA=G/A;GM=NM_012174.1,NM_153348.2,XM_005253866.1,XM_005253867.1,XM_005253868.1;GL=FBXW8;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=ARG/GLN,ARG/GLN,ARG/GLN,GLU/LYS,GLU/LYS;PP=126/533,192/599,192/423,41/418,41/418;CDP=377,575,575,121,121;GS=43,43,43,56,56;PH=1;CP=0.990;CG=5.190;AA=G;HA=22.8;HE=17.4;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=A:8670/G:4336;PPI=SKP1:994|CUL7:994|CUL1:932|RBX1:929;PAC=NP_036306.1,NP_699179.2,XP_005253923.1,XP_005253924.1,XP_005253925.1 GT:GQ:PL 0/1:23:18,0,39 0/1:3:0,0,0 1/1:17:93,18,0 0/1:14:9,0,30</p>

      <p>12 120805836 . G A 14.20 lowCVG AC1=4;AF1=0.5539;DP=5;DP4=0,2,0,3;FQ=12.9;MQ=60;PV4=1,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=S81L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MSI1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002442.3;VDB=0.0000;GM=NM_002442.3;GL=MSI1;FG=missense;FD=unknown;AAC=SER/LEU;PP=81/363;CDP=242;GS=145;PH=0.539;CP=1.000;CG=3.240;AA=G;DSP=26;TFBS=V$CEBPA_01,V$CEBPB_02,V$CREB_Q2,V$GATA1_01,V$GFI1_01;GESP=G:13006;PAC=NP_002433.1 GT:GQ:PL 0/1:3:0,0,0/1:3:0,0,0 1/1:6:51,9,0 0/0:6:0,6,48</p>

      <p>12 125467158 rs11558556 C T 62 highCVG AC1=2;AF1=0.2591;DB;DP=25;DP4=3,16,2,4;FQ=63.3;G3=0.75,7.094e-07,0.25;HWE=0.0457;MQ=60;PV4=0.56,0.034,1,1;SNPEFF_AMINO_ACID_CHANGE=M96I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHX37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032656.3;VDB=0.0142;DN=120;DA=C/G/T;GM=NM_032656.3,XM_005253590.1;GL=DHX37;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=96/1158,96/1176;CDP=288;GS=10;PH=0;CP=1.000;CG=4.050;AA=C;HA=41.2;HE=31.7;HC=38.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12;GESP=T:4715/C:8291;PAC=NP_116045.2,XP_005253647.1 GT:GQ:PL 0/0:9:0,6,41 0/0:32:0,30,147 1/1:11:108,18,0 0/0:23:0,21,121</p>

      <p>12 133423662 rs2306536 C T 17.40 lowCVG AC1=5;AF1=0.666;DB;DP=8;DP4=3,0,3,0;FQ=12.8;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V580M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHFR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001161344.1;VDB=0.0028;DN=100;DA=C/T;GM=NM_001161344.1,NM_001161345.1,NM_001161346.1,NM_001161347.1,NM_018223.2,XM_005266170.1;GL=CHFR;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=VAL/MET;PP=580/665,579/664,568/653,488/573,539/624,621/706;CDP=1738,1735,1702,1462,1615,1861;GS=21;PH=0.099;CP=0.012;CG=-2.610;AA=C;HA=31.0;HE=16.7;HC=4.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=34;GESP=T:2911/C:10095;PPI=UBC:937|AURKA:846;PAC=NP_001154816.1,NP_001154817.1,NP_001154818.1,NP_001154819.1,NP_060693.2,XP_005266227.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:24,3,0 0/1:32:28,0,43</p>

      <p>12 133803551 rs36146434 G C 26.70 midCVG AC1=4;AF1=0.4988;DB;DP=12;DP4=7,1,2,2;FQ=29.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S230C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANHX;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001191054.1;VDB=0.0086;DN=126;DA=G/C;GM=NM_001191054.1;GL=ANHX;FG=missense;FD=missense;AAC=SER/CYS;PP=230/380;CDP=689;GS=112;PH=0.983;CP=0.001;CG=3.110;AA=G;DV=by-cluster;DSP=30;PAC=NP_001177983.1 GT:GQ:PL 0/1:22:20,0,80 0/1:3:0,0,1/1:4:47,6,0 0/0:5:0,6,44</p>

      <p>13 46170609 rs3014939 C T 28.50 lowCVG AC1=3;AF1=0.4056;DB;DP=8;DP4=0,4,4,0;FQ=30.6;MQ=60;PV4=0.029,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=E178K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ERICH6B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182542.2;VDB=0.0092;DN=101;DA=C/T;GM=NM_182542.2;GL=FAM194B;FG=missense;FD=missense;AAC=GLU/LYS;PP=178/697;CDP=532;GS=56;PH=0.988;CP=0.002;CG=1.170;AA=C;HA=13.3;HE=49.7;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=106;GESP=T:1855/C:2711;PAC=NP_872348.2 GT:GQ:PL 0/0:4:0,3,26 0/1:22:32,0,17 1/1:3:39,6,0 0/0:6:0,6,44</p>

      <p>13 50141345 rs4942848 G A 41.70 midCVG AC1=4;AF1=0.5416;DB;DP=11;DP4=4,1,0,5;FQ=35.5;MQ=60;PV4=0.048,1,1,0.0057;SNPEFF_AMINO_ACID_CHANGE=A24V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RCBTB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018191.3;VDB=0.0072;DN=111;DA=G/A;GM=NM_018191.3,XM_005266440.1,XM_005266441.1,XM_005266442.1;GL=RCBTB1;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/VAL;PP=24/532,24/532,24/532,24/356;CDP=71;GS=64;PH=1;CP=0.989;CG=5.840;AA=G;HA=38.1;HE=32.9;HC=36.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=56;GESP=A:7547/G:5459;PAC=NP_060661.3,XP_005266497.1,XP_005266498.1,XP_005266499.1 GT:GQ:PL 0/1:42:40,0,50 0/1:3:0,0,0 1/1:5:41,6,0 0/0:3:0,3,22</p>

      <p>13 51854694 . C A 42.70 highCVG AC1=3;AF1=0.3358;DP=25;DP4=6,7,12,0;FQ=44.4;MQ=60;PV4=0.0052,0.15,1,1;SNPEFF_AMINO_ACID_CHANGE=P351T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM124A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145019.3;VDB=0.0000;GM=NM_001242312.1,NM_145019.3;GL=FAM124A;FG=missense;FD=unknown;AAC=PRO/THR;PP=315/547,351/583;CDP=943,1051;GS=38;PH=0.034;CP=0.000;CG=0.795;AA=C;DSP=109;GESP=C:13006;PAC=NP_001229241.1,NP_659456.3 GT:GQ:PL 0/0:15:0,15,93 0/0:3:0,0,0 1/1:16:87,22,0 0/0:21:0,21,125</p>

      <p>13 92345579 rs553717 C T 77.30 highCVG AC1=4;AF1=0.4896;DB;DP=16;DP4=4,6,0,6;FQ=80.1;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A155V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004466.5;VDB=0.0059;DN=83;DA=C/T;GM=NM_004466.4;GL=GPC5;FG=missense;FD=missense;AAC=ALA/VAL;PP=155/573;CDP=464;GS=64;PH=0.013;CP=0.790;CG=4.190;AA=C;HA=19.4;HE=14.8;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=139;GESP=T:1792/C:11214;PAC=NP_004457.1 GT:GQ:PL 0/0:22:0,24,144 0/0:5:0,6,49 1/1:6:66,9,0 1/1:6:61,9,0</p>

      <p>13 96540204 rs12876018 T G 17.80 midCVG AC1=3;AF1=0.3733;DB;DP=13;DP4=1,7,2,2;FQ=19.7;MQ=60;PV4=0.24,0.25,1,0.15;SNPEFF_AMINO_ACID_CHANGE=M994L;SNPEFF_CODON_CHANGE=Atg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=26;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UGGT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020121.3;VDB=0.0164;DN=121;DA=T/G;GM=NM_020121.3,XM_005254066.1,XM_005254067.1,XM_005254068.1,XM_005254069.1;GL=UGGT2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=MET/LEU;PP=994/1517,992/1515,994/1485,950/1473,640/1163;CDP=2980,2974,2980,2848,1918;GS=15;PH=0.131;CP=0.995;CG=2.860;AA=T;HA=26.0;HE=44.2;HC=28.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4835/T:8169;PPI=BRE:892|BABAM1:871;PAC=NP_064506.3,XP_005254123.1,XP_005254124.1,XP_005254125.1,XP_005254126.1 GT:GQ:PL 0/0:12:0,12,66 0/0:5:0,3,21 1/1:3:31,6,0 0/1:30:30,0,50</p>

      <p>13 103384824 rs9300756 G A 50.40 highCVG AC1=3;AF1=0.3894;DB;DP=15;DP4=2,6,5,0;FQ=52.5;MQ=60;PV4=0.021,1,1,0.032;SNPEFF_AMINO_ACID_CHANGE=R6075C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC168;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146197.1;VDB=0.0090;DN=119;DA=G/A;GM=NM_001146197.1;GL=CCDC168;FG=missense;FD=missense;AAC=ARG/CYS;PP=6075/7082;CDP=18223;GS=180;PH=0;CP=0.000;CG=0.010;AA=G;HA=35.0;HE=41.0;HC=16.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17915;GESP=A:2470/G:2094;PAC=NP_001139669.1 GT:GQ:PL 0/0:14:0,15,100 0/0:6:0,6,34 1/1:8:75,12,0 0/1:20:22,0,19</p>

      <p>13 109438084 rs911973 T A 59.60 midCVG AC1=5;AF1=0.5535;DB;DP=11;DP4=0,5,1,5;FQ=45;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D203E;SNPEFF_CODON_CHANGE=gaT/gaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198950.1;VDB=0.0067;DN=86;DA=T/A;GM=NM_001198950.1,NM_015011.1;GL=MYO16;FG=missense;FD=unknown;AAC=ASP/GLU;PP=203/1881,181/1859;CDP=609,543;GS=45;PH=0.025;CP=1.000;CG=2.710;AA=T;HA=0.0;HE=2.5;HC=22.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=8;GESP=A:306/T:12700;PAC=NP_001185879.1,NP_055826.1 GT:GQ:PL 0/1:21:18,0,29 0/1:3:0,3,26 1/1:10:75,12,0 0/1:15:11,0,31</p>

      <p>13 109661359 rs3825491 C G 999 maxCVG AC1=6;AF1=0.7326;DB;DP=50;DP4=3,3,13,30;FQ=35.7;MQ=60;PV4=0.38,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=P853A;SNPEFF_CODON_CHANGE=Cct/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYO16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198950.1;VDB=0.0041;DN=107;DA=C/G;GM=NM_001198950.1,NM_015011.1;GL=MYO16;FG=missense;FD=unknown;AAC=PRO/ALA;PP=853/1881,831/1859;CDP=2557,2491;GS=27;PH=0.97;CP=1.000;CG=5.660;AA=C;HA=28.8;HE=39.6;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=39;GESP=G:4727/C:8279;PAC=NP_001185879.1,NP_055826.1 GT:GQ:PL 0/1:18:21,0,19 0/1:55:84,0,56 1/1:97:160,96,0 1/1:8:46,6,0</p>

      <p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

      <p>14 20404736 rs2792146 G A 143 highCVG AC1=4;AF1=0.4995;DB;DP=17;DP4=3,1,8,3;FQ=42.3;MQ=60;PV4=1,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=R304H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004063.2;VDB=0.0150;DN=100;DA=G/A;GM=NM_001004063.2;GL=OR4K1;FG=missense;FD=missense;AAC=ARG/HIS;PP=304/312;CDP=911;GS=29;PH=0.003;CP=0.000;CG=-4.590;AA=G;HA=23.3;HC=30.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=222291;KP=hsa04740:OlfactoryTransduction;GESP=A:4807/G:8199;PAC=NP_001004063.2 GT:GQ:PL 0/1:34:39,0,31 0/0:3:0,3,28 1/1:21:149,24,0 0/0:3:0,3,25</p>

      <p>14 20482919 rs17308108 A C 27.60 highCVG AC1=2;AF1=0.2756;DB;DP=32;DP4=12,11,5,1;FQ=28.9;MQ=60;PV4=0.35,4.3e-07,1,0.024;SNPEFF_AMINO_ACID_CHANGE=L145R;SNPEFF_CODON_CHANGE=cTg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004712.1;VDB=0.0161;DN=123;DA=A/C;GM=NM_001004712.1;GL=OR4K14;FG=missense;FD=missense;AAC=LEU/ARG;PP=145/311;CDP=434;GS=102;PH=0.994;CP=0.198;CG=4.040;AA=A;HA=5.8;HE=10.2;HC=11.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=274987;KP=hsa04740:OlfactoryTransduction;GESP=C:894/A:12112;PAC=NP_001004712.1 GT:GQ:PL 0/0:44:0,42,160 0/0:6:0,3,19 1/1:11:73,18,0 0/0:26:0,24,111</p>

      <p>14 20528207 rs1958715 G A 143 highCVG AC1=5;AF1=0.58;DB;DP=15;DP4=3,1,7,4;FQ=17.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D2N;SNPEFF_CODON_CHANGE=Gat/Aat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004717.1;VDB=0.0157;DN=92;DA=G/A;GM=NM_001004717.1;GL=OR4L1;FG=missense;FD=missense;AAC=ASP/ASN;PP=2/313;CDP=4;GS=23;PH=0.002;CP=0.786;CG=1.400;HA=45.1;HE=44.5;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=229699;KP=hsa04740:OlfactoryTransduction;GESP=A:5603/G:7401;PAC=NP_001004717.1 GT:GQ:PL 0/1:26:34,0,24 0/1:4:0,3,25 1/1:19:113,21,0 0/1:12:41,0,10</p>

      <p>14 20528321 rs1958716 A G 62.10 midCVG AC1=4;AF1=0.5177;DB;DP=12;DP4=2,4,3,3;FQ=65.2;MQ=60;PV4=1,0.028,1,0.098;SNPEFF_AMINO_ACID_CHANGE=M40V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004717.1;VDB=0.0138;DN=92;DA=A/G;GM=NM_001004717.1;GL=OR4L1;FG=missense;FD=missense;AAC=MET/VAL;PP=40/313;CDP=118;GS=21;PH=0;CP=0.001;CG=-5.830;HA=45.1;HE=44.5;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=229585;KP=hsa04740:OlfactoryTransduction;GESP=G:5619/A:7387;PAC=NP_001004717.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,77 1/1:9:79,12,0 0/1:29:26,0,46</p>

      <p>14 20897676 rs1953225 T C 48 highCVG AC1=4;AF1=0.4622;DB;DP=33;DP4=22,4,0,5;FQ=50.6;MQ=60;PV4=0.00074,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E345G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHL33;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001109997.2;VDB=0.0074;DN=92;DA=T/C;GM=NM_001109997.2,XM_005267347.1;GL=KLHL33;FG=missense;FD=missense,unknown;AAC=GLU/GLY;PP=345/534,609/798;CDP=1034,1826;GS=98;PH=0.018;CP=0.986;CG=5.340;AA=T;HA=38.4;HE=21.5;HC=25.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;GESP=C:1053/T:3513;PAC=NP_001103467.2,XP_005267404.1 GT:GQ:PL 0/0:67:0,69,127 0/1:3:0,0,0 1/1:6:56,9,0 0/1:37:35,0,47</p>

      <p>14 21770730 rs6571751 A G 150 midCVG AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3 GT:GQ:PL 0/1:18:39,0,18 0/1:21:83,0,21 1/1:7:49,6,0 1/1:5:22,3,0</p>

      <p>14 39556185 rs8018720 G C 105 midCVG AC1=5;AF1=0.6629;DB;DP=14;DP4=1,3,6,3;FQ=23.7;MQ=59;PV4=0.27,0.14,0.26,0.26;SNPEFF_AMINO_ACID_CHANGE=L211V;SNPEFF_CODON_CHANGE=Ctt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006364.2;VDB=0.0026;DN=116;DA=G/C;GM=NM_006364.2,XM_005267262.1,XM_005267263.1;GL=SEC23A;FG=missense;FD=missense,unknown,unknown;AAC=LEU/VAL;PP=211/766,211/790,182/737;CDP=631,631,544;GS=32;PH=0;CP=0.826;CG=4.770;AA=C;HA=10.8;HE=12.5;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;TFBS=V$MEF2_03;GESP=C:10877/G:2129;PPI=SEC24D:959|SEC24A:900|SEC24C:845;PAC=NP_006355.2,XP_005267319.1,XP_005267320.1 GT:GQ:PL 0/1:3:0,0,0 0/1:43:41,0,46 1/1:12:74,12,0 0/1:15:31,0,14</p>

      <p>14 58681951 . T C 41 lowCVG AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1 GT:GQ:PL 0/1:33:31,0,37 0/0:4:0,6,50 1/1:3:26,3,0 1/1:3:27,3,0</p>

      <p>14 60903757 rs1254319 G A 70.80 midCVG AC1=6;AF1=0.7392;DB;DP=12;DP4=1,2,1,5;FQ=25.5;MQ=60;PV4=1,0.16,1,0.093;SNPEFF_AMINO_ACID_CHANGE=L524F;SNPEFF_CODON_CHANGE=Ctt/Ttt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C14orf39;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174978.2;VDB=0.0032;DN=87;DA=G/A;GM=NM_174978.2;GL=C14orf39;FG=missense;FD=unknown;AAC=LEU/PHE;PP=524/588;CDP=1570;GS=22;PH=0;CP=0.998;CG=4.030;AA=A;HA=45.3;HE=28.7;HC=36.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=9;GESP=A:4716/G:8288;PAC=NP_777638.2 GT:GQ:PL 0/1:32:26,0,57 0/1:3:0,0,0 1/1:12:82,12,0 0/1:3:0,0,0</p>

      <p>14 63174960 rs4902176 C T 17 highCVG AC1=4;AF1=0.4355;DB;DP=15;DP4=2,8,1,3;FQ=19.4;MQ=60;PV4=1,1,1,0.21;SNPEFF_AMINO_ACID_CHANGE=A745T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139318.4;VDB=0.0080;DN=111;DA=C/T;GM=NM_139318.4,NM_172375.2;GL=KCNH5;FG=missense,3-prime-UTR;FD=unknown;AAC=ALA/THR,none;PP=745/989,NA;CDP=2233,NA;GS=58,NA;PH=0.017;CP=0.997;CG=5.520;AA=C;HA=19.0;HE=34.4;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=214;GESP=T:3891/C:9115;PAC=NP_647479.2,NA GT:GQ:PL 0/0:14:0,15,81 0/1:13:12,0,62 1/1:3:46,6,0 0/1:3:0,0,0</p>

      <p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

      <p>14 74759477 rs4148077 C T 120 midCVG AC1=6;AF1=0.7273;DB;DP=13;DP4=3,0,6,3;FQ=8.59;MQ=60;PV4=0.51,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A304T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005050.3;VDB=0.0175;DN=110;DA=C/T;GM=NM_005050.3,NR_003256.2,XM_005267935.1,XM_005267936.1,XM_005267937.1,XM_005267938.1,XM_005267939.1,XM_005267940.1,XM_005267941.1,XM_005267942.1,XM_005267943.1,XM_005267944.1,XM_005267945.1,XM_005267946.1,XM_005267947.1,XM_005267948.1,XM_005267949.1,XM_005267950.1,XM_005267951.1,XM_005267952.1,XM_005267953.1,XM_005267954.1;GL=ABCD4;FG=missense,non-coding-exon,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR,none,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR;PP=304/607,NA,287/576,304/529,200/489,304/475,166/469,166/469,166/469,166/469,304/455,145/448,145/448,145/448,145/448,145/448,145/448,145/448,41/344,41/344,41/344,41/344;CDP=910,934,859,910,598,910,496,496,496,496,910,433,433,433,433,433,433,433,121,121,121,121;GS=58,NA,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58;PH=0.003;CP=0.998;CG=2.140;AA=C;HA=16.7;HE=29.2;HC=23.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=27;KP=hsa02010:ABCTransporters|hsa04146:Peroxisome;GESP=T:3999/C:9007;PAC=NP_005041.1,NA,XP_005267992.1,XP_005267993.1,XP_005267994.1,XP_005267995.1,XP_005267996.1,XP_005267997.1,XP_005267998.1,XP_005267999.1,XP_005268000.1,XP_005268001.1,XP_005268002.1,XP_005268003.1,XP_005268004.1,XP_005268005.1,XP_005268006.1,XP_005268007.1,XP_005268008.1,XP_005268009.1,XP_005268010.1,XP_005268011.1 GT:GQ:PL 0/1:18:12,0,27 0/1:18:17,0,20 1/1:16:88,15,0 1/1:8:48,6,0</p>

      <p>14 75475817 rs145117455 A G 117 highCVG AC1=4;AF1=0.4529;DB;DP=18;DP4=3,4,8,2;FQ=64.3;MQ=60;PV4=0.16,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I328V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EIF2B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014239.3;VDB=0.0051;DN=134;DA=A/G;GM=NM_014239.3;GL=EIF2B2;FG=missense;FD=missense;AAC=ILE/VAL;PP=328/352;CDP=982;GS=29;PH=0.146;CP=1.000;CG=5.770;AA=A;DV=by-frequency,by-cluster;DSP=84;GESP=G:4/A:13002;PAC=NP_055054.1 GT:GQ:PL 0/0:11:0,12,73 0/1:3:0,0,0 1/1:4:44,6,0 0/1:34:114,0,30</p>

      <p>14 88459795 rs73312836 C T 22.20 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=3,5,2,1;FQ=24.4;MQ=60;PV4=0.55,1,1,0.005;SNPEFF_AMINO_ACID_CHANGE=A17T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GALC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001201402.1;VDB=0.0092;DN=130;DA=C/T;GM=NM_001201402.1;GL=GALC;FG=missense;FD=missense;AAC=ALA/THR;PP=17/660;CDP=49;GS=58;CP=0.000;CG=-0.125;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=69;KP=hsa00600:SphingolipidMetabolism|hsa01100:MetabolicPathways|hsa04142:Lysosome;PAC=NP_001188331.1 GT:GQ:PL 0/0:15:0,15,85 0/1:4:20,3,0 1/1:3:48,6,0 0/0:9:0,9,64</p>

      <p>14 92465749 rs35007347 C T 31.40 highCVG AC1=3;AF1=0.362;DB;DP=18;DP4=9,5,2,2;FQ=33.3;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=R1576H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIP11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004239.3;VDB=0.0065;DN=126;DA=C/T;GM=NM_004239.3,XM_005268213.1,XM_005268214.1,XM_005268215.1;GL=TRIP11;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/HIS;PP=1576/1980,1547/1951,1134/1538,566/970;CDP=4727,4640,3401,1697;GS=29;PH=1;CP=0.895;CG=5.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=T:254/C:12752;PAC=NP_004230.2,XP_005268270.1,XP_005268271.1,XP_005268272.1 GT:GQ:PL 0/0:30:0,30,154 0/1:36:36,0,57 1/1:3:39,6,0 0/0:5:0,3,21</p>

      <p>14 94088435 . A G 9.11 lowCVG AC1=7;AF1=0.9732;DP=8;DP4=1,0,4,0;FQ=-25.6;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=E1442G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UNC79;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020818.3;VDB=0.0000;GM=NM_020818.3,XM_005267917.1;GL=UNC79;FG=missense;FD=unknown;AAC=GLU/GLY;PP=1442/2459,1642/2659;CDP=4325,4925;GS=98;PH=1;CP=0.501;CG=5.650;AA=A;DSP=469;GESP=A:13006;PAC=NP_065869.3,XP_005267974.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:14,3,0 0/1:5:27,0,2</p>

      <p>14 102815042 rs7011 C T 107 midCVG AC1=5;AF1=0.6413;DB;DP=13;DP4=0,5,3,5;FQ=6.84;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R164H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CINP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032630.2;VDB=0.0109;DN=52;DA=C/T;GM=NM_032630.2,XM_005267753.1;GL=CINP;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=ARG/HIS,none;PP=164/213,NA;CDP=491,NA;GS=29,NA;PH=0.054;CP=0.701;CG=2.760;AA=C;HA=5.8;HE=31.7;HC=11.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=T:3032/C:9974;PAC=NP_116019.1,NA GT:GQ:PL 0/1:3:0,0,0 0/1:12:53,0,11 1/1:6:48,6,0 0/1:31:46,0,30</p>

      <p>14 104139526 . T C 20.20 midCVG AC1=2;AF1=0.2343;DP=13;DP4=3,6,3,0;FQ=21.3;MQ=60;PV4=0.18,0.47,1,1;VDB=0.0000;GM=NM_001130107.1,NM_005552.4,NM_182923.3,XM_005267599.1,XM_005267600.1,XM_005267601.1,XM_005267602.1,XM_005267603.1,XM_005267604.1,XM_005267605.1,XM_005267606.1,XM_005267607.1,XM_005267608.1,XM_005267609.1,XM_005267610.1,XM_005267611.1,XM_005267612.1,XM_005267613.1,XM_005267614.1,XM_005267615.1,XM_005267616.1,XM_005267617.1,XM_005267618.1,XM_005267619.1,XM_005267620.1,XM_005267621.1,XM_005267622.1,XM_005267623.1,XM_005267624.1,XM_005267625.1,XM_005267626.1,XM_005267627.1,XM_005267628.1,XM_005267629.1,XM_005267630.1,XM_005267631.1;GL=KLC1;FG=splice-donor;FD=unknown;CP=0.923;CG=5.560;AA=T;DSP=0;GESP=T:13006;PPI=KIF5B:937|APP:846|KIF5A:845|KIF5C:845;PAC=NP_001123579.1,NP_005543.2,NP_891553.2,XP_005267656.1,XP_005267657.1,XP_005267658.1,XP_005267659.1,XP_005267660.1,XP_005267661.1,XP_005267662.1,XP_005267663.1,XP_005267664.1,XP_005267665.1,XP_005267666.1,XP_005267667.1,XP_005267668.1,XP_005267669.1,XP_005267670.1,XP_005267671.1,XP_005267672.1,XP_005267673.1,XP_005267674.1,XP_005267675.1,XP_005267676.1,XP_005267677.1,XP_005267678.1,XP_005267679.1,XP_005267680.1,XP_005267681.1,XP_005267682.1,XP_005267683.1,XP_005267684.1,XP_005267685.1,XP_005267686.1,XP_005267687.1,XP_005267688.1 GT:GQ:PL 0/0:12:0,9,64 0/0:7:0,3,26 1/1:3:62,9,0 0/0:18:0,15,104</p>

      <p>14 105413204 rs2582514 G T 89 midCVG AC1=6;AF1=0.707;DB;DP=10;DP4=1,2,5,2;FQ=7.98;MQ=55;PV4=0.5,1,0.15,0.061;SNPEFF_AMINO_ACID_CHANGE=R2862S;SNPEFF_CODON_CHANGE=Cgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0059;DN=100;DA=G/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=2862/5796,2762/5696;CDP=8584,8284;GS=110;PH=0;CP=0.000;CG=0.525;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7933;GESP=T:7253/G:4947;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:39:79,0,38</p>

      <p>14 105415608 rs11850848 G C 19.70 midCVG AC1=4;AF1=0.4707;DB;DP=12;DP4=8,0,4,0;FQ=22.3;MQ=57;PV4=1,3.7e-05,0.00042,1;SNPEFF_AMINO_ACID_CHANGE=H2060Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0001;DN=120;DA=G/C;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=HIS/GLN;PP=2060/5796,1960/5696;CDP=6180,5880;GS=24;PH=0.022;CP=0.000;CG=-2.470;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=5529;GESP=C:2565/G:6715;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/0:23:0,24,114 1/1:8:59,12,0 0/1:3:0,0,0</p>

      <p>14 105418391 rs11625007 C T 60.30 midCVG AC1=5;AF1=0.6662;DB;DP=10;DP4=2,1,0,6;FQ=18.1;MQ=60;PV4=0.083,1,0.26,1;SNPEFF_AMINO_ACID_CHANGE=V1133I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0023;DN=120;DA=C/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=1133/5796,1033/5696;CDP=3397,3097;GS=29;PH=0.017;CP=0.001;CG=-7.740;AA=T;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2746;GESP=T:5916/C:6260;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:50:70,0,48</p>

      <p>15 22368862 rs1835183 G A 35.10 lowCVG AC1=4;AF1=0.4751;DB;DP=8;DP4=2,2,3,1;FQ=27.6;MQ=57;PV4=1,1,0.45,0.28;SNPEFF_AMINO_ACID_CHANGE=G96E;SNPEFF_CODON_CHANGE=gGa/gAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4M2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004719.2;VDB=0.0087;DN=92;DA=G/A;GM=NM_001004719.2,NR_015416.1;GL=OR4M2/LOC727924;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=GLY/GLU,none;PP=96/314,NA;CDP=287,1319;GS=98,NA;PH=0.97;CP=0.990;CG=2.500;HE=1.9;HC=28.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=55;KP=hsa04740:OlfactoryTransduction;GESP=A:7271/G:5735;PAC=NP_001004719.2,NA GT:GQ:PL 0/0:8:0,9,53 0/1:3:0,0,0 1/1:4:42,6,0 0/1:17:34,0,13</p>

      <p>15 29418520 rs2256273 C T 23.10 lowCVG AC1=5;AF1=0.5568;DB;DP=8;DP4=4,1,3,0;FQ=26.6;MQ=60;PV4=1,0.058,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R393H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM189A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015307.1;VDB=0.0105;DN=100;DA=C/T;GM=NM_015307.1,XM_005254261.1,XM_005254262.1;GL=FAM189A1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=393/540,430/577,252/399;CDP=1178,1289,755;GS=29;PH=0.005;CP=0.000;CG=-2.140;AA=C;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=51;TFBS=V$P300_01;GESP=T:3037/C:1529;PAC=NP_056122.1,XP_005254318.1,XP_005254319.1 GT:GQ:PL 0/1:3:22,3,0 0/1:3:0,0,1/1:5:43,6,0 0/0:12:0,15,102</p>

      <p>15 41105926 rs690347 T G 87.70 midCVG AC1=6;AF1=0.706;DB;DP=11;DP4=2,0,2,6;FQ=-18.1;MQ=59;PV4=0.13,1,0.32,1;SNPEFF_AMINO_ACID_CHANGE=S376A;SNPEFF_CODON_CHANGE=Tcc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZFYVE19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001077268.1;VDB=0.0034;DN=83;DA=T/G;GM=NM_001077268.1,NM_001258420.1,NM_001258421.1,NM_032850.4;GL=ZFYVE19;FG=missense;FD=missense;AAC=SER/ALA;PP=376/472,308/404,201/297,366/462;CDP=1126,922,601,1096;GS=99;PH=0;CP=1.000;CG=5.270;AA=G;HA=15.5;HE=28.1;HC=7.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;GESP=G:9475/T:3081;PAC=NP_001070736.1,NP_001245349.1,NP_001245350.1,NP_116239.3 GT:GQ:PL 0/1:6:63,0,4 0/1:3:0,0,0 1/1:7:41,6,0 0/1:11:22,0,11</p>

      <p>15 42154738 rs16972193 G A 26.10 midCVG AC1=5;AF1=0.5822;DB;DP=9;DP4=2,0,4,0;FQ=9.38;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R2462W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016642.3;VDB=0.0109;DN=123;DA=G/A;GM=NM_016642.3,XM_005254438.1;GL=SPTBN5;FG=missense-near-splice;FD=missense,unknown;AAC=ARG/TRP;PP=2462/3675,2482/3695;CDP=7384,7444;GS=101;PH=0.213;CP=0.837;CG=-4.960;AA=G;HE=13.8;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=A:1149/G:11369;PAC=NP_057726.4,XP_005254495.1 GT:GQ:PL 0/0:5:0,6,43 0/1:3:0,0,0 1/1:9:64,12,0 0/1:3:0,0,0</p>

      <p>15 42570718 rs8024732 A G 106 highCVG AC1=4;AF1=0.501;DB;DP=26;DP4=3,9,5,8;FQ=108;MQ=60;PV4=0.67,0.04,1,1;SNPEFF_AMINO_ACID_CHANGE=Q44R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GANC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198141.2;VDB=0.0051;DN=116;DA=A/G;GM=NM_198141.2,XM_005254286.1;GL=GANC;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=44/915,44/913;CDP=131;GS=43;PH=0;CP=0.454;CG=-2.850;AA=A;HA=31.5;HE=11.0;HC=11.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;KP=hsa00052:GalactoseMetabolism|hsa00500:StarchAndSucroseMetabolism|hsa01100:MetabolicPathways;GESP=G:9331/A:3673;PAC=NP_937784.2,XP_005254343.1 GT:GQ:PL 0/1:13:11,0,15 0/0:11:0,14,130 1/1:12:59,15,0 0/1:18:83,0,15</p>

      <p>15 44120559 rs678084 T G 24.40 lowCVG AC1=4;AF1=0.5372;DB;DP=7;DP4=3,1,3,0;FQ=27.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S153A;SNPEFF_CODON_CHANGE=Tcc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024908.3;VDB=0.0062;DN=83;DA=T/G;GM=NM_001167941.1,NM_024908.3,XM_005254680.1,XM_005254681.1;GL=WDR76;FG=missense;FD=missense,missense,unknown,unknown;AAC=SER/ALA;PP=89/563,153/627,89/563,153/473;CDP=265,457,265,457;GS=99;PH=0;CP=0.524;CG=2.540;AA=G;HA=30.1;HE=20.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=G:10282/T:2710;PAC=NP_001161413.1,NP_079184.2,XP_005254737.1,XP_005254738.1 GT:GQ:PL 0/1:19:20,0,19 0/0:7:0,9,58 1/1:3:23,3,0 1/1:3:25,3,0</p>

      <p>15 48057210 . G A 92.30 maxCVG AC1=3;AF1=0.3739;DP=51;DP4=2,8,39,2;FQ=94.3;MQ=60;PV4=2.9e-06,2.1e-06,1,1;SNPEFF_AMINO_ACID_CHANGE=D462N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEMA6D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153618.1;VDB=0.0007;GM=NM_001198999.1,NM_020858.1,NM_024966.2,NM_153616.1,NM_153617.1,NM_153618.1,NM_153619.1,XM_005254685.1,XM_005254686.1,XM_005254687.1,XM_005254688.1,XM_005254689.1,XM_005254690.1;GL=SEMA6D;FG=missense;FD=unknown;AAC=ASP/ASN;PP=462/1012,462/1012,462/477,462/999,462/1018,462/1074,462/598,462/1087,462/1087,462/1074,462/1068,462/1055,462/1031;CDP=1384;GS=23;PH=0.993;CP=1.000;CG=5.790;AA=G;DSP=44;KP=hsa04360:AxonGuidance;GESP=G:12990;PAC=NP_001185928.1,NP_065909.1,NP_079242.2,NP_705869.1,NP_705870.1,NP_705871.1,NP_705872.1,XP_005254742.1,XP_005254743.1,XP_005254744.1,XP_005254745.1,XP_005254746.1,XP_005254747.1 GT:GQ:PL 0/0:9:0,9,67 0/0:12:0,12,80 1/1:99:134,120,0 0/1:7:5,0,60</p>

      <p>15 65676726 rs33918653 C T 78.50 midCVG AC1=4;AF1=0.5302;DB;DP=10;DP4=0,4,5,1;FQ=24.8;MQ=60;PV4=0.048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C1125Y;SNPEFF_CODON_CHANGE=tGc/tAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0003;DN=126;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=CYS/TYR;PP=1125/1251;CDP=3374;GS=194;PH=0.981;CP=1.000;CG=4.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=T:2295/C:10705;PAC=NP_066013.1 GT:GQ:PL 0/0:5:0,6,38 0/1:3:0,0,0 1/1:7:70,9,0 0/1:28:50,0,25</p>

      <p>15 65684513 rs148990036 C T 29.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,3,0;FQ=21.1;MQ=59;PV4=0.4,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R694Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0000;DN=134;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=ARG/GLN;PP=694/1251;CDP=2081;GS=43;PH=0.968;CP=0.939;CG=5.490;AA=C;DV=by-frequency,by-cluster;DSP=42;GESP=T:88/C:12878;PAC=NP_066013.1 GT:GQ:PL 0/0:9:0,9,56 0/1:3:0,0,0 1/1:6:67,9,0 0/1:3:0,0,0</p>

      <p>15 81598269 rs11556218 T G 14.60 lowCVG AC1=2;AF1=0.3027;DB;DP=7;DP4=3,1,1,2;FQ=16;MQ=60;PV4=0.49,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=N1147K;SNPEFF_CODON_CHANGE=aaT/aaG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_172217.3;VDB=0.0093;DN=120;DA=T/G;GM=NM_001172128.1,NM_004513.5,NM_172217.3,XM_005254342.1,XM_005254343.1,XM_005254344.1,XM_005254345.1,XM_005254346.1;GL=IL16;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=ASN/LYS;PP=1147/1332,446/632,1147/1333,1194/1380,1194/1379,537/723,537/722,446/632;CDP=3441,1338,3441,3582,3582,1611,1611,1338;GS=94;PH=1;CP=0.007;CG=-6.300;AA=T;HA=25.0;HE=10.0;HC=20.6;DG;DV=by-frequency,by-cluster,by-1000G;DSP=21;TFBS=V$YY1_01;GESP=G:1601/T:11405;PPI=CD4:846;PAC=NP_001165599.1,NP_004504.3,NP_757366.2,XP_005254399.1,XP_005254400.1,XP_005254401.1,XP_005254402.1,XP_005254403.1 GT:GQ:PL 0/0:6:0,3,19 0/0:8:0,6,35 1/1:4:55,9,0 0/0:6:0,3,26</p>

      <p>15 83215190 . C G 63.20 highCVG AC1=2;AF1=0.2897;DP=17;DP4=3,4,0,8;FQ=62.9;MQ=60;PV4=0.077,1,1,1;VDB=0.0006;GM=NM_001079533.1,NM_001079534.1,NM_001079535.1,NM_030594.3,XM_005272441.1,XM_005272442.1,XM_005272443.1,XM_005272444.1,XM_005272445.1,XM_005272446.1;GL=CPEB1;FG=splice-donor;FD=unknown;CP=1.000;CG=4.850;AA=C;DSP=0;KP=hsa04114:OocyteMeiosis|hsa04320:Dorso-ventralAxisFormation|hsa04914:Progesterone-mediatedOocyteMaturation;TFBS=V$AREB6_02,V$SREBP1_01;GESP=C:12084;PPI=UBC:943|SPI1:845|GRIN3A:830|MRTO4:830;PAC=NP_001073001.1,NP_001073002.1,NP_001073003.1,NP_085097.3,XP_005272498.1,XP_005272499.1,XP_005272500.1,XP_005272501.1,XP_005272502.1,XP_005272503.1 GT:GQ:PL 0/0:8:0,6,37 0/0:6:0,3,27 1/1:6:106,12,0 0/0:11:0,9,45</p>

      <p>15 90174824 rs12900805 C T 23.70 midCVG AC1=6;AF1=0.8319;DB;DP=9;DP4=1,0,1,2;FQ=-24.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G1005R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198525.2;VDB=0.0106;DN=121;DA=C/T;GM=NM_198525.2,XM_005254902.1,XM_005254903.1,XM_005254904.1;GL=KIF7;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ARG;PP=1005/1344,1005/1344,1004/1343,1004/1343;CDP=3013,3013,3010,3010;GS=125;PH=0.004;CP=0.067;CG=-0.473;AA=C;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=99;TFBS=V$SREBP1_02;GESP=T:6862/C:6054;PAC=NP_940927.2,XP_005254959.1,XP_005254960.1,XP_005254961.1 GT:GQ:PL 0/1:3:0,0,0/1:6:29,0,4 1/1:4:28,3,0 0/1:3:0,0,0</p>

      <p>16 715990 rs7190775 G A 37.20 lowCVG AC1=5;AF1=0.6489;DB;DP=5;DP4=0,1,4,0;FQ=-10.8;MQ=60;PV4=0.2,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R1492H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR90;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145294.4;VDB=0.0001;DN=116;DA=G/A;GM=NM_145294.4,XM_005255160.1,XM_005255161.1;GL=WDR90;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=1492/1749,1495/1752,1494/1751;CDP=4475,4484,4481;GS=29;PH=0.169;CP=0.000;CG=-4.830;AA=G;HA=11.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;GESP=A:5864/G:6472;PAC=NP_660337.3,XP_005255217.1,XP_005255218.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,22 1/1:10:74,12,0 0/1:3:0,0,0</p>

      <p>16 840378 rs3765263 A G 66.70 midCVG AC1=3;AF1=0.3829;DB;DP=14;DP4=6,0,8,0;FQ=58.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K244R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHTF18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022092.2;VDB=0.0000;DN=107;DA=A/G;GM=NM_022092.2,XM_005255470.1,XM_005255471.1,XM_005255472.1;GL=CHTF18;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=LYS/ARG,LYS/ARG,LYS/ARG,none;PP=244/976,128/860,272/1004,NA;CDP=731,383,815,NA;GS=26,26,26,NA;PH=0;CP=0.081;CG=-5.390;AA=G;HA=33.2;HE=22.3;HC=42.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4000/A:8074;PPI=PCNA:939|RFC4:846|DSCC1:845|RFC3:845|RFC2:845;PAC=NP_071375.1,XP_005255527.1,XP_005255528.1,NA GT:GQ:PL 0/0:12:0,12,60 0/1:3:0,0,0 1/1:19:110,24,0 0/0:7:0,6,40</p>

      <p>16 1838640 rs344359 T C 5.70 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,3,0;FQ=-28.6;MQ=60;SNPEFF_AMINO_ACID_CHANGE=W99R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001284502.1;VDB=0.0025;DN=79;DA=T/C;GM=NM_012225.2,XM_005255025.1,XM_005255026.1,XM_005255027.1,XM_005255028.1,XM_005255029.1,XM_005255030.1;GL=NUBP2;FG=synonymous;FD=synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO;PP=247/272,201/226,187/212,187/212,187/212,106/131,106/131;CDP=741,603,561,561,561,318,318;CP=0.022;CG=0.180;AA=C;HA=3.6;HE=20.7;HC=32.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;GESP=C:11162/T:1824;PAC=NP_036357.1,XP_005255082.1,XP_005255083.1,XP_005255084.1,XP_005255085.1,XP_005255086.1,XP_005255087.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:25,6,0 1/1:4:13,3,0</p>

      <p>16 2854507 rs61747737 T G 133 highCVG AC1=4;AF1=0.4823;DB;DP=35;DP4=6,1,23,2;FQ=81.4;MQ=60;PV4=0.54,1,0.3,1;SNPEFF_AMINO_ACID_CHANGE=F233C;SNPEFF_CODON_CHANGE=tTt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS41;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135086.1;VDB=0.0087;DN=129;DA=T/G;GM=NM_001135086.1;GL=PRSS41;FG=missense;FD=missense;AAC=PHE/CYS;PP=233/319;CDP=698;GS=205;PH=0.99;CP=0.167;CG=4.230;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=41;GESP=G:264/T:4302;PAC=NP_001128558.1 GT:GQ:PL 0/0:7:0,9,62 0/1:3:24,3,0 1/1:48:158,52,0 0/0:7:0,9,65</p>

      <p>16 2905689 rs199712440 G A 23.60 lowCVG AC1=3;AF1=0.4317;DB;DP=7;DP4=2,1,2,1;FQ=25.1;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R149C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022119.3;VDB=0.0096;DN=137;DA=G/A;GM=NM_022119.3,XM_005255473.1;GL=PRSS22;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=149/318;CDP=445;GS=180;PH=0.99;CP=0.006;CG=3.430;AA=G;DV=by-cluster;DSP=115;TFBS=V$PAX4_01;GESP=A:1/G:12995;PAC=NP_071402.1,XP_005255530.1 GT:GQ:PL 0/0:4:0,3,24 0/0:4:0,3,29 1/1:4:50,6,0 0/1:16:15,0,19</p>

      <p>16 5139186 rs12928528 G C 82.20 highCVG AC1=4;AF1=0.5055;DB;DP=16;DP4=2,5,2,6;FQ=74.9;MQ=54;PV4=1,0.19,1,0.18;SNPEFF_AMINO_ACID_CHANGE=H272D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM86A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_201400.3;VDB=0.0194;DN=121;DA=G/C;GM=NM_201400.2,NM_201598.2,XM_005255157.1,XM_005255158.1,XM_005255159.1;GL=FAM86A;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ASP;PP=272/331,238/297,245/304,172/231,172/231;CDP=814,712,733,514,514;GS=81;PH=0;CP=0.000;CG=-2.010;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=72;GESP=C:3557/G:4861;PAC=NP_958802.1,NP_963892.1,XP_005255214.1,XP_005255215.1,XP_005255216.1 GT:GQ:PL 0/0:5:0,6,25 0/0:13:0,15,97 1/1:15:88,18,0 1/1:5:44,6,0</p>

      <p>16 10837913 rs2233531 C G 30.10 highCVG AC1=3;AF1=0.3384;DB;DP=23;DP4=2,15,4,1;FQ=31.8;MQ=60;PV4=0.0093,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P39A;SNPEFF_CODON_CHANGE=Ccg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NUBP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002484.3;VDB=0.0101;DN=98;DA=C/G;GM=NM_001278506.1,NM_002484.3;GL=NUBP1;FG=missense;FD=unknown,missense;AAC=PRO/ALA;PP=39/310,39/321;CDP=115;GS=27;PH=0.002;CP=0.509;CG=3.830;AA=C;HA=15.8;HE=10.0;HC=2.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;CPG=10837585-10838196;GESP=G:1334/C:11630;PAC=NP_001265435.1,NP_002475.2 GT:GQ:PL 0/0:3:0,0,0 0/0:5:0,3,23 1/1:4:70,9,0 0/0:18:0,17,85</p>

      <p>16 12897578 rs3748976 G T 56.70 midCVG AC1=5;AF1=0.5742;DB;DP=11;DP4=1,2,2,3;FQ=7.33;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A19D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018340.2;VDB=0.0072;DN=107;DA=G/T;GM=NM_001099455.1,NM_018340.2;GL=CPPED1;FG=missense;FD=missense;AAC=ALA/ASP;PP=19/173,19/315;CDP=56;GS=126;PH=0;CP=1.000;CG=4.810;AA=T;HA=26.9;HE=11.4;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;CPG=12897366-12897889;GESP=T:3654/G:8256;PAC=NP_001092925.1,NP_060810.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,16 1/1:7:69,9,0 0/1:27:28,0,27</p>

      <p>16 15111218 rs4985162 C T 42.50 highCVG AC1=5;AF1=0.5528;DB;DP=27;DP4=8,9,4,3;FQ=46;MQ=32;PV4=1,1,0.18,1;SNPEFF_AMINO_ACID_CHANGE=P301L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PDXDC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015027.3;VDB=0.0062;DN=111;DA=C/T;GM=NM_015027.2,XM_005255173.1,XM_005255174.1,XM_005255175.1,XM_005255176.1,XM_005255177.1,XM_005255178.1,XM_005255179.1,XM_005255180.1,XM_005255181.1;GL=PDXDC1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=301/789,319/807,310/798,300/788,286/774,286/774,282/770,274/762,273/761,210/698;CDP=902,956,929,899,857,857,845,821,818,629;GS=98;PH=0.015;CP=0.484;CG=4.490;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;TFBS=V$PPARA_01;GESP=T:2213/C:10775;PAC=NP_055842.2,XP_005255230.1,XP_005255231.1,XP_005255232.1,XP_005255233.1,XP_005255234.1,XP_005255235.1,XP_005255236.1,XP_005255237.1,XP_005255238.1 GT:GQ:PL 0/0:35:0,39,138 0/1:18:14,0,36 1/1:10:62,12,0 1/1:3:14,3,0</p>

      <p>16 15131962 rs1135999 A G 59.20 highCVG AC1=5;AF1=0.6381;DB;DP=21;DP4=3,1,1,13;FQ=9.29;MQ=57;PV4=0.019,0.29,0.23,1;SNPEFF_AMINO_ACID_CHANGE=S287P;SNPEFF_CODON_CHANGE=Tct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0035;DN=86;DA=A/G;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=SER/PRO;PP=182/206,182/206,287/311;CDP=544,544,859;GS=74;PH=0;CP=1.000;CG=5.870;AA=G;HA=13.3;HE=27.7;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;TFBS=V$FREAC3_01,V$STAT1_01,V$STAT3_01,V$STAT5A_01,V$STAT5B_01;GESP=G:3550/A:9444;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/0:4:0,6,44 0/1:3:0,0,0 1/1:5:23,6,0 0/1:3:76,2,0</p>

      <p>16 15131974 rs1136001 G T 95.50 highCVG AC1=6;AF1=0.7476;DB;DP=17;DP4=2,0,2,12;FQ=-10.1;MQ=57;PV4=0.05,0.05,0.28,1;SNPEFF_AMINO_ACID_CHANGE=H283N;SNPEFF_CODON_CHANGE=Cac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NTAN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173474.3;VDB=0.0015;DN=86;DA=G/T;GM=NM_001270766.1,NM_001270767.1,NM_173474.3;GL=NTAN1;FG=missense;FD=missense;AAC=HIS/ASN;PP=178/206,178/206,283/311;CDP=532,532,847;GS=68;PH=0;CP=0.000;CG=1.840;AA=T;HA=13.4;HE=27.5;HC=41.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=94;GESP=T:3575/G:9419;PAC=NP_001257695.1,NP_001257696.1,NP_775745.1 GT:GQ:PL 0/1:5:0,3,25 0/1:3:0,0,0 1/1:5:20,3,0 1/1:16:116,16,0</p>

      <p>16 55562466 rs837550 G A 132 highCVG AC1=5;AF1=0.5658;DB;DP=21;DP4=11,0,4,5;FQ=111;MQ=60;PV4=0.0081,1,1,0.063;SNPEFF_AMINO_ACID_CHANGE=M163I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LPCAT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017839.4;VDB=0.0146;DN=86;DA=G/A;GM=NM_017839.4,XM_005256006.1;GL=LPCAT2;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=163/545,163/331;CDP=489;GS=10;PH=0;CP=0.006;CG=-3.800;AA=G;HA=38.7;HE=35.2;HC=41.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00564:GlycerophospholipidMetabolism|hsa00565:EtherLipidMetabolism|hsa01100:MetabolicPathways;GESP=A:7296/G:5700;PAC=NP_060309.2,XP_005256063.1 GT:GQ:PL 0/1:14:10,0,42 0/0:20:0,24,113 1/1:3:31,3,0 1/1:19:139,21,0</p>

      <p>16 56673227 rs11640851 C A 13.60 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=2,0,1,1;FQ=13.6;MQ=60;PV4=1,0.23,1,1;SNPEFF_AMINO_ACID_CHANGE=T27N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MT1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005946.2;VDB=0.0011;DN=120;DA=C/A;GM=NM_005946.2;GL=MT1A;FG=missense;FD=missense;AAC=THR/ASN;PP=27/62;CDP=80;GS=65;PH=0.928;CP=0.839;CG=1.960;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=A:6838/C:6158;PAC=NP_005937.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:49,6,0 0/0:7:0,6,49</p>

      <p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

      <p>16 70690989 rs8046424 G C 61.30 lowCVG AC1=5;AF1=0.603;DB;DP=7;DP4=0,2,5,0;FQ=10.6;MQ=60;PV4=0.048,0.29,1,1;SNPEFF_AMINO_ACID_CHANGE=E123Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL34;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172772.1;VDB=0.0024;DN=116;DA=G/C;GM=NM_001172771.1,NM_001172772.1,NM_152456.2,XM_005255810.1;GL=IL34;FG=missense;FD=missense,missense,missense,unknown;AAC=GLU/GLN;PP=122/242,123/243,123/243,123/243;CDP=364,367,367,367;GS=29;PH=0;CP=0.115;CG=3.040;AA=C;HA=0.0;HE=45.8;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=C:8157/G:4839;PPI=CSF1R:959;PAC=NP_001166242.1,NP_001166243.1,NP_689669.2,XP_005255867.1 GT:GQ:PL 0/1:3:0,3,24 0/1:3:0,3,27 1/1:5:44,6,0 1/1:8:62,9,0</p>

      <p>16 75563330 rs3826107 G A 36.20 lowCVG AC1=5;AF1=0.6482;DB;DP=5;DP4=0,1,4,0;FQ=-9.91;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T318M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHST5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024533.4;VDB=0.0001;DN=107;DA=G/A;GM=NM_024533.4,XM_005255880.1;GL=CHST5;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=318/412,324/418;CDP=953,971;GS=81;PH=0.999;CP=0.994;CG=2.840;AA=G;HA=9.7;HE=15.6;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=906;CPG=75563114-75564112;GESP=A:2238/G:10758;PAC=NP_078809.2,XP_005255937.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:10:73,12,0 0/1:3:0,0,0</p>

      <p>16 89350038 rs2279348 G A 53.40 lowCVG AC1=6;AF1=0.6958;DB;DP=6;DP4=0,2,1,3;FQ=8.26;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A971V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256182.1;VDB=0.0084;DN=100;DA=G/A;GM=NM_001256182.1,NM_001256183.1,NM_013275.5;GL=ANKRD11;FG=missense;FD=missense;AAC=ALA/VAL;PP=971/2664;CDP=2912;GS=64;PH=0.014;CP=0.002;CG=-3.500;AA=G;HA=40.8;HE=27.5;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2020;GESP=A:9057/G:3937;PAC=NP_001243111.1,NP_001243112.1,NP_037407.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,44 1/1:6:46,6,0 1/1:6:49,6,0</p>

      <p>17 649547 rs910925 G C 39.60 midCVG AC1=4;AF1=0.5306;DB;DP=10;DP4=2,1,4,0;FQ=11.2;MQ=60;PV4=0.43,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=A579G;SNPEFF_CODON_CHANGE=gCc/gGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GEMIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015721.2;VDB=0.0040;DN=86;DA=G/C;GM=NM_015721.2,XM_005256667.1,XM_005256668.1,XM_005256669.1,XM_005256670.1;GL=GEMIN4;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=579/1059,583/1063,583/1063,568/1048,568/1048;CDP=1736,1748,1748,1703,1703;GS=60;PH=1;CP=0.997;CG=5.840;AA=G;HA=11.7;HE=47.5;HC=33.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1726;GESP=C:8046/G:4332;PPI=DDX20:973|SMN1:904|SMN2:898|AGO2:885;PAC=NP_056536.2,XP_005256724.1,XP_005256725.1,XP_005256726.1,XP_005256727.1 GT:GQ:PL 0/0:3:0,3,24 0/0:5:0,6,28 1/1:7:59,9,0 0/1:3:25,3,0</p>

      <p>17 663472 rs77166377 C A 77.80 highCVG AC1=3;AF1=0.3934;DB;DP=18;DP4=4,8,3,3;FQ=79.9;MQ=59;PV4=0.63,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A280S;SNPEFF_CODON_CHANGE=Gca/Tca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GLOD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016080.3;VDB=0.0035;DN=131;DA=C/A;GM=NM_016080.3;GL=GLOD4;FG=missense;FD=missense;AAC=ALA/SER;PP=280/299;CDP=838;GS=99;PH=0.103;CP=0.999;CG=5.050;AA=C;DV=by-frequency,by-cluster;DSP=7;GESP=A:318/C:12688;PAC=NP_057164.3 GT:GQ:PL 0/0:6:0,6,40 0/1:18:17,0,31 1/1:10:108,15,0 0/0:23:0,24,101</p>

      <p>17 1690752 rs11549830 G A 24.30 lowCVG AC1=3;AF1=0.3495;DB;DP=8;DP4=2,3,3,0;FQ=26.1;MQ=60;PV4=0.2,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R562W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMYD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052928.2;VDB=0.0000;DN=120;DA=G/A;GM=NM_052928.2,XM_005256435.1,XM_005256436.1;GL=SMYD4;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=562/805,562/680,562/636;CDP=1684;GS=101;PH=0.997;CP=0.266;CG=2.540;AA=G;HA=45.0;HE=30.0;HC=45.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=37;GESP=A:4552/G:8438;PAC=NP_443160.2,XP_005256492.1,XP_005256493.1 GT:GQ:PL 0/0:13:0,12,73 0/0:3:0,0,0 1/1:4:64,9,0 0/0:5:0,3,21</p>

      <p>17 5058808 rs9899177 G A 14.40 midCVG AC1=4;AF1=0.4652;DB;DP=9;DP4=1,5,0,3;FQ=17.1;MQ=56;PV4=1,1,6.1e-05,0.17;SNPEFF_AMINO_ACID_CHANGE=R912Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004505.2;VDB=0.0065;DN=119;DA=G/A;GM=NM_004505.2,XM_005256845.1,XM_005256846.1;GL=USP6;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=912/1407,912/1407,595/1090;CDP=2735,2735,1784;GS=43;PH=0.999;CP=1.000;CG=2.910;AA=G;HA=15.2;HE=33.3;HC=10.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT1_01;GESP=A:6766/G:6240;PAC=NP_004496.2,XP_005256902.1,XP_005256903.1 GT:GQ:PL 0/0:8:0,9,57 0/1:3:0,0,0 1/1:4:41,6,0 0/1:15:14,0,40</p>

      <p>17 5087040 rs9908414 A G 60.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=4,4,0,5;FQ=63.2;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I171T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF594;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032530.1;VDB=0.0105;DN=119;DA=A/G;GM=NM_032530.1,XM_005256827.1;GL=ZNF594;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=171/808;CDP=512;GS=89;PH=0;CP=0.203;CG=-2.720;AA=A;HA=9.7;HE=47.3;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=532;GESP=A:12386;PAC=NP_115919.1,XP_005256884.1 GT:GQ:PL 0/0:16:0,18,107 0/0:6:0,6,37 1/1:6:70,9,0 1/1:4:39,6,0</p>

      <p>17 7250481 . C A 31.70 midCVG AC1=5;AF1=0.6147;DP=10;DP4=3,0,5,0;FQ=-5.48;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=C421*;SNPEFF_CODON_CHANGE=tgC/tgA;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACAP1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_014716.3;VDB=0.0000;GM=NM_014716.3,XM_005256878.1;GL=ACAP1;FG=stop-gained;FD=unknown;AAC=CYS/stop;PP=421/741,347/667;CDP=1263,1041;CP=0.931;CG=-5.370;AA=C;DSP=81;KP=hsa04144:Endocytosis;GESP=C:13006;PAC=NP_055531.1,XP_005256935.1 GT:GQ:PL 0/1:19:17,0,20 0/1:3:0,0,1/1:4:25,3,0 0/1:12:28,0,10</p>

      <p>ZNF594 17 7951819 rs4792147 A G 25 lowCVG AC1=5;AF1=0.6327;DB;DP=8;DP4=2,1,3,1;FQ=27.7;MQ=60;PV4=1,0.054,1,1;SNPEFF_AMINO_ACID_CHANGE=Q656R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALOX15B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001141.2;VDB=0.0083;DN=111;DA=A/C/G;GM=NM_001039130.1,NM_001039131.1,NM_001141.2,XM_005256591.1;GL=ALOX15B;FG=missense;FD=missense,missense,missense,unknown;AAC=GLN/ARG;PP=627/648,582/603,656/677,644/665;CDP=1880,1745,1967,1931;GS=43;PH=0.001;CP=0.017;CG=-1.690;AA=G;HA=7.1;HE=47.0;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=116;KP=hsa00590:ArachidonicAcidMetabolism|hsa01100:MetabolicPathways;GESP=G:7866/A:5140;PAC=NP_001034219.1,NP_001034220.1,NP_001132.2,XP_005256648.1 GT:GQ:PL 0/0:6:0,9,70 0/1:3:0,0,0 1/1:8:52,9,0 1/1:3:15,3,0</p>

      <p>17 8216468 rs871841 T C 58.40 lowCVG AC1=5;AF1=0.6499;DB;DP=8;DP4=0,3,2,3;FQ=10.5;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L277P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025014.1;VDB=0.0003;DN=86;DA=T/C;GM=NM_025014.1,NM_173728.3,XM_005256536.1;GL=ARHGEF15;FG=missense;FD=missense,missense,unknown;AAC=LEU/PRO;PP=277/842,277/842,277/778;CDP=830;GS=98;PH=0;CP=0.995;CG=5.130;AA=C;HA=29.6;HE=47.3;HC=26.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=105;GESP=C:7328/T:5678;PAC=NP_079290.1,NP_776089.2,XP_005256593.1 GT:GQ:PL 0/0:6:0,9,48 0/1:3:0,0,0 1/1:11:83,12,0 1/1:4:18,3,0</p>

      <p>17 10223697 rs2074876 G T 61.70 lowCVG AC1=5;AF1=0.5935;DB;DP=8;DP4=1,1,5,1;FQ=9.15;MQ=60;PV4=0.46,0.34,1,0.18;SNPEFF_AMINO_ACID_CHANGE=D1076E;SNPEFF_CODON_CHANGE=gaC/gaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=25;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYH13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003802.2;VDB=0.0194;DN=96;DA=G/T;GM=NM_003802.2,XM_005256843.1;GL=MYH13;FG=missense;FD=missense,unknown;AAC=ASP/GLU;PP=1076/1939;CDP=3228;GS=45;PH=0.001;CP=1.000;CG=3.680;AA=G;HA=12.7;HE=8.7;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=30;KP=hsa04530:TightJunction|hsa05416:ViralMyocarditis;GESP=T:1345/G:10557;PAC=NP_003793.2,XP_005256900.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,3,23 1/1:3:27,3,0 1/1:13:79,15,0</p>

      <p>17 13980350 rs2072279 G A 65.90 highCVG AC1=4;AF1=0.4922;DB;DP=16;DP4=0,8,6,0;FQ=68.8;MQ=60;PV4=0.00033,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R159Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COX10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001303.3;VDB=0.0150;DN=96;DA=G/A;GM=NM_001303.3,XM_005256458.1,XM_005256459.1;GL=COX10;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=159/444,159/240,159/238;CDP=476;GS=43;PH=0.178;CP=0.605;CG=5.490;AA=G;HA=37.0;HE=44.2;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=24;KP=hsa00190:OxidativePhosphorylation|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa01100:MetabolicPathways;GESP=A:6378/G:6628;PAC=NP_001294.2,XP_005256515.1,XP_005256516.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:25,3,0 1/1:12:88,15,0 0/0:19:0,21,111</p>

      <p>17 19861458 rs2108978 C T 98.40 highCVG AC1=3;AF1=0.3904;DB;DP=18;DP4=7,4,1,6;FQ=101;MQ=60;PV4=0.066,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R249H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AKAP10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007202.3;VDB=0.0017;DN=96;DA=C/T;GM=NM_007202.3;GL=AKAP10;FG=missense;FD=missense;AAC=ARG/HIS;PP=249/663;CDP=746;GS=29;PH=0;CP=0.995;CG=2.360;AA=T;HA=44.6;HE=39.4;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=132;TFBS=V$GATA1_03;GESP=T:5787/C:7219;PPI=PDZK1:959|PRKAR1A:784;PAC=NP_009133.2 GT:GQ:PL 0/0:9:0,9,75 0/0:11:0,12,64 1/1:10:115,15,0 0/1:32:31,0,84</p>

      <p>17 21318867 rs73979893 G A 999 highCVG AC1=5;AF1=0.6076;DB;DP=37;DP4=5,10,16,5;FQ=193;MQ=60;PV4=0.017,0.45,1,0.069;SNPEFF_AMINO_ACID_CHANGE=M71I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNJ12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021012.4;VDB=0.0094;DN=130;DA=G/A;GM=NM_021012.4,XM_005256625.1;GL=KCNJ12;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=71/434;CDP=213;GS=10;PH=0;CP=1.000;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=269;CPG=21318651-21320041;TFBS=V$MEIS1AHOXA9_01,V$OCT1_06;PPI=DLG1:972|CASK:833|LIN7B:833|LIN7A:833|APBA1:833|LIN7C:833|DLG4:833;PAC=NP_066292.2,XP_005256682.1 GT:GQ:PL 0/1:54:49,0,91 0/1:35:62,0,34 1/1:8:120,9,0 0/1:59:54,0,108</p>

      <p>17 26694861 rs704 G A 31.80 highCVG AC1=4;AF1=0.4318;DB;DP=15;DP4=3,7,1,3;FQ=34;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T400M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000638.3;VDB=0.0089;DN=36;DA=G/A;GM=NM_000638.3;GL=VTN;FG=missense;FD=missense;AAC=THR/MET;PP=400/479;CDP=1199;GS=81;PH=0.943;CP=0.000;CG=-8.940;AA=G;HA=35.0;HE=46.9;HC=25.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=126;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=A:6627/G:6379;PAC=NP_000629.3 GT:GQ:PL 0/0:12:0,12,67 0/1:3:0,0,0 1/1:5:69,9,0 0/1:5:3,0,95</p>

      <p>17 27284443 rs12941884 A G 58.10 midCVG AC1=5;AF1=0.587;DB;DP=9;DP4=2,0,4,1;FQ=5.4;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=M806T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEZ6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178860.4;VDB=0.0169;DN=121;DA=A/G;GM=NM_001098635.1,NM_178860.4,XM_005257919.1;GL=SEZ6;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=806/994,806/995,681/870;CDP=2417,2417,2042;GS=81;PH=0;CP=0.999;CG=3.650;AA=G;HA=20.8;HE=14.6;HC=8.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=78;GESP=G:1806/A:10402;PAC=NP_001092105.1,NP_849191.3,XP_005257976.1 GT:GQ:PL 0/1:3:0,0,0 0/1:18:21,0,17 1/1:10:78,12,0 0/1:3:0,3,24</p>

      <p>17 33585770 . G C 33.60 midCVG AC1=2;AF1=0.3076;DP=14;DP4=6,4,4,0;FQ=35.1;MQ=60;PV4=0.25,1,1,0;SNPEFF_AMINO_ACID_CHANGE=V21L;SNPEFF_CODON_CHANGE=Gtc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLFN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144975.3;VDB=0.0001;GM=NM_144975.3,XM_005257934.1;GL=SLFN5;FG=missense;FD=unknown;AAC=VAL/LEU;PP=21/892,21/390;CDP=61;GS=32;PH=0.936;CP=0.006;CG=-3.630;AA=G;DSP=101;GESP=G:13006;PAC=NP_659412.3,XP_005257991.1 GT:GQ:PL 0/0:19:0,18,118 0/0:3:0,0,1/1:6:76,12,0 0/0:13:0,12,76</p>

      <p>17 37093510 rs113358400 T C 115 highCVG AC1=4;AF1=0.5505;DB;DP=17;DP4=3,1,5,7;FQ=13.6;MQ=60;PV4=0.57,0.38,1,0.38;SNPEFF_AMINO_ACID_CHANGE=N426S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXO47;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008777.2;VDB=0.0129;DN=132;DA=T/C;GM=NM_001008777.2;GL=FBXO47;FG=missense;FD=missense;AAC=ASN/SER;PP=426/453;CDP=1277;GS=46;PH=0.208;CP=0.984;CG=4.810;AA=T;DV=by-frequency,by-cluster;DSP=34;GESP=C:292/T:12714;PAC=NP_001008777.2 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,36 1/1:10:124,12,0 0/1:19:33,0,16</p>

      <p>17 38121993 rs3894194 G A 35.30 lowCVG AC1=6;AF1=0.7326;DB;DP=5;DP4=1,0,2,2;FQ=-12.4;MQ=60;PV4=1,0.21,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R18Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSDMA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178171.4;VDB=0.0024;DN=108;DA=G/A;GM=NM_178171.4;GL=GSDMA;FG=missense;FD=missense;AAC=ARG/GLN;PP=18/446;CDP=53;GS=43;PH=0.016;CP=1.000;CG=3.340;AA=G;HA=28.6;HE=47.7;HC=46.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=58;GESP=A:4966/G:7324;PAC=NP_835465.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,22 1/1:9:54,9,0 1/1:4:21,3,0</p>

      <p>17 38936659 rs17558560 C T 89.80 highCVG AC1=4;AF1=0.4888;DB;DP=16;DP4=3,3,0,9;FQ=51.7;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G193S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT27;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181537.3;VDB=0.0046;DN=123;DA=C/T;GM=NM_181537.3;GL=KRT27;FG=missense;FD=missense;AAC=GLY/SER;PP=193/460;CDP=577;GS=56;PH=0.152;CP=0.032;CG=3.410;AA=C;HA=11.4;HE=45.7;HC=39.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=50;TFBS=V$CDPCR3HD_01;GESP=T:4441/C:8565;PAC=NP_853515.2 GT:GQ:PL 0/0:10:0,12,63 0/0:5:0,6,34 1/1:21:112,24,0 0/1:3:26,3,0</p>

      <p>17 39983808 rs1046404 G C 71.20 lowCVG AC1=6;AF1=0.6722;DB;DP=8;DP4=1,1,1,5;FQ=6.96;MQ=60;PV4=0.46,0.14,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S213C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0127;DN=86;DA=G/C;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/CYS,none,none;PP=213/301,NA,NA;CDP=638,784,940;GS=112,NA,NA;PH=0.991;CP=0.812;CG=5.070;AA=G;HA=20.0;HE=24.2;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=71;GESP=C:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:18,0,20 0/1:4:0,3,26 1/1:9:55,9,0 1/1:7:43,6,0</p>

      <p>17 39983820 rs1046403 G A 58.40 lowCVG AC1=5;AF1=0.6474;DB;DP=7;DP4=1,1,1,4;FQ=7.67;MQ=60;PV4=1,0.17,1,0.19;SNPEFF_AMINO_ACID_CHANGE=A209V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0201;DN=86;DA=G/A;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ALA/VAL,none,none;PP=209/301,NA,NA;CDP=626,772,928;GS=64,NA,NA;PH=0.001;CP=0.060;CG=2.870;AA=G;HA=20.8;HE=23.8;HC=17.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;GESP=A:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:17,0,20 0/1:4:0,3,26 1/1:9:64,9,0 1/1:4:21,3,0</p>

      <p>17 41361960 rs8482 A G 76.10 midCVG AC1=4;AF1=0.5419;DB;DP=12;DP4=0,4,2,5;FQ=27.5;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H923R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NBR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005899.4;VDB=0.0215;DN=52;DA=A/G;GM=NM_005899.3,NM_031858.2,NM_031862.2;GL=NBR1;FG=missense;FD=missense;AAC=HIS/ARG;PP=923/967;CDP=2768;GS=29;PH=0.968;CP=1.000;CG=5.860;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;GESP=G:3148/A:7152;PPI=SQSTM1:994|MAP1LC3A:937|UBC:757|MAP1LC3C:756;PAC=NP_005890.2,NP_114064.1,NP_114068.1 GT:GQ:PL 0/1:9:6,0,31 0/1:3:0,0,0 1/1:15:112,18,0 0/0:5:0,6,35</p>

      <p>17 46847364 rs2032844 C A 24.80 lowCVG AC1=4;AF1=0.5405;DB;DP=5;DP4=1,1,1,2;FQ=9.57;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E712D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTLL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130918.1;VDB=0.0135;DN=94;DA=C/A;GM=NM_001130918.1,NM_173623.3;GL=TTLL6;FG=missense;FD=missense;AAC=GLU/ASP;PP=712/892,405/585;CDP=2136,1215;GS=45;PH=0.999;CP=0.002;CG=-3.840;AA=C;HA=24.4;HE=29.9;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=96;GESP=A:3281/C:9725;PAC=NP_001124390.1,NP_775894.2 GT:GQ:PL 0/0:3:0,3,24 0/1:3:20,3,0 1/1:4:47,6,0 0/0:3:0,3,24</p>

      <p>17 48598785 rs1380657 G A 174 highCVG AC1=6;AF1=0.7336;DB;DP=19;DP4=3,2,5,7;FQ=44.9;MQ=60;PV4=0.62,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=A424T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYCBPAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032133.4;VDB=0.0177;DN=88;DA=G/A;GM=NM_032133.4,XM_005257725.1,XM_005257726.1,XM_005257727.1;GL=MYCBPAP;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=424/985,439/1001,439/1000,424/986;CDP=1270,1315,1315,1270;GS=58;PH=0;CP=0.000;CG=-4.290;AA=A;HA=7.6;HE=27.6;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=A:10167/G:2839;PPI=MYCBP:846;PAC=NP_115509.4,XP_005257782.1,XP_005257783.1,XP_005257784.1 GT:GQ:PL 0/1:14:16,0,16 0/1:38:31,0,68 1/1:22:130,21,0 1/1:8:42,6,0</p>

      <p>17 48629458 rs8065903 A G 42.80 lowCVG AC1=5;AF1=0.6395;DB;DP=8;DP4=3,0,3,2;FQ=9.67;MQ=60;PV4=0.46,1,1,0.071;SNPEFF_AMINO_ACID_CHANGE=K625R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022827.3;VDB=0.0000;DN=116;DA=A/G;GM=NM_001258372.1,NM_001258373.1,NM_022827.3,XM_005257591.1;GL=SPATA20;FG=missense;FD=missense,missense,missense,unknown;AAC=LYS/ARG;PP=609/787,565/743,625/803,609/787;CDP=1826,1694,1874,1826;GS=26;PH=0;CP=0.996;CG=1.610;AA=G;HA=0.4;HE=23.2;HC=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;TFBS=V$GR_Q6,V$OLF1_01;GESP=G:10556/A:2450;PAC=NP_001245301.1,NP_001245302.1,NP_073738.2,XP_005257648.1 GT:GQ:PL 0/1:4:0,3,20 0/1:12:7,0,28 1/1:8:57,9,0 1/1:4:22,3,0</p>

      <p>17 56233034 rs7218964 C A 103 midCVG AC1=5;AF1=0.6659;DB;DP=10;DP4=0,2,6,2;FQ=-6.33;MQ=60;PV4=0.13,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=L174I;SNPEFF_CODON_CHANGE=Cta/Ata;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012374.1;VDB=0.0134;DN=116;DA=C/A;GM=NM_012374.1;GL=OR4D1;FG=missense;FD=missense;AAC=LEU/ILE;PP=174/311;CDP=520;GS=5;PH=0.329;CP=0.984;CG=5.630;AA=C;HA=48.5;HE=22.7;HC=12.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=37250;KP=hsa04740:OlfactoryTransduction;GESP=A:3410/C:9596;PAC=NP_036506.1 GT:GQ:PL 0/1:3:0,0,0 0/1:17:36,0,16 1/1:12:76,12,0 0/1:16:32,0,15</p>

      <p>17 56388238 rs2680704 C G 16.90 highCVG AC1=3;AF1=0.3661;DB;DP=16;DP4=6,4,4,0;FQ=18.7;MQ=60;PV4=0.25,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=A1140P;SNPEFF_CODON_CHANGE=Gca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BZRAP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004758.3;VDB=0.0162;DN=100;DA=C/G;GM=NM_001261835.1,NM_004758.3,NM_024418.2,XM_005257807.1,XM_005257808.1,XM_005257809.1,XM_005257810.1;GL=BZRAP1;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/PRO;PP=1140/1849,1140/1858,1080/1798,1140/1899,1133/1892,1080/1839,990/1749;CDP=3418,3418,3238,3418,3397,3238,2968;GS=27;PH=0;CP=0.000;CG=-1.050;AA=G;HA=22.5;HE=38.3;HC=21.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42;GESP=G:3461/C:9395;PAC=NP_001248764.1,NP_004749.2,NP_077729.1,XP_005257864.1,XP_005257865.1,XP_005257866.1,XP_005257867.1 GT:GQ:PL 0/0:5:0,3,17 0/0:28:0,27,161 1/1:7:58,12,0 0/0:3:0,0,0</p>

      <p>17 56388303 rs3744099 T A 41.60 midCVG AC1=2;AF1=0.3053;DB;DP=10;DP4=2,4,3,1;FQ=43.1;MQ=60;PV4=0.52,0.25,1,0.057;SNPEFF_AMINO_ACID_CHANGE=H1118L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BZRAP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004758.3;VDB=0.0175;DN=107;DA=T/A;GM=NM_001261835.1,NM_004758.3,NM_024418.2,XM_005257807.1,XM_005257808.1,XM_005257809.1,XM_005257810.1;GL=BZRAP1;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown;AAC=HIS/LEU;PP=1118/1849,1118/1858,1058/1798,1118/1899,1111/1892,1058/1839,968/1749;CDP=3353,3353,3173,3353,3332,3173,2903;GS=99;PH=0.05;CP=0.000;CG=-11.500;AA=T;HA=0.0;HE=6.0;HC=10.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=107;TFBS=V$MYOGNF1_01,V$PAX5_02;GESP=A:660/T:12322;PAC=NP_001248764.1,NP_004749.2,NP_077729.1,XP_005257864.1,XP_005257865.1,XP_005257866.1,XP_005257867.1 GT:GQ:PL 0/0:6:0,3,26 0/0:14:0,12,84 1/1:6:84,12,0 0/0:6:0,3,25</p>

      <p>17 59544932 . C A 13.30 midCVG AC1=3;AF1=0.3359;DP=11;DP4=2,6,0,3;FQ=14.9;MQ=60;PV4=1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=P155T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018488.2;VDB=0.0039;GM=NM_018488.2,XM_005257835.1,XM_005257836.1,XM_005257837.1,XM_005257838.1;GL=TBX4;FG=missense;FD=unknown;AAC=PRO/THR;PP=155/546,174/566,174/565,155/547,155/547;CDP=463,520,520,463,463;GS=38;PH=1;CP=1.000;CG=5.770;AA=C;DSP=62;GESP=C:13006;PAC=NP_060958.2,XP_005257892.1,XP_005257893.1,XP_005257894.1,XP_005257895.1 GT:GQ:PL 0/0:6:0,3,19 0/0:3:0,0,0 1/1:4:53,9,0 0/0:22:0,21,120</p>

      <p>17 66246416 rs3213690 A G 90.80 midCVG AC1=5;AF1=0.6141;DB;DP=10;DP4=1,1,3,5;FQ=-8.56;MQ=59;PV4=1,1,0.32,1;SNPEFF_AMINO_ACID_CHANGE=N30D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AMZ2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001033569.1;VDB=0.0143;DN=106;DA=A/G;GM=NM_001033569.1,NM_001033570.1,NM_001033571.1,NM_001033572.1,NM_001033574.1,NM_016627.4,XM_005257432.1,XM_005257433.1,XM_005257434.1,XM_005257435.1,XM_005257436.1,XM_005257437.1,XM_005257438.1;GL=AMZ2;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ASN/ASP;PP=30/361,30/361,30/361,30/361,30/303,30/361,94/425,30/361,30/361,30/361,30/361,30/361,94/266;CDP=88,88,88,88,88,88,280,88,88,88,88,88,280;GS=23;PH=0;CP=0.009;CG=1.320;AA=G;HA=2.0;HE=37.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=G:9868/A:3138;PAC=NP_001028741.1,NP_001028742.1,NP_001028743.1,NP_001028744.1,NP_001028746.1,NP_057711.3,XP_005257489.1,XP_005257490.1,XP_005257491.1,XP_005257492.1,XP_005257493.1,XP_005257494.1,XP_005257495.1 GT:GQ:PL 0/1:4:0,3,19 0/1:3:0,0,0 1/1:16:107,18,0 0/1:12:25,0,10</p>

      <p>17 72346868 rs2382644 C T 25.90 lowCVG AC1=5;AF1=0.6043;DB;DP=8;DP4=1,2,1,2;FQ=24.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R471W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153209.3;VDB=0.0128;DN=100;DA=C/T;GM=NM_153209.3,XM_005257029.1,XM_005257030.1;GL=KIF19;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=471/999,471/1000,429/958;CDP=1411,1411,1285;GS=101;PH=0.999;CP=0.000;CG=-11.500;DG;DV=by-frequency,by-cluster,by-1000G;DSP=23;TFBS=V$HOX13_01;GESP=T:1007/C:11997;PAC=NP_694941.2,XP_005257086.1,XP_005257087.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,63 1/1:3:17,3,0 1/1:5:50,6,0</p>

      <p>17 73886888 rs3760128 A G 19.20 highCVG AC1=3;AF1=0.3332;DB;DP=24;DP4=12,3,1,3;FQ=20.8;MQ=60;PV4=0.071,0.1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=L509P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM65;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173547.3;VDB=0.0187;DN=107;DA=A/G;GM=NM_001256124.1,NM_173547.3;GL=TRIM65;FG=missense;FD=missense;AAC=LEU/PRO;PP=487/496,509/518;CDP=1460,1526;GS=98;PH=0;CP=0.001;CG=3.690;AA=G;HA=7.2;HE=33.3;HC=28.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=541;GESP=G:6549/A:6433;PAC=NP_001243053.1,NP_775818.2 GT:GQ:PL 0/0:3:0,0,0 0/0:8:0,6,42 1/1:6:61,12,0 0/0:40:0,39,144</p>

      <p>17 74006201 . C T 36.70 highCVG AC1=2;AF1=0.2266;DP=18;DP4=9,4,0,4;FQ=37.8;MQ=60;PV4=0.029,1,1,1;SNPEFF_AMINO_ACID_CHANGE=D1029N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EVPL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001988.2;VDB=0.0002;GM=NM_001988.2,XM_005257143.1;GL=EVPL;FG=missense;FD=unknown;AAC=ASP/ASN;PP=1029/2034,1051/2056;CDP=3085,3151;GS=23;PH=0.964;CP=0.992;CG=5.340;AA=C;DSP=424;GESP=C:13004;PPI=PPL:937;PAC=NP_001979.2,XP_005257200.1 GT:GQ:PL 0/0:12:0,9,55 0/0:15:0,12,74 1/1:5:81,12,0 0/0:21:0,18,102</p>

      <p>17 74151720 rs201646108 C T 56.70 highCVG AC1=3;AF1=0.3694;DB;DP=26;DP4=6,12,3,3;FQ=58.7;MQ=59;PV4=0.63,1,0.041,0.25;SNPEFF_AMINO_ACID_CHANGE=A541T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RNF157;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052916.2;VDB=0.0188;DN=137;DA=C/T;GM=NM_052916.2,XM_005257006.1,XM_005257007.1,XM_005257008.1;GL=RNF157;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=541/680,541/658,541/654,474/613;CDP=1621,1621,1621,1420;GS=58;PH=1;CP=0.755;CG=5.430;AA=C;DV=by-cluster;DSP=78;GESP=T:5/C:13001;PAC=NP_443148.1,XP_005257063.1,XP_005257064.1,XP_005257065.1 GT:GQ:PL 0/0:9:0,9,60 0/1:29:29,0,78 1/1:7:75,12,0 0/0:24:0,24,130</p>

      <p>17 76433899 rs61742072 G A 107 midCVG AC1=5;AF1=0.5795;DB;DP=14;DP4=6,0,2,6;FQ=60.9;MQ=60;PV4=0.0097,0.39,1,1;SNPEFF_AMINO_ACID_CHANGE=H3953Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=74;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH17;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173628.3;VDB=0.0018;DN=129;DA=G/A;GM=NM_173628.3;GL=DNAH17;FG=missense;FD=missense;AAC=HIS/TYR;PP=3953/4463;CDP=11857;GS=83;PH=0;CP=0.997;CG=2.960;AA=G;DV=by-frequency,by-cluster;DSP=52;GESP=A:1641/G:11359;PAC=NP_775899.3 GT:GQ:PL 0/1:21:20,0,22 0/0:11:0,15,83 1/1:4:26,3,0 1/1:16:109,18,0</p>

      <p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

      <p>17 79634774 rs61742303 A T 27.20 midCVG AC1=2;AF1=0.2865;DB;DP=13;DP4=4,5,1,2;FQ=28.6;MQ=60;PV4=1,1,0.041,0.3;SNPEFF_AMINO_ACID_CHANGE=K50N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC137;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199287.2;VDB=0.0222;DN=129;DA=A/T;GM=NM_199287.2;GL=CCDC137;FG=missense;FD=unknown;AAC=LYS/ASN;PP=50/290;CDP=150;GS=94;PH=0.982;CP=0.997;CG=0.759;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=16;GESP=T:2092/A:9860;PAC=NP_954981.1 GT:GQ:PL 0/0:20:0,18,97 0/0:4:0,0,0 1/1:4:68,9,0 0/0:11:0,9,59</p>

      <p>17 80086389 rs7406163 C T 19.80 lowCVG AC1=4;AF1=0.4474;DB;DP=6;DP4=0,2,0,3;FQ=3.11;MQ=60;PV4=1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=D776N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC57;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198082.2;VDB=0.0077;DN=116;DA=C/T;GM=NM_198082.2,XM_005256343.1,XM_005256344.1;GL=CCDC57;FG=missense;FD=missense,unknown,unknown;AAC=ASP/ASN;PP=776/916,888/1033,888/1028;CDP=2326,2662,2662;GS=23;PH=0;CP=0.000;CG=-3.210;AA=C;HA=49.1;HE=44.6;HC=9.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=21;GESP=T:7026/C:5548;PAC=NP_932348.2,XP_005256400.1,XP_005256401.1 GT:GQ:PL 0/0:4:0,3,22 0/1:3:0,0,0 1/1:5:58,9,0 0/0:4:0,3,14</p>

      <p>18 3215158 rs1791085 C G 46.10 lowCVG AC1=3;AF1=0.4432;DB;DP=8;DP4=1,2,5,0;FQ=30.6;MQ=60;PV4=0.11,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V22L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYOM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003803.3;VDB=0.0017;DN=89;DA=C/G;GM=NM_003803.3,NM_019856.1;GL=MYOM1;FG=missense;FD=missense;AAC=VAL/LEU;PP=22/1686,22/1590;CDP=64;GS=32;PH=0;CP=0.984;CG=5.670;AA=G;HA=13.4;HE=8.7;HC=15.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;GESP=G:1230/C:11464;PAC=NP_003794.3,NP_062830.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,22 1/1:10:87,15,0 0/0:6:0,6,47</p>

      <p>18 5956238 rs3737353 C T 87.70 highCVG AC1=3;AF1=0.4121;DB;DP=16;DP4=7,0,2,5;FQ=85.9;MQ=60;PV4=0.021,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G618D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=L3MBTL4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173464.3;VDB=0.0033;DN=107;DA=C/T;GM=NM_173464.3,XM_005258165.1,XM_005258166.1,XM_005258167.1;GL=L3MBTL4;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ASP;PP=618/624,618/624,609/615,588/594;CDP=1853,1853,1826,1763;GS=94;PH=0.053;CP=0.010;CG=2.780;AA=C;HA=33.1;HE=38.3;HC=46.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=149;GESP=T:4306/C:7788;PAC=NP_775735.2,XP_005258222.1,XP_005258223.1,XP_005258224.1 GT:GQ:PL 0/0:11:0,12,74 0/1:18:19,0,17 1/1:13:116,18,0 0/0:6:0,6,42</p>

      <p>18 11610052 rs74410464 T C 70.60 highCVG AC1=5;AF1=0.6397;DB;DP=15;DP4=3,5,2,4;FQ=70.8;MQ=55;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V153A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=T/C;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=VAL/ALA;PP=153/339;CDP=458;GS=64;CP=0.000;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79887;PAC=XP_934053.1 GT:GQ:PL 0/1:10:5,0,68 0/1:3:0,0,0 1/1:11:93,12,0 0/1:18:13,0,41</p>

      <p>18 11610073 rs75301503 G A 72.40 midCVG AC1=4;AF1=0.5075;DB;DP=14;DP4=2,5,3,3;FQ=75.1;MQ=53;PV4=0.59,1,0.44,0.48;SNPEFF_AMINO_ACID_CHANGE=C160Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0063;DN=131;DA=G/A;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=CYS/TYR;PP=160/339;CDP=479;GS=194;CP=0.000;AA=G;DG;DV=by-cluster,by-1000G;DSP=79866;PAC=XP_934053.1 GT:GQ:PL 0/0:10:0,12,79 0/1:3:0,0,1/1:12:100,15,0 0/1:19:16,0,48</p>

      <p>18 11610163 rs62084100 G C 19.60 highCVG AC1=4;AF1=0.4585;DB;DP=15;DP4=4,4,3,3;FQ=22.1;MQ=47;PV4=1,0.45,0.072,0.13;SNPEFF_AMINO_ACID_CHANGE=G190A;SNPEFF_CODON_CHANGE=gGc/gCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC35G4P;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282300.1;VDB=0.0171;DN=129;DA=G/C/T;GM=XM_928960.4;GL=SLC35G4;FG=missense;FD=missense;AAC=GLY/ALA;PP=190/339;CDP=569;GS=60;CP=0.105;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79776;PAC=XP_934053.1 GT:GQ:PL 0/0:6:0,6,100 0/1:3:0,0,0 1/1:8:59,12,0 0/0:3:0,1,17</p>

      <p>18 14779969 rs9675365 C G 147 highCVG AC1=4;AF1=0.5542;DB;DP=24;DP4=0,8,5,11;FQ=15.9;MQ=55;PV4=0.13,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F477L;SNPEFF_CODON_CHANGE=ttC/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145029.1;VDB=0.0092;DN=119;DA=C/G;GM=NM_001145029.1;GL=ANKRD30B;FG=missense;FD=missense;AAC=PHE/LEU;PP=477/1393;CDP=1431;GS=22;PH=0.286;CP=0.000;CG=-2.400;AA=C;HA=41.2;HE=48.8;HC=36.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=11;GESP=C:4566;PAC=NP_001138501.1 GT:GQ:PL 0/0:4:0,6,34 0/1:5:92,0,1 1/1:5:69,6,0 0/1:22:29,0,20</p>

      <p>18 28986333 rs4799570 A C 83.20 midCVG AC1=5;AF1=0.5962;DB;DP=10;DP4=3,1,3,3;FQ=24.2;MQ=60;PV4=0.57,1,1,0.43;SNPEFF_AMINO_ACID_CHANGE=I644L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DSG4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001134453.1;VDB=0.0057;DN=111;DA=A/C;GM=NM_001134453.1,NM_177986.3;GL=DSG4;FG=missense;FD=missense;AAC=ILE/LEU;PP=644/1060,644/1041;CDP=1930;GS=5;PH=0;CP=0.762;CG=4.770;AA=C;HA=0.0;HE=11.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=4;GESP=C:12022/A:984;PAC=NP_001127925.1,NP_817123.1 GT:GQ:PL 0/1:3:0,3,13 0/1:3:0,0,0 1/1:10:93,12,0 0/1:35:31,0,48</p>

      <p>18 29339975 . C T 29.30 lowCVG AC1=3;AF1=0.4714;DP=7;DP4=2,1,0,4;FQ=-8.34;MQ=24;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G227D;SNPEFF_CODON_CHANGE=gGt/gAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A52;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001034172.2;VDB=0.0001;GM=NM_001034172.2;GL=SLC25A52;FG=missense;FD=unknown;AAC=GLY/ASP;PP=227/308;CDP=680;GS=94;PH=0.006;CP=0.077;CG=-1.260;AA=C;DSP=35519;GESP=C:13006;PAC=NP_001029344.3 GT:GQ:PL 0/0:4:0,3,22 0/1:3:0,0,0 1/1:8:69,12,0 0/0:5:0,6,7</p>

      <p>18 29797921 rs616114 C T 34.30 midCVG AC1=3;AF1=0.4037;DB;DP=9;DP4=1,3,0,5;FQ=36.2;MQ=60;PV4=0.44,0.28,1,1;SNPEFF_AMINO_ACID_CHANGE=P695L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MEP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005925.2;VDB=0.0113;DN=83;DA=C/T;GM=NM_005925.2,XM_005258275.1;GL=MEP1B;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=695/702,695/701;CDP=2084;GS=98;PH=0;CP=0.004;CG=0.414;AA=C;HA=45.9;HE=37.9;HC=26.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=8;GESP=T:4754/C:7364;PAC=NP_005916.2,XP_005258332.1 GT:GQ:PL 0/0:7:0,6,40 0/1:3:0,0,0 1/1:10:76,15,0 0/0:7:0,6,47</p>

      <p>18 43497710 rs3744998 A G 87.20 highCVG AC1=5;AF1=0.6466;DB;DP=18;DP4=6,1,2,6;FQ=33.7;MQ=60;PV4=0.041,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=V1058A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPG5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020964.2;VDB=0.0029;DN=107;DA=A/G;GM=NM_020964.2,XM_005258323.1,XM_005258324.1;GL=EPG5;FG=missense;FD=missense,unknown,unknown;AAC=VAL/ALA;PP=1058/2580,1058/2579,703/2225;CDP=3173,3173,2108;GS=64;PH=0.884;CP=1.000;CG=5.400;AA=A;HA=26.5;HE=41.1;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=67;GESP=G:5759/A:6247;PAC=NP_066015.2,XP_005258380.1,XP_005258381.1 GT:GQ:PL 0/1:36:35,0,38 0/1:3:0,0,0 1/1:9:44,9,0 0/1:33:49,0,32</p>

      <p>18 65179829 rs2279269 G A 25.80 lowCVG AC1=4;AF1=0.4987;DB;DP=8;DP4=2,2,1,2;FQ=27;MQ=60;PV4=1,1,1,0.04;SNPEFF_AMINO_ACID_CHANGE=P683S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DSEL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032160.2;VDB=0.0085;DN=100;DA=G/A;GM=NM_032160.2;GL=DSEL;FG=missense;FD=missense;AAC=PRO/SER;PP=683/1223;CDP=2047;GS=74;PH=0;CP=0.987;CG=2.920;AA=A;HA=29.6;HE=47.7;HC=50.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=2901;GESP=A:5871/G:7135;PAC=NP_115536.1 GT:GQ:PL 0/1:19:18,0,23 0/1:3:0,0,0 1/1:4:48,6,0 0/0:5:0,6,46</p>

      <p>18 72103782 rs2278154 T C 999 highCVG AC1=5;AF1=0.603;DB;DP=20;DP4=5,1,8,6;FQ=35.4;MQ=60;PV4=0.35,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q405R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM69C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001044369.2;VDB=0.0039;DN=100;DA=T/C;GM=NM_001044369.2,XM_005266647.1;GL=FAM69C;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=405/420,264/279;CDP=1214,791;GS=43;PH=0;CP=0.021;CG=4.080;AA=C;HA=27.7;HE=36.4;HC=29.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=173;GESP=C:4387/T:7701;PAC=NP_001037834.2,XP_005266704.1 GT:GQ:PL 0/1:3:0,0,0 0/1:53:53,0,54 1/1:28:173,30,0 0/1:4:0,3,19</p>

      <p>19 1010629 . G A 33.70 lowCVG AC1=6;AF1=0.8319;DP=6;DP4=1,0,4,0;FQ=-24.4;MQ=60;PV4=1,1,1,0.043;SNPEFF_AMINO_ACID_CHANGE=A528V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM259;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001033026.1;VDB=0.0000;GM=NM_001033026.1,NM_033420.3,XM_005259675.1,XM_005259676.1,XM_005259677.1,XM_005259678.1,XM_005259679.1;GL=TMEM259;FG=missense,3-prime-UTR,missense,missense,missense,missense,missense;FD=unknown;AAC=ALA/VAL,none,ALA/VAL,ALA/VAL,ALA/VAL,ALA/VAL,ALA/VAL;PP=528/621,NA,569/662,563/656,552/645,534/627,348/441;CDP=1583,NA,1706,1688,1655,1601,1043;GS=64,NA,64,64,64,64,64;PH=0.99;CP=0.057;CG=3.370;DSP=266;CPG=1007552-1012222;GESP=G:8630;PAC=NP_001028198.1,NA,XP_005259732.1,XP_005259733.1,XP_005259734.1,XP_005259735.1,XP_005259736.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:26,3,0 0/1:6:41,0,4</p>

      <p>19 1068738 rs1801284 G A 31.20 lowCVG AC1=6;AF1=0.749;DB;DP=5;DP4=0,1,2,2;FQ=-19.7;MQ=60;PV4=1,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R155H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HMHA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001258328.2;VDB=0.0157;DN=89;DA=G/A;GM=NM_001258328.1,NM_012292.3;GL=HMHA1;FG=missense;FD=missense;AAC=ARG/HIS;PP=155/1153,139/1137;CDP=464,416;GS=29;PH=0.008;CP=0.000;CG=-8.440;AA=A;HA=45.1;HE=34.4;HC=41.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;CPG=1068439-1068764;GESP=A:5253/G:7735;PAC=NP_001245257.1,NP_036424.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:60,9,0 0/1:10:7,0,11</p>

      <p>19 4512669 rs55901236 T C 66.40 midCVG AC1=6;AF1=0.6769;DB;DP=10;DP4=0,2,6,1;FQ=-15;MQ=60;PV4=0.083,1,1,0.31;SNPEFF_AMINO_ACID_CHANGE=T421A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080400.1;VDB=0.0122;DN=129;DA=T/C;GM=NM_001080400.1,XM_005259640.1,XM_005259641.1;GL=PLIN4;FG=missense;FD=missense,unknown,unknown;AAC=THR/ALA;PP=421/1358,436/1373,435/1372;CDP=1261,1306,1303;GS=58;PH=0.009;CP=0.001;CG=0.087;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1045;TFBS=V$GR_Q6;GESP=C:5321/T:7027;PAC=NP_001073869.1,XP_005259697.1,XP_005259698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:17:76,0,15</p>

      <p>19 4538904 . G A 14.10 lowCVG AC1=5;AF1=0.6551;DP=7;DP4=0,2,3,0;FQ=-10.9;MQ=60;PV4=0.1,1,1,2.3e-05;SNPEFF_AMINO_ACID_CHANGE=A31V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052972.2;VDB=0.0085;GM=NM_052972.2;GL=LRG1;FG=missense;FD=unknown;AAC=ALA/VAL;PP=31/348;CDP=92;GS=64;PH=0.006;CP=0.003;CG=0.951;AA=G;DSP=60;GESP=G:13006;PAC=NP_443204.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,25 1/1:5:40,6,0 1/1:3:14,3,0</p>

      <p>19 6374890 rs7540 G A 55.70 midCVG AC1=4;AF1=0.4563;DB;DP=10;DP4=5,0,3,1;FQ=47.1;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R191Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALKBH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032306.3;VDB=0.0086;DN=52;DA=G/A;GM=NM_032306.3,XM_005259658.1;GL=ALKBH7;FG=missense;FD=missense,unknown;AAC=ARG/GLN;PP=191/222,165/196;CDP=572,494;GS=43;PH=0.275;CP=0.021;CG=3.030;AA=G;HA=0.0;HE=10.3;HC=2.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=69;TFBS=V$OLF1_01;GESP=A:862/G:12144;PAC=NP_115682.1,XP_005259715.1 GT:GQ:PL 0/1:18:43,0,14 0/1:3:0,0,0 1/1:4:54,6,0 0/0:11:0,12,72</p>

      <p>19 8997482 . C T 52.20 highCVG AC1=5;AF1=0.5614;DP=30;DP4=3,9,6,12;FQ=55.8;MQ=39;PV4=0.7,1,3.5e-17,1;SNPEFF_AMINO_ACID_CHANGE=R13647Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0008;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=ARG/GLN;PP=13647/14508,5561/6405,5561/6025,442/1303;CDP=40940,16682,16682,1325;GS=43;PH=0.984;CP=0.000;CG=-5.720;AA=C;DSP=55;GESP=C:12276;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/1:3:0,3,59 0/1:17:15,0,19 1/1:13:69,15,0 0/1:17:13,0,77</p>

      <p>19 8997507 . G A 30.10 highCVG AC1=3;AF1=0.3713;DP=29;DP4=8,10,2,8;FQ=32.1;MQ=43;PV4=0.25,0.045,0.00017,0.46;SNPEFF_AMINO_ACID_CHANGE=L13639F;SNPEFF_CODON_CHANGE=Ctc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0023;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=LEU/PHE;PP=13639/14508,5553/6405,5553/6025,434/1303;CDP=40915,16657,16657,1300;GS=22;PH=0.984;CP=0.046;CG=-2.050;AA=G;DSP=30;GESP=G:12308;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/0:21:0,21,114 0/1:16:16,0,29 1/1:7:61,12,0 0/0:8:0,8,111</p>

      <p>19 9024994 rs67631215 C T 90.10 highCVG AC1=5;AF1=0.6778;DB;DP=16;DP4=2,2,11,0;FQ=-17;MQ=54;PV4=0.057,0.07,1,0.16;SNPEFF_AMINO_ACID_CHANGE=E12290K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0020;DN=130;DA=C/T;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=GLU/LYS;PP=12290/14508,4204/6405,4204/6025;CDP=36868,12610,12610;GS=56;PH=0.009;CP=0.004;CG=1.510;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=5;GESP=T:4021/C:7885;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:3:0,0,0 0/1:9:41,0,8 1/1:6:33,6,0 0/1:10:55,0,9</p>

      <p>19 9056982 rs11670318 G A 109 midCVG AC1=5;AF1=0.6629;DB;DP=11;DP4=1,2,6,2;FQ=5.06;MQ=60;PV4=0.49,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T10155I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0045;DN=120;DA=G/A;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=THR/ILE;PP=10155/14508,2069/6405,2069/6025;CDP=30464,6206,6206;GS=89;PH=0.997;CP=0.000;CG=0.467;AA=G;HA=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=810;GESP=A:3367/G:8779;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:30:47,0,29 0/1:3:0,0,0 1/1:9:61,9,0 0/1:11:41,0,10</p>

      <p>19 9069792 rs1862458 G A 52.70 highCVG AC1=4;AF1=0.4499;DB;DP=16;DP4=1,9,4,2;FQ=55.2;MQ=60;PV4=0.036,0.29,1,1;SNPEFF_AMINO_ACID_CHANGE=S5885F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0050;DN=92;DA=G/A;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=SER/PHE;PP=5885/14508;CDP=17654;GS=155;PH=0.995;CP=0.002;CG=1.870;AA=G;HA=12.4;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8074;GESP=A:1913/G:10339;PAC=NP_078966.2 GT:GQ:PL 0/1:13:11,0,32 0/0:19:0,21,95 1/1:4:44,6,0 0/1:19:43,0,15</p>

      <p>19 9069892 rs1862460 A C 33.40 highCVG AC1=3;AF1=0.3838;DB;DP=20;DP4=2,9,3,1;FQ=35.4;MQ=60;PV4=0.077,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=F5852V;SNPEFF_CODON_CHANGE=Ttt/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0084;DN=92;DA=A/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=PHE/VAL;PP=5852/14508;CDP=17554;GS=50;PH=0;CP=0.000;CG=-0.715;AA=C;HA=30.1;HE=16.1;HC=36.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7974;GESP=C:2522/A:9570;PAC=NP_078966.2 GT:GQ:PL 0/1:13:13,0,15 0/0:27:0,27,130 1/1:5:65,9,0 0/0:4:0,3,28</p>

      <p>19 9070225 rs1559172 G C 130 midCVG AC1=6;AF1=0.682;DB;DP=12;DP4=0,3,4,5;FQ=5.24;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H5741D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0092;DN=88;DA=G/C;GM=NM_024690.2;GL=MUC16;FG=missense;FD=missense;AAC=HIS/ASP;PP=5741/14508;CDP=17221;GS=81;PH=0;CP=0.000;CG=-0.660;AA=C;HA=28.3;HE=15.0;HC=11.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7641;GESP=C:10349/G:2229;PAC=NP_078966.2 GT:GQ:PL 0/1:22:52,0,22 0/1:5:0,3,22 1/1:12:76,12,0 1/1:7:47,6,0</p>

      <p>19 9764230 . G A 25.50 lowCVG AC1=2;AF1=0.3027;DP=7;DP4=2,2,0,3;FQ=26.8;MQ=60;PV4=0.43,1,1,0;SNPEFF_AMINO_ACID_CHANGE=H226Y;SNPEFF_CODON_CHANGE=Cac/Tac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF562;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130031.1;VDB=0.0003;GM=NM_001130031.1,NM_001130032.1,NM_017656.3,XM_005259941.1,XM_005259942.1,XM_005259943.1,XM_005259944.1,XM_005259945.1,XM_005259946.1,XM_005259947.1,XM_005259948.1,XM_005259949.1,XM_005259950.1,XM_005259951.1,XM_005259952.1,XM_005259953.1;GL=ZNF562;FG=missense;FD=unknown;AAC=HIS/TYR;PP=226/427,226/427,154/355,226/427,225/426,218/419,189/390,157/358,157/358,157/358,157/358,148/349,111/312,110/311,83/284,83/284;CDP=676,676,460,676,673,652,565,469,469,469,469,442,331,328,247,247;GS=83;PH=0.965;CP=0.000;CG=-0.749;AA=G;DSP=325;GESP=G:13006;PAC=NP_001123503.1,NP_001123504.1,NP_060126.2,XP_005259998.1,XP_005259999.1,XP_005260000.1,XP_005260001.1,XP_005260002.1,XP_005260003.1,XP_005260004.1,XP_005260005.1,XP_005260006.1,XP_005260007.1,XP_005260008.1,XP_005260009.1,XP_005260010.1 GT:GQ:PL 0/0:8:0,6,34 0/0:6:0,3,28 1/1:4:66,9,0 0/0:6:0,3,19</p>

      <p>19 10112346 rs2287803 T C 55.50 lowCVG AC1=5;AF1=0.647;DB;DP=8;DP4=3,0,0,5;FQ=23;MQ=60;PV4=0.018,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R322G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL5A3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015719.3;VDB=0.0071;DN=100;DA=T/C;GM=NM_015719.3;GL=COL5A3;FG=missense-near-splice;FD=missense;AAC=ARG/GLY;PP=322/1746;CDP=964;GS=125;PH=0;CP=0.002;CG=2.170;AA=C;HA=3.2;HE=2.3;HC=13.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:12695/T:311;PPI=COL5A1:999;PAC=NP_056534.2 GT:GQ:PL 0/1:17:20,0,16 0/1:17:12,0,43 1/1:9:64,9,0 0/1:3:0,0,0</p>

      <p>19 10224526 rs3745600 C T 89.90 lowCVG AC1=6;AF1=0.7365;DB;DP=8;DP4=0,2,2,4;FQ=11;MQ=60;PV4=1,0.13,1,0.14;SNPEFF_AMINO_ACID_CHANGE=P520L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPAN-P2RY11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001198690.1;VDB=0.0011;DN=107;DA=C/T;GM=NM_001040664.2,NM_001198690.1,NM_002566.4;GL=P2RY11/PPAN-P2RY11;FG=synonymous,missense,synonymous;FD=synonymous-codon,missense,synonymous-codon;AAC=ALA,PRO/LEU,ALA;PP=499/795,520/521,79/375;CDP=1497,1559,237;GS=NA,98,NA;PH=0;CP=0.553;CG=-4.370;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=218;KP=hsa04080:NeuroactiveLigand-receptorInteraction;CPG=10224443-10225138;GESP=T:4862/C:8140;PAC=NP_001035754.1,NP_001185619.1,NP_002557.2 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,48 1/1:9:71,9,0 1/1:9:62,9,0</p>

      <p>19 12154799 rs67102109 G C 83.90 lowCVG AC1=6;AF1=0.7469;DB;DP=7;DP4=0,1,2,4;FQ=-14.5;MQ=60;PV4=1,0.36,1,0.21;SNPEFF_AMINO_ACID_CHANGE=H473D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF878;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080404.2;VDB=0.0169;DN=130;DA=G/C;GM=NM_001080404.2;GL=ZNF878;FG=missense;FD=missense;AAC=HIS/ASP;PP=473/532;CDP=1417;GS=81;PH=0.997;CP=0.016;CG=1.300;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1226;GESP=C:908/G:12058;PAC=NP_001073873.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:12:87,12,0 0/1:17:34,0,17</p>

      <p>19 12502457 rs4804669 A G 15.20 midCVG AC1=3;AF1=0.3874;DB;DP=12;DP4=4,3,2,3;FQ=17.3;MQ=50;PV4=1,0.014,0.37,1;SNPEFF_AMINO_ACID_CHANGE=L252P;SNPEFF_CODON_CHANGE=cTg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF799;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080821.2;VDB=0.0207;DN=111;DA=A/G;GM=NM_001080821.2,XM_005260140.1;GL=ZNF799;FG=missense;FD=missense,unknown;AAC=LEU/PRO;PP=252/644,254/646;CDP=755,761;GS=98;PH=0;CP=0.001;CG=1.310;AA=G;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=564;GESP=G:10882/A:2116;PAC=NP_001074290.1,XP_005260197.1 GT:GQ:PL 0/0:7:0,6,51 0/1:11:11,0,15 1/1:7:50,12,0 0/0:12:0,12,85</p>

      <p>19 12542653 rs4239550 T G 25.90 midCVG AC1=3;AF1=0.3275;DB;DP=11;DP4=4,1,1,2;FQ=27.6;MQ=50;PV4=0.46,0.23,0.01,1;SNPEFF_AMINO_ACID_CHANGE=K111N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF443;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005815.4;VDB=0.0205;DN=111;DA=T/A/G;GM=NM_005815.4,XM_005259705.1;GL=ZNF443;FG=missense;FD=missense,unknown;AAC=LYS/ASN;PP=111/672,113/674;CDP=333,339;GS=94;PH=0.009;CP=0.000;CG=-0.007;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=142;GESP=G:4380/T:8626;PAC=NP_005806.2,XP_005259762.1 GT:GQ:PL 0/0:11:0,9,62 0/0:8:0,6,48 1/1:4:66,9,0 0/0:3:0,0,0</p>

      <p>19 12542654 rs28599549 T A 22.40 midCVG AC1=2;AF1=0.2468;DB;DP=12;DP4=4,4,1,2;FQ=23.6;MQ=52;PV4=1,0.082,0.0011,1;SNPEFF_AMINO_ACID_CHANGE=K111I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF443;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005815.4;VDB=0.0205;DN=125;DA=T/A;GM=NM_005815.4,XM_005259705.1;GL=ZNF443;FG=missense;FD=missense,unknown;AAC=LYS/ILE;PP=111/672,113/674;CDP=332,338;GS=102;PH=0;CP=0.000;CG=-1.340;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=141;GESP=A:4376/T:8630;PAC=NP_005806.2,XP_005259762.1 GT:GQ:PL 0/0:18:0,15,110 0/0:9:0,6,48 1/1:3:64,9,0 0/0:7:0,3,28</p>

      <p>19 12772165 rs1054487 G A 27.30 lowCVG AC1=3;AF1=0.3966;DB;DP=8;DP4=1,3,3,1;FQ=29.4;MQ=60;PV4=0.49,0.0045,1,1;SNPEFF_AMINO_ACID_CHANGE=T312I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAN2B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000528.3;VDB=0.0063;DN=86;DA=G/A/T;GM=NM_000528.3,NM_001173498.1,XM_005259913.1;GL=MAN2B1;FG=missense;FD=missense,missense,unknown;AAC=THR/ILE;PP=312/1012,312/1011,312/1013;CDP=935;GS=89;PH=0.003;CP=0.015;CG=-2.190;AA=A;HA=40.2;HE=38.7;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=A:5899/G:7107;PAC=NP_000519.2,NP_001166969.1,XP_005259970.1 GT:GQ:PL 0/0:5:0,3,25 0/1:3:0,0,0 1/1:7:68,12,0 0/0:9:0,9,72</p>

      <p>19 14185226 rs7258963 T C 15.40 lowCVG AC1=4;AF1=0.4664;DB;DP=7;DP4=1,3,2,1;FQ=18.1;MQ=60;PV4=0.49,0.27,1,0.35;SNPEFF_AMINO_ACID_CHANGE=V198A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LOC113230;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291291.1;VDB=0.0059;DN=116;DA=T/C;GM=NR_024282.1;GL=LOC113230;FG=non-coding-exon;FD=nc-transcript-variant;CDP=894;CP=0.984;CG=3.330;AA=C;HA=21.4;HE=37.5;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;CPG=14185108-14185601 GT:GQ:PL 0/1:3:0,0,0 0/0:8:0,9,62 1/1:4:42,6,0 0/1:15:14,0,19</p>

      <p>19 14875388 rs35612307 G A 41.60 highCVG AC1=3;AF1=0.4114;DB;DP=35;DP4=3,23,3,1;FQ=43.9;MQ=60;PV4=0.018,0.35,1,1;SNPEFF_AMINO_ACID_CHANGE=A314V;SNPEFF_CODON_CHANGE=gCc/gTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EMR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013447.3;VDB=0.0063;DN=126;DA=G/A;GM=NM_001271052.1,NM_013447.3,XM_005259884.1,XM_005259885.1,XM_005259886.1,XM_005259887.1;GL=EMR2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/VAL;PP=314/766,314/824,383/893,383/835,53/552,383/462;CDP=941,941,1148,1148,158,1148;GS=64;PH=0.294;CP=0.000;CG=-3.320;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=A:3066/G:9936;PAC=NP_001257981.1,NP_038475.2,XP_005259941.1,XP_005259942.1,XP_005259943.1,XP_005259944.1 GT:GQ:PL 0/0:68:0,69,135 0/1:4:24,3,0 1/1:5:65,9,0 0/0:9:0,9,34</p>

      <p>19 14910573 rs10415562 C T 54.20 lowCVG AC1=6;AF1=0.7302;DB;DP=6;DP4=0,1,2,2;FQ=-8.83;MQ=60;PV4=1,0.37,1,0.34;SNPEFF_AMINO_ACID_CHANGE=V126I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR7C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198944.1;VDB=0.0040;DN=119;DA=C/T;GM=NM_198944.1,XM_005259853.1;GL=OR7C1;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=126/321;CDP=376;GS=29;PH=0.007;CP=0.999;CG=2.590;AA=T;HA=14.3;HE=25.8;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=389;KP=hsa04740:OlfactoryTransduction;GESP=T:10304/C:2702;PAC=NP_945182.1,XP_005259910.1 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:4:29,3,0 1/1:9:65,9,0</p>

      <p>19 15198631 rs8105737 A C 43.30 lowCVG AC1=5;AF1=0.6343;DB;DP=7;DP4=3,0,3,1;FQ=15.8;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=Y252S;SNPEFF_CODON_CHANGE=tAt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004713.1;VDB=0.0061;DN=116;DA=A/C;GM=NM_001004713.1;GL=OR1I1;FG=missense;FD=missense;AAC=TYR/SER;PP=252/356;CDP=755;GS=144;PH=0.999;CP=0.743;CG=3.590;AA=A;HA=36.4;HE=33.0;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19702;KP=hsa04740:OlfactoryTransduction;TFBS=V$GR_Q6,V$YY1_02;GESP=C:7526/A:5480;PAC=NP_001004713.1 GT:GQ:PL 0/1:14:17,0,13 0/1:3:0,0,0 1/1:6:49,6,0 0/1:22:17,0,38</p>

      <p>19 15587185 rs733731 C T 22.30 lowCVG AC1=5;AF1=0.6333;DB;DP=6;DP4=1,1,4,0;FQ=-3.17;MQ=60;PV4=0.33,0.097,1,1;SNPEFF_AMINO_ACID_CHANGE=R99Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PGLYRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052890.3;VDB=0.0069;DN=86;DA=C/T;GM=NM_052890.3;GL=PGLYRP2;FG=missense;FD=missense;AAC=ARG/GLN;PP=99/577;CDP=296;GS=43;PH=0.03;CP=0.001;CG=-5.850;AA=C;HA=34.3;HE=39.5;HC=38.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=235;GESP=T:4832/C:8174;PAC=NP_443122.3 GT:GQ:PL 0/1:14:15,0,14 0/1:4:0,3,24 1/1:4:19,3,0 1/1:6:31,6,0</p>

      <p>19 15740220 rs4239614 C T 30.90 lowCVG AC1=6;AF1=0.7447;DB;DP=5;DP4=0,1,0,4;FQ=-12.7;MQ=60;PV4=1,0.091,1,0.25;VDB=0.0093;DN=111;DA=C/T;GM=NM_007253.3;GL=CYP4F8;FG=3-prime-UTR;FD=utr-variant-3-prime;CP=0.000;CG=-0.816;AA=C;HA=24.3;HE=26.8;HC=28.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=218;GESP=T:5207/C:1801 GT:GQ:PL 0/1:4:0,3,22 0/1:3:0,0,0 1/1:7:35,6,0 1/1:7:36,6,0</p>

      <p>19 15852363 rs11670007 G A 21 lowCVG AC1=3;AF1=0.3936;DB;DP=6;DP4=1,2,0,3;FQ=22.5;MQ=58;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R54H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10H3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013938.1;VDB=0.0067;DN=120;DA=G/A;GM=NM_013938.1;GL=OR10H3;FG=missense;FD=missense;AAC=ARG/HIS;PP=54/317;CDP=161;GS=29;PH=0.001;CP=0.225;CG=-4.700;AA=G;HA=43.3;HE=31.4;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18868;KP=hsa04740:OlfactoryTransduction;GESP=A:4268/G:8738;PAC=NP_039226.1 GT:GQ:PL 0/0:5:0,3,26 0/0:3:0,0,0 1/1:5:60,9,0 0/0:7:0,6,43</p>

      <p>19 17017878 rs56408444 G A 20.80 lowCVG AC1=4;AF1=0.4992;DB;DP=6;DP4=2,1,1,2;FQ=23.3;MQ=60;PV4=1,0.004,1,0.41;SNPEFF_AMINO_ACID_CHANGE=A1351V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPAMD8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015692.2;VDB=0.0285;DN=129;DA=G/A;GM=NM_015692.2;GL=CPAMD8;FG=missense;FD=missense;AAC=ALA/VAL;PP=1351/1933;CDP=4052;GS=64;PH=0.068;CP=0.000;CG=-4.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=50;GESP=A:1872/G:10942;PAC=NP_056507.2GT:GQ:PL 0/0:3:0,3,27 0/1:3:22,3,0 1/1:4:42,6,0 0/0:5:0,6,47</p>

      <p>19 17170885 rs1130222 C T 99.90 highCVG AC1=5;AF1=0.5805;DB;DP=17;DP4=4,4,3,5;FQ=74.7;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G83R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033417.1;VDB=0.0107;DN=86;DA=C/A/G/T;GM=NM_001011699.1,NM_033417.1,XM_005260154.1;GL=HAUS8;FG=missense,missense,5-prime-UTR;FD=missense/stop-gained,missense/stop-gained,unknown;AAC=GLY/ARG,GLY/ARG,none;PP=82/410,83/411,NA;CDP=244,247,NA;GS=125,125,NA;PH=0.062;CP=0.001;CG=1.070;AA=C;HA=45.0;HE=33.3;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;GESP=T:5621/C:7385;PPI=HAUS1:999|HAUS6:994|HAUS2:987|HAUS4:987;PAC=NP_001011699.1,NP_219485.1,NA GT:GQ:PL 0/1:46:42,0,63 0/0:9:0,12,55 1/1:4:28,3,0 1/1:10:77,12,0</p>

      <p>19 17743622 rs75595663 C A 43.40 highCVG AC1=2;AF1=0.2548;DB;DP=15;DP4=4,7,3,1;FQ=44.6;MQ=60;PV4=0.28,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A1133S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UNC13A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080421.2;VDB=0.0071;DN=132;DA=C/A;GM=NM_001080421.2,XM_005259821.1;GL=UNC13A;FG=missense;FD=missense,unknown;AAC=ALA/SER;PP=1133/1704,1133/1681;CDP=3397;GS=99;PH=0.001;CP=0.987;CG=-0.725;DG;DV=by-frequency,by-cluster,by-1000G;DSP=32;GESP=A:782/C:11986;PAC=NP_001073890.2,XP_005259878.1 GT:GQ:PL 0/0:14:0,12,89 0/0:6:0,3,25 1/1:5:87,12,0 0/0:20:0,18,104</p>

      <p>19 20133846 rs7255165 C T 23.50 midCVG AC1=3;AF1=0.4339;DB;DP=12;DP4=5,4,2,1;FQ=25.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V65M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF682;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033196.2;VDB=0.0044;DN=116;DA=C/T;GM=NM_001077349.1,NM_033196.2;GL=ZNF682;FG=missense;FD=missense;AAC=VAL/MET;PP=33/467,65/499;CDP=97,193;GS=21;PH=0.005;CP=0.300;CG=0.898;AA=C;HA=42.2;HE=18.1;HC=6.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=34;GESP=T:2867/C:9861;PAC=NP_001070817.1,NP_149973.1 GT:GQ:PL 0/0:20:0,21,136 0/1:3:0,0,0 1/1:4:48,6,0 0/1:18:17,0,43</p>

      <p>19 21909948 rs7246400 G A 38.50 lowCVG AC1=5;AF1=0.6627;DB;DP=8;DP4=0,1,0,6;FQ=-6.85;MQ=34;PV4=1,0.01,1,0.36;SNPEFF_AMINO_ACID_CHANGE=T389I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF100;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173531.3;VDB=0.0289;DN=116;DA=G/A;GM=NM_173531.3,XM_005259783.1,XM_005259784.1,XM_005259785.1;GL=ZNF100;FG=missense;FD=missense,unknown,unknown,unknown;AAC=THR/ILE;PP=389/543,388/542,356/510,174/328;CDP=1166,1163,1067,521;GS=89;PH=0.804;CP=0.029;CG=0.841;AA=G;HA=30.4;HC=40.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=844;GESP=A:6476/G:6444;PAC=NP_775802.2,XP_005259840.1,XP_005259841.1,XP_005259842.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:16:76,18,0 0/1:3:0,3,27</p>

      <p>19 22941591 rs62119159 A T 42.80 midCVG AC1=6;AF1=0.739;DB;DP=13;DP4=2,1,0,10;FQ=19.6;MQ=46;PV4=0.038,4.5e-06,1,1;SNPEFF_AMINO_ACID_CHANGE=C374S;SNPEFF_CODON_CHANGE=Tgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080409.2;VDB=0.0101;DN=129;DA=A/T;GM=NM_001080409.2;GL=ZNF99;FG=missense;FD=missense;AAC=CYS/SER;PP=374/865;CDP=1120;GS=112;PH=1;CP=0.032;CG=1.120;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=894;GESP=T:1032/A:11442;PAC=NP_001073878.2 GT:GQ:PL 0/1:12:6,0,51 0/1:3:0,0,0 1/1:27:74,27,0 0/1:3:0,0,0</p>

      <p>19 23158847 rs1433111 T A 17 lowCVG AC1=3;AF1=0.4009;DB;DP=8;DP4=1,3,3,0;FQ=19;MQ=58;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K431I;SNPEFF_CODON_CHANGE=aAa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0063;DN=88;DA=T/A;GM=NM_001267716.1;GL=ZNF728;FG=missense;FD=missense;AAC=LYS/ILE;PP=431/623;CDP=1292;GS=102;CP=0.729;CG=-1.320;AA=T;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1066;PAC=NP_001254645.1 GT:GQ:PL 0/0:4:0,3,16 0/0:7:0,6,42 1/1:3:46,6,0 0/1:13:13,0,16</p>

      <p>19 23159486 rs289318 T C 119 highCVG AC1=4;AF1=0.5102;DB;DP=20;DP4=6,3,3,7;FQ=106;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Y218C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF728;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001267716.1;VDB=0.0021;DN=79;DA=T/C;GM=NM_001267716.1,XM_005259908.1;GL=ZNF728;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=TYR/CYS,none;PP=218/623,NA;CDP=653,NA;GS=194,NA;CP=0.272;CG=0.945;AA=T;HE=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=427;PAC=NP_001254645.1,NA GT:GQ:PL 0/0:9:0,12,78 0/1:20:34,0,17 1/1:18:121,21,0 0/1:15:12,0,56</p>

      <p>19 35435069 rs765746 A G 47.60 midCVG AC1=4;AF1=0.4586;DB;DP=12;DP4=6,1,4,1;FQ=50.2;MQ=60;PV4=1,0.4,1,0.13;SNPEFF_AMINO_ACID_CHANGE=Y401C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF30;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099437.1;VDB=0.0179;DN=86;DA=A/G;GM=NM_001099437.1,NM_001099438.1,NM_194325.2,NR_024018.1;GL=ZNF30;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=TYR/CYS,TYR/CYS,TYR/CYS,none;PP=401/625,401/625,400/624,NA;CDP=1202,1202,1199,1657;GS=194,194,194,NA;PH=1;CP=0.040;CG=1.040;AA=A;HA=22.5;HE=18.8;HC=33.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=943;GESP=G:2295/A:10709;PAC=NP_001092907.1,NP_001092908.1,NP_919306.2,NA GT:GQ:PL 0/0:6:0,6,39 0/0:13:0,15,92 1/1:8:75,12,0 0/1:4:20,3,0</p>

      <p>19 36303664 rs3848666 G C 17.50 lowCVG AC1=4;AF1=0.5;DB;DP=8;DP4=2,1,2,1;FQ=20;MQ=60;PV4=1,0.19,1,0.28;SNPEFF_AMINO_ACID_CHANGE=P91R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRODH2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021232.1;VDB=0.0075;DN=108;DA=G/C;GM=NM_021232.1,XM_005259123.1;GL=PRODH2;FG=missense;FD=missense,unknown;AAC=PRO/ARG;PP=91/537,15/379;CDP=272,44;GS=103;PH=0.004;CP=0.000;CG=0.647;AA=A;HA=0.8;HE=14.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=131;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;GESP=C:1572/G:11434;PAC=NP_067055.1,XP_005259180.1 GT:GQ:PL 0/0:9:0,9,65 0/1:3:0,0,0 1/1:5:55,9,0 0/1:3:0,0,0</p>

      <p>19 37117302 rs3108171 A G 50.10 lowCVG AC1=5;AF1=0.5824;DB;DP=8;DP4=1,2,5,0;FQ=19.9;MQ=60;PV4=0.11,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=E168G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0027;DN=103;DA=A/G;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=GLU/GLY;PP=167/550,168/551,168/551,167/550;CDP=500,503,503,500;GS=98;PH=0.103;CP=1.000;CG=3.820;AA=A;HA=28.0;HE=40.7;HC=16.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=271;GESP=G:8458/A:4542;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:3:0,3,26 0/1:3:0,0,0 1/1:10:75,12,0 0/1:19:16,0,31</p>

      <p>19 37118439 rs61732180 C T 83.80 midCVG AC1=5;AF1=0.6451;DB;DP=9;DP4=3,0,3,3;FQ=15.4;MQ=60;PV4=0.46,1,1,0.002;SNPEFF_AMINO_ACID_CHANGE=T547M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0028;DN=129;DA=C/T;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=THR/MET;PP=546/550,547/551,547/551,546/550;CDP=1637,1640,1640,1637;GS=81;PH=0;CP=0.009;CG=1.990;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1408;GESP=T:2043/C:10963;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:17:27,0,16 0/0:4:0,6,41 1/1:9:59,9,0 1/1:6:44,6,0</p>

      <p>19 37488472 rs1644698 C G 25.20 lowCVG AC1=6;AF1=0.7081;DB;DP=8;DP4=1,0,0,3;FQ=-11;MQ=60;PV4=0.25,0.34,1,0.21;SNPEFF_AMINO_ACID_CHANGE=P627A;SNPEFF_CODON_CHANGE=Ccc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0049;DN=89;DA=C/G;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=PRO/ALA;PP=627/636,563/572;CDP=1879,1687;GS=27;PH=0;CP=0.004;CG=-3.900;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=1254;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:47,6,0 0/1:17:14,0,20</p>

      <p>19 37488481 rs1363753 G C 93.70 midCVG AC1=4;AF1=0.5436;DB;DP=10;DP4=1,2,1,6;FQ=6.29;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G630R;SNPEFF_CODON_CHANGE=Ggt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0155;DN=88;DA=G/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=GLY/ARG;PP=630/636,566/572;CDP=1888,1696;GS=125;PH=0;CP=0.022;CG=1.640;AA=C;HA=29.2;HE=49.2;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1263;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:0,0,0 1/1:5:48,6,0 0/1:24:85,0,21</p>

      <p>19 37488499 rs1667366 T C 105 midCVG AC1=6;AF1=0.6771;DB;DP=10;DP4=1,1,1,7;FQ=-10.4;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*636R;SNPEFF_CODON_CHANGE=Tga/Cga;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0136;DN=89;DA=T/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=stop-lost;FD=stop-lost;AAC=stop/ARG;PP=636/636,572/572;CDP=1906,1714;CP=0.084;CG=2.660;AA=C;HA=43.3;HE=48.3;HC=26.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1281;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:6:47,6,0 0/1:3:96,1,0</p>

      <p>19 39401520 . G C 15.90 midCVG AC1=4;AF1=0.497;DP=11;DP4=1,6,4,0;FQ=18.8;MQ=60;PV4=0.015,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q132E;SNPEFF_CODON_CHANGE=Cag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCER2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001243212.1;VDB=0.0133;GM=NM_001243212.1;GL=LOC643669;FG=missense;FD=unknown;AAC=GLN/GLU;PP=132/267;CDP=394;GS=29;CP=0.080;CG=2.700;DSP=204;PAC=NP_001230141.1 GT:GQ:PL 0/0:6:0,6,38 0/1:3:0,0,0 1/1:4:40,6,0 0/1:18:16,0,58</p>

      <p>19 40363926 rs3746013 C G 69.60 midCVG AC1=3;AF1=0.4126;DB;DP=13;DP4=3,3,3,3;FQ=71.8;MQ=60;PV4=1,0.033,1,1;SNPEFF_AMINO_ACID_CHANGE=D4906H;SNPEFF_CODON_CHANGE=Gat/Cat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FCGBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003890.2;VDB=0.0085;DN=107;DA=C/G;GM=NM_003890.2,XM_005259365.1;GL=FCGBP;FG=missense;FD=missense,unknown;AAC=ASP/HIS;PP=4906/5406;CDP=14716;GS=81;PH=0.997;CP=0.005;CG=-1.040;AA=C;HA=34.2;HE=19.2;HC=44.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:2841/C:10165;PAC=NP_003881.2,XP_005259422.1 GT:GQ:PL 0/0:9:0,9,75 0/0:4:0,3,26 1/1:5:63,9,0 0/1:42:51,0,38</p>

      <p>19 40729096 rs147729392 C G 71.20 highCVG AC1=4;AF1=0.5608;DB;DP=20;DP4=3,0,1,15;FQ=25;MQ=60;PV4=0.0041,0.00036,1,0.34;SNPEFF_AMINO_ACID_CHANGE=D263H;SNPEFF_CODON_CHANGE=Gac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNTD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024877.3;VDB=0.0024;DN=134;DA=C/G;GM=NM_024877.3;GL=CNTD2;FG=missense;FD=missense;AAC=ASP/HIS;PP=263/308;CDP=787;GS=81;PH=0.742;CP=0.000;CG=-0.127;AA=C;DV=by-cluster;DSP=115;CPG=40729088-40729430;GESP=C:4564;PAC=NP_079153.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:45:111,48,0 0/0:7:0,9,61</p>

      <p>19 40901614 rs268671 A G 70.30 highCVG AC1=4;AF1=0.4851;DB;DP=16;DP4=3,6,2,4;FQ=73.2;MQ=60;PV4=1,0.33,1,0.16;SNPEFF_AMINO_ACID_CHANGE=V882A;SNPEFF_CODON_CHANGE=gTg/gCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRX;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181882.2;VDB=0.0157;DN=79;DA=A/G;GM=NM_020956.2,NM_181882.2,XM_005259111.1;GL=PRX;FG=3-prime-UTR,missense,missense;FD=utr-variant-3-prime,missense,unknown;AAC=none,VAL/ALA,VAL/ALA;PP=NA,882/1462,856/1436;CDP=NA,2645,2567;GS=NA,64,64;PH=0;CP=0.033;CG=2.700;AA=G;HA=9.5;HE=49.1;HC=20.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1088;GESP=G:7930/A:5076;PAC=NA,NP_870998.2,XP_005259168.1 GT:GQ:PL 0/0:16:0,18,111 0/1:3:0,0,1/1:9:84,12,0 0/1:32:30,0,49</p>

      <p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

      <p>19 41035024 . C T 33.50 midCVG AC1=4;AF1=0.5209;DP=10;DP4=0,3,5,0;FQ=22.5;MQ=60;PV4=0.018,0.066,1,0.086;SNPEFF_AMINO_ACID_CHANGE=R1312*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_020971.2;VDB=0.0182;GM=NM_020971.2,XM_005259112.1,XM_005259113.1;GL=SPTBN4;FG=stop-gained;FD=unknown;AAC=ARG/stop;PP=1312/2565,1307/2560,55/1308;CDP=3934,3919,163;CP=1.000;CG=2.100;AA=C;DSP=15;GESP=C:13006;PPI=DISC1:789;PAC=NP_066022.2,XP_005259169.1,XP_005259170.1 GT:GQ:PL 0/1:11:8,0,21 0/0:3:0,3,28 1/1:5:45,6,0 0/1:16:23,0,13</p>

      <p>SERTAD1 19 43243238 rs11559136 A G 43.70 lowCVG AC1=5;AF1=0.6561;DB;DP=7;DP4=2,1,1,3;FQ=11.7;MQ=51;PV4=0.49,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L23S;SNPEFF_CODON_CHANGE=tTa/tCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PSG3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021016.3;VDB=0.0075;DN=120;DA=A/G;GM=NM_021016.3,XM_005259068.1;GL=PSG3;FG=missense;FD=missense,unknown;AAC=LEU/SER;PP=23/429,45/459;CDP=68,134;GS=145;PH=0;CP=0.977;CG=1.390;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4;GESP=G:8155/A:4843;PAC=NP_066296.2,XP_005259125.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,47 1/1:6:43,6,0 1/1:4:41,3,0</p>

      <p>19 44591360 rs8113249 G A 146 highCVG AC1=4;AF1=0.5099;DB;DP=21;DP4=3,5,6,7;FQ=114;MQ=60;PV4=1,0.021,1,0.056;SNPEFF_AMINO_ACID_CHANGE=E577K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF284;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001037813.2;VDB=0.0032;DN=116;DA=G/A;GM=NM_001037813.2,XM_005258859.1;GL=ZNF284;FG=missense;FD=missense,unknown;AAC=GLU/LYS;PP=577/594;CDP=1729;GS=56;PH=0;CP=0.707;CG=-0.036;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=1494;GESP=A:10995/G:1991;PAC=NP_001032902.1,XP_005258916.1 GT:GQ:PL 0/0:12:0,15,106 0/1:3:0,0,0 1/1:27:160,30,0 0/1:33:30,0,49</p>

      <p>19 44610798 rs4239529 A T 111 highCVG AC1=4;AF1=0.5334;DB;DP=15;DP4=3,3,2,7;FQ=45.2;MQ=60;PV4=0.33,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H162L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF224;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013398.2;VDB=0.0096;DN=111;DA=A/T;GM=NM_013398.2,NR_033341.1,XM_005259221.1;GL=ZNF224/LOC100379224;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=HIS/LEU,none,HIS/LEU;PP=162/708,NA,162/708;CDP=485,2055,485;GS=99,NA,99;PH=0;CP=0.000;CG=-1.700;AA=T;HA=9.2;HE=12.5;HC=37.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=250;GESP=T:11402/A:1604;PAC=NP_037530.2,NA,XP_005259278.1 GT:GQ:PL 0/0:12:0,15,81 0/1:3:0,0,0 1/1:12:78,15,0 0/1:8:76,0,4</p>

      <p>19 45716364 rs189063316 C T 11.60 lowCVG AC1=5;AF1=0.5837;DB;DP=5;DP4=1,0,2,0;FQ=-12.4;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R398Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXOC3L2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138568.3;VDB=0.0116;DN=135;DA=C/T;GM=NM_138568.3;GL=EXOC3L2;FG=missense;FD=missense;AAC=ARG/GLN;PP=398/410;CDP=1193;GS=43;PH=0.46;CP=0.112;CG=1.260;AA=C;DV=by-cluster;DSP=252;GESP=T:121/C:12673;PAC=NP_612635.3 GT:GQ:PL 0/1:3:0,0,0/0:4:0,3,19 1/1:4:46,6,0 0/1:3:0,0,0</p>

      <p>19 46269076 rs2014576 G A 95.10 midCVG AC1=6;AF1=0.6715;DB;DP=11;DP4=2,1,3,4;FQ=14.1;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P635S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SIX5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175875.4;VDB=0.0122;DN=92;DA=G/A;GM=NM_175875.4;GL=SIX5;FG=missense;FD=missense;AAC=PRO/SER;PP=635/740;CDP=1903;GS=74;PH=0.004;CP=0.983;CG=-0.760;AA=G;HA=33.3;HE=47.5;HC=29.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=294;GESP=A:5213/G:7717;PAC=NP_787071.2 GT:GQ:PL 0/1:29:32,0,29 0/1:4:0,3,25 1/1:7:54,6,0 1/1:9:54,9,0</p>

      <p>19 48495304 rs60213124 G A 79 midCVG AC1=4;AF1=0.5008;DB;DP=12;DP4=3,3,3,3;FQ=79.1;MQ=60;PV4=1,0.42,1,0.17;SNPEFF_AMINO_ACID_CHANGE=T12M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BSPH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128326.2;VDB=0.0230;DN=129;DA=G/A;GM=NM_001128326.2;GL=BSPH1;FG=missense;FD=missense;AAC=THR/MET;PP=12/133;CDP=35;GS=81;PH=0.962;CP=0.000;CG=-0.642;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=G:4566;PAC=NP_001121798.1 GT:GQ:PL 0/0:13:0,15,105 0/1:3:27,3,0 1/1:12:99,15,0 0/0:3:0,3,21</p>

      <p>19 48525507 rs2303690 G A 67.70 midCVG AC1=5;AF1=0.6528;DB;DP=13;DP4=3,0,5,3;FQ=4.34;MQ=60;PV4=0.49,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E199K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ELSPBP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022142.4;VDB=0.0107;DN=100;DA=G/A;GM=NM_022142.4;GL=ELSPBP1;FG=missense;FD=missense;AAC=GLU/LYS;PP=199/224;CDP=595;GS=56;PH=0.011;CP=0.044;CG=1.540;AA=A;HA=31.1;HE=47.0;HC=13.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=81;GESP=A:7578/G:5428;PPI=MYF5:832|MYOG:832|MYOD1:832;PAC=NP_071425.3 GT:GQ:PL 0/1:10:52,0,8 0/1:3:0,0,0 1/1:8:53,11,2 0/1:9:4,0,29</p>

      <p>19 48608598 rs2307279 C G 24.60 lowCVG AC1=5;AF1=0.6092;DB;DP=5;DP4=0,2,0,3;FQ=-7.18;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A48P;SNPEFF_CODON_CHANGE=Gct/Cct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLA2G4C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159322.1;DN=100;DA=C/G;GM=NM_001159322.1,NM_001159323.1,NM_003706.2,XM_005259361.1;GL=PLA2G4C;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/PRO;PP=48/552,38/528,38/542,48/534;CDP=142,112,112,142;GS=27;PH=0.74;CP=0.000;CG=0.300;AA=C;HA=38.7;HE=4.2;HC=30.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=9;GESP=G:2512/C:10494;PAC=NP_001152794.1,NP_001152795.1,NP_003697.2,XP_005259418.1 GT:GQ:PL 0/1:15:17,0,14 0/1:4:0,3,17 1/1:4:24,3,0 1/1:4:25,3,0</p>

      <p>19 49377319 rs610308 A G 17.80 midCVG AC1=3;AF1=0.4136;DB;DP=11;DP4=3,1,0,3;FQ=20;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K277E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R15A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014330.3;VDB=0.0071;DN=83;DA=A/G;GM=NM_014330.3;GL=PPP1R15A;FG=missense;FD=missense;AAC=LYS/GLU;PP=277/675;CDP=829;GS=56;PH=0;CP=0.000;CG=-3.110;AA=G;HA=22.5;HE=32.3;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=837;GESP=G:6297/A:6709;PPI=PPP1CA:942|PPP1CC:937|SMARCB1:846|PPP1CB:845;PAC=NP_055145.3 GT:GQ:PL 0/0:6:0,6,46 0/0:4:0,3,24 1/1:3:39,6,0 0/1:13:21,0,8</p>

      <p>19 49438363 rs2270941 G A 47.70 midCVG AC1=5;AF1=0.5687;DB;DP=10;DP4=0,4,2,3;FQ=37.1;MQ=60;PV4=0.44,0.036,1,1;SNPEFF_AMINO_ACID_CHANGE=S66N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHDH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014475.3;VDB=0.0081;DN=100;DA=G/A;GM=NM_014475.3,XM_005258748.1;GL=DHDH;FG=missense;FD=missense,unknown;AAC=SER/ASN,ALA/THR;PP=66/335,9/223;CDP=197,25;GS=46,58;PH=0;CP=0.282;CG=4.740;AA=A;HA=22.1;HE=21.4;HC=45.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00980:MetabolismOfXenobioticsByCytochromeP450;GESP=A:4923/G:8051;PAC=NP_055290.1,XP_005258805.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:7:65,9,0 0/1:26:23,0,51</p>

      <p>19 52496149 rs16983353 C T 125 highCVG AC1=5;AF1=0.6541;DB;DP=17;DP4=1,5,4,6;FQ=50.5;MQ=59;PV4=0.59,1,0.23,0.15;SNPEFF_AMINO_ACID_CHANGE=R738K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF615;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199324.1;VDB=0.0050;DN=123;DA=C/T;GM=NM_001199324.1,NM_198480.3,XM_005258801.1,XM_005258802.1,XM_005258803.1,XM_005258804.1,XM_005258805.1,XM_005258806.1;GL=ZNF615;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=738/743,727/732,743/748,738/743,732/737,727/732,619/624,619/624;CDP=2213,2180,2228,2213,2195,2180,1856,1856;GS=26;PH=0;CP=0.003;CG=2.500;AA=T;HA=35.3;HE=28.2;HC=33.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1942;GESP=T:4942/C:8064;PAC=NP_001186253.1,NP_940882.3,XP_005258858.1,XP_005258859.1,XP_005258860.1,XP_005258861.1,XP_005258862.1,XP_005258863.1 GT:GQ:PL 0/1:18:13,0,48 0/1:3:0,0,0 1/1:11:88,12,0 0/1:40:65,0,39</p>

      <p>19 52888245 rs55748277 A G 23.70 midCVG AC1=4;AF1=0.4732;DB;DP=11;DP4=3,3,2,3;FQ=26.4;MQ=59;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=K471R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF880;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145434.1;VDB=0.0110;DN=129;DA=A/G;GM=NM_001145434.1;GL=ZNF880;FG=missense;FD=missense;AAC=LYS/ARG;PP=471/578;CDP=1412;GS=26;PH=0.991;CP=0.007;CG=-0.460;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1144;GESP=G:1745/A:2821;PAC=NP_001138906.1 GT:GQ:PL 0/1:6:3,0,37 0/1:13:24,0,9 1/1:4:39,6,0 0/0:6:0,6,45</p>

      <p>19 53086124 rs373554 C A 100 highCVG AC1=5;AF1=0.6181;DB;DP=15;DP4=5,1,5,4;FQ=54.4;MQ=59;PV4=0.58,0.21,0.22,0.18;SNPEFF_AMINO_ACID_CHANGE=T337K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF701;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172655.1;VDB=0.0155;DN=80;DA=C/A;GM=NM_001172655.1,NM_018260.2;GL=ZNF701;FG=missense;FD=missense;AAC=THR/LYS;PP=337/532,271/466;CDP=1010,812;GS=78;PH=0;CP=0.001;CG=-0.451;AA=A;HA=0.0;HE=17.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=670;GESP=A:9195/C:3811;PAC=NP_001166126.1,NP_060730.2 GT:GQ:PL 0/1:22:17,0,33 0/1:16:21,0,15 1/1:11:58,12,0 0/1:47:49,0,47</p>

      <p>19 53117531 rs1056185 C T 9.83 lowCVG AC1=6;AF1=0.6684;DB;DP=7;DP4=1,0,1,1;FQ=-10.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S96N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF83;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001105549.1;VDB=0.0014;DN=86;DA=C/T;GM=NM_001105549.1,NM_001105550.1,NM_001105551.1,NM_001105552.1,NM_001277945.1,NM_001277946.1,NM_001277947.1,NM_001277948.1,NM_001277949.1,NM_001277951.1,NM_001277952.1,NM_018300.3;GL=ZNF83;FG=missense;FD=missense;AAC=SER/ASN;PP=96/517;CDP=287;GS=46;PH=0;CP=0.000;CG=-2.410;AA=T;HA=8.7;HE=37.5;HC=27.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=520;GESP=T:8689/C:4317;PAC=NP_001099019.1,NP_001099020.1,NP_001099021.1,NP_001099022.1,NP_001264874.1,NP_001264875.1,NP_001264876.1,NP_001264877.1,NP_001264878.1,NP_001264880.1,NP_001264881.1,NP_060770.3 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,0,0 1/1:3:27,3,0 1/1:3:20,3,0</p>

      <p>19 53959343 rs2708742 G A 58.30 lowCVG AC1=5;AF1=0.6547;DB;DP=7;DP4=0,1,3,1;FQ=-14.5;MQ=60;PV4=0.4,1,1,0.35;SNPEFF_AMINO_ACID_CHANGE=G528S;SNPEFF_CODON_CHANGE=Ggc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF761;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008401.3;VDB=0.0002;DN=100;DA=G/A;GM=NM_001008401.3;GL=ZNF761;FG=missense;FD=missense;AAC=GLY/SER;PP=528/747;CDP=1583;GS=56;CP=0.001;CG=1.140;AA=G;HA=29.9;HE=41.9;HC=24.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1440;GESP=A:4990/G:8016;PAC=NP_001008401.3 GT:GQ:PL 0/1:3:0,3,18 0/1:3:0,0,0 1/1:10:95,12,0 0/1:3:0,0,0</p>

      <p>19 53959568 rs2617726 G C 74.70 highCVG AC1=4;AF1=0.5121;DB;DP=15;DP4=3,2,5,2;FQ=27;MQ=52;PV4=1,0.28,1,1;SNPEFF_AMINO_ACID_CHANGE=E603Q;SNPEFF_CODON_CHANGE=Gag/Cag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF761;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001008401.3;VDB=0.0125;DN=100;DA=G/C;GM=NM_001008401.3;GL=ZNF761;FG=missense;FD=missense;AAC=GLU/GLN;PP=603/747;CDP=1808;GS=29;CP=0.228;CG=1.130;AA=G;HA=39.2;HE=41.4;HC=24.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1665;GESP=C:5269/G:7733;PAC=NP_001008401.3 GT:GQ:PL 0/0:3:0,3,19 0/0:7:0,9,50 1/1:15:98,18,0 0/1:4:22,1,0</p>

      <p>19 54755922 rs12977062 C T 24.20 highCVG AC1=3;AF1=0.4158;DB;DP=22;DP4=1,11,4,0;FQ=26.4;MQ=48;PV4=0.0027,1,0.05,1;SNPEFF_AMINO_ACID_CHANGE=A542T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081442.1;VDB=0.0083;DN=121;DA=C/T;GM=NM_001081442.1,NM_001081443.1,NM_006840.3,XM_005258435.1;GL=LILRB5;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=542/592,442/492,541/591,525/575;CDP=1624,1324,1621,1573;GS=58;PH=0.013;CP=0.000;CG=-2.320;AA=C;HA=0.0;HE=4.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=T:2224/C:10782;PAC=NP_001074911.1,NP_001074912.1,NP_006831.1,XP_005258492.1 GT:GQ:PL 0/0:26:0,27,82 0/0:9:0,9,70 1/1:4:37,6,0 1/1:4:35,6,0</p>

      <p>19 55106778 rs10418391 G A 52.20 lowCVG AC1=6;AF1=0.7307;DB;DP=6;DP4=0,1,0,4;FQ=-9.7;MQ=55;PV4=1,1,0.34,1;SNPEFF_AMINO_ACID_CHANGE=R191H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006863.3;VDB=0.0197;DN=119;DA=G/A;GM=NM_001278318.1,NM_001278319.1,NM_006863.3,NR_103501.1,NR_103502.1,NR_103503.1;GL=LILRA1;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=unknown,unknown,missense,unknown,unknown,unknown;AAC=ARG/HIS,ARG/HIS,ARG/HIS,none,none,none;PP=191/290,191/440,191/490,NA,NA,NA;CDP=572,572,572,744,988,744;GS=29,29,29,NA,NA,NA;PH=0.042;CP=0.000;CG=-0.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=90;GESP=A:2052/G:10954;PAC=NP_001265247.1,NP_001265248.1,NP_006854.1,NA,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:4:27,3,0 1/1:9:65,9,0</p>

      <p>19 55143083 rs1061679 T C 34 lowCVG AC1=5;AF1=0.5573;DB;DP=8;DP4=5,0,2,1;FQ=32.8;MQ=58;PV4=0.37,1,1,0.017;SNPEFF_AMINO_ACID_CHANGE=L68P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081637.2;VDB=0.0034;DN=86;DA=T/C;GM=NM_001081637.1,NM_001081638.2,NM_001081639.2,NM_001278398.1,NM_001278399.1,NM_006669.5,NR_103518.1;GL=LILRB1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,unknown,unknown,missense,unknown;AAC=LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,LEU/PRO,none;PP=68/653,68/652,68/652,68/635,68/457,68/651,NA;CDP=203,203,203,203,203,203,368;GS=98,98,98,98,98,98,NA;PH=0;CP=0.000;CG=-3.160;AA=C;HA=0.8;HC=6.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=133;GESP=C:5322/T:7684;PPI=B2M:959;PAC=NP_001075106.1,NP_001075107.1,NP_001075108.1,NP_001265327.1,NP_001265328.1,NP_006660.3,NA GT:GQ:PL 0/0:12:0,15,73 0/1:3:0,0,0 1/1:5:50,6,0 0/1:3:26,3,0</p>

      <p>19 55174498 rs28366008 T C 53.10 midCVG AC1=3;AF1=0.4334;DB;DP=10;DP4=2,3,0,5;FQ=52;MQ=58;PV4=0.44,1,0.17,1;SNPEFF_AMINO_ACID_CHANGE=F46L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001278429.3;VDB=0.0117;DN=125;DA=T/C;GM=NM_001278426.2,NM_001278427.2,NM_001278428.2,NM_001278429.2,NM_001278430.2;GL=LILRB4;FG=missense;FD=unknown;AAC=PHE/LEU;PP=5/449,5/448,5/449,46/479,5/255;CDP=13,13,13,136,13;GS=22;PH=0.001;CP=0.000;CG=0.476;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=22;GESP=C:4017/T:8989;PPI=PTPN11:846|PTPN6:846;PAC=NP_001265355.1,NP_001265356.1,NP_001265357.1,NP_001265358.1,NP_001265359.1 GT:GQ:PL 0/0:6:0,6,44 0/0:4:0,3,22 1/1:6:66,9,0 0/1:30:31,0,30</p>

      <p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>

      <p>19 55494157 rs4306647 G A 14.20 highCVG AC1=3;AF1=0.3892;DB;DP=15;DP4=6,3,3,1;FQ=16.2;MQ=41;PV4=1,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R364K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001174081.1;VDB=0.0098;DN=111;DA=G/A;GM=NM_001174081.1,NM_001174082.1,NM_001174083.1,NM_017852.3,XM_005259050.1;GL=NLRP2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/LYS;PP=364/1063,342/1041,341/1040,364/1063,361/1060;CDP=1091,1025,1022,1091,1082;GS=26;PH=0.363;CP=0.019;CG=1.640;AA=G;HA=19.6;HE=2.2;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=628;GESP=A:915/G:12091;PAC=NP_001167552.1,NP_001167553.1,NP_001167554.1,NP_060322.1,XP_005259107.1 GT:GQ:PL 0/0:5:0,3,19 0/1:11:10,0,32 1/1:5:48,9,0 0/0:12:0,12,50</p>

      <p>19 56499279 rs306457 G C 68.10 midCVG AC1=4;AF1=0.4813;DB;DP=12;DP4=0,6,0,6;FQ=46;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=*1049Y;SNPEFF_CODON_CHANGE=taG/taC;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP8;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_176811.2;VDB=0.0070;DN=79;DA=G/C;GM=NM_176811.2,XM_005258511.1;GL=NLRP8;FG=stop-lost;FD=stop-lost,unknown;AAC=stop/TYR;PP=1049/1049,1030/1030;CDP=3147,3090;CP=0.000;CG=0.372;AA=C;HA=26.7;HE=30.0;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=100;GESP=C:9537/G:3469;PAC=NP_789781.2,XP_005258568.1 GT:GQ:PL 0/0:7:0,9,50 0/1:17:21,0,15 1/1:4:45,6,0 0/1:27:47,0,23</p>

      <p>19 57641146 rs1027392 A G 124 highCVG AC1=5;AF1=0.6123;DB;DP=15;DP4=1,4,2,8;FQ=44.9;MQ=60;PV4=1,0.41,1,0.22;SNPEFF_AMINO_ACID_CHANGE=N368S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020903.2;VDB=0.0091;DN=86;DA=A/G;GM=NM_020903.2;GL=USP29;FG=missense;FD=missense;AAC=ASN/SER;PP=368/923;CDP=1103;GS=46;PH=0.916;CP=0.000;CG=-0.648;AA=G;HA=25.9;HE=41.4;HC=6.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1119;GESP=G:8646/A:4358;PAC=NP_065954.1 GT:GQ:PL 0/0:8:0,12,80 0/1:10:75,0,8 1/1:8:57,9,0 1/1:6:39,6,0</p>

      <p>19 57646570 rs4801433 T C 80.30 midCVG AC1=5;AF1=0.6545;DB;DP=14;DP4=3,2,0,7;FQ=66.9;MQ=60;PV4=0.045,0.067,1,0.21;SNPEFF_AMINO_ACID_CHANGE=I379V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZIM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052882.1;VDB=0.0156;DN=111;DA=T/C;GM=NM_052882.1;GL=ZIM3;FG=missense;FD=missense;AAC=ILE/VAL;PP=379/473;CDP=1135;GS=29;PH=0.978;CP=0.326;CG=1.680;AA=T;HA=24.0;HE=49.7;HC=38.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=894;GESP=C:5945/T:7061;PAC=NP_443114.1 GT:GQ:PL 0/0:11:0,15,108 0/1:3:0,0,0 1/1:11:66,12,0 1/1:8:60,9,0</p>

      <p>19 57647236 rs7252632 T C 26.30 midCVG AC1=4;AF1=0.4772;DB;DP=11;DP4=1,5,1,3;FQ=29;MQ=60;PV4=1,0.41,1,0.12;SNPEFF_AMINO_ACID_CHANGE=N157D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZIM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052882.1;VDB=0.0086;DN=116;DA=T/C;GM=NM_052882.1;GL=ZIM3;FG=missense;FD=missense;AAC=ASN/ASP;PP=157/473;CDP=469;GS=23;PH=0;CP=0.000;CG=-4.360;AA=C;HA=24.3;HE=40.9;HC=6.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=228;GESP=C:8716/T:4290;PAC=NP_443114.1 GT:GQ:PL 0/0:4:0,3,20 0/1:5:1,0,45 1/1:4:48,6,0 0/1:20:18,0,35</p>

      <p>19 57760018 rs2014572 G A 89.90 midCVG AC1=4;AF1=0.5523;DB;DP=10;DP4=2,1,5,2;FQ=19.5;MQ=46;PV4=1,0.47,0.1,1;SNPEFF_AMINO_ACID_CHANGE=G68E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0081;DN=92;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense,5-prime-UTR;FD=missense,utr-variant-5-prime;AAC=GLY/GLU,none;PP=68/628,NA;CDP=203,NA;GS=98,NA;PH=0.682;CP=0.000;CG=-0.004;AA=A;HA=26.7;HE=42.5;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=46;GESP=A:2030/G:2536;PAC=NP_001018857.2,NA GT:GQ:PL 0/0:5:0,6,44 0/1:3:0,0,0 1/1:10:82,12,0 0/1:17:50,0,14</p>

      <p>19 57760057 rs3746231 G A 82.30 midCVG AC1=4;AF1=0.5287;DB;DP=11;DP4=2,2,5,2;FQ=27.7;MQ=47;PV4=0.58,1,0.055,0.28;SNPEFF_AMINO_ACID_CHANGE=G81D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0081;DN=107;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense,5-prime-UTR;FD=missense,utr-variant-5-prime;AAC=GLY/ASP,none;PP=81/628,NA;CDP=242,NA;GS=94,NA;PH=0;CP=0.000;CG=2.540;AA=G;HA=26.7;HE=42.2;HC=21.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=12;GESP=A:2029/G:2537;PAC=NP_001018857.2,NA GT:GQ:PL 0/0:7:0,9,57 0/1:3:0,0,0 1/1:9:87,12,0 0/1:13:38,0,10</p>

      <p>19 57764770 rs10421632 G A 40 midCVG AC1=5;AF1=0.612;DB;DP=10;DP4=2,2,3,2;FQ=32.8;MQ=60;PV4=1,0.13,1,0.11;SNPEFF_AMINO_ACID_CHANGE=V195I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF805;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001023563.3;VDB=0.0083;DN=119;DA=G/A;GM=NM_001023563.3,NM_001145078.1;GL=ZNF805;FG=missense;FD=missense;AAC=VAL/ILE;PP=195/628,62/495;CDP=583,184;GS=29;PH=0.026;CP=0.000;CG=-8.700;AA=A;HA=24.6;HE=41.9;HC=20.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=330;GESP=A:2022/G:2544;PAC=NP_001018857.2,NP_001138550.1 GT:GQ:PL 0/0:4:0,6,39 0/1:18:14,0,35 1/1:8:54,9,0 1/1:4:17,3,0</p>

      <p>19 58038964 rs12461014 T A 13.70 lowCVG AC1=6;AF1=0.7458;DB;DP=5;DP4=2,0,3,0;FQ=3.71;MQ=60;PV4=1,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=I8N;SNPEFF_CODON_CHANGE=aTt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF549;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199295.1;DN=120;DA=T/A;GM=NM_001199295.1,NM_153263.2;GL=ZNF549;FG=missense;FD=unknown;AAC=ILE/ASN;PP=8/641,8/628;CDP=23;GS=149;PH=0;CP=0.000;CG=0.703;AA=A;HA=0.0;HE=5.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=11;CPG=58038573-58039208;GESP=A:12747/T:259;PAC=NP_001186224.1,NP_694995.2 GT:GQ:PL 0/1:16:11,0,33 0/1:3:0,0,0 1/1:4:21,3,0 1/1:4:19,3,0</p>

      <p>19 58928786 rs7257872 A G 26.90 highCVG AC1=2;AF1=0.2698;DB;DP=17;DP4=12,2,3,0;FQ=28.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T301A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF584;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173548.1;VDB=0.0062;DN=116;DA=A/G;GM=NM_173548.1,XM_005258628.1,XM_005258629.1,XM_005258630.1,XM_005258631.1;GL=ZNF584;FG=missense,missense,missense,3-prime-UTR,3-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=THR/ALA,THR/ALA,THR/ALA,none,none;PP=301/422,293/414,256/377,NA,NA;CDP=901,877,766,NA,NA;GS=58,58,58,NA,NA;PH=0.001;CP=0.822;CG=0.242;AA=A;HA=49.6;HE=26.3;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=609;GESP=G:4324/A:8682;PAC=NP_775819.1,XP_005258685.1,XP_005258686.1,NA,NA GT:GQ:PL 0/0:29:0,27,139 0/0:4:0,0,0 1/1:3:68,9,0 0/0:17:0,15,76</p>

      <p>19 58946056 rs1465789 G A 19.10 midCVG AC1=3;AF1=0.3689;DB;DP=9;DP4=5,1,1,2;FQ=21.1;MQ=60;PV4=0.23,0.3,1,0.085;SNPEFF_AMINO_ACID_CHANGE=P252L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF132;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003433.3;VDB=0.0079;DN=88;DA=G/A;GM=NM_003433.3,XM_005259210.1;GL=ZNF132;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=252/707,239/694;CDP=755,716;GS=98;PH=0;CP=0.002;CG=0.980;AA=A;HA=14.9;HE=46.1;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=523;GESP=A:7487/G:5519;PAC=NP_003424.3,XP_005259267.1 GT:GQ:PL 0/0:7:0,6,53 0/0:7:0,6,49 1/1:3:49,6,0 0/1:13:13,0,30</p>

      <p>19 59023166 rs34415062 G A 24 lowCVG AC1=4;AF1=0.5357;DB;DP=7;DP4=2,1,2,1;FQ=25.4;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=R53W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=G/A;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=53/691,53/607,53/396;CDP=157;GS=101;PH=0;CP=0.000;CG=-8.620;AA=A;HA=22.8;HE=22.2;HC=5.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=280;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=A:2195/G:10597;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,26 1/1:4:50,6,0 0/1:16:13,0,40</p>

      <p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

      <p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

      <p>20 17716416 rs1053993 C G 89 midCVG AC1=5;AF1=0.6077;DB;DP=14;DP4=3,4,3,4;FQ=77.9;MQ=60;PV4=1,0.18,1,0.1;SNPEFF_AMINO_ACID_CHANGE=T85S;SNPEFF_CODON_CHANGE=aCt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BANF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159495.1;VDB=0.0033;DN=86;DA=C/G;GM=NM_001014977.3,NM_001159495.1,NM_178477.4,XM_005260668.1;GL=BANF2;FG=missense;FD=missense,missense,missense,unknown;AAC=THR/SER;PP=78/91,85/98,78/91,78/91;CDP=233,254,233,233;GS=58;PH=0;CP=0.107;CG=3.360;AA=G;HA=45.0;HE=22.5;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=107;GESP=G:4533/C:8473;PAC=NP_001014977.2,NP_001152967.1,NP_848572.3,XP_005260725.1 GT:GQ:PL 0/1:8:3,0,60 0/0:6:0,9,61 1/1:8:66,9,0 1/1:8:67,9,0</p>

      <p>20 18505200 rs36023150 G T 94.50 highCVG AC1=5;AF1=0.6403;DB;DP=15;DP4=1,6,4,4;FQ=80.8;MQ=60;PV4=0.28,0.1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V164L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172745.1;VDB=0.0134;DN=126;DA=G/T;GM=NM_001172745.1,NM_001172746.1,NM_006363.4,NM_032985.4,NM_032986.3;GL=SEC23B;FG=missense;FD=missense;AAC=VAL/LEU;PP=164/768,146/750,164/768,164/768,164/768;CDP=490,436,490,490,490;GS=32;PH=0.994;CP=0.996;CG=4.910;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=T:97/G:12909;PAC=NP_001166216.1,NP_001166217.1,NP_006354.2,NP_116780.1,NP_116781.1 GT:GQ:PL 0/1:3:0,0,0 0/1:34:29,0,53 1/1:14:103,15,0 0/1:8:3,0,64</p>

      <p>20 33874720 rs2425049 C T 49.50 midCVG AC1=3;AF1=0.4207;DB;DP=14;DP4=1,6,0,5;FQ=51.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R621Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM83C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178468.5;VDB=0.0155;DN=100;DA=C/T;GM=NM_178468.5;GL=FAM83C;FG=missense;FD=missense;AAC=ARG/GLN;PP=621/748;CDP=1862;GS=43;PH=0;CP=0.334;CG=2.850;AA=T;HA=26.5;HE=21.9;HC=15.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1056;GESP=T:4644/C:8362;PAC=NP_848563.1 GT:GQ:PL 0/0:4:0,3,23 0/0:14:0,15,87 1/1:8:76,12,0 0/1:12:19,0,8</p>

      <p>20 34214173 rs11543239 G A 32.30 midCVG AC1=5;AF1=0.5743;DB;DP=10;DP4=1,3,1,2;FQ=34.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P540L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003915.5;VDB=0.0111;DN=120;DA=G/A;GM=NM_001198863.1,NM_003915.5,NM_152925.2,NM_152926.2,NM_152927.2,NM_152928.2,NR_037188.1;GL=CPNE1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,none;PP=534/537,540/543,535/538,535/538,535/538,535/538,NA;CDP=1601,1619,1604,1604,1604,1604,1978;GS=98,98,98,98,98,98,NA;PH=0.008;CP=0.016;CG=5.550;AA=G;HA=11.2;HE=5.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=131;GESP=A:911/G:12095;PAC=NP_001185792.1,NP_003906.2,NP_690902.1,NP_690903.1,NP_690904.1,NP_690905.1,NA GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:24,3,0</p>

      <p>20 52645167 rs61754122 C A 66.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=6,2,0,5;FQ=69.2;MQ=60;PV4=0.021,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V163F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BCAS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003657.2;VDB=0.0055;DN=129;DA=C/A;GM=NM_003657.2,XM_005260587.1,XM_005260588.1,XM_005260589.1,XM_005260590.1,XM_005260591.1,XM_005260592.1,XM_005260593.1,XM_005260594.1,XM_005260595.1,XM_005260596.1;GL=BCAS1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/PHE;PP=163/585,163/630,163/630,163/616,163/608,163/594,124/591,163/571,163/552,163/538,163/507;CDP=487,487,487,487,487,487,370,487,487,487,487;GS=50;PH=0.868;CP=0.003;CG=0.450;AA=C;DV=by-frequency,by-cluster;DSP=237;GESP=A:93/C:12913;PAC=NP_003648.2,XP_005260644.1,XP_005260645.1,XP_005260646.1,XP_005260647.1,XP_005260648.1,XP_005260649.1,XP_005260650.1,XP_005260651.1,XP_005260652.1,XP_005260653.1 GT:GQ:PL 0/0:16:0,18,101 0/0:6:0,6,48 1/1:4:50,6,0 1/1:6:65,9,0</p>

      <p>20 60887027 . C T 11.30 lowCVG AC1=4;AF1=0.554;DP=5;DP4=2,0,0,3;FQ=4.23;MQ=56;PV4=0.1,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G3195D;SNPEFF_CODON_CHANGE=gGc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=70;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0028;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=unknown;AAC=GLY/ASP;PP=3195/3696;CDP=9584;GS=94;PH=0.299;CP=0.001;CG=-2.870;AA=C;DSP=52;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:13002;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:48,9,0 0/0:6:0,6,36</p>

      <p>20 60899206 rs2427283 C T 73.40 midCVG AC1=5;AF1=0.6657;DB;DP=10;DP4=0,1,1,6;FQ=-7.71;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=V1900M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMA5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005560.4;VDB=0.0025;DN=100;DA=C/T;GM=NM_005560.4;GL=LAMA5;FG=missense;FD=missense;AAC=VAL/MET;PP=1900/3696;CDP=5698;GS=21;PH=0.839;CP=0.017;CG=1.990;AA=C;HE=31.7;HC=20.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=T:8377/C:4565;PPI=LAMC2:988|LAMC1:980|LAMB2:951|LAMC3:951|LAMB1:951|BCAM:846|COL7A1:846;PAC=NP_005551.3 GT:GQ:PL 0/1:4:0,3,26 0/1:3:0,0,0 1/1:19:111,21,0 0/1:3:0,0,0</p>

      <p>21 31812772 rs2832873 C A 95.50 highCVG AC1=6;AF1=0.744;DB;DP=17;DP4=3,3,4,6;FQ=70.4;MQ=60;PV4=1,0.023,1,0.032;SNPEFF_AMINO_ACID_CHANGE=L43M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRTAP15-1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181623.1;VDB=0.0148;DN=100;DA=C/A;GM=NM_181623.1;GL=KRTAP15-1;FG=missense;FD=missense;AAC=LEU/MET;PP=43/138;CDP=127;GS=15;PH=0.956;CP=0.961;CG=0.826;AA=C;HA=26.2;HE=45.1;HC=34.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=228721;GESP=A:7882/C:5124;PAC=NP_853654.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:124,24,0 0/1:15:9,0,102</p>

      <p>21 34907047 . T C 80.50 highCVG AC1=3;AF1=0.4181;DP=25;DP4=16,1,5,2;FQ=82.8;MQ=59;PV4=0.19,0.071,1,0.48;SNPEFF_AMINO_ACID_CHANGE=N85S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GART;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000819.4;VDB=0.0037;GM=NM_000819.4,NM_001136005.1,NM_001136006.1,NM_175085.2,XM_005260941.1;GL=GART;FG=missense;FD=unknown;AAC=ASN/SER;PP=85/1011,85/1011,85/1011,85/434,85/1011;CDP=254;GS=46;PH=0;CP=0.763;CG=2.160;AA=T;DSP=13;KP=hsa00230:PurineMetabolism|hsa00670:OneCarbonPoolByFolate|hsa01100:MetabolicPathways;GESP=T:13006;PPI=UBC:730;PAC=NP_000810.1,NP_001129477.1,NP_001129478.1,NP_780294.1,XP_005260998.1 GT:GQ:PL 0/1:32:31,0,53 0/0:38:0,39,130 1/1:11:96,15,0 0/0:4:0,3,18</p>

      <p>21 37661405 rs3827183 G A 106 highCVG AC1=3;AF1=0.4579;DB;DP=17;DP4=3,2,6,3;FQ=57.2;MQ=60;PV4=1,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=G2139E;SNPEFF_CODON_CHANGE=gGg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DOPEY2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005128.2;VDB=0.0104;DN=107;DA=G/A;GM=NM_005128.2;GL=DOPEY2;FG=missense;FD=missense;AAC=GLY/GLU;PP=2139/2299;CDP=6416;GS=98;PH=0.215;CP=0.199;CG=2.870;AA=G;HA=11.5;HE=11.2;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=A:1551/G:11455;PAC=NP_005119.2 GT:GQ:PL 0/0:6:0,6,53 0/1:6:85,0,0 1/1:6:64,9,0 0/0:6:0,6,45</p>

      <p>21 38525356 rs1053808 T C 81.50 midCVG AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1 GT:GQ:PL 0/1:30:24,0,88 0/1:14:50,0,13 1/1:5:28,3,0 1/1:5:21,3,0</p>

      <p>21 43547873 rs220159 G A 23.40 midCVG AC1=5;AF1=0.6013;DB;DP=12;DP4=5,3,3,1;FQ=27.2;MQ=60;PV4=1,0.006,1,1;SNPEFF_AMINO_ACID_CHANGE=D1336N;SNPEFF_CODON_CHANGE=Gac/Aac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UMODL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173568.3;VDB=0.0075;DN=79;DA=G/A;GM=NM_001004416.2,NM_001199527.1,NM_001199528.2,NM_173568.3;GL=UMODL1;FG=missense;FD=missense;AAC=ASP/ASN;PP=1208/1319,1264/1375,1136/1247,1336/1447;CDP=3622,3790,3406,4006;GS=23;PH=0.001;CP=0.924;CG=2.420;AA=A;HA=34.4;HE=36.0;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=69;CPG=43547872-43548089;TFBS=V$ZID_01;GESP=A:4181/G:8217;PAC=NP_001004416.2,NP_001186456.1,NP_001186457.2,NP_775839.3 GT:GQ:PL 0/1:8:4,0,87 0/1:3:0,0,0 1/1:5:47,6,0 0/1:15:11,0,63</p>

      <p>21 46271460 rs2236442 T C 20.70 midCVG AC1=2;AF1=0.3147;DB;DP=9;DP4=4,2,0,3;FQ=22.3;MQ=58;PV4=0.17,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=E84G;SNPEFF_CODON_CHANGE=gAg/gGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PTTG1IP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001286822.1;VDB=0.0020;DN=98;DA=T/C;GM=NM_004339.3,XM_005261175.1,XM_005261176.1,XM_005261177.1;GL=PTTG1IP;FG=3-prime-UTR,missense,missense,3-prime-UTR;FD=utr-variant-3-prime,unknown,unknown,unknown;AAC=none,GLU/GLY,GLU/GLY,none;PP=NA,104/163,84/143,NA;CDP=NA,311,251,NA;GS=NA,98,98,NA;PH=0.914;CP=0.000;CG=0.454;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=83;GESP=C:902/T:12104;PAC=NA,XP_005261232.1,XP_005261233.1,NA GT:GQ:PL 0/0:14:0,12,80 0/0:3:0,0,0 1/1:4:61,9,0 0/0:8:0,6,40</p>

      <p>21 47786524 rs2839227 A G 27.90 midCVG AC1=4;AF1=0.5236;DB;DP=10;DP4=6,0,0,4;FQ=31;MQ=60;PV4=0.0048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T879A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0116;DN=100;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1,XM_005261130.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ALA;PP=879/3337,879/3348,879/3347,879/3269,879/3259,761/3230,761/3140,879/2707;CDP=2635,2635,2635,2635,2635,2281,2281,2635;GS=58;PH=0.264;CP=0.002;CG=1.810;AA=A;HA=38.8;HE=17.7;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26;GESP=G:3367/A:9597;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1,XP_005261187.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,40 1/1:6:57,9,0 0/1:15:12,0,61</p>

      <p>21 47851753 rs2073376 A G 39.40 lowCVG AC1=6;AF1=0.6957;DB;DP=6;DP4=1,1,1,3;FQ=10.9;MQ=60;PV4=1,0.27,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q2792R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0093;DN=96;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=2792/3337,2803/3348,2802/3347,2803/3269,2803/3259,2685/3230,2674/3140;CDP=8375,8408,8405,8408,8408,8054,8021;GS=43;PH=0;CP=0.314;CG=2.320;AA=G;HA=15.0;HE=33.5;HC=31.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=140;GESP=G:8998/A:4008;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:6:34,6,0 1/1:6:47,6,0</p>

      <p>22 21357119 rs71314757 G A 67 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,1,5;FQ=8.97;MQ=60;PV4=1,0.3,1,1;VDB=0.0209;DN=130;DA=G/A;GM=NR_027051.1,NR_027052.1;GL=THAP7-AS1;FG=splice-donor;FD=splice-donor-variant;CP=0.010;CG=-2.790;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=0 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,45 1/1:14:81,15,0 1/1:4:28,3,0</p>

      <p>22 21800806 . T C 11.20 highCVG AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1 GT:GQ:PL 0/0:39:0,36,185 0/0:4:0,0,0 1/1:3:52,9,0 0/0:18:0,15,98</p>

      <p>22 22868841 . A G 36.10 highCVG AC1=3;AF1=0.3333;DP=17;DP4=2,11,0,4;FQ=37.8;MQ=59;PV4=1,0.42,0.11,1;SNPEFF_AMINO_ACID_CHANGE=S372P;SNPEFF_CODON_CHANGE=Tca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF280A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080740.4;VDB=0.0001;GM=NM_080740.3;GL=ZNF280A;FG=missense;FD=unknown;AAC=SER/PRO;PP=372/543;CDP=1114;GS=74;PH=1;CP=0.993;CG=2.860;AA=A;DSP=1185;GESP=A:13006;PAC=NP_542778.1 GT:GQ:PL 0/0:8:0,6,48 0/0:3:0,0,0 1/1:6:78,12,0 0/0:34:0,33,165</p>

      <p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

      <p>22 30888527 rs17738540 C T 33.50 lowCVG AC1=3;AF1=0.4062;DB;DP=8;DP4=2,2,3,1;FQ=34.5;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=V200M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC14L4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_174977.3;VDB=0.0080;DN=123;DA=C/T;GM=NM_001161368.1,NM_174977.3,XM_005261583.1,XM_005261584.1,XM_005261585.1,XM_005261586.1;GL=SEC14L4;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=200/361,200/407,201/408,185/392,146/361,117/324;CDP=598,598,601,553,436,349;GS=21;PH=1;CP=0.855;CG=2.760;AA=C;HA=4.6;HE=24.7;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18;TFBS=V$HNF4_01,V$MIF1_01;GESP=T:2478/C:10528;PAC=NP_001154840.1,NP_777637.1,XP_005261640.1,XP_005261641.1,XP_005261642.1,XP_005261643.1 GT:GQ:PL 0/0:6:0,6,44 0/0:4:0,3,20 1/1:3:43,6,0 0/1:22:33,0,17</p>

      <p>22 32112249 rs117719230 G A 43.30 midCVG AC1=3;AF1=0.4398;DB;DP=12;DP4=5,1,2,2;FQ=45.9;MQ=60;PV4=0.5,0.41,1,0.33;SNPEFF_AMINO_ACID_CHANGE=P526S;SNPEFF_CODON_CHANGE=Cct/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRR14L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173566.2;VDB=0.0050;DN=132;DA=G/A;GM=NM_173566.2,XM_005261480.1;GL=PRR14L;FG=missense;FD=missense,unknown;AAC=PRO/SER;PP=526/2152;CDP=1576;GS=74;PH=0.048;CP=0.020;CG=0.731;AA=G;DV=by-cluster;DSP=1029;GESP=G:4566;PPI=PPP2CA:808|PPP2CB:808|PPP2R1A:808;PAC=NP_775837.2,XP_005261537.1 GT:GQ:PL 0/0:8:0,9,71 0/0:8:0,9,59 1/1:6:69,9,0 0/1:4:21,3,0</p>

      <p>22 32586854 rs136467 C T 6.51 midCVG AC1=3;AF1=0.3919;DB;DP=12;DP4=0,7,3,2;FQ=8.17;MQ=45;PV4=0.045,0.095,0.028,1;SNPEFF_AMINO_ACID_CHANGE=V348I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=C/T;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=VAL/ILE;PP=348/379,258/289,258/289,287/318,258/289;CDP=1042,772,772,859,772;GS=29;PH=0;CP=0.000;CG=-2.480;AA=T;HA=16.7;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=486;GESP=C:12726;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1 GT:GQ:PL 0/0:6:0,3,26 0/1:7:7,0,26 1/1:6:43,12,0 0/0:16:0,15,85</p>

      <p>22 32586875 rs136468 G A 7 midCVG AC1=3;AF1=0.3215;DB;DP=12;DP4=0,7,4,1;FQ=8.3;MQ=41;PV4=0.01,1,0.0013,0.38;SNPEFF_AMINO_ACID_CHANGE=R341C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=G/A;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/CYS;PP=341/379,251/289,251/289,280/318,251/289;CDP=1021,751,751,838,751;GS=180;PH=0.999;CP=0.036;CG=0.582;AA=G;HA=1.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=465;GESP=G:12936;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1 GT:GQ:PL 0/0:6:0,3,27 0/0:5:0,1,22 1/1:5:48,12,0 0/0:17:0,15,80</p>

      <p>22 36585242 rs12781 A G 11.10 lowCVG AC1=4;AF1=0.4427;DB;DP=5;DP4=2,0,0,3;FQ=5.79;MQ=60;PV4=0.1,0.27,1,1;VDB=0.0021;DN=52;DA=A/G;GM=NM_030643.3,NM_145660.1;GL=APOL4;FG=3-prime-UTR;FD=utr-variant-3-prime;CP=0.001;CG=-0.274;AA=G;HA=14.4;HE=44.9;HC=21.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;RM=LTR57;DSP=2717 GT:GQ:PL 0/0:5:0,3,19 0/0:3:0,0,0 1/1:5:49,9,0 0/0:5:0,3,22</p>

      <p>22 36585609 rs5756090 C A 11.10 midCVG AC1=5;AF1=0.5958;DB;DP=9;DP4=4,0,2,1;FQ=11.4;MQ=60;PV4=0.43,1,1,1;DN=114;DA=C/A;GM=NM_030643.3,NM_145660.1;GL=APOL4;FG=3-prime-UTR;FD=utr-variant-3-prime;CP=0.000;CG=-1.380;AA=A;HA=12.1;HE=45.5;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2350 GT:GQ:PL 0/1:13:11,0,16 0/1:3:0,0,0 1/1:3:23,3,0 0/1:18:15,0,32</p>

      <p>22 36585721 rs5756091 T G 27.50 lowCVG AC1=4;AF1=0.5293;DB;DP=7;DP4=3,0,2,2;FQ=7.54;MQ=58;PV4=0.43,1,0.22,0.12;VDB=0.0164;DN=114;DA=T/G;GM=NM_030643.3,NM_145660.1;GL=APOL4;FG=3-prime-UTR;FD=utr-variant-3-prime;CP=0.000;CG=-1.710;AA=G;HA=23.7;HE=46.5;HC=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2238 GT:GQ:PL 0/1:12:10,0,14 0/1:3:0,3,19 1/1:5:40,6,0 0/1:15:20,0,13</p>

      <p>22 36586499 rs5756097 G A 103 lowCVG AC1=6;AF1=0.7546;DB;DP=8;DP4=0,1,5,2;FQ=-17.4;MQ=48;PV4=0.37,0.47,0.15,0.3;VDB=0.0062;DN=114;DA=G/A;GM=NM_030643.3,NM_145660.1;GL=APOL4;FG=3-prime-UTR;FD=utr-variant-3-prime;CP=0.015;DG;DV=by-cluster,by-hapmap,by-1000G;RM=MER11C;DSP=1460 GT:GQ:PL 0/1:3:0,0,0 0/1:14:62,0,14 1/1:12:78,12,0 0/1:3:0,0,0</p>

      <p>22 36587952 rs2227169 C T 153 highCVG AC1=5;AF1=0.5665;DB;DP=22;DP4=5,2,8,7;FQ=64.5;MQ=60;PV4=0.65,0.18,0.25,1;VDB=0.0124;DN=96;DA=C/T;GM=NM_030643.3,NM_145660.1;GL=APOL4;FG=coding-unknown;FD=synonymous-codon;CDP=216,225;CP=0.017;CG=0.035;AA=C;HA=22.5;HE=48.3;HC=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7;GESP=T:6897/C:5563;PAC=NP_085146.2,NP_663693.1 GT:GQ:PL 0/1:29:39,0,27 0/1:4:0,3,29 1/1:31:149,33,0 0/1:14:10,0,50</p>

      <p>22 42486723 rs1801311 G A 45.20 lowCVG AC1=5;AF1=0.6464;DB;DP=6;DP4=0,1,2,2;FQ=-6.55;MQ=60;PV4=1,0.24,1,0.25;SNPEFF_AMINO_ACID_CHANGE=A35V;SNPEFF_CODON_CHANGE=gCt/gTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NDUFA6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002490.3;VDB=0.0099;DN=89;DA=G/A;GM=NM_002490.3;GL=NDUFA6;FG=missense;FD=missense;AAC=ALA/VAL;PP=35/155;CDP=104;GS=64;PH=0.024;CP=0.066;CG=2.390;AA=G;HA=49.1;HE=31.4;HC=18.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=114;KP=hsa00190:OxidativePhosphorylation|hsa01100:MetabolicPathways|hsa05010:Alzheimer'sDisease|hsa05012:Parkinson'sDisease|hsa05016:Huntington'sDisease;CPG=42486350-42487246;GESP=A:5063/G:7943;PPI=NDUFA2:999;PAC=NP_002481.2 GT:GQ:PL 0/1:3:0,3,27 0/1:3:0,0,0 1/1:10:82,12,0 0/1:3:0,0,0</p>

      <p>22 46644168 rs9627281 A G 73.20 highCVG AC1=3;AF1=0.3788;DB;DP=17;DP4=1,9,2,4;FQ=75.2;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V5A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDPF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207327.4;VDB=0.0179;DN=119;DA=A/G;GM=NM_207327.4;GL=CDPF1;FG=missense;FD=missense;AAC=VAL/ALA;PP=5/124;CDP=14;GS=64;PH=0;CP=0.000;CG=0.371;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=14;GESP=G:4606/A:8400;PAC=NP_997210.3 GT:GQ:PL 0/0:12:0,12,87 0/0:14:0,15,85 1/1:10:101,15,0 0/1:15:20,0,11</p>

      <p>22 46655779 rs36125344 G C 91.40 midCVG AC1=5;AF1=0.6118;DB;DP=12;DP4=3,2,1,6;FQ=52.5;MQ=60;PV4=0.22,1,1,0.45;SNPEFF_AMINO_ACID_CHANGE=I1147M;SNPEFF_CODON_CHANGE=atC/atG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PKDREJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006071.1;VDB=0.0056;DN=126;DA=G/C;GM=NM_006071.1,XM_005261302.1;GL=PKDREJ;FG=missense;FD=missense,unknown;AAC=ILE/MET;PP=1147/2254,1147/2163;CDP=3441;GS=10;PH=0.022;CP=0.000;CG=-8.750;AA=G;HA=3.3;HE=15.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=643;GESP=C:1242/G:11764;PAC=NP_006062.1,XP_005261359.1 GT:GQ:PL 0/0:8:0,12,77 0/1:19:19,0,20 1/1:8:65,9,0 1/1:8:56,9,0</p>

      <p>22 50555686 rs760749 A C 999 highCVG AC1=4;AF1=0.5314;DB;DP=31;DP4=5,4,3,11;FQ=93.2;MQ=60;PV4=0.18,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I454L;SNPEFF_CODON_CHANGE=Att/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MOV10L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018995.2;VDB=0.0115;DN=86;DA=A/C;GM=NM_001164104.1,NM_001164105.1,NM_018995.2,XM_005261923.1,XM_005261924.1;GL=MOV10L1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=ILE/LEU;PP=454/1166,434/1166,454/1212,434/1192,213/971;CDP=1360,1300,1360,1300,637;GS=5;PH=0.472;CP=0.048;CG=5.760;AA=A;HA=0.9;HE=14.8;HC=23.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=95;GESP=C:1564/A:11442;PAC=NP_001157576.1,NP_001157577.1,NP_061868.1,XP_005261980.1,XP_005261981.1 GT:GQ:PL 0/1:14:81,0,11 0/1:50:47,0,57 1/1:19:130,21,0 0/0:7:0,9,69</p>

      <p>22 50728062 rs28379706 T C 62.50 midCVG AC1=4;AF1=0.5564;DB;DP=11;DP4=1,2,6,0;FQ=26.1;MQ=60;PV4=0.083,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K318E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012401.3;VDB=0.0214;DN=125;DA=T/C;GM=NM_012401.3,XM_005261908.1,XM_005261909.1,XM_005261910.1,XM_005261911.1;GL=PLXNB2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=318/1839,400/1921,318/1839,318/1839,318/1839;CDP=952,1198,952,952,952;GS=56;PH=0;CP=0.001;CG=-1.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=117;KP=hsa04360:AxonGuidance;CPG=50727442-50728654;GESP=C:5524/T:6698;PPI=TUBA3E:962|MYC:846|ARHGEF11:845;PAC=NP_036533.2,XP_005261965.1,XP_005261966.1,XP_005261967.1,XP_005261968.1 GT:GQ:PL 0/1:3:0,3,24 0/0:5:0,6,44 1/1:3:21,3,0 1/1:13:87,15,0</p>

      <p>22 51009953 rs470117 C T 122 highCVG AC1=4;AF1=0.5;DB;DP=21;DP4=5,3,7,6;FQ=74.8;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=E531K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145135.1;VDB=0.0103;DN=80;DA=C/T;GM=NM_001145134.1,NM_001145135.1,NM_001145136.1,NM_001145137.1,NM_004377.3,NM_152245.2,NM_152246.2,NR_027928.2;GL=CPT1B/CHKB-CPT1B;FG=missense,missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,none;PP=497/739,531/773,450/692,531/773,531/773,531/773,531/773,NA;CDP=1489,1591,1348,1591,1591,1591,1591,3161;GS=56,56,56,56,56,56,56,NA;PH=0.966;CP=0.993;CG=5.310;AA=C;HA=10.1;HE=48.3;HC=47.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway|hsa04920:AdipocytokineSignalingPathway;GESP=T:4624/C:8382;PAC=NP_001138606.1,NP_001138607.1,NP_001138608.1,NP_001138609.1,NP_004368.1,NP_689451.1,NP_689452.1,NA GT:GQ:PL 0/0:8:0,9,56 0/1:62:106,0,58 1/1:6:58,9,0 0/1:3:0,0,0</p>

      <p>22 51045190 rs1140555 C T 26.90 lowCVG AC1=6;AF1=0.7232;DB;DP=5;DP4=1,0,3,0;FQ=-21.3;MQ=60;PV4=1,1,1,0.21;VDB=0.0048;DN=86;DA=C/T;GM=NM_012324.3,NM_016431.3;GL=MAPK8IP2;FG=synonymous;FD=synonymous-codon;PP=743/825,716/798;CDP=2228,2147;CP=0.989;CG=-1.270;AA=C;HC=2.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=4;KP=hsa04010:MAPKSignalingPathway;GESP=T:6729/C:5159;PPI=MAP2K7:846|MAPK13:845|MAPK8IP1:845|LRP2:809;PAC=NP_036456.1,NP_057515.1 GT:GQ:PL 0/1:3:0,0,0 0/1:10:40,0,8 1/1:4:21,3,0 0/1:3:0,0,0</p>

      <p>X 12708329 . C A 66.40 highCVG AC1=2;AF1=0.276;DP=28;DP4=11,3,12,0;FQ=67.8;G3=0.75,7.644e-07,0.25;HWE=0.0436;MQ=60;PV4=0.22,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=L233I;SNPEFF_CODON_CHANGE=Ctt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMPD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014728.3;VDB=0.0000;GM=NM_014728.3,XM_005274632.1;GL=FRMPD4;FG=missense;FD=unknown;AAC=LEU/ILE;PP=233/1323,233/1763;CDP=697;GS=5;PH=0.999;CP=0.982;CG=3.770;AA=C;DSP=16;TFBS=V$CDC5_01;GESP=C:13006;PAC=NP_055543.2,XP_005274689.1 GT:GQ:PL 0/0:13:0,12,66 0/0:25:0,24,77 1/1:29:113,36,0 0/0:8:0,6,47</p>

      <p>X 23693193 . C A 54.30 highCVG AC1=2;AF1=0.2857;DP=20;DP4=6,6,0,7;FQ=55.8;MQ=60;PV4=0.044,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S149Y;SNPEFF_CODON_CHANGE=tCt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRDX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006406.1;VDB=0.0080;GM=NM_006406.1,XM_005274438.1;GL=PRDX4;FG=missense;FD=unknown;AAC=SER/TYR;PP=149/272,135/258;CDP=446,404;GS=144;PH=1;CP=1.000;CG=5.840;DSP=31;GESP=C:13006;PAC=NP_006397.1,XP_005274495.1 GT:GQ:PL 0/0:16:0,15,96 0/0:6:0,3,25 1/1:14:100,21,0 0/0:19:0,18,118</p>

      <p>X 27839617 rs12557898 G A 18.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,0,3;FQ=16.7;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=R65Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGEB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182506.3;VDB=0.0106;DN=120;DA=G/A;GM=NM_182506.3;GL=MAGEB10;FG=missense;FD=missense;AAC=ARG/GLN;PP=65/348;CDP=194;GS=43;PH=0.012;CP=0.000;CG=-5.040;AA=G;HA=14.0;HE=44.2;HC=8.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=243;GESP=A:4054/G:6506;PAC=NP_872312.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:56,9,0 0/0:9:0,9,53</p>

      <p>X 48418659 rs235836 G A 55.30 midCVG AC1=6;AF1=0.6676;DB;DP=9;DP4=2,0,7,0;FQ=-7.74;MQ=60;PV4=1,0.14,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A455T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D25;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002536.2;VDB=0.0091;DN=79;DA=G/A;GM=NM_002536.2;GL=TBC1D25;FG=missense;FD=missense;AAC=ALA/THR;PP=455/689;CDP=1363;GS=58;PH=0;CP=0.015;CG=-7.340;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=658;GESP=A:5802/G:4761;PAC=NP_002527.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:22,3,0 0/1:25:68,0,23</p>

      <p>X 49179719 rs6520418 C G 20.80 midCVG AC1=5;AF1=0.592;DB;DP=14;DP4=0,8,2,4;FQ=24.7;MQ=26;PV4=0.16,0.013,7.5e-06,0.011;SNPEFF_AMINO_ACID_CHANGE=P16R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0167;DN=116;DA=C/G;GM=NM_001098406.1;GL=GAGE12J;FG=missense;FD=missense;AAC=PRO/ARG;PP=16/118;CDP=47;GS=103;PH=0;CP=0.000;CG=0.012;AA=G;DV=by-cluster,by-2hit-2allele;DSP=38;GESP=G:1901/C:4835;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:20:0,24,118 1/1:3:17,3,0 1/1:13:48,15,0</p>

      <p>X 49179755 rs7064530 G A 5.88 midCVG AC1=5;AF1=0.6096;DB;DP=13;DP4=0,6,0,5;FQ=9.09;MQ=25;PV4=1,1,6.2e-05,1;SNPEFF_AMINO_ACID_CHANGE=R28Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0001;DN=116;DA=G/A;GM=NM_001098406.1;GL=GAGE12J;FG=missense-near-splice;FD=missense;AAC=ARG/GLN;PP=28/118;CDP=83;GS=43;PH=0.881;CP=0.000;CG=-1.910;AA=G;DV=by-cluster;DSP=2;GESP=A:2418/G:4398;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:17:0,18,67 1/1:4:19,6,0 1/1:6:30,9,0</p>

      <p>X 88008807 rs5984611 G A 58 lowCVG AC1=6;AF1=0.7071;DB;DP=7;DP4=0,2,1,4;FQ=4.26;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=R131H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPXCR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184771.1;VDB=0.0098;DN=114;DA=G/A;GM=NM_001184771.1,NM_033048.5;GL=CPXCR1;FG=missense;FD=missense;AAC=ARG/HIS;PP=131/302;CDP=392;GS=29;PH=0.995;CP=0.000;CG=0.096;AA=G;HA=16.1;HE=40.7;HC=5.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=400;GESP=A:3478/G:7085;PAC=NP_001171700.1,NP_149037.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:40,6,0 0/1:34:54,0,33</p>

      <p>X 118281545 . A G 25.70 midCVG AC1=4;AF1=0.4454;DP=10;DP4=1,1,3,0;FQ=5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F101L;SNPEFF_CODON_CHANGE=Ttt/Ctt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1210;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020721.1;VDB=0.0000;GM=NM_020721.1;GL=KIAA1210;FG=missense;FD=unknown;AAC=PHE/LEU;PP=101/1710;CDP=301;GS=22;PH=0.6;CP=0.000;CG=-2.860;AA=A;DSP=12;GESP=A:12338;PAC=NP_065772.1 GT:GQ:PL 0/0:4:0,3,25 0/1:3:0,0,0 1/1:5:64,9,0 0/0:4:0,3,15</p>

      <p>X 120183030 rs9697983 T G 66.90 midCVG AC1=3;AF1=0.4074;DB;DP=11;DP4=5,0,5,1;FQ=62.1;MQ=60;PV4=1,0.25,1,0.26;SNPEFF_AMINO_ACID_CHANGE=S498A;SNPEFF_CODON_CHANGE=Tcg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GLUD2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012084.3;VDB=0.0053;DN=119;DA=T/G;GM=NM_012084.3;GL=GLUD2;FG=missense;FD=missense;AAC=SER/ALA;PP=498/559;CDP=1492;GS=99;PH=0;CP=0.092;CG=0.710;AA=T;HA=4.9;HE=3.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=64389;KP=hsa00250:Alanine,AspartateAndGlutamateMetabolism|hsa00330:ArginineAndProlineMetabolism|hsa00471:D-GlutamineAndD-glutamateMetabolism|hsa00910:NitrogenMetabolism|hsa01100:MetabolicPathways|hsa04964:ProximalTubuleBicarbonateReclamation;GESP=G:390/T:10173;PAC=NP_036216.2 GT:GQ:PL 0/1:3:0,0,0 0/0:12:0,12,76 1/1:13:109,18,0 0/0:4:0,3,27</p>

      <p>X 128691863 . G T 18.30 lowCVG AC1=5;AF1=0.6661;DP=7;DP4=0,3,0,4;FQ=6.16;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=E125D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OCRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000276.3;VDB=0.0039;GM=NM_000276.3,NM_001587.3,XM_005262421.1,XM_005262422.1;GL=OCRL;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=GLU/ASP,GLU/ASP,GLU/ASP,none;PP=125/902,125/894,59/836,NA;CDP=375,375,177,NA;GS=45,45,45,NA;PH=0.001;CP=0.999;CG=0.803;AA=G;DSP=26;KP=hsa00562:InositolPhosphateMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=G:13006;PPI=RAB8A:959|CLTC:937;PAC=NP_000267.2,NP_001578.2,XP_005262478.1,NA GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:24,3,0 0/1:32:29,0,35</p>

      <p>X 148038075 . A C 11.70 midCVG AC1=5;AF1=0.5727;DP=14;DP4=3,1,0,3;FQ=15.3;MQ=60;PV4=0.14,0.022,1,1;SNPEFF_AMINO_ACID_CHANGE=K834Q;SNPEFF_CODON_CHANGE=Aaa/Caa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AFF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002025.3;VDB=0.0005;GM=NM_001169122.1,NM_001169123.1,NM_001169124.1,NM_001169125.1,NM_001170628.1,NM_002025.3;GL=AFF2;FG=missense;FD=unknown;AAC=LYS/GLN;PP=801/1277,824/1302,799/1277,795/1273,475/953,834/1312;CDP=2401,2470,2395,2383,1423,2500;GS=53;PH=1;CP=0.878;CG=5.870;AA=A;DSP=69;TFBS=V$PAX6_01;GESP=A:13006;PAC=NP_001162593.1,NP_001162594.1,NP_001162595.1,NP_001162596.1,NP_001164099.1,NP_002016.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:18,3,0 1/1:4:35,6,0 0/0:10:0,12,77</p>

      <p>X 150349871 rs13440581 A G 56.20 highCVG AC1=3;AF1=0.3587;DB;DP=15;DP4=5,4,4,2;FQ=58.1;MQ=60;PV4=1,0.31,1,0.021;SNPEFF_AMINO_ACID_CHANGE=I606V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR50;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004224.3;VDB=0.0088;DN=121;DA=A/G;GM=NM_004224.3;GL=GPR50;FG=missense;FD=missense;AAC=ILE/VAL;PP=606/618;CDP=1816;GS=29;PH=0.01;CP=0.002;CG=2.130;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;CA;DSP=969;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4866/A:5438;PAC=NP_004215.2 GT:GQ:PL 0/0:9:0,9,73 0/0:7:0,6,42 1/1:3:43,6,0 0/1:55:57,0,53</p>

      <p>X 153039502 rs6643791 G C 15.60 lowCVG AC1=6;AF1=0.683;DB;DP=5;DP4=0,2,3,0;FQ=-16.8;MQ=60;PV4=0.1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=E1179D;SNPEFF_CODON_CHANGE=gaG/gaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163257.1;VDB=0.0123;DN=116;DA=G/C;GM=NM_001163257.1,NM_005393.2,XM_005274694.1,XM_005274695.1,XM_005274696.1;GL=PLXNB3;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=GLU/ASP;PP=1179/1933,1156/1910,1203/1957,1203/1933,809/1563;CDP=3537,3468,3609,3609,2427;GS=45;PH=0.005;CP=0.008;CG=-0.654;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=52;KP=hsa04360:AxonGuidance;GESP=C:7729/G:2554;PPI=ARHGEF11:895;PAC=NP_001156729.1,NP_005384.2,XP_005274751.1,XP_005274752.1,XP_005274753.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:27,3,0 0/1:15:23,0,13</p>

      <p>X 153284192 rs1059702 A G 86.90 midCVG AC1=5;AF1=0.64;DB;DP=10;DP4=2,0,5,3;FQ=-11.8;MQ=60;PV4=1,0.45,1,0.18;SNPEFF_AMINO_ACID_CHANGE=F196S;SNPEFF_CODON_CHANGE=tTt/tCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IRAK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001569.3;VDB=0.0123;DN=86;DA=A/G;GM=NM_001025242.1,NM_001025243.1,NM_001569.3,XM_005274668.1;GL=IRAK1;FG=missense;FD=missense,missense,missense,unknown;AAC=PHE/SER;PP=196/683,196/634,196/713,222/709;CDP=587,587,587,665;GS=155;PH=0.002;CP=0.002;CG=3.680;AA=G;HA=1.8;HE=20.5;HC=19.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=47;KP=hsa04210:Apoptosis|hsa04620:Toll-likeReceptorSignalingPathway|hsa04722:NeurotrophinSignalingPathway|hsa05140:Leishmaniasis|hs</p>

        <p>
          <strong>IDs and variants associated:</strong> H00293;DSC2|DSG2|DSP|JUP|PKP2|RYR2|TGFB3|TMEM43;hsa05412
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 10
        </p>
        <p>
          <strong>Missense Mutations:</strong> 624
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.724
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse52">Dilated cardiomyopathy (DCM)</a>
      </h4>
    </div>
    <div id="collapse52" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

<p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>6 152443744 rs2295190 G T 29 lowCVG AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:34,6,0 0/1:24:31,0,23</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>9 117846580 rs1061494 T C 25.30 midCVG AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1 GT:GQ:PL 0/1:22:18,0,120 0/1:3:0,0,0 1/1:6:35,6,0 0/1:15:12,0,20</p>

<p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

        <p>
          <strong>IDs and variants associated:</strong> H00294;ABCC9|LDB3|DES|DMD|EYA4|FKTN|RBM20|DPA1|DPB1|DQA1|DQB1|DRB1|LMNA|MYBPC3|MYH6|MYH7|PLN|PSEN1|PSEN2|SCN5A|SDHA|SGCD|TAZ|ACTC1|TMPO|TNNC1|TNNI3|TNNT2|TPM1|TTN|VCL|CSRP3|TCAP|ACTN2|NEXN;hsa05414
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 8
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.8
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse53">Leprosy; Hansen disease</a>
      </h4>
    </div>
    <div id="collapse53" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          4	38774889	rs4129009	T	C	35.10	midCVG	AC1=3;AF1=0.3337;DB;DP=9;DP4=1,3,3,1;FQ=36.8;MQ=60;PV4=0.49,0.0012,1,0.18;SNPEFF_AMINO_ACID_CHANGE=I775V;SNPEFF_CODON_CHANGE=Att/Gtt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TLR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001017388.2;VDB=0.0028;DN=108;DA=T/A/C/G;GM=NM_001017388.2,NM_001195106.1,NM_001195107.1,NM_001195108.1,NM_030956.3,XM_005262671.1;GL=TLR10;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=ILE/VAL;PP=775/812,775/812,775/812,761/798,775/812,761/798;CDP=2323,2323,2323,2281,2323,2281;GS=29;PH=0;CP=0.010;CG=-5.340;AA=T;HE=20.1;HC=43.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=2288;GESP=C:1621/T:11385;PAC=NP_001017388.1,NP_001182035.1,NP_001182036.1,NP_001182037.1,NP_112218.2,XP_005262728.1	GT:GQ:PL	0/0:8:0,6,48	0/0:5:0,3,26	1/1:6:77,12,0	0/0:5:0,3,26

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00344;LTA|PARK2|TLR1|TLR2;hsa04650,hsa04620,hsa04612,hsa04060
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse54">Disorders of nucleotide excision repair, including: Xeroderma pigmentosum (XP); Cockayne syndrome</a>
      </h4>
    </div>
    <div id="collapse54" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          2	100055158	rs3087399	T	C	126	highCVG	AC1=5;AF1=0.5732;DB;DP=19;DP4=8,3,4,4;FQ=128;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N373S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=REV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016316.2;VDB=0.0092;DN=102;DA=T/C;GM=NM_001037872.1,NM_016316.2,XM_005263967.1,XM_005263968.1,XM_005263969.1;GL=REV1;FG=missense,missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown,unknown;AAC=ASN/SER,ASN/SER,ASN/SER,ASN/SER,none;PP=373/1251,373/1252,373/1252,303/1182,NA;CDP=1118,1118,1118,908,NA;GS=46,46,46,46,NA;PH=0.069;CP=0.794;CG=-0.648;AA=T;HA=22.9;HE=10.8;HC=3.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=96;GESP=C:2153/T:10853;PPI=MAD2L2:999|REV3L:998|POLH:995|POLI:846|POLK:846|UBC:845;PAC=NP_001032961.1,NP_057400.1,XP_005264024.1,XP_005264025.1,NA	GT:GQ:PL	0/1:19:54,0,17	0/0:26:0,30,163	1/1:8:72,9,0	1/1:5:48,6,0

        </p>
        <p>
          5	86695274	rs2230641	A	G	50.70	midCVG	AC1=4;AF1=0.4542;DB;DP=9;DP4=1,4,2,2;FQ=51.2;MQ=60;PV4=0.52,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=V270A;SNPEFF_CODON_CHANGE=gTt/gCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCNH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001239.3;VDB=0.0061;DN=98;DA=A/G/T;GM=NM_001199189.1,NM_001239.3,XM_005248627.1,XM_005248628.1,XM_005248629.1;GL=CCNH;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=VAL/ALA;PP=217/271,270/324,270/330,270/326,217/277;CDP=650,809,809,809,650;GS=64;PH=0.965;CP=1.000;CG=5.680;AA=A;HA=5.0;HE=20.3;HC=8.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;KP=hsa03420:NucleotideExcisionRepair|hsa04110:CellCycle;GESP=G:2278/A:10728;PPI=CDK7:999|ERCC3:997|GTF2H1:989|CDK2:989|MNAT1:905|CDK8:846|ERCC2:845;PAC=NP_001186118.1,NP_001230.1,XP_005248684.1,XP_005248685.1,XP_005248686.1	GT:GQ:PL	0/0:11:0,12,76	0/1:3:0,0,0	1/1:4:48,6,0	0/1:22:44,0,18

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00403;ERCC8|DDB1|DDB2|ERCC1|ERCC2|ERCC3|ERCC4|ERCC5|ERCC6|GTF2H5|POLH|XPA|XPC;hsa03420
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse55">Type I diabetes mellitus</a>
      </h4>
    </div>
    <div id="collapse55" class="panel-collapse collapse">
      <div class="panel-body">
        <p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>7 77789554 . G A 51.40 highCVG AC1=2;AF1=0.2978;DP=17;DP4=3,6,8,0;FQ=53;MQ=60;PV4=0.009,0.27,1,0.052;SNPEFF_AMINO_ACID_CHANGE=S878F;SNPEFF_CODON_CHANGE=tCt/tTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGI2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012301.3;VDB=0.0000;GM=NM_012301.3,XM_005250725.1;GL=MAGI2;FG=missense;FD=unknown;AAC=SER/PHE;PP=878/1456,864/1442;CDP=2633,2591;GS=155;PH=0.996;CP=0.999;CG=5.510;AA=G;DSP=37;KP=hsa04530:TightJunction;GESP=G:13006;PPI=ADRB1:846|ACVR2A:845;PAC=NP_036433.2,XP_005250782.1 GT:GQ:PL 0/0:13:0,12,79 0/0:13:0,12,69 1/1:17:97,24,0 0/0:5:0,3,25</p>

<p>8 32453358 rs3924999 G A 31.20 lowCVG AC1=6;AF1=0.7171;DB;DP=6;DP4=0,1,2,1;FQ=-15.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R38Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013956.3;VDB=0.0076;DN=108;DA=G/A;GM=NM_001159995.1,NM_001159999.1,NM_001160001.1,NM_001160002.1,NM_001160004.1,NM_001160005.1,NM_001160007.1,NM_001160008.1,NM_004495.3,NM_013956.3,NM_013957.3,NM_013958.3,NM_013960.3,NM_013962.2,NM_013964.3,XM_005273488.1;GL=NRG1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,unknown;AAC=ARG/GLN;PP=17/608,17/625,17/591,38/195,38/460,38/208,38/178,38/421,38/212,38/646,38/638,38/242,38/463,253/423,38/641,45/636;CDP=50,50,50,113,113,113,113,113,113,113,113,113,113,758,113,134;GS=43;PH=0.966;CP=0.998;CG=5.650;AA=G;HA=1.3;HE=43.2;HC=21.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=13;KP=hsa04012:ErbBSignalingPathway;GESP=A:3990/G:9016;PPI=ERBB3:989|ERBB4:983;PAC=NP_001153467.1,NP_001153471.1,NP_001153473.1,NP_001153474.1,NP_001153476.1,NP_001153477.1,NP_001153479.1,NP_001153480.1,NP_004486.2,NP_039250.2,NP_039251.2,NP_039252.2,NP_039254.1,NP_039256.2,NP_039258.1,XP_005273545.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,18 1/1:4:20,3,0 1/1:6:50,6,0</p>

<p>19 55281315 rs2304224 G T 78.20 lowCVG AC1=6;AF1=0.7291;DB;DP=8;DP4=1,1,5,1;FQ=-4.87;MQ=54;PV4=0.46,1,0.21,0.3;SNPEFF_AMINO_ACID_CHANGE=V5F;SNPEFF_CODON_CHANGE=Gtc/Ttc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIR2DL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014218.2;VDB=0.0061;DN=100;DA=G/T;GM=NM_014218.2,XM_005258889.1;GL=KIR2DL1;FG=missense;FD=missense,unknown;AAC=VAL/PHE;PP=5/349,5/375;CDP=13;GS=50;PH=0.754;CP=0.011;CG=-1.030;AA=G;HA=4.2;HE=23.3;HC=14.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa04612:AntigenProcessingAndPresentation|hsa04650:NaturalKillerCellMediatedCytotoxicity|hsa05332:Graft-versus-hostDisease;GESP=T:3077/G:9625;PPI=HLA-C:995|B2M:900;PAC=NP_055033.2,XP_005258946.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:7:45,6,0 1/1:12:76,12,0</p>


        <p>
          <strong>IDs and variants associated:</strong> H00408;SH2B3|CD226|1QTNF6|CTLA4|CTSH|CYP27B1|ERBB3|PTPN22|HLA|DQA1|HLA|DQB1|HLA|DRB1|IL2|IL2RA|INS|ITPR3|PRKCQ|PTPN2|IL21|BACH2|IFIH1|NAA25;hsa04940,hsa04722,hsa04660,hsa04630,hsa04060
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse56">Type II diabetes mellitus</a>
      </h4>
    </div>
    <div id="collapse56" class="panel-collapse collapse">
      <div class="panel-body">

        <p>1 156642841 . G T 17.30 lowCVG AC1=3;AF1=0.3522;DP=7;DP4=4,0,3,0;FQ=17.7;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=A380D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NES;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006617.1;VDB=0.0007;GM=NM_006617.1;GL=NES;FG=missense;FD=unknown;AAC=ALA/ASP;PP=380/1622;CDP=1139;GS=126;PH=0.94;CP=0.515;CG=3.090;AA=G;DSP=157;TFBS=V$HOX13_01;GESP=G:13006;PPI=CDK5:937|CDK5R1:845;PAC=NP_006608.1 GT:GQ:PL 0/0:8:0,6,32 0/0:8:0,6,29 1/1:4:57,9,0 0/0:3:0,0,0</p>

<p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

<p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

<p>5 178540975 rs1054480 G A 49.80 lowCVG AC1=6;AF1=0.6833;DB;DP=6;DP4=2,0,1,3;FQ=-5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1177S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADAMTS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014244.4;VDB=0.0112;DN=86;DA=G/A;GM=NM_014244.4;GL=ADAMTS2;FG=missense;FD=missense;AAC=PRO/SER;PP=1177/1212;CDP=3529;GS=74;PH=0.651;CP=0.973;CG=5.050;AA=G;HA=2.7;HE=25.0;HC=32.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=351;GESP=A:2884/G:10122;PAC=NP_055059.2 GT:GQ:PL 0/0:4:0,6,32 0/1:3:0,0,0 1/1:4:26,3,0 1/1:8:65,9,0</p>

<p>8 39362859 rs62510500 C T 82.70 highCVG AC1=4;AF1=0.5274;DB;DP=16;DP4=5,4,5,2;FQ=86;MQ=60;PV4=0.63,0.077,1,1;VDB=0.0134;DN=129;DA=C/T;GM=NR_001569.3,NR_024106.2,NR_024107.2,NR_073423.1;GL=ADAM3A;FG=splice-donor;FD=splice-donor-variant;CP=0.704;CG=3.850;DG;DV=by-frequency,by-cluster,by-1000G;DSP=0 GT:GQ:PL 0/1:29:26,0,105 0/1:3:0,0,0 1/1:12:100,15,0 0/0:7:0,9,72</p>

<p>9 123170733 rs4837768 C G 142 highCVG AC1=6;AF1=0.7872;DB;DP=17;DP4=3,0,5,5;FQ=-19.3;MQ=60;PV4=0.23,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=V1540L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDK5RAP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018249.5;VDB=0.0119;DN=111;DA=C/G;GM=NM_001011649.2,NM_001272039.1,NM_018249.5,NR_073554.1,NR_073555.1,NR_073556.1,NR_073557.1,NR_073558.1;GL=CDK5RAP2;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant,nc-transcript-variant;AAC=VAL/LEU,VAL/LEU,VAL/LEU,none,none,none,none,none;PP=1540/1815,1310/1664,1540/1894,NA,NA,NA,NA,NA;CDP=4618,3928,4618,4887,4810,5017,4890,4887;GS=32,32,32,NA,NA,NA,NA,NA;PH=0.105;CP=0.112;CG=-0.555;AA=C;HA=41.5;HE=27.5;HC=20.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:9047/C:3959;PPI=CDK5R1:845;PAC=NP_001011649.1,NP_001258968.1,NP_060719.4,NA,NA,NA,NA,NA GT:GQ:PL 0/1:4:27,0,4 0/1:10:31,0,12 1/1:6:21,3,0 1/1:17:105,15,0</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

<p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

<p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

        <p>
          <strong>IDs and variants associated:</strong> H00409;CDKN2A|CDKN2B|IGF2BP2|CAPN10|SLC30A8|JAZF1|HHEX|KCNJ11|KCNQ1|MTNR1B|NOTCH2|ENPP1|PPARG|CDKAL1|CDK5|CDK5|ADAMTS9|ADAM|HNF1B|TCF7L2|WFS1|FTO;hsa04950,hsa04930,hsa04350,hsa04330,hsa04310,hsa04115,hsa04110,hsa03320
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 10
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 1.0
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse57">Maturity onset diabetes of the young (MODY)</a>
      </h4>
    </div>
    <div id="collapse57" class="panel-collapse collapse">
      <div class="panel-body">

        <p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>5 42719239 rs6180 A C 98.30 midCVG AC1=6;AF1=0.705;DB;DP=10;DP4=0,3,0,7;FQ=25.8;MQ=60;PV4=1,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=I551L;SNPEFF_CODON_CHANGE=Atc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GHR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242399.2;VDB=0.0032;DN=52;DA=A/C;GM=NM_000163.4,NM_001242399.2,NM_001242400.2,NM_001242401.3,NM_001242402.2,NM_001242403.2,NM_001242404.2,NM_001242405.2,NM_001242406.2,NM_001242460.1,NM_001242462.1,XM_005248288.1;GL=GHR;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,3-prime-UTR,missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,utr-variant-3-prime,unknown;AAC=ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,none,ILE/LEU;PP=544/639,551/646,544/639,544/639,544/639,544/639,544/639,544/639,544/639,522/617,NA,522/617;CDP=1630,1651,1630,1630,1630,1630,1630,1630,1630,1564,NA,1564;GS=5,5,5,5,5,5,5,5,5,5,NA,5;PH=0.933;CP=0.951;CG=3.430;AA=A;HA=41.6;HE=46.2;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=685;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04630:Jak-STATSignalingPathway;TFBS=V$PAX4_04;GESP=C:5675/A:7331;PPI=CSHL1:998|SOCS2:937|JAK2:937|PTPN11:846|UBC:846|BTRC:846|NCOA6:845|SHC1:845|CISH:845;PAC=NP_000154.1,NP_001229328.1,NP_001229329.1,NP_001229330.1,NP_001229331.1,NP_001229332.1,NP_001229333.1,NP_001229334.1,NP_001229335.1,NP_001229389.1,NA,XP_005248345.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:12:83,12,0 1/1:9:60,9,0</p>

<p>10 115529576 rs41292632 T A 52.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=1,1,4,1;FQ=13.3;MQ=60;PV4=1,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=F151I;SNPEFF_CODON_CHANGE=Ttc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLEKHS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182601.1;VDB=0.0055;DN=127;DA=T/A;GM=NM_001193434.1,NM_001193435.1,NM_024889.4,NM_182601.1,XM_005270162.1,XM_005270163.1;GL=PLEKHS1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown;AAC=PHE/ILE;PP=69/367,69/367,157/364,151/466,157/469,157/455;CDP=205,205,469,451,469,469;GS=21;PH=0.893;CP=0.009;CG=4.760;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=34;TFBS=V$PAX4_04;GESP=A:1962/T:11044;PAC=NP_001180363.1,NP_001180364.1,NP_079165.3,NP_872407.1,XP_005270219.1,XP_005270220.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,50 1/1:6:35,6,0 1/1:9:60,9,0</p>

<p>12 49952449 . C T 14.40 lowCVG AC1=4;AF1=0.5125;DP=6;DP4=3,0,0,3;FQ=12.6;MQ=60;PV4=0.1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A469T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCRS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001012300.1;VDB=0.0104;GM=NM_001012300.1,NM_001278341.1,NM_006337.4,XM_005268572.1,XM_005268573.1;GL=MCRS1;FG=missense;FD=unknown;AAC=ALA/THR;PP=469/476,265/272,456/463,456/463,367/374;CDP=1405,793,1366,1366,1099;GS=58;PH=0.997;CP=1.000;CG=5.280;AA=C;DSP=64;TFBS=V$PAX4_01;GESP=C:13006;PPI=INO80E:892|DAXX:846|INO80:811|PBX2:786;PAC=NP_001012300.1,NP_001265270.1,NP_006328.2,XP_005268629.1,XP_005268630.1 GT:GQ:PL 0/1:7:5,0,32 0/1:3:0,0,0 1/1:4:47,6,0 0/0:4:0,3,17</p>

<p>16 2905689 rs199712440 G A 23.60 lowCVG AC1=3;AF1=0.4317;DB;DP=7;DP4=2,1,2,1;FQ=25.1;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R149C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022119.3;VDB=0.0096;DN=137;DA=G/A;GM=NM_022119.3,XM_005255473.1;GL=PRSS22;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=149/318;CDP=445;GS=180;PH=0.99;CP=0.006;CG=3.430;AA=G;DV=by-cluster;DSP=115;TFBS=V$PAX4_01;GESP=A:1/G:12995;PAC=NP_071402.1,XP_005255530.1 GT:GQ:PL 0/0:4:0,3,24 0/0:4:0,3,29 1/1:4:50,6,0 0/1:16:15,0,19</p>
        <p>
          <strong>IDs and variants associated:</strong> H00410;CEL|GCK|HNF4A|INS|PDX1|NEUROD1|PAX4|BLK|HNF1A|HNF1B|KLF11;hsa04950,hsa04930,hsa04910
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 5
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.5
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse58">Familial partial lipodystrophy (FPL)</a>
      </h4>
    </div>
    <div id="collapse58" class="panel-collapse collapse">
      <div class="panel-body">

        <p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

<p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

<p>6 152443744 rs2295190 G T 29 lowCVG AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:34,6,0 0/1:24:31,0,23</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

<p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

<p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

        <p>
          <strong>IDs and variants associated:</strong> H00420;AKT2|LMNA|PPARG|LMNB2;hsa04910,hsa03320
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 8
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.8
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse59">Mitochondrial DNA depletion syndrome (MDS)</a>
      </h4>
    </div>
    <div id="collapse59" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          2	37105041	rs77353404	G	A	36.60	highCVG	AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1	GT:GQ:PL	0/0:7:0,6,42	0/0:7:0,6,45	1/1:3:31,6,0	0/1:49:49,0,68

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00469;DGUOK|TYMP|MPV17|RRM2B|POLG|TK2|SUCLG1|SUCLA2;hsa00240,hsa00230,hsa00020
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse60">Mitochondrial respiratory chain deficiencies (MRCD)</a>
      </h4>
    </div>
    <div id="collapse60" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          8	104412688	rs141856398	T	C	58.40	highCVG	AC1=3;AF1=0.4045;DB;DP=19;DP4=7,3,4,3;FQ=60.7;MQ=60;PV4=0.64,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=Y300C;SNPEFF_CODON_CHANGE=tAt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A32;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030780.4;VDB=0.0120;DN=134;DA=T/C;GM=NM_030780.4,NR_102337.1,NR_102338.1;GL=SLC25A32;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=TYR/CYS,none,none;PP=300/316,NA,NA;CDP=899,1211,1406;GS=194,NA,NA;PH=1;CP=0.997;CG=5.660;AA=T;DV=by-frequency,by-cluster;DSP=87;TFBS=V$OCT1_01;GESP=C:7/T:12999;PAC=NP_110407.2,NA,NA	GT:GQ:PL	0/0:4:0,3,24	0/0:6:0,6,40	1/1:3:38,6,0	0/1:64:63,0,86

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00473;LRPPRC|AFG3L2|NDUFA11|COX6B1|COX15|NDUFAF6|FASTKD2|NDUFAF3|HIBCH|UQCRQ|ACAD9|NDUFAF4|HADHA|HADH|NDUFS7|TRNL1|NDUFA1|NDUFS1|NDUFS2|NDUFS3|NDUFV1|NDUFS4|NDUFS6|NDUFS8|NDUFV2|COA5|OPA1|SLC25A3|TTC19|TMEM70|MTPAP|FOXRED1|TRMU|BCS1L|SDHA|SDHAF1|SURF1|UQCRB|NDUFAF5|NUBPL|ATPAF2|NDUFAF2;hsa05016,hsa05012,hsa05010,hsa00280,hsa00190
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse61">Non-syndromic X-linked mental retardation</a>
      </h4>
    </div>
    <div id="collapse61" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          X	148038075	.	A	C	11.70	midCVG	AC1=5;AF1=0.5727;DP=14;DP4=3,1,0,3;FQ=15.3;MQ=60;PV4=0.14,0.022,1,1;SNPEFF_AMINO_ACID_CHANGE=K834Q;SNPEFF_CODON_CHANGE=Aaa/Caa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AFF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002025.3;VDB=0.0005;GM=NM_001169122.1,NM_001169123.1,NM_001169124.1,NM_001169125.1,NM_001170628.1,NM_002025.3;GL=AFF2;FG=missense;FD=unknown;AAC=LYS/GLN;PP=801/1277,824/1302,799/1277,795/1273,475/953,834/1312;CDP=2401,2470,2395,2383,1423,2500;GS=53;PH=1;CP=0.878;CG=5.870;AA=A;DSP=69;TFBS=V$PAX6_01;GESP=A:13006;PAC=NP_001162593.1,NP_001162594.1,NP_001162595.1,NP_001162596.1,NP_001164099.1,NP_002016.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:18,3,0	1/1:4:35,6,0	0/0:10:0,12,77

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00480;PQBP1|IL1RAPL1|ARX|DLG3|AGTR2|ACSL4|AFF2|FTSJ1|GDI1|HSD17B10|ZNF81|PAK3|ATRX|NLGN4X|RPS6KA3|SLC6A8|TSPAN7|ZNF41|MAGT1|ARHGEF6;hsa04810
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse62">Cone-rod dystrophy and cone dystrophy</a>
      </h4>
    </div>
    <div id="collapse62" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          14	21770730	rs6571751	A	G	150	midCVG	AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3	GT:GQ:PL	0/1:18:39,0,18	0/1:21:83,0,21	1/1:7:49,6,0	1/1:5:22,3,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00481;CRX|KCNV2|RIMS1|ABCA4|GUCA1A|GUCY2D|PDE6C|PDE6H|RPGRIP1|RPGR|SEMA4A|CACNA1F|PITPNM3|RAX2|ADAM9|PROM1|CDHR1|CACNA2D4;hsa04744,hsa00230
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse63">3M syndrome</a>
      </h4>
    </div>
    <div id="collapse63" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          6	43011188	.	G	T	66.80	highCVG	AC1=2;AF1=0.3167;DP=22;DP4=7,1,10,0;FQ=68.4;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1202H;SNPEFF_CODON_CHANGE=cCc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CUL7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001168370.1;VDB=0.0000;GM=NM_001168370.1,NM_014780.4,XM_005249503.1;GL=CUL7;FG=missense;FD=unknown;AAC=PRO/HIS;PP=1202/1783,1118/1699,1170/1751;CDP=3605,3353,3509;GS=77;PH=0.99;CP=1.000;CG=5.520;AA=G;DSP=3;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=G:13004;PPI=FBXW8:994|RBX1:971|TP53:937|SKP1:826;PAC=NP_001161842.1,NP_055595.2,XP_005249560.1	GT:GQ:PL	0/0:7:0,6,35	0/0:5:0,3,19	1/1:23:112,30,0	0/0:16:0,15,97

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00509;OBSL1|CUL7;hsa04120
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse64">Noonan syndrome and related disorders</a>
      </h4>
    </div>
    <div id="collapse64" class="panel-collapse collapse">
      <div class="panel-body">
        <p>3 111901099 rs74840030 T C 15.40 midCVG AC1=3;AF1=0.3387;DB;DP=11;DP4=4,3,2,2;FQ=17;MQ=60;PV4=1,0.00097,1,0.41;SNPEFF_AMINO_ACID_CHANGE=M844V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC9C1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_183061.1;VDB=0.0154;DN=132;DA=T/C;GM=NM_183061.1,XM_005247381.1;GL=SLC9C1;FG=missense;FD=missense,unknown;AAC=MET/VAL;PP=844/1178,796/1130;CDP=2530,2386;GS=21;PH=0.39;CP=1.000;CG=4.590;AA=T;DV=by-frequency,by-cluster;DSP=7;TFBS=V$HNF1_01;GESP=C:495/T:12501;PAC=NP_898884.1,XP_005247438.1 GT:GQ:PL 0/0:10:0,9,74 0/1:5:4,0,40 1/1:4:54,9,0 0/0:8:0,6,51</p>

<p>4 128996129 . G T 28.60 midCVG AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1 GT:GQ:PL 0/0:8:0,6,51 0/0:6:0,3,17 1/1:6:71,12,0 0/0:8:0,6,45</p>

<p>5 42719239 rs6180 A C 98.30 midCVG AC1=6;AF1=0.705;DB;DP=10;DP4=0,3,0,7;FQ=25.8;MQ=60;PV4=1,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=I551L;SNPEFF_CODON_CHANGE=Atc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GHR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001242399.2;VDB=0.0032;DN=52;DA=A/C;GM=NM_000163.4,NM_001242399.2,NM_001242400.2,NM_001242401.3,NM_001242402.2,NM_001242403.2,NM_001242404.2,NM_001242405.2,NM_001242406.2,NM_001242460.1,NM_001242462.1,XM_005248288.1;GL=GHR;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,3-prime-UTR,missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,utr-variant-3-prime,unknown;AAC=ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,ILE/LEU,none,ILE/LEU;PP=544/639,551/646,544/639,544/639,544/639,544/639,544/639,544/639,544/639,522/617,NA,522/617;CDP=1630,1651,1630,1630,1630,1630,1630,1630,1630,1564,NA,1564;GS=5,5,5,5,5,5,5,5,5,5,NA,5;PH=0.933;CP=0.951;CG=3.430;AA=A;HA=41.6;HE=46.2;HC=39.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=685;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04630:Jak-STATSignalingPathway;TFBS=V$PAX4_04;GESP=C:5675/A:7331;PPI=CSHL1:998|SOCS2:937|JAK2:937|PTPN11:846|UBC:846|BTRC:846|NCOA6:845|SHC1:845|CISH:845;PAC=NP_000154.1,NP_001229328.1,NP_001229329.1,NP_001229330.1,NP_001229331.1,NP_001229332.1,NP_001229333.1,NP_001229334.1,NP_001229335.1,NP_001229389.1,NA,XP_005248345.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,9,65 1/1:12:83,12,0 1/1:9:60,9,0</p>

<p>5 127685135 rs154001 C T 84.40 midCVG AC1=2;AF1=0.3246;DB;DP=13;DP4=3,3,3,4;FQ=86;MQ=60;PV4=1,0.09,1,1;SNPEFF_AMINO_ACID_CHANGE=V965I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001999.3;VDB=0.0006;DN=79;DA=C/T;GM=NM_001999.3;GL=FBN2;FG=missense;FD=missense;AAC=VAL/ILE;PP=965/2913;CDP=2893;GS=29;PH=0.118;CP=0.989;CG=3.320;AA=C;HA=40.8;HE=31.3;HC=11.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=30;TFBS=V$ATF_01,V$CEBP_C,V$MYOGNF1_01;GESP=T:8612/C:4394;PAC=NP_001990.2 GT:GQ:PL 0/0:10:0,9,72 0/0:5:0,3,23 1/1:15:129,21,0 0/0:7:0,6,52</p>

<p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

<p>10 70641860 rs1341667 T C 76 midCVG AC1=5;AF1=0.6593;DB;DP=10;DP4=1,2,3,4;FQ=26.9;MQ=60;PV4=1,0.21,1,0.26;SNPEFF_AMINO_ACID_CHANGE=Y153H;SNPEFF_CODON_CHANGE=Tac/Cac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STOX1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001130161.2;VDB=0.0014;DN=88;DA=T/C;GM=NM_001130159.2,NM_001130160.2,NM_001130161.2,NM_001130162.2,NM_152709.4,XM_005269605.1;GL=STOX1;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=TYR/HIS;PP=153/228,153/170,153/990,43/880,153/990,43/880;CDP=457,457,457,127,457,127;GS=83;PH=0.687;CP=0.990;CG=4.440;AA=T;HA=33.8;HE=38.1;HC=11.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=7;TFBS=V$HNF1_C;GESP=C:6772/T:5074;PAC=NP_001123631.1,NP_001123632.1,NP_001123633.1,NP_001123634.1,NP_689922.3,XP_005269662.1 GT:GQ:PL 0/0:6:0,9,65 0/1:3:0,0,0 1/1:17:106,18,0 1/1:4:13,3,0</p>

<p>10 126678087 rs111888382 C T 16.70 highCVG AC1=6;AF1=0.6827;DB;DP=36;DP4=3,5,13,14;FQ=21.4;MQ=11;PV4=0.7,0.29,1.9e-05,0.13;SNPEFF_AMINO_ACID_CHANGE=*986;SNPEFF_CODON_CHANGE=taG/taA;SNPEFF_EFFECT=SYNONYMOUS_STOP;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0149;DN=132;DA=C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=synonymous;FD=synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=stop;PP=446/446,446/446,986/986,514/514,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,446/446,421/421,421/421;CDP=1338,1338,2958,1542,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1338,1263,1263;CP=1.000;CG=5.120;AA=C;DV=by-cluster;DSP=181;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:4:0,3,38 1/1:11:16,12,0 1/1:30:45,31,0</p>

<p>10 126678092 rs73373169 G A 25.10 highCVG AC1=5;AF1=0.6731;DB;DP=36;DP4=2,5,12,13;FQ=29.5;MQ=12;PV4=0.43,1,1.3e-05,0.07;SNPEFF_AMINO_ACID_CHANGE=Q985*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0155;DN=131;DA=G/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=stop-gained;FD=stop-gained,stop-gained,stop-gained,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/stop;PP=445/446,445/446,985/986,513/514,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,445/446,420/421,420/421;CDP=1333,1333,2953,1537,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1333,1258,1258;CP=1.000;CG=5.120;AA=G;DG;DV=by-cluster;DSP=176;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$MYB_Q6;GESP=G:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:7:1,0,41 0/1:3:0,5,41 1/1:17:22,18,0 1/1:28:49,28,0</p>

<p>CTBP2 10 126678177 . G T 39.20 highCVG AC1=6;AF1=0.7163;DP=34;DP4=3,4,8,17;FQ=44.8;MQ=28;PV4=0.67,0.00015,8e-15,1;SNPEFF_AMINO_ACID_CHANGE=N956K;SNPEFF_CODON_CHANGE=aaC/aaA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0013;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=unknown;AAC=ASN/LYS;PP=416/446,416/446,956/986,484/514,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,416/446,391/421,391/421;CDP=1248,1248,2868,1452,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1248,1173,1173;GS=94;PH=0.613;CP=1.000;CG=3.120;AA=G;DSP=91;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;TFBS=V$BACH1_01,V$TAXCREB_02;GESP=G:12998;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:5:1,0,7 0/1:4:0,4,98 1/1:24:24,24,0 1/1:33:60,33,0</p>

<p>10 126678221 rs77752853 C A 43.30 highCVG AC1=6;AF1=0.7316;DB;DP=30;DP4=3,4,3,17;FQ=49.1;MQ=30;PV4=0.29,0.031,2.4e-12,0.15;SNPEFF_AMINO_ACID_CHANGE=A942S;SNPEFF_CODON_CHANGE=Gcc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=131;DA=C/A;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/SER;PP=402/446,402/446,942/986,470/514,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,402/446,377/421,377/421;CDP=1204,1204,2824,1408,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1204,1129,1129;GS=99;PH=0.599;CP=0.998;CG=5.120;AA=C;DV=by-cluster;DSP=47;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;GESP=C:13006;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/1:6:0,1,11 0/1:11:4,0,114 1/1:21:24,21,0 1/1:25:60,24,0</p>

<p>10 126678241 rs113985797 G C 8.10 highCVG AC1=4;AF1=0.4981;DB;DP=26;DP4=4,7,2,13;FQ=10.6;MQ=39;PV4=0.35,1.5e-07,2.9e-13,1;SNPEFF_AMINO_ACID_CHANGE=A935G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CTBP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022802.2;VDB=0.0006;DN=132;DA=G/C/T;GM=NM_001083914.1,NM_001329.2,NM_022802.2,XM_005269560.1,XM_005269561.1,XM_005269562.1,XM_005269563.1,XM_005269564.1,XM_005269565.1,XM_005269566.1,XM_005269567.1,XM_005269568.1,XM_005269569.1,XM_005269570.1,XM_005269571.1,XM_005269572.1,XM_005269573.1,XM_005269574.1;GL=CTBP2;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/GLY;PP=395/446,395/446,935/986,463/514,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,395/446,370/421,370/421;CDP=1184,1184,2804,1388,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1184,1109,1109;GS=60;PH=0.016;CP=1.000;CG=5.120;AA=G;DV=by-frequency,by-cluster;DSP=27;KP=hsa04310:WntSignalingPathway|hsa04330:NotchSignalingPathway|hsa05200:PathwaysInCancer|hsa05220:ChronicMyeloidLeukemia;PPI=ZEB1:974|HIC1:846|MDM2:846|EP300:846|ZNF217:846|KLF3:845|NRIP1:708;PAC=NP_001077383.1,NP_001320.1,NP_073713.2,XP_005269617.1,XP_005269618.1,XP_005269619.1,XP_005269620.1,XP_005269621.1,XP_005269622.1,XP_005269623.1,XP_005269624.1,XP_005269625.1,XP_005269626.1,XP_005269627.1,XP_005269628.1,XP_005269629.1,XP_005269630.1,XP_005269631.1 GT:GQ:PL 0/0:4:0,2,39 0/0:6:0,6,121 1/1:10:17,18,0 0/1:10:34,0,4</p>

<p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

<p>17 56388303 rs3744099 T A 41.60 midCVG AC1=2;AF1=0.3053;DB;DP=10;DP4=2,4,3,1;FQ=43.1;MQ=60;PV4=0.52,0.25,1,0.057;SNPEFF_AMINO_ACID_CHANGE=H1118L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BZRAP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004758.3;VDB=0.0175;DN=107;DA=T/A;GM=NM_001261835.1,NM_004758.3,NM_024418.2,XM_005257807.1,XM_005257808.1,XM_005257809.1,XM_005257810.1;GL=BZRAP1;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown,unknown;AAC=HIS/LEU;PP=1118/1849,1118/1858,1058/1798,1118/1899,1111/1892,1058/1839,968/1749;CDP=3353,3353,3173,3353,3332,3173,2903;GS=99;PH=0.05;CP=0.000;CG=-11.500;AA=T;HA=0.0;HE=6.0;HC=10.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=107;TFBS=V$MYOGNF1_01,V$PAX5_02;GESP=A:660/T:12322;PAC=NP_001248764.1,NP_004749.2,NP_077729.1,XP_005257864.1,XP_005257865.1,XP_005257866.1,XP_005257867.1 GT:GQ:PL 0/0:6:0,3,26 0/0:14:0,12,84 1/1:6:84,12,0 0/0:6:0,3,25</p>

<p>17 74151720 rs201646108 C T 56.70 highCVG AC1=3;AF1=0.3694;DB;DP=26;DP4=6,12,3,3;FQ=58.7;MQ=59;PV4=0.63,1,0.041,0.25;SNPEFF_AMINO_ACID_CHANGE=A541T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RNF157;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052916.2;VDB=0.0188;DN=137;DA=C/T;GM=NM_052916.2,XM_005257006.1,XM_005257007.1,XM_005257008.1;GL=RNF157;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=541/680,541/658,541/654,474/613;CDP=1621,1621,1621,1420;GS=58;PH=1;CP=0.755;CG=5.430;AA=C;DV=by-cluster;DSP=78;GESP=T:5/C:13001;PAC=NP_443148.1,XP_005257063.1,XP_005257064.1,XP_005257065.1 GT:GQ:PL 0/0:9:0,9,60 0/1:29:29,0,78 1/1:7:75,12,0 0/0:24:0,24,130</p>

<p>19 21909948 rs7246400 G A 38.50 lowCVG AC1=5;AF1=0.6627;DB;DP=8;DP4=0,1,0,6;FQ=-6.85;MQ=34;PV4=1,0.01,1,0.36;SNPEFF_AMINO_ACID_CHANGE=T389I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF100;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173531.3;VDB=0.0289;DN=116;DA=G/A;GM=NM_173531.3,XM_005259783.1,XM_005259784.1,XM_005259785.1;GL=ZNF100;FG=missense;FD=missense,unknown,unknown,unknown;AAC=THR/ILE;PP=389/543,388/542,356/510,174/328;CDP=1166,1163,1067,521;GS=89;PH=0.804;CP=0.029;CG=0.841;AA=G;HA=30.4;HC=40.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=844;GESP=A:6476/G:6444;PAC=NP_775802.2,XP_005259840.1,XP_005259841.1,XP_005259842.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:16:76,18,0 0/1:3:0,3,27</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 44591360 rs8113249 G A 146 highCVG AC1=4;AF1=0.5099;DB;DP=21;DP4=3,5,6,7;FQ=114;MQ=60;PV4=1,0.021,1,0.056;SNPEFF_AMINO_ACID_CHANGE=E577K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF284;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001037813.2;VDB=0.0032;DN=116;DA=G/A;GM=NM_001037813.2,XM_005258859.1;GL=ZNF284;FG=missense;FD=missense,unknown;AAC=GLU/LYS;PP=577/594;CDP=1729;GS=56;PH=0;CP=0.707;CG=-0.036;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=1494;GESP=A:10995/G:1991;PAC=NP_001032902.1,XP_005258916.1 GT:GQ:PL 0/0:12:0,15,106 0/1:3:0,0,0 1/1:27:160,30,0 0/1:33:30,0,49</p>

<p>19 44610798 rs4239529 A T 111 highCVG AC1=4;AF1=0.5334;DB;DP=15;DP4=3,3,2,7;FQ=45.2;MQ=60;PV4=0.33,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H162L;SNPEFF_CODON_CHANGE=cAc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF224;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013398.2;VDB=0.0096;DN=111;DA=A/T;GM=NM_013398.2,NR_033341.1,XM_005259221.1;GL=ZNF224/LOC100379224;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=HIS/LEU,none,HIS/LEU;PP=162/708,NA,162/708;CDP=485,2055,485;GS=99,NA,99;PH=0;CP=0.000;CG=-1.700;AA=T;HA=9.2;HE=12.5;HC=37.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=250;GESP=T:11402/A:1604;PAC=NP_037530.2,NA,XP_005259278.1 GT:GQ:PL 0/0:12:0,15,81 0/1:3:0,0,0 1/1:12:78,15,0 0/1:8:76,0,4</p>

<p>19 55174498 rs28366008 T C 53.10 midCVG AC1=3;AF1=0.4334;DB;DP=10;DP4=2,3,0,5;FQ=52;MQ=58;PV4=0.44,1,0.17,1;SNPEFF_AMINO_ACID_CHANGE=F46L;SNPEFF_CODON_CHANGE=Ttc/Ctc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001278429.3;VDB=0.0117;DN=125;DA=T/C;GM=NM_001278426.2,NM_001278427.2,NM_001278428.2,NM_001278429.2,NM_001278430.2;GL=LILRB4;FG=missense;FD=unknown;AAC=PHE/LEU;PP=5/449,5/448,5/449,46/479,5/255;CDP=13,13,13,136,13;GS=22;PH=0.001;CP=0.000;CG=0.476;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=22;GESP=C:4017/T:8989;PPI=PTPN11:846|PTPN6:846;PAC=NP_001265355.1,NP_001265356.1,NP_001265357.1,NP_001265358.1,NP_001265359.1 GT:GQ:PL 0/0:6:0,6,44 0/0:4:0,3,22 1/1:6:66,9,0 0/1:30:31,0,30</p>

<p>19 58946056 rs1465789 G A 19.10 midCVG AC1=3;AF1=0.3689;DB;DP=9;DP4=5,1,1,2;FQ=21.1;MQ=60;PV4=0.23,0.3,1,0.085;SNPEFF_AMINO_ACID_CHANGE=P252L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF132;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003433.3;VDB=0.0079;DN=88;DA=G/A;GM=NM_003433.3,XM_005259210.1;GL=ZNF132;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=252/707,239/694;CDP=755,716;GS=98;PH=0;CP=0.002;CG=0.980;AA=A;HA=14.9;HE=46.1;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=523;GESP=A:7487/G:5519;PAC=NP_003424.3,XP_005259267.1 GT:GQ:PL 0/0:7:0,6,53 0/0:7:0,6,49 1/1:3:49,6,0 0/1:13:13,0,30</p>

<p>20 17716416 rs1053993 C G 89 midCVG AC1=5;AF1=0.6077;DB;DP=14;DP4=3,4,3,4;FQ=77.9;MQ=60;PV4=1,0.18,1,0.1;SNPEFF_AMINO_ACID_CHANGE=T85S;SNPEFF_CODON_CHANGE=aCt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BANF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159495.1;VDB=0.0033;DN=86;DA=C/G;GM=NM_001014977.3,NM_001159495.1,NM_178477.4,XM_005260668.1;GL=BANF2;FG=missense;FD=missense,missense,missense,unknown;AAC=THR/SER;PP=78/91,85/98,78/91,78/91;CDP=233,254,233,233;GS=58;PH=0;CP=0.107;CG=3.360;AA=G;HA=45.0;HE=22.5;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=107;GESP=G:4533/C:8473;PAC=NP_001014977.2,NP_001152967.1,NP_848572.3,XP_005260725.1 GT:GQ:PL 0/1:8:3,0,60 0/0:6:0,9,61 1/1:8:66,9,0 1/1:8:67,9,0</p>

<p>22 22868841 . A G 36.10 highCVG AC1=3;AF1=0.3333;DP=17;DP4=2,11,0,4;FQ=37.8;MQ=59;PV4=1,0.42,0.11,1;SNPEFF_AMINO_ACID_CHANGE=S372P;SNPEFF_CODON_CHANGE=Tca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF280A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_080740.4;VDB=0.0001;GM=NM_080740.3;GL=ZNF280A;FG=missense;FD=unknown;AAC=SER/PRO;PP=372/543;CDP=1114;GS=74;PH=1;CP=0.993;CG=2.860;AA=A;DSP=1185;GESP=A:13006;PAC=NP_542778.1 GT:GQ:PL 0/0:8:0,6,48 0/0:3:0,0,0 1/1:6:78,12,0 0/0:34:0,33,165</p>


        <p>
          <strong>IDs and variants associated:</strong> H00523;SPRED1|HRAS|ARAF|KRAS|NF1|NF2|NRAS|MAP2K1|MAP2K2|PTPN11|SOS1|BRAF|SHOC2|CBL;hsa04010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 1
        </p>
        <p>
          <strong>Missense Mutations:</strong> 20
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.30
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse65">Disorders of fatty-acid oxidation</a>
      </h4>
    </div>
    <div id="collapse65" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	53676448	rs1799821	G	A	24.80	lowCVG	AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1	GT:GQ:PL	0/1:3:0,0,0	0/0:3:0,3,22	1/1:5:45,6,0	0/1:20:19,0,22

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00525;CPT1A|CPT2|HADHA|HADHB|ACADM|ACADS|ACADSB|ACADVL|SLC22A5|SLC25A20;hsa00280,hsa00071
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse66">Retinitis pigmentosa (RP)</a>
      </h4>
    </div>
    <div id="collapse66" class="panel-collapse collapse">
      <div class="panel-body">

        <p>4 128996129 . G T 28.60 midCVG AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1 GT:GQ:PL 0/0:8:0,6,51 0/0:6:0,3,17 1/1:6:71,12,0 0/0:8:0,6,45</p>

<p>5 95091201 rs34899 A G 48.40 highCVG AC1=3;AF1=0.4188;DB;DP=17;DP4=3,6,3,2;FQ=50.4;MQ=60;PV4=0.58,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N262D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RHOBTB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014899.3;VDB=0.0132;DN=76;DA=A/G;GM=NM_014899.3,XM_005271925.1;GL=RHOBTB3;FG=missense;FD=missense,unknown;AAC=ASN/ASP;PP=262/612,294/644;CDP=784,880;GS=23;PH=0;CP=0.013;CG=5.210;AA=G;HA=30.5;HE=41.1;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=102;GESP=G:5290/A:7716;PAC=NP_055714.3,XP_005271982.1 GT:GQ:PL 0/0:4:0,3,27 0/0:4:0,3,15 1/1:7:88,12,0 0/1:4:1,0,63</p>

<p>6 66005888 rs9342464 C T 35.50 highCVG AC1=4;AF1=0.47;DB;DP=19;DP4=2,11,2,3;FQ=38.2;MQ=60;PV4=0.53,0.3,1,0.074;SNPEFF_AMINO_ACID_CHANGE=G631S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0077;DN=119;DA=C/T;GM=NM_001142800.1;GL=EYS;FG=missense;FD=missense;AAC=GLY/SER;PP=631/3145;CDP=1891;GS=56;PH=0;CP=0.025;CG=1.710;AA=C;HA=23.3;HE=45.0;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=125;GESP=T:2272/C:2294;PAC=NP_001136272.1 GT:GQ:PL 0/0:10:0,12,80 0/1:11:9,0,78 1/1:6:62,9,0 0/1:12:10,0,69</p>

<p>6 66053934 rs61753611 T G 82 midCVG AC1=4;AF1=0.5267;DB;DP=13;DP4=4,2,2,4;FQ=59.3;MQ=60;PV4=0.57,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K532N;SNPEFF_CODON_CHANGE=aaA/aaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EYS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001292009.1;VDB=0.0126;DN=129;DA=T/G;GM=NM_001142800.1,NM_001142801.1,NM_198283.1;GL=EYS;FG=missense;FD=missense;AAC=LYS/ASN;PP=532/3145,532/620,532/595;CDP=1596;GS=94;PH=0.246;CP=0.001;CG=-3.870;AA=T;DV=by-frequency,by-cluster;DSP=4;GESP=G:108/T:12896;PAC=NP_001136272.1,NP_001136273.1,NP_938024.1 GT:GQ:PL 0/0:12:0,15,90 0/0:3:0,3,14 1/1:9:87,12,0 1/1:5:43,6,0</p>

<p>8 10468976 . G C 25.70 lowCVG AC1=5;AF1=0.6557;DP=5;DP4=2,0,2,1;FQ=8.94;MQ=60;PV4=1,0.4,1,0.25;SNPEFF_AMINO_ACID_CHANGE=Q878E;SNPEFF_CODON_CHANGE=Caa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RP1L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178857.5;VDB=0.0119;GM=NM_178857.5;GL=RP1L1;FG=missense;FD=unknown;AAC=GLN/GLU;PP=878/2401;CDP=2632;GS=29;PH=0.005;CP=0.002;CG=2.580;AA=G;DSP=1881;GESP=G:10756;PAC=NP_849188.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,44 1/1:4:24,3,0 1/1:5:42,6,0</p>

<p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

<p>12 53605545 rs2229774 G A 41.20 midCVG AC1=4;AF1=0.5599;DB;DP=12;DP4=0,3,0,9;FQ=13.2;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=S427L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RARG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000966.5;VDB=0.0000;DN=98;DA=G/A;GM=NM_000966.5,NM_001042728.2,NM_001243730.1,NM_001243731.1,NM_001243732.1,XM_005269054.1,XM_005269055.1,XM_005269056.1,XM_005269057.1;GL=RARG;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/LEU;PP=427/455,416/444,355/383,306/334,405/433,514/542,514/542,427/455,427/455;CDP=1280,1247,1064,917,1214,1541,1541,1280,1280;GS=145;PH=0.525;CP=1.000;CG=4.530;AA=G;HA=11.1;HE=6.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=103;TFBS=V$ARP1_01;GESP=A:949/G:12057;PAC=NP_000957.1,NP_001036193.1,NP_001230659.1,NP_001230660.1,NP_001230661.1,XP_005269111.1,XP_005269112.1,XP_005269113.1,XP_005269114.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:81,27,0 0/0:7:0,9,49</p>

<p>14 21770730 rs6571751 A G 150 midCVG AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3 GT:GQ:PL 0/1:18:39,0,18 0/1:21:83,0,21 1/1:7:49,6,0 1/1:5:22,3,0</p>

<p>19 15587185 rs733731 C T 22.30 lowCVG AC1=5;AF1=0.6333;DB;DP=6;DP4=1,1,4,0;FQ=-3.17;MQ=60;PV4=0.33,0.097,1,1;SNPEFF_AMINO_ACID_CHANGE=R99Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PGLYRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052890.3;VDB=0.0069;DN=86;DA=C/T;GM=NM_052890.3;GL=PGLYRP2;FG=missense;FD=missense;AAC=ARG/GLN;PP=99/577;CDP=296;GS=43;PH=0.03;CP=0.001;CG=-5.850;AA=C;HA=34.3;HE=39.5;HC=38.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=235;GESP=T:4832/C:8174;PAC=NP_443122.3 GT:GQ:PL 0/1:14:15,0,14 0/1:4:0,3,24 1/1:4:19,3,0 1/1:6:31,6,0</p>

<p>19 55494157 rs4306647 G A 14.20 highCVG AC1=3;AF1=0.3892;DB;DP=15;DP4=6,3,3,1;FQ=16.2;MQ=41;PV4=1,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R364K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001174081.1;VDB=0.0098;DN=111;DA=G/A;GM=NM_001174081.1,NM_001174082.1,NM_001174083.1,NM_017852.3,XM_005259050.1;GL=NLRP2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/LYS;PP=364/1063,342/1041,341/1040,364/1063,361/1060;CDP=1091,1025,1022,1091,1082;GS=26;PH=0.363;CP=0.019;CG=1.640;AA=G;HA=19.6;HE=2.2;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=628;GESP=A:915/G:12091;PAC=NP_001167552.1,NP_001167553.1,NP_001167554.1,NP_060322.1,XP_005259107.1 GT:GQ:PL 0/0:5:0,3,19 0/1:11:10,0,32 1/1:5:48,9,0 0/0:12:0,12,50</p>

<p>22 51045190 rs1140555 C T 26.90 lowCVG AC1=6;AF1=0.7232;DB;DP=5;DP4=1,0,3,0;FQ=-21.3;MQ=60;PV4=1,1,1,0.21;VDB=0.0048;DN=86;DA=C/T;GM=NM_012324.3,NM_016431.3;GL=MAPK8IP2;FG=synonymous;FD=synonymous-codon;PP=743/825,716/798;CDP=2228,2147;CP=0.989;CG=-1.270;AA=C;HC=2.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=4;KP=hsa04010:MAPKSignalingPathway;GESP=T:6729/C:5159;PPI=MAP2K7:846|MAPK13:845|MAPK8IP1:845|LRP2:809;PAC=NP_036456.1,NP_057515.1 GT:GQ:PL 0/1:3:0,0,0 0/1:10:40,0,8 1/1:4:21,3,0 0/1:3:0,0,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00527;NR2E3|TOPORS|MERTK|PRPF8|CNGB1|CNGA1|ZNF513|SNRNP200|CRB1|ABCA4|FSCN2|PRPF31|GUCA1B|IDH3B|EYS|IMPDH1|CERKL|NRL|IMPG2|PDE6A|PDE6G|PDE6B|SPATA7|KLHL7|PRPH2|RGR|RHO|RLBP1|RP9|RP1|RP2|RPGR|RPE65|SAG|SEMA4A|TULP1|USH2A|BEST1|CA4|PRCD|ARL6|FAM161A|PROM1|PRPF3|LRAT;hsa04744,hsa03040,hsa00830
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 10
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse67">von Hippel-Lindau syndrome</a>
      </h4>
    </div>
    <div id="collapse67" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          11	61165741	rs10897158	G	C	74.40	midCVG	AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1	GT:GQ:PL	0/1:10:35,0,8	0/0:6:0,9,70	1/1:4:24,3,0	1/1:8:61,9,0

        </p>
        <p>
          11	69466030	.	G	A	20.20	lowCVG	AC1=2;AF1=0.2865;DP=8;DP4=4,1,3,0;FQ=21.6;MQ=59;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=V290M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCND1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053056.2;VDB=0.0176;GM=NM_053056.2;GL=CCND1;FG=missense;FD=unknown;AAC=VAL/MET;PP=290/296;CDP=868;GS=21;PH=0.998;CP=1.000;CG=4.680;AA=G;DSP=145;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway|hsa04310:WntSignalingPathway|hsa04510:FocalAdhesion|hsa04630:Jak-STATSignalingPathway|hsa05200:PathwaysInCancer|hsa05210:ColorectalCancer|hsa05212:PancreaticCancer|hsa05213:EndometrialCancer|hsa05214:Glioma|hsa05215:ProstateCancer|hsa05216:ThyroidCancer|hsa05218:Melanoma|hsa05219:BladderCancer|hsa05220:ChronicMyeloidLeukemia|hsa05221:AcuteMyeloidLeukemia|hsa05222:SmallCellLungCancer|hsa05223:Non-smallCellLungCancer|hsa05416:ViralMyocarditis;TFBS=V$AP2_Q6,V$PAX2_01,V$SEF1_C;GESP=G:12974;PPI=CDKN1A:999|CDK4:999|CDK6:999|RB1:999|UBC:999|CDKN1B:998|CDK2:974|HDAC3:974|STAT3:937|PCNA:937|ESR1:937;PAC=NP_444284.1	GT:GQ:PL	0/0:6:0,3,17	0/0:6:0,3,14	1/1:4:61,9,0	0/0:11:0,9,66

        </p>

        <p>
          <strong>IDs and variants associated:</strong>
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse68">Kabuki syndrome</a>
      </h4>
    </div>
    <div id="collapse68" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          12	49443799	rs112236653	G	A	49.50	midCVG	AC1=4;AF1=0.5106;DB;DP=12;DP4=5,2,2,3;FQ=52.5;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1191L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KMT2D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003482.3;VDB=0.0155;DN=132;DA=G/A;GM=NM_003482.3,XM_005269162.1;GL=KMT2D;FG=missense;FD=missense,unknown;AAC=PRO/LEU;PP=1191/5538;CDP=3572;GS=98;PH=0.764;CP=0.993;CG=5.400;AA=G;DV=by-frequency,by-cluster;DSP=335;GESP=A:10/G:12452;PPI=WDR5:996|NCOA6:971|ASH2L:968|RBBP5:968|CTNNB1:845|ESR1:833|KDM6A:830|PAXIP1:824;PAC=NP_003473.3,XP_005269219.1	GT:GQ:PL	0/1:14:11,0,64	0/1:3:0,0,0	1/1:9:81,12,0	0/0:7:0,9,52

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00570;KMT2D;hsa00310
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse69">Syndromic X-linked mental retardation with epilepsy or seizures</a>
      </h4>
    </div>
    <div id="collapse69" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          11	71169547	rs2276360	G	C	15.50	lowCVG	AC1=5;AF1=0.5737;DB;DP=8;DP4=2,2,0,3;FQ=19.2;MQ=60;PV4=0.43,0.24,1,0.37;SNPEFF_AMINO_ACID_CHANGE=V74L;SNPEFF_CODON_CHANGE=Gtg/Ctg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NADSYN1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018161.4;VDB=0.0099;DN=100;DA=G/C;GM=NM_018161.4;GL=NADSYN1;FG=missense;FD=missense;AAC=VAL/LEU;PP=74/707;CDP=220;GS=32;PH=0;CP=0.037;CG=-2.340;AA=C;HA=33.2;HE=27.2;HC=40.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=44;KP=hsa00760:NicotinateAndNicotinamideMetabolism|hsa01100:MetabolicPathways;GESP=C:8267/G:4721;PAC=NP_060631.2	GT:GQ:PL	0/1:3:21,3,0	0/1:3:0,0,0	1/1:5:36,6,0	0/0:10:0,12,81

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00577;PQBP1|ATP6AP2|SLC9A6|RAB39B|ZDHHC15|ARX|FLNA|SRPX2|GRIA3|1CAM|OPHN1|NSDHL|ATRX|SHROOM4|PCDH19|SLC6A8|SYN1|SYP|KDM5C|PHF6|CASK;hsa04730,hsa04514,hsa04510,hsa04360,hsa04080,hsa04010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse70">Epidermolysis bullosa, junctional, including: Epidermolysis bullosa, junctional, Herlitz type (JEB-H
</a>
      </h4>
    </div>
    <div id="collapse70" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          7	107593989	rs20556	T	C	80.60	midCVG	AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1	GT:GQ:PL	0/1:17:41,0,14	0/1:3:0,0,0	1/1:13:82,15,0	0/0:4:0,6,52

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00586;COL17A1|ITGA6|ITGB4|LAMA3|LAMB3|LAMC2;hsa04810,hsa04512,hsa04510
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse71">Congenital muscular dystrophies (CMD/MDC)</a>
      </h4>
    </div>
    <div id="collapse71" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          6	152443744	rs2295190	G	T	29	lowCVG	AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:34,6,0	0/1:24:31,0,23

        </p>
        <p>
          7	107593989	rs20556	T	C	80.60	midCVG	AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1	GT:GQ:PL	0/1:17:41,0,14	0/1:3:0,0,0	1/1:13:82,15,0	0/0:4:0,6,52

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00590;POMT1|COL6A1|COL6A2|COL6A3|FKTN|POMT2|ITGA7|ITGA9|LAMA2|LMNA|POMGNT1|SEPN1|FKRP|LARGE;hsa04810,hsa04510
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse72">Limb-girdle muscular dystrophy (LGMD)</a>
      </h4>
    </div>
    <div id="collapse72" class="panel-collapse collapse">
      <div class="panel-body">

        <p>
          6	152443744	rs2295190	G	T	29	lowCVG	AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:34,6,0	0/1:24:31,0,23

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00593;POMT1|DAG1|ANO5|FKTN|TRIM32|POMT2|LMNA|POMGNT1|SGCA|SGCB|SGCD|SGCG|FKRP|CAPN3|DYSF|TCAP|CAV3|MYOT;hsa04510,hsa04144,hsa04120
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse73">Distal muscular dystrophies</a>
      </h4>
    </div>
    <div id="collapse73" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

<p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>10 69933969 rs7916821 G A 187 highCVG AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA GT:GQ:PL 0/1:45:68,0,45 0/1:5:0,3,28 1/1:15:86,15,0 1/1:13:80,12,0</p>


        <p>
          <strong>IDs and variants associated:</strong> H00594;GNE|LDB3|CRYAB|DES|ANO5|MYH7|NEB|TTN|DYSF|MYOT|MATR3;hsa04530,hsa04260,hsa00520
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 5
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.5
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse74">Myofibrillar myopathies (MFM)</a>
      </h4>
    </div>
    <div id="collapse74" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          2	152527572	rs7426114	C	T	226	highCVG	AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1	GT:GQ:PL	0/1:13:55,0,13	0/1:3:0,6,35	1/1:15:90,15,0	1/1:21:129,21,0

        </p>
        <p>
          2	152536498	rs6735208	T	A	123	highCVG	AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1	GT:GQ:PL	0/1:12:5,0,108	0/1:5:71,0,5	1/1:11:55,9,0	1/1:8:34,6,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong>
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse75">46,XX disorders of sex development (Disorders of gonadal development)</a>
      </h4>
    </div>
    <div id="collapse75" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          14	58681951	.	T	C	41	lowCVG	AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1	GT:GQ:PL	0/1:33:31,0,37	0/0:4:0,6,50	1/1:3:26,3,0	1/1:3:27,3,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00598;FSHR|RSPO1|WNT4|SOX3|SOX9|SOX10|SRY|BMP15;hsa04310
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse76">46,XY disorders of sex development (Disorders of gonadal development)</a>
      </h4>
    </div>
    <div id="collapse76" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          14	58681951	.	T	C	41	lowCVG	AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1	GT:GQ:PL	0/1:33:31,0,37	0/0:4:0,6,50	1/1:3:26,3,0	1/1:3:27,3,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00607;DMRT1|NR0B1|NR5A1|DHH|WNT4|ATRX|SOX9|SRY|TSPY1|WT1|CBX2;hsa04340,hsa04310
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse77">46,XY disorders of sex development (Disorders in androgen synthesis or action)</a>
      </h4>
    </div>
    <div id="collapse77" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 12855878 rs1052908 A C 37.70 midCVG AC1=6;AF1=0.6449;DB;DP=11;DP4=3,1,1,4;FQ=-6.46;MQ=20;PV4=0.21,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R386S;SNPEFF_CODON_CHANGE=agA/agC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRAMEF1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_023013.4;VDB=0.0080;DN=86;DA=A/C;GM=NM_023013.2,XM_005263490.1;GL=PRAMEF1;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=386/475,141/230;CDP=1158,423;GS=110;PH=0;CP=0.004;CG=0.600;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,suspect;DSP=292;GESP=A:12994;PAC=NP_075389.1,XP_005263547.1 GT:GQ:PL 0/1:2:4,3,0 0/1:3:0,0,0 1/1:3:70,4,0 0/0:3:0,3,27</p>

<p>1 15909850 rs6429757 C G 23.50 midCVG AC1=3;AF1=0.403;DB;DP=14;DP4=9,2,3,0;FQ=25.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G105R;SNPEFF_CODON_CHANGE=Ggg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AGMAT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024758.4;VDB=0.0052;DN=116;DA=C/G;GM=NM_024758.4;GL=AGMAT;FG=missense;FD=missense;AAC=GLY/ARG;PP=105/353;CDP=313;GS=125;PH=0;CP=0.010;CG=-1.410;AA=G;HA=7.9;HE=49.6;HC=39.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;TFBS=V$COMP1_01;GESP=G:8504/C:4502;PAC=NP_079034.3 GT:GQ:PL 0/0:6:0,6,49 0/1:4:22,3,0 1/1:3:47,6,0 0/0:26:0,27,74</p>

<p>1 24201920 rs2501432 T C 86.40 highCVG AC1=5;AF1=0.5586;DB;DP=20;DP4=8,2,3,5;FQ=69.9;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q63R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CNR2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001841.2;VDB=0.0031;DN=100;DA=T/C;GM=NM_001841.2,XM_005245736.1,XM_005245737.1,XM_005245738.1;GL=CNR2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLN/ARG;PP=63/361;CDP=188;GS=43;PH=0;CP=0.910;CG=5.120;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=233;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=C:7940/T:5066;PAC=NP_001832.1,XP_005245793.1,XP_005245794.1,XP_005245795.1 GT:GQ:PL 0/1:23:46,0,21 0/0:20:0,24,93 1/1:3:16,3,0 1/1:10:72,12,0</p>

<p>1 36550658 rs12043423 C T 16.90 lowCVG AC1=4;AF1=0.4561;DB;DP=8;DP4=2,3,2,1;FQ=19.5;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=R46C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TEKT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014466.2;VDB=0.0094;DN=120;DA=C/T;GM=NM_014466.2,XM_005270753.1;GL=TEKT2;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=46/431;CDP=136;GS=180;PH=1;CP=1.000;CG=3.680;AA=C;HA=6.1;HE=0.0;HC=19.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=21;GESP=T:273/C:12733;PAC=NP_055281.2,XP_005270810.1 GT:GQ:PL 0/1:3:0,0,0 0/0:11:0,12,80 1/1:4:37,6,0 0/1:16:21,0,13</p>

<p>1 53676448 rs1799821 G A 24.80 lowCVG AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,22 1/1:5:45,6,0 0/1:20:19,0,22</p>

<p>1 54661150 rs946448 C T 61.80 highCVG AC1=4;AF1=0.5145;DB;DP=17;DP4=6,2,5,1;FQ=64.8;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=R47K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYB5RL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001031672.2;VDB=0.0105;DN=86;DA=C/T;GM=NM_001031672.2,XM_005271096.1;GL=CYB5RL;FG=missense;FD=missense,unknown;AAC=ARG/LYS;PP=47/316,47/237;CDP=140;GS=26;PH=0.159;CP=1.000;CG=4.240;AA=C;HA=17.4;HE=37.5;HC=24.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=59;GESP=T:3741/C:8713;PAC=NP_001026842.2,XP_005271153.1 GT:GQ:PL 0/0:3:0,3,18 0/0:18:0,21,113 1/1:9:69,12,0 1/1:5:41,6,0</p>

<p>1 152285603 . G A 73.80 midCVG AC1=5;AF1=0.6441;DP=9;DP4=2,1,1,5;FQ=9.97;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R587C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002016.1;VDB=0.0003;GM=NM_002016.1;GL=FLG;FG=missense;FD=unknown;AAC=ARG/CYS;PP=587/4062;CDP=1759;GS=180;PH=1;CP=0.000;CG=-3.110;AA=G;DSP=912;GESP=G:13006;PPI=FGFR1:845;PAC=NP_002007.1 GT:GQ:PL 0/1:10:19,0,9 0/0:4:0,6,41 1/1:4:26,3,0 1/1:11:74,12,0</p>

<p>1 152770521 rs41268492 G A 13.80 lowCVG AC1=6;AF1=0.699;DB;DP=5;DP4=1,0,3,0;FQ=-21.4;MQ=47;PV4=1,1,0.48,1;SNPEFF_AMINO_ACID_CHANGE=R84H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LCE1D;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178352.2;VDB=0.0045;DN=127;DA=G/A;GM=NM_178352.2;GL=LCE1D;FG=missense;FD=missense;AAC=ARG/HIS;PP=84/115;CDP=251;GS=29;PH=0;CP=0.000;CG=-7.330;AA=A;DV=by-frequency,by-cluster;DSP=273;GESP=A:2436/G:9006;PAC=NP_848129.1 GT:GQ:PL 0/1:3:0,0,0/1:4:0,1,8 1/1:4:47,6,0 0/1:3:0,0,0</p>

<p>1 156883546 rs12137505 A G 17.30 lowCVG AC1=4;AF1=0.4665;DB;DP=8;DP4=3,1,2,1;FQ=20;MQ=60;PV4=1,0.24,1,0.14;SNPEFF_AMINO_ACID_CHANGE=N903D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PEAR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080471.1;VDB=0.0107;DN=120;DA=A/G;GM=NM_001080471.1,XM_005245141.1,XM_005245142.1;GL=PEAR1;FG=missense;FD=missense,unknown,unknown;AAC=ASN/ASP;PP=903/1038,903/1038,839/974;CDP=2707,2707,2515;GS=23;PH=0.094;CP=0.000;CG=-3.910;AA=A;HA=20.6;HE=35.7;HC=48.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=7;GESP=G:4616/A:8390;PAC=NP_001073940.1,XP_005245198.1,XP_005245199.1 GT:GQ:PL 0/0:4:0,3,27 0/0:8:0,9,64 1/1:4:44,6,0 0/1:4:17,3,0</p>

<p>1 169823718 rs4656197 T C 47.80 highCVG AC1=4;AF1=0.5453;DB;DP=15;DP4=3,3,0,7;FQ=39.1;MQ=60;PV4=0.07,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=Q621R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCYL3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181093.3;VDB=0.0088;DN=111;DA=T/A/C/G;GM=NM_020423.6,NM_181093.3;GL=SCYL3;FG=missense;FD=missense;AAC=GLN/ARG;PP=567/689,621/743;CDP=1700,1862;GS=43;PH=0.589;CP=0.000;CG=-1.020;AA=T;HA=27.3;HE=43.9;HC=34.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=308;GESP=C:8150/T:4856;PAC=NP_065156.5,NP_851607.2 GT:GQ:PL 0/1:5:52,0,0 0/1:3:0,0,0 1/1:7:37,9,0 0/0:12:0,15,77</p>

<p>1 182025558 . T C 23 midCVG AC1=2;AF1=0.2266;DP=12;DP4=0,9,0,3;FQ=24;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R530G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF648;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001009992.1;VDB=0.0015;GM=NM_001009992.1;GL=ZNF648;FG=missense;FD=unknown;AAC=ARG/GLY;PP=530/569;CDP=1588;GS=125;PH=0.797;CP=0.999;CG=1.640;AA=T;DSP=1651;GESP=T:13006;PAC=NP_001009992.1 GT:GQ:PL 0/0:9:0,6,35 0/0:9:0,6,44 1/1:3:65,9,0 0/0:18:0,15,92</p>

<p>1 206225011 rs33990840 G C 30 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=2,0,1,2;FQ=11.7;MQ=60;PV4=0.4,0.36,1,0.25;SNPEFF_AMINO_ACID_CHANGE=G191R;SNPEFF_CODON_CHANGE=Ggc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AVPR1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000707.3;VDB=0.0112;DN=126;DA=G/C;GM=NM_000707.3;GL=AVPR1B;FG=missense;FD=missense;AAC=GLY/ARG;PP=191/425;CDP=571;GS=125;PH=0;CP=1.000;CG=4.980;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=370;KP=hsa04020:CalciumSignalingPathway|hsa04080:NeuroactiveLigand-receptorInteraction|hsa04270:VascularSmoothMuscleContraction;GESP=C:802/G:12188;PAC=NP_000698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:67,9,0 0/0:6:0,6,45</p>

<p>1 207790088 rs3811381 C G 67.70 midCVG AC1=5;AF1=0.5957;DB;DP=9;DP4=0,4,1,4;FQ=29.7;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2277R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=41;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000651.4;VDB=0.0076;DN=107;DA=C/G;GM=NM_000573.3,NM_000651.4,XM_005273064.1;GL=CR1;FG=missense;FD=missense,missense,unknown;AAC=PRO/ARG;PP=1827/2040,2277/2490,2149/2362;CDP=5480,6830,6446;GS=103;PH=0.643;CP=0.000;CG=-5.580;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;KP=hsa04610:ComplementAndCoagulationCascades|hsa04640:HematopoieticCellLineage|hsa05140:Leishmaniasis;GESP=G:2006/C:10176;PPI=CR2:846;PAC=NP_000564.2,NP_000642.3,XP_005273121.1 GT:GQ:PL 0/1:35:36,0,34 0/0:4:0,6,36 1/1:6:51,6,0 1/1:4:25,3,0</p>

<p>1 214830617 rs438034 A G 57 lowCVG AC1=5;AF1=0.6559;DB;DP=8;DP4=2,0,2,4;FQ=7.46;MQ=60;PV4=0.43,0.015,1,1;SNPEFF_AMINO_ACID_CHANGE=R2943G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPF;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016343.3;VDB=0.0106;DN=80;DA=A/G;GM=NM_016343.3,XM_005273008.1;GL=CENPF;FG=missense;FD=missense,unknown;AAC=ARG/GLY;PP=2943/3115,2921/3093;CDP=8827,8761;GS=125;PH=0;CP=0.000;CG=2.520;AA=A;HA=36.6;HE=47.7;HC=9.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=136;GESP=G:6399/A:6607;PPI=NDEL1:718;PAC=NP_057427.3,XP_005273065.1 GT:GQ:PL 0/1:18:16,0,20 0/1:18:14,0,23 1/1:11:68,12,0 0/1:3:0,0,0</p>

<p>1 228559430 rs41305136 G A 31.70 lowCVG AC1=5;AF1=0.6552;DB;DP=6;DP4=0,2,2,1;FQ=8.11;MQ=60;PV4=0.4,1,1,0.44;SNPEFF_AMINO_ACID_CHANGE=R7941Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=105;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OBSCN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271223.2;VDB=0.0019;DN=127;DA=G/A;GM=NM_001098623.2,NM_001271223.2,XM_005273287.1,XM_005273288.1,XM_005273289.1,XM_005273290.1,XM_005273291.1,XM_005273292.1,XM_005273293.1,XM_005273294.1,XM_005273295.1,XM_005273296.1,XM_005273297.1,XM_005273298.1,XM_005273299.1,XM_005273300.1,XM_005273301.1,XM_005273302.1,XM_005273303.1,XM_005273304.1,XM_005273305.1,XM_005273306.1,XM_005273307.1,XM_005273308.1;GL=OBSCN;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLN;PP=6984/7969,7941/8924,8043/9028,8042/9027,8042/9027,8042/9027,8031/9016,7955/8940,7955/8940,7955/8940,7951/8936,7951/8936,7951/8936,7940/8925,7867/8852,7867/8852,7867/8852,7859/8844,7852/8837,7779/8764,7691/8676,7675/8660,7669/8654,7163/8148;CDP=20951,23822,24128,24125,24125,24125,24092,23864,23864,23864,23852,23852,23852,23819,23600,23600,23600,23576,23555,23336,23072,23024,23006,21488;GS=43;PH=0;CP=0.000;CG=-3.530;AA=G;DV=by-frequency,by-cluster;DSP=707;GESP=A:271/G:11683;PPI=TTN:974|ANK1:937;PAC=NP_001092093.2,NP_001258152.2,XP_005273344.1,XP_005273345.1,XP_005273346.1,XP_005273347.1,XP_005273348.1,XP_005273349.1,XP_005273350.1,XP_005273351.1,XP_005273352.1,XP_005273353.1,XP_005273354.1,XP_005273355.1,XP_005273356.1,XP_005273357.1,XP_005273358.1,XP_005273359.1,XP_005273360.1,XP_005273361.1,XP_005273362.1,XP_005273363.1,XP_005273364.1,XP_005273365.1 GT:GQ:PL 0/0:4:0,6,43 0/1:3:0,0,0 1/1:4:20,3,0 1/1:5:52,6,0</p>

<p>1 247420421 rs41308154 C T 25.10 midCVG AC1=3;AF1=0.3809;DB;DP=11;DP4=2,3,2,2;FQ=27.1;MQ=60;PV4=1,0.0023,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R350C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VN1R5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173858.1;VDB=0.0123;DN=127;DA=C/T;GM=NM_173858.1;GL=VN1R5;FG=missense;FD=unknown;AAC=ARG/CYS;PP=350/358;CDP=1048;GS=180;PH=0.987;CP=0.337;AA=C;DG;DV=by-cluster,by-1000G;RM=MSTC;DSP=44158;GESP=T:2053/C:9883;PAC=NP_776257.1 GT:GQ:PL 0/0:3:0,0,0 0/0:12:0,12,89 1/1:7:66,12,0 0/0:5:0,3,23</p>

<p>1 248458718 rs79635253 T C 14.40 midCVG AC1=4;AF1=0.443;DB;DP=9;DP4=4,1,1,2;FQ=16.8;MQ=45;PV4=0.46,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R55G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004692.1;VDB=0.0077;DN=131;DA=T/C;GM=NM_001004692.1;GL=OR2T12;FG=missense;FD=missense;AAC=ARG/GLY;PP=55/321;CDP=163;GS=125;PH=0.005;CP=0.159;CG=1.550;AA=T;DV=by-frequency,by-cluster;DSP=196059;KP=hsa04740:OlfactoryTransduction;GESP=C:613/T:12389;PAC=NP_001004692.1 GT:GQ:PL 0/0:4:0,3,23 0/1:4:13,3,0 1/1:3:45,6,0 0/0:12:0,12,73</p>

<p>1 248722777 rs77589892 T A 89 highCVG AC1=5;AF1=0.7783;DB;DP=29;DP4=7,2,14,5;FQ=-25.8;MQ=12;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R6W;SNPEFF_CODON_CHANGE=Agg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2T29;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004694.2;VDB=0.0185;DN=131;DA=T/A;GM=NM_001004694.2;GL=OR2T29;FG=missense;FD=missense;AAC=ARG/TRP;PP=6/316;CDP=16;GS=101;CP=0.001;CG=-0.604;AA=A;DG;DV=by-frequency,by-cluster;DSP=122849;KP=hsa04740:OlfactoryTransduction;GESP=A:4597/T:8405;PAC=NP_001004694.2 GT:GQ:PL 0/0:4:0,12,14 0/1:3:6,0,2 1/1:22:86,20,0 1/1:23:44,21,0</p>

<p>2 25022598 rs1550116 A G 39.40 lowCVG AC1=6;AF1=0.7255;DB;DP=6;DP4=1,1,1,3;FQ=3.46;MQ=60;PV4=1,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q34R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CENPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024322.2;VDB=0.0085;DN=88;DA=A/G;GM=NM_001199803.1,NM_024322.2;GL=CENPO;FG=missense;FD=missense;AAC=GLN/ARG;PP=28/295,34/301;CDP=83,101;GS=43;PH=0;CP=0.000;CG=-3.630;AA=A;HA=5.8;HE=18.0;HC=19.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=55;GESP=G:1761/A:11245;PAC=NP_001186732.1,NP_077298.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:64,9,0 0/1:17:12,0,32</p>

<p>2 131414176 rs62165276 C T 7.93 midCVG AC1=4;AF1=0.5176;DB;DP=10;DP4=1,3,3,2;FQ=10.6;MQ=18;PV4=0.52,0.35,0.061,0.14;SNPEFF_AMINO_ACID_CHANGE=R615W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POTEJ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277083.1;VDB=0.0090;DN=129;DA=C/T;GM=NM_001277083.1,XM_005263739.1,XM_005263740.1;GL=POTEJ;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=615/1039,497/921,492/916;CDP=1843,1489,1474;GS=101;CP=0.246;CG=0.993;AA=C;DG;DV=by-frequency,by-cluster;DSP=55;PAC=NP_001264012.1,XP_005263796.1,XP_005263797.1 GT:GQ:PL 0/0:6:0,6,33 0/1:3:0,0,0 1/1:5:39,9,0 0/1:10:9,0,34</p>

<p>2 167099158 rs6746030 A G 150 highCVG AC1=6;AF1=0.7278;DB;DP=16;DP4=2,4,7,3;FQ=65.7;MQ=60;PV4=0.3,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W1150R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCN9A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002977.3;VDB=0.0076;DN=116;DA=A/G;GM=NM_002977.3,XM_005246757.1;GL=SCN9A;FG=missense;FD=missense,unknown;AAC=TRP/ARG;PP=1150/1978,1161/1989;CDP=3448,3481;GS=101;PH=0;CP=0.788;CG=3.830;AA=G;HA=13.1;HE=12.3;HC=4.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=9;GESP=G:10303/A:1439;PAC=NP_002968.1,XP_005246814.1 GT:GQ:PL 0/1:43:47,0,44 0/1:22:15,0,61 1/1:10:59,9,0 1/1:10:74,9,0</p>

<p>2 178417142 rs11694988 C T 97.90 midCVG AC1=5;AF1=0.5858;DB;DP=13;DP4=2,4,3,4;FQ=78.7;MQ=50;PV4=1,0.46,0.044,0.32;SNPEFF_AMINO_ACID_CHANGE=R117H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152517.2;VDB=0.0034;DN=120;DA=C/T;GM=NM_152517.2;GL=TTC30B;FG=missense;FD=missense;AAC=ARG/HIS;PP=117/666;CDP=350;GS=29;PH=0.944;CP=0.996;CG=4.300;AA=C;HA=15.8;HE=14.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14341;GESP=T:11079/C:1927;PAC=NP_689730.2 GT:GQ:PL 0/0:11:0,15,107 0/1:18:35,0,16 1/1:8:64,9,0 1/1:6:47,6,0</p>

<p>2 207570529 rs56067462 G C 69.40 midCVG AC1=4;AF1=0.5258;DB;DP=14;DP4=2,5,4,2;FQ=67.2;MQ=60;PV4=0.29,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P122R;SNPEFF_CODON_CHANGE=cCt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DYTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001093730.1;VDB=0.0048;DN=129;DA=G/C;GM=NM_001093730.1;GL=DYTN;FG=missense;FD=missense;AAC=PRO/ARG;PP=122/579;CDP=365;GS=103;PH=1;CP=1.000;CG=4.150;AA=G;DV=by-frequency,by-cluster;DSP=18;GESP=C:854/G:11154;PAC=NP_001087199.1 GT:GQ:PL 0/1:3:0,0,0 0/0:15:0,18,106 1/1:7:57,9,0 0/1:6:54,0,1</p>

<p>2 240969312 rs10187574 A G 13.60 midCVG AC1=3;AF1=0.4226;DB;DP=12;DP4=4,3,1,4;FQ=-18.1;MQ=19;PV4=0.29,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=C179R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6B2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005853.1;VDB=0.0214;DN=119;DA=A/G;GM=NM_001005853.1,XM_005247004.1;GL=OR6B2;FG=missense;FD=missense,unknown;AAC=CYS/ARG;PP=179/313;CDP=535;GS=180;PH=1;CP=1.000;CG=4.360;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=560;KP=hsa04740:OlfactoryTransduction;TFBS=V$ATF_01,V$PPARG_01;GESP=A:12038;PAC=NP_001005853.1,XP_005247061.1 GT:GQ:PL 0/0:11:0,12,14 0/1:3:6,3,0 1/1:7:53,12,0 0/0:8:0,9,11</p>

<p>3 101283792 rs3762735 C G 93 highCVG AC1=4;AF1=0.4986;DB;DP=20;DP4=6,4,7,2;FQ=96;MQ=60;PV4=0.63,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=P56R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRMT10C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017819.3;VDB=0.0174;DN=107;DA=C/G;GM=NM_017819.3;GL=TRMT10C;FG=missense;FD=missense;AAC=PRO/ARG;PP=56/404;CDP=167;GS=103;PH=0.001;CP=0.084;CG=5.160;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=179;GESP=G:1711/C:10225;PAC=NP_060289.2 GT:GQ:PL 0/0:15:0,18,120 0/1:29:26,0,47 1/1:15:98,18,0 0/1:20:17,0,29</p>

<p>3 121208833 rs3218649 G C 208 highCVG AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3 GT:GQ:PL 0/1:37:59,0,36 0/0:43:0,48,136 1/1:20:109,21,0 1/1:11:91,12,0</p>

<p>3 185990096 rs2193587 C T 115 midCVG AC1=5;AF1=0.651;DB;DP=13;DP4=1,3,1,7;FQ=49.8;MQ=60;PV4=1,0.2,1,0.13;SNPEFF_AMINO_ACID_CHANGE=R316K;SNPEFF_CODON_CHANGE=aGa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DGKG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001346.2;VDB=0.0103;DN=96;DA=C/T;GM=NM_001080744.1,NM_001080745.1,NM_001346.2,XM_005247156.1,XM_005247157.1,XM_005247158.1;GL=DGKG;FG=missense;FD=missense,missense,missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=316/767,316/753,316/792,316/768,258/734,316/733;CDP=947,947,947,947,773,947;GS=26;PH=0;CP=1.000;CG=3.920;AA=T;HA=11.8;HE=14.2;HC=29.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=37;KP=hsa00561:GlycerolipidMetabolism|hsa00564:GlycerophospholipidMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=T:10726/C:2280;PAC=NP_001074213.1,NP_001074214.1,NP_001337.2,XP_005247213.1,XP_005247214.1,XP_005247215.1 GT:GQ:PL 0/0:3:0,6,53 0/1:37:36,0,39 1/1:11:72,12,0 1/1:6:54,6,0</p>

<p>4 2451694 rs3108494 T C 127 midCVG AC1=6;AF1=0.6856;DB;DP=10;DP4=1,1,6,2;FQ=6.6;MQ=60;PV4=1,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=C253R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CFAP99;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001193282.2;VDB=0.0035;DN=103;DA=T/C;GM=NM_001193282.1;GL=LOC402160;FG=missense;FD=missense;AAC=CYS/ARG;PP=253/647;CDP=757;GS=180;CP=0.976;CG=3.340;AA=C;DG;DV=by-frequency,by-cluster,by-1000G,by-other-pop;DSP=34;PAC=NP_001180211.1 GT:GQ:PL 0/1:20:22,0,20 0/1:5:0,3,26 1/1:7:51,6,0 1/1:15:99,15,0</p>

<p>4 8603105 rs115392829 G A 26.80 lowCVG AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA GT:GQ:PL 0/0:7:0,6,47 0/0:4:0,3,18 1/1:7:67,12,0 0/1:3:0,0,0</p>

<p>4 71232388 rs10031844 G A 39.10 lowCVG AC1=5;AF1=0.5822;DB;DP=6;DP4=0,2,2,2;FQ=11.2;MQ=58;PV4=0.47,0.22,1,0.3;SNPEFF_AMINO_ACID_CHANGE=G28R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMR3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012390.3;VDB=0.0103;DN=119;DA=G/A;GM=NM_012390.3;GL=SMR3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=28/135;CDP=82;GS=125;PH=0.001;CP=0.000;CG=-5.140;AA=G;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=28;GESP=A:3028/G:9978;PAC=NP_036522.3 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:77,12,0 0/0:5:0,6,45</p>

<p>4 77631425 rs3821979 T A 48.70 midCVG AC1=5;AF1=0.5697;DB;DP=13;DP4=2,6,0,5;FQ=52.1;G3=2.929e-30,1,6.915e-06;HWE=0.0461;MQ=60;PV4=0.49,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=L147H;SNPEFF_CODON_CHANGE=cTt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SHROOM3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020859.3;VDB=-0.0036;DN=107;DA=T/A;GM=NM_020859.3,XM_005263162.1,XM_005263163.1,XM_005263164.1,XM_005263165.1;GL=SHROOM3;FG=missense,missense,5-prime-UTR,5-prime-UTR,5-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=LEU/HIS,LEU/HIS,none,none,none;PP=147/1997,147/1997,NA,NA,NA;CDP=440,440,NA,NA,NA;GS=99,99,NA,NA,NA;PH=0.998;CP=1.000;CG=3.780;AA=T;HA=21.0;HE=7.5;HC=26.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$BRACH_01,V$PPARG_01;GESP=A:1891/T:11115;PAC=NP_065910.3,XP_005263219.1,NA,NA,NA GT:GQ:PL 0/1:35:32,0,38 0/1:21:20,0,22 1/1:4:23,3,0 0/1:20:16,0,43</p>

<p>4 88415636 rs1049544 G C 88.60 midCVG AC1=4;AF1=0.5004;DB;DP=12;DP4=2,3,5,2;FQ=51.6;MQ=60;PV4=0.56,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H106D;SNPEFF_CODON_CHANGE=Cac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPARCL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128310.2;VDB=0.0073;DN=86;DA=G/C;GM=NM_001128310.1,NM_004684.4,XM_005263272.1,XM_005263273.1;GL=SPARCL1;FG=missense,missense,missense,5-prime-UTR;FD=missense,missense,unknown,unknown;AAC=HIS/ASP,HIS/ASP,HIS/ASP,none;PP=106/665,106/665,106/665,NA;CDP=316,316,316,NA;GS=81,81,81,NA;PH=0;CP=0.015;CG=4.250;AA=C;HA=8.4;HE=33.5;HC=40.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=115;GESP=C:6754/G:6252;PAC=NP_001121782.1,NP_004675.3,XP_005263329.1,NA GT:GQ:PL 0/0:7:0,9,68 0/1:15:50,0,12 1/1:7:74,9,0 0/1:12:10,0,14</p>

<p>4 95170839 rs11722476 G A 30.20 highCVG AC1=3;AF1=0.351;DB;DP=19;DP4=5,7,0,5;FQ=32;MQ=60;PV4=0.24,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=S247N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0126;DN=120;DA=G/A;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none;PP=247/1029,247/1029,247/1027,NA;CDP=740,740,740,1066;GS=46,46,46,NA;PH=0;CP=0.987;CG=0.199;AA=G;HA=20.5;HE=45.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;GESP=A:4424/G:8574;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/0:10:0,9,68 0/1:8:7,0,75 1/1:7:69,12,0 0/0:15:0,15,90</p>

<p>4 95173779 rs7439869 T C 130 midCVG AC1=3;AF1=0.4198;DB;DP=14;DP4=2,3,3,6;FQ=65;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=V301A;SNPEFF_CODON_CHANGE=gTa/gCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMARCAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128429.2;VDB=0.0162;DN=116;DA=T/C;GM=NM_001128429.2,NM_001128430.1,NM_020159.4,NR_045644.1;GL=SMARCAD1;FG=missense,missense,missense,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant;AAC=VAL/ALA,VAL/ALA,VAL/ALA,none;PP=301/1029,301/1029,301/1027,NA;CDP=902,902,902,1228;GS=64,64,64,NA;PH=0;CP=0.012;CG=4.400;AA=C;HA=5.0;HE=41.6;HC=28.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=C:9055/T:3845;PPI=TRIM28:721;PAC=NP_001121901.1,NP_001121902.1,NP_064544.2,NA GT:GQ:PL 0/1:11:100,0,6 0/0:6:0,6,45 1/1:6:74,9,0 0/0:6:0,6,56</p>

<p>4 128996129 . G T 28.60 midCVG AC1=2;AF1=0.3067;DP=9;DP4=2,3,4,0;FQ=30.1;MQ=60;PV4=0.17,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=S8I;SNPEFF_CODON_CHANGE=aGt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARP1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018078.3;VDB=0.0001;GM=NM_001278604.1,NM_018078.3,NM_032239.3,NM_178043.2,XM_005263086.1,XM_005263087.1,XM_005263088.1,XM_005263089.1,XM_005263090.1,XM_005263091.1,XM_005263092.1,XM_005263093.1,XM_005263094.1,XM_005263095.1,XM_005263096.1,XM_005263097.1,XM_005263098.1,XM_005263099.1,XM_005263101.1;GL=LARP1B;FG=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,5-prime-UTR,missense;FD=unknown;AAC=SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,SER/ILE,none,SER/ILE;PP=8/336,8/915,8/359,8/523,209/1104,209/1091,209/1077,209/1056,209/1045,209/1001,209/977,209/977,209/760,209/760,209/741,209/720,209/711,NA,209/537;CDP=23,23,23,23,626,626,626,626,626,626,626,626,626,626,626,626,626,NA,626;GS=142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,142,NA,142;PH=0.974;CP=1.000;CG=4.530;AA=G;DSP=20;TFBS=V$HNF1_C;GESP=G:13006;PAC=NP_001265533.1,NP_060548.2,NP_115615.2,NP_835144.1,XP_005263143.1,XP_005263144.1,XP_005263145.1,XP_005263146.1,XP_005263147.1,XP_005263148.1,XP_005263149.1,XP_005263150.1,XP_005263151.1,XP_005263152.1,XP_005263153.1,XP_005263154.1,XP_005263155.1,NA,XP_005263158.1 GT:GQ:PL 0/0:8:0,6,51 0/0:6:0,3,17 1/1:6:71,12,0 0/0:8:0,6,45</p>

<p>4 155287535 rs10017772 T C 73.60 midCVG AC1=5;AF1=0.5889;DB;DP=10;DP4=2,1,5,2;FQ=5.24;MQ=60;PV4=1,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=H174R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DCHS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017639.3;VDB=0.0092;DN=119;DA=T/C;GM=NM_001142552.1,NM_017639.3;GL=DCHS2;FG=missense;FD=missense;AAC=HIS/ARG;PP=768/1370,174/2917;CDP=2303,521;GS=29;PH=0.007;CP=0.001;CG=2.100;AA=T;HA=36.4;HE=48.0;HC=45.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;TFBS=V$CART1_01;GESP=C:5817/T:7189;PAC=NP_001136024.1,NP_060109.2 GT:GQ:PL 0/1:7:54,0,3 0/1:3:0,0,0 1/1:7:60,9,0 0/0:4:0,6,37</p>

<p>4 169611811 . C T 14.60 midCVG AC1=2;AF1=0.2475;DP=11;DP4=6,2,0,3;FQ=15.7;MQ=60;PV4=0.061,1,1,0.052;SNPEFF_AMINO_ACID_CHANGE=R465C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PALLD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001166108.1;VDB=0.0128;GM=NM_001166108.1,NM_001166109.1,NM_016081.3,XM_005262861.1,XM_005262862.1,XM_005262863.1,XM_005262864.1,XM_005262865.1,XM_005262866.1;GL=PALLD;FG=missense;FD=unknown;AAC=ARG/CYS;PP=465/1124,83/778,465/1107,465/1401,465/1384,465/1348,465/1310,465/1277,83/1019;CDP=1393,247,1393,1393,1393,1393,1393,1393,247;GS=180;PH=1;CP=0.789;CG=3.960;AA=C;DSP=58;TFBS=V$TCF11MAFG_01;GESP=C:13006;PPI=EZR:846|ACTN1:846;PAC=NP_001159580.1,NP_001159581.1,NP_057165.3,XP_005262918.1,XP_005262919.1,XP_005262920.1,XP_005262921.1,XP_005262922.1,XP_005262923.1 GT:GQ:PL 0/0:7:0,3,16 0/0:9:0,6,36 1/1:3:56,9,0 0/0:18:0,15,104</p>

<p>4 187627792 rs11939575 T C 10.40 lowCVG AC1=4;AF1=0.5539;DB;DP=5;DP4=0,2,3,0;FQ=9.79;MQ=60;PV4=0.1,0.00034,1,1;SNPEFF_AMINO_ACID_CHANGE=R1064G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAT1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005245.3;VDB=0.0075;DN=120;DA=T/C;GM=NM_005245.3,XM_005262834.1,XM_005262835.1;GL=FAT1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLY;PP=1064/4589,1064/4601,1064/4601;CDP=3190;GS=125;PH=0;CP=0.635;CG=4.600;AA=C;HA=9.0;HE=23.5;HC=49.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=76;GESP=C:10198/T:2130;PPI=ENAH:846;PAC=NP_005236.2,XP_005262891.1,XP_005262892.1 GT:GQ:PL 0/0:6:0,6,45 0/1:3:0,0,0 1/1:5:47,9,0 0/1:3:0,0,0</p>

<p>5 54439466 rs1021580 G A 260 highCVG AC1=6;AF1=0.805;DB;DP=19;DP4=1,2,4,12;FQ=-17.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R121C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CDC20B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170402.1;VDB=0.0082;DN=86;DA=G/A;GM=NM_001145734.2,NM_001170402.1,NM_152623.2,XM_005248450.1;GL=CDC20B;FG=missense;FD=missense,missense,missense,unknown;AAC=ARG/CYS;PP=121/478,121/520,121/516,121/373;CDP=361;GS=180;PH=0;CP=0.001;CG=-1.690;AA=A;HA=49.1;HE=18.6;HC=28.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;GESP=A:9758/G:3248;PAC=NP_001139206.2,NP_001163873.1,NP_689836.2,XP_005248507.1 GT:GQ:PL 0/1:4:87,0,4 0/1:13:40,0,15 1/1:12:63,9,0 1/1:18:113,15,0</p>

<p>5 75003678 rs2307111 T C 211 highCVG AC1=6;AF1=0.7083;DB;DP=19;DP4=2,3,3,11;FQ=53.7;MQ=60;PV4=0.57,0.46,1,0.25;SNPEFF_AMINO_ACID_CHANGE=H36R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099271.1;VDB=0.0147;DN=100;DA=T/C;GM=NM_001099271.1,NM_152408.2,XM_005248435.1,XM_005248436.1,XM_005248437.1;GL=POC5;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=HIS/ARG;PP=36/576,11/551,9/549,36/548,36/399;CDP=107,32,26,107,107;GS=29;PH=0;CP=0.937;CG=5.850;AA=C;HA=6.7;HE=38.0;HC=46.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=23;GESP=C:6453/T:5401;PAC=NP_001092741.1,NP_689621.2,XP_005248492.1,XP_005248493.1,XP_005248494.1 GT:GQ:PL 0/1:53:47,0,73 0/1:5:0,3,22 1/1:22:132,21,0 1/1:13:79,12,0</p>

<p>5 78181477 rs1065757 C T 31.80 midCVG AC1=5;AF1=0.5709;DB;DP=14;DP4=7,0,0,4;FQ=35.4;MQ=59;PV4=0.003,0.029,0.1,0.1;SNPEFF_AMINO_ACID_CHANGE=V358M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARSB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000046.3;VDB=0.0048;DN=86;DA=C/A/G/T;GM=NM_000046.3,NM_198709.2,XM_005248505.1,XM_005248506.1;GL=ARSB;FG=missense;FD=missense,missense,unknown,unknown;AAC=VAL/MET;PP=358/534,358/414,358/534,358/405;CDP=1072;GS=21;PH=1;CP=0.410;CG=5.460;AA=C;HA=3.2;HE=44.2;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=71;KP=hsa00531:GlycosaminoglycanDegradation|hsa01100:MetabolicPathways|hsa04142:Lysosome;GESP=T:4223/C:8783;PAC=NP_000037.2,NP_942002.1,XP_005248562.1,XP_005248563.1 GT:GQ:PL 0/0:18:0,21,102 0/1:3:0,0,0 1/1:7:56,9,0 1/1:3:19,3,0</p>

<p>5 111504479 rs7703522 C T 58.60 midCVG AC1=4;AF1=0.4913;DB;DP=11;DP4=0,5,2,3;FQ=36.6;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R630H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L4A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022140.3;VDB=0.0074;DN=116;DA=C/T;GM=NM_022140.3,XM_005272043.1,XM_005272044.1;GL=EPB41L4A;FG=missense;FD=unknown;AAC=ARG/HIS;PP=630/687,602/659,257/339;CDP=1889,1805,770;GS=29;PH=1;CP=0.728;CG=3.770;AA=C;HA=4.9;HE=35.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;GESP=T:8586/C:3412;PAC=NP_071423.3,XP_005272100.1,XP_005272101.1 GT:GQ:PL 0/0:10:0,12,66 0/1:3:0,0,0 1/1:6:68,9,0 0/1:14:33,0,10</p>

<p>5 134782450 rs12520799 T A 123 midCVG AC1=4;AF1=0.5277;DB;DP=14;DP4=3,2,5,4;FQ=60;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=R117*;SNPEFF_CODON_CHANGE=Aga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C5orf20;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_130848.2;VDB=0.0030;DN=120;DA=T/A;GM=NM_130848.2;GL=C5orf20;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=117/245;CDP=349;CP=0.000;CG=-5.910;AA=A;HA=5.0;HE=48.3;HC=46.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=3190;GESP=A:7784/T:5222;PAC=NP_570900.1 GT:GQ:PL 0/1:39:61,0,36 0/1:3:0,0,0 1/1:12:105,15,0 0/0:7:0,9,64</p>

<p>EPB41L4A 5 140572763 rs2907323 C G 71.10 midCVG AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1 GT:GQ:PL 0/0:10:0,12,73 0/0:5:0,6,45 1/1:9:74,12,0 1/1:4:46,6,0</p>

<p>5 145499996 rs10988 C T 67.30 midCVG AC1=5;AF1=0.6441;DB;DP=12;DP4=4,1,4,3;FQ=17.8;MQ=60;PV4=0.58,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=R1088K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LARS;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020117.9;VDB=0.0086;DN=52;DA=C/T;GM=NM_020117.9,XM_005268458.1,XM_005268459.1,XM_005268460.1;GL=LARS;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/LYS;PP=1088/1177,1061/1150,1042/1131,1034/1123;CDP=3263,3182,3125,3101;GS=26;PH=0.073;CP=1.000;CG=4.460;AA=C;HA=1.4;HE=29.1;HC=22.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;KP=hsa00290:Valine,LeucineAndIsoleucineBiosynthesis|hsa00970:Aminoacyl-tRNABiosynthesis;GESP=T:2696/C:10310;PAC=NP_064502.9,XP_005268515.1,XP_005268516.1,XP_005268517.1 GT:GQ:PL 0/1:13:39,0,11 0/1:3:0,0,0 1/1:8:63,9,0 0/1:10:5,0,42</p>

<p>5 149006726 rs7732714 A C 48.70 midCVG AC1=4;AF1=0.5;DB;DP=10;DP4=1,4,2,3;FQ=50.5;MQ=60;PV4=1,0.047,1,0.025;SNPEFF_AMINO_ACID_CHANGE=S518R;SNPEFF_CODON_CHANGE=Agt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF37;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001669.2;VDB=0.0090;DN=116;DA=A/C;GM=NM_001001669.2,XM_005268447.1,XM_005268448.1;GL=ARHGEF37;FG=missense;FD=missense,unknown,unknown;AAC=SER/ARG;PP=518/676,518/676,304/462;CDP=1552,1552,910;GS=110;PH=0.906;CP=0.938;CG=3.930;AA=A;HA=32.4;HE=7.0;HC=11.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=88;GESP=C:1584/A:11172;PAC=NP_001001669.2,XP_005268504.1,XP_005268505.1 GT:GQ:PL 0/0:5:0,6,47 0/1:3:0,0,0 1/1:4:44,6,0 0/1:46:45,0,47</p>

<p>5 180030322 rs79620092 C T 69.90 lowCVG AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2 GT:GQ:PL 0/1:15:20,0,13 0/1:3:0,0,0 1/1:13:91,15,0 0/1:3:0,3,26</p>

<p>5 180582604 rs2546423 A G 65.20 midCVG AC1=5;AF1=0.6414;DB;DP=10;DP4=3,0,2,4;FQ=12;MQ=60;PV4=0.17,0.39,1,0.2;SNPEFF_AMINO_ACID_CHANGE=H221R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR2V2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206880.1;VDB=0.0064;DN=100;DA=A/G;GM=NM_206880.1;GL=OR2V2;FG=missense;FD=missense;AAC=HIS/ARG;PP=221/316;CDP=662;GS=29;PH=0;CP=0.001;CG=0.494;AA=G;HA=19.4;HE=41.3;HC=39.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26085;KP=hsa04740:OlfactoryTransduction;GESP=G:8019/A:4987;PAC=NP_996763.1 GT:GQ:PL 0/1:11:8,0,13 0/0:4:0,6,40 1/1:6:43,6,0 1/1:8:60,9,0</p>

<p>6 5431340 rs11243011 A G 68.10 highCVG AC1=4;AF1=0.5135;DB;DP=19;DP4=6,3,5,3;FQ=71.2;MQ=60;PV4=1,0.29,1,0.13;SNPEFF_AMINO_ACID_CHANGE=N280S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FARS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006567.3;VDB=0.0157;DN=120;DA=A/G;GM=NM_006567.3,XM_005248811.1,XM_005248812.1,XM_005248813.1;GL=FARS2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ASN/SER;PP=280/452,280/452,280/452,280/337;CDP=839;GS=46;PH=0.001;CP=1.000;CG=3.130;AA=A;HA=6.4;HE=15.1;HC=37.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=66;KP=hsa00970:Aminoacyl-tRNABiosynthesis;GESP=G:1979/A:11027;PAC=NP_006558.1,XP_005248868.1,XP_005248869.1,XP_005248870.1 GT:GQ:PL 0/1:3:0,0,0 0/0:10:0,12,63 1/1:15:94,18,0 0/1:21:18,0,80</p>

<p>6 7211818 rs1334576 G A 14 lowCVG AC1=4;AF1=0.4604;DB;DP=7;DP4=0,4,2,1;FQ=15.2;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G195R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RREB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001003699.3;VDB=0.0098;DN=88;DA=G/A;GM=NM_001003698.3,NM_001003699.3,NM_001003700.1,NM_001168344.1,XM_005249272.1,XM_005249273.1,XM_005249274.1,XM_005249275.1,XM_005249276.1,XM_005249277.1;GL=RREB1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/ARG;PP=195/1688,195/1743,195/1477,195/1688,195/1743,195/1743,195/1743,195/1743,195/1743,195/1688;CDP=583;GS=125;PH=0.998;CP=0.039;CG=2.930;AA=G;HA=35.5;HE=45.1;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=13;GESP=A:4926/G:8080;PAC=NP_001003698.1,NP_001003699.1,NP_001003700.1,NP_001161816.1,XP_005249329.1,XP_005249330.1,XP_005249331.1,XP_005249332.1,XP_005249333.1,XP_005249334.1 GT:GQ:PL 0/1:4:21,3,0 0/0:6:0,6,32 1/1:4:37,6,0 0/0:6:0,6,30</p>

<p>6 54173413 rs2297980 A G 54.70 midCVG AC1=5;AF1=0.5683;DB;DP=12;DP4=3,2,4,3;FQ=51.3;MQ=60;PV4=1,0.035,1,1;SNPEFF_AMINO_ACID_CHANGE=Q22R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0132;DN=100;DA=A/G;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=GLN/ARG;PP=22/477;CDP=65;GS=43;PH=0;CP=0.000;CG=-4.040;AA=A;HA=9.2;HE=10.5;HC=0.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=291;GESP=G:1365/A:11641;PAC=NP_055279.3 GT:GQ:PL 0/1:53:50,0,61 0/1:3:0,0,0 1/1:7:45,9,0 0/1:3:0,3,28</p>

<p>6 54219326 rs3195579 G A 24.40 midCVG AC1=4;AF1=0.4386;DB;DP=11;DP4=6,1,2,1;FQ=26.9;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=R381H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TINAG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014464.3;VDB=0.0086;DN=105;DA=G/A/C;GM=NM_014464.3;GL=TINAG;FG=missense;FD=missense;AAC=ARG/HIS;PP=381/477;CDP=1142;GS=29;PH=0;CP=0.594;CG=2.850;AA=G;HA=10.2;HE=17.7;HC=1.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=16;TFBS=V$PAX2_01;GESP=A:2158/G:10842;PAC=NP_055279.3 GT:GQ:PL 0/0:17:0,18,123 0/0:4:0,3,25 1/1:4:43,6,0 0/1:4:26,3,0</p>

<p>6 97063522 rs9373985 C G 49.80 highCVG AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1 GT:GQ:PL 0/0:9:0,9,63 0/1:27:26,0,61 1/1:13:72,18,0 0/0:35:0,36,176</p>

<p>6 117150008 rs6907580 G A 108 highCVG AC1=4;AF1=0.4941;DB;DP=17;DP4=2,6,5,3;FQ=107;MQ=60;PV4=0.31,0.089,1,0.23;SNPEFF_AMINO_ACID_CHANGE=R57*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPRC6A;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_148963.3;VDB=0.0127;DN=116;DA=G/A;GM=NM_148963.2,XM_005266863.1,XM_005266864.1;GL=GPRC6A;FG=stop-gained;FD=stop-gained,unknown,unknown;AAC=ARG/stop;PP=57/927,57/856,57/752;CDP=169;CP=0.862;CG=-2.840;AA=G;HA=11.5;HE=1.3;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=26;GESP=A:908/G:12098;PAC=NP_683766.2,XP_005266920.1,XP_005266921.1 GT:GQ:PL 0/1:15:35,0,12 0/1:21:39,0,18 1/1:9:81,12,0 0/0:15:0,18,123</p>

<p>FHL5 6 155451352 rs931312 G A 154 highCVG AC1=5;AF1=0.6241;DB;DP=19;DP4=5,1,3,10;FQ=43.7;MQ=60;PV4=0.041,0.11,1,0.15;SNPEFF_AMINO_ACID_CHANGE=R332H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TIAM2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012454.3;VDB=0.0045;DN=86;DA=G/A;GM=NM_012454.3,XM_005266927.1;GL=TIAM2;FG=missense;FD=missense,unknown;AAC=ARG/HIS;PP=332/1702,332/1731;CDP=995;GS=29;PH=1;CP=0.427;CG=4.990;AA=G;HA=9.0;HE=27.2;HC=32.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=200;KP=hsa04062:ChemokineSignalingPathway|hsa04810:RegulationOfActinCytoskeleton;GESP=A:2063/G:10943;PAC=NP_036586.2,XP_005266984.1 GT:GQ:PL 0/1:18:43,0,16 0/1:52:48,0,63 1/1:5:103,5,0 0/1:3:0,0,0</p>

<p>7 2646796 rs61729413 G A 20 lowCVG AC1=4;AF1=0.5539;DB;DP=6;DP4=2,0,0,3;FQ=12.4;MQ=60;PV4=0.1,0.33,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R635K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IQCE;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001287499.1;VDB=0.0117;DN=129;DA=G/A;GM=NM_001100390.1,NM_152558.3,XM_005249675.1,XM_005249676.1,XM_005249677.1,XM_005249678.1;GL=IQCE;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=619/680,635/696,635/703,619/687,570/638,570/631;CDP=1856,1904,1904,1856,1709,1709;GS=26;PH=0;CP=0.000;CG=-3.560;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=A:1500/G:10556;PAC=NP_001093860.1,NP_689771.3,XP_005249732.1,XP_005249733.1,XP_005249734.1,XP_005249735.1 GT:GQ:PL 0/0:6:0,6,46 0/1:3:0,0,0 1/1:6:57,9,0 0/1:3:0,0,0</p>

<p>7 4213975 rs671694 A G 129 highCVG AC1=5;AF1=0.6104;DB;DP=18;DP4=4,5,5,4;FQ=109;MQ=60;PV4=1,1,1,0.36;SNPEFF_AMINO_ACID_CHANGE=H1641R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152744.3;VDB=0.0088;DN=83;DA=A/G;GM=NM_001079653.1,NM_152744.3;GL=SDK1;FG=missense;FD=missense;AAC=HIS/ARG;PP=128/680,1641/2214;CDP=383,4922;GS=29;PH=0;CP=0.086;CG=4.910;AA=G;HA=0.0;HE=29.8;HC=23.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=G:10231/A:2775;PAC=NP_001073121.1,NP_689957.3 GT:GQ:PL 0/1:37:32,0,78 0/0:8:0,12,76 1/1:6:47,6,0 1/1:14:99,15,0</p>

<p>7 6656830 rs2243563 G A 41.40 lowCVG AC1=6;AF1=0.6978;DB;DP=7;DP4=0,2,3,1;FQ=-9.9;MQ=60;PV4=0.4,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=G8R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF853;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017560.1;VDB=0.0058;DN=100;DA=G/A;GM=NM_017560.1,XM_005249790.1;GL=ZNF853;FG=missense;FD=missense,unknown;AAC=GLY/ARG;PP=8/660,8/203;CDP=22;GS=125;PH=0.001;CP=0.000;CG=-7.700;AA=G;HA=14.0;HE=49.1;HC=26.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=10;GESP=A:2834/G:1732;PAC=NP_060030.1,XP_005249847.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:46,6,0 1/1:6:37,6,0</p>

<p>7 72397435 . A G 6.99 lowCVG AC1=5;AF1=0.625;DP=5;DP4=0,1,0,3;FQ=-8.69;MQ=50;PV4=1,0.036,0.14,1;SNPEFF_AMINO_ACID_CHANGE=K54R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POM121;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001257190.2;VDB=0.0004;GM=NM_001257190.2,NM_172020.4,XM_005250726.1,XM_005250727.1,XM_005250728.1,XM_005250729.1,XM_005250730.1,XM_005250731.1,XM_005250732.1;GL=POM121;FG=missense;FD=unknown;AAC=LYS/ARG;PP=54/1000,54/985,319/1250,319/1191,54/985,54/985,54/985,54/985,54/985;CDP=161,161,956,956,161,161,161,161,161;GS=26;PH=0.018;CP=0.966;AA=-;DSP=67;GESP=A:13006;PAC=NP_001244119.1,NP_742017.1,XP_005250783.1,XP_005250784.1,XP_005250785.1,XP_005250786.1,XP_005250787.1,XP_005250788.1,XP_005250789.1 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,3,24 1/1:5:42,9,0 0/1:3:0,0,0</p>

<p>7 99306685 rs2257401 C G 76 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:64:67,0,63</p>

<p>7 107593989 rs20556 T C 80.60 midCVG AC1=4;AF1=0.5615;DB;DP=11;DP4=1,2,3,5;FQ=27.2;MQ=60;PV4=1,0.032,1,1;SNPEFF_AMINO_ACID_CHANGE=Q1022R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LAMB1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002291.2;VDB=0.0141;DN=67;DA=T/C;GM=NM_002291.2,XM_005250339.1;GL=LAMB1;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=1022/1787,1108/1873;CDP=3065,3323;GS=43;PH=0;CP=0.289;CG=-1.990;AA=C;HA=18.6;HE=38.4;HC=16.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction|hsa05200:PathwaysInCancer|hsa05222:SmallCellLungCancer;GESP=C:8484/T:4522;PPI=LAMA4:951|LAMA5:951|LAMA3:951|LAMA1:951|LAMA2:951;PAC=NP_002282.2,XP_005250396.1 GT:GQ:PL 0/1:17:41,0,14 0/1:3:0,0,0 1/1:13:82,15,0 0/0:4:0,6,52</p>

<p>7 122635024 rs860170 C T 171 highCVG AC1=5;AF1=0.609;DB;DP=24;DP4=3,8,2,10;FQ=110;MQ=60;PV4=0.64,1,1,0.064;SNPEFF_AMINO_ACID_CHANGE=R222H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TAS2R16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016945.2;VDB=0.0143;DN=86;DA=C/T;GM=NM_016945.2;GL=TAS2R16;FG=missense;FD=missense;AAC=ARG/HIS;PP=222/292;CDP=665;GS=29;PH=0;CP=0.138;CG=2.210;AA=T;HA=3.2;HE=27.6;HC=47.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=108972;KP=hsa04742:TasteTransduction;GESP=T:9806/C:3200;PAC=NP_058641.1 GT:GQ:PL 0/1:13:69,0,12 0/0:25:0,30,144 1/1:14:84,15,0 1/1:8:68,9,0</p>

<p>8 12879539 rs608052 A G 159 highCVG AC1=5;AF1=0.5689;DB;DP=16;DP4=3,2,4,7;FQ=62.2;MQ=60;PV4=0.6,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R451G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1456;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020844.2;VDB=0.0150;DN=83;DA=A/G;GM=NM_001099677.1,NM_020844.2,XM_005273584.1,XM_005273585.1,XM_005273586.1,XM_005273587.1,XM_005273588.1,XM_005273589.1,XM_005273590.1,XM_005273591.1,XM_005273592.1,XM_005273593.1,XM_005273594.1,XM_005273595.1;GL=KIAA1456;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/GLY;PP=325/329,451/455,520/524,471/475,451/455,420/424,420/424,415/419,415/419,415/419,415/419,364/368,325/329,364/368;CDP=973,1351,1558,1411,1351,1258,1258,1243,1243,1243,1243,1090,973,1090;GS=125;PH=0;CP=0.034;CG=0.894;AA=G;HA=45.0;HE=18.9;HC=25.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1023;GESP=G:8818/A:3100;PAC=NP_001093147.1,NP_065895.2,XP_005273641.1,XP_005273642.1,XP_005273643.1,XP_005273644.1,XP_005273645.1,XP_005273646.1,XP_005273647.1,XP_005273648.1,XP_005273649.1,XP_005273650.1,XP_005273651.1,XP_005273652.1 GT:GQ:PL 0/0:9:0,12,85 0/1:3:0,3,23 1/1:8:69,9,0 1/1:22:140,24,0</p>

<p>8 32453358 rs3924999 G A 31.20 lowCVG AC1=6;AF1=0.7171;DB;DP=6;DP4=0,1,2,1;FQ=-15.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R38Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NRG1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013956.3;VDB=0.0076;DN=108;DA=G/A;GM=NM_001159995.1,NM_001159999.1,NM_001160001.1,NM_001160002.1,NM_001160004.1,NM_001160005.1,NM_001160007.1,NM_001160008.1,NM_004495.3,NM_013956.3,NM_013957.3,NM_013958.3,NM_013960.3,NM_013962.2,NM_013964.3,XM_005273488.1;GL=NRG1;FG=missense;FD=missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,unknown;AAC=ARG/GLN;PP=17/608,17/625,17/591,38/195,38/460,38/208,38/178,38/421,38/212,38/646,38/638,38/242,38/463,253/423,38/641,45/636;CDP=50,50,50,113,113,113,113,113,113,113,113,113,113,758,113,134;GS=43;PH=0.966;CP=0.998;CG=5.650;AA=G;HA=1.3;HE=43.2;HC=21.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=13;KP=hsa04012:ErbBSignalingPathway;GESP=A:3990/G:9016;PPI=ERBB3:989|ERBB4:983;PAC=NP_001153467.1,NP_001153471.1,NP_001153473.1,NP_001153474.1,NP_001153476.1,NP_001153477.1,NP_001153479.1,NP_001153480.1,NP_004486.2,NP_039250.2,NP_039251.2,NP_039252.2,NP_039254.1,NP_039256.2,NP_039258.1,XP_005273545.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,18 1/1:4:20,3,0 1/1:6:50,6,0</p>

<p>8 48739305 rs4278157 G A 137 midCVG AC1=4;AF1=0.5579;DB;DP=14;DP4=2,2,4,6;FQ=14.3;MQ=60;PV4=1,1,1,1;VDB=0.0263;DN=111;DA=G/A;GM=NM_001081640.1,NM_006904.6;GL=PRKDC;FG=missense;FD=missense;AAC=ARG/CYS;PP=2899/4098,2899/4129;CDP=8694;GS=180;PH=0.966;CP=0.005;CG=4.670;AA=G;HA=0.0;HE=5.8;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=89;KP=hsa03450:Non-homologousEnd-joining|hsa04110:CellCycle;GESP=A:442/G:12052;PAC=NP_001075109.1,NP_008835.5 GT:GQ:PL 0/0:5:0,6,35 0/1:19:21,0,18 1/1:13:158,15,0 0/1:3:0,0,0</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 67380528 rs1060242 T C 74.90 highCVG AC1=5;AF1=0.62;DB;DP=19;DP4=3,3,4,4;FQ=64;MQ=60;PV4=1,0.14,1,1;SNPEFF_AMINO_ACID_CHANGE=C449R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADHFE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_144650.2;VDB=0.0170;DN=86;DA=T/C;GM=NM_144650.2,XM_005251152.1,XM_005251153.1,XM_005251154.1,XM_005251155.1,XM_005251156.1,XM_005251157.1,XM_005251158.1,XM_005251159.1,XM_005251160.1,XM_005251161.1,XM_005251162.1,XM_005251163.1;GL=ADHFE1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=CYS/ARG;PP=449/468,430/449,401/420,401/420,401/420,401/420,401/420,379/398,401/420,219/238,219/238,219/238,219/238;CDP=1345,1288,1201,1201,1201,1201,1201,1135,1201,655,655,655,655;GS=180;PH=0;CP=0.818;CG=3.680;AA=C;HA=23.7;HE=42.6;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;GESP=C:7238/T:5768;PAC=NP_653251.2,XP_005251209.1,XP_005251210.1,XP_005251211.1,XP_005251212.1,XP_005251213.1,XP_005251214.1,XP_005251215.1,XP_005251216.1,XP_005251217.1,XP_005251218.1,XP_005251219.1,XP_005251220.1 GT:GQ:PL 0/0:8:0,12,80 0/1:21:17,0,29 1/1:8:35,9,0 1/1:11:72,12,0</p>

<p>8 124449466 rs3824250 C T 38.30 midCVG AC1=4;AF1=0.4784;DB;DP=10;DP4=5,1,1,3;FQ=41.1;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R134C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDYHV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018024.2;VDB=0.0061;DN=107;DA=C/T;GM=NM_018024.1,XM_005250997.1,XM_005250998.1;GL=WDYHV1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/CYS;PP=134/206,74/146,66/138;CDP=400,220,196;GS=180;PH=0.99;CP=1.000;CG=5.480;AA=C;HA=34.8;HE=28.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;TFBS=V$AHRARNT_02,V$TCF11MAFG_01;GESP=T:4627/C:8379;PAC=NP_060494.1,XP_005251054.1,XP_005251055.1 GT:GQ:PL 0/1:16:14,0,41 0/0:7:0,9,51 1/1:4:47,6,0 0/1:17:22,0,14</p>

<p>8 142367335 rs10875472 T C 48.60 lowCVG AC1=6;AF1=0.6934;DB;DP=6;DP4=1,0,3,2;FQ=-18.7;MQ=60;PV4=1,0.17,1,0.27;SNPEFF_AMINO_ACID_CHANGE=H230R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005293.2;VDB=0.0059;DN=120;DA=T/C;GM=NM_005293.2;GL=GPR20;FG=missense;FD=missense;AAC=HIS/ARG;PP=230/359;CDP=689;GS=29;PH=0;CP=0.989;CG=-0.037;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=713;CPG=142367051-142367860;GESP=C:10416/T:2188;PAC=NP_005284.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:21,3,0 0/1:13:62,0,11</p>

<p>8 142517227 rs2613648 C T 79.90 midCVG AC1=5;AF1=0.6558;DB;DP=13;DP4=3,2,8,0;FQ=19.6;MQ=59;PV4=0.13,1,0.23,0.41;SNPEFF_AMINO_ACID_CHANGE=R8K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MROH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_207414.2;VDB=0.0030;DN=100;DA=C/T;GM=NM_207414.2,NR_102363.1,NR_102364.1;GL=MROH5;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ARG/LYS,none,none;PP=8/1319,NA,NA;CDP=23,104,104;GS=26,NA,NA;CP=0.000;CG=-2.700;AA=C;HA=21.0;HE=49.1;HC=36.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=58;GESP=T:4996/C:7690;PAC=NP_997297.2,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:21:47,0,20 1/1:11:61,12,0 0/1:18:13,0,36</p>

<p>9 26978259 rs11555693 A G 90.20 highCVG AC1=6;AF1=0.7286;DB;DP=22;DP4=0,6,4,5;FQ=48.8;MQ=60;PV4=0.1,0.013,1,0.18;SNPEFF_AMINO_ACID_CHANGE=K85R;SNPEFF_CODON_CHANGE=aAa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IFT74;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001099222.1;VDB=0.0093;DN=120;DA=A/G;GM=NM_001099222.1,NM_001099223.1,NM_001099224.1,NM_025103.2;GL=IFT74;FG=missense;FD=missense;AAC=LYS/ARG;PP=85/601,85/601,85/373,85/601;CDP=254;GS=26;PH=0.459;CP=1.000;CG=5.620;AA=A;DV=by-frequency,by-cluster;DSP=3;GESP=G:735/A:11169;PAC=NP_001092692.1,NP_001092693.1,NP_001092694.1,NP_079379.2 GT:GQ:PL 0/1:78:77,0,80 0/1:3:0,0,0 1/1:9:50,9,0 0/1:3:0,0,0</p>

<p>9 33385863 rs62542746 G T 17.60 highCVG AC1=5;AF1=0.6224;DB;DP=17;DP4=2,5,5,5;FQ=21.6;MQ=33;PV4=0.62,1,1.5e-05,0.36;SNPEFF_AMINO_ACID_CHANGE=A176E;SNPEFF_CODON_CHANGE=gCg/gAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0114;DN=129;DA=G/T;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense-near-splice;FD=missense,unknown,unknown;AAC=ALA/GLU;PP=176/343,176/343,176/274;CDP=527;GS=107;PH=0.006;CP=0.040;CG=1.570;AA=T;DV=by-cluster;DSP=2;KP=hsa03320:PPARSignalingPathway;GESP=G:13002;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:11:7,0,29 0/0:4:0,5,74 1/1:5:18,6,0 1/1:5:37,6,0</p>

<p>9 33387047 rs4008658 T G 66.90 highCVG AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:27:37,0,25 0/1:17:22,0,15 1/1:10:52,12,0 0/1:4:0,2,50</p>

<p>9 88651370 rs2297002 T C 82.10 midCVG AC1=3;AF1=0.3754;DB;DP=12;DP4=5,1,4,2;FQ=82;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=H217R;SNPEFF_CODON_CHANGE=cAc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GOLM1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016548.3;VDB=0.0035;DN=100;DA=T/C;GM=NM_016548.3,NM_177937.2,XM_005252040.1,XM_005252041.1,XM_005252042.1,XM_005252043.1,XM_005252044.1;GL=GOLM1;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=HIS/ARG;PP=217/402,217/402,217/459,217/459,217/403,217/385,217/385;CDP=650;GS=29;PH=0;CP=0.000;CG=-3.200;AA=C;HA=49.2;HE=2.5;HC=39.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=53;GESP=C:1785/T:11221;PAC=NP_057632.2,NP_808800.1,XP_005252097.1,XP_005252098.1,XP_005252099.1,XP_005252100.1,XP_005252101.1 GT:GQ:PL 0/0:12:0,12,76 0/0:7:0,6,51 1/1:12:125,18,0 0/1:3:0,0,0</p>

<p>9 104433235 rs10989589 C T 82.80 highCVG AC1=4;AF1=0.5477;DB;DP=16;DP4=2,2,4,3;FQ=34.3;MQ=60;PV4=1,0.22,1,0.16;SNPEFF_AMINO_ACID_CHANGE=G487R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GRIN3A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133445.2;VDB=0.0068;DN=120;DA=C/T;GM=NM_133445.2;GL=GRIN3A;FG=missense;FD=missense;AAC=GLY/ARG;PP=487/1116;CDP=1459;GS=125;PH=0.956;CP=0.252;CG=5.760;AA=C;HA=10.0;HE=35.8;HC=20.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=155;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=T:4290/C:8716;PPI=GRIN1:846|CPEB1:830;PAC=NP_597702.2 GT:GQ:PL 0/0:5:0,6,49 0/1:3:0,0,0 1/1:10:83,12,0 0/1:27:42,0,24</p>

<p>9 117050998 rs10982134 G A 55.90 lowCVG AC1=6;AF1=0.7419;DB;DP=6;DP4=1,0,3,1;FQ=-9.04;MQ=60;PV4=1,0.38,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R1354Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL27A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032888.2;VDB=0.0146;DN=120;DA=G/A;GM=NM_032888.2;GL=COL27A1;FG=missense;FD=missense;AAC=ARG/GLN;PP=1354/1861;CDP=4061;GS=43;PH=0.016;CP=0.999;CG=1.840;AA=G;HA=21.2;HE=25.9;HC=38.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=A:3908/G:9098;PAC=NP_116277.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,26 1/1:7:48,6,0 1/1:7:48,6,0</p>

<p>9 117846580 rs1061494 T C 25.30 midCVG AC1=5;AF1=0.6256;DB;DP=14;DP4=4,4,2,3;FQ=29.5;MQ=60;PV4=1,0.0026,1,0.016;SNPEFF_AMINO_ACID_CHANGE=Q680R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TNC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002160.3;VDB=0.0126;DN=86;DA=T/C;GM=NM_002160.3,XM_005251972.1,XM_005251973.1,XM_005251974.1,XM_005251975.1;GL=TNC;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=680/2202,680/2111,680/1838,680/1656,680/1565;CDP=2039;GS=43;PH=0.001;CP=0.977;CG=1.040;AA=C;HA=39.1;HE=44.2;HC=26.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=93;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:6253/T:6753;PPI=TNNC1:846;PAC=NP_002151.2,XP_005252029.1,XP_005252030.1,XP_005252031.1,XP_005252032.1 GT:GQ:PL 0/1:22:18,0,120 0/1:3:0,0,0 1/1:6:35,6,0 0/1:15:12,0,20</p>

<p>9 125315557 rs1831370 T C 80.20 highCVG AC1=4;AF1=0.548;DB;DP=15;DP4=2,5,4,3;FQ=74.2;MQ=60;PV4=0.59,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W37R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1N2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004457.1;VDB=0.0200;DN=92;DA=T/C;GM=NM_001004457.1;GL=OR1N2;FG=missense;FD=missense;AAC=TRP/ARG;PP=37/331;CDP=109;GS=101;PH=0;CP=0.002;CG=-2.620;AA=C;HA=41.0;HE=45.5;HC=40.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=42490;KP=hsa04740:OlfactoryTransduction;GESP=C:6740/T:6266;PAC=NP_001004457.1 GT:GQ:PL 0/1:3:0,3,19 0/0:5:0,7,98 1/1:12:96,15,0 1/1:3:30,3,0</p>

<p>9 125330125 rs10739614 C G 163 highCVG AC1=6;AF1=0.7385;DB;DP=20;DP4=0,5,11,3;FQ=29.2;MQ=60;PV4=0.0048,1,1,0.19;SNPEFF_AMINO_ACID_CHANGE=R211P;SNPEFF_CODON_CHANGE=cGt/cCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0083;DN=120;DA=C/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=ARG/PRO;PP=211/310;CDP=632;GS=103;PH=0;CP=0.008;CG=3.600;AA=G;HA=32.8;HE=45.0;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57058;KP=hsa04740:OlfactoryTransduction;GESP=G:8273/C:4733;PAC=NP_001004454.1 GT:GQ:PL 0/1:12:16,0,13 0/1:46:40,0,55 1/1:8:43,6,0 1/1:25:109,24,0</p>

<p>9 125391241 rs1476860 G A 91.10 midCVG AC1=6;AF1=0.6853;DB;DP=9;DP4=2,1,1,5;FQ=19.3;MQ=60;PV4=0.23,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R192*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1B1;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001004450.1;VDB=0.0081;DN=88;DA=G/A;GM=NM_001004450.1;GL=OR1B1;FG=stop-gained;FD=stop-gained;AAC=ARG/stop;PP=192/319;CDP=574;CP=0.142;CG=-6.050;AA=G;HA=3.8;HE=30.8;HC=46.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=118174;KP=hsa04740:OlfactoryTransduction;GESP=A:2952/G:10054;PAC=NP_001004450.1 GT:GQ:PL 0/0:5:0,9,58 0/1:3:0,0,0 1/1:6:45,6,0 1/1:11:90,12,0</p>

<p>OR1L8 9 140201604 rs28685844 C T 15.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=3,0,1,1;FQ=15;MQ=60;PV4=0.4,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R810Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXD3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017820.4;VDB=0.0130;DN=125;DA=C/T;GM=NM_017820.3,XM_005266091.1,XM_005266092.1;GL=EXD3;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=810/877,490/557,448/515;CDP=2429,1469,1343;GS=43;PH=0.193;CP=0.000;CG=-3.990;AA=C;HA=3.6;HE=0.4;HC=8.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=112;GESP=T:123/C:12191;PAC=NP_060290.3,XP_005266148.1,XP_005266149.1 GT:GQ:PL 0/1:3:0,0,0 0/0:9:0,9,51 1/1:3:52,6,0 0/1:3:0,0,0</p>

<p>10 5415954 rs10904481 A G 49.50 lowCVG AC1=6;AF1=0.685;DB;DP=8;DP4=0,3,1,4;FQ=13.7;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R91G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_053049.2;VDB=0.0100;DN=120;DA=A/G;GM=NM_053049.2;GL=UCN3;FG=missense;FD=missense;AAC=ARG/GLY;PP=91/162;CDP=271;GS=125;PH=0;CP=0.000;CG=2.520;AA=G;HA=29.2;HE=35.7;HC=48.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=277;KP=hsa04080:NeuroactiveLigand-receptorInteraction;GESP=G:4033/A:8969;PAC=NP_444277.2 GT:GQ:PL 0/1:17:11,0,30 0/1:19:16,0,21 1/1:7:46,6,0 1/1:5:19,3,0</p>

<p>10 69933969 rs7916821 G A 187 highCVG AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA GT:GQ:PL 0/1:45:68,0,45 0/1:5:0,3,28 1/1:15:86,15,0 1/1:13:80,12,0</p>

<p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

<p>11 5528918 rs2227271 C T 10.70 lowCVG AC1=5;AF1=0.5785;DB;DP=5;DP4=0,1,2,0;FQ=-6.4;MQ=60;PV4=0.33,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=R624Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UBQLN3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017481.2;VDB=0.0061;DN=98;DA=C/T;GM=NM_017481.2;GL=UBQLN3;FG=missense;FD=missense;AAC=ARG/GLN;PP=624/656;CDP=1871;GS=43;PH=0.045;CP=0.213;CG=1.230;AA=C;HA=6.4;HE=37.7;HC=6.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=1907;GESP=T:3709/C:9287;PAC=NP_059509.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:45,6,0 0/0:4:0,3,26</p>

<p>11 5809673 rs10769224 C T 24 highCVG AC1=2;AF1=0.2252;DB;DP=16;DP4=3,9,2,1;FQ=25;MQ=60;PV4=0.24,0.058,1,0.012;SNPEFF_AMINO_ACID_CHANGE=C125Y;SNPEFF_CODON_CHANGE=tGt/tAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52N1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001913.1;VDB=0.0010;DN=120;DA=C/T;GM=NM_001001913.1;GL=OR52N1;FG=missense;FD=missense;AAC=CYS/TYR;PP=125/321;CDP=374;GS=194;PH=0;CP=0.875;CG=2.130;AA=T;HA=49.4;HE=27.8;HC=17.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15427;KP=hsa04740:OlfactoryTransduction;TFBS=V$PPARG_01;GESP=T:4534/C:8460;PAC=NP_001001913.1 GT:GQ:PL 0/0:24:0,21,130 0/0:7:0,3,25 1/1:3:66,9,0 0/0:15:0,12,88</p>

<p>11 5906205 rs4757987 G A 63.50 midCVG AC1=4;AF1=0.4597;DB;DP=12;DP4=3,4,2,3;FQ=66.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R228H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52E4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005165.1;VDB=0.0135;DN=111;DA=G/A;GM=NM_001005165.1;GL=OR52E4;FG=missense;FD=missense;AAC=ARG/HIS;PP=228/313;CDP=683;GS=29;PH=0.003;CP=0.023;CG=0.047;AA=G;HA=47.5;HC=42.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=64694;KP=hsa04740:OlfactoryTransduction;GESP=A:5225/G:7769;PAC=NP_001005165.1 GT:GQ:PL 0/0:6:0,6,44 0/1:4:25,3,0 1/1:8:86,12,0 0/0:13:0,15,95</p>

<p>11 6007680 rs4436524 A G 189 highCVG AC1=6;AF1=0.7132;DB;DP=17;DP4=0,2,8,7;FQ=-4.85;MQ=47;PV4=0.47,0.4,0.081,0.3;SNPEFF_AMINO_ACID_CHANGE=C161R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0140;DN=111;DA=A/G/T;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=CYS/ARG;PP=161/330;CDP=481;GS=180;PH=0;CP=0.012;CG=0.241;AA=G;HA=43.3;HE=15.3;HC=11.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=166169;KP=hsa04740:OlfactoryTransduction;GESP=G:9200/A:3246;PAC=NP_001005173.2 GT:GQ:PL 0/1:16:39,0,17 0/1:5:0,3,21 1/1:13:53,12,0 1/1:28:144,27,0</p>

<p>11 6341684 rs2682123 C G 35.60 lowCVG AC1=6;AF1=0.7558;DB;DP=5;DP4=0,1,2,2;FQ=-21.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R8P;SNPEFF_CODON_CHANGE=cGg/cCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRKCDBP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145040.2;VDB=0.0113;DN=100;DA=C/G;GM=NM_145040.2;GL=PRKCDBP;FG=missense;FD=missense;AAC=ARG/PRO;PP=8/262;CDP=23;GS=103;PH=0;CP=0.004;CG=2.010;AA=G;HA=13.6;HE=1.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=362;CPG=6340446-6341909;GESP=G:11875/C:705;PAC=NP_659477.2 GT:GQ:PL 0/1:10:28,0,9 0/1:3:0,0,0 1/1:6:43,6,0 0/1:3:0,0,0</p>

<p>11 47270255 rs2167079 C T 15.90 midCVG AC1=3;AF1=0.4192;DB;DP=11;DP4=4,2,1,2;FQ=18.3;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R29Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001610.2;VDB=0.0055;DN=96;DA=C/T;GM=NM_001131064.1,NM_001251934.1,NM_001251935.1,NM_001610.2,XM_005252982.1,XM_005252983.1,XM_005252984.1,XM_005252985.1;GL=ACP2/NR1H3;FG=missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,5-prime-UTR,missense;FD=missense,utr-variant-5-prime,utr-variant-5-prime,missense,unknown,unknown,unknown,unknown;AAC=ARG/GLN,none,none,ARG/GLN,ARG/GLN,ARG/GLN,none,ARG/GLN;PP=29/161,NA,NA,29/424,29/392,29/361,NA,29/218;CDP=86,NA,NA,86,86,86,NA,86;GS=43,NA,NA,43,43,43,NA,43;PH=0.003;CP=0.987;CG=5.690;AA=T;HA=38.3;HE=31.2;HC=27.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;KP=hsa00740:RiboflavinMetabolism|hsa04142:Lysosome|hsa03320:PPARSignalingPathway;CPG=47270235-47270633;TFBS=V$AHR_01;GESP=T:4930/C:8048;PPI=RXRA:937|RXRB:900;PAC=NP_001124536.1,NA,NA,NP_001601.1,XP_005253039.1,XP_005253040.1,NA,XP_005253042.1 GT:GQ:PL 0/1:4:24,3,0 0/0:6:0,6,30 1/1:3:37,6,0 0/0:11:0,12,67</p>

<p>11 55406524 rs61896975 C T 78.80 highCVG AC1=4;AF1=0.4977;DB;DP=15;DP4=2,3,1,9;FQ=54.7;MQ=60;PV4=0.24,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R231C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4P4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004124.1;VDB=0.0025;DN=129;DA=C/T;GM=NM_001004124.1;GL=OR4P4;FG=missense;FD=missense;AAC=ARG/CYS;PP=231/313;CDP=691;GS=180;PH=0.407;CP=0.000;CG=-1.540;AA=C;DG;DV=by-cluster,by-1000G;DSP=244337;KP=hsa04740:OlfactoryTransduction;GESP=T:719/C:11695;PAC=NP_001004124.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,55 1/1:4:102,6,0 0/1:19:17,0,37</p>

<p>11 55541620 rs7124871 G T 59.50 midCVG AC1=4;AF1=0.4901;DB;DP=13;DP4=3,4,4,2;FQ=62.4;MQ=59;PV4=0.59,1,0.15,1;SNPEFF_AMINO_ACID_CHANGE=R236L;SNPEFF_CODON_CHANGE=cGc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5D13;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001967.1;VDB=0.0105;DN=116;DA=G/A/T;GM=NM_001001967.1;GL=OR5D13;FG=missense;FD=missense;AAC=ARG/LEU;PP=236/315;CDP=707;GS=102;PH=0.01;CP=0.000;CG=2.850;AA=G;HA=0.0;HE=7.6;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=109241;KP=hsa04740:OlfactoryTransduction;GESP=T:576/G:12416;PAC=NP_001001967.1 GT:GQ:PL 0/1:3:0,0,0 0/0:13:0,15,100 1/1:9:74,12,0 0/1:31:29,0,37</p>

<p>11 55703861 rs17597625 T C 22.50 midCVG AC1=4;AF1=0.508;DB;DP=12;DP4=3,1,4,0;FQ=25.4;MQ=60;PV4=1,0.019,1,0.18;SNPEFF_AMINO_ACID_CHANGE=R6G;SNPEFF_CODON_CHANGE=Aga/Gga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR5I1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006637.1;VDB=0.0044;DN=123;DA=T/C;GM=NM_006637.1;GL=OR5I1;FG=missense;FD=missense;AAC=ARG/GLY;PP=6/315;CDP=16;GS=125;PH=0;CP=0.238;CG=4.060;AA=C;HA=0.6;HE=10.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=43693;KP=hsa04740:OlfactoryTransduction;GESP=C:787/T:12195;PAC=NP_006628.1 GT:GQ:PL 0/0:8:0,9,62 0/0:3:0,3,22 1/1:4:30,6,0 1/1:4:38,6,0</p>

<p>11 61165741 rs10897158 G C 74.40 midCVG AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1 GT:GQ:PL 0/1:10:35,0,8 0/0:6:0,9,70 1/1:4:24,3,0 1/1:8:61,9,0</p>

<p>11 87013438 rs2276102 G A 98 midCVG AC1=6;AF1=0.7212;DB;DP=11;DP4=0,2,5,3;FQ=-16.6;MQ=60;PV4=0.44,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=G218R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMEM135;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022918.3;VDB=0.0208;DN=100;DA=G/A;GM=NM_001168724.1,NM_022918.3,NR_033149.1;GL=TMEM135;FG=missense,missense,non-coding-exon;FD=missense,missense,nc-transcript-variant;AAC=GLY/ARG,GLY/ARG,none;PP=196/437,218/459,NA;CDP=586,652,820;GS=125,125,NA;PH=0;CP=0.664;CG=4.310;AA=A;HA=43.6;HE=31.1;HC=46.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;GESP=A:8629/G:4371;PAC=NP_001162195.1,NP_075069.3,NA GT:GQ:PL 0/1:14:15,0,14 0/1:3:0,0,0 1/1:5:26,3,0 0/1:3:93,0,0</p>

<p>11 89531540 rs12417980 C T 84.10 highCVG AC1=6;AF1=0.7306;DB;DP=18;DP4=3,3,3,9;FQ=8.51;MQ=21;PV4=0.34,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G373R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM49;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020358.2;VDB=0.0085;DN=120;DA=C/T;GM=NM_020358.2;GL=TRIM49;FG=missense;FD=missense;AAC=GLY/ARG;PP=373/453;CDP=1117;GS=125;PH=0.834;CP=0.000;CG=-1.800;AA=C;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=258;GESP=T:3992/C:8964;PAC=NP_065091.1 GT:GQ:PL 0/1:4:13,0,1 0/1:3:0,0,0 1/1:8:80,7,0 0/1:34:29,0,39</p>

<p>11 124908352 rs7107487 T C 999 highCVG AC1=4;AF1=0.5057;DB;DP=29;DP4=6,9,4,10;FQ=205;MQ=60;PV4=0.7,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=C813R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025004.2;VDB=0.0034;DN=116;DA=T/C;GM=NM_025004.2;GL=CCDC15;FG=missense;FD=missense;AAC=CYS/ARG;PP=813/952;CDP=2437;GS=180;PH=0;CP=0.973;CG=3.110;AA=C;HA=36.7;HE=12.8;HC=17.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;GESP=C:2453/T:9283;PAC=NP_079280.2 GT:GQ:PL 0/1:48:45,0,74 0/1:63:60,0,82 1/1:18:134,21,0 0/0:12:0,15,94</p>

<p>12 53605545 rs2229774 G A 41.20 midCVG AC1=4;AF1=0.5599;DB;DP=12;DP4=0,3,0,9;FQ=13.2;MQ=60;PV4=1,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=S427L;SNPEFF_CODON_CHANGE=tCg/tTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RARG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000966.5;VDB=0.0000;DN=98;DA=G/A;GM=NM_000966.5,NM_001042728.2,NM_001243730.1,NM_001243731.1,NM_001243732.1,XM_005269054.1,XM_005269055.1,XM_005269056.1,XM_005269057.1;GL=RARG;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown;AAC=SER/LEU;PP=427/455,416/444,355/383,306/334,405/433,514/542,514/542,427/455,427/455;CDP=1280,1247,1064,917,1214,1541,1541,1280,1280;GS=145;PH=0.525;CP=1.000;CG=4.530;AA=G;HA=11.1;HE=6.2;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=103;TFBS=V$ARP1_01;GESP=A:949/G:12057;PAC=NP_000957.1,NP_001036193.1,NP_001230659.1,NP_001230660.1,NP_001230661.1,XP_005269111.1,XP_005269112.1,XP_005269113.1,XP_005269114.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:24:81,27,0 0/0:7:0,9,49</p>

<p>12 117383320 rs4076700 G A 78.70 midCVG AC1=5;AF1=0.6582;DB;DP=13;DP4=3,1,7,1;FQ=32.3;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R192Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FBXW8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153348.2;VDB=0.0037;DN=108;DA=G/A;GM=NM_012174.1,NM_153348.2,XM_005253866.1,XM_005253867.1,XM_005253868.1;GL=FBXW8;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=ARG/GLN,ARG/GLN,ARG/GLN,GLU/LYS,GLU/LYS;PP=126/533,192/599,192/423,41/418,41/418;CDP=377,575,575,121,121;GS=43,43,43,56,56;PH=1;CP=0.990;CG=5.190;AA=G;HA=22.8;HE=17.4;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=14;KP=hsa04120:UbiquitinMediatedProteolysis;GESP=A:8670/G:4336;PPI=SKP1:994|CUL7:994|CUL1:932|RBX1:929;PAC=NP_036306.1,NP_699179.2,XP_005253923.1,XP_005253924.1,XP_005253925.1 GT:GQ:PL 0/1:23:18,0,39 0/1:3:0,0,0 1/1:17:93,18,0 0/1:14:9,0,30</p>

<p>13 103384824 rs9300756 G A 50.40 highCVG AC1=3;AF1=0.3894;DB;DP=15;DP4=2,6,5,0;FQ=52.5;MQ=60;PV4=0.021,1,1,0.032;SNPEFF_AMINO_ACID_CHANGE=R6075C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC168;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001146197.1;VDB=0.0090;DN=119;DA=G/A;GM=NM_001146197.1;GL=CCDC168;FG=missense;FD=missense;AAC=ARG/CYS;PP=6075/7082;CDP=18223;GS=180;PH=0;CP=0.000;CG=0.010;AA=G;HA=35.0;HE=41.0;HC=16.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17915;GESP=A:2470/G:2094;PAC=NP_001139669.1 GT:GQ:PL 0/0:14:0,15,100 0/0:6:0,6,34 1/1:8:75,12,0 0/1:20:22,0,19</p>

<p>13 113479820 rs368865 A G 144 highCVG AC1=4;AF1=0.5148;DB;DP=25;DP4=6,8,6,5;FQ=147;MQ=60;PV4=0.7,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M317V;SNPEFF_CODON_CHANGE=Atg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATP11A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032189.3;VDB=0.0062;DN=80;DA=A/C/G/T;GM=NM_015205.2,NM_032189.3,XM_005268299.1,XM_005268300.1,XM_005268301.1,XM_005268302.1,XM_005268303.1,XM_005268304.1,XM_005268305.1,XM_005268306.1;GL=ATP11A;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/VAL;PP=317/1135,317/1192,317/1193,317/1172,317/1140,317/1140,317/1140,317/1140,317/1135,317/1135;CDP=949;GS=21;PH=0;CP=0.993;CG=5.140;AA=G;HE=25.2;HC=12.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=75;TFBS=V$PPARG_02;GESP=G:10386/A:2620;PAC=NP_056020.2,NP_115565.3,XP_005268356.1,XP_005268357.1,XP_005268358.1,XP_005268359.1,XP_005268360.1,XP_005268361.1,XP_005268362.1,XP_005268363.1 GT:GQ:PL 0/0:10:0,13,159 0/1:3:0,0,0 1/1:21:145,24,0 0/1:46:43,0,56</p>

<p>14 20404736 rs2792146 G A 143 highCVG AC1=4;AF1=0.4995;DB;DP=17;DP4=3,1,8,3;FQ=42.3;MQ=60;PV4=1,0.37,1,1;SNPEFF_AMINO_ACID_CHANGE=R304H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004063.2;VDB=0.0150;DN=100;DA=G/A;GM=NM_001004063.2;GL=OR4K1;FG=missense;FD=missense;AAC=ARG/HIS;PP=304/312;CDP=911;GS=29;PH=0.003;CP=0.000;CG=-4.590;AA=G;HA=23.3;HC=30.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=222291;KP=hsa04740:OlfactoryTransduction;GESP=A:4807/G:8199;PAC=NP_001004063.2 GT:GQ:PL 0/1:34:39,0,31 0/0:3:0,3,28 1/1:21:149,24,0 0/0:3:0,3,25</p>

<p>14 20482919 rs17308108 A C 27.60 highCVG AC1=2;AF1=0.2756;DB;DP=32;DP4=12,11,5,1;FQ=28.9;MQ=60;PV4=0.35,4.3e-07,1,0.024;SNPEFF_AMINO_ACID_CHANGE=L145R;SNPEFF_CODON_CHANGE=cTg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4K14;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004712.1;VDB=0.0161;DN=123;DA=A/C;GM=NM_001004712.1;GL=OR4K14;FG=missense;FD=missense;AAC=LEU/ARG;PP=145/311;CDP=434;GS=102;PH=0.994;CP=0.198;CG=4.040;AA=A;HA=5.8;HE=10.2;HC=11.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=274987;KP=hsa04740:OlfactoryTransduction;GESP=C:894/A:12112;PAC=NP_001004712.1 GT:GQ:PL 0/0:44:0,42,160 0/0:6:0,3,19 1/1:11:73,18,0 0/0:26:0,24,111</p>

<p>14 58681951 . T C 41 lowCVG AC1=4;AF1=0.5652;DP=8;DP4=1,3,1,3;FQ=43;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=C183R;SNPEFF_CODON_CHANGE=Tgt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACTR10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018477.2;VDB=0.0091;GM=NM_018477.2;GL=ACTR10;FG=missense;FD=unknown;AAC=CYS/ARG;PP=183/418;CDP=547;GS=180;PH=0.427;CP=1.000;CG=5.850;AA=T;DSP=29;TFBS=V$SOX5_01,V$SOX9_B1;GESP=T:13006;PAC=NP_060947.1 GT:GQ:PL 0/1:33:31,0,37 0/0:4:0,6,50 1/1:3:26,3,0 1/1:3:27,3,0</p>

<p>14 92465749 rs35007347 C T 31.40 highCVG AC1=3;AF1=0.362;DB;DP=18;DP4=9,5,2,2;FQ=33.3;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=R1576H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=13;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIP11;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004239.3;VDB=0.0065;DN=126;DA=C/T;GM=NM_004239.3,XM_005268213.1,XM_005268214.1,XM_005268215.1;GL=TRIP11;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ARG/HIS;PP=1576/1980,1547/1951,1134/1538,566/970;CDP=4727,4640,3401,1697;GS=29;PH=1;CP=0.895;CG=5.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=29;GESP=T:254/C:12752;PAC=NP_004230.2,XP_005268270.1,XP_005268271.1,XP_005268272.1 GT:GQ:PL 0/0:30:0,30,154 0/1:36:36,0,57 1/1:3:39,6,0 0/0:5:0,3,21</p>

<p>14 102815042 rs7011 C T 107 midCVG AC1=5;AF1=0.6413;DB;DP=13;DP4=0,5,3,5;FQ=6.84;MQ=60;PV4=0.23,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R164H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CINP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032630.2;VDB=0.0109;DN=52;DA=C/T;GM=NM_032630.2,XM_005267753.1;GL=CINP;FG=missense,3-prime-UTR;FD=missense,unknown;AAC=ARG/HIS,none;PP=164/213,NA;CDP=491,NA;GS=29,NA;PH=0.054;CP=0.701;CG=2.760;AA=C;HA=5.8;HE=31.7;HC=11.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=T:3032/C:9974;PAC=NP_116019.1,NA GT:GQ:PL 0/1:3:0,0,0 0/1:12:53,0,11 1/1:6:48,6,0 0/1:31:46,0,30</p>

<p>14 105413204 rs2582514 G T 89 midCVG AC1=6;AF1=0.707;DB;DP=10;DP4=1,2,5,2;FQ=7.98;MQ=55;PV4=0.5,1,0.15,0.061;SNPEFF_AMINO_ACID_CHANGE=R2862S;SNPEFF_CODON_CHANGE=Cgc/Agc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AHNAK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138420.2;VDB=0.0059;DN=100;DA=G/T;GM=NM_138420.2,XM_005267299.1;GL=AHNAK2;FG=missense;FD=missense,unknown;AAC=ARG/SER;PP=2862/5796,2762/5696;CDP=8584,8284;GS=110;PH=0;CP=0.000;CG=0.525;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=7933;GESP=T:7253/G:4947;PAC=NP_612429.2,XP_005267356.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:46,6,0 0/1:39:79,0,38</p>

<p>15 29418520 rs2256273 C T 23.10 lowCVG AC1=5;AF1=0.5568;DB;DP=8;DP4=4,1,3,0;FQ=26.6;MQ=60;PV4=1,0.058,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R393H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM189A1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015307.1;VDB=0.0105;DN=100;DA=C/T;GM=NM_015307.1,XM_005254261.1,XM_005254262.1;GL=FAM189A1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=393/540,430/577,252/399;CDP=1178,1289,755;GS=29;PH=0.005;CP=0.000;CG=-2.140;AA=C;HE=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=51;TFBS=V$P300_01;GESP=T:3037/C:1529;PAC=NP_056122.1,XP_005254318.1,XP_005254319.1 GT:GQ:PL 0/1:3:22,3,0 0/1:3:0,0,1/1:5:43,6,0 0/0:12:0,15,102</p>

<p>15 42154738 rs16972193 G A 26.10 midCVG AC1=5;AF1=0.5822;DB;DP=9;DP4=2,0,4,0;FQ=9.38;MQ=60;PV4=1,0.26,1,1;SNPEFF_AMINO_ACID_CHANGE=R2462W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016642.3;VDB=0.0109;DN=123;DA=G/A;GM=NM_016642.3,XM_005254438.1;GL=SPTBN5;FG=missense-near-splice;FD=missense,unknown;AAC=ARG/TRP;PP=2462/3675,2482/3695;CDP=7384,7444;GS=101;PH=0.213;CP=0.837;CG=-4.960;AA=G;HE=13.8;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=2;GESP=A:1149/G:11369;PAC=NP_057726.4,XP_005254495.1 GT:GQ:PL 0/0:5:0,6,43 0/1:3:0,0,0 1/1:9:64,12,0 0/1:3:0,0,0</p>

<p>15 42570718 rs8024732 A G 106 highCVG AC1=4;AF1=0.501;DB;DP=26;DP4=3,9,5,8;FQ=108;MQ=60;PV4=0.67,0.04,1,1;SNPEFF_AMINO_ACID_CHANGE=Q44R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GANC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198141.2;VDB=0.0051;DN=116;DA=A/G;GM=NM_198141.2,XM_005254286.1;GL=GANC;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=44/915,44/913;CDP=131;GS=43;PH=0;CP=0.454;CG=-2.850;AA=A;HA=31.5;HE=11.0;HC=11.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;KP=hsa00052:GalactoseMetabolism|hsa00500:StarchAndSucroseMetabolism|hsa01100:MetabolicPathways;GESP=G:9331/A:3673;PAC=NP_937784.2,XP_005254343.1 GT:GQ:PL 0/1:13:11,0,15 0/0:11:0,14,130 1/1:12:59,15,0 0/1:18:83,0,15</p>

<p>15 65684513 rs148990036 C T 29.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,3,0;FQ=21.1;MQ=59;PV4=0.4,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=R694Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IGDCC4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020962.1;VDB=0.0000;DN=134;DA=C/T;GM=NM_020962.1;GL=IGDCC4;FG=missense;FD=missense;AAC=ARG/GLN;PP=694/1251;CDP=2081;GS=43;PH=0.968;CP=0.939;CG=5.490;AA=C;DV=by-frequency,by-cluster;DSP=42;GESP=T:88/C:12878;PAC=NP_066013.1 GT:GQ:PL 0/0:9:0,9,56 0/1:3:0,0,0 1/1:6:67,9,0 0/1:3:0,0,0</p>

<p>15 83215190 . C G 63.20 highCVG AC1=2;AF1=0.2897;DP=17;DP4=3,4,0,8;FQ=62.9;MQ=60;PV4=0.077,1,1,1;VDB=0.0006;GM=NM_001079533.1,NM_001079534.1,NM_001079535.1,NM_030594.3,XM_005272441.1,XM_005272442.1,XM_005272443.1,XM_005272444.1,XM_005272445.1,XM_005272446.1;GL=CPEB1;FG=splice-donor;FD=unknown;CP=1.000;CG=4.850;AA=C;DSP=0;KP=hsa04114:OocyteMeiosis|hsa04320:Dorso-ventralAxisFormation|hsa04914:Progesterone-mediatedOocyteMaturation;TFBS=V$AREB6_02,V$SREBP1_01;GESP=C:12084;PPI=UBC:943|SPI1:845|GRIN3A:830|MRTO4:830;PAC=NP_001073001.1,NP_001073002.1,NP_001073003.1,NP_085097.3,XP_005272498.1,XP_005272499.1,XP_005272500.1,XP_005272501.1,XP_005272502.1,XP_005272503.1 GT:GQ:PL 0/0:8:0,6,37 0/0:6:0,3,27 1/1:6:106,12,0 0/0:11:0,9,45</p>

<p>15 90174824 rs12900805 C T 23.70 midCVG AC1=6;AF1=0.8319;DB;DP=9;DP4=1,0,1,2;FQ=-24.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G1005R;SNPEFF_CODON_CHANGE=Ggg/Agg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198525.2;VDB=0.0106;DN=121;DA=C/T;GM=NM_198525.2,XM_005254902.1,XM_005254903.1,XM_005254904.1;GL=KIF7;FG=missense;FD=missense,unknown,unknown,unknown;AAC=GLY/ARG;PP=1005/1344,1005/1344,1004/1343,1004/1343;CDP=3013,3013,3010,3010;GS=125;PH=0.004;CP=0.067;CG=-0.473;AA=C;HC=9.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=99;TFBS=V$SREBP1_02;GESP=T:6862/C:6054;PAC=NP_940927.2,XP_005254959.1,XP_005254960.1,XP_005254961.1 GT:GQ:PL 0/1:3:0,0,0/1:6:29,0,4 1/1:4:28,3,0 0/1:3:0,0,0</p>

<p>16 715990 rs7190775 G A 37.20 lowCVG AC1=5;AF1=0.6489;DB;DP=5;DP4=0,1,4,0;FQ=-10.8;MQ=60;PV4=0.2,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R1492H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=36;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDR90;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145294.4;VDB=0.0001;DN=116;DA=G/A;GM=NM_145294.4,XM_005255160.1,XM_005255161.1;GL=WDR90;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=1492/1749,1495/1752,1494/1751;CDP=4475,4484,4481;GS=29;PH=0.169;CP=0.000;CG=-4.830;AA=G;HA=11.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=41;GESP=A:5864/G:6472;PAC=NP_660337.3,XP_005255217.1,XP_005255218.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,22 1/1:10:74,12,0 0/1:3:0,0,0</p>

<p>16 840378 rs3765263 A G 66.70 midCVG AC1=3;AF1=0.3829;DB;DP=14;DP4=6,0,8,0;FQ=58.4;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K244R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHTF18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022092.2;VDB=0.0000;DN=107;DA=A/G;GM=NM_022092.2,XM_005255470.1,XM_005255471.1,XM_005255472.1;GL=CHTF18;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=LYS/ARG,LYS/ARG,LYS/ARG,none;PP=244/976,128/860,272/1004,NA;CDP=731,383,815,NA;GS=26,26,26,NA;PH=0;CP=0.081;CG=-5.390;AA=G;HA=33.2;HE=22.3;HC=42.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;GESP=G:4000/A:8074;PPI=PCNA:939|RFC4:846|DSCC1:845|RFC3:845|RFC2:845;PAC=NP_071375.1,XP_005255527.1,XP_005255528.1,NA GT:GQ:PL 0/0:12:0,12,60 0/1:3:0,0,0 1/1:19:110,24,0 0/0:7:0,6,40</p>

<p>16 2905689 rs199712440 G A 23.60 lowCVG AC1=3;AF1=0.4317;DB;DP=7;DP4=2,1,2,1;FQ=25.1;MQ=60;PV4=1,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R149C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRSS22;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022119.3;VDB=0.0096;DN=137;DA=G/A;GM=NM_022119.3,XM_005255473.1;GL=PRSS22;FG=missense;FD=missense,unknown;AAC=ARG/CYS;PP=149/318;CDP=445;GS=180;PH=0.99;CP=0.006;CG=3.430;AA=G;DV=by-cluster;DSP=115;TFBS=V$PAX4_01;GESP=A:1/G:12995;PAC=NP_071402.1,XP_005255530.1 GT:GQ:PL 0/0:4:0,3,24 0/0:4:0,3,29 1/1:4:50,6,0 0/1:16:15,0,19</p>

<p>16 15111218 rs4985162 C T 42.50 highCVG AC1=5;AF1=0.5528;DB;DP=27;DP4=8,9,4,3;FQ=46;MQ=32;PV4=1,1,0.18,1;SNPEFF_AMINO_ACID_CHANGE=P301L;SNPEFF_CODON_CHANGE=cCg/cTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PDXDC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015027.3;VDB=0.0062;DN=111;DA=C/T;GM=NM_015027.2,XM_005255173.1,XM_005255174.1,XM_005255175.1,XM_005255176.1,XM_005255177.1,XM_005255178.1,XM_005255179.1,XM_005255180.1,XM_005255181.1;GL=PDXDC1;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/LEU;PP=301/789,319/807,310/798,300/788,286/774,286/774,282/770,274/762,273/761,210/698;CDP=902,956,929,899,857,857,845,821,818,629;GS=98;PH=0.015;CP=0.484;CG=4.490;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;TFBS=V$PPARA_01;GESP=T:2213/C:10775;PAC=NP_055842.2,XP_005255230.1,XP_005255231.1,XP_005255232.1,XP_005255233.1,XP_005255234.1,XP_005255235.1,XP_005255236.1,XP_005255237.1,XP_005255238.1 GT:GQ:PL 0/0:35:0,39,138 0/1:18:14,0,36 1/1:10:62,12,0 1/1:3:14,3,0</p>

<p>16 67183799 . A G 7.55 lowCVG AC1=4;AF1=0.4439;DP=5;DP4=0,2,3,0;FQ=3.73;MQ=60;PV4=0.1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=F197S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=B3GNT9;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033309.2;VDB=0.0006;GM=NM_033309.2;GL=B3GNT9;FG=missense;FD=unknown;AAC=PHE/SER;PP=197/403;CDP=590;GS=155;PH=1;CP=1.000;CG=4.740;DSP=776;CPG=67183209-67184968;TFBS=V$PPARG_03;GESP=A:12378;PAC=NP_171608.2 GT:GQ:PL 0/0:5:0,3,20 0/0:5:0,3,18 1/1:4:45,9,0 0/0:3:0,0,0</p>

<p>17 1690752 rs11549830 G A 24.30 lowCVG AC1=3;AF1=0.3495;DB;DP=8;DP4=2,3,3,0;FQ=26.1;MQ=60;PV4=0.2,1,1,0.37;SNPEFF_AMINO_ACID_CHANGE=R562W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SMYD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052928.2;VDB=0.0000;DN=120;DA=G/A;GM=NM_052928.2,XM_005256435.1,XM_005256436.1;GL=SMYD4;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=562/805,562/680,562/636;CDP=1684;GS=101;PH=0.997;CP=0.266;CG=2.540;AA=G;HA=45.0;HE=30.0;HC=45.0;DG;DV=by-frequency,by-cluster,by-1000G;DSP=37;GESP=A:4552/G:8438;PAC=NP_443160.2,XP_005256492.1,XP_005256493.1 GT:GQ:PL 0/0:13:0,12,73 0/0:3:0,0,0 1/1:4:64,9,0 0/0:5:0,3,21</p>

<p>17 5058808 rs9899177 G A 14.40 midCVG AC1=4;AF1=0.4652;DB;DP=9;DP4=1,5,0,3;FQ=17.1;MQ=56;PV4=1,1,6.1e-05,0.17;SNPEFF_AMINO_ACID_CHANGE=R912Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=23;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004505.2;VDB=0.0065;DN=119;DA=G/A;GM=NM_004505.2,XM_005256845.1,XM_005256846.1;GL=USP6;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=912/1407,912/1407,595/1090;CDP=2735,2735,1784;GS=43;PH=0.999;CP=1.000;CG=2.910;AA=G;HA=15.2;HE=33.3;HC=10.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=92;TFBS=V$STAT1_01;GESP=A:6766/G:6240;PAC=NP_004496.2,XP_005256902.1,XP_005256903.1 GT:GQ:PL 0/0:8:0,9,57 0/1:3:0,0,0 1/1:4:41,6,0 0/1:15:14,0,40</p>

<p>17 7951819 rs4792147 A G 25 lowCVG AC1=5;AF1=0.6327;DB;DP=8;DP4=2,1,3,1;FQ=27.7;MQ=60;PV4=1,0.054,1,1;SNPEFF_AMINO_ACID_CHANGE=Q656R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALOX15B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001141.2;VDB=0.0083;DN=111;DA=A/C/G;GM=NM_001039130.1,NM_001039131.1,NM_001141.2,XM_005256591.1;GL=ALOX15B;FG=missense;FD=missense,missense,missense,unknown;AAC=GLN/ARG;PP=627/648,582/603,656/677,644/665;CDP=1880,1745,1967,1931;GS=43;PH=0.001;CP=0.017;CG=-1.690;AA=G;HA=7.1;HE=47.0;HC=43.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=116;KP=hsa00590:ArachidonicAcidMetabolism|hsa01100:MetabolicPathways;GESP=G:7866/A:5140;PAC=NP_001034219.1,NP_001034220.1,NP_001132.2,XP_005256648.1 GT:GQ:PL 0/0:6:0,9,70 0/1:3:0,0,0 1/1:8:52,9,0 1/1:3:15,3,0</p>

<p>17 8216468 rs871841 T C 58.40 lowCVG AC1=5;AF1=0.6499;DB;DP=8;DP4=0,3,2,3;FQ=10.5;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L277P;SNPEFF_CODON_CHANGE=cTc/cCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ARHGEF15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025014.1;VDB=0.0003;DN=86;DA=T/C;GM=NM_025014.1,NM_173728.3,XM_005256536.1;GL=ARHGEF15;FG=missense;FD=missense,missense,unknown;AAC=LEU/PRO;PP=277/842,277/842,277/778;CDP=830;GS=98;PH=0;CP=0.995;CG=5.130;AA=C;HA=29.6;HE=47.3;HC=26.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=105;GESP=C:7328/T:5678;PAC=NP_079290.1,NP_776089.2,XP_005256593.1 GT:GQ:PL 0/0:6:0,9,48 0/1:3:0,0,0 1/1:11:83,12,0 1/1:4:18,3,0</p>

<p>17 13980350 rs2072279 G A 65.90 highCVG AC1=4;AF1=0.4922;DB;DP=16;DP4=0,8,6,0;FQ=68.8;MQ=60;PV4=0.00033,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=R159Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COX10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001303.3;VDB=0.0150;DN=96;DA=G/A;GM=NM_001303.3,XM_005256458.1,XM_005256459.1;GL=COX10;FG=missense;FD=missense,unknown,unknown;AAC=ARG/GLN;PP=159/444,159/240,159/238;CDP=476;GS=43;PH=0.178;CP=0.605;CG=5.490;AA=G;HA=37.0;HE=44.2;HC=49.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=24;KP=hsa00190:OxidativePhosphorylation|hsa00860:PorphyrinAndChlorophyllMetabolism|hsa01100:MetabolicPathways;GESP=A:6378/G:6628;PAC=NP_001294.2,XP_005256515.1,XP_005256516.1 GT:GQ:PL 0/0:3:0,3,26 0/1:3:25,3,0 1/1:12:88,15,0 0/0:19:0,21,111</p>

<p>17 19861458 rs2108978 C T 98.40 highCVG AC1=3;AF1=0.3904;DB;DP=18;DP4=7,4,1,6;FQ=101;MQ=60;PV4=0.066,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R249H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AKAP10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007202.3;VDB=0.0017;DN=96;DA=C/T;GM=NM_007202.3;GL=AKAP10;FG=missense;FD=missense;AAC=ARG/HIS;PP=249/663;CDP=746;GS=29;PH=0;CP=0.995;CG=2.360;AA=T;HA=44.6;HE=39.4;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=132;TFBS=V$GATA1_03;GESP=T:5787/C:7219;PPI=PDZK1:959|PRKAR1A:784;PAC=NP_009133.2 GT:GQ:PL 0/0:9:0,9,75 0/0:11:0,12,64 1/1:10:115,15,0 0/1:32:31,0,84</p>

<p>17 38121993 rs3894194 G A 35.30 lowCVG AC1=6;AF1=0.7326;DB;DP=5;DP4=1,0,2,2;FQ=-12.4;MQ=60;PV4=1,0.21,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R18Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSDMA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178171.4;VDB=0.0024;DN=108;DA=G/A;GM=NM_178171.4;GL=GSDMA;FG=missense;FD=missense;AAC=ARG/GLN;PP=18/446;CDP=53;GS=43;PH=0.016;CP=1.000;CG=3.340;AA=G;HA=28.6;HE=47.7;HC=46.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=58;GESP=A:4966/G:7324;PAC=NP_835465.2 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,22 1/1:9:54,9,0 1/1:4:21,3,0</p>

<p>17 41361960 rs8482 A G 76.10 midCVG AC1=4;AF1=0.5419;DB;DP=12;DP4=0,4,2,5;FQ=27.5;MQ=60;PV4=0.49,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=H923R;SNPEFF_CODON_CHANGE=cAt/cGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NBR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005899.4;VDB=0.0215;DN=52;DA=A/G;GM=NM_005899.3,NM_031858.2,NM_031862.2;GL=NBR1;FG=missense;FD=missense;AAC=HIS/ARG;PP=923/967;CDP=2768;GS=29;PH=0.968;CP=1.000;CG=5.860;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=41;GESP=G:3148/A:7152;PPI=SQSTM1:994|MAP1LC3A:937|UBC:757|MAP1LC3C:756;PAC=NP_005890.2,NP_114064.1,NP_114068.1 GT:GQ:PL 0/1:9:6,0,31 0/1:3:0,0,0 1/1:15:112,18,0 0/0:5:0,6,35</p>

<p>17 48629458 rs8065903 A G 42.80 lowCVG AC1=5;AF1=0.6395;DB;DP=8;DP4=3,0,3,2;FQ=9.67;MQ=60;PV4=0.46,1,1,0.071;SNPEFF_AMINO_ACID_CHANGE=K625R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_022827.3;VDB=0.0000;DN=116;DA=A/G;GM=NM_001258372.1,NM_001258373.1,NM_022827.3,XM_005257591.1;GL=SPATA20;FG=missense;FD=missense,missense,missense,unknown;AAC=LYS/ARG;PP=609/787,565/743,625/803,609/787;CDP=1826,1694,1874,1826;GS=26;PH=0;CP=0.996;CG=1.610;AA=G;HA=0.4;HE=23.2;HC=5.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=84;TFBS=V$GR_Q6,V$OLF1_01;GESP=G:10556/A:2450;PAC=NP_001245301.1,NP_001245302.1,NP_073738.2,XP_005257648.1 GT:GQ:PL 0/1:4:0,3,20 0/1:12:7,0,28 1/1:8:57,9,0 1/1:4:22,3,0</p>

<p>17 72346868 rs2382644 C T 25.90 lowCVG AC1=5;AF1=0.6043;DB;DP=8;DP4=1,2,1,2;FQ=24.6;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R471W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIF19;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_153209.3;VDB=0.0128;DN=100;DA=C/T;GM=NM_153209.3,XM_005257029.1,XM_005257030.1;GL=KIF19;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=471/999,471/1000,429/958;CDP=1411,1411,1285;GS=101;PH=0.999;CP=0.000;CG=-11.500;DG;DV=by-frequency,by-cluster,by-1000G;DSP=23;TFBS=V$HOX13_01;GESP=T:1007/C:11997;PAC=NP_694941.2,XP_005257086.1,XP_005257087.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,63 1/1:3:17,3,0 1/1:5:50,6,0</p>

<p>18 72103782 rs2278154 T C 999 highCVG AC1=5;AF1=0.603;DB;DP=20;DP4=5,1,8,6;FQ=35.4;MQ=60;PV4=0.35,1,1,1;SNPEFF_AMINO_ACID_CHANGE=Q405R;SNPEFF_CODON_CHANGE=cAa/cGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM69C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001044369.2;VDB=0.0039;DN=100;DA=T/C;GM=NM_001044369.2,XM_005266647.1;GL=FAM69C;FG=missense;FD=missense,unknown;AAC=GLN/ARG;PP=405/420,264/279;CDP=1214,791;GS=43;PH=0;CP=0.021;CG=4.080;AA=C;HA=27.7;HE=36.4;HC=29.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=173;GESP=C:4387/T:7701;PAC=NP_001037834.2,XP_005266704.1 GT:GQ:PL 0/1:3:0,0,0 0/1:53:53,0,54 1/1:28:173,30,0 0/1:4:0,3,19</p>

<p>19 1068738 rs1801284 G A 31.20 lowCVG AC1=6;AF1=0.749;DB;DP=5;DP4=0,1,2,2;FQ=-19.7;MQ=60;PV4=1,1,1,0.25;SNPEFF_AMINO_ACID_CHANGE=R155H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HMHA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001258328.2;VDB=0.0157;DN=89;DA=G/A;GM=NM_001258328.1,NM_012292.3;GL=HMHA1;FG=missense;FD=missense;AAC=ARG/HIS;PP=155/1153,139/1137;CDP=464,416;GS=29;PH=0.008;CP=0.000;CG=-8.440;AA=A;HA=45.1;HE=34.4;HC=41.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;CPG=1068439-1068764;GESP=A:5253/G:7735;PAC=NP_001245257.1,NP_036424.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:9:60,9,0 0/1:10:7,0,11</p>

<p>19 6374890 rs7540 G A 55.70 midCVG AC1=4;AF1=0.4563;DB;DP=10;DP4=5,0,3,1;FQ=47.1;MQ=60;PV4=0.44,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R191Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ALKBH7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032306.3;VDB=0.0086;DN=52;DA=G/A;GM=NM_032306.3,XM_005259658.1;GL=ALKBH7;FG=missense;FD=missense,unknown;AAC=ARG/GLN;PP=191/222,165/196;CDP=572,494;GS=43;PH=0.275;CP=0.021;CG=3.030;AA=G;HA=0.0;HE=10.3;HC=2.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=69;TFBS=V$OLF1_01;GESP=A:862/G:12144;PAC=NP_115682.1,XP_005259715.1 GT:GQ:PL 0/1:18:43,0,14 0/1:3:0,0,0 1/1:4:54,6,0 0/0:11:0,12,72</p>

<p>19 8997482 . C T 52.20 highCVG AC1=5;AF1=0.5614;DP=30;DP4=3,9,6,12;FQ=55.8;MQ=39;PV4=0.7,1,3.5e-17,1;SNPEFF_AMINO_ACID_CHANGE=R13647Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=59;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0008;GM=NM_024690.2,XM_005260161.1,XM_005260162.1,XM_005260163.1;GL=MUC16;FG=missense;FD=unknown;AAC=ARG/GLN;PP=13647/14508,5561/6405,5561/6025,442/1303;CDP=40940,16682,16682,1325;GS=43;PH=0.984;CP=0.000;CG=-5.720;AA=C;DSP=55;GESP=C:12276;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1,XP_005260220.1 GT:GQ:PL 0/1:3:0,3,59 0/1:17:15,0,19 1/1:13:69,15,0 0/1:17:13,0,77</p>

<p>19 10112346 rs2287803 T C 55.50 lowCVG AC1=5;AF1=0.647;DB;DP=8;DP4=3,0,0,5;FQ=23;MQ=60;PV4=0.018,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R322G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL5A3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015719.3;VDB=0.0071;DN=100;DA=T/C;GM=NM_015719.3;GL=COL5A3;FG=missense-near-splice;FD=missense;AAC=ARG/GLY;PP=322/1746;CDP=964;GS=125;PH=0;CP=0.002;CG=2.170;AA=C;HA=3.2;HE=2.3;HC=13.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:12695/T:311;PPI=COL5A1:999;PAC=NP_056534.2 GT:GQ:PL 0/1:17:20,0,16 0/1:17:12,0,43 1/1:9:64,9,0 0/1:3:0,0,0</p>

<p>19 15587185 rs733731 C T 22.30 lowCVG AC1=5;AF1=0.6333;DB;DP=6;DP4=1,1,4,0;FQ=-3.17;MQ=60;PV4=0.33,0.097,1,1;SNPEFF_AMINO_ACID_CHANGE=R99Q;SNPEFF_CODON_CHANGE=cGa/cAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PGLYRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052890.3;VDB=0.0069;DN=86;DA=C/T;GM=NM_052890.3;GL=PGLYRP2;FG=missense;FD=missense;AAC=ARG/GLN;PP=99/577;CDP=296;GS=43;PH=0.03;CP=0.001;CG=-5.850;AA=C;HA=34.3;HE=39.5;HC=38.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=235;GESP=T:4832/C:8174;PAC=NP_443122.3 GT:GQ:PL 0/1:14:15,0,14 0/1:4:0,3,24 1/1:4:19,3,0 1/1:6:31,6,0</p>

<p>19 15852363 rs11670007 G A 21 lowCVG AC1=3;AF1=0.3936;DB;DP=6;DP4=1,2,0,3;FQ=22.5;MQ=58;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R54H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR10H3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013938.1;VDB=0.0067;DN=120;DA=G/A;GM=NM_013938.1;GL=OR10H3;FG=missense;FD=missense;AAC=ARG/HIS;PP=54/317;CDP=161;GS=29;PH=0.001;CP=0.225;CG=-4.700;AA=G;HA=43.3;HE=31.4;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=18868;KP=hsa04740:OlfactoryTransduction;GESP=A:4268/G:8738;PAC=NP_039226.1 GT:GQ:PL 0/0:5:0,3,26 0/0:3:0,0,0 1/1:5:60,9,0 0/0:7:0,6,43</p>

<p>19 17170885 rs1130222 C T 99.90 highCVG AC1=5;AF1=0.5805;DB;DP=17;DP4=4,4,3,5;FQ=74.7;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G83R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033417.1;VDB=0.0107;DN=86;DA=C/A/G/T;GM=NM_001011699.1,NM_033417.1,XM_005260154.1;GL=HAUS8;FG=missense,missense,5-prime-UTR;FD=missense/stop-gained,missense/stop-gained,unknown;AAC=GLY/ARG,GLY/ARG,none;PP=82/410,83/411,NA;CDP=244,247,NA;GS=125,125,NA;PH=0.062;CP=0.001;CG=1.070;AA=C;HA=45.0;HE=33.3;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;GESP=T:5621/C:7385;PPI=HAUS1:999|HAUS6:994|HAUS2:987|HAUS4:987;PAC=NP_001011699.1,NP_219485.1,NA GT:GQ:PL 0/1:46:42,0,63 0/0:9:0,12,55 1/1:4:28,3,0 1/1:10:77,12,0</p>

<p>19 36303664 rs3848666 G C 17.50 lowCVG AC1=4;AF1=0.5;DB;DP=8;DP4=2,1,2,1;FQ=20;MQ=60;PV4=1,0.19,1,0.28;SNPEFF_AMINO_ACID_CHANGE=P91R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRODH2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021232.1;VDB=0.0075;DN=108;DA=G/C;GM=NM_021232.1,XM_005259123.1;GL=PRODH2;FG=missense;FD=missense,unknown;AAC=PRO/ARG;PP=91/537,15/379;CDP=272,44;GS=103;PH=0.004;CP=0.000;CG=0.647;AA=A;HA=0.8;HE=14.7;HC=0.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=131;KP=hsa00330:ArginineAndProlineMetabolism|hsa01100:MetabolicPathways;GESP=C:1572/G:11434;PAC=NP_067055.1,XP_005259180.1 GT:GQ:PL 0/0:9:0,9,65 0/1:3:0,0,0 1/1:5:55,9,0 0/1:3:0,0,0</p>

<p>19 37488481 rs1363753 G C 93.70 midCVG AC1=4;AF1=0.5436;DB;DP=10;DP4=1,2,1,6;FQ=6.29;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G630R;SNPEFF_CODON_CHANGE=Ggt/Cgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0155;DN=88;DA=G/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=missense;FD=missense;AAC=GLY/ARG;PP=630/636,566/572;CDP=1888,1696;GS=125;PH=0;CP=0.022;CG=1.640;AA=C;HA=29.2;HE=49.2;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=1263;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/0:3:0,3,20 0/1:3:0,0,0 1/1:5:48,6,0 0/1:24:85,0,21</p>

<p>19 37488499 rs1667366 T C 105 midCVG AC1=6;AF1=0.6771;DB;DP=10;DP4=1,1,1,7;FQ=-10.4;MQ=60;PV4=0.38,1,1,1;SNPEFF_AMINO_ACID_CHANGE=*636R;SNPEFF_CODON_CHANGE=Tga/Cga;SNPEFF_EFFECT=STOP_LOST;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF568;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_001204838.1;VDB=0.0136;DN=89;DA=T/C;GM=NM_001204838.1,NM_001204839.1;GL=ZNF568;FG=stop-lost;FD=stop-lost;AAC=stop/ARG;PP=636/636,572/572;CDP=1906,1714;CP=0.084;CG=2.660;AA=C;HA=43.3;HE=48.3;HC=26.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1281;PAC=NP_001191767.1,NP_001191768.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:6:47,6,0 0/1:3:96,1,0</p>

<p>19 41035024 . C T 33.50 midCVG AC1=4;AF1=0.5209;DP=10;DP4=0,3,5,0;FQ=22.5;MQ=60;PV4=0.018,0.066,1,0.086;SNPEFF_AMINO_ACID_CHANGE=R1312*;SNPEFF_CODON_CHANGE=Cga/Tga;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=18;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPTBN4;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_020971.2;VDB=0.0182;GM=NM_020971.2,XM_005259112.1,XM_005259113.1;GL=SPTBN4;FG=stop-gained;FD=unknown;AAC=ARG/stop;PP=1312/2565,1307/2560,55/1308;CDP=3934,3919,163;CP=1.000;CG=2.100;AA=C;DSP=15;GESP=C:13006;PPI=DISC1:789;PAC=NP_066022.2,XP_005259169.1,XP_005259170.1 GT:GQ:PL 0/1:11:8,0,21 0/0:3:0,3,28 1/1:5:45,6,0 0/1:16:23,0,13</p>

<p>ZNF568 19 45716364 rs189063316 C T 11.60 lowCVG AC1=5;AF1=0.5837;DB;DP=5;DP4=1,0,2,0;FQ=-12.4;MQ=60;PV4=1,1,1,0;SNPEFF_AMINO_ACID_CHANGE=R398Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXOC3L2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_138568.3;VDB=0.0116;DN=135;DA=C/T;GM=NM_138568.3;GL=EXOC3L2;FG=missense;FD=missense;AAC=ARG/GLN;PP=398/410;CDP=1193;GS=43;PH=0.46;CP=0.112;CG=1.260;AA=C;DV=by-cluster;DSP=252;GESP=T:121/C:12673;PAC=NP_612635.3 GT:GQ:PL 0/1:3:0,0,0/0:4:0,3,19 1/1:4:46,6,0 0/1:3:0,0,0</p>

<p>19 49377319 rs610308 A G 17.80 midCVG AC1=3;AF1=0.4136;DB;DP=11;DP4=3,1,0,3;FQ=20;MQ=60;PV4=0.14,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K277E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PPP1R15A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014330.3;VDB=0.0071;DN=83;DA=A/G;GM=NM_014330.3;GL=PPP1R15A;FG=missense;FD=missense;AAC=LYS/GLU;PP=277/675;CDP=829;GS=56;PH=0;CP=0.000;CG=-3.110;AA=G;HA=22.5;HE=32.3;HC=12.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=837;GESP=G:6297/A:6709;PPI=PPP1CA:942|PPP1CC:937|SMARCB1:846|PPP1CB:845;PAC=NP_055145.3 GT:GQ:PL 0/0:6:0,6,46 0/0:4:0,3,24 1/1:3:39,6,0 0/1:13:21,0,8</p>

<p>19 52496149 rs16983353 C T 125 highCVG AC1=5;AF1=0.6541;DB;DP=17;DP4=1,5,4,6;FQ=50.5;MQ=59;PV4=0.59,1,0.23,0.15;SNPEFF_AMINO_ACID_CHANGE=R738K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF615;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001199324.1;VDB=0.0050;DN=123;DA=C/T;GM=NM_001199324.1,NM_198480.3,XM_005258801.1,XM_005258802.1,XM_005258803.1,XM_005258804.1,XM_005258805.1,XM_005258806.1;GL=ZNF615;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ARG/LYS;PP=738/743,727/732,743/748,738/743,732/737,727/732,619/624,619/624;CDP=2213,2180,2228,2213,2195,2180,1856,1856;GS=26;PH=0;CP=0.003;CG=2.500;AA=T;HA=35.3;HE=28.2;HC=33.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1942;GESP=T:4942/C:8064;PAC=NP_001186253.1,NP_940882.3,XP_005258858.1,XP_005258859.1,XP_005258860.1,XP_005258861.1,XP_005258862.1,XP_005258863.1 GT:GQ:PL 0/1:18:13,0,48 0/1:3:0,0,0 1/1:11:88,12,0 0/1:40:65,0,39</p>

<p>19 52888245 rs55748277 A G 23.70 midCVG AC1=4;AF1=0.4732;DB;DP=11;DP4=3,3,2,3;FQ=26.4;MQ=59;PV4=1,0.19,1,1;SNPEFF_AMINO_ACID_CHANGE=K471R;SNPEFF_CODON_CHANGE=aAg/aGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF880;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145434.1;VDB=0.0110;DN=129;DA=A/G;GM=NM_001145434.1;GL=ZNF880;FG=missense;FD=missense;AAC=LYS/ARG;PP=471/578;CDP=1412;GS=26;PH=0.991;CP=0.007;CG=-0.460;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1144;GESP=G:1745/A:2821;PAC=NP_001138906.1 GT:GQ:PL 0/1:6:3,0,37 0/1:13:24,0,9 1/1:4:39,6,0 0/0:6:0,6,45</p>

<p>19 55106778 rs10418391 G A 52.20 lowCVG AC1=6;AF1=0.7307;DB;DP=6;DP4=0,1,0,4;FQ=-9.7;MQ=55;PV4=1,1,0.34,1;SNPEFF_AMINO_ACID_CHANGE=R191H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRA1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006863.3;VDB=0.0197;DN=119;DA=G/A;GM=NM_001278318.1,NM_001278319.1,NM_006863.3,NR_103501.1,NR_103502.1,NR_103503.1;GL=LILRA1;FG=missense,missense,missense,non-coding-exon,non-coding-exon,non-coding-exon;FD=unknown,unknown,missense,unknown,unknown,unknown;AAC=ARG/HIS,ARG/HIS,ARG/HIS,none,none,none;PP=191/290,191/440,191/490,NA,NA,NA;CDP=572,572,572,744,988,744;GS=29,29,29,NA,NA,NA;PH=0.042;CP=0.000;CG=-0.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=90;GESP=A:2052/G:10954;PAC=NP_001265247.1,NP_001265248.1,NP_006854.1,NA,NA,NA GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:4:27,3,0 1/1:9:65,9,0</p>

<p>19 55494157 rs4306647 G A 14.20 highCVG AC1=3;AF1=0.3892;DB;DP=15;DP4=6,3,3,1;FQ=16.2;MQ=41;PV4=1,0.41,1,1;SNPEFF_AMINO_ACID_CHANGE=R364K;SNPEFF_CODON_CHANGE=aGg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NLRP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001174081.1;VDB=0.0098;DN=111;DA=G/A;GM=NM_001174081.1,NM_001174082.1,NM_001174083.1,NM_017852.3,XM_005259050.1;GL=NLRP2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/LYS;PP=364/1063,342/1041,341/1040,364/1063,361/1060;CDP=1091,1025,1022,1091,1082;GS=26;PH=0.363;CP=0.019;CG=1.640;AA=G;HA=19.6;HE=2.2;HC=22.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=628;GESP=A:915/G:12091;PAC=NP_001167552.1,NP_001167553.1,NP_001167554.1,NP_060322.1,XP_005259107.1 GT:GQ:PL 0/0:5:0,3,19 0/1:11:10,0,32 1/1:5:48,9,0 0/0:12:0,12,50</p>

<p>19 59023166 rs34415062 G A 24 lowCVG AC1=4;AF1=0.5357;DB;DP=7;DP4=2,1,2,1;FQ=25.4;MQ=60;PV4=1,0.31,1,1;SNPEFF_AMINO_ACID_CHANGE=R53W;SNPEFF_CODON_CHANGE=Cgg/Tgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=G/A;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=ARG/TRP;PP=53/691,53/607,53/396;CDP=157;GS=101;PH=0;CP=0.000;CG=-8.620;AA=A;HA=22.8;HE=22.2;HC=5.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=280;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=A:2195/G:10597;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,26 1/1:4:50,6,0 0/1:16:13,0,40</p>

<p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

<p>20 1210647 rs450739 T C 57.60 lowCVG AC1=6;AF1=0.6884;DB;DP=8;DP4=1,1,5,1;FQ=-8.74;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=C90R;SNPEFF_CODON_CHANGE=Tgc/Cgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RAD21L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001136566.2;VDB=0.0021;DN=80;DA=T/C;GM=NM_001136566.2,XM_005260794.1,XM_005260795.1;GL=RAD21L1;FG=missense;FD=missense,unknown,unknown;AAC=CYS/ARG;PP=90/557,90/557,90/556;CDP=268;GS=180;PH=0;CP=1.000;CG=5.160;AA=C;HA=23.4;HE=47.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=7;TFBS=V$OCT1_06,V$PPARG_01;GESP=T:4566;PAC=NP_001130038.2,XP_005260851.1,XP_005260852.1 GT:GQ:PL 0/1:18:17,0,20 0/1:3:0,0,0 1/1:12:62,12,0 0/1:8:19,0,7</p>

<p>20 33874720 rs2425049 C T 49.50 midCVG AC1=3;AF1=0.4207;DB;DP=14;DP4=1,6,0,5;FQ=51.8;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R621Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM83C;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178468.5;VDB=0.0155;DN=100;DA=C/T;GM=NM_178468.5;GL=FAM83C;FG=missense;FD=missense;AAC=ARG/GLN;PP=621/748;CDP=1862;GS=43;PH=0;CP=0.334;CG=2.850;AA=T;HA=26.5;HE=21.9;HC=15.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1056;GESP=T:4644/C:8362;PAC=NP_848563.1 GT:GQ:PL 0/0:4:0,3,23 0/0:14:0,15,87 1/1:8:76,12,0 0/1:12:19,0,8</p>

<p>21 34907047 . T C 80.50 highCVG AC1=3;AF1=0.4181;DP=25;DP4=16,1,5,2;FQ=82.8;MQ=59;PV4=0.19,0.071,1,0.48;SNPEFF_AMINO_ACID_CHANGE=N85S;SNPEFF_CODON_CHANGE=aAc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GART;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000819.4;VDB=0.0037;GM=NM_000819.4,NM_001136005.1,NM_001136006.1,NM_175085.2,XM_005260941.1;GL=GART;FG=missense;FD=unknown;AAC=ASN/SER;PP=85/1011,85/1011,85/1011,85/434,85/1011;CDP=254;GS=46;PH=0;CP=0.763;CG=2.160;AA=T;DSP=13;KP=hsa00230:PurineMetabolism|hsa00670:OneCarbonPoolByFolate|hsa01100:MetabolicPathways;GESP=T:13006;PPI=UBC:730;PAC=NP_000810.1,NP_001129477.1,NP_001129478.1,NP_780294.1,XP_005260998.1 GT:GQ:PL 0/1:32:31,0,53 0/0:38:0,39,130 1/1:11:96,15,0 0/0:4:0,3,18</p>

<p>21 47851753 rs2073376 A G 39.40 lowCVG AC1=6;AF1=0.6957;DB;DP=6;DP4=1,1,1,3;FQ=10.9;MQ=60;PV4=1,0.27,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q2792R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0093;DN=96;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=2792/3337,2803/3348,2802/3347,2803/3269,2803/3259,2685/3230,2674/3140;CDP=8375,8408,8405,8408,8408,8054,8021;GS=43;PH=0;CP=0.314;CG=2.320;AA=G;HA=15.0;HE=33.5;HC=31.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=140;GESP=G:8998/A:4008;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:6:34,6,0 1/1:6:47,6,0</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

<p>22 32586875 rs136468 G A 7 midCVG AC1=3;AF1=0.3215;DB;DP=12;DP4=0,7,4,1;FQ=8.3;MQ=41;PV4=0.01,1,0.0013,0.38;SNPEFF_AMINO_ACID_CHANGE=R341C;SNPEFF_CODON_CHANGE=Cgc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RFPL2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098527.2;VDB=0.0002;DN=78;DA=G/A;GM=NM_001098527.2,NM_001159545.1,NM_001159546.1,NM_006605.3,XM_005261310.1;GL=RFPL2;FG=missense;FD=missense,missense,missense,missense,unknown;AAC=ARG/CYS;PP=341/379,251/289,251/289,280/318,251/289;CDP=1021,751,751,838,751;GS=180;PH=0.999;CP=0.036;CG=0.582;AA=G;HA=1.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=465;GESP=G:12936;PAC=NP_001091997.2,NP_001153017.1,NP_001153018.1,NP_006596.2,XP_005261367.1 GT:GQ:PL 0/0:6:0,3,27 0/0:5:0,1,22 1/1:5:48,12,0 0/0:17:0,15,80</p>

<p>22 50728062 rs28379706 T C 62.50 midCVG AC1=4;AF1=0.5564;DB;DP=11;DP4=1,2,6,0;FQ=26.1;MQ=60;PV4=0.083,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K318E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012401.3;VDB=0.0214;DN=125;DA=T/C;GM=NM_012401.3,XM_005261908.1,XM_005261909.1,XM_005261910.1,XM_005261911.1;GL=PLXNB2;FG=missense;FD=missense,unknown,unknown,unknown,unknown;AAC=LYS/GLU;PP=318/1839,400/1921,318/1839,318/1839,318/1839;CDP=952,1198,952,952,952;GS=56;PH=0;CP=0.001;CG=-1.270;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=117;KP=hsa04360:AxonGuidance;CPG=50727442-50728654;GESP=C:5524/T:6698;PPI=TUBA3E:962|MYC:846|ARHGEF11:845;PAC=NP_036533.2,XP_005261965.1,XP_005261966.1,XP_005261967.1,XP_005261968.1 GT:GQ:PL 0/1:3:0,3,24 0/0:5:0,6,44 1/1:3:21,3,0 1/1:13:87,15,0</p>

<p>22 51009953 rs470117 C T 122 highCVG AC1=4;AF1=0.5;DB;DP=21;DP4=5,3,7,6;FQ=74.8;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=E531K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145135.1;VDB=0.0103;DN=80;DA=C/T;GM=NM_001145134.1,NM_001145135.1,NM_001145136.1,NM_001145137.1,NM_004377.3,NM_152245.2,NM_152246.2,NR_027928.2;GL=CPT1B/CHKB-CPT1B;FG=missense,missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,none;PP=497/739,531/773,450/692,531/773,531/773,531/773,531/773,NA;CDP=1489,1591,1348,1591,1591,1591,1591,3161;GS=56,56,56,56,56,56,56,NA;PH=0.966;CP=0.993;CG=5.310;AA=C;HA=10.1;HE=48.3;HC=47.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway|hsa04920:AdipocytokineSignalingPathway;GESP=T:4624/C:8382;PAC=NP_001138606.1,NP_001138607.1,NP_001138608.1,NP_001138609.1,NP_004368.1,NP_689451.1,NP_689452.1,NA GT:GQ:PL 0/0:8:0,9,56 0/1:62:106,0,58 1/1:6:58,9,0 0/1:3:0,0,0</p>

<p>X 27839617 rs12557898 G A 18.40 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=1,2,0,3;FQ=16.7;MQ=60;PV4=1,1,1,0.22;SNPEFF_AMINO_ACID_CHANGE=R65Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAGEB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182506.3;VDB=0.0106;DN=120;DA=G/A;GM=NM_182506.3;GL=MAGEB10;FG=missense;FD=missense;AAC=ARG/GLN;PP=65/348;CDP=194;GS=43;PH=0.012;CP=0.000;CG=-5.040;AA=G;HA=14.0;HE=44.2;HC=8.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=243;GESP=A:4054/G:6506;PAC=NP_872312.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:5:56,9,0 0/0:9:0,9,53</p>

<p>X 49179719 rs6520418 C G 20.80 midCVG AC1=5;AF1=0.592;DB;DP=14;DP4=0,8,2,4;FQ=24.7;MQ=26;PV4=0.16,0.013,7.5e-06,0.011;SNPEFF_AMINO_ACID_CHANGE=P16R;SNPEFF_CODON_CHANGE=cCc/cGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0167;DN=116;DA=C/G;GM=NM_001098406.1;GL=GAGE12J;FG=missense;FD=missense;AAC=PRO/ARG;PP=16/118;CDP=47;GS=103;PH=0;CP=0.000;CG=0.012;AA=G;DV=by-cluster,by-2hit-2allele;DSP=38;GESP=G:1901/C:4835;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:20:0,24,118 1/1:3:17,3,0 1/1:13:48,15,0</p>

<p>X 49179755 rs7064530 G A 5.88 midCVG AC1=5;AF1=0.6096;DB;DP=13;DP4=0,6,0,5;FQ=9.09;MQ=25;PV4=1,1,6.2e-05,1;SNPEFF_AMINO_ACID_CHANGE=R28Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GAGE12J;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001098406.2;VDB=0.0001;DN=116;DA=G/A;GM=NM_001098406.1;GL=GAGE12J;FG=missense-near-splice;FD=missense;AAC=ARG/GLN;PP=28/118;CDP=83;GS=43;PH=0.881;CP=0.000;CG=-1.910;AA=G;DV=by-cluster;DSP=2;GESP=A:2418/G:4398;PAC=NP_001091876.1 GT:GQ:PL 0/1:3:0,0,0 0/0:17:0,18,67 1/1:4:19,6,0 1/1:6:30,9,0</p>

<p>X 88008807 rs5984611 G A 58 lowCVG AC1=6;AF1=0.7071;DB;DP=7;DP4=0,2,1,4;FQ=4.26;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=R131H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPXCR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184771.1;VDB=0.0098;DN=114;DA=G/A;GM=NM_001184771.1,NM_033048.5;GL=CPXCR1;FG=missense;FD=missense;AAC=ARG/HIS;PP=131/302;CDP=392;GS=29;PH=0.995;CP=0.000;CG=0.096;AA=G;HA=16.1;HE=40.7;HC=5.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=400;GESP=A:3478/G:7085;PAC=NP_001171700.1,NP_149037.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:40,6,0 0/1:34:54,0,33</p>

<p>X 153039502 rs6643791 G C 15.60 lowCVG AC1=6;AF1=0.683;DB;DP=5;DP4=0,2,3,0;FQ=-16.8;MQ=60;PV4=0.1,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=E1179D;SNPEFF_CODON_CHANGE=gaG/gaC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=21;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLXNB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001163257.1;VDB=0.0123;DN=116;DA=G/C;GM=NM_001163257.1,NM_005393.2,XM_005274694.1,XM_005274695.1,XM_005274696.1;GL=PLXNB3;FG=missense;FD=missense,missense,unknown,unknown,unknown;AAC=GLU/ASP;PP=1179/1933,1156/1910,1203/1957,1203/1933,809/1563;CDP=3537,3468,3609,3609,2427;GS=45;PH=0.005;CP=0.008;CG=-0.654;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=52;KP=hsa04360:AxonGuidance;GESP=C:7729/G:2554;PPI=ARHGEF11:895;PAC=NP_001156729.1,NP_005384.2,XP_005274751.1,XP_005274752.1,XP_005274753.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:27,3,0 0/1:15:23,0,13</p>


        <p>
          <strong>IDs and variants associated:</strong> H00608;CYP11A1|CYP17A1|DHCR7|HSD3B2|HSD17B3|AR|LHCGR|POR|SRD5A2|STAR;hsa04114,hsa04080,hsa04020,hsa00140
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 4
        </p>
        <p>
          <strong>Missense Mutations:</strong> 154
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.194
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse78">Cornelia de Lange syndrome (CdLS)</a>
      </h4>
    </div>
    <div id="collapse78" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	36995903	.	G	A	18	lowCVG	AC1=3;AF1=0.3939;DP=7;DP4=2,1,3,0;FQ=18.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E1101K;SNPEFF_CODON_CHANGE=Gaa/Aaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPBL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_133433.3;VDB=0.0000;GM=NM_015384.4,NM_133433.3,XM_005248280.1,XM_005248281.1,XM_005248282.1,XM_005248283.1;GL=NIPBL;FG=missense;FD=unknown;AAC=GLU/LYS;PP=1101/2698,1101/2805,1101/2699,981/2685,881/2585,562/2266;CDP=3301,3301,3301,2941,2641,1684;GS=56;PH=0.61;CP=1.000;CG=5.430;AA=G;DSP=4;GESP=G:13006;PPI=CBX5:765;PAC=NP_056199.2,NP_597677.2,XP_005248337.1,XP_005248338.1,XP_005248339.1,XP_005248340.1	GT:GQ:PL	0/0:7:0,6,39	0/0:5:0,3,23	1/1:5:57,9,0	0/0:3:0,0,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00631;NIPBL|SMC1A|SMC3;hsa04114,hsa04110
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse79">Heterotaxy; Situs ambiguus</a>
      </h4>
    </div>
    <div id="collapse79" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          4	8603105	rs115392829	G	A	26.80	lowCVG	AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA	GT:GQ:PL	0/0:7:0,6,47	0/0:4:0,3,18	1/1:7:67,12,0	0/1:3:0,0,0

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00632;NODAL|CFC1|ZIC3|ACVR2B;hsa04350,hsa04060
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse80">Lowe syndrome; Oculocerebrorenal Dystrophy (OCRL)</a>
      </h4>
    </div>
    <div id="collapse80" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          X	128691863	.	G	T	18.30	lowCVG	AC1=5;AF1=0.6661;DP=7;DP4=0,3,0,4;FQ=6.16;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=E125D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OCRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000276.3;VDB=0.0039;GM=NM_000276.3,NM_001587.3,XM_005262421.1,XM_005262422.1;GL=OCRL;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=GLU/ASP,GLU/ASP,GLU/ASP,none;PP=125/902,125/894,59/836,NA;CDP=375,375,177,NA;GS=45,45,45,NA;PH=0.001;CP=0.999;CG=0.803;AA=G;DSP=26;KP=hsa00562:InositolPhosphateMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=G:13006;PPI=RAB8A:959|CLTC:937;PAC=NP_000267.2,NP_001578.2,XP_005262478.1,NA	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:24,3,0	0/1:32:29,0,35

        </p>

        <p>
          <strong>IDs and variants associated:</strong>  H00692;OCRL;hsa04070,hsa00562
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse81">Dent disease</a>
      </h4>
    </div>
    <div id="collapse81" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          X	128691863	.	G	T	18.30	lowCVG	AC1=5;AF1=0.6661;DP=7;DP4=0,3,0,4;FQ=6.16;MQ=60;PV4=1,0.42,1,1;SNPEFF_AMINO_ACID_CHANGE=E125D;SNPEFF_CODON_CHANGE=gaG/gaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OCRL;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000276.3;VDB=0.0039;GM=NM_000276.3,NM_001587.3,XM_005262421.1,XM_005262422.1;GL=OCRL;FG=missense,missense,missense,5-prime-UTR;FD=unknown;AAC=GLU/ASP,GLU/ASP,GLU/ASP,none;PP=125/902,125/894,59/836,NA;CDP=375,375,177,NA;GS=45,45,45,NA;PH=0.001;CP=0.999;CG=0.803;AA=G;DSP=26;KP=hsa00562:InositolPhosphateMetabolism|hsa01100:MetabolicPathways|hsa04070:PhosphatidylinositolSignalingSystem;GESP=G:13006;PPI=RAB8A:959|CLTC:937;PAC=NP_000267.2,NP_001578.2,XP_005262478.1,NA	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:3:24,3,0	0/1:32:29,0,35

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00694;CLCN5|OCRL;hsa04070,hsa00562
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse82">Nemaline myopathy</a>
      </h4>
    </div>
    <div id="collapse82" class="panel-collapse collapse">
      <div class="panel-body">

        <p>2 152527572 rs7426114 C T 226 highCVG AC1=6;AF1=0.6942;DB;DP=19;DP4=1,2,7,8;FQ=6.18;MQ=60;PV4=1,1,1,0.46;SNPEFF_AMINO_ACID_CHANGE=V1491M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0167;DN=116;DA=C/T;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=VAL/MET;PP=1491/8526,1491/8526,1491/8561,1491/6670,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8495,1491/8464,1491/8464,1491/8464,1491/8433,1491/8433,1491/8402,1491/8402,1491/8371,1491/8371,1491/8371,1491/8371,1491/8340,1491/8340,1491/8309,1491/8309,1491/8309,1491/8283,1491/8283,1491/8278,1491/8278,1491/8019,1491/7554;CDP=4471;GS=21;PH=0.82;CP=1.000;CG=5.420;AA=C;HA=14.5;HE=19.2;HC=36.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8386/C:4292;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:13:55,0,13 0/1:3:0,6,35 1/1:15:90,15,0 1/1:21:129,21,0</p>

<p>2 152536498 rs6735208 T A 123 highCVG AC1=6;AF1=0.7592;DB;DP=24;DP4=9,2,6,5;FQ=76.9;MQ=60;PV4=0.36,0.46,1,0.29;SNPEFF_AMINO_ACID_CHANGE=K1027N;SNPEFF_CODON_CHANGE=aaA/aaT;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=31;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NEB;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271208.1;VDB=0.0118;DN=116;DA=T/A;GM=NM_001164507.1,NM_001164508.1,NM_001271208.1,NM_004543.4,XM_005246590.1,XM_005246591.1,XM_005246592.1,XM_005246593.1,XM_005246594.1,XM_005246595.1,XM_005246596.1,XM_005246597.1,XM_005246598.1,XM_005246599.1,XM_005246600.1,XM_005246601.1,XM_005246602.1,XM_005246603.1,XM_005246604.1,XM_005246605.1,XM_005246606.1,XM_005246607.1,XM_005246608.1,XM_005246609.1,XM_005246610.1,XM_005246611.1,XM_005246612.1,XM_005246613.1,XM_005246614.1,XM_005246615.1,XM_005246616.1,XM_005246617.1;GL=NEB;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=LYS/ASN;PP=1027/8526,1027/8526,1027/8561,1027/6670,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8495,1027/8464,1027/8464,1027/8464,1027/8433,1027/8433,1027/8402,1027/8402,1027/8371,1027/8371,1027/8371,1027/8371,1027/8340,1027/8340,1027/8309,1027/8309,1027/8309,1027/8283,1027/8283,1027/8278,1027/8278,1027/8019,1027/7554;CDP=3081;GS=94;PH=0.982;CP=0.992;CG=-1.620;AA=T;HA=29.2;HE=21.8;HC=39.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=39;TFBS=V$PAX4_04;GESP=A:8489/T:3663;PPI=TTN:937|DES:846|MYPN:845|TRIM63:765;PAC=NP_001157979.1,NP_001157980.1,NP_001258137.1,NP_004534.2,XP_005246647.1,XP_005246648.1,XP_005246649.1,XP_005246650.1,XP_005246651.1,XP_005246652.1,XP_005246653.1,XP_005246654.1,XP_005246655.1,XP_005246656.1,XP_005246657.1,XP_005246658.1,XP_005246659.1,XP_005246660.1,XP_005246661.1,XP_005246662.1,XP_005246663.1,XP_005246664.1,XP_005246665.1,XP_005246666.1,XP_005246667.1,XP_005246668.1,XP_005246669.1,XP_005246670.1,XP_005246671.1,XP_005246672.1,XP_005246673.1,XP_005246674.1 GT:GQ:PL 0/1:12:5,0,108 0/1:5:71,0,5 1/1:11:55,9,0 1/1:8:34,6,0</p>

<p>10 69933969 rs7916821 G A 187 highCVG AC1=6;AF1=0.7039;DB;DP=20;DP4=0,4,9,5;FQ=31.7;MQ=60;PV4=0.082,0.0054,1,0.096;SNPEFF_AMINO_ACID_CHANGE=S707N;SNPEFF_CODON_CHANGE=aGt/aAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYPN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256267.1;VDB=0.0087;DN=116;DA=G/A;GM=NM_001256267.1,NM_001256268.1,NM_032578.3,NR_045662.3,NR_045663.3;GL=MYPN;FG=missense,missense,missense,non-coding-exon,non-coding-exon;FD=missense,missense,missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/ASN,SER/ASN,SER/ASN,none,none;PP=707/1321,413/1027,707/1321,NA,NA;CDP=2120,1238,2120,1547,2388;GS=46,46,46,NA,NA;PH=0.091;CP=1.000;CG=4.680;AA=G;HA=17.6;HE=45.9;HC=24.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=147;TFBS=V$ARNT_02;GESP=A:5103/G:7903;PPI=ANKRD1:845|NEB:845;PAC=NP_001243196.1,NP_001243197.1,NP_115967.2,NA,NA GT:GQ:PL 0/1:45:68,0,45 0/1:5:0,3,28 1/1:15:86,15,0 1/1:13:80,12,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00698;CFL2|KBTBD13|NEB|ACTA1|TNNT1|TPM2|TPM3;hsa04810,hsa04666,hsa04360,hsa04260
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 3
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.3
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse83">Atrial fibrillation</a>
      </h4>
    </div>
    <div id="collapse83" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	32786389	rs2270915	A	G	30.80	midCVG	AC1=4;AF1=0.5475;DB;DP=9;DP4=1,4,2,2;FQ=33;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N522D;SNPEFF_CODON_CHANGE=Aac/Gac;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NPR3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001204375.1;VDB=0.0096;DN=100;DA=A/C/G/T;GM=NM_000908.3,NM_001204375.1,NM_001204376.1,XM_005248309.1;GL=NPR3;FG=missense;FD=missense,missense,missense,unknown;AAC=ASN/ASP;PP=521/541,522/542,305/325,306/326;CDP=1561,1564,913,916;GS=23;PH=0.001;CP=0.741;CG=5.000;AA=A;HA=8.1;HE=15.1;HC=23.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=50;GESP=G:2077/A:9753;PPI=NPPC:846|NPPB:846|NPPA:846;PAC=NP_000899.1,NP_001191304.1,NP_001191305.1,XP_005248366.1	GT:GQ:PL	0/0:5:0,6,41	0/1:3:21,3,0	1/1:5:51,6,0	0/1:5:1,0,37

        </p>

        <p>
          <strong>IDs and variants associated:</strong> H00731;ABCC9|GJA5|KCNA5|KCNJ2|KCNQ1|NPPA|SCN5A|KCNE2;hsa04270
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse84">3-Methylglutaconic aciduria (MGCA)</a>
      </h4>
    </div>
    <div id="collapse84" class="panel-collapse collapse">
      <div class="panel-body">
        <p>2 74755959 . G A 6.27 midCVG AC1=3;AF1=0.3379;DP=10;DP4=6,1,0,3;FQ=7.61;MQ=60;PV4=0.033,0.067,1,0.092;SNPEFF_AMINO_ACID_CHANGE=S148F;SNPEFF_CODON_CHANGE=tCc/tTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AUP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181575.3;VDB=0.0037;GM=NM_181575.3,XM_005264392.1;GL=AUP1;FG=missense;FD=unknown;AAC=SER/PHE;PP=148/411,205/468;CDP=443,614;GS=155;PH=0.99;CP=1.000;CG=4.820;AA=G;DSP=82;GESP=G:12058;PPI=UBE2G2:873|UBC:852|SEL1L:846;PAC=NP_853553.1,XP_005264449.1 GT:GQ:PL 0/0:4:0,0,0 0/0:7:0,3,19 1/1:3:45,9,0 0/0:21:0,18,92</p>

<p>9 19058743 rs10511670 G C 22.10 highCVG AC1=4;AF1=0.463;DB;DP=20;DP4=7,8,4,1;FQ=24.7;MQ=28;PV4=0.32,1,0.41,0.28;SNPEFF_AMINO_ACID_CHANGE=H674Q;SNPEFF_CODON_CHANGE=caC/caG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017645.4;VDB=0.0076;DN=119;DA=G/C;GM=NM_001270890.1,NM_017645.4,XM_005251491.1;GL=HAUS6;FG=missense;FD=missense,missense,unknown;AAC=HIS/GLN;PP=639/921,674/956,674/939;CDP=1917,2022,2022;GS=24;PH=0.008;CP=0.000;CG=-4.360;AA=G;HA=6.2;HE=6.7;HC=13.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=257;GESP=C:1067/G:11939;PPI=HAUS1:999|HAUS4:994|HAUS3:994|HAUS8:994|HAUS2:994|HAUS7:994;PAC=NP_001257819.1,NP_060115.3,XP_005251548.1 GT:GQ:PL 0/1:25:23,0,67 0/1:3:0,0,0 1/1:6:42,9,0 0/0:22:0,24,114</p>

<p>12 133423662 rs2306536 C T 17.40 lowCVG AC1=5;AF1=0.666;DB;DP=8;DP4=3,0,3,0;FQ=12.8;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=V580M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHFR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001161344.1;VDB=0.0028;DN=100;DA=C/T;GM=NM_001161344.1,NM_001161345.1,NM_001161346.1,NM_001161347.1,NM_018223.2,XM_005266170.1;GL=CHFR;FG=missense;FD=missense,missense,missense,missense,missense,unknown;AAC=VAL/MET;PP=580/665,579/664,568/653,488/573,539/624,621/706;CDP=1738,1735,1702,1462,1615,1861;GS=21;PH=0.099;CP=0.012;CG=-2.610;AA=C;HA=31.0;HE=16.7;HC=4.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=34;GESP=T:2911/C:10095;PPI=UBC:937|AURKA:846;PAC=NP_001154816.1,NP_001154817.1,NP_001154818.1,NP_001154819.1,NP_060693.2,XP_005266227.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:24,3,0 0/1:32:28,0,43</p>

<p>19 17170885 rs1130222 C T 99.90 highCVG AC1=5;AF1=0.5805;DB;DP=17;DP4=4,4,3,5;FQ=74.7;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=G83R;SNPEFF_CODON_CHANGE=Gga/Aga;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HAUS8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_033417.1;VDB=0.0107;DN=86;DA=C/A/G/T;GM=NM_001011699.1,NM_033417.1,XM_005260154.1;GL=HAUS8;FG=missense,missense,5-prime-UTR;FD=missense/stop-gained,missense/stop-gained,unknown;AAC=GLY/ARG,GLY/ARG,none;PP=82/410,83/411,NA;CDP=244,247,NA;GS=125,125,NA;PH=0.062;CP=0.001;CG=1.070;AA=C;HA=45.0;HE=33.3;HC=41.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=15;GESP=T:5621/C:7385;PPI=HAUS1:999|HAUS6:994|HAUS2:987|HAUS4:987;PAC=NP_001011699.1,NP_219485.1,NA GT:GQ:PL 0/1:46:42,0,63 0/0:9:0,12,55 1/1:4:28,3,0 1/1:10:77,12,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00754;DNAJC19|AUH|AU|TAZ|OPA3;hsa00280
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse85">Pitt-Hopkins syndrome</a>
      </h4>
    </div>
    <div id="collapse85" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	23885498	rs11574	T	C	82.30	midCVG	AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3	GT:GQ:PL	0/1:16:12,0,38	0/0:6:0,9,64	1/1:4:21,3,0	1/1:14:96,15,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00756;CNTNAP2|TCF4|NRXN1;hsa04514
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse86">Nonsyndromic autosomal recessive mental retardation (NS-ARMR)</a>
      </h4>
    </div>
    <div id="collapse86" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          8	141468645	.	G	A	29.60	midCVG	AC1=3;AF1=0.3656;DP=12;DP4=6,0,5,0;FQ=31.5;MQ=60;PV4=1,0.034,1,1.3e-09;SNPEFF_AMINO_ACID_CHANGE=Q7*;SNPEFF_CODON_CHANGE=Caa/Taa;SNPEFF_EFFECT=STOP_GAINED;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=NONSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRAPPC9;SNPEFF_IMPACT=HIGH;SNPEFF_TRANSCRIPT_ID=NM_031466.6;VDB=0.0000;GM=NM_031466.5;GL=TRAPPC9;FG=stop-gained;FD=unknown;AAC=GLN/stop;PP=7/1247;CDP=19;CP=0.000;CG=-2.520;AA=G;DSP=266;GESP=G:12786;PAC=NP_113654.4	GT:GQ:PL	0/0:3:0,0,0	0/0:7:0,6,51	1/1:9:72,15,0	0/0:12:0,12,70

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00768;GRIK2|CRBN|CC2D1A|ST3GAL3|TUSC3|TRAPPC9|PRSS12;hsa04080,hsa00601,hsa00533,hsa00510
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 1
        </p>
        <p>
          <strong>Missense Mutations:</strong> 0
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse87">Febrile seizures</a>
      </h4>
    </div>
    <div id="collapse87" class="panel-collapse collapse">
      <div class="panel-body">
        <p>2 167099158 rs6746030 A G 150 highCVG AC1=6;AF1=0.7278;DB;DP=16;DP4=2,4,7,3;FQ=65.7;MQ=60;PV4=0.3,1,1,1;SNPEFF_AMINO_ACID_CHANGE=W1150R;SNPEFF_CODON_CHANGE=Tgg/Cgg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=19;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SCN9A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002977.3;VDB=0.0076;DN=116;DA=A/G;GM=NM_002977.3,XM_005246757.1;GL=SCN9A;FG=missense;FD=missense,unknown;AAC=TRP/ARG;PP=1150/1978,1161/1989;CDP=3448,3481;GS=101;PH=0;CP=0.788;CG=3.830;AA=G;HA=13.1;HE=12.3;HC=4.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=9;GESP=G:10303/A:1439;PAC=NP_002968.1,XP_005246814.1 GT:GQ:PL 0/1:43:47,0,44 0/1:22:15,0,61 1/1:10:59,9,0 1/1:10:74,9,0</p>

<p>5 89988504 rs2366926 A G 62.10 midCVG AC1=5;AF1=0.6067;DB;DP=9;DP4=2,0,7,0;FQ=4.33;MQ=60;PV4=1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=N2345S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=32;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0009;DN=100;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2345/6307,NA,51/4013;CDP=7034,7047,152;GS=46,NA,46;PH=1;CP=1.000;CG=5.950;AA=A;HA=17.9;HE=29.9;HC=47.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=83;GESP=G:3478/A:8458;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:5:0,6,37 0/1:3:0,0,0 1/1:18:101,21,0 0/1:3:0,0,0</p>

<p>5 89990324 rs1878878 A G 184 highCVG AC1=6;AF1=0.7366;DB;DP=16;DP4=1,3,8,4;FQ=36.4;MQ=60;PV4=0.26,1,1,0.041;SNPEFF_AMINO_ACID_CHANGE=N2584S;SNPEFF_CODON_CHANGE=aAt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=33;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0007;DN=92;DA=A/G;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=ASN/SER,none,ASN/SER;PP=2584/6307,NA,290/4013;CDP=7751,7764,869;GS=46,NA,46;PH=0;CP=0.995;CG=5.880;AA=G;HA=4.0;HE=40.4;HC=18.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;CA;DSP=195;GESP=G:8914/A:3276;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/1:25:18,0,41 0/1:34:51,0,35 1/1:16:97,15,0 1/1:11:63,9,0</p>

<p>5 90012379 rs13157270 G A 132 highCVG AC1=4;AF1=0.4566;DB;DP=24;DP4=6,6,2,8;FQ=135;MQ=60;PV4=0.2,1,1,0.054;SNPEFF_AMINO_ACID_CHANGE=V3094I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=43;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR98;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032119.3;VDB=0.0154;DN=121;DA=G/A;GM=NM_032119.3,NR_003149.1,XM_005248613.1;GL=GPR98;FG=missense,non-coding-exon,missense;FD=missense,nc-transcript-variant,unknown;AAC=VAL/ILE,none,VAL/ILE;PP=3094/6307,NA,800/4013;CDP=9280,9293,2398;GS=29,NA,29;PH=0.01;CP=0.001;CG=-2.950;AA=G;HA=0.5;HE=9.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=96;GESP=A:774/G:11128;PAC=NP_115495.3,NA,XP_005248670.1 GT:GQ:PL 0/0:16:0,18,131 0/1:28:101,0,24 1/1:4:47,6,0 0/1:32:30,0,52</p>


        <p>
          <strong>IDs and variants associated:</strong> H00783;GABRD|GABRG2|SCN1A|SCN1B|SCN9A|GPR98;hsa04080
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse88">Congenital stationary night blindness (CSNB)</a>
      </h4>
    </div>
    <div id="collapse88" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	95091201	rs34899	A	G	48.40	highCVG	AC1=3;AF1=0.4188;DB;DP=17;DP4=3,6,3,2;FQ=50.4;MQ=60;PV4=0.58,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N262D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RHOBTB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014899.3;VDB=0.0132;DN=76;DA=A/G;GM=NM_014899.3,XM_005271925.1;GL=RHOBTB3;FG=missense;FD=missense,unknown;AAC=ASN/ASP;PP=262/612,294/644;CDP=784,880;GS=23;PH=0;CP=0.013;CG=5.210;AA=G;HA=30.5;HE=41.1;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=102;GESP=G:5290/A:7716;PAC=NP_055714.3,XP_005271982.1	GT:GQ:PL	0/0:4:0,3,27	0/0:4:0,3,15	1/1:7:88,12,0	0/1:4:1,0,63

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00787;GNAT1|GRM6|TRPM1|PDE6B|CABP4|RHO|GRK1|NYX|SAG|CACNA1F;hsa04744,hsa04080,hsa04020,hsa04010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse89">Ehlers-Danlos syndrome (EDS)</a>
      </h4>
    </div>
    <div id="collapse89" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	178540975	rs1054480	G	A	49.80	lowCVG	AC1=6;AF1=0.6833;DB;DP=6;DP4=2,0,1,3;FQ=-5.57;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P1177S;SNPEFF_CODON_CHANGE=Ccc/Tcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=22;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ADAMTS2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014244.4;VDB=0.0112;DN=86;DA=G/A;GM=NM_014244.4;GL=ADAMTS2;FG=missense;FD=missense;AAC=PRO/SER;PP=1177/1212;CDP=3529;GS=74;PH=0.651;CP=0.973;CG=5.050;AA=G;HA=2.7;HE=25.0;HC=32.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=351;GESP=A:2884/G:10122;PAC=NP_055059.2	GT:GQ:PL	0/0:4:0,6,32	0/1:3:0,0,0	1/1:4:26,3,0	1/1:8:65,9,0

        </p>
        <p>
          19	10112346	rs2287803	T	C	55.50	lowCVG	AC1=5;AF1=0.647;DB;DP=8;DP4=3,0,0,5;FQ=23;MQ=60;PV4=0.018,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R322G;SNPEFF_CODON_CHANGE=Agg/Ggg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=COL5A3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015719.3;VDB=0.0071;DN=100;DA=T/C;GM=NM_015719.3;GL=COL5A3;FG=missense-near-splice;FD=missense;AAC=ARG/GLY;PP=322/1746;CDP=964;GS=125;PH=0;CP=0.002;CG=2.170;AA=C;HA=3.2;HE=2.3;HC=13.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=C:12695/T:311;PPI=COL5A1:999;PAC=NP_056534.2	GT:GQ:PL	0/1:17:20,0,16	0/1:17:12,0,43	1/1:9:64,9,0	0/1:3:0,0,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00802;4GALT7|CHST14|COL1A1|COL1A2|COL3A1|COL5A1|COL5A2|PLOD1|TNXB|ZNF469|SLC39A13|ADAMTS2;hsa04512,hsa04510,hsa00534,hsa00532,hsa00310
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse90">Progressive myoclonic epilepsy (PME)</a>
      </h4>
    </div>
    <div id="collapse90" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          12	42863266	rs79087668	C	T	63.10	midCVG	AC1=5;AF1=0.5836;DB;DP=10;DP4=2,2,2,4;FQ=39.8;MQ=60;PV4=1,0.25,1,0.22;SNPEFF_AMINO_ACID_CHANGE=A124T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRICKLE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144881.1;VDB=0.0077;DN=131;DA=C/T;GM=NM_001144881.1,NM_001144882.1,NM_001144883.1,NM_153026.2;GL=PRICKLE1;FG=missense;FD=missense;AAC=ALA/THR;PP=124/832;CDP=370;GS=58;PH=0.002;CP=0.927;CG=3.550;AA=C;DV=by-frequency,by-cluster;DSP=15;KP=hsa04310:WntSignalingPathway;GESP=T:22/C:12984;PAC=NP_001138353.1,NP_001138354.1,NP_001138355.1,NP_694571.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,3,26	1/1:10:83,12,0	0/1:25:21,0,51

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00810;TPP1|CLN3|CLN5|PRICKLE1|CSTB|CTSD|KCTD7|CLN8|MFSD8|GBA|NHLRC1|TRNF|TRNH|TRNK|TRNL1|TRNS1|TRNS2|NEU1|CLN6|PPT1|EPM2A|SCARB2;hsa04310,hsa04142,hsa04120,hsa00970,hsa00600,hsa00511,hsa00062
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse91">Macular degeneration</a>
      </h4>
    </div>
    <div id="collapse91" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          14	21770730	rs6571751	A	G	150	midCVG	AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3	GT:GQ:PL	0/1:18:39,0,18	0/1:21:83,0,21	1/1:7:49,6,0	1/1:5:22,3,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00821;FBLN5|CFHR3|CST3|CX3CR1|ERCC6|ABCA4|CFH|CFHR1|ARMS2|HTRA1|PRPH2|RPGR|CFB|TLR4|HMCN1|RAX2|PROM1;hsa04620,hsa04062,hsa04060,hsa03420,hsa02010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse92">Familial flecked retina syndrome</a>
      </h4>
    </div>
    <div id="collapse92" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	95091201	rs34899	A	G	48.40	highCVG	AC1=3;AF1=0.4188;DB;DP=17;DP4=3,6,3,2;FQ=50.4;MQ=60;PV4=0.58,1,1,1;SNPEFF_AMINO_ACID_CHANGE=N262D;SNPEFF_CODON_CHANGE=Aat/Gat;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RHOBTB3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014899.3;VDB=0.0132;DN=76;DA=A/G;GM=NM_014899.3,XM_005271925.1;GL=RHOBTB3;FG=missense;FD=missense,unknown;AAC=ASN/ASP;PP=262/612,294/644;CDP=784,880;GS=23;PH=0;CP=0.013;CG=5.210;AA=G;HA=30.5;HE=41.1;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=102;GESP=G:5290/A:7716;PAC=NP_055714.3,XP_005271982.1	GT:GQ:PL	0/0:4:0,3,27	0/0:4:0,3,15	1/1:7:88,12,0	0/1:4:1,0,63

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00825;EFEMP1|CYP4V2|CFH|RDH5|RHO|RLBP1;hsa04744,hsa00830
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse93">Primary dystonia</a>
      </h4>
    </div>
    <div id="collapse93" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 11015165 rs75130475 A G 149 midCVG AC1=5;AF1=0.6109;DB;DP=14;DP4=0,2,7,4;FQ=-4.32;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I286T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C1orf127;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170754.1;VDB=0.0208;DN=131;DA=A/G;GM=NM_001170754.1,XM_005263428.1,XM_005263429.1,XM_005263430.1;GL=C1orf127;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=286/824,228/766,129/667,147/659;CDP=857,683,386,440;GS=89;PH=0.397;CP=0.000;CG=-6.010;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=79;GESP=G:1746/A:11260;PAC=NP_001164225.1,XP_005263485.1,XP_005263486.1,XP_005263487.1 GT:GQ:PL 0/1:4:0,3,24 0/1:4:0,3,14 1/1:4:26,3,0 1/1:28:168,30,0</p>

<p>1 17660468 rs34309058 C A 35.70 midCVG AC1=2;AF1=0.3104;DB;DP=13;DP4=5,4,2,2;FQ=37.2;MQ=60;PV4=1,0.34,1,0.089;SNPEFF_AMINO_ACID_CHANGE=P102T;SNPEFF_CODON_CHANGE=Cca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PADI4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012387.2;VDB=0.0067;DN=126;DA=C/A;GM=NM_012387.2;GL=PADI4;FG=missense;FD=missense;AAC=PRO/THR;PP=102/664;CDP=304;GS=38;PH=0.006;CP=0.000;CG=-9.480;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=31;GESP=A:184/C:12822;PPI=TP53:846;PAC=NP_036519.2 GT:GQ:PL 0/0:16:0,15,97 0/0:3:0,0,0 1/1:6:78,12,0 0/0:13:0,12,80</p>

<p>1 18808526 rs2992752 A C 43.10 lowCVG AC1=6;AF1=0.7115;DB;DP=8;DP4=2,0,0,6;FQ=-12;MQ=60;PV4=0.036,0.43,1,0.35;SNPEFF_AMINO_ACID_CHANGE=T351P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KLHDC7A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152375.2;VDB=0.0098;DN=101;DA=A/C;GM=NM_152375.2,XM_005245739.1,XM_005245740.1;GL=KLHDC7A;FG=missense;FD=missense,unknown,unknown;AAC=THR/PRO;PP=351/778,351/759,208/635;CDP=1051,1051,622;GS=38;PH=0;CP=0.000;CG=0.690;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=255;GESP=C:8092/A:4532;PAC=NP_689588.2,XP_005245796.1,XP_005245797.1 GT:GQ:PL 0/1:20:44,0,19 0/1:3:0,0,0 1/1:6:35,6,0 0/1:3:0,0,0</p>

<p>1 23885498 rs11574 T C 82.30 midCVG AC1=5;AF1=0.6016;DB;DP=13;DP4=4,1,4,3;FQ=59.4;MQ=60;PV4=0.58,0.2,1,1;SNPEFF_AMINO_ACID_CHANGE=T105A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ID3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002167.4;VDB=0.0021;DN=52;DA=T/C;GM=NM_002167.4;GL=ID3;FG=missense;FD=missense;AAC=THR/ALA;PP=105/120;CDP=313;GS=58;PH=0;CP=0.004;CG=-2.760;AA=C;HA=0.9;HE=28.2;HC=2.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=13;KP=hsa04350:TGF-betaSignalingPathway;GESP=C:10690/T:2316;PPI=TCF3:982|TCF4:907|MYOD1:832;PAC=NP_002158.3 GT:GQ:PL 0/1:16:12,0,38 0/0:6:0,9,64 1/1:4:21,3,0 1/1:14:96,15,0</p>

<p>1 55638075 rs1165222 G A 13.90 lowCVG AC1=8;AF1=1;DB;DP=7;DP4=0,0,0,2;FQ=-26.8;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T226I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=USP24;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015306.2;VDB=0.0081;DN=87;DA=G/A;GM=NM_015306.2,XM_005270689.1;GL=USP24;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=226/2621,51/2444;CDP=677,152;GS=89;PH=0;CP=0.988;CG=2.190;AA=A;HA=12.5;HE=16.1;HC=0.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=26;TFBS=V$BRACH_01,V$CDPCR3_01;GESP=A:10080/G:1770;PPI=UBC:709;PAC=NP_056121.2,XP_005270746.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:3:0,0,0</p>

<p>1 93672723 rs2815413 A C 107 highCVG AC1=6;AF1=0.7028;DB;DP=15;DP4=0,6,5,3;FQ=45.9;MQ=60;PV4=0.031,0.36,1,1;SNPEFF_AMINO_ACID_CHANGE=N326T;SNPEFF_CODON_CHANGE=aAt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CCDC18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_206886.4;VDB=0.0073;DN=100;DA=A/C;GM=NM_206886.3,XM_005270815.1,XM_005270816.1;GL=CCDC18;FG=missense;FD=missense,unknown,unknown;AAC=ASN/THR;PP=326/1300,444/1574,444/1418;CDP=977,1331,1331;GS=65;PH=0.649;CP=1.000;CG=4.580;AA=A;HA=25.6;HE=15.0;HC=3.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=60;GESP=C:8247/A:3559;PAC=NP_996769.3,XP_005270872.1,XP_005270873.1 GT:GQ:PL 0/1:18:44,0,18 0/1:38:32,0,66 1/1:10:53,9,0 1/1:5:21,3,0</p>

<p>1 112308953 rs197412 T C 58.10 midCVG AC1=5;AF1=0.6066;DB;DP=10;DP4=1,1,7,0;FQ=8.93;MQ=60;PV4=0.22,1,1,0.045;SNPEFF_AMINO_ACID_CHANGE=I636T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DDX20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007204.4;VDB=0.0159;DN=79;DA=T/C;GM=NM_007204.4;GL=DDX20;FG=missense;FD=missense;AAC=ILE/THR;PP=636/825;CDP=1907;GS=89;PH=0;CP=0.331;CG=3.860;AA=C;HA=19.7;HE=43.3;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=595;GESP=C:6295/T:6711;PPI=SMN1:984|SMN2:983|GEMIN4:973|GEMIN2:933|AGO2:885|GEMIN5:836|PRRC2A:786|HDAC5:716;PAC=NP_009135.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:97,21,0 0/0:5:0,6,43</p>

<p>1 205801872 rs1361754 A G 26.70 lowCVG AC1=5;AF1=0.6576;DB;DP=6;DP4=2,0,0,3;FQ=-9.17;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I380T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PM20D1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_152491.4;VDB=0.0040;DN=88;DA=A/G;GM=NM_152491.4;GL=PM20D1;FG=missense;FD=missense;AAC=ILE/THR;PP=380/503;CDP=1139;GS=89;PH=0.016;CP=0.270;CG=-0.108;AA=A;HA=33.9;HE=42.0;HC=0.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=23;GESP=G:6267/A:6739;PAC=NP_689704.4 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,27 1/1:6:41,6,0 1/1:4:26,3,0</p>

<p>1 248039451 rs3811444 C T 107 highCVG AC1=4;AF1=0.5108;DB;DP=24;DP4=8,4,4,7;FQ=110;MQ=60;PV4=0.22,0.14,1,0.15;SNPEFF_AMINO_ACID_CHANGE=T374M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TRIM58;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015431.3;VDB=0.0144;DN=107;DA=C/T;GM=NM_015431.3,XM_005273091.1,XM_005273092.1;GL=TRIM58;FG=missense;FD=missense,unknown,unknown;AAC=THR/MET;PP=374/487,230/343,230/343;CDP=1121,689,689;GS=81;PH=0.999;CP=0.011;CG=2.210;AA=C;HA=3.3;HE=33.2;HC=23.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=250;GESP=T:3365/C:9641;PAC=NP_056246.3,XP_005273148.1,XP_005273149.1 GT:GQ:PL 0/0:12:0,15,107 0/1:13:19,0,11 1/1:21:112,24,0 0/1:27:24,0,91</p>

<p>2 1320143 rs4927635 C T 220 highCVG AC1=6;AF1=0.7391;DB;DP=19;DP4=0,2,8,6;FQ=-7.92;MQ=60;PV4=0.47,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T495M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SNTG2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018968.3;VDB=0.0181;DN=111;DA=C/T;GM=NM_018968.3,XM_005264695.1;GL=SNTG2;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=495/540,368/413;CDP=1484,1103;GS=81;PH=0.042;CP=0.935;CG=-0.929;AA=C;HA=41.0;HE=41.6;HC=44.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=5;GESP=T:2274/C:2292;PAC=NP_061841.2,XP_005264752.1 GT:GQ:PL 0/1:6:0,3,25 0/1:8:20,0,8 1/1:19:119,18,0 1/1:22:127,21,0</p>

<p>2 26536694 rs2052937 C T 46.40 midCVG AC1=4;AF1=0.5336;DB;DP=10;DP4=5,0,1,4;FQ=47.7;MQ=60;PV4=0.048,0.27,1,1;SNPEFF_AMINO_ACID_CHANGE=A404T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GPR113;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145168.1;VDB=0.0034;DN=94;DA=C/T;GM=NM_001145168.1,NM_001145169.1,NM_153835.3,XM_005264163.1;GL=GPR113;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=404/1080,335/998,205/874,335/1004;CDP=1210,1003,613,1003;GS=58;PH=0.066;CP=0.007;CG=3.470;AA=T;HA=34.1;HE=32.5;HC=25.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=67;GESP=T:8972/C:4034;PAC=NP_001138640.1,NP_001138641.1,NP_722577.2,XP_005264220.1 GT:GQ:PL 0/0:7:0,9,60 0/1:35:49,0,32 1/1:3:26,3,0 1/1:3:15,3,0</p>

<p>2 29295186 rs75276619 C T 29.50 midCVG AC1=3;AF1=0.3733;DB;DP=12;DP4=4,4,3,1;FQ=31.5;MQ=60;PV4=0.58,0.41,1,0.46;SNPEFF_AMINO_ACID_CHANGE=A648T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=C2orf71;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001029883.2;VDB=0.0002;DN=131;DA=C/T;GM=NM_001029883.2;GL=C2orf71;FG=missense;FD=missense;AAC=ALA/THR;PP=648/1289;CDP=1942;GS=58;PH=0.2;CP=0.000;CG=-3.320;AA=C;DV=by-frequency,by-cluster;DSP=1727;GESP=T:113/C:12423;PAC=NP_001025054.1 GT:GQ:PL 0/1:11:10,0,51 0/0:9:0,9,67 1/1:5:64,9,0 0/0:7:0,6,37</p>

<p>2 37105041 rs77353404 G A 36.60 highCVG AC1=3;AF1=0.376;DB;DP=15;DP4=8,1,4,2;FQ=38.7;MQ=60;PV4=0.53,0.013,1,0.035;SNPEFF_AMINO_ACID_CHANGE=T439I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=STRN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003162.3;VDB=0.0026;DN=131;DA=G/A;GM=NM_003162.3,XM_005264519.1;GL=STRN;FG=missense;FD=missense,unknown;AAC=THR/ILE;PP=439/781,402/744;CDP=1316,1205;GS=89;PH=0.534;CP=0.995;CG=5.920;AA=G;DV=by-frequency,by-cluster;DSP=8;GESP=A:195/G:12811;PPI=CTTNBP2NL:999|CCT6A:999|CCT5:999|CCT8:999|SLMAP:999|CCT2:999|PPP2R1B:999|MOB4:999|TCP1:999|PPP2R1A:999|STK25:999|CCT6B:999|PDCD10:999|STRN3:999|CCT3:999|SIKE1:999|STRIP1:999|STK24:999|STRN4:999|MST4:999|CCT4:999|STK3:999|PPP2CA:999|TRAF3IP3:999|CTTNBP2:999|PPP2CB:999|FGFR1OP2:999|STRIP2:999|CCT7:999;PAC=NP_003153.2,XP_005264576.1 GT:GQ:PL 0/0:7:0,6,42 0/0:7:0,6,45 1/1:3:31,6,0 0/1:49:49,0,68</p>

<p>2 96993757 rs2279105 C T 69.10 lowCVG AC1=5;AF1=0.6603;DB;DP=8;DP4=0,2,2,4;FQ=-3.65;MQ=58;PV4=1,1,0.24,1;SNPEFF_AMINO_ACID_CHANGE=T471M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ITPRIPL1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178495.5;VDB=0.0196;DN=100;DA=C/T;GM=NM_001008949.2,NM_001163523.1,NM_001163524.1,NM_178495.5;GL=ITPRIPL1;FG=missense;FD=missense;AAC=THR/MET;PP=463/556,455/548,455/548,471/564;CDP=1388,1364,1364,1412;GS=81;PH=0.998;CP=0.363;CG=5.500;AA=C;HA=26.6;HE=32.6;HC=26.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1378;GESP=T:3982/C:9024;PAC=NP_001008949.1,NP_001156995.1,NP_001156996.1,NP_848590.3 GT:GQ:PL 0/1:16:12,0,21 0/1:4:0,3,17 1/1:4:21,3,0 1/1:12:80,12,0</p>

<p>2 120885310 rs1034489 G A 53.80 highCVG AC1=4;AF1=0.5281;DB;DP=21;DP4=0,9,6,2;FQ=57.1;MQ=60;PV4=0.0023,0.00019,1,1;SNPEFF_AMINO_ACID_CHANGE=A462T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EPB41L5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020909.3;VDB=0.0023;DN=86;DA=G/A;GM=NM_001184937.1,NM_020909.3,XM_005263719.1,XM_005263720.1,XM_005263721.1,XM_005263722.1;GL=EPB41L5;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=462/688,462/734,465/737,465/736,462/734,390/662;CDP=1384,1384,1393,1393,1384,1168;GS=58;PH=0;CP=0.973;CG=1.730;AA=A;HA=41.4;HE=26.4;HC=32.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=47;TFBS=V$ROAZ_01;GESP=A:8169/G:4837;PAC=NP_001171866.1,NP_065960.2,XP_005263776.1,XP_005263777.1,XP_005263778.1,XP_005263779.1 GT:GQ:PL 0/1:35:32,0,51 0/1:3:0,3,21 1/1:5:32,6,0 0/1:36:33,0,69</p>

<p>2 185802243 rs1366842 C A 30.40 highCVG AC1=2;AF1=0.2953;DB;DP=17;DP4=3,8,4,0;FQ=31.8;MQ=60;PV4=0.026,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T707K;SNPEFF_CODON_CHANGE=aCa/aAa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF804A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_194250.1;VDB=0.0182;DN=88;DA=C/A;GM=NM_194250.1;GL=ZNF804A;FG=missense;FD=missense;AAC=THR/LYS;PP=707/1210;CDP=2120;GS=78;PH=0.002;CP=0.000;CG=1.860;AA=C;HA=4.5;HE=40.1;HC=13.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=1734;GESP=A:5568/C:7416;PAC=NP_919226.1 GT:GQ:PL 0/0:6:0,3,26 0/0:29:0,27,124 1/1:6:73,12,0 0/0:6:0,3,28</p>

<p>2 186656956 rs17229201 T C 27.70 lowCVG AC1=5;AF1=0.6553;DB;DP=5;DP4=2,0,2,1;FQ=12.6;MQ=60;PV4=1,0.22,1,0.25;SNPEFF_AMINO_ACID_CHANGE=M1787T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FSIP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173651.2;VDB=0.0061;DN=123;DA=T/C;GM=NM_173651.2;GL=FSIP2;FG=missense;FD=missense;AAC=MET/THR;PP=1787/6997;CDP=5360;GS=81;CP=0.371;CG=3.390;AA=C;HA=42.4;HE=48.8;HC=46.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3587;PAC=NP_775922.2 GT:GQ:PL 0/1:3:0,0,0 0/0:4:0,6,48 1/1:4:21,3,0 1/1:5:47,6,0</p>

<p>2 220353532 rs13026308 C A 58.90 midCVG AC1=6;AF1=0.7369;DB;DP=11;DP4=1,1,5,1;FQ=-4.99;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P2687T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=34;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPEG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005876.4;VDB=0.0094;DN=121;DA=C/A;GM=NM_005876.4,XM_005246235.1,XM_005246236.1,XM_005246237.1,XM_005246238.1,XM_005246239.1,XM_005246240.1,XM_005246241.1,XM_005246242.1;GL=SPEG;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=PRO/THR;PP=2687/3268,2628/3209,2626/3207,2593/3174,2534/3115,1895/2476,1838/2419,1838/2419,1900/2481;CDP=8059,7882,7876,7777,7600,5683,5512,5512,5698;GS=38;PH=0.997;CP=0.999;CG=4.380;AA=C;HA=0.0;HE=16.1;HC=3.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=33;GESP=A:1183/C:11149;PAC=NP_005867.3,XP_005246292.1,XP_005246293.1,XP_005246294.1,XP_005246295.1,XP_005246296.1,XP_005246297.1,XP_005246298.1,XP_005246299.1 GT:GQ:PL 0/0:3:0,6,34 0/1:3:0,0,0 1/1:9:54,9,0 1/1:9:48,9,0</p>

<p>2 231865090 rs2271376 T C 84.80 midCVG AC1=4;AF1=0.5465;DB;DP=9;DP4=1,2,4,2;FQ=5.61;MQ=60;PV4=0.52,1,1,0.27;SNPEFF_AMINO_ACID_CHANGE=I104T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SPATA3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139073.3;VDB=0.0134;DN=100;DA=T/C;GM=NM_139073.3,XM_005246303.1;GL=SPATA3;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=104/193;CDP=311;GS=89;PH=0;CP=0.001;CG=-0.717;AA=C;HA=48.8;HE=35.5;HC=40.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=10;GESP=C:1865/T:2701;PAC=NP_620712.2,XP_005246360.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,16 1/1:5:47,6,0 0/1:27:77,0,24</p>

<p>2 237149941 rs6756597 C T 24.40 highCVG AC1=2;AF1=0.254;DB;DP=18;DP4=7,5,4,0;FQ=25.6;MQ=60;PV4=0.24,0.36,1,0.041;SNPEFF_AMINO_ACID_CHANGE=A104T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ASB18;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_212556.2;VDB=0.0223;DN=116;DA=C/T;GM=NM_212556.2;GL=ASB18;FG=missense;FD=missense;AAC=ALA/THR;PP=104/467;CDP=310;GS=58;PH=0.76;CP=0.928;CG=4.820;AA=C;HA=29.5;HE=46.2;HC=42.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=19;TFBS=V$P53_01,V$SRF_Q6;GESP=T:5089/C:6967;PAC=NP_997721.2 GT:GQ:PL 0/0:6:0,3,20 0/0:23:0,21,119 1/1:5:68,12,0 0/0:14:0,12,79</p>

<p>3 45637026 . C A 59.40 midCVG AC1=4;AF1=0.5;DP=14;DP4=1,3,0,9;FQ=9.86;MQ=60;PV4=0.31,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P219T;SNPEFF_CODON_CHANGE=Ccc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIMD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014240.2;VDB=0.0059;GM=NM_014240.2;GL=LIMD1;FG=missense;FD=unknown;AAC=PRO/THR;PP=219/677;CDP=655;GS=38;PH=0.004;CP=0.002;CG=-0.748;AA=C;DSP=754;GESP=C:13006;PAC=NP_055055.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:97,9,0 0/0:8:0,9,44</p>

<p>3 57136593 rs368070640 A G 14.20 midCVG AC1=3;AF1=0.3818;DB;DP=12;DP4=6,0,4,0;FQ=16.2;MQ=60;PV4=1,0.044,1,1;SNPEFF_AMINO_ACID_CHANGE=I298T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=IL17RD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017563.3;VDB=0.0210;DN=138;DA=A/G;GM=NM_017563.3,XM_005265237.1,XM_005265238.1,XM_005265239.1;GL=IL17RD;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ILE/THR;PP=298/740,274/716,270/712,154/596;CDP=893,821,809,461;GS=89;PH=0.986;CP=1.000;CG=5.480;AA=A;DSP=25;TFBS=V$GFI1_01;GESP=A:13006;PAC=NP_060033.3,XP_005265294.1,XP_005265295.1,XP_005265296.1 GT:GQ:PL 0/0:7:0,6,34 0/0:9:0,9,61 1/1:5:45,9,0 0/1:14:14,0,19</p>

<p>3 57414071 rs4681982 G A 107 highCVG AC1=4;AF1=0.4999;DB;DP=19;DP4=3,7,5,4;FQ=110;MQ=60;PV4=0.37,0.32,1,1;SNPEFF_AMINO_ACID_CHANGE=T1786I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DNAH12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001291661.1;VDB=0.0121;DN=111;DA=G/A;GM=NM_178504.4;GL=DNAH12;FG=missense;FD=missense;AAC=THR/ILE;PP=1763/3093;CDP=5288;GS=89;PH=0;CP=0.003;CG=-0.547;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6;GESP=G:4506;PAC=NP_848599.3 GT:GQ:PL 0/1:32:30,0,41 0/0:5:0,6,44 1/1:5:50,6,0 0/1:74:71,0,91</p>

<p>3 66419956 rs13874 C T 145 highCVG AC1=4;AF1=0.5354;DB;DP=17;DP4=3,2,7,5;FQ=57.7;MQ=60;PV4=1,0.15,1,0.37;SNPEFF_AMINO_ACID_CHANGE=T208M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC25A26;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173471.3;VDB=0.0202;DN=52;DA=C/T;GM=NM_001164796.1,NR_028475.1;GL=SLC25A26;FG=missense,non-coding-exon;FD=missense,nc-transcript-variant;AAC=THR/MET,none;PP=120/187,NA;CDP=359,904;GS=81,NA;PH=0;CP=1.000;CG=5.490;AA=T;HA=13.3;HE=46.3;HC=28.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=11;GESP=T:7243/C:5763;PAC=NP_001158268.1,NA GT:GQ:PL 0/0:7:0,9,70 0/1:3:0,0,0 1/1:18:112,21,0 0/1:31:77,0,28</p>

<p>3 69245454 rs9310141 T C 6.32 lowCVG AC1=8;AF1=1;DB;DP=5;DP4=0,0,2,0;FQ=-27.9;MQ=60;SNPEFF_AMINO_ACID_CHANGE=T396A;SNPEFF_CODON_CHANGE=Acg/Gcg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRMD4B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015123.2;VDB=0.0175;DN=119;DA=T/C;GM=NM_015123.1,XM_005264720.1,XM_005264721.1,XM_005264722.1,XM_005264723.1;GL=FRMD4B;FG=missense;FD=unknown;AAC=THR/ALA;PP=396/1035,365/1004,363/1002,351/990,48/687;CDP=1186,1093,1087,1051,142;GS=58;PH=0;CP=0.803;CG=6.070;AA=C;HA=13.4;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=49;GESP=C:11705/T:393;PAC=NP_055938.1,XP_005264777.1,XP_005264778.1,XP_005264779.1,XP_005264780.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:13,3,0 1/1:4:25,3,0</p>

<p>3 98512540 rs28489284 G A 55.60 midCVG AC1=3;AF1=0.4126;DB;DP=12;DP4=5,1,1,4;FQ=57.2;MQ=60;PV4=0.08,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A364T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ST3GAL6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001271145.1;VDB=0.0070;DN=125;DA=G/A;GM=NM_001271142.1,NM_001271145.1,NM_001271146.1,NM_001271147.1,NM_001271148.1,NM_006100.3,XM_005247067.1,XM_005247068.1,XM_005247069.1,XM_005247070.1,XM_005247071.1,XM_005247072.1;GL=ST3GAL6;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=225/246,364/385,311/332,193/214,167/188,311/332,334/355,333/354,311/332,311/332,167/188,167/188;CDP=673,1090,931,577,499,931,1000,997,931,931,499,499;GS=58;PH=0.934;CP=1.000;CG=5.930;AA=G;HA=13.0;HE=2.9;HC=2.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=22;KP=hsa00601:GlycosphingolipidBiosynthesis-LactoAndNeolactoSeries|hsa01100:MetabolicPathways;GESP=A:682/G:12324;PAC=NP_001258071.1,NP_001258074.1,NP_001258075.1,NP_001258076.1,NP_001258077.1,NP_006091.1,XP_005247124.1,XP_005247125.1,XP_005247126.1,XP_005247127.1,XP_005247128.1,XP_005247129.1 GT:GQ:PL 0/0:4:0,3,25 0/0:9:0,9,53 1/1:5:69,9,0 0/1:30:31,0,30</p>

<p>3 121208833 rs3218649 G C 208 highCVG AC1=5;AF1=0.6146;DB;DP=33;DP4=1,17,4,11;FQ=125;MQ=60;PV4=0.15,0.0063,1,0.47;SNPEFF_AMINO_ACID_CHANGE=T982R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=POLQ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_199420.3;VDB=0.0100;DN=106;DA=G/C;GM=NM_199420.3;GL=POLQ;FG=missense;FD=missense;AAC=THR/ARG;PP=982/2591;CDP=2945;GS=71;PH=0;CP=0.000;CG=-3.860;AA=C;HA=48.3;HE=40.8;HC=9.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=423;GESP=C:7836/G:5164;PAC=NP_955452.3 GT:GQ:PL 0/1:37:59,0,36 0/0:43:0,48,136 1/1:20:109,21,0 1/1:11:91,12,0</p>

<p>3 121838319 rs1129055 G A 43.30 midCVG AC1=5;AF1=0.6034;DB;DP=9;DP4=1,1,1,4;FQ=8.82;MQ=60;PV4=1,0.11,1,1;SNPEFF_AMINO_ACID_CHANGE=A310T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CD86;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_175862.4;VDB=0.0085;DN=86;DA=G/A;GM=NM_001206924.1,NM_001206925.1,NM_006889.4,NM_175862.4,NM_176892.1,XM_005247905.1,XM_005247906.1;GL=CD86;FG=missense;FD=missense,missense,missense,missense,missense,unknown,unknown;AAC=ALA/THR;PP=198/218,228/248,304/324,310/330,256/276,398/418,350/370;CDP=592,682,910,928,766,1192,1048;GS=58;PH=0.932;CP=0.027;CG=3.200;AA=G;HA=11.6;HE=29.9;HC=38.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=35;KP=hsa04514:CellAdhesionMolecules(CAMs)|hsa04620:Toll-likeReceptorSignalingPathway|hsa04672:IntestinalImmuneNetworkForIgAProduction|hsa04940:TypeIDiabetesMellitus|hsa05320:AutoimmuneThyroidDisease|hsa05322:SystemicLupusErythematosus|hsa05330:AllograftRejection|hsa05332:Graft-versus-hostDisease|hsa05416:ViralMyocarditis;GESP=A:2991/G:10015;PPI=CD28:995|CTLA4:991|UBC:937;PAC=NP_001193853.1,NP_001193854.1,NP_008820.3,NP_787058.4,NP_795711.1,XP_005247962.1,XP_005247963.1 GT:GQ:PL 0/1:3:0,3,23 0/1:3:0,3,26 1/1:8:53,9,0 1/1:5:35,6,0</p>

<p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>3 194062519 rs3732477 C T 62.10 highCVG AC1=3;AF1=0.3624;DB;DP=16;DP4=0,8,1,5;FQ=64;MQ=60;PV4=0.43,0.0087,1,1;SNPEFF_AMINO_ACID_CHANGE=A305T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080513.3;VDB=0.0065;DN=107;DA=C/T;GM=NM_001080513.2,XM_005269280.1,XM_005269281.1;GL=CPN2;FG=missense;FD=missense,unknown,unknown;AAC=ALA/THR;PP=305/546;CDP=913;GS=58;PH=0.017;CP=0.000;CG=-10.600;AA=C;HA=44.2;HE=26.5;HC=32.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=913;GESP=T:4238/C:8768;PAC=NP_001073982.2,XP_005269337.1,XP_005269338.1 GT:GQ:PL 0/0:9:0,9,59 0/0:12:0,12,79 1/1:5:63,9,0 0/1:19:45,0,13</p>

<p>3 195452799 rs2550232 C T 11.30 midCVG AC1=8;AF1=1;DB;DP=9;DP4=4,1,1,3;FQ=-26.1;MQ=21;PV4=0.21,0.34,1,0.42;SNPEFF_AMINO_ACID_CHANGE=T442I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC20;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001282506.1;VDB=0.0102;DN=100;DA=C/T;GM=NM_001098516.1,NM_152673.2,XM_005269303.1,XM_005269304.1,XM_005269305.1,XM_005269306.1,XM_005269307.1,XM_005269308.1,XM_005269309.1;GL=MUC20;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ILE;PP=236/504,271/539,442/727,442/710,407/692,407/692,407/692,271/553,272/540;CDP=707,812,1325,1325,1220,1220,1220,812,815;GS=89;PH=0.002;CP=0.000;CG=-2.630;AA=C;DG;DV=by-cluster,by-2hit-2allele;DSP=131;GESP=C:12316;PAC=NP_001091986.1,NP_689886.2,XP_005269360.1,XP_005269361.1,XP_005269362.1,XP_005269363.1,XP_005269364.1,XP_005269365.1,XP_005269366.1 GT:GQ:PL 0/0:2:0,6,0/1:3:16,0,0 1/1:4:20,3,0 1/1:4:14,3,0</p>

<p>4 71068489 rs3196714 T C 34.90 lowCVG AC1=5;AF1=0.5673;DB;DP=7;DP4=0,2,0,4;FQ=7.37;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=I222T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ODAM;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_017855.3;VDB=0.0087;DN=105;DA=T/C;GM=NM_017855.3,XM_005265698.1;GL=ODAM;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=222/280,239/297;CDP=665,716;GS=89;PH=0.007;CP=0.000;CG=0.676;AA=T;HA=15.6;HE=26.6;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;GESP=C:3310/T:9690;PAC=NP_060325.3,XP_005265755.1 GT:GQ:PL 0/1:15:13,0,17 0/1:3:0,0,0 1/1:7:62,9,0 0/1:3:0,3,26</p>

<p>4 100504664 rs3816873 T C 35.40 lowCVG AC1=6;AF1=0.695;DB;DP=7;DP4=1,1,3,2;FQ=10.9;MQ=60;PV4=1,0.018,1,0.15;SNPEFF_AMINO_ACID_CHANGE=I155T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MTTP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300785.1;VDB=0.0031;DN=107;DA=T/C;GM=NM_000253.2,XM_005263025.1;GL=MTTP;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=128/895,155/922;CDP=383,464;GS=89;PH=0.007;CP=0.810;CG=4.100;AA=T;HA=22.5;HE=25.3;HC=14.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=11;GESP=C:3415/T:9587;PPI=APOB:846;PAC=NP_000244.2,XP_005263082.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:11:61,12,0 1/1:4:16,3,0</p>

<p>5 140023238 rs17208187 C G 65.40 midCVG AC1=5;AF1=0.6275;DB;DP=12;DP4=3,1,1,6;FQ=18.7;MQ=60;PV4=0.088,1,1,0.029;SNPEFF_AMINO_ACID_CHANGE=T305S;SNPEFF_CODON_CHANGE=aCc/aGc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TMCO6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001300980.1;VDB=0.0001;DN=123;DA=C/G;GM=NM_018502.3,XM_005268476.1,XM_005268477.1;GL=TMCO6;FG=missense;FD=missense,unknown,unknown;AAC=THR/SER;PP=299/494,305/500,300/495;CDP=896,914,899;GS=58;PH=0.001;CP=0.341;CG=2.370;AA=C;HA=7.1;HE=26.0;HC=31.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=23;GESP=G:2322/C:9960;PAC=NP_060972.3,XP_005268533.1,XP_005268534.1 GT:GQ:PL 0/1:16:11,0,28 0/0:4:0,6,32 1/1:4:16,3,0 1/1:14:84,15,0</p>

<p>5 140572763 rs2907323 C G 71.10 midCVG AC1=4;AF1=0.5;DB;DP=12;DP4=0,6,4,2;FQ=70;MQ=55;PV4=0.061,1,1,0.025;SNPEFF_AMINO_ACID_CHANGE=T213R;SNPEFF_CODON_CHANGE=aCa/aGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018930.3;VDB=0.0102;DN=101;DA=C/G;GM=NM_018930.3;GL=PCDHB10;FG=missense;FD=missense;AAC=THR/ARG;PP=213/801;CDP=638;GS=71;PH=0.977;CP=0.991;CG=3.410;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=6171;GESP=G:1618/C:11388;PAC=NP_061753.1 GT:GQ:PL 0/0:10:0,12,73 0/0:5:0,6,45 1/1:9:74,12,0 1/1:4:46,6,0</p>

<p>6 52617731 rs2180314 C G 37.70 midCVG AC1=4;AF1=0.5406;DB;DP=9;DP4=4,0,3,1;FQ=23;MQ=60;PV4=1,1,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S112T;SNPEFF_CODON_CHANGE=aGt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GSTA2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000846.4;VDB=0.0058;DN=96;DA=C/G;GM=NM_000846.4;GL=GSTA2;FG=missense;FD=missense;AAC=SER/THR;PP=112/223;CDP=335;GS=58;PH=0;CP=0.000;CG=-4.510;AA=C;HA=22.9;HE=43.0;HC=33.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;KP=hsa00480:GlutathioneMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450;GESP=G:6507/C:6499;PPI=GSTM2:937;PAC=NP_000837.3 GT:GQ:PL 0/0:3:0,3,27 0/1:3:0,0,1/1:5:48,6,0 0/1:30:29,0,32</p>

<p>6 118886961 rs3734382 G T 108 midCVG AC1=5;AF1=0.5972;DB;DP=14;DP4=1,3,6,2;FQ=43.4;MQ=60;PV4=0.22,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P254T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CEP85L;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001178035.1;VDB=0.0162;DN=107;DA=G/T;GM=NM_001042475.2,NM_001178035.1,NM_206921.2,XM_005266970.1,XM_005266971.1;GL=CEP85L;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=PRO/THR;PP=251/806,254/809,251/497,149/704,149/704;CDP=751,760,751,445,445;GS=38;PH=0;CP=0.066;CG=-4.210;AA=T;HA=17.8;HE=26.0;HC=31.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=270;GESP=T:9623/G:3383;PAC=NP_001035940.1,NP_001171506.1,NP_996804.2,XP_005267027.1,XP_005267028.1 GT:GQ:PL 0/1:4:0,3,23 0/1:3:0,0,0 1/1:16:112,18,0 0/1:42:38,0,58</p>

<p>7 99306685 rs2257401 C G 76 midCVG AC1=6;AF1=0.7069;DB;DP=12;DP4=2,2,0,7;FQ=32.2;MQ=58;PV4=0.11,1,0.1,1;SNPEFF_AMINO_ACID_CHANGE=R409T;SNPEFF_CODON_CHANGE=aGa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYP3A7-CYP3AP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256497.2;VDB=0.0041;DN=100;DA=C/G;GM=NM_000765.3,NM_001256497.1,XM_005250168.1;GL=CYP3A7/CYP3A7-CYP3AP1;FG=missense;FD=missense,missense,unknown;AAC=ARG/THR;PP=409/504,409/536,408/503;CDP=1226,1226,1223;GS=71;PH=0;CP=0.373;CG=1.540;AA=G;HE=6.7;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=28;KP=hsa00140:SteroidHormoneBiosynthesis|hsa00591:LinoleicAcidMetabolism|hsa00830:RetinolMetabolism|hsa00980:MetabolismOfXenobioticsByCytochromeP450|hsa00982:DrugMetabolism-CytochromeP450|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;GESP=G:9936/C:3070;PAC=NP_000756.2,NP_001243426.1,XP_005250225.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:45,6,0 0/1:64:67,0,63</p>

<p>7 101200727 rs17393069 G A 47.60 lowCVG AC1=6;AF1=0.7176;DB;DP=8;DP4=2,0,4,1;FQ=9.59;MQ=60;PV4=1,0.015,1,0.29;VDB=0.0078;DN=123;DA=G/A;GM=NM_001278563.1,NM_133457.3;GL=COL26A1;FG=missense;FD=unknown,missense;AAC=ALA/THR;PP=415/442,413/440;CDP=1242,1236;GS=58;PH=0.226;CP=0.199;CG=-0.864;AA=G;HA=45.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=77;GESP=A:4643/G:7635;PAC=NP_001265492.1,NP_597714.2 GT:GQ:PL 0/0:3:0,6,46 0/1:3:0,0,0 1/1:9:54,9,0 1/1:6:36,6,0</p>

<p>8 53586706 rs17337252 A G 65.40 midCVG AC1=4;AF1=0.5;DB;DP=14;DP4=2,2,2,7;FQ=40.4;MQ=60;PV4=0.53,0.22,1,1;SNPEFF_AMINO_ACID_CHANGE=M234T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RB1CC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014781.4;VDB=0.0199;DN=123;DA=A/G;GM=NM_001083617.1,NM_014781.4,XM_005251330.1;GL=RB1CC1;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=234/1592,234/1595,234/1595;CDP=701;GS=81;PH=0;CP=0.893;CG=2.990;AA=A;HA=12.7;HE=41.3;HC=16.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=129;TFBS=V$EVI1_04;GESP=G:5333/A:7673;PPI=SMARCB1:846|TP53:846;PAC=NP_001077086.1,NP_055596.3,XP_005251387.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:103,9,0 0/0:8:0,9,75</p>

<p>8 97614625 rs3816208 G A 21 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1 GT:GQ:PL 0/0:9:0,6,45 0/0:18:0,15,111 1/1:3:63,9,0 0/0:9:0,6,47</p>

<p>8 144801593 rs60732298 C A 16.80 lowCVG AC1=5;AF1=0.6552;DB;DP=5;DP4=2,0,3,0;FQ=7.18;MQ=60;PV4=1,0.027,1,1;SNPEFF_AMINO_ACID_CHANGE=T221K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAPK15;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139021.2;VDB=0.0066;DN=129;DA=C/A;GM=NM_139021.2;GL=MAPK15;FG=missense;FD=missense;AAC=THR/LYS;PP=221/545;CDP=662;GS=78;PH=0.992;CP=0.104;CG=4.140;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=60;GESP=A:1162/C:11844;PAC=NP_620590.2 GT:GQ:PL 0/0:4:0,6,42 0/1:3:0,0,0 1/1:3:20,3,0 1/1:5:37,6,0</p>

<p>9 33387047 rs4008658 T G 66.90 highCVG AC1=5;AF1=0.5699;DB;DP=20;DP4=2,4,1,11;FQ=49.5;MQ=24;PV4=0.25,0.21,1.6e-05,0.46;SNPEFF_AMINO_ACID_CHANGE=K63T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AQP7;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170.1;VDB=0.0095;DN=108;DA=T/G;GM=NM_001170.1,XM_005251453.1,XM_005251454.1;GL=AQP7;FG=missense;FD=missense,unknown,unknown;AAC=LYS/THR;PP=63/343,63/343,63/274;CDP=188;GS=78;PH=0;CP=0.000;CG=-4.790;AA=G;HA=0.0;HE=0.0;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,suspect;DSP=44;KP=hsa03320:PPARSignalingPathway;PAC=NP_001161.1,XP_005251510.1,XP_005251511.1 GT:GQ:PL 0/1:27:37,0,25 0/1:17:22,0,15 1/1:10:52,12,0 0/1:4:0,2,50</p>

<p>9 107267352 rs7847413 T C 13.20 midCVG AC1=2;AF1=0.2756;DB;DP=14;DP4=2,9,3,0;FQ=14.5;MQ=60;PV4=0.027,0.17,1,1;SNPEFF_AMINO_ACID_CHANGE=I270T;SNPEFF_CODON_CHANGE=aTa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR13F1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004485.1;VDB=0.0024;DN=116;DA=T/C;GM=NM_001004485.1;GL=OR13F1;FG=missense;FD=missense;AAC=ILE/THR;PP=270/320;CDP=809;GS=89;PH=0.016;CP=0.942;CG=1.920;AA=T;HA=40.2;HE=8.4;HC=33.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=242782;KP=hsa04740:OlfactoryTransduction;TFBS=V$NFAT_Q6;GESP=C:2246/T:10760;PAC=NP_001004485.1 GT:GQ:PL 0/0:14:0,12,74 0/0:23:0,21,130 1/1:3:54,9,0 0/0:4:0,0,0</p>

<p>9 125330678 rs10985704 T G 52.30 midCVG AC1=5;AF1=0.6368;DB;DP=9;DP4=2,0,6,0;FQ=-3.17;MQ=60;PV4=1,0.0061,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T27P;SNPEFF_CODON_CHANGE=Aca/Cca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR1L8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001004454.1;VDB=0.0026;DN=120;DA=T/G;GM=NM_001004454.1;GL=OR1L8;FG=missense;FD=missense;AAC=THR/PRO;PP=27/310;CDP=79;GS=38;PH=0;CP=0.733;CG=3.480;AA=G;HA=43.0;HE=49.1;HC=1.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=57611;KP=hsa04740:OlfactoryTransduction;TFBS=V$FAC1_01;GESP=G:7411/T:5595;PAC=NP_001004454.1 GT:GQ:PL 0/1:4:0,3,25 0/1:14:37,0,13 1/1:4:20,3,0 1/1:6:38,6,0</p>

<p>9 130165973 rs34064803 T C 79.40 highCVG AC1=3;AF1=0.4231;DB;DP=25;DP4=0,4,2,15;FQ=51.8;MQ=60;PV4=1,0.0002,1,0.08;SNPEFF_AMINO_ACID_CHANGE=I253T;SNPEFF_CODON_CHANGE=aTc/aCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC2A8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014580.4;VDB=0.0144;DN=126;DA=T/C;GM=NM_001271711.1,NM_001271712.1,NM_014580.4,NR_073416.1,XM_005251949.1;GL=SLC2A8;FG=missense,missense,missense,non-coding-exon,missense;FD=missense,missense,missense,nc-transcript-variant,unknown;AAC=ILE/THR,ILE/THR,ILE/THR,none,ILE/THR;PP=253/412,90/315,253/478,NA,90/315;CDP=758,269,758,284,269;GS=89,89,89,NA,89;PH=0.878;CP=0.927;CG=5.420;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=35;GESP=C:1021/T:11985;PAC=NP_001258640.1,NP_001258641.1,NP_055395.2,NA,XP_005252006.1 GT:GQ:PL 0/0:6:0,6,47 0/1:3:0,0,0 1/1:46:122,51,0 0/0:6:0,6,45</p>

<p>9 134401335 rs2296957 C T 63.50 midCVG AC1=4;AF1=0.5532;DB;DP=9;DP4=1,2,1,5;FQ=24.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A180T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=UCK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001135954.1;VDB=0.0243;DN=100;DA=C/A/T;GM=NM_001135954.1,NM_001261450.1,NM_001261451.1,NM_031432.2,XM_005272224.1,XM_005272225.1;GL=UCK1;FG=missense,synonymous,synonymous,synonymous,synonymous,synonymous;FD=missense,synonymous-codon,synonymous-codon,synonymous-codon,unknown,unknown;AAC=ALA/THR,PRO,PRO,PRO,PRO,PRO;PP=180/202,202/269,216/283,211/278,225/292,188/255;CDP=538,606,648,633,675,564;GS=58,NA,NA,NA,NA,NA;PH=0.104;CP=0.908;CG=-5.080;AA=C;HA=41.2;HE=8.1;HC=7.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=20;KP=hsa00240:PyrimidineMetabolism|hsa00983:DrugMetabolism-OtherEnzymes|hsa01100:MetabolicPathways;TFBS=V$SREBP1_02;GESP=T:10799/C:2207;PAC=NP_001129426.1,NP_001248379.1,NP_001248380.1,NP_113620.1,XP_005272281.1,XP_005272282.1 GT:GQ:PL 0/1:3:0,0,0 0/0:7:0,9,60 1/1:15:103,18,0 0/1:3:0,0,0</p>

<p>9 135786904 rs1073123 A G 103 highCVG AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1 GT:GQ:PL 0/0:4:0,3,23 0/1:84:83,0,87 1/1:6:64,9,0 0/0:6:0,6,39</p>

<p>10 91007360 rs1051338 T G 24.80 lowCVG AC1=4;AF1=0.5432;DB;DP=5;DP4=1,1,0,3;FQ=4.88;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T16P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LIPA;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000235.3;VDB=0.0127;DN=86;DA=T/G;GM=NM_000235.2,NM_001127605.1;GL=LIPA;FG=missense;FD=missense;AAC=THR/PRO;PP=16/400;CDP=46;GS=38;PH=0.002;CP=0.997;CG=3.720;AA=T;HA=8.8;HE=33.0;HC=28.5;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=47;KP=hsa00100:SteroidBiosynthesis|hsa04142:Lysosome;GESP=G:3134/T:9872;PAC=NP_000226.2,NP_001121077.1 GT:GQ:PL 0/1:19:20,0,18 0/1:3:0,0,0 1/1:5:44,6,0 0/0:3:0,3,21</p>

<p>10 95459817 rs726817 C T 164 highCVG AC1=5;AF1=0.6264;DB;DP=16;DP4=2,3,6,5;FQ=31.4;MQ=60;PV4=1,1,1,0.14;SNPEFF_AMINO_ACID_CHANGE=R16H;SNPEFF_CODON_CHANGE=cGc/cAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FRA10AC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145246.4;VDB=0.0066;DN=86;DA=C/T;GM=NM_145246.4,XM_005269515.1,XM_005269516.1;GL=FRA10AC1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/HIS;PP=16/316;CDP=47;GS=29;PH=0.001;CP=0.794;CG=4.070;AA=T;HA=45.6;HE=24.6;HC=32.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=31;GESP=T:8629/C:4377;PPI=DGCR14:991|IK:974|PRPF40A:974|CHERP:974|THOC1:974|MFAP1:974|U2AF1:974|TTC14:974|PRPF3:974|SF3B2:974|ZCCHC10:974|SAP30BP:974|ZNF830:974|NKAP:974;PAC=NP_660289.2,XP_005269572.1,XP_005269573.1 GT:GQ:PL 0/1:34:33,0,36 0/1:13:61,0,12 1/1:14:98,15,0 0/1:20:17,0,24</p>

<p>11 6007613 rs4354673 T G 168 highCVG AC1=6;AF1=0.6957;DB;DP=18;DP4=0,3,5,9;FQ=4.33;MQ=48;PV4=0.51,1,0.054,1;SNPEFF_AMINO_ACID_CHANGE=K183T;SNPEFF_CODON_CHANGE=aAa/aCa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR52L1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005173.3;VDB=0.0083;DN=111;DA=T/G;GM=NM_001005173.2;GL=OR52L1;FG=missense;FD=missense;AAC=LYS/THR;PP=183/330;CDP=548;GS=78;PH=0.549;CP=0.001;CG=-0.167;AA=T;HA=44.5;HE=13.2;HC=10.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=166102;KP=hsa04740:OlfactoryTransduction;GESP=G:8663/T:3675;PAC=NP_001005173.2 GT:GQ:PL 0/1:11:38,0,11 0/1:3:0,6,34 1/1:15:87,15,0 1/1:15:91,15,0</p>

<p>11 17632651 rs7111528 C T 102 midCVG AC1=5;AF1=0.5888;DB;DP=10;DP4=1,2,4,3;FQ=15.1;MQ=60;PV4=1,1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=T1947M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=35;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OTOG;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001277269.1;VDB=0.0076;DN=116;DA=C/T;GM=NM_001277269.1,XM_005252894.1;GL=OTOG;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=1947/2926,953/1509;CDP=5840,2858;GS=81;PH=0.011;CP=0.000;CG=-0.815;AA=T;HA=48.7;HE=42.3;HC=17.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=320;PAC=NP_001264198.1,XP_005252951.1 GT:GQ:PL 0/0:4:0,6,33 0/1:3:0,3,25 1/1:5:50,6,0 1/1:13:99,15,0</p>

<p>11 56468047 rs2865519 A G 112 highCVG AC1=5;AF1=0.5869;DB;DP=23;DP4=3,4,8,7;FQ=50.7;MQ=40;PV4=1,1,0.0065,0.36;SNPEFF_AMINO_ACID_CHANGE=T62A;SNPEFF_CODON_CHANGE=Act/Gct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR9G1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005213.1;VDB=0.0113;DN=101;DA=A/G;GM=NM_001005213.1;GL=OR9G1;FG=missense;FD=missense;AAC=THR/ALA;PP=62/306;CDP=184;GS=58;PH=0.004;CP=0.961;CG=4.540;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=482111;TFBS=V$POU3F2_01;GESP=A:12994;PAC=NP_001005213.1 GT:GQ:PL 0/1:28:24,0,64 0/1:11:62,0,9 1/1:13:70,15,0 0/1:4:0,2,18</p>

<p>11 59940599 rs7232 T A 17.90 lowCVG AC1=4;AF1=0.4461;DB;DP=5;DP4=0,3,1,1;FQ=18;MQ=60;PV4=0.4,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T213S;SNPEFF_CODON_CHANGE=Act/Tct;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MS4A6A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001247999.1;VDB=0.0040;DN=52;DA=T/A;GM=NM_001247999.1,NM_022349.3,NM_152851.2,NM_152852.2,XM_005274177.1,XM_005274178.1,XM_005274179.1,XM_005274180.1,XM_005274181.1;GL=MS4A6A;FG=missense;FD=unknown,unknown,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=THR/SER,THR/SER,ASN/ILE,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER,THR/SER;PP=213/254,185/226,150/179,185/249,213/277,213/277,185/249,168/232,168/209;CDP=637,553,449,553,637,637,553,502,502;GS=58,58,149,58,58,58,58,58,58;PH=0.827;CP=0.001;CG=-5.110;AA=T;HA=1.8;HE=44.6;HC=8.9;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;CA;DSP=4;GESP=A:3608/T:9384;PAC=NP_001234928.1,NP_071744.2,NP_690590.1,NP_690591.1,XP_005274234.1,XP_005274235.1,XP_005274236.1,XP_005274237.1,XP_005274238.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:54,6,0 0/0:9:0,9,55</p>

<p>11 61165741 rs10897158 G C 74.40 midCVG AC1=5;AF1=0.6065;DB;DP=10;DP4=3,1,1,5;FQ=36.7;MQ=58;PV4=0.19,1,1,1;VDB=0.0083;DN=120;DA=G/C;GM=NM_001173990.2,NM_001173991.2,NM_016499.5,XM_005274039.1;GL=TMEM216;FG=splice-acceptor,missense,missense,missense;FD=splice-acceptor-variant,missense,missense,unknown;AAC=none,ARG/THR,ARG/THR,ARG/THR;PP=NA,147/149,86/88,86/88;CDP=NA,440,257,257;GS=NA,71,71,71;CP=1.000;CG=4.100;AA=G;HA=29.8;HE=16.9;HC=2.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=0;GESP=C:8600/G:3618;PAC=NP_001167461.1,NP_001167462.1,NP_057583.2,XP_005274096.1 GT:GQ:PL 0/1:10:35,0,8 0/0:6:0,9,70 1/1:4:24,3,0 1/1:8:61,9,0</p>

<p>11 63233721 rs149917445 G A 18 midCVG AC1=2;AF1=0.2266;DB;DP=12;DP4=4,5,3,0;FQ=19.1;MQ=60;PV4=0.2,0.33,1,1;SNPEFF_AMINO_ACID_CHANGE=T203M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HRASLS5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_054108.3;VDB=0.0092;DN=134;DA=G/A/T;GM=NM_001146728.1,NM_001146729.1,NM_054108.3;GL=HRASLS5;FG=missense;FD=unknown;AAC=THR/MET;PP=203/254,193/270,203/280;CDP=608,578,608;GS=81;PH=0.652;CP=0.562;CG=2.310;AA=G;DSP=90;GESP=A:4/G:12994;PAC=NP_001140200.1,NP_001140201.1,NP_473449.1 GT:GQ:PL 0/0:18:0,15,108 0/0:9:0,6,46 1/1:3:60,9,0 0/0:9:0,6,41</p>

<p>11 75152243 rs571353 C T 59.30 midCVG AC1=5;AF1=0.5871;DB;DP=10;DP4=4,1,4,1;FQ=40.4;MQ=60;PV4=1,1,1,0.17;SNPEFF_AMINO_ACID_CHANGE=A480T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GDPD5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_030792.6;VDB=0.0080;DN=83;DA=C/T;GM=NM_030792.6,XM_005274313.1,XM_005274314.1,XM_005274315.1,XM_005274316.1,XM_005274317.1,XM_005274318.1,XM_005274319.1;GL=GDPD5;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR;PP=480/606,480/607,480/607,480/607,480/607,480/606,235/362,235/361;CDP=1438,1438,1438,1438,1438,1438,703,703;GS=58;PH=0;CP=0.014;CG=-0.094;HA=44.1;HE=26.6;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=36;GESP=T:8514/C:4472;PAC=NP_110419.5,XP_005274370.1,XP_005274371.1,XP_005274372.1,XP_005274373.1,XP_005274374.1,XP_005274375.1,XP_005274376.1 GT:GQ:PL 0/1:7:30,0,4 0/0:9:0,12,77 1/1:4:26,3,0 1/1:5:48,6,0</p>

<p>11 123624658 rs12364099 G T 84.70 midCVG AC1=4;AF1=0.5037;DB;DP=14;DP4=1,3,8,1;FQ=24.8;MQ=60;PV4=0.052,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T190N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR6X1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001005188.1;VDB=0.0009;DN=120;DA=G/T;GM=NM_001005188.1;GL=OR6X1;FG=missense;FD=missense;AAC=THR/ASN;PP=190/313;CDP=569;GS=65;PH=0.029;CP=0.386;CG=3.460;AA=G;HA=1.7;HE=14.0;HC=2.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=12402;KP=hsa04740:OlfactoryTransduction;TFBS=V$YY1_01;GESP=T:1529/G:11473;PAC=NP_001005188.1 GT:GQ:PL 0/1:19:84,0,15 0/1:3:0,0,0 1/1:4:41,6,0 0/0:5:0,6,47</p>

<p>12 9307373 rs2377741 T G 29.10 midCVG AC1=6;AF1=0.7339;DB;DP=14;DP4=0,2,5,1;FQ=-4.32;MQ=50;PV4=0.11,0.007,0.18,1;SNPEFF_AMINO_ACID_CHANGE=T1205P;SNPEFF_CODON_CHANGE=Acc/Ccc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=29;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PZP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002864.2;VDB=0.0199;DN=100;DA=T/G;GM=NM_002864.2,XM_005253459.1;GL=PZP;FG=missense;FD=missense,unknown;AAC=THR/PRO;PP=1205/1483,991/1269;CDP=3613,2971;GS=38;PH=0;CP=0.039;CG=0.627;AA=G;HA=4.9;HE=0.5;HC=1.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=63;GESP=G:12716/T:290;PAC=NP_002855.2,XP_005253516.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:14:60,15,0 0/1:12:6,0,29</p>

<p>12 32135186 rs3759302 T A 62.90 midCVG AC1=4;AF1=0.5521;DB;DP=9;DP4=2,1,1,4;FQ=12.7;MQ=60;PV4=0.46,1,1,1;SNPEFF_AMINO_ACID_CHANGE=S433T;SNPEFF_CODON_CHANGE=Tct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KIAA1551;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018169.3;VDB=0.0034;DN=107;DA=T/A;GM=NM_018169.3,XM_005253405.1;GL=KIAA1551;FG=missense;FD=missense,unknown;AAC=SER/THR;PP=433/1748;CDP=1297;GS=58;PH=0.999;CP=0.001;CG=-1.130;AA=T;HA=3.4;HE=18.1;HC=11.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=1375;GESP=A:2107/T:10899;PAC=NP_060639.3,XP_005253462.1 GT:GQ:PL 0/1:3:0,3,16 0/0:5:0,6,38 1/1:3:26,3,0 1/1:10:82,12,0</p>

<p>12 40758652 rs3761863 T C 14.60 lowCVG AC1=6;AF1=0.6706;DB;DP=5;DP4=1,0,2,1;FQ=-12.1;MQ=60;PV4=1,0.14,1,0.29;SNPEFF_AMINO_ACID_CHANGE=M2397T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=49;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LRRK2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_198578.3;VDB=0.0050;DN=107;DA=T/C;GM=NM_198578.3,XM_005268629.1,XM_005268630.1,XM_005268631.1;GL=LRRK2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=MET/THR;PP=2397/2528,2397/2522,2397/2528,2357/2488;CDP=7190,7190,7190,7070;GS=81;PH=0;CP=0.005;CG=3.030;AA=C;HA=46.5;HE=34.3;HC=42.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=9;KP=hsa05012:Parkinson'sDisease;GESP=C:7962/T:5042;PPI=STUB1:846;PAC=NP_940980.3,XP_005268686.1,XP_005268687.1,XP_005268688.1 GT:GQ:PL 0/1:20:30,0,18 0/1:3:0,0,0 1/1:3:19,3,0 0/1:3:0,0,0</p>

<p>12 42863266 rs79087668 C T 63.10 midCVG AC1=5;AF1=0.5836;DB;DP=10;DP4=2,2,2,4;FQ=39.8;MQ=60;PV4=1,0.25,1,0.22;SNPEFF_AMINO_ACID_CHANGE=A124T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PRICKLE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001144881.1;VDB=0.0077;DN=131;DA=C/T;GM=NM_001144881.1,NM_001144882.1,NM_001144883.1,NM_153026.2;GL=PRICKLE1;FG=missense;FD=missense;AAC=ALA/THR;PP=124/832;CDP=370;GS=58;PH=0.002;CP=0.927;CG=3.550;AA=C;DV=by-frequency,by-cluster;DSP=15;KP=hsa04310:WntSignalingPathway;GESP=T:22/C:12984;PAC=NP_001138353.1,NP_001138354.1,NP_001138355.1,NP_694571.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,26 1/1:10:83,12,0 0/1:25:21,0,51</p>

<p>12 49952449 . C T 14.40 lowCVG AC1=4;AF1=0.5125;DP=6;DP4=3,0,0,3;FQ=12.6;MQ=60;PV4=0.1,1,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A469T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MCRS1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001012300.1;VDB=0.0104;GM=NM_001012300.1,NM_001278341.1,NM_006337.4,XM_005268572.1,XM_005268573.1;GL=MCRS1;FG=missense;FD=unknown;AAC=ALA/THR;PP=469/476,265/272,456/463,456/463,367/374;CDP=1405,793,1366,1366,1099;GS=58;PH=0.997;CP=1.000;CG=5.280;AA=C;DSP=64;TFBS=V$PAX4_01;GESP=C:13006;PPI=INO80E:892|DAXX:846|INO80:811|PBX2:786;PAC=NP_001012300.1,NP_001265270.1,NP_006328.2,XP_005268629.1,XP_005268630.1 GT:GQ:PL 0/1:7:5,0,32 0/1:3:0,0,0 1/1:4:47,6,0 0/0:4:0,3,17</p>

<p>12 53165940 rs6580904 C T 76.30 midCVG AC1=5;AF1=0.5683;DB;DP=11;DP4=2,1,5,3;FQ=13.2;MQ=60;PV4=1,0.37,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A359T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KRT76;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015848.4;VDB=0.0153;DN=116;DA=C/T;GM=NM_015848.4;GL=KRT76;FG=missense;FD=missense;AAC=ALA/THR;PP=359/639;CDP=1075;GS=58;PH=0.986;CP=0.890;CG=4.420;AA=C;HA=39.2;HE=5.8;HC=36.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=63;GESP=T:9195/C:3811;PAC=NP_056932.2 GT:GQ:PL 0/1:3:0,3,15 0/1:15:12,0,21 1/1:13:85,15,0 0/1:19:24,0,18</p>

<p>13 51854694 . C A 42.70 highCVG AC1=3;AF1=0.3358;DP=25;DP4=6,7,12,0;FQ=44.4;MQ=60;PV4=0.0052,0.15,1,1;SNPEFF_AMINO_ACID_CHANGE=P351T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FAM124A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_145019.3;VDB=0.0000;GM=NM_001242312.1,NM_145019.3;GL=FAM124A;FG=missense;FD=unknown;AAC=PRO/THR;PP=315/547,351/583;CDP=943,1051;GS=38;PH=0.034;CP=0.000;CG=0.795;AA=C;DSP=109;GESP=C:13006;PAC=NP_001229241.1,NP_659456.3 GT:GQ:PL 0/0:15:0,15,93 0/0:3:0,0,0 1/1:16:87,22,0 0/0:21:0,21,125</p>

<p>14 63174960 rs4902176 C T 17 highCVG AC1=4;AF1=0.4355;DB;DP=15;DP4=2,8,1,3;FQ=19.4;MQ=60;PV4=1,1,1,0.21;SNPEFF_AMINO_ACID_CHANGE=A745T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNH5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_139318.4;VDB=0.0080;DN=111;DA=C/T;GM=NM_139318.4,NM_172375.2;GL=KCNH5;FG=missense,3-prime-UTR;FD=unknown;AAC=ALA/THR,none;PP=745/989,NA;CDP=2233,NA;GS=58,NA;PH=0.017;CP=0.997;CG=5.520;AA=C;HA=19.0;HE=34.4;HC=15.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=214;GESP=T:3891/C:9115;PAC=NP_647479.2,NA GT:GQ:PL 0/0:14:0,15,81 0/1:13:12,0,62 1/1:3:46,6,0 0/1:3:0,0,0</p>

<p>14 64491695 rs4902264 T C 97.50 midCVG AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:18:122,18,0 0/1:19:13,0,55</p>

<p>14 74759477 rs4148077 C T 120 midCVG AC1=6;AF1=0.7273;DB;DP=13;DP4=3,0,6,3;FQ=8.59;MQ=60;PV4=0.51,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A304T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ABCD4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005050.3;VDB=0.0175;DN=110;DA=C/T;GM=NM_005050.3,NR_003256.2,XM_005267935.1,XM_005267936.1,XM_005267937.1,XM_005267938.1,XM_005267939.1,XM_005267940.1,XM_005267941.1,XM_005267942.1,XM_005267943.1,XM_005267944.1,XM_005267945.1,XM_005267946.1,XM_005267947.1,XM_005267948.1,XM_005267949.1,XM_005267950.1,XM_005267951.1,XM_005267952.1,XM_005267953.1,XM_005267954.1;GL=ABCD4;FG=missense,non-coding-exon,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense,missense;FD=missense,nc-transcript-variant,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=ALA/THR,none,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR,ALA/THR;PP=304/607,NA,287/576,304/529,200/489,304/475,166/469,166/469,166/469,166/469,304/455,145/448,145/448,145/448,145/448,145/448,145/448,145/448,41/344,41/344,41/344,41/344;CDP=910,934,859,910,598,910,496,496,496,496,910,433,433,433,433,433,433,433,121,121,121,121;GS=58,NA,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58,58;PH=0.003;CP=0.998;CG=2.140;AA=C;HA=16.7;HE=29.2;HC=23.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=27;KP=hsa02010:ABCTransporters|hsa04146:Peroxisome;GESP=T:3999/C:9007;PAC=NP_005041.1,NA,XP_005267992.1,XP_005267993.1,XP_005267994.1,XP_005267995.1,XP_005267996.1,XP_005267997.1,XP_005267998.1,XP_005267999.1,XP_005268000.1,XP_005268001.1,XP_005268002.1,XP_005268003.1,XP_005268004.1,XP_005268005.1,XP_005268006.1,XP_005268007.1,XP_005268008.1,XP_005268009.1,XP_005268010.1,XP_005268011.1 GT:GQ:PL 0/1:18:12,0,27 0/1:18:17,0,20 1/1:16:88,15,0 1/1:8:48,6,0</p>

<p>14 88459795 rs73312836 C T 22.20 midCVG AC1=3;AF1=0.3888;DB;DP=11;DP4=3,5,2,1;FQ=24.4;MQ=60;PV4=0.55,1,1,0.005;SNPEFF_AMINO_ACID_CHANGE=A17T;SNPEFF_CODON_CHANGE=Gcg/Acg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=GALC;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001201402.1;VDB=0.0092;DN=130;DA=C/T;GM=NM_001201402.1;GL=GALC;FG=missense;FD=missense;AAC=ALA/THR;PP=17/660;CDP=49;GS=58;CP=0.000;CG=-0.125;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=69;KP=hsa00600:SphingolipidMetabolism|hsa01100:MetabolicPathways|hsa04142:Lysosome;PAC=NP_001188331.1 GT:GQ:PL 0/0:15:0,15,85 0/1:4:20,3,0 1/1:3:48,6,0 0/0:9:0,9,64</p>

<p>16 56673227 rs11640851 C A 13.60 lowCVG AC1=4;AF1=0.5;DB;DP=6;DP4=2,0,1,1;FQ=13.6;MQ=60;PV4=1,0.23,1,1;SNPEFF_AMINO_ACID_CHANGE=T27N;SNPEFF_CODON_CHANGE=aCc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MT1A;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_005946.2;VDB=0.0011;DN=120;DA=C/A;GM=NM_005946.2;GL=MT1A;FG=missense;FD=missense;AAC=THR/ASN;PP=27/62;CDP=80;GS=65;PH=0.928;CP=0.839;CG=1.960;AA=C;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;GESP=A:6838/C:6158;PAC=NP_005937.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:3:49,6,0 0/0:7:0,6,49</p>

<p>16 75563330 rs3826107 G A 36.20 lowCVG AC1=5;AF1=0.6482;DB;DP=5;DP4=0,1,4,0;FQ=-9.91;MQ=60;PV4=0.2,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T318M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CHST5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024533.4;VDB=0.0001;DN=107;DA=G/A;GM=NM_024533.4,XM_005255880.1;GL=CHST5;FG=missense;FD=missense,unknown;AAC=THR/MET;PP=318/412,324/418;CDP=953,971;GS=81;PH=0.999;CP=0.994;CG=2.840;AA=G;HA=9.7;HE=15.6;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=906;CPG=75563114-75564112;GESP=A:2238/G:10758;PAC=NP_078809.2,XP_005255937.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:10:73,12,0 0/1:3:0,0,0</p>

<p>17 5087040 rs9908414 A G 60.50 midCVG AC1=4;AF1=0.4687;DB;DP=13;DP4=4,4,0,5;FQ=63.2;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=I171T;SNPEFF_CODON_CHANGE=aTt/aCt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF594;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032530.1;VDB=0.0105;DN=119;DA=A/G;GM=NM_032530.1,XM_005256827.1;GL=ZNF594;FG=missense;FD=missense,unknown;AAC=ILE/THR;PP=171/808;CDP=512;GS=89;PH=0;CP=0.203;CG=-2.720;AA=A;HA=9.7;HE=47.3;HC=7.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=532;GESP=A:12386;PAC=NP_115919.1,XP_005256884.1 GT:GQ:PL 0/0:16:0,18,107 0/0:6:0,6,37 1/1:6:70,9,0 1/1:4:39,6,0</p>

<p>17 26694861 rs704 G A 31.80 highCVG AC1=4;AF1=0.4318;DB;DP=15;DP4=3,7,1,3;FQ=34;MQ=60;PV4=1,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=T400M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=VTN;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000638.3;VDB=0.0089;DN=36;DA=G/A;GM=NM_000638.3;GL=VTN;FG=missense;FD=missense;AAC=THR/MET;PP=400/479;CDP=1199;GS=81;PH=0.943;CP=0.000;CG=-8.940;AA=G;HA=35.0;HE=46.9;HC=25.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=126;KP=hsa04510:FocalAdhesion|hsa04512:ECM-receptorInteraction;GESP=A:6627/G:6379;PAC=NP_000629.3 GT:GQ:PL 0/0:12:0,12,67 0/1:3:0,0,0 1/1:5:69,9,0 0/1:5:3,0,95</p>

<p>17 27284443 rs12941884 A G 58.10 midCVG AC1=5;AF1=0.587;DB;DP=9;DP4=2,0,4,1;FQ=5.4;MQ=60;PV4=1,0.48,1,1;SNPEFF_AMINO_ACID_CHANGE=M806T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEZ6;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_178860.4;VDB=0.0169;DN=121;DA=A/G;GM=NM_001098635.1,NM_178860.4,XM_005257919.1;GL=SEZ6;FG=missense;FD=missense,missense,unknown;AAC=MET/THR;PP=806/994,806/995,681/870;CDP=2417,2417,2042;GS=81;PH=0;CP=0.999;CG=3.650;AA=G;HA=20.8;HE=14.6;HC=8.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=78;GESP=G:1806/A:10402;PAC=NP_001092105.1,NP_849191.3,XP_005257976.1 GT:GQ:PL 0/1:3:0,0,0 0/1:18:21,0,17 1/1:10:78,12,0 0/1:3:0,3,24</p>

<p>17 48598785 rs1380657 G A 174 highCVG AC1=6;AF1=0.7336;DB;DP=19;DP4=3,2,5,7;FQ=44.9;MQ=60;PV4=0.62,0.38,1,1;SNPEFF_AMINO_ACID_CHANGE=A424T;SNPEFF_CODON_CHANGE=Gca/Aca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MYCBPAP;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032133.4;VDB=0.0177;DN=88;DA=G/A;GM=NM_032133.4,XM_005257725.1,XM_005257726.1,XM_005257727.1;GL=MYCBPAP;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=424/985,439/1001,439/1000,424/986;CDP=1270,1315,1315,1270;GS=58;PH=0;CP=0.000;CG=-4.290;AA=A;HA=7.6;HE=27.6;HC=11.3;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=8;GESP=A:10167/G:2839;PPI=MYCBP:846;PAC=NP_115509.4,XP_005257782.1,XP_005257783.1,XP_005257784.1 GT:GQ:PL 0/1:14:16,0,16 0/1:38:31,0,68 1/1:22:130,21,0 1/1:8:42,6,0</p>

<p>17 59544932 . C A 13.30 midCVG AC1=3;AF1=0.3359;DP=11;DP4=2,6,0,3;FQ=14.9;MQ=60;PV4=1,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=P155T;SNPEFF_CODON_CHANGE=Cct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBX4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018488.2;VDB=0.0039;GM=NM_018488.2,XM_005257835.1,XM_005257836.1,XM_005257837.1,XM_005257838.1;GL=TBX4;FG=missense;FD=unknown;AAC=PRO/THR;PP=155/546,174/566,174/565,155/547,155/547;CDP=463,520,520,463,463;GS=38;PH=1;CP=1.000;CG=5.770;AA=C;DSP=62;GESP=C:13006;PAC=NP_060958.2,XP_005257892.1,XP_005257893.1,XP_005257894.1,XP_005257895.1 GT:GQ:PL 0/0:6:0,3,19 0/0:3:0,0,0 1/1:4:53,9,0 0/0:22:0,21,120</p>

<p>17 74151720 rs201646108 C T 56.70 highCVG AC1=3;AF1=0.3694;DB;DP=26;DP4=6,12,3,3;FQ=58.7;MQ=59;PV4=0.63,1,0.041,0.25;SNPEFF_AMINO_ACID_CHANGE=A541T;SNPEFF_CODON_CHANGE=Gcc/Acc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RNF157;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052916.2;VDB=0.0188;DN=137;DA=C/T;GM=NM_052916.2,XM_005257006.1,XM_005257007.1,XM_005257008.1;GL=RNF157;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=541/680,541/658,541/654,474/613;CDP=1621,1621,1621,1420;GS=58;PH=1;CP=0.755;CG=5.430;AA=C;DV=by-cluster;DSP=78;GESP=T:5/C:13001;PAC=NP_443148.1,XP_005257063.1,XP_005257064.1,XP_005257065.1 GT:GQ:PL 0/0:9:0,9,60 0/1:29:29,0,78 1/1:7:75,12,0 0/0:24:0,24,130</p>

<p>17 76697783 . A G 12.40 lowCVG AC1=2;AF1=0.2844;DP=8;DP4=1,4,3,0;FQ=13.7;MQ=60;PV4=0.14,1,1,0.34;SNPEFF_AMINO_ACID_CHANGE=F136S;SNPEFF_CODON_CHANGE=tTc/tCc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CYTH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_004762.3;VDB=0.0000;GM=NM_004762.2,NM_017456.2;GL=CYTH1;FG=missense;FD=unknown;AAC=PHE/SER;PP=136/399,136/398;CDP=407;GS=155;PH=1;CP=1.000;CG=4.950;AA=A;DSP=31;GESP=A:13006;PPI=ITGB2:937|CYTIP:846;PAC=NP_004753.1,NP_059430.2 GT:GQ:PL 0/0:6:0,3,20 0/0:11:0,9,45 1/1:3:53,9,0 0/0:6:0,3,26</p>

<p>19 4512669 rs55901236 T C 66.40 midCVG AC1=6;AF1=0.6769;DB;DP=10;DP4=0,2,6,1;FQ=-15;MQ=60;PV4=0.083,1,1,0.31;SNPEFF_AMINO_ACID_CHANGE=T421A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PLIN4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080400.1;VDB=0.0122;DN=129;DA=T/C;GM=NM_001080400.1,XM_005259640.1,XM_005259641.1;GL=PLIN4;FG=missense;FD=missense,unknown,unknown;AAC=THR/ALA;PP=421/1358,436/1373,435/1372;CDP=1261,1306,1303;GS=58;PH=0.009;CP=0.001;CG=0.087;AA=T;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1045;TFBS=V$GR_Q6;GESP=C:5321/T:7027;PAC=NP_001073869.1,XP_005259697.1,XP_005259698.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:25,3,0 0/1:17:76,0,15</p>

<p>19 9056982 rs11670318 G A 109 midCVG AC1=5;AF1=0.6629;DB;DP=11;DP4=1,2,6,2;FQ=5.06;MQ=60;PV4=0.49,1,1,0.28;SNPEFF_AMINO_ACID_CHANGE=T10155I;SNPEFF_CODON_CHANGE=aCa/aTa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUC16;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024690.2;VDB=0.0045;DN=120;DA=G/A;GM=NM_024690.2,XM_005260161.1,XM_005260162.1;GL=MUC16;FG=missense;FD=missense,unknown,unknown;AAC=THR/ILE;PP=10155/14508,2069/6405,2069/6025;CDP=30464,6206,6206;GS=89;PH=0.997;CP=0.000;CG=0.467;AA=G;HA=16.4;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=810;GESP=A:3367/G:8779;PAC=NP_078966.2,XP_005260218.1,XP_005260219.1 GT:GQ:PL 0/1:30:47,0,29 0/1:3:0,0,0 1/1:9:61,9,0 0/1:11:41,0,10</p>

<p>19 12772165 rs1054487 G A 27.30 lowCVG AC1=3;AF1=0.3966;DB;DP=8;DP4=1,3,3,1;FQ=29.4;MQ=60;PV4=0.49,0.0045,1,1;SNPEFF_AMINO_ACID_CHANGE=T312I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MAN2B1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000528.3;VDB=0.0063;DN=86;DA=G/A/T;GM=NM_000528.3,NM_001173498.1,XM_005259913.1;GL=MAN2B1;FG=missense;FD=missense,missense,unknown;AAC=THR/ILE;PP=312/1012,312/1011,312/1013;CDP=935;GS=89;PH=0.003;CP=0.015;CG=-2.190;AA=A;HA=40.2;HE=38.7;HC=7.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=26;KP=hsa00511:OtherGlycanDegradation|hsa04142:Lysosome;GESP=A:5899/G:7107;PAC=NP_000519.2,NP_001166969.1,XP_005259970.1 GT:GQ:PL 0/0:5:0,3,25 0/1:3:0,0,0 1/1:7:68,12,0 0/0:9:0,9,72</p>

<p>19 21909948 rs7246400 G A 38.50 lowCVG AC1=5;AF1=0.6627;DB;DP=8;DP4=0,1,0,6;FQ=-6.85;MQ=34;PV4=1,0.01,1,0.36;SNPEFF_AMINO_ACID_CHANGE=T389I;SNPEFF_CODON_CHANGE=aCt/aTt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF100;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173531.3;VDB=0.0289;DN=116;DA=G/A;GM=NM_173531.3,XM_005259783.1,XM_005259784.1,XM_005259785.1;GL=ZNF100;FG=missense;FD=missense,unknown,unknown,unknown;AAC=THR/ILE;PP=389/543,388/542,356/510,174/328;CDP=1166,1163,1067,521;GS=89;PH=0.804;CP=0.029;CG=0.841;AA=G;HA=30.4;HC=40.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=844;GESP=A:6476/G:6444;PAC=NP_775802.2,XP_005259840.1,XP_005259841.1,XP_005259842.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:16:76,18,0 0/1:3:0,3,27</p>

<p>19 37118439 rs61732180 C T 83.80 midCVG AC1=5;AF1=0.6451;DB;DP=9;DP4=3,0,3,3;FQ=15.4;MQ=60;PV4=0.46,1,1,0.002;SNPEFF_AMINO_ACID_CHANGE=T547M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF382;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_032825.4;VDB=0.0028;DN=129;DA=C/T;GM=NM_001256838.1,NM_032825.4,XM_005259347.1,XM_005259348.1;GL=ZNF382;FG=missense;FD=missense,missense,unknown,unknown;AAC=THR/MET;PP=546/550,547/551,547/551,546/550;CDP=1637,1640,1640,1637;GS=81;PH=0;CP=0.009;CG=1.990;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=1408;GESP=T:2043/C:10963;PAC=NP_001243767.1,NP_116214.2,XP_005259404.1,XP_005259405.1 GT:GQ:PL 0/1:17:27,0,16 0/0:4:0,6,41 1/1:9:59,9,0 1/1:6:44,6,0</p>

<p>19 40929363 rs268687 T C 115 highCVG AC1=6;AF1=0.7126;DB;DP=15;DP4=1,5,4,4;FQ=41.3;MQ=60;PV4=0.3,1,1,0.11;SNPEFF_AMINO_ACID_CHANGE=T31A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SERTAD1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_013376.3;VDB=0.0100;DN=79;DA=T/C;GM=NM_013376.3;GL=SERTAD1;FG=missense;FD=missense;AAC=THR/ALA;PP=31/237;CDP=91;GS=58;PH=0;CP=0.074;CG=1.500;AA=C;HA=8.0;HE=48.6;HC=21.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=91;TFBS=V$MYOGNF1_01;GESP=C:8054/T:4886;PPI=CREBBP:846|CDKN2A:845|CDK4:845;PAC=NP_037508.2 GT:GQ:PL 0/1:34:35,0,35 0/1:14:7,0,45 1/1:7:52,6,0 1/1:10:65,9,0</p>

<p>19 48495304 rs60213124 G A 79 midCVG AC1=4;AF1=0.5008;DB;DP=12;DP4=3,3,3,3;FQ=79.1;MQ=60;PV4=1,0.42,1,0.17;SNPEFF_AMINO_ACID_CHANGE=T12M;SNPEFF_CODON_CHANGE=aCg/aTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BSPH1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128326.2;VDB=0.0230;DN=129;DA=G/A;GM=NM_001128326.2;GL=BSPH1;FG=missense;FD=missense;AAC=THR/MET;PP=12/133;CDP=35;GS=81;PH=0.962;CP=0.000;CG=-0.642;AA=G;DV=by-frequency,by-cluster;DSP=39;GESP=G:4566;PAC=NP_001121798.1 GT:GQ:PL 0/0:13:0,15,105 0/1:3:27,3,0 1/1:12:99,15,0 0/0:3:0,3,21</p>

<p>19 49438363 rs2270941 G A 47.70 midCVG AC1=5;AF1=0.5687;DB;DP=10;DP4=0,4,2,3;FQ=37.1;MQ=60;PV4=0.44,0.036,1,1;SNPEFF_AMINO_ACID_CHANGE=S66N;SNPEFF_CODON_CHANGE=aGc/aAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=DHDH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_014475.3;VDB=0.0081;DN=100;DA=G/A;GM=NM_014475.3,XM_005258748.1;GL=DHDH;FG=missense;FD=missense,unknown;AAC=SER/ASN,ALA/THR;PP=66/335,9/223;CDP=197,25;GS=46,58;PH=0;CP=0.282;CG=4.740;AA=A;HA=22.1;HE=21.4;HC=45.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=6;KP=hsa00040:PentoseAndGlucuronateInterconversions|hsa00980:MetabolismOfXenobioticsByCytochromeP450;GESP=A:4923/G:8051;PAC=NP_055290.1,XP_005258805.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,23 1/1:7:65,9,0 0/1:26:23,0,51</p>

<p>19 53086124 rs373554 C A 100 highCVG AC1=5;AF1=0.6181;DB;DP=15;DP4=5,1,5,4;FQ=54.4;MQ=59;PV4=0.58,0.21,0.22,0.18;SNPEFF_AMINO_ACID_CHANGE=T337K;SNPEFF_CODON_CHANGE=aCg/aAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF701;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172655.1;VDB=0.0155;DN=80;DA=C/A;GM=NM_001172655.1,NM_018260.2;GL=ZNF701;FG=missense;FD=missense;AAC=THR/LYS;PP=337/532,271/466;CDP=1010,812;GS=78;PH=0;CP=0.001;CG=-0.451;AA=A;HA=0.0;HE=17.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=670;GESP=A:9195/C:3811;PAC=NP_001166126.1,NP_060730.2 GT:GQ:PL 0/1:22:17,0,33 0/1:16:21,0,15 1/1:11:58,12,0 0/1:47:49,0,47</p>

<p>19 54755922 rs12977062 C T 24.20 highCVG AC1=3;AF1=0.4158;DB;DP=22;DP4=1,11,4,0;FQ=26.4;MQ=48;PV4=0.0027,1,0.05,1;SNPEFF_AMINO_ACID_CHANGE=A542T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=LILRB5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001081442.1;VDB=0.0083;DN=121;DA=C/T;GM=NM_001081442.1,NM_001081443.1,NM_006840.3,XM_005258435.1;GL=LILRB5;FG=missense;FD=missense,missense,missense,unknown;AAC=ALA/THR;PP=542/592,442/492,541/591,525/575;CDP=1624,1324,1621,1573;GS=58;PH=0.013;CP=0.000;CG=-2.320;AA=C;HA=0.0;HE=4.5;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=6;GESP=T:2224/C:10782;PAC=NP_001074911.1,NP_001074912.1,NP_006831.1,XP_005258492.1 GT:GQ:PL 0/0:26:0,27,82 0/0:9:0,9,70 1/1:4:37,6,0 1/1:4:35,6,0</p>

<p>19 58928786 rs7257872 A G 26.90 highCVG AC1=2;AF1=0.2698;DB;DP=17;DP4=12,2,3,0;FQ=28.2;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T301A;SNPEFF_CODON_CHANGE=Aca/Gca;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ZNF584;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173548.1;VDB=0.0062;DN=116;DA=A/G;GM=NM_173548.1,XM_005258628.1,XM_005258629.1,XM_005258630.1,XM_005258631.1;GL=ZNF584;FG=missense,missense,missense,3-prime-UTR,3-prime-UTR;FD=missense,unknown,unknown,unknown,unknown;AAC=THR/ALA,THR/ALA,THR/ALA,none,none;PP=301/422,293/414,256/377,NA,NA;CDP=901,877,766,NA,NA;GS=58,58,58,NA,NA;PH=0.001;CP=0.822;CG=0.242;AA=A;HA=49.6;HE=26.3;HC=23.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=609;GESP=G:4324/A:8682;PAC=NP_775819.1,XP_005258685.1,XP_005258686.1,NA,NA GT:GQ:PL 0/0:29:0,27,139 0/0:4:0,0,0 1/1:3:68,9,0 0/0:17:0,15,76</p>

<p>19 59023174 rs35350976 A G 35.70 lowCVG AC1=5;AF1=0.5468;DB;DP=8;DP4=3,1,2,2;FQ=38.9;MQ=60;PV4=1,0.061,1,1;SNPEFF_AMINO_ACID_CHANGE=M50T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SLC27A5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_012254.2;VDB=0.0051;DN=126;DA=A/G;GM=NM_012254.2,XM_005258437.1,XM_005258438.1;GL=SLC27A5;FG=missense;FD=missense,unknown,unknown;AAC=MET/THR;PP=50/691,50/607,50/396;CDP=149;GS=81;PH=0;CP=0.000;CG=-0.944;AA=A;DG;DV=by-frequency,by-cluster,by-1000G;DSP=288;KP=hsa00120:PrimaryBileAcidBiosynthesis|hsa01100:MetabolicPathways|hsa03320:PPARSignalingPathway;GESP=G:2198/A:10646;PAC=NP_036386.1,XP_005258494.1,XP_005258495.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,33 1/1:6:69,9,0 0/1:9:6,0,61</p>

<p>20 17716416 rs1053993 C G 89 midCVG AC1=5;AF1=0.6077;DB;DP=14;DP4=3,4,3,4;FQ=77.9;MQ=60;PV4=1,0.18,1,0.1;SNPEFF_AMINO_ACID_CHANGE=T85S;SNPEFF_CODON_CHANGE=aCt/aGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=BANF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001159495.1;VDB=0.0033;DN=86;DA=C/G;GM=NM_001014977.3,NM_001159495.1,NM_178477.4,XM_005260668.1;GL=BANF2;FG=missense;FD=missense,missense,missense,unknown;AAC=THR/SER;PP=78/91,85/98,78/91,78/91;CDP=233,254,233,233;GS=58;PH=0;CP=0.107;CG=3.360;AA=G;HA=45.0;HE=22.5;HC=48.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=107;GESP=G:4533/C:8473;PAC=NP_001014977.2,NP_001152967.1,NP_848572.3,XP_005260725.1 GT:GQ:PL 0/1:8:3,0,60 0/0:6:0,9,61 1/1:8:66,9,0 1/1:8:67,9,0</p>

<p>21 38525356 rs1053808 T C 81.50 midCVG AC1=6;AF1=0.6741;DB;DP=14;DP4=3,3,5,2;FQ=64.4;MQ=57;PV4=0.59,0.32,0.0091,0.078;SNPEFF_AMINO_ACID_CHANGE=M840T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=27;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TTC3;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001001894.1;VDB=0.0087;DN=86;DA=T/C;GM=NM_001001894.1,NM_003316.3,XM_005261045.1,XM_005261046.1,XM_005261047.1,XM_005261048.1,XM_005261049.1,XM_005261050.1,XM_005261051.1,XM_005261052.1,XM_005261053.1,XM_005261054.1,XM_005261055.1,XM_005261056.1,XM_005261057.1,XM_005261058.1;GL=TTC3;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=840/2026,840/2026,878/2082,862/2066,878/2064,856/2060,856/2060,856/2060,862/2048,840/2044,840/2044,840/2044,530/1734,530/1734,118/1322,878/1078;CDP=2519,2519,2633,2585,2633,2567,2567,2567,2585,2519,2519,2519,1589,1589,353,2633;GS=81;PH=0;CP=0.888;CG=-0.107;AA=C;HA=25.0;HE=46.7;HC=34.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=106;GESP=C:7210/T:5796;PAC=NP_001001894.1,NP_003307.3,XP_005261102.1,XP_005261103.1,XP_005261104.1,XP_005261105.1,XP_005261106.1,XP_005261107.1,XP_005261108.1,XP_005261109.1,XP_005261110.1,XP_005261111.1,XP_005261112.1,XP_005261113.1,XP_005261114.1,XP_005261115.1 GT:GQ:PL 0/1:30:24,0,88 0/1:14:50,0,13 1/1:5:28,3,0 1/1:5:21,3,0</p>

<p>21 47786524 rs2839227 A G 27.90 midCVG AC1=4;AF1=0.5236;DB;DP=10;DP4=6,0,0,4;FQ=31;MQ=60;PV4=0.0048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T879A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0116;DN=100;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1,XM_005261130.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ALA;PP=879/3337,879/3348,879/3347,879/3269,879/3259,761/3230,761/3140,879/2707;CDP=2635,2635,2635,2635,2635,2281,2281,2635;GS=58;PH=0.264;CP=0.002;CG=1.810;AA=A;HA=38.8;HE=17.7;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26;GESP=G:3367/A:9597;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1,XP_005261187.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,40 1/1:6:57,9,0 0/1:15:12,0,61</p>

<p>22 21357119 rs71314757 G A 67 lowCVG AC1=6;AF1=0.7034;DB;DP=8;DP4=0,2,1,5;FQ=8.97;MQ=60;PV4=1,0.3,1,1;VDB=0.0209;DN=130;DA=G/A;GM=NR_027051.1,NR_027052.1;GL=THAP7-AS1;FG=splice-donor;FD=splice-donor-variant;CP=0.010;CG=-2.790;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=0 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,6,45 1/1:14:81,15,0 1/1:4:28,3,0</p>

<p>22 21800806 . T C 11.20 highCVG AC1=2;AF1=0.2696;DP=20;DP4=7,10,3,0;FQ=12.4;MQ=60;PV4=0.21,0.048,1,0.00028;SNPEFF_AMINO_ACID_CHANGE=M541T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=HIC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015094.2;VDB=0.0009;GM=NM_015094.2,XM_005261395.1;GL=HIC2;FG=missense;FD=unknown;AAC=MET/THR;PP=541/616;CDP=1622;GS=81;PH=0.963;CP=1.000;CG=4.350;AA=T;DSP=1596;CPG=21800413-21801052;GESP=T:13006;PAC=NP_055909.2,XP_005261452.1 GT:GQ:PL 0/0:39:0,36,185 0/0:4:0,0,0 1/1:3:52,9,0 0/0:18:0,15,98</p>

<p>22 29913272 rs737976 C T 154 midCVG AC1=5;AF1=0.6744;DB;DP=14;DP4=0,2,6,6;FQ=-8.35;MQ=60;PV4=0.47,0.21,1,1;SNPEFF_AMINO_ACID_CHANGE=V525I;SNPEFF_CODON_CHANGE=Gtt/Att;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=17;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=THOC5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001002877.1;VDB=0.0074;DN=86;DA=C/T;GM=NM_001002877.1,NM_001002878.1,NM_001002879.1,NM_003678.4,XM_005261797.1,XM_005261798.1,XM_005261799.1,XM_005261800.1,XM_005261801.1;GL=THOC5;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown;AAC=VAL/ILE;PP=525/684,525/684,525/684,525/684,536/695,536/695,536/695,252/411,186/345;CDP=1573,1573,1573,1573,1606,1606,1606,754,556;GS=29;PH=0;CP=0.005;CG=-2.490;AA=T;HA=34.1;HE=42.1;HC=26.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=21;TFBS=V$PPARG_02;GESP=T:5876/C:7130;PPI=THOC1:995|ZNF830:995|LSM3:987|PPIL1:980;PAC=NP_001002877.1,NP_001002878.1,NP_001002879.1,NP_003669.4,XP_005261854.1,XP_005261855.1,XP_005261856.1,XP_005261857.1,XP_005261858.1 GT:GQ:PL 0/1:3:0,0,0 0/1:4:0,3,25 1/1:14:103,15,0 0/1:5:91,0,2</p>

<p>X 48418659 rs235836 G A 55.30 midCVG AC1=6;AF1=0.6676;DB;DP=9;DP4=2,0,7,0;FQ=-7.74;MQ=60;PV4=1,0.14,1,0.23;SNPEFF_AMINO_ACID_CHANGE=A455T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TBC1D25;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002536.2;VDB=0.0091;DN=79;DA=G/A;GM=NM_002536.2;GL=TBC1D25;FG=missense;FD=missense;AAC=ALA/THR;PP=455/689;CDP=1363;GS=58;PH=0;CP=0.015;CG=-7.340;AA=A;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=658;GESP=A:5802/G:4761;PAC=NP_002527.1 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:4:22,3,0 0/1:25:68,0,23</p>

        <p>
          <strong>IDs and variants associated:</strong> H00831;TOR1A|PNKD|GCH1|ATP1A3|THAP1|SLC2A1|SPR|TAF1|TH|PRKRA|SGCE;hsa05012,hsa04920,hsa04260,hsa00790,hsa00350
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 94
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.94
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse94">Neurodegeneration with brain iron accumulation (NBIA)</a>
      </h4>
    </div>
    <div id="collapse94" class="panel-collapse collapse">
      <div class="panel-body">
        <p>1 53676448 rs1799821 G A 24.80 lowCVG AC1=4;AF1=0.5412;DB;DP=6;DP4=1,1,1,2;FQ=8.71;MQ=60;PV4=1,0.34,1,1;SNPEFF_AMINO_ACID_CHANGE=V368I;SNPEFF_CODON_CHANGE=Gtc/Atc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000098.2;VDB=0.0125;DN=89;DA=G/A;GM=NM_000098.2,XM_005270484.1;GL=CPT2;FG=missense;FD=missense,unknown;AAC=VAL/ILE;PP=368/659,368/636;CDP=1102;GS=29;PH=0.011;CP=0.032;CG=-0.261;AA=G;HA=23.4;HE=44.8;HC=25.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=475;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway;GESP=A:5936/G:7070;PAC=NP_000089.1,XP_005270541.1 GT:GQ:PL 0/1:3:0,0,0 0/0:3:0,3,22 1/1:5:45,6,0 0/1:20:19,0,22</p>

<p>2 207820276 rs13420911 G A 43.40 highCVG AC1=2;AF1=0.2979;DB;DP=18;DP4=6,8,2,2;FQ=44.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M85I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPO;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_173077.2;VDB=0.0134;DN=121;DA=G/A;GM=NM_173077.2;GL=CPO;FG=missense;FD=missense;AAC=MET/ILE;PP=85/375;CDP=255;GS=10;PH=0.021;CP=0.999;CG=5.110;AA=G;HA=33.2;HE=44.8;HC=39.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=13;GESP=A:5799/G:7207;PAC=NP_775100.1 GT:GQ:PL 0/0:20:0,18,108 0/0:3:0,0,0 1/1:6:86,12,0 0/0:25:0,24,153</p>

<p>3 194062519 rs3732477 C T 62.10 highCVG AC1=3;AF1=0.3624;DB;DP=16;DP4=0,8,1,5;FQ=64;MQ=60;PV4=0.43,0.0087,1,1;SNPEFF_AMINO_ACID_CHANGE=A305T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPN2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001080513.3;VDB=0.0065;DN=107;DA=C/T;GM=NM_001080513.2,XM_005269280.1,XM_005269281.1;GL=CPN2;FG=missense;FD=missense,unknown,unknown;AAC=ALA/THR;PP=305/546;CDP=913;GS=58;PH=0.017;CP=0.000;CG=-10.600;AA=C;HA=44.2;HE=26.5;HC=32.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=913;GESP=T:4238/C:8768;PAC=NP_001073982.2,XP_005269337.1,XP_005269338.1 GT:GQ:PL 0/0:9:0,9,59 0/0:12:0,12,79 1/1:5:63,9,0 0/1:19:45,0,13</p>

<p>4 8603105 rs115392829 G A 26.80 lowCVG AC1=3;AF1=0.4273;DB;DP=7;DP4=2,1,0,4;FQ=25.1;MQ=60;PV4=0.14,0.3,1,1;SNPEFF_AMINO_ACID_CHANGE=R126Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPZ;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001014447.2;VDB=0.0092;DN=132;DA=G/A;GM=NM_001014447.2,NM_001014448.2,NM_003652.3,XM_005248035.1,XM_005248036.1;GL=CPZ;FG=missense,5-prime-UTR,missense,missense,5-prime-UTR;FD=unknown;AAC=ARG/GLN,none,ARG/GLN,ARG/GLN,none;PP=126/653,NA,115/642,126/582,NA;CDP=377,NA,344,377,NA;GS=43,NA,43,43,NA;PH=0.886;CP=0.331;CG=2.560;AA=G;DSP=120;TFBS=V$AP2_Q6,V$ZIC3_01;GESP=A:134/G:12864;PAC=NP_001014447.1,NA,NP_003643.2,XP_005248092.1,NA GT:GQ:PL 0/0:7:0,6,47 0/0:4:0,3,18 1/1:7:67,12,0 0/1:3:0,0,0</p>

<p>7 120776097 rs41281692 C G 153 midCVG AC1=6;AF1=0.7444;DB;DP=14;DP4=2,1,4,7;FQ=14.1;MQ=60;PV4=0.54,0.2,1,0.31;SNPEFF_AMINO_ACID_CHANGE=A551G;SNPEFF_CODON_CHANGE=gCa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_024913.4;VDB=0.0084;DN=127;DA=C/G;GM=NM_001105533.1,NM_024913.4,XM_005250604.1,XM_005250605.1;GL=CPED1;FG=missense;FD=missense,missense,unknown,unknown;AAC=ALA/GLY;PP=551/784,551/1027,331/807,331/564;CDP=1652,1652,992,992;GS=60;PH=0.403;CP=0.000;CG=-0.512;AA=C;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4;GESP=G:4925/C:8057;PAC=NP_001099003.1,NP_079189.4,XP_005250661.1,XP_005250662.1 GT:GQ:PL 0/1:12:17,0,13 0/1:20:13,0,40 1/1:16:101,15,0 1/1:11:67,9,0</p>

<p>7 129950740 rs2171492 G T 30.70 lowCVG AC1=4;AF1=0.5413;DB;DP=5;DP4=0,2,3,0;FQ=11.4;MQ=60;PV4=0.1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=G303C;SNPEFF_CODON_CHANGE=Ggc/Tgc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=9;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPA4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_016352.3;VDB=0.0110;DN=96;DA=G/T;GM=NM_001163446.1,NM_016352.3;GL=CPA4;FG=missense;FD=missense;AAC=GLY/CYS;PP=270/389,303/422;CDP=808,907;GS=159;PH=0.012;CP=0.992;CG=0.479;AA=T;HA=17.1;HE=34.6;HC=37.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=87;GESP=T:4313/G:8693;PPI=LXN:959;PAC=NP_001156918.1,NP_057436.2 GT:GQ:PL 0/1:3:26,3,0 0/0:3:0,3,25 1/1:5:47,6,0 0/0:3:0,3,25</p>

<p>16 12897578 rs3748976 G T 56.70 midCVG AC1=5;AF1=0.5742;DB;DP=11;DP4=1,2,2,3;FQ=7.33;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=A19D;SNPEFF_CODON_CHANGE=gCc/gAc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPPED1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018340.2;VDB=0.0072;DN=107;DA=G/T;GM=NM_001099455.1,NM_018340.2;GL=CPPED1;FG=missense;FD=missense;AAC=ALA/ASP;PP=19/173,19/315;CDP=56;GS=126;PH=0;CP=1.000;CG=4.810;AA=T;HA=26.9;HE=11.4;HC=29.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=15;CPG=12897366-12897889;GESP=T:3654/G:8256;PAC=NP_001092925.1,NP_060810.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,3,16 1/1:7:69,9,0 0/1:27:28,0,27</p>

<p>19 17017878 rs56408444 G A 20.80 lowCVG AC1=4;AF1=0.4992;DB;DP=6;DP4=2,1,1,2;FQ=23.3;MQ=60;PV4=1,0.004,1,0.41;SNPEFF_AMINO_ACID_CHANGE=A1351V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPAMD8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_015692.2;VDB=0.0285;DN=129;DA=G/A;GM=NM_015692.2;GL=CPAMD8;FG=missense;FD=missense;AAC=ALA/VAL;PP=1351/1933;CDP=4052;GS=64;PH=0.068;CP=0.000;CG=-4.180;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=50;GESP=A:1872/G:10942;PAC=NP_056507.2GT:GQ:PL 0/0:3:0,3,27 0/1:3:22,3,0 1/1:4:42,6,0 0/0:5:0,6,47</p>

<p>20 34214173 rs11543239 G A 32.30 midCVG AC1=5;AF1=0.5743;DB;DP=10;DP4=1,3,1,2;FQ=34.9;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=P540L;SNPEFF_CODON_CHANGE=cCc/cTc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=16;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_003915.5;VDB=0.0111;DN=120;DA=G/A;GM=NM_001198863.1,NM_003915.5,NM_152925.2,NM_152926.2,NM_152927.2,NM_152928.2,NR_037188.1;GL=CPNE1;FG=missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,PRO/LEU,none;PP=534/537,540/543,535/538,535/538,535/538,535/538,NA;CDP=1601,1619,1604,1604,1604,1604,1978;GS=98,98,98,98,98,98,NA;PH=0.008;CP=0.016;CG=5.550;AA=G;HA=11.2;HE=5.9;HC=3.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=131;GESP=A:911/G:12095;PAC=NP_001185792.1,NP_003906.2,NP_690902.1,NP_690903.1,NP_690904.1,NP_690905.1,NA GT:GQ:PL 0/0:9:0,12,79 0/1:3:0,0,0 1/1:5:50,6,0 1/1:3:24,3,0</p>

<p>22 51009953 rs470117 C T 122 highCVG AC1=4;AF1=0.5;DB;DP=21;DP4=5,3,7,6;FQ=74.8;MQ=60;PV4=1,0.05,1,1;SNPEFF_AMINO_ACID_CHANGE=E531K;SNPEFF_CODON_CHANGE=Gag/Aag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPT1B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145135.1;VDB=0.0103;DN=80;DA=C/T;GM=NM_001145134.1,NM_001145135.1,NM_001145136.1,NM_001145137.1,NM_004377.3,NM_152245.2,NM_152246.2,NR_027928.2;GL=CPT1B/CHKB-CPT1B;FG=missense,missense,missense,missense,missense,missense,missense,non-coding-exon;FD=missense,missense,missense,missense,missense,missense,missense,nc-transcript-variant;AAC=GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,GLU/LYS,none;PP=497/739,531/773,450/692,531/773,531/773,531/773,531/773,NA;CDP=1489,1591,1348,1591,1591,1591,1591,3161;GS=56,56,56,56,56,56,56,NA;PH=0.966;CP=0.993;CG=5.310;AA=C;HA=10.1;HE=48.3;HC=47.7;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=16;KP=hsa00071:FattyAcidMetabolism|hsa03320:PPARSignalingPathway|hsa04920:AdipocytokineSignalingPathway;GESP=T:4624/C:8382;PAC=NP_001138606.1,NP_001138607.1,NP_001138608.1,NP_001138609.1,NP_004368.1,NP_689451.1,NP_689452.1,NA GT:GQ:PL 0/0:8:0,9,56 0/1:62:106,0,58 1/1:6:58,9,0 0/1:3:0,0,0</p>

<p>X 88008807 rs5984611 G A 58 lowCVG AC1=6;AF1=0.7071;DB;DP=7;DP4=0,2,1,4;FQ=4.26;MQ=60;PV4=1,0.12,1,1;SNPEFF_AMINO_ACID_CHANGE=R131H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=CPXCR1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184771.1;VDB=0.0098;DN=114;DA=G/A;GM=NM_001184771.1,NM_033048.5;GL=CPXCR1;FG=missense;FD=missense;AAC=ARG/HIS;PP=131/302;CDP=392;GS=29;PH=0.995;CP=0.000;CG=0.096;AA=G;HA=16.1;HE=40.7;HC=5.7;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=400;GESP=A:3478/G:7085;PAC=NP_001171700.1,NP_149037.4 GT:GQ:PL 0/1:3:0,0,0 0/1:3:0,0,0 1/1:6:40,6,0 0/1:34:54,0,33</p>

        <p>
          <strong>IDs and variants associated:</strong> 00833;CP|ATP13A2|FTL|FA2H|PANK2|PLA2G6;hsa04730,hsa04666,hsa04664,hsa00860,hsa00770,hsa00591,hsa00590,hsa00565,hsa00564
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 11
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.11
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse95">Leber congenital amaurosis (LCR)</a>
      </h4>
    </div>
    <div id="collapse95" class="panel-collapse collapse">
      <div class="panel-body">
        <p>2 98165911 rs1839230 T C 170 highCVG AC1=5;AF1=0.6034;DB;DP=35;DP4=0,3,12,4;FQ=5.73;MQ=50;PV4=0.036,1,1,1;SNPEFF_AMINO_ACID_CHANGE=E483G;SNPEFF_CODON_CHANGE=gAa/gGa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=20;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD36B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_025190.3;VDB=0.0137;DN=92;DA=T/C;GM=NM_025190.3,XM_005263995.1,XM_005263996.1,XM_005263997.1,XM_005263998.1,XM_005263999.1,XM_005264000.1,XM_005264001.1;GL=ANKRD36B;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLU/GLY;PP=483/1354,483/1354,483/1354,483/1353,483/1320,251/1122,232/1103,483/939;CDP=1448,1448,1448,1448,1448,752,695,1448;GS=98;PH=0;CP=0.002;CG=-0.893;AA=T;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-1000G;DSP=30;GESP=T:11082;PAC=NP_079466.3,XP_005264052.1,XP_005264053.1,XP_005264054.1,XP_005264055.1,XP_005264056.1,XP_005264057.1,XP_005264058.1 GT:GQ:PL 0/1:4:0,3,24 0/1:3:0,0,0 1/1:34:154,36,0 0/1:20:58,0,18</p>

<p>14 21770730 rs6571751 A G 150 midCVG AC1=6;AF1=0.6916;DB;DP=13;DP4=2,1,6,4;FQ=3.81;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=K192E;SNPEFF_CODON_CHANGE=Aaa/Gaa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=RPGRIP1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_020366.3;VDB=0.0044;DN=116;DA=A/G;GM=NM_020366.3;GL=RPGRIP1;FG=missense;FD=missense;AAC=LYS/GLU;PP=192/1287;CDP=574;GS=56;PH=0;CP=0.062;CG=3.110;AA=G;HA=47.4;HE=46.2;HC=33.2;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;CA;DSP=14;GESP=G:5809/A:6231;PPI=RPGR:846;PAC=NP_065099.3 GT:GQ:PL 0/1:18:39,0,18 0/1:21:83,0,21 1/1:7:49,6,0 1/1:5:22,3,0</p>

<p>18 14779969 rs9675365 C G 147 highCVG AC1=4;AF1=0.5542;DB;DP=24;DP4=0,8,5,11;FQ=15.9;MQ=55;PV4=0.13,1,1,1;SNPEFF_AMINO_ACID_CHANGE=F477L;SNPEFF_CODON_CHANGE=ttC/ttG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ANKRD30B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001145029.1;VDB=0.0092;DN=119;DA=C/G;GM=NM_001145029.1;GL=ANKRD30B;FG=missense;FD=missense;AAC=PHE/LEU;PP=477/1393;CDP=1431;GS=22;PH=0.286;CP=0.000;CG=-2.400;AA=C;HA=41.2;HE=48.8;HC=36.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=11;GESP=C:4566;PAC=NP_001138501.1 GT:GQ:PL 0/0:4:0,6,34 0/1:5:92,0,1 1/1:5:69,6,0 0/1:22:29,0,20</p>
        <p>
          <strong>IDs and variants associated:</strong> H00837;CRX|RDH12|LCA5|CRB1|AIPL1|GUCY2D|RD3|IMPDH1|KCNJ13|SPATA7|RPGRIP1|RPE65|TULP1|CEP290|LRAT;hsa04744,hsa00830,hsa00230
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 3
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.3
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse96">Mismatch repair deficiency</a>
      </h4>
    </div>
    <div id="collapse96" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	242035438	rs4149965	G	A	6.86	lowCVG	AC1=4;AF1=0.4545;DB;DP=8;DP4=3,2,2,1;FQ=8.98;MQ=60;PV4=1,0.0043,1,0.27;SNPEFF_AMINO_ACID_CHANGE=V458M;SNPEFF_CODON_CHANGE=Gtg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=EXO1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006027.4;VDB=0.0122;DN=110;DA=G/A;GM=NM_003686.4,NM_006027.4,NM_130398.3,XM_005273350.1;GL=EXO1;FG=missense;FD=missense,missense,missense,unknown;AAC=VAL/MET;PP=458/804,458/847,458/847,457/846;CDP=1372,1372,1372,1369;GS=21;PH=0.016;CP=0.020;CG=-3.180;AA=G;HA=0.3;HE=29.9;HC=0.0;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=102;KP=hsa03430:MismatchRepair;GESP=A:2269/G:10737;PPI=MLH1:998|MSH2:995|PMS2:885|YWHAE:845|YWHAB:845|YWHAZ:845;PAC=NP_003677.4,NP_006018.4,NP_569082.2,XP_005273407.1	GT:GQ:PL	0/0:3:0,0,0	0/0:3:0,0,0	1/1:4:44,9,0	0/0:16:0,15,101

        </p>
        <p>
          3	142281612	rs2227928	A	G	136	midCVG	AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2	GT:GQ:PL	0/1:9:65,0,8	0/1:5:0,3,26	1/1:7:37,6,0	1/1:12:79,12,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00876;MLH3|MSH6|EPCAM|MLH1|MSH2|PMS2;hsa05213,hsa05210,hsa03430
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse97">FG syndrome (FGS); Opitz-Kaveggia syndrome</a>
      </h4>
    </div>
    <div id="collapse97" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          8	97614625	rs3816208	G	A	21	midCVG	AC1=2;AF1=0.2266;DB;DP=12;DP4=3,6,1,2;FQ=22;MQ=60;PV4=1,0.04,1,0.059;SNPEFF_AMINO_ACID_CHANGE=A59T;SNPEFF_CODON_CHANGE=Gct/Act;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SDC2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002998.3;VDB=0.0142;DN=107;DA=G/A;GM=NM_002998.3,XM_005251019.1,XM_005251020.1,XM_005251021.1;GL=SDC2;FG=missense;FD=missense,unknown,unknown,unknown;AAC=ALA/THR;PP=59/202,50/193,30/173,30/173;CDP=175,148,88,88;GS=58;PH=0.018;CP=0.959;CG=4.150;AA=G;HA=0.6;HE=3.2;HC=23.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=3;KP=hsa04512:ECM-receptorInteraction|hsa04514:CellAdhesionMolecules(CAMs);GESP=A:685/G:12321;PPI=CASK:845;PAC=NP_002989.2,XP_005251076.1,XP_005251077.1,XP_005251078.1	GT:GQ:PL	0/0:9:0,6,45	0/0:18:0,15,111	1/1:3:63,9,0	0/0:9:0,6,47

        </p>
        <p>
          17	21318867	rs73979893	G	A	999	highCVG	AC1=5;AF1=0.6076;DB;DP=37;DP4=5,10,16,5;FQ=193;MQ=60;PV4=0.017,0.45,1,0.069;SNPEFF_AMINO_ACID_CHANGE=M71I;SNPEFF_CODON_CHANGE=atG/atA;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=3;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KCNJ12;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_021012.4;VDB=0.0094;DN=130;DA=G/A;GM=NM_021012.4,XM_005256625.1;GL=KCNJ12;FG=missense;FD=missense,unknown;AAC=MET/ILE;PP=71/434;CDP=213;GS=10;PH=0;CP=1.000;CG=5.330;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=269;CPG=21318651-21320041;TFBS=V$MEIS1AHOXA9_01,V$OCT1_06;PPI=DLG1:972|CASK:833|LIN7B:833|LIN7A:833|APBA1:833|LIN7C:833|DLG4:833;PAC=NP_066292.2,XP_005256682.1	GT:GQ:PL	0/1:54:49,0,91	0/1:35:62,0,34	1/1:8:120,9,0	0/1:59:54,0,108

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00894;FLNA|CASK|MED12;hsa04530,hsa04510,hsa04010
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse98">Lymphangioleiomyomatosis (LAM)</a>
      </h4>
    </div>
    <div id="collapse98" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          9	135786904	rs1073123	A	G	103	highCVG	AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1	GT:GQ:PL	0/0:4:0,3,23	0/1:84:83,0,87	1/1:6:64,9,0	0/0:6:0,6,39

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00896;TSC1|TSC2;hsa04910,hsa04150,hsa04115
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse99">Tuberous sclerosis complex (TSC); Bourneville-Pringle disease</a>
      </h4>
    </div>
    <div id="collapse99" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          9	135786904	rs1073123	A	G	103	highCVG	AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1	GT:GQ:PL	0/0:4:0,3,23	0/1:84:83,0,87	1/1:6:64,9,0	0/0:6:0,6,39

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H00915;TSC1|TSC2;hsa04910,hsa04150,hsa04115
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse100">Factor XIII deficiency</a>
      </h4>
    </div>
    <div id="collapse100" class="panel-collapse collapse">
      <div class="panel-body">
        <p>9 107533175 rs3739741 C G 117 highCVG AC1=5;AF1=0.5707;DB;DP=19;DP4=1,7,4,7;FQ=84.8;MQ=60;PV4=0.34,0.018,1,1;SNPEFF_AMINO_ACID_CHANGE=A159G;SNPEFF_CODON_CHANGE=gCt/gGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NIPSNAP3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018376.2;VDB=0.0089;DN=107;DA=C/G;GM=NM_018376.2;GL=NIPSNAP3B;FG=missense;FD=unknown;AAC=ALA/GLY;PP=159/248;CDP=476;GS=60;PH=0.999;CP=1.000;CG=2.440;AA=C;HA=0.0;HE=20.7;HC=27.8;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=46;GESP=G:1830/C:11176;PAC=NP_060846.2 GT:GQ:PL 0/1:28:67,0,26 0/0:14:0,18,103 1/1:4:27,3,0 1/1:13:71,15,0</p>

<p>17 39983808 rs1046404 G C 71.20 lowCVG AC1=6;AF1=0.6722;DB;DP=8;DP4=1,1,1,5;FQ=6.96;MQ=60;PV4=0.46,0.14,1,0.18;SNPEFF_AMINO_ACID_CHANGE=S213C;SNPEFF_CODON_CHANGE=tCt/tGt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0127;DN=86;DA=G/C;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=SER/CYS,none,none;PP=213/301,NA,NA;CDP=638,784,940;GS=112,NA,NA;PH=0.991;CP=0.812;CG=5.070;AA=G;HA=20.0;HE=24.2;HC=18.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=71;GESP=C:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:18,0,20 0/1:4:0,3,26 1/1:9:55,9,0 1/1:7:43,6,0</p>

<p>17 39983820 rs1046403 G A 58.40 lowCVG AC1=5;AF1=0.6474;DB;DP=7;DP4=1,1,1,4;FQ=7.67;MQ=60;PV4=1,0.17,1,0.19;SNPEFF_AMINO_ACID_CHANGE=A209V;SNPEFF_CODON_CHANGE=gCg/gTg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=8;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=NT5C3B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_052935.4;VDB=0.0201;DN=86;DA=G/A;GM=NM_052935.4,NR_033464.1,NR_033465.1;GL=NT5C3B;FG=missense,non-coding-exon,non-coding-exon;FD=missense,nc-transcript-variant,nc-transcript-variant;AAC=ALA/VAL,none,none;PP=209/301,NA,NA;CDP=626,772,928;GS=64,NA,NA;PH=0.001;CP=0.060;CG=2.870;AA=G;HA=20.8;HE=23.8;HC=17.3;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=59;GESP=A:9892/G:3114;PAC=NP_443167.4,NA,NA GT:GQ:PL 0/1:19:17,0,20 0/1:4:0,3,26 1/1:9:64,9,0 1/1:4:21,3,0</p>

<p>20 18505200 rs36023150 G T 94.50 highCVG AC1=5;AF1=0.6403;DB;DP=15;DP4=1,6,4,4;FQ=80.8;MQ=60;PV4=0.28,0.1,1,0.12;SNPEFF_AMINO_ACID_CHANGE=V164L;SNPEFF_CODON_CHANGE=Gtg/Ttg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SEC23B;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001172745.1;VDB=0.0134;DN=126;DA=G/T;GM=NM_001172745.1,NM_001172746.1,NM_006363.4,NM_032985.4,NM_032986.3;GL=SEC23B;FG=missense;FD=missense;AAC=VAL/LEU;PP=164/768,146/750,164/768,164/768,164/768;CDP=490,436,490,490,490;GS=32;PH=0.994;CP=0.996;CG=4.910;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=70;GESP=T:97/G:12909;PAC=NP_001166216.1,NP_001166217.1,NP_006354.2,NP_116780.1,NP_116781.1 GT:GQ:PL 0/1:3:0,0,0 0/1:34:29,0,53 1/1:14:103,15,0 0/1:8:3,0,64</p>

        <p>
          <strong>IDs and variants associated:</strong> H00945;3A1|3B;hsa04610
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse101">Seckel syndrome</a>
      </h4>
    </div>
    <div id="collapse101" class="panel-collapse collapse">
      <div class="panel-body">
        <p>3 142281612 rs2227928 A G 136 midCVG AC1=6;AF1=0.7062;DB;DP=12;DP4=2,0,0,10;FQ=-6.32;MQ=60;PV4=0.015,1,1,0.24;SNPEFF_AMINO_ACID_CHANGE=M211T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ATR;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001184.3;VDB=0.0075;DN=98;DA=A/G;GM=NM_001184.3;GL=ATR;FG=missense;FD=missense;AAC=MET/THR;PP=211/2645;CDP=632;GS=81;PH=0;CP=0.839;CG=4.600;AA=G;HA=15.4;HE=45.3;HC=49.0;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=340;KP=hsa04110:CellCycle|hsa04115:p53SignalingPathway;GESP=G:8614/A:4392;PPI=TP53:999|ATRIP:999|BRCA1:988|CHEK1:971|CHEK2:937|WRN:929|TELO2:923|TTI1:846|MSH2:846|MTA1:846;PAC=NP_001175.2 GT:GQ:PL 0/1:9:65,0,8 0/1:5:0,3,26 1/1:7:37,6,0 1/1:12:79,12,0</p>

<p>21 47786524 rs2839227 A G 27.90 midCVG AC1=4;AF1=0.5236;DB;DP=10;DP4=6,0,0,4;FQ=31;MQ=60;PV4=0.0048,1,1,1;SNPEFF_AMINO_ACID_CHANGE=T879A;SNPEFF_CODON_CHANGE=Acc/Gcc;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=15;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0116;DN=100;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1,XM_005261130.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=THR/ALA;PP=879/3337,879/3348,879/3347,879/3269,879/3259,761/3230,761/3140,879/2707;CDP=2635,2635,2635,2635,2635,2281,2281,2635;GS=58;PH=0.264;CP=0.002;CG=1.810;AA=A;HA=38.8;HE=17.7;HC=23.8;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;DSP=26;GESP=G:3367/A:9597;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1,XP_005261187.1 GT:GQ:PL 0/1:3:0,0,0 0/0:5:0,6,40 1/1:6:57,9,0 0/1:15:12,0,61</p>

<p>21 47851753 rs2073376 A G 39.40 lowCVG AC1=6;AF1=0.6957;DB;DP=6;DP4=1,1,1,3;FQ=10.9;MQ=60;PV4=1,0.27,1,0.27;SNPEFF_AMINO_ACID_CHANGE=Q2792R;SNPEFF_CODON_CHANGE=cAg/cGg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=38;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCNT;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_006031.5;VDB=0.0093;DN=96;DA=A/G;GM=NM_006031.5,XM_005261124.1,XM_005261125.1,XM_005261126.1,XM_005261127.1,XM_005261128.1,XM_005261129.1;GL=PCNT;FG=missense;FD=missense,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/ARG;PP=2792/3337,2803/3348,2802/3347,2803/3269,2803/3259,2685/3230,2674/3140;CDP=8375,8408,8405,8408,8408,8054,8021;GS=43;PH=0;CP=0.314;CG=2.320;AA=G;HA=15.0;HE=33.5;HC=31.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=140;GESP=G:8998/A:4008;PPI=CALM1:846|DISC1:790;PAC=NP_006022.3,XP_005261181.1,XP_005261182.1,XP_005261183.1,XP_005261184.1,XP_005261185.1,XP_005261186.1 GT:GQ:PL 0/0:4:0,6,47 0/1:3:0,0,0 1/1:6:34,6,0 1/1:6:47,6,0</p>

        <p>
          <strong>IDs and variants associated:</strong> H00992;CEP152|PCNT|ATR|CENPJ|RBBP8;hsa04115,hsa04110
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 3
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.3
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse102">Diseases of the tricarboxylic acid cycle</a>
      </h4>
    </div>
    <div id="collapse102" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          6	97063522	rs9373985	C	G	49.80	highCVG	AC1=3;AF1=0.3859;DB;DP=27;DP4=12,7,6,2;FQ=51.9;MQ=60;PV4=0.68,0.037,1,1;SNPEFF_AMINO_ACID_CHANGE=S243R;SNPEFF_CODON_CHANGE=agC/agG;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=6;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FHL5;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001170807.1;VDB=0.0013;DN=119;DA=C/G;GM=NM_001170807.1,NM_020482.4,XM_005267227.1;GL=FHL5;FG=missense;FD=missense,missense,unknown;AAC=SER/ARG;PP=243/285;CDP=729;GS=110;PH=0;CP=1.000;CG=3.900;AA=C;HA=42.4;HE=39.8;HC=41.6;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=38;GESP=G:5005/C:8001;PPI=CREM:846;PAC=NP_001164278.1,NP_065228.4,XP_005267284.1	GT:GQ:PL	0/0:9:0,9,63	0/1:27:26,0,61	1/1:13:72,18,0	0/0:35:0,36,176

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01022;FH|OGDH|SDHA;hsa00020
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse103">Familial adenomatous polyposis</a>
      </h4>
    </div>
    <div id="collapse103" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          1	45797505	rs3219489	C	G	19	midCVG	AC1=4;AF1=0.4355;DB;DP=11;DP4=3,5,1,2;FQ=21.3;MQ=60;PV4=1,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=Q338H;SNPEFF_CODON_CHANGE=caG/caC;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=12;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=MUTYH;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001128425.1;VDB=0.0034;DN=106;DA=C/G;GM=NM_001048171.1,NM_001048172.1,NM_001048173.1,NM_001048174.1,NM_001128425.1,NM_012222.2,XM_005270880.1,XM_005270881.1,XM_005270882.1,XM_005270883.1,XM_005270884.1,XM_005270885.1,XM_005270886.1;GL=MUTYH;FG=missense;FD=missense,missense,missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLN/HIS;PP=324/536,311/523,310/522,310/522,338/550,335/547,325/537,321/533,321/533,282/494,218/430,218/430,184/396;CDP=972,933,930,930,1014,1005,975,963,963,846,654,654,552;GS=24;PH=0.645;CP=1.000;CG=1.260;AA=C;HA=30.5;HE=28.4;HC=45.4;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=17;KP=hsa03410:BaseExcisionRepair;GESP=G:3304/C:9698;PPI=APEX1:846|PCNA:846;PAC=NP_001041636.1,NP_001041637.1,NP_001041638.1,NP_001041639.1,NP_001121897.1,NP_036354.1,XP_005270937.1,XP_005270938.1,XP_005270939.1,XP_005270940.1,XP_005270941.1,XP_005270942.1,XP_005270943.1	GT:GQ:PL	0/1:13:12,0,51	0/0:14:0,15,79	1/1:4:48,6,0	0/1:3:0,0,0

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01025;APC|MUTYH;hsa05210,hsa05200,hsa04810,hsa04310,hsa03410
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse104">Anophthalmia and microphthalmia (A/M)</a>
      </h4>
    </div>
    <div id="collapse104" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          9	139118673	rs12380852	T	C	17.20	midCVG	AC1=3;AF1=0.3906;DB;DP=12;DP4=3,5,1,2;FQ=19.3;MQ=60;PV4=1,1,1,0.26;SNPEFF_AMINO_ACID_CHANGE=K126E;SNPEFF_CODON_CHANGE=Aag/Gag;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=2;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=QSOX2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_181701.3;VDB=0.0103;DN=120;DA=T/C;GM=NM_181701.3;GL=QSOX2;FG=missense;FD=missense;AAC=LYS/GLU;PP=126/699;CDP=376;GS=56;PH=0.002;CP=0.615;CG=2.180;AA=T;HA=4.2;HE=7.5;HC=25.8;DG;DV=by-frequency,by-cluster,by-1000G;DSP=48;GESP=C:1203/T:11803;PAC=NP_859052.3	GT:GQ:PL	0/0:9:0,9,60	0/1:4:26,3,0	1/1:3:37,6,0	0/0:15:0,15,95

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01027;CRYBA4|RAX|HCCS|VSX2|GDF6|SIX6|OTX2|BCOR|STRA6|SHH|BMP4|SOX2|MFRP|GDF3;hsa04350,hsa04340,hsa00860
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse105">Peters anomaly</a>
      </h4>
    </div>
    <div id="collapse105" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          X	148038075	.	A	C	11.70	midCVG	AC1=5;AF1=0.5727;DP=14;DP4=3,1,0,3;FQ=15.3;MQ=60;PV4=0.14,0.022,1,1;SNPEFF_AMINO_ACID_CHANGE=K834Q;SNPEFF_CODON_CHANGE=Aaa/Caa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AFF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002025.3;VDB=0.0005;GM=NM_001169122.1,NM_001169123.1,NM_001169124.1,NM_001169125.1,NM_001170628.1,NM_002025.3;GL=AFF2;FG=missense;FD=unknown;AAC=LYS/GLN;PP=801/1277,824/1302,799/1277,795/1273,475/953,834/1312;CDP=2401,2470,2395,2383,1423,2500;GS=53;PH=1;CP=0.878;CG=5.870;AA=A;DSP=69;TFBS=V$PAX6_01;GESP=A:13006;PAC=NP_001162593.1,NP_001162594.1,NP_001162595.1,NP_001162596.1,NP_001164099.1,NP_002016.2	GT:GQ:PL	0/1:3:0,0,0	0/1:3:18,3,0	1/1:4:35,6,0	0/0:10:0,12,77

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01075;3GALTL|CYP1B1|FOXC1|PAX6|PITX2;hsa04350,hsa00980,hsa00380,hsa00140
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse106">Cerebral palsy</a>
      </h4>
    </div>
    <div id="collapse106" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          9	742388	rs140893133	G	A	40.80	midCVG	AC1=2;AF1=0.3159;DB;DP=13;DP4=7,1,3,1;FQ=42.4;MQ=60;PV4=1,1,1,0.4;SNPEFF_AMINO_ACID_CHANGE=G1294S;SNPEFF_CODON_CHANGE=Ggt/Agt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=14;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=KANK1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001256876.1;VDB=0.0053;DN=134;DA=G/A;GM=NM_001256876.1,NM_001256877.1,NM_015158.3,NM_153186.4,XM_005251410.1,XM_005251411.1,XM_005251412.1,XM_005251413.1,XM_005251414.1,XM_005251415.1,XM_005251416.1,XM_005251417.1,XM_005251418.1,XM_005251419.1;GL=KANK1;FG=missense;FD=missense,missense,missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=GLY/SER;PP=1294/1353,1294/1353,1294/1353,1136/1195,1294/1365,1294/1365,1294/1365,1214/1285,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207,1136/1207;CDP=3880,3880,3880,3406,3880,3880,3880,3640,3406,3406,3406,3406,3406,3406;GS=56;PH=0.073;CP=0.955;CG=5.330;AA=G;DV=by-frequency,by-cluster;DSP=18;TFBS=V$CHOP_01;GESP=A:16/G:12990;PPI=YWHAQ:846;PAC=NP_001243805.1,NP_001243806.1,NP_055973.2,NP_694856.1,XP_005251467.1,XP_005251468.1,XP_005251469.1,XP_005251470.1,XP_005251471.1,XP_005251472.1,XP_005251473.1,XP_005251474.1,XP_005251475.1,XP_005251476.1	GT:GQ:PL	0/0:13:0,12,85	0/0:3:0,0,0	1/1:6:83,12,0	0/0:13:0,12,70

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01097;AP4B1|AP4S1|KANK1|AP4E1|GAD1|AP4M1;hsa04142,hsa00650,hsa00410,hsa00430,hsa00250
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse107">Pituitary adenomas</a>
      </h4>
    </div>
    <div id="collapse107" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          17	19861458	rs2108978	C	T	98.40	highCVG	AC1=3;AF1=0.3904;DB;DP=18;DP4=7,4,1,6;FQ=101;MQ=60;PV4=0.066,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R249H;SNPEFF_CODON_CHANGE=cGt/cAt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=4;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AKAP10;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_007202.3;VDB=0.0017;DN=96;DA=C/T;GM=NM_007202.3;GL=AKAP10;FG=missense;FD=missense;AAC=ARG/HIS;PP=249/663;CDP=746;GS=29;PH=0;CP=0.995;CG=2.360;AA=T;HA=44.6;HE=39.4;HC=19.1;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G,by-other-pop;DSP=132;TFBS=V$GATA1_03;GESP=T:5787/C:7219;PPI=PDZK1:959|PRKAR1A:784;PAC=NP_009133.2	GT:GQ:PL	0/0:9:0,9,75	0/0:11:0,12,64	1/1:10:115,15,0	0/1:32:31,0,84

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01102;CDKN1B|MEN1|PRKAR1A|AIP;hsa04910,hsa04210,hsa04110,hsa04012
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse108">Acid phosphatase deficiency</a>
      </h4>
    </div>
    <div id="collapse108" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          11	47270255	rs2167079	C	T	15.90	midCVG	AC1=3;AF1=0.4192;DB;DP=11;DP4=4,2,1,2;FQ=18.3;MQ=60;PV4=0.52,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R29Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=ACP2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001610.2;VDB=0.0055;DN=96;DA=C/T;GM=NM_001131064.1,NM_001251934.1,NM_001251935.1,NM_001610.2,XM_005252982.1,XM_005252983.1,XM_005252984.1,XM_005252985.1;GL=ACP2/NR1H3;FG=missense,5-prime-UTR,5-prime-UTR,missense,missense,missense,5-prime-UTR,missense;FD=missense,utr-variant-5-prime,utr-variant-5-prime,missense,unknown,unknown,unknown,unknown;AAC=ARG/GLN,none,none,ARG/GLN,ARG/GLN,ARG/GLN,none,ARG/GLN;PP=29/161,NA,NA,29/424,29/392,29/361,NA,29/218;CDP=86,NA,NA,86,86,86,NA,86;GS=43,NA,NA,43,43,43,NA,43;PH=0.003;CP=0.987;CG=5.690;AA=T;HA=38.3;HE=31.2;HC=27.9;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=25;KP=hsa00740:RiboflavinMetabolism|hsa04142:Lysosome|hsa03320:PPARSignalingPathway;CPG=47270235-47270633;TFBS=V$AHR_01;GESP=T:4930/C:8048;PPI=RXRA:937|RXRB:900;PAC=NP_001124536.1,NA,NA,NP_001601.1,XP_005253039.1,XP_005253040.1,NA,XP_005253042.1	GT:GQ:PL	0/1:4:24,3,0	0/0:6:0,6,30	1/1:3:37,6,0	0/0:11:0,12,67

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01113;ACP2;hsa04142,hsa00740
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse109">Cataract</a>
      </h4>
    </div>
    <div id="collapse109" class="panel-collapse collapse">
      <div class="panel-body">
        <p>4 169611811 . C T 14.60 midCVG AC1=2;AF1=0.2475;DP=11;DP4=6,2,0,3;FQ=15.7;MQ=60;PV4=0.061,1,1,0.052;SNPEFF_AMINO_ACID_CHANGE=R465C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=7;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PALLD;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_001166108.1;VDB=0.0128;GM=NM_001166108.1,NM_001166109.1,NM_016081.3,XM_005262861.1,XM_005262862.1,XM_005262863.1,XM_005262864.1,XM_005262865.1,XM_005262866.1;GL=PALLD;FG=missense;FD=unknown;AAC=ARG/CYS;PP=465/1124,83/778,465/1107,465/1401,465/1384,465/1348,465/1310,465/1277,83/1019;CDP=1393,247,1393,1393,1393,1393,1393,1393,247;GS=180;PH=1;CP=0.789;CG=3.960;AA=C;DSP=58;TFBS=V$TCF11MAFG_01;GESP=C:13006;PPI=EZR:846|ACTN1:846;PAC=NP_001159580.1,NP_001159581.1,NP_057165.3,XP_005262918.1,XP_005262919.1,XP_005262920.1,XP_005262921.1,XP_005262922.1,XP_005262923.1 GT:GQ:PL 0/0:7:0,3,16 0/0:9:0,6,36 1/1:3:56,9,0 0/0:18:0,15,104</p>

<p>5 140559596 rs2697541 T G 19.30 lowCVG AC1=3;AF1=0.352;DB;DP=7;DP4=3,1,0,3;FQ=21.1;MQ=56;PV4=0.14,0.29,0.42,0.14;SNPEFF_AMINO_ACID_CHANGE=L661V;SNPEFF_CODON_CHANGE=Ttg/Gtg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=1;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=PCDHB8;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_019120.3;VDB=0.0059;DN=100;DA=T/G;GM=NM_019120.3;GL=PCDHB8;FG=missense;FD=missense;AAC=LEU/VAL;PP=661/802;CDP=1981;GS=32;PH=0;CP=1.000;CG=4.220;AA=G;DG;DV=by-frequency,by-cluster,by-1000G;DSP=4823;CPG=140558932-140559789;TFBS=V$TCF11MAFG_01;GESP=G:10886/T:1780;PAC=NP_061993.2 GT:GQ:PL 0/0:8:0,6,49 0/0:3:0,0,0 1/1:4:59,9,0 0/0:8:0,6,47</p>

<p>8 124449466 rs3824250 C T 38.30 midCVG AC1=4;AF1=0.4784;DB;DP=10;DP4=5,1,1,3;FQ=41.1;MQ=60;PV4=0.19,1,1,1;SNPEFF_AMINO_ACID_CHANGE=R134C;SNPEFF_CODON_CHANGE=Cgt/Tgt;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=5;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=WDYHV1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_018024.2;VDB=0.0061;DN=107;DA=C/T;GM=NM_018024.1,XM_005250997.1,XM_005250998.1;GL=WDYHV1;FG=missense;FD=missense,unknown,unknown;AAC=ARG/CYS;PP=134/206,74/146,66/138;CDP=400,220,196;GS=180;PH=0.99;CP=1.000;CG=5.480;AA=C;HA=34.8;HE=28.6;HC=40.2;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=17;TFBS=V$AHRARNT_02,V$TCF11MAFG_01;GESP=T:4627/C:8379;PAC=NP_060494.1,XP_005251054.1,XP_005251055.1 GT:GQ:PL 0/1:16:14,0,41 0/0:7:0,9,51 1/1:4:47,6,0 0/1:17:22,0,14</p>

<p>X 148038075 . A C 11.70 midCVG AC1=5;AF1=0.5727;DP=14;DP4=3,1,0,3;FQ=15.3;MQ=60;PV4=0.14,0.022,1,1;SNPEFF_AMINO_ACID_CHANGE=K834Q;SNPEFF_CODON_CHANGE=Aaa/Caa;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=11;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=AFF2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_002025.3;VDB=0.0005;GM=NM_001169122.1,NM_001169123.1,NM_001169124.1,NM_001169125.1,NM_001170628.1,NM_002025.3;GL=AFF2;FG=missense;FD=unknown;AAC=LYS/GLN;PP=801/1277,824/1302,799/1277,795/1273,475/953,834/1312;CDP=2401,2470,2395,2383,1423,2500;GS=53;PH=1;CP=0.878;CG=5.870;AA=A;DSP=69;TFBS=V$PAX6_01;GESP=A:13006;PAC=NP_001162593.1,NP_001162594.1,NP_001162595.1,NP_001162596.1,NP_001164099.1,NP_002016.2 GT:GQ:PL 0/1:3:0,0,0 0/1:3:18,3,0 1/1:4:35,6,0 0/0:10:0,12,77</p>

        <p>
          <strong>IDs and variants associated:</strong> H01202;CHMP4B|CRYAA|CRYAB|CRYBA1|CRYBA4|CRYBB1|CRYBB2|CRYBB3|CRYGA|CRYGC|CRYGD|CRYGS|EPHA2|EYA1|FOXE3|FTL|GCNT2|GJA3|GJA8|HSF4|SLC16A12|LIM2|MAF|MIP|NHS|PAX6|PITX3|BFSP1|SIL1|VIM|BFSP2;hsa04360,hsa04144,hsa00860,hsa00601
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 4
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.4
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse110">Hypomagnesemia</a>
      </h4>
    </div>
    <div id="collapse110" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          5	180030322	rs79620092	C	T	69.90	lowCVG	AC1=5;AF1=0.5992;DB;DP=8;DP4=1,1,0,6;FQ=4.03;MQ=60;PV4=0.25,1,1,0.3;SNPEFF_AMINO_ACID_CHANGE=R1321Q;SNPEFF_CODON_CHANGE=cGg/cAg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=30;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=FLT4;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182925.4;VDB=0.0070;DN=131;DA=C/T;GM=NM_182925.4;GL=FLT4;FG=missense;FD=missense;AAC=ARG/GLN;PP=1321/1364;CDP=3962;GS=43;PH=0.001;CP=0.001;CG=1.360;AA=C;DV=by-frequency,by-cluster;DSP=69;KP=hsa04060:Cytokine-cytokineReceptorInteraction|hsa04510:FocalAdhesion;GESP=T:154/C:12850;PPI=KDR:974|SHC1:974|VEGFC:937|GRB2:846;PAC=NP_891555.2	GT:GQ:PL	0/1:15:20,0,13	0/1:3:0,0,0	1/1:13:91,15,0	0/1:3:0,3,26

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01210;CLDN16|TRPM6|CLDN19|EGF|FXYD2;hsa04540,hsa04530,hsa04514,hsa04510
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse111">Left ventricular noncompaction (LVNC)</a>
      </h4>
    </div>
    <div id="collapse111" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          6	152443744	rs2295190	G	T	29	lowCVG	AC1=6;AF1=0.7087;DB;DP=6;DP4=1,1,3,1;FQ=-8.42;MQ=60;PV4=1,1,1,1;SNPEFF_AMINO_ACID_CHANGE=L8741M;SNPEFF_CODON_CHANGE=Ctg/Atg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=146;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182961.3;VDB=0.0061;DN=100;DA=G/T;GM=NM_033071.3,NM_182961.3,XM_005266877.1;GL=SYNE1;FG=missense;FD=missense,missense,unknown;AAC=LEU/MET;PP=8693/8750,8741/8798,8769/8826;CDP=26077,26221,26305;GS=15;PH=0.994;CP=0.840;CG=0.627;AA=G;HA=0.8;HE=17.5;HC=10.6;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G;DSP=68;GESP=T:1354/G:11652;PPI=SUN2:959|LMNA:846;PAC=NP_149062.1,NP_892006.3,XP_005266934.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:6:34,6,0	0/1:24:31,0,23

        </p>
        <p>
          14	64491695	rs4902264	T	C	97.50	midCVG	AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:18:122,18,0	0/1:19:13,0,55

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01216;LDB3|DTNA|LMNA|MYH7|TAZ|ACTC1|TNNT2;hsa04260
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 2
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.2
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse112">Restrictive cardiomyopathy (RCM)</a>
      </h4>
    </div>
    <div id="collapse112" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          14	64491695	rs4902264	T	C	97.50	midCVG	AC1=6;AF1=0.7452;DB;DP=10;DP4=1,2,5,2;FQ=23.4;MQ=59;PV4=0.5,1,0.27,1;SNPEFF_AMINO_ACID_CHANGE=M1969T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=40;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=SYNE2;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_182914.2;VDB=0.0140;DN=111;DA=T/C;GM=NM_015180.4,NM_182914.2,XM_005267454.1,XM_005267455.1,XM_005267456.1,XM_005267457.1,XM_005267458.1,XM_005267459.1,XM_005267460.1;GL=SYNE2;FG=missense;FD=missense,missense,unknown,unknown,unknown,unknown,unknown,unknown,unknown;AAC=MET/THR;PP=1969/6886,1969/6908,1969/6923,1969/6923,1969/6922,1969/6909,1969/6900,1969/6885,1969/6884;CDP=5906;GS=81;PH=0.003;CP=0.024;CG=-1.290;AA=T;HA=33.2;HE=12.8;HC=21.5;DG;DV=by-frequency,by-cluster,by-2hit-2allele,by-hapmap,by-1000G;DSP=55;GESP=C:8797/T:3081;PPI=SUN2:959|ACTC1:846;PAC=NP_055995.4,NP_878918.2,XP_005267511.1,XP_005267512.1,XP_005267513.1,XP_005267514.1,XP_005267515.1,XP_005267516.1,XP_005267517.1	GT:GQ:PL	0/1:3:0,0,0	0/1:3:0,0,0	1/1:18:122,18,0	0/1:19:13,0,55

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01219;DES|ACTC1|TNNI3|TNNT2;hsa04260
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

  <div class="panel panel-default">
    <div class="panel-heading">
      <h4 class="panel-title">
        <a data-toggle="collapse" data-parent="#accordion" href="#collapse113">Focal cortical dysplasia of Taylor; Focal cortical dysplasia type II</a>
      </h4>
    </div>
    <div id="collapse113" class="panel-collapse collapse">
      <div class="panel-body">
        <p>
          9	135786904	rs1073123	A	G	103	highCVG	AC1=3;AF1=0.4333;DB;DP=22;DP4=1,9,0,11;FQ=103;MQ=60;PV4=0.48,1,1,1;SNPEFF_AMINO_ACID_CHANGE=M322T;SNPEFF_CODON_CHANGE=aTg/aCg;SNPEFF_EFFECT=NON_SYNONYMOUS_CODING;SNPEFF_EXON_ID=10;SNPEFF_FUNCTIONAL_CLASS=MISSENSE;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=TSC1;SNPEFF_IMPACT=MODERATE;SNPEFF_TRANSCRIPT_ID=NM_000368.4;VDB=0.0099;DN=86;DA=A/G;GM=NM_000368.4,NM_001162426.1,NM_001162427.1,XM_005272211.1,XM_005272212.1;GL=TSC1;FG=missense;FD=missense,missense,missense,unknown,unknown;AAC=MET/THR;PP=322/1165,322/1164,271/1114,322/1165,322/1165;CDP=965,965,812,965,965;GS=81;PH=0.006;CP=0.009;CG=2.950;AA=G;HA=22.3;HE=13.4;HC=5.1;DG;DV=by-frequency,by-cluster,by-hapmap,by-1000G,by-other-pop;CA;DSP=52;KP=hsa04150:mTORSignalingPathway|hsa04910:InsulinSignalingPathway;GESP=G:2152/A:10854;PPI=TSC2:999|UBC:861|CDK1:846|EZR:845|NF2:845|AKT1:845|YWHAG:733;PAC=NP_000359.1,NP_001155898.1,NP_001155899.1,XP_005272268.1,XP_005272269.1	GT:GQ:PL	0/0:4:0,3,23	0/1:84:83,0,87	1/1:6:64,9,0	0/0:6:0,6,39

        </p>
        <p>
          <strong>IDs and variants associated:</strong> H01251;TSC1;hsa04910,hsa04150
        </p>
        <p>
          <strong>Nonsense Mutations:</strong> 0
        </p>
        <p>
          <strong>Missense Mutations:</strong> 1
        </p>
        <p>
          <strong>Disease Rank Score:</strong> 0.1
        </p>
      </div>
    </div>
  </div>

</div>
</div>
</div>

</body>
</html>
